diff --git a/.gn b/.gn
index 84c68fbae243a60a676b0234cf850331adbb87d9..2c969cc92a9d12057c530c71c73c036945381b6d 100644
--- a/.gn
+++ b/.gn
@@ -2,6 +2,7 @@
 # tree and to set startup options. For documentation on the values set in this
 # file, run "gn help dotfile" at the command line.
 
+import("//brave/build/circular_includes_brave.gni")
 import("//build/circular_includes_bedrock.gni")
 import("//build/circular_includes_untriaged.gni")
 import("//build/dotfile_settings.gni")
@@ -475,3 +476,4 @@ expand_directory_allowlist =
 allow_circular_includes_from_allowlist =
     allow_circular_includes_from_untriaged + project_bedrock_circular_includes +
     []
+allow_circular_includes_from_allowlist += allow_circular_includes_from_brave
