diff --git a/.rustfmt.toml b/.rustfmt.toml
index 7b3b53017cd6421e5f3998a15165b3a09d985a4f..2dbcf71b6115bdb686850cf0a9890974e0fb56a3 100644
--- a/.rustfmt.toml
+++ b/.rustfmt.toml
@@ -35,4 +35,8 @@ ignore = [
   # Explicitly ignore third-party code vendored from crates.io.
   # (This entry "wins" over the `!third_party/rust/**` one because it is last.)
   "third_party/rust/chromium_crates_io/vendor/**",
+  "brave/third_party/",
+  "brave/tools/crates/vendor/",
+  "brave/third_party/wasm/vendor/",
 ]
+skip_children = true  # Do not format submodules.
