diff --git a/.vpython3 b/.vpython3
index 21e15085072f03737725b10588369eb4c2b620c6..2fc76f02b61fc8bc7f62df9e9fd3c62385b7cc8b 100644
--- a/.vpython3
+++ b/.vpython3
@@ -59,6 +59,17 @@ wheel: <
 wheel: <
   name: "infra/python/wheels/numpy/${vpython_platform}"
   version: "version:1.20.3"
+  # A newer version of numpy is required on ARM64, but it breaks older OS versions.
+  not_match_tag <
+    platform: "macosx_11_0_arm64"
+  >
+>
+wheel: <
+  name: "infra/python/wheels/numpy/mac-arm64_cp38_cp38"
+  version: "version:1.21.1"
+  match_tag <
+    platform: "macosx_11_0_arm64"
+  >
 >
 wheel: <
   name: "infra/python/wheels/psutil/${vpython_platform}"
