diff --git a/android_webview/system_webview_apk_tmpl.gni b/android_webview/system_webview_apk_tmpl.gni
index a71c7e1b7fc96c8558808542cb79970ad907b551..46a9e4bd5c4d54a5ea1cb035d6438a874ef6ce43 100644
--- a/android_webview/system_webview_apk_tmpl.gni
+++ b/android_webview/system_webview_apk_tmpl.gni
@@ -58,6 +58,7 @@ template("system_webview_apk_or_module_tmpl") {
     }
     _android_manifest = "$target_gen_dir/$target_name/AndroidManifest.xml"
     _android_manifest_target_name = "${target_name}__android_manifest"
+    #group(_android_manifest_target_name){}
     jinja_template(_android_manifest_target_name) {
       if (!defined(_android_sdk_release)) {
         _android_sdk_release = android_sdk_release
