diff --git a/android_webview/browser/cookie_manager.cc b/android_webview/browser/cookie_manager.cc
index 4344f15143f75d56a2253586792f7b644eaf25c4..9245d3e8dd35f5172f93fca36c6155e30a8a44bc 100644
--- a/android_webview/browser/cookie_manager.cc
+++ b/android_webview/browser/cookie_manager.cc
@@ -478,7 +478,7 @@ void CookieManager::SetCookieHelper(const GURL& host,
 
   net::CanonicalCookie::CookieInclusionStatus status;
   std::unique_ptr<net::CanonicalCookie> cc(
-      net::CanonicalCookie::Create(new_host, value, base::Time::Now(),
+      net::CanonicalCookie::Create(BRAVE_SETCOOKIEHELPER, new_host, value, base::Time::Now(),
                                    base::nullopt /* server_time */, &status));
 
   if (!cc || !should_allow_cookie) {
