diff --git a/base/BUILD.gn b/base/BUILD.gn
index 521cb942f526a8cb0873f9ae3d72053caf14faa9..3f9731eb2f100705a75f7ba0dc65f0472718f39d 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4515,6 +4515,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
       "android/java/src/org/chromium/base/WrappedClassLoader.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -5052,6 +5053,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
