diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index 9377661e94e74ab6f74d67306c97713ef794d2ac..2199cd824ccd1c46c645fd7b1b3fd7b278fc160f 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -61,6 +61,10 @@ PERFETTO_DEFINE_TEST_CATEGORY_PREFIXES("cat",
 PERFETTO_DEFINE_CATEGORIES_IN_NAMESPACE_WITH_ATTRS(
     base,
     BASE_EXPORT,
+    perfetto::Category("brave"),
+    perfetto::Category("brave.adblock"),
+    perfetto::Category("brave.ads"),
+    perfetto::Category("brave.news"),
     /* The rest of the list is in alphabetical order */
     perfetto::Category("__metadata"),
     perfetto::Category("accessibility"),
