diff --git a/base/BUILD.gn b/base/BUILD.gn
index b4c3abd70029072c7371aa6424ffc7289040551f..27e1985c0176c658af2aafc999d70450fdfcdae1 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4640,6 +4640,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5293,6 +5294,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
