diff --git a/base/BUILD.gn b/base/BUILD.gn
index 0865b69d10656c050739e8173c5bc0e09240c594..8a8c2e7f0f84d4b5fef48acce384e00468811cce 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5239,6 +5239,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5933,6 +5934,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
