diff --git a/base/BUILD.gn b/base/BUILD.gn
index 4dae6c29c574c95b99dc2671aaf9e0c299e1d385..205f54a1887486b44914fa4fccb202b4fb7e6693 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5015,6 +5015,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5689,6 +5690,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
