diff --git a/base/BUILD.gn b/base/BUILD.gn
index 1c989c7e922168fdd8ff40ce821c6c5af20ee1e1..cb6652b218b82dfba511795dd8e657f5da178ab5 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -3960,6 +3960,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
