diff --git a/base/BUILD.gn b/base/BUILD.gn
index 125fe6e8f461f9f5fbea98fca598ce30f2967024..1e3342404de28178cd6c1308eb2d6e286cb1fa09 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4322,6 +4322,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
