diff --git a/base/BUILD.gn b/base/BUILD.gn
index a1f4787d5608b12e24f33193b7a430c68241a5e7..fe3bd2e94c6c604a41dbf88b30a9ff14c760f386 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4641,6 +4641,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5294,6 +5295,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
