diff --git a/base/BUILD.gn b/base/BUILD.gn
index 92f38639628933381c0218f41a0a506a03c60383..0075ab340d749dd8fcc6d85673fe520308ca2566 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4348,6 +4348,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
