diff --git a/base/BUILD.gn b/base/BUILD.gn
index d273dae18a596ac14b05d1eb8f4f3b4d9e39652a..f905982e08689442718231d050429e2b0eceba2d 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -3993,6 +3993,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
