diff --git a/base/feature_list.cc b/base/feature_list.cc
index 6ae9d0d2a482b4cd578cd5e5c697b93789b9f2fc..d253010cea7e7b6fa7327aa6e2b0aa5f0abe37c4 100644
--- a/base/feature_list.cc
+++ b/base/feature_list.cc
@@ -747,6 +747,9 @@ base::flat_set<std::string> FeatureList::GetFeaturesAssociatedWithTrial(
 }
 
 bool FeatureList::IsFeatureOverridden(std::string_view feature_name) const {
+  if (internal::IsCompileOverriddenFeature(feature_name)) {
+    return true;
+  }
   return GetOverrideEntryByFeatureName(feature_name);
 }
 
@@ -1223,6 +1226,7 @@ bool FeatureList::IsFeatureEnabled(const Feature& feature) const {
 
 std::optional<bool> FeatureList::IsFeatureEnabledIfOverridden(
     const Feature& feature) const {
+  std::optional<bool> state = [&]() -> std::optional<bool> {
   OverrideState overridden_state = GetOverrideState(feature);
 
   // If marked as OVERRIDE_USE_DEFAULT, fall through to returning empty.
@@ -1231,6 +1235,11 @@ std::optional<bool> FeatureList::IsFeatureEnabledIfOverridden(
   }
 
   return std::nullopt;
+  }();
+  if (!state.has_value() && internal::IsCompileOverriddenFeature(feature.name)) {
+    return IsFeatureEnabled(feature);
+  }
+  return state;
 }
 
 FeatureList::OverrideState FeatureList::GetOverrideState(
