diff --git a/base/BUILD.gn b/base/BUILD.gn
index 9b6cdba48e0c7d7979883927f46ea2df18f12a60..0706ed613dc49745a7cce4f1cb0fe377956bdda1 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4837,6 +4837,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5498,6 +5499,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
