diff --git a/base/win/embedded_i18n/generate_embedded_i18n.gni b/base/win/embedded_i18n/generate_embedded_i18n.gni
index 79c704925623b9a47cdec425af11ad7913f24af9..672cf17fa1c4d7367c2e3927796171d4712c7922 100644
--- a/base/win/embedded_i18n/generate_embedded_i18n.gni
+++ b/base/win/embedded_i18n/generate_embedded_i18n.gni
@@ -97,6 +97,10 @@ template("generate_embedded_i18n") {
       output_rc,
     ]
 
+    if (defined(invoker.sources)) {
+      sources = invoker.sources
+    }
+
     args = [
       "--header-file",
       "$output_dir/$output_file_name_base.h",
