diff --git a/base/BUILD.gn b/base/BUILD.gn
index 20e49fa40d823f2bd8d457241f1c69fb4911395d..99d270911108501e769eefc64d4248ca7e27588f 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4288,6 +4288,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
