diff --git a/base/BUILD.gn b/base/BUILD.gn
index ddcc03bf9bec0c0b8d32eabbf01cddd269c887b7..1a2d74bf1fc0686c940041e7085ce47c9ab40727 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5084,6 +5084,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5758,6 +5759,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
