diff --git a/base/BUILD.gn b/base/BUILD.gn
index 4249616290f5b25203b33b3c5d21ebf39515a48c..08e7f1acfc4d5aab6009631c8f7043c71bd44106 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4204,6 +4204,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
       "android/java/src/org/chromium/base/time/Timer.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
