diff --git a/base/BUILD.gn b/base/BUILD.gn
index c13c24d5ccbaa3d2a52ffdad17bc6edd7e9b9c90..70d12cd52ad2a38bd10540d4797e268cfebfba93 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -3720,6 +3720,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
