diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
index 075e8173bcf1cfe420b87afe216bb06459a0d8bc..b78f6d31ec8380bc33f67297c2280fd56a8c5aea 100644
--- a/base/threading/thread_restrictions.h
+++ b/base/threading/thread_restrictions.h
@@ -726,6 +726,7 @@ class BASE_EXPORT ScopedAllowBaseSyncPrimitives {
   ScopedAllowBaseSyncPrimitives& operator=(
       const ScopedAllowBaseSyncPrimitives&) = delete;
 
+  BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H
  private:
   // This can only be instantiated by friends. Use
   // ScopedAllowBaseSyncPrimitivesForTesting in unit tests to avoid the friend
