diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
index 5d28e0a1eceff66747e8472c7c0744d2f62194c3..0204b07cc1d9b46a7911036721ca8db5291ac6e9 100644
--- a/base/threading/thread_restrictions.h
+++ b/base/threading/thread_restrictions.h
@@ -468,6 +468,7 @@ INLINE_IF_DCHECK_IS_OFF void DisallowBaseSyncPrimitives()
     EMPTY_BODY_IF_DCHECK_IS_OFF;
 
 class BASE_EXPORT ScopedAllowBaseSyncPrimitives {
+  BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H
  private:
   // This can only be instantiated by friends. Use
   // ScopedAllowBaseSyncPrimitivesForTesting in unit tests to avoid the friend
