diff --git a/base/BUILD.gn b/base/BUILD.gn
index dbef16bc357905655a9a3f8f6a1aabc6f41051d4..e065d039455ffdf4522b513dd4eec73558df5c71 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4176,6 +4176,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
