diff --git a/base/mac/foundation_util.mm b/base/mac/foundation_util.mm
index 15fc15ba307e18f438852f00f41b2f5ecf7ff85f..1cce370b0973e56ff26dc94ca2c425de07a4d305 100644
--- a/base/mac/foundation_util.mm
+++ b/base/mac/foundation_util.mm
@@ -252,7 +252,7 @@ const char* BaseBundleID() {
 #if defined(GOOGLE_CHROME_BUILD)
   return "com.google.Chrome";
 #else
-  return "org.chromium.Chromium";
+  return "org.brave.Brave";
 #endif
 }
 
