diff --git a/base/BUILD.gn b/base/BUILD.gn
index 68dd910d0e7ce964b4f9d1a4ab23fc674533ede5..86667056d09ab7e6ecf28bc33e3f3ef4629da3a1 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5128,6 +5128,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5802,6 +5803,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
