diff --git a/base/BUILD.gn b/base/BUILD.gn
index 123c0998a7e20532c9ed02fbbe0606e142fc9916..437fde79796f5a5e2949cddd1511e15cbc1ea2ab 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -3648,6 +3648,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
