diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
index bf028545a03f9250412e484c67369b481b5a0dbe..fc392a6ed0d954171c2c16e337ee1ce354645dba 100644
--- a/base/threading/thread_restrictions.h
+++ b/base/threading/thread_restrictions.h
@@ -458,6 +458,7 @@ INLINE_IF_DCHECK_IS_OFF void DisallowBaseSyncPrimitives()
     EMPTY_BODY_IF_DCHECK_IS_OFF;
 
 class BASE_EXPORT ScopedAllowBaseSyncPrimitives {
+  BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H
  private:
   // This can only be instantiated by friends. Use
   // ScopedAllowBaseSyncPrimitivesForTesting in unit tests to avoid the friend
