diff --git a/base/BUILD.gn b/base/BUILD.gn
index 23cb4c9b86f24f746cf65ac40fa8c18d3d8bbc23..e1841634e3c292682d301d6ab06dfaa699aba6f4 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5080,6 +5080,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5755,6 +5756,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
