diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 690c755ca403d0c8a1aae53d100a93c8b01f03f7..809dd5d81c99cff89819c30312d4c4a55e184868 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -372,6 +372,7 @@ static_library("test_support") {
   # TODO(crbug.com/40031409): Fix code that adds exit-time destructors and
   # enable the diagnostic by removing this line.
   configs += [ "//build/config/compiler:no_exit_time_destructors" ]
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
