diff --git a/base/BUILD.gn b/base/BUILD.gn
index 304a8d1891a9896fa770c1b90503aeb7fe972dd0..6db56ec0e693d61286e5c93ec632bfbaaa478b34 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4368,6 +4368,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
