diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index 52d4c2888d80f7c80887609a337f402b4d072451..1706ea32eb2632cbfa5c57ea05f2594f0401d398 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -279,6 +279,7 @@ PERFETTO_DEFINE_CATEGORIES_IN_NAMESPACE_WITH_ATTRS(
     perfetto::Category("webrtc").SetTags("audio", "video"),
     perfetto::Category("webrtc_stats"),
     perfetto::Category("xr"),
+    BRAVE_INTERNAL_TRACE_LIST_BUILTIN_CATEGORIES
     perfetto::Category(TRACE_DISABLED_BY_DEFAULT("android_view_hierarchy"))
         .SetTags("slow"),
     perfetto::Category(TRACE_DISABLED_BY_DEFAULT("animation-worklet"))
