diff --git a/base/BUILD.gn b/base/BUILD.gn
index 370a19ef0b556afff8a78c308003878767273c50..99a8ce640e56f4bab0c0aee2528e7f92df6cad4a 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -3660,6 +3660,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
