diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index 60bba6a85d393ddb19e954503c663b06244ad160..d268a1c848d6dcd5417e7341f16f8005bd448166 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -173,6 +173,7 @@
   X("WebCore")                                                           \
   X("webrtc")                                                            \
   X("xr")                                                                \
+  BRAVE_INTERNAL_TRACE_LIST_BUILTIN_CATEGORIES(X) /*see choromium_src*/  \
   X(TRACE_DISABLED_BY_DEFAULT("android_view_hierarchy"))                 \
   X(TRACE_DISABLED_BY_DEFAULT("animation-worklet"))                      \
   X(TRACE_DISABLED_BY_DEFAULT("audio"))                                  \
