diff --git a/base/trace_event/memory_infra_background_allowlist.cc b/base/trace_event/memory_infra_background_allowlist.cc
index c253b40acf2fe6414113b8321df011b1ae460410..4a2741bc5d417581e7d5c6cb73c8c6b1284b48d0 100644
--- a/base/trace_event/memory_infra_background_allowlist.cc
+++ b/base/trace_event/memory_infra_background_allowlist.cc
@@ -387,6 +387,9 @@ bool IsMemoryDumpProviderInAllowlist(const char* mdp_name) {
 }
 
 bool IsMemoryAllocatorDumpNameInAllowlist(const std::string& name) {
+  if (IsMemoryAllocatorDumpNameInBraveAllowlist(name)) {
+    return true;
+  }
   // Global dumps that are of hex digits are all allowed for background use.
   if (base::StartsWith(name, "global/", CompareCase::SENSITIVE)) {
     for (size_t i = strlen("global/"); i < name.size(); i++) {
