diff --git a/base/BUILD.gn b/base/BUILD.gn
index 8f468a26cbe190cb98a17a38029d69d33a1305fa..c0fa10bdfe2b6708c0e81c7ead5d5abed67168b6 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4198,6 +4198,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
       "android/java/src/org/chromium/base/time/Timer.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
