diff --git a/base/BUILD.gn b/base/BUILD.gn
index 4516b3575892563a291c968b030ccac8398e84a1..c6e846c7ad286825762e2e93947d117a623d7e76 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -3790,6 +3790,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
