diff --git a/base/BUILD.gn b/base/BUILD.gn
index 250dde1336335da280a8046fe8f2666cca8e2724..9785a0c54b390b723061f624b0ed7fbd0e57f059 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4148,6 +4148,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
