diff --git a/base/BUILD.gn b/base/BUILD.gn
index 7ff21a5adbcbb16e109ecdc7f63f9b5f0b12e57a..224d9e7103b837602270e18300b09c4744b05aea 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -3991,6 +3991,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
