diff --git a/base/BUILD.gn b/base/BUILD.gn
index 4740e8e370b97583dcee2bab3b5aa91942d5eaa5..f56c4eca53d3668fb123bcd22586d6fba24eef24 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4147,6 +4147,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
