diff --git a/base/BUILD.gn b/base/BUILD.gn
index ac10b537c21d8b9622eedeccbe9801bb1e78d57d..d5803aeddd13a850c239ac3eeb871ae028b2b77e 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4311,6 +4311,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
