diff --git a/base/BUILD.gn b/base/BUILD.gn
index 7d3506c3587c5c42f19250254295458b8984e118..c93b670d3940a2db6ea135716e56c85e4be4a04a 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -3937,6 +3937,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
