diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index 9377661e94e74ab6f74d67306c97713ef794d2ac..4d9690984ed962c0dabb5c38f4f1f8593b8d5795 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -61,6 +61,11 @@ PERFETTO_DEFINE_TEST_CATEGORY_PREFIXES("cat",
 PERFETTO_DEFINE_CATEGORIES_IN_NAMESPACE_WITH_ATTRS(
     base,
     BASE_EXPORT,
+    perfetto::Category("brave"),
+    perfetto::Category("brave.adblock"),
+    perfetto::Category("brave.ads"),
+    perfetto::Category("brave.ai_chat"),
+    perfetto::Category("brave.news"),
     /* The rest of the list is in alphabetical order */
     perfetto::Category("__metadata"),
     perfetto::Category("accessibility"),
