diff --git a/base/metrics/histogram_functions.h b/base/metrics/histogram_functions.h
index eda83e423676e27631ff7b15bff03a89315fb3a6..0dd9bafa643ad34c2205398eb502f4cbeba8218f 100644
--- a/base/metrics/histogram_functions.h
+++ b/base/metrics/histogram_functions.h
@@ -89,6 +89,7 @@ void UmaHistogramEnumeration(std::string_view name, T sample) {
                 "Enumeration's kMaxValue is out of range of "
                 "LinearHistogram::kBucketCount_MAX. Use a sparse histogram "
                 "instead.");
+  BRAVE_HISTOGRAM_FUNCTIONS_UMA_HISTOGRAM_ENUMERATION
   DCHECK_LE(static_cast<uintmax_t>(sample),
             static_cast<uintmax_t>(T::kMaxValue));
   return UmaHistogramExactLinear(name, static_cast<int>(sample), kBucketCount);
