diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 37a7e20c415f7db3bee798e0f925c6a3edcf7f32..cc805a8cff290539625f27334040359d4457af9d 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -357,6 +357,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
