diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
index 5f83cbf9019dbe9c7fba75a5d6ebf0c4133e421c..bdd9d7577685b7b90de472a812efbb115c67a867 100644
--- a/base/threading/thread_restrictions.h
+++ b/base/threading/thread_restrictions.h
@@ -486,6 +486,7 @@ INLINE_IF_DCHECK_IS_OFF void DisallowBaseSyncPrimitives()
     EMPTY_BODY_IF_DCHECK_IS_OFF;
 
 class BASE_EXPORT ScopedAllowBaseSyncPrimitives {
+  BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H
  private:
   // This can only be instantiated by friends. Use
   // ScopedAllowBaseSyncPrimitivesForTesting in unit tests to avoid the friend
