diff --git a/base/BUILD.gn b/base/BUILD.gn
index 9b83881f61c52789ff377911f65a5e7440988aea..6f2a148073c3c70e1e041f533a3e2afb472628d2 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4788,6 +4788,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5444,6 +5445,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
