diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
index 385b540b5a4291070eff5e1e6a407223de097cdb..76ea6d80e8674c17fd317d35499c48c3df006d6c 100644
--- a/base/threading/thread_restrictions.h
+++ b/base/threading/thread_restrictions.h
@@ -731,6 +731,7 @@ class BASE_EXPORT [[maybe_unused, nodiscard]] ScopedAllowBaseSyncPrimitives {
   ScopedAllowBaseSyncPrimitives& operator=(
       const ScopedAllowBaseSyncPrimitives&) = delete;
 
+  BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H
  private:
   // This can only be instantiated by friends. Use
   // ScopedAllowBaseSyncPrimitivesForTesting in unit tests to avoid the friend
