diff --git a/base/BUILD.gn b/base/BUILD.gn
index 8417fcda40d49733b5fa6cac56e6caf623efc237..b7aeb5db260f7f95dd8248eff65b4cd83cccd8b6 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4359,6 +4359,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
