diff --git a/base/BUILD.gn b/base/BUILD.gn
index a52faec8e0a2d9401df9e620d66f39feb8873a39..27a106fe7b00fc9408978bce422a2394d3ffb0e3 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4334,6 +4334,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
