diff --git a/base/BUILD.gn b/base/BUILD.gn
index 53ed8ae4e4c4a29951c0d92242b48f8e9f65625e..3cb08e45bf3074df7a614bf16e476bbbda85b93a 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -3855,6 +3855,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
