diff --git a/base/BUILD.gn b/base/BUILD.gn
index 2edc8252e3e83ab0238383091ebea5a21df3186f..0b2ec4e7ed35a4fd4cae5dcfe4f1c10be5604700 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4699,6 +4699,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5338,6 +5339,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
