diff --git a/base/BUILD.gn b/base/BUILD.gn
index e66559755a083fd7dfe52e908447c9bafd5da01f..a9304e8957b0cabc6ee3a7c15767df924d32cda5 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4654,6 +4654,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5305,6 +5306,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
