diff --git a/base/BUILD.gn b/base/BUILD.gn
index aa1654d428420e99bdce3abf3140f764d73e7c48..cbe7fe8fa1f99202d4904ee3d9ccb6355312146b 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4417,6 +4417,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
       "android/java/src/org/chromium/base/WrappedClassLoader.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4882,6 +4883,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   # Most cached_flags code is in components/cached_flags. These classes are
@@ -4900,6 +4902,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/cached_flags/ValuesOverridden.java",
       "android/java/src/org/chromium/base/cached_flags/ValuesReturned.java",
     ]
+    sources += brave_base_cached_flags_java_sources
   }
 
   android_aidl("process_launcher_aidl") {
