diff --git a/base/BUILD.gn b/base/BUILD.gn
index a707f0bd86d7da91f11b1058af18c8d141ba97e8..b5f90d25b557566fac331eb6b958151bcd1ee6b7 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4278,6 +4278,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
