diff --git a/base/BUILD.gn b/base/BUILD.gn
index 1c383e0424d13120d87f1881df7bee74ce2f76e0..7e53fcffe3a22383e8c63b7f54471a4b72d2e9b1 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4651,6 +4651,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5312,6 +5313,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
