diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 4e6fba7b058f0dacd6bfc8cab861bfcd9a5af819..77c9e21e0c7388376288e667b86da11de9c5c25e 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -337,6 +337,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
