diff --git a/base/BUILD.gn b/base/BUILD.gn
index d441042f555c20d6b2d69c9b809399fbbbc11bac..9c2a7ff960c420eb386a1e1867ae5669171d51e2 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -3912,6 +3912,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
