diff --git a/base/BUILD.gn b/base/BUILD.gn
index cd6eff970d23dee24b6bf27501a94fb0c48329db..b9b1a2740bb5e29704a9ecee6ed7eaee678e68bf 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4628,6 +4628,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5289,6 +5290,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
