diff --git a/base/BUILD.gn b/base/BUILD.gn
index afd0690c72de4785d85b9d5bd64cdf772251f2f7..f4f353a97bc2fc2d7bc0773d141c22bb92adca62 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4043,6 +4043,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
