diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index ff5675fc9e304cc148634d24d8a988cb1ca336d6..f7034c28e9af87bfaaf8e7b7899357c6a325c0ba 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -379,6 +379,7 @@ static_library("test_support") {
   # TODO(crbug.com/40031409): Fix code that adds exit-time destructors and
   # enable the diagnostic by removing this line.
   configs += [ "//build/config/compiler:no_exit_time_destructors" ]
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
