diff --git a/base/win/default_apps_util.cc b/base/win/default_apps_util.cc
index 0f5f86ce6b7c4c63c32b85be9a0945693d7b97b0..e3c5f9e3aae88f5716a9e47fa64f5ac7aab18a51 100644
--- a/base/win/default_apps_util.cc
+++ b/base/win/default_apps_util.cc
@@ -24,6 +24,7 @@ std::wstring GetTargetForDefaultAppsSettings(base::WStringPiece protocol) {
     return base::StrCat({kSystemSettingsDefaultAppsPrefix, L"Browser"});
   if (base::EqualsCaseInsensitiveASCII(protocol, L"mailto"))
     return base::StrCat({kSystemSettingsDefaultAppsPrefix, L"Email"});
+  BRAVE_GET_TARGET_FOR_DEFAULT_APP_SETTINGS
   return L"SettingsPageAppsDefaultsProtocolView";
 }
 
