diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 983f36dd7c712b7e0613a4cc857fb915f657b443..c5540877df72cb239823c1d1f115a2d0be52ea1b 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -379,6 +379,7 @@ static_library("test_support") {
   # TODO(crbug.com/40031409): Fix code that adds exit-time destructors and
   # enable the diagnostic by removing this line.
   configs += [ "//build/config/compiler:no_exit_time_destructors" ]
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
