diff --git a/base/BUILD.gn b/base/BUILD.gn
index d72367da30bbbfb7b024c0a90c9a257d10326e63..7070d5d4f82ed8700a77d80ff10258cc06b70f2c 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4300,6 +4300,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
