diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index 2c2cde3260fb1d104d01fcc0fb7b160dc29fc4c0..cd79611a02bc85463a9273fb7f08e78e7e2e2a88 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -181,6 +181,7 @@
   X("WebCore")                                                           \
   X("webrtc")                                                            \
   X("xr")                                                                \
+  BRAVE_INTERNAL_TRACE_LIST_BUILTIN_CATEGORIES(X) /*see chromium_src*/   \
   X(TRACE_DISABLED_BY_DEFAULT("android_view_hierarchy"))                 \
   X(TRACE_DISABLED_BY_DEFAULT("animation-worklet"))                      \
   X(TRACE_DISABLED_BY_DEFAULT("audio"))                                  \
