diff --git a/base/task/sequenced_task_runner.h b/base/task/sequenced_task_runner.h
index d5a6d286b0d1b3081ec59a53b86a154b49ff6ae2..cfae0eb214e64979803523e61479145ea9761919 100644
--- a/base/task/sequenced_task_runner.h
+++ b/base/task/sequenced_task_runner.h
@@ -20,6 +20,7 @@
 
 namespace blink {
 class LowPrecisionTimer;
+class ScriptedIdleTaskController;
 class TimerBase;
 class TimerBasedTickProvider;
 class WebRtcTaskQueue;
@@ -67,6 +68,7 @@ class PostDelayedTaskPassKey {
   friend class base::DeadlineTimer;
   friend class base::MetronomeTimer;
   friend class blink::LowPrecisionTimer;
+  friend class blink::ScriptedIdleTaskController;
   friend class blink::TimerBase;
   friend class blink::TimerBasedTickProvider;
   friend class blink::WebRtcTaskQueue;
