diff --git a/base/feature_list.cc b/base/feature_list.cc
index bba1fd091685b33f4963f693d0194899d9b44d21..3f3007fc71828f0e76c78fa8194c850113bc001a 100644
--- a/base/feature_list.cc
+++ b/base/feature_list.cc
@@ -728,6 +728,9 @@ base::flat_set<std::string> FeatureList::GetFeaturesAssociatedWithTrial(
 }
 
 bool FeatureList::IsFeatureOverridden(std::string_view feature_name) const {
+  if (internal::IsCompileOverriddenFeature(feature_name)) {
+    return true;
+  }
   return GetOverrideEntryByFeatureName(feature_name);
 }
 
@@ -1184,6 +1187,7 @@ bool FeatureList::IsFeatureEnabled(const Feature& feature) const {
 
 std::optional<bool> FeatureList::IsFeatureEnabledIfOverridden(
     const Feature& feature) const {
+  std::optional<bool> state = [&]() -> std::optional<bool> {
   OverrideState overridden_state = GetOverrideState(feature);
 
   // If marked as OVERRIDE_USE_DEFAULT, fall through to returning empty.
@@ -1192,6 +1196,11 @@ std::optional<bool> FeatureList::IsFeatureEnabledIfOverridden(
   }
 
   return std::nullopt;
+  }();
+  if (!state.has_value() && internal::IsCompileOverriddenFeature(feature.name)) {
+    return IsFeatureEnabled(feature);
+  }
+  return state;
 }
 
 FeatureList::OverrideState FeatureList::GetOverrideState(
