diff --git a/base/BUILD.gn b/base/BUILD.gn
index aaba93f3af194f3c07c8f5883eb23001ffd08650..903c1b4f02938817c4ad13703509884e138ab987 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4779,6 +4779,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5435,6 +5436,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
