diff --git a/base/metrics/histogram_functions.h b/base/metrics/histogram_functions.h
index 9baec5417d6a097fb1d596b5c63a06bbf6cabf26..3f6577709bbede7b012b52b5247b5fd36a7f7109 100644
--- a/base/metrics/histogram_functions.h
+++ b/base/metrics/histogram_functions.h
@@ -80,6 +80,7 @@ void UmaHistogramEnumeration(std::string_view name, T sample) {
   static_assert(static_cast<uintmax_t>(T::kMaxValue) <=
                     static_cast<uintmax_t>(INT_MAX) - 1,
                 "Enumeration's kMaxValue is out of range of INT_MAX!");
+  BRAVE_HISTOGRAM_FUNCTIONS_UMA_HISTOGRAM_ENUMERATION
   DCHECK_LE(static_cast<uintmax_t>(sample),
             static_cast<uintmax_t>(T::kMaxValue));
   // While UmaHistogramExactLinear’s documentation states that the third
