diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 3e64db0c2d97f28a3ac59ec50b747ead25a9fbc6..4cd9e95002a910dbaadd7fd34c1ba89e4bf0da76 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -397,6 +397,7 @@ static_library("test_support") {
   # TODO(crbug.com/40031409): Fix code that adds exit-time destructors and
   # enable the diagnostic by removing this line.
   configs += [ "//build/config/compiler:no_exit_time_destructors" ]
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
