diff --git a/base/BUILD.gn b/base/BUILD.gn
index 73f29943ba0f9851d9b141e31bf7807b8ffbdfde..e12c25d614c794201cc72908f5e841a1654ba5e3 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4206,6 +4206,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
       "android/java/src/org/chromium/base/time/Timer.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
