diff --git a/base/BUILD.gn b/base/BUILD.gn
index 7d76b3464102b926fdcc5a36998bc88b8c142be4..e42b94c45a63d7114566a5ee5cf44b73b1bb9c66 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4667,6 +4667,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5319,6 +5320,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
