diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
index 6f000c21239c85de00733ccaef02feeabdf994de..bef7dcc2bd333dde8e796c1d7a578463e17f36fe 100644
--- a/base/threading/thread_restrictions.h
+++ b/base/threading/thread_restrictions.h
@@ -735,6 +735,7 @@ class BASE_EXPORT ScopedAllowBaseSyncPrimitives {
   ScopedAllowBaseSyncPrimitives& operator=(
       const ScopedAllowBaseSyncPrimitives&) = delete;
 
+  BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H
  private:
   // This can only be instantiated by friends. Use
   // ScopedAllowBaseSyncPrimitivesForTesting in unit tests to avoid the friend
