diff --git a/base/allocator/partition_alloc_features.cc b/base/allocator/partition_alloc_features.cc
index b2942fc2bdac0a0b464f0f78f6d380b9ff12f11c..359eb202d2555c3d4acdf5563f15ac3f89aeb52c 100644
--- a/base/allocator/partition_alloc_features.cc
+++ b/base/allocator/partition_alloc_features.cc
@@ -488,7 +488,7 @@ BASE_FEATURE(kPartitionAllocUseSmallSingleSlotSpans,
 BASE_FEATURE(kPartitionAllocShadowMetadata,
              "PartitionAllocShadowMetadata",
 #if BUILDFLAG(IS_LINUX)
-             FEATURE_ENABLED_BY_DEFAULT);
+             FEATURE_DISABLED_BY_DEFAULT);  // Brave: crbug.com/40238514
 #else
              FEATURE_DISABLED_BY_DEFAULT);
 #endif
