diff --git a/base/BUILD.gn b/base/BUILD.gn
index f73ed42f84ca486bc0f103370823dbaa14afa35f..39e58dd0cb1fa429f8d1cfbda509a2c3d41cb610 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4597,6 +4597,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5258,6 +5259,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
