diff --git a/base/BUILD.gn b/base/BUILD.gn
index f990dad216453ba9cd9047478653f14bd3626daa..d894737095342ad9bc177a8f4dd9cccd1ecae5a3 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4198,6 +4198,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
       "android/java/src/org/chromium/base/time/Timer.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
