diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
index 0b572372308a44409fc5e58d92644e644da8ce6f..f3c8a9ecdcd3c40e279a2ad7e45208c408e5d91d 100644
--- a/base/threading/thread_restrictions.h
+++ b/base/threading/thread_restrictions.h
@@ -531,6 +531,7 @@ class BASE_EXPORT ScopedDisallowBaseSyncPrimitives {
 };
 
 class BASE_EXPORT ScopedAllowBaseSyncPrimitives {
+  BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H
  private:
   // This can only be instantiated by friends. Use
   // ScopedAllowBaseSyncPrimitivesForTesting in unit tests to avoid the friend
