diff --git a/base/win/embedded_i18n/generate_embedded_i18n.gni b/base/win/embedded_i18n/generate_embedded_i18n.gni
index 08da8f8d38e5295d6e1d869b1a5cb00f904c7aad..0276031a3592ac394bd648b9aa9ae3b7a5a781c1 100644
--- a/base/win/embedded_i18n/generate_embedded_i18n.gni
+++ b/base/win/embedded_i18n/generate_embedded_i18n.gni
@@ -97,6 +97,10 @@ template("generate_embedded_i18n") {
       output_rc,
     ]
 
+    if (defined(invoker.sources)) {
+      sources = invoker.sources
+    }
+
     args = [
       "--header-file",
       "$output_dir/$output_file_name_base.h",
