diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 82fab133d08d67d9b7829f16605ac76413bcf1fd..9838e1538378c62d45ffc3fbd1c71ab2b522cd30 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -355,6 +355,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
