diff --git a/base/BUILD.gn b/base/BUILD.gn
index fdfe3630b4c0fa473e012f21b57dd826c5603167..3727ca1c390c2a5a5a8aaa67a03dbfa01e1de94a 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5151,6 +5151,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5851,6 +5852,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
