diff --git a/base/BUILD.gn b/base/BUILD.gn
index 98c3dbe91452543c4cbe58f51268633346f3851f..dcb7083357ee5943387d25acf7e8acc791059ae3 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4344,6 +4344,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
