diff --git a/base/BUILD.gn b/base/BUILD.gn
index f820c9e57f92e83950a7f54f5fa539a712711f5c..d848c870baf8f20d233bc7d525e107f3a18710a6 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4739,6 +4739,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5378,6 +5379,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
