diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index 37bbac5815969906f4d75798c37f748e9fc75216..563f0bb6d75138a9be43f76154c76565d8f00ddd 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -185,6 +185,7 @@
   X("webrtc")                                                            \
   X("webrtc_stats")                                                      \
   X("xr")                                                                \
+  BRAVE_INTERNAL_TRACE_LIST_BUILTIN_CATEGORIES(X) /*see chromium_src*/   \
   X(TRACE_DISABLED_BY_DEFAULT("android_view_hierarchy"))                 \
   X(TRACE_DISABLED_BY_DEFAULT("animation-worklet"))                      \
   X(TRACE_DISABLED_BY_DEFAULT("audio"))                                  \
