diff --git a/base/BUILD.gn b/base/BUILD.gn
<<<<<<< HEAD
index aa1654d428420e99bdce3abf3140f764d73e7c48..cbe7fe8fa1f99202d4904ee3d9ccb6355312146b 100644
=======
index 3a08c3bde5977eeb0bfa1f14713b715394f2cd6f..1a891b80b9b1e93ce7a40e1a3bdd02d55507533a 100644
>>>>>>> a6a09ab89ec ([Android] Move base.cached_flags to components.cached_flags)
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4417,6 +4417,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
       "android/java/src/org/chromium/base/WrappedClassLoader.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4882,6 +4883,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   # Most cached_flags code is in components/cached_flags. These classes are
