diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
index f459e521c831c7a3cbf9c6b8d829e4b84549b723..cb8aca9460abaf47c2b1f70d31bd15a8407f565e 100644
--- a/base/threading/thread_restrictions.h
+++ b/base/threading/thread_restrictions.h
@@ -517,6 +517,7 @@ class BASE_EXPORT ScopedDisallowBaseSyncPrimitives {
 };
 
 class BASE_EXPORT ScopedAllowBaseSyncPrimitives {
+  BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H
  private:
   // This can only be instantiated by friends. Use
   // ScopedAllowBaseSyncPrimitivesForTesting in unit tests to avoid the friend
