diff --git a/base/BUILD.gn b/base/BUILD.gn
index 998d8906b6bd33f48199dc5a67bbd149742eb561..3b8422d9ccabfc7669ea5a1cd2e482f8c2e18417 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4495,6 +4495,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
       "android/java/src/org/chromium/base/WrappedClassLoader.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -5032,6 +5033,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
