diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index 440349df6c5767fe3f93b51f78b33bf9d3bb5c1a..e8ecc20f5af9110649d3ede401c19cd070931f21 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -224,7 +224,7 @@ PERFETTO_DEFINE_CATEGORIES_IN_NAMESPACE_WITH_ATTRS(
       "processes, etc. and their properties.").SetTags("toplevel"),
     perfetto::Category("persistent_cache"),
     perfetto::Category("PlatformMalloc"),
-    perfetto::Category("ppapi"),
+    // perfetto::Category("ppapi"),
     perfetto::Category("ppapi_proxy"),
     perfetto::Category("print"),
     perfetto::Category("raf_investigation"),
@@ -284,6 +284,7 @@ PERFETTO_DEFINE_CATEGORIES_IN_NAMESPACE_WITH_ATTRS(
     perfetto::Category("webrtc").SetTags("audio", "video"),
     perfetto::Category("webrtc_stats"),
     perfetto::Category("xr"),
+    BRAVE_INTERNAL_TRACE_LIST_BUILTIN_CATEGORIES
     perfetto::Category(TRACE_DISABLED_BY_DEFAULT("android_view_hierarchy"))
         .SetTags("slow"),
     perfetto::Category(TRACE_DISABLED_BY_DEFAULT("animation-worklet"))
