diff --git a/base/BUILD.gn b/base/BUILD.gn
index 37e006910a1bb91bc506a59319d8021ea9b7660c..c2f39c2569842d5621eadfb3ed5e7aa439ccc714 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5136,6 +5136,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5821,6 +5822,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
