diff --git a/base/BUILD.gn b/base/BUILD.gn
index 18ee47c7efc7a35a3a254998494f62315991847b..9e57a3f4b9cc959e8b265ed3d147014b473018fb 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4193,6 +4193,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
       "android/java/src/org/chromium/base/time/Timer.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
