diff --git a/base/feature_list.cc b/base/feature_list.cc
index e8f9091d2a1c1126a6b01abb70535ac5bce94aff..debcb43b392bc7eee7d95fe3b65a6c833d8c6aec 100644
--- a/base/feature_list.cc
+++ b/base/feature_list.cc
@@ -748,6 +748,9 @@ base::flat_set<std::string> FeatureList::GetFeaturesAssociatedWithTrial(
 }
 
 bool FeatureList::IsFeatureOverridden(std::string_view feature_name) const {
+  if (internal::IsCompileOverriddenFeature(feature_name)) {
+    return true;
+  }
   return GetOverrideEntryByFeatureName(feature_name);
 }
 
@@ -1224,6 +1227,7 @@ bool FeatureList::IsFeatureEnabled(const Feature& feature) const {
 
 std::optional<bool> FeatureList::IsFeatureEnabledIfOverridden(
     const Feature& feature) const {
+  std::optional<bool> state = [&]() -> std::optional<bool> {
   OverrideState overridden_state = GetOverrideState(feature);
 
   // If marked as OVERRIDE_USE_DEFAULT, fall through to returning empty.
@@ -1232,6 +1236,11 @@ std::optional<bool> FeatureList::IsFeatureEnabledIfOverridden(
   }
 
   return std::nullopt;
+  }();
+  if (!state.has_value() && internal::IsCompileOverriddenFeature(feature.name)) {
+    return IsFeatureEnabled(feature);
+  }
+  return state;
 }
 
 FeatureList::OverrideState FeatureList::GetOverrideState(
