diff --git a/base/version_info/BUILD.gn b/base/version_info/BUILD.gn
index b1f13fa6e555a874d1f96eacefcd5a8401739c75..aa0a9cd343d9ba3134e3673f295a2c73c7104474 100644
--- a/base/version_info/BUILD.gn
+++ b/base/version_info/BUILD.gn
@@ -23,6 +23,7 @@ static_library("version_info") {
     ":generate_version_info",
     "//base",
   ]
+  configs += [ "//brave/build:version" ] deps += [ "//brave/components/version_info" ]
 }
 
 static_library("version_string") {
