diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 3b2fcb0c84b6d67c8ca31a7a25157add9c40af80..020070110aba19a22713588178effc7485fca4d3 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -357,6 +357,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
