diff --git a/base/memory/memory_pressure_listener.h b/base/memory/memory_pressure_listener.h
index 0c361d83fb4c0086ec30d2556a141baf493a7889..566e49d97657c4a3bfcb4120eaf02f3752294cbf 100644
--- a/base/memory/memory_pressure_listener.h
+++ b/base/memory/memory_pressure_listener.h
@@ -102,6 +102,8 @@ enum class MemoryPressureListenerTag {
   kGrShaderCache = 68,
   kMemoryPressureListenerPolicy = 69,
   kLongScreenshotsTabService = 70,
+  kRewardsDatabase = 1000,
+  kAdsDatabase = 1001,
 };
 
 // base::MemoryPressureListener is deprecated. Use base::MemoryConsumer instead.
