diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index 4ee70d069ea8a63e2ff37b3088602e4bc6ee758d..772e99d9d24e4d5c03837cc543275a24f02537f5 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -254,6 +254,7 @@ PERFETTO_DEFINE_CATEGORIES_IN_NAMESPACE_WITH_ATTRS(
     perfetto::Category("webrtc").SetTags("audio", "video"),
     perfetto::Category("webrtc_stats"),
     perfetto::Category("xr"),
+    BRAVE_INTERNAL_TRACE_LIST_BUILTIN_CATEGORIES
     perfetto::Category(TRACE_DISABLED_BY_DEFAULT("android_view_hierarchy"))
         .SetTags("slow"),
     perfetto::Category(TRACE_DISABLED_BY_DEFAULT("animation-worklet"))
