diff --git a/base/BUILD.gn b/base/BUILD.gn
index 7f98c195c12b141a792913b3e9888d8eeabc74ed..76dbfd2d5c93442ce604358124fcccbf571f1db1 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4670,6 +4670,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5322,6 +5323,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
