diff --git a/base/BUILD.gn b/base/BUILD.gn
index 73d636ded85feb3db6582d5eab8c713fb7304b31..28b849d68ef83d27e1980ac69831d15b7b548a4e 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4698,6 +4698,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5337,6 +5338,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
