diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index ca4a54e4335ae206f5b96431e7bb2b61edc65ae4..eb1fa2e9c6e2552e63ca9eebfc6973d121a7443f 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1622,6 +1622,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags

   configs = [ ":default_warnings" ]
 }
