diff --git a/build/config/chrome_build.gni b/build/config/chrome_build.gni
index ddb1d392ef6b7fcbcb26c57e23761e14b39927df..dc05b5fa491c9557d82b8027b88c26f6962a4aa9 100644
--- a/build/config/chrome_build.gni
+++ b/build/config/chrome_build.gni
@@ -1,6 +1,7 @@
 # Copyright 2015 The Chromium Authors
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
+import("//brave/build/config/brave_build.gni")
 
 declare_args() {
   # Select the desired branding flavor. False means normal Chromium branding,
