diff --git a/build/config/android/BUILD.gn b/build/config/android/BUILD.gn
index 63b37e0c17c678dca03b18b9cbfc3a5b778f208b..a3722c8bad6efbd8233c47b55e535f849acc5e32 100644
--- a/build/config/android/BUILD.gn
+++ b/build/config/android/BUILD.gn
@@ -39,6 +39,7 @@ config("compiler") {
     # http://crbug.com/448386
     "-Wl,--exclude-libs=libvpx_assembly_arm.a",
   ]
+  ldflags += ["-Wl,--allow-multiple-definition"] # Resolve conflict between libgcc++ and rustup symbols (e.g. __divdi3)
 
   if (current_cpu == "arm64") {
     # Reduce the page size from 65536 in order to reduce binary size slightly
