diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 461e62da2d5025de79dd6913cebfc0ad19150543..1c6fd92574dcd82f71a4b74e42bcec1b02e07508 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1965,7 +1965,7 @@ config("default_optimization") {
     # The NaCl IRT is a special case and always wants its own config.
     # It gets optimized the same way regardless of the type of build.
     configs = [ "//build/config/nacl:irt_optimize" ]
-  } else if (is_debug) {
+  } else if (is_debug || !is_official_build) {
     configs = [ ":no_optimize" ]
   } else if (optimize_for_fuzzing) {
     assert(!is_win, "Fuzzing optimize level not supported on Windows")
