diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
index df2289ab13cab0a0f0d6551bdbc94e3737a9fd8c..074f6134f07869c1ed2abe64a0c37bf8152b1ece 100644
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -658,6 +658,7 @@ if (enable_java_templates) {
   #     output = "$target_gen_dir/AndroidManifest.xml"
   #   }
   template("jinja_template") {
+    if (defined(invoker.includes)) { invoker.includes += brave_android_manifest_includes } else { invoker.includes = brave_android_manifest_includes }
     action_with_pydeps(target_name) {
       forward_variables_from(invoker,
                              [
