diff --git a/build/android/gyp/lint.py b/build/android/gyp/lint.py
index 61986747e600d91d1eed8a684ee88e3d109b5859..53193d0516be592008a11607d6831749c3a6ce50 100755
--- a/build/android/gyp/lint.py
+++ b/build/android/gyp/lint.py
@@ -525,5 +525,6 @@ def main():
   server_utils.MaybeTouch(args.stamp)
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
   sys.exit(main())
