diff --git a/build/config/siso/main.star b/build/config/siso/main.star
index 6a6c788c0e7416ada53854d51383c521c32c6b7e..50d7a5f36ece5881fc7a3da68972f2e8254c5b1e 100644
--- a/build/config/siso/main.star
+++ b/build/config/siso/main.star
@@ -24,6 +24,7 @@ load("./rust.star", "rust")
 load("./simple.star", "simple")
 load("./typescript_all.star", "typescript_all")
 load("./windows.star", chromium_windows = "chromium")
+load("./brave_siso_config.star", "brave_siso_config")
 
 def __setup_python(ctx, step_config):
     for rule in step_config["rules"]:
@@ -135,6 +136,7 @@ def init(ctx):
     handlers.update(reclient.handlers)
     handlers.update(typescript_all.handlers)
 
+    brave_siso_config.configure(ctx, step_config, filegroups, handlers)
     return module(
         "config",
         step_config = json.encode(step_config),
