diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index d893ebc344219f85a244e7530e633949507ee0b8..9d50ee72c76e5739ed3ec9fdf2e3c6d5a54b3652 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -8,6 +8,7 @@
 # toolchain, this GNI file may be read and processed from within Chrome OS
 # toolchains. Checking |is_android| here would therefore be too restrictive.
 if (is_android || is_chromeos) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/chromecast_build.gni")
   import("//build/config/dcheck_always_on.gni")
