diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 6bfbd46d0c894e20484d807e8b324b00fa2bfded..2b7e49e6ecfc8fc79f66a470fdabb5ca6d56ae42 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1730,6 +1730,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
