diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 93a97ff2803fe335be4537bc2d21a493468a2a23..b2073ca826d7f488ffb4ebcbb6cf00af58cdf37f 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -46,6 +46,8 @@
 # When writing build files, to do something only for the host:
 #   if (current_toolchain == host_toolchain) { ...
 
+import("//brave/brave_init_settings.gni")
+
 if (target_os == "") {
   target_os = host_os
 }
@@ -528,7 +530,14 @@ default_compiler_configs = [
   "//build/config/coverage:default_coverage",
   "//build/config/sanitizers:default_sanitizer_flags",
 ]
-
+if (brave_chromium_build) {
+  default_compiler_configs -= [ "//build/config/compiler:default_include_dirs" ]
+  default_compiler_configs += [
+    "//brave/build:feature_flags",
+    "//brave/build:brave_include_dirs",
+    "//build/config/compiler:default_include_dirs"
+  ]
+}
 if (is_win) {
   default_compiler_configs += [
     "//build/config/win:default_crt",
