diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index 53ec7373755170ce0f0364b855a34e69b5fc51cd..388e7f2689ba9cf3092f52c9a8eafb28cc94a388 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -376,6 +376,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
