diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index 0dc26faf653852aeec3c3de2a7f79a175b6f772b..f9865d4d4c079dfbb28a360446890bc304405bf7 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -393,6 +393,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
