diff --git a/build/rust/std/BUILD.gn b/build/rust/std/BUILD.gn
index 9f95c130d64bdd2ad3f8dd4c698a0c66bbcd8570..c55c17c0124a81f57783a3b9611974bc61a1cdb8 100644
--- a/build/rust/std/BUILD.gn
+++ b/build/rust/std/BUILD.gn
@@ -200,6 +200,9 @@ if (toolchain_has_rust) {
         sources = []
         output_name = "librustc-nightly_rt.$rt_name"
         output_dir = "$local_rustc_sysroot/$sysroot_lib_subdir"
+        if (is_win) {
+          arflags = [ "/llvmlibempty" ]
+        }
       }
     }
     rustc_sanitizer_runtimes = []
