diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index d196afd1c101ad3e775c4e776b494d4647312afb..d1b50c0f785270658407a2bf52a50f7db1ee1692 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1732,6 +1732,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
