diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index c78a201b1c7c25bbd497789eea62413558bf33cd..f888184705b2ace5805c7ea6a75a4d67b749b276 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -29,6 +29,7 @@ assert(!(incremental_install && !is_java_debug),
 robolectric_toolchain = "//build/toolchain/android:robolectric_$host_cpu"
 
 if (is_android) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/clang/clang.gni")
   import("//build/config/dcheck_always_on.gni")
