diff --git a/build/config/locales.gni b/build/config/locales.gni
index e1f23cb53d9d41b28fcdd1562e2a7898433932b6..b5936d50d8a93a5025c3895075e7ec07d0ea15db 100644
--- a/build/config/locales.gni
+++ b/build/config/locales.gni
@@ -154,6 +154,7 @@ extended_locales = [
   "zh-HK",
   "zu",
 ]
+import("//brave/build/config/locales.gni") extended_locales -= brave_extended_locales_exclusions
 
 # Chrome locales not on Android.
 # These locales have not yet been tested yet. Specifically, AOSP has not been
