diff --git a/build/rust/cargo_crate.gni b/build/rust/cargo_crate.gni
index 3d8a929adcfc8b16d9258b8dc2ac68a8ee0646c0..159a578d1bedb325049e59a307b60a8c22c7325b 100644
--- a/build/rust/cargo_crate.gni
+++ b/build/rust/cargo_crate.gni
@@ -281,6 +281,7 @@ template("cargo_crate") {
       testonly = _testonly
       if (defined(invoker.visibility)) {
         visibility = invoker.visibility
+      visibility += [ "//brave/third_party/*", "//brave/components/*",  ]
       }
       crate_name = _crate_name
 
