diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 538bb3d72967da509ae3173e74149ec7b530172f..35e26fba89f52f1167fe3ca746a798fedc7a6573 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -363,6 +363,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config/compiler/pgo:default_pgo_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
