diff --git a/build/util/android_chrome_version.py b/build/util/android_chrome_version.py
index c06bb38b74b7f8c4e02186d7056ee4a05a2dae04..fcfaecf1f64a198a7a4e6e1e239a4f63e2767f98 100644
--- a/build/util/android_chrome_version.py
+++ b/build/util/android_chrome_version.py
@@ -191,7 +191,7 @@ def GenerateVersionCodes(version_values, arch, is_next_build):
   """
 
   base_version_code = int(
-      '%s%03d00' % (version_values['BUILD'], int(version_values['PATCH'])))
+      '%02d%02d%03d00' % (int(version_values['MINOR']) + 40, int(version_values['BUILD']), int(version_values['PATCH'])))
 
   if is_next_build:
     base_version_code += _NEXT_BUILD_VERSION_CODE_DIFF
