diff --git a/build/config/siso/main.star b/build/config/siso/main.star
index c87d0eef4b56952a318868bab5d6d17eaa6a57b4..d6d9203eb2213594032aba24e3ebd1a131d535db 100644
--- a/build/config/siso/main.star
+++ b/build/config/siso/main.star
@@ -22,6 +22,7 @@ load("./rust.star", "rust")
 load("./simple.star", "simple")
 load("./typescript_all.star", "typescript_all")
 load("./windows.star", chromium_windows = "chromium")
+load("./brave_siso_config.star", "brave_siso_config")
 
 def __disable_remote(ctx, step_config):
     gn_logs_data = gn_logs.read(ctx)
@@ -103,6 +104,7 @@ def init(ctx):
     handlers.update(reproxy.handlers)
     handlers.update(typescript_all.handlers)
 
+    brave_siso_config.configure(ctx, step_config, filegroups, handlers)
     return module(
         "config",
         step_config = json.encode(step_config),
