diff --git a/build/config/android/internal_rules.gni b/build/config/android/internal_rules.gni
index 1494124a991658eef51c5b2803beeff769636d02..13cc2304c109e9b037e1e98e144cdcf5b6e33f88 100644
--- a/build/config/android/internal_rules.gni
+++ b/build/config/android/internal_rules.gni
@@ -1798,6 +1798,7 @@ if (enable_java_templates) {
         }
         if (_enable_bytecode_checks) {
           args += [ "--enable-check-class-path" ]
+          args -= [ "--enable-check-class-path" ]
         }
         args += [
           "--missing-classes-allowlist",
