diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index e0514b45ffc7cef5440bec757d668016d2f09447..f82645b8d10ea11310ceda3a782a13a4c68d840f 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -385,6 +385,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
