diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index 701181f55e0e60ab45764a98ea6bb7a329fc7d77..1bedb174268b07ddab50060a1cc323454dc67578 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -31,6 +31,7 @@ assert(!(incremental_install && !is_java_debug),
 robolectric_toolchain = "//build/toolchain/android:robolectric_$host_cpu"
 
 if (is_android) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/clang/clang.gni")
   import("//build/config/dcheck_always_on.gni")
