diff --git a/build/config/chrome_build.gni b/build/config/chrome_build.gni
index 99758f6df440fbbadcf639c164f5d137b7f5652e..20fa72d5cfa5fcd197e4184365b35884d54d1c67 100644
--- a/build/config/chrome_build.gni
+++ b/build/config/chrome_build.gni
@@ -1,6 +1,7 @@
 # Copyright 2015 The Chromium Authors
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
+import("//brave/build/config/brave_build.gni")
 
 import("//build/config/gclient_args.gni")
 import("//build_overrides/build.gni")
@@ -77,6 +78,7 @@ assert(is_chrome_for_testing || !is_chrome_for_testing_branded,
        "`is_chrome_for_testing_branded` requires `is_chrome_for_testing`")
 
 declare_args() {
+  branding_base_dir = "//chrome/app"
   # Refers to the subdirectory for branding in various places including
   # chrome/app/theme.
   #
@@ -97,9 +99,12 @@ declare_args() {
   }
 }
 
+# Location of theme resources in branding_base_dir
+branding_theme_dir = "$branding_base_dir/theme"
+
 declare_args() {
   # The path to the BRANDING file in chrome/app/theme.
-  branding_file_path = "//chrome/app/theme/$branding_path_component/BRANDING"
+  branding_file_path = "$branding_theme_dir/$branding_path_component/BRANDING"
 }
 
 # Whether to enable built-in branding assets (logos & marketing snippets) for
