diff --git a/build/dotfile_settings.gni b/build/dotfile_settings.gni
index dc47c40070b238996d694663e540c75c36fabb88..432e7874fa75514125f2cfc57458e5d7746e0145 100644
--- a/build/dotfile_settings.gni
+++ b/build/dotfile_settings.gni
@@ -63,3 +63,4 @@ build_dotfile_settings = {
     "//build/modules/BUILD.gn",
   ]
 }
+import("//brave/build/dotfile_settings.gni") build_dotfile_settings.exec_script_allowlist += brave_exec_script_allowlist
