diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 69f7eb1ac466fbd4e528e3cf2e4e2f96e622424f..9cddd247278ff1b8bece6bac8bc5ff9ed5c198dc 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -345,6 +345,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config/compiler/pgo:default_pgo_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
