diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 427296f350788b3dbfbcec34e3157aa48913d3d7..53bf2c375b67b4930823e9830a7a53dcf47e7495 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1680,6 +1680,7 @@ config("chromium_code") {
       ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [
     ":default_warnings",
