diff --git a/build/util/action_remote.py b/build/util/action_remote.py
index 040d5118ffe822f265e3c9ce13896682302c96ee..e7bd51a3c415c2f6015835d8ed8ef0a77bfaa28d 100755
--- a/build/util/action_remote.py
+++ b/build/util/action_remote.py
@@ -99,6 +99,7 @@ def _get_mojom_parser_inputs(exec_root, output_files, extra_args):
              processed_inputs)
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 def main():
   # Set up argparser with some rewrapper flags.
   argparser = argparse.ArgumentParser(description='rewrapper executor for gn',
