diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index 0c19c020290c96dbe3874cbea07d022bdeebaf9e..322b0eb2c5d11cffd10e61d4b85f0b425f6930c5 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -8,6 +8,7 @@
 # toolchain, this GNI file may be read and processed from within Chrome OS
 # toolchains. Checking |is_android| here would therefore be too restrictive.
 if (is_android || is_chromeos) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/chromecast_build.gni")
   import("//build/config/dcheck_always_on.gni")
