diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
index 727c3f7dac98554e9193f953dfd720b67b82861c..e0489c3c4885e0282a86d5ffc88c24dab5759c74 100644
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -208,6 +208,7 @@ if (!is_robolectric && enable_java_templates) {
   #     ]
   #   }
   template("java_cpp_enum") {
+    foreach(source, invoker.sources) { foreach(filter, brave_java_cpp_enum_filter) { if(source == filter) { invoker.sources -= [filter] }}}
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "sources" ])
 
@@ -397,6 +398,7 @@ if (!is_robolectric && enable_java_templates) {
   #     output = "$target_gen_dir/AndroidManifest.xml"
   #   }
   template("jinja_template") {
+    if (defined(invoker.includes)) { invoker.includes += brave_android_manifest_includes } else { invoker.includes = brave_android_manifest_includes }
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "deps" ])
       inputs = [ invoker.input ]
