diff --git a/build/android/gyp/util/params_json_util.py b/build/android/gyp/util/params_json_util.py
index e032e18e10acf388b08a21b663322bf199d514c5..b0b69bc6dddaec32f5cbb7bfd64cbdae527010ad 100644
--- a/build/android/gyp/util/params_json_util.py
+++ b/build/android/gyp/util/params_json_util.py
@@ -355,11 +355,7 @@ class ParamsJson(dict):
     return self.type in _MERGES_MANIFESTS_TYPES
 
   def needs_full_javac_classpath(self):
-    """Returns True if the target type runs manifest_merger.py."""
-    return self.type in ('android_apk', 'android_app_bundle_module') or (
-        self.is_library() and self.get('needs_full_javac_classpath',
-                                       False)) or (self.type == 'dist_jar'
-                                                   and self.requires_android())
+    return True
 
   def collects_dex_paths(self):
     """Returns True if the target type collects transitive .dex files."""
