diff --git a/build/android/gyp/dex.py b/build/android/gyp/dex.py
index 9252a0a5649c5aedbc41c873f738b3dfc4c8751c..0487ba0962b223b991ed75a3cf98abb1050e304c 100755
--- a/build/android/gyp/dex.py
+++ b/build/android/gyp/dex.py
@@ -54,7 +54,7 @@ _IGNORE_WARNINGS = (
 _SKIPPED_CLASS_FILE_NAMES = (
     'module-info.class',  # Explicitly skipped by r8/utils/FileUtils#isClassFile
 )
-
+_IGNORE_WARNINGS += ( r'.Brave', )
 
 def _ParseArgs(args):
   args = build_utils.ExpandFileArgs(args)
