diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index 593744682227f08d24690956ff639bbcd4e46926..ea4cedec5ca74f1ec222ab897171d836f1549b64 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -31,6 +31,7 @@ assert(!(incremental_install && !is_java_debug),
 robolectric_toolchain = "//build/toolchain/android:robolectric_$host_cpu"
 
 if (is_android) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/clang/clang.gni")
   import("//build/config/dcheck_always_on.gni")
