diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index 0584179abfef1cb215e7a2c93f431d3cb0df8051..6e5b56e2ed6216622c05d16164855afd34a81eff 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -3,6 +3,7 @@
 # found in the LICENSE file.
 
 # This file contains common system config stuff for the Android build.
+import("//brave/build/android/config.gni")
 
 # NOTE: Because Chrome OS builds may depend on targets built with the Android
 # toolchain, this GNI file may be read and processed from within Chrome OS
