diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 6ee83a41a05488f016f86f90b026835f4b19ea54..8ad38f54d4333df734b49064673e07eabf2df6dc 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -329,6 +329,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
