diff --git a/build/dotfile_settings.gni b/build/dotfile_settings.gni
index 4911002e5c18b9532af94ea1ef725d9e497ecc59..aeb5c222833b43b62d7df6653c649eb6bb56f74c 100644
--- a/build/dotfile_settings.gni
+++ b/build/dotfile_settings.gni
@@ -38,3 +38,4 @@ build_dotfile_settings = {
     "//build/util/branding.gni",
   ]
 }
+import("//brave/build/dotfile_settings.gni") build_dotfile_settings.exec_script_allowlist += brave_exec_script_allowlist
