diff --git a/build/dotfile_settings.gni b/build/dotfile_settings.gni
index f0a0a01d9444faf87b8c7e246749f6a0a27b5fee..76df86aa4a55c61922f88033f15fdfcd51fb98af 100644
--- a/build/dotfile_settings.gni
+++ b/build/dotfile_settings.gni
@@ -46,3 +46,4 @@ build_dotfile_settings = {
   # so that we don't need to keep supporting `exec_script_whitelist`.
   exec_script_whitelist = exec_script_allowlist
 }
+import("//brave/build/dotfile_settings.gni") build_dotfile_settings.exec_script_allowlist += brave_exec_script_allowlist
