diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 8cf72db378c4582229933e09c6062b6e5365dfdd..e8b9254671ce6c5865981f5931f4b634916f71e3 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1660,6 +1660,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
