diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index b14de6a6e78e6af3d3a02e11b1b483f33c708209..92abde0e1131a5008b5bbbf9e273c6c3f0a0e0b6 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -380,6 +380,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
