diff --git a/build/android/bytecode/BUILD.gn b/build/android/bytecode/BUILD.gn
index 742a46b2ee7a1edb0f3e0be7db25e1486f3dde9a..4dbd10595f0eb0a94e2d373d224e4d13e1aa8cac 100644
--- a/build/android/bytecode/BUILD.gn
+++ b/build/android/bytecode/BUILD.gn
@@ -16,6 +16,7 @@ java_binary("java_bytecode_rewriter") {
     "java/org/chromium/bytecode/ThreadAssertionClassAdapter.java",
     "java/org/chromium/bytecode/TypeUtils.java",
   ]
+  sources += brave_java_bytecode_files
   main_class = "org.chromium.bytecode.ByteCodeProcessor"
   deps = [
     "//third_party/ow2_asm:asm_java",
