diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index f33cba178d7710b1ad624fcd1b649b370bb401a4..626a655d7afa30a5853d837aaf974d2b0b59ca7a 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -519,6 +519,7 @@ set_sources_assignment_filter(sources_assignment_filter)
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
