diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 3abdf7d232cf9fd998d32cd52217b2148fe1daac..19b7dfde6d34011d2392526f144826646c54df20 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1727,6 +1727,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
