diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
index 232fd4e136978d9c6b0737c336613ab8be9e4823..bd31bf57e346fab5ca2bfb6d6df4ea7ce8365057 100644
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -513,6 +513,7 @@ if (enable_java_templates) {
   #   }
   template("java_cpp_enum") {
     set_sources_assignment_filter([])
+    foreach(source, invoker.sources) { foreach(filter, brave_java_cpp_enum_filter) { if(source == filter) { invoker.sources -= [filter] }}}
     action_with_pydeps(target_name) {
       forward_variables_from(invoker,
                              [
@@ -626,6 +627,7 @@ if (enable_java_templates) {
   #     output = "$target_gen_dir/AndroidManifest.xml"
   #   }
   template("jinja_template") {
+    if (defined(invoker.includes)) { invoker.includes += brave_android_manifest_includes } else { invoker.includes = brave_android_manifest_includes }
     action_with_pydeps(target_name) {
       forward_variables_from(invoker,
                              [
