diff --git a/build/android/gyp/java_cpp_enum.py b/build/android/gyp/java_cpp_enum.py
index ffd896d9bc7ecb937e9f367400eea0cb18defbd9..8c52096a70a94fd505361b3e0563f536e62bf86f 100755
--- a/build/android/gyp/java_cpp_enum.py
+++ b/build/android/gyp/java_cpp_enum.py
@@ -506,5 +506,6 @@ def DoMain(argv):
         zip_helpers.add_to_zip_hermetic(srcjar, output_path, data=data)
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
   DoMain(sys.argv[1:])
