diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index bafffe27168dbbc1f7e9b87beeca923f61a57abb..f972ee5a69709cab4fc7673f81ace391efbd22f5 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1651,6 +1651,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
