diff --git a/build/gn_logs.gni b/build/gn_logs.gni
index d581105e3861a2a78edbb3099e29d3829b31157f..174928b6c66a89a275dad6ffa182dee331a4030f 100644
--- a/build/gn_logs.gni
+++ b/build/gn_logs.gni
@@ -37,7 +37,7 @@ if (target_os == "ios") {
   build_gn_logs += [ "" ]
 }
 
-if (target_os == "mac") {
+if (host_os == "mac") {
   import("//build/config/mac/mac_sdk.gni")
   build_gn_logs += [ "### mac_sdk.gni ###" ] + mac_sdk_logs
   build_gn_logs += [ "" ]
