diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 1cff821865f06fec2efa37bf68781af94a4127e7..aa2c054f8e9cfbb83cdd1e77484e7e92310a493d 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -323,6 +323,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
