diff --git a/build/android/gyp/dex.py b/build/android/gyp/dex.py
index b893e51ad8a4cc4f21197210a879a6b57d404baf..828a253bf4698967f0b1343972c7fec6a056ccfd 100755
--- a/build/android/gyp/dex.py
+++ b/build/android/gyp/dex.py
@@ -53,7 +53,7 @@ _IGNORE_WARNINGS = (
 _SKIPPED_CLASS_FILE_NAMES = (
     'module-info.class',  # Explicitly skipped by r8/utils/FileUtils#isClassFile
 )
-
+_IGNORE_WARNINGS += ( r'.Brave', )
 
 def _ParseArgs(args):
   args = build_utils.ExpandFileArgs(args)
