diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index ddad619b207d724d8dda1d3a2a4a44e64bfec537..0a7ba90dfd12b350db5389a856bbb3e08a60ba5a 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -3,6 +3,7 @@
 # found in the LICENSE file.
 
 # This file contains common system config stuff for the Android build.
+import("//brave/build/android/config.gni")
 
 # NOTE: Because Chrome OS builds may depend on targets built with the Android
 # toolchain, this GNI file may be read and processed from within Chrome OS
