diff --git a/build/config/android/BUILD.gn b/build/config/android/BUILD.gn
index 8eed45e80903f0289ea8759f1c18d62c044150a7..bbdd23a638ec32aab64d564089629b6b0a2a6cc6 100644
--- a/build/config/android/BUILD.gn
+++ b/build/config/android/BUILD.gn
@@ -50,6 +50,7 @@ config("compiler") {
     # http://crbug.com/448386
     "-Wl,--exclude-libs=libvpx_assembly_arm.a",
   ]
+  ldflags += ["-Wl,--allow-multiple-definition"] # Resolve conflict between libgcc++ and rustup symbols (e.g. __divdi3)
 
   # TODO(crbug.com/1184398): Move to compiler-rt when we are ready.
   ldflags += [ "--rtlib=libgcc" ]
