diff --git a/build/android/gyp/create_app_bundle.py b/build/android/gyp/create_app_bundle.py
index 43efec3ce978c0c425d6da169ef2e7ae47e04bb3..59b42cdace895df93d90f8ad3400d2fd1b30098d 100755
--- a/build/android/gyp/create_app_bundle.py
+++ b/build/android/gyp/create_app_bundle.py
@@ -79,6 +79,7 @@ _ALLOWLISTED_NON_BASE_SERVICES = {
     'com.google.apps.tiktok.concurrent.InternalForegroundService',
 }
 
+from brave_app_bundle_utils import extend_allowlisted_non_base_services; _ALLOWLISTED_NON_BASE_SERVICES = extend_allowlisted_non_base_services(_ALLOWLISTED_NON_BASE_SERVICES)
 
 def _ParseArgs(args):
   parser = argparse.ArgumentParser()
