diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index bd09f8b97e58c040e588993c75c060b02b0f718a..ccc8d812c294981dafe48a6b851c34920bacc0df 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -8,6 +8,7 @@
 # toolchain, this GNI file may be read and processed from within Chrome OS
 # toolchains. Checking |is_android| here would therefore be too restrictive.
 if (is_android || is_chromeos) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/chromecast_build.gni")
   import("//build/config/dcheck_always_on.gni")
