diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
index 0db0395c6619b2b544dccfed39d91cc3a350d1b8..39eb27a8ce26a7c01c7a3cb689f4b1e809df7d32 100644
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -207,6 +207,7 @@ if (!is_robolectric && enable_java_templates) {
   #     ]
   #   }
   template("java_cpp_enum") {
+    foreach(source, invoker.sources) { foreach(filter, brave_java_cpp_enum_filter) { if(source == filter) { invoker.sources -= [filter] }}}
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "sources" ])
 
@@ -382,6 +383,7 @@ if (!is_robolectric && enable_java_templates) {
   #     output = "$target_gen_dir/AndroidManifest.xml"
   #   }
   template("jinja_template") {
+    if (defined(invoker.includes)) { invoker.includes += brave_android_manifest_includes } else { invoker.includes = brave_android_manifest_includes }
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "deps" ])
       inputs = [ invoker.input ]
