diff --git a/build/android/gyp/dex.py b/build/android/gyp/dex.py
index ba90872e26ddec96ea4cc233791768df2c1bdaa0..f0140e74e9d73d08fd4e2e9075368c66fde96b9d 100755
--- a/build/android/gyp/dex.py
+++ b/build/android/gyp/dex.py
@@ -58,7 +58,7 @@ _IGNORE_WARNINGS = (
 _SKIPPED_CLASS_FILE_NAMES = (
     'module-info.class',  # Explicitly skipped by r8/utils/FileUtils#isClassFile
 )
-
+_IGNORE_WARNINGS += ( r'.Brave', )
 
 def _ParseArgs(args):
   args = build_utils.ExpandFileArgs(args)
