diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index f44478e5f96211bc696cbcf1704c26cedf9f0a5b..1ea2580c59c38ffff4f17074f1ff8e071847e0d9 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -354,6 +354,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config/compiler/pgo:default_pgo_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
