diff --git a/build/android/bytecode/BUILD.gn b/build/android/bytecode/BUILD.gn
index 742a46b2ee7a1edb0f3e0be7db25e1486f3dde9a..9a83750ccffc0acc75e6d1d72be98c0b929a71af 100644
--- a/build/android/bytecode/BUILD.gn
+++ b/build/android/bytecode/BUILD.gn
@@ -16,6 +16,7 @@ java_binary("java_bytecode_rewriter") {
     "java/org/chromium/bytecode/ThreadAssertionClassAdapter.java",
     "java/org/chromium/bytecode/TypeUtils.java",
   ]
+  java_files += brave_java_bytecode_files
   main_class = "org.chromium.bytecode.ByteCodeProcessor"
   deps = [
     "//third_party/ow2_asm:asm_java",
