diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 2656f800f7da4dfd6f69f9c78f932710617057c0..a7b4fea2ae99d2c3047e7d74d9eddc9c980c2337 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1638,6 +1638,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags

   configs = [ ":default_warnings" ]
 }
