diff --git a/build/config/chrome_build.gni b/build/config/chrome_build.gni
index 99758f6df440fbbadcf639c164f5d137b7f5652e..e1900fef458a1cb0c79cabc62b2ac5e412d90416 100644
--- a/build/config/chrome_build.gni
+++ b/build/config/chrome_build.gni
@@ -1,6 +1,7 @@
 # Copyright 2015 The Chromium Authors
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
+import("//brave/build/config/brave_build.gni")
 
 import("//build/config/gclient_args.gni")
 import("//build_overrides/build.gni")
