diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index c9a3ce41465b38594c5eba3f152e4f127859ebb2..3d9c3eb009302e99c933e51435d19d7ad0d816ec 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -335,6 +335,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
