diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index 4d5c8f777245672cb0620fbbdb463cd3904c8ee9..312d893746bc4ae573a52adb77fb2889dfc4c794 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -394,6 +394,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
