diff --git a/build/android/gyp/create_app_bundle.py b/build/android/gyp/create_app_bundle.py
index d74d6fa8a6d663b0bac7682376d3ab60f889a3f7..ca1d842eb9d8cb9fb9484bdc4dc41e6bdfc1d99c 100755
--- a/build/android/gyp/create_app_bundle.py
+++ b/build/android/gyp/create_app_bundle.py
@@ -80,6 +80,7 @@ _ALLOWLISTED_NON_BASE_SERVICES = {
     'com.google.apps.tiktok.concurrent.InternalForegroundService',
 }
 
+from brave_app_bundle_utils import extend_allowlisted_non_base_services; _ALLOWLISTED_NON_BASE_SERVICES = extend_allowlisted_non_base_services(_ALLOWLISTED_NON_BASE_SERVICES)
 
 def _ParseArgs(args):
   parser = argparse.ArgumentParser()
