diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index bc176d5e0317dd4dab9c1e1bfcf20955c254674f..bec1208b56ef9bca7a785dd6d186cd3af59ac3d1 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -7,6 +7,7 @@
 import("//build/config/chromeos/ui_mode.gni")
 
 # This file contains common system config stuff for the Android build.
+import("//brave/build/android/config.gni")
 
 # NOTE: Because Chrome OS builds may depend on targets built with the Android
 # toolchain, this GNI file may be read and processed from within Chrome OS
