diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 959a592317462abb07eefbf03853e1610034c5f3..d3eaa9e595978b030ed4aad3c8ae18bc56b227fa 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -2046,7 +2046,7 @@ config("default_optimization") {
     # The NaCl IRT is a special case and always wants its own config.
     # It gets optimized the same way regardless of the type of build.
     configs = [ "//build/config/nacl:irt_optimize" ]
-  } else if (is_debug) {
+  } else if (is_debug || !is_official_build) {
     configs = [ ":no_optimize" ]
   } else if (optimize_for_fuzzing) {
     assert(!is_win, "Fuzzing optimize level not supported on Windows")
