diff --git a/build/android/gyp/util/resource_utils.py b/build/android/gyp/util/resource_utils.py
index 60e272bd410802144520166379b964a66b67a751..7f30c3b346ef43ebe8f07daecf090ba8803edf8c 100644
--- a/build/android/gyp/util/resource_utils.py
+++ b/build/android/gyp/util/resource_utils.py
@@ -286,6 +286,7 @@ class ResourceInfoFile:
             return
         source_path = os.path.normpath(source_path)
         new_value = self._entries.setdefault(archive_path, source_path)
+        return
         if new_value != source_path:
             raise Exception(
                 'Duplicate AddMapping for "{}". old={} new={}'.format(
