diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index 88321e9bac4d144f0afec7ebb7ddf2029bd48dcd..aaa0328c012546b9d047948a38685156e8ea027c 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -392,6 +392,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
