diff --git a/build/util/android_chrome_version.py b/build/util/android_chrome_version.py
index 91d31e97ad6b5fdc55128a3a5104092323d84c9c..c92067f369c32cfe46cdc6c365661cc19af51a49 100644
--- a/build/util/android_chrome_version.py
+++ b/build/util/android_chrome_version.py
@@ -191,7 +191,7 @@ def GenerateVersionCodes(version_values, arch, is_next_build):
   """
 
   base_version_code = int(
-      '%s%03d00' % (version_values['BUILD'], int(version_values['PATCH'])))
+      '%02d%02d%03d00' % (int(version_values['MINOR']) + 40, int(version_values['BUILD']), int(version_values['PATCH'])))
 
   if is_next_build:
     base_version_code += _NEXT_BUILD_VERSION_CODE_DIFF
