diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 2ac26091c028d5c41af230885f25b3e4f4a62b88..28d0a2fcdb52ac657991d35bd6292fac9a2c17e9 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -588,7 +588,6 @@ config("compiler") {
   }
 
   if (is_clang && !is_nacl && current_os != "zos") {
-    cflags += [ "-fcrash-diagnostics-dir=" + clang_diagnostic_dir ]
     if (save_reproducers_on_lld_crash && use_lld) {
       ldflags += [
         "-fcrash-diagnostics=all",
