diff --git a/build/config/android/internal_rules.gni b/build/config/android/internal_rules.gni
index f79f26ef501ed326c56b070dbe2cc993d2f858b0..0a2da0edc7823bdf036b57f98e24e1aa7904d70d 100644
--- a/build/config/android/internal_rules.gni
+++ b/build/config/android/internal_rules.gni
@@ -2072,6 +2072,7 @@ if (enable_java_templates) {
         _deps = [ ":$_desugar_target" ]
         _previous_output_jar = _desugar_output_jar
       }
+      foreach(bytecode_jar, brave_bytecode_jars) { if (rebase_path(invoker.input_jar_path, root_build_dir) == bytecode_jar) { _bytecode_rewriter_output_jar = "$target_out_dir/$target_name.bytecode-rewritten.jar" brave_bytecode_rewriter("${target_name}__bytecode_rewrite"){} _deps = [] _deps = [ ":${target_name}__bytecode_rewrite" ] _previous_output_jar = _bytecode_rewriter_output_jar } }
 
       if (invoker.jacoco_instrument) {
         _filter_jar_target_name = "${target_name}_device__filter_jar"
