diff --git a/build/toolchain/win/BUILD.gn b/build/toolchain/win/BUILD.gn
index 07319b7989989adc542383935e760bb3e0a6bbcd..515133b38cbf56df62cfc27da67ffe24d092cd75 100644
--- a/build/toolchain/win/BUILD.gn
+++ b/build/toolchain/win/BUILD.gn
@@ -30,6 +30,8 @@ if (use_goma) {
     goma_prefix = "$goma_dir/gomacc "
   }
   clang_prefix = goma_prefix
+} else if (cc_wrapper != "") {
+  goma_prefix = "$cc_wrapper "
 } else {
   goma_prefix = ""
   if (cc_wrapper != "") {
