diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 3dfb041f1a812f3a7a286965773a994243ee2203..33f3f00e1113cf7e720fd1d1a94db3d3a05c5f68 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -347,6 +347,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
