diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 031b87dd48983ff3071430d2c8b08b65574c622a..37e0942fd33df8c9d8e0cd31ca04a5634251cdca 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -354,6 +354,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
