diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index bb656fa19ff32fa407a01f302ea989ffcacc45f8..882a1b75b079151296497dbc9938dbba83924710 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -328,6 +328,7 @@ set_sources_assignment_filter(sources_assignment_filter)
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
