diff --git a/build/config/win/BUILD.gn b/build/config/win/BUILD.gn
index 1edf10346da4406a7ffb64fd9bca30f30117b66b..dcf4279da32b6f82f0b15cc210e8284368678d91 100644
--- a/build/config/win/BUILD.gn
+++ b/build/config/win/BUILD.gn
@@ -229,6 +229,12 @@ config("compiler") {
     # the source file is a no-op.
     "/ignore:4221",
   ]
+  import("//brave/build/config/win/config.gni")
+  if (brave_win_use_large_pdb &&
+      filter_include(ldflags, [ "/pdbpagesize:8192" ]) != []) {
+    ldflags -= [ "/pdbpagesize:8192" ]
+    ldflags += [ "/pdbpagesize:16384" ]
+  }
 }
 
 # This is included by reference in the //build/config/compiler:runtime_library
