diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 9534c7f4de867b6c1eb55a73cadbebe5230867f5..daf2855b99906fd53de64cffebae82d29315070e 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1700,6 +1700,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
