diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index f0a9a086a90ff5cdaeef802d714e4a88b86d0290..a8c633213a7acae116d70b469985133e5b57a40d 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1682,6 +1682,7 @@ config("chromium_code") {
       ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [
     ":default_warnings",
