diff --git a/build/config/siso/main.star b/build/config/siso/main.star
index 548330e241485073c1cb0dd0366018137d4117b0..36d10f6d48b3fd432d81b2053d588a9f5c923e06 100644
--- a/build/config/siso/main.star
+++ b/build/config/siso/main.star
@@ -23,6 +23,7 @@ load("./rust.star", "rust")
 load("./simple.star", "simple")
 load("./typescript_all.star", "typescript_all")
 load("./windows.star", chromium_windows = "chromium")
+load("./brave_siso_config.star", "brave_siso_config")
 
 def __disable_remote(ctx, step_config):
     gn_logs_data = gn_logs.read(ctx)
@@ -106,6 +107,7 @@ def init(ctx):
     handlers.update(reproxy.handlers)
     handlers.update(typescript_all.handlers)
 
+    brave_siso_config.configure(ctx, step_config, filegroups, handlers)
     return module(
         "config",
         step_config = json.encode(step_config),
