diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index efba84719e620954fd093dcac4ce2c60b6b97def..6de9b34d81eeffa41b3c8643c5322a52436207d1 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -329,6 +329,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
