diff --git a/build/android/gyp/lint.py b/build/android/gyp/lint.py
index e22f17ac81f0294ca195c12280f06849505743f1..3337191309b80c66eb4dd63b4db18ad7738c4509 100755
--- a/build/android/gyp/lint.py
+++ b/build/android/gyp/lint.py
@@ -554,5 +554,6 @@ def main():
   server_utils.MaybeTouch(args.stamp)
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
   sys.exit(main())
