diff --git a/build/config/siso/main.star b/build/config/siso/main.star
index 5ea41e811fb32e181e0a864437c2601f776b803b..7dd65a91b82d1197922008585eb325a6025c0754 100644
--- a/build/config/siso/main.star
+++ b/build/config/siso/main.star
@@ -22,6 +22,7 @@ load("./rust.star", "rust")
 load("./simple.star", "simple")
 load("./typescript_all.star", "typescript_all")
 load("./windows.star", chromium_windows = "chromium")
+load("./brave_siso_config.star", "brave_siso_config")
 
 def __disable_remote(ctx, step_config):
     gn_logs_data = gn_logs.read(ctx)
@@ -94,6 +95,7 @@ def init(ctx):
     handlers.update(reproxy.handlers)
     handlers.update(typescript_all.handlers)
 
+    brave_siso_config.configure(ctx, step_config, filegroups, handlers)
     return module(
         "config",
         step_config = json.encode(step_config),
