diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 6d705aa0f09dc8c9ff5985d851534fa98b17cdb0..44092a58ef13987c558d5f0be2a5209fe4694169 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -151,6 +151,7 @@ declare_args() {
   host_toolchain = ""
 
   # DON'T ADD MORE FLAGS HERE. Read the comment above.
+  antimuon_chromium_build = false
 }
 
 declare_args() {
@@ -530,6 +531,13 @@ default_compiler_configs = [
   "//build/config/coverage:default_coverage",
   "//build/config/sanitizers:default_sanitizer_flags",
 ]
+if (!antimuon_chromium_build) {
+  default_compiler_configs -= [ "//build/config/compiler:default_include_dirs" ]
+  default_compiler_configs += [
+    "//brave/build:antimuon_include_dirs",
+    "//build/config/compiler:default_include_dirs"
+  ]
+}
 if (is_win) {
   default_compiler_configs += [
     "//build/config/win:default_crt",
