diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index 5d1e683e37341473c72a2886a63232ca0d52773e..5a0a5d0cc5409d2028dbf47c1af4526700e69ccf 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -394,6 +394,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
