diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 0bb155a83dfe698b80f02e5378a4e480e1b0a44f..0a285872acc8a4143d378b716b458b1cf0c3ff2b 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1734,6 +1734,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
