diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index d40843b44f9ee81f026bf52e3f39c9475fe87ac3..c5400b845bb92482893cee5388b910185cca8286 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1679,6 +1679,7 @@ config("chromium_code") {
       ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [
     ":default_warnings",
