diff --git a/build/config/android/BUILD.gn b/build/config/android/BUILD.gn
index 88f18975688bf262de6bad34711e77b2174f430f..9cbba14b433b7c17dc5c856477ca160f5573c540 100644
--- a/build/config/android/BUILD.gn
+++ b/build/config/android/BUILD.gn
@@ -63,6 +63,7 @@ config("compiler") {
       rustflags = [ "-Zbranch-protection=pac-ret" ]
     }
   }
+  import("//brave/build/config/android/brave_compiler_ldflags.gni") ldflags += brave_build_config_android_compiler_ldflags
 
   # Disable TLSDESC for riscv64 until the Android NDK supports it. While
   # Chromium's clang turns it on by default for Android RISC-V devices, NDK r27
