diff --git a/build/android/bytecode/BUILD.gn b/build/android/bytecode/BUILD.gn
index 7a9d3044259a3f7dfca7de1acaf075797bd9cc47..bf9bda47d146461d3d9fa24dd8341abc9e6cd2e7 100644
--- a/build/android/bytecode/BUILD.gn
+++ b/build/android/bytecode/BUILD.gn
@@ -15,6 +15,7 @@ java_binary("java_bytecode_rewriter") {
     "java/org/chromium/bytecode/ThreadAssertionClassAdapter.java",
     "java/org/chromium/bytecode/TypeUtils.java",
   ]
+  sources += brave_java_bytecode_files
   main_class = "org.chromium.bytecode.ByteCodeProcessor"
   deps = [
     "//third_party/android_deps:org_ow2_asm_asm_java",
