diff --git a/build/util/android_chrome_version.py b/build/util/android_chrome_version.py
index 876f485466959e1d6ac57fcf237ea305880e64ab..e0457dbb68e0e4cb4d31a2767bda92ed00d9219e 100755
--- a/build/util/android_chrome_version.py
+++ b/build/util/android_chrome_version.py
@@ -286,7 +286,7 @@ def GenerateVersionCodes(version_values, arch, is_next_build):
   """
 
   base_version_code = int(
-      '%s%03d00' % (version_values['BUILD'], int(version_values['PATCH'])))
+      '%02d%02d%03d00' % (int(version_values['MINOR']) + 40, int(version_values['BUILD']), int(version_values['PATCH'])))
 
   if is_next_build:
     base_version_code += _NEXT_BUILD_VERSION_CODE_DIFF
