diff --git a/build/util/android_chrome_version.py b/build/util/android_chrome_version.py
index 4d21c03a65fdec4b85e0f3cbf77a1dd5bb69f76b..da2f7801d57f22282f81b0b16eaaadfd2455ecde 100644
--- a/build/util/android_chrome_version.py
+++ b/build/util/android_chrome_version.py
@@ -273,7 +273,7 @@ def GenerateVersionCodes(version_values, arch, is_next_build):
   """
 
   base_version_code = int(
-      '%s%03d00' % (version_values['BUILD'], int(version_values['PATCH'])))
+      '%02d%02d%03d00' % (int(version_values['MINOR']) + 40, int(version_values['BUILD']), int(version_values['PATCH'])))
 
   if is_next_build:
     base_version_code += _NEXT_BUILD_VERSION_CODE_DIFF
