diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 3f3ea354f3ee6882c85dc7071edee0384c68d595..d2a68c26b830a03d91ca5a1b3458449bb9214d73 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -341,6 +341,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
