diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index a97032d327f8ca4a71d9366476d0e65c43e991a4..fe95fc6e4225af70667632148cf208496cb57532 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1742,6 +1742,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
