diff --git a/build/config/android/internal_rules.gni b/build/config/android/internal_rules.gni
index af22e3e45c0b0f1d90e73a9eb2785bc5eb67c2f8..0e1d64b187e2c5c2c788278ded92513764a63173 100644
--- a/build/config/android/internal_rules.gni
+++ b/build/config/android/internal_rules.gni
@@ -2028,6 +2028,7 @@ if (enable_java_templates) {
         _deps = [ ":$_desugar_target" ]
         _previous_output_jar = _desugar_output_jar
       }
+      foreach(bytecode_jar, brave_bytecode_jars) { if (rebase_path(invoker.input_jar_path, root_build_dir) == bytecode_jar) { _bytecode_rewriter_output_jar = "$target_out_dir/$target_name.bytecode-rewritten.jar" brave_bytecode_rewriter("${target_name}__bytecode_rewrite"){} _deps = [] _deps = [ ":${target_name}__bytecode_rewrite" ] _previous_output_jar = _bytecode_rewriter_output_jar } }
 
       if (invoker.jacoco_instrument) {
         _filter_jar_target_name = "${target_name}_device__filter_jar"
