diff --git a/build/config/android/BUILD.gn b/build/config/android/BUILD.gn
index 697d598f0b0c35a4cbeff9eebc5e564e1881fb10..164c499e7457c3827b0ca4ff0ec7a906272b567f 100644
--- a/build/config/android/BUILD.gn
+++ b/build/config/android/BUILD.gn
@@ -50,6 +50,7 @@ config("compiler") {
     # segments to be mapped adjacently, which breakpad relies on.
     "-Wl,-z,max-page-size=$_max_page_size",
   ]
+  ldflags += ["-Wl,--allow-multiple-definition"] # Resolve conflict between libgcc++ and rustup symbols (e.g. __divdi3)
 
   if (current_cpu == "arm64") {
     if (arm_control_flow_integrity == "standard") {
