diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 0aa5b35cd8a237980dadd2b847d65f31320f0baa..60e1112aef5d78cb4dc8dbdc8b8765aba5cde86f 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1693,6 +1693,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [
     ":default_warnings",
