diff --git a/build/android/gyp/dex.py b/build/android/gyp/dex.py
index 883e0faac64ebb6b88cfa5f045da4a8f711fd0a1..8c54b95fc5737d8079d598a6d28628b77edaee5a 100755
--- a/build/android/gyp/dex.py
+++ b/build/android/gyp/dex.py
@@ -54,7 +54,7 @@ _IGNORE_WARNINGS = (
 _SKIPPED_CLASS_FILE_NAMES = (
     'module-info.class',  # Explicitly skipped by r8/utils/FileUtils#isClassFile
 )
-
+_IGNORE_WARNINGS += ( r'.Brave', )
 
 def _ParseArgs(args):
   args = build_utils.ExpandFileArgs(args)
