diff --git a/build/toolchain/rbe.gni b/build/toolchain/rbe.gni
index a0201de92592b82673eb3bb8e154c9a38471a006..1d052ad2edc38de826424a085d5c6768d40c226d 100644
--- a/build/toolchain/rbe.gni
+++ b/build/toolchain/rbe.gni
@@ -1,9 +1,9 @@
 # Defines the configuration of Remote Build Execution (RBE).
 
-# The directory where the re-client tooling binaries are.
-rbe_bin_dir = rebase_path("//buildtools/reclient", root_build_dir)
-
 declare_args() {
+  # The directory where the re-client tooling binaries are.
+  rbe_bin_dir = rebase_path("//buildtools/reclient", root_build_dir)
+
   # Execution root - this should be the root of the source tree.
   # This is defined here instead of in the config file because
   # this will vary depending on where the user has placed the
