diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 164ec91bb8bd377467e3614601450d62efb2d93f..441e5071bbbf76be5f9d27631fbf4b921fa3f962 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -335,6 +335,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
