diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 284a7da8868dc7fc46768c9ea92cc1506d887c02..3d891864428b542384c9f49c3ac56fbb46e68b83 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -419,6 +419,7 @@ set_sources_assignment_filter(sources_assignment_filter)
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
