diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index bd1fefeff6f604474cbe1bafb89513b12abf8dba..d67093b26f09ce2e12edb013d21189ec69ddcf92 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1643,6 +1643,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
