diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 6f5d5f123f89c5ef4dd601eddb6025bd4dc3b9ab..c0acd817d30aa9ad98063b3b10c56e685e501058 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -419,6 +419,7 @@ set_sources_assignment_filter(sources_assignment_filter)
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
