diff --git a/build/dotfile_settings.gni b/build/dotfile_settings.gni
index 4b8d16a5d5c83bc8a1fbcaac32f57650484f1309..b1c97d606a5a3fe428135f6e17993c9fc2e33cc3 100644
--- a/build/dotfile_settings.gni
+++ b/build/dotfile_settings.gni
@@ -40,3 +40,4 @@ build_dotfile_settings = {
     "//build/util/branding.gni",
   ]
 }
+import("//brave/build/dotfile_settings.gni") build_dotfile_settings.exec_script_allowlist += brave_exec_script_allowlist
