diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index 763bba2956e51357733b53d70cb013a1b6b7dc62..9a2c1a0c16624b7d4837492936385cbd799daa9b 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -374,6 +374,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
