diff --git a/build/config/locales.gni b/build/config/locales.gni
index cd5c7a1c656a1735b9342b649c57aa088767f47d..e2dfb719290a4be63ef8ef22479da4698227d59c 100644
--- a/build/config/locales.gni
+++ b/build/config/locales.gni
@@ -172,6 +172,7 @@ extended_locales = [
   "zh-HK",
   "zu",
 ]
+import("//brave/build/config/locales.gni") extended_locales -= brave_extended_locales_exclusions
 
 # Chrome locales not on Android.
 # These locales have not yet been tested yet. Specifically, AOSP has not been
