diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index 2975eb28a682355f81447515a5b2f47fc78be0c1..765fd3775fc1537d5900424dba31ae5471feb235 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -32,6 +32,7 @@ robolectric_toolchain = "//build/toolchain/android:robolectric_$host_cpu"
 # toolchain, this GNI file may be read and processed from within Chrome OS
 # toolchains. Checking |is_android| here would therefore be too restrictive.
 if (is_android || is_chromeos) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/clang/clang.gni")
   import("//build/config/dcheck_always_on.gni")
