diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 5a8c7bc62567110c31a251af44421b7328055823..f4963af1bac4320ed39f5fe7bf9f7b748b011f75 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1641,6 +1641,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
