diff --git a/build/config/android/internal_rules.gni b/build/config/android/internal_rules.gni
index d3427ebd6c2a64f739e3ae7f7d947c27c9666e8d..38257b7cf27e7d0c9d6e88db6ebf09e1562fcd4e 100644
--- a/build/config/android/internal_rules.gni
+++ b/build/config/android/internal_rules.gni
@@ -4032,6 +4032,7 @@ if (enable_java_templates) {
       } else {
         not_needed(invoker, [ "missing_classes_allowlist" ])
       }
+      foreach(bytecode_jar, brave_bytecode_jars) { if (rebase_path(_unprocessed_jar_path, root_build_dir) == bytecode_jar) { _bytecode_rewriter_output_jar = "$target_out_dir/$target_name.bytecode-rewritten.jar" brave_bytecode_rewriter("${target_name}__bytecode_rewrite") { build_config = _build_config } _unprocessed_jar_deps = [] _unprocessed_jar_deps = [ ":${target_name}__bytecode_rewrite" ] _unprocessed_jar_path = _bytecode_rewriter_output_jar } }
 
       if (_build_host_jar) {
         _process_host_jar_target_name = "${target_name}__host"
