diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
index 113b60905e683794a344940a6c3aa68f029c748e..797448d63eeb2d3705d4084f1037bc53497e1c1b 100644
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -213,6 +213,7 @@ if (enable_java_templates && is_android) {
   #     ]
   #   }
   template("java_cpp_enum") {
+    foreach(source, invoker.sources) { foreach(filter, brave_java_cpp_enum_filter) { if(source == filter) { invoker.sources -= [filter] }}}
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "sources" ])
 
@@ -388,6 +389,7 @@ if (enable_java_templates && is_android) {
   #     output = "$target_gen_dir/AndroidManifest.xml"
   #   }
   template("jinja_template") {
+    if (defined(invoker.includes)) { invoker.includes += brave_android_manifest_includes } else { invoker.includes = brave_android_manifest_includes }
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "deps" ])
       inputs = [ invoker.input ]
