diff --git a/build/BUILD.gn b/build/BUILD.gn
index 3fa5cae7e11f653c84af0e091277c98e3fd1d189..80403a4d76fd66cf83793218051f3baacff1e35b 100644
--- a/build/BUILD.gn
+++ b/build/BUILD.gn
@@ -54,6 +54,7 @@ buildflag_header("branding_buildflags") {
   }
 
   flags += [ "ENABLE_BUILTIN_SEARCH_PROVIDER_ASSETS=$enable_builtin_search_provider_assets" ]
+  public_deps = [ "//brave/components/brave_origin/buildflags" ]
 }
 
 buildflag_header("blink_buildflags") {
