diff --git a/build/android/gyp/dex.py b/build/android/gyp/dex.py
index 6db26715338a0d2bde6cedeae81a3889d89f4e7e..eb4d1a0d47354ae569ba7d77bd3ca54215b6dc33 100755
--- a/build/android/gyp/dex.py
+++ b/build/android/gyp/dex.py
@@ -30,6 +30,7 @@ DEFAULT_IGNORE_WARNINGS = (
     # META-INF/. This means that all rules with a '-upto-' qualifier will be
     # excluded and all rules with a -from- qualifier will be included.
     r'Running R8 version main', )
+DEFAULT_IGNORE_WARNINGS += ( r'.Brave', )
 
 _IGNORE_SERVICE_ENTRIES = (
     # ServiceLoader call is used only for ProtoBuf full (non-lite).
