diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index 41b32719c2cf0bfc72a7416670f7eb3066a4dfce..cbecbb866f0ead369c2930268779d42e3ec0d100 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -7,6 +7,7 @@
 import("//build/config/chromeos/ui_mode.gni")
 
 # This file contains common system config stuff for the Android build.
+import("//brave/build/android/config.gni")
 
 # NOTE: Because Chrome OS builds may depend on targets built with the Android
 # toolchain, this GNI file may be read and processed from within Chrome OS
