diff --git a/build/config/siso/main.star b/build/config/siso/main.star
index a019e96df5c4778c44780fa25a68b951f28ac4d5..07b602e10915c0b99d1cd966184700a8124e625e 100644
--- a/build/config/siso/main.star
+++ b/build/config/siso/main.star
@@ -24,6 +24,7 @@ load("./rust.star", "rust")
 load("./simple.star", "simple")
 load("./typescript_all.star", "typescript_all")
 load("./windows.star", chromium_windows = "chromium")
+load("./brave_siso_config.star", "brave_siso_config")
 
 def __setup_python(ctx, step_config):
     for rule in step_config["rules"]:
@@ -154,6 +155,7 @@ def init(ctx):
     handlers.update(reclient.handlers)
     handlers.update(typescript_all.handlers)
 
+    brave_siso_config.configure(ctx, step_config, filegroups, handlers)
     return module(
         "config",
         step_config = json.encode(step_config),
