diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index 11b66eb957f78c42f9854341ff2474c72390cbe2..a4a8b6ed4db28d94e5ebd1a7b9cbe613a646fa65 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -8,6 +8,7 @@
 # toolchain, this GNI file may be read and processed from within Chrome OS
 # toolchains. Checking |is_android| here would therefore be too restrictive.
 if (is_android || is_chromeos) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/chromecast_build.gni")
   import("//build/config/dcheck_always_on.gni")
