diff --git a/build/config/chrome_build.gni b/build/config/chrome_build.gni
index 4bb4a043cbbcb562aed8926867660547cd7f018c..8b0f251cc3f7170f73bf5bc2ab4724686c1c2b17 100644
--- a/build/config/chrome_build.gni
+++ b/build/config/chrome_build.gni
@@ -22,5 +22,5 @@ declare_args() {
 if (is_chrome_branded) {
   branding_path_component = "google_chrome"
 } else {
-  branding_path_component = "chromium"
+  branding_path_component = "brave"
 }
