diff --git a/build/android/gyp/util/resource_utils.py b/build/android/gyp/util/resource_utils.py
index 36d890417d5665de7b4ca48a4402fd6ff699a0c0..4cf64c6145f363f475aae61973b56249c4af1de7 100644
--- a/build/android/gyp/util/resource_utils.py
+++ b/build/android/gyp/util/resource_utils.py
@@ -256,6 +256,7 @@ class ResourceInfoFile:
       return
     source_path = os.path.normpath(source_path)
     new_value = self._entries.setdefault(archive_path, source_path)
+    return
     if new_value != source_path:
       raise Exception('Duplicate AddMapping for "{}". old={} new={}'.format(
           archive_path, new_value, source_path))
