diff --git a/build/android/gyp/lint.py b/build/android/gyp/lint.py
index 691a0e38ebc2816496d478a3f59621767b526366..80b2f9b4a622d712708731fb0b403330a8853fca 100755
--- a/build/android/gyp/lint.py
+++ b/build/android/gyp/lint.py
@@ -550,5 +550,6 @@ def main():
     action_helpers.write_depfile(args.depfile, args.stamp, depfile_deps)
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
   sys.exit(main())
