diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index dbcddb392df8c51ad40d443bcbde31bcbcf79018..1730da17a9717894f5ad7f4703569e0ed7b9e5ef 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1635,6 +1635,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
