diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index e49fc28c0893d1399432ccbfe3ec8c137e657999..13e5b1d0fbcbfa65c240aa56e22c02e81475146f 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1664,6 +1664,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [
     ":default_warnings",
