diff --git a/build/toolchain/win/setup_toolchain.py b/build/toolchain/win/setup_toolchain.py
index 47309866ca790e6753d20cdd097cce9ee2b3f57b..99e676117f6f60ef06fb84162ecea7d0efdf614b 100644
--- a/build/toolchain/win/setup_toolchain.py
+++ b/build/toolchain/win/setup_toolchain.py
@@ -277,6 +277,7 @@ def main():
         return s if re.match(r'^[a-zA-Z0-9._/\\:-]*$', s) else '"' + s + '"'
       include_I = ' '.join([q('/I' + i) for i in include])
       include_imsvc = ' '.join([q('-imsvc' + i) for i in include])
+      open(os.path.abspath(os.path.join(SCRIPT_DIR, '..', '..', '..', "include_imsvc")), "w").writelines([(q('-imsvc' + i) + '\n') for i in include])
       libpath_flags = ' '.join([q('-libpath:' + i) for i in lib])
 
       if (environment_block_name != ''):
