diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index 8c76ac7b1160439b5176c0f745ce44a8e635275e..c64776938de9cac796f6875fc3abca069627b3fc 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -29,6 +29,7 @@ assert(!(incremental_install && !is_java_debug),
 robolectric_toolchain = "//build/toolchain/android:robolectric_$host_cpu"
 
 if (is_android) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/clang/clang.gni")
   import("//build/config/dcheck_always_on.gni")
