diff --git a/build/android/gyp/dex.py b/build/android/gyp/dex.py
index c8495191ad86dc803c56be0e1bce27dd6cc5b9d2..b60e4d62b3211a713143046e6fbedfd858183310 100755
--- a/build/android/gyp/dex.py
+++ b/build/android/gyp/dex.py
@@ -53,7 +53,7 @@ _IGNORE_WARNINGS = (
 _SKIPPED_CLASS_FILE_NAMES = (
     'module-info.class',  # Explicitly skipped by r8/utils/FileUtils#isClassFile
 )
-
+_IGNORE_WARNINGS += ( r'.Brave', )
 
 def _ParseArgs(args):
   args = build_utils.ExpandFileArgs(args)
