diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index 65a35b1f20167702c77dc304416ae6671c217401..9c78269933d7beeedaa918c16e1620e0e438b621 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -274,6 +274,7 @@ def _OptimizeWithR8(options,
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.disable_checks:
       cmd += ['--map-diagnostics:CheckDiscardDiagnostic', 'error', 'none']
