diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index f227c0aacb664eb933920a7c9dffa8c2e4479333..e22258017c556e539beef0fabd4812dd5cff9e3a 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -389,6 +389,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
