diff --git a/build/config/chrome_build.gni b/build/config/chrome_build.gni
index 883c6614d14b1932a6eaea36de5abd54ebe2f9e7..c3d7f0e25afa34d88a2126523637ba5f45ce2d60 100644
--- a/build/config/chrome_build.gni
+++ b/build/config/chrome_build.gni
@@ -1,6 +1,7 @@
 # Copyright 2015 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
+import("//brave/build/config/brave_build.gni")
 
 declare_args() {
   # Select the desired branding flavor. False means normal Chromium branding,
@@ -26,6 +27,7 @@ declare_args() {
   } else {
     branding_path_component = "chromium"
   }
+  if (brave_branding_path_component != "") { branding_path_component = brave_branding_path_component }
 }
 
 declare_args() {
