diff --git a/build/dotfile_settings.gni b/build/dotfile_settings.gni
index cd9991adcf895a2c39a7fccbb1b4dbfbfe1aa572..7a4d7ddecc41b766072b37a194836f29ca577ae7 100644
--- a/build/dotfile_settings.gni
+++ b/build/dotfile_settings.gni
@@ -7,6 +7,7 @@
 
 build_dotfile_settings = {
   exec_script_whitelist = [
+    "//build/config/BUILDCONFIG.gn",
     "//build/config/android/config.gni",
     "//build/config/android/internal_rules.gni",
     "//build/config/android/rules.gni",
