diff --git a/build/config/android/BUILD.gn b/build/config/android/BUILD.gn
index db4025262767d571bc832e973a5d5283e5ea5910..c77608ccb93bf62854a72ce29056f756f45f831e 100644
--- a/build/config/android/BUILD.gn
+++ b/build/config/android/BUILD.gn
@@ -53,6 +53,7 @@ config("compiler") {
     # segments to be mapped adjacently, which breakpad relies on.
     "-Wl,-z,max-page-size=$_max_page_size",
   ]
+  ldflags += ["-Wl,--allow-multiple-definition"] # Resolve conflict between libgcc++ and rustup symbols (e.g. __divdi3)
 
   if (current_cpu == "arm64") {
     if (arm_control_flow_integrity == "standard") {
@@ -63,6 +64,7 @@ config("compiler") {
       rustflags = [ "-Zbranch-protection=pac-ret" ]
     }
   }
+  if ((current_cpu == "arm64" || current_cpu == "x64") && use_thin_lto) { ldflags += ["-Wl,-mllvm", "-Wl,-generate-type-units"] }
 
   # Disable TLSDESC for riscv64 until the Android NDK supports it. While
   # Chromium's clang turns it on by default for Android RISC-V devices, NDK r27
