diff --git a/build/config/android/internal_rules.gni b/build/config/android/internal_rules.gni
index 04dc65115fb7f677bfa36087c1f08c96e44b95b2..7fedf4b0c86497b7f23a523e991097b188551109 100644
--- a/build/config/android/internal_rules.gni
+++ b/build/config/android/internal_rules.gni
@@ -4006,6 +4006,7 @@ if (enable_java_templates) {
           "*/webengine_shell_apk/*",
           "local_modifications/pdf/java/androidx/pdf/viewer/fragment/PdfViewerFragment.kt",
         ]
+        import("//brave/browser/playlist/android/sources.gni") _kt_allowlist += brave_playlist_kt_allowlist
         _found_kt = filter_exclude(_abs_kt_files, _kt_allowlist)
         assert(
             _found_kt == [],
