diff --git a/build/android/bytecode/BUILD.gn b/build/android/bytecode/BUILD.gn
index f9e1baf2604134ca19f50fb228863d076840ddc3..86f3281dae27884b620e716158a36e2596c8e413 100644
--- a/build/android/bytecode/BUILD.gn
+++ b/build/android/bytecode/BUILD.gn
@@ -13,6 +13,7 @@ java_binary("java_bytecode_rewriter") {
     "java/org/chromium/bytecode/ThreadAssertionClassAdapter.java",
     "java/org/chromium/bytecode/TypeUtils.java",
   ]
+  sources += brave_java_bytecode_files
   main_class = "org.chromium.bytecode.ByteCodeProcessor"
   deps = [
     "//third_party/android_deps:org_ow2_asm_asm_java",
