diff --git a/build/rust/BUILD.gn b/build/rust/BUILD.gn
index 17c7efbb7abd2376b2c83656c9dab9d6fc732e05..609dab84217b4d17e1c53ff572c23460431b1f72 100644
--- a/build/rust/BUILD.gn
+++ b/build/rust/BUILD.gn
@@ -28,10 +28,15 @@ if (toolchain_has_rust) {
 
     defines = [ "RUST_CXX_NO_EXCEPTIONS" ]
 
-    if (is_win) {
-      defines += [ "CXX_RS_EXPORT=__declspec(dllexport)" ]
+    # We cannot depend on base/base_export.h because base depends on us.
+    if (is_component_build) {
+      if (is_win) {
+        defines += [ "CXX_RS_EXPORT=__declspec(dllexport)" ]
+      } else {
+        defines += [ "CXX_RS_EXPORT=__attribute__((visibility(\"default\")))" ]
+      }
     } else {
-      defines += [ "CXX_RS_EXPORT=__attribute__((visibility(\"default\")))" ]
+      defines += [ "CXX_RS_EXPORT=" ]
     }
 
     # Depending on the C++ bindings side of cxx then requires also depending
