diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index f5f4e8e11e25fd35054ebcbc23870bff82608e21..faf0ee2b6797eb3efc7718f17600c8064b93def8 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -334,6 +334,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
