diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index af37ee76ac033473aac56c3f1d89516f95325374..eb7ed27f20c836c604fb382f35c473a486943022 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1665,6 +1665,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [
     ":default_warnings",
