diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
index 09e4a28029ac4b6240f35398763d94714a459781..128b73cb4dad4a04ccef904233c594e97d467213 100644
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -479,6 +479,7 @@ if (enable_java_templates) {
   #     ]
   #   }
   template("java_cpp_enum") {
+    foreach(source, invoker.sources) { foreach(filter, brave_java_cpp_enum_filter) { if(source == filter) { invoker.sources -= [filter] }}}
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "sources" ])
 
@@ -650,6 +651,7 @@ if (enable_java_templates) {
   #     output = "$target_gen_dir/AndroidManifest.xml"
   #   }
   template("jinja_template") {
+    if (defined(invoker.includes)) { invoker.includes += brave_android_manifest_includes } else { invoker.includes = brave_android_manifest_includes }
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "deps" ])
       inputs = [ invoker.input ]
