diff --git a/build/rust/std/remap_alloc.cc b/build/rust/std/remap_alloc.cc
index bfd39b724b88a8de963840c617430eb0d3875f03..41c3b7317d5310c70156e5aafca5a30cefa08e0f 100644
--- a/build/rust/std/remap_alloc.cc
+++ b/build/rust/std/remap_alloc.cc
@@ -130,7 +130,15 @@ REMAP_ALLOC_ATTRIBUTES void* __rust_alloc(size_t size, size_t align) {
 }
 
 REMAP_ALLOC_ATTRIBUTES void __rust_dealloc(void* p, size_t size, size_t align) {
+#if defined(COMPILER_MSVC)
+  if (align <= alignof(std::max_align_t)) {
+    free(p);
+  } else {
+    _aligned_free(p);
+  }
+#else
   free(p);
+#endif
 }
 
 REMAP_ALLOC_ATTRIBUTES void* __rust_realloc(void* p,
