diff --git a/build/android/gyp/lint.py b/build/android/gyp/lint.py
index 177981aa62403af3dc6874bc3a2ab2c4064bce29..472fe290d02ed9ca999d4873bc3d1f8ff2b5f16a 100755
--- a/build/android/gyp/lint.py
+++ b/build/android/gyp/lint.py
@@ -519,5 +519,6 @@ def main():
   server_utils.MaybeTouch(args.stamp)
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
   sys.exit(main())
