diff --git a/build/config/android/internal_rules.gni b/build/config/android/internal_rules.gni
index ca50d6a26a70e9b823528c2a46b5a4a5947b9c3d..af1ff58b6ea05fdb6b7859e63223a2f9c8c64a1f 100644
--- a/build/config/android/internal_rules.gni
+++ b/build/config/android/internal_rules.gni
@@ -1981,6 +1981,7 @@ if (enable_java_templates) {
         _deps = [ ":$_desugar_target" ]
         _previous_output_jar = _desugar_output_jar
       }
+      foreach(bytecode_jar, brave_bytecode_jars) { if (rebase_path(invoker.input_jar_path, root_build_dir) == bytecode_jar) { _bytecode_rewriter_output_jar = "$target_out_dir/$target_name.bytecode-rewritten.jar" brave_bytecode_rewriter("${target_name}__bytecode_rewrite"){} _deps = [] _deps = [ ":${target_name}__bytecode_rewrite" ] _previous_output_jar = _bytecode_rewriter_output_jar } }
 
       if (invoker.jacoco_instrument) {
         _filter_jar_target_name = "${target_name}_device__filter_jar"
