diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index 579501c7ded30e0073ee6ac512ed733f5c5b0c48..d0de3af3037bcc8bff4937909b3dcc69df52e219 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -279,6 +279,7 @@ def _OptimizeWithR8(options,
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.disable_checks:
       cmd += ['--map-diagnostics:CheckDiscardDiagnostic', 'error', 'none']
