diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index b2e8f54968d526eb6661a8a88e2a9d37bc509dda..5c4dc42a48d6d5202b70c1467e74a6ee2d63012f 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1712,6 +1712,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
