diff --git a/build/toolchain/rbe.gni b/build/toolchain/rbe.gni
index 78cd9d5c3f5205ebedb6adff6903218e70f6b30c..86f795f702f3be72d28f089175e4386946d74d73 100644
--- a/build/toolchain/rbe.gni
+++ b/build/toolchain/rbe.gni
@@ -138,7 +138,7 @@ declare_args() {
 
 if (use_reclient) {
   rewrapper_path = "${reclient_bin_dir}/rewrapper"
-  if (is_win) {
+  if (host_os == "win") {
     rewrapper_path = "${reclient_bin_dir}/rewrapper.exe"
   }
   assert(path_exists(rewrapper_path),
