diff --git a/build/toolchain/gcc_toolchain.gni b/build/toolchain/gcc_toolchain.gni
index 1d0ac288ab9dff824c7c39c53261a9edfbb0bccc..e34b54182b75850637c622c7d6d5305d8d0af4f8 100644
--- a/build/toolchain/gcc_toolchain.gni
+++ b/build/toolchain/gcc_toolchain.gni
@@ -759,6 +759,15 @@ template("gcc_toolchain") {
     }
 
     forward_variables_from(invoker, [ "deps" ])
+    if (toolchain_cc_wrapper != "" && cc_wrapper_dep != "") {
+      if (get_label_info(":" + target_name, "label_no_toolchain") !=
+          get_label_info(cc_wrapper_dep, "toolchain")) {
+        if (!defined(deps)) {
+          deps = []
+        }
+        deps += [ cc_wrapper_dep ]
+      }
+    }
   }
 }
 
