diff --git a/build/android/gyp/dex.py b/build/android/gyp/dex.py
index 4e447ae519cc8d310edce1223a3638af6970dd21..ad6857bb365a708b65fcaa9193cb9bceec62e870 100755
--- a/build/android/gyp/dex.py
+++ b/build/android/gyp/dex.py
@@ -35,6 +35,7 @@ DEFAULT_IGNORE_WARNINGS = (
     # https://crbug.com/408280256
     r'MethodHandle.invoke',
 )
+DEFAULT_IGNORE_WARNINGS += ( r'.Brave', )
 
 _MERGE_SERVICE_ENTRIES = (
     # Uses ServiceLoader to find all implementing classes, so multiple are
@@ -485,6 +486,7 @@ def MergeDexForIncrementalInstall(r8_jar_path, src_paths, dest_dex_jar,
 def main(args):
   build_utils.InitLogging('DEX_DEBUG')
   options = _ParseArgs(args)
+  options.intermediate = options.intermediate and not any('com_wireguard_android_java' in c for c in options.class_inputs)
 
   options.class_inputs += options.class_inputs_filearg
   options.dex_inputs += options.dex_inputs_filearg
