diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index bc4236a4e7080b005e18bacea92fe1d5d71b5cb1..634363585b59e9ae6665cc4c06357b06b3fb5232 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -365,6 +365,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config/compiler/pgo:default_pgo_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
