diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 4a22dfc9ee6541c3691d7fe0101b2d9ebdfd7a71..9f4de907b8e3aecb3b075eee24d221412194d9c2 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1718,6 +1718,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
