diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 944de32ba1a2c8af12f1cdc9edfcf23e36bfa96d..7680b790c603bded5d94313d09cd57dd6535ce50 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1718,6 +1718,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
