diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 4b24c762b6e9f1542f7d6e276d1d87dbecdc3d46..b421057b0288d141541de9b8d0cc70a25fc8ef1d 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -2124,7 +2124,7 @@ config("default_optimization") {
     # The NaCl IRT is a special case and always wants its own config.
     # It gets optimized the same way regardless of the type of build.
     configs = [ "//build/config/nacl:irt_optimize" ]
-  } else if (is_debug) {
+  } else if (is_debug || !is_official_build) {
     configs = [ ":no_optimize" ]
   } else if (optimize_for_fuzzing) {
     assert(!is_win, "Fuzzing optimize level not supported on Windows")
