diff --git a/build/config/android/BUILD.gn b/build/config/android/BUILD.gn
index 4fc5565e27a596fe4aa1a093cfc0567fbd24b2a0..893b3ed9478b34de60352070259642bae938973b 100644
--- a/build/config/android/BUILD.gn
+++ b/build/config/android/BUILD.gn
@@ -46,6 +46,7 @@ config("compiler") {
     # http://crbug.com/448386
     "-Wl,--exclude-libs=libvpx_assembly_arm.a",
   ]
+  ldflags += ["-Wl,--allow-multiple-definition"] # Resolve conflict between libgcc++ and rustup symbols (e.g. __divdi3)
 
   if (current_cpu == "arm64") {
     if (arm_control_flow_integrity == "standard") {
