diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index c577e8352b9562637fae692704e3e04310eacf75..0a3879c917377dc9a7c387dc3420827a25b6fbd9 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1696,6 +1696,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [
     ":default_warnings",
