diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 9164ee0578686c88fc76ea3c59a7d3e50d15147d..e28a25faf5989baa7d2390b213de5a89c8fbd7bc 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -340,6 +340,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
