diff --git a/buildtools/checkdeps/checkdeps.py b/buildtools/checkdeps/checkdeps.py
index 1019914aac41e9f3a029053bb70a8f5137224734..ddb3575be43eb2b679a133b136a7dc5062942c17 100755
--- a/buildtools/checkdeps/checkdeps.py
+++ b/buildtools/checkdeps/checkdeps.py
@@ -24,7 +24,7 @@ import cpp_checker
 import java_checker
 import results
 
-from builddeps import DepsBuilder
+from buildtools.checkdeps.brave_builddeps import DepsBuilder
 from rules import Rule, Rules
 
 
