diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index 77ccb7b3baf528f6ec18ce2f7fa456bf9587d214..d43fa74aa9fd184f3ba4e9f39e2b4851b1008fc2 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -32,6 +32,7 @@ robolectric_toolchain = "//build/toolchain/android:robolectric_$host_cpu"
 # toolchain, this GNI file may be read and processed from within Chrome OS
 # toolchains. Checking |is_android| here would therefore be too restrictive.
 if (is_android || is_chromeos) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/clang/clang.gni")
   import("//build/config/dcheck_always_on.gni")
