diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 46ab685bf00f5604d24b7001805f7f196e4267fd..528fc35ffb74bc93473798b8f9e724dd103a50c9 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1622,6 +1622,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags

   configs = [ ":default_warnings" ]
 }
