diff --git a/build/android/gyp/dex.py b/build/android/gyp/dex.py
index a7f024a277d567335091af8c5f90eedaedd0594f..71d47c73906e68b55828614bd798a26fe3f53b5f 100755
--- a/build/android/gyp/dex.py
+++ b/build/android/gyp/dex.py
@@ -63,7 +63,7 @@ _IGNORE_WARNINGS = (
 _SKIPPED_CLASS_FILE_NAMES = (
     'module-info.class',  # Explicitly skipped by r8/utils/FileUtils#isClassFile
 )
-
+_IGNORE_WARNINGS += ( r'.Brave', )
 
 def _ParseArgs(args):
   args = build_utils.ExpandFileArgs(args)
