diff --git a/build/config/chrome_build.gni b/build/config/chrome_build.gni
index 2207912e73af6a792abefd7ade4b5c36199387d8..e339972df8b6a2e9d4daf67a32be9daa237b5cda 100644
--- a/build/config/chrome_build.gni
+++ b/build/config/chrome_build.gni
@@ -1,6 +1,7 @@
 # Copyright 2015 The Chromium Authors
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
+import("//brave/build/config/brave_build.gni")
 
 declare_args() {
   # Select the desired branding flavor. False means normal Chromium branding,
