diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index 8b8cf6bc8f8a449cf70b66d37fa74f8599b9abda..053738075fd628e36b0a7b5795fa43f4da94e595 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -329,6 +329,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
