diff --git a/build/android/gyp/lint.py b/build/android/gyp/lint.py
index 2d2c60d521ca218b4ea9cde7f1617ae4a323f628..d09059fff25c7992996378b91b07f6a4c75c434c 100755
--- a/build/android/gyp/lint.py
+++ b/build/android/gyp/lint.py
@@ -454,5 +454,6 @@ def main():
   server_utils.MaybeTouch(args.stamp)
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
   sys.exit(main())
