diff --git a/build/config/win/BUILD.gn b/build/config/win/BUILD.gn
index de1c0f19002621046387cd7b78d7ccdd968bd84e..7237692a3a8b3248164c2302eb82be1d56c9318c 100644
--- a/build/config/win/BUILD.gn
+++ b/build/config/win/BUILD.gn
@@ -177,6 +177,7 @@ config("compiler") {
     ldflags += [ "/pdbpagesize:16384" ]
   } else {
     ldflags += [ "/pdbpagesize:8192" ]
+    import("//brave/build/config/win/config.gni") if (brave_win_use_large_pdb) { ldflags -= [ "/pdbpagesize:8192" ] ldflags += [ "/pdbpagesize:16384" ] }
   }
 
   if (!is_debug && !is_component_build) {
