diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
index 71cd964ce9c9968410426e033f70bb5169f97824..4433726b4a5bfef828317963e53f0027ff0dadd7 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
@@ -229,6 +229,7 @@ void BookmarkBubbleView::ShowBubble(
   bookmark_bubble_ = bubble.get();
   bubble->SetAnchorView(anchor_view);
   bubble->SetArrow(views::BubbleBorder::TOP_RIGHT);
+  BRAVE_BOOKMARK_BUBBLE_VIEW
   if (highlighted_button)
     bubble->SetHighlightedButton(highlighted_button);
 
