diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 2cc88304b36202e3baa0f2a22728d0ae973f15c4..8fde03a0b7722b92518a2f263b9c846aa9759b90 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -1263,6 +1263,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
