diff --git a/chrome/browser/lifetime/BUILD.gn b/chrome/browser/lifetime/BUILD.gn
index e42b9c589d1d0687a79b6c638efa0da59f797b49..931340ac8423a7dc48157d4e78099d910dd50c4e 100644
--- a/chrome/browser/lifetime/BUILD.gn
+++ b/chrome/browser/lifetime/BUILD.gn
@@ -131,6 +131,7 @@ source_set("impl") {
   }
 
   if (!is_android) {
+    import("//brave/chromium_src/chrome/browser/lifetime/sources.gni") deps += brave_chrome_browser_lifetime_impl_deps
     sources += [
       "application_lifetime_desktop.cc",
       "browser_close_manager.cc",
