diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java
index 6dc5ea9b7e3b3d26537fe4b94c9258a5124381ae..4e1d8af87bcad1bca184f06016b6a5b4f9c8d86d 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java
@@ -73,7 +73,7 @@ public class AndroidSyncSettings {
     public static AndroidSyncSettings get() {
         ThreadUtils.assertOnUiThread();
         if (sInstance == null) {
-            sInstance = new AndroidSyncSettings(getSyncAccount());
+            sInstance = new BraveAndroidSyncSettings(getSyncAccount());
         }
         return sInstance;
     }
