diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index f5816070e5999f959ca4f522d86d283932fd02ed..0e4bc09a60d76897e634b00097ce1def55b3785a 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -1798,6 +1798,7 @@ split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_android) {
