diff --git a/chrome/test/data/webui/settings/BUILD.gn b/chrome/test/data/webui/settings/BUILD.gn
index 83bcffe17c565091da8b41775f643781232a8621..22ea18b436418dbd30920071895928bb8c30b454 100644
--- a/chrome/test/data/webui/settings/BUILD.gn
+++ b/chrome/test/data/webui/settings/BUILD.gn
@@ -248,6 +248,7 @@ build_webui_tests("build") {
     ts_definitions +=
         [ "//tools/typescript/definitions/quick_unlock_private.d.ts" ]
   }
+  import("//brave/browser/resources/settings/sources.gni") ts_definitions += brave_settings_ts_definitions
 
   ts_deps = [
     "//chrome/browser/resources/settings:build_ts",
