diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index b1c488cfce337ffc1fd6a6019d4af12ef518b493..a8e8fda55f06d695d1867a21d3cb2bb54e719f97 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -35,6 +35,7 @@ if (optimize_webui) {
 
     deps = [
       ":unpak",
+      "//brave/browser/resources:unpak",
     ]
   }
 
