diff --git a/chrome/browser/supervised_user/supervised_user_settings_service_factory.cc b/chrome/browser/supervised_user/supervised_user_settings_service_factory.cc
index 5f85a90f2f2800c00bf0367c1de83ff36d8b8fe7..9ab88b8a25b36afdbe3016fd20cc29343395e634 100644
--- a/chrome/browser/supervised_user/supervised_user_settings_service_factory.cc
+++ b/chrome/browser/supervised_user/supervised_user_settings_service_factory.cc
@@ -36,6 +36,7 @@ SupervisedUserSettingsServiceFactory::BuildServiceInstanceFor(
 
 SimpleFactoryKey* SupervisedUserSettingsServiceFactory::GetKeyToUse(
     SimpleFactoryKey* key) const {
+  BRAVE_GET_KEY_TO_USE
   ProfileKey* profile_key = ProfileKey::FromSimpleFactoryKey(key);
   return profile_key->GetOriginalKey();
 }
