diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index f8dbb8bd1c0bd44e759f104e3e6dfb689c95c213..b3e2e18d037039c3df478e6219f3560f58760872 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -34,7 +34,9 @@ function addPrivacyChildRoutes(r) {
 
   // TODO(tommycli): Find a way to refactor these repetitive category
   // routes.
+      // <if expr="_google_chrome">
   r.SITE_SETTINGS_ADS = r.SITE_SETTINGS.createChild('ads');
+      // </if>
   r.SITE_SETTINGS_AR = r.SITE_SETTINGS.createChild('ar');
   r.SITE_SETTINGS_AUTOMATIC_DOWNLOADS =
       r.SITE_SETTINGS.createChild('automaticDownloads');
@@ -176,7 +178,9 @@ function createBrowserSettingsRoutes() {
     }
 
     r.PRINTING = r.ADVANCED.createSection('/printing', 'printing');
+      // <if expr="_google_chrome">
     r.CLOUD_PRINTERS = r.PRINTING.createChild('/cloudPrinters');
+      // </if>
 
     r.ACCESSIBILITY = r.ADVANCED.createSection('/accessibility', 'a11y');
 
