diff --git a/chrome/browser/ui/webui/signin/BUILD.gn b/chrome/browser/ui/webui/signin/BUILD.gn
index 4e165b0f5af4d35f7d3be5b578947cfc334a7a19..e7a960506cc5736c309af03cc7c147700c595113 100644
--- a/chrome/browser/ui/webui/signin/BUILD.gn
+++ b/chrome/browser/ui/webui/signin/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/ui/webui/custom_profile_image/buildflags/buildflags.gni")
 import("//chrome/common/features.gni")
 import("//components/signin/features.gni")
 
@@ -482,6 +483,13 @@ if (!is_chromeos) {
     if (is_win || is_mac || is_linux) {
       deps += [ "//chrome/browser/ui/views/profiles:profile_management_types" ]
     }
+    deps += [ "//brave/ui/webui/custom_profile_image/buildflags" ]
+    if (enable_brave_custom_profile_image_webui) {
+      deps += [
+        "//brave/browser/ui/webui/custom_profile_image",
+        "//brave/app:brave_generated_resources_grit",
+      ]
+    }
   }
 }
 
