diff --git a/chrome/installer/util/prebuild/create_installer_string_rc.py b/chrome/installer/util/prebuild/create_installer_string_rc.py
index 9b174616ccee685ce6e672084795a25b8f067426..509876a90b75a18cb9155abecbc5990940725488 100755
--- a/chrome/installer/util/prebuild/create_installer_string_rc.py
+++ b/chrome/installer/util/prebuild/create_installer_string_rc.py
@@ -104,3 +104,5 @@ MODE_SPECIFIC_STRINGS = {
     ],
   },
 }
+
+from chrome.installer.util.prebuild.create_installer_string_rc import MODE_SPECIFIC_STRINGS
