diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index a88c4e6c1947ad5e813325837e80d438ae94ec3c..88dd0b2aa22e4a8cb665535b17f45146ac2772dc 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -120,6 +120,7 @@ source_set("internal") {
       deps += [ "//chrome/browser/ui:overscroll_pref_manager" ]
     }
   }
+  deps += [ "//brave/browser/ui/browser_window/internal" ]
 }
 
 source_set("unit_tests") {
