diff --git a/chrome/browser/extensions/component_loader.h b/chrome/browser/extensions/component_loader.h
index 59dea62f37096618eec5134de92d7eeb90db7006..804048aa2c5d80b71ef5ff27320bcd0bd1a281da 100644
--- a/chrome/browser/extensions/component_loader.h
+++ b/chrome/browser/extensions/component_loader.h
@@ -81,7 +81,7 @@ class ComponentLoader {
   // the loader will skip loading component extensions that weren't supposed to
   // be loaded unless we are in signed user session (ChromeOS). For all other
   // platforms this |skip_session_components| is expected to be unset.
-  void AddDefaultComponentExtensions(bool skip_session_components);
+  virtual void AddDefaultComponentExtensions(bool skip_session_components);
 
   // Similar to above but adds the default component extensions for kiosk mode.
   void AddDefaultComponentExtensionsForKioskMode(bool skip_session_components);
@@ -116,6 +116,7 @@ class ComponentLoader {
   }
 
  private:
+  friend class BraveComponentLoader;
   FRIEND_TEST_ALL_PREFIXES(ComponentLoaderTest, ParseManifest);
 
   // Information about a registered component extension.
@@ -161,6 +162,7 @@ class ComponentLoader {
   void AddDefaultComponentExtensionsWithBackgroundPagesForKioskMode();
 
 #if BUILDFLAG(ENABLE_HANGOUT_SERVICES_EXTENSION)
+  virtual
   void AddHangoutServicesExtension();
 #endif  // BUILDFLAG(ENABLE_HANGOUT_SERVICES_EXTENSION)
 
