diff --git a/chrome/renderer/content_settings_observer.h b/chrome/renderer/content_settings_observer.h
index 0a0e420dfc70143c63144f61157bc673791dee5f..0a01eced5df36d25361704b93edddf6816504001 100644
--- a/chrome/renderer/content_settings_observer.h
+++ b/chrome/renderer/content_settings_observer.h
@@ -105,6 +105,7 @@ class ContentSettingsObserver
   }
 
  private:
+  friend class BraveContentSettingsObserver;
   FRIEND_TEST_ALL_PREFIXES(ContentSettingsObserverTest, WhitelistedSchemes);
   FRIEND_TEST_ALL_PREFIXES(ContentSettingsObserverBrowserTest,
                            ContentSettingsInterstitialPages);
