diff --git a/chrome/browser/ui/chrome_pages.cc b/chrome/browser/ui/chrome_pages.cc
index 57f5b56223003be5567b22048b68c780d302ce0d..0c497bfe6687416d4c74ba7a8f347135b0bc5fdc 100644
--- a/chrome/browser/ui/chrome_pages.cc
+++ b/chrome/browser/ui/chrome_pages.cc
@@ -305,7 +305,7 @@ void ShowSlow(Browser* browser) {
 #endif
 }
 
-GURL GetSettingsUrl(const std::string& sub_page) {
+BRAVE_GET_SETTINGS_URL
   return GURL(std::string(kChromeUISettingsURL) + sub_page);
 }
 
