diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 0c5467af2bce341e0b3f0835899c820f60adbc87..8d6d326588be988f2cc8c90f254559b7c38c4583 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -31,7 +31,7 @@ import("java_sources.gni")
 import("static_initializers.gni")
 import("trichrome.gni")
 
-chrome_public_manifest_package = "org.chromium.chrome"
+declare_args() { chrome_public_manifest_package = "org.chromium.chrome" }
 chrome_public_test_manifest_package = "org.chromium.chrome.tests"
 
 chrome_public_jinja_variables =
