diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index 143e7442ca050594dfb51a8e2fb5eb662a3f24c0..b68aeb1e147898eb013e5444009b1030683abc94 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -205,6 +205,7 @@ class BrowserProcessImpl : public BrowserProcess,
   std::string actual_locale();
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
