diff --git a/chrome/browser/tab_ui/android/BUILD.gn b/chrome/browser/tab_ui/android/BUILD.gn
index d1c8aa091948593b7ee9ed6e091485ad3e14b363..abcc04a268c9a76ddb68ffad5d76e30a14b5fc0d 100644
--- a/chrome/browser/tab_ui/android/BUILD.gn
+++ b/chrome/browser/tab_ui/android/BUILD.gn
@@ -36,6 +36,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/tab_ui/TabThumbnailView.java",
     "java/src/org/chromium/chrome/browser/tab_ui/ThumbnailProvider.java",
   ]
+  sources += brave_tab_ui_sources
   deps = [
     ":java_resources",
     "//base:base_java",
