diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 10e4e2021cb39f2384f81d7ed8783e85286fa37b..cf7ac291b4305411f6a9545894aa5d6e3b0fac14 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -720,6 +720,7 @@ class Browser : public TabStripModelObserver,
 
   StatusBubble* GetStatusBubbleForTesting();
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
