diff --git a/chrome/browser/renderer_context_menu/render_view_context_menu.cc b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
index f115804eba19b2762efafa473769ecf3e28d9276..1c0ea00719e23f006ed7d94b427be541707396e9 100644
--- a/chrome/browser/renderer_context_menu/render_view_context_menu.cc
+++ b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
@@ -1064,6 +1064,7 @@ void RenderViewContextMenu::AppendLinkItems() {
                                       IDS_CONTENT_CONTEXT_OPENLINKOFFTHERECORD);
     }
 
+    AppendBraveLinkItems();
     AppendOpenWithLinkItems();
 
     // While ChromeOS supports multiple profiles, only one can be open at a
