diff --git a/chrome/browser/net/proxy_config_monitor.h b/chrome/browser/net/proxy_config_monitor.h
index 89784b4d6eab9938c475abb6b58c3e08e8801371..927be1e7cb7df2a4742eb918082d79dce5b76696 100644
--- a/chrome/browser/net/proxy_config_monitor.h
+++ b/chrome/browser/net/proxy_config_monitor.h
@@ -66,7 +66,7 @@ class ProxyConfigMonitor : public net::ProxyConfigService::Observer,
   // they're received, to allow tests to wait until all pending proxy
   // configuration changes have been applied.
   void FlushForTesting();
-
+  BRAVE_PROXY_CONFIG_MONITOR_H
  private:
   // net::ProxyConfigService::Observer implementation:
   void OnProxyConfigChanged(
