diff --git a/chrome/browser/extensions/api/developer_private/extension_info_generator.cc b/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
index 3905ec19874e708bfe4fa889b64050173e3595cf..f61cc0f405d43931342642a226f80e6333493dbf 100644
--- a/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
+++ b/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
@@ -710,6 +710,7 @@ void ExtensionInfoGenerator::FillExtensionInfo(const Extension& extension,
 #else
   info.incognito_access_pending_change = false;
 #endif
+  BRAVE_CREATE_EXTENSION_INFO_HELPER
 
   // User Scripts toggle.
   info.user_scripts_access.is_enabled =
