diff --git a/chrome/browser/ui/views/frame/tab_strip_region_view.cc b/chrome/browser/ui/views/frame/tab_strip_region_view.cc
index 41abe63a36f4eb8ba7f98006772e1fbbbc10a911..b1e700a40b36a3278d574d4e68ceecd2a51ca0a3 100644
--- a/chrome/browser/ui/views/frame/tab_strip_region_view.cc
+++ b/chrome/browser/ui/views/frame/tab_strip_region_view.cc
@@ -119,7 +119,7 @@ std::unique_ptr<TabStrip> CreateTabStrip(BrowserView* browser_view) {
       std::move(tab_menu_model_factory));
   BrowserTabStripController* tabstrip_controller_ptr =
       tabstrip_controller.get();
-  auto tab_strip = std::make_unique<TabStrip>(std::move(tabstrip_controller));
+  BRAVE_TAB_STRIP_REGION_VIEW_CREATE_TAB_STRIP
   tabstrip_controller_ptr->InitFromModel(tab_strip.get());
   return tab_strip;
 }
