diff --git a/chrome/android/java/src/org/chromium/chrome/browser/browsing_data/ClearBrowsingDataFragment.java b/chrome/android/java/src/org/chromium/chrome/browser/browsing_data/ClearBrowsingDataFragment.java
index 85ec3b34682d36e773d2d20181c10f419e0f803b..23ab3fa63e8d1840bfcfdafce6b9524d68cdac73 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/browsing_data/ClearBrowsingDataFragment.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/browsing_data/ClearBrowsingDataFragment.java
@@ -356,7 +356,7 @@ public class ClearBrowsingDataFragment extends ChromeBaseSettingsFragment
     }
 
     /** Called when browsing data is about to be cleared. */
-    private void onClearBrowsingData() {
+    protected void onClearBrowsingData() {
         RecordUserAction.record("ClearBrowsingData_AdvancedTab");
     }
 
