diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 589d635d17884050b20b491d65833b7893f5e3e6..114f7a9a9b62f0efcc14271041b670a88988294c 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2509,6 +2509,7 @@ static_library("browser") {
   } else {
     sources += [ "net/net_error_diagnostics_dialog_stub.cc" ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
@@ -8448,6 +8449,7 @@ static_library("browser_public_dependencies") {
       "//rlz:rlz_lib",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
