diff --git a/chrome/browser/importer/external_process_importer_host.h b/chrome/browser/importer/external_process_importer_host.h
index d6d2ad0e41581c001d1364a9833ca46dee0d0668..171ac72ca757a9cc55b23afda22a467d88d7af0b 100644
--- a/chrome/browser/importer/external_process_importer_host.h
+++ b/chrome/browser/importer/external_process_importer_host.h
@@ -72,6 +72,7 @@ class ExternalProcessImporterHost
   void NotifyImportItemEnded(importer::ImportItem item);
   void NotifyImportEnded();
 
+  BRAVE_EXTERNAL_PROCESS_IMPORTER_HOST_H
  private:
   // ExternalProcessImporterHost deletes itself in OnImportEnded().
   ~ExternalProcessImporterHost() override;
