diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index 5eb8c871506a8264c8a5110c0eab4c162933e9c5..b5a8fb36240e0aeb63fcbedca3618119e6bf6b77 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -346,6 +346,8 @@ content::WebUIDataSource* CreateExtensionsSource(Profile* profile,
       kEnableEnhancedSiteControls,
       base::FeatureList::IsEnabled(features::kExtensionsMenuAccessControl));
 
+  BraveAddExtensionsResources(source);
+
   return source;
 }
 
