diff --git a/chrome/browser/net/net_error_diagnostics_dialog_posix.cc b/chrome/browser/net/net_error_diagnostics_dialog_posix.cc
index e2866f851389a1e645b133419834bab26c5f68dc..2c21794a89878b5921c8fe8cf0bc32620946aabe 100644
--- a/chrome/browser/net/net_error_diagnostics_dialog_posix.cc
+++ b/chrome/browser/net/net_error_diagnostics_dialog_posix.cc
@@ -6,7 +6,7 @@
 
 #include "base/logging.h"
 
-bool CanShowNetworkDiagnosticsDialog() {
+bool CanShowNetworkDiagnosticsDialog(content::WebContents* web_contents) {
   return false;
 }
 
