diff --git a/chrome/browser/ui/views/frame/browser_view.cc b/chrome/browser/ui/views/frame/browser_view.cc
index c56cb845088fb2c5f8f3ed0587f6ebb5d31aab75..093c1a7562505ed671b7709a3a80f01e13ec328d 100644
--- a/chrome/browser/ui/views/frame/browser_view.cc
+++ b/chrome/browser/ui/views/frame/browser_view.cc
@@ -639,6 +639,7 @@ bool ConvertedHitTest(views::View* src, views::View* dst, gfx::Point* point) {
   DCHECK(src);
   DCHECK(dst);
   DCHECK(point);
+  BRAVE_BROWSER_VIEW_LAYOUT_CONVERTED_HIT_TEST
   views::View::ConvertPointToTarget(src, dst, point);
   return dst->HitTestPoint(*point);
 }
@@ -1959,6 +1960,7 @@ void BrowserView::OnActiveTabChanged(content::WebContents* old_contents,
       !browser_->tab_strip_model()->closing_all() && GetWidget()->IsActive() &&
       GetWidget()->IsVisible() && !tab_change_in_split_view;
 #endif
+  BRAVE_BROWSER_VIEW_ON_ACTIVE_TAB_CHANGED
   // Update various elements that are interested in knowing the current
   // WebContents.
 
