diff --git a/chrome/browser/file_select_helper.h b/chrome/browser/file_select_helper.h
index d61db71bcfcff81600555523cc4906bb3dd5b14b..5d9382d090be47db441d56fe9146c6a4c1425bc3 100644
--- a/chrome/browser/file_select_helper.h
+++ b/chrome/browser/file_select_helper.h
@@ -79,6 +79,9 @@ class FileSelectHelper : public base::RefCountedThreadSafe<
       scoped_refptr<content::FileSelectListener> listener,
       const base::FilePath& path);
 
+ protected:
+  bool image_metadata_processed_ = false;
+
  private:
   friend class base::RefCountedThreadSafe<FileSelectHelper>;
   friend class base::DeleteHelper<FileSelectHelper>;
