diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 430ba501b49cc60b103219a54517e6d3e9b937ce..faca667fe68e858061ba82892e0f4b5b6b30fc39 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -868,6 +868,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
