diff --git a/chrome/browser/resources/settings/about_page/about_page.js b/chrome/browser/resources/settings/about_page/about_page.js
index b51b9fd58d656edda4d93406724a4bd0c8d67ce5..66a603199251ff23376c954e5adcbc3a929b3e86 100644
--- a/chrome/browser/resources/settings/about_page/about_page.js
+++ b/chrome/browser/resources/settings/about_page/about_page.js
@@ -306,7 +306,14 @@ Polymer({
 
   /** @private */
   onRelaunchTap_: function() {
+    // <if expr="is_macosx">
+    // Sparkle framework's relaunch api is used.
+    this.lifetimeBrowserProxy_.relaunchOnMac();
+    // </if>
+
+    // <if expr="not is_macosx">
     this.lifetimeBrowserProxy_.relaunch();
+    // </if>
   },
 
   /** @private */
