diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index 8d33a8d9447f8dfde5dfea6a144d65773f0b00e0..4ec6f8f71160e063698180f77c05c0a48f116f75 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -379,7 +379,7 @@ const char kGlicActorUiDebounceTimerName[] = "glic-actor-ui-debounce-timer";
 
 // Controls whether the task icon in the actor ui is enabled.
 const base::FeatureParam<bool> kGlicActorUiTaskIcon{
-    &kGlicActorUi, kGlicActorUiTaskIconName, true};
+    &kGlicActorUi, kGlicActorUiTaskIconName, false};
 // Controls whether the Actor Overlay in the actor ui is enabled.
 const base::FeatureParam<bool> kGlicActorUiOverlay{
     &kGlicActorUi, kGlicActorUiOverlayName, true};
