diff --git a/chrome/app/theme/BUILD.gn b/chrome/app/theme/BUILD.gn
index 361f09f7fd3aaf2b391dac51b3f2f85c4277e4b0..b1a95240477d8ff1d0f41670536814e8fe581503 100644
--- a/chrome/app/theme/BUILD.gn
+++ b/chrome/app/theme/BUILD.gn
@@ -70,6 +70,7 @@ grit("webui_theme_resources") {
     ":chrome_unscaled_resources",
     "//ui/resources",
   ]
+  if (!is_android) { deps += ["//brave/app/theme:brave_theme_resources"] }
 
   output_dir = "$root_gen_dir/chrome"
 }
@@ -81,6 +82,7 @@ grit("chrome_unscaled_resources") {
     "grit/chrome_unscaled_resources.h",
     "chrome_unscaled_resources.pak",
   ]
+  deps = [] if (!is_android) { deps += ["//brave/app/theme:brave_unscaled_resources"] }
 
   output_dir = "$root_gen_dir/chrome"
 }
