diff --git a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
index de88201232782323b43be112a5a4170e17fe043e..20e5d90722612d42c70b156206ed469465a8837b 100644
--- a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
+++ b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
@@ -616,6 +616,7 @@ SkPath IconLabelBubbleView::GetHighlightPath() const {
   const float corner_radius = highlight_bounds.height() / 2.f;
   const SkRect rect = RectToSkRect(highlight_bounds);
 
+  BRAVE_ICON_LABEL_BUBBLE_VIEW_GET_HIGHLIGHT_PATH
   return SkPath().addRoundRect(rect, corner_radius, corner_radius);
 }
 
