diff --git a/chrome/browser/ui/views/page_info/page_info_bubble_view.cc b/chrome/browser/ui/views/page_info/page_info_bubble_view.cc
index 4afea8380a67f02e76e5f5c255645ab83d0d2a6a..9105d58538d4ed6d33e8bf79227a2e647b15f74a 100644
--- a/chrome/browser/ui/views/page_info/page_info_bubble_view.cc
+++ b/chrome/browser/ui/views/page_info/page_info_bubble_view.cc
@@ -256,7 +256,7 @@ views::BubbleDialogDelegateView* PageInfoBubbleView::CreatePageInfoBubble(
                                           web_contents, url);
   }
 
-  PageInfoBubbleView* const bubble = new PageInfoBubbleView(
+  PageInfoBubbleView* const bubble = new BravePageInfoBubbleView(
       anchor, anchor_rect, parent_view, web_contents, url,
       specification->initialized_callback(),
       specification->page_info_closing_callback(),
