diff --git a/chrome/installer/mini_installer/BUILD.gn b/chrome/installer/mini_installer/BUILD.gn
index 52905752585a955297754821bf07c9e4fc79c79c..a663582adf50a294b55f6fb20ec82a79e601e657 100644
--- a/chrome/installer/mini_installer/BUILD.gn
+++ b/chrome/installer/mini_installer/BUILD.gn
@@ -137,7 +137,7 @@ if (enable_uncompressed_archive) {
       "$target_gen_dir/mini_installer/packed_files_uncompressed.rc"
 }
 
-action("mini_installer_archive") {
+import("//brave/chromium_src/chrome/installer/mini_installer/build.gni") brave_mini_installer_archive_action("mini_installer_archive") {
   script = "//chrome/tools/build/win/create_installer_archive.py"
 
   release_file = "chrome.release"
@@ -312,6 +312,7 @@ template("generate_mini_installer") {
       no_default_deps = true
       ldflags += [ "/ENTRY:MainEntryPoint" ]
     }
+if (defined(invoker.extra_deps)) { deps += invoker.extra_deps } }} generate_mini_installer("delta_installer_unsigned") { rc_file = "$target_gen_dir/packed_delta_files.rc" extra_deps = [ "//brave:delta_installer_archive" ] if (false) {
   }
 }
 
