diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index a41af3bd5160288bc1f3bba2a2f38a704c377990..1e15b68b6e3f517d6d742a3694e6f931a820b0b3 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -156,6 +156,7 @@ class AvatarToolbarButton : public ToolbarButton {
   CreateScopedZeroDelayOverrideSigninPendingTextForTesting();
 
  private:
+  BRAVE_AVATAR_BUTTON_H
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
