diff --git a/chrome/browser/resources/certificate_manager/certificate_manager.ts b/chrome/browser/resources/certificate_manager/certificate_manager.ts
index 36f29e1137ebb11e3cf9e6a28e25b8f87a2383ab..8a3533cabaf4b9f5ba33f0dc3f6af6de636e5285 100644
--- a/chrome/browser/resources/certificate_manager/certificate_manager.ts
+++ b/chrome/browser/resources/certificate_manager/certificate_manager.ts
@@ -537,4 +537,4 @@ declare global {
   }
 }
 
-customElements.define(CertificateManagerElement.is, CertificateManagerElement);
+// customElements.define(CertificateManagerElement.is, CertificateManagerElement);
