diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index b514273518b656712b49f936bcf7d51f70ecb213..615e19f3628f24af588e4d57f167a5f5a3db730c 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -5132,6 +5132,7 @@ const FeatureEntry::FeatureVariation kContextualTaskEntryPointVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
