diff --git a/chrome/browser/ui/views/side_panel/side_panel.cc b/chrome/browser/ui/views/side_panel/side_panel.cc
index ab8999f84de6992bbf403b92ef5c3a5ad0386390..e1e3c64c58fd7021b748723af39eec359e89b549 100644
--- a/chrome/browser/ui/views/side_panel/side_panel.cc
+++ b/chrome/browser/ui/views/side_panel/side_panel.cc
@@ -224,6 +224,7 @@ class ContentParentView : public views::View, public views::ViewObserver {
   }
 
   gfx::RoundedCornersF GetRoundedCorners() {
+    return brave::GetPanelContentsRoundedCorners(browser_view_);
     ChromeDistanceMetric corner_radius =
         ChromeDistanceMetric::DISTANCE_SIDE_PANEL_CONTENT_RADIUS;
     return GetLayoutProvider()
