diff --git a/chrome/browser/extensions/chrome_component_extension_resource_manager.h b/chrome/browser/extensions/chrome_component_extension_resource_manager.h
index f270d14deb8f7cb592d50b39e3f5b436424f950e..ad35aaa598ea1140c758021b07eb4cb612cd4e00 100644
--- a/chrome/browser/extensions/chrome_component_extension_resource_manager.h
+++ b/chrome/browser/extensions/chrome_component_extension_resource_manager.h
@@ -24,6 +24,7 @@ class ChromeComponentExtensionResourceManager
                                     int* resource_id) const override;
   const ui::TemplateReplacements* GetTemplateReplacementsForExtension(
       const std::string& extension_id) const override;
+  BRAVE_CHROME_COMPONENT_EXTENSION_RESOURCE_MANAGER_H_
 
  private:
   class Data;
