diff --git a/chrome/browser/resources/settings/people_page/sync_controls.ts b/chrome/browser/resources/settings/people_page/sync_controls.ts
index a1dc3e64ff30111b1a241d46ed5a571d3e393421..990136ad26a3a195873e89cbeedc623c52e5b9a7 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.ts
+++ b/chrome/browser/resources/settings/people_page/sync_controls.ts
@@ -144,7 +144,7 @@ export class SettingsSyncControlsElement extends
         'sync-prefs-changed', this.handleSyncPrefsChanged_.bind(this));
 
     const router = Router.getInstance();
-    if (router.getCurrentRoute() === routes.SYNC_ADVANCED) {
+    if (router.getCurrentRoute() === routes.BRAVE_SYNC_SETUP) {
       this.browserProxy_.didNavigateToSyncPage();
     }
   }
@@ -270,6 +270,7 @@ export class SettingsSyncControlsElement extends
   private disableTypeCheckBox_(
       syncStatus: SyncStatus, syncAllDataTypes: boolean,
       dataTypeManaged: boolean): boolean {
+    if (dataTypeManaged) { return true; } if (!syncStatus || !this.syncPrefs) { return syncAllDataTypes }
     if (!syncStatus) {
       return true;
     }
