diff --git a/chrome/browser/profiles/profile_impl.cc b/chrome/browser/profiles/profile_impl.cc
index 98c0522cc5e9be7eb2a1d8ee9712d9347fd7ecc8..7d259a217a0fcfd4c5973e16488b0fa711ec5cd7 100644
--- a/chrome/browser/profiles/profile_impl.cc
+++ b/chrome/browser/profiles/profile_impl.cc
@@ -362,7 +362,7 @@ std::unique_ptr<Profile> Profile::CreateProfile(const base::FilePath& path,
     NOTREACHED();
   }
 
-  std::unique_ptr<Profile> profile = base::WrapUnique(new ProfileImpl(
+  std::unique_ptr<Profile> profile = base::WrapUnique(new BraveProfileImpl(
       path, delegate, create_mode, creation_time, io_task_runner));
   return profile;
 }
