diff --git a/chrome/browser/bookmarks/android/BUILD.gn b/chrome/browser/bookmarks/android/BUILD.gn
index d406d951b82d7b3c216ab632cd0e9765b7765a44..c3b04c3b8bc698bf863811a27b3bd76dca104f54 100644
--- a/chrome/browser/bookmarks/android/BUILD.gn
+++ b/chrome/browser/bookmarks/android/BUILD.gn
@@ -38,6 +38,7 @@ source_set("impl") {
     "//components/query_parser",
     "//components/undo",
   ]
+  deps += brave_browser_bookmarks_android_impl_deps
 }
 
 android_resources("java_resources") {
@@ -122,6 +123,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/bookmarks/bar/BookmarkBarViewBinder.java",
     "java/src/org/chromium/chrome/browser/bookmarks/bar/BookmarkBarVisibilityProvider.java",
   ]
+  sources += brave_browser_bookmarks_android_sources
 
   deps = [
     ":java_resources",
