diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 80c69ea700c9804d69bd7daa9ff6df80aae2eb1d..e6b7312b517e8a0ed10cc8e2e41ed2024c194349 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -2976,6 +2976,7 @@ constexpr FeatureEntry::FeatureVariation kLensStandaloneVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
