diff --git a/chrome/browser/sync/sync_service_factory.cc b/chrome/browser/sync/sync_service_factory.cc
index 64323bef78dd3822ed299f75f5d7bbcfc636c46c..850d4433366fbe43268cc5c96a9c07e2e667ee3d 100644
--- a/chrome/browser/sync/sync_service_factory.cc
+++ b/chrome/browser/sync/sync_service_factory.cc
@@ -303,6 +303,7 @@ syncer::DataTypeController::TypeVector CreateCommonControllers(
   builder.SetNotebooksService(
       notebooks::NotebooksServiceFactory::GetForProfile(profile));
 
+  BRAVE_COMMON_CONTROLLER_BUILDER_SET_SERVICES
   return builder.Build(/*disabled_types=*/{}, sync_service,
                        chrome::GetChannel());
 }
@@ -472,7 +473,7 @@ std::unique_ptr<KeyedService> BuildSyncService(
   }
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize(CreateControllers(profile, sync_service.get()));
 
   // Notify the PasswordStore of complete initialisation to resolve a circular
