diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index b58905ddb371f96377320c2948fe64bb17257663..f5fd9c4f920348caebd681e33503f5d0efd14eca 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/sources.gni")
 import("//base/allocator/allocator.gni")
 import("//build/buildflag_header.gni")
 import("//build/config/buildflags_paint_preview.gni")
@@ -2014,6 +2015,7 @@ source_set("browser") {
     # should be added to browser_generated_files.
     ":browser_public_dependencies",
     ":core",
+    "//brave/browser:core",
     "//chrome/browser/accessibility/accessibility_prefs",
     "//chrome/browser/browsing_data",
     "//chrome/browser/download",
@@ -3441,6 +3443,7 @@ group("browser_public_dependencies") {
     "//sql",
     "//ui/lottie",
   ]
+  public_deps += brave_chrome_browser_public_deps
   if (is_android) {
     public_deps += [
       "//chrome/android/features/dev_ui:buildflags",
