diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java
index 97ffd7bd67ab326fd71925ccc3eac76a2b69b04d..bf120b4ef9d90c24984bb38fa94e5fd71f163c62 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java
@@ -177,7 +177,7 @@ public class ManageSyncSettings extends ChromeBaseSettingsFragment
     private boolean mShouldUpdatePrefs;
 
     /** Maps {@link UserSelectableType} to the corresponding preference. */
-    private Map<Integer, ChromeSwitchPreference> mSyncTypeSwitchPreferencesMap = new HashMap<>();
+    protected Map<Integer, ChromeSwitchPreference> mSyncTypeSwitchPreferencesMap = new HashMap<>();
 
     private Preference mGoogleActivityControls;
     private Preference mSyncEncryption;
@@ -266,6 +266,7 @@ public class ManageSyncSettings extends ChromeBaseSettingsFragment
 
     @Override
     public boolean onPreferenceChange(Preference preference, Object newValue) {
+        if (false)  // Suppress the "addresses not encrypted" warning (not applicable to Brave's sync model).
         if (mSyncService.isUsingExplicitPassphrase()
                 && preference
                         .getKey()
@@ -460,7 +461,7 @@ public class ManageSyncSettings extends ChromeBaseSettingsFragment
      * this state.
      */
     private void updateSyncPreferences() {
-        @Nullable AccountInfo primaryAccount = getIdentityManager().getPrimaryAccountInfo();
+        @Nullable AccountInfo primaryAccount = new AccountInfo.Builder("", new org.chromium.google_apis.gaia.GaiaId("")).build();
         // May happen if account is removed from the device while this screen is shown.
         if (primaryAccount == null) {
             finishCurrentSettings();
