diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
index c38d03923119b07ee31ea1665bf2079da5c8c423..babfb335e21c4485226f48a9c72bffb2aafa811f 100644
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -209,6 +209,10 @@ if (!is_android && !is_mac) {
         "common/crash_keys.cc",
         "common/crash_keys.h",
       ]
+      if (brave_chromium_build) {
+        sources -= [ "app/chrome_exe.rc", ]
+        sources += [ "//brave/app/brave_exe.rc", ]
+      }
 
       deps += [
         ":chrome_dll",
@@ -575,6 +579,11 @@ if (is_win) {
     ]
   }
 
+if (brave_chromium_build) {
+  group("chrome_app") {
+    public_deps = [ "//brave:chrome_app", ]
+  }
+} else {
   mac_app_bundle("chrome_app") {
     output_name = chrome_product_full_name
 
@@ -620,6 +629,7 @@ if (is_win) {
       ]
     }
   }
+}
 
   compiled_action("chrome_app_strings") {
     tool = "//chrome/tools/build/mac:infoplist_strings_tool"
@@ -649,7 +659,7 @@ if (is_win) {
     args =
         [
           "-b",
-          "${branding_path_component}_strings",
+          _strings_file,
           "-v",
           rebase_path(chrome_version_file, root_build_dir),
           "-g",
@@ -730,7 +740,7 @@ if (is_win) {
       # framework itself, that would cause a cyclical dependency. Instead,
       # this dependency directly copies the file into the framework's
       # resources directory.
-      public_deps += [ ":chrome_framework_widevine_signature" ]
+      #public_deps += [ ":chrome_framework_widevine_signature" ]
     }
   }
 
@@ -1220,6 +1230,10 @@ if (is_win) {
     if (is_chrome_branded) {
       deps += [ ":default_apps" ]
     }
+    if (brave_chromium_build) {
+      deps += [ "//brave:framework_bundle_data", ]
+      if (defined(chrome_framework_version)) { framework_contents += [ "Frameworks", ] }
+    }
 
     ldflags = [ "-Wl,-install_name,@executable_path/../Frameworks/$chrome_framework_name.framework/Versions/$chrome_version_full/$chrome_framework_name" ]
 
@@ -1388,6 +1402,7 @@ if (is_win) {
 
 group("browser_dependencies") {
   public_deps = [
+    "//brave:browser_dependencies",
     "//chrome/browser",
     "//chrome/common",
     "//components/gwp_asan/buildflags",
@@ -1460,13 +1475,14 @@ group("child_dependencies") {
       # this is OK because all of content is linked into one library.
       "//content/browser",
     ]
+    assert_no_deps -= [ "//content/browser" ]
   }
 }
 
 if (is_win) {
   process_version_rc_template("chrome_exe_version") {
     sources = [
-      "app/chrome_exe.ver",
+      "//brave/app/brave_exe.ver",
     ]
     output = "$target_gen_dir/chrome_exe_version.rc"
   }
@@ -1529,6 +1545,7 @@ group("resources") {
     "//chrome/browser:resources",
     "//chrome/common:resources",
     "//chrome/renderer:resources",
+    "//brave:resources",
   ]
 }
 
