diff --git a/chrome/browser/ui/web_modal/browser_window_modal_dialog_delegate.h b/chrome/browser/ui/web_modal/browser_window_modal_dialog_delegate.h
index 54ff1d613d36187021834e1322dc34b74fc14efe..a91135a75bbf1be9fd6e0ed8d1a235c398d653c3 100644
--- a/chrome/browser/ui/web_modal/browser_window_modal_dialog_delegate.h
+++ b/chrome/browser/ui/web_modal/browser_window_modal_dialog_delegate.h
@@ -38,6 +38,8 @@ class BrowserWindowModalDialogDelegate
 
   void NotifyModalDialogsPositionRequiresUpdate();
 
+  bool IsWebContentsVisible(content::WebContents* web_contents) override;
+
  private:
   const raw_ptr<BrowserWindowInterface> browser_;
   ui::ScopedUnownedUserData<BrowserWindowModalDialogDelegate>
