diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index 2c9d816e1aff340615061e888cbbcd69300befcd..ff9c60969b10b9037097ea0186cc35c96f0a5a33 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -434,7 +434,7 @@ Browser::BrowserCreationStatus Browser::GetBrowserCreationStatusForProfile(
 Browser* Browser::Create(const CreateParams& params) {
   CHECK_EQ(BrowserCreationStatus::kOk,
            GetBrowserCreationStatusForProfile(params.profile));
-  return new Browser(params);
+  return new BraveBrowser(params);
 }
 
 Browser::Browser(const CreateParams& params)
@@ -445,7 +445,7 @@ Browser::Browser(const CreateParams& params)
       tab_strip_model_delegate_(
           std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
       tab_strip_model_(
-          std::make_unique<TabStripModel>(tab_strip_model_delegate_.get(),
+          std::make_unique<BraveTabStripModel>(tab_strip_model_delegate_.get(),
                                           params.profile)),
       app_name_(params.app_name),
       is_trusted_source_(params.trusted_source),
