diff --git a/chrome/browser/ui/views/tabs/compound_tab_container.cc b/chrome/browser/ui/views/tabs/compound_tab_container.cc
index c741e1b66fc2e..8efefcd506b73 100644
--- a/chrome/browser/ui/views/tabs/compound_tab_container.cc
+++ b/chrome/browser/ui/views/tabs/compound_tab_container.cc
@@ -62,17 +62,17 @@ class PinnedTabContainerController final : public TabContainerController {
   }
 
   bool IsGroupCollapsed(const tab_groups::TabGroupId& group) const override {
-    NOTREACHED_NORETURN();  // Pinned container can't have groups.
+    return base_controller_->IsGroupCollapsed(group);
   }
 
   std::optional<int> GetFirstTabInGroup(
       const tab_groups::TabGroupId& group) const override {
-    NOTREACHED_NORETURN();  // Pinned container can't have groups.
+    return base_controller_->GetFirstTabInGroup(group);
   }
 
   gfx::Range ListTabsInGroup(
       const tab_groups::TabGroupId& group) const override {
-    NOTREACHED_NORETURN();  // Pinned container can't have groups.
+    return base_controller_->ListTabsInGroup(group);
   }
 
   bool CanExtendDragHandle() const override {
