diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SettingsSearchCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SettingsSearchCoordinator.java
index c8264df9bda937b587f83f5bacee651d8961a3fa..e1af9d3c5bfc50999453a2372cf5b43aa2948a5d 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SettingsSearchCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SettingsSearchCoordinator.java
@@ -4,6 +4,8 @@
 
 package org.chromium.chrome.browser.settings.search;
 
+import org.chromium.chrome.browser.theme.BraveDynamicColors;
+import org.chromium.components.browser_ui.settings.BraveSettingsIconTintUtils;
 import static androidx.annotation.VisibleForTesting.PRIVATE;
 
 import static org.chromium.base.CallbackUtils.emptyRunnable;
@@ -307,6 +309,7 @@ public class SettingsSearchCoordinator
         TooltipCompat.setTooltipText(
                 searchBox.requireViewById(R.id.search_icon),
                 mActivity.getString(R.string.search_in_settings_hint));
+        BraveSettingsIconTintUtils.applyIconTint(searchBox.requireViewById(R.id.search_icon), BraveDynamicColors.isDynamicColorsEnabled());
 
         View query = requireViewById(R.id.search_query_container);
         Drawable bg = ContextCompat.getDrawable(mActivity, R.drawable.pill_background);
