diff --git a/chrome/browser/download/download_item_model.h b/chrome/browser/download/download_item_model.h
index 31e5bdd29de7e3b4b73250aa1b49706e0641b757..eac4924f6c93c524b6877d5dbda3fe30c0be2160 100644
--- a/chrome/browser/download/download_item_model.h
+++ b/chrome/browser/download/download_item_model.h
@@ -98,6 +98,7 @@ class DownloadItemModel : public DownloadUIModel,
   void OnDownloadDestroyed(download::DownloadItem* download) override;
 
  private:
+  friend class BraveDownloadItemModel;
   // DownloadUIModel implementation.
   std::string GetMimeType() const override;
   bool IsExtensionDownload() const override;
