diff --git a/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java b/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
index 9d66bd160ce83a6804723d219eaea6e8304f78a6..e14b366235d3e0871a37cb001fd35622600f9c6e 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
@@ -301,6 +301,7 @@ public abstract class FullscreenHtmlApiHandlerBase
 
                     @Override
                     public void onHidden(Tab tab, @TabHidingType int reason) {
+                        BraveFullscreenHtmlApiHandlerBase.class.cast(FullscreenHtmlApiHandlerBase.this).mTabHiddenByChangedTabs = reason == TabHidingType.CHANGED_TABS;
                         // Clean up any fullscreen state that might impact other tabs.
                         exitPersistentFullscreenMode();
                     }
