diff --git a/chrome/browser/chrome_content_browser_client.cc b/chrome/browser/chrome_content_browser_client.cc
index 54e5fca63b106a61a916498423c50aedc69abfb4..10767bacdd6b4cfd07a8c22859264f4a4214ec8b 100644
--- a/chrome/browser/chrome_content_browser_client.cc
+++ b/chrome/browser/chrome_content_browser_client.cc
@@ -9079,6 +9079,7 @@ base::FilePath ChromeContentBrowserClient::GetChildProcessPath(int flags) {
 
 bool ChromeContentBrowserClient::ShouldUseFirstPartyStorageKey(
     const url::Origin& origin) {
+  if (BraveShouldUseFirstPartyStorageKey(origin)) return true;
 #if BUILDFLAG(ENABLE_EXTENSIONS_CORE)
   return origin.scheme() == extensions::kExtensionScheme;
 #else
