diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 8d66d50525f97a06f9cc6501017e4b4a1e42f4d5..dde22ef0e8da16259eaa45f1cf3fb43681d8383b 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -275,7 +275,7 @@ void ToolbarView::Init() {
   } else {
     // TODO(crbug.com/932818): Remove this once the
     // |kAutofillEnableToolbarStatusChip| is fully launched.
-    avatar_ = AddChildView(std::make_unique<AvatarToolbarButton>(browser_));
+    avatar_ = AddChildView(std::make_unique<BraveAvatarToolbarButton>(browser_));
     avatar_->SetVisible(show_avatar_toolbar_button);
   }
 
