diff --git a/chrome/browser/external_protocol/external_protocol_handler.cc b/chrome/browser/external_protocol/external_protocol_handler.cc
index 5d0b30ab58c344e5a1c4220ea5b5de6b08104933..cf4bd77edf5cba3ff5e9a069c04c805053210414 100644
--- a/chrome/browser/external_protocol/external_protocol_handler.cc
+++ b/chrome/browser/external_protocol/external_protocol_handler.cc
@@ -371,7 +371,7 @@ ExternalProtocolHandler::BlockState ExternalProtocolHandler::GetBlockState(
     if (candidate == scheme) {
       base::UmaHistogramEnumeration(kBlockStateMetric,
                                     BlockStateMetric::kAllowedDefaultMail);
-      return DONT_BLOCK;
+      return UNKNOWN;
     }
   }
   for (const auto* candidate : kNewsSchemes) {
