diff --git a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabManagementDelegateImpl.java b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabManagementDelegateImpl.java
index 56863b428fbbca5f9a9b027d50b372685d84f728..5ad95282f39297a100a7f798ca4221e274af71f2 100644
--- a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabManagementDelegateImpl.java
+++ b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabManagementDelegateImpl.java
@@ -240,7 +240,7 @@ public class TabManagementDelegateImpl implements TabManagementDelegate {
             DataSharingTabManager dataSharingTabManager) {
         LazyOneshotSupplier<TabModel> tabModelSupplier =
                 LazyOneshotSupplier.fromSupplier(() -> tabModelSelector.getModel(false));
-        return new TabGroupsPane(
+        return new BraveTabGroupsPane(
                 context,
                 tabModelSupplier,
                 onToolbarAlphaChange,
