diff --git a/chrome/browser/resources/extensions/BUILD.gn b/chrome/browser/resources/extensions/BUILD.gn
index bae9844d79099b625f39423cfaaa2a5296456eab..9c21ff78077e61474d27f8b19a4f700ec99bdc69 100644
--- a/chrome/browser/resources/extensions/BUILD.gn
+++ b/chrome/browser/resources/extensions/BUILD.gn
@@ -33,6 +33,7 @@ if (optimize_webui) {
       "../../../../ui/webui/resources:preprocess",
     ]
     excludes = [ "chrome://resources/js/cr.m.js" ]
+    deps += brave_deps_chrome_browser_resources_extensions_build
   }
 }
 
@@ -54,6 +55,7 @@ generate_grd("build_grd") {
     deps = [ ":build_ts" ]
     manifest_files = [ "$target_gen_dir/tsconfig.manifest" ]
   }
+  deps += [ "//brave/browser/resources/extensions:resources" ]
 }
 
 preprocess_if_expr("preprocess") {
