diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 62430f9f3ce6767280d888b43e62d629092788e3..a5ab63a5177ee370f44de4af72ed6c9cdadc8a9a 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2459,6 +2459,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps defines += brave_chrome_browser_defines

   if (is_android) {
     sources += [
