diff --git a/chrome/browser/first_run/upgrade_util_win.cc b/chrome/browser/first_run/upgrade_util_win.cc
index c3b12ea3877ef69808bae2f779629dc2930f3456..31f3f2d119b2581e119825b0d213db893daf7ed4 100644
--- a/chrome/browser/first_run/upgrade_util_win.cc
+++ b/chrome/browser/first_run/upgrade_util_win.cc
@@ -61,7 +61,7 @@
 #include "third_party/abseil-cpp/absl/cleanup/cleanup.h"
 #include "ui/base/ui_base_switches.h"
 
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OFFICIAL_BUILD)
 #include "chrome/updater/app/server/win/updater_legacy_idl.h"
 #endif
 
@@ -74,7 +74,7 @@ bool GetNewerChromeFile(base::FilePath* path) {
   return true;
 }
 
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OFFICIAL_BUILD)
 // Holds the result of the IPC to CoCreate `GoogleUpdate3Web`.
 struct CreateGoogleUpdate3WebResult
     : public base::RefCountedThreadSafe<CreateGoogleUpdate3WebResult> {
@@ -216,7 +216,7 @@ Microsoft::WRL::ComPtr<IUnknown> CreateGoogleUpdate3Web() {
 #endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
 
 bool InvokeGoogleUpdateForRename() {
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OFFICIAL_BUILD)
   // This has been identified as very slow on some startups. Detailed trace
   // events below try to shine a light on each steps. crbug.com/1252004
   TRACE_EVENT0("startup", "upgrade_util::InvokeGoogleUpdateForRename");
