diff --git a/chrome/browser/resources/extensions/BUILD.gn b/chrome/browser/resources/extensions/BUILD.gn
index 017696b5d7f4f7a3bebbbec6164ae55a45623487..64cadc465117ecdbbbe3f2922682b8af2bbdec85 100644
--- a/chrome/browser/resources/extensions/BUILD.gn
+++ b/chrome/browser/resources/extensions/BUILD.gn
@@ -56,6 +56,7 @@ generate_grd("build_grd") {
       "$target_gen_dir/$preprocess_gen_manifest",
     ]
   }
+  deps += [ "//brave/browser/resources/extensions:resources" ]
 }
 
 preprocess_grit("preprocess") {
