diff --git a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java
index 8051698a9f277e4cd4a255cc0b30d2f41b38da46..25bb25c47850b4ced07568c866a7c3f9cc19c142 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java
@@ -63,7 +63,7 @@ class ChromeContextMenuItem {
         Item.OPEN_IN_NEW_TAB_IN_GROUP,
         Item.SHARE_HIGHLIGHT,
         Item.REMOVE_HIGHLIGHT,
-        Item.LEARN_MORE
+        Item.LEARN_MORE, Item.COPY_CLEAN_LINK
     })
     @Retention(RetentionPolicy.SOURCE)
     public @interface Item {
@@ -112,7 +112,7 @@ class ChromeContextMenuItem {
         int REMOVE_HIGHLIGHT = 33;
         int LEARN_MORE = 34;
         // ALWAYS UPDATE!
-        int NUM_ENTRIES = 35;
+        int COPY_CLEAN_LINK = 35;int NUM_ENTRIES = 36;
     }
 
     /** Mapping from {@link Item} to the ID found in the ids.xml. */
@@ -152,6 +152,7 @@ class ChromeContextMenuItem {
         R.id.contextmenu_share_highlight, // Item.SHARE_HIGHLIGHT
         R.id.contextmenu_remove_highlight, // Item.REMOVE_HIGHLIGHT
         R.id.contextmenu_learn_more, // Item.LEARN_MORE
+        R.id.contextmenu_copy_clean_link, // Item.COPY_CLEAN_LINK
     };
 
     /** Mapping from {@link Item} to the ID of the string that describes the action of the item. */
@@ -191,6 +192,7 @@ class ChromeContextMenuItem {
         R.string.contextmenu_share_highlight, // Item.SHARE_HIGHLIGHT
         R.string.contextmenu_remove_highlight, // Item.REMOVE_HIGHLIGHT
         R.string.contextmenu_learn_more, // Item.LEARN_MORE
+        R.string.contextmenu_copy_clean_link, // Item.COPY_CLEAN_LINK
     };
 
     /**
