diff --git a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java
index ba5eef852b735d49db72578939def3dad490a446..22fa66ab6764f103c1db718d7c588bdad2835e2d 100644
--- a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java
+++ b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java
@@ -588,7 +588,7 @@ public class TabSwitcherPaneCoordinator implements BackPressHandler {
                     recyclerViewTimer.getElapsedMillis());
 
             mTabListEditorManager =
-                    new TabListEditorManager(
+                    new BraveTabListEditorManager(
                             activity,
                             mModalDialogManager,
                             coordinatorView,
