diff --git a/chrome/browser/ui/omnibox/omnibox_theme.h b/chrome/browser/ui/omnibox/omnibox_theme.h
index 0adeb47d814d12199710c2e7527c34e49d0d28aa..7609ad04529b529cb61fbe86926505884b7a94af 100644
--- a/chrome/browser/ui/omnibox/omnibox_theme.h
+++ b/chrome/browser/ui/omnibox/omnibox_theme.h
@@ -28,7 +28,7 @@ enum class OmniboxPart {
 };
 
 // The tint of the omnibox theme. E.g. Incognito may use a DARK tint.
-enum class OmniboxTint { DARK, LIGHT };
+enum class OmniboxTint_Chromium { DARK, LIGHT };
 
 // An optional state for a given |OmniboxPart|.
 enum class OmniboxPartState {
