diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
index cd62f9d3ffab41de7781a83a4a98807b28d013a7..52b14c74c1173b0a7b7b68eeed418863e265efba 100644
--- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
+++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
@@ -284,8 +284,8 @@ public class ToolbarPhone extends ToolbarLayout
     private @Nullable PhoneCaptureStateToken mPhoneCaptureStateToken;
     private @Nullable ButtonData mButtonData;
 
-    private final @ColorInt int mToolbarBackgroundColorForNtp;
-    private final @ColorInt int mLocationBarBackgroundColorForNtp;
+    @ColorInt int mToolbarBackgroundColorForNtp;
+    @ColorInt int mLocationBarBackgroundColorForNtp;
     private boolean mUseAdjustedTintColorForNtp;
     private boolean mIsToolbarExpandedOnNtp;
 
@@ -581,6 +581,7 @@ public class ToolbarPhone extends ToolbarLayout
 
     /** Set the background color of the location bar to appropriately match the theme color. */
     private void updateModernLocationBarColor(@ColorInt int color) {
+        BraveToolbarLayout.class.cast(this).updateModernLocationBarColorImpl(color);
         if (mCurrentLocationBarColor == color) return;
         mCurrentLocationBarColor = color;
         mLocationBarBackground.setBackgroundColor(color);
@@ -1289,6 +1290,7 @@ public class ToolbarPhone extends ToolbarLayout
      * Tab Page.
      */
     private void updateLocationBarLayoutForExpansionAnimation() {
+        if (!BraveToolbarLayout.class.cast(this).isLocationBarValid(mLocationBar)) return;
         TraceEvent.begin("ToolbarPhone.updateLocationBarLayoutForExpansionAnimation");
         if (isInTabSwitcherMode()) return;
 
@@ -1705,6 +1707,7 @@ public class ToolbarPhone extends ToolbarLayout
             }
         }
 
+        BraveToolbarLayout.class.cast(this).drawAnimationOverlay(mToolbarButtonsContainer, canvas);
         // Draw the tab stack button and associated text if necessary.
         ToggleTabStackButtonCoordinator tabSwitcherButtonCoordinator =
                 getTabSwitcherButtonCoordinator();
@@ -2097,7 +2100,7 @@ public class ToolbarPhone extends ToolbarLayout
 
     @Override
     public void onHomeButtonIsEnabledUpdate(boolean homeButtonEnabled) {
-        mIsHomeButtonEnabled = homeButtonEnabled;
+        mIsHomeButtonEnabled = false;
         updateButtonVisibility();
     }
 
@@ -2503,6 +2506,7 @@ public class ToolbarPhone extends ToolbarLayout
     }
 
     private void createAndRunFocusAnimatorLegacy(boolean hasFocus, boolean shouldShowKeyboard) {
+        if (!BraveToolbarLayout.class.cast(this).isLocationBarValid(mLocationBar)) return;
         List<Animator> animators = new ArrayList<>();
         if (hasFocus) {
             populateUrlExpansionAnimatorSet(animators);
@@ -3661,6 +3665,7 @@ public class ToolbarPhone extends ToolbarLayout
      * @return The offset for the location bar when showing the dse icon.
      */
     int getLocationBarOffsetForFocusAnimation(boolean hasFocus) {
+        if (!urlHasFocus()) return 0;
         StatusCoordinator statusCoordinator = mLocationBar.getStatusCoordinator();
         if (statusCoordinator == null) return 0;
 
