diff --git a/chrome/browser/importer/in_process_importer_bridge.h b/chrome/browser/importer/in_process_importer_bridge.h
index 3c6695c20420e80d6680176e3a9f6c4ba6d77cf9..76bca0f2e15015c6cecba4a6fd82daef00727381 100644
--- a/chrome/browser/importer/in_process_importer_bridge.h
+++ b/chrome/browser/importer/in_process_importer_bridge.h
@@ -58,6 +58,7 @@ class InProcessImporterBridge : public ImporterBridge {
   std::u16string GetLocalizedString(int message_id) override;
   // End ImporterBridge implementation.
 
+  BRAVE_IN_PROCESS_IMPORTER_BRIDGE_H
  private:
   ~InProcessImporterBridge() override;
 
