diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 2e454efbd39a0ebdc3c0e02f2f35ca955d1456d2..8b912fd166953def82e2137880b399fd10dab5ca 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -895,6 +895,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
