diff --git a/chrome/common/chrome_content_client_constants.cc b/chrome/common/chrome_content_client_constants.cc
index a8365688d3ea165e26bc6a9e1b1851cbacaf59b2..659229e8b0774574393d4212af9a42a98ed657f2 100644
--- a/chrome/common/chrome_content_client_constants.cc
+++ b/chrome/common/chrome_content_client_constants.cc
@@ -14,7 +14,7 @@ const base::FilePath::CharType ChromeContentClient::kNaClPluginFileName[] =
     FILE_PATH_LITERAL("internal-nacl-plugin");
 #endif
 
-#if defined(GOOGLE_CHROME_BUILD)
+#if 1  // Always enabled for Brave's Chromium build
 const char ChromeContentClient::kPDFExtensionPluginName[] = "Chrome PDF Viewer";
 const char ChromeContentClient::kPDFInternalPluginName[] = "Chrome PDF Plugin";
 #else
