diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page_browser_proxy.ts b/chrome/browser/resources/settings/privacy_page/privacy_page_browser_proxy.ts
index 6e946be0031b2639e9023459002f188cfd85a135..08b5d26a5818dea5f3ff160e3dc8eb031d4e4ef8 100644
--- a/chrome/browser/resources/settings/privacy_page/privacy_page_browser_proxy.ts
+++ b/chrome/browser/resources/settings/privacy_page/privacy_page_browser_proxy.ts
@@ -46,7 +46,7 @@ export interface SecureDnsSetting {
 }
 
 export interface PrivacyPageBrowserProxy {
-  // <if expr="_google_chrome and not chromeos_ash">
+  // <if expr="True and not chromeos_ash">
   getMetricsReporting(): Promise<MetricsReporting>;
   setMetricsReportingEnabled(enabled: boolean): void;
 
@@ -84,7 +84,7 @@ export interface PrivacyPageBrowserProxy {
 }
 
 export class PrivacyPageBrowserProxyImpl implements PrivacyPageBrowserProxy {
-  // <if expr="_google_chrome and not chromeos_ash">
+  // <if expr="True and not chromeos_ash">
   getMetricsReporting() {
     return sendWithPromise('getMetricsReporting');
   }
