diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
index 44524674567cbc11981c4cabe4be882d3aaa14ce..ace664b8094264e763ec92f88e1a263f3a6fa274 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
@@ -259,6 +259,7 @@ void BookmarkBubbleView::ShowBubble(
       dialog_model_builder.Build(), anchor_view,
       views::BubbleBorder::TOP_RIGHT);
   bookmark_bubble_ = bubble.get();
+  BRAVE_BOOKMARK_BUBBLE_VIEW
   if (highlighted_button)
     bubble->SetHighlightedButton(highlighted_button);
 
