diff --git a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationPlatformBridge.java b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationPlatformBridge.java
index 6fa8b0d1833b92dc486b861d2d3a1469d814c574..954c6ac08c4d44a8fcae198788a2b07bd5079736 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationPlatformBridge.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationPlatformBridge.java
@@ -81,7 +81,7 @@ public class NotificationPlatformBridge {
 
     private static final int[] EMPTY_VIBRATION_PATTERN = new int[0];
 
-    private static NotificationPlatformBridge sInstance;
+    protected static NotificationPlatformBridge sInstance;
 
     private static NotificationManagerProxy sNotificationManagerOverride;
 
@@ -134,7 +134,7 @@ public class NotificationPlatformBridge {
         sNotificationManagerOverride = notificationManager;
     }
 
-    private NotificationPlatformBridge(long nativeNotificationPlatformBridge) {
+    protected NotificationPlatformBridge(long nativeNotificationPlatformBridge) {
         mNativeNotificationPlatformBridge = nativeNotificationPlatformBridge;
         Context context = ContextUtils.getApplicationContext();
         if (sNotificationManagerOverride != null) {
@@ -574,7 +574,7 @@ public class NotificationPlatformBridge {
         });
     }
 
-    private NotificationBuilderBase prepareNotificationBuilder(String notificationId,
+    protected NotificationBuilderBase prepareNotificationBuilder(String notificationId,
             @NotificationType int notificationType, String origin, String scopeUrl,
             String profileId, boolean incognito, boolean vibrateEnabled, String title, String body,
             Bitmap image, Bitmap icon, Bitmap badge, int[] vibrationPattern, long timestamp,
@@ -682,7 +682,7 @@ public class NotificationPlatformBridge {
                         notificationId /* notificationTag */, PLATFORM_ID /* notificationId */));
     }
 
-    private NotificationBuilderBase createNotificationBuilder(Context context, boolean hasImage) {
+    protected NotificationBuilderBase createNotificationBuilder(Context context, boolean hasImage) {
         return useCustomLayouts(hasImage) ? new CustomNotificationBuilder(context)
                                           : new StandardNotificationBuilder(context);
     }
