diff --git a/chrome/browser/profiles/profile_manager.h b/chrome/browser/profiles/profile_manager.h
index 8ba4e0ba9f3c014fb94794d50afe8d54e94e3fdf..4551ca035aabe5844d3676bb093678ada0099c7f 100644
--- a/chrome/browser/profiles/profile_manager.h
+++ b/chrome/browser/profiles/profile_manager.h
@@ -355,6 +355,7 @@ class ProfileManager : public Profile::Delegate {
       std::unique_ptr<AccountProfileMapper> mapper);
 #endif
 
+  BRAVE_PROFILE_MANAGER_H
  private:
   friend class TestingProfileManager;
   FRIEND_TEST_ALL_PREFIXES(ProfileManagerBrowserTest, DeleteAllProfiles);
