diff --git a/chrome/browser/ui/views/download/download_item_view.h b/chrome/browser/ui/views/download/download_item_view.h
index dff79a4cc6b0c9d6661535285ecf0f9fe534abab..acf63aefd2681280e0a43151d907f395b41ecbb7 100644
--- a/chrome/browser/ui/views/download/download_item_view.h
+++ b/chrome/browser/ui/views/download/download_item_view.h
@@ -115,6 +115,7 @@ class DownloadItemView : public views::View,
   void OnPaintBackground(gfx::Canvas* canvas) override;
   void OnPaint(gfx::Canvas* canvas) override;
   void OnThemeChanged() override;
+  BRAVE_DOWNLOAD_DOWNLOAD_ITEM_VIEW_H_
 
  private:
   // Returns the mode that best reflects the current model state.
