diff --git a/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc b/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc
index 2068a1131186e99fc9f675e2dc9f4bc2cc4f135d..6124ccd39eacac4c46f4a89d9fe4622ca4dd59be 100644
--- a/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc
+++ b/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc
@@ -136,6 +136,7 @@ PermissionPromptBubbleView::PermissionPromptBubbleView(
 
   for (permissions::PermissionRequest* request : GetVisibleRequests())
     AddRequestLine(request);
+  BRAVE_PERMISSION_PROMPT_BUBBLE_VIEW
 
   absl::optional<std::u16string> extra_text = GetExtraText();
   if (extra_text.has_value()) {
