diff --git a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
index e783bf65be65ee6a38d2f9609c3f1baa9c83a752..a94f1976a8453b69128cb85fe8d203615d402ce2 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
@@ -270,6 +270,7 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             Action.TRANSLATE,
             Action.CREATE_QR_CODE,
             Action.ASK_GEMINI,
+            Action.COPY_CLEAN_LINK,
         })
         @Retention(RetentionPolicy.SOURCE)
         public @interface Action {
@@ -333,7 +334,8 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             int TRANSLATE = 57;
             int CREATE_QR_CODE = 58;
             int ASK_GEMINI = 59;
-            int NUM_ENTRIES = 60;
+            int COPY_CLEAN_LINK = 60;
+            int NUM_ENTRIES = COPY_CLEAN_LINK + 1;
         }
 
         // LINT.ThenChange(/tools/metrics/histograms/enums.xml:ContextMenuOptionAndroid)
@@ -727,6 +729,7 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             if (!MailTo.isMailTo(mParams.getLinkUrl().getSpec())
                     && !UrlUtilities.isTelScheme(mParams.getLinkUrl())) {
                 linkGroup.add(createListItem(Item.COPY_LINK_ADDRESS));
+                linkGroup.add(createListItem(Item.COPY_CLEAN_LINK));
                 if (!mParams.getLinkText().trim().isEmpty() && !mParams.isImage()) {
                     linkGroup.add(createListItem(Item.COPY_LINK_TEXT));
                 }
