diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index c0d41e69687a0ae4e10e9dc3e5fb8dfe0468fa7e..658487fd554ea1fa8949dcfe5df8f3d5a3058cac 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -216,6 +216,7 @@ source_set("internal") {
       "//chrome/browser/ui:browser_select_file_dialog_controller",
     ]
   }
+  import("//brave/browser/ui/browser_window/internal/sources.gni") deps += brave_browser_window_internal_deps
 }
 
 source_set("unit_tests") {
