diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 521a10b99592ba36c86327e6bf2235daef506828..31a7464faebf1b5215fcc705127c8ab1c3aebb30 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -221,6 +221,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  sources += brave_chrome_renderer_sources public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
