diff --git a/chrome/chrome_proxy/chrome_proxy_main_win.cc b/chrome/chrome_proxy/chrome_proxy_main_win.cc
index 4fcbf2229155734a3cdceabe76184f69266678de..b48350a29f2728e8de579e97a404617b69015f95 100644
--- a/chrome/chrome_proxy/chrome_proxy_main_win.cc
+++ b/chrome/chrome_proxy/chrome_proxy_main_win.cc
@@ -15,7 +15,10 @@
 namespace {
 
 constexpr base::FilePath::CharType kChromeExecutable[] =
+    FILE_PATH_LITERAL("brave.exe");
+#if 0
     FILE_PATH_LITERAL("chrome.exe");
+#endif
 
 constexpr base::FilePath::CharType kChromeProxyExecutable[] =
     FILE_PATH_LITERAL("chrome_proxy.exe");
