diff --git a/chrome/browser/profiles/profile_io_data.cc b/chrome/browser/profiles/profile_io_data.cc
index 1c925a2bff238c437e4a37cdeb47785e95acdbe4..611dd06bd451e960df5339b93ae3e9295bb02d78 100644
--- a/chrome/browser/profiles/profile_io_data.cc
+++ b/chrome/browser/profiles/profile_io_data.cc
@@ -24,6 +24,7 @@
 #include "base/task/post_task.h"
 #include "base/task/task_traits.h"
 #include "base/threading/thread_task_runner_handle.h"
+#include "brave/browser/net/brave_profile_network_delegate.h"
 #include "build/build_config.h"
 #include "chrome/browser/browser_process.h"
 #include "chrome/browser/chrome_notification_types.h"
@@ -640,6 +641,9 @@ bool ProfileIOData::IsHandledProtocol(const std::string& scheme) {
     extensions::kExtensionScheme,
 #endif
     content::kChromeUIScheme,
+#if defined(BRAVE_CHROMIUM_BUILD)
+    kBraveUIScheme,
+#endif
     url::kDataScheme,
 #if defined(OS_CHROMEOS)
     content::kExternalFileScheme,
@@ -926,7 +930,7 @@ void ProfileIOData::Init(
         std::make_unique<network::URLRequestContextBuilderMojo>();
 
     std::unique_ptr<ChromeNetworkDelegate> chrome_network_delegate(
-        new ChromeNetworkDelegate(
+        new BraveProfileNetworkDelegate(
 #if BUILDFLAG(ENABLE_EXTENSIONS)
             io_thread_globals->extension_event_router_forwarder.get()));
 #else
