diff --git a/chrome/browser/sync/chrome_sync_client.h b/chrome/browser/sync/chrome_sync_client.h
index c89058c8815f8983dafaeb88edf1a239f11a9b08..bb22a19336d91fcf476607a3399bbbe850fa5f47 100644
--- a/chrome/browser/sync/chrome_sync_client.h
+++ b/chrome/browser/sync/chrome_sync_client.h
@@ -60,6 +60,7 @@ class ChromeSyncClient : public browser_sync::BrowserSyncClient {
   scoped_refptr<syncer::ModelSafeWorker> CreateModelWorkerForGroup(
       syncer::ModelSafeGroup group) override;
   syncer::SyncApiComponentFactory* GetSyncApiComponentFactory() override;
+  brave_sync::BraveSyncClient* GetBraveSyncClient() override;
 
  private:
   Profile* const profile_;
@@ -82,6 +83,8 @@ class ChromeSyncClient : public browser_sync::BrowserSyncClient {
   // Generates and monitors the ExtensionsActivity object used by sync.
   ExtensionsActivityMonitor extensions_activity_monitor_;
 
+  std::unique_ptr<brave_sync::BraveSyncClient> brave_sync_client_;
+
   DISALLOW_COPY_AND_ASSIGN(ChromeSyncClient);
 };
 
