diff --git a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
index 4a462e7afe9c38181f0cc560f7cc9eb160dc46d2..154243236b6c01b9e1087232e9ddf5ecff74b395 100644
--- a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
+++ b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
@@ -1475,7 +1475,13 @@ void TabDragController::AttachToNewContext(
 
       const bool pinned = detached_tab_collection->get()->pinned_;
 
-      if (std::holds_alternative<std::unique_ptr<tabs::TabGroupTabCollection>>(
+      if (std::holds_alternative<
+              std::unique_ptr<tabs::TreeTabNodeTabCollection>>(
+              detached_tab_collection->get()->collection_)) {
+        collection_indices =
+            attached_context_->GetTabStripModel()->InsertDetachedTreeTabNodeAt(
+                std::move(*detached_tab_collection), index);
+      } else if (std::holds_alternative<std::unique_ptr<tabs::TabGroupTabCollection>>(
               detached_tab_collection->get()->collection_)) {
         collection_indices =
             attached_context_->GetTabStripModel()->InsertDetachedTabGroupAt(
