diff --git a/chrome/browser/ui/tabs/tab_style.h b/chrome/browser/ui/tabs/tab_style.h
index b2dcced3575a18131a634b4ddba46baec958dc85..d8a9ce61a27dfb45f9b5e04ab6a96d785232b38a 100644
--- a/chrome/browser/ui/tabs/tab_style.h
+++ b/chrome/browser/ui/tabs/tab_style.h
@@ -111,7 +111,7 @@ class TabStyle {
 
   TabStyle(const TabStyle&) = delete;
   TabStyle& operator=(const TabStyle&) = delete;
-  ~TabStyle();
+  virtual ~TabStyle();
 
   // Returns the standard height of a single Tab.
   int GetStandardHeight() const;
