diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index aae3ce608dec01da00bc6bd9c2648c68924e3251..6304fcbbb319794b488b89f1be59d78a1b2b960b 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -72,6 +72,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
