diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index fa0820880c28a78960d10fc918d121be18d7823f..40c1ad3e4990527f77b71a762d5f59a372f0d966 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3157,6 +3157,7 @@ const FeatureEntry::FeatureVariation kDmTokenDeletionVariation[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
