diff --git a/chrome/browser/themes/theme_service.h b/chrome/browser/themes/theme_service.h
index b215ccd3941a82cf740fd559b3556f880a8fc2d6..9039b49a2d911dd9cc6e551f48829748773808a3 100644
--- a/chrome/browser/themes/theme_service.h
+++ b/chrome/browser/themes/theme_service.h
@@ -205,6 +205,7 @@ class ThemeService : public content::NotificationObserver,
   // ThemeSource no longer uses the ThemeService when it is not ready.
   bool ready_;
 
+  THEMES_THEME_SERVICE_H_
  private:
   // This class implements ui::ThemeProvider on behalf of ThemeService and keeps
   // track of the incognito state of the calling code.
