diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 0fcf061bf93b4b11603ef944cf8dd4d456cf5f45..4e3251cdf1c6b9a830230a4538cc3b2838740314 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -1307,6 +1307,7 @@ if (!is_android && !is_fuchsia) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
