diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 289ca80107b57eda6d99041889b6a3cfab6eb079..d4dc0f7888fc22012f65d593235620c9e81de4ce 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4743,6 +4743,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
