diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 7540a4823c30c7a7219aeab246a79a2144961adf..a892ee1556a757ede5b87a6f72becd066c4da6aa 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -2866,6 +2866,7 @@ constexpr FeatureEntry::FeatureVariation kLensStandaloneVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
