diff --git a/chrome/browser/resources/downloads/BUILD.gn b/chrome/browser/resources/downloads/BUILD.gn
index f388e81c0cc50be62ac7471254730dfa54a95644..55384e1dbcfd48387791d9d82362cbfd576e9bb8 100644
--- a/chrome/browser/resources/downloads/BUILD.gn
+++ b/chrome/browser/resources/downloads/BUILD.gn
@@ -71,4 +71,5 @@ build_webui("build") {
     optimize_webui_host = "downloads"
     optimize_webui_in_files = [ "downloads.js" ]
   }
+  import("//brave/browser/resources/downloads/sources.gni") non_web_component_files += brave_downloads_non_web_component_files exclude_ts_preprocess_files = brave_downloads_ts_local_files preprocess_deps = brave_downloads_preprocess_extra_deps
 }
