diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 6d355943715348a94af7df53337a96422a0f95c5..c8d06b7b4f785ede823d28eb256a9cc47c2aea53 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -78,6 +78,7 @@ source_set("channel_info") {
   if (is_android) {
     public_deps += [ "//components/version_info/android:channel_getter" ]
   }
+  if (brave_chromium_build) { deps = [ "//brave/common:channel_info" ] }
 }
 
 source_set("ini_parser") {
@@ -542,6 +543,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:ui_features",
   ]
+  if (brave_chromium_build) { configs += [ "//brave/common:constants_configs" ] }
 }
 
 static_library("constants") {
