diff --git a/chrome/browser/ui/accelerator_table.cc b/chrome/browser/ui/accelerator_table.cc
index 9323183fc9ead2cadc1fe33eddbd017691f56ce3..d727666f148134e24e32dd8ef98e888788a3f872 100644
--- a/chrome/browser/ui/accelerator_table.cc
+++ b/chrome/browser/ui/accelerator_table.cc
@@ -309,7 +309,7 @@ std::vector<AcceleratorMapping>* GetAcceleratorsPointer() {
 
 }  // namespace
 
-std::vector<AcceleratorMapping> GetAcceleratorList() {
+std::vector<AcceleratorMapping> GetAcceleratorList_ChromiumImpl() {
   std::vector<AcceleratorMapping>* accelerators = GetAcceleratorsPointer();
 
   if (accelerators->empty()) {
