diff --git a/chrome/browser/ui/webui/history/history_ui.cc b/chrome/browser/ui/webui/history/history_ui.cc
index 1bcf321faec3e4582ea28dec93a1fff2fb8c3014..a553aee814893b22c0ab4f57da597807f00d4075 100644
--- a/chrome/browser/ui/webui/history/history_ui.cc
+++ b/chrome/browser/ui/webui/history/history_ui.cc
@@ -88,6 +88,7 @@ content::WebUIDataSource* CreateAndAddHistoryUIHTMLSource(Profile* profile) {
 #endif  // BUILDFLAG!(IS_CHROMEOS)
 
   HistoryUtil::PopulateCommonSourceForHistory(source, profile);
+  BRAVE_CREATE_HISTORY_UI_HTML_SOURCE
 
   static constexpr webui::LocalizedString kStrings[] = {
       // Localized strings (alphabetical order).
