diff --git a/chrome/browser/ui/views/tabs/tab_style_views.cc b/chrome/browser/ui/views/tabs/tab_style_views.cc
index 202e116be8fb3bd6dc948de9d725eaa14941d81e..c0a1f1287ea497a5cc8388eb457ea4568571f134 100644
--- a/chrome/browser/ui/views/tabs/tab_style_views.cc
+++ b/chrome/browser/ui/views/tabs/tab_style_views.cc
@@ -126,6 +126,7 @@ class GM2TabStyleViews : public TabStyleViews {
                                              const Tab* other_tab) const;
 
  private:
+  BRAVE_GM2_TAB_STYLE_H
   // Gets the bounds for the leading and trailing separators for a tab.
   TabStyle::SeparatorBounds GetSeparatorBounds(float scale) const;
 
@@ -274,6 +275,7 @@ SkPath GM2TabStyleViews::GetPath(TabStyle::PathType path_type,
   const ShapeModifier shape_modifier = GetShapeModifier(path_type);
   const bool extend_left_to_bottom = shape_modifier & kNoLowerLeftArc;
   const bool extend_right_to_bottom = shape_modifier & kNoLowerRightArc;
+  extension_corner_radius = 0;
 
   SkPath path;
 
@@ -548,8 +550,7 @@ TabStyle::SeparatorBounds GM2TabStyleViews::GetSeparatorBounds(
       aligned_bounds.x() + corner_radius - separator_margin.right() -
           separator_size.width(),
       aligned_bounds.y() + (aligned_bounds.height() - separator_size.height() -
-                            separator_margin.bottom()) /
-                               2,
+                            separator_margin.bottom()),
       separator_size.width(), separator_size.height());
 
   separator_bounds.trailing = separator_bounds.leading;
@@ -923,12 +924,16 @@ void GM2TabStyleViews::PaintSeparators(gfx::Canvas* canvas) const {
                                                    SK_AlphaOPAQUE));
   };
 
+  // Even if |separator_radius| becomes 1 native pixel the 'roundedness'
+  // will be approximated with color, although extremely subtle and
+  // likely only on screens >= 2x (i.e. separator width is 2+px)!
+  const int separator_radius =  separator_bounds.leading.width() / 2;
   cc::PaintFlags flags;
   flags.setAntiAlias(true);
   flags.setColor(separator_color(separator_opacities.left));
-  canvas->DrawRect(separator_bounds.leading, flags);
+  canvas->DrawRoundRect(separator_bounds.leading, separator_radius, flags);
   flags.setColor(separator_color(separator_opacities.right));
-  canvas->DrawRect(separator_bounds.trailing, flags);
+  canvas->DrawRoundRect(separator_bounds.trailing, separator_radius, flags);
 }
 
 float GM2TabStyleViews::GetTopCornerRadiusForWidth(int width) const {
