diff --git a/chrome/browser/importer/in_process_importer_bridge.h b/chrome/browser/importer/in_process_importer_bridge.h
index 61190844025f04ef83bcfe2c056f9f61661ca132..49f75bcbb4b1c6ae09488d8968dfea2bef2b77e1 100644
--- a/chrome/browser/importer/in_process_importer_bridge.h
+++ b/chrome/browser/importer/in_process_importer_bridge.h
@@ -60,6 +60,7 @@ class InProcessImporterBridge : public ImporterBridge {
   std::u16string GetLocalizedString(int message_id) override;
   // End ImporterBridge implementation.
 
+  BRAVE_IN_PROCESS_IMPORTER_BRIDGE_H
  private:
   ~InProcessImporterBridge() override;
 
