diff --git a/chrome/browser/ui/startup/startup_browser_creator_impl.cc b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
index 3f4c0479b171960191d2e9d0dc45aabc9a4e69a8..ab36248b2a0ef1717641a5022df009375e7cc0df 100644
--- a/chrome/browser/ui/startup/startup_browser_creator_impl.cc
+++ b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
@@ -421,6 +421,7 @@ BrowserWindowInterface* StartupBrowserCreatorImpl::OpenTabsInBrowser(
       params.disposition = WindowOpenDisposition::CURRENT_TAB;
       params.tabstrip_add_types = ADD_NONE;
       first_tab = false;
+      BraveModifyStartupTabNavigationParams(tab, browser, params);
       Navigate(&params);
       continue;
     }
@@ -451,6 +452,7 @@ BrowserWindowInterface* StartupBrowserCreatorImpl::OpenTabsInBrowser(
     }
 #endif  // BUILDFLAG(ENABLE_RLZ)
 
+    BraveModifyStartupTabNavigationParams(tab, browser, params);
     Navigate(&params);
     first_tab = false;
   }
