diff --git a/chrome/browser/ui/views/page_action/page_action_controller.h b/chrome/browser/ui/views/page_action/page_action_controller.h
index 4524d269031a0387a2de6644207b934cc44f6107..2c3eb05a0b1bebc80c60f095379abf7397cafcfb 100644
--- a/chrome/browser/ui/views/page_action/page_action_controller.h
+++ b/chrome/browser/ui/views/page_action/page_action_controller.h
@@ -108,6 +108,17 @@ class PageActionController {
  public:
   virtual ~PageActionController() = default;
 
+  // Force the label to be shown regardless of the available space
+  virtual void SetAlwaysShowLabel(actions::ActionId action_id,
+                                  bool always_show) = 0;
+
+  // Override the colors of the chip
+  virtual void OverrideChipColors(
+      actions::ActionId action_id,
+      std::optional<SkColor> override_background_color,
+      std::optional<SkColor> override_foreground_color) = 0;
+  virtual void ClearOverrideChipColors(actions::ActionId action_id) = 0;
+
   // Requests that the page action be shown or hidden.
   virtual void Show(actions::ActionId action_id) = 0;
   virtual void Hide(actions::ActionId action_id) = 0;
@@ -242,6 +253,13 @@ class PageActionControllerImpl : public PageActionController,
   PageActionControllerImpl& operator=(const PageActionControllerImpl&) = delete;
   ~PageActionControllerImpl() override;
 
+  void SetAlwaysShowLabel(actions::ActionId action_id,
+                          bool always_show) override;
+  void OverrideChipColors(actions::ActionId action_id,
+      std::optional<SkColor> override_background_color,
+      std::optional<SkColor> override_foreground_color) override;
+  void ClearOverrideChipColors(actions::ActionId action_id) override;
+
   void Initialize(
       tabs::TabInterface& tab_interface,
       const std::vector<actions::ActionId>& action_ids,
