diff --git a/chrome/browser/ui/views/frame/tab_strip_region_view.h b/chrome/browser/ui/views/frame/tab_strip_region_view.h
index 56c5935b07c3d1d9c5234be10da37c8bf13122f9..d8d24ab70be417acc9a099d99f6278add7ca1c39 100644
--- a/chrome/browser/ui/views/frame/tab_strip_region_view.h
+++ b/chrome/browser/ui/views/frame/tab_strip_region_view.h
@@ -29,7 +29,7 @@ class TabSearchPositionMetricsLogger;
 
 // Container for the tabstrip and the other views sharing space with it -
 // with the exception of the caption buttons.
-class TabStripRegionView final : public views::AccessiblePaneView,
+class TabStripRegionView : public views::AccessiblePaneView,
                                  public TabStripViewInterface {
   METADATA_HEADER(TabStripRegionView, views::AccessiblePaneView)
 
