diff --git a/chrome/browser/ui/startup/startup_browser_creator_impl.cc b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
index e989e5b80d255bb7983b8f4d1c1b6bc39109eb41..c65a9c0d306b852917befeeb404dc22ce6121e2d 100644
--- a/chrome/browser/ui/startup/startup_browser_creator_impl.cc
+++ b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
@@ -417,6 +417,7 @@ Browser* StartupBrowserCreatorImpl::OpenTabsInBrowser(
       params.disposition = WindowOpenDisposition::CURRENT_TAB;
       params.tabstrip_add_types = ADD_NONE;
       first_tab = false;
+      BraveModifyStartupTabNavigationParams(tab, browser, params);
       Navigate(&params);
       continue;
     }
@@ -447,6 +448,7 @@ Browser* StartupBrowserCreatorImpl::OpenTabsInBrowser(
     }
 #endif  // BUILDFLAG(ENABLE_RLZ)
 
+    BraveModifyStartupTabNavigationParams(tab, browser, params);
     Navigate(&params);
     first_tab = false;
   }
