diff --git a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
index 96269a9aea8c8947fb1d7a2f1e18d416e59fe568..437356df9a04d3b8aa8d48710d2b52059ffb91ae 100644
--- a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
+++ b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
@@ -580,6 +580,9 @@ void RecentTabsSubMenuModel::BuildTabsFromOtherDevices() {
   std::vector<raw_ptr<const sync_sessions::SyncedSession, VectorExperimental>>
       sessions;
   if (!open_tabs || !open_tabs->GetAllForeignSessions(&sessions)) {
+    AddItemWithStringId(IDC_RECENT_TABS_NO_DEVICE_TABS,
+                        IDS_RECENT_TABS_NO_DEVICE_TABS);
+    return;
     if (open_tabs) {
       AddItemWithStringId(IDC_RECENT_TABS_NO_DEVICE_TABS,
                           IDS_RECENT_TABS_NO_DEVICE_TABS);
@@ -623,6 +626,8 @@ void RecentTabsSubMenuModel::BuildTabsFromOtherDevices() {
     DCHECK(!session->GetSessionName().empty());
     std::unique_ptr<ui::SimpleMenuModel> device_menu_model =
         CreateOtherDeviceSubMenu(session, tabs_in_session);
+    MaybeAppendOverflowStubTab(tabs_in_session, kMaxSessionsToShow,
+                               device_menu_model.get());
     const int command_id = GetAndIncrementNextMenuID();
     AddSubMenu(command_id, base::UTF8ToUTF16(session->GetSessionName()),
                device_menu_model.get());
