diff --git a/chrome/browser/profiles/profile_impl.cc b/chrome/browser/profiles/profile_impl.cc
index 46c6cdc32ea5edeb0662e1e0f75f5c174f632295..3a783b1358b23ec942b053a0c1529a2d2e237f07 100644
--- a/chrome/browser/profiles/profile_impl.cc
+++ b/chrome/browser/profiles/profile_impl.cc
@@ -378,7 +378,7 @@ std::unique_ptr<Profile> Profile::CreateProfile(const base::FilePath& path,
     NOTREACHED();
   }
 
-  std::unique_ptr<Profile> profile = base::WrapUnique(new ProfileImpl(
+  std::unique_ptr<Profile> profile = base::WrapUnique(new BraveProfileImpl(
       path, delegate, create_mode, creation_time, io_task_runner));
   return profile;
 }
