diff --git a/chrome/browser/ui/views/extensions/extension_popup.cc b/chrome/browser/ui/views/extensions/extension_popup.cc
index d61bc7c817e97ff03148540967c503a1dd000844..07b187408a70e5b2d6d707be94b5ca5efc61b6a9 100644
--- a/chrome/browser/ui/views/extensions/extension_popup.cc
+++ b/chrome/browser/ui/views/extensions/extension_popup.cc
@@ -66,7 +66,7 @@ gfx::Size ExtensionPopup::CalculatePreferredSize() const {
 
 void ExtensionPopup::AddedToWidget() {
   BubbleDialogDelegateView::AddedToWidget();
-  const int radius = GetBubbleFrameView()->corner_radius();
+  BRAVE_ADDED_TO_WIDGET
   const bool contents_has_rounded_corners =
       GetExtensionView()->holder()->SetCornerRadius(radius);
   SetBorder(views::CreateEmptyBorder(
