diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 26f8d389be1f26e7f21e3225b9c367b4b5f71e07..d05fedd41baa2d4e0cb6f82a583ce061078fa541 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -75,6 +75,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
