diff --git a/chrome/browser/ui/views/download/download_item_view.h b/chrome/browser/ui/views/download/download_item_view.h
index 1e5ee9c7eb94ea6da4e71d3fb236fc44c6cb2fe1..df8a169b22b06637d13ad10b63079253b770ec04 100644
--- a/chrome/browser/ui/views/download/download_item_view.h
+++ b/chrome/browser/ui/views/download/download_item_view.h
@@ -115,6 +115,7 @@ class DownloadItemView : public views::View,
   void OnPaintBackground(gfx::Canvas* canvas) override;
   void OnPaint(gfx::Canvas* canvas) override;
   void OnThemeChanged() override;
+  BRAVE_DOWNLOAD_DOWNLOAD_ITEM_VIEW_H_
 
   // ui::LayerDelegate:
   void OnDeviceScaleFactorChanged(float old_device_scale_factor,
