diff --git a/chrome/browser/updater/BUILD.gn b/chrome/browser/updater/BUILD.gn
index b9c442efe4805721eeb732eed905d4e4a668a7aa..441b3d417093ec1dac90978db7c5e5ecfb34c1f2 100644
--- a/chrome/browser/updater/BUILD.gn
+++ b/chrome/browser/updater/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/updater/sources.gni")
 import("//build/config/chrome_build.gni")
 import("//chrome/browser/buildflags.gni")
 
@@ -76,6 +77,7 @@ source_set("updater") {
       ]
       frameworks = [ "OpenDirectory.framework" ]
     }
+    deps += brave_chrome_updater_scheduler_deps
   } else if (enable_updater && is_linux) {
     # Chromium Updater is only partially supported on Linux. Linux support in
     # the browser exists to facilitate the implementation of cross-platform
