diff --git a/chrome/browser/notifications/notification_platform_bridge_android.h b/chrome/browser/notifications/notification_platform_bridge_android.h
index b3c1e534a347c83bf77f2a40cb38be5f4eb0d0be..5de85b72e3a53778cd20e453431a2eed8ffda83c 100644
--- a/chrome/browser/notifications/notification_platform_bridge_android.h
+++ b/chrome/browser/notifications/notification_platform_bridge_android.h
@@ -105,6 +105,7 @@ class NotificationPlatformBridgeAndroid : public NotificationPlatformBridge {
  private:
   void OnNotificationProcessed(const std::string& notification_id);
 
+  friend class BraveNotificationPlatformBridgeHelperAndroid;
   // Contains information necessary in order to enable closing notifications
   // that were not created by this instance of the manager. This list may not
   // contain the notifications that have not been interacted with since the last
