diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 92db44edb49e8e58389c4ac95c3016e43d8718dc..36559e2f11a6287632647fabd51997c9c8609c80 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2538,6 +2538,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
