diff --git a/chrome/browser/ui/actions/chrome_action_id.h b/chrome/browser/ui/actions/chrome_action_id.h
index 4f062f3be33895fe8c807f554bb0c13e616d921a..f1c710f61da78aa9bd97cd77f0c54736bb33d5d7 100644
--- a/chrome/browser/ui/actions/chrome_action_id.h
+++ b/chrome/browser/ui/actions/chrome_action_id.h
@@ -551,6 +551,9 @@
 // Should be reinserted into CHROME_COMMON_ACTION_IDS when general solution to
 // add action id mappings is implemented.
 #define SIDE_PANEL_ACTION_IDS \
+  E(kActionSidePanelShowPlaylist) \
+  E(kActionSidePanelShowChatUI) \
+  E(kActionSidePanelShowBraveNews) \
   /* Side Panel items */ \
   E(kActionSidePanelShowAboutThisSite) \
   E(kActionSidePanelShowAssistant) \
@@ -576,6 +579,8 @@
     IDC_SHOW_TABS_FROM_OTHER_DEVICES_SIDE_PANEL)
 
 #define TOOLBAR_PINNABLE_ACTION_IDS \
+  E(kActionShowPartitionedStorage) \
+  E(kActionShowPsstIcon) \
   E(kActionHome, IDC_HOME) \
   E(kActionForward, IDC_FORWARD) \
   E(kActionNewIncognitoWindow, IDC_NEW_INCOGNITO_WINDOW) \
