diff --git a/chrome/browser/ui/views/overlay/overlay_window_views.cc b/chrome/browser/ui/views/overlay/overlay_window_views.cc
index 7d91ecafdfc2b977aeb73c764cee6bb65f80b5a3..8c79edb7226b7b2907f2f84ae4362cede22be094 100644
--- a/chrome/browser/ui/views/overlay/overlay_window_views.cc
+++ b/chrome/browser/ui/views/overlay/overlay_window_views.cc
@@ -533,6 +533,7 @@ void OverlayWindowViews::UpdateMaxSize(const gfx::Rect& work_area) {
 
   max_size_ = new_max_size;
 
+  BRAVE_UPDATE_MAX_SIZE
   if (!native_widget())
     return;
 
