diff --git a/chrome/browser/ui/views/page_action/page_action_view.cc b/chrome/browser/ui/views/page_action/page_action_view.cc
index dfd4bc11fdd9d0c85a6a67f3872b0e52afec4ea2..02eb3bcb991263ff6420bab82287412db3baa937 100644
--- a/chrome/browser/ui/views/page_action/page_action_view.cc
+++ b/chrome/browser/ui/views/page_action/page_action_view.cc
@@ -167,7 +167,7 @@ void PageActionView::OnPageActionModelChanged(
   if (!visible) {
     ResetSlideAnimation(/*show=*/false);
     NotifyIsChipShowingChange();
-  } else if (model.ShouldShowSuggestionChip()) {
+  } else if (model.ShouldShowSuggestionChip() || model.GetAlwaysShowLabel()) {
     if (model.GetShouldAnimateChipIn()) {
       AnimateIn(/*string_id=*/std::nullopt);
     } else {
