diff --git a/chrome/android/chrome_test_apk_tmpl.gni b/chrome/android/chrome_test_apk_tmpl.gni
index 113c1e404526eb7d449d554d322e9e1c70016cdf..17f7337c8c460214b48a1bee504ceb1e79d9e77d 100644
--- a/chrome/android/chrome_test_apk_tmpl.gni
+++ b/chrome/android/chrome_test_apk_tmpl.gni
@@ -100,6 +100,7 @@ template("monochrome_test_apk_tmpl") {
              "//third_party/android_sdk:android_test_runner_java",
              "//third_party/android_support_test_runner:runner_java",
            ]
+    deps -= [ "//android_webview:platform_service_bridge_upstream_implementation_java" ]
 
     if (webview_includes_weblayer) {
       deps += [ "//weblayer/browser/java:upstream_java" ]
