diff --git a/chrome/browser/ui/views/tabs/tab_strip.cc b/chrome/browser/ui/views/tabs/tab_strip.cc
index 5aed267573ab4c17c08689ec142d6327f9fb1f9e..49ea6719639d73e2d229369066b0f379758cf37f 100644
--- a/chrome/browser/ui/views/tabs/tab_strip.cc
+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
@@ -488,6 +488,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
 
   std::vector<gfx::Rect> CalculateBoundsForDraggedViews(
       const std::vector<TabSlotView*>& views) override {
+    BRAVE_CALCULATE_BOUNDS_FOR_DRAGGED_VIEWS
     DCHECK(!views.empty());
 
     std::vector<gfx::Rect> bounds;
@@ -754,6 +755,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
         continue;
       }
 
+      BRAVE_CALCULATE_INSERTION_INDEX
       // If there's a group header here, and we're dragging a group, we might
       // end up on either side of that header. Check both cases to find the
       // best option.
