diff --git a/chrome/browser/ui/views/profiles/profile_menu_view.h b/chrome/browser/ui/views/profiles/profile_menu_view.h
index 2b876009054a70bff3801b7a7706d3bcadcb2f70..a42b8d27e42f4cc92094a57adbff52e66da6f085 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view.h
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.h
@@ -64,6 +64,7 @@ class ProfileMenuView : public ProfileMenuViewBase {
   }
 
  private:
+  friend class BraveProfileMenuView;
   friend class ProfileMenuViewExtensionsTest;
   friend class ProfileMenuViewSigninPendingTest;
   friend class ProfileMenuViewSignoutTest;
@@ -82,7 +83,7 @@ class ProfileMenuView : public ProfileMenuViewBase {
   void OnProfileManagementButtonClicked();
   void OnManageGoogleAccountButtonClicked();
   void OnGuestProfileButtonClicked();
-  void OnExitProfileButtonClicked();
+  virtual void OnExitProfileButtonClicked();
   void OnSyncSettingsButtonClicked();
   void OnGoogleServicesSettingsButtonClicked();
   void OnAccountSettingsButtonClicked();
@@ -123,12 +124,12 @@ class ProfileMenuView : public ProfileMenuViewBase {
   void MaybeBuildChromeAccountSettingsButton();
   void MaybeBuildGoogleServicesSettingsButton();
   void MaybeBuildManageGoogleAccountButton();
-  void MaybeBuildCloseBrowsersButton();
+  virtual void MaybeBuildCloseBrowsersButton();
 #if BUILDFLAG(ENABLE_DICE_SUPPORT)
   void MaybeBuildCrossDeviceSigninButton();
 #endif  // BUILDFLAG(ENABLE_DICE_SUPPORT)
   void MaybeBuildSignoutButton();
-  void BuildFeatureButtons();
+  virtual void BuildFeatureButtons();
   IdentitySectionParams GetIdentitySectionParams(
       const ProfileAttributesEntry& entry);
   void BuildIdentityWithCallToAction();
