diff --git a/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java
index fae1ca1624d5269e2f4424d2e18014d965dc81f2..199aae002b8f4bcc3e93aa9fbddc0a152357e602 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java
@@ -52,7 +52,7 @@ import java.util.Set;
 
 /** Implements {@link MultiInstanceOrchestrator} as a singleton. */
 @NullMarked
-/* package */ final class MultiInstanceOrchestratorImpl implements MultiInstanceOrchestrator {
+/* package */ class MultiInstanceOrchestratorImpl implements MultiInstanceOrchestrator {
     private static @Nullable MultiInstanceOrchestrator sInstance;
     private static @Nullable TabReparentingDelegate sTabReparentingDelegateForTesting;
 
@@ -74,7 +74,7 @@ import java.util.Set;
         return sInstance;
     }
 
-    private MultiInstanceOrchestratorImpl(TabReparentingDelegate tabReparentingDelegate) {
+    MultiInstanceOrchestratorImpl(TabReparentingDelegate tabReparentingDelegate) {
         mTabReparentingDelegate = tabReparentingDelegate;
         ApplicationStatus.registerStateListenerForAllActivities(this::onActivityStateChange);
 
