diff --git a/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarManager.java b/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarManager.java
index cc51dcbb68b9a309d0f4f8bdda17b70ead4431b6..278ac3c39dff44b5f67742d657a79998e8cc95cb 100644
--- a/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarManager.java
+++ b/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarManager.java
@@ -461,7 +461,7 @@ public class SnackbarManager
             boolean viewChanged = true;
             if (mView == null) {
                 mView =
-                        new SnackbarView(
+                        new BraveSnackbarView(
                                 mActivity,
                                 this,
                                 currentSnackbar,
