diff --git a/chrome/browser/ui/views/frame/browser_frame_view_win.cc b/chrome/browser/ui/views/frame/browser_frame_view_win.cc
index bd59ccd1b178450c7eba720ea8b235f24416a15e..40d21c79c71f8fb76dcaa8910ebd011db0443d19 100644
--- a/chrome/browser/ui/views/frame/browser_frame_view_win.cc
+++ b/chrome/browser/ui/views/frame/browser_frame_view_win.cc
@@ -488,6 +488,7 @@ int BrowserFrameViewWin::FrameTopBorderThickness(bool restored) const {
       // default. When maximized, the OS sizes the window such that the border
       // extends beyond the screen edges. In that case, we must return the
       // default value.
+      BRAVE_BROWSER_FRAME_VIEW_WIN_FRAME_TOP_BORDER_THICKNESS
       const int kTopResizeFrameArea = features::IsChromeRefresh2023() ? 1 : 5;
       return kTopResizeFrameArea;
     }
@@ -547,6 +548,7 @@ int BrowserFrameViewWin::TopAreaHeight(bool restored) const {
                      : caption_button_container_->GetPreferredSize().height();
     return top;
   }
+  BRAVE_BROWSER_FRAME_VIEW_WIN_TOP_AREA_HEIGHT
 
   // In Refresh, the tabstrip controls its own top padding.
   if (features::IsChromeRefresh2023()) {
