diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index d08a945c6b11a918fa5faedcb1c87c269772e877..1da2eecbb8d9421063a2adec7345424e2acbdbed 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -528,6 +528,15 @@ static_library("non_code_constants") {
     "env_vars.h",
   ]
 
+  if (brave_chromium_build) {
+    sources -= [
+      "chrome_icon_resources_win.h",
+    ]
+    sources += [
+      "//brave/chromium_src/chrome/common/chrome_icon_resources_win.h",
+    ]
+  }
+
   deps = [
     ":version_header",
     "//base",
@@ -554,6 +563,15 @@ static_library("constants") {
     "pref_names.h",
   ]
 
+  if (brave_chromium_build) {
+    if (is_mac) {
+      sources += [
+        "//brave/common/brave_paths_mac.mm",
+        "//brave/common/brave_paths_mac.h",
+      ]
+    }
+  }
+
   public_deps = [
     ":features",
     ":non_code_constants",
