diff --git a/chrome/browser/content_settings/BUILD.gn b/chrome/browser/content_settings/BUILD.gn
index 4130f3bd222983d433a54cc399dcf8f8dbe14ea6..05c62a12c5b5eee6a196c78b379883815cf1dd28 100644
--- a/chrome/browser/content_settings/BUILD.gn
+++ b/chrome/browser/content_settings/BUILD.gn
@@ -116,6 +116,7 @@ source_set("impl") {
   if (enable_pdf) {
     deps += [ "//chrome/browser/pdf" ]
   }
+  import("//brave/browser/content_settings/sources.gni") deps += brave_browser_content_settings_deps
 }
 
 # Fixes circular deps with different components. For example -
