diff --git a/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc b/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc
index 8de3e4300f760a7c5a9c82a27939eb3217342e4c..6a8ab7da6a5a95c74fc33cbdb4aad8292ac21e63 100644
--- a/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc
+++ b/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc
@@ -117,6 +117,7 @@ PermissionPromptBubbleView::PermissionPromptBubbleView(
 
   for (permissions::PermissionRequest* request : visible_requests_)
     AddPermissionRequestLine(request);
+  BRAVE_PERMISSION_PROMPT_BUBBLE_VIEW
 
   base::Optional<base::string16> extra_text = GetExtraText();
   if (extra_text.has_value()) {
