diff --git a/chrome/browser/ui/views/tabs/tab.h b/chrome/browser/ui/views/tabs/tab.h
index 2633dda37993f6f278769137cad245d4aaec5cc6..625a05a7548f85a8f17cf5f46a7d63c8356ddd82 100644
--- a/chrome/browser/ui/views/tabs/tab.h
+++ b/chrome/browser/ui/views/tabs/tab.h
@@ -307,7 +307,7 @@ class Tab : public gfx::AnimationDelegate,
 
   // Whether we are showing the close button. It is cached so that we can
   // detect when it changes and layout appropriately.
-  bool showing_close_button_ = false;
+  ControllableCloseButtonState showing_close_button_{*controller_, *this};
 
   // Whether the tab is currently animating from a pinned to an unpinned state.
   bool is_animating_from_pinned_ = false;
