diff --git a/chrome/browser/net/stub_resolver_config_reader.cc b/chrome/browser/net/stub_resolver_config_reader.cc
index c5c6ef452661a1fdf5b5b0aee98a7722872bfb06..618d1bd293809ddbd13dba804c44d7cc7ed90aec 100644
--- a/chrome/browser/net/stub_resolver_config_reader.cc
+++ b/chrome/browser/net/stub_resolver_config_reader.cc
@@ -357,6 +357,7 @@ SecureDnsConfig StubResolverConfigReader::GetAndUpdateConfiguration(
 
   std::string doh_templates =
       local_state_->GetString(prefs::kDnsOverHttpsTemplates);
+  BRAVE_GET_AND_UPDATE_CONFIGURATION
   std::string server_method;
   std::vector<net::DnsOverHttpsServerConfig> dns_over_https_servers;
   base::Optional<std::vector<network::mojom::DnsOverHttpsServerPtr>>
