diff --git a/chrome/browser/ui/views/profiles/profile_menu_view.h b/chrome/browser/ui/views/profiles/profile_menu_view.h
index 0420cbae191ec3a8f0a983834ba212a7024a31dd..d7c1ebc18bc8f9be09beec9871f41793b53dc362 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view.h
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.h
@@ -46,6 +46,7 @@ class ProfileMenuView : public ProfileMenuViewBase {
   gfx::ImageSkia GetSyncIcon() const override;
 
  private:
+  BRAVE_PROFILE_MENU_VIEW_H
   friend class ProfileMenuViewExtensionsTest;
   friend class ProfileMenuViewSignoutTest;
 
