diff --git a/chrome/browser/resources/settings/privacy_page/personalization_options.ts b/chrome/browser/resources/settings/privacy_page/personalization_options.ts
index 94286e33fbe61d710171ed9337c49a0d5d68857a..52e8617fd3ac8c1c9fe5a9fbfdb4c5346f112504 100644
--- a/chrome/browser/resources/settings/privacy_page/personalization_options.ts
+++ b/chrome/browser/resources/settings/privacy_page/personalization_options.ts
@@ -86,7 +86,7 @@ export class SettingsPersonalizationOptionsElement extends
 
       syncStatus: Object,
 
-      // <if expr="_google_chrome and not chromeos_ash">
+      // <if expr="not chromeos_ash">
       // TODO(dbeam): make a virtual.* pref namespace and set/get this normally
       // (but handled differently in C++).
       metricsReportingPref_: {
@@ -145,7 +145,7 @@ export class SettingsPersonalizationOptionsElement extends
   focusConfig: FocusConfig;
   syncStatus: SyncStatus;
 
-  // <if expr="_google_chrome and not chromeos_ash">
+  // <if expr="not chromeos_ash">
   private metricsReportingPref_: chrome.settingsPrivate.PrefObject<boolean>;
   private showRestart_: boolean;
   // </if>
@@ -187,7 +187,7 @@ export class SettingsPersonalizationOptionsElement extends
   override ready() {
     super.ready();
 
-    // <if expr="_google_chrome and not chromeos_ash">
+    // <if expr="not chromeos_ash">
     const setMetricsReportingPref = (metricsReporting: MetricsReporting) =>
         this.setMetricsReportingPref_(metricsReporting);
     this.addWebUiListener('metrics-reporting-change', setMetricsReportingPref);
@@ -202,9 +202,11 @@ export class SettingsPersonalizationOptionsElement extends
         this.setChromeSigninUserChoiceInfo_.bind(this));
     // </if>
 
+    // <if expr="_google_chrome">
     this.registerHelpBubble(
         ANONYMIZED_URL_COLLECTION_ID,
         this.$.urlCollectionToggle.getBubbleAnchor(), {anchorPaddingTop: 10});
+    // </if>
   }
 
   // <if expr="chromeos_ash">
@@ -225,7 +227,7 @@ export class SettingsPersonalizationOptionsElement extends
   }
   // </if>
 
-  // <if expr="_google_chrome and not chromeos_ash">
+  // <if expr="not chromeos_ash">
   private onMetricsReportingChange_() {
     const enabled = this.$.metricsReportingControl.checked;
     this.browserProxy_.setMetricsReportingEnabled(enabled);
