diff --git a/chrome/browser/global_features.cc b/chrome/browser/global_features.cc
index 3d21bd90a96ed2a7dcea5a7d97a47df7b640d1a7..342578b0e80572f0f2ae1a262ffd79189aa09f03 100644
--- a/chrome/browser/global_features.cc
+++ b/chrome/browser/global_features.cc
@@ -75,6 +75,7 @@
 #include "chrome/browser/on_device_translation/installer_impl.h"
 #endif  // BUILDFLAG(ENABLE_ON_DEVICE_TRANSLATION)
 
+GlobalFeatures* CreateBraveGlobalFeatures();
 namespace {
 
 // This is the generic entry point for test code to stub out browser
@@ -93,7 +94,7 @@ std::unique_ptr<GlobalFeatures> GlobalFeatures::CreateGlobalFeatures() {
     return GetFactory().Run();
   }
   // Constructor is protected.
-  return base::WrapUnique(new GlobalFeatures());
+  return base::WrapUnique(CreateBraveGlobalFeatures());
 }
 
 GlobalFeatures::~GlobalFeatures() = default;
