diff --git a/chrome/browser/resources/settings_shared/privacy_page/privacy_page_browser_proxy.ts b/chrome/browser/resources/settings_shared/privacy_page/privacy_page_browser_proxy.ts
index 95e28f9cb02b42589bfbb0760d6d4c7daac882f9..2c9fe5068c0c4f31e70d1699d975e886af5bf7b0 100644
--- a/chrome/browser/resources/settings_shared/privacy_page/privacy_page_browser_proxy.ts
+++ b/chrome/browser/resources/settings_shared/privacy_page/privacy_page_browser_proxy.ts
@@ -62,7 +62,7 @@ export interface SecureDnsSetting {
 }
 
 export interface PrivacyPageBrowserProxy {
-  // <if expr="_google_chrome and not is_chromeos">
+  // <if expr="True and not is_chromeos">
   getMetricsReporting(): Promise<MetricsReporting>;
   setMetricsReportingEnabled(enabled: boolean): void;
 
@@ -85,7 +85,7 @@ export interface PrivacyPageBrowserProxy {
 }
 
 export class PrivacyPageBrowserProxyImpl implements PrivacyPageBrowserProxy {
-  // <if expr="_google_chrome and not is_chromeos">
+  // <if expr="True and not is_chromeos">
   getMetricsReporting() {
     return sendWithPromise('getMetricsReporting');
   }
