diff --git a/chrome/browser/sync/sync_service_factory.cc b/chrome/browser/sync/sync_service_factory.cc
index d8f58453d9c970d3191d9da7ccbb1111a0c38ac0..0d36f204000405ea6fb6fb91598ab37d7dba2ba8 100644
--- a/chrome/browser/sync/sync_service_factory.cc
+++ b/chrome/browser/sync/sync_service_factory.cc
@@ -434,7 +434,7 @@ std::unique_ptr<KeyedService> BuildSyncService(
   }
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize(CreateControllers(profile, sync_service.get()));
 
   // Notify the PasswordStore of complete initialisation to resolve a circular
