diff --git a/chrome/android/chrome_java_sources.gni b/chrome/android/chrome_java_sources.gni
index 643c526d0b6083c0f353f89287bfbec5289d1f83..b992647e83aae3422d87525948860502476bb4c7 100644
--- a/chrome/android/chrome_java_sources.gni
+++ b/chrome/android/chrome_java_sources.gni
@@ -1174,3 +1174,4 @@ if (enable_screen_capture) {
     "java/src/org/chromium/chrome/browser/media/MediaCapturePickerItemViewBinder.java",
   ]
 }
+chrome_java_sources -= ["java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPageView.java",]
