diff --git a/chrome/browser/ui/views/page_info/page_info_main_view.cc b/chrome/browser/ui/views/page_info/page_info_main_view.cc
index 6bff9b2a9f5ed4d1862fa280dfc3bdd457242d10..39ad7785834cedbe4e0c29c69399def340cefec9 100644
--- a/chrome/browser/ui/views/page_info/page_info_main_view.cc
+++ b/chrome/browser/ui/views/page_info/page_info_main_view.cc
@@ -525,6 +525,7 @@ gfx::Size PageInfoMainView::CalculatePreferredSize() const {
   if (site_settings_view_) {
     width = std::max(width, site_settings_view_->GetPreferredSize().width());
     width = std::max(width, permissions_view_->GetPreferredSize().width());
+    BRAVE_PAGE_INFO_MAIN_VIEW_CALCULATE_PREFERRED_SIZE
   }
   return gfx::Size(width, views::View::GetHeightForWidth(width));
 }
