diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index e5744a9d05d3099a61815954b08c7c030d081071..41ba370a6aee0b7d492e65c529d0cd1877cab57b 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -247,7 +247,7 @@ grit("google_chrome_strings") {
 }
 
 grit("chromium_strings") {
-  source = "chromium_strings.grd"
+  source = "${branding_path_component}_strings.grd"
   defines = chrome_grit_defines
   output_dir = "$root_gen_dir/chrome"
   outputs = [
