diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index df1e9276ae1ac46f1b25344c53052059a59f49fc..28608889c4b353b2b21345ffc143c5c244f57b57 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -105,6 +105,7 @@ source_set("channel_info") {
   } else if (is_posix) {
     sources += [ "channel_info_posix.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -281,6 +282,7 @@ static_library("common_lib") {
   if (enable_ppapi) {
     public_deps += [ "//ppapi/shared_impl" ]
   }
+  import("//brave/common/sources.gni") sources += brave_chrome_common_sources deps += brave_chrome_common_deps public_deps += brave_chrome_common_public_deps
 
   if (enable_extensions) {
     sources += [
@@ -546,6 +548,7 @@ static_library("non_code_constants") {
   if (is_win) {
     sources += [ "chrome_icon_resources_win.h" ]
   }
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
