diff --git a/chrome/common/extensions/BUILD.gn b/chrome/common/extensions/BUILD.gn
index 371904a3984e27a7d3c259410d4d0c0e23e6f035..4190c3b9dbea1b22edc3bc1e8fdf39e5ddd4d813 100644
--- a/chrome/common/extensions/BUILD.gn
+++ b/chrome/common/extensions/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/common/extensions/sources.gni")
 import("//extensions/buildflags/buildflags.gni")
 import("//tools/json_schema_compiler/json_features.gni")
 
@@ -26,6 +27,7 @@ source_set("extensions") {
     "permissions/chrome_permission_message_rules.h",
     "sync_helper.h",
   ]
+  public += brave_chrome_common_extensions_public
 
   # Files that can only be included from here.
   sources = [
@@ -50,6 +52,7 @@ source_set("extensions") {
     "permissions/chrome_permission_message_rules.cc",
     "sync_helper.cc",
   ]
+  sources += brave_chrome_common_extensions_sources
 
   public_deps = [
     "//base",
@@ -62,6 +65,7 @@ source_set("extensions") {
     "//ui/gfx/geometry",
     "//url",
   ]
+  public_deps += brave_chrome_common_extensions_public_deps
 
   deps = [
     "//chrome/app:branded_strings",
@@ -88,6 +92,7 @@ source_set("extensions") {
     "//ui/message_center/public/cpp",
     "//url",
   ]
+  deps += brave_chrome_common_extensions_deps
 
   # TODO(https://crbug.com/356905053): These sources do not currently build
   # cleanly on the experimental desktop-android configuration. Gradually include
