diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 93495ccb9c54464ebaf44b8ed44ebd7103f1dc40..4fef12949c0bb97a87cba5d5c6095cb313a920d8 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3266,6 +3266,7 @@ const FeatureEntry::FeatureVariation kPrivacySandboxSettings4Variations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
