diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index d9d984964068c06a9be4659f353fbe0162090e43..c7cb0c0e9f740cfd5353b76544b1bfcb7ea91fde 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -91,6 +91,7 @@ grit_strings("generated_resources") {
   if (is_android) {
     create_android_resources = true
   }
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
