diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 5aa3c3fee46d27c4dd363e7398404966bd755e90..c6a50819abae1fdadfc294b854988c0215cda4ae 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3260,6 +3260,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
