diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 147de9967aee1ecc53987d7a038c42f609351801..1b4605173d2bda9b0937107870fb827e787a79a9 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -4607,6 +4607,7 @@ source_set("browser") {
   if (!is_android) {
     deps += [ "//chrome/browser/contextual_cueing/internals" ]
   }
+  public_deps += [ "//brave/browser:core" ]
 
   # This target intentionally has no sources of its own and no
   # allow_circular_includes_from; it only aggregates dependencies. Do NOT add
@@ -4776,6 +4777,7 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
