diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 388633d91746d1d613c8b53455c6f81ca71d61d1..23fafc7202c53175f18f24bd1ceedbec4dc9817f 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -437,6 +437,8 @@ test("browser_tests") {
     "//third_party/blink/public:buildflags",
   ]
 
+  deps += [ "//brave/test:brave_browser_tests_deps", ]
+
   data_deps = [
     "//testing/buildbot/filters:browser_tests_filters",
     "//tools/media_engagement_preload:generator",
