diff --git a/chrome/browser/ssl/https_upgrades_util.cc b/chrome/browser/ssl/https_upgrades_util.cc
index 330ce07feb36770d4604155266b375794c836ad0..92e98c0f61aea89279815e80b58fd438bffecca2 100644
--- a/chrome/browser/ssl/https_upgrades_util.cc
+++ b/chrome/browser/ssl/https_upgrades_util.cc
@@ -72,11 +72,11 @@ ComputeInterstitialState(content::WebContents* web_contents, const GURL& url) {
   security_interstitials::https_only_mode::HttpInterstitialState
       interstitial_state;
   interstitial_state.enabled_by_pref =
-      prefs && prefs->GetBoolean(prefs::kHttpsOnlyModeEnabled);
+      prefs && prefs->GetBooleanOr(prefs::kHttpsOnlyModeEnabled, NormalWindowHttpsOnly(url, profile));
 
   if (base::FeatureList::IsEnabled(features::kHttpsFirstModeIncognito)) {
     if (profile->IsIncognitoProfile() && prefs &&
-        prefs->GetBoolean(prefs::kHttpsFirstModeIncognito)) {
+        prefs->GetBooleanOr(prefs::kHttpsFirstModeIncognito, NormalWindowHttpsOnly(url, profile))) {
       interstitial_state.enabled_by_incognito = true;
     }
   }
