diff --git a/chrome/browser/resources/bookmarks/command_manager.ts b/chrome/browser/resources/bookmarks/command_manager.ts
index 46e8b719c14d0ac57209b847425b6866e2520349..fcbbb07b01259ddc98590013cae534c200e67a06 100644
--- a/chrome/browser/resources/bookmarks/command_manager.ts
+++ b/chrome/browser/resources/bookmarks/command_manager.ts
@@ -434,7 +434,7 @@ export class BookmarksCommandManagerElement extends
         chrome.bookmarkManagerPrivate.export();
         break;
       case Command.HELP_CENTER:
-        window.open('https://support.google.com/chrome/?p=bookmarks');
+        window.open('https://community.brave.com', undefined, 'noreferrer');
         break;
       default:
         assertNotReached();
