diff --git a/chrome/browser/ui/views/tabs/tab_strip.cc b/chrome/browser/ui/views/tabs/tab_strip.cc
index 7695b7feb4c4600d0ecf433f4cf29c2cce4be2f4..20a1834f07057b4b5501fbbe19cd52c28523a155 100644
--- a/chrome/browser/ui/views/tabs/tab_strip.cc
+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
@@ -476,6 +476,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
 
   std::vector<gfx::Rect> CalculateBoundsForDraggedViews(
       const std::vector<TabSlotView*>& views) override {
+    BRAVE_CALCULATE_BOUNDS_FOR_DRAGGED_VIEWS
     DCHECK(!views.empty());
 
     std::vector<gfx::Rect> bounds;
@@ -720,6 +721,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
         continue;
       }
 
+      BRAVE_CALCULATE_INSERTION_INDEX
       // If there's a group header here, and we're dragging a group, we might
       // end up on either side of that header. Check both cases to find the
       // best option.
