diff --git a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
index e7a086296bb065c2bb2cade4d9661e3741679d81..63f3acfea930d9e981fbde7c1f018d63c2a67739 100644
--- a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
+++ b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
@@ -86,6 +86,7 @@ class FrameGrabHandle : public views::View {
   void Layout(PassKey) override {
     LayoutSuperclass<views::View>(this);
 
+    if (!dialog_anchor_) { return; }
     int x = width() * 0.4;
     int y = height() * 0.7;
     dialog_anchor_->MaybeUpdateAnchor(gfx::Rect(x, y, 0, 0));
@@ -148,14 +149,14 @@ std::unique_ptr<TabStrip> CreateTabStrip(
             ->GetTabMenuModelFactory();
   }
 
-  auto tabstrip_controller = std::make_unique<BrowserTabStripController>(
+  auto tabstrip_controller = std::make_unique<BraveBrowserTabStripController>(
       browser_view->browser()->GetTabStripModel(), browser_view,
       std::move(tab_menu_model_factory));
 
-  std::unique_ptr<TabHoverCardController> hover_card_controller(
-      std::make_unique<TabHoverCardController>(tab_strip_region_view,
+  std::unique_ptr<BraveTabHoverCardController> hover_card_controller(
+      std::make_unique<BraveTabHoverCardController>(tab_strip_region_view,
                                                browser_view->browser()));
-  auto tab_strip = std::make_unique<TabStrip>(std::move(tabstrip_controller),
+  auto tab_strip = std::make_unique<BraveTabStrip>(std::move(tabstrip_controller),
                                               std::move(hover_card_controller));
   return tab_strip;
 }
@@ -232,7 +233,7 @@ HorizontalTabStripRegionViewOld::HorizontalTabStripRegionViewOld(
 
   if (ShouldShowNewTabButton(browser)) {
     std::unique_ptr<TabStripControlButton> tab_strip_control_button =
-        std::make_unique<NewTabButton>(
+        std::make_unique<BraveNewTabButton>(
             base::BindRepeating(&TabStrip::NewTabButtonPressed,
                                 base::Unretained(tab_strip_)),
             features::IsRoundedIconsEnabled() ? vector_icons::kAddWeight500Icon
@@ -475,14 +476,14 @@ void HorizontalTabStripRegionViewOld::InitializeTabStrip() {
   }
 
   tab_strip_->Initialize();
-  static_cast<BrowserTabStripController*>(tab_strip_->controller())
+  static_cast<BraveBrowserTabStripController*>(tab_strip_->controller())
       ->InitFromModel(tab_strip_);
   tab_strip_set_ = true;
 }
 
 void HorizontalTabStripRegionViewOld::ResetTabStrip() {
   tab_strip_set_ = false;
-  static_cast<BrowserTabStripController*>(tab_strip_->controller())->Reset();
+  static_cast<BraveBrowserTabStripController*>(tab_strip_->controller())->Reset();
   tab_strip_->Reset();
 }
 
@@ -605,7 +606,7 @@ void HorizontalTabStripRegionViewOld::UpdateButtonBorders() {
   const int extra_vertical_space =
       GetLayoutConstant(LayoutConstant::kTabStripHeight) -
       GetLayoutConstant(LayoutConstant::kTabstripToolbarOverlap) -
-      NewTabButton::kButtonSize.height();
+      BraveNewTabButton::GetButtonSize().height();
   const int top_inset = extra_vertical_space / 2;
   const int bottom_inset =
       extra_vertical_space - top_inset +
