diff --git a/chrome/android/javatests/BUILD.gn b/chrome/android/javatests/BUILD.gn
index 09485ce2bb525b1d9276e8facd81878c547e77ac..096e640ba23c082997b89f71216a0430c968f030 100644
--- a/chrome/android/javatests/BUILD.gn
+++ b/chrome/android/javatests/BUILD.gn
@@ -1712,6 +1712,7 @@ android_library("chrome_unit_test_java_various") {
     "src/org/chromium/chrome/browser/toolbar/ToolbarSecurityIconTest.java",
     "src/org/chromium/chrome/browser/webapps/WebappLaunchCauseMetricsTest.java",
   ]
+  sources -= [ "src/org/chromium/chrome/browser/ntp/IncognitoDescriptionViewRenderTest.java" ]
 
   deps = chrome_unit_test_java_deps
 }
