diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index dfef0e5bc5ef235eeff2dd4048bb81d58b10e2eb..cd5f16857213deccc186963721b3cae456b94bdf 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -875,6 +875,7 @@ void BookmarkBarView::Layout(PassKey) {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   const bool show_bookmarks_overflow =
       bookmark_service_->loaded() &&
