diff --git a/chrome/browser/ui/tabs/tab_strip_model.cc b/chrome/browser/ui/tabs/tab_strip_model.cc
index abc905f35921d0ae3bd8d5e3b19213aead81fb06..81fa2ea4f76d8c9f773a9787ca54411a679b9749 100644
--- a/chrome/browser/ui/tabs/tab_strip_model.cc
+++ b/chrome/browser/ui/tabs/tab_strip_model.cc
@@ -1820,6 +1820,7 @@ void TabStripModel::AddTab(std::unique_ptr<tabs::TabModel> tab,
   if (ui::PageTransitionTypeIncludingQualifiersIs(transition,
                                                   ui::PAGE_TRANSITION_TYPED) &&
       index == count()) {
+    tab->set_opener_was_set_for_empty_new_tab();
     // Also, any tab opened at the end of the TabStrip with a "TYPED"
     // transition inherit opener as well. This covers the cases where the user
     // creates a New Tab (e.g. Ctrl+T, or clicks the New Tab button), or types
@@ -2627,6 +2628,15 @@ void TabStripModel::ExecuteContextMenuCommand(int context_index,
     return;
   }
   switch (command_id) {
+    case CommandRestoreTab:
+    case CommandBookmarkAllTabs:
+    case CommandShowVerticalTabs:
+    case CommandToggleTabMuted:
+    case CommandBringAllTabsToThisWindow:
+    case CommandCloseDuplicateTabs:
+    case CommandOpenInContainer:
+    case CommandRenameTab:
+      NOTREACHED();
     case CommandNewTabToRight: {
       base::UmaHistogramCounts1000(
           "Tab.ContextMenu.NewTabToRight.SelectedTabsCount",
@@ -4177,7 +4187,7 @@ void TabStripModel::MoveTabRelative(TabRelativeDirection direction) {
 
 std::pair<std::optional<int>, std::optional<int>>
 TabStripModel::GetAdjacentTabsAfterSelectedMove(
-    base::PassKey<DraggingTabsSession>,
+    base::PassKey<DraggingTabsSessionChromium>,
     int destination_index) {
   const int pinned_tab_count = IndexOfFirstNonPinnedTab();
   const std::vector<int> pinned_selected_indices = GetSelectedPinnedTabs();
@@ -4650,7 +4660,9 @@ void TabStripModel::InsertTabAtIndexImpl(
   }
 
   tabs::TabInterface* old_active_tab = GetActiveTab();
-  contents_data_->AddTabRecursive(std::move(tab_model), index, group, pin);
+  auto* opener = tab_model->opener_was_set_for_empty_new_tab() ? nullptr : tab_model->opener();
+  contents_data_->AddTabRecursive(std::move(tab_model), index, group, pin, opener);
+
   selection_model_.InvalidateListSelectionModel(base::PassKey<TabStripModel>());
 
   // Start computing selection change after updating the indices in
