diff --git a/chrome/browser/upgrade_detector/upgrade_detector_impl.cc b/chrome/browser/upgrade_detector/upgrade_detector_impl.cc
index d9d001527f27..899fde129c96 100644
--- a/chrome/browser/upgrade_detector/upgrade_detector_impl.cc
+++ b/chrome/browser/upgrade_detector/upgrade_detector_impl.cc
@@ -115,7 +115,11 @@ UpgradeDetectorImpl::~UpgradeDetectorImpl() {
 
 // static
 base::Version UpgradeDetectorImpl::GetCurrentlyInstalledVersion() {
+#if !defined(OS_ANDROID)
   return GetInstalledVersion().installed_version;
+#else
+  return base::Version();
+#endif
 }
 
 void UpgradeDetectorImpl::StartUpgradeNotificationTimer() {
@@ -460,7 +464,7 @@ void UpgradeDetectorImpl::Init() {
 
   // On Windows, only enable upgrade notifications for Google Chrome builds.
   // Chromium does not use an auto-updater.
-#if !defined(OS_WIN) || BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if !defined(OS_WIN) || BUILDFLAG(GOOGLE_CHROME_BRANDING) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))
 
   // On macOS, only enable upgrade notifications if the updater (Keystone) is
   // present.
