diff --git a/chrome/browser/ui/views/page_action/page_action_properties_provider.cc b/chrome/browser/ui/views/page_action/page_action_properties_provider.cc
index e29cd0260c78d383db9ca04e4d824ef630190616..b604aa71533f3e626352b20399444869b3153a65 100644
--- a/chrome/browser/ui/views/page_action/page_action_properties_provider.cc
+++ b/chrome/browser/ui/views/page_action/page_action_properties_provider.cc
@@ -191,6 +191,13 @@ constexpr auto kPageActionProperties = base::MakeFixedFlatMap<
             .element_identifier = kAutofillAddressPageActionElementId,
         },
     },
+    {
+        kActionShowPartitionedStorage,
+        {
+            .histogram_name = "PartitionedStorage",
+            .type = brave::kPartitionedStorageActionIconType,
+        },
+    },
     {
         kActionVirtualCardEnroll,
         {
