diff --git a/chrome/browser/google/BUILD.gn b/chrome/browser/google/BUILD.gn
index d8af9201cc4798efa1643db445a20e3bf1800937..ea7148500a57efa27d5d7cbc5bbe9e5be51d2b8f 100644
--- a/chrome/browser/google/BUILD.gn
+++ b/chrome/browser/google/BUILD.gn
@@ -78,6 +78,7 @@ source_set("google") {
       "//chrome/updater:tagging",
     ]
   }
+  import("//brave/chromium_src/chrome/browser/google/sources.gni") sources += brave_chrome_browser_google_sources
 }
 
 source_set("unit_tests") {
