diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 245a7102ec7bb8f568bd760b8bea8e5e96c2655c..d861f6a0a89641c7cc789108424d9b0257794a48 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -121,6 +121,7 @@ jumbo_static_library("renderer") {
   defines = []
 
   deps = [
+    "//brave/renderer",
     "//chrome:resources",
     "//chrome:strings",
     "//chrome/common",
