diff --git a/chrome/browser/actor/ui/task_list_bubble/actor_task_list_bubble_row_button.cc b/chrome/browser/actor/ui/task_list_bubble/actor_task_list_bubble_row_button.cc
index 2ee4069426f78c09690124b79f641c8f7c60846e..eb5789abac729f864d8b741925329e89c0db7745 100644
--- a/chrome/browser/actor/ui/task_list_bubble/actor_task_list_bubble_row_button.cc
+++ b/chrome/browser/actor/ui/task_list_bubble/actor_task_list_bubble_row_button.cc
@@ -41,10 +41,12 @@ const gfx::VectorIcon& GetRowIcon(actor::ActorTask::State state) {
 
 ui::ColorId GetRowColor(actor::ActorTask::State state,
                         bool requires_processing) {
+#if BUILDFLAG(ENABLE_GLIC)
   if (requires_processing &&
       tabs::GlicActorTaskIconManager::RequiresAttention(state)) {
     return ui::kColorSysPrimary;
   }
+#endif
   return ui::kColorMenuIcon;
 }
 
@@ -53,10 +55,13 @@ std::u16string GetRowSubtitle(actor::ActorTask::State state, bool has_tab) {
     return l10n_util::GetStringUTF16(
         IDR_ACTOR_TASK_LIST_BUBBLE_ROW_TAB_CLOSED_SUBTITLE);
   }
+#if BUILDFLAG(ENABLE_GLIC)
   if (tabs::GlicActorTaskIconManager::RequiresAttention(state)) {
     return l10n_util::GetStringUTF16(
         IDR_ACTOR_TASK_LIST_BUBBLE_ROW_CHECK_TASK_SUBTITLE);
-  } else if (state == actor::ActorTask::State::kFinished) {
+  } else
+#endif
+  if (state == actor::ActorTask::State::kFinished) {
     return l10n_util::GetStringUTF16(
         IDR_ACTOR_TASK_LIST_BUBBLE_ROW_COMPLETED_TASK_SUBTITLE);
   } else if (state == actor::ActorTask::State::kFailed) {
