diff --git a/chrome/browser/prefs/chrome_pref_service_factory.cc b/chrome/browser/prefs/chrome_pref_service_factory.cc
index f7ee08f67436ff5c763321bba557e44fd9651b78..7e06aa760fc7afe7d2660ac26c1e95da5a0cf14c 100644
--- a/chrome/browser/prefs/chrome_pref_service_factory.cc
+++ b/chrome/browser/prefs/chrome_pref_service_factory.cc
@@ -179,6 +179,7 @@ const auto kTrackedPrefs = std::to_array<prefs::TrackedPreferenceMetadata>({
 
     // See note at top, new items added here also need to be added to
     // histograms.xml's TrackedPreference enum.
+    BRAVE_TRACKED_PREFS_EXTEND
 });
 
 // One more than the last tracked preferences ID above.
