diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index e63afefc769d6ea3657eafa3013bfa851393a239..01f8c54a4442fcb421646fc68f0e30de298026ba 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -203,6 +203,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
