diff --git a/chrome/browser/web_applications/components/BUILD.gn b/chrome/browser/web_applications/components/BUILD.gn
index 1ef11da47440..9ce495a5c143 100644
--- a/chrome/browser/web_applications/components/BUILD.gn
+++ b/chrome/browser/web_applications/components/BUILD.gn
@@ -138,6 +138,7 @@ source_set("components") {
     "//services/preferences/public/cpp",
     "//skia",
   ]
+  if (is_android) { deps += ["//brave/browser/web_applications/components"] }
 
   if (is_win) {
     deps += [ "//chrome/browser/web_applications/chrome_pwa_launcher:util" ]
@@ -222,4 +223,5 @@ source_set("browser_tests") {
     "//chrome/test:test_support_ui",
     "//net:test_support",
   ]
+  if (is_android) { deps -= ["//chrome/test:test_support", "//chrome/test:test_support_ui",] }
 }
