diff --git a/chrome/browser/resources/settings/people_page/sync_controls.ts b/chrome/browser/resources/settings/people_page/sync_controls.ts
index 1e5afaf147008936350b929ba416d992423bee00..ead63e5d272c6bbde6d42db1a07c62291c4843aa 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.ts
+++ b/chrome/browser/resources/settings/people_page/sync_controls.ts
@@ -145,7 +145,7 @@ export class SettingsSyncControlsElement extends
 
     const router = Router.getInstance();
     const currentRoute = router.getCurrentRoute();
-    if (currentRoute === routes.SYNC_ADVANCED) {
+    if (currentRoute === routes.BRAVE_SYNC_SETUP) {
       this.syncBrowserProxy_.didNavigateToSyncPage();
     }
     if (loadTimeData.getBoolean('replaceSyncPromosWithSignInPromos') &&
@@ -377,6 +377,7 @@ export class SettingsSyncControlsElement extends
 
   protected disableTypeCheckBox_(dataTypeManaged: boolean|undefined|null):
       boolean {
+    if (dataTypeManaged) { return true; } if (!this.syncStatus || !this.syncPrefs) { return !!this.syncPrefs?.syncAllDataTypes; }
     if (!this.syncStatus) {
       return true;
     }
@@ -386,7 +387,7 @@ export class SettingsSyncControlsElement extends
     }
 
     if (this.syncStatus.signedInState === SignedInState.SYNCING) {
-      return !!this.syncPrefs?.syncAllDataTypes;
+      return this.syncPrefs?.syncAllDataTypes;
     }
 
     // Toggles should be disabled on the account settings page if sync is
