diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index fdb844c5ad1bfaa3da7dd6e91bba7140951e62b8..6d7d59df660befa3a102aad7857ee041931bb4c8 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -78,6 +78,7 @@ source_set("configs") {
   if (enable_glic) {
     deps += [ "//chrome/browser/glic" ]
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("webui") {
