diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 38d736c0ee18a36ae6da0f7d7d2a30d56d6ece30..c8c6365c4aa52e23f6e1de8bff9811d894554d58 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -81,6 +81,7 @@ source_set("channel_info") {
   } else if (is_posix) {
     sources += [ "channel_info_posix.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -548,6 +549,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
