diff --git a/chrome/browser/ui/browser_window/public/browser_window_features.h b/chrome/browser/ui/browser_window/public/browser_window_features.h
index d4424e85e152d728de64abe2a2f57769b137328a..23184f61ccf16435d0e77545372001d41be7f1a5 100644
--- a/chrome/browser/ui/browser_window/public/browser_window_features.h
+++ b/chrome/browser/ui/browser_window/public/browser_window_features.h
@@ -260,7 +260,7 @@ class AiOverlayDialogController;
 class BrowserWindowFeatures {
  public:
   BrowserWindowFeatures();
-  ~BrowserWindowFeatures();
+  virtual ~BrowserWindowFeatures();
 
   BrowserWindowFeatures(const BrowserWindowFeatures&) = delete;
   BrowserWindowFeatures& operator=(const BrowserWindowFeatures&) = delete;
