diff --git a/chrome/browser/net/profile_network_context_service.cc b/chrome/browser/net/profile_network_context_service.cc
index 04ea60a87f56e5f08998d61d2d16a86487468d34..47ceb2761858876865edbe58b82c9159b25bd7d2 100644
--- a/chrome/browser/net/profile_network_context_service.cc
+++ b/chrome/browser/net/profile_network_context_service.cc
@@ -641,6 +641,7 @@ network::mojom::CTPolicyPtr ProfileNetworkContextService::GetCTPolicy() {
   std::vector<std::string> excluded_spkis(
       TranslateStringArray(ct_excluded_spkis));
 
+  BRAVE_PROFILE_NETWORK_CONTEXT_SERVICE_GET_CT_POLICY
   return network::mojom::CTPolicy::New(std::move(excluded),
                                        std::move(excluded_spkis));
 }
