diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page.ts b/chrome/browser/resources/settings/privacy_page/privacy_page.ts
index 4ab4d784d4fe5b04e6e5406e2a4a4c2399292380..c26e1367c1c900e8272dd03a0b480f9268a5b3fb 100644
--- a/chrome/browser/resources/settings/privacy_page/privacy_page.ts
+++ b/chrome/browser/resources/settings/privacy_page/privacy_page.ts
@@ -16,6 +16,7 @@ import '../icons.html.js';
 import '../privacy_icons.html.js';
 import '../settings_page/settings_section.js';
 import '../settings_shared.css.js';
+import './personalization_options.js';
 import './privacy_guide/privacy_guide_dialog.js';
 
 import {PrefsMixin} from '/shared/settings/prefs/prefs_mixin.js';
