diff --git a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java
index bb7fa6b2bcdb853e6f4936b03bce28ca1a2185d6..e1acd87263bbb9160cc9b5e7420f465ee995136d 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java
@@ -157,7 +157,7 @@ public class MostVisitedTilesCoordinator implements ConfigurationChangedObserver
         mUserEducationHelper = new UserEducationHelper(mActivity, profile, handler);
 
         mContextMenuManager =
-                new ContextMenuManager(
+                new org.chromium.chrome.browser.native_page.BraveContextMenuManager(
                         suggestionsUiDelegate.getNavigationDelegate(),
                         touchEnabledDelegate,
                         mActivity::closeContextMenu,
