diff --git a/chrome/test/base/testing_browser_process.cc b/chrome/test/base/testing_browser_process.cc
index 2d753f99ec6859e5dfa67c70fd722b3255cf19fd..571b3ebd7e2730e82cd6df3bfd7f4e5ce40c493e 100644
--- a/chrome/test/base/testing_browser_process.cc
+++ b/chrome/test/base/testing_browser_process.cc
@@ -130,10 +130,12 @@ void TestingBrowserProcess::CreateInstance() {
         std::move(fake_geolocation_system_permission_manager));
   }
 #endif  // BUILDFLAG(OS_LEVEL_GEOLOCATION_PERMISSION_SUPPORTED)
+  BRAVE_TESTING_BROWSER_PROCESS_CREATE_INSTANCE
 }
 
 // static
 void TestingBrowserProcess::DeleteInstance() {
+  BRAVE_TESTING_BROWSER_PROCESS_DELETE_INSTANCE
   // g_browser_process must be null during its own destruction.
   BrowserProcess* browser_process = g_browser_process;
   g_browser_process = nullptr;
@@ -142,6 +144,7 @@ void TestingBrowserProcess::DeleteInstance() {
 
 // static
 void TestingBrowserProcess::TearDownAndDeleteInstance() {
+  BRAVE_TESTING_BROWSER_PROCESS_TEAR_DOWN_AND_DELETE_INSTANCE
   TestingBrowserProcess::DeleteInstance();
 }
 
