diff --git a/chrome/browser/external_protocol/external_protocol_handler.cc b/chrome/browser/external_protocol/external_protocol_handler.cc
index 9d7f383e3878f0410ca7e7f34d7941b60847c097..4b9db45c891ea676f0f19c2769bdf26d945b521d 100644
--- a/chrome/browser/external_protocol/external_protocol_handler.cc
+++ b/chrome/browser/external_protocol/external_protocol_handler.cc
@@ -414,7 +414,7 @@ ExternalProtocolHandler::BlockState ExternalProtocolHandler::GetBlockState(
   if (scheme == "mailto") {
     base::UmaHistogramEnumeration(kBlockStateMetric,
                                   BlockStateMetric::kAllowedDefaultMail);
-    return DONT_BLOCK;
+    return UNKNOWN;
   }
 
   PrefService* profile_prefs = profile->GetPrefs();
