diff --git a/chrome/browser/prefs/pref_service_incognito_allowlist.cc b/chrome/browser/prefs/pref_service_incognito_allowlist.cc
index db262016de186c9deb760feea11c7a8b1bf97163..40e72dcab6ce831be7c6035c85da216c79caeb99 100644
--- a/chrome/browser/prefs/pref_service_incognito_allowlist.cc
+++ b/chrome/browser/prefs/pref_service_incognito_allowlist.cc
@@ -28,6 +28,7 @@ namespace {
 // List of keys that can be changed in the user prefs file by the incognito
 // profile.
 const char* const kPersistentPrefNames[] = {
+    bookmarks::prefs::kAlwaysShowBookmarkBarOnNTP,
 #if BUILDFLAG(IS_CHROMEOS)
     // Accessibility preferences should be persisted if they are changed in
     // incognito mode.
@@ -227,10 +228,16 @@ const char* const kPersistentPrefNames[] = {
 namespace prefs {
 
 std::vector<const char*> GetIncognitoPersistentPrefsAllowlist() {
+  std::vector<const char*> result = [&]() -> std::vector<const char*> {
   std::vector<const char*> allowlist;
   allowlist.insert(allowlist.end(), std::begin(kPersistentPrefNames),
                    std::end(kPersistentPrefNames));
   return allowlist;
+  }();
+  for (auto pref : brave::GetBravePersistentPrefNames()) {
+    result.push_back(pref.data());
+  }
+  return result;
 }
 
 }  // namespace prefs
