diff --git a/chrome/browser/ui/color/BUILD.gn b/chrome/browser/ui/color/BUILD.gn
index 3dba3ad26c4770bfe4320625357b05b49f0e9c46..7a3db7510317c1c59af938d028b6da8e0d8d65f2 100644
--- a/chrome/browser/ui/color/BUILD.gn
+++ b/chrome/browser/ui/color/BUILD.gn
@@ -53,6 +53,7 @@ source_set("mixers") {
     "//ui/color:color",
     "//ui/color:mixers",
   ]
+  sources += brave_browser_ui_color_mixers_sources deps += brave_browser_ui_color_mixers_deps
 
   if (is_chromeos) {
     sources += [ "chromeos/native_chrome_color_mixer_chromeos.cc" ]
