diff --git a/chrome/browser/resources/settings/settings_page/main_page_mixin.ts b/chrome/browser/resources/settings/settings_page/main_page_mixin.ts
index ba490b2af49db5ea3bb46fa4068d5e90719cd221..5e0856719d87c69595425b648956c9dfedc52c1a 100644
--- a/chrome/browser/resources/settings/settings_page/main_page_mixin.ts
+++ b/chrome/browser/resources/settings/settings_page/main_page_mixin.ts
@@ -41,7 +41,7 @@ guestTopLevelRoute = routes.PRIVACY;
 // </if>
 
 const TOP_LEVEL_EQUIVALENT_ROUTE: Route =
-    loadTimeData.getBoolean('isGuest') ? guestTopLevelRoute : routes.PEOPLE;
+    loadTimeData.getBoolean('isGuest') ? guestTopLevelRoute : routes.GET_STARTED;
 
 function classifyRoute(route: Route|null): RouteState {
   if (!route) {
