diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index 85ea634a7c6c38ba0c9fa1a8dc3987fbe85897eb..31d8ecafb34f0fe5678f0c36a318997c0b30cf28 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -8,6 +8,7 @@ import {pageVisibility} from './page_visibility.js';
 import {Route, Router} from './router.js';
 import {SettingsRoutes} from './settings_routes.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -215,6 +216,7 @@ function createBrowserSettingsRoutes(): SettingsRoutes {
       // </if>
     }
   }
+  addBraveRoutes(r);
   return r;
 }
 
