diff --git a/chrome/browser/resources/password_manager/BUILD.gn b/chrome/browser/resources/password_manager/BUILD.gn
index 678502e308c36d813ee0fe3c7ad1a08fd709933b..7d59e81d9b51c25484c1ca88992f6312c02128c8 100644
--- a/chrome/browser/resources/password_manager/BUILD.gn
+++ b/chrome/browser/resources/password_manager/BUILD.gn
@@ -140,4 +140,5 @@ build_webui("build") {
               "$root_gen_dir/chrome/browser/resources/settings_shared/tsc",
               root_build_dir) ]
   }
+  import("//brave/browser/resources/password_manager/sources.gni") non_web_component_files += brave_password_manager_non_web_component_files exclude_ts_preprocess_files = brave_password_manager_ts_local_files preprocess_deps = brave_password_manager_preprocess_extra_deps
 }
