diff --git a/chrome/installer/linux/debian/update_dist_package_versions.py b/chrome/installer/linux/debian/update_dist_package_versions.py
index 0b0b35963c7eb07d037cd741e8009dc7d818ab42..123a17ba3513341e706f4103c5a59aa890413053 100755
--- a/chrome/installer/linux/debian/update_dist_package_versions.py
+++ b/chrome/installer/linux/debian/update_dist_package_versions.py
@@ -38,6 +38,7 @@ PACKAGE_FILTER = {
     "libatspi2.0-0",
     "libc6",
     "libcairo2",
+    "libcairo-gobject2",
     "libcups2",
     "libdbus-1-3",
     "libdrm2",
@@ -70,6 +71,7 @@ PACKAGE_FILTER = {
     "libxshmfence1",
     "libxss1",
     "libxtst6",
+    "zlib1g",
 }
 
 
