diff --git a/chrome/browser/ui/startup/startup_tab_provider.cc b/chrome/browser/ui/startup/startup_tab_provider.cc
index 440876d5ed95095ee4658923050803e4d0f1b25e..79eacba865d8e228dda69143342e99f176a5442a 100644
--- a/chrome/browser/ui/startup/startup_tab_provider.cc
+++ b/chrome/browser/ui/startup/startup_tab_provider.cc
@@ -142,6 +142,7 @@ StartupTabs StartupTabProviderImpl::GetPostCrashTabs(
 bool StartupTabProviderImpl::CanShowWelcome(bool is_signin_allowed,
                                             bool is_supervised_user,
                                             bool is_force_signin_enabled) {
+  return true;
   return is_signin_allowed && !is_supervised_user && !is_force_signin_enabled;
 }
 
