diff --git a/chrome/browser/shell_integration.h b/chrome/browser/shell_integration.h
index 0690cf22a9e1acc617ef17dd09f2ce2d4251f4a1..7257f004bdba6feacacb0598939bf96053c93c5e 100644
--- a/chrome/browser/shell_integration.h
+++ b/chrome/browser/shell_integration.h
@@ -300,6 +300,7 @@ class DefaultBrowserWorker : public DefaultWebClientWorker {
 
   static void DisableSetAsDefaultForTesting();
 
+  BRAVE_DEFAULT_BROWSER_WORKER_H
  protected:
   ~DefaultBrowserWorker() override;
 
