diff --git a/chrome/browser/download/internal/android/BUILD.gn b/chrome/browser/download/internal/android/BUILD.gn
index 2731bf67092b2f91ebf96ab5a2fee408046911c1..59b644368a48577ddba525f8c2c5c37cdd38e5bd 100644
--- a/chrome/browser/download/internal/android/BUILD.gn
+++ b/chrome/browser/download/internal/android/BUILD.gn
@@ -200,6 +200,7 @@ android_resources("java_resources") {
     "java/res/values/ids.xml",
     "java/res/values/styles.xml",
   ]
+  sources += brave_download_internal_java_resources
 
   deps = [
     "//chrome/browser/download/android:java_resources",
