diff --git a/chrome/browser/policy/profile_policy_connector.cc b/chrome/browser/policy/profile_policy_connector.cc
index 95f9790fecee8b66483242154ac52c65045b7256..08c64bbea8bb072a7a10671a1b30ac48b94b92b3 100644
--- a/chrome/browser/policy/profile_policy_connector.cc
+++ b/chrome/browser/policy/profile_policy_connector.cc
@@ -429,6 +429,7 @@ void ProfilePolicyConnector::Init(
   }
 #endif
 
+  BRAVE_PROFILE_POLICY_CONNECTOR_INIT
   std::vector<std::unique_ptr<PolicyMigrator>> migrators;
 #if BUILDFLAG(IS_WIN)
   migrators.push_back(
@@ -497,6 +498,7 @@ void ProfilePolicyConnector::Shutdown() {
     special_user_policy_provider_->Shutdown();
 #endif
 
+  BRAVE_PROFILE_POLICY_CONNECTOR_SHUTDOWN
 #if BUILDFLAG(IS_CHROMEOS)
   if (restricted_mgs_policy_provider)
     restricted_mgs_policy_provider->Shutdown();
