diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 5302bf39e1303513660715d02d8fe3d64b41f577..4626b0fe760b8319ab5570e54212073c3a7f770c 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -169,6 +169,7 @@ if (current_toolchain == default_toolchain) {
       "java/res_chromium_base/mipmap-xxxhdpi/layered_app_icon_background.png",
       "java/res_chromium_base/values/channel_constants.xml",
     ]
+    sources += brave_base_module_resources
   }
 
   android_resources("chrome_app_java_resources") {
@@ -235,6 +236,7 @@ if (current_toolchain == default_toolchain) {
       "//third_party/androidx:androidx_gridlayout_gridlayout_java",
       "//third_party/androidx:androidx_preference_preference_java",
     ]
+    deps += brave_chrome_app_java_resources_deps sources += brave_java_resources resource_overlay = true
   }
 
   android_library("app_hooks_java") {
@@ -621,6 +623,7 @@ if (current_toolchain == default_toolchain) {
     ]
 
     deps += feed_deps
+    deps += brave_chrome_java_deps
 
     srcjar_deps = [
       ":chrome_android_java_enums_srcjar",
@@ -641,6 +644,7 @@ if (current_toolchain == default_toolchain) {
       "//components/supervised_user/core/browser:supervised_user_url_filter_enum_javagen",
       "//net:effective_connection_type_java",
     ]
+    srcjar_deps += brave_chrome_java_srcjar_deps
 
     # From java_sources.gni.
     sources = chrome_java_sources + [ app_hooks_impl ]
@@ -770,6 +774,7 @@ if (current_toolchain == default_toolchain) {
       "//components/messages/android/internal:java",
       "//components/segmentation_platform/internal:internal_java",
     ]
+    deps += brave_chrome_all_java_deps
   }
 
   action_with_pydeps("chrome_android_java_google_api_keys_srcjar") {
@@ -2027,6 +2032,7 @@ if (current_toolchain == default_toolchain) {
       "java/res_chromium_base/mipmap-xxxhdpi/layered_app_icon_background.png",
       "java/res_chromium_base/values/channel_constants.xml",
     ]
+    sources += brave_base_module_resources
 
     # Dep needed to ensure override works properly.
     deps = [ ":chrome_base_module_resources" ]
@@ -2246,6 +2252,7 @@ if (current_toolchain == default_toolchain) {
         ":${_variant}_locale_pak_assets",
         ":${_variant}_paks",
       ]
+      deps += [ "//brave:brave_pak_assets" ]
       if (_is_monochrome) {
         deps += [ "//android_webview:locale_pak_assets" ]
         if (webview_includes_weblayer && !_is_bundle_module) {
@@ -2516,6 +2523,7 @@ if (current_toolchain == default_toolchain) {
       "//components/payments/content/android:service_java",
       "//third_party/androidx:androidx_browser_browser_java",
     ]
+    sources += brave_java_base_module_sources deps += brave_java_base_module_deps
 
     # More deps for DFMs.
     if (dfmify_dev_ui) {
