diff --git a/chrome/browser/updater/BUILD.gn b/chrome/browser/updater/BUILD.gn
index 4da6a64d959c9eada3d48631c7536678eef24b31..fc373c1e25f4f2470fb59de11309f6c34c246b87 100644
--- a/chrome/browser/updater/BUILD.gn
+++ b/chrome/browser/updater/BUILD.gn
@@ -38,6 +38,7 @@ source_set("updater") {
       "//chrome/updater/mojom",
     ]
 
+    import("//brave/browser/updater/sources.gni") deps += brave_chrome_updater_scheduler_deps
     if (is_win) {
       sources += [
         "browser_updater_client_util_win.cc",
