diff --git a/chromecast/browser/cast_browser_context.cc b/chromecast/browser/cast_browser_context.cc
index fe51d1d32742..8752a04b4b72 100644
--- a/chromecast/browser/cast_browser_context.cc
+++ b/chromecast/browser/cast_browser_context.cc
@@ -79,13 +79,13 @@ void CastBrowserContext::InitWhileIOAllowed() {
 #endif  // defined(OS_ANDROID)
 }
 
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
 std::unique_ptr<content::ZoomLevelDelegate>
 CastBrowserContext::CreateZoomLevelDelegate(
     const base::FilePath& partition_path) {
   return nullptr;
 }
-#endif  // !defined(OS_ANDROID)
+//#endif  // !defined(OS_ANDROID)
 
 base::FilePath CastBrowserContext::GetPath() {
   return path_;
