diff --git a/chrome/browser/ui/tab_contents/chrome_web_contents_view_handle_drop.cc b/chrome/browser/ui/tab_contents/chrome_web_contents_view_handle_drop.cc
index 52603574ba6277ef6071afb71354d20529125562..d10e8a84423473bac51fbee6098e978a9b893143 100644
--- a/chrome/browser/ui/tab_contents/chrome_web_contents_view_handle_drop.cc
+++ b/chrome/browser/ui/tab_contents/chrome_web_contents_view_handle_drop.cc
@@ -12,6 +12,7 @@
 #include "content/public/browser/web_contents_view_delegate.h"
 #include "content/public/common/drop_data.h"
 
+#if !defined(OS_ANDROID)
 namespace {
 
 void DeepScanCompletionCallback(
@@ -32,7 +33,9 @@ void DeepScanCompletionCallback(
 }
 
 }  // namespace
+#endif
 
+#if !defined(OS_ANDROID)
 void HandleOnPerformDrop(
     content::WebContents* web_contents,
     const content::DropData& drop_data,
@@ -69,3 +72,4 @@ void HandleOnPerformDrop(
       base::BindOnce(&DeepScanCompletionCallback, std::move(callback)),
       safe_browsing::DeepScanAccessPoint::DRAG_AND_DROP);
 }
+#endif
