diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 52ae4877ab7bd03044ac42a221a4551cda057132..264fedaaa0fd82f473e75d11bd4a5669e3473d23 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -61,6 +61,7 @@ source_set("channel_info") {
     "channel_info.h",
   ]
   deps = [ "//build:branding_buildflags" ]
+  if (brave_chromium_build) { deps += [ "//brave/common:channel_info" ] }
   public_deps = [
     "//base",
     "//components/version_info",
@@ -146,6 +147,7 @@ static_library("common") {
   ]
 
   public_deps = [
+    "//brave/common",
     ":available_offline_content_mojom",
     ":buildflags",
     ":channel_info",
@@ -549,6 +551,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  if (brave_chromium_build) { configs += [ "//brave/common:constants_configs" ] }
 }
 
 component("chrome_features") {
