diff --git a/chrome/browser/file_select_helper.cc b/chrome/browser/file_select_helper.cc
index f21e8f38bcbdeb4cb5cad7004ecc6523d2870df9..26c9601c4f5e3f5723b76dc7e6268168e6f572d2 100644
--- a/chrome/browser/file_select_helper.cc
+++ b/chrome/browser/file_select_helper.cc
@@ -463,6 +463,11 @@ void FileSelectHelper::ContentAnalysisCompletionCallback(
 
 void FileSelectHelper::NotifyListenerAndEnd(
     std::vector<blink::mojom::FileChooserFileInfoPtr> list) {
+  if (brave::MaybeStripImageMetadataForUpload(
+          image_metadata_processed_, temporary_files_, list,
+          base::BindOnce(&FileSelectHelper::NotifyListenerAndEnd, this))) {
+    return;
+  }
   listener_->FileSelected(std::move(list), base_dir_, dialog_mode_);
   listener_.reset();
 
