diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index 188d34fc4634f56f0798e50c12e742eb234236fe..061f87098d56f96a544bd9e8b0f5e2a2b8305b5c 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -88,6 +88,7 @@ grit_strings("generated_resources") {
   if (is_android) {
     create_android_resources = true
   }
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
@@ -106,7 +107,7 @@ if (is_android) {
 }
 
 grit_strings("branded_strings") {
-  source = "${branding_path_product}_strings.grd"
+  source = "${branding_base_dir}/${branding_path_product}_strings.grd"
   defines = chrome_grit_defines
   output_dir = "$root_gen_dir/chrome"
   outputs = [ "grit/branded_strings.h" ]
