diff --git a/chrome/browser/ui/webui/settings/people_handler.cc b/chrome/browser/ui/webui/settings/people_handler.cc
index 044ccd3f5ea12ac6a1abcc37e2dadc5d145b639e..22d53844d3004852fd3b79915202b4bb3d9c6942 100644
--- a/chrome/browser/ui/webui/settings/people_handler.cc
+++ b/chrome/browser/ui/webui/settings/people_handler.cc
@@ -1147,6 +1147,7 @@ void PeopleHandler::OnStateChanged(syncer::SyncService* sync_service) {
 
 void PeopleHandler::BeforeUnloadDialogCancelled() {
   // The before unload dialog is only shown during the first sync setup.
+  if (false)
   DCHECK(IdentityManagerFactory::GetForProfile(profile_)->HasPrimaryAccount(
       signin::ConsentLevel::kSync));
   syncer::SyncService* service = GetSyncService();
@@ -1196,6 +1197,7 @@ base::Value::Dict PeopleHandler::GetSyncStatusDictionary() const {
       service && !disallowed_by_policy && service->IsSetupInProgress() &&
           !service->GetUserSettings()->IsInitialSyncFeatureSetupComplete() &&
           identity_manager->HasPrimaryAccount(signin::ConsentLevel::kSync));
+  BRAVE_GET_SYNC_STATUS_DICTIONARY
 
   SyncStatusLabels status_labels;
 
@@ -1401,6 +1403,7 @@ void PeopleHandler::MaybeMarkSyncConfiguring() {
 }
 
 bool PeopleHandler::IsProfileAuthNeededOrHasErrors() {
+  return false;
   return !IdentityManagerFactory::GetForProfile(profile_)->HasPrimaryAccount(
              signin::ConsentLevel::kSync) ||
          SigninErrorControllerFactory::GetForProfile(profile_)->HasError();
