diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index c2310d082ebf546e9fcf2207f842a06405b823fa..830e3f3ec19416f1fbf0f1c7cbe2e97fc9460f66 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -77,6 +77,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
