diff --git a/chrome/android/java_sources.gni b/chrome/android/java_sources.gni
index b2f24064da7e3cd7b0283009c2d8c4ddd6fe6cd9..88911a437321cc7fc83cde137b2b5b2510f3bee1 100644
--- a/chrome/android/java_sources.gni
+++ b/chrome/android/java_sources.gni
@@ -20,6 +20,7 @@ import("//components/offline_pages/buildflags/features.gni")
 import("//device/vr/buildflags/buildflags.gni")
 
 chrome_java_sources += public_autofill_assistant_java_sources
+chrome_java_sources += brave_java_sources
 
 if (enable_vr) {
   chrome_java_sources += [ "java/src/org/chromium/chrome/browser/component_updater/VrAssetsComponentInstaller.java" ]
