diff --git a/chrome/browser/notifications/notification_platform_bridge_linux.cc b/chrome/browser/notifications/notification_platform_bridge_linux.cc
index aa6d15f6ebbdcc6e70374a46ca9c11d4c1bff854..5c2bdec672a402edbc3602887fdad9e3cc0056a8 100644
--- a/chrome/browser/notifications/notification_platform_bridge_linux.cc
+++ b/chrome/browser/notifications/notification_platform_bridge_linux.cc
@@ -632,6 +632,7 @@ class NotificationPlatformBridgeLinuxImpl
       actions.push_back("Activate");
       // Always add a settings button for web notifications.
       if (notification_type != NotificationHandler::Type::EXTENSION &&
+          notification_type != NotificationHandler::Type::BRAVE_ADS &&
           notification_type != NotificationHandler::Type::SEND_TAB_TO_SELF) {
         actions.push_back(kSettingsButtonId);
         actions.push_back(
