diff --git a/chrome/browser/ui/webui/settings/import_data_handler.cc b/chrome/browser/ui/webui/settings/import_data_handler.cc
index a2d1ae566df6352f21c0a437ffd5e30ba7b1abdc..a5c818276daf6376184da4f7d5e78f0318db56fe 100644
--- a/chrome/browser/ui/webui/settings/import_data_handler.cc
+++ b/chrome/browser/ui/webui/settings/import_data_handler.cc
@@ -130,6 +130,7 @@ void ImportDataHandler::HandleImportData(const base::Value::List& args) {
     selected_items |= importer::PASSWORDS;
   if (*types.FindBoolKey(prefs::kImportDialogSearchEngine))
     selected_items |= importer::SEARCH_ENGINES;
+  BRAVE_IMPORT_DATA
 
   const importer::SourceProfile& source_profile =
       importer_list_->GetSourceProfileAt(browser_index);
@@ -210,6 +211,7 @@ void ImportDataHandler::SendBrowserProfileData(const std::string& callback_id) {
     browser_profile->SetBoolKey(
         "autofillFormData",
         (browser_services & importer::AUTOFILL_FORM_DATA) != 0);
+    BRAVE_SEND_BROWSER_PROFILE_DATA
 
     browser_profiles.Append(std::move(browser_profile));
   }
