diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.cc b/chrome/browser/ui/views/location_bar/location_bar_view.cc
index fb4091acc5e5ee7721ba864060e7283ec7c6ff1a..7ca0edf04ed4750e24dac6bb928dcd288fe1dc1c 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
@@ -736,6 +736,7 @@ void LocationBarView::Layout(PassKey) {
   // Indentation add padding when the permission chip is visible and replacing
   // the LHS icon.
   int text_overriding_permission_chip_indent = 0;
+  BRAVE_LOCATION_BAR_VIEW_LAYOUT_LEADING_DECORATIONS
   if (should_indent) {
     icon_left += icon_indent;
     text_left += text_indent;
@@ -808,6 +809,7 @@ void LocationBarView::Layout(PassKey) {
       }
       selected_keyword_view_->SetCustomImage(image);
     }
+  BRAVE_LOCATION_BAR_VIEW_LAYOUT_HANDLE_SEARCH_PROMOTION_BUTTON_VISIBILITY
   } else if (location_icon_view_->GetShowText() &&
              !ShouldChipOverrideLocationIcon()) {
     location_icon_view_->SetVisible(true);
@@ -855,6 +857,7 @@ void LocationBarView::Layout(PassKey) {
     }
   };
 
+  BRAVE_LOCATION_BAR_VIEW_LAYOUT_RIGHT_MOST_TRAILING_DECORATIONS
   add_trailing_decoration(page_action_icon_container_,
                           /*intra_item_padding=*/0);
   add_trailing_decoration(page_action_container_,
@@ -870,6 +873,7 @@ void LocationBarView::Layout(PassKey) {
   }
 
   add_trailing_decoration(clear_all_button_, /*intra_item_padding=*/0);
+  BRAVE_LOCATION_BAR_VIEW_LAYOUT_LEFT_MOST_TRAILING_DECORATIONS
 
   // Perform layout.
   int entry_width = width();
