diff --git a/chrome/browser/actor/execution_engine.cc b/chrome/browser/actor/execution_engine.cc
index 77a0a38c774f483b33a1e8ccc618ba59fbff1cbf..24fd0020e117bbf762ed296971a06ae3cd8dee51 100644
--- a/chrome/browser/actor/execution_engine.cc
+++ b/chrome/browser/actor/execution_engine.cc
@@ -1416,6 +1416,16 @@ void ExecutionEngine::SafetyChecksForNextAction() {
   // TODO(mcnee): Add UMA for the outcomes.
   content::WebContents& web_contents = *(tab->GetContents());
   const GURL& url = web_contents.GetPrimaryMainFrame()->GetLastCommittedURL();
+  BRAVE_MAY_ACT_ON_URL_INTERNAL(
+      base::BindOnce([](MayActOnUrlBlockReason block_reason) {
+        return BlockReasonToResultCode(block_reason,
+                                       /*for_navigation=*/false);
+      })
+          .Then(base::BindOnce(&ExecutionEngine::DidFinishAsyncSafetyChecks,
+                               GetActionSequenceWeakPtr(),
+                               tab->GetContents()
+                                   ->GetPrimaryMainFrame()
+                                   ->GetLastCommittedOrigin())))
   auto event = GateableEvent::kPageAction;
   GetOriginGatingChecker().ComputeGatingDecision(
       std::make_unique<PageActionGatingContext>(web_contents.GetWeakPtr()),
@@ -1729,6 +1739,7 @@ const EnterprisePolicyChecker& ExecutionEngine::GetEnterprisePolicyChecker()
 void ExecutionEngine::IsAcceptableNavigationDestination(
     const GURL& url,
     DecisionCallbackWithReason callback) {
+  BRAVE_MAY_ACT_ON_URL_INTERNAL(std::move(callback))
   auto event = GateableEvent::kNavigationRequest;
   GetOriginGatingChecker().ComputeGatingDecision(
       std::make_unique<OriginGatingDecisionContext>(), event, /*source=*/GURL(),
