diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.h b/chrome/browser/ui/views/location_bar/location_bar_view.h
index 7b11704e9f09fcf6500feabe000754a056762c5f..778680a9a805311be70fd07561b71cc1bd8000b8 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.h
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.h
@@ -138,6 +138,7 @@ class LocationBarView
   int GetBorderRadius() const;
 
   // Initializes the LocationBarView.
+  virtual
   void Init();
 
   bool in_popup_state_transition() const { return in_popup_state_transition_; }
@@ -201,6 +202,7 @@ class LocationBarView
 
   // Updates the controller, and, if |contents| is non-null, restores saved
   // state that the tab holds.
+  virtual
   void Update(content::WebContents* contents);
 
   // Clears the location bar's state for |contents|.
@@ -299,6 +301,7 @@ class LocationBarView
   GetContentSettingViewsForTest() {
     return content_setting_views_;
   }
+  BRAVE_LOCATION_BAR_VIEW_H_
 
   SkColor GetBackgroundColorForTesting() const { return background_color_; }
 
