diff --git a/chrome/browser/plugins/plugins_resource_service.cc b/chrome/browser/plugins/plugins_resource_service.cc
index 2f6b793ce0eaa4f95bd5ea08c263e11c5792c22d..cbd677a89a739dec2a4204c97e6e6a49cb744a1a 100644
--- a/chrome/browser/plugins/plugins_resource_service.cc
+++ b/chrome/browser/plugins/plugins_resource_service.cc
@@ -55,7 +55,7 @@ const int kStartResourceFetchDelayMs = 60 * 1000;
 const int kCacheUpdateDelayMs = 24 * 60 * 60 * 1000;
 
 const char kPluginsServerUrl[] =
-    "https://www.gstatic.com/chrome/config/plugins_3/";
+    "https://static1.brave.com/chrome/config/plugins_3/";
 
 GURL GetPluginsServerURL() {
   std::string filename;
