diff --git a/chrome/android/java/src/org/chromium/chrome/browser/MonochromeApplicationImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/MonochromeApplicationImpl.java
index 80f97ffce49b2a6cf04ccfdc1df390e72bb43b2a..7360b03625556ecaaa88cfa0b00fd56f2abfa62f 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/MonochromeApplicationImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/MonochromeApplicationImpl.java
@@ -4,7 +4,7 @@
 
 package org.chromium.chrome.browser;
 
-import org.chromium.android_webview.nonembedded.WebViewApkApplication;
+//import org.chromium.android_webview.nonembedded.WebViewApkApplication;
 import org.chromium.base.ActivityState;
 import org.chromium.base.ApplicationStatus;
 import org.chromium.chrome.browser.base.SplitCompatApplication;
@@ -35,14 +35,14 @@ public class MonochromeApplicationImpl extends ChromeApplicationImpl {
             if (getApplication().isWebViewProcess()) {
                 // Whenever a monochrome webview process is launched (WebView service or
                 // developer UI), post a background task to show/hide the DevTools icon.
-                WebViewApkApplication.postDeveloperUiLauncherIconTask();
+//                WebViewApkApplication.postDeveloperUiLauncherIconTask();
             } else if (SplitCompatApplication.isBrowserProcess()) {
                 // Frequently check current system webview provider and show/hide the icon
                 // accordingly by listening to Monochrome browser Activities status (whenever a
                 // browser activity comes to the foreground).
                 ApplicationStatus.registerStateListenerForAllActivities((activity, state) -> {
                     if (state == ActivityState.STARTED) {
-                        WebViewApkApplication.postDeveloperUiLauncherIconTask();
+//                        WebViewApkApplication.postDeveloperUiLauncherIconTask();
                     }
                 });
             }
