diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index 2d0961a58426023d8c0b0bfc6db1fd64a6a9d65a..8f69502625653cc58518974e76d61567caf4c2d9 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -392,6 +392,7 @@ content::WebUIDataSource* CreateExtensionsSource(Profile* profile,
       "showAccessRequestsInToolbarLearnMoreLink",
       chrome_extension_constants::kShowAccessRequestsInToolbarHelpURL);
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
