diff --git a/chrome/browser/importer/external_process_importer_client.cc b/chrome/browser/importer/external_process_importer_client.cc
index 7320911bb2c1e848eafe487f9b376df2177521e8..6e7278eb8fd6fc9fd2ad339192e4eb7cf7ba3964 100644
--- a/chrome/browser/importer/external_process_importer_client.cc
+++ b/chrome/browser/importer/external_process_importer_client.cc
@@ -69,6 +69,12 @@ void ExternalProcessImporterClient::Start() {
   localized_strings.try_emplace(
       IDS_BOOKMARK_BAR_FOLDER_NAME,
       l10n_util::GetStringUTF8(IDS_BOOKMARK_BAR_FOLDER_NAME));
+  localized_strings.try_emplace(
+      IDS_IMPORT_FROM_BRAVE,
+      l10n_util::GetStringUTF8(IDS_IMPORT_FROM_BRAVE));
+  localized_strings.try_emplace(
+      IDS_BOOKMARK_GROUP_FROM_BRAVE,
+      l10n_util::GetStringUTF8(IDS_BOOKMARK_GROUP_FROM_BRAVE));
 
   // If the utility process hasn't started yet the message will queue until it
   // does.
