diff --git a/chrome/browser/profiles/profile_impl.cc b/chrome/browser/profiles/profile_impl.cc
index cd7cb4e41199883ad37a43764536d5db1805f689..c6fb29518e39cdc3ca148d108bd6e5c467e3df39 100644
--- a/chrome/browser/profiles/profile_impl.cc
+++ b/chrome/browser/profiles/profile_impl.cc
@@ -611,6 +611,7 @@ void ProfileImpl::TakePrefsFromStartupData() {
       CreateExtensionPrefStore(this, /*incognito_pref_store=*/false));
 
   ProfileKeyStartupAccessor::GetInstance()->Reset();
+  BRAVE_PROFILE_IMPL_TAKE_PREFS_FROM_STARTUP_DATA
 }
 #endif
 
