diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.h b/chrome/browser/ui/views/toolbar/toolbar_view.h
index da32d80ee4f08ff7bca6de8934f861076053fa6b..7787a5c27fadf80e53b8f399ec8c0701f55c6542 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.h
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.h
@@ -101,12 +101,14 @@ class ToolbarView : public views::AccessiblePaneView,
   ~ToolbarView() override;
 
   // Create the contents of the Browser Toolbar.
+  virtual
   void Init();
 
   // Forces the toolbar (and transitively the location bar) to update its
   // current state.  If |tab| is non-NULL, we're switching (back?) to this tab
   // and should restore any previous location bar state (such as user editing)
   // as well.
+  virtual
   void Update(content::WebContents* tab);
 
   // Updates the visibility of the custom tab bar, potentially animating the
@@ -137,6 +139,7 @@ class ToolbarView : public views::AccessiblePaneView,
       IntentPickerResponse callback);
 
   // Shows a bookmark bubble and anchors it appropriately.
+  virtual
   void ShowBookmarkBubble(const GURL& url,
                           bool already_bookmarked,
                           bookmarks::BookmarkBubbleObserver* observer);
@@ -209,6 +212,7 @@ class ToolbarView : public views::AccessiblePaneView,
   friend class AvatarToolbarButtonBrowserTest;
 
  protected:
+  friend class BraveToolbarView;
   // This controls Toolbar, LocationBar and CustomTabBar visibility.
   // If we don't set all three, tab navigation from the app menu breaks
   // on Chrome OS.
@@ -264,6 +268,7 @@ class ToolbarView : public views::AccessiblePaneView,
   void OnChromeLabsPrefChanged();
 
   // Loads the images for all the child views.
+  virtual
   void LoadImages();
 
   // Shows the critical notification bubble against the app menu.
