diff --git a/chrome/browser/ui/views/tabs/compound_tab_container.cc b/chrome/browser/ui/views/tabs/compound_tab_container.cc
index e0bc23beddade8782ed5e05ceecebd0ef6407d17..712cba9bf79e0c0aba322a17aafaf153a479da40 100644
--- a/chrome/browser/ui/views/tabs/compound_tab_container.cc
+++ b/chrome/browser/ui/views/tabs/compound_tab_container.cc
@@ -52,7 +52,7 @@ class PinnedTabContainerController final : public TabContainerController {
   }
 
   int NumPinnedTabsInModel() const override {
-    return base_controller_->NumPinnedTabsInModel();
+    return compound_tab_container_->NumPinnedTabs();
   }
 
   void OnDropIndexUpdate(const std::optional<int> index,
