diff --git a/chrome/browser/ui/views/page_action/page_action_controller.h b/chrome/browser/ui/views/page_action/page_action_controller.h
index 801b4cac8c05050089c9b4a5457c1a60acc77359..59240c6f19522d38aa385c28b8078da0746efd9f 100644
--- a/chrome/browser/ui/views/page_action/page_action_controller.h
+++ b/chrome/browser/ui/views/page_action/page_action_controller.h
@@ -107,6 +107,17 @@ class PageActionController {
  public:
   virtual ~PageActionController() = default;
 
+  // Force the label to be shown regardless of the available space
+  virtual void SetAlwaysShowLabel(actions::ActionId action_id,
+                                  bool always_show) = 0;
+
+  // Override the colors of the chip
+  virtual void OverrideChipColors(
+      actions::ActionId action_id,
+      std::optional<SkColor> override_background_color,
+      std::optional<SkColor> override_foreground_color) = 0;
+  virtual void ClearOverrideChipColors(actions::ActionId action_id) = 0;
+
   // Requests that the page action be shown or hidden.
   virtual void Show(actions::ActionId action_id) = 0;
   virtual void Hide(actions::ActionId action_id) = 0;
@@ -241,6 +252,13 @@ class PageActionControllerImpl : public PageActionController,
   PageActionControllerImpl& operator=(const PageActionControllerImpl&) = delete;
   ~PageActionControllerImpl() override;
 
+  void SetAlwaysShowLabel(actions::ActionId action_id,
+                          bool always_show) override;
+  void OverrideChipColors(actions::ActionId action_id,
+      std::optional<SkColor> override_background_color,
+      std::optional<SkColor> override_foreground_color) override;
+  void ClearOverrideChipColors(actions::ActionId action_id) override;
+
   void Initialize(
       tabs::TabInterface& tab_interface,
       const std::vector<actions::ActionId>& action_ids,
