diff --git a/chrome/browser/ui/views/omnibox/omnibox_popup_contents_view.cc b/chrome/browser/ui/views/omnibox/omnibox_popup_contents_view.cc
index 81a62db9dab92349c990efbeff49d88fb271ce62..23a1dee91bcfc1982ac55fa90f747b3b4c1daec6 100644
--- a/chrome/browser/ui/views/omnibox/omnibox_popup_contents_view.cc
+++ b/chrome/browser/ui/views/omnibox/omnibox_popup_contents_view.cc
@@ -347,7 +347,7 @@ void OmniboxPopupContentsView::UpdatePopupAppearance() {
     if (children().size() == i) {
       AddChildView(std::make_unique<OmniboxRowView>(
           i, edit_model_,
-          std::make_unique<OmniboxResultView>(this, edit_model_, i),
+          std::make_unique<BraveOmniboxResultView>(this, edit_model_, i),
           pref_service));
     }
 
