diff --git a/chrome/browser/ui/android/theme/java/src/org/chromium/chrome/browser/theme/ThemeUtils.java b/chrome/browser/ui/android/theme/java/src/org/chromium/chrome/browser/theme/ThemeUtils.java
index 3aa1dbc62756976ebe6453c6b13b62da21edffe6..db00310b307080321e38dcac76796acb061eaab3 100644
--- a/chrome/browser/ui/android/theme/java/src/org/chromium/chrome/browser/theme/ThemeUtils.java
+++ b/chrome/browser/ui/android/theme/java/src/org/chromium/chrome/browser/theme/ThemeUtils.java
@@ -87,6 +87,7 @@ public class ThemeUtils {
             Context context, @ColorInt int color, boolean isIncognito) {
         // Text box color on default toolbar background in incognito mode is a pre-defined
         // color. We calculate the equivalent opaque color from the pre-defined translucent color.
+        if (isIncognito) {return ApiCompatibilityUtils.getColor(res, R.color.toolbar_text_box_background_incognito);}
         if (isIncognito) {
             final int overlayColor = ApiCompatibilityUtils.getColor(
                     context.getResources(), R.color.toolbar_text_box_background_incognito);
