diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 3f89c70d1fe370ef15db842266d767d214b57992..7fc8163f42af8edb05f804f5eeb0c99a0bb8f37f 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2266,6 +2266,7 @@ static_library("browser") {
     "//ui/web_dialogs",
     "//ui/webui",
     "//ui/webui/resources/cr_components/customize_themes:mojom",
+    "//brave/browser:browser_process",
   ]
   if (is_chromeos) {
     deps += [
@@ -4001,6 +4002,7 @@ static_library("browser") {
         "task_manager/sampling/shared_sampler_posix.cc",
       ]
     }
+    sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps
   }
 
   if (is_chromeos) {
