diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index d9790e02fdfe69ae6798dfe4553e205ad0b30369..0ed21c51ded139aeb9a1fb81702191cd17bb97a9 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -125,6 +125,7 @@ bool Profile::OTRProfileID::AllowsBrowserWindows() const {
   // DevTools::BrowserContext, MediaRouter::Presentation, and
   // CaptivePortal::Signin are exceptions to this ban.
   if (*this == PrimaryID() || IsDevTools() ||
+      BRAVE_ALLOWS_BROWSER_WINDOWS
       base::StartsWith(profile_id_, kMediaRouterOTRProfileIDPrefix,
                        base::CompareCase::SENSITIVE)) {
     return true;
