diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index c5157e934b328fa30eca11f7c8728b933570d35a..28de46dd92a56bf9c8135a7b9e09d2ac228874e0 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -8,6 +8,7 @@ import {pageVisibility} from './page_visibility.js';
 import {Route, Router} from './router.js';
 import {SettingsRoutes} from './settings_routes.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -215,6 +216,7 @@ function createBrowserSettingsRoutes(): SettingsRoutes {
       // </if>
     }
   }
+  addBraveRoutes(r);
   return r;
 }
 
