diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index 5e9e4797a9add24acad64387683ced62d130feaf..be074c0b9c18bab3298573318b33e95fa8f7bbe0 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -220,6 +220,8 @@ ts_library("build_ts") {
     ":preprocess",
     ":preprocess_generated",
   ]
+  in_files += brave_deps_chrome_browser_resources_settings_in_files extra_deps += brave_deps_chrome_browser_resources_settings_extra_deps
+
 }
 
 generate_grd("build_grd") {
@@ -308,4 +311,5 @@ generate_grd("build_grd") {
     manifest_files =
         filter_include(get_target_outputs(":build_ts"), [ "*.manifest" ])
   }
+  deps += [ "//brave/browser/resources/settings:resources" ]
 }
