diff --git a/chrome/browser/chrome_browser_main.cc b/chrome/browser/chrome_browser_main.cc
index b2619baa8f61dfc4699a27f17e44fc4a94dd94f7..1f7274ae617d93e4ead1090641531dfcd9325d6a 100644
--- a/chrome/browser/chrome_browser_main.cc
+++ b/chrome/browser/chrome_browser_main.cc
@@ -1061,6 +1061,11 @@ void ChromeBrowserMainParts::ToolkitInitialized() {
 
 #if BUILDFLAG(IS_MAC) && BUILDFLAG(ENABLE_UPDATER)
 void PromptUpdaterPromotion() {
+  #if BUILDFLAG(ENABLE_OMAHA4)
+  if (!brave_updater::ShouldUseOmaha4()) {
+    return;
+  }
+  #endif
   content::GetUIThreadTaskRunner({})->PostTask(
       FROM_HERE, base::BindOnce([]() {
         if (base::CommandLine::ForCurrentProcess()->HasSwitch(
