diff --git a/chrome/android/java_sources.gni b/chrome/android/java_sources.gni
index 2dcef8bd76e082f8c011601e6017f38c3eddfeed..b7e31de137a56b379563c3874dc7b4c6b8dc2ae4 100644
--- a/chrome/android/java_sources.gni
+++ b/chrome/android/java_sources.gni
@@ -20,6 +20,7 @@ import("//components/offline_pages/buildflags/features.gni")
 import("//device/vr/buildflags/buildflags.gni")
 
 chrome_java_sources += public_autofill_assistant_java_sources
+chrome_java_sources += brave_java_sources
 
 if (enable_vr) {
   chrome_java_sources += [ "java/src/org/chromium/chrome/browser/component_updater/VrAssetsComponentInstaller.java" ]
