diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 70c3930b4d357b2284b79dddb6191050cb42eab6..03a2247e92e0a5336d244e461cf78e8899be3ff9 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2751,6 +2751,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
