diff --git a/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_bubble_view_impl.cc b/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_bubble_view_impl.cc
index edf8a7a104de2b461a5db3729b3fddfe7cf6a0a6..2b1fb0556e13b918726060c2ef5762aa55ddbd4e 100644
--- a/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_bubble_view_impl.cc
+++ b/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_bubble_view_impl.cc
@@ -139,6 +139,7 @@ void SendTabToSelfBubbleViewImpl::Init() {
   CreateHintTextLabel();
   CreateDevicesScrollView();
 
+  BRAVE_SENT_TAB_TO_SELF_BUBBLE_VIEW_IMPL_INIT
   AddChildView(std::make_unique<views::Separator>());
   CreateManageDevicesLink();
 }
