diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 48138252497a51f25e7145d119f6b07268f68a25..3fa089274dcf06d36a3c0995d86cccf69fb46588 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -267,6 +267,7 @@ static_library("utility") {
   if (is_android) {
     deps += [ "//chrome/services/readaloud" ]
   }
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
 }
 
 if (!is_android) {
