diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 1ae2a847ad4987e42d78370f748d8093c9ad74e7..7dd29ec729433df0bde19000bbc74e3d151a7b95 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -185,6 +185,7 @@ if (_is_default_toolchain) {
       "java/res_chromium_base/mipmap-xxxhdpi/layered_app_icon_background.png",
       "java/res_chromium_base/values/channel_constants.xml",
     ]
+    sources += brave_base_module_resources
   }
 
   android_resources("chrome_app_java_resources") {
@@ -285,6 +286,7 @@ if (_is_default_toolchain) {
     if (enable_extensions_core) {
       deps += [ "//chrome/browser/ui/android/extensions:java_resources" ]
     }
+    deps += brave_chrome_app_java_resources_deps sources += brave_java_resources resource_overlay = true
   }
 
   android_resources("java_overlay_resources") {
@@ -807,6 +809,7 @@ if (_is_default_toolchain) {
     ]
 
     deps += feed_deps
+    deps += brave_chrome_java_deps
 
     srcjar_deps = [
       ":chrome_android_java_enums_srcjar",
@@ -823,6 +826,7 @@ if (_is_default_toolchain) {
       "//components/supervised_user/core/browser:supervised_user_utils_enum_javagen",
       "//components/touch_to_search/core/browser:quick_action_category_enum_javagen",
     ]
+    srcjar_deps += brave_chrome_java_srcjar_deps
 
     # From java_sources.gni.
     sources = chrome_java_sources
@@ -960,6 +964,7 @@ if (_is_default_toolchain) {
         "//chrome/browser/ui/browser_window/stub:java",
       ]
     }
+    deps += brave_chrome_all_java_deps
   }
 
   action_with_pydeps("chrome_android_java_google_api_keys_srcjar") {
@@ -1682,6 +1687,7 @@ if (_is_default_toolchain) {
       "java/res_chromium_base/mipmap-xxxhdpi/layered_app_icon_background.png",
       "java/res_chromium_base/values/channel_constants.xml",
     ]
+    sources += brave_base_module_resources
 
     # Dep needed to ensure override works properly.
     deps = [ ":chrome_base_module_resources" ]
@@ -1900,6 +1906,7 @@ if (_is_default_toolchain) {
         ":${_variant}_locale_pak_assets",
         ":${_variant}_paks",
       ]
+      deps += [ "//brave:brave_pak_assets" ]
       if (_is_monochrome) {
         deps += [ "//android_webview:locale_pak_assets" ]
       }
@@ -2191,6 +2198,7 @@ if (_is_default_toolchain) {
       "//third_party/androidx:androidx_annotation_annotation_experimental_java",
       "//third_party/androidx:androidx_browser_browser_java",
     ]
+    sources += brave_java_base_module_sources deps += brave_java_base_module_deps
 
     # More deps for DFMs.
     if (dfmify_dev_ui) {
@@ -3285,6 +3293,7 @@ if (_is_default_toolchain) {
       "java/src/org/chromium/chrome/browser/webapps/WebApkUpdateManager.java",
       "java/src/org/chromium/chrome/browser/webapps/WebappRegistry.java",
     ]
+    sources += brave_jni_headers_sources
 
     if (enable_screen_capture) {
       sources += [ "java/src/org/chromium/chrome/browser/media/MediaCapturePickerDialogBridge.java" ]
