diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 6d3b7f5eb87b2363fa788d527aa5365a66d74c49..a01aa9a0d030c09d4d2b5f5b77e00a3cb106fbd6 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -95,6 +95,7 @@ source_set("configs") {
   if (enable_glic) {
     deps += [ "//chrome/browser/glic" ]
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("webui") {
