diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index a9652ed1c3f4f4fe8c47a55320b0a34fa62017a0..d40afa33b298b775e432f4167a979c579db2686c 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -236,6 +236,7 @@ android_resources("chrome_app_java_resources") {
     "//third_party/androidx:androidx_preference_preference_java",
     "//third_party/androidx:androidx_recyclerview_recyclerview_java",
   ]
+  deps += brave_chrome_app_java_resources_deps sources += brave_java_resources resource_overlay = true
 }
 
 if (enable_vr) {
@@ -552,6 +553,7 @@ android_library("chrome_java") {
   ]
 
   deps += feed_deps
+  deps += brave_chrome_java_deps
 
   srcjar_deps = [
     ":chrome_android_java_enums_srcjar",
@@ -584,6 +586,7 @@ android_library("chrome_java") {
     "//components/ui_metrics:ui_metrics_enums_java",
     "//net:effective_connection_type_java",
   ]
+  srcjar_deps += brave_chrome_java_srcjar_deps
 
   # From java_sources.gni.
   sources = chrome_java_sources + [ app_hooks_impl ]
@@ -712,6 +715,7 @@ group("jni_headers") {
     "//chrome/browser/webauthn/android:jni_headers",
     "//components/media_router/browser/android:jni_headers",
   ]
+  public_deps += ["//brave/build/android:jni_headers"]
 }
 
 # TODO(crbug.com/1112471): Get this to run cleanly under Python 3.
@@ -1900,6 +1904,7 @@ if (current_toolchain == default_toolchain) {
         ":${_variant}_locale_pak_assets",
         ":${_variant}_paks",
       ]
+      deps += [ "//brave:brave_pak_assets" ]
       if (_is_monochrome) {
         deps += [ "//android_webview:locale_pak_assets" ]
         if (webview_includes_weblayer && !_is_bundle_module) {
