diff --git a/chrome/browser/ui/webui/settings/import_data_handler.cc b/chrome/browser/ui/webui/settings/import_data_handler.cc
index 73ad5260441a8fec0482801847957c1d02e7ef5c..87fe3fd7f86251ddf6c7b9e186b9cfc80c5be58c 100644
--- a/chrome/browser/ui/webui/settings/import_data_handler.cc
+++ b/chrome/browser/ui/webui/settings/import_data_handler.cc
@@ -130,6 +130,7 @@ void ImportDataHandler::HandleImportData(const base::Value::List& args) {
     selected_items |= importer::PASSWORDS;
   if (*types.FindBoolKey(prefs::kImportDialogSearchEngine))
     selected_items |= importer::SEARCH_ENGINES;
+  BRAVE_IMPORT_DATA
 
   const importer::SourceProfile& source_profile =
       importer_list_->GetSourceProfileAt(browser_index);
@@ -208,6 +209,7 @@ void ImportDataHandler::SendBrowserProfileData(const std::string& callback_id) {
     browser_profile.Set("autofillFormData",
                         (browser_services & importer::AUTOFILL_FORM_DATA) != 0);
 
+    BRAVE_SEND_BROWSER_PROFILE_DATA
     browser_profiles.Append(std::move(browser_profile));
   }
 
