diff --git a/chrome/browser/ui/views/tabs/tab_group_views.cc b/chrome/browser/ui/views/tabs/tab_group_views.cc
index b5b49abff4d9ac680586d48cd7ebfc13bf36e687..2f5d84956e7c3a7aa15761f5207b10895ea0b1de 100644
--- a/chrome/browser/ui/views/tabs/tab_group_views.cc
+++ b/chrome/browser/ui/views/tabs/tab_group_views.cc
@@ -166,6 +166,7 @@ TabGroupViews::GetLeadingTrailingGroupViews(
   views::View* trailing_child = nullptr;
   gfx::Rect trailing_child_widget_bounds;
 
+  BRAVE_TAB_GROUP_VIEWS_GET_LEADING_TRAILING_GROUP_VIEWS
   for (views::View* child : children) {
     TabSlotView* tab_slot_view = views::AsViewClass<TabSlotView>(child);
     if (!tab_slot_view || tab_slot_view->group() != group_ ||
