diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 187b2df80268aff900fadc0679b42f6ffb707613..53e81484fe280b67227609490ce34434f8134ebe 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -449,6 +449,7 @@ void ToolbarView::Init() {
     media_button_ = container_view_->AddChildView(std::move(media_button));
   }
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = container_view_->AddChildView(
       std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
