diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 2b750852cf3d53686121e9b919cafea1ae6ad43d..153dece8e6df445807dc0bc36ee32ca5f39e8f59 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -265,6 +265,7 @@ static_library("utility") {
       "readaloud/read_aloud_playback_controller.h",
     ]
   }
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
 }
 
 if (!is_android) {
