diff --git a/chrome/browser/extensions/extension_management.h b/chrome/browser/extensions/extension_management.h
index 53e7ae9056f2b6458836e274a74b5d7338900f0e..e07d64b193c222c51d4d77fe5207ee4dafe38a04 100644
--- a/chrome/browser/extensions/extension_management.h
+++ b/chrome/browser/extensions/extension_management.h
@@ -198,6 +198,7 @@ class ExtensionManagement : public KeyedService {
   bool is_child() const { return is_child_; }
 
  private:
+  friend class BraveExtensionManagement;
   using SettingsIdMap =
       std::unordered_map<ExtensionId,
                          std::unique_ptr<internal::IndividualSettings>>;
