diff --git a/chrome/browser/ui/views/frame/browser_frame_view_mac.h b/chrome/browser/ui/views/frame/browser_frame_view_mac.h
index 34fd5d5edfac71ff5304aef7320a7d70fd7fc1b6..633894606a1f9d8da20c9c454edf30d6cf2d9a42 100644
--- a/chrome/browser/ui/views/frame/browser_frame_view_mac.h
+++ b/chrome/browser/ui/views/frame/browser_frame_view_mac.h
@@ -90,9 +90,6 @@ class BrowserFrameViewMac : public BrowserFrameView,
   FRIEND_TEST_ALL_PREFIXES(BrowserFrameViewMacTest,
                            GetCaptionButtonPlaceholderBounds);
 
-  // Fetches the caption button bounds from the buttons themselves.
-  BoundsAndMargins GetCaptionButtonBoundsNative() const;
-
   // Creates an inset from the caption button size which controls for which edge
   // the captions buttons exists on. Used to position elements like the tabstrip
   // that are adjacent to the caption buttons.
