diff --git a/chrome/browser/ui/page_action/page_action_controller.h b/chrome/browser/ui/page_action/page_action_controller.h
index 58f223cd5dfc0ab6dc2704f5b23efb4be585eb0c..589b4d4fb51997b2ebfd57fa934b188dc34a3598 100644
--- a/chrome/browser/ui/page_action/page_action_controller.h
+++ b/chrome/browser/ui/page_action/page_action_controller.h
@@ -374,6 +374,7 @@ class PageActionController {
   virtual void DecrementActivityCounter(actions::ActionId action_id) = 0;
 };
 
+namespace chromium_impl {
 class PageActionControllerImpl : public PageActionController,
                                  public PinnedToolbarActionsModel::Observer {
  public:
@@ -464,6 +465,7 @@ class PageActionControllerImpl : public PageActionController,
   void OnActionsChanged() override;
 
  private:
+  friend class ::page_actions::PageActionControllerImpl;
   using PageActionModelsMap =
       std::map<actions::ActionId, std::unique_ptr<PageActionModelInterface>>;
 
@@ -494,7 +496,7 @@ class PageActionControllerImpl : public PageActionController,
   void ActionItemChanged(const actions::ActionItem* action_item);
   void PinnedActionsModelChanged();
 
-  std::unique_ptr<PageActionModelInterface> CreateModel(
+  virtual std::unique_ptr<PageActionModelInterface> CreateModel(
       actions::ActionId action_id,
       bool is_ephemeral);
 
@@ -554,6 +556,8 @@ class PageActionControllerImpl : public PageActionController,
 
   base::WeakPtrFactory<PageActionControllerImpl> weak_factory_{this};
 };
+}  // namespace chromium_impl
+
 
 // Returns page action IDs that are present in the browser window's root action
 // item.
