diff --git a/chrome/browser/ui/extensions/icon_with_badge_image_source.cc b/chrome/browser/ui/extensions/icon_with_badge_image_source.cc
index 9f316f14815790dc40cfc83ce6f8211f3d0545b8..457a8f8fcf86bb18096a9e7807b33ff0f25fb721 100644
--- a/chrome/browser/ui/extensions/icon_with_badge_image_source.cc
+++ b/chrome/browser/ui/extensions/icon_with_badge_image_source.cc
@@ -179,6 +179,7 @@ void IconWithBadgeImageSource::Draw(gfx::Canvas* canvas) {
   gfx::ImageSkiaRep rep = skia.GetRepresentation(canvas->image_scale());
   if (rep.scale() != canvas->image_scale()) {
     skia.AddRepresentation(
+        BRAVE_ICON_WITH_BADGE_IMAGE_SOURCE_DRAW_1
         ScaleImageSkiaRep(rep, extensions::ExtensionAction::ActionIconSize(),
                           canvas->image_scale()));
   }
@@ -190,6 +191,7 @@ void IconWithBadgeImageSource::Draw(gfx::Canvas* canvas) {
       (size().width() - extensions::ExtensionAction::ActionIconSize()) / 2.0);
   int y_offset = std::floor(
       (size().height() - extensions::ExtensionAction::ActionIconSize()) / 2.0);
+  BRAVE_ICON_WITH_BADGE_IMAGE_SOURCE_DRAW_2
   canvas->DrawImageInt(skia, x_offset, y_offset);
 
   // Draw a badge on the provided browser action icon's canvas.
