diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index f4d455d5dd7a24c1e50008f0a45cd614356bc6b0..84f3e7ef923b802a3fc3fef725d359a5c70c1065 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3338,6 +3338,7 @@ const FeatureEntry::FeatureVariation kVcSegmentationModelVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
