diff --git a/chrome/browser/net/system_network_context_manager.cc b/chrome/browser/net/system_network_context_manager.cc
index 688523d311b1f77b91b923ac9f0e23c649752f64..7eea43dd5ec8fcbfe9b20b462a6290b0b26e9823 100644
--- a/chrome/browser/net/system_network_context_manager.cc
+++ b/chrome/browser/net/system_network_context_manager.cc
@@ -295,7 +295,7 @@ class SystemNetworkContextManager::URLLoaderFactoryForSystem
     if (!manager_)
       return;
     manager_->GetURLLoaderFactory()->CreateLoaderAndStart(
-        std::move(request), routing_id, request_id, options, url_request,
+        std::move(request), routing_id, request_id, options, brave::OnBeforeSystemRequest(url_request),
         std::move(client), traffic_annotation);
   }
 
