diff --git a/chrome/browser/ui/toolbar/chrome_location_bar_model_delegate.h b/chrome/browser/ui/toolbar/chrome_location_bar_model_delegate.h
index 33f2e683f54f40d1fa5f6b58f0ed3a9a874efd50..5cd6507645dcea4470431e2c56c4f6c3554216af 100644
--- a/chrome/browser/ui/toolbar/chrome_location_bar_model_delegate.h
+++ b/chrome/browser/ui/toolbar/chrome_location_bar_model_delegate.h
@@ -65,6 +65,7 @@ class ChromeLocationBarModelDelegate : public LocationBarModelDelegate {
 
   // Helper method to get the navigation entry from the navigation controller.
   content::NavigationEntry* GetNavigationEntry() const;
+  BRAVE_CHROME_LOCATION_BAR_MODEL_DELEGATE_H_
 
  private:
   // The state of URL elision in the omnibox.
