diff --git a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
index 453996a664fcb99f9acb0c8e314ff85eaaeffeae..947a836b4ce11eb9e4ea1cfa948a8f075cb4afa5 100644
--- a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
+++ b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
@@ -162,7 +162,7 @@ std::unique_ptr<TabStrip> CreateTabStrip(
   std::unique_ptr<TabHoverCardController> hover_card_controller(
       std::make_unique<TabHoverCardController>(tab_strip_region_view,
                                                browser_view->browser()));
-  auto tab_strip = std::make_unique<TabStrip>(std::move(tabstrip_controller),
+  auto tab_strip = std::make_unique<BraveTabStrip>(std::move(tabstrip_controller),
                                               std::move(hover_card_controller));
   return tab_strip;
 }
