diff --git a/chrome/browser/global_features.cc b/chrome/browser/global_features.cc
index e1c76d0788c262d1e2f4fe20329d3a3c31c1d646..9b816385699617b3e9d522f10a841abb2da503e8 100644
--- a/chrome/browser/global_features.cc
+++ b/chrome/browser/global_features.cc
@@ -71,6 +71,7 @@
 #include "chrome/browser/on_device_translation/installer_impl.h"
 #endif  // BUILDFLAG(ENABLE_ON_DEVICE_TRANSLATION)
 
+GlobalFeatures* CreateBraveGlobalFeatures();
 namespace {
 
 // This is the generic entry point for test code to stub out browser
@@ -89,7 +90,7 @@ std::unique_ptr<GlobalFeatures> GlobalFeatures::CreateGlobalFeatures() {
     return GetFactory().Run();
   }
   // Constructor is protected.
-  return base::WrapUnique(new GlobalFeatures());
+  return base::WrapUnique(CreateBraveGlobalFeatures());
 }
 
 GlobalFeatures::~GlobalFeatures() = default;
