diff --git a/chrome/common/pref_names.h b/chrome/common/pref_names.h
index dfefd3d134ec5c6d4365d502fcb0da04fc3d591c..573d94b671b25b55c22b96c6d976e2a073524ce3 100644
--- a/chrome/common/pref_names.h
+++ b/chrome/common/pref_names.h
@@ -359,6 +359,8 @@ extern const char kImportDialogBookmarks[];
 extern const char kImportDialogHistory[];
 extern const char kImportDialogSavedPasswords[];
 extern const char kImportDialogSearchEngine[];
+extern const char kImportDialogCookies[];
+extern const char kImportDialogStats[];
 
 extern const char kProfileAvatarIndex[];
 extern const char kProfileLocalAvatarIndex[];
