diff --git a/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc b/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc
index a712f9eb70c1d74832a92c68cd58d1c672d2342f..2c86b156304a9df9a8639965ac0abfde3066fbc9 100644
--- a/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc
+++ b/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc
@@ -99,6 +99,8 @@ class MockPage : public tab_search::mojom::Page {
   MOCK_METHOD(void, TabUpdated, (tab_search::mojom::TabUpdateInfoPtr));
   MOCK_METHOD(void, TabsRemoved, (tab_search::mojom::TabsRemovedInfoPtr));
   MOCK_METHOD(void, TabUnsplit, ());
+  MOCK_METHOD(void, TabFocusShowFREChanged, (bool show));
+  MOCK_METHOD(void, TabFocusEnabledChanged, (bool enabled));
 };
 
 void ExpectNewTab(const tab_search::mojom::Tab* tab,
