diff --git a/chrome/installer/util/BUILD.gn b/chrome/installer/util/BUILD.gn
index fce0a3a5c27711bfba8487b02e8ad3f628414257..0eebe3a5515b87d90b85eb2c87c7563c26022e2d 100644
--- a/chrome/installer/util/BUILD.gn
+++ b/chrome/installer/util/BUILD.gn
@@ -202,6 +202,7 @@ static_library("with_no_strings") {
       "initial_preferences.h",
     ]
   }
+  import("//brave/chromium_src/chrome/installer/util/sources.gni") deps += brave_chromium_src_chrome_installer_util_deps sources += brave_chromium_src_chrome_installer_util_sources
 }
 
 # Use this version of installer_util to link to the generated strings in .rc
@@ -234,6 +235,7 @@ generate_embedded_i18n("generate_strings") {
   output_file_name_base = "installer_util_strings"
 
   branding = branding_path_product
+  sources = [ "//chrome/app/brave_strings.grd" ] import("//brave/installer/util/sources.gni") extractor_datafile = brave_installer_util_generate_strings_extractor_datafile
 }
 
 # Compile the generated .rc file.
