diff --git a/chrome/browser/extensions/extension_service.h b/chrome/browser/extensions/extension_service.h
index ee355da335aed9346b4a758975778998e7c93e08..e776c626fda16d5b41549dff3d670ac160494075 100644
--- a/chrome/browser/extensions/extension_service.h
+++ b/chrome/browser/extensions/extension_service.h
@@ -446,6 +446,7 @@ class ExtensionService : public ExtensionServiceInterface,
   }
 
  private:
+  friend class BraveExtensionService;
   // Loads extensions specified via a command line flag/switch.
   void LoadExtensionsFromCommandLineFlag(const char* switch_name);
 #if defined(OS_CHROMEOS)
