diff --git a/chrome/browser/ui/tabs/tab_model.h b/chrome/browser/ui/tabs/tab_model.h
index 3d4ff71a69f481102361cfd0f64c305fd40c51b1..446f2aaf86dd4383ef086e39e016445e020b01c0 100644
--- a/chrome/browser/ui/tabs/tab_model.h
+++ b/chrome/browser/ui/tabs/tab_model.h
@@ -161,8 +161,10 @@ class TabModel final : public TabInterface,
       TabInterfaceCallback callback) override;
 
   bool IsInNormalWindow() const override;
+#if !BUILDFLAG(IS_ANDROID)
   BrowserWindowInterface* GetBrowserWindowInterface() override;
   const BrowserWindowInterface* GetBrowserWindowInterface() const override;
+#endif  // !BUILDFLAG(IS_ANDROID)
   tabs::TabFeatures* GetTabFeatures() override;
   const tabs::TabFeatures* GetTabFeatures() const override;
   bool IsPinned() const override;
