diff --git a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
index 9130ae3712b6f4a0934077b3ae652d5d2da2bff5..216df04e561aa212b7b1b2ef174633109b050c04 100644
--- a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
+++ b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
@@ -34,7 +34,7 @@ ${function() {
             data-index="${index}">
         </profile-card>
       `)}
-      <cr-button id="addProfile" class="profile-item"
+      <cr-button id="addProfile" class="profile-item plain"
           @click="${this.onAddProfileClick_}"
           ?hidden="${!this.profileCreationAllowed_}"
           aria-labelledby="addProfileButtonLabel">
