diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index 2f7ac35fe7d61216a62a1ad9f69769ff45a73afc..c9a962e7b3de976394cdea1c4f950fa8d8dd4233 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -88,6 +88,7 @@ grit_strings("generated_resources") {
   if (is_android) {
     create_android_resources = true
   }
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
