diff --git a/chrome/test/base/testing_profile.cc b/chrome/test/base/testing_profile.cc
index 6cbe567313ee72ab643a26e00be3463a930be915..8eb36f2a7df8c2c606af5d729bd04fd379f71551 100644
--- a/chrome/test/base/testing_profile.cc
+++ b/chrome/test/base/testing_profile.cc
@@ -414,6 +414,7 @@ void TestingProfile::Init() {
       EnsureBrowserContextKeyedServiceFactoriesBuilt();
 
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
+  BRAVE_INIT
   if (!IsOffTheRecord()) {
     SupervisedUserSettingsService* settings_service =
         SupervisedUserSettingsServiceFactory::GetForKey(key_.get());
