diff --git a/chrome/browser/extensions/component_loader.h b/chrome/browser/extensions/component_loader.h
index fb2867f727fb3f7264bf5a41db5e8bb19d44959d..efc11ba68d9aa5873eff426cffb0bc778485ef63 100644
--- a/chrome/browser/extensions/component_loader.h
+++ b/chrome/browser/extensions/component_loader.h
@@ -113,7 +113,7 @@ class ComponentLoader : public KeyedService {
   // the loader will skip loading component extensions that weren't supposed to
   // be loaded unless we are in signed user session (ChromeOS). For all other
   // platforms this `skip_session_components` is expected to be unset.
-  void AddDefaultComponentExtensions(bool skip_session_components);
+  virtual void AddDefaultComponentExtensions(bool skip_session_components);
 
   // Similar to above but adds the default component extensions for kiosk mode.
   void AddDefaultComponentExtensionsForKioskMode(bool skip_session_components);
@@ -161,6 +161,7 @@ class ComponentLoader : public KeyedService {
   void set_profile_for_testing(Profile* profile) { profile_ = profile; }
 
  private:
+  friend class BraveComponentLoader;
   friend class ComponentLoaderFactory;
   friend class TtsApiTest;
   FRIEND_TEST_ALL_PREFIXES(ComponentLoaderTest, ParseManifest);
