diff --git a/chrome/browser/notifications/BUILD.gn b/chrome/browser/notifications/BUILD.gn
index a7a8f5ad87966aac876b85c19f9ed3fc5d19324c..99f0c007b96ce3b7f7569c52882cb52c1f12ad5d 100644
--- a/chrome/browser/notifications/BUILD.gn
+++ b/chrome/browser/notifications/BUILD.gn
@@ -79,6 +79,7 @@ if (is_android) {
       "//third_party/androidx:androidx_annotation_annotation_java",
       "//ui/android:ui_no_recycler_view_java",
     ]
+    deps += [ "//brave/browser/notifications:brave_java", ]
 
     srcjar_deps = [ ":notifications_java_enum_srcjar" ]
     resources_package = "org.chromium.chrome.browser.notifications"
