diff --git a/chrome/browser/ui/views/frame/multi_contents_view.h b/chrome/browser/ui/views/frame/multi_contents_view.h
index 4e08c7f6c050a37cea1767fac012a2e287493a82..3bf5e6a45745fc25679e0168182b8aace6c5b975 100644
--- a/chrome/browser/ui/views/frame/multi_contents_view.h
+++ b/chrome/browser/ui/views/frame/multi_contents_view.h
@@ -23,7 +23,7 @@
 class BrowserView;
 class ContentsWebView;
 class MultiContentsDropTargetView;
-class MultiContentsResizeArea;
+// class MultiContentsResizeArea;
 class MultiContentsViewDelegate;
 class MultiContentsViewDropTargetController;
 class MultiContentsViewMiniToolbar;
