diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 8867f81df66f641d2ccb2cdabd7b44abc611363f..d5eb282521cd14c810138aea833d52f9c7d8e319 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -69,6 +69,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
