diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index e06cdcd322877a65763bde7e5a82c7e5c14d5414..557e23611667f0d860a8c2c9ab56f4005f058a20 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2769,6 +2769,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
