diff --git a/chrome/browser/resources/side_panel/bookmarks/BUILD.gn b/chrome/browser/resources/side_panel/bookmarks/BUILD.gn
index bbf41a6fed4b7b92afc4ce5af72316af4b37ba79..f5ec1f755680d50660e5612046146da281779c61 100644
--- a/chrome/browser/resources/side_panel/bookmarks/BUILD.gn
+++ b/chrome/browser/resources/side_panel/bookmarks/BUILD.gn
@@ -74,4 +74,5 @@ build_webui("build") {
   html_to_wrapper_template = "detect"
   webui_context_type = "trusted"
   generate_code_cache = enable_webui_generate_code_cache
+  import("//brave/browser/resources/side_panel/bookmarks/sources.gni") non_web_component_files += brave_side_panel_bookmarks_non_web_component_files exclude_ts_preprocess_files = brave_side_panel_bookmarks_ts_local_files preprocess_deps = brave_side_panel_bookmarks_preprocess_extra_deps
 }
