diff --git a/chrome/browser/resources/settings/people_page/sync_controls.ts b/chrome/browser/resources/settings/people_page/sync_controls.ts
index be8ab78b417a98a8a671efcab0e694f9590d8eb3..7b4225c317267e480b41e28307f6a4e51c3075d9 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.ts
+++ b/chrome/browser/resources/settings/people_page/sync_controls.ts
@@ -143,7 +143,7 @@ export class SettingsSyncControlsElement extends
         'sync-prefs-changed', this.handleSyncPrefsChanged_.bind(this));
 
     const router = Router.getInstance();
-    if (router.getCurrentRoute() === routes.SYNC_ADVANCED) {
+    if (router.getCurrentRoute() === routes.BRAVE_SYNC_SETUP) {
       this.browserProxy_.didNavigateToSyncPage();
     }
   }
