diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
index 136dc3b7e9c04af1d1594f3f478a7cca42545062..5e5cbceacf6867098571c67f8d20e92b1b87161b 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
@@ -730,7 +730,7 @@ public abstract class ChromeActivity extends AsyncInitializationActivity
             var modalDialogManager = getModalDialogManager();
             assert modalDialogManager != null;
             SnackbarManager snackbarManager =
-                    new SnackbarManager(
+                    new org.chromium.chrome.browser.ui.messages.snackbar.BraveSnackbarManager(
                             this,
                             mBottomContainer,
                             windowAndroid,
@@ -3517,6 +3517,7 @@ public abstract class ChromeActivity extends AsyncInitializationActivity
      */
     public void onExitVr() {}
 
+    public void onTabStateInitializedHandler() {}
     private void clearToolbarResourceCache() {
         View v = findViewById(R.id.control_container);
         try {
