diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/notifications/NotificationsSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/notifications/NotificationsSettings.java
index 5ff9a1084c3567634e2ddbfa1e7d4cb9030d8e4d..7ca5ebc97387adf8492f2347e5732becec99c536 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/notifications/NotificationsSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/notifications/NotificationsSettings.java
@@ -27,7 +27,7 @@ import org.chromium.components.content_settings.ContentSettingsType;
  * notification channels at the top level and links to website specific notifications. This is only
  * used on pre-O devices, devices on Android O+ will link to the Android notification settings.
  */
-public class NotificationsSettings extends PreferenceFragmentCompat {
+public class NotificationsSettings extends org.chromium.chrome.browser.settings.BravePreferenceFragment {
     // These are package-private to be used in tests.
     static final String PREF_FROM_WEBSITES = "from_websites";
     static final String PREF_SUGGESTIONS = "content_suggestions";
