diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index 2c3ac524ed8bf228b9667cc49e8b37732da6eabe..1c0267290ff29caec5f10c05f7d1847cd1b2e4aa 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -32,6 +32,7 @@ if (optimize_webui) {
     deps = [
       ":unpak",
       "../../../../ui/webui/resources:preprocess",
+      "//brave/browser/resources/settings:unpak",
     ]
     excludes = [ "chrome://resources/js/cr.m.js" ]
   }
