diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 9cc63fbd174db7f37f69b3ab258c843c0b7a12b9..058789121b9a1c0f0206829c1aa95ff82df128b2 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -235,6 +235,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  import("//brave/renderer/sources.gni") sources += brave_chrome_renderer_sources deps += brave_chrome_renderer_deps public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
