diff --git a/chrome/installer/mac/signing/signing.py b/chrome/installer/mac/signing/signing.py
index 7ac09b875f7e6521a205730f3c7d09255712b139..0b5b76a5562b8ac09cbd460e38901f5fdcb5a89d 100644
--- a/chrome/installer/mac/signing/signing.py
+++ b/chrome/installer/mac/signing/signing.py
@@ -23,7 +23,7 @@ class InvalidAppGeometryException(ValueError):
 class Invoker(invoker.Base):
 
     def codesign(self, config, product, path):
-        command = ['codesign', '--sign', config.identity]
+        command = ['codesign', '--force', '--sign', config.identity]
         if config.notarize.should_notarize():
             # If the products will be notarized, the signature requires a secure
             # timestamp.
