diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index f1c66751bff73836e58c6550f16dba060b88bd8d..17c3b6b47cc8fb65d9d474fc41ebe79250cdbed6 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -701,6 +701,7 @@ class Browser : public TabStripModelObserver,
 
   StatusBubble* GetStatusBubbleForTesting();
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
