diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 93d76a4dcead4cf1386e3c99b829d49e770aef35..28d5775a660645c68ef5f8c20c13430a4e498922 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2521,6 +2521,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
