diff --git a/chrome/android/java/src/org/chromium/chrome/browser/datareduction/settings/DataReductionPreferenceFragment.java b/chrome/android/java/src/org/chromium/chrome/browser/datareduction/settings/DataReductionPreferenceFragment.java
index fc53fd37f25f8a0b87891fdabb131e51da3ab294..4c2afc95223142929c4e1fbeb0af10517e026e5c 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/datareduction/settings/DataReductionPreferenceFragment.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/datareduction/settings/DataReductionPreferenceFragment.java
@@ -35,7 +35,7 @@ import java.util.Locale;
 /**
  * Settings fragment that allows the user to configure Data Saver.
  */
-public class DataReductionPreferenceFragment extends PreferenceFragmentCompat {
+public class DataReductionPreferenceFragment extends org.chromium.chrome.browser.settings.BravePreferenceFragment {
     public static final String FROM_MAIN_MENU = "FromMainMenu";
 
     public static final String PREF_DATA_REDUCTION_SWITCH = "data_reduction_switch";
