diff --git a/chrome/browser/chrome_content_browser_client.h b/chrome/browser/chrome_content_browser_client.h
index 71dfd396b405e4344341590f446bf268f4b483c2..23b1274cda34e91c0bdce4653e0bb2724874a3c9 100644
--- a/chrome/browser/chrome_content_browser_client.h
+++ b/chrome/browser/chrome_content_browser_client.h
@@ -793,6 +793,7 @@ class ChromeContentBrowserClient : public content::ContentBrowserClient {
   }
 
  private:
+  friend class BraveContentBrowserClient;
   friend class DisableWebRtcEncryptionFlagTest;
   friend class InProcessBrowserTest;
 
