diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index f68bd44ecd82fb076ffe9fa08f533f2f728d48fa..f3eb14c6709f8f49b45ca968cf32c94c1c262b88 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -553,7 +553,7 @@ Browser* Browser::Create(const CreateParams& params) {
   // should handle this; see e.g. crbug.com/1141608 and crbug.com/1261628.
   CHECK_EQ(CreationStatus::kOk, GetCreationStatusForProfile(params.profile));
 
-  std::unique_ptr<Browser> browser = base::WrapUnique(new Browser(params));
+  std::unique_ptr<Browser> browser = base::WrapUnique(new BraveBrowser(params));
   Browser* const browser_ptr = browser.get();
   BrowserManagerServiceFactory::GetForProfile(params.profile)
       ->AddBrowser(std::move(browser));
@@ -578,7 +578,7 @@ Browser::Browser(const CreateParams& params)
       window_(nullptr),
       tab_strip_model_delegate_(
           std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
-      tab_strip_model_(std::make_unique<TabStripModel>(
+      tab_strip_model_(std::make_unique<BraveTabStripModel>(
           tab_strip_model_delegate_.get(),
           params.profile,
           // Tab groups are disabled for app browsers.
