diff --git a/chrome/browser/extensions/extension_management.cc b/chrome/browser/extensions/extension_management.cc
index b3eec417cffd098be797515c47ef09a415f91a87..fa7b747711c9f269849c8b4fd1121de1c3cae69b 100644
--- a/chrome/browser/extensions/extension_management.cc
+++ b/chrome/browser/extensions/extension_management.cc
@@ -1084,7 +1084,7 @@ ExtensionManagementFactory::BuildServiceInstanceForBrowserContext(
     content::BrowserContext* context) const {
   TRACE_EVENT0("browser,startup",
                "ExtensionManagementFactory::BuildServiceInstanceFor");
-  return std::make_unique<ExtensionManagement>(
+  return std::make_unique<BraveExtensionManagement>(
       Profile::FromBrowserContext(context));
 }
 
