diff --git a/chrome/browser/ui/startup/startup_tab_provider.cc b/chrome/browser/ui/startup/startup_tab_provider.cc
index f0875670077d29f4d62b3095f3bac5a67793fcca..f5c1bb3fc0f1cb294aba2e766d6ff1ed04e30e2f 100644
--- a/chrome/browser/ui/startup/startup_tab_provider.cc
+++ b/chrome/browser/ui/startup/startup_tab_provider.cc
@@ -292,6 +292,7 @@ StartupTabs StartupTabProviderImpl::GetPrivacySandboxTabs(
 bool StartupTabProviderImpl::CanShowWelcome(bool is_signin_allowed,
                                             bool is_child_account,
                                             bool is_force_signin_enabled) {
+  if ((true)) return true;
   return is_signin_allowed && !is_child_account && !is_force_signin_enabled;
 }
 
