diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java
index 949ac0ed0ac7d0c2b5ebcbc628e60d7e8d2d6e08..79e88e544c9d45a2eb2c219ffcf5ffe7d50f8df3 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java
@@ -39,7 +39,7 @@ import org.chromium.components.feature_engagement.Tracker;
  */
 class BottomToolbarCoordinator {
     /** The browsing mode bottom toolbar component */
-    private final BrowsingModeBottomToolbarCoordinator mBrowsingModeCoordinator;
+    protected final BrowsingModeBottomToolbarCoordinator mBrowsingModeCoordinator;
 
     /** The tab switcher mode bottom toolbar component */
     private TabSwitcherBottomToolbarCoordinator mTabSwitcherModeCoordinator;
@@ -51,8 +51,8 @@ class BottomToolbarCoordinator {
     private final ThemeColorProvider mThemeColorProvider;
 
     /** The overview mode manager. */
-    private OverviewModeBehavior mOverviewModeBehavior;
-    private OverviewModeObserver mOverviewModeObserver;
+    protected OverviewModeBehavior mOverviewModeBehavior;
+    protected OverviewModeObserver mOverviewModeObserver;
 
     /** The activity tab provider. */
     private ActivityTabProvider mTabProvider;
@@ -99,7 +99,7 @@ class BottomToolbarCoordinator {
             setUrlBarFocusAction.onResult(LocationBar.OmniboxFocusReason.ACCELERATOR_TAP);
         };
 
-        mBrowsingModeCoordinator = new BrowsingModeBottomToolbarCoordinator(root, tabProvider,
+        mBrowsingModeCoordinator = new BraveBrowsingModeBottomToolbarCoordinator(root, tabProvider,
                 homeButtonListener, searchAcceleratorListener, mShareButtonListenerSupplier,
                 tabsSwitcherLongClickListner);
 
@@ -228,13 +228,16 @@ class BottomToolbarCoordinator {
 
     private void onShareDelegateAvailable(ShareDelegate shareDelegate) {
         final OnClickListener shareButtonListener = v -> {
+/*
             if (BottomToolbarVariationManager.isShareButtonOnBottom()) {
                 recordBottomToolbarUseForIPH();
                 RecordUserAction.record("MobileBottomToolbarShareButton");
             }
 
+*/
             Tab tab = mTabProvider.get();
-            shareDelegate.share(tab, /*shareDirectly=*/false);
+            tab.GetActivity().addOrEditBookmark(tab);
+//        shareDelegate.share(tab, /*shareDirectly=*/false);
         };
 
         mShareButtonListenerSupplier.set(shareButtonListener);
@@ -260,4 +263,5 @@ class BottomToolbarCoordinator {
             listener.onClick(v);
         };
     }
+    public void updateBookmarkButton(boolean isBookmarked, boolean editingAllowed) {if (mBrowsingModeCoordinator != null) {((BraveBrowsingModeBottomToolbarCoordinator)mBrowsingModeCoordinator).updateBookmarkButton(isBookmarked, editingAllowed);}}
 }
