diff --git a/chrome/browser/net/system_network_context_manager.cc b/chrome/browser/net/system_network_context_manager.cc
index 2293a2ecb4857e1fa37833d70d753a08c3e2a310..31dbf025ef4895ed69f3d72f7afa955fb2491713 100644
--- a/chrome/browser/net/system_network_context_manager.cc
+++ b/chrome/browser/net/system_network_context_manager.cc
@@ -476,7 +476,7 @@ class SystemNetworkContextManager::URLLoaderFactoryForSystem
     if (!manager_)
       return;
     manager_->GetURLLoaderFactory()->CreateLoaderAndStart(
-        std::move(receiver), request_id, options, url_request,
+        std::move(receiver), request_id, options, BRAVE_CREATE_LOADER_AND_START,
         std::move(client), traffic_annotation);
   }
 
