diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index faa7b52a0ffa0df0461ea2cdfa3b0e888a571a82..83757a70d2bff971949cbb17dae5826b25cf2420 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -392,6 +392,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data_deps = []
 
   if (enable_pdf) {
