diff --git a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
index eb35127aad902296397584fab726cd0a95bb423c..c216b32ee813f0cd6336b97795708e712448e996 100644
--- a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
+++ b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
@@ -105,13 +105,20 @@ class HorizontalTabStripRegionViewOld : public TabStripRegionView {
 
   bool HasLeadingButtons() const;
 
+  views::Button* new_tab_button() { return new_tab_button_; }
+  TabStripComboButton* GetComboButton() { return combo_button_; }
+
  private:
+  friend class BraveVerticalTabStripRegionView;
+  friend class BraveTabStrip;
+  friend class BraveHorizontalTabStripRegionView;
+  FRIEND_TEST_ALL_PREFIXES(VerticalTabStripBrowserTest, MinHeight);
   // Updates the border padding for `new_tab_button_`.  This should be called
   // whenever any input of the computation of the border's sizing changes.
   void UpdateButtonBorders();
 
   // Updates the left and right margins for the tab strip.
-  void UpdateTabStripMargin();
+  virtual void UpdateTabStripMargin();
 
   // Gets called on `Layout` and adjusts the x-axis position of the `view` based
   // on `offset`. This should only used for views that show before tab strip.
