diff --git a/chrome/browser/content_settings/BUILD.gn b/chrome/browser/content_settings/BUILD.gn
index bc8123d22dae41b883cd3824ae3097a352845844..b16770a8560af2bfc1a3e55e10b0320c321b9fed 100644
--- a/chrome/browser/content_settings/BUILD.gn
+++ b/chrome/browser/content_settings/BUILD.gn
@@ -117,6 +117,7 @@ source_set("impl") {
   if (enable_pdf) {
     deps += [ "//chrome/browser/pdf" ]
   }
+  import("//brave/browser/content_settings/sources.gni") deps += brave_browser_content_settings_deps
 }
 
 # Fixes circular deps with different components. For example -
