diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index c6dabe9887f15e22fac5e1aeeeac0c8fe61603a0..1982696d735fe4da06bbdbac28724fc6300df162 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4853,6 +4853,7 @@ constexpr char kWebiumFlag[] = "webium";
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
