diff --git a/chrome/browser/sessions/tab_restore_service_factory.cc b/chrome/browser/sessions/tab_restore_service_factory.cc
index 83855dacb105b888ed643ad2501375e72d483120..dae85d14204869b656377e58e02fe4e06a822116 100644
--- a/chrome/browser/sessions/tab_restore_service_factory.cc
+++ b/chrome/browser/sessions/tab_restore_service_factory.cc
@@ -15,7 +15,6 @@ namespace {
 std::unique_ptr<KeyedService> BuildTemplateService(
     content::BrowserContext* browser_context) {
   Profile* profile = Profile::FromBrowserContext(browser_context);
-  DCHECK(!profile->IsOffTheRecord());
   auto client = std::make_unique<ChromeTabRestoreServiceClient>(profile);
   return std::make_unique<sessions::TabRestoreServiceImpl>(
       std::move(client), profile->GetPrefs(), nullptr);
@@ -59,7 +58,7 @@ TabRestoreServiceFactory::TabRestoreServiceFactory()
     : ProfileKeyedServiceFactory(
           "sessions::TabRestoreService",
           ProfileSelections::Builder()
-              .WithRegular(ProfileSelection::kOriginalOnly)
+              .WithRegular(ProfileSelection::kOwnInstance)
               // TODO(crbug.com/40257657): Check if this service is needed in
               // Guest mode.
               .WithGuest(ProfileSelection::kOriginalOnly)
