diff --git a/chrome/browser/signin/chrome_signin_helper.cc b/chrome/browser/signin/chrome_signin_helper.cc
index dfb7b597b28b4e7b740dc4d64343e3a3a9302d88..e866ca9d5a262e0ca82f1a8d692ea858f560d379 100644
--- a/chrome/browser/signin/chrome_signin_helper.cc
+++ b/chrome/browser/signin/chrome_signin_helper.cc
@@ -248,6 +248,7 @@ void ProcessMirrorHeader(
   // Do not allow non-Google origins to open incognito windows.
   // TODO(crbug.com/40064889): Expand this check to all Mirror headers,
   //                          regardless of `service_type`.
+  BRAVE_PROCESS_MIRROR_HEADERS_IF_FALSE
   if (service_type == GAIA_SERVICE_TYPE_INCOGNITO) {
     GURL initiator_url =
         request_initiator ? request_initiator->GetURL() : GURL();
