diff --git a/chrome/browser/ui/views/frame/tab_strip_region_view.h b/chrome/browser/ui/views/frame/tab_strip_region_view.h
index 139509b4d834a87f457c55d6a01b7dcd3901165d..20244f56d43f83f34eab7f0cfec09889cae3ce97 100644
--- a/chrome/browser/ui/views/frame/tab_strip_region_view.h
+++ b/chrome/browser/ui/views/frame/tab_strip_region_view.h
@@ -24,7 +24,7 @@ class ProductSpecificationsButton;
 
 // Container for the tabstrip and the other views sharing space with it -
 // with the exception of the caption buttons.
-class TabStripRegionView final : public views::AccessiblePaneView {
+class TabStripRegionView : public views::AccessiblePaneView {
   METADATA_HEADER(TabStripRegionView, views::AccessiblePaneView)
 
  public:
