diff --git a/chrome/browser/chrome_browser_main.cc b/chrome/browser/chrome_browser_main.cc
index b2619baa8f61dfc4699a27f17e44fc4a94dd94f7..3f89705462bef958e0cfb27adeec33e651d7c048 100644
--- a/chrome/browser/chrome_browser_main.cc
+++ b/chrome/browser/chrome_browser_main.cc
@@ -1061,6 +1061,7 @@ void ChromeBrowserMainParts::ToolkitInitialized() {
 
 #if BUILDFLAG(IS_MAC) && BUILDFLAG(ENABLE_UPDATER)
 void PromptUpdaterPromotion() {
+  if (!ShouldPromptUpdaterPromotion()) return;
   content::GetUIThreadTaskRunner({})->PostTask(
       FROM_HERE, base::BindOnce([]() {
         if (base::CommandLine::ForCurrentProcess()->HasSwitch(
