diff --git a/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc b/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
index 3a3d1a6a490dfbf29ebff7b2d7d6637ebd5caac4..fe234f23b8269e7a295eed76e190e62dd7cb00d8 100644
--- a/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
+++ b/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
@@ -742,7 +742,7 @@ BrowserViewTabbedLayoutImpl::CalculateProposedLayout(
   }
 
   BrowserLayoutParams params = layout_data_->revised_params;
-  bool needs_exclusion = true;
+  bool needs_exclusion = !delegate().ShouldShowVerticalTabs();
   const HorizontalLayout& horizontal_layout = layout_data_->horizontal_layout;
 
   // Lay out horizontal tab strip region if present.
