diff --git a/chrome/browser/resources/settings/site_settings/pdf_documents.js b/chrome/browser/resources/settings/site_settings/pdf_documents.js
index 66f61a6db59bc5a549db3ec505518462f72b5cb7..367b6e80aebaea557fba9aa216ce70dc9fec1bdf 100644
--- a/chrome/browser/resources/settings/site_settings/pdf_documents.js
+++ b/chrome/browser/resources/settings/site_settings/pdf_documents.js
@@ -17,4 +17,4 @@ Polymer({
       notify: true,
     },
   },
-});
+});
\ No newline at end of file
