diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index 68c047f2ec4d625a04aa33d30f24e41154375d87..ca6da400355ba7b7f357c7edfae02a4adcfb12d4 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -975,6 +975,7 @@ void BookmarkBarView::Layout() {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   // Only render the tab group separtor if there are groups and bookmarks
   // visible. Must done after the bookmark buttons are rendered.
