diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index c830709a54d249ebec59377f20ed6eae26e9603b..89aa4366d33916cded956fb1aeb76295ccd453b5 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -158,6 +158,7 @@ source_set("configs") {
   if (is_mac) {
     deps += [ "//chrome/browser/ui/webui/unexportable_keys_internals" ]
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("webui") {
