diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 9c16559df8a273359154a727edf826bd4d257dd2..ece0be1ea083fff9bd3c66df71699634766bec0b 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2465,6 +2465,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
