diff --git a/chrome/browser/ui/webui/settings/people_handler.cc b/chrome/browser/ui/webui/settings/people_handler.cc
index 6dc8e03ca5ac6e10a59708d75796fa99a24eada1..29393972bcac72094e938970da2f0df137319847 100644
--- a/chrome/browser/ui/webui/settings/people_handler.cc
+++ b/chrome/browser/ui/webui/settings/people_handler.cc
@@ -337,6 +337,7 @@ void PeopleHandler::RegisterMessages() {
       "SyncStartKeyRetrieval",
       base::BindRepeating(&PeopleHandler::HandleStartKeyRetrieval,
                           base::Unretained(this)));
+  BRAVE_REGISTER_MESSAGES
 }
 
 void PeopleHandler::OnJavascriptAllowed() {
@@ -616,6 +617,7 @@ void PeopleHandler::HandleShowSetupUI(const base::ListValue* args) {
 
   syncer::SyncService* service = GetSyncService();
 
+  BRAVE_HANDLE_SHOW_SETUP_UI
   if (service && !sync_blocker_)
     sync_blocker_ = service->GetSetupInProgressHandle();
 
@@ -896,8 +898,7 @@ std::unique_ptr<base::DictionaryValue> PeopleHandler::GetSyncStatusDictionary()
   sync_status->SetBoolean(
       "firstSetupInProgress",
       service && !disallowed_by_policy && service->IsSetupInProgress() &&
-          !service->GetUserSettings()->IsFirstSetupComplete() &&
-          identity_manager->HasPrimaryAccount());
+          !service->GetUserSettings()->IsFirstSetupComplete());
 
   const sync_ui_util::StatusLabels status_labels =
       sync_ui_util::GetStatusLabels(profile_);
@@ -1045,6 +1046,7 @@ void PeopleHandler::MarkFirstSetupComplete() {
 }
 
 bool PeopleHandler::IsProfileAuthNeededOrHasErrors() {
+  return false;
   return !IdentityManagerFactory::GetForProfile(profile_)
               ->HasPrimaryAccount() ||
          SigninErrorControllerFactory::GetForProfile(profile_)->HasError();
