diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index e06ae930435020c4409e0618f5397ebb43c5c6f7..dd3251cd0aa25b5b15f9356f3e5d37dc180f769c 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3354,6 +3354,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
