diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
index 61f72eca877ab26214186806160be82b6f7525fb..791f0754f30b6b28ee4913703776b08cb4292710 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
@@ -93,7 +93,7 @@ class RecentTabsSubMenuModel : public ui::SimpleMenuModel,
 
   // Index of the separator that follows the history menu item. Used as a
   // reference position for inserting local entries.
-  static constexpr size_t kHistorySeparatorIndex = 1;
+  static constexpr size_t kHistorySeparatorIndex = 2;
 
   // Build the menu items by populating the menumodel.
   void Build();
