diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 6256a29956a8cd67767a15b2b3ed1f009a4d8891..810f8d70a7df9ce15668b9d5cab9d3e731f37364 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2522,6 +2522,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
