diff --git a/chrome/browser/ui/views/permission_bubble/permission_prompt_impl.cc b/chrome/browser/ui/views/permission_bubble/permission_prompt_impl.cc
index 489db17efe96db01666c57e01f6be0c2d37be873..84d5a95b37b2219a4abbb3c28554be0168b12562 100644
--- a/chrome/browser/ui/views/permission_bubble/permission_prompt_impl.cc
+++ b/chrome/browser/ui/views/permission_bubble/permission_prompt_impl.cc
@@ -148,6 +148,7 @@ PermissionsBubbleDialogDelegateView::PermissionsBubbleDialogDelegateView(
     AddChildView(label_container);
   }
 
+  AddAdditionalWidevineViewControlsIfNeeded(this, requests);
   chrome::RecordDialogCreation(chrome::DialogIdentifier::PERMISSIONS);
 }
 
