diff --git a/chrome/browser/ui/views/profiles/profile_menu_view.cc b/chrome/browser/ui/views/profiles/profile_menu_view.cc
index f8a4a95717c5de2b1e092c108c21018483e7aa5d..5a6da9992830dad342260aa3608056718744e957 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view.cc
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.cc
@@ -1041,6 +1041,7 @@ void ProfileMenuView::AddOptionsView(bool display_lock,
   if (is_guest) {
     CreateAndAddButton(
         CreateVectorIcon(kCloseAllIcon),
+        BRAVE_ADDOPTIONSVIEW_ADD_TOR_EXIT_BUTTON_
         l10n_util::GetStringUTF16(IDS_PROFILES_EXIT_GUEST),
         base::BindRepeating(&ProfileMenuView::OnExitProfileButtonClicked,
                             base::Unretained(this)));
