diff --git a/chrome/browser/ui/webui/settings/import_data_handler.h b/chrome/browser/ui/webui/settings/import_data_handler.h
index f10aab2d6ed4e76568cac16860d20b2198d0287e..f518b7f2958f7b5d3206039dd9bbcd7f3e36ebbd 100644
--- a/chrome/browser/ui/webui/settings/import_data_handler.h
+++ b/chrome/browser/ui/webui/settings/import_data_handler.h
@@ -37,6 +37,7 @@ class ImportDataHandler : public SettingsPageUIHandler,
   void OnJavascriptAllowed() override {}
   void OnJavascriptDisallowed() override;
 
+  BRAVE_IMPORT_DATA_HANDLER_H
  private:
   void StartImport(const importer::SourceProfile& source_profile,
                    uint16_t imported_items);
