diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index 761cd13202557f33be5dec6a0b9193679210467e..ba35f7af0c7385279fe2d68a51e8dc9b56a6be84 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -430,6 +430,7 @@ Browser::CreationStatus Browser::GetCreationStatusForProfile(Profile* profile) {
 // static
 Browser* Browser::Create(const CreateParams& params) {
   CHECK_EQ(CreationStatus::kOk, GetCreationStatusForProfile(params.profile));
+  BRAVE_BROWSER_CREATE
   return new Browser(params);
 }
 
@@ -441,7 +442,7 @@ Browser::Browser(const CreateParams& params)
       tab_strip_model_delegate_(
           std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
       tab_strip_model_(
-          std::make_unique<TabStripModel>(tab_strip_model_delegate_.get(),
+          std::make_unique<BraveTabStripModel>(tab_strip_model_delegate_.get(),
                                           params.profile)),
       app_name_(params.app_name),
       is_trusted_source_(params.trusted_source),
