diff --git a/chrome/browser/ui/views/tabs/new_tab_button.h b/chrome/browser/ui/views/tabs/new_tab_button.h
index b53bac3e26964d030725641d973fd5693b903425..b5a82e8d69725ee60707a2fcde5aaad38087cf49 100644
--- a/chrome/browser/ui/views/tabs/new_tab_button.h
+++ b/chrome/browser/ui/views/tabs/new_tab_button.h
@@ -66,6 +66,7 @@ class NewTabButton : public views::ImageButton,
   void RemoveLayerBeneathView(ui::Layer* old_layer) override;
 
  private:
+  friend class BraveNewTabButton;
 // views::ImageButton:
 #if defined(OS_WIN)
   void OnMouseReleased(const ui::MouseEvent& event) override;
@@ -89,12 +90,14 @@ class NewTabButton : public views::ImageButton,
   void PaintFill(gfx::Canvas* canvas) const;
 
   // Paints a properly sized plus (+) icon into the center of the button.
+  virtual
   void PaintPlusIcon(gfx::Canvas* canvas) const;
 
   SkColor GetButtonFillColor() const;
 
   // Returns the path for the given |origin| and |scale|.  If |extend_to_top| is
   // true, the path is extended vertically to y = 0.
+  virtual
   SkPath GetBorderPath(const gfx::Point& origin,
                        float scale,
                        bool extend_to_top) const;
