diff --git a/chrome/browser/permissions/permission_uma_util.cc b/chrome/browser/permissions/permission_uma_util.cc
index f6dd452d5f2fa800cd39cec3611fa01c899f6b98..0a681059978b661add0f1406aa6b2248565fe1f6 100644
--- a/chrome/browser/permissions/permission_uma_util.cc
+++ b/chrome/browser/permissions/permission_uma_util.cc
@@ -65,7 +65,7 @@ namespace {
 
 const int kPriorCountCap = 10;
 
-std::string GetPermissionRequestString(PermissionRequestType type) {
+std::string GetPermissionRequestString_ChromiumImpl(PermissionRequestType type) {
   switch (type) {
     case PermissionRequestType::MULTIPLE:
       return "AudioAndVideoCapture";
@@ -472,6 +472,9 @@ void PermissionUmaUtil::RecordPermissionAction(
       base::UmaHistogramEnumeration("Permissions.Action.Nfc", action,
                                     PermissionAction::NUM);
       break;
+    case ContentSettingsType::AUTOPLAY:
+      BraveRecordPermissionAction(permission, secure_origin, action);
+      break;
     // The user is not prompted for these permissions, thus there is no
     // permission action recorded for them.
     default:
