diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 535c4f04c0f39b38a8831913437064c63af67998..2de4861340dc65721a166e20b2b07d64d9f01c4e 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -558,6 +558,9 @@ jumbo_split_static_library("ui") {
     "//ui/webui",
     "//v8:v8_version",
   ]
+  deps += [
+    "//brave/browser/ui",
+  ]
   allow_circular_includes_from +=
       [ "//chrome/browser/ui/webui/bluetooth_internals" ]
 
@@ -2282,10 +2285,13 @@ jumbo_split_static_library("ui") {
       ]
       deps += [ "//google_update" ]
     } else {
-      sources += [
-        "webui/help/version_updater_basic.cc",
-        "webui/help/version_updater_basic.h",
-      ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [
+          "webui/help/version_updater_basic.cc",
+          "webui/help/version_updater_basic.h",
+        ]
+      }
     }
   } else {  # 'OS!="win"
     # The Cast Linux build sets toolkit_views to false, but many files in this
