diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index 006e0379877f11c18b690264d1671450d4eb92de..db4d9c261fdf391280ff81e4a71d5c96f74975c8 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -226,6 +226,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
