diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index 12390b95903a69f209c8bd708c944011f1d8c682..5aa06724c469e3d1b4aef4f8178deaa0df91a59f 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -9,6 +9,7 @@ import {pageVisibility} from './page_visibility.js';
 import type {SettingsRoutes} from './router.js';
 import {Route, Router} from './router.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -329,6 +330,7 @@ function createRoutes(): SettingsRoutes {
           loadTimeData.getString('performancePageTitle'));
     }
   }
+  addBraveRoutes(r);
   return r as unknown as SettingsRoutes;
 }
 
