diff --git a/chrome/browser/ui/views/page_info/page_info_view_factory.cc b/chrome/browser/ui/views/page_info/page_info_view_factory.cc
index b38d6049f7451e05e8b82fbb785374eaa0600304..f542d140e4d19c9bd271bd8138bd34dad5bbb7ba 100644
--- a/chrome/browser/ui/views/page_info/page_info_view_factory.cc
+++ b/chrome/browser/ui/views/page_info/page_info_view_factory.cc
@@ -466,6 +466,7 @@ const ui::ImageModel PageInfoViewFactory::GetPermissionIcon(
 
   icon = &gfx::VectorIcon::EmptyIcon();
   switch (info.type) {
+    BRAVE_PAGE_INFO_VIEW_FACTORY_GET_PERMISSION_ICON
     case ContentSettingsType::COOKIES:
       icon = &vector_icons::kDatabaseIcon;
       break;
