diff --git a/chrome/browser/resources/bookmarks/api_listener.ts b/chrome/browser/resources/bookmarks/api_listener.ts
index 49dbba8a713601549c5d93bf8a8c2dcd2431888c..8b7295b922c22c14b5534d1a36d99d337cca40d3 100644
--- a/chrome/browser/resources/bookmarks/api_listener.ts
+++ b/chrome/browser/resources/bookmarks/api_listener.ts
@@ -118,6 +118,7 @@ function onChildrenReordered(
  */
 function onImportBegan() {
   chrome.bookmarks.onCreated.removeListener(onBookmarkCreated);
+  chrome.bookmarks.onMoved.removeListener(onBookmarkMoved);
   document.dispatchEvent(new CustomEvent('import-began'));
 }
 
@@ -126,6 +127,7 @@ function onImportEnded() {
     dispatch(refreshNodes(normalizeNodes(results[0]!)));
   });
   chrome.bookmarks.onCreated.addListener(onBookmarkCreated);
+  chrome.bookmarks.onMoved.addListener(onBookmarkMoved);
   document.dispatchEvent(new CustomEvent('import-ended'));
 }
 
