diff --git a/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc b/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
index 8e22a9f4cac677ec3694f9c8ebda7d18597ab357..dfc68130032f770c8c6c23b2083ff765f0f494f8 100644
--- a/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
+++ b/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
@@ -554,6 +554,7 @@ void CheckClientDownloadRequestBase::SendRequest() {
   request->set_archive_directory_count(directory_count_);
   request->set_request_ap_verdicts(is_under_advanced_protection_);
 
+  BRAVE_SEND_REQUEST_FILTER
   if (!request->SerializeToString(&client_download_request_data_)) {
     FinishRequest(DownloadCheckResult::UNKNOWN, REASON_INVALID_REQUEST_PROTO);
     return;
