diff --git a/chrome/browser/ssl/https_upgrades_util.cc b/chrome/browser/ssl/https_upgrades_util.cc
index 7e66c7bc95f53b1db95bee38e29551c957608c67..11c844c36661ff4f2e0b34f20ab46953b5f5745b 100644
--- a/chrome/browser/ssl/https_upgrades_util.cc
+++ b/chrome/browser/ssl/https_upgrades_util.cc
@@ -75,11 +75,11 @@ ComputeInterstitialState(content::WebContents* web_contents, const GURL& url) {
   security_interstitials::https_only_mode::HttpInterstitialState
       interstitial_state;
   interstitial_state.enabled_by_pref =
-      prefs && prefs->GetBoolean(prefs::kHttpsOnlyModeEnabled);
+      prefs && prefs->GetBooleanOr(prefs::kHttpsOnlyModeEnabled, NormalWindowHttpsOnly(url, profile));
 
   if (base::FeatureList::IsEnabled(features::kHttpsFirstModeIncognito)) {
     if (profile->IsIncognitoProfile() && prefs &&
-        prefs->GetBoolean(prefs::kHttpsFirstModeIncognito)) {
+        prefs->GetBooleanOr(prefs::kHttpsFirstModeIncognito, NormalWindowHttpsOnly(url, profile))) {
       interstitial_state.enabled_by_incognito = true;
     }
   }
