diff --git a/chrome/browser/actor/site_policy.cc b/chrome/browser/actor/site_policy.cc
index 78aad92e13dca22911587d7faee8ad8344e066db..66c0a1b388fed8708828d06ce7a2fcd7dba02b48 100644
--- a/chrome/browser/actor/site_policy.cc
+++ b/chrome/browser/actor/site_policy.cc
@@ -160,6 +160,7 @@ void MayActOnUrlInternal(const GURL& url,
     return;
   }
 
+  BRAVE_MAY_ACT_ON_URL_INTERNAL
   if (IsActorSafetyCheckDisabled()) {
     decision_wrapper->Accept();
     return;
