diff --git a/chrome/browser/resources/history/history.ts b/chrome/browser/resources/history/history.ts
index cff9b4ac74757abba28d3eb993787ee14a4c7f63..98382b14322e7632a29266fbe91294855972a3f1 100644
--- a/chrome/browser/resources/history/history.ts
+++ b/chrome/browser/resources/history/history.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'chrome://brave-resources/polymer_overriding.js'
 import './app.js';
 
 export {ensureLazyLoaded, listenForPrivilegedLinkClicks} from './app.js';
