diff --git a/chrome/browser/prefs/BUILD.gn b/chrome/browser/prefs/BUILD.gn
index 90512dd781e7c329b65e1bd768d4b4f540360944..92b3daa8b5cb3819285ee518fa9db8a2e0a92f66 100644
--- a/chrome/browser/prefs/BUILD.gn
+++ b/chrome/browser/prefs/BUILD.gn
@@ -162,6 +162,7 @@ source_set("impl") {
     "//services/preferences/tracked",
     "//services/preferences/tracked:features",
   ]
+  import("//brave/browser/brave_news/sources.gni") import("//brave/components/brave_wallet/sources.gni") deps += brave_browser_brave_news_prefs_deps + brave_chrome_browser_prefs_impl_deps
 
   if (enable_extensions_core) {
     deps += [ "//extensions/common" ]
