diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index 71b39d448839289b827fc80c3dc590a06fa13da7..299dab18d9225a7ef2b5a7bf01e98b9cc66dbf90 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -73,6 +73,8 @@ grit("generated_resources") {
   if (is_android) {
     outputs += android_generated_java_resources
   }
+
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
@@ -95,7 +97,7 @@ grit("google_chrome_strings") {
 }
 
 grit("chromium_strings") {
-  source = "chromium_strings.grd"
+  source = "${branding_path_component}_strings.grd"
   defines = chrome_grit_defines
   output_dir = "$root_gen_dir/chrome"
   outputs =
