diff --git a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
index 9b892ae0a0de5597d69d10c83486171e9759521e..d31856eee73b02036add6845f4aa442d61e24b12 100644
--- a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
+++ b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
@@ -74,7 +74,7 @@ content::WebUIDataSource* CreateAndAddBookmarksUIHTMLSource(Profile* profile) {
       {"editDialogInvalidUrl", IDS_BOOKMARK_MANAGER_INVALID_URL},
       {"editDialogNameInput", IDS_BOOKMARK_MANAGER_NAME_INPUT_PLACE_HOLDER},
       {"editDialogUrlInput", IDS_BOOKMARK_MANAGER_URL_INPUT_PLACE_HOLDER},
-      {"emptyList", IDS_BOOKMARK_MANAGER_EMPTY_LIST},
+      {"emptyList", IDS_BRAVE_BOOKMARK_MANAGER_EMPTY_LIST},
       {"emptyUnmodifiableList", IDS_BOOKMARK_MANAGER_EMPTY_UNMODIFIABLE_LIST},
       {"folderLabel", IDS_BOOKMARK_MANAGER_FOLDER_LABEL},
       {"importBegan", IDS_BOOKMARK_MANAGER_MENU_IMPORT_BEGAN},
