diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
index 67ce688c4bbc1bf0b1f88e90e5279bc2b73f07e2..9321fa77d25132d5a4eb56f9753fef2c903e1ddf 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
@@ -708,7 +708,7 @@ public abstract class ChromeActivity extends AsyncInitializationActivity
             var modalDialogManager = getModalDialogManager();
             assert modalDialogManager != null;
             SnackbarManager snackbarManager =
-                    new SnackbarManager(
+                    new org.chromium.chrome.browser.ui.messages.snackbar.BraveSnackbarManager(
                             this,
                             mBottomContainer,
                             windowAndroid,
@@ -3384,6 +3384,7 @@ public abstract class ChromeActivity extends AsyncInitializationActivity
      */
     public void onExitVr() {}
 
+    public void onTabStateInitializedHandler() {}
     private void clearToolbarResourceCache() {
         View v = findViewById(R.id.control_container);
         try {
