diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 95b88e50fca92ca847662d1861db7de835a1cd37..51e4041b42eab4cd9a48eb88160a2e358e6b6580 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/utility/sources.gni")
 import("//build/config/sysroot.gni")
 import("//chrome/common/features.gni")
 import("//chrome/common/request_header_integrity/buildflags.gni")
@@ -24,6 +25,7 @@ static_library("utility") {
     "services.cc",
     "services.h",
   ]
+  sources += brave_utility_sources
 
   defines = []
   configs += [
@@ -32,6 +34,7 @@ static_library("utility") {
   ]
 
   public_deps = []
+  public_deps += brave_utility_public_deps
   deps = [
     "//base",
     "//build:chromeos_buildflags",
@@ -80,6 +83,7 @@ static_library("utility") {
     "//third_party/zlib/google:zip",
     "//ui/base:buildflags",
   ]
+  deps += brave_utility_deps
 
   if (is_win) {
     deps += [
