diff --git a/chrome/browser/resources/settings/people_page/sync_controls.js b/chrome/browser/resources/settings/people_page/sync_controls.js
index 37932cbc8c8d3962706a8ce436117d736440492f..a232e52bb4c74792f0dcc7ef646c6a536992af1e 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.js
+++ b/chrome/browser/resources/settings/people_page/sync_controls.js
@@ -108,7 +108,7 @@ Polymer({
     const router = Router.getInstance();
     if (router.getCurrentRoute() ===
         /** @type {{ SYNC_ADVANCED: !Route }} */
-        (router.getRoutes()).SYNC_ADVANCED) {
+        (router.getRoutes()).BRAVE_SYNC_SETUP) {
       this.browserProxy_.didNavigateToSyncPage();
     }
   },
@@ -241,6 +241,7 @@ Polymer({
    * @private
    */
   syncControlsHidden_() {
+    return false;
     if (!this.syncStatus) {
       // Show sync controls by default.
       return false;
