diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index a38a5476d7158b7b2092963c2f721ea3258bbfe7..bc778fb902fbe6d4cc9945a418918b4db06e8e77 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4604,6 +4604,7 @@ const FeatureEntry::FeatureVariation kEnableCanvasNoiseVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
