diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
index c3d782dce51966627fc040fc063f2b734db26b59..b8c3b7a7b7be575c55cac0eb0bb2b81d85b50211 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
@@ -710,7 +710,7 @@ public abstract class ChromeActivity extends AsyncInitializationActivity
             var modalDialogManager = getModalDialogManager();
             assert modalDialogManager != null;
             SnackbarManager snackbarManager =
-                    new SnackbarManager(
+                    new org.chromium.chrome.browser.ui.messages.snackbar.BraveSnackbarManager(
                             this,
                             mBottomContainer,
                             windowAndroid,
@@ -3396,6 +3396,7 @@ public abstract class ChromeActivity extends AsyncInitializationActivity
      */
     public void onExitVr() {}
 
+    public void onTabStateInitializedHandler() {}
     private void clearToolbarResourceCache() {
         View v = findViewById(R.id.control_container);
         try {
