diff --git a/chrome/renderer/chrome_content_renderer_client.cc b/chrome/renderer/chrome_content_renderer_client.cc
index 530707c9c77ab8646c4b4fefaf13f24da24ed067..1bc90c98bafd0380dc19b4209204a2b0a0acaa9b 100644
--- a/chrome/renderer/chrome_content_renderer_client.cc
+++ b/chrome/renderer/chrome_content_renderer_client.cc
@@ -600,7 +600,7 @@ void ChromeContentRendererClient::RenderFrameCreated(
       ChromeExtensionsRendererClient::GetInstance()->extension_dispatcher());
 #endif
   content_settings::ContentSettingsAgentImpl* content_settings =
-      new content_settings::ContentSettingsAgentImpl(
+      new content_settings::BraveContentSettingsAgentImpl(
           render_frame, should_allow_for_content_settings,
           std::move(content_settings_delegate));
   if (chrome_observer_.get()) {
@@ -789,6 +789,7 @@ void ChromeContentRendererClient::RenderFrameCreated(
                                 render_frame));
   }
 #endif
+  BRAVE_RENDER_FRAME_CREATED
 }
 
 void ChromeContentRendererClient::WebViewCreated(
