diff --git a/chrome/browser/ui/tabs/tab_strip_model.cc b/chrome/browser/ui/tabs/tab_strip_model.cc
index a08a0071d3eb45d9adf887f960da660183231428..2fd99e5df898aa0dc91042d82f2f085874184112 100644
--- a/chrome/browser/ui/tabs/tab_strip_model.cc
+++ b/chrome/browser/ui/tabs/tab_strip_model.cc
@@ -1785,6 +1785,7 @@ void TabStripModel::AddTab(std::unique_ptr<tabs::TabModel> tab,
     // in the address bar and presses Alt+Enter. This allows for opening a new
     // Tab to quickly look up something. When this Tab is closed, the old one
     // is re-activated, not the next-adjacent.
+    BRAVE_TAB_STRIP_MODEL_ADD_TAB
     inherit_opener = true;
   }
   tabs::TabModel* tab_ptr = tab.get();
