diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index c128db868840e68713bf0c3e93bb633a0445c13b..249d8ce8eedc372fc0bf5851b27b78504c08e13f 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -127,7 +127,7 @@ class BrowserProcessImpl : public BrowserProcess,
   const ui::UnownedUserDataHost& GetUnownedUserDataHost() const override;
 
   // Called to complete initialization.
-  void Init();
+  virtual void Init();
 
 #if !BUILDFLAG(IS_ANDROID)
   // Sets a closure to be run to break out of a run loop on browser shutdown
@@ -152,15 +152,15 @@ class BrowserProcessImpl : public BrowserProcess,
   // Called after the threads have been created but before the message loops
   // starts running. Allows the browser process to do any initialization that
   // requires all threads running.
-  void PreMainMessageLoopRun();
+  virtual void PreMainMessageLoopRun();
 
   // Most cleanup is done by these functions, driven from
   // ChromeBrowserMain based on notifications from the content
   // framework, rather than in the destructor, so that we can
   // interleave cleanup with threads being stopped.
 #if !BUILDFLAG(IS_ANDROID)
-  void StartTearDown();
-  void PostDestroyThreads();
+  virtual void StartTearDown();
+  virtual void PostDestroyThreads();
 #endif
 
   // BrowserProcess implementation.
@@ -257,6 +257,7 @@ class BrowserProcessImpl : public BrowserProcess,
   GlobalFeatures* GetFeatures() override;
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
