diff --git a/chrome/browser/ui/page_action/page_action_model.cc b/chrome/browser/ui/page_action/page_action_model.cc
index d2ae65374d9984e41878b236a75169d79a15c7ec..899cd0295f7d7984f780183e9a8c743e1ac4b0b1 100644
--- a/chrome/browser/ui/page_action/page_action_model.cc
+++ b/chrome/browser/ui/page_action/page_action_model.cc
@@ -16,6 +16,7 @@ namespace {
 using ::actions::ActionItem;
 }  // namespace
 
+namespace chromium_impl {
 PageActionModel::PageActionModel(actions::ActionId action_id, bool is_ephemeral)
     : action_id_(action_id), is_ephemeral_(is_ephemeral) {}
 
@@ -386,4 +387,5 @@ PageActionModel::GetAnchoredMessageExpandableContent() const {
   return expandable_content_;
 }
 
+}  // namespace chromium_impl
 }  // namespace page_actions
