diff --git a/chrome/browser/ui/views/location_bar/location_icon_view.cc b/chrome/browser/ui/views/location_bar/location_icon_view.cc
index f428a01b632919c3afe0531e15be8f8654a8272c..85701238832af5209790d2b39c7f5d46002323b7 100644
--- a/chrome/browser/ui/views/location_bar/location_icon_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_icon_view.cc
@@ -308,6 +308,7 @@ void LocationIconView::UpdateBackground() {
 
   SetBackgroundColor(GetColorProvider()->GetColor(id));
 
+  BRAVE_LOCATION_ICON_VIEW_UPDATE_BACKGROUND
   if (is_text_dangerous) {
     ConfigureInkDropForRefresh2023(this, kColorOmniboxSecurityChipInkDropHover,
                                    kColorOmniboxSecurityChipInkDropRipple);
