diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 7e5783950e3a4e3b110b82e1737b42bf8fab6d3b..39c2795730aa6e0503e661cf5b55aba8cb057d7a 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -659,6 +659,7 @@ class Browser : public TabStripModelObserver,
   // dragging.
   void SetIsInTabDragging(bool is_in_tab_dragging);
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
