diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index d6b316a4e249033efc5c05416007b979895072af..456182bcedd9210b053357c2c4f379707c2ee95b 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2189,6 +2189,7 @@ static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (build_with_tflite_lib) {
@@ -4001,6 +4002,7 @@ static_library("browser") {
         "task_manager/sampling/shared_sampler_posix.cc",
       ]
     }
+    sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps
   }
 
   if (is_chromeos) {
