diff --git a/chrome/browser/ui/browser_window/BUILD.gn b/chrome/browser/ui/browser_window/BUILD.gn
index fe9d1b818a2822a039a1620c9bedd6982418ffcd..3a81dcf421dea82270ff0616a6fa72771f421237 100644
--- a/chrome/browser/ui/browser_window/BUILD.gn
+++ b/chrome/browser/ui/browser_window/BUILD.gn
@@ -27,6 +27,7 @@ source_set("browser_window") {
       "//ui/base:types",
     ]
   }
+  deps += [ "//brave/browser/ui/browser_window" ]
 }
 
 source_set("impl") {
@@ -78,4 +79,5 @@ source_set("impl") {
       deps += [ "//chrome/browser/glic" ]
     }
   }
+  deps += [ "//brave/browser/ui/browser_window:impl" ]
 }
