diff --git a/chrome/browser/resources/signin/profile_picker/profile_card.html.ts b/chrome/browser/resources/signin/profile_picker/profile_card.html.ts
index 5774fc71a51f05341e0089a7308fec915d68537f..ed319fdf69e354921bd5c2ed9462fbbf0bdd6148 100644
--- a/chrome/browser/resources/signin/profile_picker/profile_card.html.ts
+++ b/chrome/browser/resources/signin/profile_picker/profile_card.html.ts
@@ -10,7 +10,7 @@ import {isGlicVersion} from './profile_picker_flags.js';
 export function getHtml(this: ProfileCardElement) {
   return html`<!--_html_template_start_-->
 <div id="profileCardContainer">
-  <cr-button id="profileCardButton" @click="${this.onProfileClick_}"
+  <cr-button id="profileCardButton" class="plain" @click="${this.onProfileClick_}"
       aria-label="${this.profileState.profileCardButtonLabel}">
     <div id="avatarContainer">
       <img class="profile-avatar" alt="" .src="${this.profileState.avatarIcon}">
