diff --git a/chrome/common/pref_names.cc b/chrome/common/pref_names.cc
index acb5c9d57c34810d93b9d0d28c8bce55c664462b..03097b0618e6d0f886bb8b7ad9f563b66dae72d1 100644
--- a/chrome/common/pref_names.cc
+++ b/chrome/common/pref_names.cc
@@ -1067,6 +1067,7 @@ const char kImportDialogBookmarks[] = "import_dialog_bookmarks";
 const char kImportDialogHistory[] = "import_dialog_history";
 const char kImportDialogSavedPasswords[] = "import_dialog_saved_passwords";
 const char kImportDialogSearchEngine[] = "import_dialog_search_engine";
+const char kImportDialogCookies[] = "import_dialog_cookies";
 
 // Profile avatar and name
 const char kProfileAvatarIndex[] = "profile.avatar_index";
