diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index bcd7439e3258b323e7d7c896b9beeae7f5a58246..45b575deda0a255472ff2106f54f62dd3235afc9 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -72,6 +72,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
