diff --git a/chrome/tools/build/mac/infoplist_strings_util.mm b/chrome/tools/build/mac/infoplist_strings_util.mm
index 5a25e3663fb09d18ee8495c2dc9032dcbdc40120..f14cc47e47a76f827fecac0d6f48e9253ad77de9 100644
--- a/chrome/tools/build/mac/infoplist_strings_util.mm
+++ b/chrome/tools/build/mac/infoplist_strings_util.mm
@@ -241,8 +241,7 @@ int main(int argc, char* const argv[]) {
 
       // Fetch the strings.
       NSString* name =
-            LoadStringFromDataPack(branded_data_pack.get(), cur_lang,
-                                   name_id, name_id_str);
+          [NSString stringWithUTF8String:BRAVE_PRODUCT_STRING];
       NSString* copyright_format =
           LoadStringFromDataPack(branded_data_pack.get(), cur_lang,
                                  IDS_ABOUT_VERSION_COPYRIGHT,
