diff --git a/chrome/browser/ui/browser_window/public/browser_window_features.h b/chrome/browser/ui/browser_window/public/browser_window_features.h
index fc79024a2077b823a14804a64028eb3b5b521126..e503646d36d3705dfaff0a6b9b2ee253fd71a4c7 100644
--- a/chrome/browser/ui/browser_window/public/browser_window_features.h
+++ b/chrome/browser/ui/browser_window/public/browser_window_features.h
@@ -201,7 +201,7 @@ class AiModePageActionController;
 class BrowserWindowFeatures {
  public:
   BrowserWindowFeatures();
-  ~BrowserWindowFeatures();
+  virtual ~BrowserWindowFeatures();
 
   BrowserWindowFeatures(const BrowserWindowFeatures&) = delete;
   BrowserWindowFeatures& operator=(const BrowserWindowFeatures&) = delete;
