diff --git a/chrome/browser/ui/browser_command_controller.h b/chrome/browser/ui/browser_command_controller.h
index e05b4895fe6ad581f0c7c574edfbd7de1c53bcdb..61446e4a22726e0c461d2d0eba17d394fedef493 100644
--- a/chrome/browser/ui/browser_command_controller.h
+++ b/chrome/browser/ui/browser_command_controller.h
@@ -123,6 +123,7 @@ class BrowserCommandController : public CommandUpdater,
       Profile* profile);
 
  private:
+  friend class BraveBrowserCommandController;
 #if BUILDFLAG(IS_CHROMEOS)
   friend class BrowserCommandControllerBrowserTestLockedFullscreen;
 #endif
@@ -199,7 +200,7 @@ class BrowserCommandController : public CommandUpdater,
 
   // Update commands whose state depends on the type of fullscreen mode the
   // window is in.
-  void UpdateCommandsForFullscreenMode();
+  virtual void UpdateCommandsForFullscreenMode();
 
   // Update commands whose state depends on whether they're available to hosted
   // app windows.
