diff --git a/chrome/browser/prefs/browser_prefs.cc b/chrome/browser/prefs/browser_prefs.cc
index 06086e902052b7d7700464c67e54c06da4bdac07..c4cb322a24c198cc6fe4bbd0f8910cd71d8c5e14 100644
--- a/chrome/browser/prefs/browser_prefs.cc
+++ b/chrome/browser/prefs/browser_prefs.cc
@@ -1816,6 +1816,7 @@ void RegisterLocalState(PrefRegistrySimple* registry) {
   registry->RegisterIntegerPref(prefs::kToastAlertLevel, 0);
 
   // This is intentionally last.
+  brave::RegisterLocalStatePrefs(registry);
   RegisterLocalStatePrefsForMigration(registry);
 }
 
@@ -2249,6 +2250,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry,
   registry->RegisterIntegerPref(prefs::kLensOverlayStartCount, 0);
 
   registry->RegisterDictionaryPref(prefs::kReportingEndpoints);
+  brave::RegisterProfilePrefs(registry);
 }
 
 void RegisterUserProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
