diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 6cf62a9adcbed8e2829b2cba7410f6ada6a2103a..962df2fe25f56fd4556f48db9fdf4e308ac4a9db 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -617,6 +617,7 @@ class Browser : public TabStripModelObserver,
   // dragging.
   void SetIsInTabDragging(bool is_in_tab_dragging);
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class FullscreenControllerInteractiveTest;
