diff --git a/chrome/browser/startup/startup_features.cc b/chrome/browser/startup/startup_features.cc
index a2bfc75e840e2b98b8363d7d0353dfbc9f98adbd..aa64319095f5134ea71804df7a477d4f3b948b0f 100644
--- a/chrome/browser/startup/startup_features.cc
+++ b/chrome/browser/startup/startup_features.cc
@@ -9,7 +9,8 @@
 
 namespace features {
 
-BASE_FEATURE(kLaunchOnStartup, base::FEATURE_ENABLED_BY_DEFAULT);
+// kLaunchOnStartup feature state is enforced via plaster rewrite.
+BASE_FEATURE(kLaunchOnStartup, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE_ENUM_PARAM(LaunchOnStartupMode,
                         kLaunchOnStartupModeParam,
