diff --git a/chrome/browser/ui/views/translate/translate_bubble_view.h b/chrome/browser/ui/views/translate/translate_bubble_view.h
index 078a51664712fd5a59da8e56103c0308c1924cf1..d56ed6e9f81853b905328de13859833744b8a0ea 100644
--- a/chrome/browser/ui/views/translate/translate_bubble_view.h
+++ b/chrome/browser/ui/views/translate/translate_bubble_view.h
@@ -117,6 +117,7 @@ class TranslateBubbleView : public LocationBarBubbleDelegateView,
 
   // Returns the current view state.
   TranslateBubbleModel::ViewState GetViewState() const;
+  BRAVE_TRANSLATE_BUBBLE_VIEW_H_
 
  protected:
   // LocationBarBubbleDelegateView:
