diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
index 55aa36caddc7e777338114f42ed6959afd9baaf5..405b78e7a4319a192e22aecd27b3db1c15c0b057 100644
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -2,6 +2,10 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/sources.gni")
+import("//brave/browser/updater/buildflags.gni")
+import("//brave/components/brave_origin/buildflags/buildflags.gni")
+import("//brave/updater/config.gni")
 import("//build/config/chrome_build.gni")
 import("//build/config/chromeos/args.gni")
 import("//build/config/compiler/compiler.gni")
@@ -407,6 +411,13 @@ if (!is_android && !is_mac) {
     if (is_chrome_for_testing) {
       deps += [ "//chrome/browser/chrome_for_testing:browser" ]
     }
+    if (is_win) {
+      sources -= [ "app/chrome_exe.rc" ]
+      sources += [ "//brave/app/brave_exe.rc" ]
+      if (is_brave_origin_branded) {
+        defines += [ "IS_BRAVE_ORIGIN_BRANDED" ]
+      }
+    }
   }
 
   # TODO(crbug.com/40204298): Make this work on other platforms.
@@ -541,6 +552,12 @@ if (is_win) {
     }
   }
 
+  copy("brave_app_plist") {
+    deps = [ "//brave:brave_app_plist" ]
+    sources = [ "$root_gen_dir/brave/brave_app_plist_tweaked.plist" ]
+    outputs = [ "$root_gen_dir/chrome/{{source_file_part}}" ]
+  }
+
   mac_app_bundle("chrome_app") {
     output_name = chrome_product_full_name
 
@@ -567,6 +584,7 @@ if (is_win) {
       "//chrome/common:buildflags",
       "//chrome/common:version_header",
     ]
+    deps += [ "//brave:brave_app" ]
 
     bundle_deps = [
       ":chrome_app_strings_bundle_data",
@@ -602,6 +620,7 @@ if (is_win) {
       # for runtime_deps at create_bundle nodes (https://crbug.com/40101435).
       data_deps = [ ":chrome_framework" ]
     }
+    info_plist_target = ":brave_app_plist"
   }
 
   if (verify_dynamic_libraries) {
@@ -720,6 +739,9 @@ if (is_win) {
       # resources directory.
       public_deps += [ ":chrome_framework_widevine_signature" ]
     }
+    if (enable_widevine_cdm_host_verification) {
+      public_deps -= [ ":chrome_framework_widevine_signature" ]
+    }
   }
 
   if (enable_updater) {
@@ -763,8 +785,15 @@ if (is_win) {
     ]
   }
 
+  copy("brave_helper_plist") {
+    deps = [ "//brave:brave_helper_plist" ]
+    sources = [ "$root_gen_dir/brave/brave_helper_plist_tweaked.plist" ]
+    outputs = [ "$root_gen_dir/chrome/{{source_file_part}}" ]
+  }
+
   compile_entitlements("entitlements") {
     entitlements_templates = [ "app/app-entitlements.plist" ]
+    entitlements_templates += brave_entitlements_templates
     if (is_chrome_branded && include_branded_entitlements) {
       # These entitlements are bound to the official Google Chrome signing
       # certificate and will not necessarily work in any other build.
@@ -789,6 +818,7 @@ if (is_win) {
         info_plist_target = invoker.info_plist_target
       } else {
         info_plist_target = ":chrome_helper_plist"
+        info_plist_target = ":brave_helper_plist"
       }
 
       extra_substitutions = [
@@ -1072,6 +1102,11 @@ if (is_win) {
     if (enable_updater) {
       if (is_chrome_branded) {
         sources += [ "//third_party/updater/chrome_mac_universal_prod/cipd/${updater_product_full_name}.app" ]
+      } else if (!enable_omaha4) {
+      } else if (use_prebuilt_omaha4) {
+        sources += [
+          "//brave/third_party/updater/mac/${updater_product_full_name}.app",
+        ]
       } else {
         sources += [ "$root_out_dir/${updater_product_full_name}.app" ]
 
@@ -1381,6 +1416,7 @@ if (is_win) {
       "Libraries",
       "Resources",
     ]
+    framework_contents += brave_chrome_framework_contents
 
     configs += [ "//build/config/compiler:wexit_time_destructors" ]
     configs -= [ "//build/config/compiler:thinlto_optimize_default" ]
@@ -1393,6 +1429,7 @@ if (is_win) {
     ]
 
     public_deps = [ ":chrome_dll" ]
+    public_deps += brave_chrome_framework_public_deps
 
     bundle_deps = [
       ":aperitif_library",
@@ -1405,6 +1442,7 @@ if (is_win) {
       "//chrome/browser/web_applications/isolated_web_apps/key_distribution/preload:component_bundle",
       "//components/privacy_sandbox/privacy_sandbox_attestations/preload:component_bundle",
     ]
+    bundle_deps += brave_chrome_framework_bundle_deps
 
     if (is_chrome_branded) {
       bundle_deps += [ ":chromecompaneros_bundle_data" ]
@@ -1642,6 +1680,7 @@ if (is_win) {
 
 group("dependencies") {
   public_deps = [
+    "//brave:browser_dependencies",
     "//build:branding_buildflags",
     "//build:chromeos_buildflags",
     "//chrome/browser",
@@ -1688,7 +1727,7 @@ group("dependencies") {
 
 if (is_win) {
   process_version_rc_template("chrome_exe_version") {
-    sources = [ "app/chrome_exe.ver" ]
+    sources = [ "//brave/app/brave_exe.ver" ]
     output = "$target_gen_dir/chrome_exe_version.rc"
   }
 
@@ -1734,6 +1773,7 @@ copy("visual_elements_resources") {
 
 group("resources") {
   public_deps = [
+    "//brave:resources",
     "//chrome/browser:resources",
     "//chrome/common:resources",
     "//chrome/renderer:resources",
@@ -1762,6 +1802,10 @@ if (!is_android) {
     if (enable_resource_allowlist_generation) {
       repack_allowlist = _chrome_resource_allowlist
       deps = [ ":resource_allowlist" ]
+      deps += [ "//brave:merge_allowlists" ]
+    }
+    if (enable_resource_allowlist_generation) {
+      repack_allowlist = brave_allowlist_file
     }
   }
 }
