diff --git a/chrome/browser/ui/cocoa/tabs/tab_strip_controller.mm b/chrome/browser/ui/cocoa/tabs/tab_strip_controller.mm
index 8f0fc0a5711f1f011b90c1cbcb27d78977d70263..b9ec879393c5029b34650455b8d594ca929f6cbe 100644
--- a/chrome/browser/ui/cocoa/tabs/tab_strip_controller.mm
+++ b/chrome/browser/ui/cocoa/tabs/tab_strip_controller.mm
@@ -92,7 +92,7 @@
 const CGFloat kLastPinnedTabSpacing = 2.0;
 
 // The amount by which the new tab button is offset (from the tabs).
-const CGFloat kNewTabButtonOffset = 10.0;
+const CGFloat kNewTabButtonOffset = -7.0;
 
 const NSTimeInterval kAnimationDuration = 0.125;
 
@@ -592,7 +592,7 @@ + (CGFloat)tabOverlap {
   // tab. Else, every time the throbber is painted, the throbber's invalidation
   // will also invalidate parts of the tab to the left, and two tabs's
   // backgrounds need to be painted on each throbber frame instead of one.
-  const CGFloat kTabOverlap = 18.0;
+  const CGFloat kTabOverlap = 2.0;
   return kTabOverlap;
 }
 
