diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index 162b5ec9f9d6451a6723d10e96f71ac94622729f..8f0463b5d35670b01c34190fe5f1a8e7d2c2174f 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -34,7 +34,9 @@ function addPrivacyChildRoutes(r) {
 
   // TODO(tommycli): Find a way to refactor these repetitive category
   // routes.
+      // <if expr="_google_chrome">
   r.SITE_SETTINGS_ADS = r.SITE_SETTINGS.createChild('ads');
+      // </if>
   r.SITE_SETTINGS_AR = r.SITE_SETTINGS.createChild('ar');
   r.SITE_SETTINGS_AUTOMATIC_DOWNLOADS =
       r.SITE_SETTINGS.createChild('automaticDownloads');
@@ -181,7 +183,9 @@ function createBrowserSettingsRoutes() {
     }
 
     r.PRINTING = r.ADVANCED.createSection('/printing', 'printing');
+      // <if expr="_google_chrome">
     r.CLOUD_PRINTERS = r.PRINTING.createChild('/cloudPrinters');
+      // </if>
 
     r.ACCESSIBILITY = r.ADVANCED.createSection('/accessibility', 'a11y');
 
