diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index e67bc0288c238a0cc0a117d3902d096400ca3645..3de091586dc100b67097da4af03af1b85ea60440 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -62,6 +62,7 @@ source_set("chrome_dll_resources") {
       "//build:branding_buildflags",
       "//printing/buildflags",
     ]
+    import("//brave/components/brave_origin/buildflags/buildflags.gni") if (is_brave_origin_branded) { defines = [ "IS_BRAVE_ORIGIN_BRANDED" ] }
   }
 }
 
@@ -107,6 +108,7 @@ grit_strings("generated_resources") {
   if (is_android) {
     create_android_resources = true
   }
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
