diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.h b/chrome/browser/ui/views/location_bar/location_bar_view.h
index 6fd3631dc90b578ca1adaef3d69b97611734ebf0..a0ab959e60515842c5b7ee77edf7c822f737dfb7 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.h
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.h
@@ -109,6 +109,7 @@ class LocationBarView : public LocationBar,
   int GetBorderRadius() const;
 
   // Initializes the LocationBarView.
+  virtual
   void Init();
 
   // True if this instance has been initialized by calling Init, which can only
@@ -161,6 +162,7 @@ class LocationBarView : public LocationBar,
 
   // Updates the controller, and, if |contents| is non-null, restores saved
   // state that the tab holds.
+  virtual
   void Update(content::WebContents* contents);
 
   // Clears the location bar's state for |contents|.
@@ -243,6 +245,7 @@ class LocationBarView : public LocationBar,
   std::vector<ContentSettingImageView*>& GetContentSettingViewsForTest() {
     return content_setting_views_;
   }
+  BRAVE_LOCATION_BAR_VIEW_H_
 
   void RecordPageInfoMetrics();
 
