diff --git a/chrome/browser/extensions/chrome_component_extension_resource_manager.cc b/chrome/browser/extensions/chrome_component_extension_resource_manager.cc
index 9d0819104de106a32a708bcbcffa7578219d9e30..02ed3a9b8a3d8b407a5bf4426ca34925203c87d7 100644
--- a/chrome/browser/extensions/chrome_component_extension_resource_manager.cc
+++ b/chrome/browser/extensions/chrome_component_extension_resource_manager.cc
@@ -85,6 +85,7 @@ class ChromeComponentExtensionResourceManager::Data {
 };
 
 ChromeComponentExtensionResourceManager::Data::Data() {
+  [&]() -> void {
   static const webui::ResourcePath kExtraComponentExtensionResources[] = {
 #if BUILDFLAG(IS_CHROMEOS)
     {"web_store/webstore_icon_128.png", IDR_WEBSTORE_APP_ICON_128},
@@ -166,6 +167,9 @@ ChromeComponentExtensionResourceManager::Data::Data() {
         std::move(pdf_viewer_replacements);
   }
 #endif
+  }();
+  AddComponentResourceEntries(kBraveExtension);
+  AddComponentResourceEntries(kBraveExtensionGenerated);
 }
 
 void ChromeComponentExtensionResourceManager::Data::AddComponentResourceEntries(
