diff --git a/chrome/browser/resources/settings/system_page/system_page.ts b/chrome/browser/resources/settings/system_page/system_page.ts
index c50e369220ac71b72ed6059756a2cfd7ce0340c5..fec23495a7b3acac22d331e4f86d31b2613575d6 100644
--- a/chrome/browser/resources/settings/system_page/system_page.ts
+++ b/chrome/browser/resources/settings/system_page/system_page.ts
@@ -282,5 +282,3 @@ declare global {
     'settings-system-page': SettingsSystemPageElement;
   }
 }
-
-customElements.define(SettingsSystemPageElement.is, SettingsSystemPageElement);
