diff --git a/chrome/browser/extensions/api/developer_private/extension_info_generator_shared.cc b/chrome/browser/extensions/api/developer_private/extension_info_generator_shared.cc
index e6c15e15d3157b3f7484b06c942264d7e58b98a4..d7ac7131ff49a8080d8e345d119d41cb35911b69 100644
--- a/chrome/browser/extensions/api/developer_private/extension_info_generator_shared.cc
+++ b/chrome/browser/extensions/api/developer_private/extension_info_generator_shared.cc
@@ -652,6 +652,7 @@ void ExtensionInfoGeneratorShared::FillExtensionInfo(
 #else
   info.incognito_access_pending_change = false;
 #endif
+  BRAVE_CREATE_EXTENSION_INFO_HELPER
 
   // User Scripts toggle.
   info.user_scripts_access.is_enabled =
