diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index fe0b00acf2dc56f4cb17cf19664347e75bdd13d3..c96e21bc4535c159f55c12476b5e0216c995a8aa 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -93,6 +93,7 @@ source_set("channel_info") {
   } else if (is_posix) {
     sources += [ "channel_info_posix.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -280,6 +281,7 @@ static_library("common_lib") {
     public_deps += [ "//components/cdm/common" ]
   }
 
+  import("//brave/common/sources.gni") sources += brave_chrome_common_sources deps += brave_chrome_common_deps public_deps += brave_chrome_common_public_deps
   if (enable_extensions_core) {
     public_deps += [
       "//chrome/common/extensions",
@@ -472,6 +474,7 @@ static_library("non_code_constants") {
   if (is_win) {
     sources += [ "chrome_icon_resources_win.h" ]
   }
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
@@ -577,6 +580,7 @@ static_library("constants") {
     "//third_party/widevine/cdm:buildflags",
   ]
 
+  import("//brave/common/sources.gni") deps += brave_chrome_constants_deps
   if (is_android) {
     sources += [ "chrome_paths_android.cc" ]
 
