diff --git a/chrome/browser/extensions/extension_management.h b/chrome/browser/extensions/extension_management.h
index 66a2b462ff40bfcff9258081c9a35e4f2363cf24..c0182f323eba0406555b9f1c874076c2c44f25db 100644
--- a/chrome/browser/extensions/extension_management.h
+++ b/chrome/browser/extensions/extension_management.h
@@ -199,6 +199,7 @@ class ExtensionManagement : public KeyedService,
       const ExtensionId& extension_id);
 
  private:
+  friend class BraveExtensionManagement;
   using SettingsIdMap =
       base::flat_map<ExtensionId,
                      std::unique_ptr<internal::IndividualSettings>>;
