diff --git a/chrome/browser/metrics/chrome_feature_list_creator.cc b/chrome/browser/metrics/chrome_feature_list_creator.cc
index 2bb85926405715dffea09f2e29eedcff44e0925f..2f2203c7ff9fb448b71422e55f56d48f8a9ea4da 100644
--- a/chrome/browser/metrics/chrome_feature_list_creator.cc
+++ b/chrome/browser/metrics/chrome_feature_list_creator.cc
@@ -298,5 +298,6 @@ void ChromeFeatureListCreator::SetupInitialPrefs() {
     local_state_->SetTime(variations::prefs::kVariationsSeedDate,
                           base::Time::Now());
   }
+  BRAVE_CHROME_FEATURE_LIST_CREATOR_SETUP_INITIAL_PREFS
 #endif  // !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_CHROMEOS)
 }
