diff --git a/chrome/browser/extensions/chrome_content_verifier_delegate.cc b/chrome/browser/extensions/chrome_content_verifier_delegate.cc
index 0c25c94ffd763200ab6f58898bc30b1fd8a3d08e..90f75878810a54bc76cdb89c852648897c0fe440 100644
--- a/chrome/browser/extensions/chrome_content_verifier_delegate.cc
+++ b/chrome/browser/extensions/chrome_content_verifier_delegate.cc
@@ -103,7 +103,7 @@ ChromeContentVerifierDelegate::GetDefaultMode() {
 #endif  // BUILDFLAG(PLATFORM_CFM)
 
   VerifyInfo::Mode experiment_value;
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if defined(OFFICIAL_BUILD)
   experiment_value = VerifyInfo::Mode::ENFORCE_STRICT;
 #else
   experiment_value = VerifyInfo::Mode::NONE;
