diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 805bae464aea9cb6adad1ef2d925a749dceedb7d..30d392c3348e66c571a7d8d25b70bbdcfa92763d 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -65,6 +65,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
