diff --git a/chrome/android/java_sources.gni b/chrome/android/java_sources.gni
index d71e9f6da63f3c8384ac85fbe6e2baa81baac49b..a2b6910f95633fc86a9ed0df292751b360306763 100644
--- a/chrome/android/java_sources.gni
+++ b/chrome/android/java_sources.gni
@@ -20,6 +20,7 @@ import("//components/offline_pages/buildflags/features.gni")
 import("//device/vr/buildflags/buildflags.gni")
 
 chrome_java_sources += public_autofill_assistant_java_sources
+chrome_java_sources += brave_java_sources
 
 if (enable_vr) {
   chrome_java_sources += [ "java/src/org/chromium/chrome/browser/component_updater/VrAssetsComponentInstaller.java" ]
