diff --git a/chrome/browser/prefs/browser_prefs.cc b/chrome/browser/prefs/browser_prefs.cc
index 3523036ac2556432e2783708d3c92a4a8f8d046e..b5ab26e5f59bdb23f1fa87e66c6bdb8c113a2b3b 100644
--- a/chrome/browser/prefs/browser_prefs.cc
+++ b/chrome/browser/prefs/browser_prefs.cc
@@ -583,6 +583,7 @@ void RegisterLocalState(PrefRegistrySimple* registry) {
 #if defined(TOOLKIT_VIEWS)
   RegisterBrowserViewLocalPrefs(registry);
 #endif
+  brave::RegisterLocalStatePrefs(registry);
 }
 
 // Register prefs applicable to all profiles.
