diff --git a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java
index a06bf1af0ebce6ae59bfcc8ed5a2234f6927424b..b0b38915c18fc687f79c340145cc46bf570c5392 100644
--- a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java
+++ b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java
@@ -587,7 +587,7 @@ public class TabSwitcherPaneCoordinator implements BackPressHandler {
                     recyclerViewTimer.getElapsedMillis());
 
             mTabListEditorManager =
-                    new TabListEditorManager(
+                    new BraveTabListEditorManager(
                             activity,
                             mModalDialogManager,
                             coordinatorView,
