diff --git a/chrome/android/junit/BUILD.gn b/chrome/android/junit/BUILD.gn
index 1295155c289b9dec7e9215e45f48eec729b3200d..df458214913ca2ce61f79e57e3112637e54758a0 100644
--- a/chrome/android/junit/BUILD.gn
+++ b/chrome/android/junit/BUILD.gn
@@ -421,6 +421,7 @@ if (is_android) {
       "//url:gurl_junit_test_support",
       "//url/mojom:url_mojom_gurl_java",
     ]
+    import("//brave/android/junit/sources.gni") public_deps -= brave_android_junit_chrome_junit_tests_helper_excluded_public_deps
   }
 
   # Partition tests into one target per-package to keep incremental build times
