diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 07714995a3a9f73fa4611ae1ae104be3da7bbaf9..0df1231fe63c48cb703e0718ba22087b9c6373d5 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2164,6 +2164,7 @@ static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
