diff --git a/chrome/browser/ui/webui/history/history_ui.cc b/chrome/browser/ui/webui/history/history_ui.cc
index 820bf5ab5864322e2fa053ddc026cd1d91a63d6c..9fedca5602845415be8b0fb5a506791c9bf7249b 100644
--- a/chrome/browser/ui/webui/history/history_ui.cc
+++ b/chrome/browser/ui/webui/history/history_ui.cc
@@ -88,6 +88,7 @@ content::WebUIDataSource* CreateAndAddHistoryUIHTMLSource(Profile* profile) {
 #endif  // BUILDFLAG!(IS_CHROMEOS)
 
   HistoryUtil::PopulateCommonSourceForHistory(source, profile);
+  BRAVE_CREATE_HISTORY_UI_HTML_SOURCE
 
   static constexpr webui::LocalizedString kStrings[] = {
       // Localized strings (alphabetical order).
