diff --git a/chrome/browser/net/chrome_network_delegate.h b/chrome/browser/net/chrome_network_delegate.h
index 63d22f1ed9d80118f32a2be91ba728e1c889d9cc..d62180bc4aabcee0e713cd49fc02002c7fae0b4c 100644
--- a/chrome/browser/net/chrome_network_delegate.h
+++ b/chrome/browser/net/chrome_network_delegate.h
@@ -105,6 +105,7 @@ class ChromeNetworkDelegate : public net::NetworkDelegateImpl {
   static void EnableAccessToAllFilesForTesting(bool enabled);
 
  private:
+  friend class BraveNetworkDelegateBase;
   // NetworkDelegate implementation.
   int OnBeforeURLRequest(net::URLRequest* request,
                          net::CompletionOnceCallback callback,
