diff --git a/chrome/browser/resources/extensions/BUILD.gn b/chrome/browser/resources/extensions/BUILD.gn
index 81705a7264ac77e78cb1f34b06c98d40aa276e78..5b6f85320d4d1cb27ff25842de276341da392753 100644
--- a/chrome/browser/resources/extensions/BUILD.gn
+++ b/chrome/browser/resources/extensions/BUILD.gn
@@ -31,6 +31,7 @@ if (optimize_webui) {
       "../../../../ui/webui/resources:preprocess",
     ]
     excludes = [ "chrome://resources/js/cr.m.js" ]
+    deps += brave_deps_chrome_browser_resources_extensions_build
   }
 }
 
@@ -48,6 +49,7 @@ generate_grd("build_grd") {
     deps = [ ":build_ts" ]
     manifest_files = [ "$target_gen_dir/tsconfig.manifest" ]
   }
+  deps += [ "//brave/browser/resources/extensions:resources" ]
 }
 
 preprocess_if_expr("preprocess") {
