diff --git a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
index 15e829f8b90e479b2d83de7a7b21140e40bd005f..22cb778e8d19b8daeec1980be686afed6b06e0e3 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
@@ -275,6 +275,7 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             Action.TRANSLATE,
             Action.CREATE_QR_CODE,
             Action.ASK_GEMINI,
+            Action.COPY_CLEAN_LINK,
         })
         @Retention(RetentionPolicy.SOURCE)
         public @interface Action {
@@ -338,7 +339,8 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             int TRANSLATE = 57;
             int CREATE_QR_CODE = 58;
             int ASK_GEMINI = 59;
-            int NUM_ENTRIES = 60;
+            int COPY_CLEAN_LINK = 60;
+            int NUM_ENTRIES = COPY_CLEAN_LINK + 1;
         }
 
         // LINT.ThenChange(/tools/metrics/histograms/enums.xml:ContextMenuOptionAndroid)
@@ -735,6 +737,7 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             if (!MailTo.isMailTo(mParams.getLinkUrl().getSpec())
                     && !UrlUtilities.isTelScheme(mParams.getLinkUrl())) {
                 linkGroup.add(createListItem(Item.COPY_LINK_ADDRESS));
+                linkGroup.add(createListItem(Item.COPY_CLEAN_LINK));
                 if (!mParams.getLinkText().trim().isEmpty() && !mParams.isImage()) {
                     linkGroup.add(createListItem(Item.COPY_LINK_TEXT));
                 }
