diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc
index c3e5a1afca5f697ce57dd21217c7641786ae6422..dd2082aeefb4afee68d304a177c1c2d7942ec834 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc
@@ -465,6 +465,8 @@ void RecentTabsSubMenuModel::BuildTabsFromOtherDevices() {
          ++k) {
       BuildOtherDevicesTabItem(session_tag, *tabs_in_session[k]);
     }  // for all tabs in one session
+
+    BRAVE_BUILD_TABS_FROM_OTHER_DEVICES
     ++num_sessions_added;
   }  // for all sessions
 
