diff --git a/chrome/browser/permissions/permission_uma_util.cc b/chrome/browser/permissions/permission_uma_util.cc
index 14efe3ccb3906516ba5c13a1c1ba78fb2ccd2d7d..cf3ac17e8b5e3c59a4c880ddc3861875dda1c3c8 100644
--- a/chrome/browser/permissions/permission_uma_util.cc
+++ b/chrome/browser/permissions/permission_uma_util.cc
@@ -64,7 +64,7 @@ namespace {
 
 const int kPriorCountCap = 10;
 
-std::string GetPermissionRequestString(PermissionRequestType type) {
+std::string GetPermissionRequestString_ChromiumImpl(PermissionRequestType type) {
   switch (type) {
     case PermissionRequestType::MULTIPLE:
       return "AudioAndVideoCapture";
@@ -401,6 +401,9 @@ void PermissionUmaUtil::RecordPermissionAction(
       UMA_HISTOGRAM_ENUMERATION("Permissions.Action.PaymentHandler", action,
                                 PermissionAction::NUM);
       break;
+    case CONTENT_SETTINGS_TYPE_AUTOPLAY:
+      BraveRecordPermissionAction(permission, secure_origin, action);
+      break;
     // The user is not prompted for these permissions, thus there is no
     // permission action recorded for them.
     default:
