diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index c9d0dcd8f2839e1451194b3e483e0fa03be9ce81..8f958c6b449b62ca2cd6f0c39417c9aec53f8019 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -7750,6 +7750,7 @@ const FeatureEntry kFeatureEntries[] = {
     // "LoginCustomFlags" in tools/metrics/histograms/enums.xml. See "Flag
     // Histograms" in tools/metrics/histograms/README.md (run the
     // AboutFlagsHistogramTest unit test to verify this process).
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 };
 
 class FlagsStateSingleton : public flags_ui::FlagsState::Delegate {
