diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 0a7a4ad4c28702ee0703ef3838926574eb68a4ab..afeaa135d3b38861452648115fc17c1bcaa18f39 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -97,6 +97,7 @@ source_set("channel_info") {
   } else if (is_fuchsia) {
     sources += [ "channel_info_fuchsia.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -276,6 +277,7 @@ static_library("common_lib") {
   if (enable_ppapi) {
     public_deps += [ "//ppapi/shared_impl" ]
   }
+  import("//brave/common/sources.gni") sources += brave_chrome_common_sources deps += brave_chrome_common_deps public_deps += brave_chrome_common_public_deps
 
   if (enable_extensions) {
     sources += [
@@ -547,6 +549,7 @@ static_library("non_code_constants") {
   if (is_win) {
     sources += [ "chrome_icon_resources_win.h" ]
   }
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
