diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 0096bead803507ffb0856548bae26431daf24d9d..d064fdfc39745dbb8aa2ac6e45d9f38778b4884c 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -868,6 +868,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
