diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
index f64a099146f77563ec1280b2bdd22f23bd0a734e..f5aa739d8a32bbbaad96cb22bffd3a2944b9bc64 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
@@ -369,7 +369,7 @@ public abstract class ChromeActivity<C extends ChromeActivityComponent>
             new ArrayList<>();
 
     protected ChromeActivity() {
-        mIntentHandler = new IntentHandler(this, createIntentHandlerDelegate());
+        mIntentHandler = new org.chromium.chrome.browser.BraveIntentHandler(this, createIntentHandlerDelegate());
     }
 
     @Override
