diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 0bae6f32f3f02de3179bf306b94b5adb125a4d1a..36fe98eb343c4f4000343213e3593bf656045e04 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -56,9 +56,9 @@
 #include "ui/gfx/geometry/rect.h"
 #include "ui/shell_dialogs/select_file_dialog.h"
 
-#if defined(OS_ANDROID)
-#error This file should only be included on desktop.
-#endif
+//#if defined(OS_ANDROID)
+//#error This file should only be included on desktop.
+//#endif
 
 class BackgroundContents;
 class BrowserContentSettingBubbleModelDelegate;
@@ -660,7 +660,7 @@ class Browser : public TabStripModelObserver,
   // This information is used to decide if fast resize will be used during
   // dragging.
   void SetIsInTabDragging(bool is_in_tab_dragging);
-
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class FullscreenControllerInteractiveTest;
