diff --git a/chrome/browser/extensions/extension_service.h b/chrome/browser/extensions/extension_service.h
index e51f6485fc17242e579809c63b104f8df42fcae6..d30d4f0232feeb76812c868c47db6fc6ceae8e9e 100644
--- a/chrome/browser/extensions/extension_service.h
+++ b/chrome/browser/extensions/extension_service.h
@@ -425,6 +425,7 @@ class ExtensionService : public ExtensionServiceInterface,
   }
 
  private:
+  friend class BraveExtensionService;
   // Loads extensions specified via a command line flag/switch.
   void LoadExtensionsFromCommandLineFlag(const char* switch_name);
 
