diff --git a/chrome/browser/profiles/profile_manager.h b/chrome/browser/profiles/profile_manager.h
index e457f7d56fe3268a8599d973398e596e56b077a0..ca5f1d309b8509ff0d612ebf481e8dbac23a711d 100644
--- a/chrome/browser/profiles/profile_manager.h
+++ b/chrome/browser/profiles/profile_manager.h
@@ -278,6 +278,7 @@ class ProfileManager : public Profile::Delegate {
   void set_do_final_services_init(bool do_final_services_init) {
     do_final_services_init_ = do_final_services_init;
   }
+  BRAVE_PROFILE_MANAGER_H
 
  private:
   friend class TestingProfileManager;
