diff --git a/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/channels/ChromeChannelDefinitions.java b/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/channels/ChromeChannelDefinitions.java
index 81c39210ba410b8261047cee432615b268226999..9df765ff0db7ef8fcd54e1c7228055a638b5c4b0 100644
--- a/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/channels/ChromeChannelDefinitions.java
+++ b/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/channels/ChromeChannelDefinitions.java
@@ -69,6 +69,7 @@ public class ChromeChannelDefinitions extends ChannelDefinitions {
      * more detailed instructions.
      */
     @StringDef({ChannelId.BROWSER, ChannelId.DOWNLOADS, ChannelId.INCOGNITO,
+            BraveChannelDefinitions.ChannelId.BRAVE_ADS, BraveChannelDefinitions.ChannelId.BRAVE_ADS_BACKGROUND,
             ChannelId.MEDIA_PLAYBACK, ChannelId.SCREEN_CAPTURE, ChannelId.CONTENT_SUGGESTIONS,
             ChannelId.WEBAPP_ACTIONS, ChannelId.SITES, ChannelId.SHARING, ChannelId.UPDATES,
             ChannelId.COMPLETED_DOWNLOADS, ChannelId.PERMISSION_REQUESTS,
@@ -101,7 +102,7 @@ public class ChromeChannelDefinitions extends ChannelDefinitions {
         String CHROME_TIPS = "chrome_tips";
     }
 
-    @StringDef({ChannelGroupId.GENERAL, ChannelGroupId.SITES})
+    @StringDef({BraveChannelDefinitions.ChannelGroupId.BRAVE_ADS, ChannelGroupId.GENERAL, ChannelGroupId.SITES})
     @Retention(RetentionPolicy.SOURCE)
     public @interface ChannelGroupId {
         String SITES = "sites";
@@ -130,6 +131,7 @@ public class ChromeChannelDefinitions extends ChannelDefinitions {
             Map<String, PredefinedChannel> map = new HashMap<>();
             Set<String> startup = new HashSet<>();
 
+            BraveChannelDefinitions.addBraveChannels(map, startup);
             map.put(ChannelId.BROWSER,
                     PredefinedChannel.create(ChannelId.BROWSER,
                             R.string.notification_category_browser,
@@ -266,6 +268,7 @@ public class ChromeChannelDefinitions extends ChannelDefinitions {
         static final Map<String, PredefinedChannelGroup> MAP;
         static {
             Map<String, PredefinedChannelGroup> map = new HashMap<>();
+            BraveChannelDefinitions.addBraveChannelGroups(map);
             map.put(ChannelGroupId.GENERAL,
                     new PredefinedChannelGroup(
                             ChannelGroupId.GENERAL, R.string.notification_category_group_general));
