diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 0a39e033b39bd1dfaaf80c8c3cff0c125d8d27e7..2fc2414f6eb860dc870f675cd1b12d1600fb5d1e 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -1006,6 +1006,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
