diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index be1c5ca3b52f11b730295e869bff79c3394ad59f..3cab817b4d3e6f1b33f92128b5718cf3d0f1d6ba 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -165,6 +165,7 @@ source_set("internal") {
   if (enable_extensions_core) {
     deps += [ "//chrome/browser/extensions" ]
   }
+  import("//brave/browser/ui/browser_window/internal/sources.gni") deps += brave_browser_window_internal_deps
 }
 
 source_set("unit_tests") {
