diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index dde2eafb88f0702a73b44b6b45decc43d04ae2bf..d246392a6f3bdb4eb5961cdf3847d001241742ef 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -745,6 +745,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome/browser/resources/media/mei_preload:component",
