diff --git a/chrome/renderer/chrome_content_renderer_client.cc b/chrome/renderer/chrome_content_renderer_client.cc
index 6c9d906d290ff91edcd40f202f37a23d9f08c5ed..8984c8329e86a9259d10db0d449d648f2ba82e4f 100644
--- a/chrome/renderer/chrome_content_renderer_client.cc
+++ b/chrome/renderer/chrome_content_renderer_client.cc
@@ -575,7 +575,7 @@ void ChromeContentRendererClient::RenderFrameCreated(
       ChromeExtensionsRendererClient::GetInstance()->extension_dispatcher());
 #endif
   content_settings::ContentSettingsAgentImpl* content_settings =
-      new content_settings::ContentSettingsAgentImpl(
+      new content_settings::BraveContentSettingsAgentImpl(
           render_frame, should_allow_for_content_settings,
           std::move(content_settings_delegate));
   if (chrome_observer_.get()) {
@@ -760,6 +760,7 @@ void ChromeContentRendererClient::RenderFrameCreated(
                                 render_frame));
   }
 #endif
+  BRAVE_RENDER_FRAME_CREATED
 }
 
 void ChromeContentRendererClient::WebViewCreated(
