diff --git a/chrome/browser/ui/omnibox/omnibox_theme.cc b/chrome/browser/ui/omnibox/omnibox_theme.cc
index 499923574cf0caee89876a84eccabc883af5d62f..d0b8884614eb9e078041d49e04a350a2c2d982af 100644
--- a/chrome/browser/ui/omnibox/omnibox_theme.cc
+++ b/chrome/browser/ui/omnibox/omnibox_theme.cc
@@ -39,7 +39,7 @@ SkColor GetSecurityChipColor(OmniboxTint tint, OmniboxPartState state) {
 
 }  // namespace
 
-SkColor GetOmniboxColor(OmniboxPart part,
+SkColor GetOmniboxColor_ChromiumImpl(OmniboxPart part,
                         OmniboxTint tint,
                         OmniboxPartState state) {
   using NativeId = ui::NativeTheme::ColorId;
