diff --git a/chrome/browser/extensions/webstore_inline_installer_factory.cc b/chrome/browser/extensions/webstore_inline_installer_factory.cc
index 39e0b83a7aa1b2b75fa412a762134a163e3ae0d7..b61d998422a012fb7bdbe623108cc16b211eac1e 100644
--- a/chrome/browser/extensions/webstore_inline_installer_factory.cc
+++ b/chrome/browser/extensions/webstore_inline_installer_factory.cc
@@ -6,6 +6,7 @@
 
 #include <memory>
 
+#include "brave/browser/extensions/brave_webstore_inline_installer.h"
 #include "chrome/browser/extensions/webstore_inline_installer.h"
 #include "content/public/browser/web_contents.h"
 
@@ -17,8 +18,8 @@ WebstoreInlineInstaller* WebstoreInlineInstallerFactory::CreateInstaller(
     const std::string& webstore_item_id,
     const GURL& requestor_url,
     const WebstoreStandaloneInstaller::Callback& callback) {
-  return new WebstoreInlineInstaller(contents, host, webstore_item_id,
-                                     requestor_url, callback);
+  return new BraveWebstoreInlineInstaller(contents, host, webstore_item_id,
+                                          requestor_url, callback);
 }
 
 }  // namespace extensions
