diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
index 36bd9c4e2538617070592f2ec51061c7a3c37101..d6650ee394a8d3bad38b8d1c77a43ef0b4f76b60 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
@@ -73,7 +73,7 @@ views::Widget* BookmarkBubbleView::ShowBubble(
   // Bookmark bubble should always anchor TOP_RIGHT, but the
   // LocationBarBubbleDelegateView does not know that and may use different
   // arrow anchoring.
-  bookmark_bubble_->SetArrow(views::BubbleBorder::TOP_RIGHT);
+  bookmark_bubble_->SetArrow(views::BubbleBorder::TOP_LEFT);
   if (!anchor_view) {
     bookmark_bubble_->SetAnchorRect(anchor_rect);
     bookmark_bubble_->set_parent_window(parent_window);
