diff --git a/chrome/browser/ui/views/omnibox/omnibox_popup_contents_view.cc b/chrome/browser/ui/views/omnibox/omnibox_popup_contents_view.cc
index 93949fd1aa1dcc712f29788fd0392daf1c0532b0..1cc0dc99eaa48fb4ff004c68e9fd80a1a14229db 100644
--- a/chrome/browser/ui/views/omnibox/omnibox_popup_contents_view.cc
+++ b/chrome/browser/ui/views/omnibox/omnibox_popup_contents_view.cc
@@ -345,7 +345,7 @@ void OmniboxPopupContentsView::UpdatePopupAppearance() {
     if (children().size() == i) {
       AddChildView(std::make_unique<OmniboxRowView>(
           i, edit_model_,
-          std::make_unique<OmniboxResultView>(this, edit_model_, i),
+          std::make_unique<BraveOmniboxResultView>(this, edit_model_, i),
           pref_service));
     }
 
