diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index bb635e9dd9391093e5291b530b49c5cff1458c7d..08d78adb37acbf520ab069d3d7235fc588944b2d 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -9,6 +9,7 @@ import {pageVisibility} from './page_visibility.js';
 import type {SettingsRoutes} from './router.js';
 import {Route, Router} from './router.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -330,6 +331,7 @@ function createRoutes(): SettingsRoutes {
           loadTimeData.getString('performancePageTitle'));
     }
   }
+  addBraveRoutes(r);
   return r as unknown as SettingsRoutes;
 }
 
