diff --git a/chrome/browser/ui/views/toolbar/app_menu.h b/chrome/browser/ui/views/toolbar/app_menu.h
index def1151fc7ee00303f5a8c8531f85a07d9449d05..f3abd671ec02620fb36cd62b9e4835d14535620b 100644
--- a/chrome/browser/ui/views/toolbar/app_menu.h
+++ b/chrome/browser/ui/views/toolbar/app_menu.h
@@ -32,7 +32,7 @@ class MenuRunner;
 }  // namespace views
 
 // AppMenu adapts the AppMenuModel to view's menu related classes.
-class AppMenu final : public views::MenuDelegate,
+class AppMenu : public views::MenuDelegate,
                       public bookmarks::BaseBookmarkModelObserver,
                       public GlobalErrorObserver {
  public:
