diff --git a/chrome/renderer/content_settings_observer.h b/chrome/renderer/content_settings_observer.h
index b77c34d973c8c0760894620cda8673adeb699a40..7bd97747ac267c9cd7bac43f5acb085c2e88c7e5 100644
--- a/chrome/renderer/content_settings_observer.h
+++ b/chrome/renderer/content_settings_observer.h
@@ -111,6 +111,7 @@ class ContentSettingsObserver
   }
 
  private:
+  friend class BraveContentSettingsObserver;
   FRIEND_TEST_ALL_PREFIXES(ContentSettingsObserverTest, WhitelistedSchemes);
   FRIEND_TEST_ALL_PREFIXES(ContentSettingsObserverBrowserTest,
                            ContentSettingsInterstitialPages);
