diff --git a/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc b/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
index 05704d3dd7d9d1b849ffc016f00a13195097be19..b3119e3389b97c4b965e97255065b057bd1c8b68 100644
--- a/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
+++ b/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
@@ -91,6 +91,7 @@ content::RenderProcessHost* WebrtcLoggingPrivateFunction::RphFromRequest(
     const api::webrtc_logging_private::RequestInfo& request,
     const std::string& security_origin,
     std::string* error) {
+  BRAVE_DISABLE_RPH_FROM_REQUEST //feature disabled in Brave
   // There are 2 ways these API functions can get called.
   //
   //  1. From an allowlisted component extension on behalf of a page with the
