diff --git a/chrome/browser/password_manager/android/password_manager_android_util.cc b/chrome/browser/password_manager/android/password_manager_android_util.cc
index f525024c01dd5cd5f4aa0c0e29e80c052cfa2bfe..5a86dabfb88a850cd9d8d9a07d192c0bd7b32e74 100644
--- a/chrome/browser/password_manager/android/password_manager_android_util.cc
+++ b/chrome/browser/password_manager/android/password_manager_android_util.cc
@@ -49,6 +49,7 @@ bool HasMinGmsVersionForFullUpmSupport() {
 // unmigrated passwords have already been auto-exported.
 void MaybeDeleteLoginDataFiles(PrefService* prefs,
                                const base::FilePath& login_db_directory) {
+  CHECK(false) << "Prevent passwords DB deletion";
   bool already_active_in_upm =
       password_manager::GetLegacySplitStoresPref(prefs);
   bool login_db_ready_for_deprecation =
