diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index 4f74e74cfda63e4a9e9cb64c32a490c85a9a42d1..9aafcebcea46f2f5550ff329f7a0b0ad4b80f8a9 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -250,7 +250,7 @@ grit("google_chrome_strings") {
 }
 
 grit("chromium_strings") {
-  source = "chromium_strings.grd"
+  source = "${branding_path_component}_strings.grd"
   defines = chrome_grit_defines
   output_dir = "$root_gen_dir/chrome"
   outputs = [
