diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ntp/RecentTabsPage.java b/chrome/android/java/src/org/chromium/chrome/browser/ntp/RecentTabsPage.java
index 394fd2b10ed607d8ff0f9aa505eb6f252de2d15f..ce88acafffeb90437807031d289fd79c337f6dad 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ntp/RecentTabsPage.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ntp/RecentTabsPage.java
@@ -114,7 +114,7 @@ public class RecentTabsPage
         mListView = mView.findViewById(R.id.odp_listview);
 
         mContextMenuManager =
-                new ContextMenuManager(
+                new org.chromium.chrome.browser.native_page.BraveContextMenuManager(
                         navigationDelegate, this, mActivity::closeContextMenu, "RecentTabs");
 
         mAdapter = new RecentTabsRowAdapter(activity, recentTabsManager, mContextMenuManager);
