diff --git a/chrome/browser/resources/bookmarks/api_listener.ts b/chrome/browser/resources/bookmarks/api_listener.ts
index 27948861dcd9c7e7cf069e8d362536f828522824..eaf197953a355985387a65dd4ca426a48b2aa0ec 100644
--- a/chrome/browser/resources/bookmarks/api_listener.ts
+++ b/chrome/browser/resources/bookmarks/api_listener.ts
@@ -118,6 +118,7 @@ function onChildrenReordered(
  */
 function onImportBegan() {
   chrome.bookmarks.onCreated.removeListener(onBookmarkCreated);
+  chrome.bookmarks.onMoved.removeListener(onBookmarkMoved);
   document.dispatchEvent(new CustomEvent('import-began'));
 }
 
@@ -126,6 +127,7 @@ function onImportEnded() {
     dispatch(refreshNodes(normalizeNodes(results[0]!)));
   });
   chrome.bookmarks.onCreated.addListener(onBookmarkCreated);
+  chrome.bookmarks.onMoved.addListener(onBookmarkMoved);
   document.dispatchEvent(new CustomEvent('import-ended'));
 }
 
