diff --git a/chrome/browser/ui/views/bookmarks/saved_tab_groups/saved_tab_group_button.h b/chrome/browser/ui/views/bookmarks/saved_tab_groups/saved_tab_group_button.h
index 0dfe9338c4182a2148154f0d9aadd459e3385309..eaaf51b37471676c2217b44783c2f1a0f4803640 100644
--- a/chrome/browser/ui/views/bookmarks/saved_tab_groups/saved_tab_group_button.h
+++ b/chrome/browser/ui/views/bookmarks/saved_tab_groups/saved_tab_group_button.h
@@ -96,13 +96,14 @@ class SavedTabGroupButton : public views::MenuButton,
   const base::Uuid guid() const { return guid_; }
 
  private:
+  friend class BraveSavedTabGroupButton;
   FRIEND_TEST_ALL_PREFIXES(SavedTabGroupBarComponentBrowserTest,
                            AccessibleName);
   FRIEND_TEST_ALL_PREFIXES(SavedTabGroupBarComponentBrowserTest, TooltipText);
 
   std::u16string GetAccessibleNameForButton() const;
   void SetTextProperties(const SavedTabGroup& group);
-  void UpdateButtonLayout();
+  virtual void UpdateButtonLayout();
   void UpdateAccessibleName();
   void SetText(std::u16string_view text) override;
   int GetAndIncrementLatestCommandId();
