diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 4f9844f6f8546efe591e679c41c7cd3b43c371a8..89006c99c02315f0c439dbf244d5ae3ce5cb7a0b 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2142,6 +2142,7 @@ jumbo_split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
