diff --git a/chrome/browser/themes/theme_service_win.h b/chrome/browser/themes/theme_service_win.h
index eba1cb2596d71b91e06183209503f3fc83b0e593..966a408a3080c6941db428b554ace6c66f6aead6 100644
--- a/chrome/browser/themes/theme_service_win.h
+++ b/chrome/browser/themes/theme_service_win.h
@@ -18,6 +18,9 @@ class ThemeServiceWin : public ThemeService {
   ~ThemeServiceWin() override;
 
  private:
+#if defined(BRAVE_CHROMIUM_BUILD)
+  friend class BraveThemeServiceWin;
+#endif
   // ThemeService:
   bool ShouldUseNativeFrame() const override;
   SkColor GetDefaultColor(int id, bool incognito) const override;
