diff --git a/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc b/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
index a9069959fb0534ef0795b25125c144211e9fa26b..35daa25203a13d111e7df4e6bd1f3f0f5793fcde 100644
--- a/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
+++ b/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
@@ -320,6 +320,7 @@ void CheckClientDownloadRequestBase::OnRequestBuilt(
     std::unique_ptr<ClientDownloadRequest> request) {
   client_download_request_ = std::move(request);
   SanitizeRequest();
+  BRAVE_ON_REQUEST_BUILT_FILTER
 
   // If it's an archive with no archives or executables, finish early.
   if ((client_download_request_->download_type() ==
