diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index b2207f0d9cf4ec80195b39ce4bba224b4ba33130..73de33ff43a5483a1b7bb6336e218a183bae3db7 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -7,7 +7,7 @@ import {loadTimeData} from 'chrome://resources/js/load_time_data.m.js';
 import {pageVisibility} from './page_visibility.js';
 import {Route, Router} from './router.js';
 import {SettingsRoutes} from './settings_routes.js';
-
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -216,6 +216,7 @@ function createBrowserSettingsRoutes() {
       // </if>
     }
   }
+  addBraveRoutes(r)
   return r;
 }
 
