diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index 8799ad360d51c3a0764185ad2b567550276a5119..e2a52158e126c10f54594e098c0a74f606d64bae 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -203,6 +203,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
