diff --git a/chrome/browser/net/profile_network_context_service.cc b/chrome/browser/net/profile_network_context_service.cc
index e50641649a959a17233ebf36f1d92041e329146a..c6a45234683cccb541e9fcc0a6b6b439a65c0456 100644
--- a/chrome/browser/net/profile_network_context_service.cc
+++ b/chrome/browser/net/profile_network_context_service.cc
@@ -517,6 +517,7 @@ network::mojom::CTPolicyPtr ProfileNetworkContextService::GetCTPolicy() {
   std::vector<std::string> excluded_legacy_spkis(
       TranslateStringArray(ct_excluded_legacy_spkis));
 
+  BRAVE_PROFILE_NETWORK_CONTEXT_SERVICE_GET_CT_POLICY
   return network::mojom::CTPolicy::New(std::move(excluded),
                                        std::move(excluded_spkis),
                                        std::move(excluded_legacy_spkis));
