diff --git a/chrome/browser/importer/external_process_importer_host.cc b/chrome/browser/importer/external_process_importer_host.cc
index 9087093838f249ca73cc9fc82d54d5b44e9bb5f2..d8eb7cbb25bf4ef9f931830b1ebdd71453f86b95 100644
--- a/chrome/browser/importer/external_process_importer_host.cc
+++ b/chrome/browser/importer/external_process_importer_host.cc
@@ -60,6 +60,7 @@ void ExternalProcessImporterHost::StartImportSettings(
     return;
   }
 
+  BRAVE_START_IMPORT_SETTINGS
   CheckForLoadedModels(items);
 
   LaunchImportIfReady();
