diff --git a/chrome/browser/extensions/component_loader_factory.cc b/chrome/browser/extensions/component_loader_factory.cc
index 2d2c332880ffeced368c11b004eb3c96f6ae823d..580e22a60d02a75f94547793914ebc5d177939b0 100644
--- a/chrome/browser/extensions/component_loader_factory.cc
+++ b/chrome/browser/extensions/component_loader_factory.cc
@@ -56,7 +56,7 @@ ComponentLoaderFactory::BuildServiceInstanceForBrowserContext(
     content::BrowserContext* context) const {
   Profile* profile = Profile::FromBrowserContext(context);
   // Use `new` to access private constructor.
-  return base::WrapUnique(new ComponentLoader(profile));
+  return base::WrapUnique(new BraveComponentLoader(profile));
 }
 
 }  // namespace extensions
