diff --git a/chrome/browser/ui/views/tabs/tab.cc b/chrome/browser/ui/views/tabs/tab.cc
index 57fe65bc63bc6f65a4332f8cd07676d25e552cb0..f71b5839ba812d145b26a2baf907b25adec1808f 100644
--- a/chrome/browser/ui/views/tabs/tab.cc
+++ b/chrome/browser/ui/views/tabs/tab.cc
@@ -385,6 +385,7 @@ void Tab::Layout() {
     int title_right = contents_rect.right();
     if (showing_alert_indicator_) {
       title_right = alert_indicator_->x() - after_title_padding;
+      BRAVE_UI_VIEWS_TABS_TAB_ALERT_INDICATOR_POSITION
     } else if (showing_close_button_) {
       // Allow the title to overlay the close button's empty border padding.
       title_right = close_x - after_title_padding;
@@ -991,6 +992,7 @@ void Tab::UpdateIconVisibility() {
       available_width -= favicon_width;
 
     showing_close_button_ = large_enough_for_close_button;
+    BRAVE_UI_VIEWS_TABS_TAB_UPDATE_ICON_VISIBILITY
     if (showing_close_button_)
       available_width -= close_button_width;
 
