diff --git a/chrome/installer/mac/signing/pipeline.py b/chrome/installer/mac/signing/pipeline.py
index 58578b567b62be29915a36266d12b3a7dd4e8ab9..19bcb56a6030225ea4f2d23744bb72383a41cbf5 100644
--- a/chrome/installer/mac/signing/pipeline.py
+++ b/chrome/installer/mac/signing/pipeline.py
@@ -319,7 +319,6 @@ def _package_installer_tools(paths, config):
         'dirdiffer.sh',
         'dirpatcher.sh',
         'dmgdiffer.sh',
-        'keystone_install.sh',
         'pkg-dmg',
     )
 
