diff --git a/chrome/browser/sessions/BUILD.gn b/chrome/browser/sessions/BUILD.gn
index 651b3dc2887cc75ee9599a45e768e1120734ec65..713a4e4f0879811ec1460652fb4564e5df2334a2 100644
--- a/chrome/browser/sessions/BUILD.gn
+++ b/chrome/browser/sessions/BUILD.gn
@@ -164,6 +164,7 @@ source_set("impl") {
         "//chromeos/components/kiosk",
       ]
     }
+    import("//brave/chromium_src/chrome/browser/sessions/sources.gni") deps += brave_chromium_src_chrome_browser_sessions_deps
   }
 }
 
