diff --git a/chrome/browser/ui/views/frame/browser_view.h b/chrome/browser/ui/views/frame/browser_view.h
index 75b245723b424cd4575aa5fac437491ca21d1871..b52bfd75b5004d18b288eff27f54b4d8f932c307 100644
--- a/chrome/browser/ui/views/frame/browser_view.h
+++ b/chrome/browser/ui/views/frame/browser_view.h
@@ -748,6 +748,7 @@ class BrowserView : public BrowserWindow,
       const std::map<std::string, std::string>& extra_data) override;
 #endif
 
+  BRAVE_BROWSER_VIEW_H
  private:
   // Do not friend BrowserViewLayout. Use the BrowserViewLayoutDelegate
   // interface to keep these two classes decoupled and testable.
