diff --git a/chrome/browser/ui/page_action/page_action_properties_provider.cc b/chrome/browser/ui/page_action/page_action_properties_provider.cc
index e91b9c3673a132ac55fede5259578ffb9bf7a51a..6a715f1e603e9635ec761bae80cf7149e2548b2b 100644
--- a/chrome/browser/ui/page_action/page_action_properties_provider.cc
+++ b/chrome/browser/ui/page_action/page_action_properties_provider.cc
@@ -215,6 +215,13 @@ constexpr auto kPageActionProperties = base::MakeFixedFlatMap<
             .element_identifier = kAutofillAddressPageActionElementId,
         },
     },
+    {
+        kActionShowPartitionedStorage,
+        {
+            .histogram_name = "PartitionedStorage",
+            .type = brave::kPartitionedStorageActionIconType,
+        },
+    },
     {
         kActionVirtualCardEnroll,
         {
