diff --git a/chrome/browser/flags/android/chrome_feature_list.cc b/chrome/browser/flags/android/chrome_feature_list.cc
index 2097a93069c8b9bfe354d64d97ef814382e9ff2f..ad748c7b87729517c5feee3a5b509504a0620d47 100644
--- a/chrome/browser/flags/android/chrome_feature_list.cc
+++ b/chrome/browser/flags/android/chrome_feature_list.cc
@@ -343,6 +343,7 @@ const base::Feature* const kFeaturesExposedToJava[] = {
 };
 
 const base::Feature* FindFeatureExposedToJava(const std::string& feature_name) {
+  BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA
   for (const auto* feature : kFeaturesExposedToJava) {
     if (feature->name == feature_name)
       return feature;
