diff --git a/chrome/browser/ui/views/tabs/alert_indicator.h b/chrome/browser/ui/views/tabs/alert_indicator.h
index 364fef52f0e8ade9a9099cc879de0e0db132204e..b0f781db92edca41ec6275bd65f10dddfccc66af 100644
--- a/chrome/browser/ui/views/tabs/alert_indicator.h
+++ b/chrome/browser/ui/views/tabs/alert_indicator.h
@@ -50,6 +50,7 @@ class AlertIndicator : public views::ImageView {
   View* GetTooltipHandlerForPoint(const gfx::Point& point) override;
 
  private:
+  friend class BraveAlertIndicator;
   friend class AlertIndicatorTest;
   friend class TabTest;
   class FadeAnimationDelegate;
