diff --git a/chrome/browser/resources/tab_search/BUILD.gn b/chrome/browser/resources/tab_search/BUILD.gn
index d8bc1b7393fd55b984adbc850909b9f13a2c195f..6aad94f29dd6e0d9d4227093da2644917c8b8965 100644
--- a/chrome/browser/resources/tab_search/BUILD.gn
+++ b/chrome/browser/resources/tab_search/BUILD.gn
@@ -130,4 +130,5 @@ build_webui("build") {
       "tab_search.js",
     ]
   }
+  import("//brave/browser/resources/tab_search/sources.gni") extra_grdp_deps = brave_tab_search_extra_grdp_deps extra_grdp_files = brave_tab_search_extra_grdp_files
 }
