diff --git a/chrome/tools/build/mac/infoplist_strings_util.cc b/chrome/tools/build/mac/infoplist_strings_util.cc
index 89801229291a63e69574bc13cfe563aac360012f..038a9a66222dedad219c83b06be3e53a00b70f94 100644
--- a/chrome/tools/build/mac/infoplist_strings_util.cc
+++ b/chrome/tools/build/mac/infoplist_strings_util.cc
@@ -153,6 +153,7 @@ int main(int argc, char* const argv[]) {
     // Fetch the strings.
     std::string name = LoadStringFromDataPack(branded_data_pack.get(), cur_lang,
                                               name_id, name_id_str);
+    name = BRAVE_PRODUCT_STRING;
     std::string copyright_format = LoadStringFromDataPack(
         branded_data_pack.get(), cur_lang, IDS_ABOUT_VERSION_COPYRIGHT,
         "IDS_ABOUT_VERSION_COPYRIGHT");
