diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 70a4fa14560b92599bcfa4aadc4cda73bb8cc5a0..092c239a62902f06a3f6f0f3e4fd80d1a32b7e6c 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -1271,6 +1271,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
