diff --git a/chrome/browser/permissions/permission_manager.cc b/chrome/browser/permissions/permission_manager.cc
index b6c6d202c03df12aa2fa86983b445d6252f65337..de029c0e54eee56490919955376e06addd6fd133 100644
--- a/chrome/browser/permissions/permission_manager.cc
+++ b/chrome/browser/permissions/permission_manager.cc
@@ -85,7 +85,7 @@ PermissionStatus ContentSettingToPermissionStatus(ContentSetting setting) {
 }
 
 // Helper method to convert PermissionType to ContentSettingType.
-ContentSettingsType PermissionTypeToContentSetting(PermissionType permission) {
+ContentSettingsType PermissionTypeToContentSetting_ChromiumImpl(PermissionType permission) {
   switch (permission) {
     case PermissionType::MIDI:
       return CONTENT_SETTINGS_TYPE_MIDI;
@@ -123,6 +123,7 @@ ContentSettingsType PermissionTypeToContentSetting(PermissionType permission) {
     case PermissionType::PAYMENT_HANDLER:
       return CONTENT_SETTINGS_TYPE_PAYMENT_HANDLER;
     case PermissionType::NUM:
+    default:
       // This will hit the NOTREACHED below.
       break;
   }
