diff --git a/chrome/browser/ui/views/frame/browser_view.h b/chrome/browser/ui/views/frame/browser_view.h
index 72454b49ce1fe10862b4a5e1870f4409ef6ea01b..a421c936f3e417f477b69e09b614ac37681473cd 100644
--- a/chrome/browser/ui/views/frame/browser_view.h
+++ b/chrome/browser/ui/views/frame/browser_view.h
@@ -703,6 +703,7 @@ class BrowserView : public BrowserWindow,
       const std::map<std::string, base::Value>& extra_data) override;
 #endif
 
+  BRAVE_BROWSER_VIEW_H
  private:
   // Do not friend BrowserViewLayout. Use the BrowserViewLayoutDelegate
   // interface to keep these two classes decoupled and testable.
