diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index b4796a1ddc7805096e3d9ab20403faa1042221d7..17b822cc7b7ebd33e7b95953adb145ce107884dc 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -149,6 +149,7 @@ source_set("internal") {
   if (enable_extensions_core) {
     deps += [ "//chrome/browser/extensions" ]
   }
+  import("//brave/browser/ui/browser_window/internal/sources.gni") deps += brave_browser_window_internal_deps
 }
 
 source_set("unit_tests") {
