diff --git a/chrome/browser/resources/settings/settings_page/main_page_mixin.ts b/chrome/browser/resources/settings/settings_page/main_page_mixin.ts
index 6c5d4cb887e6741d5d319b1103f1a5fe4fddf72e..0026b2ff4d75b1d2acb7c64bdcbbffb1448ece24 100644
--- a/chrome/browser/resources/settings/settings_page/main_page_mixin.ts
+++ b/chrome/browser/resources/settings/settings_page/main_page_mixin.ts
@@ -40,7 +40,7 @@ guestTopLevelRoute = routes.PRIVACY;
 // </if>
 
 const TOP_LEVEL_EQUIVALENT_ROUTE: Route =
-    loadTimeData.getBoolean('isGuest') ? guestTopLevelRoute : routes.PEOPLE;
+    loadTimeData.getBoolean('isGuest') ? guestTopLevelRoute : routes.GET_STARTED;
 
 function classifyRoute(route: Route|null): RouteState {
   if (!route) {
