diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.cc b/chrome/browser/ui/views/location_bar/location_bar_view.cc
index bf6d1bf97e0e604851c0cc89c962799f6ad5b929..066b057f8627f5521ad8fa28aa2e24625e6b27d9 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
@@ -714,6 +714,7 @@ void LocationBarView::Layout(PassKey) {
   if (show_overriding_permission_chip)
     text_left += text_overriding_permission_chip_indent;
 
+  BRAVE_LAYOUT_LEADING_DECORATIONS
   LocationBarLayout leading_decorations(LocationBarLayout::Position::kLeftEdge,
                                         text_left);
   LocationBarLayout trailing_decorations(
@@ -793,6 +794,7 @@ void LocationBarView::Layout(PassKey) {
     }
   };
 
+  BRAVE_LAYOUT_TRAILING_DECORATIONS
   add_trailing_decoration(page_action_icon_container_,
                           /*intra_item_padding=*/0);
   for (ContentSettingImageView* view : base::Reversed(content_setting_views_)) {
