diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/homepage/HomepageSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/homepage/HomepageSettings.java
index adc23a019d6efc4d2e7400c8d47f8c89481cf154..5bf845b378aa7dd94d10944807962120f63feb01 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/homepage/HomepageSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/homepage/HomepageSettings.java
@@ -21,7 +21,7 @@ import org.chromium.chrome.browser.settings.SettingsUtils;
 /**
  * Fragment that allows the user to configure homepage related preferences.
  */
-public class HomepageSettings extends PreferenceFragmentCompat {
+public class HomepageSettings extends org.chromium.chrome.browser.settings.BravePreferenceFragment {
     @VisibleForTesting
     public static final String PREF_HOMEPAGE_SWITCH = "homepage_switch";
     @VisibleForTesting
