diff --git a/chrome/browser/resources/bookmarks/command_manager.ts b/chrome/browser/resources/bookmarks/command_manager.ts
index d28158479a5f2fdfb42e50856d98d1889e6c02fa..dc31bcabce383dfc8600e44bed2aff675b29e61a 100644
--- a/chrome/browser/resources/bookmarks/command_manager.ts
+++ b/chrome/browser/resources/bookmarks/command_manager.ts
@@ -428,7 +428,7 @@ export class BookmarksCommandManagerElement extends
         chrome.bookmarks.export();
         break;
       case Command.HELP_CENTER:
-        window.open('https://support.google.com/chrome/?p=bookmarks');
+        window.open('https://community.brave.com');
         break;
       default:
         assertNotReached();
