diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 6cd4bb240c669f351f502a86a98f3645d8cc23b8..a919fa7959fa0c3abf26aa7fc84377c158b423bc 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3332,6 +3332,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
