diff --git a/chrome/browser/media/router/providers/cast/cast_activity_record.cc b/chrome/browser/media/router/providers/cast/cast_activity_record.cc
index 7e25f7b98dd3..b82caaba750c 100644
--- a/chrome/browser/media/router/providers/cast/cast_activity_record.cc
+++ b/chrome/browser/media/router/providers/cast/cast_activity_record.cc
@@ -159,12 +159,16 @@ bool CastActivityRecord::CanJoinSession(const CastMediaSource& cast_source,
 bool CastActivityRecord::HasJoinableClient(AutoJoinPolicy policy,
                                            const url::Origin& origin,
                                            int tab_id) const {
+#if !defined(OS_ANDROID)
   return std::any_of(connected_clients_.begin(), connected_clients_.end(),
                      [policy, &origin, tab_id](const auto& client) {
                        return IsAutoJoinAllowed(policy, origin, tab_id,
                                                 client.second->origin(),
                                                 client.second->tab_id());
                      });
+#else
+  return false;
+#endif
 }
 
 }  // namespace media_router
