diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index e71281c19320c47001b9ffc758d2f914098dd228..bccbb95f9220b97922ef70d103f237e2a26888b4 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -31,6 +31,7 @@ if (optimize_webui) {
 
     deps = [
       ":unpak",
+      "//brave/browser/resources/settings:unpak",
       "../../../../ui/webui/resources:modulize",
     ]
     excludes = [ "chrome://resources/js/cr.m.js" ]
