diff --git a/chrome/installer/mini_installer/BUILD.gn b/chrome/installer/mini_installer/BUILD.gn
index cc3782f4a91da503a743d38740210f8f8a973e8d..e5da0757587fca503577892afd84ed4eaf35787e 100644
--- a/chrome/installer/mini_installer/BUILD.gn
+++ b/chrome/installer/mini_installer/BUILD.gn
@@ -130,11 +130,12 @@ template("generate_mini_installer") {
 
     inputs = [
       "$chrome_dll_file",
-      "$root_out_dir/chrome.exe",
+      "$root_out_dir/brave.exe",
       "$root_out_dir/locales/en-US.pak",
       "$root_out_dir/setup.exe",
       "$root_out_dir/v8_context_snapshot.bin",
       release_file,
+      "$root_out_dir/brave_resources.pak",
     ]
 
     outputs = [
@@ -179,6 +180,7 @@ template("generate_mini_installer") {
       "//third_party/icu:icudata",
       "//tools/v8_context_snapshot",
       chrome_dll_target,
+      "//brave:packed_extra_resources",
     ]
 
     if (enable_hidpi) {
