diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index b067a3f7af514bc48632365427ba8ecffa4885df..dadadab8dd95e47bc228053bd1335cbd6d87e3b6 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -236,6 +236,7 @@ android_resources("chrome_app_java_resources") {
     "//third_party/androidx:androidx_preference_preference_java",
     "//third_party/androidx:androidx_recyclerview_recyclerview_java",
   ]
+  deps += brave_chrome_app_java_resources_deps sources += brave_java_resources resource_overlay = true
 }
 
 if (enable_vr) {
@@ -551,6 +552,7 @@ android_library("chrome_java") {
   ]
 
   deps += feed_deps
+  deps += brave_chrome_java_deps
 
   srcjar_deps = [
     ":chrome_android_java_enums_srcjar",
@@ -583,6 +585,7 @@ android_library("chrome_java") {
     "//components/ui_metrics:ui_metrics_enums_java",
     "//net:effective_connection_type_java",
   ]
+  srcjar_deps += brave_chrome_java_srcjar_deps
 
   # From java_sources.gni.
   sources = chrome_java_sources + [ app_hooks_impl ]
@@ -710,6 +713,7 @@ group("jni_headers") {
     "//chrome/browser/webauthn/android:jni_headers",
     "//components/media_router/browser/android:jni_headers",
   ]
+  public_deps += ["//brave/build/android:jni_headers"]
 }
 
 # TODO(crbug.com/1112471): Get this to run cleanly under Python 3.
@@ -1897,6 +1901,7 @@ if (current_toolchain == default_toolchain) {
         ":${_variant}_locale_pak_assets",
         ":${_variant}_paks",
       ]
+      deps += [ "//brave:brave_pak_assets" ]
       if (_is_monochrome) {
         deps += [ "//android_webview:locale_pak_assets" ]
         if (webview_includes_weblayer && !_is_bundle_module) {
