diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 1ad2ed58b40859ed878c81445e4d556e0fc33ea5..72401bd3e51926aa03749d5c0eadd69514342535 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -81,6 +81,7 @@ source_set("channel_info") {
   } else if (is_posix) {
     sources += [ "channel_info_posix.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -544,6 +545,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
