diff --git a/chrome/browser/ui/location_bar/location_bar.h b/chrome/browser/ui/location_bar/location_bar.h
index 239d3bac4ecff88a3a12b5cc6cf1c2fa8ff0c500..5d5072c10236b2e324f756c069cf98ac767f968e 100644
--- a/chrome/browser/ui/location_bar/location_bar.h
+++ b/chrome/browser/ui/location_bar/location_bar.h
@@ -81,7 +81,7 @@ class LocationBar {
   virtual ~LocationBar();
 
   // Checks if any extension has requested that the bookmark star be hidden.
-  bool IsBookmarkStarHiddenByExtension() const;
+  virtual bool IsBookmarkStarHiddenByExtension() const;
 
  private:
   class ExtensionLoadObserver;
