diff --git a/chrome/app/chrome_command_ids.h b/chrome/app/chrome_command_ids.h
index 1f25ecad4be9fc5863888dba76b70b22dafed861..92e61f7ad1bd1ba5384647ed4b028a41fdc9b1ae 100644
--- a/chrome/app/chrome_command_ids.h
+++ b/chrome/app/chrome_command_ids.h
@@ -5,6 +5,10 @@
 #ifndef CHROME_APP_CHROME_COMMAND_IDS_H_
 #define CHROME_APP_CHROME_COMMAND_IDS_H_
 
+#if defined(BRAVE_CHROMIUM_BUILD)
+#include "brave/app/brave_command_ids.h"
+#endif
+
 // This file lists all the command IDs understood by e.g. the browser.
 // It is used by Windows RC files, Mac NIB files, and other platforms too.
 
