diff --git a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts
index 49053ca3cf3cf4c2e816815f535401a4f8d38740..547e0eb6fea835eccabf51e7b2339071f01f2a00 100644
--- a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts
+++ b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts
@@ -109,6 +109,7 @@ export class ProfilePickerMainViewElement extends
   private eventTracker_: EventTracker = new EventTracker();
 
   override firstUpdated() {
+    this.$.addProfile.style.borderRadius = '8px';
     this.addEventListener('view-enter-finish', this.onViewEnterFinish_);
 
     this.addEventListener('toggle-drag', this.toggleDrag_);
