diff --git a/chrome/browser/resources/bookmarks/command_manager.js b/chrome/browser/resources/bookmarks/command_manager.js
index 11e79acf9c00396e969a2172974cc517729f37b1..d43a0cb763973c65118301f6864e388a77843b12 100644
--- a/chrome/browser/resources/bookmarks/command_manager.js
+++ b/chrome/browser/resources/bookmarks/command_manager.js
@@ -443,7 +443,7 @@ export const CommandManager = Polymer({
         chrome.bookmarks.export();
         break;
       case Command.HELP_CENTER:
-        window.open('https://support.google.com/chrome/?p=bookmarks');
+        window.open('https://community.brave.com');
         break;
       default:
         assert(false);
