diff --git a/chrome/browser/ui/chrome_pages.cc b/chrome/browser/ui/chrome_pages.cc
index f1d4aefe3ffade61c1e86c7ac1c55707d12ccea5..32ee8a3086fc3de52618b5dac34a27f61948adf1 100644
--- a/chrome/browser/ui/chrome_pages.cc
+++ b/chrome/browser/ui/chrome_pages.cc
@@ -414,7 +414,7 @@ void ShowSlow(Browser* browser) {
 #endif
 }
 
-GURL GetSettingsUrl(std::string_view sub_page) {
+BRAVE_GET_SETTINGS_URL
   return GURL(base::StrCat({kChromeUISettingsURL, sub_page}));
 }
 
