diff --git a/chrome/browser/metrics/chrome_metrics_services_manager_client.cc b/chrome/browser/metrics/chrome_metrics_services_manager_client.cc
index c75a9c8727d566512c210297707f10262c907f01..c83aa3c92c5cbd549db8d93ba5219696fd1ebf52 100644
--- a/chrome/browser/metrics/chrome_metrics_services_manager_client.cc
+++ b/chrome/browser/metrics/chrome_metrics_services_manager_client.cc
@@ -153,7 +153,7 @@ class ChromeMetricsServicesManagerClient::ChromeEnabledStateProvider
   }
 
   bool IsReportingEnabled() const override {
-    return metrics::EnabledStateProvider::IsReportingEnabled() &&
+    return false && metrics::EnabledStateProvider::IsReportingEnabled() &&
            IsClientInSampleImpl(local_state_);
   }
 
