diff --git a/chrome/browser/ui/webui/settings/import_data_handler.h b/chrome/browser/ui/webui/settings/import_data_handler.h
index 8f1fc60d23ea7ab9d8b186a1de1b75f5251c20ee..159874741d1fd43995cecf61bd4e0496a1912de7 100644
--- a/chrome/browser/ui/webui/settings/import_data_handler.h
+++ b/chrome/browser/ui/webui/settings/import_data_handler.h
@@ -37,6 +37,7 @@ class ImportDataHandler : public SettingsPageUIHandler,
   void OnJavascriptAllowed() override {}
   void OnJavascriptDisallowed() override;
 
+  BRAVE_IMPORT_DATA_HANDLER_H
  private:
   void StartImport(const importer::SourceProfile& source_profile,
                    uint16_t imported_items);
