diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
index e2d52f4385d8f71bfa4bf928c88542532f5ca445..96f0a49f90dba716d8b5bf112c1f8c53d2871d33 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
@@ -81,6 +81,7 @@ class RecentTabsSubMenuModel : public ui::SimpleMenuModel,
   bool GetURLAndTitleForItemAtIndex(int index,
                                     std::string* url,
                                     std::u16string* title);
+  BRAVE_RECENT_TABS_SUB_MENU_MODEL_H_
 
  private:
   struct TabNavigationItem;
