diff --git a/chrome/browser/ui/messages/android/BUILD.gn b/chrome/browser/ui/messages/android/BUILD.gn
index 6aea71fdfe3e90a464c9326075a3a5a37b1ca5a3..8eedf226a056bc822d8048866fab2200b10d5814 100644
--- a/chrome/browser/ui/messages/android/BUILD.gn
+++ b/chrome/browser/ui/messages/android/BUILD.gn
@@ -30,6 +30,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarSwipeHandler.java",
     "java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarView.java",
   ]
+  sources += brave_browser_ui_messages_java_sources
 
   deps = [
     ":java_resources",
