diff --git a/chrome/browser/resources/settings/settings.ts b/chrome/browser/resources/settings/settings.ts
index b041a6f32aaf6009cb06e22461c456d06bd0e523..c29d4a8c7b051aa672f3f4e59d6207ab246bad64 100644
--- a/chrome/browser/resources/settings/settings.ts
+++ b/chrome/browser/resources/settings/settings.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './settings_ui/settings_ui.js';
 
 export {ControlledRadioButtonElement} from '/shared/settings/controls/controlled_radio_button.js';
