diff --git a/chrome/browser/ui/extensions/icon_with_badge_image_source.cc b/chrome/browser/ui/extensions/icon_with_badge_image_source.cc
index c566b0319a6d757d72096503bd66f3f9fa408432..89e98a66acada18044ac4bc78ca7f3199afd679a 100644
--- a/chrome/browser/ui/extensions/icon_with_badge_image_source.cc
+++ b/chrome/browser/ui/extensions/icon_with_badge_image_source.cc
@@ -146,14 +146,18 @@ void IconWithBadgeImageSource::Draw(gfx::Canvas* canvas) {
   gfx::ImageSkiaRep rep = skia.GetRepresentation(canvas->image_scale());
   if (rep.scale() != canvas->image_scale()) {
     skia.AddRepresentation(ScaleImageSkiaRep(
-        rep, ExtensionAction::ActionIconSize(), canvas->image_scale()));
+        rep, GetCustomGraphicSize().value_or(ExtensionAction::ActionIconSize()), canvas->image_scale()));
   }
   if (grayscale_)
     skia = gfx::ImageSkiaOperations::CreateHSLShiftedImage(skia, {-1, 0, 0.6});
 
   int x_offset =
+    GetCustomGraphicXOffset().has_value() ?
+      GetCustomGraphicXOffset().value() :
       std::floor((size().width() - ExtensionAction::ActionIconSize()) / 2.0);
   int y_offset =
+    GetCustomGraphicYOffset().has_value() ?
+      GetCustomGraphicYOffset().value() :
       std::floor((size().height() - ExtensionAction::ActionIconSize()) / 2.0);
   canvas->DrawImageInt(skia, x_offset, y_offset);
 
