diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 63bd79598ded0350612843ab5a60df9d4fa56f61..978bc620e6320332156aa5dd8457e93db06dbd1c 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -90,6 +90,7 @@ class AvatarToolbarButton : public ToolbarButton,
   void OnBoundsChanged(const gfx::Rect& previous_bounds) override;
 
  private:
+  friend class BraveAvatarToolbarButton;
   // Used by tests to access the private state_manager_ for verification.
   friend class ::AvatarToolbarButtonTestAccessor;
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
