diff --git a/chrome/browser/net/proxy_config_monitor.cc b/chrome/browser/net/proxy_config_monitor.cc
index ecc6f393533139045b9153c428bfb34a45e9c272..33c0023bacb77db9c888edf5ba4928ff5dd08a37 100644
--- a/chrome/browser/net/proxy_config_monitor.cc
+++ b/chrome/browser/net/proxy_config_monitor.cc
@@ -52,6 +52,7 @@ ProxyConfigMonitor::ProxyConfigMonitor(Profile* profile) {
             profile->GetPrefs(), g_browser_process->local_state());
   }
 
+  BRAVE_PROXY_CONFIG_MONITOR
   proxy_config_service_ = ProxyServiceFactory::CreateProxyConfigService(
       pref_proxy_config_tracker_.get(), profile);
 
