diff --git a/chrome/browser/importer/external_process_importer_client.h b/chrome/browser/importer/external_process_importer_client.h
index 8e4f47662a42cdde51125a3e58bc870d42f9bdd8..c950791430e482aa8028187ddd6d205b05eb0bf6 100644
--- a/chrome/browser/importer/external_process_importer_client.h
+++ b/chrome/browser/importer/external_process_importer_client.h
@@ -90,6 +90,7 @@ class ExternalProcessImporterClient
  protected:
   ~ExternalProcessImporterClient() override;
 
+  BRAVE_EXTERNAL_PROCESS_IMPORTER_CLIENT_H
  private:
   friend class base::RefCounted<ExternalProcessImporterClient>;
 
