diff --git a/chrome/browser/web_applications/extensions/BUILD.gn b/chrome/browser/web_applications/extensions/BUILD.gn
index e73d5757225d8f9ee72acde11ac401d7a70ce035..0a9af989696164832bc85f6deb3df56d218aabad 100644
--- a/chrome/browser/web_applications/extensions/BUILD.gn
+++ b/chrome/browser/web_applications/extensions/BUILD.gn
@@ -66,7 +66,6 @@ source_set("unit_tests") {
     "//chrome/browser/web_applications:web_applications_test_support",
     "//chrome/browser/web_applications/components",
     "//chrome/common",
-    "//chrome/test:test_support",
     "//components/crx_file:crx_file",
     "//content/public/browser",
     "//content/test:test_support",
@@ -75,6 +74,10 @@ source_set("unit_tests") {
     "//skia",
     "//testing/gtest",
   ]
+
+  if (!is_android) {
+    deps += ["//chrome/test:test_support",]
+  }
 }
 
 source_set("browser_tests") {
@@ -100,11 +103,15 @@ source_set("browser_tests") {
     "//chrome/browser/web_applications:web_applications_on_extensions_test_support",
     "//chrome/browser/web_applications:web_applications_test_support",
     "//chrome/browser/web_applications/components",
-    "//chrome/test:test_support",
-    "//chrome/test:test_support_ui",
     "//extensions:test_support",
     "//extensions/browser",
     "//extensions/common",
     "//net:test_support",
   ]
+
+  if (!is_android) {
+    deps += ["//chrome/test:test_support",
+      "//chrome/test:test_support_ui",
+    ]
+  }
 }
