diff --git a/chrome/browser/importer/external_process_importer_client.h b/chrome/browser/importer/external_process_importer_client.h
index 6d38b92400bbf4e1e4475959247ff74b0a9d83df..67739b6d04ef2206f36c126c738802b7449294cb 100644
--- a/chrome/browser/importer/external_process_importer_client.h
+++ b/chrome/browser/importer/external_process_importer_client.h
@@ -84,6 +84,7 @@ class ExternalProcessImporterClient
       const std::vector<ImporterAutofillFormDataEntry>&
           autofill_form_data_entry_group) override;
 
+  BRAVE_EXTERNAL_PROCESS_IMPORTER_CLIENT_H
  protected:
   ~ExternalProcessImporterClient() override;
 
