diff --git a/chrome/browser/ui/views/profiles/profile_menu_view.h b/chrome/browser/ui/views/profiles/profile_menu_view.h
index beb7d3d06a47ffb9facdbfe1f6f4d6d544f00739..2455e45596b329f5ed3899a7a373e4fbf3fe6c65 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view.h
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.h
@@ -46,6 +46,7 @@ class ProfileMenuView : public ProfileMenuViewBase {
   gfx::ImageSkia GetSyncIcon() const override;
 
  private:
+  BRAVE_PROFILE_MENU_VIEW_H
   friend class ProfileMenuViewExtensionsTest;
   friend class ProfileMenuViewSignoutTest;
 
