diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
index 24ae4d886d3dfcb3406a8333abefaf4508696ca6..c28b7a1f540ab6a8b8a7c6c1ec69f7312a0bd467 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
@@ -81,6 +81,7 @@ class RecentTabsSubMenuModel : public ui::SimpleMenuModel,
   bool GetURLAndTitleForItemAtIndex(int index,
                                     std::string* url,
                                     std::u16string* title);
+  BRAVE_RECENT_TABS_SUB_MENU_MODEL_H_
 
  private:
   struct TabNavigationItem;
