diff --git a/chrome/browser/ui/views/profiles/profile_chooser_view.h b/chrome/browser/ui/views/profiles/profile_chooser_view.h
index 68f6117e09153a3234515286d783df9226ac9791..6537a30ddb9ef6d580eff348150dc4b884ba87cb 100644
--- a/chrome/browser/ui/views/profiles/profile_chooser_view.h
+++ b/chrome/browser/ui/views/profiles/profile_chooser_view.h
@@ -62,6 +62,8 @@ class ProfileChooserView : public ProfileMenuViewBase,
   static void Hide();
 
  private:
+  friend class BraveProfileChooserView;
+  friend class BraveProfileChooserViewTest;
   friend class ProfileChooserViewExtensionsTest;
 
   typedef std::vector<size_t> Indexes;
@@ -104,7 +106,7 @@ class ProfileChooserView : public ProfileMenuViewBase,
   // Tests set this to "false" for more consistent operation.
   static bool close_on_deactivate_for_testing_;
 
-  void ResetView();
+  virtual void ResetView();
 
   // Shows the bubble with the |view_to_display|.
   void ShowView(profiles::BubbleViewMode view_to_display,
