diff --git a/chrome/browser/flags/android/chrome_feature_list.cc b/chrome/browser/flags/android/chrome_feature_list.cc
index 8417d2383422a35eabf4a74b0fc5722844e09488..c6489c9ffb1e0ecce95a80290104a29bcc60ee6d 100644
--- a/chrome/browser/flags/android/chrome_feature_list.cc
+++ b/chrome/browser/flags/android/chrome_feature_list.cc
@@ -391,6 +391,7 @@ const base::Feature* const kFeaturesExposedToJava[] = {
 };
 
 const base::Feature* FindFeatureExposedToJava(const std::string& feature_name) {
+  BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA
   static auto kFeaturesExposedToJavaMap = base::NoDestructor(
       base::MakeFlatMap<base::StringPiece, const base::Feature*>(
           kFeaturesExposedToJava, {},
