diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index f5e62acbe488bbf758d0084ae3fca7ef05595a13..63ac37a30dc9d0b6e35d238fd77885643b9bafef 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -78,6 +78,7 @@ source_set("configs") {
   if (enable_glic) {
     deps += [ "//chrome/browser/glic" ]
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("webui") {
