diff --git a/chrome/browser/ui/views/translate/translate_bubble_view.cc b/chrome/browser/ui/views/translate/translate_bubble_view.cc
index 325fac91ab832682e07d3aab6b202ea21f89ab6d..8d5acddae44d38dff3fd080487112f8f586ad550 100644
--- a/chrome/browser/ui/views/translate/translate_bubble_view.cc
+++ b/chrome/browser/ui/views/translate/translate_bubble_view.cc
@@ -182,7 +182,7 @@ views::Widget* TranslateBubbleView::ShowBubble(
           source_language, target_language));
   std::unique_ptr<TranslateBubbleModel> model(
       new TranslateBubbleModelImpl(step, std::move(ui_delegate)));
-  TranslateBubbleView* view = new TranslateBubbleView(
+  TranslateBubbleView* view = new BRAVE_TRANSLATE_BUBBLE_VIEW_(
       anchor_view, std::move(model), error_type, web_contents);
 
   if (highlighted_button)
