diff --git a/chrome/browser/extensions/extension_management.h b/chrome/browser/extensions/extension_management.h
index 7d50b4a57aa5cee47149df30ccc026dfefdc3161..def64912783c871c39aa5321b1645bd234acabdb 100644
--- a/chrome/browser/extensions/extension_management.h
+++ b/chrome/browser/extensions/extension_management.h
@@ -185,6 +185,7 @@ class ExtensionManagement : public KeyedService {
   bool IsChild() const { return is_child_; }
 
  private:
+  friend class BraveExtensionManagement;
   using SettingsIdMap =
       std::unordered_map<ExtensionId,
                          std::unique_ptr<internal::IndividualSettings>>;
