diff --git a/chrome/common/chrome_content_client.cc b/chrome/common/chrome_content_client.cc
index 5df291a12226596061c7f461fb2a7e2b2a2f2d6c..a7818b73161489966df14e2397d1dab18b7c0cce 100644
--- a/chrome/common/chrome_content_client.cc
+++ b/chrome/common/chrome_content_client.cc
@@ -122,6 +122,7 @@ void ChromeContentClient::SetGpuInfo(const gpu::GPUInfo& gpu_info) {
 
 void ChromeContentClient::AddPlugins(
     std::vector<content::WebPluginInfo>* plugins) {
+  [&]() -> void {
 #if BUILDFLAG(ENABLE_PDF)
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
   static constexpr char16_t kPDFPluginName[] = u"Chrome PDF Plugin";
@@ -144,6 +145,8 @@ void ChromeContentClient::AddPlugins(
   pdf_info.type = content::WebPluginInfo::PLUGIN_TYPE_BROWSER_INTERNAL_PLUGIN;
   plugins->push_back(pdf_info);
 #endif  // BUILDFLAG(ENABLE_PDF)
+  }();
+  RenameChromiumPdfPluginToChrome(plugins);
 }
 
 void ChromeContentClient::AddContentDecryptionModules(
