diff --git a/chrome/android/java/src/org/chromium/chrome/browser/compositor/layouts/phone/NewTabAnimationLayout.java b/chrome/android/java/src/org/chromium/chrome/browser/compositor/layouts/phone/NewTabAnimationLayout.java
index 9433bb6de83f05fe8c9b269a43119418a325578b..37f981bd9f86da89cbef7c89b285bbdf307e6f20 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/compositor/layouts/phone/NewTabAnimationLayout.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/compositor/layouts/phone/NewTabAnimationLayout.java
@@ -700,6 +700,7 @@ public class NewTabAnimationLayout extends Layout {
                 isRegularNtp || ToolbarPositionController.shouldShowToolbarOnTop(animationTab);
         int toolbarHeight = toolbarPosition[1] + getTopInsetIfNeeded(animationTab);
 
+        BraveNewTabAnimationLayout.BraveTabSwitcherState braveState = BraveNewTabAnimationLayout.compute(mAnimationHostView, tabSwitcherRect, tabSwitcherButtonIsVisible, toolbarHeight, isTopToolbar); tabSwitcherRect = braveState.rect; tabSwitcherButtonIsVisible = braveState.isVisible; toolbarHeight = braveState.toolbarHeight; isTopToolbar = braveState.isTopToolbar;
         Rect compositorViewRect = new Rect();
         mCompositorViewHolder.getGlobalVisibleRect(compositorViewRect);
 
