diff --git a/chrome/browser/bookmarks/android/BUILD.gn b/chrome/browser/bookmarks/android/BUILD.gn
index 56e7c0862cbc4be395a496823891bfa68623bcdc..ccb569d6f127779dace685074ba7baf61566752a 100644
--- a/chrome/browser/bookmarks/android/BUILD.gn
+++ b/chrome/browser/bookmarks/android/BUILD.gn
@@ -36,6 +36,7 @@ source_set("impl") {
     "//components/query_parser",
     "//components/undo",
   ]
+  deps += brave_browser_bookmarks_android_impl_deps
 }
 
 android_resources("java_resources") {
@@ -103,6 +104,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/bookmarks/bar/BookmarkBarViewBinder.java",
     "java/src/org/chromium/chrome/browser/bookmarks/bar/BookmarkBarVisibilityProvider.java",
   ]
+  sources += brave_browser_bookmarks_android_sources
 
   deps = [
     ":java_resources",
