diff --git a/chrome/browser/resources/tools/build_webui.gni b/chrome/browser/resources/tools/build_webui.gni
index 568fd447935225efa177075545638acb35771aae..9345f9b470cb2e212f23dc76c607b2ae1a0fbb76 100644
--- a/chrome/browser/resources/tools/build_webui.gni
+++ b/chrome/browser/resources/tools/build_webui.gni
@@ -132,6 +132,7 @@ template("build_webui") {
     if (defined(invoker.css_files)) {
       in_files += invoker.css_files
     }
+    if (defined(invoker.exclude_preprocess_files)) { in_files -= invoker.exclude_preprocess_files } if (defined(invoker.preprocess_deps)) { public_deps = invoker.preprocess_deps }
   }
 
   if (defined(html_files)) {
