diff --git a/chrome/browser/password_manager/password_change_delegate_impl.cc b/chrome/browser/password_manager/password_change_delegate_impl.cc
index 8f4480cd29995c069c42049f514df396052901ca..6cd7f35cb21211447831fa51d5a0ba3260d48c9c 100644
--- a/chrome/browser/password_manager/password_change_delegate_impl.cc
+++ b/chrome/browser/password_manager/password_change_delegate_impl.cc
@@ -146,6 +146,7 @@ std::unique_ptr<BrowserSavePasswordProgressLogger> GetLoggerIfAvailable(
 }
 
 std::unique_ptr<content::WebContents> CreateWebContents(Profile* profile,
+                                                        base::raw_ptr<content::WebContents> originator,
                                                         const GURL& url) {
   scoped_refptr<content::SiteInstance> initial_site_instance_for_new_contents =
       tab_util::GetSiteInstanceForNewTab(profile, url);
@@ -296,7 +297,7 @@ void PasswordChangeDelegateImpl::StartPasswordChangeFlow() {
   LogPasswordSavedOnStart(originator_);
   UpdateState(State::kWaitingForChangePasswordForm);
 
-  executor_ = CreateWebContents(profile_, change_password_url_);
+  executor_ = CreateWebContents(profile_, originator_, change_password_url_);
   CHECK(executor_);
 
   auto* client = ChromePasswordManagerClient::FromWebContents(executor_.get());
