diff --git a/chrome/common/extensions/api/developer_private.idl b/chrome/common/extensions/api/developer_private.idl
index d3cc055d5780a31dac9c0bfa17937fc1548884d9..fcdaeee9e4c4ea442d6c0df66415c74383c4b043 100644
--- a/chrome/common/extensions/api/developer_private.idl
+++ b/chrome/common/extensions/api/developer_private.idl
@@ -231,6 +231,7 @@ namespace developerPrivate {
     DOMString iconUrl;
     DOMString id;
     AccessModifier incognitoAccess;
+    boolean isSplitMode;
     DOMString[] installWarnings;
     DOMString? launchUrl;
     Location location;
