diff --git a/chrome/browser/ui/browser_window/BUILD.gn b/chrome/browser/ui/browser_window/BUILD.gn
index 4d1833b1b03e2c7e235c56fd0d3facb5372ae7f4..e73c2944b0e0e0ee4596a127a8e5db71442c76ce 100644
--- a/chrome/browser/ui/browser_window/BUILD.gn
+++ b/chrome/browser/ui/browser_window/BUILD.gn
@@ -34,6 +34,7 @@ source_set("browser_window") {
       "//ui/base:types",
     ]
   }
+  deps += [ "//brave/browser/ui/browser_window" ]
 }
 
 source_set("create_browser_window") {
