diff --git a/chrome/browser/ui/browser_commands.cc b/chrome/browser/ui/browser_commands.cc
index 43e2b81f9903ff4ea44c9fa9307b628071bdd1c3..f88cd1245c8b55f8217264e2cc9cb29da5f6c005 100644
--- a/chrome/browser/ui/browser_commands.cc
+++ b/chrome/browser/ui/browser_commands.cc
@@ -1187,6 +1187,7 @@ void CloseTab(BrowserWindowInterface* browser) {
   }
 
   ToastController* toast_controller = browser->GetFeatures().toast_controller();
+  BRAVE_CLOSE_TAB
   if (!toast_controller) {
     CloseSelectedTabAndRecordTabCountMetric(browser);
     return;
