diff --git a/chrome/browser/ui/page_action/page_action_model.h b/chrome/browser/ui/page_action/page_action_model.h
index bea88558bf05a7ebbd6c3bc13ef3dedcdd5f460a..5df133150275c30339d5805797ec9054960afd07 100644
--- a/chrome/browser/ui/page_action/page_action_model.h
+++ b/chrome/browser/ui/page_action/page_action_model.h
@@ -123,9 +123,28 @@ class PageActionModelInterface {
   virtual PageActionColorSource GetColorSource() const = 0;
 
   virtual bool IsEphemeral() const = 0;
+  virtual void SetOverrideChipColors(
+      PageActionPassKey,
+      std::optional<SkColor> override_background_color,
+      std::optional<SkColor> override_foreground_color) = 0;
+  virtual void SetAlwaysShowLabel(PageActionPassKey,
+                                  bool always_show) = 0;
+  virtual void SetOverrideHeight(PageActionPassKey,
+                                 std::optional<int> height) = 0;
+  virtual void SetOverrideTriggerableEvent(PageActionPassKey,
+                                           std::optional<int> event_flags) = 0;
+  virtual void SetOverrideBorder(PageActionPassKey,
+                                 std::optional<gfx::Insets> border) = 0;
+  virtual std::optional<SkColor> GetOverrideBackgroundColor() const = 0;
+  virtual std::optional<SkColor> GetOverrideForegroundColor() const = 0;
+  virtual bool GetAlwaysShowLabel() const = 0;
+  virtual std::optional<int> GetOverrideHeight() const = 0;
+  virtual std::optional<int> GetOverrideTriggerableEvent() const = 0;
+  virtual std::optional<gfx::Insets> GetOverrideBorder() const = 0;
 };
 
 // PageActionModel represents the page action's state, scoped to a single tab.
+namespace chromium_impl {
 class PageActionModel : public PageActionModelInterface {
  public:
   explicit PageActionModel(actions::ActionId action_id,
@@ -229,6 +248,7 @@ class PageActionModel : public PageActionModelInterface {
   bool IsEphemeral() const override;
 
  private:
+  friend class ::page_actions::PageActionModel;
   // Identifies which property triggered a NotifyChange call, used for
   // per-property reentrancy checks.
   enum class Property {
@@ -252,7 +272,12 @@ class PageActionModel : public PageActionModelInterface {
     kIsAnchoredMessageShowing,
     kAnchoredMessageIcon,
     kAnchoredMessageExpandableContent,
-    kMaxValue = kAnchoredMessageExpandableContent,
+    kAlwaysShowLabel,
+    kOverrideChipColors,
+    kOverrideHeight,
+    kOverrideTriggerableEvent,
+    kOverrideBorder,
+    kMaxValue = kOverrideBorder,
   };
   using PropertySet =
       base::EnumSet<Property, Property::kShowRequested, Property::kMaxValue>;
@@ -356,6 +381,7 @@ class PageActionModel : public PageActionModelInterface {
 
   base::ObserverList<PageActionModelObserver> observer_list_;
 };
+}  // namespace chromium_impl
 
 class PageActionModelFactory {
  public:
