diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 4eecf4c09f42ea90a74b2b00329f17711c421fa6..e43af101f72668a08409e4f498180faed1268fa0 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3272,6 +3272,7 @@ const FeatureEntry::FeatureVariation kDesktopSharePreviewVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
