diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index e48bf445f05234ab9f8e1d6498ceec220b11baab..d83f2262f7dafe360ebfc28a351876f7f0747212 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -871,6 +871,7 @@ void BookmarkBarView::Layout(PassKey) {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   const bool show_bookmarks_overflow =
       bookmark_service_->loaded() &&
