diff --git a/chrome/browser/sessions/session_service.cc b/chrome/browser/sessions/session_service.cc
index 374769c05f505d7b5c3341c2a0db39b7192fb639..ceb013f8008d3e36ce4793e987e66f240fa52482 100644
--- a/chrome/browser/sessions/session_service.cc
+++ b/chrome/browser/sessions/session_service.cc
@@ -383,6 +383,7 @@ void SessionService::TabClosed(SessionID window_id, SessionID tab_id) {
               window_id) == window_closing_ids_.end()) &&
         IsOnlyOneTabLeft()) {
       // This is the last tab in the last tabbed browser.
+      BRAVE_SESSION_SERVICE_TAB_CLOSED
       has_open_trackable_browsers_ = false;
     }
   }
