diff --git a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/DoNotTrackSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/DoNotTrackSettings.java
index 842f3291586b323ba845d67638d0ae130b1eca30..f7679b32e4cebcd0fa0abc46615af37916980cc3 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/DoNotTrackSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/DoNotTrackSettings.java
@@ -19,7 +19,7 @@ import org.chromium.components.user_prefs.UserPrefs;
 /**
  * Fragment to manage 'Do Not Track' preference and to explain to the user what it does.
  */
-public class DoNotTrackSettings extends PreferenceFragmentCompat {
+public class DoNotTrackSettings extends org.chromium.chrome.browser.settings.BravePreferenceFragment {
     // Must match key in do_not_track_preferences.xml.
     private static final String PREF_DO_NOT_TRACK_SWITCH = "do_not_track_switch";
 
