diff --git a/chrome/installer/gcapi/gcapi.cc b/chrome/installer/gcapi/gcapi.cc
index d115d6f43abbaee4b4142591d77a233f294fba58..82346778d92143a6a452987aa859da1cf5ef7117 100644
--- a/chrome/installer/gcapi/gcapi.cc
+++ b/chrome/installer/gcapi/gcapi.cc
@@ -56,6 +56,16 @@ using Microsoft::WRL::ComPtr;
 
 namespace {
 
+#if defined(OFFICIAL_BUILD)
+const wchar_t kGCAPITempKey[] = L"Software\\BraveSoftware\\GCAPITemp";
+const wchar_t kChromeRegVersion[] = L"pv";
+const wchar_t kNoChromeOfferUntil[] =
+    L"SOFTWARE\\BraveSoftware\\No Chrome Offer Until";
+const wchar_t kC1FPendingKey[] =
+    L"Software\\BraveSoftware\\Common\\Rlz\\Events\\C";
+const wchar_t kC1FSentKey[] =
+    L"Software\\BraveSoftware\\Common\\Rlz\\StatefulEvents\\C";
+#else
 const wchar_t kGCAPITempKey[] = L"Software\\Google\\GCAPITemp";
 
 const wchar_t kChromeRegVersion[] = L"pv";
@@ -65,6 +75,7 @@ const wchar_t kNoChromeOfferUntil[] =
 const wchar_t kC1FPendingKey[] = L"Software\\Google\\Common\\Rlz\\Events\\C";
 const wchar_t kC1FSentKey[] =
     L"Software\\Google\\Common\\Rlz\\StatefulEvents\\C";
+#endif
 const wchar_t kC1FKey[] = L"C1F";
 
 const wchar_t kRelaunchBrandcodeValue[] = L"RelaunchBrandcode";
