diff --git a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java
index 832da6703671206ee8f2ac5366306915ef25ffb4..f3613467d498bd342248d379b6e2fe47912878cb 100644
--- a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java
+++ b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java
@@ -435,6 +435,9 @@ public class FuseboxCoordinator implements TemplateUrlServiceObserver {
     // TemplateUrlServiceObserver
     @Override
     public void onTemplateURLServiceChanged() {
+        if (!mTemplateUrlService.hasNativeService()) {
+            return;
+        }
         boolean isDseGoogle = mTemplateUrlService.isDefaultSearchEngineGoogle();
         if (isDseGoogle == mDefaultSearchEngineIsGoogle) return;
         mDefaultSearchEngineIsGoogle = isDseGoogle;
