diff --git a/chrome/browser/resources/extensions/extensions.ts b/chrome/browser/resources/extensions/extensions.ts
index 67185bb055f409023dbf3b33563a54313cd81766..1e1fe04f3049a347b2a290e5455baeaa442c697f 100644
--- a/chrome/browser/resources/extensions/extensions.ts
+++ b/chrome/browser/resources/extensions/extensions.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './manager.js';
 
 export {getToastManager} from 'chrome://resources/cr_elements/cr_toast/cr_toast_manager.js';
