diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index eae7b4cef4dc9e633c975b63fcf89ebe59dffff6..f9ed57e409391138f51922353a3e317884f25656 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -781,6 +781,7 @@ static_library("ui") {
     # gets componentized.
     "//chrome/browser/ui/extensions:impl",
   ]
+  import("//brave/browser/sources.gni") import("//brave/components/brave_wallet/sources.gni") public_deps += [ "//brave/browser/ui" ] deps += brave_chrome_browser_ui_allow_circular_includes_from + brave_chrome_browser_ui_deps allow_circular_includes_from += brave_chrome_browser_ui_allow_circular_includes_from
 
   if (enable_vr && is_win) {
     deps += [ "//chrome/browser/vr:vr_base" ]
@@ -3342,7 +3343,10 @@ static_library("ui") {
         "//chrome/updater/app/server/win:updater_legacy_idl",
       ]
     } else {
-      sources += [ "webui/help/version_updater_basic.cc" ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [ "webui/help/version_updater_basic.cc" ]
+      }
     }
   }
 
@@ -5478,6 +5482,7 @@ static_library("ui_public_dependencies") {
     "//components/enterprise/buildflags",
     "//components/paint_preview/buildflags",
     "//components/safe_browsing:buildflags",
+    "//components/saved_tab_groups/public",
     "//components/segmentation_platform/public",
     "//components/sync",
     "//components/sync_user_events",
@@ -5489,6 +5494,7 @@ static_library("ui_public_dependencies") {
   if (enable_extensions_core) {
     public_deps += [ "//extensions/browser" ]
   }
+  public_deps += [ "//brave/browser/ui:ui_public_dependencies" ]
   if (!is_android) {
     public_deps += [
       "//build:branding_buildflags",
@@ -5612,6 +5618,7 @@ source_set("browser_navigator_params_headers") {
     "//url",
   ]
   deps = []
+  public_deps += [ "//brave/browser/ui:browser_navigator_params_headers_public_dependencies" ]
 }
 
 if (!is_android) {
