diff --git a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
index 3544f1b569ee1629933deb89657d25278d754de6..80d222844d645c38219bf3648003a7470682f12d 100644
--- a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
+++ b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
@@ -826,6 +826,7 @@ SkPath IconLabelBubbleView::GetHighlightPath() const {
                                 {radii.lower_right(), radii.lower_right()},
                                 {radii.lower_left(), radii.lower_left()}};
 
+  BRAVE_ICON_LABEL_BUBBLE_VIEW_GET_HIGHLIGHT_PATH
   return SkPath::RRect(SkRRect::MakeRectRadii(rect, sk_radii));
 }
 
