diff --git a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
index 7c991f9d6181ae649b1dc92d2b4466bba37ffc22..0161bd8ad9f94cabfe70f16b88d07656820ac4bd 100644
--- a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
+++ b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
@@ -160,7 +160,7 @@ std::unique_ptr<TabStrip> CreateTabStrip(
   std::unique_ptr<TabHoverCardController> hover_card_controller(
       std::make_unique<TabHoverCardController>(tab_strip_region_view,
                                                browser_view->browser()));
-  auto tab_strip = std::make_unique<TabStrip>(std::move(tabstrip_controller),
+  auto tab_strip = std::make_unique<BraveTabStrip>(std::move(tabstrip_controller),
                                               std::move(hover_card_controller));
   return tab_strip;
 }
