diff --git a/chrome/browser/autocomplete/autocomplete_classifier_factory.cc b/chrome/browser/autocomplete/autocomplete_classifier_factory.cc
index c106fe59a239c026852617d842f2f2c1a789cb01..34b0207a815fdf71530b48a2a8664afd88ebcb00 100644
--- a/chrome/browser/autocomplete/autocomplete_classifier_factory.cc
+++ b/chrome/browser/autocomplete/autocomplete_classifier_factory.cc
@@ -68,7 +68,12 @@ AutocompleteClassifierFactory::~AutocompleteClassifierFactory() {
 
 content::BrowserContext* AutocompleteClassifierFactory::GetBrowserContextToUse(
     content::BrowserContext* context) const {
+#if defined(BRAVE_CHROMIUM_BUILD)
+  // See BraveAutocompleteProviderClient about why separate services are used.
+  return chrome::GetBrowserContextOwnInstanceInIncognito(context);
+#else
   return chrome::GetBrowserContextRedirectedInIncognito(context);
+#endif
 }
 
 bool AutocompleteClassifierFactory::ServiceIsNULLWhileTesting() const {
