diff --git a/chrome/browser/ui/views/frame/browser_view.h b/chrome/browser/ui/views/frame/browser_view.h
index 11982628e6f3436c299277e4108a6d8613564773..2b54c13d405325ec94441e8a2858633a236a2783 100644
--- a/chrome/browser/ui/views/frame/browser_view.h
+++ b/chrome/browser/ui/views/frame/browser_view.h
@@ -734,6 +734,7 @@ class BrowserView : public BrowserWindow,
       const std::map<std::string, std::string>& extra_data) override;
 #endif
 
+  BRAVE_BROWSER_VIEW_H
  private:
   // Do not friend BrowserViewLayout. Use the BrowserViewLayoutDelegate
   // interface to keep these two classes decoupled and testable.
