diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 8d577144a67e44e95ec2dcdb356fb85ec9f005d0..c94fb59ba44e02ce2514cba20d7b2af1b9b11664 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3462,6 +3462,7 @@ const FeatureEntry::FeatureVariation kVcSegmentationModelVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
