diff --git a/chrome/browser/browsing_data/BUILD.gn b/chrome/browser/browsing_data/BUILD.gn
index 90b8b57f96d0e98966d2b66e843c1e49f2fe1a22..8cf5e0d35cad0fd3e8d4b4ee21c0d2c4c1639165 100644
--- a/chrome/browser/browsing_data/BUILD.gn
+++ b/chrome/browser/browsing_data/BUILD.gn
@@ -12,6 +12,7 @@ source_set("constants") {
     "//content/public/browser",
     "//extensions/buildflags",
   ]
+  public_deps = [ "//brave/components/ai_chat/core/common/buildflags" ]
 }
 
 source_set("browsing_data") {
