diff --git a/chrome/browser/extensions/api/developer_private/extension_info_generator.cc b/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
index d4ef0e322ba66bede49ebc4ece75bcd39ea71928..8d83ccd36c110593724313fd2637f402e7fb2104 100644
--- a/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
+++ b/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
@@ -565,7 +565,7 @@ void ExtensionInfoGenerator::FillExtensionInfo(const Extension& extension,
                                                   extension_prefs_);
   switch (blocklist_state) {
     case BitMapBlocklistState::BLOCKLISTED_MALWARE:
-      blocklist_text = IDS_EXTENSIONS_BLOCKLISTED_MALWARE;
+      blocklist_text = BraveGetMalwareBlocklistText(extension.id());
       break;
     case BitMapBlocklistState::BLOCKLISTED_SECURITY_VULNERABILITY:
       blocklist_text = IDS_EXTENSIONS_BLOCKLISTED_SECURITY_VULNERABILITY;
@@ -748,6 +748,7 @@ void ExtensionInfoGenerator::FillExtensionInfo(const Extension& extension,
 #else
   info.incognito_access_pending_change = false;
 #endif
+  info.is_split_mode = IncognitoInfo::IsSplitMode(&extension);
 
   // User Scripts toggle.
   info.user_scripts_access.is_enabled =
@@ -902,6 +903,7 @@ void ExtensionInfoGenerator::FillExtensionInfo(const Extension& extension,
   CHECK(mv2_handler);
   info.is_affected_by_mv2_deprecation =
       mv2_handler->IsExtensionAffected(extension);
+  ProcessKnownMV2Extensions(info);
   if (info.web_store_url.length() > 0) {
     info.recommendations_url =
         extension_urls::GetNewWebstoreItemRecommendationsUrl(extension.id())
