diff --git a/chrome/browser/download/internal/android/BUILD.gn b/chrome/browser/download/internal/android/BUILD.gn
index b4f0f0ab5c61c0036191c189cac7ad3cfc6ebadb..4601cbf442a88571a59c2985cc77293b0d144955 100644
--- a/chrome/browser/download/internal/android/BUILD.gn
+++ b/chrome/browser/download/internal/android/BUILD.gn
@@ -222,6 +222,7 @@ android_resources("java_resources") {
     "//content/public/android:content_java_resources",
     "//ui/android:ui_java_resources",
   ]
+  deps += [ "//brave/android:brave_download_internal_java_resources" ]
 }
 
 android_library("javatests") {
