diff --git a/chrome/browser/resources/optimize_webui.gni b/chrome/browser/resources/optimize_webui.gni
index 9427d71b23d6cfb09fd43a7afb0284b6b63dfb4f..845c8d56a331153bc77b09f3bdc11f1968b85252 100644
--- a/chrome/browser/resources/optimize_webui.gni
+++ b/chrome/browser/resources/optimize_webui.gni
@@ -45,6 +45,7 @@ template("optimize_webui") {
     }
 
     deps = invoker.deps
+    deps += [ "//brave/ui/webui/resources:modulize", ]
 
     # Note that we have to manually pass the sources to our script if the
     # script needs them as inputs.
