diff --git a/chrome/browser/resources/password_manager/password_manager.ts b/chrome/browser/resources/password_manager/password_manager.ts
index 7f500bb17a599eba994e9d8f76c1bf54afca3427..01d394423d3a96214dc8b02857a92bcf07b9bd1c 100644
--- a/chrome/browser/resources/password_manager/password_manager.ts
+++ b/chrome/browser/resources/password_manager/password_manager.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './password_manager_app.js';
 
 export {SettingsPrefsElement} from 'chrome://resources/cr_components/settings_prefs/prefs.js';
