diff --git a/chrome/browser/net/proxy_config_monitor.cc b/chrome/browser/net/proxy_config_monitor.cc
index 34584b44002762c875b68eb8547f651b8d7d7a0e..af52519bcf741c1a6892b8cad04b9630866127d5 100644
--- a/chrome/browser/net/proxy_config_monitor.cc
+++ b/chrome/browser/net/proxy_config_monitor.cc
@@ -52,6 +52,7 @@ ProxyConfigMonitor::ProxyConfigMonitor(Profile* profile) {
             profile->GetPrefs(), g_browser_process->local_state());
   }
 
+  BRAVE_PROXY_CONFIG_MONITOR
   proxy_config_service_ = ProxyServiceFactory::CreateProxyConfigService(
       pref_proxy_config_tracker_.get(), profile);
 
