diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index b125cb113fddae080600bf8535c1c72077143223..14ec533d6c34ec67b9e17809398316a59907dcc4 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -96,6 +96,7 @@ source_set("channel_info") {
   } else if (is_posix) {
     sources += [ "channel_info_posix.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -273,6 +274,7 @@ static_library("common_lib") {
     public_deps += [ "//components/cdm/common" ]
   }
 
+  import("//brave/common/sources.gni") sources += brave_chrome_common_sources deps += brave_chrome_common_deps public_deps += brave_chrome_common_public_deps
   if (enable_extensions_core) {
     public_deps += [
       "//chrome/common/extensions",
@@ -469,6 +471,7 @@ static_library("non_code_constants") {
   if (is_win) {
     sources += [ "chrome_icon_resources_win.h" ]
   }
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
