diff --git a/chrome/browser/extensions/favicon/favicon_util.cc b/chrome/browser/extensions/favicon/favicon_util.cc
index 4c59a428e7d6715a1d4c534526370478919afdd4..777fa494ef6e391ac09be3f466cc2c3402ae21e6 100644
--- a/chrome/browser/extensions/favicon/favicon_util.cc
+++ b/chrome/browser/extensions/favicon/favicon_util.cc
@@ -37,10 +37,12 @@ int GetResourceID(int size_in_pixels) {
       ui::NativeTheme::GetInstanceForNativeUi()->preferred_color_scheme() ==
       ui::NativeTheme::PreferredColorScheme::kDark;
   if (size_in_pixels >= 64) {
-    return is_dark ? IDR_DEFAULT_FAVICON_DARK_64 : IDR_DEFAULT_FAVICON_64;
+    return is_dark ? BRAVE_IDR_DEFAULT_FAVICON_DARK_64
+                   : BRAVE_IDR_DEFAULT_FAVICON_64;
   }
   if (size_in_pixels >= 32) {
-    return is_dark ? IDR_DEFAULT_FAVICON_DARK_32 : IDR_DEFAULT_FAVICON_32;
+    return is_dark ? BRAVE_IDR_DEFAULT_FAVICON_DARK_32
+                   : BRAVE_IDR_DEFAULT_FAVICON_32;
   }
   return is_dark ? IDR_DEFAULT_FAVICON_DARK : IDR_DEFAULT_FAVICON;
 }
