diff --git a/chrome/browser/apps/platform_apps/BUILD.gn b/chrome/browser/apps/platform_apps/BUILD.gn
index 00bf7253317c..2b4c3cd06947 100644
--- a/chrome/browser/apps/platform_apps/BUILD.gn
+++ b/chrome/browser/apps/platform_apps/BUILD.gn
@@ -65,6 +65,7 @@ source_set("platform_apps") {
     "//services/preferences/public/cpp:cpp",
     "//ui/gfx",
   ]
+  if (is_android) { deps -= ["//chrome/browser/media/router/discovery",] }
 
   if (is_mac) {
     deps += [ "//chrome/browser/apps/app_shim" ]
