diff --git a/chrome/browser/prefs/browser_prefs.cc b/chrome/browser/prefs/browser_prefs.cc
index 1317d71ce1812deaf77440e812cdfeb2a963fa65..80bf6aa4e9a96c348329ee6512bdb4e43c36cf8f 100644
--- a/chrome/browser/prefs/browser_prefs.cc
+++ b/chrome/browser/prefs/browser_prefs.cc
@@ -1730,6 +1730,7 @@ void RegisterLocalState(PrefRegistrySimple* registry) {
       false);
 
   // This is intentionally last.
+  brave::RegisterLocalStatePrefs(registry);
   RegisterLocalStatePrefsForMigration(registry);
 }
 
@@ -2259,6 +2260,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry,
 #endif  // BUILDFLAG(IS_ANDROID)
 
   registry->RegisterBooleanPref(prefs::kStaticStorageQuotaEnabled, false);
+  brave::RegisterProfilePrefs(registry);
 }
 
 void RegisterUserProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -2558,6 +2560,7 @@ void MigrateObsoleteProfilePrefs(PrefService* profile_prefs,
   profile_prefs->ClearPref(kObsoletePasswordsUseUPMLocalAndSeparateStores);
   profile_prefs->ClearPref(kObsoleteEmptyProfileStoreLoginDatabase);
   profile_prefs->ClearPref(kObsoleteUpmAutoExportCsvNeedsDeletion);
+#if 0
   base::DeleteFile(profile_path.Append(FILE_PATH_LITERAL("Login Data")));
   base::DeleteFile(
       profile_path.Append(FILE_PATH_LITERAL("Login Data For Account")));
@@ -2565,6 +2568,7 @@ void MigrateObsoleteProfilePrefs(PrefService* profile_prefs,
       profile_path.Append(FILE_PATH_LITERAL("Login Data-journal")));
   base::DeleteFile(
       profile_path.Append(FILE_PATH_LITERAL("Login Data For Account-journal")));
+#endif
 #endif  // BUILDFLAG(IS_ANDROID)
 
   // Added 09/2025.
