diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/about/AboutChromeSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/about/AboutChromeSettings.java
index fb84c0ca78431cc5339cd51092f61c959b1c6838..d9ee30ca43aeffe9b756458ec446af58a55ee5d0 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/about/AboutChromeSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/about/AboutChromeSettings.java
@@ -24,7 +24,7 @@ import java.util.Calendar;
  * Settings fragment that displays information about Chrome.
  */
 public class AboutChromeSettings
-        extends PreferenceFragmentCompat implements Preference.OnPreferenceClickListener {
+        extends org.chromium.chrome.browser.settings.BravePreferenceFragment implements Preference.OnPreferenceClickListener {
     private static final int TAPS_FOR_DEVELOPER_SETTINGS = 7;
 
     private static final String PREF_APPLICATION_VERSION = "application_version";
