diff --git a/chrome/browser/ui/passwords/settings/password_manager_presenter.cc b/chrome/browser/ui/passwords/settings/password_manager_presenter.cc
index c06184bc1cac56aec339b24e6307ce1ed704467a..0985fdb84c188711028f457f378db2f7b41f79bc 100644
--- a/chrome/browser/ui/passwords/settings/password_manager_presenter.cc
+++ b/chrome/browser/ui/passwords/settings/password_manager_presenter.cc
@@ -44,9 +44,9 @@
 #include "components/undo/undo_operation.h"
 #include "content/public/browser/browser_thread.h"
 
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
 #include "chrome/browser/extensions/api/passwords_private/passwords_private_utils.h"
-#endif
+//#endif
 
 using base::StringPiece;
 using password_manager::PasswordStore;
@@ -342,7 +342,7 @@ void PasswordManagerPresenter::UndoRemoveSavedPasswordOrException() {
   undo_manager_.Undo();
 }
 
-#if !defined(OS_ANDROID)  // This is never called on Android.
+//#if !defined(OS_ANDROID)  // This is never called on Android.
 void PasswordManagerPresenter::RequestShowPassword(
     const std::string& sort_key,
     base::OnceCallback<void(base::Optional<base::string16>)> callback) const {
@@ -374,7 +374,7 @@ void PasswordManagerPresenter::RequestShowPassword(
       password_manager::metrics_util::ACCESS_PASSWORD_VIEWED,
       password_manager::metrics_util::ACCESS_PASSWORD_COUNT);
 }
-#endif
+//#endif
 
 void PasswordManagerPresenter::AddLogin(const autofill::PasswordForm& form) {
   PasswordStore* store =
