diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index 039035dce20878b29523ad7ebce2bf189eadfa0c..85bded6b295f7f523b4084c2af365be1e1ca117b 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -127,6 +127,7 @@ bool Profile::OTRProfileID::AllowsBrowserWindows() const {
   // Non-Primary OTR profiles are not supposed to create Browser windows.
   // DevTools::BrowserContext is an exception to this ban.
   return *this == PrimaryID() ||
+         BRAVE_ALLOWS_BROWSER_WINDOWS
          base::StartsWith(profile_id_, kDevToolsOTRProfileIDPrefix,
                           base::CompareCase::SENSITIVE);
 }
@@ -277,7 +278,7 @@ const char Profile::kProfileKey[] = "__PROFILE__";
 void Profile::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
       prefs::kSearchSuggestEnabled,
-      true,
+      false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 #if defined(OS_ANDROID)
   registry->RegisterStringPref(
