diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 71fa7739e26cfaf129ef3f04a1d7dda658ce8c5a..8adea906da5dbd1c42d29a66140ce288148f0cb8 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -900,6 +900,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
