diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 2f12bfd6d0c057aa28a95a043aa229d270a53dd7..37fe42b15123b0acea66e4cb8dec6454fd2ebd76 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -689,6 +689,7 @@ class Browser : public TabStripModelObserver,
 
   StatusBubble* GetStatusBubbleForTesting();
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
