diff --git a/chrome/browser/ui/views/permissions/chip_controller.cc b/chrome/browser/ui/views/permissions/chip_controller.cc
index e5eeb0e09089c14c839e3e57cf144f0a5c9403aa..eeb6418c5603e35a20fd0c3bd31c3194eb3a8fca 100644
--- a/chrome/browser/ui/views/permissions/chip_controller.cc
+++ b/chrome/browser/ui/views/permissions/chip_controller.cc
@@ -397,6 +397,7 @@ void ChipController::CollapsePrompt(bool allow_restart) {
 void ChipController::HideChip() {
   chip_->SetVisible(false);
   // When the chip is hidden the locationbar layout should be updated.
+  BRAVE_CHIP_CONTROLLER_HIDE_CHIP
   GetLocationBarView()->InvalidateLayout();
 }
 
