diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
index 78a127358e8b5ad409efc6857f2498a0e22a9dcf..686dbee01633f8455b6fe7512eacb9bd2d4ca9bf 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
@@ -864,6 +864,7 @@ public class ChromeTabbedActivity extends ChromeActivity {
                         private void closeIfNoTabsAndHomepageEnabled(
                                 boolean isPendingClosure, boolean shouldRemoveWindowWithZeroTabs) {
                             if (getTabModelSelector().getTotalTabCount() == 0) {
+                                shouldRemoveWindowWithZeroTabs = shouldRemoveWindowWithZeroTabs && org.chromium.chrome.browser.partnercustomizations.CloseBraveManager.shouldCloseAppWithZeroTabs();
                                 if (shouldRemoveWindowWithZeroTabs) {
                                     finishAndRemoveTask();
                                 } else if (HomepageManager.getInstance()
