diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index 354072b39cccfe4b343915a4bb434e1714136d97..a680fa310e9118d16295509a333620043abcd02d 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -428,6 +428,7 @@ Browser::CreationStatus Browser::GetCreationStatusForProfile(Profile* profile) {
 // static
 Browser* Browser::Create(const CreateParams& params) {
   CHECK_EQ(CreationStatus::kOk, GetCreationStatusForProfile(params.profile));
+  BRAVE_BROWSER_CREATE
   return new Browser(params);
 }
 
@@ -439,7 +440,7 @@ Browser::Browser(const CreateParams& params)
       tab_strip_model_delegate_(
           std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
       tab_strip_model_(
-          std::make_unique<TabStripModel>(tab_strip_model_delegate_.get(),
+          std::make_unique<BraveTabStripModel>(tab_strip_model_delegate_.get(),
                                           params.profile)),
       app_name_(params.app_name),
       is_trusted_source_(params.trusted_source),
