diff --git a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
index 6bd71ecf31afe27120fb5122f6960ae2c9c95f50..5de2012e91ceb4efde5573f04f02746282d8e243 100644
--- a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
+++ b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
@@ -129,6 +129,7 @@ content::WebUIDataSource* CreateBookmarksUIHTMLSource(Profile* profile) {
   for (const auto& str : kStrings)
     AddLocalizedString(source, str.name, str.id);
 
+  BRAVE_CREATE_BOOKMARKS_UI_HTML_SOURCE
   return source;
 }
 
