diff --git a/chrome/browser/resources/settings/about_page/about_page.ts b/chrome/browser/resources/settings/about_page/about_page.ts
index a5dcbb3ee70a59be9e4d269e62f88c0cf2d16160..d54d233e4ad0c4a92d845d87e2c59b8c816e7347 100644
--- a/chrome/browser/resources/settings/about_page/about_page.ts
+++ b/chrome/browser/resources/settings/about_page/about_page.ts
@@ -190,7 +190,14 @@ export class SettingsAboutPageElement extends SettingsAboutPageElementBase {
   }
 
   private onRelaunchTap_() {
+    // <if expr="is_macosx">
+    // Sparkle framework's relaunch api is used.
+    LifetimeBrowserProxyImpl.getInstance().relaunchOnMac();
+    // </if>
+
+    // <if expr="not is_macosx">
     LifetimeBrowserProxyImpl.getInstance().relaunch();
+    // </if>
   }
 
   // <if expr="not chromeos">
