diff --git a/chrome/browser/sync/chrome_sync_client.cc b/chrome/browser/sync/chrome_sync_client.cc
index 9a6cbd434fe4fa0c30b7632f100b481375287873..bcb8ce82b94e3578316beae7d00ebb6e9c366295 100644
--- a/chrome/browser/sync/chrome_sync_client.cc
+++ b/chrome/browser/sync/chrome_sync_client.cc
@@ -12,6 +12,8 @@
 #include "base/feature_list.h"
 #include "base/path_service.h"
 #include "base/syslog_logging.h"
+#include "brave/browser/brave_rewards/vg_sync_service.h"
+#include "brave/browser/brave_rewards/vg_sync_service_factory.h"
 #include "build/build_config.h"
 #include "build/chromeos_buildflags.h"
 #include "chrome/browser/bookmarks/bookmark_model_factory.h"
@@ -670,6 +672,12 @@ ChromeSyncClient::GetControllerDelegateForModelType(syncer::ModelType type) {
     case syncer::USER_EVENTS:
       return browser_sync::UserEventServiceFactory::GetForProfile(profile_)
           ->GetControllerDelegate();
+    case syncer::VG_BODIES:
+      return VgSyncServiceFactory::GetForProfile(profile_)
+          ->GetControllerDelegateForVgBodies();
+    case syncer::VG_SPEND_STATUSES:
+      return VgSyncServiceFactory::GetForProfile(profile_)
+          ->GetControllerDelegateForVgSpendStatuses();
 #if BUILDFLAG(ENABLE_EXTENSIONS)
     case syncer::WEB_APPS: {
       auto* provider = web_app::WebAppProvider::GetForWebApps(profile_);
