diff --git a/chrome/renderer/chrome_content_renderer_client.h b/chrome/renderer/chrome_content_renderer_client.h
index 315fadea197fdd5836f3a71647cc0e52660d125e..d4a11cccdac968793b5ee0b656b8cbc6224a914f 100644
--- a/chrome/renderer/chrome_content_renderer_client.h
+++ b/chrome/renderer/chrome_content_renderer_client.h
@@ -239,6 +239,7 @@ class ChromeContentRendererClient
   base::WeakPtr<ChromeRenderThreadObserver> GetChromeObserver() const;
 
  private:
+  friend class BraveContentRendererClient;
   FRIEND_TEST_ALL_PREFIXES(ChromeContentRendererClientTest, NaClRestriction);
   FRIEND_TEST_ALL_PREFIXES(ChromeContentRendererClientTest,
                            ShouldSuppressErrorPage);
