diff --git a/chrome/browser/notifications/notification_display_service_impl.cc b/chrome/browser/notifications/notification_display_service_impl.cc
index 73ad845380e82fc015e810751fdf52d51c118849..2caf87c091627a8c26a0caf71efce79677dd0cc8 100644
--- a/chrome/browser/notifications/notification_display_service_impl.cc
+++ b/chrome/browser/notifications/notification_display_service_impl.cc
@@ -175,6 +175,7 @@ NotificationDisplayServiceImpl::NotificationDisplayServiceImpl(Profile* profile)
                              std::make_unique<NearbyNotificationHandler>());
     }
 #endif
+    BRAVE_ADD_BRAVE_ADS_NOTIFICATION_HANDLER;
   }
 
   // Initialize the bridge if native notifications are available, otherwise
