diff --git a/chrome/browser/sync/test/integration/single_client_polling_sync_test.cc b/chrome/browser/sync/test/integration/single_client_polling_sync_test.cc
index bde5311d6539b4d0cefe03adbd69ea659319a152..aa84597008ed31aaa2fdcc178cc05bfbddd3afbf 100644
--- a/chrome/browser/sync/test/integration/single_client_polling_sync_test.cc
+++ b/chrome/browser/sync/test/integration/single_client_polling_sync_test.cc
@@ -73,6 +73,7 @@ IN_PROC_BROWSER_TEST_F(SingleClientPollingSyncTest,
 
   SyncPrefs sync_prefs(GetProfile(0)->GetPrefs());
   EXPECT_THAT(sync_prefs.GetPollInterval().InSeconds(), Eq(67));
+  BRAVE_SET_DEFAULT_STARTUP_PREF
 }
 
 IN_PROC_BROWSER_TEST_F(SingleClientPollingSyncTest,
@@ -121,6 +122,7 @@ IN_PROC_BROWSER_TEST_F(SingleClientPollingSyncTest,
   // Simulate elapsed time so that the poll interval expired upon restart.
   remote_prefs.SetLastPollTime(base::Time::Now() -
                                remote_prefs.GetPollInterval());
+  BRAVE_SET_DEFAULT_STARTUP_PREF
 }
 
 IN_PROC_BROWSER_TEST_F(SingleClientPollingSyncTest,
