diff --git a/chrome/browser/ui/chrome_pages.cc b/chrome/browser/ui/chrome_pages.cc
index 02b6d6d91417533a855f98256ba92f18df76054a..235b211b80c0ab9074cf2c519cec2d15c8bb52a0 100644
--- a/chrome/browser/ui/chrome_pages.cc
+++ b/chrome/browser/ui/chrome_pages.cc
@@ -405,7 +405,7 @@ void ShowSlow(Browser* browser) {
 #endif
 }
 
-GURL GetSettingsUrl(std::string_view sub_page) {
+BRAVE_GET_SETTINGS_URL
   return GURL(base::StrCat({kChromeUISettingsURL, sub_page}));
 }
 
