diff --git a/chrome/browser/resources/extensions/BUILD.gn b/chrome/browser/resources/extensions/BUILD.gn
index 017696b5d7f4f7a3bebbbec6164ae55a45623487..bc9208a33fad4d165bb9c1b28eeac1096b9e744b 100644
--- a/chrome/browser/resources/extensions/BUILD.gn
+++ b/chrome/browser/resources/extensions/BUILD.gn
@@ -27,6 +27,7 @@ if (optimize_webui) {
       ":preprocess",
       ":preprocess_generated",
       "../../../../ui/webui/resources:preprocess",
+      "//brave/browser/resources/extensions:unpak",
     ]
     excludes = [ "chrome://resources/js/cr.m.js" ]
   }
