diff --git a/chrome/browser/ui/tabs/public/tab_features.h b/chrome/browser/ui/tabs/public/tab_features.h
index dbf1f0577b2806f1c5cef7b4f92f001fe2e2ad28..7a605b609dbb20d46d227a18ffe88ad74ef3c03a 100644
--- a/chrome/browser/ui/tabs/public/tab_features.h
+++ b/chrome/browser/ui/tabs/public/tab_features.h
@@ -148,7 +148,7 @@ class TabCreationMetricsController;
 class TabFeatures {
  public:
   TabFeatures();
-  ~TabFeatures();
+  virtual ~TabFeatures();
 
   TabFeatures(const TabFeatures&) = delete;
   TabFeatures& operator=(const TabFeatures&) = delete;
