diff --git a/chrome/android/java/src/org/chromium/chrome/browser/preferences/ChromePreferenceManager.java b/chrome/android/java/src/org/chromium/chrome/browser/preferences/ChromePreferenceManager.java
index 0d6f6b2d17a62954d7558ae5023340846ee55af4..e40842339336fd65df072907162464c21ea20287 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/preferences/ChromePreferenceManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/preferences/ChromePreferenceManager.java
@@ -15,7 +15,7 @@ import java.util.Set;
  *
  * TODO(crbug.com/1022102): Finish moving feature-specific methods out of this class and delete it.
  */
-public class ChromePreferenceManager {
+public class ChromePreferenceManager extends BravePreferenceManager {
     // For new int values with a default of 0, just document the key and its usage, and call
     // #readInt and #writeInt directly.
     // For new boolean values, document the key and its usage, call #readBoolean and #writeBoolean
