diff --git a/chrome/browser/profiles/profile_io_data.cc b/chrome/browser/profiles/profile_io_data.cc
index 2f5ed4b15589ffa5bae18d7f06e6776f705f5229..a4bd6ab7ef01d4740211eff1de086fe1493f2129 100644
--- a/chrome/browser/profiles/profile_io_data.cc
+++ b/chrome/browser/profiles/profile_io_data.cc
@@ -248,6 +248,7 @@ bool ProfileIOData::IsHandledProtocol(const std::string& scheme) {
 #endif
     content::kChromeUIScheme,
     content::kChromeUIUntrustedScheme,
+    kBraveUIScheme,
     url::kDataScheme,
 #if BUILDFLAG(IS_CHROMEOS_ASH)
     content::kExternalFileScheme,
