diff --git a/chrome/browser/ui/views/tabs/dragging/dragging_tabs_session.h b/chrome/browser/ui/views/tabs/dragging/dragging_tabs_session.h
index cddf52f7a1691ce99d9ff2d6418663c1e36ec25a..7ba5ed49b89d46ae25f3c13c624cd939d39e0a44 100644
--- a/chrome/browser/ui/views/tabs/dragging/dragging_tabs_session.h
+++ b/chrome/browser/ui/views/tabs/dragging/dragging_tabs_session.h
@@ -15,7 +15,7 @@
 
 // Handles dragging tabs within a single TabDragContext on behalf of
 // TabDragController.
-class DraggingTabsSession final : public TabDragWithScrollManager {
+class DraggingTabsSession : public TabDragWithScrollManager {
  public:
   // `drag_data` is a copy of the drag configuration for the full session.
   // `attached_context` is the context in which the tabs are being dragged.
@@ -29,7 +29,7 @@ class DraggingTabsSession final : public TabDragWithScrollManager {
                                float offset_to_width_ratio_,
                                bool initial_move,
                                gfx::Point point_in_screen);
-  ~DraggingTabsSession() final;
+  ~DraggingTabsSession() override;
 
   // TabDragWithScrollManager:
   void MoveAttached(gfx::Point point_in_screen) override;
