diff --git a/chrome/browser/ui/views/tabs/new_tab_button.h b/chrome/browser/ui/views/tabs/new_tab_button.h
index 344125c7e95ae25066ce350dc482ab56ff12c1b0..18ff6f4f868a4e47882f9fb7e3177cd745678731 100644
--- a/chrome/browser/ui/views/tabs/new_tab_button.h
+++ b/chrome/browser/ui/views/tabs/new_tab_button.h
@@ -60,6 +60,7 @@ class NewTabButton : public views::ImageButton,
   FeaturePromoBubbleView* new_tab_promo() { return new_tab_promo_; }
 
  private:
+  friend class BraveNewTabButton;
 // views::ImageButton:
 #if defined(OS_WIN)
   void OnMouseReleased(const ui::MouseEvent& event) override;
@@ -83,12 +84,14 @@ class NewTabButton : public views::ImageButton,
   void PaintFill(gfx::Canvas* canvas) const;
 
   // Paints a properly sized plus (+) icon into the center of the button.
+  virtual
   void PaintPlusIcon(gfx::Canvas* canvas) const;
 
   SkColor GetButtonFillColor() const;
 
   // Returns the path for the given |origin| and |scale|.  If |extend_to_top| is
   // true, the path is extended vertically to y = 0.
+  virtual
   SkPath GetBorderPath(const gfx::Point& origin,
                        float scale,
                        bool extend_to_top) const;
