diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 4e229c16628f69183977d5d234fd7ab6dca0544a..c53534524a72f724162fdb8ef0c084c9fb0821f4 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -128,6 +128,7 @@ jumbo_static_library("renderer") {
   defines = []
 
   deps = [
+    "//brave/renderer",
     "//chrome:resources",
     "//chrome:strings",
     "//chrome/common",
