diff --git a/chrome/android/chrome_public_apk_tmpl.gni b/chrome/android/chrome_public_apk_tmpl.gni
index 8c6cd1cb8aaf2b7a6dc38e2e5eedd05e7a2a5870..078daea2a3f1dfc49c310dc0c3063f1f4e09cb95 100644
--- a/chrome/android/chrome_public_apk_tmpl.gni
+++ b/chrome/android/chrome_public_apk_tmpl.gni
@@ -176,6 +176,7 @@ template("chrome_public_common_apk_or_module_tmpl") {
 
     # Remove unneeded entries from material design values.xml files.
     resource_values_filter_rules = [ "${_material_package}:[Tt]oolbarLayout" ]
+    resource_exclusion_exceptions += brave_resources_exclusion_exceptions
 
     if (!_is_monochrome) {
       product_config_java_packages = [ "org.chromium.chrome.browser" ]
@@ -285,6 +286,7 @@ template("chrome_public_common_apk_or_module_tmpl") {
       if (!defined(proguard_configs)) {
         proguard_configs = []
       }
+      proguard_configs += ["//brave/android/java/proguard.flags",]
       proguard_configs += [
         "//chrome/android/proguard/main.flags",
         "//base/android/proguard/chromium_apk.flags",
