diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 94886afca0c71f2285c8e1ffd6eeb738ed9ed041..90c4854630a51119dfa352ee0bd45a5e9897ba77 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -1978,6 +1978,7 @@ jumbo_split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
