diff --git a/chrome/browser/ui/views/page_info/page_info_cookies_content_view.cc b/chrome/browser/ui/views/page_info/page_info_cookies_content_view.cc
index 1289b9964a73fbeecb9051141d8a423d3e20324c..e85c3be944851bece9d6039cc92c553447a650f8 100644
--- a/chrome/browser/ui/views/page_info/page_info_cookies_content_view.cc
+++ b/chrome/browser/ui/views/page_info/page_info_cookies_content_view.cc
@@ -420,6 +420,7 @@ void PageInfoCookiesContentView::SetThirdPartyCookiesInfo(
     CookieControlsEnforcement enforcement,
     CookieBlocking3pcdStatus blocking_status,
     base::Time expiration) {
+  BRAVE_PAGE_INFO_COOKIES_CONTENT_VIEW_SET_THIRD_PARTY_COOKIES_INFO
   if (controls_state == CookieControlsState::kHidden) {
     third_party_cookies_container_->SetVisible(false);
     return;
