diff --git a/chrome/browser/profiles/renderer_updater.cc b/chrome/browser/profiles/renderer_updater.cc
index 8920c217f0b261e3d35de47b141f33624a763042..d1902d98daa8741f2141072f56a4f9c2c266f00f 100644
--- a/chrome/browser/profiles/renderer_updater.cc
+++ b/chrome/browser/profiles/renderer_updater.cc
@@ -51,6 +51,7 @@ void GetGuestViewDefaultContentSettingRules(
       base::Value::FromUniquePtrValue(
           content_settings::ContentSettingToValue(CONTENT_SETTING_BLOCK)),
       std::string(), incognito));
+  BRAVE_GET_GUEST_VIEW_DEFAULT_CONTENT_SETTING_RULES
 }
 
 #endif  // BUILDFLAG(ENABLE_EXTENSIONS)
