diff --git a/chrome/browser/resources/settings/site_settings/constants.js b/chrome/browser/resources/settings/site_settings/constants.js
index fb816c996282016f5142bcbf066002ba95dee384..7756a9aae3674ab38f17b34b2790d98b91c961c5 100644
--- a/chrome/browser/resources/settings/site_settings/constants.js
+++ b/chrome/browser/resources/settings/site_settings/constants.js
@@ -43,6 +43,7 @@ export const ContentSettingsTypes = {
   VR: 'vr',
   WINDOW_PLACEMENT: 'window-placement',
   ZOOM_LEVELS: 'zoom-levels',
+  AUTOPLAY: 'autoplay',
 };
 
 /**
