diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index d058dee3b853d66ccc5727667382b4821d75577b..615cd6e6f30290e0880f3435117873573b6fbcdd 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -484,6 +484,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
