diff --git a/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc b/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc
index 1af672e86a3d563081da2e473e493f5e9c8f8fca..d98b420395a3502025ae3c9d9d6a878f950c2d22 100644
--- a/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc
+++ b/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc
@@ -275,6 +275,7 @@ PermissionPromptBubbleView::PermissionPromptBubbleView(
   for (permissions::PermissionRequest* request :
        GetVisibleRequests(*delegate.get()))
     AddRequestLine(request);
+  BRAVE_PERMISSION_PROMPT_BUBBLE_VIEW
 
   absl::optional<std::u16string> extra_text = GetExtraText(*delegate.get());
   if (extra_text.has_value()) {
