diff --git a/chrome/installer/linux/common/installer.include b/chrome/installer/linux/common/installer.include
index 21ae32901cc38c032e9a40b4949a879e87a4b7fb..e654c2a45121241bd95c68cf3447e8bb75c29c27 100644
--- a/chrome/installer/linux/common/installer.include
+++ b/chrome/installer/linux/common/installer.include
@@ -75,6 +75,7 @@ process_template() (
     -e "s#@@PACKAGE@@#${PACKAGE}#g" \
     -e "s#@@PACKAGE_ORIG@@#${PACKAGE_ORIG}#g" \
     -e "s#@@PACKAGE_FILENAME@@#${PACKAGE_FILENAME}#g" \
+    -e "s#@@PACKAGEANDCHANNEL@@#${PACKAGEANDCHANNEL}#g" \
     -e "s#@@PROGNAME@@#${PROGNAME}#g" \
     -e "s#@@CHANNEL@@#${CHANNEL}#g" \
     -e "s#@@COMPANY_FULLNAME@@#${COMPANY_FULLNAME}#g" \
@@ -161,6 +162,25 @@ stage_install_common() {
     install -m 644 "${BUILDDIR}/ui_resources_100_percent.pak" "${STAGEDIR}/${INSTALLDIR}/"
   fi
 
+  # brave_resources.pak, contains customization, shields
+  install -m 644 "${BUILDDIR}/brave_resources.pak" "${STAGEDIR}/${INSTALLDIR}/"
+
+  # brave_[100|200]_resources.pak for scaled resources
+  install -m 644 "${BUILDDIR}/brave_100_percent.pak" "${STAGEDIR}/${INSTALLDIR}/"
+  install -m 644 "${BUILDDIR}/brave_200_percent.pak" "${STAGEDIR}/${INSTALLDIR}/"
+
+  # localization files for Brave extension
+  install -d -m 755 "${STAGEDIR}/${INSTALLDIR}/resources/brave_extension/_locales"
+  cp -a "${BUILDDIR}/resources/brave_extension/_locales" "${STAGEDIR}/${INSTALLDIR}/resources/brave_extension"
+  find "${STAGEDIR}/${INSTALLDIR}/resources/brave_extension/_locales" -type f -exec chmod 644 '{}' \;
+  find "${STAGEDIR}/${INSTALLDIR}/resources/brave_extension/_locales" -type d -exec chmod 755 '{}' \;
+
+  # localization files for Brave Rewards
+  install -d -m 755 "${STAGEDIR}/${INSTALLDIR}/resources/brave_rewards/_locales"
+  cp -a "${BUILDDIR}/resources/brave_rewards/_locales" "${STAGEDIR}/${INSTALLDIR}/resources/brave_rewards"
+  find "${STAGEDIR}/${INSTALLDIR}/resources/brave_rewards/_locales" -type f -exec chmod 644 '{}' \;
+  find "${STAGEDIR}/${INSTALLDIR}/resources/brave_rewards/_locales" -type d -exec chmod 755 '{}' \;
+
   # ICU data file; Necessary when the GN icu_use_data_file flag is true.
   install -m 644 "${BUILDDIR}/icudtl.dat" "${STAGEDIR}/${INSTALLDIR}/"
 
@@ -184,7 +204,7 @@ stage_install_common() {
   strippedfile="${buildfile}.stripped"
   debugfile="${buildfile}.debug"
   "${BUILDDIR}/installer/common/eu-strip" -o "${strippedfile}" -f "${debugfile}" "${buildfile}"
-  install -m 4755 "${strippedfile}" "${STAGEDIR}/${INSTALLDIR}/${PROGNAME}-sandbox"
+  install -m 4755 "${strippedfile}" "${STAGEDIR}/${INSTALLDIR}/chrome-sandbox"
 
   # l10n paks
   install -m 755 -d "${STAGEDIR}/${INSTALLDIR}/locales/"
@@ -285,11 +305,13 @@ stage_install_common() {
 
   # app icons
   local icon_regex=".*product_logo_[0-9]\+\."
-  if [ "$BRANDING" = "google_chrome" ]; then
+  if [ "$BRANDING" = "brave" ]; then
     if [ "$CHANNEL" = "beta" ]; then
       icon_regex=".*product_logo_[0-9]\+_beta\."
-    elif [ "$CHANNEL" = "unstable" ]; then
+    elif [ "$CHANNEL" = "unstable" -o "$CHANNEL" = "dev" ]; then
       icon_regex=".*product_logo_[0-9]\+_dev\."
+    elif [ "$CHANNEL" = "nightly" ]; then
+      icon_regex=".*product_logo_[0-9]\+_nightly\."
     fi
   fi
   LOGO_RESOURCES_PNG=$(find "${BUILDDIR}/installer/theme/" \
@@ -397,6 +419,8 @@ stage_install_common() {
       local expected_perms=644
     elif [[ "${file_type}" = *"shell script"* ]]; then
       local expected_perms=755
+    elif [[ "${base_name}" = "libwidevinecdm.so" ]]; then
+      local expected_perms=${actual_perms}
     elif [[ "${file_type}" = ELF* ]]; then
       if [[ "${base_name}" = *".so" ]]; then
         local expected_perms=${SHLIB_PERMS}
