diff --git a/chrome/browser/resources/pdf/viewport.js b/chrome/browser/resources/pdf/viewport.js
index 480664c014ce18126a4806a9ac484bed390d68ad..8f4f959091038848466d1130fd1f610035fa5212 100644
--- a/chrome/browser/resources/pdf/viewport.js
+++ b/chrome/browser/resources/pdf/viewport.js
@@ -1228,7 +1228,7 @@ Viewport.SCROLL_INCREMENT = 40;
  * chrome/browser/resources/settings/appearance_page/appearance_page.js
  */
 Viewport.ZOOM_FACTORS = [
-  0.25, 1 / 3, 0.5, 2 / 3, 0.75, 0.8, 0.9, 1, 1.1, 1.25, 1.5, 1.75, 2, 2.5, 3,
+  0.25, 1 / 3, 0.5, 2 / 3, 0.75, 0.8, 0.9, 1, 1.1, 1.25, 1.33, 1.4, 1.5, 1.75, 2, 2.5, 3,
   4, 5
 ];
 
