diff --git a/chrome/browser/resources/history/BUILD.gn b/chrome/browser/resources/history/BUILD.gn
index a5c849ab6096c7d6e6ad2bb4c358bf3cb042fd81..29e896f6deef3933e7d96c775a02210718f14f7b 100644
--- a/chrome/browser/resources/history/BUILD.gn
+++ b/chrome/browser/resources/history/BUILD.gn
@@ -117,4 +117,5 @@ build_webui("build") {
     optimize_webui_host = "history"
     optimize_webui_in_files = [ "history.js" ]
   }
+  import("//brave/browser/resources/history/sources.gni") preprocessor_defines = brave_history_preprocessor_defines mojo_files = brave_history_mojo_files mojo_files_deps = brave_history_mojo_files_deps ts_files += brave_history_ts_files exclude_ts_preprocess_files = brave_history_ts_files preprocess_deps = brave_history_preprocess_deps
 }
