diff --git a/chrome/browser/ui/extensions/extension_action_view_controller.cc b/chrome/browser/ui/extensions/extension_action_view_controller.cc
index dace9f3bcd2b1fc20be44c872bcc66fbf602d834..b360709c543b0708449aa110ca7506af746fd42d 100644
--- a/chrome/browser/ui/extensions/extension_action_view_controller.cc
+++ b/chrome/browser/ui/extensions/extension_action_view_controller.cc
@@ -61,7 +61,6 @@ ExtensionActionViewController::ExtensionActionViewController(
       extension_registry_(
           extensions::ExtensionRegistry::Get(browser_->profile())),
       popup_host_observer_(this) {
-  DCHECK(extensions_container);
   DCHECK(extension_action);
   DCHECK(extension);
 }
