diff --git a/chrome/browser/renderer_context_menu/render_view_context_menu.cc b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
index 4a5f2d8019298f58178580bc2faebc03d7843e39..79d07d60aad3f00bb10863c84d44698e591c9671 100644
--- a/chrome/browser/renderer_context_menu/render_view_context_menu.cc
+++ b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
@@ -1077,6 +1077,7 @@ void RenderViewContextMenu::AppendLinkItems() {
                : IDS_CONTENT_CONTEXT_OPENLINKOFFTHERECORD);
 
     AppendOpenInBookmarkAppLinkItems();
+    AppendBraveLinkItems();
     AppendOpenWithLinkItems();
 
     // While ChromeOS supports multiple profiles, only one can be open at a
