diff --git a/chrome/common/pref_names.cc b/chrome/common/pref_names.cc
index 22b7a7b6575c927bdd4d568f068e41e0c4369c58..408f66df31c11d584b9343517e4a7aa71997bfc2 100644
--- a/chrome/common/pref_names.cc
+++ b/chrome/common/pref_names.cc
@@ -1050,6 +1050,7 @@ const char kImportDialogBookmarks[] = "import_dialog_bookmarks";
 const char kImportDialogHistory[] = "import_dialog_history";
 const char kImportDialogSavedPasswords[] = "import_dialog_saved_passwords";
 const char kImportDialogSearchEngine[] = "import_dialog_search_engine";
+const char kImportDialogCookies[] = "import_dialog_cookies";
 
 // Profile avatar and name
 const char kProfileAvatarIndex[] = "profile.avatar_index";
