diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 030e4c1c31fd2c782c70a0177d816a2c0901c32f..6fac8f059c617cca0100d8b2d436aa3e8d6240e6 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2491,6 +2491,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps defines += brave_chrome_browser_defines
 
   if (is_android) {
     sources += [
