diff --git a/chrome/browser/download/download_item_model.h b/chrome/browser/download/download_item_model.h
index d7d554b117c275e18f674fb3323c4d29a6a8ddab..20f700b91fbca59078f9a1d27e66fdf4c4c1a36a 100644
--- a/chrome/browser/download/download_item_model.h
+++ b/chrome/browser/download/download_item_model.h
@@ -153,6 +153,7 @@ class DownloadItemModel : public DownloadUIModel,
   void OnDownloadDestroyed(download::DownloadItem* download) override;
 
  private:
+  friend class BraveDownloadItemModel;
   // DownloadUIModel implementation.
   std::string GetMimeType() const override;
 
