diff --git a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
index cb94703264e51bcd4ad0be7d2df68a1000ebb427..ab70458aac12fdbe4b3bc8b510064113c93d9921 100644
--- a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
+++ b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
@@ -81,7 +81,7 @@ content::WebUIDataSource* CreateAndAddBookmarksUIHTMLSource(Profile* profile) {
       {"editDialogInvalidUrl", IDS_BOOKMARK_MANAGER_INVALID_URL},
       {"editDialogNameInput", IDS_BOOKMARK_MANAGER_NAME_INPUT_PLACE_HOLDER},
       {"editDialogUrlInput", IDS_BOOKMARK_MANAGER_URL_INPUT_PLACE_HOLDER},
-      {"emptyList", IDS_BOOKMARK_MANAGER_EMPTY_LIST},
+      {"emptyList", IDS_BRAVE_BOOKMARK_MANAGER_EMPTY_LIST},
       {"emptyUnmodifiableList", IDS_BOOKMARK_MANAGER_EMPTY_UNMODIFIABLE_LIST},
       {"folderLabel", IDS_BOOKMARK_MANAGER_FOLDER_LABEL},
       {"importBegan", IDS_BOOKMARK_MANAGER_MENU_IMPORT_BEGAN},
