diff --git a/chrome/browser/sync/sync_service_factory.cc b/chrome/browser/sync/sync_service_factory.cc
index d905433998f149ad4372886817a71dba8d93575b..44de4f43da275e6114c43d284dd31a135fe53e02 100644
--- a/chrome/browser/sync/sync_service_factory.cc
+++ b/chrome/browser/sync/sync_service_factory.cc
@@ -461,7 +461,7 @@ std::unique_ptr<KeyedService> BuildSyncService(
   }
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize(CreateControllers(profile, sync_service.get()));
 
   // Notify the PasswordStore of complete initialisation to resolve a circular
