diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index fa166f5d8079459fa0e5bd6542b84e484aeb0488..101e6d7e49b100098af1decafdd270735c9023cd 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -1010,6 +1010,7 @@ jumbo_split_static_library("ui") {
     "//ui/webui",
     "//v8:v8_version",
   ]
+  deps += [ "//brave/browser/ui" ]
   allow_circular_includes_from +=
       [ "//chrome/browser/ui/webui/bluetooth_internals" ]
 
@@ -2647,10 +2648,13 @@ jumbo_split_static_library("ui") {
       ]
       deps += [ "//google_update" ]
     } else {
-      sources += [
-        "webui/help/version_updater_basic.cc",
-        "webui/help/version_updater_basic.h",
-      ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [
+          "webui/help/version_updater_basic.cc",
+          "webui/help/version_updater_basic.h",
+        ]
+      }
     }
   } else {  # 'OS!="win"
     # The Cast Linux build sets toolkit_views to false, but many files in this
