diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index 15fa946c5709c086dffdcabd5e2a251ddbc9e225..e68481c1e3d45a77523828a0deea4d422f3352c6 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -241,7 +241,7 @@ Browser* Browser::Create(BrowserWindowCreateParams params) {
 
   Profile* const profile_ptr = &*params.profile;
   std::unique_ptr<Browser> browser =
-      base::WrapUnique(new Browser(std::move(params)));
+      base::WrapUnique(new BraveBrowser(std::move(params)));
   Browser* const browser_ptr = browser.get();
   BrowserManagerServiceFactory::GetForProfile(profile_ptr)
       ->AddBrowser(std::move(browser));
@@ -260,7 +260,7 @@ std::unique_ptr<Browser> Browser::DeprecatedCreateOwnedForTesting(
 
   Profile* const profile_ptr = &*params.profile;
   std::unique_ptr<Browser> browser =
-      base::WrapUnique(new Browser(std::move(params)));
+      base::WrapUnique(new BraveBrowser(std::move(params)));
   BrowserManagerServiceFactory::GetForProfile(profile_ptr)
       ->AddBrowserForTesting(browser.get());
   return browser;
@@ -271,8 +271,8 @@ Browser::Browser(BrowserWindowCreateParams params)
       profile_(&*params.profile),
       window_(nullptr),
       tab_strip_model_delegate_(
-          std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
-      tab_strip_model_(std::make_unique<TabStripModel>(
+          std::make_unique<chrome::BraveTabStripModelDelegate>(this)),
+      tab_strip_model_(std::make_unique<BraveTabStripModel>(
           tab_strip_model_delegate_.get(),
           profile_,
           // Tab groups are disabled for app browsers.
