diff --git a/chrome/browser/resources/settings/site_settings/constants.ts b/chrome/browser/resources/settings/site_settings/constants.ts
index 81c9c2eafc35929c8e2ee4673458cceb785ffcb1..f4bf5e3b5dc09593902d72f2fc454ab451d15133 100644
--- a/chrome/browser/resources/settings/site_settings/constants.ts
+++ b/chrome/browser/resources/settings/site_settings/constants.ts
@@ -68,6 +68,7 @@ export enum ContentSettingsTypes {
   PDF_DOCUMENTS = 'pdfDocuments',
   SITE_DATA = 'site-data',
   OFFER_WRITING_HELP = 'offer-writing-help',
+  AUTOPLAY = 'autoplay', ETHEREUM = 'ethereum', SOLANA = 'solana', CARDANO = 'cardano', BRAVE_SHIELDS = 'braveShields', GOOGLE_SIGN_IN = 'googleSignIn', LOCALHOST_ACCESS = 'localhostAccess', BRAVE_OPEN_AI_CHAT = 'braveOpenAIChat'
 }
 
 /**
