diff --git a/chrome/browser/extensions/extension_service.h b/chrome/browser/extensions/extension_service.h
index 972396d12a9a314ae67d2360c782619c53ba5b0a..3bb6a3c28655c6fdf40be569416a358018b496cb 100644
--- a/chrome/browser/extensions/extension_service.h
+++ b/chrome/browser/extensions/extension_service.h
@@ -465,6 +465,7 @@ class ExtensionService : public ExtensionServiceInterface,
   static base::AutoReset<bool> DisableExternalUpdatesForTesting();
 
  private:
+  friend class BraveExtensionService;
   // Loads extensions specified via a command line flag/switch.
   void LoadExtensionsFromCommandLineFlag(const char* switch_name);
 #if BUILDFLAG(IS_CHROMEOS_ASH)
