diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageView.java b/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageView.java
index 7a1b1b945f89bfe7bd25e1f837a5c43f5eadcabe..cd77cc5adc6d65f910b180792597ab1c7485b794 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageView.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageView.java
@@ -198,6 +198,7 @@ public class NewTabPageView extends FrameLayout {
         mRecyclerView.setAdapter(newTabPageAdapter);
         mRecyclerView.getLinearLayoutManager().scrollToPosition(scrollPosition);
 
+        if (false)
         mRecyclerViewResizer = ViewResizer.createAndAttach(mRecyclerView, mUiConfig,
                 mRecyclerView.getResources().getDimensionPixelSize(
                         R.dimen.content_suggestions_card_modern_margin),
