diff --git a/chrome/browser/ui/views/tabs/common/tab_collection_node.cc b/chrome/browser/ui/views/tabs/common/tab_collection_node.cc
index 2f569c08d2d08bf6836cbee39cb028d4050b0073..655c5e4f015084a9e12d0e29e8222a0ef54a1eae 100644
--- a/chrome/browser/ui/views/tabs/common/tab_collection_node.cc
+++ b/chrome/browser/ui/views/tabs/common/tab_collection_node.cc
@@ -35,6 +35,7 @@ TabCollectionNode::Type GetTypeFromNode(tabs::ConstChildPtr node_data_) {
         return TabCollectionNode::Type::GROUP;
       case tabs::TabCollection::Type::SPLIT:
         return TabCollectionNode::Type::SPLIT;
+      BRAVE_GET_TYPE_FROM_NODE
     }
   }
   CHECK(std::holds_alternative<const tabs::TabInterface*>(node_data_));
@@ -105,6 +106,7 @@ std::unique_ptr<views::View> TabCollectionNode::CreateViewForNode(
       return std::make_unique<TabGroupView>(node_for_view);
     case Type::TAB:
       return std::make_unique<TabView>(node_for_view);
+    BRAVE_TAB_COLLECTION_NODE_CREATE_VIEW_FOR_NODE
   }
   return std::make_unique<CollectionTestViewImpl>(node_for_view);
 }
