diff --git a/chrome/browser/notifications/notification_platform_bridge_mac.mm b/chrome/browser/notifications/notification_platform_bridge_mac.mm
index b2c65a3ba3d19a00e750c5b1fcf83b6e2047380c..b9a8677ec06a460a508b90efa4857da527e297ba 100644
--- a/chrome/browser/notifications/notification_platform_bridge_mac.mm
+++ b/chrome/browser/notifications/notification_platform_bridge_mac.mm
@@ -226,6 +226,7 @@ void NotificationPlatformBridgeMac::Display(
       initWithCloseLabel:l10n_util::GetNSString(IDS_NOTIFICATION_BUTTON_CLOSE)
             optionsLabel:l10n_util::GetNSString(IDS_NOTIFICATION_BUTTON_MORE)
            settingsLabel:l10n_util::GetNSString(
+            notification_type == NotificationHandler::Type::BRAVE_ADS ? IDS_VIEW_MENU_MAC :
                              IDS_NOTIFICATION_BUTTON_SETTINGS)]);
 
   [builder
