diff --git a/chrome/utility/services.cc b/chrome/utility/services.cc
index 9d4ecfeb2d1fb5f20f74e736e02f58c3e160f2ff..dc5908e857132e1e521928ce22a4973c02a14660 100644
--- a/chrome/utility/services.cc
+++ b/chrome/utility/services.cc
@@ -394,6 +394,7 @@ void RegisterMainThreadServices(mojo::ServiceFactory& services) {
   services.Add(RunAssistantAudioDecoder);
 #endif
 #endif
+    BRAVE_GET_MAIN_THREAD_SERVICE_FACTORY
 }
 
 void RegisterIOThreadServices(mojo::ServiceFactory& services) {
