diff --git a/chrome/installer/mac/pkg-dmg b/chrome/installer/mac/pkg-dmg
index 4f9d03636d1bed8e774f26eb7ca17c80c3d8c8be..74d11aba03a278c3fadbcc648e4886ee2c01c0ab 100755
--- a/chrome/installer/mac/pkg-dmg
+++ b/chrome/installer/mac/pkg-dmg
@@ -334,6 +334,7 @@ my($ignore, $uname_r, $uname_s);
 if($uname_s ne 'Darwin') {
   print STDERR ($0.": warning, not running on Mac OS X, ".
    "this could be interesting.\n");
+  use File::Basename; use File::Spec; my $linux_script = dirname(File::Spec->rel2abs($0)); $linux_script =~ s|/src/.*|/src/brave/build/mac/cross_compile/pkg-dmg-linux.py|; exec($linux_script, @ARGV) or die "Failed to execute $linux_script: $!";
 }
 
 # Non-global variables used in Getopt
