diff --git a/chrome/browser/sessions/BUILD.gn b/chrome/browser/sessions/BUILD.gn
index 92c6ac9b37578c560952fe592b1fb0fba0ac8e62..f1782913a68536398c14eb4918722e5f3e10ac61 100644
--- a/chrome/browser/sessions/BUILD.gn
+++ b/chrome/browser/sessions/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/chromium_src/chrome/browser/sessions/sources.gni")
 import("//build/config/ozone.gni")
 import("//chrome/common/features.gni")
 import("//extensions/buildflags/buildflags.gni")
@@ -177,6 +178,7 @@ source_set("impl") {
         "//chromeos/components/kiosk",
       ]
     }
+    deps += brave_chromium_src_chrome_browser_sessions_deps
   }
 }
 
