diff --git a/chrome/browser/ui/extensions/icon_with_badge_image_source.h b/chrome/browser/ui/extensions/icon_with_badge_image_source.h
index eaffb48603eb622c0ab5c5d24d5748750c1dcbee..805f74ed5f752742b8d1059f5d91aa992a01aa63 100644
--- a/chrome/browser/ui/extensions/icon_with_badge_image_source.h
+++ b/chrome/browser/ui/extensions/icon_with_badge_image_source.h
@@ -58,6 +58,7 @@ class IconWithBadgeImageSource : public gfx::CanvasImageSource {
   bool paint_blocked_actions_decoration() const {
     return paint_blocked_actions_decoration_;
   }
+  BRAVE_ICON_WITH_BADGE_IMAGE_SOURCE_H_
 
  private:
   // gfx::CanvasImageSource:
