diff --git a/chrome/test/base/testing_profile.cc b/chrome/test/base/testing_profile.cc
index 5c546cc9d766ac7c8c94a222d3ac5f1bc7d738bd..e4022ad4fa0d6132ac7deee4376118491a417872 100644
--- a/chrome/test/base/testing_profile.cc
+++ b/chrome/test/base/testing_profile.cc
@@ -414,6 +414,7 @@ void TestingProfile::Init() {
       EnsureBrowserContextKeyedServiceFactoriesBuilt();
 
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
+  BRAVE_INIT
   if (!IsOffTheRecord()) {
     SupervisedUserSettingsService* settings_service =
         SupervisedUserSettingsServiceFactory::GetForKey(key_.get());
