diff --git a/chrome/browser/ui/views/download/download_item_view.h b/chrome/browser/ui/views/download/download_item_view.h
index 21811f092722692ac98b8caa028ab4f373965ad8..96dc332c83bdd35528fb9ffc99773589f9bb0168 100644
--- a/chrome/browser/ui/views/download/download_item_view.h
+++ b/chrome/browser/ui/views/download/download_item_view.h
@@ -119,6 +119,7 @@ class DownloadItemView : public views::View,
   void OnPaintBackground(gfx::Canvas* canvas) override;
   void OnPaint(gfx::Canvas* canvas) override;
   void OnThemeChanged() override;
+  BRAVE_DOWNLOAD_DOWNLOAD_ITEM_VIEW_H_
 
   // ui::LayerDelegate:
   void OnDeviceScaleFactorChanged(float old_device_scale_factor,
