diff --git a/chrome/android/chrome_public_apk_tmpl.gni b/chrome/android/chrome_public_apk_tmpl.gni
index 4438f47bcfdb8cbbc1428b35d22bff3aa3f484c4..5b4a19980bb79d750922b61e75faad5afe654e2e 100644
--- a/chrome/android/chrome_public_apk_tmpl.gni
+++ b/chrome/android/chrome_public_apk_tmpl.gni
@@ -227,6 +227,7 @@ template("chrome_public_common_apk_or_module_tmpl") {
       "${_material_package}:[Ss]lider",
       "${_material_package}:[Tt]oolbarLayout",
     ]
+    resource_exclusion_exceptions += brave_resources_exclusion_exceptions
     resource_values_filter_rules -= ["${_material_package}:[Ff]loatingActionButton"]
 
     if (!_is_monochrome) {
@@ -331,6 +332,7 @@ template("chrome_public_common_apk_or_module_tmpl") {
       if (!defined(proguard_configs)) {
         proguard_configs = []
       }
+      proguard_configs += ["//brave/android/java/proguard.flags",]
       proguard_configs += [
         "//chrome/android/java/proguard.flags",
         "//base/android/proguard/chromium_apk.flags",
