diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index b92bf49ca021b13af1c3c843bf54fd792fa74afc..c50f7a30bd1ad05bf917c810465f1b2362158b0e 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3539,6 +3539,7 @@ const FeatureEntry::FeatureVariation kFastCheckoutVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
