diff --git a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
index accdb1bb07689bc67aeca25feef87966fbe7c0aa..eb056358404e300204c33dda59bf8d02b02cc213 100644
--- a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
+++ b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
@@ -311,6 +311,7 @@ class AppMenu implements OnItemClickListener, OnKeyListener, AppMenuClickHandler
                 anchorView,
                 groupDividerResourceId,
                 anchorViewOffset);
+        if (this instanceof BraveAppMenu) ((BraveAppMenu) this).updatePopup(mPopup, isByPermanentButton);
         int[] popupPosition =
                 getPopupPosition(
                         mTempLocation,
