diff --git a/chrome/browser/extensions/extension_tab_util.cc b/chrome/browser/extensions/extension_tab_util.cc
index e3a69e27433144b479768cf2b8c7e102cc80be53..790c8d082e3bd08f982a54c6292041767f5e1075 100644
--- a/chrome/browser/extensions/extension_tab_util.cc
+++ b/chrome/browser/extensions/extension_tab_util.cc
@@ -795,6 +795,7 @@ bool ExtensionTabUtil::IsKillURL(const GURL& url) {
   };
 
   if (!url.SchemeIs(content::kChromeUIScheme))
+  if (!url.SchemeIs(content::kBraveUIScheme))
     return false;
 
   return base::Contains(kill_hosts, url.host_piece());
