diff --git a/chrome/renderer/content_settings_observer.h b/chrome/renderer/content_settings_observer.h
index 5939455cb7aa2205c617755acbab0149ff7202e7..0e4f411490478413a69d29a0dc07f80a72dcf0e5 100644
--- a/chrome/renderer/content_settings_observer.h
+++ b/chrome/renderer/content_settings_observer.h
@@ -111,6 +111,7 @@ class ContentSettingsObserver
   }
 
  private:
+  friend class BraveContentSettingsObserver;
   FRIEND_TEST_ALL_PREFIXES(ContentSettingsObserverTest, WhitelistedSchemes);
   FRIEND_TEST_ALL_PREFIXES(ContentSettingsObserverBrowserTest,
                            ContentSettingsInterstitialPages);
