diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 619f741f150975e1a93a32783f9bbfe78e9bee15..a0a09d22c024a5bacd27dfa60ae026ac0c09146b 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -461,6 +461,7 @@ void ToolbarView::Init() {
         container_view_->AddChildView(std::move(side_panel_button));
   }
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = container_view_->AddChildView(
       std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
