diff --git a/chrome/installer/mac/signing/modification.py b/chrome/installer/mac/signing/modification.py
index 62461b92268e722eaa091dbd4f5bdf1ba3f14585..5b18b0bcd6597d4aa59067caaff58e19b9b8164e 100644
--- a/chrome/installer/mac/signing/modification.py
+++ b/chrome/installer/mac/signing/modification.py
@@ -59,15 +59,8 @@ def _modify_plists(paths, dist, config):
         elif _KS_BRAND_ID in app_plist:
             del app_plist[_KS_BRAND_ID]
 
-        base_tag = app_plist.get(_KS_CHANNEL_ID)
-        base_channel_tag_components = []
-        if base_tag:
-            base_channel_tag_components.append(base_tag)
         if dist.channel:
-            base_channel_tag_components.append(dist.channel)
-        base_channel_tag = '-'.join(base_channel_tag_components)
-        if base_channel_tag:
-            app_plist[_KS_CHANNEL_ID] = base_channel_tag
+            app_plist[_KS_CHANNEL_ID] = dist.channel
         elif _KS_CHANNEL_ID in app_plist:
             del app_plist[_KS_CHANNEL_ID]
 
@@ -82,8 +75,9 @@ def _modify_plists(paths, dist, config):
         for key in app_plist.keys():
             if not key.startswith(_KS_CHANNEL_ID + '-'):
                 continue
-            ignore, extra = key.split('-')
-            app_plist[key] = '{}-{}'.format(base_channel_tag, extra)
+            orig_channel, tag = key.split('-')
+            channel_str = dist.channel if dist.channel else ''
+            app_plist[key] = '{}-{}'.format(channel_str, tag)
 
 
 def _replace_icons(paths, dist, config):
