diff --git a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog_v2.ts b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog_v2.ts
index 76f9abeac4105c36fac46bc49d07412a15e9dfb2..d8d354372fa57dcd1d39ff7d6178d8fcfed8e6a9 100644
--- a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog_v2.ts
+++ b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog_v2.ts
@@ -117,6 +117,7 @@ function getDataTypeLabel(datatypes: BrowsingDataType) {
       return loadTimeData.getString('clearDownloadHistory');
     case BrowsingDataType.HOSTED_APPS_DATA:
       return loadTimeData.getString('clearHostedAppData');
+    case BrowsingDataType.BRAVE_AI_CHAT: return loadTimeData.getString('clearBraveAIChat');
   }
 }
 
@@ -136,6 +137,7 @@ export function getDataTypePrefName(datatypes: BrowsingDataType) {
       return 'browser.clear_data.download_history';
     case BrowsingDataType.HOSTED_APPS_DATA:
       return 'browser.clear_data.hosted_apps_data';
+    case BrowsingDataType.BRAVE_AI_CHAT: return 'browser.clear_data.brave_leo';
   }
 }
 
