diff --git a/chrome/browser/resources/tools/optimize_webui.gni b/chrome/browser/resources/tools/optimize_webui.gni
index bf792a19368d0c1819658ff644ad43d7b69022cd..1930f1d7bc301027ae1d3cbed3ef7b4349d8f1b1 100644
--- a/chrome/browser/resources/tools/optimize_webui.gni
+++ b/chrome/browser/resources/tools/optimize_webui.gni
@@ -53,6 +53,8 @@ template("optimize_webui") {
                     root_build_dir)
     resources_path =
         rebase_path("$root_gen_dir/ui/webui/resources/tsc/", root_build_dir)
+    brave_resources_path =
+        rebase_path("$root_gen_dir/brave/ui/webui/resources/tsc/", root_build_dir)
 
     external_paths = []
     if (is_chromeos_ash) {
@@ -67,10 +69,12 @@ template("optimize_webui") {
 
     external_paths += [
       "chrome://resources/polymer/v3_0/|$polymer_path",
+      "chrome://resources/brave/|$brave_resources_path",
       "chrome://resources/|$resources_path",
 
       # Scheme relative versions to work with chrome-untrusted://
       "//resources/polymer/v3_0/|$polymer_path",
+      "//resources/brave/|$brave_resources_path",
       "//resources/|$resources_path",
     ]
 
