diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 84ea91c781a2f777461a4e99d1c8c2cdf138dc22..ca39af37e08065e1821fb5cffac92a34ec3e5e8f 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -483,6 +483,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
