diff --git a/chrome/browser/ui/extensions/icon_with_badge_image_source.h b/chrome/browser/ui/extensions/icon_with_badge_image_source.h
index 7e03e5e7f226ce0a05e481bff96bcc64675c8929..4febb67521a37768cf35c0aed7100651370e337d 100644
--- a/chrome/browser/ui/extensions/icon_with_badge_image_source.h
+++ b/chrome/browser/ui/extensions/icon_with_badge_image_source.h
@@ -54,6 +54,7 @@ class IconWithBadgeImageSource : public gfx::CanvasImageSource {
   bool paint_blocked_actions_decoration() const {
     return paint_blocked_actions_decoration_;
   }
+  BRAVE_ICON_WITH_BADGE_IMAGE_SOURCE_H_
 
  private:
   // gfx::CanvasImageSource:
