diff --git a/chrome/browser/ui/autofill/BUILD.gn b/chrome/browser/ui/autofill/BUILD.gn
index fe0fcbd705f7dc2b581c3e831ccfe764bb0d34b3..49ec6437c6e672c1884b6ac8868eaf2fe310b0cc 100644
--- a/chrome/browser/ui/autofill/BUILD.gn
+++ b/chrome/browser/ui/autofill/BUILD.gn
@@ -2,6 +2,8 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/ui/autofill/sources.gni")
+import("//brave/chromium_src/chrome/browser/ui/autofill/sources.gni")
 import("//build/config/chrome_build.gni")
 import("//build/config/ui.gni")
 import("//components/compose/features.gni")
@@ -255,6 +257,7 @@ source_set("impl") {
     "//components/variations/service",
     "//components/vector_icons",
   ]
+  deps += brave_chromium_src_chrome_browser_ui_autofill_deps
 
   if (safe_browsing_mode != 0) {
     deps += [ "//components/safe_browsing/content/browser/password_protection" ]
@@ -281,6 +284,7 @@ source_set("impl") {
       "//components/messages/android:feature_flags",
       "//components/password_manager/core/browser/features:password_features",
     ]
+    deps += brave_chrome_browser_ui_autofill_deps
   } else {
     sources += [
       "address_bubbles_controller.cc",
