diff --git a/chrome/browser/ui/cocoa/app_menu/app_menu_controller.mm b/chrome/browser/ui/cocoa/app_menu/app_menu_controller.mm
index 89f8b4bbba39c990b65c62159a00e51eff0b53ab..094303f8643eb1471a77a2b5e3e8e3d880b5ee7d 100644
--- a/chrome/browser/ui/cocoa/app_menu/app_menu_controller.mm
+++ b/chrome/browser/ui/cocoa/app_menu/app_menu_controller.mm
@@ -17,6 +17,7 @@
 #include "base/strings/string16.h"
 #include "base/strings/sys_string_conversions.h"
 #include "base/threading/thread_task_runner_handle.h"
+#include "brave/browser/ui/toolbar/brave_app_menu_model.h"
 #include "chrome/app/chrome_command_ids.h"
 #import "chrome/browser/app_controller_mac.h"
 #include "chrome/browser/profiles/profile.h"
@@ -495,7 +496,7 @@ - (BrowserActionsController*)browserActionsController {
 - (void)createModel {
   DCHECK(browser_);
   recentTabsMenuModelDelegate_.reset();
-  appMenuModel_.reset(new AppMenuModel(acceleratorDelegate_.get(), browser_));
+  appMenuModel_.reset(new BraveAppMenuModel(acceleratorDelegate_.get(), browser_));
   appMenuModel_->Init();
   [self setModel:appMenuModel_.get()];
 
