diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index e597b2eb93ba8758e9043e2aa7f93ddfa0d8a9b7..5d5b3fc487b2b1b12c03833d487f37ce2a9cd7db 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2472,6 +2472,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps defines += brave_chrome_browser_defines
 
   if (is_android) {
     sources += [
