diff --git a/chrome/android/java/src/org/chromium/chrome/browser/omnibox/suggestions/AutocompleteMediator.java b/chrome/android/java/src/org/chromium/chrome/browser/omnibox/suggestions/AutocompleteMediator.java
index 144eb574c67164106bfbb95dd22abb86bf9a3941..636d6d4cf94f6b5451c12756275a8baa0e2276aa 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/omnibox/suggestions/AutocompleteMediator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/omnibox/suggestions/AutocompleteMediator.java
@@ -235,7 +235,7 @@ class AutocompleteMediator implements OnSuggestionsReceivedListener, StartStopWi
         final Supplier<LargeIconBridge> iconBridgeSupplier = createIconBridgeSupplier();
 
         mEditUrlProcessor =
-                new EditUrlSuggestionProcessor(mContext, this, mDelegate, iconBridgeSupplier);
+                new org.chromium.chrome.browser.omnibox.suggestions.editurl.BraveEditUrlSuggestionProcessor(mContext, this, mDelegate, iconBridgeSupplier);
         registerSuggestionProcessor(new AnswerSuggestionProcessor(
                 mContext, this, mUrlBarEditingTextProvider, imageFetcherSupplier));
         registerSuggestionProcessor(
