diff --git a/chrome/browser/profiles/chrome_browser_main_extra_parts_profiles.cc b/chrome/browser/profiles/chrome_browser_main_extra_parts_profiles.cc
index b359e44ea989..d93311aff149 100644
--- a/chrome/browser/profiles/chrome_browser_main_extra_parts_profiles.cc
+++ b/chrome/browser/profiles/chrome_browser_main_extra_parts_profiles.cc
@@ -104,7 +104,7 @@
 #include "chrome/browser/android/explore_sites/explore_sites_service_factory.h"
 #include "chrome/browser/android/search_permissions/search_permissions_service.h"
 #include "chrome/browser/media/android/cdm/media_drm_origin_id_manager_factory.h"
-#else
+#endif
 #include "chrome/browser/apps/app_service/app_service_proxy_factory.h"
 #include "chrome/browser/feedback/feedback_uploader_factory_chrome.h"
 #include "chrome/browser/media/feeds/media_feeds_service_factory.h"
@@ -115,7 +115,6 @@
 #include "chrome/browser/ui/global_error/global_error_service_factory.h"
 #include "chrome/browser/ui/media_router/media_router_ui_service_factory.h"
 #include "chrome/browser/usb/usb_chooser_context_factory.h"
-#endif
 
 #if defined(OS_CHROMEOS)
 #include "chrome/browser/chromeos/browser_context_keyed_service_factories.h"
@@ -199,13 +198,23 @@ ChromeBrowserMainExtraPartsProfiles::~ChromeBrowserMainExtraPartsProfiles() {}
 //
 // static
 void ChromeBrowserMainExtraPartsProfiles::
+#if defined(OS_ANDROID)
+    EnsureBrowserContextKeyedServiceFactoriesBuilt(bool full_init) {
+#else
     EnsureBrowserContextKeyedServiceFactoriesBuilt() {
+#endif
 #if BUILDFLAG(ENABLE_EXTENSIONS)
+#if defined(OS_ANDROID)
+  if (full_init) {
+#endif
   apps::EnsureBrowserContextKeyedServiceFactoriesBuilt();
   chrome_apps::EnsureBrowserContextKeyedServiceFactoriesBuilt();
   chrome_apps::api::EnsureBrowserContextKeyedServiceFactoriesBuilt();
   chrome_extensions::EnsureBrowserContextKeyedServiceFactoriesBuilt();
   extensions::EnsureBrowserContextKeyedServiceFactoriesBuilt();
+#if defined(OS_ANDROID)
+  }
+#endif
 #endif
 #if defined(OS_CHROMEOS)
   chromeos::cert_provisioning::CertProvisioningSchedulerUserServiceFactory::
@@ -223,6 +232,9 @@ void ChromeBrowserMainExtraPartsProfiles::
 #endif
 #if !defined(OS_ANDROID)
   apps::AppServiceProxyFactory::GetInstance();
+#endif
+#if defined(OS_ANDROID)
+  if (full_init)
 #endif
   AutocompleteClassifierFactory::GetInstance();
   autofill::PersonalDataManagerFactory::GetInstance();
@@ -237,6 +249,9 @@ void ChromeBrowserMainExtraPartsProfiles::
   CaptivePortalServiceFactory::GetInstance();
 #endif
   CertificateReportingServiceFactory::GetInstance();
+#if defined(OS_ANDROID)
+  if (full_init)
+#endif
   ChromeBrowsingDataRemoverDelegateFactory::GetInstance();
   ChromeSigninClientFactory::GetInstance();
   ClientHintsFactory::GetInstance();
@@ -273,14 +288,17 @@ void ChromeBrowserMainExtraPartsProfiles::
 #endif
   GoogleSearchDomainMixingMetricsEmitterFactory::GetInstance();
   HistoryServiceFactory::GetInstance();
+#if defined(OS_ANDROID)
+  if (full_init)
+#endif
   HistoryUiFaviconRequestHandlerFactory::GetInstance();
   HostContentSettingsMapFactory::GetInstance();
   HttpsEngagementServiceFactory::GetInstance();
   IdentityManagerFactory::EnsureFactoryAndDependeeFactoriesBuilt();
   InMemoryURLIndexFactory::GetInstance();
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
   InstantServiceFactory::GetInstance();
-#endif
+//#endif
   LanguageModelManagerFactory::GetInstance();
 #if !defined(OS_ANDROID)
   LoginUIServiceFactory::GetInstance();
@@ -313,9 +331,9 @@ void ChromeBrowserMainExtraPartsProfiles::
   NTPResourceCacheFactory::GetInstance();
 #endif
   PasswordStoreFactory::GetInstance();
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
   PinnedTabServiceFactory::GetInstance();
-#endif
+//#endif
 #if BUILDFLAG(ENABLE_PLUGINS)
   PluginPrefsFactory::GetInstance();
 #endif
@@ -324,12 +342,18 @@ void ChromeBrowserMainExtraPartsProfiles::
   policy::UserCloudPolicyInvalidatorFactory::GetInstance();
 #if !defined(OS_CHROMEOS)
   policy::UserPolicySigninServiceFactory::GetInstance();
+#endif
+#if defined(OS_ANDROID)
+  if (full_init) {
 #endif
   predictors::AutocompleteActionPredictorFactory::GetInstance();
   predictors::LoadingPredictorFactory::GetInstance();
   predictors::PredictorDatabaseFactory::GetInstance();
   prerender::PrerenderLinkManagerFactory::GetInstance();
   prerender::PrerenderManagerFactory::GetInstance();
+#if defined(OS_ANDROID)
+  }
+#endif
   ProfileSyncServiceFactory::GetInstance();
   ProtocolHandlerRegistryFactory::GetInstance();
   RendererUpdaterFactory::GetInstance();
@@ -345,6 +369,9 @@ void ChromeBrowserMainExtraPartsProfiles::
   send_tab_to_self::SendTabToSelfClientServiceFactory::GetInstance();
 #if BUILDFLAG(ENABLE_SESSION_SERVICE)
   SessionServiceFactory::GetInstance();
+#endif
+#if defined(OS_ANDROID)
+  if (full_init)
 #endif
   SharingServiceFactory::GetInstance();
   ShortcutsBackendFactory::GetInstance();
@@ -359,6 +386,9 @@ void ChromeBrowserMainExtraPartsProfiles::
 #endif
   suggestions::SuggestionsServiceFactory::GetInstance();
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
+#if defined(OS_ANDROID)
+  if (full_init)
+#endif
   SupervisedUserServiceFactory::GetInstance();
 #endif
   TabRestoreServiceFactory::GetInstance();
@@ -382,8 +412,14 @@ void ChromeBrowserMainExtraPartsProfiles::
   UsbChooserContextFactory::GetInstance();
 #endif
 #if BUILDFLAG(ENABLE_EXTENSIONS)
+#if defined(OS_ANDROID)
+  if (full_init) {
+#endif
   web_app::WebAppMetricsFactory::GetInstance();
   web_app::WebAppProviderFactory::GetInstance();
+#if defined(OS_ANDROID)
+  }
+#endif
 #endif
   WebDataServiceFactory::GetInstance();
   webrtc_event_logging::WebRtcEventLogManagerKeyedServiceFactory::GetInstance();
