diff --git a/chrome/android/java/src/org/chromium/chrome/browser/search_engines/TemplateUrlService.java b/chrome/android/java/src/org/chromium/chrome/browser/search_engines/TemplateUrlService.java
index 55c72cd215f525a17853cd63f79ecb8b537d4102..3bec2e2c5aedcbf631aadc058c2167c1641a626d 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/search_engines/TemplateUrlService.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/search_engines/TemplateUrlService.java
@@ -11,6 +11,7 @@ import org.chromium.base.ThreadUtils;
 import org.chromium.base.VisibleForTesting;
 import org.chromium.base.annotations.CalledByNative;
 import org.chromium.base.task.PostTask;
+import org.chromium.chrome.browser.search_engines.BraveTemplateUrlService;
 import org.chromium.content_public.browser.UiThreadTaskTraits;
 
 import java.util.ArrayList;
@@ -49,7 +50,7 @@ public class TemplateUrlService {
     public static TemplateUrlService getInstance() {
         ThreadUtils.assertOnUiThread();
         if (sService == null) {
-            sService = new TemplateUrlService();
+            sService = new BraveTemplateUrlService();
         }
         return sService;
     }
@@ -59,7 +60,7 @@ public class TemplateUrlService {
     private final ObserverList<TemplateUrlServiceObserver> mObservers =
             new ObserverList<TemplateUrlServiceObserver>();
 
-    private TemplateUrlService() {
+    protected TemplateUrlService() {
         // Note that this technically leaks the native object, however, TemlateUrlService
         // is a singleton that lives forever and there's no clean shutdown of Chrome on Android
         mNativeTemplateUrlServiceAndroid = nativeInit();
