diff --git a/chrome/browser/resources/settings/people_page/sync_controls.ts b/chrome/browser/resources/settings/people_page/sync_controls.ts
index 1eff5bb2e3f2d1ce34d9209089f7e8ee1d0d993a..ef8c04a82bc98440ea749a725f1d581e2bdcba02 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.ts
+++ b/chrome/browser/resources/settings/people_page/sync_controls.ts
@@ -145,7 +145,7 @@ export class SettingsSyncControlsElement extends
 
     const router = Router.getInstance();
     const currentRoute = router.getCurrentRoute();
-    if (currentRoute === routes.SYNC_ADVANCED) {
+    if (currentRoute === routes.BRAVE_SYNC_SETUP) {
       this.browserProxy_.didNavigateToSyncPage();
     }
     // <if expr="not is_chromeos">
@@ -278,6 +278,7 @@ export class SettingsSyncControlsElement extends
   private disableTypeCheckBox_(
       syncStatus: SyncStatus, syncAllDataTypes: boolean,
       dataTypeManaged: boolean): boolean {
+    if (dataTypeManaged) { return true; } if (!syncStatus || !this.syncPrefs) { return syncAllDataTypes }
     if (!syncStatus) {
       return true;
     }
