diff --git a/chrome/browser/autocomplete/autocomplete_classifier_factory.cc b/chrome/browser/autocomplete/autocomplete_classifier_factory.cc
index 7f4dd5ffd15a0ea17fb77d712897a86370da4266..52d8469ae70f38e7a3c714f4761d4172915af138 100644
--- a/chrome/browser/autocomplete/autocomplete_classifier_factory.cc
+++ b/chrome/browser/autocomplete/autocomplete_classifier_factory.cc
@@ -68,7 +68,12 @@ AutocompleteClassifierFactory::~AutocompleteClassifierFactory() {
 
 content::BrowserContext* AutocompleteClassifierFactory::GetBrowserContextToUse(
     content::BrowserContext* context) const {
+#if defined(BRAVE_CHROMIUM_BUILD)
+  // See BraveAutocompleteProviderClient about why separate services are used.
+  return chrome::GetBrowserContextOwnInstanceInIncognito(context);
+#else
   return chrome::GetBrowserContextRedirectedInIncognito(context);
+#endif
 }
 
 bool AutocompleteClassifierFactory::ServiceIsNULLWhileTesting() const {
