diff --git a/chrome/browser/profiles/profile_avatar_icon_util.cc b/chrome/browser/profiles/profile_avatar_icon_util.cc
index 6e67b95040478f3534f21869890e217ac7406307..05edaab39f8391c3bcdd880148f2cefc869d498e 100644
--- a/chrome/browser/profiles/profile_avatar_icon_util.cc
+++ b/chrome/browser/profiles/profile_avatar_icon_util.cc
@@ -346,7 +346,7 @@ constexpr size_t kDefaultAvatarIconsCount = 1;
 #elif BUILDFLAG(IS_CHROMEOS_ASH)
 constexpr size_t kDefaultAvatarIconsCount = 27;
 #else
-constexpr size_t kDefaultAvatarIconsCount = 56;
+constexpr size_t kDefaultAvatarIconsCount = 56 + kBraveDefaultAvatarIconsCount;
 #endif
 
 #if !defined(OS_ANDROID)
@@ -459,6 +459,7 @@ size_t GetPlaceholderAvatarIndex() {
 
 size_t GetModernAvatarIconStartIndex() {
 #if !BUILDFLAG(IS_CHROMEOS_ASH) && !defined(OS_ANDROID)
+  BRAVE_GET_MODERN_AVATAR_ICON_START_INDEX
   return GetPlaceholderAvatarIndex() + 1;
 #else
   // Only use the placeholder avatar on ChromeOS and Android.
@@ -492,6 +493,7 @@ std::string GetPlaceholderAvatarIconUrl() {
 
 const IconResourceInfo* GetDefaultAvatarIconResourceInfo(size_t index) {
   CHECK_LT(index, kDefaultAvatarIconsCount);
+  BRAVE_GET_DEFAULT_AVATAR_ICON_RESOURCE_INFO
   static const IconResourceInfo resource_info[kDefaultAvatarIconsCount] = {
   // Old avatar icons:
 #if !defined(OS_ANDROID)
