diff --git a/chrome/browser/importer/importer_uma.cc b/chrome/browser/importer/importer_uma.cc
index da87e84ffe7c2193ea16d24f1ee1174c51c9503f..fb80acdfd3997df06189334a8575f39431bf9bd9 100644
--- a/chrome/browser/importer/importer_uma.cc
+++ b/chrome/browser/importer/importer_uma.cc
@@ -59,6 +59,12 @@ void LogImporterUseToMetrics(const std::string& metric_postfix,
     case TYPE_BOOKMARKS_FILE:
       metrics_type = IMPORTER_METRICS_BOOKMARKS_FILE;
       break;
+    case TYPE_CHROME:
+      // TODO: Wire this up if we want to record metrics on users who import from Chrome
+      break;
+    case TYPE_BRAVE:
+      // TODO: Wire this up if we want to record metrics on users who import from Brave
+      break;
   }
 
   // Note: This leaks memory, which is the expected behavior as the factory
