diff --git a/chrome/browser/prefs/browser_prefs.cc b/chrome/browser/prefs/browser_prefs.cc
index 1a660729062c09f3a3152db76e4a87b76e7ff039..727c13fd582b04bf53592b44f82e6107e62bb4e8 100644
--- a/chrome/browser/prefs/browser_prefs.cc
+++ b/chrome/browser/prefs/browser_prefs.cc
@@ -1204,6 +1204,7 @@ void RegisterLocalState(PrefRegistrySimple* registry) {
 
   registry->RegisterBooleanPref(
       policy::policy_prefs::kSetTimeoutWithout1MsClampEnabled, false);
+  brave::RegisterLocalStatePrefs(registry);
 
 #if BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_CHROMEOS)
   screen_ai::RegisterLocalStatePrefs(registry);
@@ -1539,6 +1540,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry,
                                 true);
 
   RegisterProfilePrefsForMigration(registry);
+  brave::RegisterProfilePrefs(registry);
 }
 
 void RegisterUserProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
