diff --git a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.ts b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.ts
index 4c5d092d29a9e282575f1da99acfe1e7879bc32b..f318ce23cfee8eb19f82bfa14b5f8fb87670ce59 100644
--- a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.ts
+++ b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.ts
@@ -80,10 +80,14 @@ export interface SettingsClearBrowsingDataDialogElement {
  * The list of all available Browsing Data types in the default order they
  * should appear in the dialog.
  */
+export
 const ALL_BROWSING_DATATYPES_LIST: BrowsingDataType[] = [
   BrowsingDataType.HISTORY,
   BrowsingDataType.SITE_DATA,
   BrowsingDataType.CACHE,
+// <if expr="enable_ai_chat">
+  BrowsingDataType.BRAVE_AI_CHAT,
+// </if>
   BrowsingDataType.DOWNLOADS,
   BrowsingDataType.FORM_DATA,
   BrowsingDataType.SITE_SETTINGS,
@@ -97,12 +101,14 @@ const DEFAULT_BROWSING_DATATYPES_LIST: BrowsingDataType[] = [
   BrowsingDataType.CACHE,
 ];
 
+export
 interface BrowsingDataTypeOption {
   label: string;
   subLabel?: string;
   prefKey: string;
 }
 
+export
 function getDataTypeLabel(datatypes: BrowsingDataType) {
   switch (datatypes) {
     case BrowsingDataType.HISTORY:
@@ -119,6 +125,9 @@ function getDataTypeLabel(datatypes: BrowsingDataType) {
       return loadTimeData.getString('clearDownloadHistory');
     case BrowsingDataType.HOSTED_APPS_DATA:
       return loadTimeData.getString('clearHostedAppData');
+// <if expr="enable_ai_chat">
+    case BrowsingDataType.BRAVE_AI_CHAT: return loadTimeData.getString('aiChatClearHistoryData');
+// </if>
     default:
       assertNotReachedCase(datatypes);
   }
@@ -140,6 +149,9 @@ export function getDataTypePrefName(datatypes: BrowsingDataType) {
       return 'browser.clear_data.download_history';
     case BrowsingDataType.HOSTED_APPS_DATA:
       return 'browser.clear_data.hosted_apps_data';
+// <if expr="enable_ai_chat">
+    case BrowsingDataType.BRAVE_AI_CHAT: return 'browser.clear_data.brave_leo';
+// </if>
     default:
       assertNotReachedCase(datatypes);
   }
@@ -515,6 +527,3 @@ declare global {
   }
 }
 
-customElements.define(
-    SettingsClearBrowsingDataDialogElement.is,
-    SettingsClearBrowsingDataDialogElement);
