diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 4f11cdb016fd53b0c7b06ea209a8ffed9fcc9bc9..426f252310e45b10f6651445f13c8928f04e67e1 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -65,6 +65,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
