diff --git a/chrome/browser/extensions/extension_service.h b/chrome/browser/extensions/extension_service.h
index 6cd67ffa63e073791359bf93811f9be74259ca91..eb4165f293b476028706e823173e486ecfda20a6 100644
--- a/chrome/browser/extensions/extension_service.h
+++ b/chrome/browser/extensions/extension_service.h
@@ -457,6 +457,7 @@ class ExtensionService : public ExtensionServiceInterface,
   static base::AutoReset<bool> DisableExternalUpdatesForTesting();
 
  private:
+  friend class BraveExtensionService;
   // Loads extensions specified via a command line flag/switch.
   void LoadExtensionsFromCommandLineFlag(const char* switch_name);
 #if BUILDFLAG(IS_CHROMEOS_ASH)
