diff --git a/chrome/browser/policy/profile_policy_connector.cc b/chrome/browser/policy/profile_policy_connector.cc
index cc4e20ff2a998c696fd22566ff71f777388a3f2d..98b510e6a87a1a7c59c4db994fc34abfcab015ed 100644
--- a/chrome/browser/policy/profile_policy_connector.cc
+++ b/chrome/browser/policy/profile_policy_connector.cc
@@ -548,6 +548,7 @@ void ProfilePolicyConnector::Init(
   }
 #endif
 
+  BRAVE_PROFILE_POLICY_CONNECTOR_INIT
   std::vector<std::unique_ptr<PolicyMigrator>> migrators;
 #if BUILDFLAG(IS_WIN)
   migrators.push_back(
