diff --git a/chrome/browser/renderer_host/chrome_navigation_ui_data.cc b/chrome/browser/renderer_host/chrome_navigation_ui_data.cc
index f6c02c2dc40abf9d882a6ac6044c35b3a7c4d745..6d9a308fda0aa0d521979dff6226b12a08c8b87e 100644
--- a/chrome/browser/renderer_host/chrome_navigation_ui_data.cc
+++ b/chrome/browser/renderer_host/chrome_navigation_ui_data.cc
@@ -52,7 +52,7 @@ ChromeNavigationUIData::CreateForMainFrameNavigation(
     content::WebContents* web_contents,
     WindowOpenDisposition disposition,
     int64_t data_reduction_proxy_page_id) {
-  auto navigation_ui_data = std::make_unique<ChromeNavigationUIData>();
+  auto navigation_ui_data = std::make_unique<BraveNavigationUIData>();
   navigation_ui_data->disposition_ = disposition;
   navigation_ui_data->data_reduction_proxy_page_id_ =
       data_reduction_proxy_page_id;
@@ -72,7 +72,7 @@ ChromeNavigationUIData::CreateForMainFrameNavigation(
 }
 
 std::unique_ptr<content::NavigationUIData> ChromeNavigationUIData::Clone() {
-  auto copy = std::make_unique<ChromeNavigationUIData>();
+  auto copy = std::make_unique<BraveNavigationUIData>();
 
   copy->disposition_ = disposition_;
   copy->data_reduction_proxy_page_id_ = data_reduction_proxy_page_id_;
