diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index 9698a4f43d0e1b027a8fa36f1accf4aa50c2d0d6..ae96b1709f46dc2b2afb69253946b6c0a3d6e017 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -26,7 +26,9 @@ cr.define('settings', function() {
 
     // TODO(tommycli): Find a way to refactor these repetitive category
     // routes.
+      // <if expr="_google_chrome">
     r.SITE_SETTINGS_ADS = r.SITE_SETTINGS.createChild('ads');
+      // </if>
     if (loadTimeData.getBoolean('enableWebXrContentSetting')) {
       r.SITE_SETTINGS_AR = r.SITE_SETTINGS.createChild('ar');
     }
@@ -161,7 +163,9 @@ cr.define('settings', function() {
       }
 
       r.PRINTING = r.ADVANCED.createSection('/printing', 'printing');
+      // <if expr="_google_chrome">
       r.CLOUD_PRINTERS = r.PRINTING.createChild('/cloudPrinters');
+      // </if>
 
       r.ACCESSIBILITY = r.ADVANCED.createSection('/accessibility', 'a11y');
 
