diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index 6608fbd987e33b3ee3a6ceb584365b75b1449f40..820c2b22d5685bd9551f234747c3adf5e7e218c4 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -554,7 +554,7 @@ Browser* Browser::Create(const CreateParams& params) {
   // should handle this; see e.g. crbug.com/1141608 and crbug.com/1261628.
   CHECK_EQ(CreationStatus::kOk, GetCreationStatusForProfile(params.profile));
 
-  std::unique_ptr<Browser> browser = base::WrapUnique(new Browser(params));
+  std::unique_ptr<Browser> browser = base::WrapUnique(new BraveBrowser(params));
   Browser* const browser_ptr = browser.get();
   BrowserManagerServiceFactory::GetForProfile(params.profile)
       ->AddBrowser(std::move(browser));
@@ -579,7 +579,7 @@ Browser::Browser(const CreateParams& params)
       window_(nullptr),
       tab_strip_model_delegate_(
           std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
-      tab_strip_model_(std::make_unique<TabStripModel>(
+      tab_strip_model_(std::make_unique<BraveTabStripModel>(
           tab_strip_model_delegate_.get(),
           params.profile,
           // Tab groups are disabled for app browsers.
@@ -1344,6 +1344,7 @@ void Browser::OnWindowClosing() {
   if (tab_restore_service && notify_restore_service) {
     tab_restore_service->BrowserClosing(GetFeatures().live_tab_context());
   }
+  BRAVE_BROWSER_ON_WINDOW_CLOSING
 
   if (!tab_strip_model_->empty()) {
     // Closing all the tabs results in eventually calling back to
