diff --git a/chrome/browser/ui/webui/devtools_ui.cc b/chrome/browser/ui/webui/devtools_ui.cc
index 55c03154ee2331eca251ed5ebfa1af3ce7d483c9..c63526a68758ecefb3fff2e802efe0db04154f77 100644
--- a/chrome/browser/ui/webui/devtools_ui.cc
+++ b/chrome/browser/ui/webui/devtools_ui.cc
@@ -270,6 +270,7 @@ void DevToolsDataSource::StartBundledDataRequest(
 void DevToolsDataSource::StartRemoteDataRequest(
     const GURL& url,
     const content::URLDataSource::GotDataCallback& callback) {
+  return; // feature disabled in Brave
   CHECK(url.is_valid());
   net::NetworkTrafficAnnotationTag traffic_annotation =
       net::DefineNetworkTrafficAnnotation("devtools_hard_coded_data_source", R"(
