diff --git a/chrome/installer/linux/common/installer.include b/chrome/installer/linux/common/installer.include
index d6f17907c9bb0c4ea59530155ca2788f8876bb83..fcc384188f10656d5d2d0744c328043c58ace7a6 100644
--- a/chrome/installer/linux/common/installer.include
+++ b/chrome/installer/linux/common/installer.include
@@ -141,6 +141,8 @@ stage_install_common() {
     install -m 644 "${OUTPUTDIR}/ui_resources_100_percent.pak" "${STAGEDIR}/${INSTALLDIR}/"
   fi
 
+  source "${OUTPUTDIR}/installer/common/brave_installer.include"
+  brave_stage_install_common
   # ICU data file; Necessary when the GN icu_use_data_file flag is true.
   install -m 644 "${OUTPUTDIR}/icudtl.dat" "${STAGEDIR}/${INSTALLDIR}/"
 
@@ -279,13 +281,15 @@ stage_install_common() {
 
   # app icons
   local icon_regex=".*product_logo_[0-9]\+\."
-  if [ "$BRANDING" = "google_chrome" ]; then
+  if [ "$BRANDING" = "brave" ]; then
     if [ "$CHANNEL" = "beta" ]; then
       icon_regex=".*product_logo_[0-9]\+_beta\."
-    elif [ "$CHANNEL" = "unstable" ]; then
+    elif [ "$CHANNEL" = "unstable" -o "$CHANNEL" = "dev" ]; then
       icon_regex=".*product_logo_[0-9]\+_dev\."
     elif [ "$CHANNEL" = "canary" ]; then
       icon_regex=".*product_logo_[0-9]\+_canary\."
+    elif [ "$CHANNEL" = "nightly" ]; then
+      icon_regex=".*product_logo_[0-9]\+_nightly\."
     fi
   fi
   LOGO_RESOURCES_PNG=$(find "${OUTPUTDIR}/installer/theme/" \
