diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 4800f267f10a186b675bba286ec53770db4ebe5d..f43a225583be006612262ceda928a4fcafc2bf55 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -1288,6 +1288,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
