diff --git a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h
index 4125cdd3752153274c64f878f5c3667cab3e5672..a1f7a6c90c8adc6e12357d2e7030bc0f58bd1c75 100644
--- a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h
+++ b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h
@@ -135,7 +135,7 @@ class TabDragController : public views::WidgetObserver,
   // call, and Liveness::ALIVE if `this` still exists.
   // Note: `dragging_views` must be ordered by their position in the source
   // tabstrip (both visually and in the model).
-  [[nodiscard]] Liveness Init(TabDragContext* source_context,
+  [[nodiscard]] virtual Liveness Init(TabDragContext* source_context,
                               TabSlotView* source_view,
                               const std::vector<TabSlotView*>& dragging_views,
                               const gfx::Point& offset_from_source_view,
