diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 71c55fc44dc45fbf2fecd24b4fe326137424caaf..39c70bc6a2e7f28ee6b2cbce20c9bb0cb6ad0f99 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -603,6 +603,7 @@ static_library("ui") {
     "//ui/webui/resources/cr_components/most_visited:mojom",
     "//v8:v8_version",
   ]
+  deps+= [ "//brave/browser/ui" ]
 
   # TODO(crbug/925153): Remove this circular dependency.
   allow_circular_includes_from = [
@@ -3412,10 +3413,13 @@ static_library("ui") {
       ]
       deps += [ "//google_update" ]
     } else {
-      sources += [
-        "webui/help/version_updater_basic.cc",
-        "webui/help/version_updater_basic.h",
-      ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [
+          "webui/help/version_updater_basic.cc",
+          "webui/help/version_updater_basic.h",
+        ]
+      }
     }
   } else {  # 'OS!="win"
     # The Cast Linux build sets toolkit_views to false, but many files in this
