diff --git a/chrome/browser/ui/views/frame/browser_view.h b/chrome/browser/ui/views/frame/browser_view.h
index 6eea8b4b899e2225c23043c46e780a5a881cf539..1e36c947cc3b3def192aa2efd0ee616893042053 100644
--- a/chrome/browser/ui/views/frame/browser_view.h
+++ b/chrome/browser/ui/views/frame/browser_view.h
@@ -705,6 +705,7 @@ class BrowserView : public BrowserWindow,
       const std::map<std::string, std::string>& extra_data) override;
 #endif
 
+  BRAVE_BROWSER_VIEW_H
  private:
   // Do not friend BrowserViewLayout. Use the BrowserViewLayoutDelegate
   // interface to keep these two classes decoupled and testable.
