diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index b772d3efd519046e2c800d3ccd4880f27f9802f1..42d56e3234d0886b57a621e783ff02d97b0de39c 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -222,6 +222,7 @@ android_resources("chrome_app_java_resources") {
     "//third_party/android_deps:androidx_recyclerview_recyclerview_java",
     "//third_party/android_deps:material_design_java",
   ]
+  deps += brave_chrome_app_java_resources_deps sources += brave_java_resources
   create_srcjar = false
 }
 
@@ -501,6 +502,7 @@ android_library("chrome_java") {
   ]
 
   deps += feed_deps
+  deps += brave_chrome_java_deps
 
   srcjar_deps = [
     ":chrome_android_java_enums_srcjar",
@@ -538,6 +540,7 @@ android_library("chrome_java") {
     "//content/public/browser:contacts_picker_properties_requested_javagen",
     "//net:effective_connection_type_java",
   ]
+  srcjar_deps += brave_chrome_java_srcjar_deps
 
   # From java_sources.gni.
   sources = chrome_java_sources + [
@@ -661,6 +664,7 @@ group("jni_headers") {
     "//chrome/browser/ui/messages/android:jni_headers",
     "//chrome/browser/util:jni_headers",
   ]
+  public_deps += ["//brave/build/android:jni_headers"]
 }
 
 action("chrome_android_java_google_api_keys_srcjar") {
@@ -1710,6 +1714,7 @@ if (current_toolchain == default_toolchain) {
         ":${_variant}_locale_pak_assets",
         ":${_variant}_paks",
       ]
+      deps += [ "//brave:brave_pak_assets" ]
       if (_is_monochrome) {
         deps += [ "//android_webview:locale_pak_assets" ]
         if (webview_includes_weblayer && !_is_bundle_module) {
