diff --git a/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc b/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
index 9a5cae665106ed5c82005da90c4d125238089018..d306d3edd8a4b6fc6a5cf90b44a4979a4cf9cd79 100644
--- a/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
+++ b/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
@@ -332,6 +332,7 @@ void CheckClientDownloadRequestBase::OnRequestBuilt(
   client_download_request_ = std::move(request);
   request_creation_details_ = details;
   SanitizeRequest();
+  BRAVE_ON_REQUEST_BUILT_FILTER
 
   // If it's an archive with no archives or executables, finish early.
   if ((client_download_request_->download_type() ==
