diff --git a/chrome/browser/ui/views/tabs/tab.h b/chrome/browser/ui/views/tabs/tab.h
index 6cd517dd3706c663a2092e9dc0de0e5229b320a9..2651092d68a43f7476e0c8bf4bbacc60b4429159 100644
--- a/chrome/browser/ui/views/tabs/tab.h
+++ b/chrome/browser/ui/views/tabs/tab.h
@@ -301,7 +301,7 @@ class Tab : public gfx::AnimationDelegate,
 
   // Whether we are showing the close button. It is cached so that we can
   // detect when it changes and layout appropriately.
-  bool showing_close_button_ = false;
+  ControllableCloseButtonState showing_close_button_{*controller_, *this};
 
   // Whether the tab is currently animating from a pinned to an unpinned state.
   bool is_animating_from_pinned_ = false;
