diff --git a/chrome/renderer/worker_content_settings_client.h b/chrome/renderer/worker_content_settings_client.h
index 7b167b87eda23b180ae7948a6aefb37e39456e1a..44b0424a8ba59bcfea377ad8196b522a9e52baa3 100644
--- a/chrome/renderer/worker_content_settings_client.h
+++ b/chrome/renderer/worker_content_settings_client.h
@@ -38,6 +38,7 @@ class WorkerContentSettingsClient : public blink::WebContentSettingsClient {
   bool AllowScriptFromSource(bool enabled_per_settings,
                              const blink::WebURL& script_url) override;
   bool ShouldAutoupgradeMixedContent() override;
+  BRAVE_WORKER_CONTENT_SETTINGS_CLIENT_H
 
  private:
   explicit WorkerContentSettingsClient(
