diff --git a/chrome/browser/ui/toasts/toast_controller_unittest.cc b/chrome/browser/ui/toasts/toast_controller_unittest.cc
index 387ee7597e9b38b67e776b0c8bca6c75896fb0db..b435124c26a5dbae8e84418bd18b539b55d9cbcc 100644
--- a/chrome/browser/ui/toasts/toast_controller_unittest.cc
+++ b/chrome/browser/ui/toasts/toast_controller_unittest.cc
@@ -70,7 +70,7 @@ class ToastControllerUnitTest : public testing::Test {
 TEST_F(ToastControllerUnitTest, ShowToast) {
   ToastRegistry* const registry = toast_registry();
   registry->RegisterToast(
-      ToastId::kLinkCopied,
+      ToastId::kLinkToHighlightCopied,
       ToastSpecification::Builder(features::IsRoundedIconsEnabled()
                                       ? vector_icons::kMailFilledIcon
                                       : vector_icons::kEmailOldIcon,
@@ -81,19 +81,19 @@ TEST_F(ToastControllerUnitTest, ShowToast) {
 
   // We should be able to show the toast because there is no toast showing.
   EXPECT_FALSE(controller->IsShowingToast());
-  EXPECT_TRUE(controller->CanShowToast(ToastId::kLinkCopied));
+  EXPECT_TRUE(controller->CanShowToast(ToastId::kLinkToHighlightCopied));
 
   EXPECT_CALL(*controller, CreateToast);
-  EXPECT_TRUE(controller->MaybeShowToast(ToastParams(ToastId::kLinkCopied)));
+  EXPECT_TRUE(controller->MaybeShowToast(ToastParams(ToastId::kLinkToHighlightCopied)));
   ::testing::Mock::VerifyAndClear(controller.get());
   EXPECT_TRUE(controller->IsShowingToast());
-  EXPECT_TRUE(controller->CanShowToast(ToastId::kLinkCopied));
+  EXPECT_TRUE(controller->CanShowToast(ToastId::kLinkToHighlightCopied));
 }
 
 TEST_F(ToastControllerUnitTest, ShowToastWithBodyStringOverride) {
   ToastRegistry* const registry = toast_registry();
   registry->RegisterToast(
-      ToastId::kLinkCopied,
+      ToastId::kLinkToHighlightCopied,
       ToastSpecification::Builder(features::IsRoundedIconsEnabled()
                                       ? vector_icons::kMailFilledIcon
                                       : vector_icons::kEmailOldIcon)
@@ -103,23 +103,23 @@ TEST_F(ToastControllerUnitTest, ShowToastWithBodyStringOverride) {
 
   // We should be able to show the toast because there is no toast showing.
   EXPECT_FALSE(controller->IsShowingToast());
-  EXPECT_TRUE(controller->CanShowToast(ToastId::kLinkCopied));
+  EXPECT_TRUE(controller->CanShowToast(ToastId::kLinkToHighlightCopied));
 
   EXPECT_CALL(*controller, CreateToast);
 
-  ToastParams params = ToastParams(ToastId::kLinkCopied);
+  ToastParams params = ToastParams(ToastId::kLinkToHighlightCopied);
   params.body_string_override = u"Some toast body";
 
   EXPECT_TRUE(controller->MaybeShowToast(std::move(params)));
   ::testing::Mock::VerifyAndClear(controller.get());
   EXPECT_TRUE(controller->IsShowingToast());
-  EXPECT_TRUE(controller->CanShowToast(ToastId::kLinkCopied));
+  EXPECT_TRUE(controller->CanShowToast(ToastId::kLinkToHighlightCopied));
 }
 
 TEST_F(ToastControllerUnitTest, ShowToastWithImage) {
   ToastRegistry* const registry = toast_registry();
   registry->RegisterToast(
-      ToastId::kLinkCopied,
+      ToastId::kLinkToHighlightCopied,
       ToastSpecification::Builder(features::IsRoundedIconsEnabled()
                                       ? vector_icons::kMailFilledIcon
                                       : vector_icons::kEmailOldIcon,
@@ -130,24 +130,24 @@ TEST_F(ToastControllerUnitTest, ShowToastWithImage) {
 
   // We should be able to show the toast because there is no toast showing.
   EXPECT_FALSE(controller->IsShowingToast());
-  EXPECT_TRUE(controller->CanShowToast(ToastId::kLinkCopied));
+  EXPECT_TRUE(controller->CanShowToast(ToastId::kLinkToHighlightCopied));
 
   EXPECT_CALL(*controller, CreateToast);
 
-  ToastParams params = ToastParams(ToastId::kLinkCopied);
+  ToastParams params = ToastParams(ToastId::kLinkToHighlightCopied);
   params.image_override =
       ui::ImageModel::FromImage(gfx::test::CreateImage(16, 16, 0xff0000));
 
   EXPECT_TRUE(controller->MaybeShowToast(std::move(params)));
   ::testing::Mock::VerifyAndClear(controller.get());
   EXPECT_TRUE(controller->IsShowingToast());
-  EXPECT_TRUE(controller->CanShowToast(ToastId::kLinkCopied));
+  EXPECT_TRUE(controller->CanShowToast(ToastId::kLinkToHighlightCopied));
 }
 
 TEST_F(ToastControllerUnitTest, ToastAutomaticallyCloses) {
   ToastRegistry* const registry = toast_registry();
   registry->RegisterToast(
-      ToastId::kLinkCopied,
+      ToastId::kLinkToHighlightCopied,
       ToastSpecification::Builder(features::IsRoundedIconsEnabled()
                                       ? vector_icons::kMailFilledIcon
                                       : vector_icons::kEmailOldIcon,
@@ -156,7 +156,7 @@ TEST_F(ToastControllerUnitTest, ToastAutomaticallyCloses) {
   auto controller = std::make_unique<TestToastController>(registry);
 
   EXPECT_CALL(*controller, CreateToast);
-  EXPECT_TRUE(controller->MaybeShowToast(ToastParams(ToastId::kLinkCopied)));
+  EXPECT_TRUE(controller->MaybeShowToast(ToastParams(ToastId::kLinkToHighlightCopied)));
   ::testing::Mock::VerifyAndClear(controller.get());
   EXPECT_TRUE(controller->IsShowingToast());
 
@@ -168,7 +168,7 @@ TEST_F(ToastControllerUnitTest, ToastAutomaticallyCloses) {
 TEST_F(ToastControllerUnitTest, ToastCloseCallbackTriggered) {
   ToastRegistry* const registry = toast_registry();
   registry->RegisterToast(
-      ToastId::kLinkCopied,
+      ToastId::kLinkToHighlightCopied,
       ToastSpecification::Builder(features::IsRoundedIconsEnabled()
                                       ? vector_icons::kMailFilledIcon
                                       : vector_icons::kEmailOldIcon,
@@ -177,7 +177,7 @@ TEST_F(ToastControllerUnitTest, ToastCloseCallbackTriggered) {
   auto controller = std::make_unique<TestToastController>(registry);
 
   EXPECT_CALL(*controller, CreateToast);
-  ToastParams params = ToastParams(ToastId::kLinkCopied);
+  ToastParams params = ToastParams(ToastId::kLinkToHighlightCopied);
   bool callback_called = false;
   params.toast_close_callback = base::ScopedClosureRunner(
       base::BindOnce([](bool* callback_called) { *callback_called = true; },
@@ -196,7 +196,7 @@ TEST_F(ToastControllerUnitTest, ToastCloseCallbackTriggered) {
 TEST_F(ToastControllerUnitTest, ToastWithActionButtonAutomaticallyCloses) {
   ToastRegistry* const registry = toast_registry();
   registry->RegisterToast(
-      ToastId::kLinkCopied,
+      ToastId::kLinkToHighlightCopied,
       ToastSpecification::Builder(features::IsRoundedIconsEnabled()
                                       ? vector_icons::kMailFilledIcon
                                       : vector_icons::kEmailOldIcon,
@@ -205,7 +205,7 @@ TEST_F(ToastControllerUnitTest, ToastWithActionButtonAutomaticallyCloses) {
   auto controller = std::make_unique<TestToastController>(registry);
 
   EXPECT_CALL(*controller, CreateToast);
-  EXPECT_TRUE(controller->MaybeShowToast(ToastParams(ToastId::kLinkCopied)));
+  EXPECT_TRUE(controller->MaybeShowToast(ToastParams(ToastId::kLinkToHighlightCopied)));
   ::testing::Mock::VerifyAndClear(controller.get());
   EXPECT_TRUE(controller->IsShowingToast());
 
@@ -217,14 +217,14 @@ TEST_F(ToastControllerUnitTest, ToastWithActionButtonAutomaticallyCloses) {
 TEST_F(ToastControllerUnitTest, CloseTimerResetsWhenToastShown) {
   ToastRegistry* const registry = toast_registry();
   registry->RegisterToast(
-      ToastId::kLinkCopied,
+      ToastId::kLinkToHighlightCopied,
       ToastSpecification::Builder(features::IsRoundedIconsEnabled()
                                       ? vector_icons::kMailFilledIcon
                                       : vector_icons::kEmailOldIcon,
                                   kTestStringResId)
           .Build());
   registry->RegisterToast(
-      ToastId::kImageCopied,
+      ToastId::kClearBrowsingData,
       ToastSpecification::Builder(features::IsRoundedIconsEnabled()
                                       ? vector_icons::kMailFilledIcon
                                       : vector_icons::kEmailOldIcon,
@@ -234,7 +234,7 @@ TEST_F(ToastControllerUnitTest, CloseTimerResetsWhenToastShown) {
   auto controller = std::make_unique<TestToastController>(registry);
 
   EXPECT_CALL(*controller, CreateToast);
-  EXPECT_TRUE(controller->MaybeShowToast(ToastParams(ToastId::kLinkCopied)));
+  EXPECT_TRUE(controller->MaybeShowToast(ToastParams(ToastId::kLinkToHighlightCopied)));
   ::testing::Mock::VerifyAndClear(controller.get());
   EXPECT_TRUE(controller->IsShowingToast());
 
@@ -245,7 +245,7 @@ TEST_F(ToastControllerUnitTest, CloseTimerResetsWhenToastShown) {
 
   // Show a different toast before the link copied toast times out.
   EXPECT_CALL(*controller, CreateToast);
-  EXPECT_TRUE(controller->MaybeShowToast(ToastParams(ToastId::kImageCopied)));
+  EXPECT_TRUE(controller->MaybeShowToast(ToastParams(ToastId::kClearBrowsingData)));
   ::testing::Mock::VerifyAndClear(controller.get());
   EXPECT_TRUE(controller->IsShowingToast());
 
