diff --git a/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_device_picker_bubble_view.cc b/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_device_picker_bubble_view.cc
index 0bfb3454d827ac708205dc654bef8c86efcb6638..4ed223e8acb12494eb55c0bffd72d70e0089566f 100644
--- a/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_device_picker_bubble_view.cc
+++ b/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_device_picker_bubble_view.cc
@@ -126,6 +126,7 @@ void SendTabToSelfDevicePickerBubbleView::Init() {
   CreateHintTextLabel();
   CreateDevicesScrollView();
 
+  BRAVE_SEND_TAB_TO_SELFDEVICE_PICKER_BUBBLE_VIEW_INIT
   AddChildView(std::make_unique<views::Separator>());
   views::View* footer = AddChildView(
       BuildManageAccountDevicesLinkView(/*show_link=*/true, controller_));
