diff --git a/chrome/browser/themes/theme_helper_win.h b/chrome/browser/themes/theme_helper_win.h
index 648620d8315d6960736afa95426c04a99f8f2c70..410a6916f17ac82f23411e3bda31ff53c8b9cd56 100644
--- a/chrome/browser/themes/theme_helper_win.h
+++ b/chrome/browser/themes/theme_helper_win.h
@@ -12,6 +12,8 @@ class ThemeHelperWin : public ThemeHelper {
   ThemeHelperWin() = default;
   ~ThemeHelperWin() override = default;
 
+  BRAVE_THEME_HELPER_WIN_H_
+
   // ThemeService:
   bool ShouldUseNativeFrame(
       const CustomThemeSupplier* theme_supplier) const override;
