diff --git a/chrome/browser/first_run/upgrade_util_win.cc b/chrome/browser/first_run/upgrade_util_win.cc
index 734c88f07a8c3166d86d14a868d24889542bfed3..5313f9462e7a2e6a5bf8bc0fac8ce1bdc0401cd4 100644
--- a/chrome/browser/first_run/upgrade_util_win.cc
+++ b/chrome/browser/first_run/upgrade_util_win.cc
@@ -58,7 +58,7 @@
 #include "third_party/abseil-cpp/absl/cleanup/cleanup.h"
 #include "ui/base/ui_base_switches.h"
 
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OFFICIAL_BUILD)
 #include "chrome/updater/app/server/win/updater_legacy_idl.h"
 #endif
 
@@ -71,7 +71,7 @@ bool GetNewerChromeFile(base::FilePath* path) {
   return true;
 }
 
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OFFICIAL_BUILD)
 // Holds the result of the IPC to CoCreate the process launcher.
 struct CreateProcessLauncherResult
     : public base::RefCountedThreadSafe<CreateProcessLauncherResult> {
@@ -211,7 +211,7 @@ Microsoft::WRL::ComPtr<IUnknown> CreateProcessLauncher() {
 #endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
 
 bool InvokeGoogleUpdateForRename() {
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OFFICIAL_BUILD)
   // This has been identified as very slow on some startups. Detailed trace
   // events below try to shine a light on each steps. crbug.com/1252004
   TRACE_EVENT0("startup", "upgrade_util::InvokeGoogleUpdateForRename");
