diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 71659fad1428689497a591e5733f7077eb11ad49..b1e287fa0cc47aa213dc38f9c78dab55c2c4bf8b 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2488,6 +2488,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
