diff --git a/chrome/browser/extensions/extension_action_manager.cc b/chrome/browser/extensions/extension_action_manager.cc
index 81278a6c02b324c7fc286ecb419aa2bc84cfc04a..bcd2e82b3686eabaeba855493333df82f8bf29e9 100644
--- a/chrome/browser/extensions/extension_action_manager.cc
+++ b/chrome/browser/extensions/extension_action_manager.cc
@@ -109,6 +109,7 @@ ExtensionAction* ExtensionActionManager::GetExtensionAction(
         ExtensionAction::ActionIconSize(),
         ExtensionAction::FallbackIcon().AsImageSkia(), nullptr));
   }
+  BRAVE_GET_EXTENSION_ACTION
 
   ExtensionAction* raw_action = action.get();
   actions_[extension.id()] = std::move(action);
