diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/themes/ThemeSettingsFragment.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/themes/ThemeSettingsFragment.java
index 972a8e0c4f5f2bcd20e71d8be037da555b6fa554..253617a5e45fd44d9dbe55ab6d08138b5ce2aabb 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/themes/ThemeSettingsFragment.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/themes/ThemeSettingsFragment.java
@@ -23,7 +23,7 @@ import org.chromium.ui.UiUtils;
 /**
  * Fragment to manage the theme user settings.
  */
-public class ThemeSettingsFragment extends PreferenceFragmentCompat {
+public class ThemeSettingsFragment extends org.chromium.chrome.browser.settings.BravePreferenceFragment {
     static final String PREF_UI_THEME_PREF = "ui_theme_pref";
 
     @Override
