diff --git a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h
index 613b9a6c78090691c37909e70849a7d7aa17f02b..f4a74a9f8bf5968dac7e058f7aeb2df14dadb67e 100644
--- a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h
+++ b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h
@@ -89,6 +89,7 @@ class ChromeBrowsingDataRemoverDelegate
   void OverrideDomainReliabilityClearerForTesting(
       DomainReliabilityClearer clearer);
 
+  BRAVE_CHROME_BROWSING_DATA_REMOVER_DELEGATE_H
  private:
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
@@ -139,7 +140,7 @@ class ChromeBrowsingDataRemoverDelegate
     kSecurePaymentConfirmationCredentials = 41,
     kWebAppHistory = 42,
     kWebAuthnCredentials = 43,
-    kMaxValue = kWebAuthnCredentials,
+    kIPFSCache = 44, kMaxValue = kIPFSCache,
   };
 
   // Called by CreateTaskCompletionClosure().
