diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 31696b9329b047c734d98754406fd7a950444bb0..328d239383405ae463a7787887820742b6d3c68f 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -5138,6 +5138,7 @@ constexpr char kEnterpriseIsolatedModeInternalName[] =
 
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
