diff --git a/chrome/browser/ui/views/page_info/page_info_bubble_view.cc b/chrome/browser/ui/views/page_info/page_info_bubble_view.cc
index b2c265ab2698479cebef4ee6dcc43b8cbd1d849d..007c8455a7411269cefd7564a61c0174a79c0d9f 100644
--- a/chrome/browser/ui/views/page_info/page_info_bubble_view.cc
+++ b/chrome/browser/ui/views/page_info/page_info_bubble_view.cc
@@ -209,7 +209,7 @@ views::BubbleDialogDelegateView* PageInfoBubbleView::CreatePageInfoBubble(
   }
 
   PageInfoBubbleView* const bubble =
-      new PageInfoBubbleView(anchor, anchor_rect, parent_view, web_contents,
+      new BravePageInfoBubbleView(anchor, anchor_rect, parent_view, web_contents,
                              url, specification->initialized_callback(),
                              specification->page_info_closing_callback(),
                              specification->show_extended_site_info());
