diff --git a/chrome/browser/ui/views/omnibox/omnibox_popup_contents_view.cc b/chrome/browser/ui/views/omnibox/omnibox_popup_contents_view.cc
index 9e2e18ec8e40b2a9f991578ba42ac121dca310f5..8637aaf8a85b7ca4c2b6d901baf171531924825b 100644
--- a/chrome/browser/ui/views/omnibox/omnibox_popup_contents_view.cc
+++ b/chrome/browser/ui/views/omnibox/omnibox_popup_contents_view.cc
@@ -351,7 +351,7 @@ void OmniboxPopupContentsView::UpdatePopupAppearance() {
     if (children().size() == i) {
       AddChildView(std::make_unique<OmniboxRowView>(
           i, edit_model_,
-          std::make_unique<OmniboxResultView>(this, edit_model_, i),
+          std::make_unique<BraveOmniboxResultView>(this, edit_model_, i),
           pref_service));
     }
 
