diff --git a/chrome/browser/global_features.cc b/chrome/browser/global_features.cc
index 227504a6d2525746d30d87ab4fafaea5dcb303d7..0fc8b1e7ed909d692612841e611870210ad92640 100644
--- a/chrome/browser/global_features.cc
+++ b/chrome/browser/global_features.cc
@@ -275,6 +275,7 @@ void GlobalFeatures::PostBrowserProcessInitCore() {
 }
 
 void GlobalFeatures::Init() {
+  brave_global_features_ = std::make_unique<BraveGlobalFeatures>();
   global_browser_collection_ = CreateGlobalBrowserCollection();
 }
 
