diff --git a/chrome/browser/ui/omnibox/omnibox_next_features.cc b/chrome/browser/ui/omnibox/omnibox_next_features.cc
index cd3b9ed04c5d0ab3c8c8594323c37e6e64b1b646..5e0ea13e8f9ddbb8dd6c22952546526055fae131 100644
--- a/chrome/browser/ui/omnibox/omnibox_next_features.cc
+++ b/chrome/browser/ui/omnibox/omnibox_next_features.cc
@@ -33,11 +33,13 @@ namespace omnibox {
 namespace internal {
 
 // If enabled, shows the omnibox suggestions in the popup in WebUI.
-BASE_FEATURE(kWebUIOmniboxPopup, ENABLED);
+// kWebUIOmniboxPopup feature state is enforced via plaster rewrite.
+BASE_FEATURE(kWebUIOmniboxPopup, DISABLED);
 
 // If enabled, Omnibox popup will transition to AI-Mode with the compose-box
 // panel taking up the whole of the popup, covering the location bar completely.
-BASE_FEATURE(kWebUIOmniboxAimPopup, ENABLED);
+// kWebUIOmniboxAimPopup feature state is enforced via plaster rewrite.
+BASE_FEATURE(kWebUIOmniboxAimPopup, DISABLED);
 
 // If enabled, the Omnibox Popup will enable a different UI state when on a
 // webpage.
