diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
index 64def9515696434a45cbb51fc6e0cf8165facbee..e25b2948741ad8a150bfa681210e606a8836968c 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
@@ -442,6 +442,7 @@ void BookmarkBubbleView::ShowBubble(views::View* anchor_view,
       dialog_model_builder.Build(), anchor_view,
       views::BubbleBorder::TOP_RIGHT);
   bookmark_bubble_ = bubble.get();
+  BRAVE_BOOKMARK_BUBBLE_VIEW_SHOW_BUBBLE_SET_ARROW
   if (highlighted_button) {
     bubble->SetHighlightedButton(highlighted_button);
   }
