diff --git a/chrome/installer/setup/unpack_archive_unittest.cc b/chrome/installer/setup/unpack_archive_unittest.cc
index bf8999a5a9fd68ba40a129928200eeba1de1b402..54a665daeefa68a4d8bd643d1e0c8dee9d6f8e4d 100644
--- a/chrome/installer/setup/unpack_archive_unittest.cc
+++ b/chrome/installer/setup/unpack_archive_unittest.cc
@@ -102,6 +102,12 @@ TEST_P(SetupUnpackArchiveTest, UnpackArchive) {
                                          FILE_PATH_LITERAL("test_data.txt"))),
                                      &actual_installer_data));
   EXPECT_STREQ(actual_installer_data.c_str(), "fakechromiumdata");
+
+  EXPECT_EQ(installer_state.uncompressed_archive,
+            GetParam().uncompressed_output_matches_input_file
+                ? chrome_archive
+                : temp_dir.GetPath().Append(FILE_PATH_LITERAL("chrome.7z")));
+  ASSERT_EQ(installer_state.archive_type, ArchiveType::FULL_ARCHIVE_TYPE);
 }
 
 TEST(SetupUnpackArchiveTest, UnpackFailsWhenCompressedAndUncompressedProvided) {
