diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index e462f7d8bef19577547cacc76fc794ab6fa17dcf..17ab7457880c8bf2cb19e71482664e0da93dc744 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2274,6 +2274,7 @@ static_library("browser") {
     "//ui/web_dialogs",
     "//ui/webui",
     "//ui/webui/resources/cr_components/customize_themes:mojom",
+    "//brave/browser:browser_process",
   ]
   if (is_chromeos) {
     deps += [
@@ -4014,6 +4015,7 @@ static_library("browser") {
         sources += [ "first_run/first_run_internal_posix.cc" ]
       }
     }
+    sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps
   }
 
   if (is_chromeos) {
