diff --git a/chrome/browser/external_protocol/external_protocol_handler.cc b/chrome/browser/external_protocol/external_protocol_handler.cc
index 2a1a5b833c48d2ff2a05a899fb5822321d68fa23..6e1c1e0bcf75550426585f14f1fbc021f95002cd 100644
--- a/chrome/browser/external_protocol/external_protocol_handler.cc
+++ b/chrome/browser/external_protocol/external_protocol_handler.cc
@@ -288,7 +288,7 @@ ExternalProtocolHandler::BlockState ExternalProtocolHandler::GetBlockState(
   // Always allow the hard-coded allowed schemes.
   for (size_t i = 0; i < base::size(kAllowedSchemes); ++i) {
     if (kAllowedSchemes[i] == scheme)
-      return DONT_BLOCK;
+      return UNKNOWN;
   }
 
   PrefService* profile_prefs = profile->GetPrefs();
