diff --git a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
index c48d1eb22d77165879862022a4154997f511be34..772bf618410aae6c14bd77cd2807730b0ec0921e 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
@@ -686,7 +686,7 @@ public class TabbedRootUiCoordinator extends RootUiCoordinator {
                             mActivity, mModalDialogManagerSupplier.get()));
             NotificationPermissionController.attach(
                     mWindowAndroid, mNotificationPermissionController);
-            didTriggerPromo = mNotificationPermissionController.requestPermissionIfNeeded(
+            if(false) didTriggerPromo = mNotificationPermissionController.requestPermissionIfNeeded(
                     false /* contextual */);
         }
 
