diff --git a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_browsertest.cc b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_browsertest.cc
index 64171bdd4f4a4eda23e7e4445e21b7d0d7c94beb..6e54c1058a71621f422a4ae489b3f820d1d70c77 100644
--- a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_browsertest.cc
+++ b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_browsertest.cc
@@ -188,6 +188,7 @@ class RecentTabsSubMenuModelTest : public InProcessBrowserTest {
     return new_browser;
   }
 
+  BRAVE_RECENT_TABS_SUB_MENU_MODEL_TEST
  private:
   std::unique_ptr<syncer::DataTypeProcessor> sync_processor_;
 };
