diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 68ca9bdafcd501ef6793d9a8bad311bd4b3af0bc..ce93230ec672a76eb3cb649803d50b887c8928ac 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -222,6 +222,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  sources += brave_chrome_renderer_sources public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
