diff --git a/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkBridge.java b/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkBridge.java
index bab4cb91f75cc8a8f3b238d6e658df93b8980af8..51f8cd1361ae41dc43eb82af9dd0e3b7587ca835 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkBridge.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkBridge.java
@@ -1004,6 +1004,7 @@ class BookmarkBridge {
         }
         return pairList;
     }
+    @CalledByNative public void bookmarksImported(boolean isSuccess) {} @CalledByNative public void bookmarksExported(boolean isSuccess) {}
 
     @VisibleForTesting(otherwise = VisibleForTesting.PACKAGE_PRIVATE)
     @NativeMethods
@@ -1082,5 +1083,6 @@ class BookmarkBridge {
         void reorderChildren(long nativeBookmarkBridge, BookmarkBridge caller, BookmarkId parent,
                 long[] orderedNodes);
         boolean isBookmarked(long nativeBookmarkBridge, GURL url);
+        void importBookmarks(long nativeBookmarkBridge, BookmarkBridge caller, org.chromium.ui.base.WindowAndroid window, String importFilePath); void exportBookmarks(long nativeBookmarkBridge, BookmarkBridge caller, org.chromium.ui.base.WindowAndroid window, String exportFilePath);
     }
 }
