diff --git a/chrome/browser/resources/app_settings/window_mode_item.html.ts b/chrome/browser/resources/app_settings/window_mode_item.html.ts
index a8047b56f1ce700dc60e45ac284ca05bf6f37c16..a5e2108e206beeb2fd84769d9df9cf103778d822 100644
--- a/chrome/browser/resources/app_settings/window_mode_item.html.ts
+++ b/chrome/browser/resources/app_settings/window_mode_item.html.ts
@@ -1,17 +1,18 @@
-// Copyright 2024 The Chromium Authors
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
 import {html} from '//resources/lit/v3_0/lit.rollup.js';
 
 import type {WindowModeItemElement} from './window_mode_item.js';
 
 export function getHtml(this: WindowModeItemElement) {
   return html`<!--_html_template_start_-->
-<app-management-toggle-row
-    id="toggle-row"
-    label="${this.windowModeLabel}"
-    ?value="${this.getValue_()}">
-</app-management-toggle-row>
+<div ?hidden="${this.app.hideWindowMode}">
+    <div class="permission-section-header">
+      <div class="header-text">$i18n{appManagementOpenModeLabel}</div>
+    </div>
+    <div class="permission-list indented-permission-block">
+      ${this.windowModes.map(mode => html`<leo-radiobutton class="subpermission-row" name="window-mode" @change="${this.onChange}" value="${mode.value}" currentvalue="${this.app.windowMode}">
+        ${mode.label}
+      </leo-radiobutton>`)}
+    </div>
+  </div>
 <!--_html_template_end_-->`;
 }
