diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index dff9f052184c5788edfd7bf69c305d4ec32e5d49..2c858d8e93b94b44ff30fdfdd8b8f535682985d5 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -240,6 +240,7 @@ class BrowserProcessImpl : public BrowserProcess,
   void CreateGlobalFeaturesForTesting() override;
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
