diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 4af645818c875e5dc2971079025e1e6c509066c1..9908e7bcab8b8475fab8d6fafe6dea40dc7a2776 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -459,6 +459,7 @@ void ToolbarView::Init() {
           browser_view_, browser_->profile()));
 #endif
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = container_view_->AddChildView(
       std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
