diff --git a/chrome/browser/ui/browser_window/internal/browser_window_features.cc b/chrome/browser/ui/browser_window/internal/browser_window_features.cc
index c2d47effffd2298e037a090dc3ad22ecd9315dbd..1f3bd1f03376929d622c25eb1015415ed77462d5 100644
--- a/chrome/browser/ui/browser_window/internal/browser_window_features.cc
+++ b/chrome/browser/ui/browser_window/internal/browser_window_features.cc
@@ -300,7 +300,7 @@ void BrowserWindowFeatures::Init(BrowserWindowInterface* browser) {
           *browser, fullscreen_controller_.get(), app_browser_controller_.get(),
           browser->GetType(),
           browser->GetBrowserForMigrationOnly()->is_trusted_source(),
-          browser->GetUnownedUserDataHost());
+          browser->GetUnownedUserDataHost(), browser->GetFeatures().vertical_tab_controller()->GetWeakPtr());
 
   side_panel_registry_ =
       GetUserDataFactory().CreateInstance<SidePanelRegistry>(*browser, browser);
