diff --git a/chrome/browser/flags/BUILD.gn b/chrome/browser/flags/BUILD.gn
index af17742f49f08bf960385f62cf1fc0e304f09890..187e56e4ac0a074a522743f33ec3053ba8cc6fec 100644
--- a/chrome/browser/flags/BUILD.gn
+++ b/chrome/browser/flags/BUILD.gn
@@ -33,6 +33,7 @@ android_library("java") {
     "//third_party/android_deps:guava_android_java",
     "//third_party/androidx:androidx_annotation_annotation_java",
   ]
+  sources += brave_flags_java_sources
   srcjar_deps = [
     ":chrome_android_java_switches_srcjar",
     ":chrome_browser_flags_enums_srcjar",
