diff --git a/chrome/browser/incognito/BUILD.gn b/chrome/browser/incognito/BUILD.gn
index bfeb79f3caf0f2a8446c09979310e4ad0bca9ecf..019efc592dc4ef928675c82f097f59757ad1dbc9 100644
--- a/chrome/browser/incognito/BUILD.gn
+++ b/chrome/browser/incognito/BUILD.gn
@@ -59,6 +59,7 @@ android_library("java") {
     "//ui/android:ui_full_java",
     "//url:gurl_java",
   ]
+  sources += brave_private_tab_reauth_java_sources deps += brave_private_tab_reauth_java_deps
   annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
   resources_package = "org.chromium.chrome.browser.incognito"
 }
