diff --git a/chrome/browser/shell_integration.h b/chrome/browser/shell_integration.h
index 6385833bbdad93e13701ae4e0656a533cca38fff..2eb86bdfc7e5d95771a2fb9e9c60e6b29ccafd3c 100644
--- a/chrome/browser/shell_integration.h
+++ b/chrome/browser/shell_integration.h
@@ -249,6 +249,7 @@ class DefaultBrowserWorker : public DefaultWebClientWorker {
   DefaultBrowserWorker(const DefaultBrowserWorker&) = delete;
   DefaultBrowserWorker& operator=(const DefaultBrowserWorker&) = delete;
 
+  BRAVE_DEFAULT_BROWSER_WORKER_H
  protected:
   ~DefaultBrowserWorker() override;
 
