diff --git a/chrome/browser/resources/settings/site_settings/category_default_setting.ts b/chrome/browser/resources/settings/site_settings/category_default_setting.ts
index 3c1a95dd14a3d8757438a1eababd0816c70e1137..c40455e7220ae909a3bb38f7dfa2f1222e48245a 100644
--- a/chrome/browser/resources/settings/site_settings/category_default_setting.ts
+++ b/chrome/browser/resources/settings/site_settings/category_default_setting.ts
@@ -179,6 +179,7 @@ class CategoryDefaultSettingElement extends CategoryDefaultSettingElementBase {
      */
     switch (this.category) {
       case ContentSettingsTypes.ADS:
+      case ContentSettingsTypes.AUTOPLAY:
       case ContentSettingsTypes.BACKGROUND_SYNC:
       case ContentSettingsTypes.IMAGES:
       case ContentSettingsTypes.JAVASCRIPT:
@@ -213,6 +214,7 @@ class CategoryDefaultSettingElement extends CategoryDefaultSettingElementBase {
       case ContentSettingsTypes.USB_DEVICES:
       case ContentSettingsTypes.VR:
       case ContentSettingsTypes.WINDOW_PLACEMENT:
+      case ContentSettingsTypes.ETHEREUM:
         // "Ask" vs "Blocked".
         this.browserProxy.setDefaultValueForContentType(
             this.category,
