diff --git a/chrome/browser/ui/views/overlay/video_overlay_window_views.cc b/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
index 749a6cd9a4877035c90e14e95c44ba4cfbfe32ac..988874c410dfd82de5f858620ee780fd473c74e2 100644
--- a/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
+++ b/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
@@ -377,7 +377,7 @@ std::unique_ptr<VideoOverlayWindowViews> VideoOverlayWindowViews::Create(
   // constructor. It's important that the constructor be private, because it
   // doesn't initialize the object fully.
   auto overlay_window =
-      base::WrapUnique(new VideoOverlayWindowViews(controller));
+      base::WrapUnique(new BraveVideoOverlayWindowViews(controller));
 
   // The 2024 updated controls use dark mode colors.
   if (Use2024UI()) {
@@ -947,6 +947,7 @@ void VideoOverlayWindowViews::UpdateMaxSize(const gfx::Rect& work_area) {
 
   max_size_ = new_max_size;
 
+  BRAVE_UPDATE_MAX_SIZE
   if (!native_widget()) {
     return;
   }
