diff --git a/chrome/browser/devtools/devtools_ui_bindings.h b/chrome/browser/devtools/devtools_ui_bindings.h
index c4586578f1c828e17959c2b18698d68134d963ff..af9584160fd5e61eeb6bc09bde54c60dd006c4a5 100644
--- a/chrome/browser/devtools/devtools_ui_bindings.h
+++ b/chrome/browser/devtools/devtools_ui_bindings.h
@@ -99,6 +99,8 @@ class DevToolsUIBindings : public DevToolsEmbedderMessageDispatcher::Delegate,
 
  private:
   using DevToolsUIBindingsList = std::vector<DevToolsUIBindings*>;
+  friend class BraveDevToolsUIBindings;
+  FRIEND_TEST_ALL_PREFIXES(BraveDevToolsUIBindingsBrowserTest, ThemeTest);
 
   void HandleMessageFromDevToolsFrontend(base::Value);
 
