diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
index 65106c86166077eed83a5f96158e3854d8df8036..65c9dcd26aebb63a9065f90348e55b271392d40b 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
@@ -856,6 +856,7 @@ public class ChromeTabbedActivity extends ChromeActivity implements PreAttachInt
                         private void closeIfNoTabsAndHomepageEnabled(
                                 boolean isPendingClosure, boolean shouldRemoveWindowWithZeroTabs) {
                             if (getTabModelSelector().getTotalTabCount() == 0) {
+                                shouldRemoveWindowWithZeroTabs = shouldRemoveWindowWithZeroTabs && org.chromium.chrome.browser.partnercustomizations.CloseBraveManager.shouldCloseAppWithZeroTabs();
                                 if (shouldRemoveWindowWithZeroTabs) {
                                     finishAndRemoveTask();
                                 } else if (HomepageManager.getInstance()
