diff --git a/chrome/browser/metrics/chrome_feature_list_creator.cc b/chrome/browser/metrics/chrome_feature_list_creator.cc
index 73c9ee5a0d93a3937a95c3fb0da6992ef0cea567..573f96a383a8fafa52a5ef449172ea56b562070b 100644
--- a/chrome/browser/metrics/chrome_feature_list_creator.cc
+++ b/chrome/browser/metrics/chrome_feature_list_creator.cc
@@ -351,5 +351,6 @@ void ChromeFeatureListCreator::SetupInitialPrefs() {
     local_state_->SetInt64(variations::prefs::kVariationsSeedDate,
                            base::Time::Now().ToInternalValue());
   }
+  BRAVE_CHROME_FEATURE_LIST_CREATOR_SETUP_INITIAL_PREFS
 #endif  // !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_CHROMEOS_ASH)
 }
