diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 0d3541d0da1d1c135be3d41a8879220c9bb102de..473687af1d465c2f8365a1e31de0c4233bad96b0 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -65,6 +65,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
