diff --git a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h
index 8024fdfff0e211bb3b42dd906648f2c157e363e2..65f2b50b374698baa2522ffd7c448a7ec90af46c 100644
--- a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h
+++ b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h
@@ -91,6 +91,7 @@ class ChromeBrowsingDataRemoverDelegate
   void OverrideDomainReliabilityClearerForTesting(
       DomainReliabilityClearer clearer);
 
+  BRAVE_CHROME_BROWSING_DATA_REMOVER_DELEGATE_H
  private:
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
@@ -147,8 +148,9 @@ class ChromeBrowsingDataRemoverDelegate
     // See also kDisableAutoSigninForProfilePasswords.
     kDisableAutoSigninForAccountPasswords = 46,
     kPrivateVerificationTokens = 47,
+    kIPFSCache = 48,
 
-    kMaxValue = kPrivateVerificationTokens,
+    kMaxValue = kIPFSCache,
   };
   // LINT.ThenChange(//tools/metrics/histograms/metadata/history/enums.xml:ChromeBrowsingDataRemoverTasks)
 
