diff --git a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
index 54fe2e48cde408898820773ba74e4c9185de98c8..847c0d2874271bd852589e0a0fe38a9be4d19729 100644
--- a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
+++ b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
@@ -135,6 +135,7 @@ content::WebUIDataSource* CreateBookmarksUIHTMLSource(Profile* profile) {
                         ? "enable-branding-update"
                         : "");
 
+  BRAVE_CREATE_BOOKMARKS_UI_HTML_SOURCE
   return source;
 }
 
