diff --git a/chrome/browser/content_settings/host_content_settings_map_factory.cc b/chrome/browser/content_settings/host_content_settings_map_factory.cc
index d5cab7562cbdfc58ea973cb15a1b77bad2ce7e7f..86d887216cbf8b9768bd01b3f53148ac5705428b 100644
--- a/chrome/browser/content_settings/host_content_settings_map_factory.cc
+++ b/chrome/browser/content_settings/host_content_settings_map_factory.cc
@@ -97,6 +97,7 @@ scoped_refptr<RefcountedKeyedService>
 
   scoped_refptr<HostContentSettingsMap> settings_map(new HostContentSettingsMap(
       profile->GetPrefs(),
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
       profile->IsOffTheRecord() || profile->IsGuestSession(),
       /*store_last_modified=*/true,
       profile->ShouldRestoreOldSessionCookies()));
