diff --git a/chrome/browser/favicon/favicon_utils.cc b/chrome/browser/favicon/favicon_utils.cc
index d318340b3d2c1420dfcda8cc66b0339d5bca9d02..151ac406ba51baa88af99294d42bc44f5b51bc82 100644
--- a/chrome/browser/favicon/favicon_utils.cc
+++ b/chrome/browser/favicon/favicon_utils.cc
@@ -174,6 +174,9 @@ bool ShouldThemifyFavicon(GURL url) {
 }
 
 bool ShouldThemifyFaviconForEntry(content::NavigationEntry* entry) {
+  if (auto brave_themify = ShouldThemifyFaviconForEntryBraveImpl(entry)) {
+    return *brave_themify;
+  }
   const GURL& virtual_url = entry->GetVirtualURL();
   const GURL& actual_url = entry->GetURL();
 
