diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index b1c488cfce337ffc1fd6a6019d4af12ef518b493..b309fec73a60e0aeae3649d4fbb436b7b51273d7 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -35,6 +35,7 @@ if (optimize_webui) {
 
     deps = [
       ":unpak",
+      "//brave/browser/resources/settings:unpak",
     ]
   }
 
