diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 9cba6c988a4026bf5561b53cb6fb85db9439f907..ad05f92d5fc006fe6ef6c709e556044d678ae266 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -29,6 +29,7 @@ static_library("utility") {
 
   public_deps = []
   deps = [
+    "//brave/utility",
     "//base",
     "//build:chromeos_buildflags",
     "//chrome:resources",
