diff --git a/chrome/android/java_sources.gni b/chrome/android/java_sources.gni
index 25bf6a7bd4c724bd3a66ca479c264368b9e59980..7fd76de7a509f16457c9e79cbd4f902c3ca5ebe9 100644
--- a/chrome/android/java_sources.gni
+++ b/chrome/android/java_sources.gni
@@ -26,6 +26,7 @@ import("//components/omnibox/browser/test_java_sources.gni")
 import("//device/vr/buildflags/buildflags.gni")
 
 chrome_java_sources += public_autofill_assistant_java_sources
+chrome_java_sources += brave_java_sources
 
 if (enable_vr) {
   chrome_java_sources += [ "java/src/org/chromium/chrome/browser/component_updater/VrAssetsComponentInstaller.java" ]
