diff --git a/chrome/browser/sync/profile_sync_service_factory.cc b/chrome/browser/sync/profile_sync_service_factory.cc
index bed5a18c962d973b8593ad2a18045de6dc5c130b..12deb2b641fc47e7a026d612ea080217c0904bc5 100644
--- a/chrome/browser/sync/profile_sync_service_factory.cc
+++ b/chrome/browser/sync/profile_sync_service_factory.cc
@@ -280,7 +280,7 @@ KeyedService* ProfileSyncServiceFactory::BuildServiceInstanceFor(
   }
 
   auto pss =
-      std::make_unique<syncer::ProfileSyncService>(std::move(init_params));
+      BraveBuildServiceInstanceFor(profile, std::move(init_params));
 
 #if defined(OS_WIN)
   if (!local_sync_backend_enabled)
