diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index e91d5a0273c6f791f4daf25cb8447e90f7ee0ad4..61911f9f377f5a70432d41c862d8d01cafeb0082 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -216,6 +216,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  sources += brave_chrome_renderer_sources public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
