diff --git a/chrome/browser/net/system_network_context_manager.cc b/chrome/browser/net/system_network_context_manager.cc
index f862945871a5748350e2398839a9daa2feab5f28..a36d4875e530b0e6bd9b2f5ebf0f61c83ed252b9 100644
--- a/chrome/browser/net/system_network_context_manager.cc
+++ b/chrome/browser/net/system_network_context_manager.cc
@@ -298,7 +298,7 @@ class SystemNetworkContextManager::URLLoaderFactoryForSystem
     if (!manager_)
       return;
     manager_->GetURLLoaderFactory()->CreateLoaderAndStart(
-        std::move(receiver), routing_id, request_id, options, url_request,
+        std::move(receiver), routing_id, request_id, options, brave::OnBeforeSystemRequest(url_request),
         std::move(client), traffic_annotation);
   }
 
