diff --git a/chrome/browser/ui/webui/about_ui.cc b/chrome/browser/ui/webui/about_ui.cc
index e04c83a8031397dd54088e1ea565227dc33a3a47..9f2bcc20dc12c34fbb0843d81724de04cb274ceb 100644
--- a/chrome/browser/ui/webui/about_ui.cc
+++ b/chrome/browser/ui/webui/about_ui.cc
@@ -579,6 +579,7 @@ std::string ChromeURLs() {
   html += "</ul>\n";
 
   AppendFooter(&html);
+  CHROME_INTERNAL_URLS_TO_BRAVE 
   return html;
 }
 
