diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index 86c3a6b34a40e8d9e6d71d1b41ab84983bc9a216..120a8ba279270366d4ed7f838f4886736736c73e 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -377,7 +377,7 @@ const char kGlicActorUiDebounceTimerName[] = "glic-actor-ui-debounce-timer";
 
 // Controls whether the task icon in the actor ui is enabled.
 const base::FeatureParam<bool> kGlicActorUiTaskIcon{
-    &kGlicActorUi, kGlicActorUiTaskIconName, true};
+    &kGlicActorUi, kGlicActorUiTaskIconName, false};
 // Controls whether the Actor Overlay in the actor ui is enabled.
 const base::FeatureParam<bool> kGlicActorUiOverlay{
     &kGlicActorUi, kGlicActorUiOverlayName, true};
