diff --git a/chrome/installer/mac/signing/config.py b/chrome/installer/mac/signing/config.py
index 8d0582f1092780d5fdf3a12fdebc382717f5abdf..19aca9f39181fbde1cb06e02a3549adcfa63762d 100644
--- a/chrome/installer/mac/signing/config.py
+++ b/chrome/installer/mac/signing/config.py
@@ -214,4 +214,4 @@ class CodeSignConfig(object):
     @property
     def packaging_dir(self):
         """Returns the path to the packaging and installer tools."""
-        return '{.product} Packaging'.format(self)
+        return '{}_Packaging'.format(self.product.replace(' ', '_'))
