diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index c3dfc2627d70f81b5a4ff17093af9c6930c49ecf..ea502516007e642471d8289deaa5578d41e4ad93 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1570,6 +1570,7 @@ source_set("extensions") {
       "api/tabs/app_window_helper.h",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 if (enable_extensions_core) {
