diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index 9a16a69974b39bc106a8a73bd7b7a97bb91a58c1..63a6d28095ec57f94ed6e8766648313f78d08c59 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -376,7 +376,9 @@ cr.define('settings', function() {
 
       // TODO(tommycli): Find a way to refactor these repetitive category
       // routes.
+      // <if expr="_google_chrome">
       r.SITE_SETTINGS_ADS = r.SITE_SETTINGS.createChild('ads');
+      // </if>
       r.SITE_SETTINGS_AUTOMATIC_DOWNLOADS =
           r.SITE_SETTINGS.createChild('automaticDownloads');
       r.SITE_SETTINGS_BACKGROUND_SYNC =
@@ -450,7 +452,9 @@ cr.define('settings', function() {
       }
 
       r.PRINTING = r.ADVANCED.createSection('/printing', 'printing');
+      // <if expr="_google_chrome">
       r.CLOUD_PRINTERS = r.PRINTING.createChild('/cloudPrinters');
+      // </if>
       // <if expr="chromeos">
       r.CUPS_PRINTERS = r.PRINTING.createChild('/cupsPrinters');
       // </if>
