diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index 7cc918bd33278a024c1d4eee0add8586b7974b30..8f0ce98783febc803cbe2ef9d7d3ca1c85bfaf98 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -117,6 +117,7 @@ grit("generated_resources") {
   if (is_android) {
     outputs += android_generated_java_resources
   }
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
