diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index 819467b388781db13d70b70cc5ebbe8c8966834b..52df670bda37b374da3d58f4b1a2454226373656 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -381,7 +381,9 @@ cr.define('settings', function() {
       }
 
       r.PRINTING = r.ADVANCED.createSection('/printing', 'printing');
+      // <if expr="_google_chrome">
       r.CLOUD_PRINTERS = r.PRINTING.createChild('/cloudPrinters');
+      // </if>
       // <if expr="chromeos">
       r.CUPS_PRINTERS = r.PRINTING.createChild('/cupsPrinters');
 
