diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index 0f6308ea44ec5c8f8a76f9fd96e5387c79174db4..56ffd5f657170035ea5380e3ab0f4176f283ef87 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -251,7 +251,7 @@ grit("google_chrome_strings") {
 }
 
 grit("chromium_strings") {
-  source = "chromium_strings.grd"
+  source = "${branding_path_component}_strings.grd"
   defines = chrome_grit_defines
   output_dir = "$root_gen_dir/chrome"
   outputs = [
@@ -465,6 +465,7 @@ service_manifest("chrome_content_packaged_services_manifest_overlay") {
       "//ui/accessibility:manifest",
     ]
   }
+  packaged_services += [ "//brave/utility/tor:tor_launcher_manifest" ]
 }
 
 # Per-profile services.
