diff --git a/chrome/browser/ui/chrome_pages.cc b/chrome/browser/ui/chrome_pages.cc
index 4cdab89531db96b4eb677cd7d3963c23e10f1545..be4e89aaaba1c41d3559267afb94deb3b1f48366 100644
--- a/chrome/browser/ui/chrome_pages.cc
+++ b/chrome/browser/ui/chrome_pages.cc
@@ -399,7 +399,7 @@ void ShowSlow(Browser* browser) {
 #endif
 }
 
-GURL GetSettingsUrl(const std::string& sub_page) {
+BRAVE_GET_SETTINGS_URL
   return GURL(std::string(kChromeUISettingsURL) + sub_page);
 }
 
