diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
index 1a7a78173b33a32db2298567c29f57a6aff85f36..65553e21164504a460860f6b7f99c23454029e62 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
@@ -172,7 +172,7 @@ import java.util.Locale;
  * This is the main activity for ChromeMobile when not running in document mode.  All the tabs
  * are accessible via a chrome specific tab switching UI.
  */
-public class ChromeTabbedActivity extends ChromeActivity implements ScreenshotMonitorDelegate {
+public class ChromeTabbedActivity extends BraveActivity implements ScreenshotMonitorDelegate {
     /**
      * The results of a system back press action.
      */
