diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 46b7b5235cd85f9c5dcd29f775d6e478f91fcd2f..23fb1c14406a43e75ac1bc78e7704885e371ccb5 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -2917,6 +2917,7 @@ const FeatureEntry::Choice kAlwaysEnableHdcpChoices[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
