diff --git a/chrome/browser/devtools/devtools_ui_bindings.h b/chrome/browser/devtools/devtools_ui_bindings.h
index 815a9dacb4ac4a591339d49f3848adf72a475a84..9831bc586d7ac62fefdee1b61dd1b110ed8c152e 100644
--- a/chrome/browser/devtools/devtools_ui_bindings.h
+++ b/chrome/browser/devtools/devtools_ui_bindings.h
@@ -104,6 +104,8 @@ class DevToolsUIBindings : public DevToolsEmbedderMessageDispatcher::Delegate,
 
  private:
   using DevToolsUIBindingsList = std::vector<DevToolsUIBindings*>;
+  friend class BraveDevToolsUIBindings;
+  FRIEND_TEST_ALL_PREFIXES(BraveDevToolsUIBindingsBrowserTest, ThemeTest);
 
   void HandleMessageFromDevToolsFrontend(base::Value);
 
