diff --git a/chrome/browser/download/download_ui_model.h b/chrome/browser/download/download_ui_model.h
index a0b2eef645615796e7b4dd373b06762687f001b9..109353aedf339025d831bb45b7dc1df4bc313072 100644
--- a/chrome/browser/download/download_ui_model.h
+++ b/chrome/browser/download/download_ui_model.h
@@ -517,6 +517,9 @@ class DownloadUIModel {
   virtual bool IsExtensionDownload() const;
 
  protected:
+  // Brave: virtual hooks for kDeleteLocalFile / kCopyDownloadLink commands.
+  virtual void DeleteLocalFile() {}
+  virtual void CopyDownloadLinkToClipboard() {}
   // Returns the MIME type of the download.
   virtual std::string GetMimeType() const;
 
