diff --git a/chrome/installer/gcapi/gcapi.cc b/chrome/installer/gcapi/gcapi.cc
index b418979309d15d592d1b905b532ef1ca861db5ff..c4b2b1f327c6ef7685ecde617d7aac59add79282 100644
--- a/chrome/installer/gcapi/gcapi.cc
+++ b/chrome/installer/gcapi/gcapi.cc
@@ -63,6 +63,16 @@ using Microsoft::WRL::ComPtr;
 
 namespace {
 
+#if defined(OFFICIAL_BUILD)
+const wchar_t kGCAPITempKey[] = L"Software\\BraveSoftware\\GCAPITemp";
+const wchar_t kChromeRegVersion[] = L"pv";
+const wchar_t kNoChromeOfferUntil[] =
+    L"SOFTWARE\\BraveSoftware\\No Chrome Offer Until";
+const wchar_t kC1FPendingKey[] =
+    L"Software\\BraveSoftware\\Common\\Rlz\\Events\\C";
+const wchar_t kC1FSentKey[] =
+    L"Software\\BraveSoftware\\Common\\Rlz\\StatefulEvents\\C";
+#else
 const wchar_t kGCAPITempKey[] = L"Software\\Google\\GCAPITemp";
 
 const wchar_t kChromeRegVersion[] = L"pv";
@@ -72,6 +82,7 @@ const wchar_t kNoChromeOfferUntil[] =
 const wchar_t kC1FPendingKey[] = L"Software\\Google\\Common\\Rlz\\Events\\C";
 const wchar_t kC1FSentKey[] =
     L"Software\\Google\\Common\\Rlz\\StatefulEvents\\C";
+#endif
 const wchar_t kC1FKey[] = L"C1F";
 
 const wchar_t kRelaunchBrandcodeValue[] = L"RelaunchBrandcode";
