diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index f51ad2e7b6df421c902c61b5c0f0d3c4ca541cb1..0b3ca7925595a742db6d45fe06a68ebe540bbfca 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -1235,6 +1235,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
