diff --git a/chrome/browser/themes/BUILD.gn b/chrome/browser/themes/BUILD.gn
index 09cba5e4e0052c88ebb11e64b5b7fc62625d9183..510ff639c289dd676fca1c44ee85aba1cf527be2 100644
--- a/chrome/browser/themes/BUILD.gn
+++ b/chrome/browser/themes/BUILD.gn
@@ -10,6 +10,7 @@ source_set("themes") {
   public = [
     "browser_theme_pack.h",
     "custom_theme_supplier.h",
+    "theme_command_line_handler.h",
     "theme_helper.h",
     "theme_local_data_batch_uploader.h",
     "theme_properties.h",
@@ -22,6 +23,7 @@ source_set("themes") {
   sources = [
     "browser_theme_pack.cc",
     "custom_theme_supplier.cc",
+    "theme_command_line_handler.cc",
     "theme_helper.cc",
     "theme_local_data_batch_uploader.cc",
     "theme_properties.cc",
