diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 38ff6fca0074f890ab9a95b03a2e80c23e46727b..5dc1f02d30dc2ff550cb85a2bdf5af4a2e62a73c 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -217,6 +217,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  sources += brave_chrome_renderer_sources public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
