diff --git a/chrome/services/printing/public/mojom/BUILD.gn b/chrome/services/printing/public/mojom/BUILD.gn
index 42b0c6205413d5aed27d4e9a00df77cbba381589..e01127d61fb891884fb102585024ae89dbe005fa 100644
--- a/chrome/services/printing/public/mojom/BUILD.gn
+++ b/chrome/services/printing/public/mojom/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/services/printing/public/mojom/sources.gni")
 import("//mojo/public/tools/bindings/mojom.gni")
 import("//printing/buildflags/buildflags.gni")
 
@@ -12,6 +13,7 @@ mojom("mojom") {
     "pdf_to_pwg_raster_converter.mojom",
     "printing_service.mojom",
   ]
+  sources += brave_services_printing_public_mojom_sources
 
   public_deps = [
     "//mojo/public/mojom/base",
@@ -20,6 +22,7 @@ mojom("mojom") {
     "//ui/gfx/geometry/mojom",
     "//url/mojom:url_mojom_gurl",
   ]
+  public_deps += brave_services_printing_public_mojom_public_deps
 
   enabled_features = []
 
