diff --git a/chrome/browser/extensions/extension_management.cc b/chrome/browser/extensions/extension_management.cc
index 4f3569c97d7c3fdc4e6bf7f16b13978a113c5189..5408bfe2d1d2595d18e9cf9cf0343dc408cea792 100644
--- a/chrome/browser/extensions/extension_management.cc
+++ b/chrome/browser/extensions/extension_management.cc
@@ -1148,6 +1148,7 @@ ExtensionManagementFactory::BuildServiceInstanceForBrowserContext(
     content::BrowserContext* context) const {
   TRACE_EVENT0("browser,startup",
                "ExtensionManagementFactory::BuildServiceInstanceFor");
+  BRAVE_EXTENSION_MANAGEMENT_FACTORY_BUILD_SERVICE_INSTANCE_FOR
   return std::make_unique<ExtensionManagement>(
       Profile::FromBrowserContext(context));
 }
