diff --git a/chrome/browser/ai/ai_data_keyed_service.cc b/chrome/browser/ai/ai_data_keyed_service.cc
index 9807c3050e531f6c86b61032ea9dcc5bf381119b..4f8ad716f4840255bfbdb976a40a8abfdbfefc94 100644
--- a/chrome/browser/ai/ai_data_keyed_service.cc
+++ b/chrome/browser/ai/ai_data_keyed_service.cc
@@ -254,7 +254,7 @@ void GetHistoryQueryResultForModelPrototyping(
                          .start_time()
                          .seconds())),
         history_query.num_history_visits(),
-        /*skip_answering=*/true, history_search_callback);
+        /*skip_answering=*/true, /*url_id_filter=*/{}, history_search_callback);
   }
 }
 
