diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 72e10ab65de8b0b0bc351f333e83854509182d8b..b8d3627feea7b86e66746554b187e1f7c8c15375 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3449,6 +3449,7 @@ const FeatureEntry::FeatureVariation kVcSegmentationModelVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
