diff --git a/chrome/browser/ui/views/page_info/page_info_main_view.cc b/chrome/browser/ui/views/page_info/page_info_main_view.cc
index 1f2dcf103cc2919a35ed3b97ba0a8353c17c8b1d..dccb11e3252edf80c7dee6240ea9327ea6d64bf6 100644
--- a/chrome/browser/ui/views/page_info/page_info_main_view.cc
+++ b/chrome/browser/ui/views/page_info/page_info_main_view.cc
@@ -464,6 +464,7 @@ gfx::Size PageInfoMainView::CalculatePreferredSize() const {
   if (site_settings_view_) {
     width = std::max(width, site_settings_view_->GetPreferredSize().width());
     width = std::max(width, permissions_view_->GetPreferredSize().width());
+    BRAVE_PAGE_INFO_MAIN_VIEW_CALCULATE_PREFERRED_SIZE
   }
   return gfx::Size(width, views::View::GetHeightForWidth(width));
 }
