diff --git a/chrome/browser/ui/views/toolbar/app_menu.h b/chrome/browser/ui/views/toolbar/app_menu.h
index 64be01957556dbf5ec7e7951ecfae3cbb3fdc19d..a21ea1c58a310eaf42974e29f81ee267a5eb9427 100644
--- a/chrome/browser/ui/views/toolbar/app_menu.h
+++ b/chrome/browser/ui/views/toolbar/app_menu.h
@@ -31,7 +31,7 @@ class MenuRunner;
 }
 
 // AppMenu adapts the AppMenuModel to view's menu related classes.
-class AppMenu final : public views::MenuDelegate,
+class AppMenu : public views::MenuDelegate,
                       public bookmarks::BaseBookmarkModelObserver,
                       public GlobalErrorObserver {
  public:
