diff --git a/chrome/browser/sync/sync_service_factory.cc b/chrome/browser/sync/sync_service_factory.cc
index 0d59517fde60b25c29e8ea0850f51e7593b3a25b..39f5755c54d2f7c38b398b0383062ca9b32dc708 100644
--- a/chrome/browser/sync/sync_service_factory.cc
+++ b/chrome/browser/sync/sync_service_factory.cc
@@ -294,6 +294,7 @@ syncer::DataTypeController::TypeVector CreateCommonControllers(
   builder.SetNotebooksService(
       notebooks::NotebooksServiceFactory::GetForProfile(profile));
 
+  BRAVE_COMMON_CONTROLLER_BUILDER_SET_SERVICES
   return builder.Build(/*disabled_types=*/{}, sync_service,
                        chrome::GetChannel());
 }
@@ -463,7 +464,7 @@ std::unique_ptr<KeyedService> BuildSyncService(
   }
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize(CreateControllers(profile, sync_service.get()));
 
   // Notify the PasswordStore of complete initialisation to resolve a circular
