diff --git a/chrome/installer/linux/debian/control.template b/chrome/installer/linux/debian/control.template
index 03224f07c09f4d6d2a7c9778f797202197161f10..764952501ef14b317dabec07f1e0655f2dd9855d 100644
--- a/chrome/installer/linux/debian/control.template
+++ b/chrome/installer/linux/debian/control.template
@@ -1,11 +1,11 @@
-Source: @@PACKAGE@@-@@CHANNEL@@
+Source: @@PACKAGEANDCHANNEL@@
 Section: web
 Priority: optional
 Maintainer: @@MAINTNAME@@ <@@MAINTMAIL@@>
 Build-Depends: dpkg-dev, devscripts, fakeroot, xz-utils
 Standards-Version: 3.8.0
 
-Package: @@PACKAGE@@-@@CHANNEL@@
+Package: @@PACKAGEANDCHANNEL@@
 Provides: @@PROVIDES@@
 Pre-Depends: @@PREDEPENDS@@
 Depends: @@DEPENDS@@
