diff --git a/chrome/android/java/src/org/chromium/chrome/browser/media/FullscreenVideoPictureInPictureController.java b/chrome/android/java/src/org/chromium/chrome/browser/media/FullscreenVideoPictureInPictureController.java
index fad9fef728445ebc7da75f6cbd8a2e826c52b923..e9e37bd29aa156a454310455c7861b255bff4cbd 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/media/FullscreenVideoPictureInPictureController.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/media/FullscreenVideoPictureInPictureController.java
@@ -281,6 +281,10 @@ public class FullscreenVideoPictureInPictureController {
             }
         }
 
+        if (BraveFullscreenVideoPictureInPictureController.class.cast(this)
+                .braveAttemptPictureInPicture(mActivity, webContents, builder, bounds)) {
+            return;
+        }
         try {
             if (!mActivity.enterPictureInPictureMode(builder.build())) return;
         } catch (IllegalStateException | IllegalArgumentException e) {
@@ -615,6 +619,16 @@ public class FullscreenVideoPictureInPictureController {
      * This will also update our auto-PiP preference with the framework, if it's changed.
      */
     private void dismissActivityIfNeeded(Activity activity, @MetricsEndReason int reason) {
+        if (BraveFullscreenVideoPictureInPictureController.class.cast(this)
+                .maybeHandleDismissActivityForYouTubePictureInPicture(
+                        activity,
+                        reason == MetricsEndReason.START,
+                        reason == MetricsEndReason.RESUME,
+                        reason == MetricsEndReason.LEFT_FULLSCREEN,
+                        reason == MetricsEndReason.WEB_CONTENTS_LEFT_FULLSCREEN,
+                        reason == MetricsEndReason.NEW_TAB)) {
+            return;
+        }
         // Something interesting happened -- make sure we've updated our preference for auto-PiP
         // with the framework, if applicable.
         Log.i(TAG, "Dismiss activity with reason " + reason);
