diff --git a/chrome/browser/ui/omnibox/omnibox_theme.h b/chrome/browser/ui/omnibox/omnibox_theme.h
index 1ad58410297798f96178e42ea36c7d2c4d0d0e0a..628bd96d29c66d97b506b126975cfea60927daab 100644
--- a/chrome/browser/ui/omnibox/omnibox_theme.h
+++ b/chrome/browser/ui/omnibox/omnibox_theme.h
@@ -28,7 +28,7 @@ enum class OmniboxPart {
 };
 
 // The tint of the omnibox theme. E.g. Incognito may use a DARK tint.
-enum class OmniboxTint { DARK, LIGHT };
+enum class OmniboxTint_Chromium { DARK, LIGHT };
 
 // An optional state for a given |OmniboxPart|.
 enum class OmniboxPartState {
