diff --git a/chrome/browser/ui/navigator/browser_navigator.cc b/chrome/browser/ui/navigator/browser_navigator.cc
index a3a2000fe57f7095a85dbbcbbff473d41fdb83b5..ccba4c08e3268825391d78c01d0ab3c60a4be484 100644
--- a/chrome/browser/ui/navigator/browser_navigator.cc
+++ b/chrome/browser/ui/navigator/browser_navigator.cc
@@ -615,6 +615,7 @@ base::WeakPtr<content::NavigationHandle> NavigateImpl(
     }
   }
 #endif  // BUILDFLAG(IS_CHROMEOS)
+  BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL
 
   // Open System Apps in their standalone window if necessary.
   // TODO(crbug.com/40136163): Remove this code after we integrate with intent
@@ -851,6 +852,7 @@ base::WeakPtr<content::NavigationHandle> NavigateImpl(
          params->user_gesture)) {
       tab_to_insert->set_opener(
           tabs::TabInterface::MaybeGetFromContents(params->source_contents));
+      BRAVE_NAVIGATE_MARK_OPENER_FOR_SYNC_NAVIGATION
     }
   }
 
@@ -868,6 +870,7 @@ base::WeakPtr<content::NavigationHandle> NavigateImpl(
            params->user_gesture)) {
         tab_to_insert->set_opener(
             tabs::TabInterface::MaybeGetFromContents(params->source_contents));
+        BRAVE_NAVIGATE_MARK_OPENER_FOR_SYNC_NAVIGATION
       }
       contents_to_navigate_or_insert = tab_to_insert->GetContents();
 
