diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index c9bd6f356c812078f6e8f01475d5c181ded82ec6..f874d795f4e53059a59cda1d9c777201bca63530 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3003,6 +3003,7 @@ const FeatureEntry::FeatureVariation kDesktopSharePreviewVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
