diff --git a/chrome/services/qrcode_generator/qrcode_generator_service_impl.cc b/chrome/services/qrcode_generator/qrcode_generator_service_impl.cc
index ad9469445ff4b7665ef78b29982e6d2b998657a1..21cc4b9939f6a1cb4798234e56b0a2653aed5c6b 100644
--- a/chrome/services/qrcode_generator/qrcode_generator_service_impl.cc
+++ b/chrome/services/qrcode_generator/qrcode_generator_service_impl.cc
@@ -78,7 +78,7 @@ void QRCodeGeneratorServiceImpl::DrawDino(SkCanvas* canvas,
                                           const int dino_border_px,
                                           const SkPaint& paint_foreground,
                                           const SkPaint& paint_background) {
-  int dino_width_px = pixels_per_dino_tile * dino_image::kDinoHeight;
+  int dino_width_px = pixels_per_dino_tile * dino_image::kDinoWidth;
   int dino_height_px = pixels_per_dino_tile * dino_image::kDinoHeight;
 
   // If we request too big a dino, we'll clip. In practice the dino size
