diff --git a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
index e267dfd4d8b245be9103f0b29176f877b4bc9c50..67d6efa2652586be474d2c6400efcb3be746d989 100644
--- a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
+++ b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
@@ -825,6 +825,7 @@ SkPath IconLabelBubbleView::GetHighlightPath() const {
       SkIntToScalar(radii.lower_right()), SkIntToScalar(radii.lower_right()),
       SkIntToScalar(radii.lower_left()),  SkIntToScalar(radii.lower_left())};
 
+  BRAVE_ICON_LABEL_BUBBLE_VIEW_GET_HIGHLIGHT_PATH
   return SkPath().addRoundRect(rect, sk_radii);
 }
 
