diff --git a/chrome/browser/global_features.cc b/chrome/browser/global_features.cc
index e25568ebab1d3dfebda3da4f48fe2bab87adc452..af05e9f831f3c885de82dbff55f9d6f4a304425d 100644
--- a/chrome/browser/global_features.cc
+++ b/chrome/browser/global_features.cc
@@ -100,6 +100,7 @@
 #endif
 #endif
 
+GlobalFeatures* CreateBraveGlobalFeatures();
 namespace {
 
 // This is the generic entry point for test code to stub out browser
@@ -118,7 +119,7 @@ std::unique_ptr<GlobalFeatures> GlobalFeatures::CreateGlobalFeatures() {
     return GetFactory().Run();
   }
   // Constructor is protected.
-  return base::WrapUnique(new GlobalFeatures());
+  return base::WrapUnique(CreateBraveGlobalFeatures());
 }
 
 GlobalFeatures::~GlobalFeatures() = default;
