diff --git a/chrome/browser/startup/startup_features.cc b/chrome/browser/startup/startup_features.cc
index 9164e5408a1fa4c3ef64539d4b40d3210d34d100..557f216d72d5d326562b435a4a2fb60669c07fe5 100644
--- a/chrome/browser/startup/startup_features.cc
+++ b/chrome/browser/startup/startup_features.cc
@@ -9,7 +9,8 @@
 
 namespace features {
 
-BASE_FEATURE(kLaunchOnStartup, base::FEATURE_ENABLED_BY_DEFAULT);
+// kLaunchOnStartup feature state is enforced via plaster rewrite.
+BASE_FEATURE(kLaunchOnStartup, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE_ENUM_PARAM(LaunchOnStartupMode,
                         kLaunchOnStartupModeParam,
