diff --git a/chrome/browser/ui/omnibox/omnibox_edit_model.cc b/chrome/browser/ui/omnibox/omnibox_edit_model.cc
index f7a4641be19896a1e19d84d7732b31190fec4d56..1daa85b01b8663fa014511f397d819e1acbcee38 100644
--- a/chrome/browser/ui/omnibox/omnibox_edit_model.cc
+++ b/chrome/browser/ui/omnibox/omnibox_edit_model.cc
@@ -712,6 +712,7 @@ void OmniboxEditModel::StartAutocomplete(bool prevent_inline_autocomplete) {
     input_.set_lens_overlay_suggest_inputs(*suggest_inputs);
   }
 
+  BRAVE_OMNIBOX_EDIT_MODEL_START_AUTOCOMPLETE
   controller_->StartAutocomplete(input_);
 }
 
