diff --git a/chrome/common/importer/importer_bridge.h b/chrome/common/importer/importer_bridge.h
index 56d4259aba433df997802109cf5a8d28c03ebd13..26bb05d70c9674cab27112704b8957972ff2dab5 100644
--- a/chrome/common/importer/importer_bridge.h
+++ b/chrome/common/importer/importer_bridge.h
@@ -15,10 +15,12 @@
 #include "chrome/common/importer/importer_data_types.h"
 #include "chrome/common/importer/importer_url_row.h"
 #include "components/favicon_base/favicon_usage_data.h"
+#include "net/cookies/canonical_cookie.h"
 
 class GURL;
 struct ImportedBookmarkEntry;
 struct ImporterAutofillFormDataEntry;
+struct BraveStats;
 
 namespace autofill {
 struct PasswordForm;
@@ -66,6 +68,12 @@ class ImporterBridge : public base::RefCountedThreadSafe<ImporterBridge> {
   virtual void SetAutofillFormData(
       const std::vector<ImporterAutofillFormDataEntry>& entries) = 0;
 
+  virtual void SetCookies(
+      const std::vector<net::CanonicalCookie>& cookies) {};
+
+  virtual void UpdateStats(
+      const BraveStats& stats) {};
+
   // Notifies the coordinator that the import operation has begun.
   virtual void NotifyStarted() = 0;
 
