diff --git a/chrome/browser/extensions/api/safe_browsing_private/safe_browsing_private_event_router.cc b/chrome/browser/extensions/api/safe_browsing_private/safe_browsing_private_event_router.cc
index e0dbf81759b2..1595d52f4aa2 100644
--- a/chrome/browser/extensions/api/safe_browsing_private/safe_browsing_private_event_router.cc
+++ b/chrome/browser/extensions/api/safe_browsing_private/safe_browsing_private_event_router.cc
@@ -831,9 +831,11 @@ void SafeBrowsingPrivateEventRouter::RealtimeReportingPrefChanged(
 
 void SafeBrowsingPrivateEventRouter::IfAuthorized(
     base::OnceCallback<void(bool)> cont) {
+#if !defined(OS_ANDROID)
   if (binary_upload_service_) {
     binary_upload_service_->IsAuthorized(std::move(cont));
   }
+#endif
 }
 
 void SafeBrowsingPrivateEventRouter::ReportRealtimeEvent(
@@ -912,6 +914,8 @@ bool SafeBrowsingPrivateEventRouter::IsRealtimeReportingAvailable() {
   // This also implies that the user is managed.
   auto* user = GetChromeOSUser();
   return user && user->IsAffiliated();
+#elif defined(OS_ANDROID)
+  return false;
 #else
   return policy::ChromeBrowserCloudManagementController::IsEnabled();
 #endif
