diff --git a/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java b/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
index 72ad6e6d1df36d3a7ac426189a1a296b1b69c8c8..2787ce1b3dc881539697c0aed8a5e26319545b0d 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
@@ -320,6 +320,7 @@ public abstract class FullscreenHtmlApiHandlerBase
 
                     @Override
                     public void onHidden(Tab tab, @TabHidingType int reason) {
+                        BraveFullscreenHtmlApiHandlerBase.class.cast(FullscreenHtmlApiHandlerBase.this).mTabHiddenByChangedTabs = reason == TabHidingType.CHANGED_TABS;
                         // Clean up any fullscreen state that might impact other tabs.
                         mFullscreenManagerDelegate.onExitFullscreen(null);
                     }
