diff --git a/chrome/browser/ui/views/frame/browser_view.h b/chrome/browser/ui/views/frame/browser_view.h
index 6c4807bbaaf0a8e12e1a2df8545b4b55f491c297..77689a13b2b50f3cc5005e2b9199e287d730ce83 100644
--- a/chrome/browser/ui/views/frame/browser_view.h
+++ b/chrome/browser/ui/views/frame/browser_view.h
@@ -681,7 +681,7 @@ class BrowserView : public BrowserWindow,
   views::ClientView* CreateClientView(views::Widget* widget) override;
   views::View* CreateOverlayView() override;
 #if BUILDFLAG(IS_MAC)
-  views::View* CreateMacOverlayView();
+  virtual views::View* CreateMacOverlayView();
 #endif
   void OnWindowBeginUserBoundsChange() override;
   void OnWindowEndUserBoundsChange() override;
