diff --git a/chrome/installer/mac/signing/signing.py b/chrome/installer/mac/signing/signing.py
index 9a425e757229484755a2f656f8a7758c7538b676..e46ebea95c8d005b5a6f9a7afcd390f1599265c8 100644
--- a/chrome/installer/mac/signing/signing.py
+++ b/chrome/installer/mac/signing/signing.py
@@ -11,6 +11,7 @@ import os.path
 
 from . import commands
 from .model import CodeSignOptions, CodeSignedProduct, VerifyOptions
+from signing_helper import AddBravePartsForSigning
 
 _PROVISIONPROFILE_EXT = '.provisionprofile'
 _PROVISIONPROFILE_DEST = 'embedded.provisionprofile'
@@ -45,7 +46,7 @@ def get_parts(config):
             CodeSignedProduct(
                 '{.app_product}.app'.format(config),
                 config.base_bundle_id,
-                options=full_hardened_runtime_options,
+                options=CodeSignOptions.RESTRICT,
                 requirements=config.codesign_requirements_outer_app,
                 identifier_requirement=False,
                 entitlements='app-entitlements.plist',
@@ -76,7 +77,7 @@ def get_parts(config):
                 '{0.framework_dir}/Helpers/{0.product} Helper.app'.format(
                     config),
                 '{}.helper'.format(uncustomized_bundle_id),
-                options=full_hardened_runtime_options,
+                options=CodeSignOptions.RESTRICT,
                 verify_options=VerifyOptions.DEEP),
         'helper-renderer-app':
             CodeSignedProduct(
@@ -136,6 +137,7 @@ def get_parts(config):
             library_basename.replace('.dylib', ''),
             verify_options=VerifyOptions.DEEP)
 
+    AddBravePartsForSigning(parts, config)
     return parts
 
 
@@ -180,7 +182,7 @@ def sign_part(paths, config, part):
         part: The |model.CodeSignedProduct| to sign. The product's |path| must
             be in |paths.work|.
     """
-    command = ['codesign', '--sign', config.identity]
+    command = ['codesign', '--force', '--sign', config.identity]
     if config.notary_user:
         # Assume if the config has notary authentication information that the
         # products will be notarized, which requires a secure timestamp.
