diff --git a/chrome/browser/translate/translate_manager_browsertest.cc b/chrome/browser/translate/translate_manager_browsertest.cc
index 5e54bcd68c4fedb8370f99a59308463043b6ac89..e196d24563f8dc1692dd6cf69985070902763464 100644
--- a/chrome/browser/translate/translate_manager_browsertest.cc
+++ b/chrome/browser/translate/translate_manager_browsertest.cc
@@ -405,7 +405,7 @@ class TranslateManagerBrowserTest : public InProcessBrowserTest {
 
   virtual void InitFeatures() {
     scoped_feature_list_.InitWithFeatures(
-        {toast_features::kTranslateToast, features::kReadAnythingImprovedUi},
+        {toast_features::kTranslateToast, ::features::kReadAnythingImprovedUi},
         {});
   }
 
