diff --git a/chrome/browser/external_protocol/external_protocol_handler.cc b/chrome/browser/external_protocol/external_protocol_handler.cc
index ac475c93034fee60a08fea240bd3e3bb02dbf9a7..220af03f4468cd5e306a05cc8291afb58175a677 100644
--- a/chrome/browser/external_protocol/external_protocol_handler.cc
+++ b/chrome/browser/external_protocol/external_protocol_handler.cc
@@ -393,7 +393,7 @@ ExternalProtocolHandler::BlockState ExternalProtocolHandler::GetBlockState(
   if (scheme == "mailto") {
     base::UmaHistogramEnumeration(kBlockStateMetric,
                                   BlockStateMetric::kAllowedDefaultMail);
-    return DONT_BLOCK;
+    return UNKNOWN;
   }
 
   PrefService* profile_prefs = profile->GetPrefs();
