diff --git a/chrome/test/data/webui/settings/BUILD.gn b/chrome/test/data/webui/settings/BUILD.gn
index bfaba09bd3b370e6edfb62fb14dcaad299b8fe30..733406ace48deda3f707d71addf24235f6aaca99 100644
--- a/chrome/test/data/webui/settings/BUILD.gn
+++ b/chrome/test/data/webui/settings/BUILD.gn
@@ -235,6 +235,7 @@ ts_library("build_ts") {
     definitions +=
         [ "//tools/typescript/definitions/quick_unlock_private.d.ts" ]
   }
+  import("//brave/browser/resources/settings/sources.gni") definitions += brave_settings_ts_definitions
   deps = [
     "..:build_ts",
     "//chrome/browser/resources/settings:build_ts",
