diff --git a/chrome/browser/ui/navigator/browser_navigator.cc b/chrome/browser/ui/navigator/browser_navigator.cc
index 90ec94c299fed230e1d768ce23b7a8bdf757244c..2c3f5fafbf7b99a8f20f5745d1e5834f6ec7abee 100644
--- a/chrome/browser/ui/navigator/browser_navigator.cc
+++ b/chrome/browser/ui/navigator/browser_navigator.cc
@@ -585,6 +585,7 @@ base::WeakPtr<content::NavigationHandle> NavigateImpl(
   }
 #endif  // BUILDFLAG(IS_CHROMEOS)
 
+BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL
   // Open System Apps in their standalone window if necessary.
   // TODO(crbug.com/40136163): Remove this code after we integrate with intent
   // handling.
