diff --git a/chrome/browser/renderer_host/chrome_navigation_ui_data.cc b/chrome/browser/renderer_host/chrome_navigation_ui_data.cc
index cc451467f27f2f5054697281ff01d56b2937c5e5..c5cdc0fc623fb6bbe498430ffa8399534ccb1032 100644
--- a/chrome/browser/renderer_host/chrome_navigation_ui_data.cc
+++ b/chrome/browser/renderer_host/chrome_navigation_ui_data.cc
@@ -49,7 +49,7 @@ ChromeNavigationUIData::CreateForMainFrameNavigation(
     content::WebContents* web_contents,
     WindowOpenDisposition disposition,
     int64_t data_reduction_proxy_page_id) {
-  auto navigation_ui_data = std::make_unique<ChromeNavigationUIData>();
+  auto navigation_ui_data = std::make_unique<BraveNavigationUIData>();
   navigation_ui_data->disposition_ = disposition;
   navigation_ui_data->data_reduction_proxy_page_id_ =
       data_reduction_proxy_page_id;
@@ -70,7 +70,7 @@ ChromeNavigationUIData::CreateForMainFrameNavigation(
 
 std::unique_ptr<content::NavigationUIData> ChromeNavigationUIData::Clone()
     const {
-  auto copy = std::make_unique<ChromeNavigationUIData>();
+  auto copy = std::make_unique<BraveNavigationUIData>();
 
   copy->disposition_ = disposition_;
   copy->data_reduction_proxy_page_id_ = data_reduction_proxy_page_id_;
