diff --git a/chrome/browser/permissions/permission_manager.cc b/chrome/browser/permissions/permission_manager.cc
index c086144beda1906d34f828dae4afdb516ece0283..7cf09fc391fc70131f919204bfebd3d2ff4a85f0 100644
--- a/chrome/browser/permissions/permission_manager.cc
+++ b/chrome/browser/permissions/permission_manager.cc
@@ -86,7 +86,7 @@ PermissionStatus ContentSettingToPermissionStatus(ContentSetting setting) {
 }
 
 // Helper method to convert PermissionType to ContentSettingType.
-ContentSettingsType PermissionTypeToContentSetting(PermissionType permission) {
+ContentSettingsType PermissionTypeToContentSetting_ChromiumImpl(PermissionType permission) {
   switch (permission) {
     case PermissionType::MIDI:
       return CONTENT_SETTINGS_TYPE_MIDI;
@@ -128,6 +128,7 @@ ContentSettingsType PermissionTypeToContentSetting(PermissionType permission) {
     case PermissionType::IDLE_DETECTION:
       return CONTENT_SETTINGS_TYPE_IDLE_DETECTION;
     case PermissionType::NUM:
+    default:
       // This will hit the NOTREACHED below.
       break;
   }
