diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index ce551a6af47222d6613fd7f6aa8c03a52616e02e..3dd4f98c84a88d3de25043b92c4646e86c833f11 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -700,6 +700,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
