diff --git a/chrome/browser/extensions/api/developer_private/extension_info_generator.cc b/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
index 0f55a475d3ba3c6169a4fd27d54b008ed295d757..3d6c2867fe0ba7c4529ed272f9ff1f0f2e212dcc 100644
--- a/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
+++ b/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
@@ -710,6 +710,7 @@ void ExtensionInfoGenerator::FillExtensionInfo(const Extension& extension,
 #else
   info.incognito_access_pending_change = false;
 #endif
+  BRAVE_CREATE_EXTENSION_INFO_HELPER
 
   // User Scripts toggle.
   info.user_scripts_access.is_enabled =
