diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 59e23baa04ac441f9562ad867b05c268e600f0be..84c92d8f373799c56315c028ab417ec106a74a20 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2509,6 +2509,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
