diff --git a/chrome/browser/extensions/extension_service.h b/chrome/browser/extensions/extension_service.h
index 19f7410c444f76439e11003e00dbd25d3aeaafe7..fad23a713b4dd4b685177bbac5276da46a68fda2 100644
--- a/chrome/browser/extensions/extension_service.h
+++ b/chrome/browser/extensions/extension_service.h
@@ -461,6 +461,7 @@ class ExtensionService : public ExtensionServiceInterface,
   static base::AutoReset<bool> DisableExternalUpdatesForTesting();
 
  private:
+  friend class BraveExtensionService;
   // Loads extensions specified via a command line flag/switch.
   void LoadExtensionsFromCommandLineFlag(const char* switch_name);
 #if BUILDFLAG(IS_CHROMEOS_ASH)
