diff --git a/chrome/browser/resources/side_panel/read_anything/BUILD.gn b/chrome/browser/resources/side_panel/read_anything/BUILD.gn
index 478bebaa5676caedb0a5114cb9ec72baa9c01af4..0e707f3dc7c3ae0e4c71ba130ac930a8656145b0 100644
--- a/chrome/browser/resources/side_panel/read_anything/BUILD.gn
+++ b/chrome/browser/resources/side_panel/read_anything/BUILD.gn
@@ -87,7 +87,7 @@ build_webui("build") {
             target_gen_dir) ]
 
   webui_context_type = "untrusted"
-  optimize = optimize_webui
+  optimize = false
   if (optimize) {
     optimize_webui_host =
         "chrome-untrusted://read-anything-side-panel.top-chrome"
