diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index 878e8ff5ecdc560ec4253078833b5a8b1e78c6ee..bfd6c0cfbc4d373ccbbadf56ab5c93d3797bb19f 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -137,7 +137,7 @@ const char Profile::kProfileKey[] = "__PROFILE__";
 void Profile::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
       prefs::kSearchSuggestEnabled,
-      true,
+      false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 #if defined(OS_ANDROID)
   registry->RegisterStringPref(
