diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index 0e3a27eb469825c1bacf799b6e51dbe17d3667ce..50a4d8665e61d7fc175a65307a9b498e3d5ae540 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -34,7 +34,9 @@ function addPrivacyChildRoutes(r) {
 
   // TODO(tommycli): Find a way to refactor these repetitive category
   // routes.
+      // <if expr="_google_chrome">
   r.SITE_SETTINGS_ADS = r.SITE_SETTINGS.createChild('ads');
+      // </if>
   r.SITE_SETTINGS_AR = r.SITE_SETTINGS.createChild('ar');
   r.SITE_SETTINGS_AUTOMATIC_DOWNLOADS =
       r.SITE_SETTINGS.createChild('automaticDownloads');
