diff --git a/chrome/browser/web_applications/os_integration/web_app_shortcut_mac.mm b/chrome/browser/web_applications/os_integration/web_app_shortcut_mac.mm
index 5295ca55864dc3fa3d58fc304beac7d2dacf75e6..3f61895b81382b10a053895229ce472eb9f378d8 100644
--- a/chrome/browser/web_applications/os_integration/web_app_shortcut_mac.mm
+++ b/chrome/browser/web_applications/os_integration/web_app_shortcut_mac.mm
@@ -793,6 +793,7 @@ base::FilePath GetChromeAppsFolder() {
   if (path.empty())
     return path;
 
+  return path.Append(GetLocalizableBraveAppShortcutsSubdirName());
   return path.Append(GetLocalizableAppShortcutsSubdirName());
 }
 
