diff --git a/chrome/browser/actor/execution_engine.cc b/chrome/browser/actor/execution_engine.cc
index fbd59c5358166c316da3494d9c2cee3b0f42de90..c7d661d7db0a6b8f178c82d63a314dfa03092f12 100644
--- a/chrome/browser/actor/execution_engine.cc
+++ b/chrome/browser/actor/execution_engine.cc
@@ -1382,6 +1382,16 @@ void ExecutionEngine::SafetyChecksForNextAction() {
   // TODO(mcnee): Add UMA for the outcomes.
   content::WebContents& web_contents = *(tab->GetContents());
   const GURL& url = web_contents.GetPrimaryMainFrame()->GetLastCommittedURL();
+  BRAVE_MAY_ACT_ON_URL_INTERNAL(
+      base::BindOnce([](MayActOnUrlBlockReason block_reason) {
+        return BlockReasonToResultCode(block_reason,
+                                       /*for_navigation=*/false);
+      })
+          .Then(base::BindOnce(&ExecutionEngine::DidFinishAsyncSafetyChecks,
+                               GetActionSequenceWeakPtr(),
+                               tab->GetContents()
+                                   ->GetPrimaryMainFrame()
+                                   ->GetLastCommittedOrigin())))
   auto event = GateableEvent::kPageAction;
   origin_gating_checker_.ComputeGatingDecision(
       std::make_unique<PageActionGatingContext>(web_contents.GetWeakPtr()),
@@ -1695,6 +1705,7 @@ const EnterprisePolicyChecker& ExecutionEngine::GetEnterprisePolicyChecker()
 void ExecutionEngine::IsAcceptableNavigationDestination(
     const GURL& url,
     DecisionCallbackWithReason callback) {
+  BRAVE_MAY_ACT_ON_URL_INTERNAL(std::move(callback))
   auto event = GateableEvent::kNavigationRequest;
   origin_gating_checker_.ComputeGatingDecision(
       std::make_unique<OriginGatingDecisionContext>(), event, /*source=*/GURL(),
