diff --git a/chrome/browser/io_thread.cc b/chrome/browser/io_thread.cc
index 25cb1333363e32f3afe13239b40159dd3444e834..b6e8725370c9bc040b9f4b1ca3e5b61a7351144a 100644
--- a/chrome/browser/io_thread.cc
+++ b/chrome/browser/io_thread.cc
@@ -237,6 +237,7 @@ net::URLRequestContextGetter* IOThread::system_url_request_context_getter() {
     system_url_request_context_getter_ =
         base::MakeRefCounted<SystemURLRequestContextGetter>(this);
   } else {
+    if (base::FeatureList::IsEnabled(network::features::kNetworkService))
     NOTREACHED();
   }
   return system_url_request_context_getter_.get();
