diff --git a/chrome/installer/mac/sign_app.sh.in b/chrome/installer/mac/sign_app.sh.in
index 89b0028a1fb782c72021c4ce7f5db54d6dcc9b22..934363520f8212cddaab525e31c7bbf4a023dd73 100644
--- a/chrome/installer/mac/sign_app.sh.in
+++ b/chrome/installer/mac/sign_app.sh.in
@@ -70,10 +70,10 @@ fi
 
 requirement="\
 designated => \
-(identifier \"com.google.Chrome\" or \
-identifier \"com.google.Chrome.beta\" or \
-identifier \"com.google.Chrome.dev\" or \
-identifier \"com.google.Chrome.canary\") \
+(identifier \"com.brave.Browser\" or \
+identifier \"com.brave.Browser.beta\" or \
+identifier \"com.brave.Browser.dev\" or \
+identifier \"com.brave.Browser.nightly\") \
 ${requirement_suffix} \
 "
 
@@ -81,7 +81,6 @@ codesign_cmd=(
   codesign --sign "${codesign_id}" --keychain "${codesign_keychain}"
   "${browser_app}"
   --options "${enforcement_flags_app}"
-  --resource-rules "${browser_app_rules}"
 )
 if [[ -z "${is_development}" ]]; then
   codesign_cmd+=(--entitlements="${entitlements_plist}")
