diff --git a/chrome/test/data/webui/settings/BUILD.gn b/chrome/test/data/webui/settings/BUILD.gn
index 19d12945c24a589e6fe439d07762aa128b12ee2e..c5c55aa4b299bce8d91f55c16729a6f180449067 100644
--- a/chrome/test/data/webui/settings/BUILD.gn
+++ b/chrome/test/data/webui/settings/BUILD.gn
@@ -218,6 +218,7 @@ build_webui_tests("build") {
     ts_definitions +=
         [ "//tools/typescript/definitions/quick_unlock_private.d.ts" ]
   }
+  import("//brave/browser/resources/settings/sources.gni") ts_definitions += brave_settings_ts_definitions
 
   ts_tsconfig_base = "tsconfig_base.json"
   ts_deps = [
