diff --git a/chrome/browser/ui/startup/startup_tab_provider.cc b/chrome/browser/ui/startup/startup_tab_provider.cc
index c9222ab4cb9cba20d7e577b04b1f7e418df7e659..a2dc7ea400c615292a00c64398e3ad53696c6e9a 100644
--- a/chrome/browser/ui/startup/startup_tab_provider.cc
+++ b/chrome/browser/ui/startup/startup_tab_provider.cc
@@ -269,6 +269,7 @@ StartupTabs StartupTabProviderImpl::GetNewFeaturesTabs(
 bool StartupTabProviderImpl::CanShowWelcome(bool is_signin_allowed,
                                             bool is_supervised_user,
                                             bool is_force_signin_enabled) {
+  return true;
   return is_signin_allowed && !is_supervised_user && !is_force_signin_enabled;
 }
 
