diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 1d56a1949724f2546968c25c294918d1d03488ba..06f115d0f497291f41e87eedbdb1a4c7199a09eb 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -78,6 +78,7 @@ source_set("channel_info") {
   if (is_android) {
     public_deps += [ "//components/version_info/android:channel_getter" ]
   }
+  if (brave_chromium_build) { deps = [ "//brave/common:channel_info" ] }
 }
 
 source_set("ini_parser") {
@@ -207,6 +208,7 @@ static_library("common") {
   ]
 
   public_deps = [
+    "//brave/common",
     ":available_offline_content_mojom",
     ":buildflags",
     ":channel_info",
@@ -551,6 +553,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  if (brave_chromium_build) { configs += [ "//brave/common:constants_configs" ] }
 }
 
 component("chrome_features") {
