diff --git a/chrome/browser/ui/views/tabs/tab_strip.cc b/chrome/browser/ui/views/tabs/tab_strip.cc
index 1f7870f9ec645bd5eb15fc7b0e688bc9e60cb6eb..cbe8b005d8e11fbb00a2515127fcad34a128a3ab 100644
--- a/chrome/browser/ui/views/tabs/tab_strip.cc
+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
@@ -1632,7 +1632,7 @@ int TabStrip::TabToFollowingNewTabButtonSpacing() const {
   // Pre-refresh, there are a variety of hardcoded values.
   const int mode = MD::GetMode();
   DCHECK_LE(mode, 2);
-  constexpr int kNewTabButtonSpacing[] = {-5, -6, 6};
+  constexpr int kNewTabButtonSpacing[] = {6, 7, 7};
   return kNewTabButtonSpacing[mode];
 }
 
