diff --git a/chrome/browser/profiles/profile.h b/chrome/browser/profiles/profile.h
index 34dbcd06cb9139761dbeeb7dc93e38a30db31dfd..8cdbd9af04bd4b1ea919ae68da87674e30b025b4 100644
--- a/chrome/browser/profiles/profile.h
+++ b/chrome/browser/profiles/profile.h
@@ -265,6 +265,8 @@ class Profile : public content::BrowserContext {
   // Returns whether it is a guest session.
   virtual bool IsGuestSession() const;
 
+  bool IsTorProfile() const override;
+
   // Returns whether it is a system profile.
   virtual bool IsSystemProfile() const;
 
