diff --git a/chrome/browser/ui/views/bubble/webui_bubble_manager.h b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
index dcb094b40eb63abe1f1c8db7756cec5c327d7ebc..ce14e7e8c4e3d93669d26967c0a29e37717076bc 100644
--- a/chrome/browser/ui/views/bubble/webui_bubble_manager.h
+++ b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
@@ -164,10 +164,12 @@ class WebUIBubbleManagerImpl : public WebUIBubbleManager {
 
  private:
   // WebUIBubbleManager:
+ protected:
   base::WeakPtr<WebUIBubbleDialogView> CreateWebUIBubbleDialog(
       Anchor anchor,
       views::BubbleBorder::Arrow arrow) override;
   WebUIContentsWrapper* GetContentsWrapper() override;
+ private:
 
   const raw_ptr<BrowserWindowInterface> browser_window_interface_;
   const GURL webui_url_;
