diff --git a/chrome/browser/ui/tabs/features.cc b/chrome/browser/ui/tabs/features.cc
index 0f394292384b5e519d7a897c6cdfcb43ffd8d26c..6efb9e6a04139806af1b66377dd17b9c819b7f78 100644
--- a/chrome/browser/ui/tabs/features.cc
+++ b/chrome/browser/ui/tabs/features.cc
@@ -34,7 +34,7 @@ BASE_FEATURE(kTabSearchPerformanceImprovements,
              base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kVerticalTabsNewBadge, base::FEATURE_ENABLED_BY_DEFAULT);
 
-BASE_FEATURE(kVerticalTabsExpandOnHover, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kVerticalTabsExpandOnHover, base::FEATURE_ENABLED_BY_DEFAULT);
 BASE_FEATURE_PARAM(bool,
                    kVerticalTabsExpandOnHoverDefaultEnabled,
                    &kVerticalTabsExpandOnHover,
