diff --git a/chrome/browser/notifications/notification_platform_bridge_mac_utils.mm b/chrome/browser/notifications/notification_platform_bridge_mac_utils.mm
index c6c1ca3d3d95d03eacd0b23bbb5c4c6186ba9879..a5b7b5d168f9267c605880ac12e3befe1f3e04dc 100644
--- a/chrome/browser/notifications/notification_platform_bridge_mac_utils.mm
+++ b/chrome/browser/notifications/notification_platform_bridge_mac_utils.mm
@@ -60,6 +60,7 @@ void DoProcessMacNotificationResponse(
 // call this method directly as SysInfo::OperatingSystemVersionNumbers might be
 // an expensive call. Instead use SupportsAlerts which caches this value.
 bool MacOSSupportsXPCAlertsImpl() {
+  BRAVE_SUPPORTSALERTSIMPL_
   int32_t major, minor, bugfix;
   base::SysInfo::OperatingSystemVersionNumbers(&major, &minor, &bugfix);
   // Allow alerts on all versions except 10.15.0, 10.15.1 & 10.15.2.
