diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model_unittest.cc b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model_unittest.cc
index 1a306e3915bc1af56734e4f70aceff58ad9099e6..f4bed6f86a74f8b229cf040d45bca84d92a3fe7f 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model_unittest.cc
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model_unittest.cc
@@ -738,6 +738,7 @@ TEST_F(RecentTabsSubMenuModelTest, MaxTabsPerSessionAndRecency) {
 
   TestRecentTabsSubMenuModel model(nullptr, browser());
   int num_items = model.GetItemCount();
+  BRAVE_MAX_TABS_PER_SESSION_AND_RECENCY
   EXPECT_EQ(9, num_items);
 
   std::vector<std::u16string> tab_titles =
