diff --git a/chrome/browser/net/proxy_config_monitor.cc b/chrome/browser/net/proxy_config_monitor.cc
index d05a82a9c369a42cddebb1af1546a5ba6ae59c9f..84e56f7c24138ec3b509026320232c6e6590f99e 100644
--- a/chrome/browser/net/proxy_config_monitor.cc
+++ b/chrome/browser/net/proxy_config_monitor.cc
@@ -52,6 +52,7 @@ ProxyConfigMonitor::ProxyConfigMonitor(Profile* profile) {
             profile->GetPrefs(), g_browser_process->local_state());
   }
 
+  BRAVE_PROXY_CONFIG_MONITOR
   proxy_config_service_ = ProxyServiceFactory::CreateProxyConfigService(
       pref_proxy_config_tracker_.get(), profile);
 
