diff --git a/chrome/android/chrome_java_sources.gni b/chrome/android/chrome_java_sources.gni
index 5c4f5c571ac9aa9cb5fd938572507135d5f3d5c7..05694c1778e0330799e2bdcf39eafe73213e8cc0 100644
--- a/chrome/android/chrome_java_sources.gni
+++ b/chrome/android/chrome_java_sources.gni
@@ -1366,3 +1366,4 @@ if (enable_screen_capture) {
     "java/src/org/chromium/chrome/browser/media/TabSharingUIManager.java",
   ]
 }
+chrome_java_sources -= ["java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPageView.java",]
