diff --git a/chrome/browser/prefs/browser_prefs.cc b/chrome/browser/prefs/browser_prefs.cc
index dd94565a7158f92a8359e9ddd68c646d0de8d63d..220c497731afdd4a6ab62d1f15acf1b57f0fb2e8 100644
--- a/chrome/browser/prefs/browser_prefs.cc
+++ b/chrome/browser/prefs/browser_prefs.cc
@@ -1609,6 +1609,7 @@ void RegisterLocalState(PrefRegistrySimple* registry) {
 #endif  // !BUILDFLAG(IS_ANDROID)
 
   // This is intentionally last.
+  brave::RegisterLocalStatePrefs(registry);
   RegisterLocalStatePrefsForMigration(registry);
 }
 
@@ -2092,6 +2093,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry,
 #endif  // BUILDFLAG(IS_ANDROID)
 
   registry->RegisterBooleanPref(prefs::kStaticStorageQuotaEnabled, false);
+  brave::RegisterProfilePrefs(registry);
 }
 
 void RegisterUserProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -2507,6 +2509,7 @@ void MigrateObsoleteProfilePrefs(PrefService* profile_prefs,
   profile_prefs->ClearPref(kObsoletePasswordsUseUPMLocalAndSeparateStores);
   profile_prefs->ClearPref(kObsoleteEmptyProfileStoreLoginDatabase);
   profile_prefs->ClearPref(kObsoleteUpmAutoExportCsvNeedsDeletion);
+#if 0
   base::DeleteFile(profile_path.Append(FILE_PATH_LITERAL("Login Data")));
   base::DeleteFile(
       profile_path.Append(FILE_PATH_LITERAL("Login Data For Account")));
@@ -2514,6 +2517,7 @@ void MigrateObsoleteProfilePrefs(PrefService* profile_prefs,
       profile_path.Append(FILE_PATH_LITERAL("Login Data-journal")));
   base::DeleteFile(
       profile_path.Append(FILE_PATH_LITERAL("Login Data For Account-journal")));
+#endif
 #endif  // BUILDFLAG(IS_ANDROID)
 
   // Added 09/2025.
