diff --git a/chrome/browser/extensions/chrome_component_extension_resource_manager.cc b/chrome/browser/extensions/chrome_component_extension_resource_manager.cc
index ff14b77e36c1881d34cfd03a5ce075fa4873b2a4..fa1b313c8d05ab8fb5ee5b6be65561565ef3bf19 100644
--- a/chrome/browser/extensions/chrome_component_extension_resource_manager.cc
+++ b/chrome/browser/extensions/chrome_component_extension_resource_manager.cc
@@ -144,6 +144,7 @@ ChromeComponentExtensionResourceManager::Data::Data() {
         std::move(pdf_viewer_replacements);
   }
 #endif
+  BRAVE_CHROME_COMPONENT_EXTENSION_RESOURCE_MANAGER_DATA_DATA
 }
 
 void ChromeComponentExtensionResourceManager::Data::AddComponentResourceEntries(
