diff --git a/chrome/browser/ui/tabs/tab_strip_model.h b/chrome/browser/ui/tabs/tab_strip_model.h
index 2733d84a065e98f48fa048f2da12360819f661c2..6cb884ecb8a8d3a5ca19ee563158322dd5e18b8c 100644
--- a/chrome/browser/ui/tabs/tab_strip_model.h
+++ b/chrome/browser/ui/tabs/tab_strip_model.h
@@ -660,6 +660,7 @@ class TabStripModel : public TabGroupController {
   tabs::TabModel* GetTabForWebContents(
       const content::WebContents* contents) const;
 
+  TAB_STRIP_MODEL_H_
  private:
   FRIEND_TEST_ALL_PREFIXES(TabStripModelTest, GetIndicesClosedByCommand);
 
