diff --git a/chrome/browser/ui/views/tabs/tab_strip.cc b/chrome/browser/ui/views/tabs/tab_strip.cc
index 6096db72868ca2ac2684c4744e836a0f8d27acc6..fe472b1e4ae0f43e483b4dbb88e49cd72cff818b 100644
--- a/chrome/browser/ui/views/tabs/tab_strip.cc
+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
@@ -352,7 +352,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
            event.type() == ui::EventType::kGestureScrollBegin))
         << "Event type must be suitable for starting a drag.";
 
-    drag_controller_ = std::make_unique<TabDragController>();
+    drag_controller_ = std::make_unique<BraveTabDragController>();
 
     // !!! Init may delete `drag_controller_` on some platforms. !!!
     // Init takes capture, which on some platforms may reenter Chrome, the
@@ -593,6 +593,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
 
   std::vector<gfx::Rect> CalculateBoundsForDraggedViews(
       const std::vector<TabSlotView*>& views) override {
+    BRAVE_TAB_DRAG_CONTEXT_IMPL_CALCULATE_BOUNDS_FOR_DRAGGED_VIEWS
     CHECK(!views.empty()) << "The views vector must not be empty.";
 
     std::vector<gfx::Rect> bounds;
@@ -942,6 +943,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
         continue;
       }
 
+      BRAVE_TAB_DRAG_CONTEXT_IMPL_CALCULATE_INSERTION_INDEX
       // If there's a group header here, and we're dragging a group, we might
       // end up on either side of that header. Check both cases to find the
       // best option.
@@ -1501,7 +1503,7 @@ void TabStrip::SetSelection(const ui::ListSelectionModel& new_selection) {
       const tab_groups::TabGroupId new_group = selected_tab->group().value();
       // If the tab that is about to be selected is in a collapsed group,
       // automatically expand the group.
-      if (IsGroupCollapsed(new_group)) {
+      if (IsGroupCollapsed(new_group) && IsTabStripEditable()) {
         // If the group is being dragged, do not expand it.
         if (drag_context_->GetDragController() &&
             drag_context_->GetDragController()->group_header_id() ==
