diff --git a/chrome/browser/ui/tab_contents/chrome_web_contents_view_handle_drop.cc b/chrome/browser/ui/tab_contents/chrome_web_contents_view_handle_drop.cc
index ed6b208d93cb..76a5057f8f5f 100644
--- a/chrome/browser/ui/tab_contents/chrome_web_contents_view_handle_drop.cc
+++ b/chrome/browser/ui/tab_contents/chrome_web_contents_view_handle_drop.cc
@@ -17,6 +17,7 @@
 #include "content/public/browser/web_contents_view_delegate.h"
 #include "content/public/common/drop_data.h"
 
+#if !defined(OS_ANDROID)
 namespace {
 
 void DeepScanCompletionCallback(
@@ -112,3 +113,4 @@ void HandleOnPerformDrop(
         base::BindOnce(&ScanData, web_contents, std::move(callback)));
   }
 }
+#endif
