diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index c679d6b5d1e8f3bd2c26f91d1c1f6d82547f2f06..a7d026d8d710b94aafa699a45fb7f9768198691f 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -78,6 +78,7 @@ source_set("channel_info") {
   if (is_android) {
     public_deps += [ "//components/version_info/android:channel_getter" ]
   }
+  if (brave_chromium_build) { deps = [ "//brave/common:channel_info" ] }
 }
 
 source_set("ini_parser") {
@@ -548,6 +549,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  if (brave_chromium_build) { configs += [ "//brave/common:constants_configs" ] }
 }
 
 component("chrome_features") {
