diff --git a/chrome/browser/ui/views/tabs/tab_strip.cc b/chrome/browser/ui/views/tabs/tab_strip.cc
index 4d333946de0433ccca2063426b35e83a5356e43d..28006495ac88fe6f3d7bff5bc5ef4a1db5c8a3c5 100644
--- a/chrome/browser/ui/views/tabs/tab_strip.cc
+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
@@ -421,7 +421,7 @@ void TabStrip::StopAllHighlighting() {
 void TabStrip::AddTabAt(int model_index, TabRendererData data, bool is_active) {
   const bool was_single_tab_mode = SingleTabMode();
 
-  Tab* tab = new Tab(this, animation_container_.get());
+  Tab* tab = new BraveTab(this, animation_container_.get());
   AddChildView(tab);
   const bool pinned = data.pinned;
   tab->SetData(std::move(data));
