diff --git a/chrome/browser/ui/views/profiles/profile_chooser_view.cc b/chrome/browser/ui/views/profiles/profile_chooser_view.cc
index 519d359adef052081e55b2431d18a363cdc12e5a..f7bde017f70536cfda717eaa014c5aab245d089b 100644
--- a/chrome/browser/ui/views/profiles/profile_chooser_view.cc
+++ b/chrome/browser/ui/views/profiles/profile_chooser_view.cc
@@ -715,9 +715,12 @@ void ProfileChooserView::AddOptionsView(bool display_lock,
           l10n_util::GetStringUTF16(IDS_PROFILES_OPEN_GUEST_PROFILE_BUTTON));
     }
   }
+  static_cast<BraveProfileChooserView*>(this)->AddTorButton(&menu_items);
 
   base::string16 text = l10n_util::GetStringUTF16(
       is_guest ? IDS_PROFILES_EXIT_GUEST : IDS_PROFILES_MANAGE_USERS_BUTTON);
+  if (browser()->profile()->IsTorProfile())
+    text = l10n_util::GetStringUTF16(IDS_PROFILES_EXIT_TOR);
   const gfx::VectorIcon& settings_icon =
       is_guest ? kCloseAllIcon : kSettingsIcon;
   users_button_ = CreateAndAddButton(CreateVectorIcon(settings_icon), text);
