diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 41a02f8a8a207a145c659b1b572badf402ff5514..56850b9da3a8887befa2bba3213d69d57cf74855 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -257,6 +257,7 @@ static_library("utility") {
   if (enable_on_device_translation) {
     deps += [ "//components/on_device_translation/service:on_device_translation_service" ]
   }
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
 }
 
 if (!is_android) {
