diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 343c8a0fbfeb6e625892b6d4d529e19c53a0bd65..0c6495042ec94420d4604802d12c816e51b21014 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -1921,6 +1921,7 @@ jumbo_split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
