diff --git a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
index db79fc3db5e8945cc463b4fadac89dde57cd91b3..d476a9086165cb6015b267a1f002b2e147c3bf02 100644
--- a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
+++ b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
@@ -47,6 +47,7 @@ class TabHoverCardBubbleView : public views::BubbleDialogDelegateView {
   void ClearPreviewImage();
   void SetPreviewImage(gfx::ImageSkia preview_image);
 
+  BRAVE_TAB_HOVER_CARD_BUBBLE_VIEW_H_
  private:
   friend class TabHoverCardBubbleViewBrowserTest;
   friend class TabHoverCardBubbleViewInteractiveUiTest;
