diff --git a/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc b/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc
index bdf3cccaff2f807fb94c7d53c320227c04937ce8..d8a222b31b2265d216f1617ee2c5ab8c6c673f5f 100644
--- a/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc
+++ b/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc
@@ -136,6 +136,7 @@ PermissionPromptBubbleView::PermissionPromptBubbleView(
 
   for (permissions::PermissionRequest* request : GetVisibleRequests())
     AddRequestLine(request);
+  BRAVE_PERMISSION_PROMPT_BUBBLE_VIEW
 
   absl::optional<std::u16string> extra_text = GetExtraText();
   if (extra_text.has_value()) {
