diff --git a/chrome/browser/ui/webui/settings/import_data_handler.cc b/chrome/browser/ui/webui/settings/import_data_handler.cc
index cdbe7b0119d5daffbabc669c08de49cae4680b03..9a08618d88c60eb3b51777de885d6ad1a66c5ea5 100644
--- a/chrome/browser/ui/webui/settings/import_data_handler.cc
+++ b/chrome/browser/ui/webui/settings/import_data_handler.cc
@@ -146,6 +146,7 @@ void ImportDataHandler::HandleImportData(const base::ListValue& args) {
   if (*type_dict.FindBool(prefs::kImportDialogSearchEngine)) {
     selected_items |= user_data_importer::SEARCH_ENGINES;
   }
+  BRAVE_IMPORT_DATA
 
   const user_data_importer::SourceProfile& source_profile =
       importer_list_->GetSourceProfileAt(browser_index);
@@ -226,6 +227,7 @@ void ImportDataHandler::SendBrowserProfileData(const std::string& callback_id) {
         "autofillFormData",
         (browser_services & user_data_importer::AUTOFILL_FORM_DATA) != 0);
 
+    BRAVE_SEND_BROWSER_PROFILE_DATA
     browser_profiles.Append(std::move(browser_profile));
   }
 
