diff --git a/chrome/browser/resources/downloads/downloads.ts b/chrome/browser/resources/downloads/downloads.ts
index f4036a3479f94b642164ac1c52b7c037389fae95..3287be58d54c34a097a1b8e404d11bed8171aa0c 100644
--- a/chrome/browser/resources/downloads/downloads.ts
+++ b/chrome/browser/resources/downloads/downloads.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './manager.js';
 
 export {CrIconElement} from 'chrome://resources/cr_elements/cr_icon/cr_icon.js';
