diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.cc b/chrome/browser/ui/views/location_bar/location_bar_view.cc
index 10bea76273f59c769553722bc2070850ed872d6b..d34a76d834e144ec35aeae55e391d241bd31c71c 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
@@ -424,7 +424,7 @@ gfx::Size LocationBarView::CalculatePreferredSize() const {
   return gfx::Size(width, height);
 }
 
-void LocationBarView::Layout() {
+void LocationBarView::Layout(views::View* right_most) {
   if (!IsInitialized())
     return;
 
@@ -517,6 +517,11 @@ void LocationBarView::Layout() {
     }
   };
 
+  if (right_most && right_most->visible())
+    trailing_decorations.AddDecoration(0,
+                              height(),
+                              false, 0, 0, right_most);
+
   if (star_view_)
     add_trailing_decoration(star_view_);
   add_trailing_decoration(page_action_icon_container_view_);
