diff --git a/chrome/browser/ui/views/extensions/extensions_menu_view_controller.cc b/chrome/browser/ui/views/extensions/extensions_menu_view_controller.cc
index de634967cf45a731acc4ec4d7fb0abad7ff7d5ff..e94d1ef550c33248f6fef9ec02685e064a4df335 100644
--- a/chrome/browser/ui/views/extensions/extensions_menu_view_controller.cc
+++ b/chrome/browser/ui/views/extensions/extensions_menu_view_controller.cc
@@ -337,6 +337,7 @@ ExtensionsMenuViewController::~ExtensionsMenuViewController() {
 }
 
 void ExtensionsMenuViewController::OpenMainPage() {
+  BRAVE_EXTENSIONS_MENU_VIEW_CONTROLLER_OPEN_MAIN_PAGE
   auto main_page = std::make_unique<ExtensionsMenuMainPageView>(browser_, this);
   UpdateMainPage(main_page.get(), GetActiveWebContents());
   PopulateMainPage(main_page.get());
