diff --git a/chrome/browser/ui/unload_controller.cc b/chrome/browser/ui/unload_controller.cc
index 6ac2d32db9d0ba5775ba07deee644a210d3b1b68..a53883f028789743a8af785db94f2ec63989db92 100644
--- a/chrome/browser/ui/unload_controller.cc
+++ b/chrome/browser/ui/unload_controller.cc
@@ -176,6 +176,9 @@ void UnloadController::OnWindowClosing() {
   if (tab_restore_service && notify_restore_service) {
     tab_restore_service->BrowserClosing(BrowserLiveTabContext::From(browser_));
   }
+  if (MaybeHandleSharedPinnedTabsClosing(browser_)) {
+    return;
+  }
 
   if (!browser_->GetTabStripModel()->empty()) {
     // Closing all the tabs results in eventually calling back to
@@ -452,6 +455,7 @@ UnloadController::GetBrowserClosingStatus() {
 bool UnloadController::TryToCloseWindow(
     bool skip_beforeunload,
     const base::RepeatingCallback<void(bool)>& on_close_confirmed) {
+  confirmed_to_close_ = true;
   cancel_download_confirmation_state_ =
       CancelDownloadConfirmationState::kResponseReceived;
   // The devtools browser gets its beforeunload events as the results of
@@ -475,6 +479,7 @@ bool UnloadController::TryToCloseWindow(
 }
 
 void UnloadController::ResetTryToCloseWindow() {
+  confirmed_to_close_ = false;
   cancel_download_confirmation_state_ =
       CancelDownloadConfirmationState::kNotPrompted;
   if (!is_calling_before_unload_handlers()) {
@@ -573,6 +578,9 @@ void UnloadController::OnTabStripModelChanged(
 }
 
 void UnloadController::TabStripEmpty() {
+  if (!browser_->GetProfile()->GetPrefs()->GetBoolean(kEnableClosingLastTab)) {
+    return;
+  }
   // Set is_attempting_to_close_browser_ here, so that extensions, etc, do not
   // attempt to add tabs to the browser before it closes.
   is_attempting_to_close_browser_ = true;
@@ -922,6 +930,9 @@ void UnloadController::InProgressDownloadResponse(bool cancel_downloads) {
 }
 
 void UnloadController::FinishWarnBeforeClosing(WarnBeforeClosingResult result) {
+  if (result == WarnBeforeClosingResult::kDoNotClose) {
+    confirmed_to_close_ = false;
+  }
   switch (result) {
     case WarnBeforeClosingResult::kOkToClose:
       chrome::CloseWindow(browser_);
