diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index ef081dc339432bc7cb4e448b2616c6f8ba5567f0..416467abd1d9989d6eedc865ad2bc79acf655a18 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -840,6 +840,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
