diff --git a/chrome/browser/ui/browser_navigator.cc b/chrome/browser/ui/browser_navigator.cc
index 8b29fe206e052ad2212bcd724410075270de68f4..7258c37bac95005f5bccb077647d5f9a0b718e21 100644
--- a/chrome/browser/ui/browser_navigator.cc
+++ b/chrome/browser/ui/browser_navigator.cc
@@ -730,6 +730,9 @@ void Navigate(NavigateParams* params) {
 bool IsHostAllowedInIncognito(const GURL& url) {
   std::string scheme = url.scheme();
   base::StringPiece host = url.host_piece();
+#if defined(BRAVE_CHROMIUM_BUILD)
+  if (!IsHostAllowedInIncognitoBraveImpl(&scheme, host)) return false;
+#endif
   if (scheme == chrome::kChromeSearchScheme) {
     return host != chrome::kChromeUIThumbnailHost &&
            host != chrome::kChromeUIThumbnailHost2 &&
