diff --git a/chrome/browser/signin/identity_manager_factory.h b/chrome/browser/signin/identity_manager_factory.h
index 57d8f7caedfb91d0a7bef5febd390d2df3497e29..ceeb6a1fddaf5c0f2df3a038cc199fcf77144c0e 100644
--- a/chrome/browser/signin/identity_manager_factory.h
+++ b/chrome/browser/signin/identity_manager_factory.h
@@ -52,6 +52,7 @@ class IdentityManagerFactory : public BrowserContextKeyedServiceFactory {
   void AddObserver(Observer* observer);
   void RemoveObserver(Observer* observer);
 
+ BRAVE_IDENTITY_MANAGER_FACTORY_H_
  private:
   friend struct base::DefaultSingletonTraits<IdentityManagerFactory>;
 
