diff --git a/chrome/browser/extensions/api/content_settings/content_settings_api.cc b/chrome/browser/extensions/api/content_settings/content_settings_api.cc
index cd71b0bd21c4..fe3d996fe817 100644
--- a/chrome/browser/extensions/api/content_settings/content_settings_api.cc
+++ b/chrome/browser/extensions/api/content_settings/content_settings_api.cc
@@ -323,7 +323,7 @@ ContentSettingsContentSettingGetResourceIdentifiersFunction::Run() {
     return RespondNow(NoArguments());
   }
 
-#if BUILDFLAG(ENABLE_PLUGINS)
+#if BUILDFLAG(ENABLE_PLUGINS) && !defined(OS_ANDROID)
   content::PluginService::GetInstance()->GetPlugins(base::BindOnce(
       &ContentSettingsContentSettingGetResourceIdentifiersFunction::
           OnGotPlugins,
@@ -333,7 +333,7 @@ ContentSettingsContentSettingGetResourceIdentifiersFunction::Run() {
   return RespondLater();
 }
 
-#if BUILDFLAG(ENABLE_PLUGINS)
+#if BUILDFLAG(ENABLE_PLUGINS) && !defined(OS_ANDROID)
 void ContentSettingsContentSettingGetResourceIdentifiersFunction::OnGotPlugins(
     const std::vector<content::WebPluginInfo>& plugins) {
   DCHECK_CURRENTLY_ON(BrowserThread::UI);
