diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index e01cd90af97607ff5c7ce49500a949a67926f0c1..83656a8a3960989e44b28a1a53e7f3eeaa3f4e19 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -32,6 +32,7 @@ if (optimize_webui) {
 
     deps = [
       ":unpak",
+      "//brave/browser/resources/settings:unpak",
     ]
   }
 
