diff --git a/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc b/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc
index af30cbec80a56a112b33f801b71df855229f0aed..4f2e73fb22a4fe6accee2ed6cec5052fc3e3d2f4 100644
--- a/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc
+++ b/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc
@@ -104,6 +104,8 @@ class MockPage : public tab_search::mojom::Page {
   MOCK_METHOD(void, TabUpdated, (tab_search::mojom::TabUpdateInfoPtr));
   MOCK_METHOD(void, TabsRemoved, (tab_search::mojom::TabsRemovedInfoPtr));
   MOCK_METHOD(void, TabUnsplit, ());
+  MOCK_METHOD(void, TabFocusShowFREChanged, (bool show));
+  MOCK_METHOD(void, TabFocusEnabledChanged, (bool enabled));
 };
 
 void ExpectNewTab(const tab_search::mojom::Tab* tab,
