diff --git a/chrome/browser/resources/extensions/BUILD.gn b/chrome/browser/resources/extensions/BUILD.gn
index adb5e2d759d3f037f70ac3e8e9ddcc86eeef0f9e..95db3f10170da490de85a79896847643c91b07ad 100644
--- a/chrome/browser/resources/extensions/BUILD.gn
+++ b/chrome/browser/resources/extensions/BUILD.gn
@@ -105,4 +105,5 @@ build_webui("build") {
     optimize_webui_resource_paths_rewrites =
         [ "extensions.rollup.js|extensions.js" ]
   }
+  extra_build_grd_deps = [ "//brave/browser/resources/extensions:resources" ] extra_build_ts_in_files = brave_deps_chrome_browser_resources_extensions_in_files extra_build_ts_extra_deps = brave_deps_chrome_browser_resources_extensions_extra_deps
 }
