diff --git a/chrome/browser/ui/tab_contents/chrome_web_contents_view_handle_drop.cc b/chrome/browser/ui/tab_contents/chrome_web_contents_view_handle_drop.cc
index 1dc42d2755f83945673d01813368fa40c77af442..9824cb2827f517aae711797ccc943ce8710c32e4 100644
--- a/chrome/browser/ui/tab_contents/chrome_web_contents_view_handle_drop.cc
+++ b/chrome/browser/ui/tab_contents/chrome_web_contents_view_handle_drop.cc
@@ -239,6 +239,8 @@ void HandleOnPerformingDrop(
     content::WebContents* web_contents,
     content::DropData drop_data,
     content::WebContentsViewDelegate::DropCompletionCallback callback) {
+  callback = brave::MaybeStripImageMetadataForDrop(web_contents,
+                                                   std::move(callback));
   CHECK(callback);
 
   content::WebContents* scan_target =
