diff --git a/chrome/browser/resources/certificate_manager/certificate_manager.ts b/chrome/browser/resources/certificate_manager/certificate_manager.ts
index dd49f8d34a3c907d6151f862bab47c708265b861..1657b321ad070ed7d8b9ef0728788959c0db619f 100644
--- a/chrome/browser/resources/certificate_manager/certificate_manager.ts
+++ b/chrome/browser/resources/certificate_manager/certificate_manager.ts
@@ -459,4 +459,4 @@ declare global {
   }
 }
 
-customElements.define(CertificateManagerElement.is, CertificateManagerElement);
+// customElements.define(CertificateManagerElement.is, CertificateManagerElement);
