diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 7151a2ff7d5f9576b2ae3d01c158e99be281ba34..7fafdf0c1c609c2c7ef157f8eea2b0fc7badfcf7 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -755,6 +755,7 @@ static_library("ui") {
 
     "//chrome/browser/task_manager:impl",
   ]
+  import("//brave/browser/sources.gni") public_deps += [ "//brave/browser/ui" ] deps += brave_chrome_browser_ui_allow_circular_includes_from allow_circular_includes_from += brave_chrome_browser_ui_allow_circular_includes_from
 
   if (enable_vr && is_win) {
     deps += [ "//chrome/browser/vr:vr_base" ]
@@ -3544,7 +3545,10 @@ static_library("ui") {
       ]
       deps += [ "//chrome/updater/app/server/win:updater_legacy_idl" ]
     } else {
-      sources += [ "webui/help/version_updater_basic.cc" ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [ "webui/help/version_updater_basic.cc" ]
+      }
     }
   }
 
@@ -5732,6 +5736,7 @@ static_library("ui_public_dependencies") {
   if (enable_extensions_core) {
     public_deps += [ "//extensions/browser" ]
   }
+  public_deps += [ "//brave/browser/ui:ui_public_dependencies" ]
   if (!is_android) {
     public_deps += [
       "//build:branding_buildflags",
