diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index 8a1ab3c04c4451192aa9e93f652d3ab64f57c6ec..47491ab52d7a01e315cba6c27396e6151d4b87ff 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -31,6 +31,7 @@ if (optimize_webui) {
 
     deps = [
       ":unpak",
+      "//brave/browser/resources/settings:unpak",
       "../../../../ui/webui/resources:modulize",
     ]
     excludes = [ "chrome://resources/js/cr.m.js" ]
