diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 1bc91a001d904e6c200e6525692e7658a02a1f4b..088f06a345e007c59eedea2d8d3d181be4f78485 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -243,6 +243,7 @@ static_library("renderer") {
     deps += [ "//components/webapps/isolated_web_apps:scheme" ]
   }
 
+  import("//brave/renderer/sources.gni") sources += brave_chrome_renderer_sources deps += brave_chrome_renderer_deps public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
