diff --git a/chrome/browser/shell_integration.h b/chrome/browser/shell_integration.h
index fdb4c9d1d59009577d4521ba32374984b109e864..c4e668791cf82b6269634c5a97381aae3b58034e 100644
--- a/chrome/browser/shell_integration.h
+++ b/chrome/browser/shell_integration.h
@@ -96,6 +96,13 @@ bool IsFirefoxDefaultBrowser();
 // Returns true if IE is likely to be the default browser for the current
 // user. This method is very fast so it can be invoked in the UI thread.
 bool IsIEDefaultBrowser();
+
+// Returns the install id of the installation set as default browser. If no
+// installation of Firefox is set as the default browser, returns an empty
+// string.
+// TODO(crbug/1011830): This should return the install id of the stable
+// version if no version of Firefox is set as default browser.
+std::string GetFirefoxProgIdSuffix();
 #endif
 
 // Attempt to determine if this instance of Chrome is the default client
