diff --git a/chrome/browser/resources/bookmarks/BUILD.gn b/chrome/browser/resources/bookmarks/BUILD.gn
index d3f2006f302ddaf57a7d9c4f4949ba44a0db61e1..70a1d7a90f09062e8d21f69b230d4f1262e731ae 100644
--- a/chrome/browser/resources/bookmarks/BUILD.gn
+++ b/chrome/browser/resources/bookmarks/BUILD.gn
@@ -74,4 +74,5 @@ build_webui("build") {
     optimize_webui_resource_paths_rewrites =
         [ "bookmarks.rollup.js|bookmarks.js" ]
   }
+  extra_build_grd_deps = [ "//brave/browser/resources/bookmarks:resources" ] extra_build_ts_in_files = brave_deps_chrome_browser_resources_bookmarks_in_files extra_build_ts_extra_deps = brave_deps_chrome_browser_resources_bookmarks_extra_deps
 }
