diff --git a/chrome/browser/ui/extensions/extension_action_view_controller.cc b/chrome/browser/ui/extensions/extension_action_view_controller.cc
index 26c9b59886eb3f944e9abff78832ca06e10d19aa..8b24719dc890eee9761edb1e88dd4ba47f6c173c 100644
--- a/chrome/browser/ui/extensions/extension_action_view_controller.cc
+++ b/chrome/browser/ui/extensions/extension_action_view_controller.cc
@@ -67,7 +67,6 @@ ExtensionActionViewController::Create(
     Browser* browser,
     ExtensionsContainer* extensions_container) {
   DCHECK(browser);
-  DCHECK(extensions_container);
 
   auto* registry = extensions::ExtensionRegistry::Get(browser->profile());
   scoped_refptr<const extensions::Extension> extension =
