diff --git a/chrome/common/importer/importer_bridge.h b/chrome/common/importer/importer_bridge.h
index b4250c91d1b83ea920b3de9cd6b1a7929b30ffc5..da3567e08fae6f01d3b518378ccaa9eac5aa81ca 100644
--- a/chrome/common/importer/importer_bridge.h
+++ b/chrome/common/importer/importer_bridge.h
@@ -78,6 +78,7 @@ class ImporterBridge : public base::RefCountedThreadSafe<ImporterBridge> {
   // external import process, since the importer only needs a few strings.
   virtual base::string16 GetLocalizedString(int message_id) = 0;
 
+  BRAVE_IMPORTER_BRIDGE_H
  protected:
   friend class base::RefCountedThreadSafe<ImporterBridge>;
 
