diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 6e8a582a7af7892de583731c9411a65c79708521..5d54b09f19a92219b83df58bdd68b91a0909db3b 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/sources.gni")
 import("//build/config/buildflags_paint_preview.gni")
 import("//build/config/chrome_build.gni")
 import("//build/config/compiler/compiler.gni")
@@ -1190,6 +1191,7 @@ if (!is_android) {
       ":ui",
       "//chrome/browser/ui/browser_window/internal",
     ]
+    visibility += brave_chrome_browser_ui_browser_header_visibility
     public_deps = [
       ":browser_window_deleter",
       "//base",
@@ -1246,6 +1248,7 @@ static_library("ui") {
     ":simple_message_box_headers",
     ":tab_dialogs",
     ":tab_modal_confirm_dialog",
+    "//brave/browser/ui",
     "//chrome/browser:browser_public_dependencies",
     "//chrome/browser/feedback",
     "//chrome/browser/headless",
@@ -1524,6 +1527,8 @@ static_library("ui") {
     "//ui/strings",
     "//ui/webui",
   ]
+  deps += brave_chrome_browser_ui_allow_circular_includes_from
+  deps += brave_chrome_browser_ui_deps
 
   allow_circular_includes_from = [
     # page_load_metrics's :impl includes headers from //chrome/browser/ui that
@@ -1547,6 +1552,8 @@ static_library("ui") {
     # - c/b/ui/views/frame/browser_view.h
     "//chrome/browser/ui/views/file_system_access:impl",
   ]
+  allow_circular_includes_from +=
+      brave_chrome_browser_ui_allow_circular_includes_from
 
   # Desktop-Android-only dependencies.
   if (is_desktop_android) {
@@ -4368,11 +4375,13 @@ static_library("ui") {
 group("ui_public_dependencies") {
   visibility = [ "//chrome/browser:browser_public_dependencies" ]
   public_deps = [
+    "//brave/browser/ui:ui_public_dependencies",
     "//components/cross_device/logging",
     "//components/dom_distiller/core",
     "//components/enterprise/buildflags",
     "//components/paint_preview/buildflags",
     "//components/safe_browsing:buildflags",
+    "//components/saved_tab_groups/public",
     "//components/segmentation_platform/public",
     "//components/sync",
     "//components/sync_user_events",
