diff --git a/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc b/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc
index 6ee8837135be6b1f9389827dac26ecfa5a2a4fd6..e38da9061e07072a56fc4b3a82a0dcff20a2d0bf 100644
--- a/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc
+++ b/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc
@@ -996,6 +996,7 @@ void AddImportDataStrings(content::WebUIDataSource* html_source) {
     {"importPasswords", IDS_SETTINGS_IMPORT_PASSWORDS_CHECKBOX},
     {"importSearch", IDS_SETTINGS_IMPORT_SEARCH_ENGINES_CHECKBOX},
     {"importAutofillFormData", IDS_SETTINGS_IMPORT_AUTOFILL_FORM_DATA_CHECKBOX},
+    {"importCookies", IDS_SETTINGS_IMPORT_COOKIES_CHECKBOX},
     {"importChooseFile", IDS_SETTINGS_IMPORT_CHOOSE_FILE},
     {"importCommit", IDS_SETTINGS_IMPORT_COMMIT},
     {"noProfileFound", IDS_SETTINGS_IMPORT_NO_PROFILE_FOUND},
