diff --git a/chrome/browser/ui/views/side_panel/BUILD.gn b/chrome/browser/ui/views/side_panel/BUILD.gn
index 777b410350cf10c1c7ddd9f657415b21ea6a835a..bf794f789e8ea8ef757c0b9edb22fc95273ad3fc 100644
--- a/chrome/browser/ui/views/side_panel/BUILD.gn
+++ b/chrome/browser/ui/views/side_panel/BUILD.gn
@@ -2,6 +2,8 @@
 # Use of this source code is governed by a BSD - style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/ui/views/side_panel/sources.gni")
+
 assert(is_win || is_mac || is_linux || is_chromeos)
 
 import("//chrome/common/features.gni")
@@ -89,6 +91,7 @@ source_set("side_panel") {
     "//ui/webui/resources/cr_components/history_embeddings:mojo_bindings",
     "//url",
   ]
+  public_deps += brave_browser_ui_views_side_panel_public_deps
   deps = [
     "//base:i18n",
     "//chrome/app:generated_resources",
@@ -174,6 +177,7 @@ source_set("side_panel") {
     "//ui/gfx/geometry:geometry_skia",
     "//ui/webui",
   ]
+  deps += brave_browser_ui_views_side_panel_deps
 
   if (enable_extensions_core) {
     public_deps += [
