diff --git a/chrome/browser/ui/views/status_bubble_views.h b/chrome/browser/ui/views/status_bubble_views.h
index 60e3dc23b6026eca189935fe8987aae1cf134e43..a29f5f67072b110157221ce7b645f237b1a478ef 100644
--- a/chrome/browser/ui/views/status_bubble_views.h
+++ b/chrome/browser/ui/views/status_bubble_views.h
@@ -73,6 +73,9 @@ class StatusBubbleViews : public StatusBubble {
   void MouseMovedAt(const gfx::Point& location, bool left_content);
 
  private:
+#if defined(BRAVE_CHROMIUM_BUILD)
+  FRIEND_TEST_ALL_PREFIXES(BraveStatusBubbleViewsTest, SetURLTest);
+#endif
   class StatusView;
   class StatusViewAnimation;
   class StatusViewExpander;
