diff --git a/chrome/browser/resources/settings/router.ts b/chrome/browser/resources/settings/router.ts
index 7e03316d8975291611bb095bd3b87c5133f1e33b..6b604189c90f73f0bdffcfcf05edfc732846093e 100644
--- a/chrome/browser/resources/settings/router.ts
+++ b/chrome/browser/resources/settings/router.ts
@@ -118,6 +118,7 @@ export interface SettingsRoutes {
   IMPORT_DATA: Route;
   SIGN_OUT: Route;
   // </if>
+  GET_STARTED: Route; SHIELDS: Route; SOCIAL_BLOCKING: Route; EXTENSIONS: Route; EXTENSIONS_V2: Route; BRAVE_SYNC: Route; BRAVE_SYNC_SETUP: Route; BRAVE_IPFS: Route; BRAVE_WALLET: Route; BRAVE_WEB3: Route; BRAVE_NEW_TAB: Route; THEMES: Route; BRAVE_SITE_SETTINGS_COOKIES_DETAILS: Route;
 }
 
 /** Class for navigable routes. */
