diff --git a/chrome/utility/services.cc b/chrome/utility/services.cc
index a2dd94e64d52d7e52958b52db32ad2844be22e12..37aa0bf5837934b6c73ba8f07c72b0efd4ad7bb7 100644
--- a/chrome/utility/services.cc
+++ b/chrome/utility/services.cc
@@ -369,6 +369,7 @@ void RegisterMainThreadServices(mojo::ServiceFactory& services) {
   services.Add(RunAssistantAudioDecoder);
 #endif
 #endif
+    BRAVE_GET_MAIN_THREAD_SERVICE_FACTORY
 }
 
 void RegisterIOThreadServices(mojo::ServiceFactory& services) {
