diff --git a/chrome/browser/ui/extensions/extension_action_view_controller.h b/chrome/browser/ui/extensions/extension_action_view_controller.h
index 8d9506242b8e8b73ccfaa4140e96831bcc4ae702..25f2da80e7e74370b8dbf9e46083dcc0235b3bb0 100644
--- a/chrome/browser/ui/extensions/extension_action_view_controller.h
+++ b/chrome/browser/ui/extensions/extension_action_view_controller.h
@@ -17,6 +17,7 @@
 #include "extensions/common/extension_id.h"
 #include "ui/gfx/image/image.h"
 
+class BraveActionViewController;
 class Browser;
 class ExtensionActionPlatformDelegate;
 class GURL;
@@ -42,6 +43,7 @@ class ExtensionActionViewController
       public extensions::ExtensionContextMenuModel::PopupDelegate,
       public extensions::ExtensionHostObserver {
  public:
+  friend class BraveActionViewController;
   // The different options for showing a popup.
   enum PopupShowAction { SHOW_POPUP, SHOW_POPUP_AND_INSPECT };
 
@@ -126,6 +128,7 @@ class ExtensionActionViewController
   // In some cases (such as when an action is shown in a menu), a substitute
   // ToolbarActionViewController should be used for showing popups. This
   // returns the preferred controller.
+  virtual
   ExtensionActionViewController* GetPreferredPopupViewController();
 
   // Executes the extension action with |show_action|. If
@@ -141,6 +144,7 @@ class ExtensionActionViewController
   // The popup may not be shown synchronously if the extension is hidden and
   // first needs to slide itself out.
   // Returns true if a popup will be shown.
+  virtual
   bool TriggerPopupWithUrl(PopupShowAction show_action,
                            const GURL& popup_url,
                            bool grant_tab_permissions);
@@ -151,6 +155,7 @@ class ExtensionActionViewController
                  PopupShowAction show_action);
 
   // Handles cleanup after the popup closes.
+  virtual
   void OnPopupClosed();
 
   // Returns the image source for the icon.
