diff --git a/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc b/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc
index 69f6abc8c5ea45d9f422c350f66b4d298dc35f66..e1c63b27a34512ef678f2e3ad00e97fb0eb35cf8 100644
--- a/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc
+++ b/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc
@@ -1209,6 +1209,7 @@ void ChromeWebUIControllerFactory::GetFaviconForURL(
 
 // static
 ChromeWebUIControllerFactory* ChromeWebUIControllerFactory::GetInstance() {
+  BRAVE_CHROME_WEBUI_CONTROLLER_FACTORY
   return base::Singleton<ChromeWebUIControllerFactory>::get();
 }
 
