diff --git a/chrome/installer/mac/signing/pipeline.py b/chrome/installer/mac/signing/pipeline.py
index 7aa84e2e9d94466bef37655f80b203815b96e470..48d4f2856f30076566a973aba1bcf8226ee563a3 100644
--- a/chrome/installer/mac/signing/pipeline.py
+++ b/chrome/installer/mac/signing/pipeline.py
@@ -783,6 +783,9 @@ async def sign_all(
                 ),
                 timeout=60 * 60 * 2,
             )
+        elif 'dest_dir' in locals():
+            # Copy the notarized app to the orig_paths.output dir where the user expects it.
+            commands.copy_files(dest_dir, orig_paths.output)
 
 
 async def _sign_and_maybe_notarize_distributions(
@@ -950,3 +953,5 @@ async def _package_and_maybe_notarize_distributions(
     if config.notarize.should_staple():
         for path in staple_paths:
             notarize.staple(path)
+
+from brave_chromium_utils import inline_file; inline_file("//brave/chromium_src/chrome/installer/mac/signing/pipeline.py", globals(), locals())
