diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 3a0202c2139b3a6d62f10d9ce965e689dc96bf4a..415c4e503392698fc4c5222acf976b459f071daa 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -703,6 +703,7 @@ class Browser : public TabStripModelObserver,
 
   StatusBubble* GetStatusBubbleForTesting();
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
