diff --git a/chrome/browser/ui/views/profiles/incognito_menu_view.h b/chrome/browser/ui/views/profiles/incognito_menu_view.h
index 186ff5fd294293f84efc1e00b7479d2cdd27a6f7..0af329f355dc0e5ae2a5907952c17cf4ee87ec20 100644
--- a/chrome/browser/ui/views/profiles/incognito_menu_view.h
+++ b/chrome/browser/ui/views/profiles/incognito_menu_view.h
@@ -32,6 +32,7 @@ class IncognitoMenuView : public ProfileMenuViewBase {
   // ProfileMenuViewBase:
   void BuildMenu() override;
 
+  BRAVE_INCOGNITO_MENU_VIEW_H_
  private:
   // views::BubbleDialogDelegateView:
   std::u16string GetAccessibleWindowTitle() const override;
