diff --git a/chrome/browser/prefs/BUILD.gn b/chrome/browser/prefs/BUILD.gn
index a81a4864630535df4a7f87b91198021c47a3bd58..16207d98ca82073f2ba4c9ec49d0f43002f995e2 100644
--- a/chrome/browser/prefs/BUILD.gn
+++ b/chrome/browser/prefs/BUILD.gn
@@ -206,6 +206,7 @@ source_set("impl") {
     "//services/preferences/tracked:features",
     "//ui/webui:buildflags",
   ]
+  import("//brave/browser/sources.gni") deps += brave_chrome_browser_prefs_impl_deps
 
   if (enable_print_preview) {
     deps += [
