diff --git a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
index 6e17e7f00b8e4d368826d00ec6e7616adaadd454..82a2cfa979e40fe2bd6d9ace19b4a3f85b6767ba 100644
--- a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
+++ b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
@@ -474,6 +474,7 @@ class AppMenu implements OnKeyListener {
                         anchorView,
                         anchorViewOffset);
 
+        if (this instanceof BraveAppMenu) ((BraveAppMenu) this).updatePopup(popup, isByPermanentButton);
         popup.setHeight(calculateMenuHeight());
 
         int[] popupPosition =
