diff --git a/chrome/browser/profiles/profile_impl.h b/chrome/browser/profiles/profile_impl.h
index db2085d8e0fc9f988caaebc6fca4622301691e0a..36271268f18037c55898fd06cd3bb37c5d52e885 100644
--- a/chrome/browser/profiles/profile_impl.h
+++ b/chrome/browser/profiles/profile_impl.h
@@ -168,6 +168,7 @@ class ProfileImpl : public Profile {
   void OnLogin() override;
   void InitChromeOSPreferences() override;
 #endif  // defined(OS_CHROMEOS)
+  BRAVE_PROFILE_IMPL_H_
 
   void SetCreationTimeForTesting(base::Time creation_time) override;
 
