diff --git a/chrome/browser/ui/views/tabs/vertical/BUILD.gn b/chrome/browser/ui/views/tabs/vertical/BUILD.gn
index ae857db027710cd43506e9f40ae44703aa378ea2..a44145ebb9a126fa65fd7f4252c20174ba5e3688 100644
--- a/chrome/browser/ui/views/tabs/vertical/BUILD.gn
+++ b/chrome/browser/ui/views/tabs/vertical/BUILD.gn
@@ -2,6 +2,8 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//chrome/common/features.gni")
+
 source_set("vertical") {
   sources = [
     "bottom_container_button.h",
@@ -57,9 +59,6 @@ source_set("impl") {
   deps = [
     ":vertical",
     "//chrome/browser:primitives",
-    "//chrome/browser/glic",
-    "//chrome/browser/glic/browser_ui:glow",
-    "//chrome/browser/glic/browser_ui:glow_controller",
     "//chrome/browser/themes",
     "//chrome/browser/ui:browser_element_identifiers",
     "//chrome/browser/ui:layout_constants",
@@ -75,6 +74,14 @@ source_set("impl") {
     "//components/contextual_tasks/public:feature_list",
   ]
 
+  if (enable_glic) {
+    deps += [
+      "//chrome/browser/glic",
+      "//chrome/browser/glic/browser_ui:glow",
+      "//chrome/browser/glic/browser_ui:glow_controller",
+    ]
+  }
+
   public_deps = [
     "//chrome/browser:browser_public_dependencies",
     "//chrome/browser/ui/tabs/alert:tab_alert",
