diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 81876b3c1a1576af78b919b00ad05e96c9a29394..6d333d7d63494b1d38c11aa69509769b3b1a3d2e 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -2865,6 +2865,7 @@ constexpr FeatureEntry::FeatureVariation kLensStandaloneVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
