diff --git a/chrome/browser/resources/bookmarks/command_manager.ts b/chrome/browser/resources/bookmarks/command_manager.ts
index b77e19ddf666c0b8fb1dbe4a3acaeb7bb0b25ef5..9ed0daa1e6b66ea8d8555643b6302a2382cf63b0 100644
--- a/chrome/browser/resources/bookmarks/command_manager.ts
+++ b/chrome/browser/resources/bookmarks/command_manager.ts
@@ -439,7 +439,7 @@ export class BookmarksCommandManagerElement extends
         chrome.bookmarkManagerPrivate.export();
         break;
       case Command.HELP_CENTER:
-        window.open('https://support.google.com/chrome/?p=bookmarks');
+        window.open('https://community.brave.com', undefined, 'noreferrer');
         break;
       default:
         assertNotReached();
