diff --git a/chrome/installer/setup/setup_main.cc b/chrome/installer/setup/setup_main.cc
index bf5b9a610a7cd3ddfce42bf848a997b6ac26c529..6861a4a8d58fb0ab7ab99aae71c3ba1c3d53b6ff 100644
--- a/chrome/installer/setup/setup_main.cc
+++ b/chrome/installer/setup/setup_main.cc
@@ -701,6 +701,7 @@ installer::InstallStatus InstallProducts(InstallationState& original_state,
     }
   }
 
+  BRAVE_UPDATE_INSTALL_STATUS
   return install_status;
 }
 
@@ -1526,6 +1527,7 @@ int SetupMain() {
                                      &exit_code)) {
     return exit_code;
   }
+  BRAVE_HANDLE_NON_INSTALL_CMD_LINE_OPTIONS
 
   if (system_install && !IsUserAnAdmin()) {
     if (!cmd_line.HasSwitch(installer::switches::kRunAsAdmin)) {
