diff --git a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
index c7db9028a77e3960a551635d0aa71c7f39e35cda..846f767f5f41ca72a9b9c3e178a4f02797d2f25c 100644
--- a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
+++ b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 import '/strings.m.js';
+import './brave_overrides/index.js';
 import './commerce/shopping_list.js';
 import './icons.html.js';
 import './power_bookmarks_context_menu.js';
@@ -178,7 +179,8 @@ export class PowerBookmarksListElement extends PolymerElement {
                sortOrder: SortOrder.kReverseAlphabetical,
                label: loadTimeData.getString('sortReverseAlphabetically'),
                lowerLabel: loadTimeData.getString('sortReverseAlphabetically'),
-             }],
+             },
+             { sortOrder: SortOrder.kCustom, label: loadTimeData.getString('sortCustom'), lowerLabel: loadTimeData.getString('sortCustomLower'), }],
       },
 
       editing_: {
