diff --git a/chrome/browser/ui/views/toolbar/browser_app_menu_button.cc b/chrome/browser/ui/views/toolbar/browser_app_menu_button.cc
index 59e399de0f7bdf299fdebfb747d471a7e0abcd34..25d44423224e8e54205c1a2b9e4209308462b4e1 100644
--- a/chrome/browser/ui/views/toolbar/browser_app_menu_button.cc
+++ b/chrome/browser/ui/views/toolbar/browser_app_menu_button.cc
@@ -11,6 +11,7 @@
 #include "base/single_thread_task_runner.h"
 #include "base/threading/thread_task_runner_handle.h"
 #include "base/time/time.h"
+#include "brave/browser/ui/toolbar/brave_app_menu_model.h"
 #include "cc/paint/paint_flags.h"
 #include "chrome/browser/ui/browser.h"
 #include "chrome/browser/ui/browser_otr_state.h"
@@ -246,7 +247,7 @@ void BrowserAppMenuButton::ShowMenu(int run_types) {
       promo_feature_ == InProductHelpFeature::kReopenTab;
 
   RunMenu(
-      std::make_unique<AppMenuModel>(toolbar_view_, browser,
+      std::make_unique<BraveAppMenuModel>(toolbar_view_, browser,
                                      toolbar_view_->app_menu_icon_controller()),
       browser, run_types, alert_reopen_tab_items);
 }
