diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.h b/chrome/browser/ui/views/toolbar/toolbar_view.h
index 41f85bf19810c703c6b00f44783e9568f54e5aa6..9d162ac6e493b220d6c1763138851dbe7db23222 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.h
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.h
@@ -98,12 +98,14 @@ class ToolbarView : public views::AccessiblePaneView,
   ~ToolbarView() override;
 
   // Create the contents of the Browser Toolbar.
+  virtual
   void Init();
 
   // Forces the toolbar (and transitively the location bar) to update its
   // current state.  If |tab| is non-NULL, we're switching (back?) to this tab
   // and should restore any previous location bar state (such as user editing)
   // as well.
+  virtual
   void Update(content::WebContents* tab);
 
   // Updates the visibility of the custom tab bar, potentially animating the
@@ -134,6 +136,7 @@ class ToolbarView : public views::AccessiblePaneView,
       IntentPickerResponse callback);
 
   // Shows a bookmark bubble and anchors it appropriately.
+  virtual
   void ShowBookmarkBubble(const GURL& url,
                           bool already_bookmarked,
                           bookmarks::BookmarkBubbleObserver* observer);
@@ -200,6 +203,7 @@ class ToolbarView : public views::AccessiblePaneView,
   friend class AvatarToolbarButtonBrowserTest;
 
  protected:
+  friend class BraveToolbarView;
   // This controls Toolbar, LocationBar and CustomTabBar visibility.
   // If we don't set all three, tab navigation from the app menu breaks
   // on Chrome OS.
@@ -251,6 +255,7 @@ class ToolbarView : public views::AccessiblePaneView,
   void OnChromeLabsPrefChanged();
 
   // Loads the images for all the child views.
+  virtual
   void LoadImages();
 
   // Shows the critical notification bubble against the app menu.
