diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index f315999073d3e7c1330d3711efd75c6d7cf091cc..32b037307fd3275b42fd02e247e8dffeaa0b290f 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4869,6 +4869,7 @@ constexpr char kWebiumFlag[] = "webium";
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
