diff --git a/chrome/browser/ui/page_action/action_ids.h b/chrome/browser/ui/page_action/action_ids.h
index da4b739fa93eb62273a0beba4413d8413a6ad9f4..adc3c2c4640ce84811b720c430e3f00dd93ecd5e 100644
--- a/chrome/browser/ui/page_action/action_ids.h
+++ b/chrome/browser/ui/page_action/action_ids.h
@@ -16,6 +16,8 @@ namespace page_actions {
 // the array.
 inline constexpr auto kActionIds = std::to_array<actions::ActionId>({
     kActionAiMode,
+    kActionShowPartitionedStorage,
+    kActionShowPsstIcon,
     kActionSidePanelShowLensOverlayResults,
     kActionLensOverlayHomework,
     kActionShowTranslate,
