diff --git a/chrome/browser/prefs/BUILD.gn b/chrome/browser/prefs/BUILD.gn
index 92182484b154f9949a0380148ed4d029b486aa2d..72d7cc44db4552bca7a74eaba6d18e13bbb2bc83 100644
--- a/chrome/browser/prefs/BUILD.gn
+++ b/chrome/browser/prefs/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/sources.gni")
 import("//chrome/common/features.gni")
 import("//components/enterprise/buildflags/buildflags.gni")
 import("//components/on_device_translation/buildflags/features.gni")
@@ -208,6 +209,7 @@ source_set("impl") {
     "//services/preferences/tracked:features",
     "//ui/webui:buildflags",
   ]
+  deps += brave_chrome_browser_prefs_impl_deps
 
   if (enable_print_preview) {
     deps += [
