diff --git a/chrome/browser/content_settings/BUILD.gn b/chrome/browser/content_settings/BUILD.gn
index a50a2a88b0420f9f740e56ac073b4967e50eedaa..3c4c1732d8536cd4a9f6bd5883dd839cc399a129 100644
--- a/chrome/browser/content_settings/BUILD.gn
+++ b/chrome/browser/content_settings/BUILD.gn
@@ -117,6 +117,7 @@ source_set("impl") {
   if (enable_pdf) {
     deps += [ "//chrome/browser/pdf" ]
   }
+  import("//brave/browser/content_settings/sources.gni") deps += brave_browser_content_settings_deps
 }
 
 # Fixes circular deps with different components. For example -
