diff --git a/chrome/installer/mac/signing/parts.py b/chrome/installer/mac/signing/parts.py
index 0a3ba9233e99ef67afa8ea34d937f77032964852..fadbe46485dd4e4a268fba2fa4cec5ce11d46b2a 100644
--- a/chrome/installer/mac/signing/parts.py
+++ b/chrome/installer/mac/signing/parts.py
@@ -10,6 +10,7 @@ import os.path
 
 from signing import commands, signing
 from signing.model import CodeSignOptions, CodeSignedProduct, VerifyOptions
+from signing_helper import BraveModifyPartsForSigning, GenerateBraveWidevineSigFile
 
 _PROVISIONPROFILE_EXT = '.provisionprofile'
 _PROVISIONPROFILE_DEST = 'embedded.provisionprofile'
@@ -202,6 +203,7 @@ def get_parts(config):
             verify_options=verify_options,
         )
 
+    parts = BraveModifyPartsForSigning(parts, config)
     return parts
 
 
@@ -269,6 +271,7 @@ async def sign_chrome(paths, config, sign_framework=False):
                 continue
             signing.sign_part(paths, config, part)
 
+        GenerateBraveWidevineSigFile(paths, config, parts['framework'])
         # Sign the framework bundle.
         signing.sign_part(paths, config, parts['framework'])
 
