diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 880d2fdcd2dde7221ce1823cead67af2d2421f8a..f3ae5b859364b0aa069a1888dbf945a09beae212 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2099,6 +2099,7 @@ jumbo_split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
