diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 52423f54913c0b68c1f168b1b933f51458e067e4..644a18fbbb82542d87a0dfcb399a99c271afaf1e 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -65,6 +65,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
