diff --git a/chrome/browser/resources/settings_shared/people_page/sync_browser_proxy.ts b/chrome/browser/resources/settings_shared/people_page/sync_browser_proxy.ts
index 2ca225f2b4fca916c4ff51e5f9870124ff327437..c4dfe0a50bc6a6d6e73ce5411e36bc54b4b20462 100644
--- a/chrome/browser/resources/settings_shared/people_page/sync_browser_proxy.ts
+++ b/chrome/browser/resources/settings_shared/people_page/sync_browser_proxy.ts
@@ -109,6 +109,9 @@ export interface SyncPrefs {
   bookmarksManaged: boolean;
   bookmarksRegistered: boolean;
   bookmarksSynced: boolean;
+  braveNewsManaged: boolean;
+  braveNewsRegistered: boolean;
+  braveNewsSynced: boolean;
   cookiesManaged: boolean;
   cookiesRegistered: boolean;
   cookiesSynced: boolean;
@@ -192,7 +195,8 @@ export enum UserSelectableType {
   SAVED_TAB_GROUPS = 10,
   PAYMENTS = 11,
   PRODUCT_COMPARISON = 12,
-  COOKIES = 13
+  COOKIES = 13,
+  BRAVE_NEWS = 14,
 }
 // LINT.ThenChange(/components/sync/base/user_selectable_type.h:UserSelectableType)
 
