diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index d38c722bb449b6bd0ad216635ed2a6eebe208eae..b7fa5f52d3b7dcd8572d3be90b02a3145bbc2ae8 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2531,6 +2531,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
