diff --git a/chrome/browser/flags/android/chrome_feature_list.cc b/chrome/browser/flags/android/chrome_feature_list.cc
index 74c5fe0d3b8f51557d9368cdd71720b7ee7c8828..45340c4fac37766161a60ab96c5dd32411eb899c 100644
--- a/chrome/browser/flags/android/chrome_feature_list.cc
+++ b/chrome/browser/flags/android/chrome_feature_list.cc
@@ -387,6 +387,7 @@ const base::Feature* const kFeaturesExposedToJava[] = {
 };
 
 const base::Feature* FindFeatureExposedToJava(const std::string& feature_name) {
+  BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA
   static auto kFeaturesExposedToJavaMap = base::NoDestructor(
       base::MakeFlatMap<base::StringPiece, const base::Feature*>(
           kFeaturesExposedToJava, {},
