diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index a74ccb7c263b0dc8d338f459a2bfe16de755f6cc..4b1105ef07564ac572bbcc0cdf438fc9b14ad8fb 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4057,6 +4057,7 @@ const flags_ui::FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
