diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 3fe9173f5875cf2e3234cc6129043d8cf75a6757..e292162807aaae393ae3f8a442baf8bcbb4a5ba5 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -5082,6 +5082,7 @@ constexpr char kEnterpriseIsolatedModeInternalName[] =
 
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
