diff --git a/chrome/tools/build/win/create_installer_archive.py b/chrome/tools/build/win/create_installer_archive.py
index 3e5dbac1391de7468aa5b1be1f99edff2edf935c..c8a68d2ca9d1daea0ce916ead56dd920bd71a8e7 100755
--- a/chrome/tools/build/win/create_installer_archive.py
+++ b/chrome/tools/build/win/create_installer_archive.py
@@ -774,6 +774,7 @@ def _ParseOptions():
                       action='store_true',
                       dest='verbose',
                       default=False)
+    parser.add_option('--skip_signing', action='store_true', default=False)
 
     options, _ = parser.parse_args()
     if not options.build_dir:
@@ -804,6 +805,7 @@ def _ParseOptions():
 
     return options
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 
 if '__main__' == __name__:
     options = _ParseOptions()
