diff --git a/chrome/installer/util/BUILD.gn b/chrome/installer/util/BUILD.gn
index 3d6e1bc2c78625723854489be12f5a1124124806..362ee4015241ec8ebc3daf0e59b1aa64e530b7e5 100644
--- a/chrome/installer/util/BUILD.gn
+++ b/chrome/installer/util/BUILD.gn
@@ -226,7 +226,20 @@ generate_embedded_i18n("generate_strings") {
 
   output_file_name_base = "installer_util_strings"
 
-  branding = branding_path_component
+  brand = "$branding_path_component"
+  if (brave_chromium_build) {
+    # When brave_strings.grd is modified, outputs should be re-generated.
+    # This is workaround. Need to change the way of using |brave_strings.grd|
+    # in |create_string_rc.py|.
+    sources = [
+      "//chrome/app/brave_strings.grd"
+    ]
+    if (!is_official_build) {
+      # TODO(shong): Remove this.
+      brand = brand + "-development"
+    }
+  }
+  branding = brand
 }
 
 # Compile the generated .rc file.
