diff --git a/chrome/browser/media/webrtc/webrtc_event_log_uploader.cc b/chrome/browser/media/webrtc/webrtc_event_log_uploader.cc
index 9c0a71246287ac8f689808ecf7f932c0270bb015..8715f803e9efd9a193d125d4ff22fbb8cee73e2a 100644
--- a/chrome/browser/media/webrtc/webrtc_event_log_uploader.cc
+++ b/chrome/browser/media/webrtc/webrtc_event_log_uploader.cc
@@ -263,6 +263,7 @@ bool WebRtcEventLogUploaderImpl::PrepareUploadData(std::string* upload_data) {
 }
 
 void WebRtcEventLogUploaderImpl::StartUpload(const std::string& upload_data) {
+  return; // feature disabled in Brave
   DCHECK(task_runner_->RunsTasksInCurrentSequence());
 
   auto resource_request = std::make_unique<network::ResourceRequest>();
