diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index b1c6b05cb71b6c97215c3659775d8996ad2293c0..771943c8043b91c327b6f8b96c7e20aeb509a779 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -286,6 +286,8 @@ source_set("configs") {
   if (enable_webui_ntp) {
     deps += [ "//chrome/browser/ui/webui/ntp_microsoft_auth" ]
   }
+  import("//brave/browser/ui/webui/sources.gni")
+  deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("webui") {
