diff --git a/chrome/browser/android/tab_features.cc b/chrome/browser/android/tab_features.cc
index f969c7758037c6b45849a13f760b87e411d84061..87297ab11a010b506ec7cdb9948c6d7288cb57e3 100644
--- a/chrome/browser/android/tab_features.cc
+++ b/chrome/browser/android/tab_features.cc
@@ -442,10 +442,10 @@ TabFeatures::TabFeatures(content::WebContents* web_contents, Profile* profile) {
       GetUserDataFactory().CreateInstance<SearchEngineTabHelper>(*tab, *tab,
                                                                  web_contents);
 
-  net_error_tab_helper_ =
+  /* net_error_tab_helper_ =
       GetUserDataFactory()
           .CreateInstance<chrome_browser_net::NetErrorTabHelper>(*tab, *tab,
-                                                                 web_contents);
+                                                                 web_contents); */
 
   if (!profile->IsOffTheRecord()) {
     supervised_user_navigation_observer_ =
