diff --git a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
index 46cb7743562ebcfedc2975d86506bbae4bf4229a..eb8d14025d58dda54986b537cb9e10489eb55d04 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
@@ -25,7 +25,7 @@ import org.chromium.chrome.browser.ui.appmenu.AppMenuHandler;
 /**
  * An {@link AppMenuPropertiesDelegateImpl} for ChromeTabbedActivity.
  */
-public class TabbedAppMenuPropertiesDelegate extends AppMenuPropertiesDelegateImpl {
+public class TabbedAppMenuPropertiesDelegate extends org.chromium.chrome.browser.appmenu.BraveAppMenuPropertiesDelegateImpl {
     AppMenuDelegate mAppMenuDelegate;
 
     public TabbedAppMenuPropertiesDelegate(Context context, ActivityTabProvider activityTabProvider,
@@ -63,6 +63,7 @@ public class TabbedAppMenuPropertiesDelegate extends AppMenuPropertiesDelegateIm
                     .initialize(appMenuHandler, mBookmarkBridge, mActivityTabProvider.get(),
                             mAppMenuDelegate);
         }
+        super.onFooterViewInflated(appMenuHandler, view);
     }
 
     @Override
