diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index f0fee1d39ff72e5f0ba19e28cd59b4bf88c08c70..718aad02f2080ee0c7b0f6fc718461f0c7652df0 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2484,6 +2484,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps defines += brave_chrome_browser_defines
 
   if (is_android) {
     sources += [
