diff --git a/chrome/browser/ui/unload_controller.cc b/chrome/browser/ui/unload_controller.cc
index 7386d1921380ca5cb5cdd93ab7a1338a086f1b04..8dd126b7887a0cbf02aa0ee1bcc66f6426cfdc76 100644
--- a/chrome/browser/ui/unload_controller.cc
+++ b/chrome/browser/ui/unload_controller.cc
@@ -206,6 +206,9 @@ void UnloadController::OnWindowClosing() {
   if (tab_restore_service && notify_restore_service) {
     tab_restore_service->BrowserClosing(BrowserLiveTabContext::From(browser_));
   }
+  if (MaybeHandleSharedPinnedTabsClosing(browser_)) {
+    return;
+  }
 
   if (!browser_->GetTabStripModel()->empty()) {
     // Closing all the tabs results in eventually calling back to
@@ -506,6 +509,7 @@ UnloadController::GetBrowserClosingStatus() const {
 bool UnloadController::RunBeforeUnloadForShutdown(
     bool skip_beforeunload,
     const base::RepeatingCallback<void(bool)>& on_close_confirmed) {
+  confirmed_to_close_ = true;
   cancel_download_confirmation_state_ =
       CancelDownloadConfirmationState::kResponseReceived;
   // The devtools browser gets its beforeunload events as the results of
@@ -529,6 +533,7 @@ bool UnloadController::RunBeforeUnloadForShutdown(
 }
 
 void UnloadController::ResetBeforeUnloadForShutdown() {
+  confirmed_to_close_ = false;
   cancel_download_confirmation_state_ =
       CancelDownloadConfirmationState::kNotPrompted;
   if (state_ != State::kRunningBeforeUnloadForShutdown &&
@@ -635,6 +640,9 @@ void UnloadController::OnTabStripModelChanged(
 }
 
 void UnloadController::TabStripEmpty() {
+  if (!browser_->GetProfile()->GetPrefs()->GetBoolean(kEnableClosingLastTab)) {
+    return;
+  }
   // Set state_ here, so that extensions, etc, do not attempt to add tabs to the
   // browser before it closes.
   if (state_ == State::kIdle) {
@@ -1085,6 +1093,9 @@ void UnloadController::InProgressDownloadResponse(bool cancel_downloads) {
 }
 
 void UnloadController::FinishWarnBeforeClosing(WarnBeforeClosingResult result) {
+  if (result == WarnBeforeClosingResult::kDoNotClose) {
+    confirmed_to_close_ = false;
+  }
   switch (result) {
     case WarnBeforeClosingResult::kOkToClose:
       chrome::CloseWindow(browser_);
