diff --git a/chrome/common/extensions/api/developer_private.idl b/chrome/common/extensions/api/developer_private.idl
index 452deaed1276eb6d63d5911fa0534d56ee318bf1..54788768845f79b2af802ed5fac7e8310d608049 100644
--- a/chrome/common/extensions/api/developer_private.idl
+++ b/chrome/common/extensions/api/developer_private.idl
@@ -231,6 +231,7 @@ namespace developerPrivate {
     DOMString iconUrl;
     DOMString id;
     AccessModifier incognitoAccess;
+    boolean isSplitMode;
     DOMString[] installWarnings;
     DOMString? launchUrl;
     Location location;
