diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index cdf4406a57c01010f065948d04f9c7a2219db82f..703af55d0a81da6195962f141a9beb5d6747b51f 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -44,6 +44,7 @@ if (optimize_webui) {
       "chrome://resources/mojo/mojo/public/js/bindings.js",
       "chrome://resources/mojo/skia/public/mojom/skcolor.mojom-webui.js",
     ]
+    deps += brave_deps_chrome_browser_resources_settings_build
   }
 }
 
@@ -91,6 +92,7 @@ generate_grd("build_grd") {
       "$target_gen_dir/$preprocess_gen_manifest",
     ]
   }
+  deps += [ "//brave/browser/resources/settings:resources" ]
 }
 
 preprocess_if_expr("preprocess") {
