diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 28236a6d24aad9cd4e53c08ab834133cb29639ab..775526f174060fe837d21fb6e7d63afb3a34e920 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2520,6 +2520,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
