diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/autofill/AndroidPaymentAppsFragment.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/autofill/AndroidPaymentAppsFragment.java
index b0589b4dbf86e0c4ac81937283344ed24d41b65f..bec10b63647fe97922f1d6e059afadf7e4474a31 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/autofill/AndroidPaymentAppsFragment.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/autofill/AndroidPaymentAppsFragment.java
@@ -26,7 +26,7 @@ import java.util.Map;
 /**
  * Preference fragment to allow users to control use of the Android payment apps on device.
  */
-public class AndroidPaymentAppsFragment extends PreferenceFragmentCompat {
+public class AndroidPaymentAppsFragment extends org.chromium.chrome.browser.settings.BravePreferenceFragment {
     @Override
     public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
         getActivity().setTitle(R.string.payment_apps_title);
