diff --git a/chrome/browser/ui/extensions/extension_action_view_controller.cc b/chrome/browser/ui/extensions/extension_action_view_controller.cc
index d0f640c2d1647a94d92e3e5e763714dd2edcc4e6..1ac0b33daee8542d8bc5c07b287b3e2f8424adcc 100644
--- a/chrome/browser/ui/extensions/extension_action_view_controller.cc
+++ b/chrome/browser/ui/extensions/extension_action_view_controller.cc
@@ -51,7 +51,6 @@ ExtensionActionViewController::Create(
     ExtensionsContainer* extensions_container,
     bool in_overflow_mode) {
   DCHECK(browser);
-  DCHECK(extensions_container);
 
   auto* registry = extensions::ExtensionRegistry::Get(browser->profile());
   scoped_refptr<const extensions::Extension> extension =
