diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index f44d02b458d142750c39867e59951607c3a57ed5..863bad749881fdc6597db3c879bc24fe026ce64a 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4685,6 +4685,7 @@ const FeatureEntry::FeatureVariation kWebAuthnAmbientSigninVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
