diff --git a/chrome/browser/permissions/permission_manager.cc b/chrome/browser/permissions/permission_manager.cc
index 6673d1591ab95d937a0c208d3ce4fce935329072..da3727e240f130c92304dee854d3522ff4e202f3 100644
--- a/chrome/browser/permissions/permission_manager.cc
+++ b/chrome/browser/permissions/permission_manager.cc
@@ -106,7 +106,7 @@ ContentSetting PermissionStatusToContentSetting(PermissionStatus status) {
 // Helper method to convert PermissionType to ContentSettingType.
 // If PermissionType is not supported or found, returns
 // CONTENT_SETTINGS_TYPE_DEFAULT.
-ContentSettingsType PermissionTypeToContentSettingSafe(
+ContentSettingsType PermissionTypeToContentSettingSafe_ChromiumImpl(
     PermissionType permission) {
   switch (permission) {
     case PermissionType::MIDI:
@@ -154,6 +154,7 @@ ContentSettingsType PermissionTypeToContentSettingSafe(
     case PermissionType::WAKE_LOCK_SYSTEM:
       return CONTENT_SETTINGS_TYPE_WAKE_LOCK_SYSTEM;
     case PermissionType::NUM:
+    default:
       break;
   }
 
