diff --git a/chrome/browser/ui/color/BUILD.gn b/chrome/browser/ui/color/BUILD.gn
index 0943df4390a8fad2acb988085863eb7af1dfb8a1..d552479c637d51e7390f6d0c9b962a69a259e5a2 100644
--- a/chrome/browser/ui/color/BUILD.gn
+++ b/chrome/browser/ui/color/BUILD.gn
@@ -63,6 +63,7 @@ source_set("mixers") {
     "//ui/color",
     "//ui/color:mixers",
   ]
+  sources += brave_browser_ui_color_mixers_sources deps += brave_browser_ui_color_mixers_deps
 
   if (is_chromeos) {
     sources += [ "chromeos/native_chrome_color_mixer_chromeos.cc" ]
