diff --git a/chrome/browser/ui/startup/BUILD.gn b/chrome/browser/ui/startup/BUILD.gn
index 0207aaf47b56534ebd297bddf2fd89c4da9bd3a6..c9dd5b97eaf3814280d2b0fe8345efc313e2bb77 100644
--- a/chrome/browser/ui/startup/BUILD.gn
+++ b/chrome/browser/ui/startup/BUILD.gn
@@ -28,6 +28,7 @@ source_set("startup_tab") {
   ]
 
   public_deps = [ "//url" ]
+  import("//brave/browser/ui/startup/sources.gni") public_deps += brave_browser_ui_startup_public_deps
 }
 
 source_set("startup") {
