diff --git a/chrome/browser/ui/views/location_bar/location_icon_view.cc b/chrome/browser/ui/views/location_bar/location_icon_view.cc
index 4031fa7f577653caac82fd64783b58db791c84d1..293d163319f2fe7872d7a1c57dbbe6dcae1749ef 100644
--- a/chrome/browser/ui/views/location_bar/location_icon_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_icon_view.cc
@@ -340,6 +340,7 @@ void LocationIconView::UpdateBackground() {
 
   SetBackgroundColor(GetColorProvider()->GetColor(id));
 
+  BRAVE_LOCATION_ICON_VIEW_UPDATE_BACKGROUND
   if (is_text_dangerous) {
     ConfigureInkDropForRefresh2023(this, kColorOmniboxSecurityChipInkDropHover,
                                    kColorOmniboxSecurityChipInkDropRipple);
