diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 8b95111f3ea3d4ea2d3c7bbd4a9a5d7250456d2d..be5e21659aad59f26f2b6bd7fbb85a167c613c79 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -705,6 +705,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome/browser/resources/media/mei_preload:component",
