diff --git a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
index 15d3e273a9f2d66f944976dbc04ff003536455a4..f3253ecf9baf74b0cd6d3132712421953d8a931a 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
@@ -206,7 +206,7 @@ import org.chromium.chrome.browser.tab_bottom_sheet.TabBottomSheetManagerImpl;
 import org.chromium.chrome.browser.tab_bottom_sheet.TabBottomSheetUtils;
 import org.chromium.chrome.browser.tab_group_suggestion.toolbar.GroupSuggestionsButtonController;
 import org.chromium.chrome.browser.tab_group_suggestion.toolbar.GroupSuggestionsButtonControllerFactory;
-import org.chromium.chrome.browser.tab_group_sync.TabGroupSyncControllerImpl;
+import org.chromium.chrome.browser.tab_group_sync.BraveTabGroupSyncControllerImpl;
 import org.chromium.chrome.browser.tab_group_sync.TabGroupSyncFeatures;
 import org.chromium.chrome.browser.tab_group_sync.TabGroupSyncServiceFactory;
 import org.chromium.chrome.browser.tab_ui.TabContentManager;
@@ -1557,7 +1557,7 @@ public class TabbedRootUiCoordinator extends RootUiCoordinator {
             var tabGroupSyncService = TabGroupSyncServiceFactory.getForProfile(originalProfile);
             assert tabGroupSyncService != null;
             mTabGroupSyncController =
-                    new TabGroupSyncControllerImpl(
+                    new BraveTabGroupSyncControllerImpl(
                             mTabModelSelectorSupplier.asNonNull().get(),
                             tabGroupSyncService,
                             UserPrefs.get(originalProfile),
