diff --git a/chrome/browser/resources/settings/privacy_page/personalization_options.ts b/chrome/browser/resources/settings/privacy_page/personalization_options.ts
index 21b6dfa46620e46845f350f1b9c87e67946b87b8..f83a183371fc622931e5148ab32786f7e2c967cf 100644
--- a/chrome/browser/resources/settings/privacy_page/personalization_options.ts
+++ b/chrome/browser/resources/settings/privacy_page/personalization_options.ts
@@ -85,7 +85,7 @@ export class SettingsPersonalizationOptionsElement extends
     return {
       syncStatus: Object,
 
-      // <if expr="_google_chrome and not is_chromeos">
+      // <if expr="not is_chromeos">
       // TODO(dbeam): make a virtual.* pref namespace and set/get this normally
       // (but handled differently in C++).
       metricsReportingPref_: {
@@ -159,7 +159,7 @@ export class SettingsPersonalizationOptionsElement extends
 
   declare syncStatus: SyncStatus;
 
-  // <if expr="_google_chrome and not is_chromeos">
+  // <if expr="not is_chromeos">
   declare private metricsReportingPref_:
       chrome.settingsPrivate.PrefObject<boolean>;
   declare private showRestart_: boolean;
@@ -215,7 +215,7 @@ export class SettingsPersonalizationOptionsElement extends
   override ready() {
     super.ready();
 
-    // <if expr="_google_chrome and not is_chromeos">
+    // <if expr="not is_chromeos">
     const setMetricsReportingPref = (metricsReporting: MetricsReporting) =>
         this.setMetricsReportingPref_(metricsReporting);
     this.addWebUiListener('metrics-reporting-change', setMetricsReportingPref);
@@ -230,9 +230,11 @@ export class SettingsPersonalizationOptionsElement extends
         this.setChromeSigninUserChoiceInfo_.bind(this));
     // </if>
 
+    // <if expr="_google_chrome">
     this.registerHelpBubble(
         ANONYMIZED_URL_COLLECTION_ID,
         this.$.urlCollectionToggle.getBubbleAnchor(), {paddingTop: 10});
+    // </if>
   }
 
   // <if expr="is_chromeos">
@@ -253,7 +255,7 @@ export class SettingsPersonalizationOptionsElement extends
   }
   // </if>
 
-  // <if expr="_google_chrome and not is_chromeos">
+  // <if expr="not is_chromeos">
   private onMetricsReportingChange_() {
     const control = this.shadowRoot!.querySelector<SettingsToggleButtonElement>(
         '#metricsReportingControl');
