diff --git a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
index 923c754ff0d35eb954d889b34566612392fe09ba..f04586a3087f0c46fa405acde6d3233f1360717d 100644
--- a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
+++ b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
@@ -345,7 +345,7 @@ class AppMenu implements OnKeyListener {
                         headerHeight,
                         anchorView,
                         anchorViewOffset);
-
+        if (this instanceof BraveAppMenu) ((BraveAppMenu) this).updatePopup(mPopup, isByPermanentButton);
         mPopup.setHeight(calculateMenuHeight());
 
         int[] popupPosition =
