diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 324d714746db7ec6c496776414deada4f845bee6..11cb12ef23443369f081ab6a2fc9d2bcd63bbb61 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -136,6 +136,7 @@ source_set("configs") {
   if (is_mac) {
     deps += [ "//chrome/browser/ui/webui/unexportable_keys_internals" ]
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("webui") {
