diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java
index 56e0504dbe8d95dac6299baa1b3be030925b96a8..6f585927bb46ea5aeffe2216b5daa8f6175373e6 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarCoordinator.java
@@ -155,4 +155,5 @@ class BottomToolbarCoordinator {
         }
         mThemeColorProvider.destroy();
     }
+    public void updateBookmarkButton(boolean isBookmarked, boolean editingAllowed) {if (mBrowsingModeCoordinator != null) {mBrowsingModeCoordinator.updateBookmarkButton(isBookmarked, editingAllowed);}}
 }
