diff --git a/chrome/browser/ui/webui/settings/site_settings_helper.cc b/chrome/browser/ui/webui/settings/site_settings_helper.cc
index 42d603bb9dc9accf799f9409b8b4e35743e22db3..1d03fd8e6a754d8a908a00ba083835cc95b94882 100644
--- a/chrome/browser/ui/webui/settings/site_settings_helper.cc
+++ b/chrome/browser/ui/webui/settings/site_settings_helper.cc
@@ -261,6 +261,7 @@ constexpr auto kContentSettingsTypeGroupNames = std::to_array<
     {ContentSettingsType::PERMISSION_ACTIONS_HISTORY, nullptr},
     {ContentSettingsType::SUSPICIOUS_NOTIFICATION_SHOW_ORIGINAL, nullptr},
     {ContentSettingsType::DEPRECATED_SUB_APP_INSTALLATION_PROMPTS, nullptr},
+    BRAVE_CONTENT_SETTINGS_TYPE_GROUP_NAMES_LIST
 });
 
 static_assert(
@@ -535,6 +536,7 @@ bool HasRegisteredGroupName(ContentSettingsType type) {
 }
 
 ContentSettingsType ContentSettingsTypeFromGroupName(std::string_view name) {
+  BRAVE_SITE_SETTINGS_HELPER_CONTENT_SETTINGS_TYPE_FROM_GROUP_NAME
   for (const auto& entry : kContentSettingsTypeGroupNames) {
     // Content setting types that aren't represented in the settings UI
     // will have `nullptr` as their `name`. However, converting `nullptr`
@@ -549,6 +551,7 @@ ContentSettingsType ContentSettingsTypeFromGroupName(std::string_view name) {
 }
 
 std::string_view ContentSettingsTypeToGroupName(ContentSettingsType type) {
+  BRAVE_SITE_SETTINGS_HELPER_CONTENT_SETTINGS_TYPE_TO_GROUP_NAME
   for (const auto& entry : kContentSettingsTypeGroupNames) {
     if (type == entry.type) {
       // Content setting types that aren't represented in the settings UI
@@ -723,6 +726,7 @@ std::string SiteSettingSourceToString(const SiteSettingSource source) {
 SiteSettingSource ProviderTypeToSiteSettingsSource(
     const ProviderType provider_type) {
   switch (provider_type) {
+    BRAVE_PROVIDER_TYPE_TO_SITE_SETTINGS_SOURCE
     case ProviderType::kWebuiAllowlistProvider:
     case ProviderType::kComponentExtensionProvider:
       return SiteSettingSource::kAllowlist;
@@ -752,6 +756,7 @@ SiteSettingSource ProviderTypeToSiteSettingsSource(
 std::string_view ProviderToDefaultSettingSourceString(
     const ProviderType provider) {
   switch (provider) {
+    BRAVE_PROVIDER_TO_DEFAULT_SETTINGS_STRING
     case ProviderType::kPolicyProvider:
       return "policy";
     case ProviderType::kSupervisedProvider:
@@ -877,6 +882,7 @@ base::DictValue GetExceptionForPage(
   exception.Set(kSource, SiteSettingSourceToString(source));
   exception.Set(kIncognito, incognito);
   exception.Set(kIsEmbargoed, is_embargoed);
+  BRAVE_GET_EXCEPTION_FOR_PAGE
   return exception;
 }
 
