diff --git a/chrome/browser/ui/views/outdated_upgrade_bubble_view.cc b/chrome/browser/ui/views/outdated_upgrade_bubble_view.cc
index 0716ab28c05a24f3bc2aefc7774ee66e7321c4c2..4a98a5d023ec5035c6cf57984e0bf76c19434d28 100644
--- a/chrome/browser/ui/views/outdated_upgrade_bubble_view.cc
+++ b/chrome/browser/ui/views/outdated_upgrade_bubble_view.cc
@@ -37,8 +37,12 @@ namespace {
 
 // The URL to be used to re-install Chrome when auto-update failed for too long.
 constexpr char kDownloadChromeUrl[] =
+#if defined(BRAVE_CHROMIUM_BUILD)
+    kDownloadBraveUrl;
+#else
     "https://www.google.com/chrome/?&brand=CHWL"
     "&utm_campaign=en&utm_source=en-et-na-us-chrome-bubble&utm_medium=et";
+#endif
 
 // The maximum number of ignored bubble we track in the NumLaterPerReinstall
 // histogram.
