diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index c8f93c048c8c881fa299f889b9b403ae1ef2addb..69b49ec25dd770c7b1b9529e33145b6e8b0b45e0 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -262,7 +262,7 @@ const char Profile::kProfileKey[] = "__PROFILE__";
 void Profile::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
       prefs::kSearchSuggestEnabled,
-      true,
+      false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 #if defined(OS_ANDROID)
   registry->RegisterStringPref(
