diff --git a/chrome/browser/download/download_item_model.h b/chrome/browser/download/download_item_model.h
index ae5068446a02a63e43ad7b18802d82f435a204a8..f6c0cb455bf6a9035d93606dcec94c69a91f5c4f 100644
--- a/chrome/browser/download/download_item_model.h
+++ b/chrome/browser/download/download_item_model.h
@@ -151,6 +151,7 @@ class DownloadItemModel : public DownloadUIModel,
   void OnDownloadDestroyed(download::DownloadItem* download) override;
 
  private:
+  friend class BraveDownloadItemModel;
   // DownloadUIModel implementation.
   std::string GetMimeType() const override;
 
