diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/about/LegalInformationSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/about/LegalInformationSettings.java
index aea41fcae53c5d615124b987e21014447ad6151f..1af120ec642d435a34229c0a510d123710aec0b2 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/about/LegalInformationSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/about/LegalInformationSettings.java
@@ -13,7 +13,7 @@ import org.chromium.chrome.browser.settings.SettingsUtils;
 /**
  * Fragment to display legal information about Chrome.
  */
-public class LegalInformationSettings extends PreferenceFragmentCompat {
+public class LegalInformationSettings extends org.chromium.chrome.browser.settings.BravePreferenceFragment {
     @Override
     public void onCreatePreferences(Bundle savedInstanceState, String s) {
         SettingsUtils.addPreferencesFromResource(this, R.xml.legal_information_preferences);
