diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ShortcutHelper.java b/chrome/android/java/src/org/chromium/chrome/browser/ShortcutHelper.java
index a52b4aed2a71a6dc95e328cc21a73a0cea94bfbe..35e454f1edb8e538676ab271f32acb8344c3b59c 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ShortcutHelper.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ShortcutHelper.java
@@ -112,7 +112,7 @@ public class ShortcutHelper {
                 // Encoding {@link icon} as a string and computing the mac are expensive.
 
                 // Encode the icon as a base64 string (Launcher drops Bitmaps in the Intent).
-                String encodedIcon = BitmapHelper.encodeBitmapAsString(icon);
+                String encodedIcon = BravePwaShortcutIconHelper.encodeDownscaledBitmapAsString(icon);
 
                 // TODO(http://crbug.com/40097069): Use action which does not require mac on O+
                 Intent shortcutIntent =
