diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index a6b64c8dc226b46451cfcf3542f6b441a174c3b8..a8ec4d41181d64d0019137a7d20e5ef2e6b7141a 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4601,6 +4601,7 @@ const FeatureEntry::FeatureVariation kGestureUserEducationVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
