diff --git a/chrome/browser/ui/tabs/tab_style.cc b/chrome/browser/ui/tabs/tab_style.cc
index 7218fd7817da37188569f1a6459c6a2b03395128..c697d44065522cebd02b9e3b8d676d0d3598aa95 100644
--- a/chrome/browser/ui/tabs/tab_style.cc
+++ b/chrome/browser/ui/tabs/tab_style.cc
@@ -189,6 +189,7 @@ float ChromeRefresh2023TabStyle::GetSelectedTabOpacity() const {
 
 // static
 const TabStyle* TabStyle::Get() {
+  BRAVE_TAB_STYLE_GET
   static TabStyle* const tab_style =
       static_cast<TabStyle*>(new ChromeRefresh2023TabStyle());
 
