diff --git a/chrome/browser/ui/android/toolbar/BUILD.gn b/chrome/browser/ui/android/toolbar/BUILD.gn
index 6943cacdf1d16546cc51907d692fe0e77f637652..fbe3422a1df3143de46c511c66f8f7324ebf0847 100644
--- a/chrome/browser/ui/android/toolbar/BUILD.gn
+++ b/chrome/browser/ui/android/toolbar/BUILD.gn
@@ -13,6 +13,7 @@ android_library("variation_utils_java") {
     "java/src/org/chromium/chrome/browser/toolbar/ToolbarVariationUtils.java",
   ]
   deps = [
+    "//brave/browser/ui/android/bottombar:prefs_java",
     "//chrome/browser/flags:java",
     "//chrome/browser/ui/android/bottombar:config_java",
   ]
@@ -232,6 +233,7 @@ android_library("core_java") {
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/HeightTransitionHandler.java",
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/TabStripTransitionCoordinator.java",
   ]
+  sources += brave_browser_ui_android_toolbar_sources
 
   deps = [
     ":adaptive_features_java",
@@ -455,6 +457,7 @@ android_resources("java_resources") {
   ]
 
   deps = [
+    "//brave/browser/ui/android/toolbar:java_resources",
     "//chrome/app:java_strings_grd",
     "//chrome/browser/ui/android/bars_common:java_resources",
     "//chrome/browser/ui/android/strings:ui_strings_grd",
