diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index ea090b62bd84b029c2e21895272963cc8c8e5656..d5b15390c34147742a2bcafaab8ccb26df2ef862 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -174,6 +174,7 @@ class AvatarToolbarButton : public ToolbarButton,
 #endif  // BUILDFLAG(ENABLE_DICE_SUPPORT)
 
  private:
+  BRAVE_AVATAR_BUTTON_H
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
