diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 7e2f40434713d95ec469f75aab413bef926004c0..247fbccabe77593de52663050e6b365bc0d9e918 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3438,6 +3438,7 @@ const FeatureEntry::FeatureVariation kVcSegmentationModelVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
