diff --git a/chrome/browser/resources/tools/rollup_plugin.js b/chrome/browser/resources/tools/rollup_plugin.js
index b541233a644bc428257dafcea7198d2d4a529d98..356397cbf7b9aa3371458c4cd6331877e1823e9c 100644
--- a/chrome/browser/resources/tools/rollup_plugin.js
+++ b/chrome/browser/resources/tools/rollup_plugin.js
@@ -71,6 +71,8 @@ export default function plugin(srcPath, genPath, rootPath, host, excludes) {
         return joinPaths(polymerSrcPath, pathFromPolymer);
       }
 
+      const resolvedByBrave = require(path.join(genPath, 'chrome', 'browser', 'resources', 'brave_rollup_overrides')).braveResolveId({srcPath, genPath, excludes}, source, origin, relativePath, joinPaths, combinePaths, chromeResourcesUrl, schemeRelativeResourcesUrl); if (resolvedByBrave) { return resolvedByBrave; }
+
       // Get path from ui/webui/resources
       let pathFromResources = '';
       if (source.startsWith(chromeResourcesUrl)) {
