diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 99b370a07c1df457e5b2ee3386f4773457eea021..2a69664cf97ea838fb2239206c5d03a899ab3537 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2490,6 +2490,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
