diff --git a/chrome/browser/ui/views/profiles/profile_menu_view_base.h b/chrome/browser/ui/views/profiles/profile_menu_view_base.h
index a70cd56cf2dc17fc30058c17d1912161b77e177f..7e63bacc36132ff971ca5a0940e0ea5141b61e9a 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view_base.h
+++ b/chrome/browser/ui/views/profiles/profile_menu_view_base.h
@@ -168,6 +168,7 @@ class ProfileMenuViewBase : public content::WebContentsDelegate,
   void set_perform_menu_actions_for_testing(bool perform_menu_actions) {
     perform_menu_actions_ = perform_menu_actions;
   }
+  BRAVE_PROFILE_MENU_VIEW_BASE_H_
 
  private:
   class AXMenuWidgetObserver;
