diff --git a/chrome/install_static/BUILD.gn b/chrome/install_static/BUILD.gn
index bebbab34ae8f822c33350214c8af8ea41690e3a0..7c53c463c09d3d45944fb63a5100603fce92ee67 100644
--- a/chrome/install_static/BUILD.gn
+++ b/chrome/install_static/BUILD.gn
@@ -45,7 +45,7 @@ static_library("install_static_util") {
     ]
   } else {
     sources += [
-      "chromium_install_modes.cc",
+      "//brave/chromium_src/chrome/install_static/chromium_install_modes.cc",
       "chromium_install_modes.h",
     ]
   }
