diff --git a/chrome/installer/setup/setup_util.h b/chrome/installer/setup/setup_util.h
index 2f16e5d079208b52942441ceea9ec4814f26767c..9e02196b40da85757a46477a7be493939dae89af 100644
--- a/chrome/installer/setup/setup_util.h
+++ b/chrome/installer/setup/setup_util.h
@@ -31,6 +31,7 @@ class Version;
 
 namespace installer {
 
+class InstallationState;
 class InstallerState;
 class InitialPreferences;
 
@@ -49,6 +50,13 @@ enum UnPackConsumer {
 // Returns the maximum version found or nullptr if no version is found.
 base::Version* GetMaxVersionFromArchiveDir(const base::FilePath& chrome_path);
 
+// Returns the uncompressed archive of the installed version that serves as the
+// source for patching.  If |desired_version| is valid, only the path to that
+// version will be returned, or empty if it doesn't exist.
+base::FilePath FindArchiveToPatch(const InstallationState& original_state,
+                                  const InstallerState& installer_state,
+                                  const base::Version& desired_version);
+
 // Spawns a new process that waits for a specified amount of time before
 // attempting to delete |path|.  This is useful for setup to delete the
 // currently running executable or a file that we cannot close right away but
