diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index de135acf580ddf090faffd95accab904d9882c42..918b4879ab49fba46fffa968ada116833080bcad 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -51,6 +51,7 @@ if (optimize_webui) {
       "chrome://resources/mojo/mojo/public/js/bindings.js",
       "chrome://resources/mojo/skia/public/mojom/skcolor.mojom-webui.js",
     ]
+    deps += brave_deps_chrome_browser_resources_settings_build
   }
 }
 
@@ -100,6 +101,7 @@ generate_grd("build_grd") {
       "$target_gen_dir/$preprocess_gen_manifest",
     ]
   }
+  deps += [ "//brave/browser/resources/settings:resources" ]
 }
 
 preprocess_if_expr("preprocess") {
