diff --git a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java
index 465debb3039f3f6a13623e36317664fc923eccdd..2a7341071aeeeabf30f4bf73a09fee623e3967ff 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java
@@ -77,7 +77,7 @@ import java.util.concurrent.TimeUnit;
 import java.util.function.Consumer;
 
 /** Fragment to keep track of the all the privacy related preferences. */
-public class PrivacySettings extends ChromeBaseSettingsFragment
+public class PrivacySettings extends org.chromium.chrome.browser.settings.BravePreferenceFragment
         implements Preference.OnPreferenceChangeListener {
     private static final String PREF_CAN_MAKE_PAYMENT = "can_make_payment";
     private static final String PREF_PRELOAD_PAGES = "preload_pages";
@@ -558,6 +558,7 @@ public class PrivacySettings extends ChromeBaseSettingsFragment
     @Override
     public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         menu.clear();
+        super.onCreateOptionsMenu(menu, inflater);
         MenuItem help =
                 menu.add(Menu.NONE, R.id.menu_id_targeted_help, Menu.NONE, R.string.menu_help);
         help.setIcon(
@@ -572,7 +573,7 @@ public class PrivacySettings extends ChromeBaseSettingsFragment
                     .show(getActivity(), getString(R.string.help_context_privacy), null);
             return true;
         }
-        return false;
+        return super.onOptionsItemSelected(item);
     }
 
     @Override
