diff --git a/chrome/browser/themes/theme_helper.h b/chrome/browser/themes/theme_helper.h
index 62e6873c50f02d7ba9297a8714a8129dac910a32..bdd83661c6a1374aafc2eee820fea927b9393901 100644
--- a/chrome/browser/themes/theme_helper.h
+++ b/chrome/browser/themes/theme_helper.h
@@ -107,6 +107,7 @@ class ThemeHelper {
       bool incognito,
       const CustomThemeSupplier* theme_supplier) const;
 
+  BRAVE_THEME_HELPER_H_
  private:
   friend class theme_service_internal::ThemeServiceTest;
 
