diff --git a/chrome/utility/importer/external_process_importer_bridge.h b/chrome/utility/importer/external_process_importer_bridge.h
index d5255e050ed703a78b4bdacc6c872d3b35e93665..501d4c59ac109e750a987b8635c3cfbe0ca72d16 100644
--- a/chrome/utility/importer/external_process_importer_bridge.h
+++ b/chrome/utility/importer/external_process_importer_bridge.h
@@ -72,6 +72,7 @@ class ExternalProcessImporterBridge : public ImporterBridge {
   std::u16string GetLocalizedString(int message_id) override;
   // End ImporterBridge implementation.
 
+  BRAVE_EXTERNAL_PROCESS_IMPORTER_BRIDGE_H_
  private:
   ~ExternalProcessImporterBridge() override;
 
