diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SettingsSearchCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SettingsSearchCoordinator.java
index aa73b51996342359fea6b1ac7ad6e4b2840316e8..60a7f3f6fcb8b140a808ca27d1e12bc9277026f4 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SettingsSearchCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SettingsSearchCoordinator.java
@@ -4,6 +4,8 @@
 
 package org.chromium.chrome.browser.settings.search;
 
+import org.chromium.chrome.browser.theme.BraveDynamicColors;
+import org.chromium.components.browser_ui.settings.BraveSettingsIconTintUtils;
 import static androidx.annotation.VisibleForTesting.PRIVATE;
 
 import static org.chromium.base.CallbackUtils.emptyRunnable;
@@ -314,6 +316,12 @@ public class SettingsSearchCoordinator
     /** Initializes search UI, sets up listeners, backpress action handler, etc. */
     @Initializer
     public void initializeSearchUi(@Nullable Bundle savedState) {
+        initializeSearchUi_ChromiumImpl(savedState);
+        View searchBox = requireViewById(R.id.search_box);
+        BraveSettingsIconTintUtils.applyIconTint(searchBox.requireViewById(R.id.search_icon), BraveDynamicColors.isDynamicColorsEnabled());
+    }
+
+    private void initializeSearchUi_ChromiumImpl(@Nullable Bundle savedState) {
         mUseMultiColumn = mUseMultiColumnSupplier.getAsBoolean();
         // AppBarLayout(app_bar_layout)
         //         +-- FrameLayout
