diff --git a/chrome/browser/profiles/profile_io_data.cc b/chrome/browser/profiles/profile_io_data.cc
index a9a5998fdf673de4ea2645cd9aa6495d9a9c00d6..ab90a2bd82558b526c8dc6a69194ca15759375c5 100644
--- a/chrome/browser/profiles/profile_io_data.cc
+++ b/chrome/browser/profiles/profile_io_data.cc
@@ -24,6 +24,7 @@
 #include "base/strings/string_util.h"
 #include "base/task_scheduler/post_task.h"
 #include "base/threading/thread_task_runner_handle.h"
+#include "brave/browser/net/brave_profile_network_delegate.h"
 #include "build/build_config.h"
 #include "chrome/browser/browser_process.h"
 #include "chrome/browser/chrome_notification_types.h"
@@ -1045,7 +1046,7 @@ void ProfileIOData::Init(
   ChromeNetworkDelegate* chrome_network_delegate_unowned = nullptr;
   if (!base::FeatureList::IsEnabled(network::features::kNetworkService)) {
     std::unique_ptr<ChromeNetworkDelegate> chrome_network_delegate(
-        new ChromeNetworkDelegate(
+        new BraveProfileNetworkDelegate(
 #if BUILDFLAG(ENABLE_EXTENSIONS)
             io_thread_globals->extension_event_router_forwarder.get(),
 #else
