diff --git a/chrome/android/chrome_public_apk_tmpl.gni b/chrome/android/chrome_public_apk_tmpl.gni
index 13ab7ec536d429e3444ab640c790b51af183be33..e9dbce7b59158cfd76a62bae84f8f5c4a118fb19 100644
--- a/chrome/android/chrome_public_apk_tmpl.gni
+++ b/chrome/android/chrome_public_apk_tmpl.gni
@@ -309,6 +309,7 @@ template("chrome_common_apk_or_module_tmpl") {
       "*ic_file_download_white*",  # Bottom edge seems misaligned.
       "*ic_lock.*",  # Bottom edge seems misaligned.
     ]
+    resource_exclusion_exceptions += brave_resources_exclusion_exceptions
 
     # Most of these, with the exception of resource_exclusion_exceptions,
     # are currently duplicated in system_webview_apk_tmpl.gni.
@@ -333,6 +334,7 @@ template("chrome_common_apk_or_module_tmpl") {
 
       proguard_enabled = true
       proguard_configs = [ "//chrome/android/proguard/main.flags" ]
+      proguard_configs += [ "//brave/android/java/proguard.flags" ]
       if (_is_monochrome) {
         proguard_configs +=
             [ "//android_webview/nonembedded/java/proguard.flags" ]
@@ -744,6 +746,7 @@ template("chrome_common_apk_or_module_tmpl") {
                                  "version_code",
                                  "version_name",
                                ])
+    _wireguard_target = "//brave/third_party/android_deps:com_wireguard_android_J__unpack_aar" deps += [ _wireguard_target] _libwireguard_dir = get_label_info("//brave/third_party/android_deps:com_wireguard_android($default_toolchain)","target_out_dir") + "/com_wireguard_android_java/jni" if (android_64bit_target_cpu) {loadable_modules +=[ "$_libwireguard_dir/$android_app_abi/libwg-go.so" ] secondary_abi_loadable_modules +=[ "$_libwireguard_dir/$android_app_secondary_abi/libwg-go.so" ]} else {loadable_modules +=[ "$_libwireguard_dir/$android_app_abi/libwg-go.so" ]}
   }
 }
 
@@ -772,6 +775,7 @@ template("monochrome_public_common_apk_or_module_tmpl") {
     if (!defined(is_monochrome)) {
       is_trichrome = true
     }
+    deps += brave_monochrome_public_apk_or_module_tmpl_deps loadable_modules = brave_monochrome_public_apk_or_module_tmpl_loadable_modules secondary_abi_loadable_modules = brave_monochrome_public_apk_or_module_tmpl_secondary_abi_loadable_modules
   }
 }
 template("chrome_public_common_apk_or_module_tmpl") {
