diff --git a/chrome/browser/resources/extensions/item_list.ts b/chrome/browser/resources/extensions/item_list.ts
index 7504fd84328f685eba76ba68e96ee8788aca412d..4a6326d988652f72df1a923dcec79118a731a60c 100644
--- a/chrome/browser/resources/extensions/item_list.ts
+++ b/chrome/browser/resources/extensions/item_list.ts
@@ -18,6 +18,7 @@ import type {ExtensionsItemElement, ItemDelegate} from './item.js';
 import {getCss} from './item_list.css.js';
 import {getHtml} from './item_list.html.js';
 import {getMv2ExperimentStage, Mv2ExperimentStage} from './mv2_deprecation_util.js';
+import './brave_item_list_more_items.js'
 
 type Filter = (info: chrome.developerPrivate.ExtensionInfo) => boolean;
 
