diff --git a/chrome/browser/ui/tabs/tab_strip_prefs.cc b/chrome/browser/ui/tabs/tab_strip_prefs.cc
index 3e21f5863698935b24c1159116ab22cf23562418..2f5ee8ca567bd39c93972fba1a6519d3752439be 100644
--- a/chrome/browser/ui/tabs/tab_strip_prefs.cc
+++ b/chrome/browser/ui/tabs/tab_strip_prefs.cc
@@ -27,6 +27,7 @@ std::optional<bool> g_tab_search_trailing_tabstrip_at_startup = std::nullopt;
 namespace tabs {
 
 bool GetDefaultTabSearchRightAligned() {
+  BRAVE_GET_DEFAULT_TAB_SEARCH_RIGHT_ALIGNED
   // These platforms are all left aligned, the others should be right.
 #if BUILDFLAG(IS_CHROMEOS) || BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_WIN)
   return false;
