diff --git a/chrome/android/chrome_public_apk_tmpl.gni b/chrome/android/chrome_public_apk_tmpl.gni
index 9153f5f5d9ecfec33cebc12cc018902f73d7fada..9315feebbd32b30180873212f98b56ebe58d7e74 100644
--- a/chrome/android/chrome_public_apk_tmpl.gni
+++ b/chrome/android/chrome_public_apk_tmpl.gni
@@ -168,6 +168,7 @@ template("chrome_public_common_apk_or_module_tmpl") {
 
     # Remove unneeded entries from material design values.xml files.
     resource_values_filter_rules = [ "${_material_package}:[Tt]oolbarLayout" ]
+    resource_exclusion_exceptions += brave_resources_exclusion_exceptions
 
     if (!_is_monochrome) {
       product_config_java_packages = [ "org.chromium.chrome.browser" ]
@@ -258,6 +259,7 @@ template("chrome_public_common_apk_or_module_tmpl") {
       if (!defined(proguard_configs)) {
         proguard_configs = []
       }
+      proguard_configs += ["//brave/android/java/proguard.flags",]
       proguard_configs += [
         "//chrome/android/proguard/main.flags",
         "//base/android/proguard/chromium_apk.flags",
