diff --git a/chrome/app_shim/app_shim_controller.mm b/chrome/app_shim/app_shim_controller.mm
index ac560719f93a75ab6df22c7b67702acebbd90bbd..f3c8395a3b51bf5aac96b9da4eaeddb34442dbd5 100644
--- a/chrome/app_shim/app_shim_controller.mm
+++ b/chrome/app_shim/app_shim_controller.mm
@@ -636,6 +636,7 @@ mojo::PlatformChannelEndpoint AppShimController::ConnectToBrowser(
 
 void AppShimController::SendBootstrapOnShimConnected(
     mojo::PlatformChannelEndpoint endpoint) {
+  AppendTestLaunchUrls(launch_urls_);
   DCHECK_EQ(init_state_, InitState::kWaitingForChromeReady);
   init_state_ = InitState::kHasSentOnShimConnected;
 
