diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index be7f0388a76d75fb1276616f65c335036d0d7031..bb236d454caff063fc230ae868547908160cedc4 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -78,6 +78,7 @@ class AvatarToolbarButton : public ToolbarButton,
   // ToolbarButton:
   void NotifyClick(const ui::Event& event) override;
 
+BRAVE_AVATAR_BUTTON_H
  private:
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
