diff --git a/chrome/browser/resources/password_manager/password_manager.ts b/chrome/browser/resources/password_manager/password_manager.ts
index 851c96d19d54ea71c751e64ea0b7b598bd37d86c..ac374839545daaf756ccc4ae91e22236156f5d77 100644
--- a/chrome/browser/resources/password_manager/password_manager.ts
+++ b/chrome/browser/resources/password_manager/password_manager.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './password_manager_app.js';
 
 export {SettingsPrefsElement} from '/shared/settings/prefs/prefs.js';
