diff --git a/chrome/browser/resources/settings/site_settings/settings_category_default_radio_group.ts b/chrome/browser/resources/settings/site_settings/settings_category_default_radio_group.ts
index 1ad00705bb9c003c7bae916ec96d622758cacd48..2b9a7c58ee6df51d182cf58ab11e2fb06334bc3f 100644
--- a/chrome/browser/resources/settings/site_settings/settings_category_default_radio_group.ts
+++ b/chrome/browser/resources/settings/site_settings/settings_category_default_radio_group.ts
@@ -175,6 +175,7 @@ export class SettingsCategoryDefaultRadioGroupElement extends
       case ContentSettingsTypes.WEB_PRINTING:
         // "Ask" vs "Blocked".
         return ContentSetting.ASK;
+      case ContentSettingsTypes.ETHEREUM: case ContentSettingsTypes.SOLANA:  case ContentSettingsTypes.CARDANO: case ContentSettingsTypes.GOOGLE_SIGN_IN: case ContentSettingsTypes.BRAVE_OPEN_AI_CHAT: return ContentSetting.ASK; case ContentSettingsTypes.AUTOPLAY: return ContentSetting.ALLOW;
       default:
         assertNotReached('Invalid category: ' + this.category);
     }
