diff --git a/chrome/browser/ui/omnibox/omnibox_edit_model.cc b/chrome/browser/ui/omnibox/omnibox_edit_model.cc
index 97d8f09eb2e11de684ee008ac59c26f3b987661a..cf165e0b4cb516ba112e9a52852ff5c4d108a5ad 100644
--- a/chrome/browser/ui/omnibox/omnibox_edit_model.cc
+++ b/chrome/browser/ui/omnibox/omnibox_edit_model.cc
@@ -537,6 +537,10 @@ bool OmniboxEditModel::ShouldShowCurrentPageIcon() const {
 
 ui::ImageModel OmniboxEditModel::GetSuperGIcon(int image_size,
                                                bool dark_mode) const {
+  if ((true)) {
+    return ui::ImageModel::FromVectorIcon(vector_icons::kGoogleColorIcon,
+                                          gfx::kPlaceholderColor, image_size);
+  }
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
   if (dark_mode) {
     return ui::ImageModel::FromVectorIcon(
@@ -683,6 +687,8 @@ void OmniboxEditModel::Revert() {
 }
 
 void OmniboxEditModel::StartAutocomplete(bool prevent_inline_autocomplete) {
+  auto pasted = SetInputIsPastedFromClipboard(
+      controller_, paste_state_ != PasteState::kNone);
   const std::u16string input_text = MaybePrependKeyword(user_text_);
 
   size_t cursor_position =
@@ -729,6 +735,9 @@ void OmniboxEditModel::StartAutocomplete(bool prevent_inline_autocomplete) {
 }
 
 bool OmniboxEditModel::CanPasteAndGo(const std::u16string& text) const {
+  if (IsBraveCommanderCommand(text)) {
+    return false;
+  }
   return searchbox::CanPasteAndGo(controller_->client(), text);
 }
 
