diff --git a/chrome/installer/mac/signing/model.py b/chrome/installer/mac/signing/model.py
index d3dbdbe7a636cf8b46629e3fa2369fc778192e95..34ce73fcb5b543db39e71078f61d7fca411f8838 100644
--- a/chrome/installer/mac/signing/model.py
+++ b/chrome/installer/mac/signing/model.py
@@ -305,7 +305,7 @@ class Paths(object):
         Returns:
             Path to the packaging directory.
         """
-        return os.path.join(self.input, '{} Packaging'.format(config.product))
+        return os.path.join(self.input, '{}_Packaging'.format(config.product.replace(' ', '_')))
 
     def replace_work(self, new_work):
         """Creates a new Paths with the same input and output directories, but
