diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index a2a10cdd71b4cfaac944cc42b2dbdf3af89af93f..88327d508f4905a8928a0e44c7e9fc2efdf370ab 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -89,6 +89,7 @@ grit_strings("generated_resources") {
   if (is_android) {
     create_android_resources = true
   }
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
