diff --git a/chrome/browser/contextual_tasks/contextual_tasks_side_panel_coordinator_unittest.cc b/chrome/browser/contextual_tasks/contextual_tasks_side_panel_coordinator_unittest.cc
index 64152cc4572445db4f177c2259e52510e3135eaa..19bd85fda4c4ba45e656d9b24aaa33e9fcce4c5a 100644
--- a/chrome/browser/contextual_tasks/contextual_tasks_side_panel_coordinator_unittest.cc
+++ b/chrome/browser/contextual_tasks/contextual_tasks_side_panel_coordinator_unittest.cc
@@ -137,6 +137,7 @@ class MockSidePanelUI : public SidePanelUI {
               SetNoDelaysForTesting,
               (bool no_delays_for_testing),
               (override));
+  BRAVE_MOCK_SIDE_PANEL_UI_TOGGLE
 };
 
 class MockActiveTaskContextProvider : public ActiveTaskContextProvider {
