diff --git a/chrome/browser/ui/browser_navigator.cc b/chrome/browser/ui/browser_navigator.cc
index 960f3ab475c3d4010e70603968f3bb5ba53e46ec..02b97de7f4b2d5a0da4dc7aeabf2d5e09b971e85 100644
--- a/chrome/browser/ui/browser_navigator.cc
+++ b/chrome/browser/ui/browser_navigator.cc
@@ -488,6 +488,7 @@ void Navigate(NavigateParams* params) {
     // Block any navigation requests in locked fullscreen mode.
     return;
   }
+  BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL
 
   // Open System Apps in their standalone window if necessary.
   // TODO(crbug.com/1096345): Remove this code after we integrate with intent
@@ -755,6 +756,7 @@ void Navigate(NavigateParams* params) {
 bool IsHostAllowedInIncognito(const GURL& url) {
   std::string scheme = url.scheme();
   base::StringPiece host = url.host_piece();
+  if (!IsHostAllowedInIncognitoBraveImpl(host)) return false;
   if (scheme == chrome::kChromeSearchScheme) {
     return host != chrome::kChromeUIThumbnailHost &&
            host != chrome::kChromeUIThumbnailHost2 &&
