diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java
index 5c6f0f445fe20e45d8dfe950ef19556fe1c8caaf..5791510014115812b08ddcc00f8ba0b1718ba351 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java
@@ -120,7 +120,7 @@ public class BottomControlsCoordinator {
                     root.findViewById(R.id.bottom_container_slot), themeColorProvider,
                     scrimCoordinator);
         } else {
-            mBottomToolbarCoordinator = new BottomToolbarCoordinator(
+            mBottomToolbarCoordinator = new BraveBottomToolbarCoordinator(root,
                     root.findViewById(R.id.bottom_toolbar_stub), tabProvider,
                     tabSwitcherLongclickListener, themeColorProvider, shareDelegateSupplier,
                     showStartSurfaceCallable, openHomepageAction, setUrlBarFocusAction,
@@ -197,4 +197,5 @@ public class BottomControlsCoordinator {
         if (mTabGroupUi != null) mTabGroupUi.destroy();
         mMediator.destroy();
     }
+    public void updateBookmarkButton(boolean isBookmarked, boolean editingAllowed) {if (mBottomToolbarCoordinator != null) { mBottomToolbarCoordinator.updateBookmarkButton(isBookmarked, editingAllowed);}}
 }
