diff --git a/chrome/browser/storage/storage_notification_service_impl.cc b/chrome/browser/storage/storage_notification_service_impl.cc
index b9b8c958e9566c24c2cba1fd8d7997cd87b250b0..b43a1cdf97444d6ae3f25d957a7af00de43917ad 100644
--- a/chrome/browser/storage/storage_notification_service_impl.cc
+++ b/chrome/browser/storage/storage_notification_service_impl.cc
@@ -7,9 +7,9 @@
 #include "build/build_config.h"
 #include "chrome/browser/profiles/profile.h"
 
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
 #include "chrome/browser/ui/storage_pressure_bubble.h"
-#endif
+//#endif
 
 base::RepeatingCallback<void(const url::Origin)>
 StorageNotificationServiceImpl::GetStoragePressureNotificationClosure() {
