diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 2bce2f66a1ec9c08ec6ae043239042c856c920db..ff609cb828f2b58436df7a8975b8ece20460527d 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -732,6 +732,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
