diff --git a/chrome/browser/component_updater/BUILD.gn b/chrome/browser/component_updater/BUILD.gn
index 60dab2ab24db6e596859d767c37ab7870a5cef7b..b27e37b5f3396a18c7225dd5fd6a59a21b10c8d2 100644
--- a/chrome/browser/component_updater/BUILD.gn
+++ b/chrome/browser/component_updater/BUILD.gn
@@ -189,6 +189,7 @@ source_set("impl") {
     "//services/cert_verifier/public/mojom",
     "//services/network/public/cpp",
   ]
+  import("//brave/chromium_src/chrome/browser/component_updater/sources.gni") deps += brave_chromium_src_chrome_browser_component_updater_impl_deps
 
   if (safe_browsing_mode == 2) {
     sources += [ "real_time_url_checks_allowlist_component_installer.cc" ]
