diff --git a/chrome/browser/download/internal/android/BUILD.gn b/chrome/browser/download/internal/android/BUILD.gn
index 5fc9c01d4feaeebae141635638e720cd13840f1a..78887fcfec78695ee1cac3c8d58c7166cd895bce 100644
--- a/chrome/browser/download/internal/android/BUILD.gn
+++ b/chrome/browser/download/internal/android/BUILD.gn
@@ -213,6 +213,7 @@ android_resources("java_resources") {
     "java/res/values/ids.xml",
     "java/res/values/styles.xml",
   ]
+  sources += brave_download_internal_java_resources
 
   deps = [
     "//chrome/browser/download/android:java_resources",
