diff --git a/chrome/browser/ui/views/tabs/tab_strip.cc b/chrome/browser/ui/views/tabs/tab_strip.cc
index f238c32d45f9c74c2f16b055619c2fb6bc843942..4962814993e84e78136fbe4f48406262625457b0 100644
--- a/chrome/browser/ui/views/tabs/tab_strip.cc
+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
@@ -1580,7 +1580,7 @@ int TabStrip::TabToFollowingNewTabButtonSpacing() const {
   // Pre-refresh, there are a variety of hardcoded values.
   const int mode = MD::GetMode();
   DCHECK_LE(mode, 2);
-  constexpr int kNewTabButtonSpacing[] = {-5, -6, 6};
+  constexpr int kNewTabButtonSpacing[] = {6, 7, 7};
   return kNewTabButtonSpacing[mode];
 }
 
