diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 8a69a486b5d7184e7e4680b3e5f98d6f6818b578..63c2da2ba00d8c6ce3f67722fb87b3a71d929092 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -228,6 +228,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  sources += brave_chrome_renderer_sources public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
