diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index cbadf12cfcdd4c6a48856fe54952875706415229..73ce1aea117b53568f24e1acf23ff9c8d8d9175c 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -7,7 +7,7 @@ import {loadTimeData} from 'chrome://resources/js/load_time_data.m.js';
 import {pageVisibility} from './page_visibility.js';
 import {Route, Router} from './router.js';
 import {SettingsRoutes} from './settings_routes.js';
-
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -217,6 +217,7 @@ function createBrowserSettingsRoutes() {
       // </if>
     }
   }
+  addBraveRoutes(r)
   return r;
 }
 
