diff --git a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts
index df3b2edf7f17fee781c677f4f6cc62d428f78102..e1a637dbd953ec95765cd1f27b816b59e7718b41 100644
--- a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts
+++ b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts
@@ -160,6 +160,7 @@ export class ProfilePickerMainViewElement extends
   }
 
   override firstUpdated() {
+    this.$.addProfile.style.borderRadius = '8px';
     this.addEventListener('view-enter-finish', this.onViewEnterFinish_);
   }
 
