diff --git a/chrome/browser/resources/signin/profile_picker/profile_card.html.ts b/chrome/browser/resources/signin/profile_picker/profile_card.html.ts
index 531d1fc9b1adf3bdd73d2cfbc9c73da32117df5c..1c5d0af1b02d0bb4892d556fa34ce93adad3ac72 100644
--- a/chrome/browser/resources/signin/profile_picker/profile_card.html.ts
+++ b/chrome/browser/resources/signin/profile_picker/profile_card.html.ts
@@ -9,7 +9,7 @@ import type {ProfileCardElement} from './profile_card.js';
 export function getHtml(this: ProfileCardElement) {
   return html`<!--_html_template_start_-->
 <div id="profileCardContainer">
-  <cr-button id="profileCardButton" @click="${this.onProfileClick_}"
+  <cr-button id="profileCardButton" class="plain" @click="${this.onProfileClick_}"
       aria-label="${this.profileState.profileCardButtonLabel}">
     <div id="avatarContainer">
       <img class="profile-avatar" alt="" .src="${this.profileState.avatarIcon}">
