diff --git a/chrome/browser/ui/cocoa/main_menu_builder.mm b/chrome/browser/ui/cocoa/main_menu_builder.mm
index 9f5703897dd3f08b175cd9bc3858567b63864365..632a8d6b1dad7372d46ebc25b81bb1b79b72c7d1 100644
--- a/chrome/browser/ui/cocoa/main_menu_builder.mm
+++ b/chrome/browser/ui/cocoa/main_menu_builder.mm
@@ -107,6 +107,7 @@ base::scoped_nsobject<NSMenuItem> BuildFileMenu(
                 Item(IDS_NEW_INCOGNITO_WINDOW_MAC)
                     .command_id(IDC_NEW_INCOGNITO_WINDOW)
                     .remove_if(is_pwa),
+                BRAVE_BUILD_FILE_MENU
                 Item(IDS_REOPEN_CLOSED_TABS_MAC)
                     .command_id(IDC_RESTORE_TAB)
                     .remove_if(is_pwa),
@@ -470,6 +471,7 @@ base::scoped_nsobject<NSMenuItem> BuildHelpMenu(
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
             Item(IDS_FEEDBACK_MAC).command_id(IDC_FEEDBACK),
 #endif
+                BRAVE_BUILD_HELP_MENU
                 Item(IDS_HELP_MAC)
                     .string_format_1(product_name)
                     .command_id(IDC_HELP_PAGE_VIA_MENU),
