diff --git a/chrome/browser/shell_integration.h b/chrome/browser/shell_integration.h
index 3929d7e117d32272ff74ebc90b82b3b8639d4957..d242def5af89766c63b1cb0c1777895bc8f3c5b9 100644
--- a/chrome/browser/shell_integration.h
+++ b/chrome/browser/shell_integration.h
@@ -236,6 +236,7 @@ class DefaultBrowserWorker : public DefaultWebClientWorker {
   DefaultBrowserWorker(const DefaultBrowserWorker&) = delete;
   DefaultBrowserWorker& operator=(const DefaultBrowserWorker&) = delete;
 
+  BRAVE_DEFAULT_BROWSER_WORKER_H
  protected:
   ~DefaultBrowserWorker() override;
 
