diff --git a/chrome/browser/profiles/profile.h b/chrome/browser/profiles/profile.h
index 2e13c0535cd576ad5eee4697e5658235d0d2b334..b087d631e753c4fa446af532dd0e6a4f0ab96827 100644
--- a/chrome/browser/profiles/profile.h
+++ b/chrome/browser/profiles/profile.h
@@ -273,6 +273,8 @@ class Profile : public content::BrowserContext {
   // Returns whether it is a guest session.
   virtual bool IsGuestSession() const;
 
+  bool IsTorProfile() const override;
+
   // Returns whether it is a system profile.
   virtual bool IsSystemProfile() const;
 
