diff --git a/chrome/browser/ui/android/toolbar/BUILD.gn b/chrome/browser/ui/android/toolbar/BUILD.gn
index c4257fcde56e90025c926a12667c61b05b97479b..15d28faea7b0f27906dd710d8f29fcf51167ea92 100644
--- a/chrome/browser/ui/android/toolbar/BUILD.gn
+++ b/chrome/browser/ui/android/toolbar/BUILD.gn
@@ -105,6 +105,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/HeightTransitionHandler.java",
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/TabStripTransitionCoordinator.java",
   ]
+  sources += brave_browser_ui_android_toolbar_sources
 
   deps = [
     ":java_resources",
