diff --git a/chrome/android/java/src/org/chromium/chrome/browser/infobar/InfoBarContainerView.java b/chrome/android/java/src/org/chromium/chrome/browser/infobar/InfoBarContainerView.java
index 9c31fed21a20cc0ee6a871354cc3077945953111..d567a81855ddcb8a8abf102188ff44350a7cf150 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/infobar/InfoBarContainerView.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/infobar/InfoBarContainerView.java
@@ -86,6 +86,7 @@ public class InfoBarContainerView extends SwipableOverlayView
      * @param isTablet Whether this view is displayed on tablet or not.
      * @deprecated Please use the constructor with the EdgeToEdgeSupplier.
      */
+    /*
     @Deprecated
     InfoBarContainerView(
             Context context,
@@ -94,6 +95,7 @@ public class InfoBarContainerView extends SwipableOverlayView
             boolean isTablet) {
         this(context, containerViewObserver, browserControlsStateProvider, null, isTablet);
     }
+     */
 
     /**
      * @param context The {@link Context} that this view is attached to.
