diff --git a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationPlatformBridge.java b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationPlatformBridge.java
index 5f5dab6c7d871dc3faab39ca8ed34c5d8149e882..4e69b83092d63507a8a82e4da5d3dbd5023b8076 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationPlatformBridge.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationPlatformBridge.java
@@ -146,7 +146,7 @@ public class NotificationPlatformBridge {
     private static long sLastPreUnsubscribePreNativeTaskStartRealMillis = -1;
 
     /** Encapsulates attributes that identify a notification and where it originates from. */
-    private static class NotificationIdentifyingAttributes {
+    public static class NotificationIdentifyingAttributes {
         public final String notificationId;
         public final @NotificationType int notificationType;
         public final String origin;
@@ -1034,7 +1034,7 @@ public class NotificationPlatformBridge {
                 .storeNotificationResourcesIfSuspended(notification);
     }
 
-    private static NotificationBuilderBase prepareNotificationBuilder(
+    protected static NotificationBuilderBase prepareNotificationBuilder(
             NotificationIdentifyingAttributes identifyingAttributes,
             boolean vibrateEnabled,
             String title,
