diff --git a/chrome/browser/ui/side_panel/side_panel_entry.h b/chrome/browser/ui/side_panel/side_panel_entry.h
index 0846458e42bede29f9cc39e25f181fbf6b6956cf..4a4012eb61098b95fc066fc1e4299f0c6eabe511 100644
--- a/chrome/browser/ui/side_panel/side_panel_entry.h
+++ b/chrome/browser/ui/side_panel/side_panel_entry.h
@@ -33,7 +33,7 @@ enum class SidePanelEntryHideReason;
 class SidePanelEntry final : public ui::PropertyHandler {
  public:
   // The default and minimum acceptable side panel content width.
-  static constexpr int kSidePanelDefaultContentWidth = 360;
+  static constexpr int kSidePanelDefaultContentWidth = 400;
   using CreateContentCallback =
       base::RepeatingCallback<SidePanelNativeView(SidePanelEntryScope&)>;
   using Id = SidePanelEntryId;
