diff --git a/chrome/installer/mac/sign_chrome.py b/chrome/installer/mac/sign_chrome.py
index b1cb1f4beaca0f61cb4a1dff0039dcc02c2614f2..321a7c4802ed60bbb91fc918f4f33fce746a79b0 100755
--- a/chrome/installer/mac/sign_chrome.py
+++ b/chrome/installer/mac/sign_chrome.py
@@ -44,6 +44,7 @@ def create_config(config_args, development):
     from signing.chromium_config import ChromiumCodeSignConfig
     config_class = ChromiumCodeSignConfig
 
+    """
     # Then search for the internal config for Google Chrome.
     try:
         from signing.internal_config import InternalCodeSignConfig
@@ -53,6 +54,7 @@ def create_config(config_args, development):
         # internal config has to be available.
         if config_class.is_chrome_branded():
             raise e
+    """
 
     if development:
 
@@ -74,6 +76,8 @@ def create_config(config_args, development):
 
         config_class = DevelopmentCodeSignConfig
 
+    from signing_helper import GetBraveSigningConfig
+    config_class = GetBraveSigningConfig(config_class, development)
     return config_class(*config_args)
 
 
