diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 09ae33ecc8b48169c5af298992b7875b7f3504e3..3ba8c023d33275a9d4f6fbaf582574ef1949cc82 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4909,6 +4909,7 @@ const FeatureEntry::FeatureVariation kMobilePromoOnDesktopVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
