diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index aa50e5bf89fb0b3c12cde19cd8d9bc78c5e91077..53e70e5f2a358b19ff3d35493a95ec1b47ae76da 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -89,6 +89,7 @@ source_set("channel_info") {
   } else if (is_fuchsia) {
     sources += [ "channel_info_fuchsia.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -535,6 +536,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
