diff --git a/chrome/browser/themes/theme_helper_win.h b/chrome/browser/themes/theme_helper_win.h
index ef35bd8a699c2d7ec53e6ef84e7761000f59fb89..478626871d4269cc70b4959b6abfb9235a9862ce 100644
--- a/chrome/browser/themes/theme_helper_win.h
+++ b/chrome/browser/themes/theme_helper_win.h
@@ -19,6 +19,7 @@ class ThemeHelperWin : public ThemeHelper {
 
   ThemeHelperWin(const ThemeHelperWin&) = delete;
   ThemeHelperWin& operator=(const ThemeHelperWin&) = delete;
+  BRAVE_THEME_HELPER_WIN_H_
 
  private:
   // ThemeService:
