diff --git a/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts b/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts
index 6a1b212f21651d116f4af2da80a14b255fd341a8..68d8dcfffe5bb59ddfbbfd671b280bda8ea0b1b3 100644
--- a/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts
+++ b/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts
@@ -9,6 +9,7 @@
  */
 
 import 'chrome://resources/cr_elements/icons.html.js';
+import '../brave_overrides/config.js'
 import 'chrome://resources/cr_elements/cr_collapse/cr_collapse.js';
 import 'chrome://resources/cr_elements/cr_expand_button/cr_expand_button.js';
 import 'chrome://resources/cr_elements/cr_link_row/cr_link_row.js';
