diff --git a/chrome/browser/resources/extensions/BUILD.gn b/chrome/browser/resources/extensions/BUILD.gn
index 8e92d58854d55a27e53d4a2b93bb299149ff4c95..bde475d2144cb31636d524d7500d731fbb4574a1 100644
--- a/chrome/browser/resources/extensions/BUILD.gn
+++ b/chrome/browser/resources/extensions/BUILD.gn
@@ -21,6 +21,7 @@ optimize_webui("build") {
 
   deps = [
     ":unpak",
+    "//brave/browser/resources/extensions:unpak",
   ]
 }
 
