diff --git a/chrome/android/java/src/org/chromium/chrome/browser/omnibox/suggestions/DropdownItemViewInfoListBuilder.java b/chrome/android/java/src/org/chromium/chrome/browser/omnibox/suggestions/DropdownItemViewInfoListBuilder.java
index 9a8adf73f731a5febebb90c618ff3a0e3c7a7137..30ef44d1b9388be630ec7c21834bd16dad7d2d3e 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/omnibox/suggestions/DropdownItemViewInfoListBuilder.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/omnibox/suggestions/DropdownItemViewInfoListBuilder.java
@@ -94,7 +94,7 @@ class DropdownItemViewInfoListBuilder {
                 () -> mShareDelegateSupplier == null ? null : mShareDelegateSupplier.get();
 
         mHeaderProcessor = new HeaderProcessor(context, host, delegate);
-        registerSuggestionProcessor(new EditUrlSuggestionProcessor(
+        registerSuggestionProcessor(new org.chromium.chrome.browser.omnibox.suggestions.editurl.BraveEditUrlSuggestionProcessor(
                 context, host, delegate, iconBridgeSupplier, mActivityTabSupplier, shareSupplier));
         registerSuggestionProcessor(
                 new AnswerSuggestionProcessor(context, host, textProvider, imageFetcherSupplier));
