diff --git a/chrome/installer/util/shell_util.cc b/chrome/installer/util/shell_util.cc
index 5b1d1c9a78b57d0c2b30828a87fc68d6fc701973..778e6119b3f2d496719b89c0ffb814126892e92f 100644
--- a/chrome/installer/util/shell_util.cc
+++ b/chrome/installer/util/shell_util.cc
@@ -355,6 +355,7 @@ void GetChromeProgIdEntries(
   app_info.delegate_clsid = install_static::GetLegacyCommandExecuteImplClsid();
 
   GetProgIdEntries(app_info, entries);
+  BRAVE_GET_CHROME_PROG_ID_ENTRIES
 
   if (!app_info.delegate_clsid.empty()) {
     auto delegate_execute_entries =
@@ -451,6 +452,7 @@ void GetShellIntegrationEntries(
 
   const std::wstring html_prog_id(GetBrowserProgId(suffix));
   for (int i = 0; ShellUtil::kPotentialFileAssociations[i] != nullptr; i++) {
+    BRAVE_GET_SHELL_INTEGRATION_ENTRIES
     entries->push_back(std::make_unique<RegistryEntry>(
         capabilities + L"\\FileAssociations",
         ShellUtil::kPotentialFileAssociations[i], html_prog_id));
@@ -475,6 +477,7 @@ void GetAppExtRegistrationEntries(
   std::wstring key_name =
       base::StrCat({ShellUtil::kRegClasses, kFilePathSeparator, ext,
                     kFilePathSeparator, ShellUtil::kRegOpenWithProgids});
+  BRAVE_GET_APP_EXT_REGISTRATION_ENTRIES
   entries->push_back(
       std::make_unique<RegistryEntry>(key_name, prog_id, std::wstring()));
 }
@@ -714,6 +717,7 @@ std::wstring GetTargetForDefaultAppsSettings(const wchar_t* protocol) {
     return base::StringPrintf(kSystemSettingsDefaultAppsFormat, L"Browser");
   if (base::EqualsCaseInsensitiveASCII(protocol, L"mailto"))
     return base::StringPrintf(kSystemSettingsDefaultAppsFormat, L"Email");
+  BRAVE_GET_TARGET_FOR_DEFAULT_APP_SETTINGS
   return L"SettingsPageAppsDefaultsProtocolView";
 }
 
@@ -1628,8 +1632,10 @@ const wchar_t* ShellUtil::kPotentialFileAssociations[] = {
     L".htm", L".html",  L".pdf",  L".shtml", L".svg",
     L".xht", L".xhtml", L".webp", nullptr};
 const wchar_t* ShellUtil::kBrowserProtocolAssociations[] = {L"ftp", L"http",
+                                                            BRAVE_BROWSER_PROTOCOL_ASSOCIATIONS
                                                             L"https", nullptr};
 const wchar_t* ShellUtil::kPotentialProtocolAssociations[] = {
+    BRAVE_POTENTIAL_PROTOCOL_ASSOCIATIONS
     L"ftp", L"http",  L"https", L"irc", L"mailto", L"mms",    L"news", L"nntp",
     L"sms", L"smsto", L"snews", L"tel", L"urn",    L"webcal", nullptr};
 const wchar_t* ShellUtil::kRegUrlProtocol = L"URL Protocol";
