diff --git a/chrome/browser/profile_resetter/reset_report_uploader.cc b/chrome/browser/profile_resetter/reset_report_uploader.cc
index fee0739a04ab8d54a5715e6c2e0fa557c6ec0952..57adb27517281380a6f999d9f7cb755a91ce858b 100644
--- a/chrome/browser/profile_resetter/reset_report_uploader.cc
+++ b/chrome/browser/profile_resetter/reset_report_uploader.cc
@@ -49,6 +49,7 @@ void ResetReportUploader::DispatchReport(
 
 void ResetReportUploader::DispatchReportInternal(
     const std::string& request_data) {
+  return; // feature disabled in Brave
   // Create traffic annotation tag.
   net::NetworkTrafficAnnotationTag traffic_annotation =
       net::DefineNetworkTrafficAnnotation("profile_resetter_upload", R"(
