diff --git a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
index bd434c4dcd6a52727d0180de54e000e3d32bbdbf..b87206c59508c1eddd4ed1d34fa96b0fa0a9264d 100644
--- a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
+++ b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
@@ -60,6 +60,7 @@ class TabHoverCardBubbleView : public views::BubbleDialogDelegateView {
     last_mouse_exit_timestamp_ = last_mouse_exit_timestamp;
   }
 
+  BRAVE_TAB_HOVER_CARD_BUBBLE_VIEW_H_
  private:
   friend class TabHoverCardBubbleViewBrowserTest;
   friend class TabHoverCardBubbleViewInteractiveUiTest;
