diff --git a/chrome/browser/profiles/profile_impl.h b/chrome/browser/profiles/profile_impl.h
index e33619d098783950b33bc970c07d197340bd565e..57eeba6884964250a89b8bd1c20fcfddaa53f7a6 100644
--- a/chrome/browser/profiles/profile_impl.h
+++ b/chrome/browser/profiles/profile_impl.h
@@ -166,6 +166,7 @@ class ProfileImpl : public Profile {
   void OnLogin() override;
   void InitChromeOSPreferences() override;
 #endif  // defined(OS_CHROMEOS)
+  BRAVE_PROFILE_IMPL_H_
 
  private:
 #if defined(OS_CHROMEOS)
