diff --git a/chrome/browser/ui/browser_window/BUILD.gn b/chrome/browser/ui/browser_window/BUILD.gn
index 0ea59c3ee88c3e9acec488ed0a97f5897c69475c..5cb26308cb623b94d0afab6c2b1a99d118dbf804 100644
--- a/chrome/browser/ui/browser_window/BUILD.gn
+++ b/chrome/browser/ui/browser_window/BUILD.gn
@@ -33,6 +33,7 @@ source_set("browser_window") {
       "//ui/base:types",
     ]
   }
+  deps += [ "//brave/browser/ui/browser_window" ]
 }
 
 source_set("create_browser_window") {
