diff --git a/chrome/installer/mac/signing/pipeline.py b/chrome/installer/mac/signing/pipeline.py
index a30e91c3c7be3d585ede5b2dbf54385ba368d5e5..deb43072b32232071ac0fa954d12b1ebe8db323d 100644
--- a/chrome/installer/mac/signing/pipeline.py
+++ b/chrome/installer/mac/signing/pipeline.py
@@ -690,5 +690,8 @@ def sign_all(orig_paths,
                         uuids_to_package_path.keys(), config):
                     package_path = uuids_to_package_path[result]
                     notarize.staple(package_path)
+        else:
+            # Copy the notarized app to the orig_paths.output dir where the user expects it.
+            commands.copy_files(dest_dir, orig_paths.output)
 
     _package_installer_tools(orig_paths, config)
