diff --git a/chrome/browser/ui/extensions/icon_with_badge_image_source.h b/chrome/browser/ui/extensions/icon_with_badge_image_source.h
index ea974293a35dd321c5a5da3c896aea7f935cf9f3..83a2d5cbf0b61552862b7760bdc07014b936f3b7 100644
--- a/chrome/browser/ui/extensions/icon_with_badge_image_source.h
+++ b/chrome/browser/ui/extensions/icon_with_badge_image_source.h
@@ -62,6 +62,7 @@ class IconWithBadgeImageSource : public gfx::CanvasImageSource {
   bool paint_blocked_actions_decoration() const {
     return paint_blocked_actions_decoration_;
   }
+  BRAVE_ICON_WITH_BADGE_IMAGE_SOURCE_H_
 
  private:
   // gfx::CanvasImageSource:
