diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
index 20c89001be4040b2691d4cfcade27120b7481543..4062268657297eef0122a40f3c8726d220349417 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
@@ -373,6 +373,7 @@ void BookmarkBubbleView::ShowBubble(
                                        bookmark_node);
 
   bool show_simplified_flow = !already_bookmarked;
+  BRAVE_BOOKMARK_BUBBLE_VIEW_SHOW_BUBBLE_SHOW_SIMPLIFIED
 
   auto bubble_delegate_unique = std::make_unique<BookmarkBubbleDelegate>(
       std::move(delegate), browser, url, show_simplified_flow);
@@ -504,6 +505,7 @@ void BookmarkBubbleView::ShowBubble(
   auto bubble = std::make_unique<views::BubbleDialogModelHost>(
       std::move(dialog_model), anchor_view, views::BubbleBorder::TOP_RIGHT);
   bookmark_bubble_ = bubble.get();
+  BRAVE_BOOKMARK_BUBBLE_VIEW_SHOW_BUBBLE_SET_ARROW
   if (highlighted_button)
     bubble->SetHighlightedButton(highlighted_button);
 
