diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index f22581ef5d5cf31c1bc9d8660ccbb71d9b96f870..e1f96733790ee47d037a7cbce1798521f91a4789 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -200,6 +200,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
