diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 017a8c3ff755f6e4142567d04740f682c2800b2b..815952e76c2425fa98b4ffad73b32a77dbc13f52 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -1260,6 +1260,7 @@ static_library("ui") {
     # split out yet.
     "//chrome/browser/webauthn:impl",
   ]
+  import("//brave/browser/sources.gni") import("//brave/components/brave_wallet/sources.gni") public_deps += [ "//brave/browser/ui" ] deps += brave_chrome_browser_ui_allow_circular_includes_from + brave_chrome_browser_ui_deps allow_circular_includes_from += brave_chrome_browser_ui_allow_circular_includes_from
 
   if (enable_vr && is_win) {
     deps += [ "//chrome/browser/vr:vr_base" ]
@@ -5614,6 +5615,7 @@ group("ui_public_dependencies") {
     "//components/enterprise/buildflags",
     "//components/paint_preview/buildflags",
     "//components/safe_browsing:buildflags",
+    "//components/saved_tab_groups/public",
     "//components/segmentation_platform/public",
     "//components/sync",
     "//components/sync_user_events",
@@ -5626,6 +5628,7 @@ group("ui_public_dependencies") {
   if (enable_extensions_core) {
     public_deps += [ "//extensions/browser" ]
   }
+  public_deps += [ "//brave/browser/ui:ui_public_dependencies" ]
   if (!is_android) {
     public_deps += [
       "//build:branding_buildflags",
