diff --git a/chrome/browser/importer/external_process_importer_client.h b/chrome/browser/importer/external_process_importer_client.h
index 9451b0917536e73b42c596527112d3119b2c4cc5..839f905b403a8103ecf22aba16dd03f52f7cf796 100644
--- a/chrome/browser/importer/external_process_importer_client.h
+++ b/chrome/browser/importer/external_process_importer_client.h
@@ -24,6 +24,7 @@
 #include "components/favicon_base/favicon_usage_data.h"
 #include "components/history/core/browser/history_types.h"
 #include "mojo/public/cpp/bindings/binding.h"
+#include "net/cookies/canonical_cookie.h"
 
 class ExternalProcessImporterHost;
 struct ImportedBookmarkEntry;
@@ -85,6 +86,10 @@ class ExternalProcessImporterClient
   void OnAutofillFormDataImportGroup(
       const std::vector<ImporterAutofillFormDataEntry>&
           autofill_form_data_entry_group) override;
+  void OnCookiesImportStart(uint32_t total_cookies_count) override {};
+  void OnCookiesImportGroup(const std::vector<net::CanonicalCookie>& cookies_group) override {};
+  void OnStatsImportReady(const BraveStats& stats) override {};
+  void OnLedgerImportReady(const BraveLedger& ledger) override {};
 
  protected:
   ~ExternalProcessImporterClient() override;
