diff --git a/chrome/browser/global_features.cc b/chrome/browser/global_features.cc
index 227504a6d2525746d30d87ab4fafaea5dcb303d7..f5ca8e9ecb965e7c786dccd12231b53aa3d2aa0f 100644
--- a/chrome/browser/global_features.cc
+++ b/chrome/browser/global_features.cc
@@ -100,6 +100,7 @@
 #endif
 #endif
 
+GlobalFeatures* CreateBraveGlobalFeatures();
 namespace {
 
 // This is the generic entry point for test code to stub out browser
@@ -118,7 +119,7 @@ std::unique_ptr<GlobalFeatures> GlobalFeatures::CreateGlobalFeatures() {
     return GetFactory().Run();
   }
   // Constructor is protected.
-  return base::WrapUnique(new GlobalFeatures());
+  return base::WrapUnique(CreateBraveGlobalFeatures());
 }
 
 GlobalFeatures::~GlobalFeatures() = default;
