diff --git a/chrome/installer/linux/BUILD.gn b/chrome/installer/linux/BUILD.gn
index b678129c30b8858fd1f126b29fa8b12690ec64a7..e7821bcc8aa8020109f9ecec69ed94343bd39db5 100644
--- a/chrome/installer/linux/BUILD.gn
+++ b/chrome/installer/linux/BUILD.gn
@@ -18,6 +18,7 @@ import("//third_party/angle/gni/angle.gni")
 import("//third_party/widevine/cdm/widevine.gni")
 import("//ui/gl/features.gni")
 import("//ui/qt/qt.gni")
+import("//brave/build/config.gni")
 
 if (bundle_widevine_cdm) {
   import("//media/cdm/library_cdm/cdm_paths.gni")
@@ -114,6 +115,7 @@ if (build_with_internal_optimization_guide) {
   }
 }
 
+packaging_files_executables -= excluded_packaging_files_executables packaging_files_executables += brave_packaging_files_executables
 packaging_files_binaries = packaging_files_executables + packaging_files_shlibs
 
 # TODO(mmoss) Any convenient way to get all the relevant build files?
@@ -221,6 +223,7 @@ action("merge_rpm_dependencies") {
 strip_binary("strip_chrome_binary") {
   binary_input = "$root_out_dir/chrome"
   deps = [ "//chrome" ]
+  binary_input = "$root_out_dir/brave" symbol_output = "$root_out_dir/brave.debug" stripped_binary_output = "$root_out_dir/brave.stripped" deps += strip_chrome_binary_deps
 }
 
 strip_binary("strip_chrome_crashpad_handler") {
@@ -236,6 +239,7 @@ strip_binary("strip_chrome_management_service") {
 strip_binary("strip_chrome_sandbox") {
   binary_input = "$root_out_dir/chrome_sandbox"
   deps = [ "//sandbox/linux:chrome_sandbox" ]
+  binary_input = "$root_out_dir/brave_sandbox" symbol_output = "$root_out_dir/brave_sandbox.debug" stripped_binary_output = "$root_out_dir/brave_sandbox.stripped" deps += strip_chrome_sandbox_deps
 }
 
 # On ChromeOS, system services depend on real ANGLE libraries rather than dummy
@@ -389,6 +393,7 @@ copy("common_packaging_files") {
   }
 
   outputs = [ "$root_out_dir/installer/common/{{source_file_part}}" ]
+  sources -= common_packaging_files_excluded_sources sources += common_packaging_files_sources
 }
 
 copy("deb_packaging_files") {
@@ -509,6 +514,7 @@ group("installer_deps") {
     "//components/privacy_sandbox/privacy_sandbox_attestations/preload:component",
     "//sandbox/linux:chrome_sandbox",
   ]
+  public_deps += brave_installer_deps
   if (bundle_widevine_cdm) {
     public_deps += [ "//third_party/widevine/cdm" ]
   }
@@ -579,11 +585,14 @@ group("installer_deps") {
 template("linux_package") {
   assert(defined(invoker.channel))
   channel = invoker.channel
+  if (channel == "unstable" ) {
+    channel = "dev"
+  }
 
   if (is_chrome_branded) {
     package = "google-chrome"
   } else {
-    package = "chromium-browser"
+    package = brave_linux_package_name chrome_version_full = brave_version
   }
 
   deb_target_name = "${target_name}_deb"
@@ -592,7 +601,7 @@ template("linux_package") {
     script = "debian/build.py"
 
     inputs = packaging_files
-    outputs = [ "$root_out_dir/${package}-${channel}_${chrome_version_full}-1_${deb_arch}.deb" ]
+    if (channel == "stable") { outputs = [ "$root_out_dir/${package}_${chrome_version_full}_${deb_arch}.deb" ] } else { outputs = [ "$root_out_dir/${package}-${channel}_${chrome_version_full}_${deb_arch}.deb" ] }
 
     args = [
       "-a",
@@ -629,7 +638,7 @@ template("linux_package") {
       script = "rpm/build.py"
 
       inputs = packaging_files
-      outputs = [ "$root_out_dir/${package}-${channel}-${chrome_version_full}-1.${rpm_arch}.rpm" ]
+      if (channel == "stable") { outputs = [ "$root_out_dir/${package}-${chrome_version_full}-1.${rpm_arch}.rpm" ] } else { outputs = [ "$root_out_dir/${package}-${channel}-${chrome_version_full}-1.${rpm_arch}.rpm" ] }
 
       args = [
         "-a",
@@ -711,6 +720,9 @@ linux_package("unstable") {
 linux_package("canary") {
   channel = "canary"
 }
+linux_package("nightly") {
+  channel = "nightly"
+}
 
 # For convenience, alias 'dev' to 'unstable'.
 group("dev") {
