diff --git a/chrome/browser/resources/optimize_webui.py b/chrome/browser/resources/optimize_webui.py
index e7804916b41259968fb33a8499c64cd522a5db05..5723d1a552531f0cc18d9d17657bada27f9ab2ca 100755
--- a/chrome/browser/resources/optimize_webui.py
+++ b/chrome/browser/resources/optimize_webui.py
@@ -45,6 +45,8 @@ _POLYMER_PATH = os.path.join(
     _SRC_PATH, 'third_party', 'polymer', 'v1_0', 'components-chromium')
 
 
+_BR_RESOURCES_PATH = os.path.join(_SRC_PATH, 'brave', 'ui', 'webui', 'resources')
+
 _VULCANIZE_BASE_ARGS = [
   # These files are already combined and minified.
   '--exclude', 'chrome://resources/html/polymer.html',
@@ -60,6 +62,8 @@ _VULCANIZE_BASE_ARGS = [
   '--exclude', 'chrome://resources/css/text_defaults.css',
   '--exclude', 'chrome://resources/css/text_defaults_md.css',
   '--exclude', 'chrome://resources/js/load_time_data.js',
+  '--exclude', 'chrome://brave-resources/fonts/muli.css',
+  '--exclude', 'chrome://brave-resources/fonts/poppins.css',
 
   '--inline-css',
   '--inline-scripts',
@@ -74,7 +78,8 @@ _URL_MAPPINGS = [
     ('chrome://resources/css/', _CSS_RESOURCES_PATH),
     ('chrome://resources/html/', _HTML_RESOURCES_PATH),
     ('chrome://resources/js/', _JS_RESOURCES_PATH),
-    ('chrome://resources/polymer/v1_0/', _POLYMER_PATH)
+    ('chrome://resources/polymer/v1_0/', _POLYMER_PATH),
+    ('chrome://brave-resources/', _BR_RESOURCES_PATH)
 ]
 
 
