diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
index d4f7def47a8241904509fac508a1b502842eeb1b..4a8955b18dfa822b33bff3cc08f8a9ac9b179b34 100644
--- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
+++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
@@ -334,10 +334,12 @@ public class ToolbarTablet extends ToolbarLayout
             mOfflineDownloader.downloadPage(getContext(), getToolbarDataProvider().getTab());
             RecordUserAction.record("MobileToolbarDownloadPage");
         }
+        BraveToolbarLayout.class.cast(this).onClickImpl(v);
     }
 
     @Override
     public boolean onLongClick(View v) {
+        if (BraveToolbarLayout.class.cast(this).onLongClickImpl(v)) return true;
         String description = null;
         Context context = getContext();
         Resources resources = context.getResources();
@@ -403,6 +405,7 @@ public class ToolbarTablet extends ToolbarLayout
         setToolbarHairlineColor(color);
         final boolean useLight = ColorUtils.shouldUseLightForegroundOnBackground(color);
         mAccessibilitySwitcherButton.setUseLightDrawables(useLight);
+        super.onThemeColorChanged(color, shouldAnimate);
     }
 
     /**
