diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index fe62a13434fc1bc3b33ccabcb8ff52ce0ba4e376..3a7c73b7a882425a2e725e352c9b6018b84220ae 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -89,6 +89,7 @@ source_set("channel_info") {
   } else if (is_fuchsia) {
     sources += [ "channel_info_fuchsia.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -541,6 +542,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
