diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
index 0d2b278f17125768d476ad0935ee034f86f3a658..5d1ab4dc75ef860c38bc05b6f70a88e22137b4d1 100644
--- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
+++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
@@ -282,8 +282,8 @@ public class ToolbarPhone extends ToolbarLayout
     private @Nullable PhoneCaptureStateToken mPhoneCaptureStateToken;
     private @Nullable ButtonData mButtonData;
 
-    private final @ColorInt int mToolbarBackgroundColorForNtp;
-    private final @ColorInt int mLocationBarBackgroundColorForNtp;
+    @ColorInt int mToolbarBackgroundColorForNtp;
+    @ColorInt int mLocationBarBackgroundColorForNtp;
     private boolean mUseAdjustedTintColorForNtp;
     private boolean mIsToolbarExpandedOnNtp;
 
@@ -573,6 +573,7 @@ public class ToolbarPhone extends ToolbarLayout
 
     /** Set the background color of the location bar to appropriately match the theme color. */
     private void updateModernLocationBarColor(@ColorInt int color) {
+        BraveToolbarLayout.class.cast(this).updateModernLocationBarColorImpl(color);
         if (mCurrentLocationBarColor == color) return;
         mCurrentLocationBarColor = color;
         mLocationBarBackground.setBackgroundColor(color);
@@ -1288,6 +1289,7 @@ public class ToolbarPhone extends ToolbarLayout
      * Tab Page.
      */
     private void updateLocationBarLayoutForExpansionAnimation() {
+        if (!BraveToolbarLayout.class.cast(this).isLocationBarValid(mLocationBar)) return;
         TraceEvent.begin("ToolbarPhone.updateLocationBarLayoutForExpansionAnimation");
         if (isInTabSwitcherMode()) return;
 
@@ -1704,6 +1706,7 @@ public class ToolbarPhone extends ToolbarLayout
             }
         }
 
+        BraveToolbarLayout.class.cast(this).drawAnimationOverlay(mToolbarButtonsContainer, canvas);
         // Draw the tab stack button and associated text if necessary.
         ToggleTabStackButtonCoordinator tabSwitcherButtonCoordinator =
                 getTabSwitcherButtonCoordinator();
@@ -2096,7 +2099,7 @@ public class ToolbarPhone extends ToolbarLayout
 
     @Override
     public void onHomeButtonIsEnabledUpdate(boolean homeButtonEnabled) {
-        mIsHomeButtonEnabled = homeButtonEnabled;
+        mIsHomeButtonEnabled = false;
         updateButtonVisibility();
     }
 
@@ -2502,6 +2505,7 @@ public class ToolbarPhone extends ToolbarLayout
     }
 
     private void createAndRunFocusAnimatorLegacy(boolean hasFocus, boolean shouldShowKeyboard) {
+        if (!BraveToolbarLayout.class.cast(this).isLocationBarValid(mLocationBar)) return;
         List<Animator> animators = new ArrayList<>();
         if (hasFocus) {
             populateUrlExpansionAnimatorSet(animators);
@@ -3665,6 +3669,7 @@ public class ToolbarPhone extends ToolbarLayout
      * @return The offset for the location bar when showing the dse icon.
      */
     int getLocationBarOffsetForFocusAnimation(boolean hasFocus) {
+        if (!urlHasFocus()) return 0;
         StatusCoordinator statusCoordinator = mLocationBar.getStatusCoordinator();
         if (statusCoordinator == null) return 0;
 
