diff --git a/chrome/browser/ui/color/BUILD.gn b/chrome/browser/ui/color/BUILD.gn
index ac646347006a07c4f019e721b95ff924a7077e00..1dfdb521509039cf26f204621cfeeaa4cc4bb749 100644
--- a/chrome/browser/ui/color/BUILD.gn
+++ b/chrome/browser/ui/color/BUILD.gn
@@ -46,6 +46,7 @@ source_set("mixers") {
     "tab_strip_color_mixer.cc",
     "tab_strip_color_mixer.h",
   ]
+  sources += brave_browser_ui_color_mixers_sources
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
 
@@ -64,6 +65,7 @@ source_set("mixers") {
     "//ui/color",
     "//ui/color:mixers",
   ]
+  deps += brave_browser_ui_color_mixers_deps
 
   if (is_chromeos) {
     sources += [ "chromeos/native_chrome_color_mixer_chromeos.cc" ]
