diff --git a/chrome/browser/ui/views/page_action/page_action_controller.h b/chrome/browser/ui/views/page_action/page_action_controller.h
index 730b49f9a3e8a9d10c3ba1357a0d8d1172e170d5..b26e564898ec31ece4ae55059cb1296ce6f3af67 100644
--- a/chrome/browser/ui/views/page_action/page_action_controller.h
+++ b/chrome/browser/ui/views/page_action/page_action_controller.h
@@ -144,6 +144,27 @@ class PageActionController {
  public:
   virtual ~PageActionController() = default;
 
+  // Force the label to be shown regardless of the available space
+  virtual void SetAlwaysShowLabel(actions::ActionId action_id,
+                                  bool always_show) = 0;
+
+  // Override the colors of the chip
+  virtual void OverrideChipColors(
+      actions::ActionId action_id,
+      std::optional<SkColor> override_background_color,
+      std::optional<SkColor> override_foreground_color) = 0;
+  virtual void ClearOverrideChipColors(actions::ActionId action_id) = 0;
+
+  // Override the height of the PageActionView
+  virtual void SetOverrideHeight(actions::ActionId action_id,
+                                 int height) = 0;
+  virtual void ClearOverrideHeight(actions::ActionId action_id) = 0;
+
+  // When set, `PageActionView::IsTriggerableEvent` uses this callback instead of
+  // the default logic. Pass nullopt to clear.
+  virtual void SetOverrideTriggerableEvent(actions::ActionId action_id, 
+                                           std::optional<int> event_flags) = 0;
+
   // Requests that the page action be shown or hidden.
   virtual void Show(actions::ActionId action_id) = 0;
   virtual void Hide(actions::ActionId action_id) = 0;
@@ -286,6 +307,18 @@ class PageActionControllerImpl : public PageActionController,
   PageActionControllerImpl& operator=(const PageActionControllerImpl&) = delete;
   ~PageActionControllerImpl() override;
 
+  void SetAlwaysShowLabel(actions::ActionId action_id,
+                          bool always_show) override;
+  void OverrideChipColors(actions::ActionId action_id,
+      std::optional<SkColor> override_background_color,
+      std::optional<SkColor> override_foreground_color) override;
+  void ClearOverrideChipColors(actions::ActionId action_id) override;
+  void SetOverrideHeight(actions::ActionId action_id,
+                         int height) override;
+  void ClearOverrideHeight(actions::ActionId action_id) override;
+  void SetOverrideTriggerableEvent(actions::ActionId action_id,
+                                   std::optional<int> event_flags) override;
+
   void Initialize(
       tabs::TabInterface& tab_interface,
       const std::vector<actions::ActionId>& action_ids,
