diff --git a/chrome/browser/ui/actions/chrome_action_id.h b/chrome/browser/ui/actions/chrome_action_id.h
index fb916f99899f5fdd7a429539b755e688fee62955..a4c28dfeef9db43f53c171d1db1e3743ccaaa9e4 100644
--- a/chrome/browser/ui/actions/chrome_action_id.h
+++ b/chrome/browser/ui/actions/chrome_action_id.h
@@ -485,6 +485,10 @@
 // Should be reinserted into CHROME_COMMON_ACTION_IDS when general solution to
 // add action id mappings is implemented.
 #define SIDE_PANEL_ACTION_IDS \
+  E(kActionSidePanelShowPlaylist) \
+  E(kActionSidePanelShowChatUI) \
+  E(kActionSidePanelShowBraveNews) \
+  E(kActionSidePanelShowWallet) \
   /* Side Panel items */ \
   E(kActionSidePanelShowAboutThisSite) \
   E(kActionSidePanelShowAssistant) \
@@ -514,6 +518,13 @@
     IDC_SHOW_TABS_FROM_OTHER_DEVICES_SIDE_PANEL)
 
 #define TOOLBAR_PINNABLE_ACTION_IDS \
+  E(kActionShowPartitionedStorage) \
+  E(kActionShowPsstIcon) \
+  E(kActionShowSpeedreader) \
+  E(kActionShowWaybackMachine) \
+  E(kActionShowPlaylistPageAction) \
+  E(kActionShowOnionLocation) \
+  E(kActionShowBraveNews) \
   E(kActionHome, IDC_HOME) \
   E(kActionForward, IDC_FORWARD) \
   E(kActionNewIncognitoWindow, IDC_NEW_INCOGNITO_WINDOW) \
