diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index b900cc8762846d0f44bacddfee207eb057494c4a..12f16a5d29e5ddf6da86268ea5ede2186ed698cf 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4955,6 +4955,7 @@ const FeatureEntry::FeatureVariation kSeamlessSigninPromoTypes[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
