diff --git a/chrome/browser/signin/account_consistency_mode_manager.cc b/chrome/browser/signin/account_consistency_mode_manager.cc
index 79f00f9b96b74b85075189745c943c5ca933154b..f5f0bb7a3db2848b67c3f6ce78e7659005acf0c0 100644
--- a/chrome/browser/signin/account_consistency_mode_manager.cc
+++ b/chrome/browser/signin/account_consistency_mode_manager.cc
@@ -192,6 +192,7 @@ AccountConsistencyModeManager::ComputeAccountConsistencyMethod(
     Profile* profile) {
   DCHECK(ShouldBuildServiceForProfile(profile));
 
+BRAVE_COMPUTE_ACCOUNT_CONSISTENCY_METHOD
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   if (!ash::IsAccountManagerAvailable(profile))
     return AccountConsistencyMethod::kDisabled;
