diff --git a/chrome/browser/profiles/profile_impl.h b/chrome/browser/profiles/profile_impl.h
index e8dfbf8c31223854fbe1bb640e61023ef82c0b90..c73c7acb3acc60492cbd7f87dd73f9fe64d07b2d 100644
--- a/chrome/browser/profiles/profile_impl.h
+++ b/chrome/browser/profiles/profile_impl.h
@@ -169,6 +169,7 @@ class ProfileImpl : public Profile {
   void OnLogin() override;
   void InitChromeOSPreferences() override;
 #endif  // defined(OS_CHROMEOS)
+  BRAVE_PROFILE_IMPL_H_
 
   void SetCreationTimeForTesting(base::Time creation_time) override;
 
