diff --git a/chrome/browser/flags/android/chrome_feature_list.cc b/chrome/browser/flags/android/chrome_feature_list.cc
index f3c99557cd1a650b60207eb149203925f0f7b835..454fe21c932a38c0040132a70574bbfde91d38d9 100644
--- a/chrome/browser/flags/android/chrome_feature_list.cc
+++ b/chrome/browser/flags/android/chrome_feature_list.cc
@@ -259,6 +259,7 @@ const base::Feature* kFeaturesExposedToJava[] = {
 };
 
 const base::Feature* FindFeatureExposedToJava(const std::string& feature_name) {
+  BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA
   for (const auto* feature : kFeaturesExposedToJava) {
     if (feature->name == feature_name)
       return feature;
