diff --git a/chrome/browser/policy/profile_policy_connector.cc b/chrome/browser/policy/profile_policy_connector.cc
index fec5d4ccd7cb71f7db38883b822464fcb3fd24d2..33532415b4cfef7538f496059dd36ecbbc1c570a 100644
--- a/chrome/browser/policy/profile_policy_connector.cc
+++ b/chrome/browser/policy/profile_policy_connector.cc
@@ -435,6 +435,7 @@ void ProfilePolicyConnector::Init(
   }
 #endif
 
+  BRAVE_PROFILE_POLICY_CONNECTOR_INIT
   std::vector<std::unique_ptr<PolicyMigrator>> migrators;
 #if BUILDFLAG(IS_WIN)
   migrators.push_back(
