diff --git a/chrome/browser/ui/cocoa/accelerators_cocoa.mm b/chrome/browser/ui/cocoa/accelerators_cocoa.mm
index 2bcdaecf985d76640ed4e996c7f0f08ff01fdd69..d349f07db552f342dc922a930289af4f6e29d675 100644
--- a/chrome/browser/ui/cocoa/accelerators_cocoa.mm
+++ b/chrome/browser/ui/cocoa/accelerators_cocoa.mm
@@ -41,6 +41,8 @@ const struct AcceleratorMapping {
      ui::VKEY_N},
     {IDC_NEW_TAB, ui::EF_COMMAND_DOWN, ui::VKEY_T},
     {IDC_NEW_WINDOW, ui::EF_COMMAND_DOWN, ui::VKEY_N},
+    {IDC_NEW_OFFTHERECORD_WINDOW_TOR, ui::EF_COMMAND_DOWN | ui::EF_ALT_DOWN,
+      ui::VKEY_N},
     {IDC_PASTE, ui::EF_COMMAND_DOWN, ui::VKEY_V},
     {IDC_PRINT, ui::EF_COMMAND_DOWN, ui::VKEY_P},
     {IDC_RESTORE_TAB, ui::EF_COMMAND_DOWN | ui::EF_SHIFT_DOWN, ui::VKEY_T},
