diff --git a/chrome/renderer/chrome_content_renderer_client.h b/chrome/renderer/chrome_content_renderer_client.h
index f28c95b801b5e475c1276d70b41604ebe79b9604..99bde6b9e9199f75fb245061ab3dcceea25f7631 100644
--- a/chrome/renderer/chrome_content_renderer_client.h
+++ b/chrome/renderer/chrome_content_renderer_client.h
@@ -237,6 +237,7 @@ class ChromeContentRendererClient
   base::WeakPtr<ChromeRenderThreadObserver> GetChromeObserver() const;
 
  private:
+  friend class BraveContentRendererClient;
   FRIEND_TEST_ALL_PREFIXES(ChromeContentRendererClientTest, NaClRestriction);
   FRIEND_TEST_ALL_PREFIXES(ChromeContentRendererClientTest,
                            ShouldSuppressErrorPage);
