diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index 6166ecb162314877315021621a86e114bf68ec68..53b2d05ea33797ce77d3709faff57ba558926d54 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -870,6 +870,7 @@ void BookmarkBarView::Layout(PassKey) {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   const bool show_bookmarks_overflow =
       bookmark_service_->loaded() &&
