diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 563383fc71d7bc73f0af0fd2130879d0844ba2af..abdda9b28dd5e4fe81a6a4ee504e7f8c0781e040 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/extensions/resources.gni")
 import("//build/config/chromebox_for_meetings/buildflags.gni")
 import("//build/config/features.gni")
 import("//build/config/ozone.gni")
@@ -646,6 +647,7 @@ source_set("extensions") {
     "//ui/native_window_tracker",
     "//ui/shell_dialogs",
   ]
+  deps += brave_extensions_resources
   public_deps = [
     "//chrome/browser/extensions/api/messaging",
     "//chrome/browser/media/webrtc",
@@ -2188,7 +2190,7 @@ source_set("unit_tests") {
       ]
     }
 
-    if (enable_service_discovery) {
+    if (enable_service_discovery && (!is_android || is_desktop_android)) {
       sources += [ "api/mdns/mdns_api_unittest.cc" ]
       deps += [ "//chrome/browser/extensions/api/mdns" ]
     }
@@ -2286,6 +2288,7 @@ source_set("unit_tests") {
 }
 
 # By default these tests run on Win, Mac, Linux, Chrome OS and desktop Android.
+if (!is_android || is_desktop_android) {
 source_set("browser_tests") {
   testonly = true
 
@@ -3186,6 +3189,7 @@ source_set("browser_tests") {
     deps += [ "//chrome/browser/ui/views/web_apps:launch_dialogs" ]
   }
 }
+}
 
 source_set("interactive_ui_tests") {
   testonly = true
