diff --git a/chrome/installer/linux/BUILD.gn b/chrome/installer/linux/BUILD.gn
index fbf86a5726a4be3c9c77f6b25e6aad24b296e3e5..1654e743d3e857668f01e121b211a15adb289c62 100644
--- a/chrome/installer/linux/BUILD.gn
+++ b/chrome/installer/linux/BUILD.gn
@@ -19,11 +19,10 @@ if (current_cpu == "x86" || current_cpu == "x64") {
 assert(is_linux)
 
 packaging_files_executables = [
-  "$root_out_dir/chrome",
-  "$root_out_dir/chrome_sandbox",
+  "$root_out_dir/brave",
+  "$root_out_dir/brave_sandbox",
 ]
 packaging_files_shlibs = []
-
 if (enable_nacl) {
   packaging_files_executables += [
     "$root_out_dir/nacl_helper",
@@ -155,7 +154,7 @@ action("merge_rpm_dependencies") {
 }
 
 action("strip_chrome_binary") {
-  prog_name = "$root_out_dir/chrome"
+  prog_name = "$root_out_dir/brave"
   debug_file = prog_name + ".debug"
   stripped_file = prog_name + ".stripped"
   deps = [
@@ -242,8 +241,8 @@ copy("common_packaging_files") {
     ]
   } else {
     sources += [
-      "common/chromium-browser/chromium-browser.appdata.xml",
-      "common/chromium-browser/chromium-browser.info",
+      "//brave/chromium_src/chrome/installer/linux/common/brave-browser/chromium-browser.appdata.xml",
+      "//brave/chromium_src/chrome/installer/linux/common/brave-browser/chromium-browser.info",
     ]
   }
 
