diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SearchIndexProviderRegistry.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SearchIndexProviderRegistry.java
index b9a515255ae779daf61e37b86cc1ec54e1215f21..6d8a5a6e116c0eb442be3e25add2ec6f3e159ffd 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SearchIndexProviderRegistry.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SearchIndexProviderRegistry.java
@@ -7,7 +7,6 @@ package org.chromium.chrome.browser.settings.search;
 import org.chromium.build.annotations.NullMarked;
 import org.chromium.chrome.browser.about_settings.AboutChromeSettings;
 import org.chromium.chrome.browser.about_settings.LegalInformationSettings;
-import org.chromium.chrome.browser.appearance.settings.AppearanceSettingsFragment;
 import org.chromium.chrome.browser.autofill.options.AutofillOptionsFragment;
 import org.chromium.chrome.browser.autofill.settings.AndroidPaymentAppsFragment;
 import org.chromium.chrome.browser.autofill.settings.AutofillBuyNowPayLaterFragment;
@@ -26,12 +25,10 @@ import org.chromium.chrome.browser.download.settings.DownloadSettings;
 import org.chromium.chrome.browser.glic.GlicSettings;
 import org.chromium.chrome.browser.homepage.settings.HomepageSettings;
 import org.chromium.chrome.browser.image_descriptions.ImageDescriptionsSettings;
-import org.chromium.chrome.browser.language.settings.LanguageSettings;
 import org.chromium.chrome.browser.night_mode.settings.ThemeSettingsFragment;
 import org.chromium.chrome.browser.prefetch.settings.PreloadPagesSettingsFragment;
 import org.chromium.chrome.browser.privacy.secure_dns.SecureDnsSettings;
 import org.chromium.chrome.browser.privacy.settings.DoNotTrackSettings;
-import org.chromium.chrome.browser.privacy.settings.PrivacySettings;
 import org.chromium.chrome.browser.privacy_sandbox.AdMeasurementFragment;
 import org.chromium.chrome.browser.privacy_sandbox.FledgeAllSitesFragment;
 import org.chromium.chrome.browser.privacy_sandbox.FledgeBlockedSitesFragment;
@@ -47,7 +44,7 @@ import org.chromium.chrome.browser.safety_check.SafetyCheckSettingsFragment;
 import org.chromium.chrome.browser.safety_hub.SafetyHubFragment;
 import org.chromium.chrome.browser.search_engines.settings.SearchEngineSettings;
 import org.chromium.chrome.browser.search_engines.settings.SiteSearchSettings;
-import org.chromium.chrome.browser.settings.MainSettings;
+import org.chromium.chrome.browser.settings.BraveMainPreferencesBase;
 import org.chromium.chrome.browser.ssl.HttpsFirstModeSettingsFragment;
 import org.chromium.chrome.browser.sync.settings.AccountManagementFragment;
 import org.chromium.chrome.browser.sync.settings.GoogleServicesSettings;
@@ -70,6 +67,31 @@ import org.chromium.components.browser_ui.site_settings.SingleWebsiteSettings;
 import org.chromium.components.browser_ui.site_settings.SiteSettings;
 import org.chromium.components.page_info.PageInfoAdPersonalizationSettings;
 import org.chromium.components.page_info.PageInfoCookiesSettings;
+import org.chromium.brave.browser.custom_search_engines.settings.AddCustomSearchEnginePreferenceFragment;
+import org.chromium.brave.browser.customize_menu.settings.BraveCustomizeMenuPreferenceFragment;
+import org.chromium.brave.browser.quick_search_engines.settings.QuickSearchEnginesFragment;
+import org.chromium.chrome.browser.language.settings.BraveLanguageSettings;
+import org.chromium.chrome.browser.password_entry_edit.BlockedCredentialFragmentView;
+import org.chromium.chrome.browser.password_entry_edit.CredentialEditFragmentView;
+import org.chromium.chrome.browser.password_entry_edit.FederatedCredentialFragmentView;
+import org.chromium.chrome.browser.password_manager.settings.PasswordSettings;
+import org.chromium.chrome.browser.playlist.settings.BravePlaylistPreferences;
+import org.chromium.chrome.browser.privacy.settings.BravePrivacySettings;
+import org.chromium.chrome.browser.settings.AppearancePreferences;
+import org.chromium.chrome.browser.settings.BraveLeoDefaultModelPreferences;
+import org.chromium.chrome.browser.settings.BraveLeoPreferences;
+import org.chromium.chrome.browser.settings.BraveLicensePreferences;
+import org.chromium.chrome.browser.settings.BraveOriginPreferences;
+import org.chromium.chrome.browser.settings.BraveSearchEnginesPreferences;
+import org.chromium.chrome.browser.settings.BraveSyncScreensPreference;
+import org.chromium.chrome.browser.settings.BraveWalletNetworksPreferenceFragment;
+import org.chromium.chrome.browser.settings.BraveWalletPreferences;
+import org.chromium.chrome.browser.settings.BraveWebrtcPolicyPreferencesFragment;
+import org.chromium.chrome.browser.settings.developer.BraveQAPreferences;
+import org.chromium.chrome.browser.site_settings.BraveWalletEthereumConnectedSites;
+import org.chromium.chrome.browser.site_settings.BraveWalletSolanaConnectedSites;
+import org.chromium.chrome.browser.vpn.settings.BraveVpnPreferences;
+import org.chromium.components.browser_ui.site_settings.BraveSiteSettingsPreferencesBase;
 
 import java.util.List;
 
@@ -90,7 +112,7 @@ public final class SearchIndexProviderRegistry {
                     // MainSettings always comes at the top. This fixes the order of the header
                     // groups in which they will be shown in the search results since the index
                     // uses LinkedHashMap internally.
-                    MainSettings.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveMainPreferencesBase.SEARCH_INDEX_DATA_PROVIDER,
                     AboutChromeSettings.SEARCH_INDEX_DATA_PROVIDER,
                     AdaptiveToolbarSettingsFragment.SEARCH_INDEX_DATA_PROVIDER,
                     AutofillOptionsFragment.SEARCH_INDEX_DATA_PROVIDER,
@@ -129,7 +151,6 @@ public final class SearchIndexProviderRegistry {
                     SafetyHubFragment.SEARCH_INDEX_DATA_PROVIDER,
                     AccountManagementFragment.SEARCH_INDEX_DATA_PROVIDER,
                     AndroidPaymentAppsFragment.SEARCH_INDEX_DATA_PROVIDER,
-                    AppearanceSettingsFragment.SEARCH_INDEX_DATA_PROVIDER,
                     AutofillBuyNowPayLaterFragment.SEARCH_INDEX_DATA_PROVIDER,
                     AutofillCardBenefitsFragment.SEARCH_INDEX_DATA_PROVIDER,
                     AutofillIdentityDocsFragment.SEARCH_INDEX_DATA_PROVIDER,
@@ -141,10 +162,8 @@ public final class SearchIndexProviderRegistry {
                     FinancialAccountsManagementFragment.SEARCH_INDEX_DATA_PROVIDER,
                     GoogleServicesSettings.SEARCH_INDEX_DATA_PROVIDER,
                     HttpsFirstModeSettingsFragment.SEARCH_INDEX_DATA_PROVIDER,
-                    LanguageSettings.SEARCH_INDEX_DATA_PROVIDER,
                     NonCardPaymentMethodsManagementFragment.SEARCH_INDEX_DATA_PROVIDER,
                     PriceNotificationSettingsFragment.SEARCH_INDEX_DATA_PROVIDER,
-                    PrivacySettings.SEARCH_INDEX_DATA_PROVIDER,
                     SafetyCheckSettingsFragment.SEARCH_INDEX_DATA_PROVIDER,
                     TabsSettings.SEARCH_INDEX_DATA_PROVIDER,
                     ManageSyncSettings.SEARCH_INDEX_DATA_PROVIDER,
@@ -153,5 +172,30 @@ public final class SearchIndexProviderRegistry {
                     DownloadSettings.SEARCH_INDEX_DATA_PROVIDER,
                     DeveloperSettings.SEARCH_INDEX_DATA_PROVIDER,
                     TracingSettings.SEARCH_INDEX_DATA_PROVIDER,
-                    TracingCategoriesSettings.SEARCH_INDEX_DATA_PROVIDER);
+                    TracingCategoriesSettings.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveSiteSettingsPreferencesBase.SEARCH_INDEX_DATA_PROVIDER,
+                    PasswordSettings.SEARCH_INDEX_DATA_PROVIDER,
+                    BravePlaylistPreferences.SEARCH_INDEX_DATA_PROVIDER,
+                    BravePrivacySettings.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveLeoDefaultModelPreferences.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveLeoPreferences.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveLicensePreferences.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveOriginPreferences.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveSearchEnginesPreferences.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveSyncScreensPreference.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveWalletNetworksPreferenceFragment.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveWalletPreferences.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveWebrtcPolicyPreferencesFragment.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveQAPreferences.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveWalletEthereumConnectedSites.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveWalletSolanaConnectedSites.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveVpnPreferences.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveCustomizeMenuPreferenceFragment.SEARCH_INDEX_DATA_PROVIDER,
+                    BlockedCredentialFragmentView.SEARCH_INDEX_DATA_PROVIDER,
+                    CredentialEditFragmentView.SEARCH_INDEX_DATA_PROVIDER,
+                    FederatedCredentialFragmentView.SEARCH_INDEX_DATA_PROVIDER,
+                    QuickSearchEnginesFragment.SEARCH_INDEX_DATA_PROVIDER,
+                    AddCustomSearchEnginePreferenceFragment.SEARCH_INDEX_DATA_PROVIDER,
+                    AppearancePreferences.SEARCH_INDEX_DATA_PROVIDER,
+                    BraveLanguageSettings.SEARCH_INDEX_DATA_PROVIDER);
 }
