diff --git a/chrome/browser/prefs/BUILD.gn b/chrome/browser/prefs/BUILD.gn
index 312c68b6e1b090cf452a61ff1b36f35a7a7aea7a..7c2c3d9d4e7b75a2fb430a02e601254626d8b6a4 100644
--- a/chrome/browser/prefs/BUILD.gn
+++ b/chrome/browser/prefs/BUILD.gn
@@ -194,6 +194,7 @@ source_set("impl") {
     "//services/preferences/tracked:features",
     "//ui/webui:buildflags",
   ]
+  import("//brave/browser/sources.gni") deps += brave_chrome_browser_prefs_impl_deps
 
   if (enable_print_preview) {
     deps += [ "//chrome/browser/printing" ]
