diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
index cfb5d65a9be8a11ff2c5bdeb35b097629063b99c..87258cc2c8d09cb26ae206eeac58b8a631a1dd99 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
@@ -59,7 +59,7 @@ import java.util.Collection;
  * The Toolbar object for Tablet screens.
  */
 @SuppressLint("Instantiatable")
-public class ToolbarTablet extends ToolbarLayout
+public class ToolbarTablet extends BraveToolbarLayout
         implements OnClickListener, View.OnLongClickListener, TabCountObserver {
     private HomeButton mHomeButton;
     private ImageButton mBackButton;
@@ -314,10 +314,12 @@ public class ToolbarTablet extends ToolbarLayout
             DownloadUtils.downloadOfflinePage(getContext(), getToolbarDataProvider().getTab());
             RecordUserAction.record("MobileToolbarDownloadPage");
         }
+        super.onClick(v);
     }
 
     @Override
     public boolean onLongClick(View v) {
+        if (super.onLongClick(v)) return true;
         String description = null;
         Context context = getContext();
         Resources resources = context.getResources();
@@ -383,6 +385,7 @@ public class ToolbarTablet extends ToolbarLayout
                 textBoxColor, PorterDuff.Mode.SRC_IN);
 
         mLocationBar.updateVisualsForState();
+        super.onThemeColorChanged(color, shouldAnimate);
     }
 
     /**
