diff --git a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
index 4a3c4f3cb08546bac4be19a177c704857ea26a75..169ae2c575c4de0ae53f3dc63c5c9c987901274e 100644
--- a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
+++ b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
@@ -132,6 +132,7 @@ content::WebUIDataSource* CreateBookmarksUIHTMLSource(Profile* profile) {
   // need this attribute.
   source->AddString("enableBrandingUpdateAttribute", "enable-branding-update");
 
+  BRAVE_CREATE_BOOKMARKS_UI_HTML_SOURCE
   return source;
 }
 
