diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index 337b8c13d19ca4ed3848057608f4a6de62dde82e..14dd9a6c8b348aedb0e482050d8c0c26e7bb84b3 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -263,6 +263,9 @@ cr.define('settings', function() {
 
     r.SEARCH = r.BASIC.createSection('/search', 'search');
     r.SEARCH_ENGINES = r.SEARCH.createChild('/searchEngines');
+    r.SHIELDS = r.BASIC.createSection('/shields', 'braveShieldsDefaults');
+    r.SOCIAL_BLOCKING = r.BASIC.createSection('/socialBlocking', 'socialBlocking');
+    r.BRAVE_SYNC = r.BASIC.createSection('/braveSync', 'braveSync');
     // <if expr="chromeos">
     if (loadTimeData.valueExists('assistantEnabled') &&
         loadTimeData.getBoolean('assistantEnabled')) {
@@ -341,9 +344,12 @@ cr.define('settings', function() {
 
       // TODO(tommycli): Find a way to refactor these repetitive category
       // routes.
+      // <if expr="_google_chrome">
       r.SITE_SETTINGS_ADS = r.SITE_SETTINGS.createChild('ads');
+      // </if>
       r.SITE_SETTINGS_AUTOMATIC_DOWNLOADS =
           r.SITE_SETTINGS.createChild('automaticDownloads');
+      r.SITE_SETTINGS_AUTOPLAY = r.SITE_SETTINGS.createChild('autoplay');
       r.SITE_SETTINGS_BACKGROUND_SYNC =
           r.SITE_SETTINGS.createChild('backgroundSync');
       r.SITE_SETTINGS_CAMERA = r.SITE_SETTINGS.createChild('camera');
@@ -401,7 +407,9 @@ cr.define('settings', function() {
       }
 
       r.PRINTING = r.ADVANCED.createSection('/printing', 'printing');
+      // <if expr="_google_chrome">
       r.CLOUD_PRINTERS = r.PRINTING.createChild('/cloudPrinters');
+      // </if>
       // <if expr="chromeos">
       r.CUPS_PRINTERS = r.PRINTING.createChild('/cupsPrinters');
       // </if>
