diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 16c831158d1ad46e68806c6f49cc5a213d0ddd9f..82492147c8bf05ae69e43abe649b2be0f9c1cfa2 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3132,6 +3132,7 @@ const FeatureEntry::FeatureVariation kDesktopSharePreviewVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
