diff --git a/chrome/browser/ui/safety_hub/password_status_check_service.cc b/chrome/browser/ui/safety_hub/password_status_check_service.cc
index 15d10e3e94e17c0de2574482be5d4cd785652d9e..0d31cc61fa38ab99cbcfa0a9c0ad2c6f6331b57f 100644
--- a/chrome/browser/ui/safety_hub/password_status_check_service.cc
+++ b/chrome/browser/ui/safety_hub/password_status_check_service.cc
@@ -719,6 +719,7 @@ base::TimeDelta PasswordStatusCheckService::GetScheduledPasswordCheckInterval()
 
 base::Value::Dict PasswordStatusCheckService::GetPasswordCardData(
     bool signed_in) {
+  BRAVE_PASSWORD_STATUS_CHECK_SERVICE_GET_PASSWORD_CARD_DATA
   if (no_passwords_saved()) {
     bool password_saving_allowed = profile_->GetPrefs()->GetBoolean(
         password_manager::prefs::kCredentialsEnableService);
