diff --git a/chrome/browser/extensions/install_verifier.cc b/chrome/browser/extensions/install_verifier.cc
index 8fd766ce2a3c6342db24822582a01f11e138df31..1292adc1cf814c4b288c90025e3889b9e33ad3df 100644
--- a/chrome/browser/extensions/install_verifier.cc
+++ b/chrome/browser/extensions/install_verifier.cc
@@ -15,6 +15,7 @@
 #include "base/metrics/histogram_macros.h"
 #include "base/stl_util.h"
 #include "base/trace_event/trace_event.h"
+#include "brave/common/extensions/extension_constants.h"
 #include "build/build_config.h"
 #include "chrome/browser/extensions/extension_management.h"
 #include "chrome/browser/extensions/extension_service.h"
@@ -213,6 +214,10 @@ bool InstallVerifier::NeedsVerification(const Extension& extension) {
 
 // static
 bool InstallVerifier::IsFromStore(const Extension& extension) {
+  // Do not fetch signatures for PDFJS which is hosted by Brave, not CWS
+  if (extension.id() == pdfjs_extension_id) {
+    return false;
+  }
   return extension.from_webstore() ||
          ManifestURL::UpdatesFromGallery(&extension);
 }
