diff --git a/chrome/browser/resources/settings/search_page/search_page.ts b/chrome/browser/resources/settings/search_page/search_page.ts
index a2e670959ce778a2f2a32e3f6b1313e6992fa121..b47c723b7002e690eb3624f0914bd183c11c0083 100644
--- a/chrome/browser/resources/settings/search_page/search_page.ts
+++ b/chrome/browser/resources/settings/search_page/search_page.ts
@@ -224,4 +224,3 @@ declare global {
   }
 }
 
-customElements.define(SettingsSearchPageElement.is, SettingsSearchPageElement);
