diff --git a/chrome/browser/chrome_content_browser_client.cc b/chrome/browser/chrome_content_browser_client.cc
index d9c675c8aca73a6a8411735a6c6a350795fa308f..4d084dc9987b0ed82ea3dfca996f472d0302b014 100644
--- a/chrome/browser/chrome_content_browser_client.cc
+++ b/chrome/browser/chrome_content_browser_client.cc
@@ -4922,7 +4922,7 @@ bool ChromeContentBrowserClient::PreSpawnChild(
     return false;
 
   // Allow loading Chrome's DLLs.
-  for (const auto* dll : {chrome::kBrowserResourcesDll, chrome::kElfDll}) {
+  for (const auto* dll : {chrome::kBrowserResourcesDll, chrome::kElfDll, L"ffmpeg.dll"}) {
     result = config->AllowExtraDlls(GetModulePath(dll).value().c_str());
     if (result != sandbox::SBOX_ALL_OK)
       return false;
@@ -4932,6 +4932,8 @@ bool ChromeContentBrowserClient::PreSpawnChild(
 }
 
 bool ChromeContentBrowserClient::IsRendererCodeIntegrityEnabled() {
+  // if ((true))
+  //   return false;
   // Emergency 'on switch' to re-enable the policy if force-disabling it causes
   // issues.
   if (base::FeatureList::IsEnabled(
