diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 5f34f77c126809a0368aa02592124f71ecc2d61f..8f7975347a82a0f3e853bf5a2215a698c6cee67f 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -33,6 +33,7 @@ class AvatarToolbarButton : public ToolbarButton,
   void UpdateText();
 
  private:
+  friend class BraveAvatarToolbarButton;
   enum class SyncState { kNormal, kPaused, kError };
 
   // ToolbarButton:
@@ -70,6 +71,7 @@ class AvatarToolbarButton : public ToolbarButton,
   bool IsIncognito() const;
   bool ShouldShowGenericIcon() const;
   base::string16 GetAvatarTooltipText() const;
+  virtual
   gfx::ImageSkia GetAvatarIcon() const;
   gfx::Image GetIconImageFromProfile() const;
   SyncState GetSyncState() const;
