diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index ad10cdfe288378c6f1e4b8a13d7efc77fc22fe81..594a8e8592c8a53f55465e1c220a9cc81467317d 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -769,6 +769,7 @@ static_library("ui") {
     # gets componentized.
     "//chrome/browser/ui/extensions:impl",
   ]
+  import("//brave/browser/sources.gni") public_deps += [ "//brave/browser/ui" ] deps += brave_chrome_browser_ui_allow_circular_includes_from allow_circular_includes_from += brave_chrome_browser_ui_allow_circular_includes_from
 
   if (enable_vr && is_win) {
     deps += [ "//chrome/browser/vr:vr_base" ]
@@ -3298,7 +3299,10 @@ static_library("ui") {
         "//chrome/updater/app/server/win:updater_legacy_idl",
       ]
     } else {
-      sources += [ "webui/help/version_updater_basic.cc" ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [ "webui/help/version_updater_basic.cc" ]
+      }
     }
   }
 
@@ -5390,6 +5394,7 @@ static_library("ui_public_dependencies") {
   if (enable_extensions_core) {
     public_deps += [ "//extensions/browser" ]
   }
+  public_deps += [ "//brave/browser/ui:ui_public_dependencies" ]
   if (!is_android) {
     public_deps += [
       "//build:branding_buildflags",
@@ -5444,6 +5449,7 @@ source_set("layout_constants") {
   if (is_chromeos) {
     deps += [ "//chromeos/constants" ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_ui_layout_constants_public_deps allow_circular_includes_from = brave_chrome_browser_ui_layout_constants_allow_circular_includes_from
 }
 
 source_set("url_identity") {
