diff --git a/chrome/browser/ui/views/tabs/common/tab_group_style.cc b/chrome/browser/ui/views/tabs/common/tab_group_style.cc
index 8fdffc5a7eff4b7d45c46e05494fb67020d55697..37537d303bee0095f7cc2a6d79af81f5aece9deb 100644
--- a/chrome/browser/ui/views/tabs/common/tab_group_style.cc
+++ b/chrome/browser/ui/views/tabs/common/tab_group_style.cc
@@ -30,6 +30,10 @@ int TabGroupStyle::GetTabGroupOverlapAdjustment() {
 
 // static
 int TabGroupStyle::GetChipCornerRadius(TabStripOrientation orientation) {
+  if (tabs::HorizontalTabsUpdateEnabled() &&
+      orientation == TabStripOrientation::kHorizontal) {
+    return tabs::kTabBorderRadius;
+  }
   return orientation == TabStripOrientation::kHorizontal
              ? kCornerRadius
              : kVerticalCornerRadius;
@@ -37,11 +41,18 @@ int TabGroupStyle::GetChipCornerRadius(TabStripOrientation orientation) {
 
 // static
 int TabGroupStyle::GetEmptyChipSize() {
+  if (tabs::HorizontalTabsUpdateEnabled()) {
+    return tabs::GetHorizontalTabHeight();
+  }
   return kEmptyChipSize;
 }
 
 // static
 gfx::Point TabGroupStyle::GetTitleChipOffset(std::optional<int> text_height) {
+  if (tabs::HorizontalTabsUpdateEnabled()) {
+    return gfx::Point(tabs::kHorizontalTabInset,
+                      tabs::GetHorizontalTabVerticalSpacing());
+  }
   const int total_space =
       GetLayoutConstant(LayoutConstant::kTabStripHeight) - GetEmptyChipSize() -
       GetLayoutConstant(LayoutConstant::kTabstripToolbarOverlap);
@@ -51,6 +62,11 @@ gfx::Point TabGroupStyle::GetTitleChipOffset(std::optional<int> text_height) {
 // static
 gfx::Insets TabGroupStyle::GetInsetsForHeaderChip(
     TabStripOrientation orientation) {
+  if (tabs::HorizontalTabsUpdateEnabled() &&
+      orientation == TabStripOrientation::kHorizontal) {
+    return gfx::Insets::VH(tabs::GetTabGroupTitleVerticalInset(),
+                           tabs::GetTabGroupTitleHorizontalInset());
+  }
   return orientation == TabStripOrientation::kHorizontal
              ? gfx::Insets::VH(kHeaderChipVerticalInset, kCornerRadius)
              : gfx::Insets::VH(0, kVerticalHeaderChipHorizontalInset);
