diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index fd1fac51fe13253f043b3f735c3a675c63617983..b6cd91a3fa4c9e2152dec339aa351d4e18789e9e 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -257,6 +257,7 @@ static_library("utility") {
   if (enable_on_device_translation) {
     deps += [ "//components/on_device_translation/service:on_device_translation_service" ]
   }
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
 }
 
 if (!is_android) {
