diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
index 4bcfb37d87bd610ac94d074523bb47ea1d8a0a7c..17f66c982f9a3005a9205d01ea58446e9e71c87e 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
@@ -85,6 +85,7 @@ class RecentTabsSubMenuModel : public ui::SimpleMenuModel,
   bool GetURLAndTitleForItemAtIndex(int index,
                                     std::string* url,
                                     std::u16string* title);
+  BRAVE_RECENT_TABS_SUB_MENU_MODEL_H_
 
  private:
   struct TabNavigationItem;
