diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.cc b/chrome/browser/ui/views/location_bar/location_bar_view.cc
index f05cedfa00c65e532ebbbac6bd960b62eaac1cfe..b2c848712826cf79a86712a3f514bc890fda5eb5 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
@@ -657,6 +657,7 @@ void LocationBarView::Layout() {
     }
   };
 
+  BRAVE_LAYOUT_TRAILING_DECORATIONS
   add_trailing_decoration(page_action_icon_container_);
   for (ContentSettingImageView* view : base::Reversed(content_setting_views_)) {
     add_trailing_decoration(view);
