diff --git a/chrome/browser/ui/accelerator_table.cc b/chrome/browser/ui/accelerator_table.cc
index 308ff4886478ec89cc77e02d789e3e31f91c332b..e9f945f652dd93a12a39c7f124b5039569162d19 100644
--- a/chrome/browser/ui/accelerator_table.cc
+++ b/chrome/browser/ui/accelerator_table.cc
@@ -314,7 +314,7 @@ std::vector<AcceleratorMapping>* GetAcceleratorsPointer() {
 
 }  // namespace
 
-std::vector<AcceleratorMapping> GetAcceleratorList() {
+std::vector<AcceleratorMapping> GetAcceleratorList_ChromiumImpl() {
   std::vector<AcceleratorMapping>* accelerators = GetAcceleratorsPointer();
 
   if (accelerators->empty()) {
