diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
index 4304cc82494f57b7bbc4d1f3dff0fab96e30aef3..5e9082000a990dfedd2d99c869cc1cc3e9d2a0c4 100644
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -183,6 +183,14 @@ if (!is_android && !is_mac) {
           "common/crash_keys.cc",
           "common/crash_keys.h",
         ]
+        if (brave_chromium_build) {
+          sources -= [
+            "app/chrome_exe.rc",
+          ]
+          sources += [
+            "//brave/app/brave_exe.rc",
+          ]
+        }
 
         deps += [
           ":chrome_dll",
@@ -246,6 +254,8 @@ if (!is_android && !is_mac) {
           "app/chrome_main.cc",
           "app/chrome_main_delegate.cc",
           "app/chrome_main_delegate.h",
+          "//brave/app/brave_main_delegate.cc",
+          "//brave/app/brave_main_delegate.h",
         ]
 
         deps += [
@@ -329,7 +339,7 @@ if (!is_android && !is_mac) {
     if (is_win) {
       output_name = "initialexe/chrome"
     } else {
-      output_name = "chrome"
+      output_name = "brave"
     }
 
     sources = []
@@ -378,6 +388,8 @@ if (is_win) {
       "app/chrome_main.cc",
       "app/chrome_main_delegate.cc",
       "app/chrome_main_delegate.h",
+      "//brave/app/brave_main_delegate.cc",
+      "//brave/app/brave_main_delegate.h",
     ]
 
     output_name = "chrome"
@@ -485,6 +497,8 @@ if (is_win) {
         "app/chrome_main.cc",
         "app/chrome_main_delegate.cc",
         "app/chrome_main_delegate.h",
+        "//brave/app/brave_main_delegate.cc",
+        "//brave/app/brave_main_delegate.h",
       ]
 
       configs += [ "//build/config/compiler:wexit_time_destructors" ]
@@ -605,6 +619,13 @@ if (is_win) {
     ]
   }
 
+if (brave_chromium_build) {
+  group("chrome_app") {
+    public_deps = [
+      "//brave:chrome_app",
+    ]
+  }
+} else {
   mac_app_bundle("chrome_app") {
     output_name = chrome_product_full_name
 
@@ -642,6 +663,7 @@ if (is_win) {
                   rebase_path("app/app.exports", root_build_dir) ]
     }
   }
+}
 
   compiled_action("chrome_app_strings") {
     tool = "//chrome/tools/build/mac:infoplist_strings_tool"
@@ -671,7 +693,7 @@ if (is_win) {
     args =
         [
           "-b",
-          "${branding_path_component}_strings",
+          _strings_file,
           "-v",
           rebase_path(chrome_version_file, root_build_dir),
           "-g",
@@ -720,6 +742,20 @@ if (is_win) {
       "app/theme/$branding_path_component/mac/document.icns",
       "browser/ui/cocoa/applescript/scripting.sdef",
     ]
+    if (brave_chromium_build) {
+      sources -= [
+        "app/theme/$branding_path_component/mac/app.icns",
+      ]
+      if (is_official_build) {
+        sources += [
+          "app/theme/$branding_path_component/mac/$brave_channel/app.icns",
+        ]
+      } else {
+        sources += [
+          "app/theme/$branding_path_component/mac/development/app.icns",
+        ]
+      }
+    }
     outputs = [
       "{{bundle_resources_dir}}/{{source_file_part}}",
     ]
@@ -1212,6 +1248,8 @@ if (is_win) {
       "app/chrome_main.cc",
       "app/chrome_main_delegate.cc",
       "app/chrome_main_delegate.h",
+      "//brave/app/brave_main_delegate.cc",
+      "//brave/app/brave_main_delegate.h",
       "app/chrome_main_mac.h",
       "app/chrome_main_mac.mm",
     ]
@@ -1295,6 +1333,17 @@ if (is_win) {
     if (is_chrome_branded) {
       deps += [ ":default_apps" ]
     }
+    if (brave_chromium_build) {
+      deps += [
+        "//brave/vendor/brave-extension:brave_extension_framework_bundle_data",
+      ]
+
+      if (defined(chrome_framework_version)) {
+        framework_contents += [
+          "Frameworks", # for Sparkle.framework
+        ]
+      }
+    }
 
     ldflags = [
       "-Wl,-install_name,@executable_path/../Versions/$chrome_version_full/$chrome_framework_name.framework/$chrome_framework_name",
@@ -1452,6 +1501,7 @@ if (is_win) {
 
 group("browser_dependencies") {
   public_deps = [
+    "//brave:browser_dependencies",
     "//chrome/browser",
     "//chrome/common",
     "//components/sync",
@@ -1488,6 +1538,7 @@ group("browser_dependencies") {
 
 group("child_dependencies") {
   public_deps = [
+    "//brave:child_dependencies",
     "//chrome/browser/devtools",
     "//chrome/child",
     "//chrome/common",
@@ -1509,7 +1560,7 @@ group("child_dependencies") {
 if (is_win) {
   process_version_rc_template("chrome_exe_version") {
     sources = [
-      "app/chrome_exe.ver",
+      "//brave/app/brave_exe.ver",
     ]
     output = "$target_gen_dir/chrome_exe_version.rc"
   }
@@ -1806,6 +1857,8 @@ if (is_android) {
       "app/android/chrome_main_delegate_android.h",
       "app/chrome_main_delegate.cc",
       "app/chrome_main_delegate.h",
+      "//brave/app/brave_main_delegate.cc",
+      "//brave/app/brave_main_delegate.h",
     ]
 
     include_dirs = [ android_ndk_include_dir ]
@@ -1854,7 +1907,7 @@ if (enable_resource_whitelist_generation) {
 if (is_linux) {
   if (is_official_build) {
     extract_symbols("linux") {
-      binary = "$root_out_dir/chrome"
+      binary = "$root_out_dir/brave"
 
       if (current_cpu == "x86") {
         # GYP used "ia32" so keep that naming for back-compat.
