diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index d92de2ce4ac878822ff36ab3df8f0f34bf6d3ee0..29323884749fdf2ecb825a0f1cb4c970147bfb75 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -216,6 +216,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  import("//brave/renderer/sources.gni") sources += brave_chrome_renderer_sources deps += brave_chrome_renderer_deps public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
