diff --git a/chrome/browser/ui/cocoa/tabs/tab_view.mm b/chrome/browser/ui/cocoa/tabs/tab_view.mm
index fc0c94001c243aba3af8ebf23174b08e80f143f5..f0c6fdfbf8bbff42c144cab6648e94fcea012c12 100644
--- a/chrome/browser/ui/cocoa/tabs/tab_view.mm
+++ b/chrome/browser/ui/cocoa/tabs/tab_view.mm
@@ -913,10 +913,11 @@ ui::ThreePartImage& GetStrokeImage(bool active, StrokeType stroke_type) {
              controlPoint1:NSMakePoint(1.805, -0.38)
              controlPoint2:NSMakePoint(2.17, 1.415)];
 
-  [bezierPath lineToPoint:NSMakePoint(14, 27)];
-  [bezierPath curveToPoint:NSMakePoint(16, 29)
-             controlPoint1:NSMakePoint(14.25, 27.25)
-             controlPoint2:NSMakePoint(14.747467, 29.118899)];
+  [bezierPath lineToPoint:NSMakePoint(2.5, 27)];
+
+  [bezierPath curveToPoint:NSMakePoint(4.5, 29)
+             controlPoint1:NSMakePoint(2.25, 27.25)
+             controlPoint2:NSMakePoint(2.747467, 29.118899)];
 
   [bezierPath lineToPoint:NSMakePoint(18, 29)];
 
