diff --git a/chrome/browser/ui/cocoa/tabs/tab_controller.mm b/chrome/browser/ui/cocoa/tabs/tab_controller.mm
index d83f7e40819805e87dcd6cd376407d6f0a285edf..ff72af7295aaaf08acc8f0294fd59923194136bf 100644
--- a/chrome/browser/ui/cocoa/tabs/tab_controller.mm
+++ b/chrome/browser/ui/cocoa/tabs/tab_controller.mm
@@ -101,8 +101,8 @@ @implementation TabController
 @synthesize url = url_;
 
 namespace {
-static const CGFloat kTabLeadingPadding = 18;
-static const CGFloat kTabTrailingPadding = 15;
+static const CGFloat kTabLeadingPadding = 9;
+static const CGFloat kTabTrailingPadding = 7;
 static const CGFloat kCloseButtonSize = 16;
 static const CGFloat kInitialTabWidth = 160;
 static const CGFloat kTitleLeadingPadding = 4;
