diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/TabSwitcherModeTTPhone.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/TabSwitcherModeTTPhone.java
index 2d5c7f94389664cdfad0e4764477043f0735bc3e..43fce29710826b322b18691dfdca43e4a5e54241 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/TabSwitcherModeTTPhone.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/top/TabSwitcherModeTTPhone.java
@@ -291,6 +291,7 @@ public class TabSwitcherModeTTPhone extends OptimizedFrameLayout
         if (BottomToolbarVariationManager.isTabSwitcherOnBottom() && !shouldShowIncognitoToggle()) {
             mToggleTabStackButton.setVisibility(isVisible ? GONE : VISIBLE);
         }
+        setMenuButtonVisibility(!isVisible || (FeatureUtilities.isBottomToolbarEnabled() && !BottomToolbarVariationManager.isMenuButtonOnBottom()));
     }
 
     private void setNewTabButtonVisibility(boolean isButtonVisible) {
