diff --git a/chrome/renderer/chrome_content_renderer_client.h b/chrome/renderer/chrome_content_renderer_client.h
index d306846f872ee27a07a67f80fc4dd8740458e47c..c811bd8de9c861b466ceea8f00815ae8c8507c08 100644
--- a/chrome/renderer/chrome_content_renderer_client.h
+++ b/chrome/renderer/chrome_content_renderer_client.h
@@ -241,6 +241,7 @@ class ChromeContentRendererClient
   ChromeRenderThreadObserver* GetChromeObserver() const;
 
  private:
+  friend class BraveContentRendererClient;
   FRIEND_TEST_ALL_PREFIXES(ChromeContentRendererClientTest, NaClRestriction);
   FRIEND_TEST_ALL_PREFIXES(ChromeContentRendererClientTest,
                            ShouldSuppressErrorPage);
