diff --git a/chrome/android/java_sources.gni b/chrome/android/java_sources.gni
index 0268b36c2351ea9b341d51c1e4dae1dade430e87..922d337e90f3ace8008f25a00ea18c862d8adf45 100644
--- a/chrome/android/java_sources.gni
+++ b/chrome/android/java_sources.gni
@@ -26,6 +26,7 @@ import("//components/omnibox/browser/test_java_sources.gni")
 import("//device/vr/buildflags/buildflags.gni")
 
 chrome_java_sources += public_autofill_assistant_java_sources
+chrome_java_sources += brave_java_sources
 
 if (enable_vr) {
   chrome_java_sources += [ "java/src/org/chromium/chrome/browser/component_updater/VrAssetsComponentInstaller.java" ]
