diff --git a/chrome/browser/resources/settings/site_settings/category_default_setting.js b/chrome/browser/resources/settings/site_settings/category_default_setting.js
index 8eef615b90f9dd9af4d724fb516a174662765a2c..4ed66d78d6c43ae4e30f29655f2fbf9c6028a4e8 100644
--- a/chrome/browser/resources/settings/site_settings/category_default_setting.js
+++ b/chrome/browser/resources/settings/site_settings/category_default_setting.js
@@ -178,6 +178,7 @@ class CategoryDefaultSettingElement extends CategoryDefaultSettingElementBase {
      */
     switch (this.category) {
       case ContentSettingsTypes.ADS:
+      case ContentSettingsTypes.AUTOPLAY:
       case ContentSettingsTypes.BACKGROUND_SYNC:
       case ContentSettingsTypes.IMAGES:
       case ContentSettingsTypes.JAVASCRIPT:
