diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java
index 10d85063b9352f883757b2c275348f03080e6079..abad2c3980cd9ac8bec8f3ee39204714a7cbb397 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java
@@ -96,7 +96,7 @@ public class BottomControlsCoordinator {
         ViewGroup.LayoutParams params = container.getLayoutParams();
         params.height = root.getResources().getDimensionPixelOffset(bottomControlsHeightId);
         mMediator =
-                new BottomControlsMediator(windowAndroid, model, controlsSizer, fullscreenManager,
+                new BraveBottomControlsMediator(windowAndroid, model, controlsSizer, fullscreenManager,
                         root.getResources().getDimensionPixelOffset(bottomControlsHeightId),
                         overlayPanelVisibilitySupplier);
 
