diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 2bc66d968cfe9ea1c2b4c35a36e4fc85227042cb..a6e3b1c7a62e092623470876d1a56f30dc0e2f52 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -1750,6 +1750,7 @@ static_library("browser") {
     "//chrome/browser/page_load_metrics:impl",
     "//chrome/browser/ui/webauthn:impl",
   ]
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_android) {
     deps += [
@@ -5422,6 +5423,7 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
