diff --git a/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc b/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
index 2424a3a1332dd3f1e1673609d114cdc2c67bc673..ef0dff855dadddd159617a145f17a58cf8735035 100644
--- a/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
+++ b/chrome/browser/safe_browsing/download_protection/check_client_download_request_base.cc
@@ -258,6 +258,7 @@ void CheckClientDownloadRequestBase::OnRequestBuilt(
     std::unique_ptr<ClientDownloadRequest> request) {
   client_download_request_ = std::move(request);
   SanitizeRequest();
+  BRAVE_ON_REQUEST_BUILT_FILTER
 
   // If it's an archive with no archives or executables, finish early.
   if ((client_download_request_->download_type() ==
