diff --git a/chrome/browser/resources/settings/site_settings/site_settings_prefs_browser_proxy.ts b/chrome/browser/resources/settings/site_settings/site_settings_prefs_browser_proxy.ts
index 3bfc16c896f4b77370c8f6dd59d455d8ff3a99b0..ef44d9902b9d19237f81216ff9551ccb1519749d 100644
--- a/chrome/browser/resources/settings/site_settings/site_settings_prefs_browser_proxy.ts
+++ b/chrome/browser/resources/settings/site_settings/site_settings_prefs_browser_proxy.ts
@@ -83,6 +83,7 @@ export interface RawSiteException {
   description?: string;
   setting: ContentSetting;
   source: SiteSettingSource;
+  braveCookieType?: undefined | "shields down" | "shields settings" | "google sign-in";  // See BraveGetExceptionForPage (site_settings_helper.cc)
 }
 
 /**
@@ -100,6 +101,7 @@ export interface SiteException {
   description?: string;
   enforcement: chrome.settingsPrivate.Enforcement|null;
   controlledBy: chrome.settingsPrivate.ControlledBy;
+  braveCookieType?: RawSiteException['braveCookieType'];
 }
 
 /**
