diff --git a/chrome/app_shim/app_shim_controller.mm b/chrome/app_shim/app_shim_controller.mm
index 6f3e9be110b2302a14901907beb30eec1ecd7891..6b358a6279eeb78887ffbfafd158748118249765 100644
--- a/chrome/app_shim/app_shim_controller.mm
+++ b/chrome/app_shim/app_shim_controller.mm
@@ -636,6 +636,7 @@ mojo::PlatformChannelEndpoint AppShimController::ConnectToBrowser(
 
 void AppShimController::SendBootstrapOnShimConnected(
     mojo::PlatformChannelEndpoint endpoint) {
+  AppendTestLaunchUrls(launch_urls_);
   DCHECK_EQ(init_state_, InitState::kWaitingForChromeReady);
   init_state_ = InitState::kHasSentOnShimConnected;
 
