diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 9b2f661481642c02a0bc0f215075ec66e14382e4..428ac34527375ef34536847d753886261d9b8f8b 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -875,6 +875,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
