diff --git a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java
index 4fa433b8303dbb36869826df0b74d7520e6f24d4..878b819b10f9798d0bde22b64ced917062a9cce6 100644
--- a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java
+++ b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java
@@ -496,6 +496,9 @@ public class FuseboxCoordinator implements TemplateUrlServiceObserver {
     // TemplateUrlServiceObserver
     @Override
     public void onTemplateURLServiceChanged() {
+        if (!mTemplateUrlService.hasNativeService()) {
+            return;
+        }
         boolean isDseGoogle = mTemplateUrlService.isDefaultSearchEngineGoogle();
         if (isDseGoogle == mDefaultSearchEngineIsGoogle) return;
         mDefaultSearchEngineIsGoogle = isDseGoogle;
