diff --git a/chrome/android/java/src/org/chromium/chrome/browser/preferences/ChromePreferenceManager.java b/chrome/android/java/src/org/chromium/chrome/browser/preferences/ChromePreferenceManager.java
index e269ba6c0436afc362804823e557dfe38c600c3b..fc3699b450c22a9731ccce0723e69dfcf813cf3c 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/preferences/ChromePreferenceManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/preferences/ChromePreferenceManager.java
@@ -21,7 +21,7 @@ import java.util.Set;
 /**
  * ChromePreferenceManager stores and retrieves various values in Android shared preferences.
  */
-public class ChromePreferenceManager {
+public class ChromePreferenceManager extends BravePreferenceManager {
     // For new int values with a default of 0, just document the key and its usage, and call
     // #readInt and #writeInt directly.
     // For new boolean values, document the key and its usage, call #readBoolean and #writeBoolean
