diff --git a/chrome/browser/download/download_item_model.h b/chrome/browser/download/download_item_model.h
index 6de605e0fbc75a1e7b4bd18b4d75484db41cecaf..2fb5d0161374e8f8d7ad925a1273d4b4070debed 100644
--- a/chrome/browser/download/download_item_model.h
+++ b/chrome/browser/download/download_item_model.h
@@ -122,6 +122,7 @@ class DownloadItemModel : public DownloadUIModel,
   void OnDownloadDestroyed(download::DownloadItem* download) override;
 
  private:
+  friend class BraveDownloadItemModel;
   // DownloadUIModel implementation.
   std::string GetMimeType() const override;
   bool IsExtensionDownload() const override;
