diff --git a/chrome/browser/profiles/profile_impl.cc b/chrome/browser/profiles/profile_impl.cc
index f1d10f03bc6f58da0c1d18fada3e32e86d29816d..df90c9dd2c220e09cbe50f927b4ec195b4811b0b 100644
--- a/chrome/browser/profiles/profile_impl.cc
+++ b/chrome/browser/profiles/profile_impl.cc
@@ -567,6 +567,7 @@ void ProfileImpl::TakePrefsFromStartupData() {
       CreateExtensionPrefStore(this, /*incognito_pref_store=*/false));
 
   ProfileKeyStartupAccessor::GetInstance()->Reset();
+  BRAVE_PROFILE_IMPL_TAKE_PREFS_FROM_STARTUP_DATA
 }
 #endif
 
