diff --git a/chrome/browser/ui/android/appmenu/internal/BUILD.gn b/chrome/browser/ui/android/appmenu/internal/BUILD.gn
index 4051da2301de49cba785304316e9f642719055a4..7b2856ed1ec02c806ad3cff60b53b39add16d967 100644
--- a/chrome/browser/ui/android/appmenu/internal/BUILD.gn
+++ b/chrome/browser/ui/android/appmenu/internal/BUILD.gn
@@ -22,6 +22,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/ui/appmenu/AppMenuItemViewBinder.java",
     "java/src/org/chromium/chrome/browser/ui/appmenu/DividerLineMenuItemViewBinder.java",
   ]
+  sources += brave_browser_ui_android_appmenu_internal_sources
 
   deps = [
     ":java_resources",
