diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 1690b661b75772dfd403e9432e8cc5310eda15d8..8857937235d7af7777f72edab4eaf313cbffc9c6 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -444,6 +444,8 @@ class Browser : public TabStripModelObserver,
 #endif
 
  private:
+  friend class BookmarkPrefsService;
+  friend class BraveBrowser;
   friend class BrowserTest;
   friend class BrowserWebContentsDelegate;
   friend class ExclusiveAccessTest;
@@ -478,7 +480,7 @@ class Browser : public TabStripModelObserver,
 
   // Handle changes to tab strip model.
   void OnTabInsertedAt(content::WebContents* contents, int index);
-  void OnTabClosing(tabs::TabInterface* tab, bool* had_active_modal_dialog);
+  virtual void OnTabClosing(tabs::TabInterface* tab, bool* had_active_modal_dialog);
   void OnTabDetached(tabs::TabInterface* tab,
                      bool was_active,
                      bool had_active_modal_dialog);
@@ -513,7 +515,7 @@ class Browser : public TabStripModelObserver,
   //   updates), then scheduled_updates_ is updated for the |source| and update
   //   pair and a task is scheduled (assuming it isn't running already)
   //   that invokes ProcessPendingUIUpdates.
-  void ScheduleUIUpdate(content::WebContents* source, unsigned changed_flags);
+  virtual void ScheduleUIUpdate(content::WebContents* source, unsigned changed_flags);
 
   // Processes all pending updates to the UI that have been scheduled by
   // ScheduleUIUpdate in scheduled_updates_.
