diff --git a/chrome/browser/ui/omnibox/omnibox_controller.cc b/chrome/browser/ui/omnibox/omnibox_controller.cc
index 9387a024cedc7bc036a0007956a5a91d0f37b1e2..1e94730cf7cc217541c4f7d1cfb198cc67763445 100644
--- a/chrome/browser/ui/omnibox/omnibox_controller.cc
+++ b/chrome/browser/ui/omnibox/omnibox_controller.cc
@@ -72,6 +72,10 @@ OmniboxController::~OmniboxController() = default;
 
 void OmniboxController::StartAutocomplete(
     const AutocompleteInput& input) const {
+  if (!IsAutocompleteEnabled(client_->GetPrefs())) {
+    ClearPopupKeywordMode();
+    return;
+  }
   TRACE_EVENT0("omnibox", "OmniboxController::StartAutocomplete");
   ClearPopupKeywordMode();
 
@@ -89,6 +93,7 @@ void OmniboxController::StopAutocomplete(bool clear_result) const {
 }
 
 void OmniboxController::StartZeroSuggestPrefetch() {
+  if ((true)) return;
   TRACE_EVENT0("omnibox", "OmniboxController::StartZeroSuggestPrefetch");
   if (base::FeatureList::IsEnabled(features::kPrewarm) &&
       base::FeatureList::IsEnabled(features::kPrewarmZeroSuggestTrigger)) {
