diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc
index 84b8122cf64800da00f95c29482a09a2194cc956..0dfbf4ea9df2569fc18c039e3b52dc2a8c3455db 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.cc
@@ -522,6 +522,7 @@ void RecentTabsSubMenuModel::BuildTabsFromOtherDevices() {
       BuildOtherDevicesTabItem(session_tag, *tabs_in_session[k]);
     }  // for all tabs in one session
 
+    BRAVE_BUILD_TABS_FROM_OTHER_DEVICES
     ++num_sessions_added;
   }  // for all sessions
 
