diff --git a/chrome/browser/ui/chrome_pages.cc b/chrome/browser/ui/chrome_pages.cc
index 681eecd12f40cf101a40fd1d1b00898e0428c43b..b453a055ca703fb5da6efca5ff1333420f70ab8f 100644
--- a/chrome/browser/ui/chrome_pages.cc
+++ b/chrome/browser/ui/chrome_pages.cc
@@ -288,7 +288,7 @@ void ShowSlow(Browser* browser) {
 #endif
 }
 
-GURL GetSettingsUrl(const std::string& sub_page) {
+BRAVE_GET_SETTINGS_URL
   return GURL(std::string(kChromeUISettingsURL) + sub_page);
 }
 
