diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 86dc7d83ec9b3ec390527fe0edb42c886348efd0..73d8055eb2fe5c9c053d5f0936bc3bf7977d59f8 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2360,6 +2360,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
