diff --git a/chrome/browser/ui/views/profiles/profile_menu_view.h b/chrome/browser/ui/views/profiles/profile_menu_view.h
index 4361ad66461f7cc41ee3173c231f4a19c6dd7fe6..694754f0b11cf8cf79bf2ff3610481606f28951d 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view.h
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.h
@@ -59,6 +59,7 @@ class ProfileMenuView : public ProfileMenuViewBase {
   void BuildMenu() override;
 
  private:
+  BRAVE_PROFILE_MENU_VIEW_H
   friend class ProfileMenuViewExtensionsTest;
   friend class ProfileMenuViewSigninPendingTest;
   friend class ProfileMenuViewSignoutTest;
