diff --git a/chrome/browser/contextual_tasks/contextual_tasks_side_panel_coordinator_unittest.cc b/chrome/browser/contextual_tasks/contextual_tasks_side_panel_coordinator_unittest.cc
index 7b6a28ed27d2e423094fe507c5203c0911eb4284..0b4849634a436b862803ace00e6aeaf60c50581a 100644
--- a/chrome/browser/contextual_tasks/contextual_tasks_side_panel_coordinator_unittest.cc
+++ b/chrome/browser/contextual_tasks/contextual_tasks_side_panel_coordinator_unittest.cc
@@ -111,6 +111,7 @@ class MockSidePanelUI : public SidePanelUI {
               SetNoDelaysForTesting,
               (bool no_delays_for_testing),
               (override));
+  BRAVE_MOCK_SIDE_PANEL_UI_TOGGLE
 };
 
 class MockActiveTaskContextProvider : public ActiveTaskContextProvider {
