diff --git a/chrome/installer/mini_installer/chrome.release b/chrome/installer/mini_installer/chrome.release
index 6523364f6870b376a65efd1593b5e87ba98fe0d5..8d08f939270da3fdbd4d9f59940270f83f8c2951 100644
--- a/chrome/installer/mini_installer/chrome.release
+++ b/chrome/installer/mini_installer/chrome.release
@@ -6,7 +6,7 @@
 #
 # Chrome Application dir entries, sorted alphabetically.
 #
-chrome.exe: %(ChromeDir)s\
+brave.exe: %(ChromeDir)s\
 chrome_proxy.exe: %(ChromeDir)s\
 #
 # Chrome version dir assembly manifest.
@@ -18,9 +18,14 @@ chrome_proxy.exe: %(ChromeDir)s\
 #
 # Chrome version dir entries, sorted alphabetically.
 #
+brave.exe.sig: %(VersionDir)s\
 chrome.dll: %(VersionDir)s\
+chrome.dll.sig: %(VersionDir)s\
+brave_resources.pak: %(VersionDir)s\
+brave_100_percent.pak: %(VersionDir)s\
 chrome_100_percent.pak: %(VersionDir)s\
 chrome_child.dll: %(VersionDir)s\
+chrome_child.dll.sig: %(VersionDir)s\
 chrome_elf.dll: %(VersionDir)s\
 chrome_pwa_launcher.exe: %(VersionDir)s\
 chrome_wer.dll: %(VersionDir)s\
@@ -63,6 +68,7 @@ MEIPreload\preloaded_data.pb: %(VersionDir)s\MEIPreload\
 
 [HIDPI]
 chrome_200_percent.pak: %(VersionDir)s\
+brave_200_percent.pak: %(VersionDir)s\
 
 [FFMPEG]
 ffmpeg.dll: %(VersionDir)s\
