diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java
index fdca8057859857c09c570516c33095ac1b6639a9..bae06b3894eeb2fb330a9067d4145e0477287918 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java
@@ -120,7 +120,7 @@ public class BottomControlsCoordinator {
                     root.findViewById(R.id.bottom_container_slot), themeColorProvider,
                     scrimCoordinator);
         } else {
-            mBottomToolbarCoordinator = new BottomToolbarCoordinator(
+            mBottomToolbarCoordinator = new BraveBottomToolbarCoordinator(root,
                     root.findViewById(R.id.bottom_toolbar_stub), tabProvider,
                     tabSwitcherLongclickListener, themeColorProvider, shareDelegateSupplier,
                     showStartSurfaceCallable, openHomepageAction, setUrlBarFocusAction,
@@ -207,4 +207,6 @@ public class BottomControlsCoordinator {
         }
         return mTabGroupUi.getTabGridDialogVisibilitySupplier();
     }
+
+    public void updateBookmarkButton(boolean isBookmarked, boolean editingAllowed) {if (mBottomToolbarCoordinator != null) { mBottomToolbarCoordinator.updateBookmarkButton(isBookmarked, editingAllowed);}}
 }
