diff --git a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
index 11fbfe4bc7aca..3c45c2ddcf672 100644
--- a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
+++ b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.cc
@@ -159,7 +159,7 @@ std::unique_ptr<TabStrip> CreateTabStrip(
   std::unique_ptr<TabHoverCardController> hover_card_controller(
       std::make_unique<TabHoverCardController>(tab_strip_region_view,
                                                browser_view->browser()));
-  auto tab_strip = std::make_unique<TabStrip>(std::move(tabstrip_controller),
+  auto tab_strip = std::make_unique<BraveTabStrip>(std::move(tabstrip_controller),
                                               std::move(hover_card_controller));
   return tab_strip;
 }
@@ -704,12 +704,12 @@ void HorizontalTabStripRegionView::LogTabSearchPositionForTesting() {
 void HorizontalTabStripRegionView::UpdateButtonBorders() {
   const int extra_vertical_space =
       GetLayoutConstant(LayoutConstant::kTabStripHeight) -
-      GetLayoutConstant(LayoutConstant::kTabstripToolbarOverlap) -
+      tabs::GetHorizontalTabControlOverlap() -
       NewTabButton::kButtonSize.height();
   const int top_inset = extra_vertical_space / 2;
   const int bottom_inset =
       extra_vertical_space - top_inset +
-      GetLayoutConstant(LayoutConstant::kTabstripToolbarOverlap);
+      tabs::GetHorizontalTabControlOverlap();
   // The new tab button is placed vertically exactly in the center of the
   // tabstrip. Extend the border of the button such that it extends to the top
   // of the tabstrip bounds. This is essential to ensure it is targetable on the
