diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
index ba4e8490ca42b169c109a7fa6b431dbb109712db..2aac12a835cae12a102a6b80fc1e6f6c76f4227c 100644
--- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
+++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
@@ -397,10 +397,12 @@ public class ToolbarTablet
             mOfflineDownloader.downloadPage(getContext(), getToolbarDataProvider().getTab());
             RecordUserAction.record("MobileToolbarDownloadPage");
         }
+        BraveToolbarLayout.class.cast(this).onClickImpl(v);
     }
 
     @Override
     public boolean onLongClick(View v) {
+        if (BraveToolbarLayout.class.cast(this).onLongClickImpl(v)) return true;
         String description = null;
         Context context = getContext();
         Resources resources = context.getResources();
@@ -506,6 +508,7 @@ public class ToolbarTablet
         mLocationBar.getTabletCoordinator().tintBackground(textBoxColor);
         mLocationBar.updateVisualsForState();
         setToolbarHairlineColor(color);
+        super.onThemeColorChanged(color, shouldAnimate);
     }
 
     /**
