diff --git a/chrome/installer/linux/rpm/update_package_provides.py b/chrome/installer/linux/rpm/update_package_provides.py
index 045f6051a390632620575819df5d4d9a28b885f0..a974aeae050347ccaf492d7f367a51c38bf3993d 100755
--- a/chrome/installer/linux/rpm/update_package_provides.py
+++ b/chrome/installer/linux/rpm/update_package_provides.py
@@ -33,6 +33,7 @@ LIBRARY_FILTER = set([
     "libatspi.so.0",
     "libc.so",
     "libcairo.so",
+    "libcairo-gobject.so.2",
     "libcups.so",
     "libdbus-1.so",
     "libdrm.so.2",
@@ -53,13 +54,17 @@ LIBRARY_FILTER = set([
     "libnssutil3.so",
     "libpango-1.0.so",
     "libpangocairo-1.0.so",
+    "libplc4.so",
+    "libplds4.so",
     "libpthread.so",
+    "libresolv.so.2",
     "librt.so",
     "libsmime3.so",
     "libstdc++.so",
     "libuuid.so",
     "libxcb.so",
     "libxcb-dri3.so.0",
+    "libz.so.1",
     "rtld(GNU_HASH)",
 ])
 
