diff --git a/chrome/browser/ui/views/profiles/profile_chooser_view.h b/chrome/browser/ui/views/profiles/profile_chooser_view.h
index b1675263d9f659f3c5f6003fd39cc87a4987d396..d5ba469dc3f1a82c8a17978bd9a4384114721805 100644
--- a/chrome/browser/ui/views/profiles/profile_chooser_view.h
+++ b/chrome/browser/ui/views/profiles/profile_chooser_view.h
@@ -51,6 +51,8 @@ class ProfileChooserView : public ProfileMenuViewBase,
   ~ProfileChooserView() override;
 
  private:
+  friend class BraveProfileChooserView;
+  friend class BraveProfileChooserViewTest;
   friend class ProfileChooserViewExtensionsTest;
 
   typedef std::vector<size_t> Indexes;
@@ -114,6 +116,7 @@ class ProfileChooserView : public ProfileMenuViewBase,
 
   // Adds a view showing the profile associated with |avatar_item| and an error
   // button below.
+  virtual
   void AddDiceSyncErrorView(const AvatarMenu::Item& avatar_item,
                             sync_ui_util::AvatarSyncErrorType error,
                             int button_string_id);
