diff --git a/chrome/test/base/testing_profile.cc b/chrome/test/base/testing_profile.cc
index 4574f3d923eff0c6c3ff5282ce52280c402f741b..607d487d803fa250906454ccfb3566a3b4c70d81 100644
--- a/chrome/test/base/testing_profile.cc
+++ b/chrome/test/base/testing_profile.cc
@@ -329,6 +329,7 @@ void TestingProfile::Init() {
       EnsureBrowserContextKeyedServiceFactoriesBuilt();
 
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
+  BRAVE_INIT
   if (!IsOffTheRecord()) {
     SupervisedUserSettingsService* settings_service =
         SupervisedUserSettingsServiceFactory::GetForKey(key_.get());
