diff --git a/chrome/browser/importer/BUILD.gn b/chrome/browser/importer/BUILD.gn
index 08b67eb41ebb07b8c740e58fd01c6d51afa4aef8..5607f69abdfef2f24c4a940aeee11689e7b3cfd5 100644
--- a/chrome/browser/importer/BUILD.gn
+++ b/chrome/browser/importer/BUILD.gn
@@ -2,6 +2,8 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/importer/sources.gni")
+
 assert(is_win || is_mac || is_linux || is_chromeos)
 
 source_set("importer") {
@@ -16,6 +18,7 @@ source_set("importer") {
     "in_process_importer_bridge.h",
     "profile_writer.h",
   ]
+  sources += brave_browser_importer_sources
 
   public_deps = [
     "//base",
@@ -31,6 +34,7 @@ source_set("importer") {
     "//ui/gfx",
     "//url",
   ]
+  public_deps += brave_browser_importer_public_deps
 }
 
 source_set("impl") {
@@ -45,6 +49,7 @@ source_set("impl") {
     "in_process_importer_bridge.cc",
     "profile_writer.cc",
   ]
+  sources += brave_browser_importer_impl_sources
 
   deps = [
     ":importer",
@@ -64,6 +69,7 @@ source_set("impl") {
     "//components/password_manager/core/browser:password_form",
     "//components/password_manager/core/browser/password_store:password_store_interface",
   ]
+  deps += brave_browser_importer_impl_deps
 
   if (is_win) {
     sources += [ "firefox_profile_lock_win.cc" ]
