diff --git a/chrome/browser/ui/browser_commands.cc b/chrome/browser/ui/browser_commands.cc
index 315465e889d2ffaa30e960bfae86fdd4269fbce3..9db93e4e871e1c7224d19dac7cbcf5b3f6e0acef 100644
--- a/chrome/browser/ui/browser_commands.cc
+++ b/chrome/browser/ui/browser_commands.cc
@@ -1067,7 +1067,7 @@ void Reload(BrowserWindowInterface* browser,
   ReloadInternal(browser, disposition, false);
 }
 
-void ReloadBypassingCache(BrowserWindowInterface* browser,
+void ReloadBypassingCache_ChromiumImpl(BrowserWindowInterface* browser,
                           WindowOpenDisposition disposition) {
   base::RecordAction(UserMetricsAction("ReloadBypassingCache"));
   ReloadInternal(browser, disposition, true);
@@ -1367,6 +1367,7 @@ void CloseTab(BrowserWindowInterface* browser) {
   }
 
   ToastController* toast_controller = browser->GetFeatures().toast_controller();
+  toast_controller = nullptr;
   if (!toast_controller) {
     CloseSelectedTabAndRecordTabCountMetric(browser);
     return;
@@ -1669,7 +1670,7 @@ void NewSplitTab(BrowserWindowInterface* browser,
   // the regular NTP which renders special content when in Incognito.
   const GURL new_tab_url = !browser->GetProfile()->IsIncognitoProfile() &&
                                    tab_strip_model->count() > 1
-                               ? GURL(chrome::kChromeUISplitViewNewTabPageURL)
+                               ? GURL(chrome::kChromeUINewTabURL)
                                : chrome::ChromeUINewTabURLAsGURL();
   tab_strip_model->delegate()->AddTabAt(
       new_tab_url, active_index + 1, true,
