diff --git a/chrome/chrome_repack_locales.gni b/chrome/chrome_repack_locales.gni
index dd37d98b467ec9bcf5ac5fd59ac56e43412a19fa..6500d8c648d260959cebc683ad29d9e9c3f9e9eb 100644
--- a/chrome/chrome_repack_locales.gni
+++ b/chrome/chrome_repack_locales.gni
@@ -35,6 +35,7 @@ template("chrome_repack_locales") {
       "${root_gen_dir}/ui/strings/app_locale_settings_",
       "${root_gen_dir}/ui/strings/ui_strings_",
     ]
+    source_patterns += brave_locale_source_patterns
     if (!defined(deps)) {
       deps = []
     }
@@ -51,6 +52,7 @@ template("chrome_repack_locales") {
       "//ui/strings:app_locale_settings",
       "//ui/strings:ui_strings",
     ]
+    deps += brave_locale_deps
     if (defined(invoker.deps)) {
       deps += invoker.deps
     }
