diff --git a/chrome/browser/ui/views/tabs/alert_indicator.h b/chrome/browser/ui/views/tabs/alert_indicator.h
index 24bdc32e6f67b91495812efe6a251f8d28c62aa8..91852813b4791c71d51c659fdaa236e040e50fe5 100644
--- a/chrome/browser/ui/views/tabs/alert_indicator.h
+++ b/chrome/browser/ui/views/tabs/alert_indicator.h
@@ -48,6 +48,7 @@ class AlertIndicator : public views::ImageView {
   View* GetTooltipHandlerForPoint(const gfx::Point& point) override;
 
  private:
+  friend class BraveAlertIndicator;
   friend class AlertIndicatorTest;
   friend class TabTest;
   class FadeAnimationDelegate;
