diff --git a/chrome/installer/util/google_update_settings.cc b/chrome/installer/util/google_update_settings.cc
index 4f0ad5378847cb9ee99569a206994c50683d06bf..6e128e4631a358208b9b3cda3213c52b9773403f 100644
--- a/chrome/installer/util/google_update_settings.cc
+++ b/chrome/installer/util/google_update_settings.cc
@@ -31,7 +31,11 @@ using base::win::RegKey;
 using installer::InstallationState;
 
 const wchar_t GoogleUpdateSettings::kPoliciesKey[] =
+#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
+    L"SOFTWARE\\Policies\\BraveSoftware\\Update";
+#else
     L"SOFTWARE\\Policies\\Google\\Update";
+#endif
 const wchar_t GoogleUpdateSettings::kUpdatePolicyValue[] = L"UpdateDefault";
 const wchar_t GoogleUpdateSettings::kDownloadPreferencePolicyValue[] =
     L"DownloadPreference";
@@ -43,6 +47,11 @@ const wchar_t GoogleUpdateSettings::kCheckPeriodOverrideMinutes[] =
 const int GoogleUpdateSettings::kCheckPeriodOverrideMinutesMax =
     60 * 24 * 7 * 6;
 
+
+#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
+#define GOOGLE_CHROME_BUILD
+#endif
+
 const GoogleUpdateSettings::UpdatePolicy
 GoogleUpdateSettings::kDefaultUpdatePolicy =
 #if defined(GOOGLE_CHROME_BUILD)
@@ -887,3 +896,7 @@ bool GoogleUpdateSettings::ReadExperimentLabels(
 
   return result == ERROR_SUCCESS;
 }
+
+#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
+#undef GOOGLE_CHROME_BUILD
+#endif
