diff --git a/chrome/browser/ui/android/toolbar/BUILD.gn b/chrome/browser/ui/android/toolbar/BUILD.gn
index 0231a777ef90b10fe9a02887fa22d03d2e009c00..6bb83555e81e09c60a5dd762a71c82ef44e9a482 100644
--- a/chrome/browser/ui/android/toolbar/BUILD.gn
+++ b/chrome/browser/ui/android/toolbar/BUILD.gn
@@ -95,6 +95,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/toolbar/top/TopToolbarSceneLayer.java",
     "java/src/org/chromium/chrome/browser/toolbar/top/ViewShiftingActionBarDelegate.java",
   ]
+  sources += brave_browser_ui_android_toolbar_sources
 
   deps = [
     ":java_resources",
