diff --git a/chrome/android/features/tab_ui/BUILD.gn b/chrome/android/features/tab_ui/BUILD.gn
index cd01fffe3a238cfa763db9864180bc06cfe2f518..b6ef9fcaf3a524b2a2db2420dd2978934263e44a 100644
--- a/chrome/android/features/tab_ui/BUILD.gn
+++ b/chrome/android/features/tab_ui/BUILD.gn
@@ -128,6 +128,7 @@ android_resources("java_resources") {
     "java/res/xml/tab_archive_time_delta_preference.xml",
     "java/res/xml/tabs_settings.xml",
   ]
+  sources += brave_android_features_java_resources
   deps = [
     ":java_strings_grd",
     "//chrome/android:chrome_app_java_resources",
