diff --git a/chrome/browser/ui/views/frame/tab_strip_region_view.cc b/chrome/browser/ui/views/frame/tab_strip_region_view.cc
index 582855d6a992251477841ca35ee40c92d0e052ca..c86540889766768f4ef321980573034af2f8ba43 100644
--- a/chrome/browser/ui/views/frame/tab_strip_region_view.cc
+++ b/chrome/browser/ui/views/frame/tab_strip_region_view.cc
@@ -123,7 +123,7 @@ std::unique_ptr<TabStrip> CreateTabStrip(BrowserView* browser_view) {
       std::move(tab_menu_model_factory));
   BrowserTabStripController* tabstrip_controller_ptr =
       tabstrip_controller.get();
-  auto tab_strip = std::make_unique<TabStrip>(std::move(tabstrip_controller));
+  BRAVE_TAB_STRIP_REGION_VIEW_CREATE_TAB_STRIP
   tabstrip_controller_ptr->InitFromModel(tab_strip.get());
   return tab_strip;
 }
