diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 753770468a1334e15d749efd682a32689e685116..f9d0e4428574bb067597a5b069e6dbb64cf0aa04 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1481,6 +1481,7 @@ source_set("extensions") {
       allow_circular_includes_from += [ "//chrome/browser/glic:impl" ]
     }
   }  # if (enable_extensions)
+  deps += brave_extensions_resources
 }
 
 if (enable_extensions_core) {
