diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index f69ad82885c4dd582895896983930bda3331be35..1cb71f892c156563810b330a783d0e208c9eb0f6 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -275,7 +275,7 @@ void ToolbarView::Init() {
   } else {
     // TODO(crbug.com/932818): Remove this once the
     // |kAutofillEnableToolbarStatusChip| is fully launched.
-    avatar_ = AddChildView(std::make_unique<AvatarToolbarButton>(browser_));
+    avatar_ = AddChildView(std::make_unique<BraveAvatarToolbarButton>(browser_));
     avatar_->SetVisible(show_avatar_toolbar_button);
   }
 
