diff --git a/chrome/browser/resources/settings/people_page/import_data_dialog.js b/chrome/browser/resources/settings/people_page/import_data_dialog.js
index 45bce2b6cda233abe4861b0743b4f71aad06377c..c19c228820dcbd6b587eb6e66930bd55ae139a53 100644
--- a/chrome/browser/resources/settings/people_page/import_data_dialog.js
+++ b/chrome/browser/resources/settings/people_page/import_data_dialog.js
@@ -86,7 +86,15 @@ Polymer({
         !(this.getPref('import_dialog_search_engine').value &&
           this.selected_.search) &&
         !(this.getPref('import_dialog_autofill_form_data').value &&
-          this.selected_.autofillFormData);
+          this.selected_.autofillFormData) &&
+        !(this.getPref('import_dialog_cookies').value &&
+          this.selected_.cookies) &&
+        !(this.getPref('import_dialog_stats').value &&
+          this.selected_.stats) &&
+        !(this.getPref('import_dialog_ledger').value &&
+          this.selected_.ledger) &&
+        !(this.getPref('import_dialog_windows').value &&
+          this.selected_.windows);
   },
 
   /**
