diff --git a/chrome/browser/ui/views/page_action/page_action_controller.h b/chrome/browser/ui/views/page_action/page_action_controller.h
index f8b60561923541d940cbc6c7fdee6fa0d025584f..31376b5e92d4581dabb2d90eb6283fa4277466e7 100644
--- a/chrome/browser/ui/views/page_action/page_action_controller.h
+++ b/chrome/browser/ui/views/page_action/page_action_controller.h
@@ -109,6 +109,17 @@ class PageActionController {
  public:
   virtual ~PageActionController() = default;
 
+  // Force the label to be shown regardless of the available space
+  virtual void SetAlwaysShowLabel(actions::ActionId action_id,
+                                  bool always_show) = 0;
+
+  // Override the colors of the chip
+  virtual void OverrideChipColors(
+      actions::ActionId action_id,
+      std::optional<SkColor> override_background_color,
+      std::optional<SkColor> override_foreground_color) = 0;
+  virtual void ClearOverrideChipColors(actions::ActionId action_id) = 0;
+
   // Requests that the page action be shown or hidden.
   virtual void Show(actions::ActionId action_id) = 0;
   virtual void Hide(actions::ActionId action_id) = 0;
@@ -246,6 +257,13 @@ class PageActionControllerImpl : public PageActionController,
   PageActionControllerImpl& operator=(const PageActionControllerImpl&) = delete;
   ~PageActionControllerImpl() override;
 
+  void SetAlwaysShowLabel(actions::ActionId action_id,
+                          bool always_show) override;
+  void OverrideChipColors(actions::ActionId action_id,
+      std::optional<SkColor> override_background_color,
+      std::optional<SkColor> override_foreground_color) override;
+  void ClearOverrideChipColors(actions::ActionId action_id) override;
+
   void Initialize(
       tabs::TabInterface& tab_interface,
       const std::vector<actions::ActionId>& action_ids,
