diff --git a/chrome/browser/ui/views/profiles/profile_menu_view.h b/chrome/browser/ui/views/profiles/profile_menu_view.h
index b1b448a9373a36127c2ec91006dc2baa8f535a53..6575a8a875bad543bda1eb39b81bffa96778b388 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view.h
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.h
@@ -41,6 +41,7 @@ class ProfileMenuView : public ProfileMenuViewBase {
   gfx::ImageSkia GetSyncIcon() const override;
 
  private:
+  BRAVE_PROFILE_MENU_VIEW_H
   friend class ProfileMenuViewExtensionsTest;
   friend class ProfileMenuViewSignoutTest;
 
