diff --git a/chrome/browser/ui/views/frame/multi_contents_view.cc b/chrome/browser/ui/views/frame/multi_contents_view.cc
index e71df1a8a0958caba9e1cd6e1e13ac0a7afb83bf..95ed89d6844b0115916cb0abbd2ff172a03338b0 100644
--- a/chrome/browser/ui/views/frame/multi_contents_view.cc
+++ b/chrome/browser/ui/views/frame/multi_contents_view.cc
@@ -60,7 +60,7 @@ MultiContentsView::MultiContentsView(
   SetProperty(views::kElementIdentifierKey, kMultiContentsViewElementId);

   contents_container_views_.push_back(
-      AddChildView(std::make_unique<ContentsContainerView>(browser_view_)));
+      AddChildView(std::make_unique<BraveContentsContainerView>(browser_view_)));
   contents_container_views_[0]
       ->contents_view()
       ->set_is_primary_web_contents_for_window(true);
@@ -69,7 +69,7 @@ MultiContentsView::MultiContentsView(
   resize_area_->SetVisible(false);

   contents_container_views_.push_back(
-      AddChildView(std::make_unique<ContentsContainerView>(browser_view_)));
+      AddChildView(std::make_unique<BraveContentsContainerView>(browser_view_)));
   contents_container_views_[1]->SetVisible(false);

   drop_target_view_ =
