diff --git a/chrome/browser/ui/webui/theme_source.cc b/chrome/browser/ui/webui/theme_source.cc
index d8a4620d0525c6665ee783ac4a80470cc20e26c8..ee3802aa65ef011d717eb54269cdecd35a956610 100644
--- a/chrome/browser/ui/webui/theme_source.cc
+++ b/chrome/browser/ui/webui/theme_source.cc
@@ -325,11 +325,13 @@ void ThemeSource::SendColorsCss(
   const auto* theme_service =
       ThemeServiceFactory::GetForProfile(profile_->GetOriginalProfile());
   std::string theme_id;
+  if (theme_service) {
   if (theme_service->GetIsGrayscale()) {
     theme_id = "--user-color-source: baseline-grayscale;";
   } else if (theme_service->GetIsBaseline()) {
     theme_id = "--user-color-source: baseline-default;";
   }
+  }
 
   std::string css_string = base::StrCat(
       {css_selector, "{", theme_id,
