diff --git a/chrome/browser/profiles/profile_io_data.cc b/chrome/browser/profiles/profile_io_data.cc
index 3bf4c0fd66c185e839f29c3b54543d6dd21a550b..e69b5f519729bbfa921c9e5ec86d9c6cf4e8bb99 100644
--- a/chrome/browser/profiles/profile_io_data.cc
+++ b/chrome/browser/profiles/profile_io_data.cc
@@ -1088,7 +1088,7 @@ void ProfileIOData::Init(
         std::make_unique<network::URLRequestContextBuilderMojo>();
 
     std::unique_ptr<ChromeNetworkDelegate> chrome_network_delegate(
-        new ChromeNetworkDelegate(
+        new BraveProfileNetworkDelegate(
 #if BUILDFLAG(ENABLE_EXTENSIONS)
             io_thread_globals->extension_event_router_forwarder.get()));
 #else
