diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 30bc52b163c69435813aa0a85314100944c12671..d5cf3e467d44133f8e0f747ce6da41644cea778a 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2559,6 +2559,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
