diff --git a/chrome/browser/updater/BUILD.gn b/chrome/browser/updater/BUILD.gn
index fddafceb5f7ed15510ce96e92f67f2bc98f5264c..65a0decb1f04370b858f77431ad09aa72749233b 100644
--- a/chrome/browser/updater/BUILD.gn
+++ b/chrome/browser/updater/BUILD.gn
@@ -36,6 +36,7 @@ source_set("updater") {
       "//chrome/updater:browser_sources",
     ]
 
+    import("//brave/browser/updater/sources.gni") deps += brave_chrome_updater_scheduler_deps
     if (is_win) {
       sources += [
         "browser_updater_client_util_win.cc",
