diff --git a/chrome/browser/ui/webui/settings/import_data_handler.cc b/chrome/browser/ui/webui/settings/import_data_handler.cc
index 1637dc014a75986cab0ec4abd327034f975a102b..1a53fdc6fb3850d502d43e62f5d3f2231950ada9 100644
--- a/chrome/browser/ui/webui/settings/import_data_handler.cc
+++ b/chrome/browser/ui/webui/settings/import_data_handler.cc
@@ -126,6 +126,7 @@ void ImportDataHandler::HandleImportData(const base::ListValue* args) {
     selected_items |= importer::PASSWORDS;
   if (*types->FindBoolKey(prefs::kImportDialogSearchEngine))
     selected_items |= importer::SEARCH_ENGINES;
+  BRAVE_IMPORT_DATA
 
   const importer::SourceProfile& source_profile =
       importer_list_->GetSourceProfileAt(browser_index);
@@ -203,6 +204,7 @@ void ImportDataHandler::SendBrowserProfileData(const std::string& callback_id) {
     browser_profile->SetBoolean(
         "autofillFormData",
         (browser_services & importer::AUTOFILL_FORM_DATA) != 0);
+    BRAVE_SEND_BROWSER_PROFILE_DATA
 
     browser_profiles.Append(std::move(browser_profile));
   }
