diff --git a/chrome/browser/plugins/plugin_utils.cc b/chrome/browser/plugins/plugin_utils.cc
index 01ea0ba7f0cb8a1ce2ee56a1a918d200159b4994..d77a8289ec8b07179b1ba2e89701d1132cd5920f 100644
--- a/chrome/browser/plugins/plugin_utils.cc
+++ b/chrome/browser/plugins/plugin_utils.cc
@@ -184,7 +184,8 @@ PluginUtils::GetMimeTypeToExtensionIdMap(
   bool profile_is_off_the_record = io_data->IsOffTheRecord();
   const scoped_refptr<const extensions::InfoMap> extension_info_map(
       io_data->GetExtensionInfoMap());
-  std::vector<std::string> whitelist = MimeTypesHandler::GetMIMETypeWhitelist();
+  std::vector<std::string> whitelist =
+      MimeTypesHandler::BraveGetMIMETypeWhitelist();
   // Go through the white-listed extensions and try to use them to intercept
   // the URL request.
   for (const std::string& extension_id : whitelist) {
