diff --git a/chrome/browser/ui/views/tabs/tab_style_views.cc b/chrome/browser/ui/views/tabs/tab_style_views.cc
index 9b48c6cc5a223a3fe531623ea0a963b0ac838fc3..06a64e0d4003a9284259d4f7402ec7a6d2059cce 100644
--- a/chrome/browser/ui/views/tabs/tab_style_views.cc
+++ b/chrome/browser/ui/views/tabs/tab_style_views.cc
@@ -136,6 +136,7 @@ class TabStyleViewsImpl : public TabStyleViews {
   TabStyle::TabSelectionState GetSelectionState() const;
 
  private:
+  BRAVE_GM2_TAB_STYLE_H
   // Gets the bounds for the leading and trailing separators for a tab.
   TabStyle::SeparatorBounds GetSeparatorBounds(float scale) const;
 
@@ -380,6 +381,7 @@ SkPath TabStyleViewsImpl::GetPath(TabStyle::PathType path_type,
     extension_corner_radius -= 0.5f * stroke_adjustment;
   }
   const bool compact_left_to_bottom = ShouldCompactLeadingEdge(path_type);
+  extension_corner_radius = 0;
 
   SkPath path;
 
@@ -1070,14 +1072,18 @@ void TabStyleViewsImpl::PaintSeparators(gfx::Canvas* canvas) const {
                                                    SK_AlphaOPAQUE));
   };
 
+  // Even if |separator_radius| becomes 1 native pixel the 'roundedness'
+  // will be approximated with color, although extremely subtle and
+  // likely only on screens >= 2x (i.e. separator width is 2+px)!
+  const int separator_radius =  separator_bounds.leading.width() / 2;
   cc::PaintFlags flags;
   flags.setAntiAlias(true);
   flags.setColor(separator_color(separator_opacities.left));
   canvas->DrawRoundRect(separator_bounds.leading,
-                        tab_style()->GetSeparatorCornerRadius() * scale, flags);
+                        separator_radius, flags);
   flags.setColor(separator_color(separator_opacities.right));
   canvas->DrawRoundRect(separator_bounds.trailing,
-                        tab_style()->GetSeparatorCornerRadius() * scale, flags);
+                        separator_radius, flags);
 }
 
 bool TabStyleViewsImpl::IsLeftSplitTab(const Tab* tab) const {
