diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 3cf0e1a732cf3eb8c9319fdbd8d22d4f32775a57..3c8bfa1b527e5fdc71a67a43a543165163281360 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -70,6 +70,7 @@ source_set("channel_info") {
   deps = [
     "//build:branding_buildflags",
   ]
+  if (brave_chromium_build) { deps += [ "//brave/common:channel_info" ] }
 
   public_deps = [
     "//base",
@@ -202,6 +203,7 @@ static_library("common") {
   ]
 
   public_deps = [
+    "//brave/common",
     ":available_offline_content_mojom",
     ":buildflags",
     ":channel_info",
@@ -572,6 +574,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  if (brave_chromium_build) { configs += [ "//brave/common:constants_configs" ] }
 }
 
 component("chrome_features") {
