diff --git a/chrome/browser/ui/tabs/tab_strip_model_observer.h b/chrome/browser/ui/tabs/tab_strip_model_observer.h
index 6b1f306dbf04b06a72fe40ff07d9a5410f30e870..35716121df0ce897d6df09052155105d70fbdc7d 100644
--- a/chrome/browser/ui/tabs/tab_strip_model_observer.h
+++ b/chrome/browser/ui/tabs/tab_strip_model_observer.h
@@ -627,6 +627,9 @@ class TabStripModelObserver {
   void StoppedObserving(ModelPasskey, TabStripModel* model);
   void ModelDestroyed(ModelPasskey, TabStripModel* model);
 
+  virtual void TabCustomTitleChanged(content::WebContents* contents, const std::optional<std::string>& custom_title) {}
+  virtual void OnTreeTabChanged(const TreeTabChange& change) {}
+
  protected:
   TabStripModelObserver();
   virtual ~TabStripModelObserver();
