diff --git a/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc b/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc
index 2ec1fb696e4f95c26a7a1c6f009d191c96a38d67..34e25db9ed3cdaf5394f7c7cbada769bdb5068d2 100644
--- a/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc
+++ b/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc
@@ -311,6 +311,7 @@ PermissionPromptBubbleView::PermissionPromptBubbleView(
   for (permissions::PermissionRequest* request :
        GetVisibleRequests(*delegate.get()))
     AddRequestLine(request);
+  BRAVE_PERMISSION_PROMPT_BUBBLE_VIEW
 
   absl::optional<std::u16string> extra_text = GetExtraText(*delegate.get());
   if (extra_text.has_value()) {
