diff --git a/chrome/browser/ui/browser_window/BUILD.gn b/chrome/browser/ui/browser_window/BUILD.gn
index a09aa66d116d72e10d999bd138ee536d36ad1874..9009080d3e55d84a3c6bf2afdcb212d80c37cb03 100644
--- a/chrome/browser/ui/browser_window/BUILD.gn
+++ b/chrome/browser/ui/browser_window/BUILD.gn
@@ -21,6 +21,7 @@ source_set("browser_window") {
     "//content/public/browser",
     "//ui/base:types",
   ]
+  deps = [ "//brave/browser/ui/browser_window" ]
 }
 
 source_set("impl") {
@@ -58,4 +59,5 @@ source_set("impl") {
   if (enable_glic) {
     deps += [ "//chrome/browser/glic" ]
   }
+  deps += [ "//brave/browser/ui/browser_window:impl" ]
 }
