diff --git a/chrome/browser/resources/bookmarks/reducers.ts b/chrome/browser/resources/bookmarks/reducers.ts
index 975bcc41fff7af9528e17a555cbb8c264047e0b8..d6f4a4b769669f1e6f8c6f6972cc27cef56ee1f4 100644
--- a/chrome/browser/resources/bookmarks/reducers.ts
+++ b/chrome/browser/resources/bookmarks/reducers.ts
@@ -193,6 +193,9 @@ function editBookmark(nodes: NodeMap, action: EditBookmarkAction): NodeMap {
 function moveBookmark(nodes: NodeMap, action: MoveBookmarkAction): NodeMap {
   const nodeModifications: NodeMap = {};
   const id = action.id;
+  // Skip when new/old parent is invisible (Pending Bookmarks)
+  if (!nodes[action.oldParentId] || !nodes[action.parentId])
+    return nodes;
 
   // Change node's parent.
   nodeModifications[id] =
@@ -330,7 +333,7 @@ export function updateFolderOpenState(
           folderOpenState, nodes[(action as SelectFolderAction).id]!.parentId!,
           nodes);
     case 'move-bookmark':
-      if (!nodes[(action as MoveBookmarkAction).id]!.children) {
+      if (!nodes[(action as MoveBookmarkAction).id] || !nodes[(action as MoveBookmarkAction).id]!.children) {
         return folderOpenState;
       }
       return openFolderAndAncestors(
