diff --git a/chrome/installer/mac/signing/model.py b/chrome/installer/mac/signing/model.py
index 5a92fcc5ce0070fbb0495d382c552a4d08182baf..a40f04cb7f0047bcdc07812103309b4d146559c6 100644
--- a/chrome/installer/mac/signing/model.py
+++ b/chrome/installer/mac/signing/model.py
@@ -415,7 +415,7 @@ class Paths(object):
         Returns:
             Path to the packaging directory.
         """
-        return os.path.join(self.input, '{} Packaging'.format(config.product))
+        return os.path.join(self.input, '{}_Packaging'.format(config.product.replace(' ', '_')))
 
     def replace_work(self, new_work):
         """Creates a new Paths with the same input and output directories, but
