diff --git a/chrome/browser/bookmarks/android/BUILD.gn b/chrome/browser/bookmarks/android/BUILD.gn
index 53338f5c94c7fcb38f86ff266e0691a2f1f620a7..a8326001adf62256522da7786ee019ad03519dcb 100644
--- a/chrome/browser/bookmarks/android/BUILD.gn
+++ b/chrome/browser/bookmarks/android/BUILD.gn
@@ -36,6 +36,7 @@ source_set("impl") {
     "//components/query_parser",
     "//components/undo",
   ]
+  deps += brave_browser_bookmarks_android_impl_deps
 }
 
 android_resources("java_resources") {
@@ -102,6 +103,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/bookmarks/bar/BookmarkBarViewBinder.java",
     "java/src/org/chromium/chrome/browser/bookmarks/bar/BookmarkBarVisibilityProvider.java",
   ]
+  sources += brave_browser_bookmarks_android_sources
 
   deps = [
     ":java_resources",
