diff --git a/chrome/browser/global_keyboard_shortcuts_mac.mm b/chrome/browser/global_keyboard_shortcuts_mac.mm
index e0c51db2f75a8617c36dab51d2e7e7b57be0be09..d351a3ed04de53662619c9d07d67f042b4b5483a 100644
--- a/chrome/browser/global_keyboard_shortcuts_mac.mm
+++ b/chrome/browser/global_keyboard_shortcuts_mac.mm
@@ -175,6 +175,7 @@ std::vector<KeyboardShortcutData> CreateKeyboardShortcutVector() {
     // Special shortcuts for Zoom in and out.
     {true,  false, false, false, kVK_ANSI_KeypadPlus,   IDC_ZOOM_PLUS},
     {true,  false, false, false, kVK_ANSI_KeypadMinus,  IDC_ZOOM_MINUS},
+    {true,  false, false, true,  kVK_ANSI_N,            IDC_NEW_OFFTHERECORD_WINDOW_TOR},
   });
   // clang-format on
 
