diff --git a/chrome/test/base/testing_profile.cc b/chrome/test/base/testing_profile.cc
index cd31628f7ff2bb0684ec250358f3e113b7a38403..e19418e465c90f04ccd3b0f0a40800a96d1547db 100644
--- a/chrome/test/base/testing_profile.cc
+++ b/chrome/test/base/testing_profile.cc
@@ -324,6 +324,7 @@ void TestingProfile::Init() {
       EnsureBrowserContextKeyedServiceFactoriesBuilt();
 
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
+  BRAVE_INIT
   if (!IsOffTheRecord()) {
     SupervisedUserSettingsService* settings_service =
         SupervisedUserSettingsServiceFactory::GetForKey(key_.get());
