diff --git a/chrome/browser/resources/history/history.ts b/chrome/browser/resources/history/history.ts
index 7ec29f17f2c04a6ca98eece4be819d1f65fa6e32..07a7d100efc5521f74e0d6566f84205e8b6042d4 100644
--- a/chrome/browser/resources/history/history.ts
+++ b/chrome/browser/resources/history/history.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './app.js';
 
 export {BrowserProxyImpl as ShoppingBrowserProxyImpl} from 'chrome://resources/cr_components/commerce/browser_proxy.js';
