diff --git a/chrome/browser/importer/external_process_importer_client.cc b/chrome/browser/importer/external_process_importer_client.cc
index 429c16dc6a56fcd0ea1d674508c1927a387e1905..433083d6802bea4213343a9315a496d9f5cb78e5 100644
--- a/chrome/browser/importer/external_process_importer_client.cc
+++ b/chrome/browser/importer/external_process_importer_client.cc
@@ -71,6 +71,12 @@ void ExternalProcessImporterClient::Start() {
   localized_strings.SetKey(
       base::IntToString(IDS_BOOKMARK_BAR_FOLDER_NAME),
       base::Value(l10n_util::GetStringUTF8(IDS_BOOKMARK_BAR_FOLDER_NAME)));
+  localized_strings.SetKey(
+      base::IntToString(IDS_IMPORT_FROM_BRAVE),
+      base::Value(l10n_util::GetStringUTF8(IDS_IMPORT_FROM_BRAVE)));
+  localized_strings.SetKey(
+      base::IntToString(IDS_BOOKMARK_GROUP_FROM_BRAVE),
+      base::Value(l10n_util::GetStringUTF8(IDS_BOOKMARK_GROUP_FROM_BRAVE)));
 
   // If the utility process hasn't started yet the message will queue until it
   // does.
