diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index 87aca6c4a5e094efb152c9cc0582fd71d2581473..d3441b1556a20079a641223fe9433a4beb2f9dc1 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -996,6 +996,7 @@ void BookmarkBarView::Layout() {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   // Only render the tab group separtor if there are groups and bookmarks
   // visible. Must done after the bookmark buttons are rendered.
