diff --git a/chrome/install_static/install_util.cc b/chrome/install_static/install_util.cc
index fa3e47a248c87efc7ade00ab47a794d119e72950..1acd13295aff67b9e538d68f36f634865fbe9241 100644
--- a/chrome/install_static/install_util.cc
+++ b/chrome/install_static/install_util.cc
@@ -620,7 +620,7 @@ void GetExecutableVersionDetails(const std::wstring& exe_path,
 }
 
 version_info::Channel GetChromeChannel() {
-#if defined(GOOGLE_CHROME_BUILD)
+#if defined(GOOGLE_CHROME_BUILD) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))
   std::wstring channel_name(GetChromeChannelName());
   if (channel_name.empty()) {
     return version_info::Channel::STABLE;
@@ -631,7 +631,7 @@ version_info::Channel GetChromeChannel() {
   if (channel_name == L"dev") {
     return version_info::Channel::DEV;
   }
-  if (channel_name == L"canary") {
+  if (channel_name == L"nightly") {
     return version_info::Channel::CANARY;
   }
 #endif
