diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index e1dd4b21c378c6c4fe2e056e875ad7b243988123..52d3d600e04f7fb413e69eb91a4cca1570ee2748 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -86,6 +86,7 @@ source_set("channel_info") {
   } else if (is_fuchsia) {
     sources += [ "channel_info_fuchsia.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -519,6 +520,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
