diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index ce932800fc12e959d8e1b741f701a355f1aa95c9..45b5c953da77c941436ecb2d6771ea0f1f8802a3 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3379,6 +3379,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
