diff --git a/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/channels/ChromeChannelDefinitions.java b/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/channels/ChromeChannelDefinitions.java
index 20a3ffecbad1a89a86acf6c7602403d84251f2a3..b63611fc594a5d56fba4a1ffbc7f70034d730eab 100644
--- a/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/channels/ChromeChannelDefinitions.java
+++ b/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/channels/ChromeChannelDefinitions.java
@@ -78,6 +78,7 @@ public class ChromeChannelDefinitions extends ChannelDefinitions {
         ChannelId.COLLABORATION,
         ChannelId.DOWNLOADS,
         ChannelId.INCOGNITO,
+        BraveChannelDefinitions.ChannelId.BRAVE_ADS, BraveChannelDefinitions.ChannelId.BRAVE_ADS_BACKGROUND,
         ChannelId.MEDIA_PLAYBACK,
         ChannelId.SCREEN_CAPTURE,
         ChannelId.CONTENT_SUGGESTIONS,
@@ -135,7 +136,8 @@ public class ChromeChannelDefinitions extends ChannelDefinitions {
         String SERIAL = "serial";
     }
 
-    @StringDef({ChannelGroupId.GENERAL, ChannelGroupId.SITES})
+    @StringDef({BraveChannelDefinitions.ChannelGroupId.BRAVE_ADS, ChannelGroupId.GENERAL,
+            ChannelGroupId.SITES})
     @Retention(RetentionPolicy.SOURCE)
     public @interface ChannelGroupId {
         String SITES = "sites";
@@ -163,6 +165,7 @@ public class ChromeChannelDefinitions extends ChannelDefinitions {
             Map<String, PredefinedChannel> map = new HashMap<>();
             Set<String> startup = new HashSet<>();
 
+            BraveChannelDefinitions.addBraveChannels(map, startup);
             map.put(
                     ChannelId.BROWSER,
                     PredefinedChannel.create(
@@ -412,6 +415,7 @@ public class ChromeChannelDefinitions extends ChannelDefinitions {
 
         static {
             Map<String, PredefinedChannelGroup> map = new HashMap<>();
+            BraveChannelDefinitions.addBraveChannelGroups(map);
             map.put(
                     ChannelGroupId.GENERAL,
                     new PredefinedChannelGroup(
