diff --git a/chrome/browser/app_controller_mac.mm b/chrome/browser/app_controller_mac.mm
index 6063284583cc19f541a3bb9b18f4f20decdc98a1..5b0c7d144b6d006be46e38e566cd07ae1efbde65 100644
--- a/chrome/browser/app_controller_mac.mm
+++ b/chrome/browser/app_controller_mac.mm
@@ -793,7 +793,7 @@ class AppControllerProfileObserver : public ProfileAttributesStorage::Observer,
 
 + (AppController*)sharedController {
   static AppController* sharedController = [] {
-    AppController* sharedController = [[AppController alloc] init];
+    AppController* sharedController = [[BraveAppController alloc] init];
     NSApp.delegate = sharedController;
     return sharedController;
   }();
@@ -1548,7 +1548,7 @@ class AppControllerProfileObserver : public ProfileAttributesStorage::Observer,
   }
 
   auto it = _profileBookmarkMenuBridgeMap.find(profilePath);
-  if (it != _profileBookmarkMenuBridgeMap.end() &&
+  if (it != _profileBookmarkMenuBridgeMap.end() && !isOffTheRecord &&
       (!base::FeatureList::IsEnabled(features::kDestroyProfileOnBrowserClose) ||
        (it->second->GetProfile() && !isOffTheRecord))) {
     // Clean up the dangling Profile* in |_profileBookmarkMenuBridgeMap|.
