diff --git a/chrome/browser/ui/omnibox/chrome_omnibox_client.h b/chrome/browser/ui/omnibox/chrome_omnibox_client.h
index aa3b72b84cd214c8afbab42372462f87518214b6..bac1f2376d9124b17cbb79da71dc2b12f75f8362 100644
--- a/chrome/browser/ui/omnibox/chrome_omnibox_client.h
+++ b/chrome/browser/ui/omnibox/chrome_omnibox_client.h
@@ -32,7 +32,7 @@ namespace omnibox {
 class OmniboxPopupCloser;
 }  // namespace omnibox
 
-class ChromeOmniboxClient final : public OmniboxClient {
+class ChromeOmniboxClient : public OmniboxClient {
  public:
   ChromeOmniboxClient(LocationBar* location_bar,
                       Browser* browser,
