diff --git a/chrome/browser/ui/android/toolbar/BUILD.gn b/chrome/browser/ui/android/toolbar/BUILD.gn
index 1fb5f286f17044c38c428b198daf32311d89a7d4..14260a2a236f6de3029adf5d6f2ce31521adaa08 100644
--- a/chrome/browser/ui/android/toolbar/BUILD.gn
+++ b/chrome/browser/ui/android/toolbar/BUILD.gn
@@ -14,6 +14,7 @@ android_library("variation_utils_java") {
   ]
   deps = [
     "//base:build_info_java",
+    "//brave/browser/ui/android/bottombar:prefs_java",
     "//chrome/browser/flags:java",
     "//third_party/androidx:androidx_annotation_annotation_java",
     "//ui/android:ui_no_recycler_view_java",
@@ -226,6 +227,7 @@ android_library("core_java") {
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/HeightTransitionHandler.java",
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/TabStripTransitionCoordinator.java",
   ]
+  sources += brave_browser_ui_android_toolbar_sources
 
   deps = [
     ":adaptive_features_java",
@@ -442,6 +444,7 @@ android_resources("java_resources") {
   ]
 
   deps = [
+    "//brave/browser/ui/android/toolbar:java_resources",
     "//chrome/app:java_strings_grd",
     "//chrome/browser/ui/android/bars_common:java_resources",
     "//chrome/browser/ui/android/strings:ui_strings_grd",
