diff --git a/chrome/browser/ui/webui/unexportable_keys_internals/unexportable_keys_internals_handler.cc b/chrome/browser/ui/webui/unexportable_keys_internals/unexportable_keys_internals_handler.cc
index 96b4b4032063149643b6f6f4bb6d8c67eb383eea..b14f61e3154568a41d39495a520f3571b3531ecf 100644
--- a/chrome/browser/ui/webui/unexportable_keys_internals/unexportable_keys_internals_handler.cc
+++ b/chrome/browser/ui/webui/unexportable_keys_internals/unexportable_keys_internals_handler.cc
@@ -32,6 +32,8 @@ std::string_view GetAlgorithmName(
       return "ECDSA_SHA256";
     case crypto::SignatureVerifier::SignatureAlgorithm::RSA_PSS_SHA256:
       return "RSA_PSS_SHA256";
+    case crypto::SignatureVerifier::SignatureAlgorithm::ECDSA_SHA384:
+      return "ECDSA_SHA384";
   }
 }
 
