diff --git a/chrome/browser/ui/omnibox/omnibox_next_features.cc b/chrome/browser/ui/omnibox/omnibox_next_features.cc
index 79cfa8e7ef6285d9137a7d69e56b73b097e8ca48..82c09738ec0d32297e0cbff1a41b27881d705c22 100644
--- a/chrome/browser/ui/omnibox/omnibox_next_features.cc
+++ b/chrome/browser/ui/omnibox/omnibox_next_features.cc
@@ -38,11 +38,13 @@ namespace omnibox {
 namespace internal {
 
 // If enabled, shows the omnibox suggestions in the popup in WebUI.
-BASE_FEATURE(kWebUIOmniboxPopup, ENABLED);
+// kWebUIOmniboxPopup feature state is enforced via plaster rewrite.
+BASE_FEATURE(kWebUIOmniboxPopup, DISABLED);
 
 // If enabled, Omnibox popup will transition to AI-Mode with the compose-box
 // panel taking up the whole of the popup, covering the location bar completely.
-BASE_FEATURE(kWebUIOmniboxAimPopup, ENABLED);
+// kWebUIOmniboxAimPopup feature state is enforced via plaster rewrite.
+BASE_FEATURE(kWebUIOmniboxAimPopup, DISABLED);
 
 // If enabled, the Omnibox Popup will enable a different UI state when on a
 // webpage.
