diff --git a/chrome/android/java_sources.gni b/chrome/android/java_sources.gni
index 3af185eb2d04268de73a444c9daea6fd7021118d..092d1188aa2531403d9a37eda8f82856e5140f24 100644
--- a/chrome/android/java_sources.gni
+++ b/chrome/android/java_sources.gni
@@ -25,6 +25,7 @@ import("//components/offline_pages/buildflags/features.gni")
 import("//device/vr/buildflags/buildflags.gni")
 
 chrome_java_sources += public_autofill_assistant_java_sources
+chrome_java_sources += brave_java_sources
 
 if (enable_vr) {
   chrome_java_sources += [ "java/src/org/chromium/chrome/browser/component_updater/VrAssetsComponentInstaller.java" ]
