diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
index 0ee14007ed077f7073a02616a3da325535e5648d..5c67971645c05af77487db4fb7957ff57b6d8d0a 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
@@ -240,6 +240,7 @@ void AvatarToolbarButton::NotifyClick(const ui::Event& event) {
 }
 
 base::string16 AvatarToolbarButton::GetAvatarTooltipText() const {
+  GET_AVATAR_TOOLTIP_TEXT_
   switch (delegate_->GetState()) {
     case State::kIncognitoProfile:
       return l10n_util::GetStringUTF16(IDS_AVATAR_BUTTON_INCOGNITO_TOOLTIP);
