diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index 496af1a2c53e9c67cc477de93c81e72f04dfa5f7..8ccdc32264edbee1307d60b3d7b933dc90ff6ff6 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -482,4 +482,5 @@ build_webui("build") {
       "lazy_load.rollup.js|lazy_load.js",
     ]
   }
+  extra_build_ts_in_files = brave_deps_chrome_browser_resources_settings_in_files extra_build_ts_extra_deps = brave_deps_chrome_browser_resources_settings_extra_deps extra_build_grd_deps = [ "//brave/browser/resources/settings:resources" ]
 }
