diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index f5ca2195a1784882918d1e78e7542cebe2ed06d3..84a9753cef49cf73da74cedebfcf30bbcd223bc7 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -467,6 +467,7 @@ void ToolbarView::Init() {
         container_view_->AddChildView(std::move(send_tab_to_self_button));
   }
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = container_view_->AddChildView(
       std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
