diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
index 8076cdd859d4acd4e96e9ebd7ad1788de1fee901..11e0ba1f5aaacff1bf054a75bce6d4379901cbcd 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
@@ -451,6 +451,7 @@ void AvatarToolbarButton::MaybeHideIdentityAnimation() {
 }
 
 base::string16 AvatarToolbarButton::GetAvatarTooltipText() const {
+  GET_AVATAR_TOOLTIP_TEXT_
   switch (GetState()) {
     case State::kIncognitoProfile:
       return l10n_util::GetStringUTF16(IDS_AVATAR_BUTTON_INCOGNITO_TOOLTIP);
