diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index c63ccb48abb54badff57a42c16009d4a20282030..adbcb5b47aad9d535c2bf87a73b64f275367e0fe 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -215,6 +215,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
