diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 673c4d84cab669aac6b2b78c6292dc46892314b9..0a3177bf12b03da6c713a1c91d9e8f90dd3e08e1 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2691,6 +2691,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
