diff --git a/chrome/installer/mac/signing/model.py b/chrome/installer/mac/signing/model.py
index 46e437e71537f234ab0f8a7053da59f304cf722a..c4f35fdfd78df691b890137df7f6ac965503d18a 100644
--- a/chrome/installer/mac/signing/model.py
+++ b/chrome/installer/mac/signing/model.py
@@ -424,7 +424,7 @@ class Paths(object):
         Returns:
             Path to the packaging directory.
         """
-        return os.path.join(self.input, '{} Packaging'.format(config.product))
+        return os.path.join(self.input, '{}_Packaging'.format(config.product.replace(' ', '_')))
 
     def replace_work(self, new_work):
         """Creates a new Paths with the same input and output directories, but
