diff --git a/chrome/browser/sessions/session_service.cc b/chrome/browser/sessions/session_service.cc
index b40fd2fe87b21b10baaeddc8689bf9402ee0a27a..6745449fca4191ebb1ebbbe73827bdc31aa13ab7 100644
--- a/chrome/browser/sessions/session_service.cc
+++ b/chrome/browser/sessions/session_service.cc
@@ -394,6 +394,7 @@ void SessionService::TabClosed(SessionID window_id, SessionID tab_id) {
     ScheduleCommand(sessions::CreateTabClosedCommand(tab_id));
     if (!window_closing_ids_.contains(window_id) && IsOnlyOneTabLeft()) {
       // This is the last tab in the last tabbed browser.
+      BRAVE_SESSION_SERVICE_TAB_CLOSED
       has_open_trackable_browsers_ = false;
     }
   }
