diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index 52f14726f3d552ce6745007354d3512bcca003a6..a792b8e913d2794c0909843a4c95ffc0ee3aa926 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -329,7 +329,7 @@ Browser* Browser::Create(const CreateParams& params) {
   CHECK_EQ(CreationStatus::kOk,
            GetBrowserWindowCreationStatusForProfile(*params.profile));
 
-  std::unique_ptr<Browser> browser = base::WrapUnique(new Browser(params));
+  std::unique_ptr<Browser> browser = base::WrapUnique(new BraveBrowser(params));
   Browser* const browser_ptr = browser.get();
   BrowserManagerServiceFactory::GetForProfile(params.profile)
       ->AddBrowser(std::move(browser));
@@ -346,7 +346,7 @@ std::unique_ptr<Browser> Browser::DeprecatedCreateOwnedForTesting(
   CHECK_EQ(CreationStatus::kOk,
            GetBrowserWindowCreationStatusForProfile(*params.profile));
 
-  std::unique_ptr<Browser> browser = base::WrapUnique(new Browser(params));
+  std::unique_ptr<Browser> browser = base::WrapUnique(new BraveBrowser(params));
   BrowserManagerServiceFactory::GetForProfile(params.profile)
       ->AddBrowserForTesting(browser.get());
   return browser;
@@ -357,8 +357,8 @@ Browser::Browser(const CreateParams& params)
       profile_(params.profile),
       window_(nullptr),
       tab_strip_model_delegate_(
-          std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
-      tab_strip_model_(std::make_unique<TabStripModel>(
+          std::make_unique<chrome::BraveTabStripModelDelegate>(this)),
+      tab_strip_model_(std::make_unique<BraveTabStripModel>(
           tab_strip_model_delegate_.get(),
           params.profile,
           // Tab groups are disabled for app browsers.
