diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 6dd44c495824833aecddbb76582836ffb7c5a593..1106dbc7e077f424635d0f0229f89cda7a53cc30 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3819,6 +3819,7 @@ const FeatureEntry::FeatureVariation kPasswordGenerationExperimentVariations[] =
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
