diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index ecae3d0e0ca718740935285addf6fc8680e70d55..0403882b162b14049cacfe92ad278c23b07c0903 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -216,6 +216,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
