diff --git a/chrome/installer/setup/install_worker.cc b/chrome/installer/setup/install_worker.cc
index feeca02961ebdec4f66112ab7cf530c3a539858f..e8f67a8586474ba01a46e03c37d2b5b7b396bee4 100644
--- a/chrome/installer/setup/install_worker.cc
+++ b/chrome/installer/setup/install_worker.cc
@@ -107,6 +107,7 @@ void AddInstallerCopyTasks(const InstallParams& install_params,
     base::FilePath active_setup_exe(installer_dir.Append(kActiveSetupExe));
     install_list->AddCopyTreeWorkItem(setup_path, active_setup_exe, temp_path);
   }
+  BRAVE_ADD_INSTALLER_COPY_TASKS
 }
 
 // A callback invoked by |work_item| that adds firewall rules for Chrome. Rules
@@ -217,6 +218,7 @@ void AddChromeWorkItems(const InstallParams& install_params,
         installer_state.GetInstallerDirectory(current_version));
     base::FilePath old_archive(
         old_installer_dir.Append(installer::kChromeArchive));
+    BRAVE_MAYBE_ABORT_ADD_CHROME_WORK_ITEMS
     auto* delete_old_archive_work_item =
         install_list->AddDeleteTreeWorkItem(old_archive, temp_path);
     // Don't cause failure of |install_list| if this WorkItem fails.
