diff --git a/chrome/browser/ui/views/frame/multi_contents_view.h b/chrome/browser/ui/views/frame/multi_contents_view.h
index ac16e8a8bd0fbdb4ab9e9fcb5a08d39972fec576..922e1c0fb0ec366721f3169ed14d83fb359922ce 100644
--- a/chrome/browser/ui/views/frame/multi_contents_view.h
+++ b/chrome/browser/ui/views/frame/multi_contents_view.h
@@ -20,7 +20,7 @@
 class BrowserView;
 class ContentsWebView;
 class MultiContentsDropTargetView;
-class MultiContentsResizeArea;
+// class MultiContentsResizeArea;
 class MultiContentsViewDelegate;
 class MultiContentsViewDropTargetController;
 class MultiContentsViewMiniToolbar;
