diff --git a/chrome/browser/password_manager/android/BUILD.gn b/chrome/browser/password_manager/android/BUILD.gn
index ba75d46392c03af28442703697b8a48b6ab72649..c3c24ad521e5787dae463ab8ce9b476c5e473907 100644
--- a/chrome/browser/password_manager/android/BUILD.gn
+++ b/chrome/browser/password_manager/android/BUILD.gn
@@ -196,6 +196,7 @@ android_library("java") {
   ]
 
   resources_package = "org.chromium.chrome.browser.password_manager"
+  import("//brave/browser/password_manager/android/java_sources.gni") sources += brave_browser_password_manager_java_sources deps += brave_browser_password_manager_deps
 }
 
 generate_jni("jni_headers") {
@@ -216,6 +217,7 @@ generate_jni("jni_headers") {
     "java/src/org/chromium/chrome/browser/password_manager/PasswordStoreCredential.java",
     "java/src/org/chromium/chrome/browser/password_manager/PasswordSyncControllerDelegateBridgeImpl.java",
   ]
+  import("//brave/browser/password_manager/android/java_sources.gni") sources += brave_browser_password_manager_jni_headers
 }
 
 android_library("utils_java") {
