diff --git a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
index 66c5983b2ff950b0715692d0dac3bb0a9a4f5c38..176403fa476a4e52724d515d11f0c6784fc0163e 100644
--- a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
+++ b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
@@ -464,6 +464,7 @@ class AppMenu implements OnKeyListener {
                         anchorView,
                         anchorViewOffset);
 
+        if (this instanceof BraveAppMenu) ((BraveAppMenu) this).updatePopup(popup, isByPermanentButton);
         popup.setHeight(calculateMenuHeight());
 
         int[] popupPosition =
