diff --git a/chrome/installer/linux/common/wrapper b/chrome/installer/linux/common/wrapper
index aaa46bf71f6a3..42a98a91438fd 100755
--- a/chrome/installer/linux/common/wrapper
+++ b/chrome/installer/linux/common/wrapper
@@ -27,6 +27,22 @@ fi
 
 export CHROME_VERSION_EXTRA="@@CHANNEL@@"
 
+# Add command-line options from a config file. Empty lines and #-comments are ignored.
+cfg_channel="$([[ "$CHROME_VERSION_EXTRA" == stable ]] && echo || echo "$CHROME_VERSION_EXTRA-")"
+cfg_file="${XDG_CONFIG_HOME:-"$HOME/.config"}/brave-${cfg_channel}flags.conf"
+cfg_options=()
+if [[ -f "$cfg_file" ]]; then
+  while read -r line; do
+    read -ra line_items <<<"${line%%[[:space:]]#*}"
+    case "${line_items[*]}" in
+      "#"*) ;;
+      --*=\"*\"|--*=\'*\') IFS="=" read -r key val <<<"${line_items[*]}"
+         cfg_options+=("$key=${val:1:${#val}-2}");;
+      *) cfg_options+=("${line_items[@]}");;
+    esac
+  done <"$cfg_file"
+fi
+
 # We don't want bug-buddy intercepting our crashes. http://crbug.com/24120
 export GNOME_DISABLE_CRASH_DIALOG=SET_BY_GOOGLE_CHROME
 
@@ -37,4 +53,4 @@ exec > >(exec cat)
 exec 2> >(exec cat >&2)
 
 # Note: exec -a below is a bashism.
-exec -a "$0" "$HERE/@@PROGNAME@@" "$@"
+exec -a "$0" "$HERE/@@PROGNAME@@" "${cfg_options[@]}" "$@"
