diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index d5cee2c9946ad6db9d76011f36c9ac003b996edd..201bab430d5bc9be4bdb8adb9571ad1502059c5d 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4896,6 +4896,7 @@ const FeatureEntry::FeatureVariation kMobilePromoOnDesktopVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
