diff --git a/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc b/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc
index d29dc6e3fb26d542b0a9ccc1e8c1d455b5b8bb3a..898825514bdbc93e1cdbccad8866ad8c9bf96071 100644
--- a/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc
+++ b/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc
@@ -1414,6 +1414,7 @@ void ChromeWebUIControllerFactory::GetFaviconForURL(
 
 // static
 ChromeWebUIControllerFactory* ChromeWebUIControllerFactory::GetInstance() {
+  BRAVE_CHROME_WEBUI_CONTROLLER_FACTORY
   return base::Singleton<ChromeWebUIControllerFactory>::get();
 }
 
