diff --git a/chrome/updater/branding.gni b/chrome/updater/branding.gni
index 50e9e0c40a411d9537752496e9261a2519d50e41..5375587caff19f8fe0419ed654d0fe68f26deafa 100644
--- a/chrome/updater/branding.gni
+++ b/chrome/updater/branding.gni
@@ -108,3 +108,4 @@ if (is_chrome_branded) {
 # Chrome and Chromium share the same endpoints for now.
 update_check_url = "https://update.googleapis.com/service/update2/json"
 updater_event_logging_url = "https://play.googleapis.com/log"
+import("//brave/chromium_src/chrome/updater/branding.gni") forward_variables_from(brave_updater_branding, "*")
