diff --git a/chrome/browser/ui/views/tabs/horizontal_tab_style_views.cc b/chrome/browser/ui/views/tabs/horizontal_tab_style_views.cc
index 4851028593cdb46210a36c7083ce2b16d747ead2..25f33b1e379e90c082e7cfa1b78388c1706c364d 100644
--- a/chrome/browser/ui/views/tabs/horizontal_tab_style_views.cc
+++ b/chrome/browser/ui/views/tabs/horizontal_tab_style_views.cc
@@ -219,6 +219,7 @@ SkPath HorizontalTabStyleViews::GetPath(TabStyle::PathType path_type,
     extension_corner_radius -= 0.5f * stroke_adjustment;
   }
 
+  extension_corner_radius = 0;
   float left_extension_corner_radius = extension_corner_radius;
   if (delegate_->IsLeftSplitTab()) {
     top_right_corner_radius = 0;
@@ -835,14 +836,18 @@ void HorizontalTabStyleViews::PaintSeparators(gfx::Canvas* canvas) const {
                                                    SK_AlphaOPAQUE));
   };
 
+  // Even if |separator_radius| becomes 1 native pixel the 'roundedness'
+  // will be approximated with color, although extremely subtle and
+  // likely only on screens >= 2x (i.e. separator width is 2+px)!
+  const int separator_radius = separator_bounds.leading.width() / 2;
   cc::PaintFlags flags;
   flags.setAntiAlias(true);
   flags.setColor(separator_color(separator_opacities.left));
   canvas->DrawRoundRect(separator_bounds.leading,
-                        tab_style()->GetSeparatorCornerRadius() * scale, flags);
+                        separator_radius, flags);
   flags.setColor(separator_color(separator_opacities.right));
   canvas->DrawRoundRect(separator_bounds.trailing,
-                        tab_style()->GetSeparatorCornerRadius() * scale, flags);
+                        separator_radius, flags);
 }
 
 BrowserFrameView* HorizontalTabStyleViews::GetBrowserFrameView() const {
