diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 09a55515091dcf8acdb23023aad8d6eb8d16de3c..1395ad873df1b6af651bed8d4564a8e46f8f975b 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -254,6 +254,7 @@ static_library("utility") {
   if (enable_on_device_translation) {
     deps += [ "//components/on_device_translation/service:on_device_translation_service" ]
   }
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
 }
 
 if (!is_android) {
