diff --git a/chrome/browser/resources/settings/site_settings_page/site_settings_page.js b/chrome/browser/resources/settings/site_settings_page/site_settings_page.js
index 8eb6db6927895617ed92587d62d5c3b5f9ff90bf..4220d89dd46d4eeffbd31414baa9a856b8d16790 100644
--- a/chrome/browser/resources/settings/site_settings_page/site_settings_page.js
+++ b/chrome/browser/resources/settings/site_settings_page/site_settings_page.js
@@ -121,7 +121,9 @@ Polymer({
       [R.SITE_SETTINGS_UNSANDBOXED_PLUGINS, 'unsandboxed-plugins'],
       [R.SITE_SETTINGS_HANDLERS, 'protocol-handlers'],
       [R.SITE_SETTINGS_MIDI_DEVICES, 'midi-devices'],
+      // <if expr="_google_chrome">
       [R.SITE_SETTINGS_ADS, 'ads'],
+      // </if>
       [R.SITE_SETTINGS_ZOOM_LEVELS, 'zoom-levels'],
       [R.SITE_SETTINGS_USB_DEVICES, 'usb-devices'],
       [R.SITE_SETTINGS_PDF_DOCUMENTS, 'pdf-documents'],
