diff --git a/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts b/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts
index c89ef3b498e14fa416b6b34bfc203e66baf026d6..6b4e23d31effd28747b8d06a84da1389eb189c4a 100644
--- a/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts
+++ b/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts
@@ -9,6 +9,7 @@
  */
 
 import 'chrome://resources/cr_elements/icons_lit.html.js';
+import '../brave_overrides/config.js'
 import 'chrome://resources/cr_elements/cr_collapse/cr_collapse.js';
 import 'chrome://resources/cr_elements/cr_expand_button/cr_expand_button.js';
 import 'chrome://resources/cr_elements/cr_link_row/cr_link_row.js';
