diff --git a/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarManager.java b/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarManager.java
index 3121ae7710a4abdeef25062e3aed9a303de9a3c6..c691361fe778f68f8e08dbd61a8490434eae094a 100644
--- a/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarManager.java
+++ b/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarManager.java
@@ -340,7 +340,7 @@ public class SnackbarManager
             boolean viewChanged = true;
             if (mView == null) {
                 mView =
-                        new SnackbarView(
+                        new BraveSnackbarView(
                                 mActivity,
                                 this,
                                 currentSnackbar,
