diff --git a/chrome/browser/ui/sync/one_click_signin_links_delegate_impl.h b/chrome/browser/ui/sync/one_click_signin_links_delegate_impl.h
index f82b74b3db08471f89d72cdb1bf338870dc9efdb..2109cc8d3440da8738a0d6e6ae6321a4b15d58b9 100644
--- a/chrome/browser/ui/sync/one_click_signin_links_delegate_impl.h
+++ b/chrome/browser/ui/sync/one_click_signin_links_delegate_impl.h
@@ -21,8 +21,10 @@ class OneClickSigninLinksDelegateImpl : public OneClickSigninLinksDelegate {
   // OneClickSigninLinksDelegate:
   void OnLearnMoreLinkClicked(bool is_dialog) override;
 
+#if !defined(OS_ANDROID)
   // Browser in which the links should be opened.
   Browser* const browser_;
+#endif
 
   DISALLOW_COPY_AND_ASSIGN(OneClickSigninLinksDelegateImpl);
 };
