diff --git a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h
index 2c3baaa9242958ce243c885a212900e4f6c8dc1f..d772ab34b8dd2b832e22df45c2aa01b5b7aefa1d 100644
--- a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h
+++ b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h
@@ -136,7 +136,7 @@ class TabDragController : public views::WidgetObserver,
   // call, and Liveness::ALIVE if `this` still exists.
   // Note: `dragging_views` must be ordered by their position in the source
   // tabstrip (both visually and in the model).
-  [[nodiscard]] Liveness Init(TabDragContext* source_context,
+  [[nodiscard]] virtual Liveness Init(TabDragContext* source_context,
                               TabSlotView* source_view,
                               const std::vector<TabSlotView*>& dragging_views,
                               const gfx::Point& offset_from_source_view,
@@ -229,6 +229,7 @@ class TabDragController : public views::WidgetObserver,
   const TabDragContext* GetAttachedContext() const override;
 
  private:
+  friend class BraveTabDragController;
   friend class TabDragControllerTest;
 
   class SourceTabStripEmptinessTracker;
@@ -424,7 +425,7 @@ class TabDragController : public views::WidgetObserver,
   // See Detach/Attach for parameter documentation. Transfers ownership of
   // `this` from `attached_context_` (which must own `this`) to
   // `target_context`.
-  void DetachAndAttachToNewContext(ReleaseCapture release_capture,
+  virtual void DetachAndAttachToNewContext(ReleaseCapture release_capture,
                                    TabDragContext* target_context);
 
   // Detaches the tabs being dragged, creates a new Browser to contain them and
@@ -553,7 +554,7 @@ class TabDragController : public views::WidgetObserver,
 
   // Initializes `dragging_tabs_session_`, and performs a first MoveAttached
   // within `attached_context_`.
-  void StartDraggingTabsSession(bool initial_move,
+  virtual void StartDraggingTabsSession(bool initial_move,
                                 gfx::Point start_point_in_screen);
 
   // Calls `SetSelectionFromModel` in the `tab_strip_model`. This centralizes
