diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
index 71c6e3af59e67a67da4257b5a4c96dbc1a9afb83..0ec1bee9a674ee215f020ec278b46d179cbb941f 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
@@ -665,7 +665,7 @@ public abstract class ChromeActivity extends AsyncInitializationActivity
             mBottomContainer = findViewById(R.id.bottom_container);
 
             mSnackbarManager =
-                    new SnackbarManager(
+                    new org.chromium.chrome.browser.ui.messages.snackbar.BraveSnackbarManager(
                             this, mBottomContainer, getWindowAndroid(), getEdgeToEdgeSupplier());
             getInsetObserver().addObserver(mSnackbarManager);
             SnackbarManagerProvider.attach(getWindowAndroid(), mSnackbarManager);
