diff --git a/chrome/browser/ui/webui/side_panel/customize_chrome/customize_toolbar/customize_toolbar_handler.h b/chrome/browser/ui/webui/side_panel/customize_chrome/customize_toolbar/customize_toolbar_handler.h
index 38bfa68c9904f92c9a32e00422623b1bb74db087..0fdffd42a8eb0f883135056e120614cf8353439a 100644
--- a/chrome/browser/ui/webui/side_panel/customize_chrome/customize_toolbar/customize_toolbar_handler.h
+++ b/chrome/browser/ui/webui/side_panel/customize_chrome/customize_toolbar/customize_toolbar_handler.h
@@ -53,6 +53,9 @@ class CustomizeToolbarHandler
  private:
   void OnActionPinnedChanged(actions::ActionId id, std::string_view pref);
   void OnActionItemChanged();
+  void ObserveBraveActions();
+  void OnBraveActionPinnedChanged(
+      side_panel::customize_chrome::mojom::ActionId action_id);
 
   PrefService* prefs() const;
 
