diff --git a/chrome/browser/ui/views/frame/multi_contents_view.h b/chrome/browser/ui/views/frame/multi_contents_view.h
index 12511f171b46863215e3904361dd39f0f8392816..86da601fec22c54ed9731a9209a22a6c276622e9 100644
--- a/chrome/browser/ui/views/frame/multi_contents_view.h
+++ b/chrome/browser/ui/views/frame/multi_contents_view.h
@@ -63,7 +63,7 @@ class MultiContentsView
     int end = 0;
   };
 
-  static constexpr int kSplitViewContentInset = 8;
+  static constexpr int kSplitViewContentInset = 0;
 
   MultiContentsView(BrowserView* browser_view,
                     std::unique_ptr<MultiContentsViewDelegate> delegate);
@@ -71,13 +71,13 @@ class MultiContentsView
   MultiContentsView& operator=(const MultiContentsView&) = delete;
   ~MultiContentsView() override;
 
-  ContentsContainerView* GetActiveContentsContainerView() const;
+  virtual ContentsContainerView* GetActiveContentsContainerView() const;
   ContentsContainerView* GetInactiveContentsContainerView() const;
-  ContentsContainerView* GetContentsContainerViewFor(
+  virtual ContentsContainerView* GetContentsContainerViewFor(
       content::WebContents* web_contents) const;
 
   // Returns the currently active ContentsWebView.
-  ContentsWebView* GetActiveContentsView() const;
+  virtual ContentsWebView* GetActiveContentsView() const;
 
   // Returns the currently inactive ContentsWebView.
   ContentsWebView* GetInactiveContentsView() const;
@@ -87,6 +87,7 @@ class MultiContentsView
   // adjusted for RtL (so e.g. `radii.upper_left()` is actually the corner that
   // displays on the upper left regardless of text direction).
   void SetBackgroundRadii(const gfx::RoundedCornersF& radii);
+  virtual void ResetResizeArea() {}
   const gfx::RoundedCornersF& GetBackgroundRadii() const;
 
   // Returns true if more than one WebContents is displayed.
@@ -129,7 +130,7 @@ class MultiContentsView
 
   // Helper method to execute an arbitrary callback on each visible contents
   // view. Will execute the callback on the active contents view first.
-  void ExecuteOnEachVisibleContentsView(
+  virtual void ExecuteOnEachVisibleContentsView(
       base::RepeatingCallback<void(ContentsWebView*)> callback);
 
   // If in a split view, swaps the positions of the two active tabs.
@@ -216,6 +217,7 @@ class MultiContentsView
       const ContentsContainerView* container) const;
 
  private:
+  friend class BraveMultiContentsView;
   FRIEND_TEST_ALL_PREFIXES(MultiContentsViewBrowserTest, DropTargetLayout);
   FRIEND_TEST_ALL_PREFIXES(MultiContentsViewBrowserTest,
                            LeadingSeparatorLayout);
@@ -257,7 +259,7 @@ class MultiContentsView
 
   int GetInactiveIndex() const;
 
-  void OnWebContentsFocused(views::WebView*);
+  virtual void OnWebContentsFocused(views::WebView*);
   void OnNtpFooterFocused(views::WebView*);
   void OnActorOverlayFocused(views::WebView*);
   void OnReadAnythingOverlayFocused(ContentsContainerView* container,
@@ -274,7 +276,7 @@ class MultiContentsView
   // `MultiContentsView`. Returns 0 if not in a split view.
   int GetMinViewSize(gfx::Rect available_space) const;
 
-  void UpdateContentsBorderAndOverlay();
+  virtual void UpdateContentsBorderAndOverlay();
 
   double CalculateRatioWithSnapPoints(double start_size,
                                       double total_size) const;
