diff --git a/chrome/app/chrome_main.cc b/chrome/app/chrome_main.cc
index 9ff589299bff368efca05c9739031c7f6b63b263..5d23a4c05067f0c5856cd2441e760da40ee2231d 100644
--- a/chrome/app/chrome_main.cc
+++ b/chrome/app/chrome_main.cc
@@ -7,6 +7,7 @@
 #include "base/callback_helpers.h"
 #include "base/command_line.h"
 #include "base/time/time.h"
+#include "brave/app/brave_main_delegate.h"
 #include "build/build_config.h"
 #include "chrome/app/chrome_main_delegate.h"
 #include "chrome/common/buildflags.h"
@@ -56,7 +57,7 @@ int ChromeMain(int argc, const char** argv) {
   install_static::InitializeFromPrimaryModule();
 #endif
 
-  ChromeMainDelegate chrome_main_delegate(
+  BraveMainDelegate chrome_main_delegate(
       base::TimeTicks::FromInternalValue(exe_entry_point_ticks));
   content::ContentMainParams params(&chrome_main_delegate);
 
