diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 7644572ee8fb6ea573d0bb3b2d759e79f7835ade..f00e592caceeb2b13f13d79f924282849d3f80b7 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -455,6 +455,8 @@ test("browser_tests") {
     "//third_party/blink/public:buildflags",
   ]
 
+  deps += [ "//brave/test:brave_browser_tests_deps", ]
+
   data_deps = [
     "//testing/buildbot/filters:browser_tests_filters",
     "//tools/media_engagement_preload:generator",
