diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 18b342e66f11a3c40a2243f424cc2ff7e825aad3..072074be162a9ca28888ab9093030a744d4c20a6 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -234,6 +234,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  import("//brave/renderer/sources.gni") sources += brave_chrome_renderer_sources deps += brave_chrome_renderer_deps public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
