diff --git a/chrome/browser/sync/profile_sync_service_factory.cc b/chrome/browser/sync/profile_sync_service_factory.cc
index 89e796cc53fc107597838419f7ce0c00db05c518..6265dcd01a9c4d803f2c57e02f3fe8e11ecc6b39 100644
--- a/chrome/browser/sync/profile_sync_service_factory.cc
+++ b/chrome/browser/sync/profile_sync_service_factory.cc
@@ -264,7 +264,7 @@ KeyedService* ProfileSyncServiceFactory::BuildServiceInstanceFor(
   }
 
   auto pss =
-      std::make_unique<syncer::ProfileSyncService>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   pss->Initialize();
 
   // Hook PSS into PersonalDataManager (a circular dependency).
