diff --git a/chrome/browser/ui/webui/realbox/realbox_handler.cc b/chrome/browser/ui/webui/realbox/realbox_handler.cc
index f2bc9c8997b3a..fac7fa689b576 100644
--- a/chrome/browser/ui/webui/realbox/realbox_handler.cc
+++ b/chrome/browser/ui/webui/realbox/realbox_handler.cc
@@ -457,7 +457,7 @@ std::string GetBase64UrlVariations(Profile* profile) {
 
 // TODO(crbug.com/1431513): Consider inheriting from `ChromeOmniboxClient`
 //  to avoid reimplementation of methods like `OnBookmarkLaunched`.
-class RealboxOmniboxClient final : public OmniboxClient {
+class RealboxOmniboxClient final : public BraveOmniboxClient {
  public:
   RealboxOmniboxClient(LocationBarModel* location_bar_model,
                        Profile* profile,
