diff --git a/chrome/browser/ui/views/bubble/webui_bubble_manager.h b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
index a554528ce9c2795bd70c4d2fadf887657a3418bf..7a8311233c0d6104cccdf1c75821117e0f0fbc2b 100644
--- a/chrome/browser/ui/views/bubble/webui_bubble_manager.h
+++ b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
@@ -162,7 +162,7 @@ class WebUIBubbleManagerImpl : public WebUIBubbleManager {
         task_manager_string_id_(task_manager_string_id) {}
   ~WebUIBubbleManagerImpl() override = default;
 
- private:
+ protected:
   // WebUIBubbleManager:
   void MaybeInitPersistentRenderer() override;
   base::WeakPtr<WebUIBubbleDialogView> CreateWebUIBubbleDialog(
@@ -247,6 +247,7 @@ WebUIBubbleManagerImpl<T>::CreateWebUIBubbleDialog(
 
   auto bubble_view = std::make_unique<WebUIBubbleDialogView>(
       anchor_view_, contents_wrapper->GetWeakPtr(), anchor, arrow);
+    BRAVE_WEBUI_BUBBLE_MANAGER_T_CREATE_WEB_UI_BUBBLE_DIALOG
 
   if (!widget_initialization_callback_.is_null()) {
     bubble_view->RegisterWidgetInitializedCallback(
