diff --git a/chrome/browser/ui/views/translate/translate_bubble_view_unittest.cc b/chrome/browser/ui/views/translate/translate_bubble_view_unittest.cc
index 40b0702f3c99824d5aac1bdf4a64bed5ead75039..5850fc897da7465a0b5e83d1710c39d961843af0 100644
--- a/chrome/browser/ui/views/translate/translate_bubble_view_unittest.cc
+++ b/chrome/browser/ui/views/translate/translate_bubble_view_unittest.cc
@@ -242,6 +242,7 @@ class TranslateBubbleViewTest : public ChromeViewsTestBase {
 
  protected:
   void SetUp() override {
+    scoped_feature_list_.InitFromCommandLine("BraveEnableAutoTranslate", "");
     ChromeViewsTestBase::SetUp();
 
     // The bubble needs the parent as an anchor.
