diff --git a/chrome/browser/ui/browser_navigator.cc b/chrome/browser/ui/browser_navigator.cc
index e805d59f87b459f106706b41be98b60aaa5786c8..047d7c3db07114f0e0bf98613090c8f2f8cfe1d2 100644
--- a/chrome/browser/ui/browser_navigator.cc
+++ b/chrome/browser/ui/browser_navigator.cc
@@ -613,6 +613,7 @@ base::WeakPtr<content::NavigationHandle> Navigate(NavigateParams* params) {
     // Block any navigation requests in locked fullscreen mode.
     return nullptr;
   }
+  BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL
 
   // Open System Apps in their standalone window if necessary.
   // TODO(crbug.com/1096345): Remove this code after we integrate with intent
@@ -984,6 +985,7 @@ bool IsHostAllowedInIncognito(const GURL& url) {
   if (scheme != content::kChromeUIScheme) {
     return true;
   }
+  BRAVE_IS_HOST_ALLOWED_IN_INCOGNITO
 
   if (host == chrome::kChromeUIChromeSigninHost) {
 #if BUILDFLAG(IS_WIN)
