diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index 4c3f9b2c3fb5eba1ae4447882b8b4b204d096ffc..c8c3cc1bc86b59157efea1e697ab4f915297f4c2 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -374,6 +374,7 @@ content::WebUIDataSource* CreateExtensionsSource(Profile* profile,
                      base::FeatureList::IsEnabled(
                          extensions_features::kExtensionsMenuAccessControl));
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
