diff --git a/chrome/browser/notifications/BUILD.gn b/chrome/browser/notifications/BUILD.gn
index d541e7c3e64d76c7fb997e0c753d603257b15953..73687fe1cb60ea65d008797e0164b2558b2f293b 100644
--- a/chrome/browser/notifications/BUILD.gn
+++ b/chrome/browser/notifications/BUILD.gn
@@ -462,6 +462,7 @@ if (is_android) {
       "//third_party/jni_zero:jni_zero_java",
       "//ui/android:ui_no_recycler_view_java",
     ]
+    deps += [ "//brave/browser/notifications/android:brave_java", ]
 
     srcjar_deps = [
       ":jni_headers",
