diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index 25969faad4be7b51489b844cc2529d928b73108d..017fd2409911502e9a79064e03d217465f27123b 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -149,6 +149,7 @@ source_set("internal") {
   if (enable_extensions_core) {
     deps += [ "//chrome/browser/extensions" ]
   }
+  import("//brave/browser/ui/browser_window/internal/sources.gni") deps += brave_browser_window_internal_deps
 }
 
 source_set("unit_tests") {
