diff --git a/chrome/browser/ui/views/side_panel/side_panel_coordinator.cc b/chrome/browser/ui/views/side_panel/side_panel_coordinator.cc
index 0c432445a4ae9d72727352e1bb4efd395796302e..f44b8ef109d8e2a994e9105e099f3144930b6024 100644
--- a/chrome/browser/ui/views/side_panel/side_panel_coordinator.cc
+++ b/chrome/browser/ui/views/side_panel/side_panel_coordinator.cc
@@ -175,6 +175,7 @@ SidePanelCoordinator::~SidePanelCoordinator() {
 void SidePanelCoordinator::Show(
     absl::optional<SidePanelEntry::Id> entry_id,
     absl::optional<SidePanelUtil::SidePanelOpenTrigger> open_trigger) {
+  BRAVE_SIDE_PANEL_COORDINATOR_SHOW
   if (entry_id.has_value()) {
     Show(SidePanelEntry::Key(entry_id.value()), open_trigger);
   } else {
@@ -511,6 +512,7 @@ std::unique_ptr<views::View> SidePanelCoordinator::CreateHeader() {
   header_combobox_->SetFocusBehavior(views::View::FocusBehavior::ALWAYS);
   header_close_button->SetFocusBehavior(views::View::FocusBehavior::ALWAYS);
 
+  BRAVE_SIDE_PANEL_COORDINATOR_CREATE_HEADER
   return header;
 }
 
