diff --git a/chrome/browser/ui/window_feature_controller/window_feature_controller.cc b/chrome/browser/ui/window_feature_controller/window_feature_controller.cc
index ec7bd3ea6cd00a5093b9bac2cfca9fe1a0bc4bfb..b7a5ba43bcca85e128ed9607ecd59b78b139b36c 100644
--- a/chrome/browser/ui/window_feature_controller/window_feature_controller.cc
+++ b/chrome/browser/ui/window_feature_controller/window_feature_controller.cc
@@ -17,12 +17,14 @@ WindowFeatureController::WindowFeatureController(
     web_app::AppBrowserController* app_controller,
     BrowserWindowInterface::Type browser_type,
     bool is_trusted_source,
-    ui::UnownedUserDataHost& host)
+    ui::UnownedUserDataHost& host,
+    BrowserWindowInterface& browser)
     : fullscreen_controller_(CHECK_DEREF(fullscreen_controller)),
       app_controller_(app_controller),
       browser_type_(browser_type),
       is_trusted_source_(is_trusted_source),
-      scoped_unowned_user_data_(host, *this) {}
+      scoped_unowned_user_data_(host, *this),
+      browser_(browser) {}
 
 WindowFeatureController::~WindowFeatureController() = default;
 
@@ -77,6 +79,14 @@ bool WindowFeatureController::UsesImmersiveFullscreenTabbedMode() const {
 bool WindowFeatureController::NormalBrowserSupportsWindowFeature(
     WindowFeature feature,
     bool check_can_support) const {
+  #if BUILDFLAG(IS_WIN)
+  if (feature == WindowFeature::kFeatureTitleBar) {
+    // In case of vertical tab strip is allowed, we need to have ability to
+    // show title bar on Windows.
+    return tabs::utils::ShouldShowBraveVerticalTabs(&browser_.get()) &&
+           tabs::utils::ShouldShowWindowTitleForVerticalTabs(&browser_.get());
+  }
+  #endif
   switch (feature) {
     case WindowFeature::kFeatureBookmarkBar:
       return true;
