diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index a7775ea16b090ee0d94a776b2c735a903cc0cf19..46d35285ac6ec7856bb90ba00b5414b505c006db 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -91,6 +91,7 @@ source_set("channel_info") {
   } else if (is_posix) {
     sources += [ "channel_info_posix.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -272,6 +273,7 @@ static_library("common_lib") {
     public_deps += [ "//components/cdm/common" ]
   }
 
+  import("//brave/common/sources.gni") sources += brave_chrome_common_sources deps += brave_chrome_common_deps public_deps += brave_chrome_common_public_deps
   if (enable_extensions_core) {
     public_deps += [
       "//chrome/common/extensions",
@@ -473,6 +475,7 @@ static_library("non_code_constants") {
   if (is_win) {
     sources += [ "chrome_icon_resources_win.h" ]
   }
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
