diff --git a/chrome/browser/ui/views/frame/browser_view.h b/chrome/browser/ui/views/frame/browser_view.h
index 93e2cd382a5781ab227cf645dc0b54e8a40c795f..902193b63f67cb8720c4bff108fdcd46c28a1fe2 100644
--- a/chrome/browser/ui/views/frame/browser_view.h
+++ b/chrome/browser/ui/views/frame/browser_view.h
@@ -704,6 +704,7 @@ class BrowserView : public BrowserWindow,
       const std::map<std::string, base::Value>& extra_data) override;
 #endif
 
+  BRAVE_BROWSER_VIEW_H
  private:
   // Do not friend BrowserViewLayout. Use the BrowserViewLayoutDelegate
   // interface to keep these two classes decoupled and testable.
