diff --git a/chrome/browser/resources/settings/site_settings/site_settings_prefs_browser_proxy.ts b/chrome/browser/resources/settings/site_settings/site_settings_prefs_browser_proxy.ts
index b5d450a6f551b4c7ba678ab353a7adf364a80645..3b5c74badae9e0f561dea752a3577023334d366e 100644
--- a/chrome/browser/resources/settings/site_settings/site_settings_prefs_browser_proxy.ts
+++ b/chrome/browser/resources/settings/site_settings/site_settings_prefs_browser_proxy.ts
@@ -83,6 +83,7 @@ export interface RawSiteException {
   description?: string;
   setting: ContentSetting;
   source: SiteSettingSource;
+  braveCookieType?: undefined | "shields down" | "shields settings" | "google sign-in";  // See BraveGetExceptionForPage (site_settings_helper.cc)
 }
 
 /**
@@ -100,6 +101,7 @@ export interface SiteException {
   description?: string;
   enforcement: chrome.settingsPrivate.Enforcement|null;
   controlledBy: chrome.settingsPrivate.ControlledBy;
+  braveCookieType?: RawSiteException['braveCookieType'];
 }
 
 /**
