diff --git a/chrome/browser/plugins/chrome_plugin_service_filter.cc b/chrome/browser/plugins/chrome_plugin_service_filter.cc
index 55801ac0fee766a5f46314137d4745ff8a1d7f24..9acdabbe23d440d7468cbc9ef3b1e589f74db12c 100644
--- a/chrome/browser/plugins/chrome_plugin_service_filter.cc
+++ b/chrome/browser/plugins/chrome_plugin_service_filter.cc
@@ -51,7 +51,7 @@ class ProfileContentSettingObserver : public content_settings::Observer {
     PluginService::GetInstance()->PurgePluginListCache(profile_, false);
 
     const GURL primary(primary_pattern.ToString());
-    if (primary.is_valid()) {
+    if (primary.is_valid() && resource_identifier.length() == 0) {
       DCHECK_EQ(ContentSettingsPattern::Relation::IDENTITY,
                 ContentSettingsPattern::Wildcard().Compare(secondary_pattern));
       PluginUtils::RememberFlashChangedForSite(map, primary);
