diff --git a/chrome/browser/bookmarks/android/BUILD.gn b/chrome/browser/bookmarks/android/BUILD.gn
index 0245a2f9d39325ad111293f8dbe5b9485e516383..c965d3965b869de47f3545e4cf01740b32c24e14 100644
--- a/chrome/browser/bookmarks/android/BUILD.gn
+++ b/chrome/browser/bookmarks/android/BUILD.gn
@@ -38,6 +38,7 @@ source_set("impl") {
     "//components/query_parser",
     "//components/undo",
   ]
+  deps += brave_browser_bookmarks_android_impl_deps
 }
 
 android_resources("java_resources") {
@@ -149,6 +150,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/bookmarks/bar/BookmarkBarViewBinder.java",
     "java/src/org/chromium/chrome/browser/bookmarks/bar/BookmarkBarVisibilityProvider.java",
   ]
+  sources += brave_browser_bookmarks_android_sources
 
   deps = [
     ":java_resources",
