diff --git a/chrome/browser/ui/browser_commands.cc b/chrome/browser/ui/browser_commands.cc
index 5131a3cc83c5015aca0855cf4458a080b54e8c3d..86e47351e975c2163786a1fef290cd90ff31eed4 100644
--- a/chrome/browser/ui/browser_commands.cc
+++ b/chrome/browser/ui/browser_commands.cc
@@ -464,7 +464,12 @@ void MoveTabsToWindowImpl(BrowserWindowInterface* source,
     } else if (auto collection =
                    std::get_if<std::unique_ptr<DetachedTabCollection>>(
                        &tab_or_collection)) {
-      if (std::holds_alternative<std::unique_ptr<tabs::TabGroupTabCollection>>(
+      if (std::holds_alternative<
+              std::unique_ptr<tabs::TreeTabNodeTabCollection>>(
+              collection->get()->collection_)) {
+        target_model->InsertDetachedTreeTabNodeAt(std::move(*collection),
+                                                  target_model->count());
+      } else if (std::holds_alternative<std::unique_ptr<tabs::TabGroupTabCollection>>(
               collection->get()->collection_)) {
         target_model->InsertDetachedTabGroupAt(std::move(*collection),
                                                target_model->count());
@@ -1130,7 +1135,7 @@ void Reload(BrowserWindowInterface* browser,
   ReloadInternal(browser, disposition, false);
 }
 
-void ReloadBypassingCache(BrowserWindowInterface* browser,
+void ReloadBypassingCache_ChromiumImpl(BrowserWindowInterface* browser,
                           WindowOpenDisposition disposition) {
   base::RecordAction(UserMetricsAction("ReloadBypassingCache"));
   ReloadInternal(browser, disposition, true);
@@ -1447,6 +1452,7 @@ void CloseTab(BrowserWindowInterface* browser) {
 #endif
 
   ToastController* toast_controller = browser->GetFeatures().toast_controller();
+  toast_controller = nullptr;
   if (!toast_controller) {
     CloseSelectedTabAndRecordTabCountMetric(browser);
     return;
@@ -1775,7 +1781,7 @@ void NewSplitTab(BrowserWindowInterface* browser,
   // the regular NTP which renders special content when in Incognito.
   const GURL new_tab_url = !browser->GetProfile()->IsIncognitoProfile() &&
                                    tab_strip_model->count() > 1
-                               ? GURL(chrome::kChromeUISplitViewNewTabPageURL)
+                               ? GURL(chrome::kChromeUINewTabURL)
                                : chrome::ChromeUINewTabURLAsGURL();
   tab_strip_model->delegate()->AddTabAt(
       new_tab_url, active_index + 1, true,
