diff --git a/chrome/browser/ui/views/user_education/browser_user_education_service.cc b/chrome/browser/ui/views/user_education/browser_user_education_service.cc
index 865c12b3e545b24fcdef3bf13552c50547d3a196..9b9c62a4ab16a68419f59d841c81b988ded75b38 100644
--- a/chrome/browser/ui/views/user_education/browser_user_education_service.cc
+++ b/chrome/browser/ui/views/user_education/browser_user_education_service.cc
@@ -323,6 +323,7 @@ void MaybeRegisterChromeFeaturePromos(
           feature_engagement::kIPHWebUiHelpBubbleTestFeature)) {
     return;
   }
+  BRAVE_MAYBE_REGISTER_CHROME_FEATURE_PROMOS
 
   // kIPHAutofillCreditCardBenefitFeature:
   registry.RegisterFeature(std::move(
