diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index b1304547ae914cc01a3d76b6c5c855cf16f73def..6c458015f0ca4b8e38367a999b9c89b6d4119173 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2526,6 +2526,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
