diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index bdb3155a8d2a86222dcace1381e094f6157c5c7a..d6f761b1d50087c9828c13e1460a100162b09e6e 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -98,6 +98,7 @@ source_set("internal") {
       deps += [ "//chrome/browser/ui:overscroll_pref_manager" ]
     }
   }
+  deps += [ "//brave/browser/ui/browser_window/internal" ]
 }
 
 source_set("unit_tests") {
