diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarVariationManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarVariationManager.java
index 9bef97dea67e0a10960c523b16d1cfbe3ecd4418..3b89eea859d0eb9dcafb157f579fa7039b24af53 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarVariationManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomToolbarVariationManager.java
@@ -50,6 +50,7 @@ public class BottomToolbarVariationManager {
      *         in the current variation.
      */
     public static boolean isShareButtonOnBottom() {
+        if (BraveBottomToolbarVariationManager.isBraveVariation()) return false;
         return BottomToolbarConfiguration.isBottomToolbarEnabled()
                 && !getVariation().equals(Variations.HOME_SEARCH_TAB_SWITCHER);
     }
@@ -59,6 +60,7 @@ public class BottomToolbarVariationManager {
      *         in portrait mode in the current variation.
      */
     public static boolean isNewTabButtonOnBottom() {
+        if (BraveBottomToolbarVariationManager.isBraveVariation()) return true;
         return BottomToolbarConfiguration.isBottomToolbarEnabled()
                 && getVariation().equals(Variations.NEW_TAB_SEARCH_SHARE);
     }
@@ -68,6 +70,7 @@ public class BottomToolbarVariationManager {
      *         in portrait mode in the current variation.
      */
     public static boolean isMenuButtonOnBottom() {
+        if (BraveBottomToolbarVariationManager.isBraveVariation()) return true;
         // If we don't have variations that put menu on bottom in the future,
         // then this method can be removed.
         return false;
@@ -78,6 +81,7 @@ public class BottomToolbarVariationManager {
      *         in the current variation.
      */
     public static boolean shouldBottomToolbarBeVisibleInOverviewMode() {
+        if (BraveBottomToolbarVariationManager.isBraveVariation()) return true;
         return (getVariation().equals(Variations.NEW_TAB_SEARCH_SHARE)
                        && !StartSurfaceConfiguration.isStartSurfaceEnabled())
                 || ((!TabUiFeatureUtilities.isGridTabSwitcherEnabled()
@@ -99,6 +103,7 @@ public class BottomToolbarVariationManager {
      *         of portrait mode in current variation.
      */
     public static boolean isTabSwitcherOnBottom() {
+        if (BraveBottomToolbarVariationManager.isBraveVariation()) return true;
         return BottomToolbarConfiguration.isBottomToolbarEnabled()
                 && getVariation().equals(Variations.HOME_SEARCH_TAB_SWITCHER);
     }
