diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 8c29ffb482c7d71164603bd039eb393386f12d70..5601eceecaf12cd28499014fdfdf56ce3c251152 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -7745,6 +7745,7 @@ const FeatureEntry kFeatureEntries[] = {
     // "LoginCustomFlags" in tools/metrics/histograms/enums.xml. See "Flag
     // Histograms" in tools/metrics/histograms/README.md (run the
     // AboutFlagsHistogramTest unit test to verify this process).
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 };

 class FlagsStateSingleton : public flags_ui::FlagsState::Delegate {
