diff --git a/chrome/browser/ui/webui/settings/people_handler.cc b/chrome/browser/ui/webui/settings/people_handler.cc
index 6eaf74064b4fcc9457c4f2ff3d7fce15a571a65c..3b9210f9264bea9094de4afa6882269cf172c205 100644
--- a/chrome/browser/ui/webui/settings/people_handler.cc
+++ b/chrome/browser/ui/webui/settings/people_handler.cc
@@ -855,6 +855,7 @@ void PeopleHandler::OnStateChanged(syncer::SyncService* sync_service) {
 
 void PeopleHandler::BeforeUnloadDialogCancelled() {
   // The before unload dialog is only shown during the first sync setup.
+  if (false)
   DCHECK(IdentityManagerFactory::GetForProfile(profile_)->HasPrimaryAccount(
       signin::ConsentLevel::kSync));
   syncer::SyncService* service = GetSyncService();
@@ -904,6 +905,7 @@ base::Value::Dict PeopleHandler::GetSyncStatusDictionary() const {
       service && !disallowed_by_policy && service->IsSetupInProgress() &&
           !service->GetUserSettings()->IsFirstSetupComplete() &&
           identity_manager->HasPrimaryAccount(signin::ConsentLevel::kSync));
+  BRAVE_GET_SYNC_STATUS_DICTIONARY
 
   const SyncStatusLabels status_labels = GetSyncStatusLabels(profile_);
   // TODO(crbug.com/1027467): Consider unifying some of the fields below to
@@ -1066,6 +1068,7 @@ void PeopleHandler::MaybeMarkSyncConfiguring() {
 }
 
 bool PeopleHandler::IsProfileAuthNeededOrHasErrors() {
+  return false;
   return !IdentityManagerFactory::GetForProfile(profile_)->HasPrimaryAccount(
              signin::ConsentLevel::kSync) ||
          SigninErrorControllerFactory::GetForProfile(profile_)->HasError();
