diff --git a/chrome/browser/prefs/chrome_pref_service_factory.cc b/chrome/browser/prefs/chrome_pref_service_factory.cc
index 50cbdfee0a804f1a84c77efaafd41befd4a2c867..6e6503aabf15968682b665134da605db96655b36 100644
--- a/chrome/browser/prefs/chrome_pref_service_factory.cc
+++ b/chrome/browser/prefs/chrome_pref_service_factory.cc
@@ -181,6 +181,7 @@ const auto kTrackedPrefs = std::to_array<prefs::TrackedPreferenceMetadata>({
 
     // See note at top, new items added here also need to be added to
     // histograms.xml's TrackedPreference enum.
+    BRAVE_TRACKED_PREFS_EXTEND
 });
 
 // One more than the last tracked preferences ID above.
