diff --git a/chrome/browser/ui/views/translate/translate_bubble_view.cc b/chrome/browser/ui/views/translate/translate_bubble_view.cc
index 7df689afe0660000064a98670151a0a372aba9e5..946e3c5d6bc4b9853e3fb1f142435990c5315564 100644
--- a/chrome/browser/ui/views/translate/translate_bubble_view.cc
+++ b/chrome/browser/ui/views/translate/translate_bubble_view.cc
@@ -182,7 +182,7 @@ views::Widget* TranslateBubbleView::ShowBubble(
           source_language, target_language));
   std::unique_ptr<TranslateBubbleModel> model(
       new TranslateBubbleModelImpl(step, std::move(ui_delegate)));
-  TranslateBubbleView* view = new TranslateBubbleView(
+  TranslateBubbleView* view = new BRAVE_TRANSLATE_BUBBLE_VIEW_(
       anchor_view, std::move(model), error_type, web_contents);
 
   if (highlighted_button)
