diff --git a/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java b/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java
index 326fe10907ba5ecb53b9a0cdd9977e94d44be74d..efac63c04941893c89a022e1eea4f606c3fb6a07 100644
--- a/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java
+++ b/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java
@@ -393,7 +393,7 @@ public class FeedSurfaceMediator
         // Start is showing, update the value of Pref.ENABLE_SNIPPETS_BY_DSE here. The
         // value should be updated before adding an observer to prevent an extra call of
         // updateContent().
-        getPrefService()
+        if (FeedFeatures.isFeedEnabled(mProfile)) getPrefService()
                 .setBoolean(
                         Pref.ENABLE_SNIPPETS_BY_DSE,
                         !mIsNewTabSearchEngineUrlAndroidEnabled
