diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index cf560fad6d2a2d240dd4d3838252ba136b4dd60b..3ec092f58a7efeefebf5e077c31516525fa5cb39 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4790,6 +4790,7 @@ const FeatureEntry::FeatureVariation kDictationVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
