diff --git a/chrome/browser/ui/views/tabs/tab.cc b/chrome/browser/ui/views/tabs/tab.cc
index 6cfde5828cb0fbe1ea0c39d3bbec1267016b4eb5..f420905f286f5180be1ebcf36aa92b0802584034 100644
--- a/chrome/browser/ui/views/tabs/tab.cc
+++ b/chrome/browser/ui/views/tabs/tab.cc
@@ -402,6 +402,7 @@ void Tab::Layout() {
     int title_right = contents_rect.right();
     if (showing_alert_indicator_) {
       title_right = alert_indicator_button_->x() - after_title_padding;
+      BRAVE_UI_VIEWS_TABS_TAB_ALERT_INDICATOR_POSITION
     } else if (showing_close_button_) {
       // Allow the title to overlay the close button's empty border padding.
       title_right = close_x - after_title_padding;
@@ -1021,6 +1022,7 @@ void Tab::UpdateIconVisibility() {
       available_width -= favicon_width;
 
     showing_close_button_ = large_enough_for_close_button;
+    BRAVE_UI_VIEWS_TABS_TAB_UPDATE_ICON_VISIBILITY
     if (showing_close_button_)
       available_width -= close_button_width;
 
