diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index e5d13c8e53c3094ee234b2a5e8dcc9a87130871a..b6b792dedb2c6da8e8f45c625dfdb798ab0d6b2b 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -8,6 +8,7 @@ import {pageVisibility} from './page_visibility.js';
 import {Route, Router} from './router.js';
 import {SettingsRoutes} from './settings_routes.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -215,6 +216,7 @@ function createBrowserSettingsRoutes(): SettingsRoutes {
       // </if>
     }
   }
+  addBraveRoutes(r);
   return r;
 }
 
