diff --git a/chrome/browser/speech/on_device_speech_recognition_impl.cc b/chrome/browser/speech/on_device_speech_recognition_impl.cc
index 9e209fe35b4b480b251d84e9709d1edcba513ffa..3650769f8acacc5b87ee525fd9c489135d5c1643 100644
--- a/chrome/browser/speech/on_device_speech_recognition_impl.cc
+++ b/chrome/browser/speech/on_device_speech_recognition_impl.cc
@@ -156,6 +156,9 @@ void OnDeviceSpeechRecognitionImpl::Available(
     const std::vector<std::string>& languages,
     media::mojom::SpeechRecognitionQuality quality,
     OnDeviceSpeechRecognitionImpl::AvailableCallback callback) {
+  if (quality == media::mojom::SpeechRecognitionQuality::kDictation) {
+    quality = media::mojom::SpeechRecognitionQuality::kCommand;
+  }
 #if BUILDFLAG(IS_ANDROID)
   std::move(callback).Run(media::mojom::AvailabilityStatus::kUnavailable);
 #else
