diff --git a/chrome/browser/ui/extensions/extension_action_view_controller.cc b/chrome/browser/ui/extensions/extension_action_view_controller.cc
index 3493b718cb952fb363ba2496ed6924d993f132cd..30cd6db2c6efe70c0ef68b114f442f420241e091 100644
--- a/chrome/browser/ui/extensions/extension_action_view_controller.cc
+++ b/chrome/browser/ui/extensions/extension_action_view_controller.cc
@@ -63,7 +63,6 @@ ExtensionActionViewController::Create(
     Browser* browser,
     ExtensionsContainer* extensions_container) {
   DCHECK(browser);
-  DCHECK(extensions_container);
 
   auto* registry = extensions::ExtensionRegistry::Get(browser->profile());
   scoped_refptr<const extensions::Extension> extension =
