diff --git a/chrome/android/java/src/org/chromium/chrome/browser/LaunchIntentDispatcher.java b/chrome/android/java/src/org/chromium/chrome/browser/LaunchIntentDispatcher.java
index 9e614cdb61a591ab8bd7f33c686aa5c5a9ec7e8b..ad5411038b468041038f30904f27e56f0f439995 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/LaunchIntentDispatcher.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/LaunchIntentDispatcher.java
@@ -250,6 +250,7 @@ public class LaunchIntentDispatcher implements IntentHandler.IntentHandlerDelega
      * @return Whether the intent is for launching a Custom Tab.
      */
     public static boolean isCustomTabIntent(Intent intent) {
+        if(!BraveLaunchIntentDispatcher.useCustomTabs()) return false;
         if (intent == null) return false;
         if (CustomTabsIntent.shouldAlwaysUseBrowserUI(intent)
                 || !intent.hasExtra(CustomTabsIntent.EXTRA_SESSION)) {
