diff --git a/chrome/browser/component_updater/chrome_component_updater_configurator.cc b/chrome/browser/component_updater/chrome_component_updater_configurator.cc
index bd35e3fa73d4acd6500916bb1a856828cca2d026..66b63e5a12a58b7803ae90c91b7403218ff5aa23 100644
--- a/chrome/browser/component_updater/chrome_component_updater_configurator.cc
+++ b/chrome/browser/component_updater/chrome_component_updater_configurator.cc
@@ -344,7 +344,10 @@ bool ChromeConfigurator::IsConnectionMetered() const {
 scoped_refptr<update_client::Configurator>
 MakeChromeComponentUpdaterConfigurator(const base::CommandLine* cmdline,
                                        PrefService* pref_service) {
-  return base::MakeRefCounted<ChromeConfigurator>(cmdline, pref_service);
+  return base::MakeRefCounted<BraveConfigurator>(
+      cmdline, pref_service,
+      g_browser_process->system_network_context_manager()
+          ->GetSharedURLLoaderFactory());
 }
 
 }  // namespace component_updater
