diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index d564e599ac39d069622c04379a322470664b5ee4..ca87b662756d7860f4be786fc9e17ac9b24eece0 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -942,6 +942,7 @@ void BookmarkBarView::Layout() {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   // Layout the right side buttons.
   x = max_x + bookmark_bar_button_padding;
