diff --git a/chrome/installer/mini_installer/BUILD.gn b/chrome/installer/mini_installer/BUILD.gn
index c365088b90b7a91faf52f5b108f64621a219b11f..c60f159ed2a94cf5401f3e979fd3d028a09140a4 100644
--- a/chrome/installer/mini_installer/BUILD.gn
+++ b/chrome/installer/mini_installer/BUILD.gn
@@ -181,6 +181,23 @@ template("generate_mini_installer") {
       "//third_party/icu:icudata",
       chrome_dll_target,
     ]
+    if (brave_chromium_build) {
+      inputs -= [
+        "$root_out_dir/chrome.exe",
+      ]
+      inputs += [
+        "$root_out_dir/brave.exe",
+        "$root_out_dir/brave_resources.pak",
+        "$root_out_dir/brave_100_percent.pak",
+        "$root_out_dir/brave_200_percent.pak",
+        "$root_gen_dir/brave/VERSION",
+      ]
+      deps += [
+        "//brave:generate_version",
+        "//brave:packed_resources",
+        "//brave/build/win:copy_exe",
+      ]
+    }
 
     if (enable_hidpi) {
       args += [ "--enable_hidpi=1" ]
