diff --git a/chrome/utility/services.cc b/chrome/utility/services.cc
index 3fc96174d7e387fb07003021ae25f38e76369f73..64c2f24008d6b79bc3dc34c75468eac6e7c240b3 100644
--- a/chrome/utility/services.cc
+++ b/chrome/utility/services.cc
@@ -376,6 +376,7 @@ void RegisterMainThreadServices(mojo::ServiceFactory& services) {
   services.Add(RunAssistantAudioDecoder);
 #endif
 #endif
+    BRAVE_GET_MAIN_THREAD_SERVICE_FACTORY
 }
 
 void RegisterIOThreadServices(mojo::ServiceFactory& services) {
