diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index da71d1b4b4a65c015b0b1dc4cd576e07db886d92..92c57dce4542530597cb557af7fab580a4245d25 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -523,6 +523,7 @@ void ToolbarView::Init() {
   }
 #endif  // BUILDFLAG(ENABLE_GLIC)
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = AddChildView(std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
 #if BUILDFLAG(IS_CHROMEOS)
