diff --git a/chrome/browser/ui/browser_commands.cc b/chrome/browser/ui/browser_commands.cc
index aa732ca6d7774505751cc30f3d2deec4e3170191..46d5ff3f414eacd7d640aa43f20c22365234e2f6 100644
--- a/chrome/browser/ui/browser_commands.cc
+++ b/chrome/browser/ui/browser_commands.cc
@@ -1185,6 +1185,7 @@ void CloseTab(BrowserWindowInterface* browser) {
   }
 
   ToastController* toast_controller = browser->GetFeatures().toast_controller();
+  BRAVE_CLOSE_TAB
   if (!toast_controller) {
     CloseSelectedTabAndRecordTabCountMetric(browser);
     return;
