diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
index ee78225ee3ae799dc3c4002defdc5bcf3a478170..bab7bc99710223fa77e1e16fae5771158cd756d2 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
@@ -866,6 +866,7 @@ public class ToolbarManager implements ScrimObserver, ToolbarTabController, UrlF
     // TODO(crbug.com/1026020): Move this logic to BottomToolbar class.
     private void onShareDelegateAvailable(ShareDelegate shareDelegate) {
         final OnClickListener shareButtonListener = v -> {
+/*
             if (BottomToolbarVariationManager.isShareButtonOnBottom()) {
                 recordBottomToolbarUseForIPH();
                 RecordUserAction.record("MobileBottomToolbarShareButton");
@@ -879,7 +880,8 @@ public class ToolbarManager implements ScrimObserver, ToolbarTabController, UrlF
                 activity = ((TabImpl) tab).getActivity();
                 isIncognito = tab.isIncognito();
             }
-            shareDelegate.share(tab, /*shareDirectly=*/false);
+*///        shareDelegate.share(tab, /*shareDirectly=*/false);
+            mActivity.addOrEditBookmark(mActivity.getActivityTab());
         };
         mShareButtonListenerSupplier.set(shareButtonListener);
     }
@@ -1883,6 +1885,7 @@ public class ToolbarManager implements ScrimObserver, ToolbarTabController, UrlF
         boolean editingAllowed = currentTab == null || mBookmarkBridge == null
                 || mBookmarkBridge.isEditBookmarksEnabled();
         mToolbar.updateBookmarkButton(isBookmarked, editingAllowed);
+        if (mBottomControlsCoordinator != null) {mBottomControlsCoordinator.updateBookmarkButton(isBookmarked, editingAllowed);}
     }
 
     private void updateReloadState(boolean tabCrashed) {
