diff --git a/chrome/browser/prefs/BUILD.gn b/chrome/browser/prefs/BUILD.gn
index d38f86333300cc6e5b430311fd9095b329005db7..d6cfee81def92ae9b6bba150ff446f8839d71352 100644
--- a/chrome/browser/prefs/BUILD.gn
+++ b/chrome/browser/prefs/BUILD.gn
@@ -203,6 +203,7 @@ source_set("impl") {
     "//services/preferences/tracked:features",
     "//ui/webui:buildflags",
   ]
+  import("//brave/browser/sources.gni") deps += brave_chrome_browser_prefs_impl_deps
 
   if (enable_print_preview) {
     deps += [
