diff --git a/chrome/browser/ui/omnibox/omnibox_edit_model.cc b/chrome/browser/ui/omnibox/omnibox_edit_model.cc
index 17b351832f11252b618dc536d7c2fdda8c67d4ed..b6b91da7121805ec5dd293ee2a332f59bb8f7b9b 100644
--- a/chrome/browser/ui/omnibox/omnibox_edit_model.cc
+++ b/chrome/browser/ui/omnibox/omnibox_edit_model.cc
@@ -681,6 +681,7 @@ void OmniboxEditModel::StartAutocomplete(bool prevent_inline_autocomplete) {
     input_.set_lens_overlay_suggest_inputs(*suggest_inputs);
   }
 
+  BRAVE_OMNIBOX_EDIT_MODEL_START_AUTOCOMPLETE
   controller_->StartAutocomplete(input_);
 }
 
