diff --git a/chrome/browser/ui/views/location_bar/location_icon_view.cc b/chrome/browser/ui/views/location_bar/location_icon_view.cc
index 2501ea3011102d75aa1080b38f7cb228c641b644..152aa7a955e551ee62f77846a0323e2c96906807 100644
--- a/chrome/browser/ui/views/location_bar/location_icon_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_icon_view.cc
@@ -355,6 +355,7 @@ void LocationIconView::UpdateBackground() {
 
   SetBackgroundColor(GetColorProvider()->GetColor(id));
 
+  BRAVE_LOCATION_ICON_VIEW_UPDATE_BACKGROUND
   if (is_text_dangerous) {
     ConfigureInkDropForRefresh2023(this, kColorOmniboxSecurityChipInkDropHover,
                                    kColorOmniboxSecurityChipInkDropRipple);
