diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 1c8687da2f37a6669581f9a815b8de8fd6d3871c..eae0c66422fc13e76ad04928630daead0d78cf6a 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2547,6 +2547,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
