diff --git a/chrome/browser/ui/views/update_recommended_message_box.h b/chrome/browser/ui/views/update_recommended_message_box.h
index adcef966dd879d608222569b83cadca01c2452ab..da13fc19533d02425bafca208250d080caf0d9ec 100644
--- a/chrome/browser/ui/views/update_recommended_message_box.h
+++ b/chrome/browser/ui/views/update_recommended_message_box.h
@@ -20,6 +20,8 @@ class UpdateRecommendedMessageBox : public views::DialogDelegate {
   static void Show(gfx::NativeWindow parent_window);
 
  private:
+  friend class UpdateRecommendedMessageBoxMac;
+
   UpdateRecommendedMessageBox();
   ~UpdateRecommendedMessageBox() override;
 
