diff --git a/chrome/browser/app_controller_mac.mm b/chrome/browser/app_controller_mac.mm
index c530233e8894872217e8b36f73415bb02e62888f..7b6372790e1ecbae4ee44458409b642e2204dc23 100644
--- a/chrome/browser/app_controller_mac.mm
+++ b/chrome/browser/app_controller_mac.mm
@@ -735,7 +735,7 @@ class AppControllerNativeThemeObserver : public ui::NativeThemeObserver {
 
 + (AppController*)sharedController {
   static AppController* sharedController = [] {
-    AppController* sharedController = [[AppController alloc] init];
+    AppController* sharedController = [[BraveAppController alloc] init];
     NSApp.delegate = sharedController;
     return sharedController;
   }();
@@ -1351,7 +1351,7 @@ class AppControllerNativeThemeObserver : public ui::NativeThemeObserver {
   }
 
   auto it = _profileBookmarkMenuBridgeMap.find(profilePath);
-  if (it != _profileBookmarkMenuBridgeMap.end() &&
+  if (it != _profileBookmarkMenuBridgeMap.end() && !isOffTheRecord &&
       (!base::FeatureList::IsEnabled(features::kDestroyProfileOnBrowserClose) ||
        (it->second->GetProfile() && !isOffTheRecord))) {
     // Clean up the dangling Profile* in |_profileBookmarkMenuBridgeMap|.
