diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index caf4aae54234a2a72ec3d0d87cd4ed780b88923d..5170a55118372684a5a8cd25394e4c4a4ccaa25a 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2596,6 +2596,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_win || is_mac || is_linux || is_chromeos_ash) {
     deps += [ "//chrome/browser/ui/webui/connectors_internals:mojo_bindings" ]
