diff --git a/chrome/installer/linux/rpm/update_package_provides.py b/chrome/installer/linux/rpm/update_package_provides.py
index 615292c11780a943ed7d7644dc640b8c9ca74f11..3cf430e908717513395a61bfb627559b8093c52c 100755
--- a/chrome/installer/linux/rpm/update_package_provides.py
+++ b/chrome/installer/linux/rpm/update_package_provides.py
@@ -34,6 +34,7 @@ LIBRARY_FILTER = set([
     "libatspi.so.0",
     "libc.so",
     "libcairo.so",
+    "libcairo-gobject.so.2",
     "libcups.so",
     "libdbus-1.so",
     "libdrm.so.2",
@@ -54,13 +55,17 @@ LIBRARY_FILTER = set([
     "libnssutil3.so",
     "libpango-1.0.so",
     "libpangocairo-1.0.so",
+    "libplc4.so",
+    "libplds4.so",
     "libpthread.so",
+    "libresolv.so.2",
     "librt.so",
     "libsmime3.so",
     "libstdc++.so",
     "libuuid.so",
     "libxcb.so",
     "libxcb-dri3.so.0",
+    "libz.so.1",
     "rtld(GNU_HASH)",
 ])
 
