diff --git a/chrome/browser/ui/views/page_action/page_action_view.cc b/chrome/browser/ui/views/page_action/page_action_view.cc
index b98eeeedfcb9276d9a95f88ebe1aea5a5e68d21c..0e9f895cece5a6df12101b552526b83c2823ca91 100644
--- a/chrome/browser/ui/views/page_action/page_action_view.cc
+++ b/chrome/browser/ui/views/page_action/page_action_view.cc
@@ -120,7 +120,7 @@ void PageActionView::OnPageActionModelChanged(
   if (!model.GetVisible()) {
     ResetSlideAnimation(/*show=*/false);
     NotifyIsChipShowingChange();
-  } else if (model.ShouldShowSuggestionChip()) {
+  } else if (model.ShouldShowSuggestionChip() || model.GetAlwaysShowLabel()) {
     if (model.GetShouldAnimateChipIn()) {
       AnimateIn(/*string_id=*/std::nullopt);
     } else {
