diff --git a/chrome/browser/ui/tabs/public/tab_features.h b/chrome/browser/ui/tabs/public/tab_features.h
index f07faf5558d748c4b634f46aa3daaedb57296391..ec27f1001f9fa50ab4ece5d993ad44a7ab5023ba 100644
--- a/chrome/browser/ui/tabs/public/tab_features.h
+++ b/chrome/browser/ui/tabs/public/tab_features.h
@@ -229,7 +229,7 @@ class TabInterface;
 class TabFeatures {
  public:
   TabFeatures();
-  ~TabFeatures();
+  virtual ~TabFeatures();
 
   TabFeatures(const TabFeatures&) = delete;
   TabFeatures& operator=(const TabFeatures&) = delete;
@@ -387,7 +387,7 @@ class TabFeatures {
   }
 
   // Called exactly once to initialize features.
-  void Init(TabInterface& tab, Profile* profile);
+  virtual void Init(TabInterface& tab, Profile* profile);
 
   static ui::UserDataFactoryWithOwner<TabInterface>&
   GetUserDataFactoryForTesting();
