diff --git a/chrome/browser/signin/account_consistency_mode_manager.cc b/chrome/browser/signin/account_consistency_mode_manager.cc
index 222ba8a3084aa84deccb932267f7f55cf6f653dd..71690d719e957c12aff0eb6afab60a2d01580001 100644
--- a/chrome/browser/signin/account_consistency_mode_manager.cc
+++ b/chrome/browser/signin/account_consistency_mode_manager.cc
@@ -189,6 +189,7 @@ AccountConsistencyModeManager::ComputeAccountConsistencyMethod(
     Profile* profile) {
   DCHECK(ShouldBuildServiceForProfile(profile));
 
+BRAVE_COMPUTE_ACCOUNT_CONSISTENCY_METHOD
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   if (!ash::IsAccountManagerAvailable(profile))
     return AccountConsistencyMethod::kDisabled;
