diff --git a/chrome/renderer/chrome_content_renderer_client.cc b/chrome/renderer/chrome_content_renderer_client.cc
index f88b293427305aecb7c445023f454c4dfda97ef1..b85681653b61c100ec9a7168b5290cda702db902 100644
--- a/chrome/renderer/chrome_content_renderer_client.cc
+++ b/chrome/renderer/chrome_content_renderer_client.cc
@@ -19,6 +19,7 @@
 #include "base/strings/utf_string_conversions.h"
 #include "base/time/time.h"
 #include "base/values.h"
+#include "brave/renderer/brave_content_settings_observer.h"
 #include "build/build_config.h"
 #include "chrome/common/buildflags.h"
 #include "chrome/common/channel_info.h"
@@ -513,7 +514,7 @@ void ChromeContentRendererClient::RenderFrameCreated(
   ext_dispatcher =
       ChromeExtensionsRendererClient::GetInstance()->extension_dispatcher();
 #endif
-  ContentSettingsObserver* content_settings = new ContentSettingsObserver(
+  ContentSettingsObserver* content_settings = new BraveContentSettingsObserver(
       render_frame, ext_dispatcher, should_whitelist_for_content_settings,
       registry);
   if (chrome_observer_.get()) {
