diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 86ebd971adfed1f728e0666e91df2db73973a18f..1c7c5e4ef08976a7df8b2699435838280953534b 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -346,6 +346,7 @@ void ToolbarView::Init() {
   if (side_panel_button)
     side_panel_button_ = AddChildView(std::move(side_panel_button));
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = AddChildView(std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
 #if BUILDFLAG(IS_CHROMEOS_ASH)
