diff --git a/chrome/browser/renderer_context_menu/render_view_context_menu.cc b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
index 55083525c00cb805e3f17fad312e099e5b0c7a2c..e1af4aa5295733777c038268bd13fcda638337f6 100644
--- a/chrome/browser/renderer_context_menu/render_view_context_menu.cc
+++ b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
@@ -1169,6 +1169,7 @@ void RenderViewContextMenu::AppendLinkItems() {
                : IDS_CONTENT_CONTEXT_OPENLINKOFFTHERECORD);
 
     AppendOpenInBookmarkAppLinkItems();
+    AppendBraveLinkItems();
     AppendOpenWithLinkItems();
 
     // While ChromeOS supports multiple profiles, only one can be open at a
