diff --git a/chrome/browser/notifications/notification_display_service_impl.cc b/chrome/browser/notifications/notification_display_service_impl.cc
index c185575c57d0f0e72edc3c9bcfc5874e4c2a63dd..44d18329a2433d73e5f49e805d2b26f234eb13f7 100644
--- a/chrome/browser/notifications/notification_display_service_impl.cc
+++ b/chrome/browser/notifications/notification_display_service_impl.cc
@@ -124,6 +124,7 @@ NotificationDisplayServiceImpl::NotificationDisplayServiceImpl(Profile* profile)
                              std::make_unique<NearbyNotificationHandler>());
     }
 #endif
+    BRAVE_NOTIFICATION_DISPLAY_SERVICE_IMPL_CONSTRUCTOR;
   }
 
   bridge_delegator_ = std::make_unique<NotificationPlatformBridgeDelegator>(
