diff --git a/chrome/browser/ui/views/tabs/tab_strip.cc b/chrome/browser/ui/views/tabs/tab_strip.cc
index 756706214a2b7ce0758b5e5420966c4d70a1fa3e..7b5436912dbe4d7277420e33a4a1d6285f1a6940 100644
--- a/chrome/browser/ui/views/tabs/tab_strip.cc
+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
@@ -547,6 +547,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
   std::vector<gfx::Rect> CalculateBoundsForDraggedViews(
       const std::vector<raw_ptr<TabSlotView, VectorExperimental>>& views)
       override {
+    BRAVE_TAB_DRAG_CONTEXT_IMPL_CALCULATE_BOUNDS_FOR_DRAGGED_VIEWS
     CHECK(!views.empty(), base::NotFatalUntil::M128)
         << "The views vector must not be empty.";
 
@@ -750,6 +751,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
     // index.
     std::stable_sort(orderable_children.begin(), orderable_children.end());
 
+    BRAVE_TAB_DRAG_CONTEXT_IMPL_PAINT_CHILDREN
     for (const ZOrderableTabContainerElement& child : orderable_children) {
       child.view()->Paint(paint_info);
     }
@@ -843,6 +845,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
         continue;
       }
 
+      BRAVE_TAB_DRAG_CONTEXT_IMPL_CALCULATE_INSERTION_INDEX
       // If there's a group header here, and we're dragging a group, we might
       // end up on either side of that header. Check both cases to find the
       // best option.
