diff --git a/chrome/browser/ui/browser_commands.cc b/chrome/browser/ui/browser_commands.cc
index 64868f777bdf7c7b8f425acb0cd60414b360b294..4ebd7dff31091d066f0d58aa670dc737babd6404 100644
--- a/chrome/browser/ui/browser_commands.cc
+++ b/chrome/browser/ui/browser_commands.cc
@@ -1093,7 +1093,7 @@ void Reload(BrowserWindowInterface* browser,
   ReloadInternal(browser, disposition, false);
 }
 
-void ReloadBypassingCache(BrowserWindowInterface* browser,
+void ReloadBypassingCache_ChromiumImpl(BrowserWindowInterface* browser,
                           WindowOpenDisposition disposition) {
   base::RecordAction(UserMetricsAction("ReloadBypassingCache"));
   ReloadInternal(browser, disposition, true);
@@ -1409,6 +1409,7 @@ void CloseTab(BrowserWindowInterface* browser) {
 #endif
 
   ToastController* toast_controller = browser->GetFeatures().toast_controller();
+  toast_controller = nullptr;
   if (!toast_controller) {
     CloseSelectedTabAndRecordTabCountMetric(browser);
     return;
@@ -1737,7 +1738,7 @@ void NewSplitTab(BrowserWindowInterface* browser,
   // the regular NTP which renders special content when in Incognito.
   const GURL new_tab_url = !browser->GetProfile()->IsIncognitoProfile() &&
                                    tab_strip_model->count() > 1
-                               ? GURL(chrome::kChromeUISplitViewNewTabPageURL)
+                               ? GURL(chrome::kChromeUINewTabURL)
                                : chrome::ChromeUINewTabURLAsGURL();
   tab_strip_model->delegate()->AddTabAt(
       new_tab_url, active_index + 1, true,
