diff --git a/chrome/android/java/src/org/chromium/chrome/browser/customtabs/features/partialcustomtab/PartialCustomTabDisplayManager.java b/chrome/android/java/src/org/chromium/chrome/browser/customtabs/features/partialcustomtab/PartialCustomTabDisplayManager.java
index 23ddd0994d66d40cbe5498ae7cbe368200ee0da6..ef8e84e80faaa66b78424160ae4282661b8d2f25 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/customtabs/features/partialcustomtab/PartialCustomTabDisplayManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/customtabs/features/partialcustomtab/PartialCustomTabDisplayManager.java
@@ -58,7 +58,7 @@ public class PartialCustomTabDisplayManager
                 BrowserServicesIntentDataProvider intentData, boolean startMaximized);
     }
 
-    private PartialCustomTabBaseStrategy mStrategy;
+    public PartialCustomTabBaseStrategy mStrategy;
     private @PartialCustomTabType int mCurrentPartialCustomTabType;
 
     private View mToolbarCoordinatorView;
