diff --git a/chrome/browser/ui/views/translate/translate_bubble_view.h b/chrome/browser/ui/views/translate/translate_bubble_view.h
index 3a0d279d42976c78fb9e99583417398cecd93af0..d5b47fb735a3dca36f42383a2058fec2ade08f60 100644
--- a/chrome/browser/ui/views/translate/translate_bubble_view.h
+++ b/chrome/browser/ui/views/translate/translate_bubble_view.h
@@ -117,6 +117,7 @@ class TranslateBubbleView : public LocationBarBubbleDelegateView,
 
   // Returns the current view state.
   TranslateBubbleModel::ViewState GetViewState() const;
+  BRAVE_TRANSLATE_BUBBLE_VIEW_H_
 
  protected:
   // LocationBarBubbleDelegateView:
