diff --git a/chrome/installer/mac/signing/config.py b/chrome/installer/mac/signing/config.py
index 3732a9d2fb2cfa6eac6d891bc6d4355460bf234d..46a886d9f1d229dd3f523035dd1b1dd2e4b949bf 100644
--- a/chrome/installer/mac/signing/config.py
+++ b/chrome/installer/mac/signing/config.py
@@ -247,4 +247,4 @@ class CodeSignConfig(object):
     @property
     def packaging_dir(self):
         """Returns the path to the packaging and installer tools."""
-        return '{.product} Packaging'.format(self)
+        return '{}_Packaging'.format(self.product.replace(' ', '_'))
