diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index b4ceefbccd083c608dcc5543c186cd884c44a75f..a9e94670547bc8fbc5ac24be312b14e442d8c581 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -205,6 +205,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
