diff --git a/chrome/android/java_sources.gni b/chrome/android/java_sources.gni
index b3165a3cbbfdafcfa1bf6e3df10516c214b5bd7b..5549b4162402400e8f976c62f1fb799620beba1f 100644
--- a/chrome/android/java_sources.gni
+++ b/chrome/android/java_sources.gni
@@ -18,6 +18,7 @@ import("//components/offline_pages/buildflags/features.gni")
 import("//device/vr/buildflags/buildflags.gni")
 
 chrome_java_sources += public_autofill_assistant_java_sources
+chrome_java_sources += brave_java_sources
 
 if (enable_vr) {
   chrome_java_sources += [ "java/src/org/chromium/chrome/browser/component_updater/VrAssetsComponentInstaller.java" ]
