diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 7ceb48d1835ad0d07dcbc03e775908717ac0727a..e9e24f2cec4698b8e39979cece5898077f46ba9c 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2502,6 +2502,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
