diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index a88fc32ba5223d4fcaba72f554169cb515e93ef7..0a81b62ee77ec5dae0e94e85831abd4dfcf182ce 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4804,6 +4804,7 @@ const FeatureEntry::FeatureVariation kHistorySyncOptinExpansionPillVariations[]
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
