diff --git a/chrome/browser/ui/views/frame/multi_contents_view.h b/chrome/browser/ui/views/frame/multi_contents_view.h
index 12e0fb3bd54253c28538086673c0d133f420fc3d..63a1d18787b856f6c7b10ba7cbcc657230ba0ca7 100644
--- a/chrome/browser/ui/views/frame/multi_contents_view.h
+++ b/chrome/browser/ui/views/frame/multi_contents_view.h
@@ -21,7 +21,7 @@
 class BrowserView;
 class ContentsWebView;
 class MultiContentsDropTargetView;
-class MultiContentsResizeArea;
+// class MultiContentsResizeArea;
 class MultiContentsViewDelegate;
 class MultiContentsViewDropTargetController;
 class MultiContentsViewMiniToolbar;
