diff --git a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationService.java b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationService.java
index baa1b123c8a648782370d8c4317479c6b2a4b382..8a415e40bab3169e8a4ed7e2d767a6b4b8e6cc97 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationService.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationService.java
@@ -117,7 +117,7 @@ public class NotificationService extends IntentService {
 
         // Now that the browser process is initialized, we pass forward the call to the
         // NotificationPlatformBridge which will take care of delivering the appropriate events.
-        if (!NotificationPlatformBridge.dispatchNotificationEvent(intent)) {
+        if (!BraveNotificationPlatformBridge.dispatchNotificationEvent(intent)) {
             Log.w(TAG, "Unable to dispatch the notification event to Chrome.");
         }
 
