diff --git a/chrome/browser/resources/settings/people_page/sync_controls.ts b/chrome/browser/resources/settings/people_page/sync_controls.ts
index afe730971495212258a609dd09b74baf7cea6bf2..2ec966a0928bc5cb7ac917bf3de8a3955c460aa8 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.ts
+++ b/chrome/browser/resources/settings/people_page/sync_controls.ts
@@ -183,7 +183,7 @@ export class SettingsSyncControlsElement extends
 
     const router = Router.getInstance();
     const currentRoute = router.getCurrentRoute();
-    if (currentRoute === routes.SYNC_ADVANCED) {
+    if (currentRoute === routes.BRAVE_SYNC_SETUP) {
       this.syncBrowserProxy_.didNavigateToSyncPage();
     }
     // <if expr="not is_chromeos">
@@ -381,6 +381,7 @@ export class SettingsSyncControlsElement extends
   private disableTypeCheckBox_(
       syncStatus: SyncStatus, syncAllDataTypes: boolean,
       dataTypeManaged: boolean): boolean {
+    if (dataTypeManaged) { return true; } if (!syncStatus || !this.syncPrefs) { return syncAllDataTypes }
     if (!syncStatus) {
       return true;
     }
