diff --git a/chrome/browser/resources/settings/settings_routes.ts b/chrome/browser/resources/settings/settings_routes.ts
index ffffa797f90256ed43abaf9b860c06410be3df0a..f901a85c45d175c18fb8491f1c241e49cd0b2864 100644
--- a/chrome/browser/resources/settings/settings_routes.ts
+++ b/chrome/browser/resources/settings/settings_routes.ts
@@ -87,4 +87,5 @@ export type SettingsRoutes = {
   SYNC_ADVANCED: Route,
   SYSTEM: Route,
   TRIGGERED_RESET_DIALOG: Route,
+  GET_STARTED: Route,
 };
