diff --git a/chrome/installer/mac/signing/signing.py b/chrome/installer/mac/signing/signing.py
index 1310932b4d81ea4550c59d619626ba4c0a27b4a1..b521950a42c52620d0de8ac823928878f11aa9b8 100644
--- a/chrome/installer/mac/signing/signing.py
+++ b/chrome/installer/mac/signing/signing.py
@@ -15,7 +15,7 @@ from signing import commands, invoker
 class Invoker(invoker.Base):
 
     def codesign(self, config, product, path, replace_existing_signature=False):
-        command = ['codesign', '--sign', config.identity]
+        command = ['codesign', '--force', '--sign', config.identity]
         if replace_existing_signature:
             command.append('--force')
         if config.notarize.should_notarize():
