diff --git a/chrome/installer/linux/common/installer.py b/chrome/installer/linux/common/installer.py
index 7646f80c84006e47ac5de006505288ccf18a4158..b0c6ffdd159473d5c74d8f0282e59fc9c76a2b99 100644
--- a/chrome/installer/linux/common/installer.py
+++ b/chrome/installer/linux/common/installer.py
@@ -200,6 +200,8 @@ def gen_changelog(config: "InstallerConfig",
     gzlog_dir = (
         config.staging_dir /
         f"usr/share/doc/{config.info_vars['PACKAGE']}-{config.channel}")
+    if config.channel == "stable":
+        gzlog_dir = (config.staging_dir / f"usr/share/doc/{config.info_vars['PACKAGE']}")
     gzlog_dir.mkdir(parents=True, exist_ok=True)
     gzlog = gzlog_dir / "changelog.gz"
 
@@ -336,6 +338,7 @@ class InstallerConfig:
     uri_scheme: str = ""
     extra_desktop_entries: str = ""
     startup_wm_class: str = ""
+    package_and_channel: str = ""
 
     @classmethod
     def from_args(cls, args: argparse.Namespace,
@@ -703,6 +706,7 @@ class InstallerConfig:
                 icon_suffix = "_dev"
             elif self.channel == "canary":
                 icon_suffix = "_canary"
+        icon_suffix = {"beta": "_beta", "dev": "_dev", "nightly": "_nightly", "unstable": "_dev"}.get(self.channel, "") if self.branding in ("brave", "brave_origin") else ""
 
         icon_sizes = [16, 24, 32, 48, 64, 128, 256]
         logo_resources_png = []
@@ -1261,3 +1265,4 @@ def compute_repo_package_hash_for_presubmit(
             return hashlib.sha256(f.read()).hexdigest()
     finally:
         os.umask(old_umask)
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
