diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index 66c121790b3e41fe625f35297822ff49fb3bcda2..c1efae2c3edad171f9ef7c8a26a8921a3b2c3a0f 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -115,6 +115,7 @@ generate_grd("build_grd") {
     deps = [ ":build_ts" ]
     manifest_files = [ "$target_gen_dir/tsconfig.manifest" ]
   }
+  deps += [ "//brave/browser/resources/settings:resources" ]
 }
 
 preprocess_if_expr("preprocess") {
@@ -263,4 +264,6 @@ ts_library("build_ts") {
     ":preprocess",
     ":preprocess_generated",
   ]
+  in_files += brave_deps_chrome_browser_resources_settings_in_files extra_deps += brave_deps_chrome_browser_resources_settings_extra_deps
+
 }
