diff --git a/chrome/browser/ui/browser_commands.cc b/chrome/browser/ui/browser_commands.cc
index c162a624c2907f6212e306f68a3e7f11222fc7d9..7cf6f62be30331c8c166fb77b39a92410409218a 100644
--- a/chrome/browser/ui/browser_commands.cc
+++ b/chrome/browser/ui/browser_commands.cc
@@ -1181,6 +1181,7 @@ void CloseTab(BrowserWindowInterface* browser) {
   }
 
   ToastController* toast_controller = browser->GetFeatures().toast_controller();
+  BRAVE_CLOSE_TAB
   if (!toast_controller) {
     CloseSelectedTabAndRecordTabCountMetric(browser);
     return;
