diff --git a/chrome/installer/linux/BUILD.gn b/chrome/installer/linux/BUILD.gn
index 291ef25d7e29b9dede8f8c84a47d94e364910c74..226dbcc7bdea64b4a25056f37583e9a6ff7a9dcc 100644
--- a/chrome/installer/linux/BUILD.gn
+++ b/chrome/installer/linux/BUILD.gn
@@ -16,6 +16,7 @@ import("//third_party/angle/gni/angle.gni")
 import("//third_party/widevine/cdm/widevine.gni")
 import("//ui/gl/features.gni")
 import("//ui/qt/qt.gni")
+import("//brave/build/config.gni")
 
 if (bundle_widevine_cdm) {
   import("//media/cdm/library_cdm/cdm_paths.gni")
@@ -63,6 +64,7 @@ if (build_with_internal_optimization_guide) {
       [ "$root_out_dir/liboptimization_guide_internal.so" ]
 }
 
+packaging_files_executables -= excluded_packaging_files_executables packaging_files_executables += brave_packaging_files_executables
 packaging_files_binaries = packaging_files_executables + packaging_files_shlibs
 
 # TODO(mmoss) Any convenient way to get all the relevant build files?
@@ -172,6 +174,7 @@ action("merge_rpm_dependencies") {
 strip_binary("strip_chrome_binary") {
   binary_input = "$root_out_dir/chrome"
   deps = [ "//chrome" ]
+  binary_input = "$root_out_dir/brave" symbol_output = "$root_out_dir/brave.debug" stripped_binary_output = "$root_out_dir/brave.stripped" deps += strip_chrome_binary_deps
 }
 
 strip_binary("strip_chrome_crashpad_handler") {
@@ -187,6 +190,7 @@ strip_binary("strip_chrome_management_service") {
 strip_binary("strip_chrome_sandbox") {
   binary_input = "$root_out_dir/chrome_sandbox"
   deps = [ "//sandbox/linux:chrome_sandbox" ]
+  binary_input = "$root_out_dir/brave_sandbox" symbol_output = "$root_out_dir/brave_sandbox.debug" stripped_binary_output = "$root_out_dir/brave_sandbox.stripped" deps += strip_chrome_sandbox_deps
 }
 
 strip_binary("strip_libEGL_shlib") {
@@ -244,9 +248,9 @@ group("linux") {
   ]
 }
 
-branding_dir = "//chrome/app/theme/$branding_path_component"
+branding_dir = "//brave/app/theme/$branding_path_component"
 branding_dir_100 =
-    "//chrome/app/theme/default_100_percent/$branding_path_component"
+    "//brave/app/theme/default_100_percent/$branding_path_component"
 
 copy("common_packaging_files") {
   visibility = [ ":*" ]
@@ -276,6 +280,7 @@ copy("common_packaging_files") {
   }
 
   outputs = [ "$root_out_dir/installer/common/{{source_file_part}}" ]
+  sources -= common_packaging_files_excluded_sources sources += common_packaging_files_sources
 }
 
 copy("deb_packaging_files") {
@@ -373,6 +378,7 @@ group("installer_deps") {
     "//components/privacy_sandbox/privacy_sandbox_attestations/preload:component",
     "//sandbox/linux:chrome_sandbox",
   ]
+  public_deps += brave_installer_deps
   if (bundle_widevine_cdm) {
     public_deps += [ "//third_party/widevine/cdm" ]
   }
@@ -427,11 +433,14 @@ group("installer_deps") {
 template("linux_package") {
   assert(defined(invoker.channel))
   channel = invoker.channel
+  if (channel == "unstable" ) {
+    channel = "dev"
+  }
 
   if (is_chrome_branded) {
     package = "google-chrome"
   } else {
-    package = "chromium-browser"
+    package = "brave-browser" chrome_version_full = brave_version
   }
 
   deb_target_name = "${target_name}_deb"
@@ -460,7 +469,7 @@ template("linux_package") {
     }
 
     inputs = packaging_files
-    outputs = [ "$root_out_dir/${package}-${channel}_${chrome_version_full}-1_${deb_arch}.deb" ]
+    if (channel == "stable") { outputs = [ "$root_out_dir/${package}_${chrome_version_full}_${deb_arch}.deb" ] } else { outputs = [ "$root_out_dir/${package}-${channel}_${chrome_version_full}_${deb_arch}.deb" ] }
 
     args = [
       "-a",
@@ -514,7 +523,7 @@ template("linux_package") {
       }
 
       inputs = packaging_files
-      outputs = [ "$root_out_dir/${package}-${channel}-${chrome_version_full}-1.${rpm_arch}.rpm" ]
+      if (channel == "stable") { outputs = [ "$root_out_dir/${package}-${chrome_version_full}-1.${rpm_arch}.rpm" ] } else { outputs = [ "$root_out_dir/${package}-${channel}-${chrome_version_full}-1.${rpm_arch}.rpm" ] }
 
       args = [
         "-a",
@@ -561,3 +570,6 @@ linux_package("unstable") {
 linux_package("canary") {
   channel = "canary"
 }
+linux_package("nightly") {
+  channel = "nightly"
+}
