diff --git a/chrome/browser/resources/settings/site_settings/category_default_setting.js b/chrome/browser/resources/settings/site_settings/category_default_setting.js
index f1aeec3e53c7c7d196111d4260b6ae6a85b02895..396dae1f7a26bb2582d19d58d37030d1dcab53c7 100644
--- a/chrome/browser/resources/settings/site_settings/category_default_setting.js
+++ b/chrome/browser/resources/settings/site_settings/category_default_setting.js
@@ -165,6 +165,7 @@ Polymer({
       case ContentSettingsTypes.PAYMENT_HANDLER:
       case ContentSettingsTypes.POPUPS:
       case ContentSettingsTypes.PROTOCOL_HANDLERS:
+      case ContentSettingsTypes.AUTOPLAY:
         // "Allowed" vs "Blocked".
         this.browserProxy.setDefaultValueForContentType(
             this.category,
