diff --git a/chrome/browser/ui/startup/BUILD.gn b/chrome/browser/ui/startup/BUILD.gn
index 8057566864cf3e7cc962c1eeb6f56aea675f600a..6e5848d57e19bda2e2cd626381d593e0e6823a48 100644
--- a/chrome/browser/ui/startup/BUILD.gn
+++ b/chrome/browser/ui/startup/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/ui/startup/sources.gni")
 import("//build/buildflag_header.gni")
 import("//build/config/chrome_build.gni")
 import("//build/config/ui.gni")
@@ -28,6 +29,7 @@ source_set("startup_tab") {
   ]
 
   public_deps = [ "//url" ]
+  public_deps += brave_browser_ui_startup_public_deps
 }
 
 source_set("startup") {
