diff --git a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesGridLayout.java b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesGridLayout.java
index 2315a2eb71a104050ec56d7a931d66bec220cf59..23973c5acf73dfc8e18bf7a4a2414b791c57602b 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesGridLayout.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesGridLayout.java
@@ -163,7 +163,7 @@ public class MostVisitedTilesGridLayout extends FrameLayout {
             }
         }
 
-        assert horizontalSpacing >= mMinHorizontalSpacing;
+        if(false)assert horizontalSpacing >= mMinHorizontalSpacing;
         assert horizontalSpacing <= mMaxHorizontalSpacing;
 
         return Pair.create(gridStart, Math.round(horizontalSpacing));
