diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 360a3851b1c99db15dfa9e3700e9072ed326d653..31953a41aca7606376ce2aeddd1f94ca4abf1b89 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3116,6 +3116,7 @@ const FeatureEntry::FeatureVariation kDesktopSharePreviewVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
