diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index 335611dcad36ab2bf058015321a9a72d8f77989b..d0034ed4502d72d467800361011703dfed1261bd 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -442,6 +442,7 @@ Browser::CreationStatus Browser::GetCreationStatusForProfile(Profile* profile) {
 // static
 Browser* Browser::Create(const CreateParams& params) {
   CHECK_EQ(CreationStatus::kOk, GetCreationStatusForProfile(params.profile));
+  BRAVE_BROWSER_CREATE
   return new Browser(params);
 }
 
@@ -453,7 +454,7 @@ Browser::Browser(const CreateParams& params)
       tab_strip_model_delegate_(
           std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
       tab_strip_model_(
-          std::make_unique<TabStripModel>(tab_strip_model_delegate_.get(),
+          std::make_unique<BraveTabStripModel>(tab_strip_model_delegate_.get(),
                                           params.profile)),
       app_name_(params.app_name),
       is_trusted_source_(params.trusted_source),
