diff --git a/chrome/browser/ssl/ask_before_http_dialog_controller.cc b/chrome/browser/ssl/ask_before_http_dialog_controller.cc
index de9f7813a8e53c16fab70438121dbc09d0330834..f281d89470a6cca5cd2d19582c81897d46e422b0 100644
--- a/chrome/browser/ssl/ask_before_http_dialog_controller.cc
+++ b/chrome/browser/ssl/ask_before_http_dialog_controller.cc
@@ -54,7 +54,7 @@ using HttpWarningReason =
 namespace {
 
 inline constexpr char kLearnMoreLink[] =
-    "https://support.google.com/chrome?p=first_mode";
+    "https://support.brave.app/hc/en-us/articles/15513090104717";
 
 // Helper to create the prompt body view based on the Ask-before-HTTP warning
 // type.
