diff --git a/chrome/browser/ui/actions/chrome_action_id.h b/chrome/browser/ui/actions/chrome_action_id.h
index efdb455d6b5912625c9f4d8c1e1bb59521e996b6..1681bef58e574f01dc4f761dd95e6a1e99e6dce0 100644
--- a/chrome/browser/ui/actions/chrome_action_id.h
+++ b/chrome/browser/ui/actions/chrome_action_id.h
@@ -469,6 +469,10 @@
 // Should be reinserted into CHROME_COMMON_ACTION_IDS when general solution to
 // add action id mappings is implemented.
 #define SIDE_PANEL_ACTION_IDS \
+  E(kActionSidePanelShowPlaylist) \
+  E(kActionSidePanelShowChatUI) \
+  E(kActionSidePanelShowBraveNews) \
+  E(kActionSidePanelShowWallet) \
   /* Side Panel items */ \
   E(kActionSidePanelShowAboutThisSite) \
   E(kActionSidePanelShowAssistant) \
@@ -497,6 +501,13 @@
     IDC_SHOW_TABS_FROM_OTHER_DEVICES_SIDE_PANEL)
 
 #define TOOLBAR_PINNABLE_ACTION_IDS \
+  E(kActionShowPartitionedStorage) \
+  E(kActionShowPsstIcon) \
+  E(kActionShowSpeedreader) \
+  E(kActionShowWaybackMachine) \
+  E(kActionShowPlaylistPageAction) \
+  E(kActionShowOnionLocation) \
+  E(kActionShowBraveNews) \
   E(kActionHome, IDC_HOME) \
   E(kActionForward, IDC_FORWARD) \
   E(kActionNewIncognitoWindow, IDC_NEW_INCOGNITO_WINDOW) \
