diff --git a/chrome/install_static/install_modes.cc b/chrome/install_static/install_modes.cc
index 8454b2ab6871c950ed976aafdbec8cfacb506e95..a2d56d3b6fcfcf23f781bcf680797a7122e4fcd9 100644
--- a/chrome/install_static/install_modes.cc
+++ b/chrome/install_static/install_modes.cc
@@ -11,6 +11,9 @@ namespace install_static {
 namespace {
 
 #if BUILDFLAG(USE_GOOGLE_UPDATE_INTEGRATION)
+#if defined(BRAVE_CHROMIUM_BUILD)
+  BRAVE_INSTALL_MODES
+#else
 std::wstring GetClientsKeyPathForApp(const wchar_t* app_guid) {
   return std::wstring(L"Software\\Google\\Update\\Clients\\").append(app_guid);
 }
@@ -24,6 +27,7 @@ std::wstring GetClientStateMediumKeyPathForApp(const wchar_t* app_guid) {
   return std::wstring(L"Software\\Google\\Update\\ClientStateMedium\\")
       .append(app_guid);
 }
+#endif
 #else
 std::wstring GetUnregisteredKeyPathForProduct(const wchar_t* product) {
   return std::wstring(L"Software\\").append(product);
