diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 8e3fde5e841e1b53e8fb16746477017f4ebbe14f..b43736dd9de6b903235947205d53f839dfee9482 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2761,6 +2761,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
