diff --git a/chrome/installer/mini_installer/BUILD.gn b/chrome/installer/mini_installer/BUILD.gn
index ce1f5a5711eeba1109bef0169a80cbe29373a843..71e9f0105edb30d649098325175f0f9928c298a3 100644
--- a/chrome/installer/mini_installer/BUILD.gn
+++ b/chrome/installer/mini_installer/BUILD.gn
@@ -143,7 +143,7 @@ if (enable_uncompressed_archive) {
       "$target_gen_dir/mini_installer/packed_files_uncompressed.rc"
 }
 
-action("mini_installer_archive") {
+import("//brave/chromium_src/chrome/installer/mini_installer/build.gni") brave_mini_installer_archive_action("mini_installer_archive") {
   script = "//chrome/tools/build/win/create_installer_archive.py"
 
   release_file = "chrome.release"
@@ -326,6 +326,7 @@ template("generate_mini_installer") {
       no_default_deps = true
       ldflags += [ "/ENTRY:MainEntryPoint" ]
     }
+if (defined(invoker.extra_deps)) { deps += invoker.extra_deps } }} generate_mini_installer("delta_installer_unsigned") { rc_file = "$target_gen_dir/packed_delta_files.rc" extra_deps = [ "//brave:delta_installer_archive" ] if (false) {
   }
 }
 
