diff --git a/chrome/android/java/src/org/chromium/chrome/browser/SwipeRefreshHandler.java b/chrome/android/java/src/org/chromium/chrome/browser/SwipeRefreshHandler.java
index 0d5401c9d2c895d8e8073f5d902bc0823199d61e..cb83d05271d97b7fa7aba794528105abf9e35460 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/SwipeRefreshHandler.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/SwipeRefreshHandler.java
@@ -99,7 +99,7 @@ public class SwipeRefreshHandler extends TabWebContentsUserData
      *
      * @param tab The Tab where the swipe occurs.
      */
-    private SwipeRefreshHandler(Tab tab) {
+    public SwipeRefreshHandler(Tab tab) {
         super(tab);
         mTab = tab;
         mTabObserver =
