diff --git a/chrome/browser/resources/extensions/BUILD.gn b/chrome/browser/resources/extensions/BUILD.gn
index 6b89fa93ad22453aa39a55ed7c97623fa9adb5da..2deee47536ad9ef9c1583656dbf0214ec6dbb9df 100644
--- a/chrome/browser/resources/extensions/BUILD.gn
+++ b/chrome/browser/resources/extensions/BUILD.gn
@@ -21,6 +21,7 @@ if (optimize_webui) {
     deps = [
       ":unpak",
       "../../../../ui/webui/resources:modulize",
+      "//brave/browser/resources/extensions:unpak",
     ]
     excludes = [ "chrome://resources/js/cr.m.js" ]
   }
