diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
index cceeec67c3b2dee2da80c8f8ba6f82b12f95f521..33e1d5513a7b4129e472c72f3ba3fe949f5cba79 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
@@ -293,6 +293,7 @@ base::string16 AvatarToolbarButton::GetAvatarTooltipText() const {
   if (IsIncognito())
     return l10n_util::GetStringUTF16(IDS_AVATAR_BUTTON_INCOGNITO_TOOLTIP);
 
+  GET_AVATAR_TOOLTIP_TEXT_
   if (profile_->IsGuestSession())
     return l10n_util::GetStringUTF16(IDS_GUEST_PROFILE_NAME);
 
