diff --git a/chrome/browser/download/internal/android/BUILD.gn b/chrome/browser/download/internal/android/BUILD.gn
index 45b3129b51cc6d6f74a51cc1f96c415ce79d90a8..294acdbecd52ec0db5a97980105c2220552c09d7 100644
--- a/chrome/browser/download/internal/android/BUILD.gn
+++ b/chrome/browser/download/internal/android/BUILD.gn
@@ -215,6 +215,7 @@ android_resources("java_resources") {
     "java/res/values/ids.xml",
     "java/res/values/styles.xml",
   ]
+  sources += brave_download_internal_java_resources
 
   deps = [
     "//chrome/browser/download/android:java_resources",
