diff --git a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
index 9fc2c17d32bf14170eda3eff9b2b36eefa68c042..ac0c7445a1746e09e95e16977e97ac279d2c2439 100644
--- a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
+++ b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
@@ -25,7 +25,7 @@ export function getHtml(this: ProfilePickerMainViewElement) {
             data-index="${index}">
         </profile-card>
       `)}
-      <cr-button id="addProfile" class="profile-item"
+      <cr-button id="addProfile" class="profile-item plain"
           @click="${this.onAddProfileClick_}"
           aria-labelledby="addProfileButtonLabel">
         <div id="addProfileButtonLabel"
