diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.cc b/chrome/browser/ui/views/location_bar/location_bar_view.cc
index 3fec6d3059a1a1376df6995cc8f0dd9fdcaa8650..d1c3457d15353c100abfa134648e0a037a6e25cd 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
@@ -750,6 +750,7 @@ void LocationBarView::Layout(PassKey) {
   // Indentation add padding when the permission chip is visible and replacing
   // the LHS icon.
   int text_overriding_permission_chip_indent = 0;
+  BRAVE_LOCATION_BAR_VIEW_LAYOUT_LEADING_DECORATIONS
   if (should_indent) {
     icon_left += icon_indent;
     text_left += text_indent;
@@ -813,6 +814,7 @@ void LocationBarView::Layout(PassKey) {
       }
       selected_keyword_view_->SetCustomImage(image);
     }
+  BRAVE_LOCATION_BAR_VIEW_LAYOUT_HANDLE_SEARCH_PROMOTION_BUTTON_VISIBILITY
   } else if (location_icon_view_->GetShowText() &&
              !ShouldChipOverrideLocationIcon()) {
     location_icon_view_->SetVisible(true);
@@ -860,6 +862,7 @@ void LocationBarView::Layout(PassKey) {
     }
   };
 
+  BRAVE_LOCATION_BAR_VIEW_LAYOUT_RIGHT_MOST_TRAILING_DECORATIONS
   add_trailing_decoration(page_action_icon_container_,
                           /*intra_item_padding=*/0);
   add_trailing_decoration(page_action_container_,
@@ -875,6 +878,7 @@ void LocationBarView::Layout(PassKey) {
   }
 
   add_trailing_decoration(clear_all_button_, /*intra_item_padding=*/0);
+  BRAVE_LOCATION_BAR_VIEW_LAYOUT_LEFT_MOST_TRAILING_DECORATIONS
 
   // Perform layout.
   int entry_width = width();
