diff --git a/chrome/browser/resources/history/BUILD.gn b/chrome/browser/resources/history/BUILD.gn
index 05321acd7f969518a8dc7781b05ce654ae2a7fe8..1b837b30229127860c7e5064688301d3e64dbac7 100644
--- a/chrome/browser/resources/history/BUILD.gn
+++ b/chrome/browser/resources/history/BUILD.gn
@@ -113,4 +113,5 @@ build_webui("build") {
     optimize_webui_host = "history"
     optimize_webui_in_files = [ "history.js" ]
   }
+  import("//brave/browser/resources/history/sources.gni") mojo_files = brave_history_mojo_files mojo_files_deps = brave_history_mojo_files_deps ts_files += brave_history_ts_files exclude_ts_preprocess_files = brave_history_ts_files preprocess_deps = brave_history_preprocess_deps
 }
