diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index a40df2ac3065c4569c62e3152faae2d8c18a442c..244e9622fee1f87081f3b919eef2c65671f7e646 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -224,6 +224,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
