diff --git a/chrome/browser/ui/extensions/extension_action_view_controller.cc b/chrome/browser/ui/extensions/extension_action_view_controller.cc
index 4b1e239146b242649c652b2b99a96d81fd56dad6..cc36febdce3cf0ac7c6e0f2fd2ae8e9e126eaf93 100644
--- a/chrome/browser/ui/extensions/extension_action_view_controller.cc
+++ b/chrome/browser/ui/extensions/extension_action_view_controller.cc
@@ -59,7 +59,6 @@ ExtensionActionViewController::ExtensionActionViewController(
       icon_factory_(browser->profile(), extension, extension_action, this),
       extension_registry_(
           extensions::ExtensionRegistry::Get(browser_->profile())) {
-  DCHECK(extensions_container);
   DCHECK(extension_action);
   DCHECK(extension);
 }
