diff --git a/chrome/browser/ui/views/page_info/page_info_bubble_view.h b/chrome/browser/ui/views/page_info/page_info_bubble_view.h
index 3fa3ca8f9466b8ef518dc3b8c36b81741921b5b7..59a3511e08125bd3d834bea6672916b7c4a3e3ca 100644
--- a/chrome/browser/ui/views/page_info/page_info_bubble_view.h
+++ b/chrome/browser/ui/views/page_info/page_info_bubble_view.h
@@ -53,6 +53,7 @@ class PageInfoBubbleView : public PageInfoBubbleViewBase,
   PageInfo* presenter_for_testing() { return presenter_.get(); }
 
  private:
+  friend class BravePageInfoBubbleView;
   PageInfoBubbleView(views::BubbleAnchor anchor,
                      const gfx::Rect& anchor_rect,
                      gfx::NativeView parent_window,
@@ -69,7 +70,7 @@ class PageInfoBubbleView : public PageInfoBubbleViewBase,
   void WebContentsDestroyed() override;
   void ChildPreferredSizeChanged(views::View* child) override;
 
-  void AnnouncePageOpened(std::u16string announcement);
+  virtual void AnnouncePageOpened(std::u16string announcement);
 
   // The presenter that controls the Page Info UI.
   std::unique_ptr<PageInfo> presenter_;
