diff --git a/chrome/elevation_service/BUILD.gn b/chrome/elevation_service/BUILD.gn
index f82cf43c78367ee2bb33dbc70ca0f8d31e68fdec..f89ebc52107d6b4a1cda94dfda69e6ac2c7054fd 100644
--- a/chrome/elevation_service/BUILD.gn
+++ b/chrome/elevation_service/BUILD.gn
@@ -11,6 +11,7 @@ import("//testing/test.gni")
 
 midl("elevation_service_idl") {
   sources = [ "elevation_service_idl.idl" ]
+  sources -= [ "elevation_service_idl.idl" ] sources += ["//brave/chromium_src/chrome/elevation_service/elevation_service_idl.idl" ]
 
   writes_tlb = true
 }
@@ -80,6 +81,7 @@ source_set("lib") {
     "crypt32.lib",
     "rpcrt4.lib",
   ]
+  deps += brave_elevation_service_lib_deps
 }
 
 source_set("service_lib") {
