diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index b5783087429ee651d21e04f6f5f11f5cb6f015a8..d7ef158012aa259d01fc51b57d7486c881bdca3f 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -370,6 +370,8 @@ static_library("browser") {
     "component_updater/trust_token_key_commitments_component_installer.h",
     "component_updater/wasm_tts_engine_component_installer.cc",
     "component_updater/wasm_tts_engine_component_installer.h",
+    "component_updater/zxcvbn_data_component_installer.cc",
+    "component_updater/zxcvbn_data_component_installer.h",
     "consent_auditor/consent_auditor_factory.cc",
     "consent_auditor/consent_auditor_factory.h",
     "content_index/content_index_metrics.cc",
@@ -2544,6 +2546,7 @@ static_library("browser") {
     "//third_party/zlib:minizip",
     "//third_party/zlib/google:compression_utils",
     "//third_party/zlib/google:zip",
+    "//third_party/zxcvbn-cpp",
     "//ui/accessibility",
     "//ui/actions",
     "//ui/base",
@@ -2582,6 +2585,7 @@ static_library("browser") {
   } else {
     sources += [ "net/net_error_diagnostics_dialog_stub.cc" ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
@@ -3610,8 +3614,6 @@ static_library("browser") {
       "component_updater/soda_component_installer.h",
       "component_updater/soda_language_pack_component_installer.cc",
       "component_updater/soda_language_pack_component_installer.h",
-      "component_updater/zxcvbn_data_component_installer.cc",
-      "component_updater/zxcvbn_data_component_installer.h",
       "contextual_cueing/contextual_cueing_helper.cc",
       "contextual_cueing/contextual_cueing_service.cc",
       "contextual_cueing/contextual_cueing_service.h",
@@ -4425,7 +4427,6 @@ static_library("browser") {
       "//services/screen_ai/public/cpp:utilities",
       "//services/screen_ai/public/mojom",
       "//third_party/crashpad/crashpad/client:common",
-      "//third_party/zxcvbn-cpp",
       "//ui/views",
       "//ui/web_dialogs",
       "//ui/webui",
@@ -8450,6 +8451,7 @@ static_library("browser_public_dependencies") {
       "//rlz:rlz_lib",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
