diff --git a/chrome/browser/ui/views/tabs/tab.cc b/chrome/browser/ui/views/tabs/tab.cc
index f5766369d48dc195eb6baced48590d4d75a0d20a..bf06a728b7a2683cd121976df9c9ac3c552d4754 100644
--- a/chrome/browser/ui/views/tabs/tab.cc
+++ b/chrome/browser/ui/views/tabs/tab.cc
@@ -283,7 +283,7 @@ Tab::Tab(TabSlotController* controller)
 
 #if BUILDFLAG(IS_CHROMEOS)
   showing_close_button_ = !controller_->IsLockedForOnTask();
-  close_button_->SetVisible(showing_close_button_);
+  close_button_->SetVisible(!controller_->ShouldAlwaysHideCloseButton() && showing_close_button_);
 #endif
 
   tab_close_button_observer_ = std::make_unique<TabCloseButtonObserver>(
@@ -372,6 +372,7 @@ void Tab::Layout(PassKey) {
   // indicator, but visually it will be smaller at kFaviconSize wide.
   gfx::Rect favicon_bounds(start, contents_rect.y(), 0, 0);
   if (showing_icon_) {
+    BRAVE_UI_VIEWS_TABS_TAB_LAYOUT_ADJUST_ICON_POSITION
     if (center_icon_) {
       // When centering the favicon, the favicon is allowed to escape the normal
       // contents rect.
@@ -425,7 +426,7 @@ void Tab::Layout(PassKey) {
          close_button_actual_size});
     close_x = visible_left - after_title_padding;
   }
-  close_button_->SetVisible(showing_close_button_);
+  close_button_->SetVisible(!controller_->ShouldAlwaysHideCloseButton() && showing_close_button_);
 
   if (showing_alert_indicator_) {
     int right = contents_rect.right();
@@ -470,6 +471,7 @@ void Tab::Layout(PassKey) {
     int title_right = contents_rect.right();
     if (showing_alert_indicator_) {
       title_right = alert_indicator_button_->x() - after_title_padding;
+      BRAVE_UI_VIEWS_TABS_TAB_ALERT_INDICATOR_POSITION
     } else if (showing_close_button_) {
       // Allow the title to overlay the close button's empty border padding.
       title_right = close_x - after_title_padding;
@@ -1198,6 +1200,7 @@ void Tab::UpdateIconVisibility() {
         !controller_->IsLockedForOnTask() &&
 #endif
         large_enough_for_close_button;
+    BRAVE_UI_VIEWS_TABS_TAB_UPDATE_ICON_VISIBILITY
     if (showing_close_button_) {
       available_width -= close_button_width;
     }
