diff --git a/chrome/installer/setup/google_chrome_behaviors.cc b/chrome/installer/setup/google_chrome_behaviors.cc
index a4d9546c583cf432c1040b02d0f220bd3c104b38..b2adfba6a355d52348e941e34def1243dc5efde7 100644
--- a/chrome/installer/setup/google_chrome_behaviors.cc
+++ b/chrome/installer/setup/google_chrome_behaviors.cc
@@ -91,8 +91,18 @@ bool IsMetricsEnabled(const base::FilePath& file_path) {
 
 }  // namespace
 
-void UpdateInstallStatus() {
-  GoogleUpdateSettings::UpdateInstallStatus();
+// If |archive_type| is INCREMENTAL_ARCHIVE_TYPE and |install_status| does not
+// indicate a successful update, "-full" is appended to Chrome's "ap" value in
+// its ClientState key if it is not present, resulting in the full installer
+// being returned from the next update check. If |archive_type| is
+// FULL_ARCHIVE_TYPE or |install_status| indicates a successful update, "-full"
+// is removed from the "ap" value. "-stage:*" values are
+// unconditionally removed from the "ap" value.
+void UpdateInstallStatus(installer::ArchiveType archive_type,
+                         installer::InstallStatus install_status) {
+  GoogleUpdateSettings::UpdateInstallStatus(
+      install_static::IsSystemInstall(), archive_type,
+      InstallUtil::GetInstallReturnCode(install_status));
 }
 
 // Returns a string holding the following URL query parameters:
