diff --git a/chrome/browser/ui/webui/whats_new/whats_new_registrar.cc b/chrome/browser/ui/webui/whats_new/whats_new_registrar.cc
index 45931a7d4aa7a1b18b30b8525e8c6612b9190a07..e951611e915a4c121770a77b99a839527a0404da 100644
--- a/chrome/browser/ui/webui/whats_new/whats_new_registrar.cc
+++ b/chrome/browser/ui/webui/whats_new/whats_new_registrar.cc
@@ -25,6 +25,7 @@ namespace features {
 void RegisterWhatsNewModules(whats_new::WhatsNewRegistry* registry) {
   // Register modules here.
   // M142
+#if BUILDFLAG(ENABLE_GLIC)
   registry->RegisterModule(WhatsNewModule(::features::kGlicIntro,
                                           "birnie@google.com",
                                           BrowserCommand::kOpenGlic));
@@ -32,6 +33,7 @@ void RegisterWhatsNewModules(whats_new::WhatsNewRegistry* registry) {
   registry->RegisterModule(WhatsNewModule(::features::kGlicLearnMore,
                                           "birnie@google.com",
                                           BrowserCommand::kOpenGlicSettings));
+#endif  // BUILDFLAG(ENABLE_GLIC)
   // 129
   registry->RegisterModule(
       WhatsNewModule("Googlepayreauth", "vinnypersky@google.com",
