diff --git a/chrome/browser/ui/webui/cr_components/searchbox/searchbox_handler.cc b/chrome/browser/ui/webui/cr_components/searchbox/searchbox_handler.cc
index c99a747c3fc33c3ee8261040b285e6a1e701fa5f..54bad0b3972d93c16c8299f65d79a9b5a35f7c13 100644
--- a/chrome/browser/ui/webui/cr_components/searchbox/searchbox_handler.cc
+++ b/chrome/browser/ui/webui/cr_components/searchbox/searchbox_handler.cc
@@ -624,6 +624,7 @@ std::string SearchboxHandler::AutocompleteIconToResourceName(
   // TODO(446953331): It's error-prone to keep the above if's up to date. When
   //   omnibox input and popup views are replaced with webUI, matches and
   //   actions can store an icon enum instead of `VectorIcon`.
+  BRAVE_AUTOCOMPLETE_ICON_TO_RESOURCE_NAME_OVERRIDE
   NOTREACHED() << "Every autocomplete icon must have an equivalent SVG "
                   "resource for the NTP Realbox. icon.name: '"
                << icon.name << "'";
