diff --git a/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/channels/ChromeChannelDefinitions.java b/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/channels/ChromeChannelDefinitions.java
index a1d4a79ba55d6fb968f632302f49860212b20a75..29d02f6d11bbe816e3d0f5eb3875087179da6735 100644
--- a/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/channels/ChromeChannelDefinitions.java
+++ b/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/channels/ChromeChannelDefinitions.java
@@ -75,6 +75,7 @@ public class ChromeChannelDefinitions extends ChannelDefinitions {
         ChannelId.COLLABORATION,
         ChannelId.DOWNLOADS,
         ChannelId.INCOGNITO,
+        BraveChannelDefinitions.ChannelId.BRAVE_ADS, BraveChannelDefinitions.ChannelId.BRAVE_ADS_BACKGROUND,
         ChannelId.MEDIA_PLAYBACK,
         ChannelId.SCREEN_CAPTURE,
         ChannelId.CONTENT_SUGGESTIONS,
@@ -142,7 +143,7 @@ public class ChromeChannelDefinitions extends ChannelDefinitions {
     // )
     // clang-format on
 
-    @StringDef({ChannelGroupId.GENERAL, ChannelGroupId.SITES})
+    @StringDef({BraveChannelDefinitions.ChannelGroupId.BRAVE_ADS, ChannelGroupId.GENERAL, ChannelGroupId.SITES})
     @Retention(RetentionPolicy.SOURCE)
     public @interface ChannelGroupId {
         String SITES = "sites";
@@ -170,6 +171,7 @@ public class ChromeChannelDefinitions extends ChannelDefinitions {
             Map<String, PredefinedChannel> map = new HashMap<>();
             Set<String> startup = new HashSet<>();
 
+            BraveChannelDefinitions.addBraveChannels(map, startup);
             map.put(
                     ChannelId.BROWSER,
                     PredefinedChannel.create(
@@ -414,6 +416,7 @@ public class ChromeChannelDefinitions extends ChannelDefinitions {
 
         static {
             Map<String, PredefinedChannelGroup> map = new HashMap<>();
+            BraveChannelDefinitions.addBraveChannelGroups(map);
             map.put(
                     ChannelGroupId.GENERAL,
                     new PredefinedChannelGroup(
