diff --git a/chrome/browser/hub/internal/BUILD.gn b/chrome/browser/hub/internal/BUILD.gn
index db11b833b402e7808c2d79d6eb11914e957e937b..534dd276105252f18ab6d1b6abe7666685aaa68e 100644
--- a/chrome/browser/hub/internal/BUILD.gn
+++ b/chrome/browser/hub/internal/BUILD.gn
@@ -43,6 +43,7 @@ android_library("java") {
     "android/java/src/org/chromium/chrome/browser/hub/TranslateHubLayoutAnimationFactory.java",
     "android/java/src/org/chromium/chrome/browser/hub/TranslateHubLayoutAnimationFactoryImpl.java",
   ]
+  sources += brave_browser_hub_internal_java_sources
   deps = [
     ":java_resources",
     "//base:base_java",
