diff --git a/chrome/browser/resources/bookmarks/command_manager.js b/chrome/browser/resources/bookmarks/command_manager.js
index 0326cdaa77299f0a8364a6583d4f3b2b3020d8a9..e232f9005a3bf1030dfbfedbdd8cc7514a4e2981 100644
--- a/chrome/browser/resources/bookmarks/command_manager.js
+++ b/chrome/browser/resources/bookmarks/command_manager.js
@@ -448,7 +448,7 @@ export const CommandManager = Polymer({
         chrome.bookmarks.export();
         break;
       case Command.HELP_CENTER:
-        window.open('https://support.google.com/chrome/?p=bookmarks');
+        window.open('https://community.brave.com');
         break;
       default:
         assert(false);
