diff --git a/chrome/browser/themes/theme_service.h b/chrome/browser/themes/theme_service.h
index e559e8d17056e4ff1fb5bdb78993775e85fd99f0..82de5cee2f169de7db7791d87a721b5fcf68e074 100644
--- a/chrome/browser/themes/theme_service.h
+++ b/chrome/browser/themes/theme_service.h
@@ -214,6 +214,7 @@ class ThemeService : public content::NotificationObserver,
   // ThemeSource no longer uses the ThemeService when it is not ready.
   bool ready_;
 
+  THEMES_THEME_SERVICE_H_
  private:
   // This class implements ui::ThemeProvider on behalf of ThemeService and keeps
   // track of the incognito state of the calling code.
