diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index b647462a8a1d16e05e8bd268fa9b5cc46c131a19..4d1a1403ddb47e85017684c507de14523c81e72e 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -357,7 +357,9 @@ cr.define('settings', function() {
 
       // TODO(tommycli): Find a way to refactor these repetitive category
       // routes.
+      // <if expr="_google_chrome">
       r.SITE_SETTINGS_ADS = r.SITE_SETTINGS.createChild('ads');
+      // </if>
       r.SITE_SETTINGS_AUTOMATIC_DOWNLOADS =
           r.SITE_SETTINGS.createChild('automaticDownloads');
       r.SITE_SETTINGS_BACKGROUND_SYNC =
@@ -421,7 +423,9 @@ cr.define('settings', function() {
       }
 
       r.PRINTING = r.ADVANCED.createSection('/printing', 'printing');
+      // <if expr="_google_chrome">
       r.CLOUD_PRINTERS = r.PRINTING.createChild('/cloudPrinters');
+      // </if>
       // <if expr="chromeos">
       r.CUPS_PRINTERS = r.PRINTING.createChild('/cupsPrinters');
       // </if>
