diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 35deffa496503f474dac1ed24bad849ae41e10d4..4ba0d392767b9f326d18f6dd23d56078a5d4517d 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4444,6 +4444,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
