diff --git a/chrome/installer/mini_installer/BUILD.gn b/chrome/installer/mini_installer/BUILD.gn
index 7189a44f5c3123e1c666feb404b30d71894a404b..3951dff756a73c9c64b44a0fb89e0a8a2709f11e 100644
--- a/chrome/installer/mini_installer/BUILD.gn
+++ b/chrome/installer/mini_installer/BUILD.gn
@@ -181,6 +181,21 @@ template("generate_mini_installer") {
       "//third_party/icu:icudata",
       chrome_dll_target,
     ]
+    if (brave_chromium_build) {
+      inputs -= [
+        "$root_out_dir/chrome.exe",
+      ]
+      inputs += [
+        "$root_out_dir/brave.exe",
+        "$root_out_dir/brave_resources.pak",
+        "$root_out_dir/brave_100_percent.pak",
+        "$root_out_dir/brave_200_percent.pak",
+      ]
+      deps += [
+        "//brave:packed_resources",
+        "//brave/build/win:copy_exe",
+      ]
+    }
 
     if (enable_hidpi) {
       args += [ "--enable_hidpi=1" ]
