diff --git a/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_device_picker_bubble_view.cc b/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_device_picker_bubble_view.cc
index 5b3f28b64d2c1f10cf143c29f3153725fe41523a..36ba760215c637b8641692f417bda1d4b4cd990e 100644
--- a/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_device_picker_bubble_view.cc
+++ b/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_device_picker_bubble_view.cc
@@ -130,6 +130,7 @@ void SendTabToSelfDevicePickerBubbleView::InitInstantSendBubble() {
   CreateHintTextLabel();
   CreateDevicesScrollView();
 
+  BRAVE_SEND_TAB_TO_SELFDEVICE_PICKER_BUBBLE_VIEW_INIT
   AddChildView(std::make_unique<views::Separator>());
   views::View* footer = AddChildView(
       BuildManageAccountDevicesLinkView(/*show_link=*/true, controller_));
