diff --git a/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc b/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc
index 1c76ec475ac5f4a44659285e92e38d5c7145283f..295220d253fe3dee083a845e9766c00add60fedc 100644
--- a/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc
+++ b/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc
@@ -957,6 +957,7 @@ void AddImportDataStrings(content::WebUIDataSource* html_source) {
     {"importPasswords", IDS_SETTINGS_IMPORT_PASSWORDS_CHECKBOX},
     {"importSearch", IDS_SETTINGS_IMPORT_SEARCH_ENGINES_CHECKBOX},
     {"importAutofillFormData", IDS_SETTINGS_IMPORT_AUTOFILL_FORM_DATA_CHECKBOX},
+    {"importCookies", IDS_SETTINGS_IMPORT_COOKIES_CHECKBOX},
     {"importChooseFile", IDS_SETTINGS_IMPORT_CHOOSE_FILE},
     {"importCommit", IDS_SETTINGS_IMPORT_COMMIT},
     {"noProfileFound", IDS_SETTINGS_IMPORT_NO_PROFILE_FOUND},
