diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index 57175d99916f76c4af2a3c74422b9871ca50e267..e68c02a50035dbd8056a7a04c9596c864ab77f09 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -414,6 +414,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
       base::FeatureList::IsEnabled(
           extensions_features::kExtensionsMenuAccessControlWithPermittedSites));
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
