diff --git a/chrome/browser/resources/settings/settings.ts b/chrome/browser/resources/settings/settings.ts
index d8783de1b83335f8dd295e1f53d3c4752a531a5c..ecb90e239e9727a2c0c3ca8585c4ebfe0ea2ea03 100644
--- a/chrome/browser/resources/settings/settings.ts
+++ b/chrome/browser/resources/settings/settings.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './settings_ui/settings_ui.js';
 
 export {ControlledRadioButtonElement} from '/shared/settings/controls/controlled_radio_button.js';
