diff --git a/chrome/browser/resources/extensions/item_list.ts b/chrome/browser/resources/extensions/item_list.ts
index c02caf2bbc1940b8be008ad1bd098f15314313b8..024f66113e4bd0e14136e4645ce53125b7c6c077 100644
--- a/chrome/browser/resources/extensions/item_list.ts
+++ b/chrome/browser/resources/extensions/item_list.ts
@@ -13,6 +13,7 @@ import {PolymerElement} from 'chrome://resources/polymer/v3_0/polymer/polymer_bu
 
 import {ExtensionsItemElement, ItemDelegate} from './item.js';
 import {getTemplate} from './item_list.html.js';
+import './brave_item_list_more_items.js'
 
 type Filter = (info: chrome.developerPrivate.ExtensionInfo) => boolean;
 
