diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 0089be5050e114497955bd70591676460d6b17bf..80040bf6503c9bd8798c5bf4ec25f392649524c1 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -576,6 +576,7 @@ void ToolbarView::Init() {
   }
 
   if (!features::IsWebUIAvatarButtonEnabled()) {
+    BRAVE_TOOLBAR_VIEW_INIT
     avatar_ =
         AddChildView(std::make_unique<AvatarToolbarButton>(browser_view_));
     bool show_avatar_toolbar_button = true;
