diff --git a/chrome/browser/devtools/devtools_ui_bindings.cc b/chrome/browser/devtools/devtools_ui_bindings.cc
index 7bb9e779ad9a7f03eaed3a0673af88fad6a9a761..656cc30353e27a7e54891f1b4d22accc13de7afb 100644
--- a/chrome/browser/devtools/devtools_ui_bindings.cc
+++ b/chrome/browser/devtools/devtools_ui_bindings.cc
@@ -2039,8 +2039,10 @@ void DevToolsUIBindings::GetHostConfig(DispatchCallback callback) {
                     std::move(starting_style_debugging));
 
   base::Value::Dict prompt_api_dict;
-  prompt_api_dict.Set("enabled", base::FeatureList::IsEnabled(
-                                     ::features::kDevToolsAiPromptApi));
+  prompt_api_dict.Set(
+      "enabled",
+      base::FeatureList::IsEnabled(::features::kDevToolsAiPromptApi) &&
+          base::FeatureList::IsEnabled(blink::features::kBuiltInAIAPI));
   prompt_api_dict.Set("allowWithoutGpu",
                       features::kDevToolsAiPromptApiAllowWithoutGpu.Get());
   response_dict.Set("devToolsAiPromptApi", std::move(prompt_api_dict));
