diff --git a/chrome/browser/ui/views/permission_bubble/permission_prompt_impl.cc b/chrome/browser/ui/views/permission_bubble/permission_prompt_impl.cc
index b1bdbf0e1ae6b49f5bd4539bda20f009065d8908..86c7a327fa0ac1e32d29ca5acee726208d075dda 100644
--- a/chrome/browser/ui/views/permission_bubble/permission_prompt_impl.cc
+++ b/chrome/browser/ui/views/permission_bubble/permission_prompt_impl.cc
@@ -132,6 +132,7 @@ PermissionsBubbleDialogDelegateView::PermissionsBubbleDialogDelegateView(
     AddChildView(label_container);
   }
 
+  AddAdditionalWidevineViewControlsIfNeeded(this, requests);
   chrome::RecordDialogCreation(chrome::DialogIdentifier::PERMISSIONS);
 }
 
