diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 7d54e51f8fab0709ee9cc494db012dea514cd007..470afde7de138b64e755dee4f659255151245e7e 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -257,6 +257,7 @@ static_library("utility") {
   if (enable_on_device_translation) {
     deps += [ "//components/on_device_translation/service:on_device_translation_service" ]
   }
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
 }
 
 if (!is_android) {
