diff --git a/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc b/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc
index 162023289249105872fa70fc7b41a4272b564e21..909d23aba6f48fc25d1d651e0ac13118caa1e4af 100644
--- a/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc
+++ b/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc
@@ -136,6 +136,7 @@ PermissionPromptBubbleView::PermissionPromptBubbleView(
 
   for (permissions::PermissionRequest* request : GetVisibleRequests())
     AddRequestLine(request);
+  BRAVE_PERMISSION_PROMPT_BUBBLE_VIEW
 
   absl::optional<std::u16string> extra_text = GetExtraText();
   if (extra_text.has_value()) {
