diff --git a/chrome/browser/resources/settings/people_page/sync_controls.js b/chrome/browser/resources/settings/people_page/sync_controls.js
index bc857c359f693190108cdd9b6547deef7a8347b4..e90e415eeb2c22d9893e4ea4c4885874d57424b4 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.js
+++ b/chrome/browser/resources/settings/people_page/sync_controls.js
@@ -77,7 +77,7 @@ Polymer({
         'sync-prefs-changed', this.handleSyncPrefsChanged_.bind(this));
 
     const router = settings.Router.getInstance();
-    if (router.getCurrentRoute() === router.getRoutes().SYNC_ADVANCED) {
+    if (router.getCurrentRoute() === router.getRoutes().BRAVE_SYNC_SETUP) {
       this.browserProxy_.didNavigateToSyncPage();
     }
   },
@@ -174,6 +174,7 @@ Polymer({
    * @private
    */
   syncControlsHidden_() {
+    return false;
     if (!this.syncStatus) {
       // Show sync controls by default.
       return false;
