diff --git a/chrome/browser/history/history_service_factory.cc b/chrome/browser/history/history_service_factory.cc
index f51a9c7b49118cddda8b75d38804c3e9875fef43..b17c0b2f00b86dd07cd46d2a127f75dcf759057f 100644
--- a/chrome/browser/history/history_service_factory.cc
+++ b/chrome/browser/history/history_service_factory.cc
@@ -30,5 +30,6 @@ std::unique_ptr<KeyedService> BuildHistoryService(
           context->GetPath(), chrome::GetChannel()))) {
     return nullptr;
   }
+  BRAVE_BUILD_HISTORY_SERVICE
   return history_service;
 }
