diff --git a/chrome/installer/util/BUILD.gn b/chrome/installer/util/BUILD.gn
index dae16bd9569eb23ce882df850bfe0295d1a9f2e9..623edadcab2b4b99fca0162670644326cb2c0138 100644
--- a/chrome/installer/util/BUILD.gn
+++ b/chrome/installer/util/BUILD.gn
@@ -1,6 +1,7 @@
 # Copyright 2014 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
+import("//brave/build/config/brave_build.gni")

 import("//base/win/embedded_i18n/generate_embedded_i18n.gni")
 import("//build/config/chrome_build.gni")
@@ -224,7 +225,20 @@ generate_embedded_i18n("generate_strings") {
 
   output_file_name_base = "installer_util_strings"
 
-  branding = branding_path_component
+  brand = "$branding_path_component"
+  if (brave_chromium_build) {
+    # When brave_strings.grd is modified, outputs should be re-generated.
+    # This is workaround. Need to change the way of using |brave_strings.grd|
+    # in |create_string_rc.py|.
+    sources = [
+      "//chrome/app/brave_strings.grd"
+    ]
+    if (!is_official_build) {
+      # TODO(shong): Remove this.
+      brand = brand + "-development"
+    }
+  }
+  branding = brand
 }
 
 # Compile the generated .rc file.
