diff --git a/chrome/browser/external_protocol/external_protocol_handler.cc b/chrome/browser/external_protocol/external_protocol_handler.cc
index 5ac521e898d4d97179a177d32105c8c3932846b4..6435e5dc1d2ff37d1c792d17d0ed84b44cf180d1 100644
--- a/chrome/browser/external_protocol/external_protocol_handler.cc
+++ b/chrome/browser/external_protocol/external_protocol_handler.cc
@@ -345,7 +345,7 @@ ExternalProtocolHandler::BlockState ExternalProtocolHandler::GetBlockState(
   // Always allow the hard-coded allowed schemes.
   for (size_t i = 0; i < std::size(kAllowedSchemes); ++i) {
     if (kAllowedSchemes[i] == scheme)
-      return DONT_BLOCK;
+      return UNKNOWN;
   }
 
   PrefService* profile_prefs = profile->GetPrefs();
