diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 26386a8c7656c4dd177228ca3c2a5287ffb1f3ad..fc9d89ada25a79993ed67231a18f10f2febabe4e 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -5004,6 +5004,7 @@ const FeatureEntry::FeatureVariation kSeamlessSigninVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
