diff --git a/chrome/browser/ui/actions/chrome_action_id.h b/chrome/browser/ui/actions/chrome_action_id.h
index d20f3fbf17491d4202ff225044caba156c2827fc..0f2eb0d3e3a2755821edbb60190918d8416f738d 100644
--- a/chrome/browser/ui/actions/chrome_action_id.h
+++ b/chrome/browser/ui/actions/chrome_action_id.h
@@ -517,6 +517,7 @@
   /* Side Panel items */ \
   E(kActionSidePanelShowAboutThisSite) \
   E(kActionSidePanelShowAssistant) \
+  E(kActionSidePanelShowPlaylist) E(kActionSidePanelShowChatUI) E(kActionSidePanelShowBraveNews) \
   E(kActionSidePanelShowBookmarks, IDC_SHOW_BOOKMARK_SIDE_PANEL) \
   E(kActionSidePanelShowComments, IDC_SHOW_COMMENTS_SIDE_PANEL) \
   E(kActionSidePanelShowCustomizeChrome, IDC_SHOW_CUSTOMIZE_CHROME_SIDE_PANEL) \
@@ -565,6 +566,8 @@
   E(kActionCopyUrl, IDC_COPY_URL) \
   E(kActionTabGroupsMenu, kSavedTabGroupsMenuId) \
   E(kActionTabSearch, IDC_TAB_SEARCH) \
+  E(kActionShowPartitionedStorage) \
+  E(kActionShowPsstIcon) \
   E(kActionSplitTab, IDC_NEW_SPLIT_TAB) \
   E(kActionFederation) \
   E(kActionGlicContextualCueing) \
