diff --git a/chrome/browser/component_updater/registration.cc b/chrome/browser/component_updater/registration.cc
index b11907b0f46d6a4aeeff96e89b731851aeac969a..f567fd73ff50aa8537d44c3e1c24e6130cae9500 100644
--- a/chrome/browser/component_updater/registration.cc
+++ b/chrome/browser/component_updater/registration.cc
@@ -142,6 +142,7 @@ void DeleteOldComponents(const base::FilePath& user_data_dir) {
 }  // namespace
 
 void RegisterComponentsForUpdate() {
+  [&]() -> void {
   auto* const cus = g_browser_process->component_updater();
 
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC)
@@ -254,6 +255,8 @@ void RegisterComponentsForUpdate() {
         FROM_HERE, {base::TaskPriority::BEST_EFFORT, base::MayBlock()},
         base::BindOnce(&DeleteOldComponents, path));
   }
+  }();
+  RegisterBraveComponentsForUpdate();
 }
 
 }  // namespace component_updater
