diff --git a/chrome/browser/sync/sync_service_factory.cc b/chrome/browser/sync/sync_service_factory.cc
index 99c819f61ce55fb740441ad2c13d095600ba29da..78fc30534cdbf22d4b38d8273dc995fd32286257 100644
--- a/chrome/browser/sync/sync_service_factory.cc
+++ b/chrome/browser/sync/sync_service_factory.cc
@@ -166,7 +166,7 @@ std::unique_ptr<KeyedService> BuildSyncService(
   }
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize();
 
   // Notify the PasswordStore of complete initialisation to resolve a circular
