diff --git a/chrome/common/extensions/BUILD.gn b/chrome/common/extensions/BUILD.gn
index e991a15a15aa9855e32eaf601dd0065edef720fa..52f9e9ba648faf1da100bd612f656d7f25bbac8a 100644
--- a/chrome/common/extensions/BUILD.gn
+++ b/chrome/common/extensions/BUILD.gn
@@ -122,6 +122,7 @@ source_set("extensions") {
 
     sources += [ "api/file_system_provider_capabilities/file_system_provider_capabilities_handler.cc" ]
   }
+  import("//brave/common/extensions/sources.gni") public += brave_chrome_common_extensions_public sources += brave_chrome_common_extensions_sources deps += brave_chrome_common_extensions_deps public_deps += brave_chrome_common_extensions_public_deps
 }
 
 # TODO(zturner): Enable this on Windows. See
