diff --git a/chrome/browser/ui/browser_commands.cc b/chrome/browser/ui/browser_commands.cc
index 59fa913a5c2366ada08ce6111a7f7c6cd62478da..eb19cf845d3653042799fb9d23a7f67294d9f75f 100644
--- a/chrome/browser/ui/browser_commands.cc
+++ b/chrome/browser/ui/browser_commands.cc
@@ -563,6 +563,7 @@ void ReloadInternal(BrowserWindowInterface* browser,
           tab_strip_model->GetWebContentsAt(selected_index));
     }
   }
+  BRAVE_RELOAD_INTERNAL
 
   base::UmaHistogramCounts100("TabStrip.Tab.ReloadCount",
                               tabs_to_reload.size());
