diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index 0f98eece3c15912858a5cf3423ee6c1f1bf6f98a..ca7a7c8a4bf06afe6cd94409d4c2c112381278b7 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -227,6 +227,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
