diff --git a/chrome/browser/resources/settings/about_page/about_page.js b/chrome/browser/resources/settings/about_page/about_page.js
index 14e8d1b3fc83229f54ee10d7bb15e2f278e94e69..23beadf6cf8cf751a487b308e1dcb599aeab4af5 100644
--- a/chrome/browser/resources/settings/about_page/about_page.js
+++ b/chrome/browser/resources/settings/about_page/about_page.js
@@ -229,7 +229,14 @@ export class SettingsAboutPageElement extends SettingsAboutPageElementBase {
 
   /** @private */
   onRelaunchTap_() {
+    // <if expr="is_macosx">
+    // Sparkle framework's relaunch api is used.
+    this.lifetimeBrowserProxy_.relaunchOnMac();
+    // </if>
+
+    // <if expr="not is_macosx">
     this.lifetimeBrowserProxy_.relaunch();
+    // </if>
   }
 
   // <if expr="not chromeos">
