diff --git a/chrome/browser/resources/settings/people_page/sync_controls.ts b/chrome/browser/resources/settings/people_page/sync_controls.ts
index be8ab78b417a98a8a671efcab0e694f9590d8eb3..fe772ddfa2e6553f22799d10f67fa6593a120a9e 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.ts
+++ b/chrome/browser/resources/settings/people_page/sync_controls.ts
@@ -143,7 +143,7 @@ export class SettingsSyncControlsElement extends
         'sync-prefs-changed', this.handleSyncPrefsChanged_.bind(this));
 
     const router = Router.getInstance();
-    if (router.getCurrentRoute() === routes.SYNC_ADVANCED) {
+    if (router.getCurrentRoute() === routes.BRAVE_SYNC_SETUP) {
       this.browserProxy_.didNavigateToSyncPage();
     }
   }
@@ -269,6 +269,7 @@ export class SettingsSyncControlsElement extends
   private disableTypeCheckBox_(
       syncStatus: SyncStatus, syncAllDataTypes: boolean,
       dataTypeManaged: boolean): boolean {
+    if (dataTypeManaged) { return true; } if (!syncStatus || !this.syncPrefs) { return syncAllDataTypes }
     if (!syncStatus) {
       return true;
     }
