diff --git a/chrome/browser/policy/profile_policy_connector.cc b/chrome/browser/policy/profile_policy_connector.cc
index ff3647a580fbdbc91141d320510a0c70ea737280..1af96229e09756b286367890ee5ec75914cc04af 100644
--- a/chrome/browser/policy/profile_policy_connector.cc
+++ b/chrome/browser/policy/profile_policy_connector.cc
@@ -429,6 +429,7 @@ void ProfilePolicyConnector::Init(
   }
 #endif
 
+  BRAVE_PROFILE_POLICY_CONNECTOR_INIT
   std::vector<std::unique_ptr<PolicyMigrator>> migrators;
 #if BUILDFLAG(IS_WIN)
   migrators.push_back(
