diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 6fa22a1e7ef1ef9d0ba72d3835f00ec6c9baf6c6..c47286e203e679397ba65c35186bcb2bbbe47aaa 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -784,6 +784,7 @@ static_library("ui") {
     # gets componentized.
     "//chrome/browser/ui/extensions:impl",
   ]
+  import("//brave/browser/sources.gni") public_deps += [ "//brave/browser/ui" ] deps += brave_chrome_browser_ui_allow_circular_includes_from allow_circular_includes_from += brave_chrome_browser_ui_allow_circular_includes_from
 
   if (enable_vr && is_win) {
     deps += [ "//chrome/browser/vr:vr_base" ]
@@ -3346,7 +3347,10 @@ static_library("ui") {
         "//chrome/updater/app/server/win:updater_legacy_idl",
       ]
     } else {
-      sources += [ "webui/help/version_updater_basic.cc" ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [ "webui/help/version_updater_basic.cc" ]
+      }
     }
   }
 
@@ -5495,6 +5499,7 @@ static_library("ui_public_dependencies") {
   if (enable_extensions_core) {
     public_deps += [ "//extensions/browser" ]
   }
+  public_deps += [ "//brave/browser/ui:ui_public_dependencies" ]
   if (!is_android) {
     public_deps += [
       "//build:branding_buildflags",
@@ -5618,6 +5623,7 @@ source_set("browser_navigator_params_headers") {
     "//url",
   ]
   deps = []
+  public_deps += [ "//brave/browser/ui:browser_navigator_params_headers_public_dependencies" ]
 }
 
 if (!is_android) {
