diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index cf418b5d1ce0c88453f391bc67b7e87e40f28f26..e5bcbefad259d5813f3dc31918937c1128765e45 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4409,6 +4409,7 @@ const FeatureEntry::FeatureVariation kEnableCanvasNoiseVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
