diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index 9e5760e10f99967294208a91ff406a502b873d3b..b3143375715e958aab6b2d03e8a50966c0626a46 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -98,6 +98,7 @@ generate_grd("build_grd") {
     deps = [ ":build_ts" ]
     manifest_files = [ "$target_gen_dir/tsconfig.manifest" ]
   }
+  deps += [ "//brave/browser/resources/settings:resources" ]
 }

 preprocess_if_expr("preprocess") {
@@ -786,4 +787,6 @@ ts_library("build_ts") {
     ":preprocess",
     ":preprocess_generated",
   ]
+  in_files += brave_deps_chrome_browser_resources_settings_in_files extra_deps += brave_deps_chrome_browser_resources_settings_extra_deps
+
 }
