diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ui/ExclusiveAccessBubble.java b/chrome/android/java/src/org/chromium/chrome/browser/ui/ExclusiveAccessBubble.java
index 55091678b955f3d817c59cc69491d1076cfe180b..45b39dfb74e0183ea1368f1642048039cf463946 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ui/ExclusiveAccessBubble.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ui/ExclusiveAccessBubble.java
@@ -18,6 +18,11 @@ import org.chromium.ui.base.DeviceInput;
  */
 @NullMarked
 public class ExclusiveAccessBubble {
+    @CalledByNative
+    public boolean shouldShowFullscreenNotice() {
+        return org.chromium.chrome.browser.preferences.ChromeSharedPreferences.getInstance()
+                .readBoolean(org.chromium.base.BravePreferenceKeys.BRAVE_SHOW_FULLSCREEN_NOTICES, true);
+    }
     private static final int EXCLUSIVE_ACCESS_SNACKBAR_DURATION_MS = 3800;
 
     private final ExclusiveAccessContext mParentContext;
