diff --git a/chrome/browser/ui/webui/signin/profile_customization_ui.cc b/chrome/browser/ui/webui/signin/profile_customization_ui.cc
index 7e701a751fcb21315113fac91394e288a2d7db33..8ce80e95008e50ac64d4da0113d642bc7c11b59d 100644
--- a/chrome/browser/ui/webui/signin/profile_customization_ui.cc
+++ b/chrome/browser/ui/webui/signin/profile_customization_ui.cc
@@ -42,6 +42,8 @@ ProfileCustomizationUI::ProfileCustomizationUI(content::WebUI* web_ui)
       profile, chrome::kChromeUIProfileCustomizationHost);
 
   static constexpr webui::ResourcePath kResources[] = {
+      {"profile_customization_app-chromium.js",
+       IDR_SIGNIN_PROFILE_CUSTOMIZATION_PROFILE_CUSTOMIZATION_APP_CHROMIUM_JS},
       {"profile_customization_app.js",
        IDR_SIGNIN_PROFILE_CUSTOMIZATION_PROFILE_CUSTOMIZATION_APP_JS},
       {"profile_customization_app.css.js",
@@ -128,6 +130,7 @@ ProfileCustomizationUI::ProfileCustomizationUI(content::WebUI* web_ui)
     // so we force it here.
     Initialize(base::DoNothing());
   }
+  AddProfileCustomizationData(source);
 }
 
 ProfileCustomizationUI::~ProfileCustomizationUI() = default;
