diff --git a/chrome/browser/ui/tabs/tab_strip_prefs.cc b/chrome/browser/ui/tabs/tab_strip_prefs.cc
index b0d357c347745dc16244d9bb1a4b30360abd55b3..f894ba26575961e50c8d1bbda82f3e5d3bf2d953 100644
--- a/chrome/browser/ui/tabs/tab_strip_prefs.cc
+++ b/chrome/browser/ui/tabs/tab_strip_prefs.cc
@@ -28,6 +28,7 @@ std::optional<bool> g_tab_search_trailing_tabstrip_at_startup;
 namespace tabs {
 
 bool GetDefaultTabSearchRightAligned() {
+  BRAVE_GET_DEFAULT_TAB_SEARCH_RIGHT_ALIGNED
   // These platforms are all left aligned, the others should be right.
 #if BUILDFLAG(IS_CHROMEOS) || BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_WIN)
   return false;
