diff --git a/chrome/android/java/src/org/chromium/chrome/browser/notifications/settings/NotificationSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/notifications/settings/NotificationSettings.java
index f96759fee495d61a4f22db081f17c067cc4001a4..b27a006de4bf07b7f719ff56a99f2444c00f0cc8 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/notifications/settings/NotificationSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/notifications/settings/NotificationSettings.java
@@ -28,7 +28,7 @@ import org.chromium.components.content_settings.ContentSettingsType;
  * notification channels at the top level and links to website specific notifications. This is only
  * used on pre-O devices, devices on Android O+ will link to the Android notification settings.
  */
-public class NotificationSettings extends PreferenceFragmentCompat {
+public class NotificationSettings extends org.chromium.chrome.browser.settings.BravePreferenceFragment {
     // These are package-private to be used in tests.
     static final String PREF_FROM_WEBSITES = "from_websites";
     static final String PREF_SUGGESTIONS = "content_suggestions";
