diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 85d4db7a958ea4a0f5f325fc9a128d60157b0f1d..41de92d40836a4bcc551a79b2d9f89b92180561b 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -868,6 +868,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
