diff --git a/chrome/chrome_repack_locales.gni b/chrome/chrome_repack_locales.gni
index 203d54004d3246cc2ac412481c3fdf9e65729619..1585055b3ebcfa17567c76d80d5934b9ec9ac9f9 100644
--- a/chrome/chrome_repack_locales.gni
+++ b/chrome/chrome_repack_locales.gni
@@ -37,6 +37,7 @@ template("chrome_repack_locales") {
       "${root_gen_dir}/ui/strings/app_locale_settings_",
       "${root_gen_dir}/ui/strings/ui_strings_",
     ]
+    source_patterns += brave_locale_source_patterns
     if (!defined(deps)) {
       deps = []
     }
@@ -54,6 +55,7 @@ template("chrome_repack_locales") {
       "//ui/strings:app_locale_settings",
       "//ui/strings:ui_strings",
     ]
+    deps += brave_locale_deps
     if (defined(invoker.deps)) {
       deps += invoker.deps
     }
