diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 7c4a3b819cfd51483374859e96adef63b95d36b3..0411b803e963f5909798e01a5ee307e478f029c7 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -671,6 +671,7 @@ class Browser : public TabStripModelObserver,
   // Sets the browser's user title. Setting it to an empty string clears it.
   void SetWindowUserTitle(const std::string& user_title);
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
