diff --git a/chrome/browser/android/browsing_data/browsing_data_bridge.cc b/chrome/browser/android/browsing_data/browsing_data_bridge.cc
index 55f33f633529d7c2d4c4fbac2abd8b2735059531..16bbc1d775a31ac0c95372045f602fb8f47f94b2 100644
--- a/chrome/browser/android/browsing_data/browsing_data_bridge.cc
+++ b/chrome/browser/android/browsing_data/browsing_data_bridge.cc
@@ -115,6 +115,7 @@ static void JNI_BrowsingDataBridge_ClearBrowsingData(
         remove_mask |= chrome_browsing_data_remover::DATA_TYPE_CONTENT_SETTINGS;
         break;
       case browsing_data::BrowsingDataType::DOWNLOADS:
+      BRAVE_CLEAR_BROWSING_DATA
       case browsing_data::BrowsingDataType::HOSTED_APPS_DATA:
         // Only implemented on Desktop.
         NOTREACHED();
