diff --git a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java
index 398a0dfa43025cbc54398fe386d2f531cbf1bf8f..bbc88e3e095ed9e91b5ab345d958f0a3a9a0c79e 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuItem.java
@@ -70,6 +70,7 @@ class ChromeContextMenuItem {
         Item.BACK,
         Item.FORWARD,
         Item.RELOAD,
+        Item.COPY_CLEAN_LINK,
     })
     @Retention(RetentionPolicy.SOURCE)
     public @interface Item {
@@ -125,7 +126,7 @@ class ChromeContextMenuItem {
         int FORWARD = 39;
         int RELOAD = 40;
         // ALWAYS UPDATE!
-        int NUM_ENTRIES = 41;
+        int COPY_CLEAN_LINK = 41;int NUM_ENTRIES = 42;
     }
 
     /** Mapping from {@link Item} to the ID found in the ids.xml. */
@@ -171,6 +172,7 @@ class ChromeContextMenuItem {
         R.id.contextmenu_back, // Item.BACK
         R.id.contextmenu_forward, // Item.FORWARD
         R.id.contextmenu_reload, // Item.RELOAD
+        R.id.contextmenu_copy_clean_link, // Item.COPY_CLEAN_LINK
     };
 
     /** Mapping from {@link Item} to the ID of the string that describes the action of the item. */
@@ -216,6 +218,7 @@ class ChromeContextMenuItem {
         R.string.contextmenu_back, // Item.BACK
         R.string.contextmenu_forward, // Item.FORWARD
         R.string.contextmenu_reload, // Item.RELOAD
+        R.string.contextmenu_copy_clean_link, // Item.COPY_CLEAN_LINK
     };
 
     /**
