diff --git a/chrome/browser/resources/side_panel/read_anything/BUILD.gn b/chrome/browser/resources/side_panel/read_anything/BUILD.gn
index a7d7017bc27e73d14a55755ce01f60f74d17f4ad..7eaaffd073e954fdf15897a32d02ca56fe9d3919 100644
--- a/chrome/browser/resources/side_panel/read_anything/BUILD.gn
+++ b/chrome/browser/resources/side_panel/read_anything/BUILD.gn
@@ -115,7 +115,7 @@ build_webui("build") {
             target_gen_dir) ]
 
   webui_context_type = "untrusted"
-  optimize = optimize_webui
+  optimize = false
   if (optimize) {
     optimize_webui_host =
         "chrome-untrusted://read-anything-side-panel.top-chrome"
