diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index cd81c1609151223e66295d3eaca301788598bf27..654a6904c31f1c0f9a7f7af2b10e61bf69448357 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -666,6 +666,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
