diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 6ded45bba5600b8f989ec29cbec760b949b1abd2..2a7e0341b6d2215e6d99aa6f0f3c0f20ad21cd8b 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -1973,6 +1973,7 @@ jumbo_split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
