diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 05b8f1996a55b29373346b8ae0051f5dc168abb1..9718bf76728f04f40d840cde69cea20d972ac12e 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -126,6 +126,7 @@ class AvatarToolbarButton : public ToolbarButton {
   void NotifyShowSigninPausedDelayEnded() const;
 
  private:
+  BRAVE_AVATAR_BUTTON_H
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
