diff --git a/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc b/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
index 61f74ef8d014086a4f36c05a60eb9807479773fb..2470f91048cd98bc44a27c643897f0e652d87ce3 100644
--- a/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
+++ b/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
@@ -738,7 +738,7 @@ BrowserViewTabbedLayoutImpl::CalculateProposedLayout(
   }
 
   BrowserLayoutParams params = layout_data_->revised_params;
-  bool needs_exclusion = true;
+  bool needs_exclusion = !delegate().ShouldShowVerticalTabs();
   const HorizontalLayout& horizontal_layout = layout_data_->horizontal_layout;
 
   // Lay out horizontal tab strip region if present.
