diff --git a/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java b/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java
index 3985c8f2b5b66482f6cd73518a3bb270843110dc..97de32ec70399ee5318a4a3a8275645a150a9ff2 100644
--- a/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java
+++ b/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java
@@ -301,7 +301,7 @@ public class FeedSurfaceMediator
         // Start is showing, update the value of Pref.ENABLE_SNIPPETS_BY_DSE here. The
         // value should be updated before adding an observer to prevent an extra call of
         // updateContent().
-        getPrefService()
+        if (FeedFeatures.isFeedEnabled(mProfile)) getPrefService()
                 .setBoolean(
                         Pref.ENABLE_SNIPPETS_BY_DSE,
                         !mIsNewTabSearchEngineUrlAndroidEnabled
