diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index f99153b36f3fac6024c6f57659c5c58aeed151df..0db4082da608eef286989fe2b34b70a8bd255b56 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -10,7 +10,7 @@ import("./settings.gni")
 
 assert(!is_android, "Android does not use these settings")
 
-build_webui("build") {
+import("//brave/browser/resources/settings/build_webui.gni") brave_settings_build_webui("build") {
   grd_prefix = "settings"
 
   static_files = [
