diff --git a/chrome/common/extensions/BUILD.gn b/chrome/common/extensions/BUILD.gn
index fdcf7eaac6b2511536d709efba7f97519a13a6fb..d98a383b3014a682965b3b7002881cb5222c01bb 100644
--- a/chrome/common/extensions/BUILD.gn
+++ b/chrome/common/extensions/BUILD.gn
@@ -119,6 +119,7 @@ source_set("extensions") {
 
     sources += [ "api/file_system_provider_capabilities/file_system_provider_capabilities_handler.cc" ]
   }
+  import("//brave/common/extensions/sources.gni") public += brave_chrome_common_extensions_public sources += brave_chrome_common_extensions_sources deps += brave_chrome_common_extensions_deps public_deps += brave_chrome_common_extensions_public_deps
 }
 
 # TODO(zturner): Enable this on Windows. See
