diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index bf29364c22715208cf40701e3ec7102d7d120cde..e0521460a6c6fba865c3d443f9b183e3d323747f 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -72,6 +72,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
