diff --git a/chrome/browser/ui/browser_actions.h b/chrome/browser/ui/browser_actions.h
index 00d799473e4b80bef2d2f0fc9db8f4b067e0171b..f7b4cca1efc76e68acb20f79c409968d3ccbbb52 100644
--- a/chrome/browser/ui/browser_actions.h
+++ b/chrome/browser/ui/browser_actions.h
@@ -22,7 +22,7 @@ class BrowserActions {
   explicit BrowserActions(Browser& browser);
   BrowserActions(const BrowserActions&) = delete;
   BrowserActions& operator=(const BrowserActions&) = delete;
-  ~BrowserActions();
+  virtual ~BrowserActions();
 
   static std::u16string GetCleanTitleAndTooltipText(std::u16string string);
 
