diff --git a/chrome/browser/password_manager/android/BUILD.gn b/chrome/browser/password_manager/android/BUILD.gn
index ce75cb153701f95c54b0938046d43f40827879f7..b32b10c9564fd18515758e3c6b5e71252b77fa6f 100644
--- a/chrome/browser/password_manager/android/BUILD.gn
+++ b/chrome/browser/password_manager/android/BUILD.gn
@@ -189,6 +189,7 @@ android_library("java") {
   ]
 
   resources_package = "org.chromium.chrome.browser.password_manager"
+  import("//brave/browser/password_manager/android/java_sources.gni") sources += brave_browser_password_manager_java_sources deps += brave_browser_password_manager_deps
 }
 
 generate_jni("jni_headers") {
@@ -209,6 +210,7 @@ generate_jni("jni_headers") {
     "java/src/org/chromium/chrome/browser/password_manager/PasswordStoreCredential.java",
     "java/src/org/chromium/chrome/browser/password_manager/PasswordSyncControllerDelegateBridgeImpl.java",
   ]
+  import("//brave/browser/password_manager/android/java_sources.gni") sources += brave_browser_password_manager_jni_headers
 }
 
 android_library("utils_java") {
