diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 5040542f30fbb8b6789226738f91b21a2815111f..31408691a856793ea0f33edf56ace182ba97d10e 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -299,14 +299,14 @@ const FeatureEntry::Choice kTouchTextSelectionStrategyChoices[] = {
 const FeatureEntry::Choice kTraceUploadURL[] = {
     {flags_ui::kGenericExperimentChoiceDisabled, "", ""},
     {flag_descriptions::kTraceUploadUrlChoiceOther, switches::kTraceUploadURL,
-     "https://performance-insights.appspot.com/upload?tags=flags,Other"},
+     "https://no-thanks.invalid"},
     {flag_descriptions::kTraceUploadUrlChoiceEmloading,
      switches::kTraceUploadURL,
-     "https://performance-insights.appspot.com/upload?tags=flags,emloading"},
+     "https://no-thanks.invalid"},
     {flag_descriptions::kTraceUploadUrlChoiceQa, switches::kTraceUploadURL,
-     "https://performance-insights.appspot.com/upload?tags=flags,QA"},
+     "https://no-thanks.invalid"},
     {flag_descriptions::kTraceUploadUrlChoiceTesting, switches::kTraceUploadURL,
-     "https://performance-insights.appspot.com/upload?tags=flags,TestingTeam"}};
+     "https://no-thanks.invalid"}};
 
 const FeatureEntry::Choice kPassiveListenersChoices[] = {
     {flags_ui::kGenericExperimentChoiceDefault, "", ""},
@@ -6784,6 +6784,7 @@ const FeatureEntry kFeatureEntries[] = {
     // "LoginCustomFlags" in tools/metrics/histograms/enums.xml. See "Flag
     // Histograms" in tools/metrics/histograms/README.md (run the
     // AboutFlagsHistogramTest unit test to verify this process).
+    BRAVE_FEATURE_ENTRIES
 };
 
 class FlagsStateSingleton : public flags_ui::FlagsState::Delegate {
