diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 5778df27f0c116633d48c8ed4729336dc7027671..84fa23f312575d64e820fa0a9f22c16762f7a500 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -70,6 +70,7 @@ class AvatarToolbarButton : public ToolbarButton,
   // ToolbarButton:
   void NotifyClick(const ui::Event& event) override;
 
+BRAVE_AVATAR_BUTTON_H
  private:
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
