diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index cb76914c7ca0268b2e39c0b50cfa5eccb65bf907..8de76f84eaba7ca7fcaa5cc7c638564f925dbbc9 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -32,7 +32,7 @@ import("java_sources.gni")
 import("static_initializers.gni")
 import("trichrome.gni")
 
-chrome_public_manifest_package = "org.chromium.chrome"
+declare_args() { chrome_public_manifest_package = "org.chromium.chrome" }
 chrome_public_test_manifest_package = "org.chromium.chrome.tests"
 
 chrome_public_jinja_variables =
@@ -58,6 +58,7 @@ chrome_jni_registration_header =
 chrome_jni_for_test_registration_header = "$root_build_dir/gen/chrome/browser/android/chrome_jni_for_test_registration.h"
 
 jinja_template("chrome_public_android_manifest") {
+  includes = brave_android_manifest_includes
   input = "java/AndroidManifest.xml"
   output = chrome_public_android_manifest
   variables = chrome_public_jinja_variables
@@ -68,6 +69,7 @@ jinja_template("chrome_public_android_manifest") {
 }
 
 jinja_template("chrome_modern_public_android_manifest") {
+  includes = brave_android_manifest_includes
   input = "java/AndroidManifest.xml"
   output = chrome_modern_public_android_manifest
   variables = chrome_public_jinja_variables
@@ -116,6 +118,7 @@ android_resources("chrome_app_java_resources") {
     "//third_party/android_deps:com_android_support_preference_v7_java",
     "//third_party/android_deps:com_android_support_recyclerview_v7_java",
   ]
+  deps += ["//brave/build/android:android_brave_strings_grd"]
   custom_package = "org.chromium.chrome"
 }
 
@@ -399,6 +402,7 @@ android_library("chrome_java") {
   ]
 
   deps += feed_deps
+  deps += brave_chrome_java_deps
 
   srcjar_deps = [
     ":browser_build_config",
@@ -551,6 +555,7 @@ group("jni_headers") {
     "//chrome/lib/image_fetcher/public/android:jni_headers",
     "//chrome/lib/util/public/android:jni_headers",
   ]
+  public_deps += ["//brave/build/android:jni_headers"]
 }
 
 android_library("bundle_canary_java") {
@@ -1423,6 +1428,7 @@ if (current_toolchain == default_toolchain) {
         ":${_variant}_locale_pak_assets",
         ":${_variant}_paks",
       ]
+      deps += [ "//brave:brave_pak_assets" ]
       if (_is_monochrome) {
         deps += [ "//android_webview:locale_pak_assets" ]
       }
