diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index d04103392bf00d8661ef08a71c2fb8aaa582bac3..79c5ccc0d35ddbc4d8339ad44a10893203ef6fc0 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2551,6 +2551,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
