diff --git a/chrome/browser/profile_resetter/reset_report_uploader.cc b/chrome/browser/profile_resetter/reset_report_uploader.cc
index 3bfa1f4eef6d6d0871766af221d5fb2a492b1bf0..a010506d76a647c0c8801c7b581aa60f6e9fe481 100644
--- a/chrome/browser/profile_resetter/reset_report_uploader.cc
+++ b/chrome/browser/profile_resetter/reset_report_uploader.cc
@@ -51,6 +51,7 @@ void ResetReportUploader::DispatchReport(
 
 void ResetReportUploader::DispatchReportInternal(
     const std::string& request_data) {
+  if ((true)) return; // feature disabled in Brave
   // Create traffic annotation tag.
   net::NetworkTrafficAnnotationTag traffic_annotation =
       net::DefineNetworkTrafficAnnotation("profile_resetter_upload", R"(
