diff --git a/chrome/browser/ui/startup/startup_browser_creator_impl.cc b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
index c19d4ba646850e49d845046f01dc46b661d71acb..2747148bc0979a7a8b0a26170067d20acc83a4dc 100644
--- a/chrome/browser/ui/startup/startup_browser_creator_impl.cc
+++ b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
@@ -367,6 +367,9 @@ BrowserWindowInterface* StartupBrowserCreatorImpl::OpenTabsInBrowser(
   custom_handlers::ProtocolHandlerRegistry* registry =
       ProtocolHandlerRegistryFactory::GetForBrowserContext(profile_);
   for (auto& tab : tabs) {
+    if (BraveIsScheduledForCleanup(tab.url, profile_)) {
+      continue;
+    }
     // We skip URLs that we'd have to launch an external protocol handler for.
     // This avoids us getting into an infinite loop asking ourselves to open
     // a URL, should the handler be (incorrectly) configured to be us. Anyone
@@ -424,7 +427,7 @@ BrowserWindowInterface* StartupBrowserCreatorImpl::OpenTabsInBrowser(
       params.disposition = WindowOpenDisposition::CURRENT_TAB;
       params.tabstrip_add_types = ADD_NONE;
       first_tab = false;
-      Navigate(&params);
+      BraveNavigateStartupTab(tab, browser, params);
       continue;
     }
 
@@ -454,7 +457,7 @@ BrowserWindowInterface* StartupBrowserCreatorImpl::OpenTabsInBrowser(
     }
 #endif  // BUILDFLAG(ENABLE_RLZ)
 
-    Navigate(&params);
+    BraveNavigateStartupTab(tab, browser, params);
     first_tab = false;
   }
   if (!browser->GetTabStripModel()->GetActiveWebContents() &&
