diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index b863c90ec58dbe5c6ebe1197064a3516c1b1cdc0..e2fb76952bd77a3bbd1d629d89a776e4a1ac4f17 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2539,6 +2539,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
