diff --git a/chrome/android/java/src/org/chromium/chrome/browser/AppHooks.java b/chrome/android/java/src/org/chromium/chrome/browser/AppHooks.java
index 9670f5b1aa017cc4969afa753c1909e2ce24bccb..9624bea5328cfb9318f91d583b780e1b02125fa3 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/AppHooks.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/AppHooks.java
@@ -73,7 +73,7 @@ public abstract class AppHooks {
 
     @CalledByNative
     public static AppHooks get() {
-        if (sInstance == null) sInstance = new AppHooksImpl();
+        if (sInstance == null) sInstance = new BraveAppHooks();
         return sInstance;
     }
 
