diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index 89254771117da2948bc176b881873e0d22f5ade6..6d7915b59f0fdc153262c2e30b1cc2c15932fdc3 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -101,6 +101,7 @@ bool Profile::OTRProfileID::AllowsBrowserWindows() const {
   // DevTools::BrowserContext, MediaRouter::Presentation, and
   // CaptivePortal::Signin are exceptions to this ban.
   if (*this == PrimaryID() ||
+      BRAVE_ALLOWS_BROWSER_WINDOWS
       base::StartsWith(profile_id_, kDevToolsOTRProfileIDPrefix,
                        base::CompareCase::SENSITIVE) ||
       base::StartsWith(profile_id_, kMediaRouterOTRProfileIDPrefix,
