diff --git a/chrome/browser/sync/sync_service_factory.cc b/chrome/browser/sync/sync_service_factory.cc
index be29547b53dc6c1cb893eb0bb05f595a7d248102..a96872ba60a0aad9ac850edd843723c77bd765ea 100644
--- a/chrome/browser/sync/sync_service_factory.cc
+++ b/chrome/browser/sync/sync_service_factory.cc
@@ -154,7 +154,7 @@ std::unique_ptr<KeyedService> BuildSyncService(
   }
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize();
 
   // Hook |sync_service| into PersonalDataManager (a circular dependency).
