diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 0a07024cb329719e4e9ca9e080c1733093e88b5a..63d42ba7b0b0475b81115e0681c91e2c7004a0b6 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4807,6 +4807,7 @@ const FeatureEntry::FeatureVariation kTouchToSearchCalloutVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
