diff --git a/chrome/tools/build/win/create_installer_archive.py b/chrome/tools/build/win/create_installer_archive.py
index 07387a69ada91e01b08043dfdc2029228890f937..dafa83866f9d4305ec2fa5173a5007c73bbe7cc4 100755
--- a/chrome/tools/build/win/create_installer_archive.py
+++ b/chrome/tools/build/win/create_installer_archive.py
@@ -645,6 +645,7 @@ def _ParseOptions():
            'with the installer archive {x86|x64}.')
   parser.add_option('-v', '--verbose', action='store_true', dest='verbose',
                     default=False)
+  parser.add_option('--skip_signing', action='store_true', default=False)
 
   options, _ = parser.parse_args()
   if not options.build_dir:
@@ -673,6 +674,7 @@ def _ParseOptions():
 
   return options
 
+from import_inline import inline_file_from_src; inline_file_from_src("brave/chromium_src/chrome/tools/build/win/create_installer_archive.py", globals(), locals())
 
 if '__main__' == __name__:
   options = _ParseOptions()
