diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index d21848953168441030c227a5b6cfb096b2d03417..60e7e0eeeee4b84eb38e0ec7478c6ccc103f0a01 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -87,6 +87,7 @@ source_set("channel_info") {
   } else if (is_fuchsia) {
     sources += [ "channel_info_fuchsia.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -525,6 +526,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
