diff --git a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list_header.ts b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list_header.ts
index 05bb5d0df736cdce169e00abc475a938caab6182..d6ae1bcd2dff52678c996f282f4713fd156053f9 100644
--- a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list_header.ts
+++ b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list_header.ts
@@ -112,6 +112,7 @@ export class PowerBookmarksListHeaderElement extends CrLitElement {
       label: loadTimeData.getString('sortReverseAlphabetically'),
       lowerLabel: loadTimeData.getString('sortReverseAlphabetically'),
     },
+    { sortOrder: SortOrder.kCustom, label: loadTimeData.getString('sortCustom'), lowerLabel: loadTimeData.getString('sortCustomLower'), },
   ];
   protected accessor webuiRoundedIconsEnabled_: boolean =
       loadTimeData.getBoolean('webuiRoundedIconsEnabled');
