diff --git a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/UrlBarMediator.java b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/UrlBarMediator.java
index 31bb1db62c3abc23805d49d97a3d4f654500be4d..83e735be7ecc0332793f4102e26a57d381f0b2a7 100644
--- a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/UrlBarMediator.java
+++ b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/UrlBarMediator.java
@@ -3,7 +3,7 @@
 // found in the LICENSE file.
 
 package org.chromium.chrome.browser.omnibox;
-
+  import org.chromium.base.Log;
 import android.content.ClipData;
 import android.content.ClipboardManager;
 import android.content.Context;
@@ -356,6 +356,10 @@ class UrlBarMediator
     /** @see UrlTextChangeListener */
     @Override
     public void onTextChanged(String textWithoutAutocomplete, String textWithAutocomplete) {
+Log.e("TAG", "UrlBarMediator.onTextChanged 000 textWithoutAutocomplete="+textWithoutAutocomplete);
+Log.e("TAG", "UrlBarMediator.onTextChanged 000 textWithAutocomplete   ="+textWithAutocomplete);
+Log.e("TAG", "UrlBarMediator.onTextChanged 001 mUrlTextChangeListeners.size()="+mUrlTextChangeListeners.size());
+
         for (int i = 0; i < mUrlTextChangeListeners.size(); i++) {
             mUrlTextChangeListeners.get(i).onTextChanged(
                     textWithoutAutocomplete, textWithAutocomplete);
