diff --git a/chrome/browser/importer/external_process_importer_host.h b/chrome/browser/importer/external_process_importer_host.h
index 69d118b9986047c4ac0418106ddff6db8377df73..ecd34b75ecb8040e53deb80086d5ce723b241609 100644
--- a/chrome/browser/importer/external_process_importer_host.h
+++ b/chrome/browser/importer/external_process_importer_host.h
@@ -75,6 +75,7 @@ class ExternalProcessImporterHost
   void NotifyImportItemEnded(importer::ImportItem item);
   void NotifyImportEnded();
 
+  BRAVE_EXTERNAL_PROCESS_IMPORTER_HOST_H
  private:
   // ExternalProcessImporterHost deletes itself in OnImportEnded().
   ~ExternalProcessImporterHost() override;
