diff --git a/chrome/browser/ui/browser_window/internal/browser_window_features.cc b/chrome/browser/ui/browser_window/internal/browser_window_features.cc
index 34c59dbb27c180ade4035ec6f9868ad8dfb0a9a0..35a92cb777b3c55fece8035e7de7d2457da4b0a9 100644
--- a/chrome/browser/ui/browser_window/internal/browser_window_features.cc
+++ b/chrome/browser/ui/browser_window/internal/browser_window_features.cc
@@ -307,7 +307,7 @@ void BrowserWindowFeatures::Init(BrowserWindowInterface* browser) {
           *browser, fullscreen_controller_.get(), app_browser_controller_.get(),
           browser->GetType(),
           BrowserInitState::From(browser)->create_params().trusted_source,
-          browser->GetUnownedUserDataHost());
+          browser->GetUnownedUserDataHost(), browser->GetFeatures().vertical_tab_controller()->GetWeakPtr());
 
   side_panel_registry_ =
       GetUserDataFactory().CreateInstance<SidePanelRegistry>(*browser, browser);
