diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
index ceb88c8fbd52876c89cc3a2dc2b776739fb078a8..9e9d59c6aff1237e97000525ef1a7dfee60f43fb 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
@@ -573,6 +573,7 @@ void BookmarkBubbleView::ShowBubble(
       dialog_model_builder.Build(), bubble_anchor,
       views::BubbleBorder::TOP_RIGHT);
   bookmark_bubble_ = bubble.get();
+  bubble->SetArrow(views::BubbleBorder::TOP_LEFT);
   bubble->SetHighlightedElement(kBookmarkStarViewElementId);
 
   if (ShouldShowShoppingCollectionFootnote(profile, bookmark_model,
