diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 540fe15f83fd57364e842ee27895b5f323a78b50..3489852e0b2a334f40b3b950d7c0e767b0955be9 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2142,6 +2142,7 @@ jumbo_split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
