diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index dbf7a49288ee8e895c02f7e8f8a17cd1ec608657..8fb0f43365777c7756f282f73ea2024871d343d2 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -314,14 +314,14 @@ const FeatureEntry::Choice kTouchTextSelectionStrategyChoices[] = {
 const FeatureEntry::Choice kTraceUploadURL[] = {
     {flags_ui::kGenericExperimentChoiceDisabled, "", ""},
     {flag_descriptions::kTraceUploadUrlChoiceOther, switches::kTraceUploadURL,
-     "https://performance-insights.appspot.com/upload?tags=flags,Other"},
+     "https://no-thanks.invalid"},
     {flag_descriptions::kTraceUploadUrlChoiceEmloading,
      switches::kTraceUploadURL,
-     "https://performance-insights.appspot.com/upload?tags=flags,emloading"},
+     "https://no-thanks.invalid"},
     {flag_descriptions::kTraceUploadUrlChoiceQa, switches::kTraceUploadURL,
-     "https://performance-insights.appspot.com/upload?tags=flags,QA"},
+     "https://no-thanks.invalid"},
     {flag_descriptions::kTraceUploadUrlChoiceTesting, switches::kTraceUploadURL,
-     "https://performance-insights.appspot.com/upload?tags=flags,TestingTeam"}};
+     "https://no-thanks.invalid"}};
 
 const FeatureEntry::Choice kLiteVideoDefaultDownlinkBandwidthKbps[] = {
     {flags_ui::kGenericExperimentChoiceDefault, "", ""},
@@ -7613,6 +7613,7 @@ const FeatureEntry kFeatureEntries[] = {
     // "LoginCustomFlags" in tools/metrics/histograms/enums.xml. See "Flag
     // Histograms" in tools/metrics/histograms/README.md (run the
     // AboutFlagsHistogramTest unit test to verify this process).
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 };
 
 class FlagsStateSingleton : public flags_ui::FlagsState::Delegate {
