diff --git a/chrome/browser/ui/views/frame/multi_contents_view.cc b/chrome/browser/ui/views/frame/multi_contents_view.cc
index cc9bfc0dd5ef36a671d3cb5b6fa10f79ade0b43b..94b7cc684d873070b9ee78c7baf66fe47d98a973 100644
--- a/chrome/browser/ui/views/frame/multi_contents_view.cc
+++ b/chrome/browser/ui/views/frame/multi_contents_view.cc
@@ -57,7 +57,7 @@ MultiContentsView::MultiContentsView(
 
   SetLayoutManager(std::make_unique<views::DelegatingLayoutManager>(this));
   contents_container_views_.push_back(
-      AddChildView(std::make_unique<ContentsContainerView>(browser_view_)));
+      AddChildView(std::make_unique<BraveContentsContainerView>(browser_view_)));
   contents_container_views_[0]
       ->GetContentsView()
       ->set_is_primary_web_contents_for_window(true);
@@ -68,7 +68,7 @@ MultiContentsView::MultiContentsView(
   resize_area_->SetVisible(false);
 
   contents_container_views_.push_back(
-      AddChildView(std::make_unique<ContentsContainerView>(browser_view_)));
+      AddChildView(std::make_unique<BraveContentsContainerView>(browser_view_)));
   contents_container_views_[1]->SetVisible(false);
   contents_container_views_[1]->GetInactiveSplitScrimView()->SetProperty(
       views::kElementIdentifierKey, kEndContainerViewScrimElementId);
