diff --git a/chrome/browser/ui/webui/theme_source.cc b/chrome/browser/ui/webui/theme_source.cc
index 927c4b9a276f2194d9bc1ddfd725eb544dfb4958..f799e72c483984a5635c8710ad6373845791fd2e 100644
--- a/chrome/browser/ui/webui/theme_source.cc
+++ b/chrome/browser/ui/webui/theme_source.cc
@@ -100,7 +100,7 @@ void ThemeSource::StartDataRequest(
   int resource_id = -1;
   if (parsed_path == "current-channel-logo") {
     switch (chrome::GetChannel()) {
-#if defined(GOOGLE_CHROME_BUILD)
+#if defined(GOOGLE_CHROME_BUILD) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))
       case version_info::Channel::CANARY:
         resource_id = IDR_PRODUCT_LOGO_32_CANARY;
         break;
@@ -122,7 +122,11 @@ void ThemeSource::StartDataRequest(
         FALLTHROUGH;
 #endif
       case version_info::Channel::UNKNOWN:
+#if defined(BRAVE_CHROMIUM_BUILD)
+        resource_id = IDR_PRODUCT_LOGO_32_DEVELOPMENT;
+#else
         resource_id = IDR_PRODUCT_LOGO_32;
+#endif
         break;
     }
   } else {
