diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 538d81f05e272b1b3f65d738c11a4e440c9da0b0..b3c364b2031c5b73a8a1f5afa440656bd2a99b4f 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -69,6 +69,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
