diff --git a/chrome/browser/enterprise/connectors/analysis/content_analysis_delegate.cc b/chrome/browser/enterprise/connectors/analysis/content_analysis_delegate.cc
index 43f09064b08d65214be4f2a4313b12c237489a93..77df1b61727daf01a0a30adf0bcd086777a11d39 100644
--- a/chrome/browser/enterprise/connectors/analysis/content_analysis_delegate.cc
+++ b/chrome/browser/enterprise/connectors/analysis/content_analysis_delegate.cc
@@ -404,7 +404,7 @@ void ContentAnalysisDelegate::CreateForWebContents(
         show_fail_closed_ui ? FinalContentAnalysisResult::FAIL_CLOSED
                             : FinalContentAnalysisResult::SUCCESS;

-#if BUILDFLAG(IS_WIN)
+#if BUILDFLAG(ENABLE_GLIC) && BUILDFLAG(IS_WIN)
     content::WebContents* top_web_contents =
         guest_view::GuestViewBase::GetTopLevelWebContents(
             web_contents->GetResponsibleWebContents());
