diff --git a/chrome/browser/resources/tools/optimize_webui.py b/chrome/browser/resources/tools/optimize_webui.py
index c8a7bd724fd63efdae3853d17d5d4cb29735933d..a479669e001242fa18e52d12c73188e96a09205b 100755
--- a/chrome/browser/resources/tools/optimize_webui.py
+++ b/chrome/browser/resources/tools/optimize_webui.py
@@ -44,6 +44,9 @@ def _request_list_path(out_path, target_name):
 
 
 def _get_dep_path(dep, host_url, in_path):
+  br_resources_path = os.path.join(_SRC_PATH, 'brave', 'ui', 'webui', 'resources','').replace('\\', '/')
+  for brave_host_url in ['chrome://brave-resources/', '//brave-resources/']:
+    if dep.startswith(brave_host_url): return dep.replace(brave_host_url, os.path.relpath(br_resources_path, _CWD))
   if dep.startswith(host_url):
     return dep.replace(host_url, os.path.relpath(in_path, _CWD))
   elif not (dep.startswith('chrome://') or dep.startswith('//')):
