diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 599a822375fc74bc5e97b37ef59fed2b18ba84b6..883f2079753435ca4ffeddd1a638c0446cc74893 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -720,6 +720,7 @@ class Browser : public TabStripModelObserver,
 
   StatusBubble* GetStatusBubbleForTesting();
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
