diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 5edda102ff59ccb30d2150a88e75e8fb65379fca..0e4efc0d13a9bcf33e09b195b54780193ae523ce 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -718,6 +718,7 @@ class Browser : public TabStripModelObserver,
 
   StatusBubble* GetStatusBubbleForTesting();
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
