diff --git a/chrome/browser/prefs/session_startup_pref.cc b/chrome/browser/prefs/session_startup_pref.cc
index 3e067a976678d9220abddfe8bc75eb8205a01693..93b04d6570e01e147329275749bb7006eb46fa50 100644
--- a/chrome/browser/prefs/session_startup_pref.cc
+++ b/chrome/browser/prefs/session_startup_pref.cc
@@ -60,6 +60,8 @@ void SessionStartupPref::RegisterProfilePrefs(
 SessionStartupPref::Type SessionStartupPref::GetDefaultStartupType() {
 #if defined(OS_CHROMEOS)
   return SessionStartupPref::LAST;
+#elif defined(BRAVE_CHROMIUM_BUILD)
+  return SessionStartupPref::LAST;
 #else
   return SessionStartupPref::DEFAULT;
 #endif
