diff --git a/chrome/browser/renderer_context_menu/render_view_context_menu.h b/chrome/browser/renderer_context_menu/render_view_context_menu.h
index 25f331b10b0132f5f59a3bfa78a7db4fc0182fdf..428e5adfd0f55e7ad2774e7df2a54dd635b06736 100644
--- a/chrome/browser/renderer_context_menu/render_view_context_menu.h
+++ b/chrome/browser/renderer_context_menu/render_view_context_menu.h
@@ -117,6 +117,7 @@ class RenderViewContextMenu : public RenderViewContextMenuBase {
   // Returns true if keyboard lock is active and requires the user to press and
   // hold escape to exit exclusive access mode.
   bool IsPressAndHoldEscRequiredToExitFullscreen() const;
+  BRAVE_RENDER_VIEW_CONTEXT_MENU_H_
 
  private:
   friend class RenderViewContextMenuTest;
