diff --git a/chrome/browser/ui/webui/cr_components/theme_color_picker/theme_color_picker_handler.cc b/chrome/browser/ui/webui/cr_components/theme_color_picker/theme_color_picker_handler.cc
index 98d6414a5b505f74113c8b212a00c5ca7edb0249..46b7852d2027d0763a8f00b2661a81904e8324a9 100644
--- a/chrome/browser/ui/webui/cr_components/theme_color_picker/theme_color_picker_handler.cc
+++ b/chrome/browser/ui/webui/cr_components/theme_color_picker/theme_color_picker_handler.cc
@@ -120,6 +120,7 @@ void ThemeColorPickerHandler::GetChromeColors(
     bool is_dark_mode,
     GetChromeColorsCallback callback) {
   std::vector<theme_color_picker::mojom::ChromeColorPtr> colors;
+  BRAVE_THEME_COLOR_PICKER_HANDLER_GET_CHROME_COLORS
   for (const auto& color_info : kDynamicCustomizeChromeColors) {
     colors.push_back(CreateDynamicChromeColor(color_info, is_dark_mode));
   }
