diff --git a/chrome/browser/ui/views/user_education/browser_user_education_service.cc b/chrome/browser/ui/views/user_education/browser_user_education_service.cc
index ae6e385e60e3cc8e156bf3d4d1d30c5aad6e81f2..19adfd539ebac5dfc529d3f5095642ed7dd37702 100644
--- a/chrome/browser/ui/views/user_education/browser_user_education_service.cc
+++ b/chrome/browser/ui/views/user_education/browser_user_education_service.cc
@@ -393,6 +393,7 @@ void MaybeRegisterChromeFeaturePromos(
           feature_engagement::kIPHWebUiHelpBubbleTestFeature)) {
     return;
   }
+  BRAVE_MAYBE_REGISTER_CHROME_FEATURE_PROMOS
 
   // kIPHAutofillCreditCardBenefitFeature:
   registry.RegisterFeature(std::move(
