diff --git a/chrome/browser/ui/views/frame/browser_view.h b/chrome/browser/ui/views/frame/browser_view.h
index 19ffaf7bb75ab0b25b973faabd5dda63712fd6cf..141accf53a0ffc7b63da04701f7d4272c1b97f9b 100644
--- a/chrome/browser/ui/views/frame/browser_view.h
+++ b/chrome/browser/ui/views/frame/browser_view.h
@@ -741,6 +741,7 @@ class BrowserView : public BrowserWindow,
       const std::map<std::string, std::string>& extra_data) override;
 #endif
 
+  BRAVE_BROWSER_VIEW_H
  private:
   // Do not friend BrowserViewLayout. Use the BrowserViewLayoutDelegate
   // interface to keep these two classes decoupled and testable.
