diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java
index bba285e86760495565a4ec622569c4761f610069..57db76ddcd7d8e3bb072e74a521ba9c9d9047e61 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java
@@ -73,7 +73,7 @@ public class AndroidSyncSettings {
         ThreadUtils.assertOnUiThread();
         if (sInstance == null) {
             SyncContentResolverDelegate contentResolver = new SystemSyncContentResolverDelegate();
-            sInstance = new AndroidSyncSettings(contentResolver);
+            sInstance = new BraveAndroidSyncSettings(contentResolver);
         }
         return sInstance;
     }
