diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 0b8467866c5a128f881c44429e61c96e405853d5..a600dbc4e8bd52a9219aeb4611e9b9dba80ef7a7 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2190,6 +2190,7 @@ static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (build_with_tflite_lib) {
@@ -4001,6 +4002,7 @@ static_library("browser") {
         "task_manager/sampling/shared_sampler_posix.cc",
       ]
     }
+    sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps
   }
 
   if (is_chromeos) {
