diff --git a/chrome/browser/ui/page_action/page_action_controller.h b/chrome/browser/ui/page_action/page_action_controller.h
index 9c81241fab39a13b1288dafeb6220b87dbeb7e4a..d64def1f662f8e6f5ad56054b8649b558373f63a 100644
--- a/chrome/browser/ui/page_action/page_action_controller.h
+++ b/chrome/browser/ui/page_action/page_action_controller.h
@@ -333,6 +333,7 @@ class PageActionController {
   virtual void DecrementActivityCounter(actions::ActionId action_id) = 0;
 };
 
+namespace chromium_impl {
 class PageActionControllerImpl : public PageActionController,
                                  public PinnedToolbarActionsModel::Observer {
  public:
@@ -413,6 +414,7 @@ class PageActionControllerImpl : public PageActionController,
   void OnActionsChanged() override;
 
  private:
+  friend class ::page_actions::PageActionControllerImpl;
   using PageActionModelsMap =
       std::map<actions::ActionId, std::unique_ptr<PageActionModelInterface>>;
 
@@ -443,7 +445,7 @@ class PageActionControllerImpl : public PageActionController,
   void ActionItemChanged(const actions::ActionItem* action_item);
   void PinnedActionsModelChanged();
 
-  std::unique_ptr<PageActionModelInterface> CreateModel(
+  virtual std::unique_ptr<PageActionModelInterface> CreateModel(
       actions::ActionId action_id,
       bool is_ephemeral);
 
@@ -503,6 +505,8 @@ class PageActionControllerImpl : public PageActionController,
 
   base::WeakPtrFactory<PageActionControllerImpl> weak_factory_{this};
 };
+}  // namespace chromium_impl
+
 
 // Returns page action IDs that are present in the browser window's root action
 // item.
