diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index 23c75e8b7d34ad078122efecaa0e461948d94567..ff04b3f0f23b6e5ed3734d7d50ff6b952d7938f6 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -7,7 +7,7 @@ import {loadTimeData} from 'chrome://resources/js/load_time_data.m.js';
 import {pageVisibility} from './page_visibility.js';
 import {Route, Router} from './router.js';
 import {SettingsRoutes} from './settings_routes.js';
-
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -215,6 +215,7 @@ function createBrowserSettingsRoutes() {
       // </if>
     }
   }
+  addBraveRoutes(r)
   return r;
 }
 
