diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index cf93dac2f899b9eabcea259261b3e5a4bcbf0c77..f2b9700266ec74f5f3127f79172217666d594d58 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -149,7 +149,7 @@ const char Profile::kProfileKey[] = "__PROFILE__";
 void Profile::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
       prefs::kSearchSuggestEnabled,
-      true,
+      false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 #if defined(OS_ANDROID)
   registry->RegisterStringPref(
