diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index e6286962272bbaf6cfd45cc4d3baebcad26eda1f..2d9dee9443d265fed72ceaa58a113ca5da6c6a4e 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -251,7 +251,7 @@ grit("google_chrome_strings") {
 }
 
 grit("chromium_strings") {
-  source = "chromium_strings.grd"
+  source = "${branding_path_component}_strings.grd"
   defines = chrome_grit_defines
   output_dir = "$root_gen_dir/chrome"
   outputs = [
