diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index 9e0d25bc3367fa5ed2a91be4b5890df122993efe..a5392897a2ede72696e85aeb0f7d4fd26e64b71d 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -149,7 +149,7 @@ const char Profile::kProfileKey[] = "__PROFILE__";
 void Profile::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
       prefs::kSearchSuggestEnabled,
-      true,
+      false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 #if defined(OS_ANDROID)
   registry->RegisterStringPref(
