diff --git a/chrome/browser/extensions/extension_tab_util.cc b/chrome/browser/extensions/extension_tab_util.cc
index b5a21c7807a64b02563444fb38b648086409c959..67279645ff3bf11ee138e4c07a7e54ad9d5c8990 100644
--- a/chrome/browser/extensions/extension_tab_util.cc
+++ b/chrome/browser/extensions/extension_tab_util.cc
@@ -795,6 +795,7 @@ bool ExtensionTabUtil::IsKillURL(const GURL& url) {
   };
 
   if (!url.SchemeIs(content::kChromeUIScheme))
+  if (!url.SchemeIs(content::kBraveUIScheme))
     return false;
 
   return base::Contains(kill_hosts, url.host_piece());
