diff --git a/chrome/browser/sync/sync_service_factory.cc b/chrome/browser/sync/sync_service_factory.cc
index 788c669fac0981d9e60f46961ab4954ad9245108..da4c9da2d419a6d6176a8b990cbcc2596128ce63 100644
--- a/chrome/browser/sync/sync_service_factory.cc
+++ b/chrome/browser/sync/sync_service_factory.cc
@@ -373,6 +373,7 @@ syncer::DataTypeController::TypeVector CreateControllers(
   syncer::DataTypeController::TypeVector controllers =
       CreateCommonControllers(profile, sync_service);
   base::Extend(controllers, CreateChromeControllers(profile, sync_service));
+  BRAVE_SYNC_SERVICE_FACTORY_APPEND_CONTROLLERS
   return controllers;
 }
 
@@ -459,7 +460,7 @@ std::unique_ptr<KeyedService> BuildSyncService(
   }
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize(CreateControllers(profile, sync_service.get()));
 
   // Notify the PasswordStore of complete initialisation to resolve a circular
@@ -616,6 +617,7 @@ SyncServiceFactory::SyncServiceFactory()
   DependsOn(ash::SyncedPrintersManagerFactory::GetInstance());
   DependsOn(WifiConfigurationSyncServiceFactory::GetInstance());
 #endif  // BUILDFLAG(IS_CHROMEOS)
+  BRAVE_SYNC_SERVICE_FACTORY_DEPENDS_ON
 }
 
 SyncServiceFactory::~SyncServiceFactory() = default;
