diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index b8bec16f630e5bcf46b5897093016b1e7ae439ab..9c18ca88998a0e13fc9538402de9d3cef606e4a8 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -511,6 +511,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
                                             ? "webui-refresh-2026"
                                             : "");
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
