diff --git a/chrome/browser/resources/settings/privacy_page/cr_lottie.ts b/chrome/browser/resources/settings/privacy_page/cr_lottie.ts
index df65d69c2f134aa3954de2bdd7b355e37744fa1f..fce0231f8b1ae96114eff63dbe0c337c07f0be67 100644
--- a/chrome/browser/resources/settings/privacy_page/cr_lottie.ts
+++ b/chrome/browser/resources/settings/privacy_page/cr_lottie.ts
@@ -89,10 +89,10 @@ export class CrLottieElement extends CrLitElement {
     };
   }
 
-  animationUrl: string = '';
-  autoplay: boolean = false;
-  override hidden: boolean = false;
-  singleLoop: boolean = false;
+  accessor animationUrl: string = '';
+  accessor autoplay: boolean = false;
+  override accessor hidden: boolean = false;
+  accessor singleLoop: boolean = false;
 
   private canvasElement_: CanvasElementWithOffscreen|null = null;
   private isAnimationLoaded_: boolean = false;
@@ -344,6 +344,7 @@ export class CrLottieElement extends CrLitElement {
       this.fire('cr-lottie-paused');
     } else if (event.data.name === 'stopped') {
       this.fire('cr-lottie-stopped');
+    } else if (event.data.name === 'complete') { this.fire('cr-lottie-complete');
     } else if (event.data.name === 'resized') {
       this.fire('cr-lottie-resized', event.data.size);
     }
