diff --git a/chrome/installer/mac/signing/config.py b/chrome/installer/mac/signing/config.py
index e2328845a3200ae86ccabe4ff9315aa6c078d7a3..72014972f933afa72818c8d5beea7da18845351f 100644
--- a/chrome/installer/mac/signing/config.py
+++ b/chrome/installer/mac/signing/config.py
@@ -266,4 +266,4 @@ class CodeSignConfig(object):
     @property
     def packaging_dir(self):
         """Returns the path to the packaging and installer tools."""
-        return '{.product} Packaging'.format(self)
+        return '{}_Packaging'.format(self.product.replace(' ', '_'))
