diff --git a/chrome/browser/web_applications/web_app_shortcut_mac.mm b/chrome/browser/web_applications/web_app_shortcut_mac.mm
index 7fe42b666718549f8f68e0b69c2fe366f5833a8f..ef24099fc4e9257cccd36f944adcf6b1baff5206 100644
--- a/chrome/browser/web_applications/web_app_shortcut_mac.mm
+++ b/chrome/browser/web_applications/web_app_shortcut_mac.mm
@@ -761,6 +761,7 @@ base::FilePath GetChromeAppsFolder() {
   if (path.empty())
     return path;
 
+  return path.Append(GetLocalizableBraveAppShortcutsSubdirName());
   return path.Append(GetLocalizableAppShortcutsSubdirName());
 }
 
