diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 77910b96039f26300d99c1f8979e655e9f5517a8..563c41d95f208886079381a619e37233f9fd70d4 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -296,7 +296,7 @@ void ToolbarView::Init() {
   } else {
     // TODO(crbug.com/932818): Remove this once the
     // |kAutofillEnableToolbarStatusChip| is fully launched.
-    avatar_ = AddChildView(std::make_unique<AvatarToolbarButton>(browser_));
+    avatar_ = AddChildView(std::make_unique<BraveAvatarToolbarButton>(browser_));
     avatar_->SetVisible(show_avatar_toolbar_button);
   }
 
