diff --git a/chrome/browser/resources/settings/privacy_page/personalization_options.js b/chrome/browser/resources/settings/privacy_page/personalization_options.js
index e8f221291df750046915055b427b68dc72b468e6..1d695673499ad788ddbbaf359646329761175d7f 100644
--- a/chrome/browser/resources/settings/privacy_page/personalization_options.js
+++ b/chrome/browser/resources/settings/privacy_page/personalization_options.js
@@ -34,7 +34,7 @@ Polymer({
     /** @type {settings.SyncStatus} */
     syncStatus: Object,
 
-    // <if expr="_google_chrome and not chromeos">
+    // <if expr="_chromium and not chromeos">
     // TODO(dbeam): make a virtual.* pref namespace and set/get this normally
     // (but handled differently in C++).
     /** @private {chrome.settingsPrivate.PrefObject} */
@@ -80,7 +80,7 @@ Polymer({
   ready() {
     this.browserProxy_ = settings.PrivacyPageBrowserProxyImpl.getInstance();
 
-    // <if expr="_google_chrome and not chromeos">
+    // <if expr="_chromium and not chromeos">
     const setMetricsReportingPref = this.setMetricsReportingPref_.bind(this);
     this.addWebUIListener('metrics-reporting-change', setMetricsReportingPref);
     this.browserProxy_.getMetricsReporting().then(setMetricsReportingPref);
@@ -111,7 +111,7 @@ Polymer({
     return /** @type {?CrToggleElement} */ (this.$$('#driveSuggestControl'));
   },
 
-  // <if expr="_google_chrome and not chromeos">
+  // <if expr="_chromium and not chromeos">
   /** @private */
   onMetricsReportingChange_() {
     const enabled = this.$.metricsReportingControl.checked;
@@ -161,6 +161,7 @@ Polymer({
   },
   // </if>
 
+  // <if expr="_google_chrome">
   /**
    * @return {boolean}
    * @private
@@ -209,6 +210,7 @@ Polymer({
     }
     this.showSignoutDialog_ = false;
   },
+  // </if>
 
   /**
    * @param {!Event} e
