diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 68285609dc85f879a90f110fd3f2c1b6f0880589..dcd1f01510b505ea4e085e789899847e5de0097a 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -72,6 +72,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
