diff --git a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationServiceImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationServiceImpl.java
index ecbbe843ea298e784d9dc06b8bcbee7c0e0f0b63..6453659ca927d0abe6e00770a4728e74f2c38556 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationServiceImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationServiceImpl.java
@@ -105,7 +105,7 @@ public class NotificationServiceImpl extends NotificationService.Impl {
 
         // Now that the browser process is initialized, we pass forward the call to the
         // NotificationPlatformBridge which will take care of delivering the appropriate events.
-        if (!NotificationPlatformBridge.dispatchNotificationEvent(intent)) {
+        if (!BraveNotificationPlatformBridge.dispatchNotificationEvent(intent)) {
             Log.w(TAG, "Unable to dispatch the notification event to Chrome.");
         }
 
