diff --git a/chrome/browser/ui/android/appmenu/internal/BUILD.gn b/chrome/browser/ui/android/appmenu/internal/BUILD.gn
index f741b3c98e2668921de0808ac3a0bc9689f79e58..d224bad084ab5aa817663eb38b237585c0163028 100644
--- a/chrome/browser/ui/android/appmenu/internal/BUILD.gn
+++ b/chrome/browser/ui/android/appmenu/internal/BUILD.gn
@@ -21,6 +21,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/ui/appmenu/AppMenuItemIcon.java",
     "java/src/org/chromium/chrome/browser/ui/appmenu/AppMenuItemViewBinder.java",
   ]
+  sources += brave_browser_ui_android_appmenu_internal_sources
 
   deps = [
     ":java_resources",
