diff --git a/chrome/browser/extensions/component_extensions_allowlist/allowlist.cc b/chrome/browser/extensions/component_extensions_allowlist/allowlist.cc
index a30faeb1eaa4316fcff2f48ce3630a80be2103c9..c54b6373891c656a497a6ad8c1105cac8980a4d7 100644
--- a/chrome/browser/extensions/component_extensions_allowlist/allowlist.cc
+++ b/chrome/browser/extensions/component_extensions_allowlist/allowlist.cc
@@ -35,6 +35,12 @@ static_assert(BUILDFLAG(ENABLE_EXTENSIONS_CORE));
 namespace extensions {
 
 bool IsComponentExtensionAllowlisted(const std::string& extension_id) {
+  if (IsComponentExtensionAllowlistedBraveImpl(extension_id)) {
+    return true;
+  }
+  if (IsComponentExtensionDenylistedBraveImpl(extension_id)) {
+    return false;
+  }
   constexpr auto kAllowed = base::MakeFixedFlatSet<std::string_view>({
       extension_misc::kContextualTasksExtensionId,
       extension_misc::kGlicExtensionId,
@@ -89,6 +95,9 @@ bool IsComponentExtensionAllowlisted(const std::string& extension_id) {
 }
 
 bool IsComponentExtensionAllowlisted(int manifest_resource_id) {
+  if (IsComponentExtensionAllowlistedBraveImpl(manifest_resource_id)) {
+    return true;
+  }
   switch (manifest_resource_id) {
     // Please keep the list in alphabetical order.
 #if BUILDFLAG(ENABLE_HANGOUT_SERVICES_EXTENSION)
