diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/MainSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/MainSettings.java
index 2fdc6b850c6addc362d04c0212a3f4e5b7e4bb73..bb1ee547df0f392e8c4c13dbda5e726befb9f18c 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/MainSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/MainSettings.java
@@ -826,7 +826,9 @@ public class MainSettings extends ChromeBaseSettingsFragment
         }
     }
 
+    @SuppressWarnings("UnnecessaryParentheses")
     private static boolean shouldShowAddressBarPref(Context context) {
+        if ((true)) return false;
         // Similar to ToolbarPositionController#isToolbarPositionCustomizationEnabled(), except
         // - no CCT checks (settings are not accessible from CCTs),
         // - showing on Foldables in unfolded (open) state.
