diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 3837f7ae674e5820e31cf8e3ab1ccac9e772f013..c763dae048d8dad39b3eb9a2690c6e6947214c0d 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -36,6 +36,7 @@ class AvatarToolbarButton : public ToolbarButton,
   void UpdateText();
 
  private:
+  friend class BraveAvatarToolbarButton;
   enum class SyncState { kNormal, kPaused, kError };
 
   // ToolbarButton:
@@ -78,6 +79,7 @@ class AvatarToolbarButton : public ToolbarButton,
   bool IsIncognitoCounterActive() const;
   bool ShouldShowGenericIcon() const;
   base::string16 GetAvatarTooltipText() const;
+  virtual
   gfx::ImageSkia GetAvatarIcon() const;
   gfx::Image GetIconImageFromProfile() const;
   SyncState GetSyncState() const;
