diff --git a/chrome/browser/permissions/permission_manager.cc b/chrome/browser/permissions/permission_manager.cc
index 7c41959fe6acb8aded0db703e0327c2251c39fe4..f6e6c9e9482a56ac2e3b12ce2609ca2e217e407e 100644
--- a/chrome/browser/permissions/permission_manager.cc
+++ b/chrome/browser/permissions/permission_manager.cc
@@ -87,7 +87,7 @@ PermissionStatus ContentSettingToPermissionStatus(ContentSetting setting) {
 }
 
 // Helper method to convert PermissionType to ContentSettingType.
-ContentSettingsType PermissionTypeToContentSetting(PermissionType permission) {
+ContentSettingsType PermissionTypeToContentSetting_ChromiumImpl(PermissionType permission) {
   switch (permission) {
     case PermissionType::MIDI:
       return CONTENT_SETTINGS_TYPE_MIDI;
@@ -129,6 +129,7 @@ ContentSettingsType PermissionTypeToContentSetting(PermissionType permission) {
     case PermissionType::IDLE_DETECTION:
       return CONTENT_SETTINGS_TYPE_IDLE_DETECTION;
     case PermissionType::NUM:
+    default:
       // This will hit the NOTREACHED below.
       break;
   }
