diff --git a/chrome/browser/ui/content_settings/content_setting_image_model.cc b/chrome/browser/ui/content_settings/content_setting_image_model.cc
index a22eb27ddccd8145dc44572d3b8f27a9400b5514..d61ee4ca61fb85ef39e18ab355a88fe21f393ac2 100644
--- a/chrome/browser/ui/content_settings/content_setting_image_model.cc
+++ b/chrome/browser/ui/content_settings/content_setting_image_model.cc
@@ -11,6 +11,7 @@
 #include "base/macros.h"
 #include "base/metrics/field_trial_params.h"
 #include "base/metrics/histogram_macros.h"
+#include "brave/browser/ui/content_settings/brave_content_setting_image_models.h"
 #include "build/build_config.h"
 #include "chrome/app/vector_icons/vector_icons.h"
 #include "chrome/browser/browser_process.h"
@@ -982,6 +983,9 @@ ContentSettingImageModel::GenerateContentSettingImageModels() {
   std::vector<std::unique_ptr<ContentSettingImageModel>> result;
   for (auto type : kContentSettingImageOrder)
     result.push_back(CreateForContentType(type));
+#if defined(BRAVE_CHROMIUM_BUILD)
+  BraveGenerateContentSettingImageModels(result);
+#endif
 
   return result;
 }
