diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 753cde04f0a35b51a9f637381f5098158fae0ea3..dc8834ed692188b0388746a22919c61816a2543d 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1479,6 +1479,7 @@ source_set("extensions") {
       ]
     }
   }  # if (enable_extensions)
+  deps += brave_extensions_resources
 }
 
 if (enable_extensions) {
