diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index 5cda67983dbededf84c50228bc8885b4ebe448a1..21aab23a5021676534287b702de88685f7cf3379 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -997,6 +997,7 @@ void BookmarkBarView::Layout() {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   // Only render the tab group separtor if there are groups and bookmarks
   // visible. Must done after the bookmark buttons are rendered.
