diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java
index f86f528b027c77f4662749f0d0ca2d449d05cc8c..6f94670e5e7108c8eb755c75836f6f9dc3bd194c 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java
@@ -200,7 +200,7 @@ public class ManageSyncSettings extends ChromeBaseSettingsFragment
     private boolean mShouldUpdatePrefs;
 
     /** Maps {@link UserSelectableType} to the corresponding preference. */
-    private Map<Integer, ChromeSwitchPreference> mSyncTypeSwitchPreferencesMap = new HashMap<>();
+    protected Map<Integer, ChromeSwitchPreference> mSyncTypeSwitchPreferencesMap = new HashMap<>();
 
     private Preference mGoogleActivityControls;
     private Preference mSyncEncryption;
@@ -289,6 +289,7 @@ public class ManageSyncSettings extends ChromeBaseSettingsFragment
 
     @Override
     public boolean onPreferenceChange(Preference preference, Object newValue) {
+        if (false)  // Suppress the "addresses not encrypted" warning (not applicable to Brave's sync model).
         if (mSyncService.isUsingExplicitPassphrase()
                 && preference
                         .getKey()
@@ -362,6 +363,9 @@ public class ManageSyncSettings extends ChromeBaseSettingsFragment
         CentralAccountCardPreference centralAccountCardPreference =
                 findPreference(PREF_CENTRAL_ACCOUNT_CARD_PREFERENCE);
         IdentityManager identityManager = getIdentityManager(profile);
+        if (identityManager.getPrimaryAccountInfo() == null) {
+            return;
+        }
         centralAccountCardPreference.initialize(
                 assumeNonNull(identityManager.getPrimaryAccountInfo()),
                 ProfileDataCache.createWithDefaultImageSizeAndNoBadge(
@@ -542,7 +546,7 @@ public class ManageSyncSettings extends ChromeBaseSettingsFragment
      * this state.
      */
     private void updateSyncPreferences() {
-        @Nullable AccountInfo primaryAccount = getIdentityManager().getPrimaryAccountInfo();
+        @Nullable AccountInfo primaryAccount = new AccountInfo.Builder("", new org.chromium.google_apis.gaia.GaiaId("")).build();
         // May happen if account is removed from the device while this screen is shown.
         if (primaryAccount == null) {
             finishCurrentSettings();
