diff --git a/chrome/browser/extensions/extension_management.h b/chrome/browser/extensions/extension_management.h
index 7894fd13acf50e6de2a6ff376e4cf5bdb9eb9eb4..eee0f936265f2f31ecc27bad46fe037f41574a04 100644
--- a/chrome/browser/extensions/extension_management.h
+++ b/chrome/browser/extensions/extension_management.h
@@ -165,6 +165,7 @@ class ExtensionManagement : public KeyedService {
                            std::string* required_version) const;
 
  private:
+  friend class BraveExtensionManagement;
   using SettingsIdMap =
       std::unordered_map<ExtensionId,
                          std::unique_ptr<internal::IndividualSettings>>;
