diff --git a/chrome/installer/setup/setup_main.cc b/chrome/installer/setup/setup_main.cc
index 276339514cdeb3c466f903d4e80302159e2b8e66..e210bba66cedb04127c5323d738f6567de45a40a 100644
--- a/chrome/installer/setup/setup_main.cc
+++ b/chrome/installer/setup/setup_main.cc
@@ -781,6 +781,7 @@ installer::InstallStatus InstallProducts(InstallationState& original_state,
     }
   }
 
+  BRAVE_INSTALL_PRODUCTS
   return install_status;
 }
 
@@ -1621,6 +1622,7 @@ int SetupMain() {
                                      &exit_code)) {
     return exit_code;
   }
+  BRAVE_SETUP_MAIN
 
   if (system_install && !IsUserAnAdmin()) {
     if (!cmd_line.HasSwitch(installer::switches::kRunAsAdmin)) {
