diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index ff8399e6c417dad97952ed1290483eb9ca070afa..4a68abdf74de7b901219f4b6c917355b438f3170 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -638,6 +638,7 @@ class Browser : public TabStripModelObserver,
   // dragging.
   void SetIsInTabDragging(bool is_in_tab_dragging);
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class FullscreenControllerInteractiveTest;
