diff --git a/chrome/browser/shell_integration.h b/chrome/browser/shell_integration.h
index 083cc0c0c232897f8c5414b935614edf9c71896b..0cd51f40618154a07f634e4dae44e4a5dd6d41d4 100644
--- a/chrome/browser/shell_integration.h
+++ b/chrome/browser/shell_integration.h
@@ -280,6 +280,7 @@ class DefaultBrowserWorker : public DefaultWebClientWorker {
 
   static void DisableSetAsDefaultForTesting();
 
+  BRAVE_DEFAULT_BROWSER_WORKER_H
  protected:
   ~DefaultBrowserWorker() override;
 
