diff --git a/chrome/browser/ui/tabs/tab_strip_prefs.cc b/chrome/browser/ui/tabs/tab_strip_prefs.cc
index 09bd26bf7c26a84ad9b1e766dfd15fdbf2b4285c..f29bb60426378b420b07e9d6f95f85274bdaa8fa 100644
--- a/chrome/browser/ui/tabs/tab_strip_prefs.cc
+++ b/chrome/browser/ui/tabs/tab_strip_prefs.cc
@@ -20,6 +20,7 @@ std::optional<bool> g_tab_search_trailing_tabstrip_at_startup = std::nullopt;
 namespace tabs {
 
 bool GetDefaultTabSearchRightAligned() {
+  BRAVE_GET_DEFAULT_TAB_SEARCH_RIGHT_ALIGNED
   // These platforms are all left aligned, the others should be right.
 #if BUILDFLAG(IS_CHROMEOS) || BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_WIN)
   return false;
