diff --git a/chrome/browser/ui/omnibox/chrome_omnibox_client.h b/chrome/browser/ui/omnibox/chrome_omnibox_client.h
index d3431a361af257b525dc2d83d1ddf0a86f97c8dd..0b5b412f8a7ca07b0acd92fb86f761fac08cbdce 100644
--- a/chrome/browser/ui/omnibox/chrome_omnibox_client.h
+++ b/chrome/browser/ui/omnibox/chrome_omnibox_client.h
@@ -23,7 +23,7 @@ class GURL;
 class LocationBar;
 class Profile;
 
-class ChromeOmniboxClient final : public OmniboxClient {
+class ChromeOmniboxClient : public OmniboxClient {
  public:
   ChromeOmniboxClient(LocationBar* location_bar,
                       Browser* browser,
