diff --git a/chrome/browser/ui/android/bottombar/BUILD.gn b/chrome/browser/ui/android/bottombar/BUILD.gn
index 5a1e6acc47dfac402a8a45b29b3848ad77786863..7e31d90a145f90b7d12505ae2e4457e85b2772bf 100644
--- a/chrome/browser/ui/android/bottombar/BUILD.gn
+++ b/chrome/browser/ui/android/bottombar/BUILD.gn
@@ -31,6 +31,7 @@ android_library("config_java") {
   sources = [ "java/src/org/chromium/chrome/browser/ui/bottombar/BottomBarConfigUtils.java" ]
   deps = [
     "//base:build_info_java",
+    "//brave/browser/ui/android/bottombar:prefs_java",
     "//build/android:build_java",
     "//chrome/browser/flags:java",
     "//chrome/browser/preferences:java",
@@ -56,6 +57,7 @@ android_library("metrics_java") {
 
 android_library("java") {
   sources = [
+    "//brave/browser/ui/android/bottombar/java/src/org/chromium/chrome/browser/ui/bottombar/BraveBottomBarButtonManager.java",
     "java/src/org/chromium/chrome/browser/ui/bottombar/AppMenuActionButtonBinder.java",
     "java/src/org/chromium/chrome/browser/ui/bottombar/BottomBar.java",
     "java/src/org/chromium/chrome/browser/ui/bottombar/BottomBarActionEligibility.java",
