diff --git a/chrome/chrome_proxy/chrome_proxy_main_win.cc b/chrome/chrome_proxy/chrome_proxy_main_win.cc
index abc75a5a435ed9afcab38626285a082e45cf6947..8dd08b70afa2471ddc0a92b42cc4fa7cef65a0a9 100644
--- a/chrome/chrome_proxy/chrome_proxy_main_win.cc
+++ b/chrome/chrome_proxy/chrome_proxy_main_win.cc
@@ -13,7 +13,10 @@
 namespace {
 
 constexpr base::FilePath::CharType kChromeExecutable[] =
+    FILE_PATH_LITERAL("brave.exe");
+#if 0
     FILE_PATH_LITERAL("chrome.exe");
+#endif
 
 constexpr base::FilePath::CharType kChromeProxyExecutable[] =
     FILE_PATH_LITERAL("chrome_proxy.exe");
