diff --git a/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc b/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
index 9a5fc00d8b8967128e4520348f14437263ea70dd..0f04c145e05d3cc28e8a7a76e3f628eb67f13e4d 100644
--- a/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
+++ b/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
@@ -90,6 +90,7 @@ content::RenderProcessHost* WebrtcLoggingPrivateFunction::RphFromRequest(
     const api::webrtc_logging_private::RequestInfo& request,
     const std::string& security_origin,
     std::string* error) {
+  BRAVE_DISABLE_RPH_FROM_REQUEST //feature disabled in Brave
   // There are 2 ways these API functions can get called.
   //
   //  1. From an allowlisted component extension on behalf of a page with the
