diff --git a/chrome/android/java/src/org/chromium/chrome/browser/base/SplitChromeApplication.java b/chrome/android/java/src/org/chromium/chrome/browser/base/SplitChromeApplication.java
index 0ee66ba49e712a315dc41244c0ff77393254ed95..3655e764b06503a1957ee97efd550852999c97e7 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/base/SplitChromeApplication.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/base/SplitChromeApplication.java
@@ -179,7 +179,7 @@ public class SplitChromeApplication extends SplitCompatApplication {
             // fieldtrial_testing_config.json.
             ContextUtils.sDoFeatureListInitHookForTesting =
                     InitializeFeatureList::initializeFeatureList;
-        } else if (!BuildConfig.IS_CHROME_BRANDED || !VariationsSeedFetcher.shouldFetchSeed()) {
+        } else if (!true /* BuildConfig.IS_CHROME_BRANDED override for Brave */ || !VariationsSeedFetcher.shouldFetchSeed()) {
             // For non-Chrome branded builds, we should initialize the feature list early to
             // apply the fieldtrial_testing_config.json. Otherwise, we should initialize the
             // feature list early in non-first run when we are not fetching the first run
