diff --git a/chrome/browser/ui/webui/theme_source.cc b/chrome/browser/ui/webui/theme_source.cc
index 2ace3bf2c47c0f4af3c00f15d2540a68bfb59877..0a2d47b0dc464f3d9bff7b511a54d42619222131 100644
--- a/chrome/browser/ui/webui/theme_source.cc
+++ b/chrome/browser/ui/webui/theme_source.cc
@@ -105,7 +105,7 @@ void ThemeSource::StartDataRequest(
   int resource_id = -1;
   if (parsed_path == "current-channel-logo") {
     switch (chrome::GetChannel()) {
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))
       case version_info::Channel::CANARY:
         resource_id = IDR_PRODUCT_LOGO_32_CANARY;
         break;
@@ -127,7 +127,11 @@ void ThemeSource::StartDataRequest(
         FALLTHROUGH;
 #endif
       case version_info::Channel::UNKNOWN:
+#if defined(BRAVE_CHROMIUM_BUILD)
+        resource_id = IDR_PRODUCT_LOGO_32_DEVELOPMENT;
+#else
         resource_id = IDR_PRODUCT_LOGO_32;
+#endif
         break;
     }
   } else {
