diff --git a/chrome/browser/ui/views/bookmarks/bookmark_context_menu.cc b/chrome/browser/ui/views/bookmarks/bookmark_context_menu.cc
index 1811e514bad87daaaa2f8ca25d835b0f099878a8..8729718e5f968b018571f37be555ab030d8194ba 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_context_menu.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_context_menu.cc
@@ -72,6 +72,7 @@ BookmarkContextMenu::BookmarkContextMenu(
   for (size_t i = 0; i < menu_model->GetItemCount(); ++i) {
     views::MenuModelAdapter::AppendMenuItemFromModel(
         menu_model, i, menu_, menu_model->GetCommandIdAt(i));
+    BRAVE_BOOKMARK_CONTEXT_MENU
     if (menu_model->GetCommandIdAt(i) == IDC_BOOKMARK_BAR_SUBMENU) {
       submodel = menu_model->GetSubmenuModelAt(i);
     }
