diff --git a/chrome/browser/resources/settings/people_page/sync_controls.ts b/chrome/browser/resources/settings/people_page/sync_controls.ts
index 33fcf88bdda5d729097da5ffe9bce4bb2c61270c..e10ce25a380c8b16ec6993f9a930766757744e15 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.ts
+++ b/chrome/browser/resources/settings/people_page/sync_controls.ts
@@ -142,7 +142,7 @@ export class SettingsSyncControlsElement extends
         'sync-prefs-changed', this.handleSyncPrefsChanged_.bind(this));
 
     const router = Router.getInstance();
-    if (router.getCurrentRoute() === routes.SYNC_ADVANCED) {
+    if (router.getCurrentRoute() === routes.BRAVE_SYNC_SETUP) {
       this.browserProxy_.didNavigateToSyncPage();
     }
   }
