diff --git a/chrome/browser/ui/views/profiles/profile_menu_view_base.cc b/chrome/browser/ui/views/profiles/profile_menu_view_base.cc
index b1f2387812161f78334d083718f5f8c71aca0e51..1b0ec4a2eede34447b4fd8412908ed10730b046b 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view_base.cc
+++ b/chrome/browser/ui/views/profiles/profile_menu_view_base.cc
@@ -603,6 +603,7 @@ void ProfileMenuViewBase::BuildProfileBackgroundContainer(
     identity_info_color_callback_ = base::BindRepeating(
         &ProfileMenuViewBase::BuildIdentityInfoColorCallback,
         base::Unretained(this));
+    BRAVE_PROFILE_MENU_VIEW_BASE_SET_BACKGROUND
   } else {
     DCHECK_EQ(SK_ColorTRANSPARENT, background_color);
     profile_background_container_->SetBackground(
