diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
index 10888e599ca499430d26ac33a88d13526531b6a5..5ebf96d7d97041fa1cbc3b4fd9b7bb6773253fc3 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
@@ -1262,6 +1262,7 @@ public class ToolbarManager implements UrlFocusChangeListener, ThemeColorObserve
         boolean editingAllowed =
                 currentTab == null || bridge == null || bridge.isEditBookmarksEnabled();
         mToolbar.updateBookmarkButton(isBookmarked, editingAllowed);
+        if (mBottomControlsCoordinator != null) {mBottomControlsCoordinator.updateBookmarkButton(isBookmarked, editingAllowed);}
     }
 
     private void updateReloadState(boolean tabCrashed) {
