diff --git a/chrome/browser/translate/translate_manager_browsertest.cc b/chrome/browser/translate/translate_manager_browsertest.cc
index 34d721c7e8cd09a7bb1c7ac065b58ffc1bb68cbc..45f754276c7e4359d582203e201409dcabcaf395 100644
--- a/chrome/browser/translate/translate_manager_browsertest.cc
+++ b/chrome/browser/translate/translate_manager_browsertest.cc
@@ -405,7 +405,7 @@ class TranslateManagerBrowserTest : public InProcessBrowserTest {
 
   virtual void InitFeatures() {
     scoped_feature_list_.InitWithFeatures(
-        {toast_features::kTranslateToast, features::kReadAnythingImprovedUi},
+        {toast_features::kTranslateToast, ::features::kReadAnythingImprovedUi},
         {});
   }
 
