diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 2203610c864eef6042936a83eb2b9797be93a0e0..a135c3be627048482127f584b7ab95d504eeb2ff 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2642,6 +2642,7 @@ static_library("browser") {
     "//chrome/browser/sharing/glic_experimental_triggering:impl",
     "//chrome/browser/ui/webauthn:impl",
   ]
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (enable_webui_ntp) {
     deps += [
@@ -5354,6 +5355,7 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
