diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 43c602c25388035e8c9fdf4bdff1f3b77e365294..fedada5a36c3c58bcf9d308ff23aab8dfd070efb 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -5190,6 +5190,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
