diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index d1519e7f3782ae6e1427c3cce1293fc6b6c5f6b0..1d972ea2e1cde88a98d33b92b473c37e9ce20abc 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -464,6 +464,7 @@ void ToolbarView::Init() {
           browser_view_, browser_->profile()));
 #endif
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = container_view_->AddChildView(
       std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
