diff --git a/chrome/browser/importer/importer_list.cc b/chrome/browser/importer/importer_list.cc
index 3bf47fa746e2a417f89201f3e3800d8639f2e323..4152d44b23d1b5b1e909e351da31f2a56193d728 100644
--- a/chrome/browser/importer/importer_list.cc
+++ b/chrome/browser/importer/importer_list.cc
@@ -7,9 +7,13 @@
 #include <stdint.h>
 
 #include "base/bind.h"
+#include "base/strings/utf_string_conversions.h"
 #include "base/task_scheduler/post_task.h"
 #include "base/task_scheduler/task_traits.h"
 #include "base/threading/thread_restrictions.h"
+#include "base/values.h"
+#include "brave/common/importer/brave_importer_utils.h"
+#include "brave/common/importer/chrome_importer_utils.h"
 #include "build/build_config.h"
 #include "chrome/browser/shell_integration.h"
 #include "chrome/common/importer/firefox_importer_utils.h"
@@ -29,6 +33,9 @@
 #include "chrome/common/importer/edge_importer_utils_win.h"
 #endif
 
+void DetectChromeProfiles(std::vector<importer::SourceProfile>* profiles);
+void DetectBraveProfiles(std::vector<importer::SourceProfile>* profiles);
+
 namespace {
 
 #if defined(OS_WIN)
@@ -119,7 +126,7 @@ void DetectFirefoxProfiles(const std::string locale,
     firefox.app_path = app_path;
   firefox.services_supported = importer::HISTORY | importer::FAVORITES |
                                importer::PASSWORDS | importer::SEARCH_ENGINES |
-                               importer::AUTOFILL_FORM_DATA;
+                               importer::AUTOFILL_FORM_DATA | importer::COOKIES;
   firefox.locale = locale;
   profiles->push_back(firefox);
 }
@@ -136,21 +143,37 @@ std::vector<importer::SourceProfile> DetectSourceProfilesWorker(
 #if defined(OS_WIN)
   if (shell_integration::IsFirefoxDefaultBrowser()) {
     DetectFirefoxProfiles(locale, &profiles);
+    DetectBraveProfiles(&profiles);
     DetectBuiltinWindowsProfiles(&profiles);
+    DetectChromeProfiles(&profiles);
   } else {
     DetectBuiltinWindowsProfiles(&profiles);
+    DetectBraveProfiles(&profiles);
     DetectFirefoxProfiles(locale, &profiles);
+    DetectChromeProfiles(&profiles);
   }
 #elif defined(OS_MACOSX)
   if (shell_integration::IsFirefoxDefaultBrowser()) {
     DetectFirefoxProfiles(locale, &profiles);
+    DetectBraveProfiles(&profiles);
     DetectSafariProfiles(&profiles);
+    DetectChromeProfiles(&profiles);
   } else {
     DetectSafariProfiles(&profiles);
+    DetectBraveProfiles(&profiles);
     DetectFirefoxProfiles(locale, &profiles);
+    DetectChromeProfiles(&profiles);
   }
 #else
-  DetectFirefoxProfiles(locale, &profiles);
+  if (shell_integration::IsFirefoxDefaultBrowser()) {
+    DetectFirefoxProfiles(locale, &profiles);
+    DetectBraveProfiles(&profiles);
+    DetectChromeProfiles(&profiles);
+  } else {
+    DetectBraveProfiles(&profiles);
+    DetectChromeProfiles(&profiles);
+    DetectFirefoxProfiles(locale, &profiles);
+  }
 #endif
   if (include_interactive_profiles) {
     importer::SourceProfile bookmarks_profile;
