diff --git a/chrome/browser/notifications/notification_display_service_impl.cc b/chrome/browser/notifications/notification_display_service_impl.cc
index f52424fc5203040ff2a68ddc8b0f6bfc7409a281..250bde945528eb536b18f696792b8c4e44499fc7 100644
--- a/chrome/browser/notifications/notification_display_service_impl.cc
+++ b/chrome/browser/notifications/notification_display_service_impl.cc
@@ -100,6 +100,7 @@ NotificationDisplayServiceImpl::NotificationDisplayServiceImpl(Profile* profile)
                              std::make_unique<NearbyNotificationHandler>());
     }
 #endif
+    BRAVE_ADD_BRAVE_ADS_NOTIFICATION_HANDLER;
   }
 
   bridge_delegator_ = std::make_unique<NotificationPlatformBridgeDelegator>(
