diff --git a/chrome/browser/profiles/profile_manager.h b/chrome/browser/profiles/profile_manager.h
index 3e1e28ea7d0e9c90d0c2189dae6ba6a394582aac..7a7746905e152dcdabaf7c3de24a31f8528fd662 100644
--- a/chrome/browser/profiles/profile_manager.h
+++ b/chrome/browser/profiles/profile_manager.h
@@ -269,6 +269,7 @@ class ProfileManager : public content::NotificationObserver,
   void set_do_final_services_init(bool do_final_services_init) {
     do_final_services_init_ = do_final_services_init;
   }
+  BRAVE_PROFILE_MANAGER_H
 
  private:
   friend class TestingProfileManager;
