diff --git a/chrome/browser/download/download_item_model.h b/chrome/browser/download/download_item_model.h
index 08de05eecb03975f7dd017d29f43ecf2c0c2809f..b186939b744c9b905b6b3bf3a01acb16a19f428e 100644
--- a/chrome/browser/download/download_item_model.h
+++ b/chrome/browser/download/download_item_model.h
@@ -112,6 +112,7 @@ class DownloadItemModel : public DownloadUIModel,
   void OnDownloadDestroyed(download::DownloadItem* download) override;
 
  private:
+  friend class BraveDownloadItemModel;
   // DownloadUIModel implementation.
   std::string GetMimeType() const override;
   bool IsExtensionDownload() const override;
