diff --git a/chrome/browser/ui/tabs/public/tab_features.h b/chrome/browser/ui/tabs/public/tab_features.h
index e20edfd974a55826c99df701efc09a67290adf76..51d4f64350950532dcff59f72d17b3461e85ae7b 100644
--- a/chrome/browser/ui/tabs/public/tab_features.h
+++ b/chrome/browser/ui/tabs/public/tab_features.h
@@ -117,7 +117,7 @@ class InactiveWindowMouseEventController;
 class TabFeatures {
  public:
   TabFeatures();
-  ~TabFeatures();
+  virtual ~TabFeatures();
 
   TabFeatures(const TabFeatures&) = delete;
   TabFeatures& operator=(const TabFeatures&) = delete;
