diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index c3d02b41f530f55bbe99094dd190aaac2cbfe70f..098aca7d4a4ac7502853bd96a7ae2a760d0a3f18 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -257,7 +257,7 @@ const char Profile::kProfileKey[] = "__PROFILE__";
 void Profile::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
       prefs::kSearchSuggestEnabled,
-      true,
+      false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 #if defined(OS_ANDROID)
   registry->RegisterStringPref(
