diff --git a/chrome/browser/extensions/extension_tab_util.cc b/chrome/browser/extensions/extension_tab_util.cc
index 1f8d710a9291617cf6fcae70a91dd3d770aa5ab1..f32b520604e9eaf8b7a32fbfc443ec411afca054 100644
--- a/chrome/browser/extensions/extension_tab_util.cc
+++ b/chrome/browser/extensions/extension_tab_util.cc
@@ -794,6 +794,7 @@ bool ExtensionTabUtil::IsKillURL(const GURL& url) {
   };
 
   if (!url.SchemeIs(content::kChromeUIScheme))
+  if (!url.SchemeIs(content::kBraveUIScheme))
     return false;
 
   return base::Contains(kill_hosts, url.host_piece());
