diff --git a/chrome/installer/linux/BUILD.gn b/chrome/installer/linux/BUILD.gn
index f5543a3271a7cd239d2f41516cb8a992bffa36dc..ae4b77895894d72c959d1f1b681e0026b134d08b 100644
--- a/chrome/installer/linux/BUILD.gn
+++ b/chrome/installer/linux/BUILD.gn
@@ -30,8 +30,8 @@ declare_args() {
 assert(is_linux || is_chromeos)
 
 packaging_files_executables = [
-  "$root_out_dir/chrome",
-  "$root_out_dir/chrome_sandbox",
+  "$root_out_dir/brave",
+  "$root_out_dir/brave_sandbox",
   "$root_out_dir/crashpad_handler",
 ]
 packaging_files_shlibs = []
@@ -167,10 +167,15 @@ action("merge_rpm_dependencies") {
 }
 
 action("strip_chrome_binary") {
-  prog_name = "$root_out_dir/chrome"
+  prog_name = "$root_out_dir/brave"
   debug_file = prog_name + ".debug"
   stripped_file = prog_name + ".stripped"
   deps = [ "//chrome" ]
+  if (brave_chromium_build) {
+    deps += [
+      "//brave/build/linux:copy_exe",
+    ]
+  }
   script = "//build/gn_run_binary.py"
   sources = [
     "//buildtools/third_party/eu-strip/bin/eu-strip",
@@ -208,7 +213,7 @@ branding_dir_100 =
 copy("common_packaging_files") {
   visibility = [ ":*" ]
   sources = [
-    "//chrome/app/resources/manpage.1.in",
+    "//brave/chromium_src/chrome/app/resources/manpage.1.in",
     "common/apt.include",
     "common/default-app-block.template",
     "common/default-app.template",
@@ -231,8 +236,8 @@ copy("common_packaging_files") {
     ]
   } else {
     sources += [
-      "common/chromium-browser/chromium-browser.appdata.xml",
-      "common/chromium-browser/chromium-browser.info",
+      "//brave/chromium_src/chrome/installer/linux/common/brave-browser/chromium-browser.appdata.xml",
+      "//brave/chromium_src/chrome/installer/linux/common/brave-browser/chromium-browser.info",
     ]
   }
 
@@ -333,6 +338,12 @@ group("installer_deps") {
     "//sandbox/linux:chrome_sandbox",
     "//third_party/crashpad/crashpad/handler:crashpad_handler",
   ]
+  if (brave_chromium_build) {
+    public_deps += [
+      "//brave/build/linux:copy_exe",
+      "//brave/build/linux:copy_sandbox",
+    ]
+  }
   if (enable_nacl) {
     public_deps += [
       "//components/nacl/loader:nacl_helper",
@@ -367,6 +378,9 @@ group("installer_deps") {
 template("linux_package") {
   assert(defined(invoker.channel))
   channel = invoker.channel
+  if (channel == "unstable" ) {
+    channel = "dev"
+  }
 
   if (is_chrome_branded) {
     package = "google-chrome"
@@ -530,3 +544,6 @@ linux_package("beta") {
 linux_package("unstable") {
   channel = "unstable"
 }
+linux_package("nightly") {
+  channel = "nightly"
+}
