diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java
index ff9f5ccac61823d09b6f98c91e6dbc16bc035ab8..683ed920aa2440e6d40911d19aeade983eb05f62 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/AndroidSyncSettings.java
@@ -74,7 +74,7 @@ public class AndroidSyncSettings {
     public static AndroidSyncSettings get() {
         ThreadUtils.assertOnUiThread();
         if (sInstance == null) {
-            sInstance = new AndroidSyncSettings(getSyncAccount());
+            sInstance = new BraveAndroidSyncSettings(getSyncAccount());
         }
         return sInstance;
     }
