diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 37546a17ce8f3c114105b117d325c3493c3ae3e9..ab2f2b0b07ccb6c75c5c254283ecee9d1addba0f 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2466,6 +2466,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
