diff --git a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
index 8966c0a6e10bddf6403d9cfaa37497f8c3033114..aa2b3fe780f6e258705153df0e52fc5d6dcef6b1 100644
--- a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
+++ b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
@@ -1568,7 +1568,13 @@ void TabDragController::AttachToNewContext(
 
       const bool pinned = detached_tab_collection->get()->pinned_;
 
-      if (std::holds_alternative<std::unique_ptr<tabs::TabGroupTabCollection>>(
+      if (std::holds_alternative<
+              std::unique_ptr<tabs::TreeTabNodeTabCollection>>(
+              detached_tab_collection->get()->collection_)) {
+        collection_indices =
+            attached_context_->GetTabStripModel()->InsertDetachedTreeTabNodeAt(
+                std::move(*detached_tab_collection), index);
+      } else if (std::holds_alternative<std::unique_ptr<tabs::TabGroupTabCollection>>(
               detached_tab_collection->get()->collection_)) {
         collection_indices =
             attached_context_->GetTabStripModel()->InsertDetachedTabGroupAt(
