diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index cd4395923de75739b829a5eb9530b025cb9bc309..5710018ff35e4d0042c41dcec729bdaee39839c9 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -229,6 +229,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  sources += brave_chrome_renderer_sources public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
