diff --git a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h
index 513a64cc3f34e4b9a88fe6fb1776fc2d60d1819e..e2e30875bd1b7e7aea88172b6fded4700c42865a 100644
--- a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h
+++ b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h
@@ -82,6 +82,7 @@ class ChromeBrowsingDataRemoverDelegate
   void OverrideDomainReliabilityClearerForTesting(
       DomainReliabilityClearer clearer);
 
+  BRAVE_CHROME_BROWSING_DATA_REMOVER_DELEGATE_H
  private:
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
@@ -130,7 +131,8 @@ class ChromeBrowsingDataRemoverDelegate
     kAccountCompromisedCredentials = 39,
     kFaviconCacheExpiration = 40,
     kSecurePaymentConfirmationInstruments = 41,
-    kMaxValue = kSecurePaymentConfirmationInstruments,
+    kIPFSCache = 42,
+    kMaxValue = kIPFSCache,
   };
 
   // Called by CreateTaskCompletionClosure().
