diff --git a/chrome/common/webui_url_constants.h b/chrome/common/webui_url_constants.h
index a85c61259b8398196e3f0fc4cbdc73144e6c0b64..82f1c490523cead1f41bb119047b633aab6b932c 100644
--- a/chrome/common/webui_url_constants.h
+++ b/chrome/common/webui_url_constants.h
@@ -161,6 +161,8 @@ extern const char kChromeUIResetPasswordHost[];
 extern const char kChromeUIResetPasswordURL[];
 extern const char kChromeUIRestartHost[];
 extern const char kChromeUIRestartURL[];
+extern const char kChromeUIRewardsSyncInternalsHost[];
+extern const char kChromeUIRewardsSyncInternalsUrl[];
 extern const char kChromeUISafetyPixelbookURL[];
 extern const char kChromeUISafetyPixelSlateURL[];
 extern const char kChromeUISegmentationInternalsHost[];
