diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 588cea7fe2d3cb649cfbc701101cfbd74f7b6bb3..056323681a6724782088d4f578c860fdebc89b69 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -670,6 +670,7 @@ static_library("ui") {
     "//ui/webui",
     "//v8:v8_version",
   ]
+  deps+= [ "//brave/browser/ui" ]
 
   # TODO(crbug/925153): Remove this circular dependency.
   allow_circular_includes_from = [
@@ -3885,10 +3886,13 @@ static_library("ui") {
       ]
       deps += [ "//google_update" ]
     } else {
-      sources += [
-        "webui/help/version_updater_basic.cc",
-        "webui/help/version_updater_basic.h",
-      ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [
+          "webui/help/version_updater_basic.cc",
+          "webui/help/version_updater_basic.h",
+        ]
+      }
     }
   } else {  # 'OS!="win"
     # The Cast Linux build sets toolkit_views to false, but many files in this
