diff --git a/chrome/browser/ui/tab_helpers.cc b/chrome/browser/ui/tab_helpers.cc
index cb5f3f3bdb28446ec344ca6d7a5aa67ab7d6db63..0e2a243d5445533d26ab602b8bfd8112cf82c850 100644
--- a/chrome/browser/ui/tab_helpers.cc
+++ b/chrome/browser/ui/tab_helpers.cc
@@ -858,4 +858,5 @@ void TabHelpers::AttachTabHelpers(WebContents* web_contents,
 
   // This is common code for all of us. PLEASE DO YOUR PART to keep it tidy and
   // organized.
+  BRAVE_TAB_HELPERS
 }
