diff --git a/chrome/browser/ui/android/toolbar/BUILD.gn b/chrome/browser/ui/android/toolbar/BUILD.gn
index e33bf182f71b7c7fe905b14a79c8f5fcb262c4aa..455707f269b80635da78d990c43f173b80cab32c 100644
--- a/chrome/browser/ui/android/toolbar/BUILD.gn
+++ b/chrome/browser/ui/android/toolbar/BUILD.gn
@@ -137,6 +137,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/StripVisibilityState.java",
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/TabStripTransitionCoordinator.java",
   ]
+  sources += brave_browser_ui_android_toolbar_sources
 
   deps = [
     ":java_resources",
