diff --git a/chrome/browser/ui/webui/settings/md_settings_ui.cc b/chrome/browser/ui/webui/settings/md_settings_ui.cc
index 368ba572922402e6e452c5db0349a76d5138c3cf..40e6171741cf834c63ef407c659b82bc8af30e81 100644
--- a/chrome/browser/ui/webui/settings/md_settings_ui.cc
+++ b/chrome/browser/ui/webui/settings/md_settings_ui.cc
@@ -136,6 +136,8 @@ void MdSettingsUI::RegisterProfilePrefs(
   registry->RegisterBooleanPref(prefs::kImportDialogHistory, true);
   registry->RegisterBooleanPref(prefs::kImportDialogSavedPasswords, true);
   registry->RegisterBooleanPref(prefs::kImportDialogSearchEngine, true);
+  registry->RegisterBooleanPref(prefs::kImportDialogCookies, true);
+  registry->RegisterBooleanPref(prefs::kImportDialogStats, true);
 }
 
 MdSettingsUI::MdSettingsUI(content::WebUI* web_ui)
