diff --git a/chrome/installer/util/BUILD.gn b/chrome/installer/util/BUILD.gn
index dae16bd9569eb23ce882df850bfe0295d1a9f2e9..9f93eae6e11e551c59c41130c474cfd54cea2347 100644
--- a/chrome/installer/util/BUILD.gn
+++ b/chrome/installer/util/BUILD.gn
@@ -224,7 +224,20 @@ generate_embedded_i18n("generate_strings") {
 
   output_file_name_base = "installer_util_strings"
 
-  branding = branding_path_component
+  brand = "$branding_path_component"
+  if (brave_chromium_build) {
+    # When brave_strings.grd is modified, outputs should be re-generated.
+    # This is workaround. Need to change the way of using |brave_strings.grd|
+    # in |create_string_rc.py|.
+    sources = [
+      "//chrome/app/brave_strings.grd"
+    ]
+    if (!is_official_build) {
+      # TODO(shong): Remove this.
+      brand = brand + "-development"
+    }
+  }
+  branding = brand
 }
 
 # Compile the generated .rc file.
