diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 2661face33160de31a37fe43f458a27e7ae0f319..1ea29d2dddf15055d5ea7e633cb84719f4f922de 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -66,4 +66,5 @@ source_set("configs") {
       ]
     }
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
