diff --git a/chrome/browser/importer/in_process_importer_bridge.h b/chrome/browser/importer/in_process_importer_bridge.h
index 3d3e8028cd062e61d0b91fdd02c15bd31905ec2f..f90a639407aa86d8d650dea42ffc74180e838039 100644
--- a/chrome/browser/importer/in_process_importer_bridge.h
+++ b/chrome/browser/importer/in_process_importer_bridge.h
@@ -59,6 +59,7 @@ class InProcessImporterBridge : public ImporterBridge {
   std::u16string GetLocalizedString(int message_id) override;
   // End ImporterBridge implementation.
 
+  BRAVE_IN_PROCESS_IMPORTER_BRIDGE_H
  private:
   ~InProcessImporterBridge() override;
 
