diff --git a/chrome/browser/ui/safety_hub/password_status_check_service.cc b/chrome/browser/ui/safety_hub/password_status_check_service.cc
index 84fa227db0263a585057bdbeb0c2746c83cea0eb..2f8b7243888b6dfe889f05fbf19fec8a6297d660 100644
--- a/chrome/browser/ui/safety_hub/password_status_check_service.cc
+++ b/chrome/browser/ui/safety_hub/password_status_check_service.cc
@@ -721,6 +721,7 @@ base::TimeDelta PasswordStatusCheckService::GetScheduledPasswordCheckInterval()
 
 base::Value::Dict PasswordStatusCheckService::GetPasswordCardData(
     bool signed_in) {
+  BRAVE_PASSWORD_STATUS_CHECK_SERVICE_GET_PASSWORD_CARD_DATA
   if (no_passwords_saved()) {
     bool password_saving_allowed = profile_->GetPrefs()->GetBoolean(
         password_manager::prefs::kCredentialsEnableService);
