diff --git a/chrome/browser/ui/toolbar/toolbar_actions_model_factory.cc b/chrome/browser/ui/toolbar/toolbar_actions_model_factory.cc
index c8d6031565de2ba8ea4a3df3fc1ac005c6c7cdab..b7a64a3c8988901fb1a9a79c6937f1482a3038ec 100644
--- a/chrome/browser/ui/toolbar/toolbar_actions_model_factory.cc
+++ b/chrome/browser/ui/toolbar/toolbar_actions_model_factory.cc
@@ -42,7 +42,7 @@ ToolbarActionsModelFactory::~ToolbarActionsModelFactory() {}
 
 KeyedService* ToolbarActionsModelFactory::BuildServiceInstanceFor(
     content::BrowserContext* context) const {
-  return new ToolbarActionsModel(
+  return new BraveToolbarActionsModel(
       Profile::FromBrowserContext(context),
       extensions::ExtensionPrefsFactory::GetForBrowserContext(context));
 }
