diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 2908b4b70815d02afff0c78c1aa54ab74c79dd83..2e994890ea12484a2dc6539a87d8f51afe0fca80 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -709,6 +709,7 @@ class Browser : public TabStripModelObserver,
 
   StatusBubble* GetStatusBubbleForTesting();
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
