diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index fdba8d4db78727a4b580f4770b8e2e66a66c2b99..9f02ed90916b9151f795bdf0454cd3e0a796f265 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -8,6 +8,7 @@ import {pageVisibility} from './page_visibility.js';
 import {Route, Router} from './router.js';
 import {SettingsRoutes} from './settings_routes.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -220,6 +221,7 @@ function createBrowserSettingsRoutes(): SettingsRoutes {
       // </if>
     }
   }
+  addBraveRoutes(r);
   return r;
 }
 
