diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index f5efeb560f35a57eee16bae3776473acb1084b71..b97cb0f481ed90a747cd47ad6b196e5569a8abb1 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4420,6 +4420,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
