diff --git a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java
index 4352e388dd9cca0d3bf67930d8665e74e0125a19..d2c5eaa9e399984f85f9f03de887217c9e70ec1e 100644
--- a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java
+++ b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java
@@ -473,6 +473,9 @@ public class FuseboxCoordinator implements TemplateUrlServiceObserver {
     // TemplateUrlServiceObserver
     @Override
     public void onTemplateURLServiceChanged() {
+        if (!mTemplateUrlService.hasNativeService()) {
+            return;
+        }
         boolean isDseGoogle = mTemplateUrlService.isDefaultSearchEngineGoogle();
         if (isDseGoogle == mDefaultSearchEngineIsGoogle) return;
         mDefaultSearchEngineIsGoogle = isDseGoogle;
