diff --git a/chrome/android/java/src/org/chromium/chrome/browser/document/ChromeLauncherActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/document/ChromeLauncherActivity.java
index 5f75116ae19c8db7ee580113f1e761a2efa2e441..7c1cdd2a0c17df91d7acfaa0d1949e66e55b7012 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/document/ChromeLauncherActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/document/ChromeLauncherActivity.java
@@ -20,7 +20,7 @@ import org.chromium.chrome.browser.vr.VrModuleProvider;
  * Dispatches incoming intents to the appropriate activity based on the current configuration and
  * Intent fired.
  */
-public class ChromeLauncherActivity extends Activity {
+public class ChromeLauncherActivity extends BraveLauncherActivity {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         // Third-party code adds disk access to Activity.onCreate. http://crbug.com/619824
