diff --git a/chrome/browser/ui/browser_actions.h b/chrome/browser/ui/browser_actions.h
index b4670400235b9963a0b446caacb384a0efff5238..dde2016b9c806b087e66f397d1b8cec5ae739566 100644
--- a/chrome/browser/ui/browser_actions.h
+++ b/chrome/browser/ui/browser_actions.h
@@ -22,7 +22,7 @@ class BrowserActions {
   explicit BrowserActions(Browser& browser);
   BrowserActions(const BrowserActions&) = delete;
   BrowserActions& operator=(const BrowserActions&) = delete;
-  ~BrowserActions();
+  virtual ~BrowserActions();
 
   static std::u16string GetCleanTitleAndTooltipText(std::u16string string);
 
