diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 501812236b5a7255a92abbbfea622a5327afba1e..dea9414c8ca59aa42217fd4791ed085a591816ee 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4730,6 +4730,7 @@ const FeatureEntry::FeatureVariation kGestureUserEducationVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
