diff --git a/chrome/browser/extensions/extension_install_prompt.h b/chrome/browser/extensions/extension_install_prompt.h
index bf9919930d0fdf0b7e25b79fca04e14f59cc20b0..bfa2fc3d3053e07b366d65ec5133a23d23f91a12 100644
--- a/chrome/browser/extensions/extension_install_prompt.h
+++ b/chrome/browser/extensions/extension_install_prompt.h
@@ -86,7 +86,7 @@ class ExtensionInstallPrompt {
   class Prompt {
    public:
     explicit Prompt(PromptType type);
-    ~Prompt();
+    virtual ~Prompt();
 
     void AddPermissionSet(const extensions::PermissionSet& permissions);
     void AddPermissionMessages(
@@ -169,6 +169,7 @@ class ExtensionInstallPrompt {
 
     bool has_webstore_data() const { return has_webstore_data_; }
 
+    BRAVE_EXTENSION_INSTALL_PROMPT_H
    private:
     bool ShouldDisplayRevokeButton() const;
 
