diff --git a/chrome/installer/setup/setup_main.h b/chrome/installer/setup/setup_main.h
index 389c7a75f2a87d54dd9b5194cd1b9423200d0c02..ed60f080117f159a4f7493d6072a3e24e489ee7f 100644
--- a/chrome/installer/setup/setup_main.h
+++ b/chrome/installer/setup/setup_main.h
@@ -22,14 +22,15 @@ class InitialPreferences;
 
 // Helper function that performs the installation of a set of products.
 // |installer_directory|, if non-nullptr, is populated with the path to the
-// directory containing the newly installed setup.exe. delegated_to_existing|
-// is set to |true| if installation was delegated to a pre-existing higher
-// version.
+// directory containing the newly installed setup.exe. |archive_type| is
+// populated with the type of archive found. |delegated_to_existing| is set to
+// |true| if installation was delegated to a pre-existing higher version.
 InstallStatus InstallProductsHelper(InstallationState& original_state,
                                     const base::FilePath& setup_exe,
                                     const base::CommandLine& cmd_line,
                                     const InitialPreferences& prefs,
-                                    InstallerState& installer_state);
+                                    InstallerState& installer_state,
+                                    ArchiveType* archive_type);
 
 }  // namespace installer
 
