diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 450ffba054c7cb988a482e2e02fed5284ca89edc..e592ee7811ed6fbd9aa0d56617834c5821135f50 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2559,6 +2559,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps defines += brave_chrome_browser_defines
 
   if (is_win || is_mac || is_linux || is_chromeos_ash) {
     deps += [ "//chrome/browser/ui/webui/connectors_internals:mojo_bindings" ]
