diff --git a/chrome/browser/profiles/chrome_browser_main_extra_parts_profiles.cc b/chrome/browser/profiles/chrome_browser_main_extra_parts_profiles.cc
index b187a831c54196cc19db3fceb4e9869ffe8c99f7..d3c944186f118cafd85982d0579212898f6ca287 100644
--- a/chrome/browser/profiles/chrome_browser_main_extra_parts_profiles.cc
+++ b/chrome/browser/profiles/chrome_browser_main_extra_parts_profiles.cc
@@ -707,7 +707,7 @@ void ChromeBrowserMainExtraPartsProfiles::
   chromeos_extensions::EnsureBrowserContextKeyedServiceFactoriesBuilt();
 #endif
   extensions::EnsureBrowserContextKeyedServiceFactoriesBuilt();
-#elif BUILDFLAG(ENABLE_DESKTOP_ANDROID_EXTENSIONS)
+#elif BUILDFLAG(IS_ANDROID) && BUILDFLAG(ENABLE_EXTENSIONS_CORE)
   chrome_extensions::EnsureBrowserContextKeyedServiceFactoriesBuilt();
   // EnsureBrowserContextKeyedServiceFactoriesBuilt() is invoked before the
   // ExtensionsBrowserClient is ready on Android. This is due to Android
