diff --git a/chrome/browser/enterprise/connectors/analysis/content_analysis_dialog.cc b/chrome/browser/enterprise/connectors/analysis/content_analysis_dialog.cc
index d6df3c2d066aa120d56d3219ce344f3ab4ae0411..6b8032886997afbc2f224a812e4837afa2aa2b7a 100644
--- a/chrome/browser/enterprise/connectors/analysis/content_analysis_dialog.cc
+++ b/chrome/browser/enterprise/connectors/analysis/content_analysis_dialog.cc
@@ -108,10 +108,7 @@ class CircleBackground : public views::Background {
     canvas->DrawCircle(center, radius, flags);
   }
 
-  void OnViewThemeChanged(views::View* view) override {
-    view->SchedulePaint();
-  }
-
+  void OnViewThemeChanged(views::View* view) override { view->SchedulePaint(); }
 };
 
 gfx::Rect GetDialogBounds(content::WebContents* contents,
@@ -310,7 +307,7 @@ void ContentAnalysisDialog::ShowDialogNow() {
 // Glic port enabled for Mac only at the moment until fixed on Windows.
 // TODO(416748209): Follow up with full port of ContentAnalysisDialog to use
 // non web modals on both Mac and Windows for all sources.
-#if BUILDFLAG(IS_MAC)
+#if BUILDFLAG(ENABLE_GLIC) && BUILDFLAG(IS_MAC)
   if (glic::IsGlicWebUI(top_level_contents_.get())) {
     // make sure only one dialog is displayed at a time. If a dialog exists we
     // just update the view.
