diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 33a36da42b37e78d496a6db0607cff58de43019e..6bab3ab321636406d1cbb06dc220a6d77d90ad28 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -1953,6 +1953,7 @@ jumbo_split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
