diff --git a/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc b/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
index be052ca27cc4df4a3072d1e85bbabd4abab97ca5..4e1599b67cadb344d19149b2dccf12ece7d313a2 100644
--- a/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
+++ b/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
@@ -683,7 +683,7 @@ BrowserViewTabbedLayoutImpl::CalculateProposedLayout(
   }
 
   BrowserLayoutParams params = layout_data_->revised_params;
-  bool needs_exclusion = true;
+  bool needs_exclusion = !delegate().ShouldShowVerticalTabs();
   const bool adjust_for_cracking = AvoidCrackingForFractionalDisplay();
   const HorizontalLayout& horizontal_layout = layout_data_->horizontal_layout;
 
