diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 039496810cf43f43c4f9365d157139ca07ce240b..565fcb85a11d0d5b8f58b79bbe1f81cb2405dc97 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -209,6 +209,7 @@ android_resources("chrome_app_java_resources") {
     "//third_party/android_deps:androidx_recyclerview_recyclerview_java",
     "//third_party/android_deps:material_design_java",
   ]
+  deps += brave_chrome_app_java_resources_deps sources += brave_java_resources
   create_srcjar = false
 }
 
@@ -497,6 +498,7 @@ android_library("chrome_java") {
   ]
 
   deps += feed_deps
+  deps += brave_chrome_java_deps
 
   srcjar_deps = [
     ":chrome_android_java_enums_srcjar",
@@ -533,6 +535,7 @@ android_library("chrome_java") {
     "//content/public/browser:contacts_picker_properties_requested_javagen",
     "//net:effective_connection_type_java",
   ]
+  srcjar_deps += brave_chrome_java_srcjar_deps
 
   # From java_sources.gni.
   sources = chrome_java_sources + [
@@ -657,6 +660,7 @@ group("jni_headers") {
     "//chrome/browser/util:jni_headers",
     "//chrome/browser/webauthn/android:jni_headers",
   ]
+  public_deps += ["//brave/build/android:jni_headers"]
 }
 
 action("chrome_android_java_google_api_keys_srcjar") {
@@ -1719,6 +1723,7 @@ if (current_toolchain == default_toolchain) {
         ":${_variant}_locale_pak_assets",
         ":${_variant}_paks",
       ]
+      deps += [ "//brave:brave_pak_assets" ]
       if (_is_monochrome) {
         deps += [ "//android_webview:locale_pak_assets" ]
         if (webview_includes_weblayer && !_is_bundle_module) {
