diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index c9579f87ddc11665974be6848df26aaace72c9a8..a83e5eeab2d80d1e16e7e9dfec97d70cb3ce25af 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4616,6 +4616,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
