diff --git a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
index 0cc1b7a5d73c6412c6351472af43b41e64d77a8d..9e755ec05d8e2fd098006cea150eead3b0d9a4b8 100644
--- a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
+++ b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
@@ -726,6 +726,7 @@ SkPath IconLabelBubbleView::GetHighlightPath() const {
   const float corner_radius = highlight_bounds.height() / 2.f;
   const SkRect rect = RectToSkRect(highlight_bounds);
 
+  BRAVE_ICON_LABEL_BUBBLE_VIEW_GET_HIGHLIGHT_PATH
   return SkPath().addRoundRect(rect, corner_radius, corner_radius);
   // return SkPath().addCircle(12, radius, radius); // size / 2
 }
