diff --git a/chrome/browser/plugins/plugin_utils.cc b/chrome/browser/plugins/plugin_utils.cc
index 8b04209fa967cf36c6f90a8c47dc22cc4fd25172..9ba805fdb55ebfec32bf05a4f4c53bf32a991fbb 100644
--- a/chrome/browser/plugins/plugin_utils.cc
+++ b/chrome/browser/plugins/plugin_utils.cc
@@ -158,7 +158,8 @@ std::string PluginUtils::GetExtensionIdForMimeType(
   bool profile_is_off_the_record = io_data->IsOffTheRecord();
   const scoped_refptr<const extensions::InfoMap> extension_info_map(
       io_data->GetExtensionInfoMap());
-  std::vector<std::string> whitelist = MimeTypesHandler::GetMIMETypeWhitelist();
+  std::vector<std::string> whitelist =
+      MimeTypesHandler::BraveGetMIMETypeWhitelist();
   // Go through the white-listed extensions and try to use them to intercept
   // the URL request.
   for (const std::string& extension_id : whitelist) {
