diff --git a/chrome/browser/speech/on_device_speech_recognition_util.cc b/chrome/browser/speech/on_device_speech_recognition_util.cc
index 2b80c9563ed6ff469a40b42705ad9d43536b3cf8..48489a878a4b90c766e927b41a7c1168be892429 100644
--- a/chrome/browser/speech/on_device_speech_recognition_util.cc
+++ b/chrome/browser/speech/on_device_speech_recognition_util.cc
@@ -114,6 +114,7 @@ media::mojom::AvailabilityStatus GetOnDeviceSpeechRecognitionAvailabilityStatus(
     content::BrowserContext* context,
     std::string_view language,
     media::mojom::SpeechRecognitionQuality quality) {
+  return GetBraveOnDeviceSpeechRecognitionAvailabilityStatus(language, quality);
 #if BUILDFLAG(IS_ANDROID)
   return media::mojom::AvailabilityStatus::kUnavailable;
 #else
