diff --git a/chrome/browser/devtools/devtools_ui_bindings.h b/chrome/browser/devtools/devtools_ui_bindings.h
index f32e33f4da0c8f6ecb8fa9a04cc0e7535dfa0e7d..93244fa307a801efa98fcc7f059494436117c0d9 100644
--- a/chrome/browser/devtools/devtools_ui_bindings.h
+++ b/chrome/browser/devtools/devtools_ui_bindings.h
@@ -106,6 +106,8 @@ class DevToolsUIBindings : public DevToolsEmbedderMessageDispatcher::Delegate,
 
  private:
   using DevToolsUIBindingsList = std::vector<DevToolsUIBindings*>;
+  friend class BraveDevToolsUIBindings;
+  FRIEND_TEST_ALL_PREFIXES(BraveDevToolsUIBindingsBrowserTest, ThemeTest);
 
   void HandleMessageFromDevToolsFrontend(base::Value);
 
