diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index b3954a5fec88eccada797be9fd23e8ccca045d86..ba22ef6fd1af5cffe6b3712ead6e6037cdc8a37f 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3954,6 +3954,7 @@ const flags_ui::FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
