diff --git a/chrome/browser/permissions/permission_uma_util.cc b/chrome/browser/permissions/permission_uma_util.cc
index e74fe0ced0587cdcd6a303e10ac04f1ad6f37698..7cebaf09cda71e77405d2c102bc7a09d91e48bf0 100644
--- a/chrome/browser/permissions/permission_uma_util.cc
+++ b/chrome/browser/permissions/permission_uma_util.cc
@@ -65,7 +65,7 @@ namespace {
 
 const int kPriorCountCap = 10;
 
-std::string GetPermissionRequestString(PermissionRequestType type) {
+std::string GetPermissionRequestString_ChromiumImpl(PermissionRequestType type) {
   switch (type) {
     case PermissionRequestType::MULTIPLE:
       return "AudioAndVideoCapture";
@@ -472,6 +472,9 @@ void PermissionUmaUtil::RecordPermissionAction(
       base::UmaHistogramEnumeration("Permissions.Action.Nfc", action,
                                     PermissionAction::NUM);
       break;
+    case ContentSettingsType::AUTOPLAY:
+      BraveRecordPermissionAction(permission, secure_origin, action);
+      break;
     // The user is not prompted for these permissions, thus there is no
     // permission action recorded for them.
     default:
