diff --git a/chrome/browser/resources/extensions/extensions.ts b/chrome/browser/resources/extensions/extensions.ts
index 378090e6f4816577f578ac418c5b449e68240ee3..40ae9e27676a5b67b0cf5fcf7f07dffab875ce6b 100644
--- a/chrome/browser/resources/extensions/extensions.ts
+++ b/chrome/browser/resources/extensions/extensions.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './manager.js';
 
 export {getToastManager} from 'chrome://resources/cr_elements/cr_toast/cr_toast_manager.js';
