diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index baa390055c1ac466f1e48ee82addbff2365259de..4abd8032330488e701794d7bfc7850289d631c78 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -377,7 +377,7 @@ const char kGlicActorUiDebounceTimerName[] = "glic-actor-ui-debounce-timer";
 
 // Controls whether the task icon in the actor ui is enabled.
 const base::FeatureParam<bool> kGlicActorUiTaskIcon{
-    &kGlicActorUi, kGlicActorUiTaskIconName, true};
+    &kGlicActorUi, kGlicActorUiTaskIconName, false};
 // Controls whether the Actor Overlay in the actor ui is enabled.
 const base::FeatureParam<bool> kGlicActorUiOverlay{
     &kGlicActorUi, kGlicActorUiOverlayName, true};
