diff --git a/chrome/browser/content_settings/BUILD.gn b/chrome/browser/content_settings/BUILD.gn
index 44113e3e1dd53208ea95905c260ee514a41a5df6..40037bece120d7a1b56f3ea368ec32d1516c34bc 100644
--- a/chrome/browser/content_settings/BUILD.gn
+++ b/chrome/browser/content_settings/BUILD.gn
@@ -121,6 +121,7 @@ source_set("impl") {
   if (enable_pdf) {
     deps += [ "//chrome/browser/pdf" ]
   }
+  import("//brave/browser/content_settings/sources.gni") deps += brave_browser_content_settings_deps
 }
 
 # Fixes circular deps with different components. For example -
