diff --git a/chrome/android/java/src/org/chromium/chrome/browser/AppHooks.java b/chrome/android/java/src/org/chromium/chrome/browser/AppHooks.java
index 34e911d6efd34d047353c5df585314842a4b539c..016574520a75283c4d0bddb37bdd4df825ea1461 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/AppHooks.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/AppHooks.java
@@ -75,7 +75,7 @@ public abstract class AppHooks {
 
     @CalledByNative
     public static AppHooks get() {
-        if (sInstance == null) sInstance = new AppHooksImpl();
+        if (sInstance == null) sInstance = new BraveAppHooks();
         return sInstance;
     }
 
