diff --git a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
index 4ecb7beb72338a44c4b71409aab29bf5c3662bde..c4600268cd6843b0720e12802c8db03b603350c3 100644
--- a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
+++ b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
@@ -34,7 +34,7 @@ ${function() {
             data-index="${index}">
         </profile-card>
       `)}
-      <cr-button id="addProfile" class="profile-item"
+      <cr-button id="addProfile" class="profile-item plain"
           @click="${this.onAddProfileClick_}"
           ?hidden="${!this.profileCreationAllowed_}"
           aria-labelledby="addProfileButtonLabel">
@@ -49,7 +49,6 @@ ${function() {
 <if expr="enable_glic">
   <div id="footer-text" class="subtitle"
       ?hidden="${this.shouldHideFooterText_()}">
-    $i18nRaw{glicAddProfileHelper}
   </div>
 </if>
 </div>
