diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 2e8ad258a2a5459e061df4a83c0d612d1cbfb310..02a856e86ef4c2d47e126a54b94bb3fd3eb95f6e 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4409,6 +4409,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
