diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.h b/chrome/browser/ui/views/toolbar/toolbar_view.h
index 21510da06867ca1e2b286010c44be71d33ae5168..1e039e2a559aaf6d767abdd1ce0c3d39fda6f8bf 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.h
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.h
@@ -93,12 +93,14 @@ class ToolbarView : public views::AccessiblePaneView,
   ~ToolbarView() override;
 
   // Create the contents of the Browser Toolbar.
+  virtual
   void Init();
 
   // Forces the toolbar (and transitively the location bar) to update its
   // current state.  If |tab| is non-NULL, we're switching (back?) to this tab
   // and should restore any previous location bar state (such as user editing)
   // as well.
+  virtual
   void Update(content::WebContents* tab);
 
   // Updates the visibility of the custom tab bar, potentially animating the
@@ -129,6 +131,7 @@ class ToolbarView : public views::AccessiblePaneView,
       IntentPickerResponse callback);
 
   // Shows a bookmark bubble and anchors it appropriately.
+  virtual
   void ShowBookmarkBubble(const GURL& url,
                           bool already_bookmarked,
                           bookmarks::BookmarkBubbleObserver* observer);
@@ -195,6 +198,7 @@ class ToolbarView : public views::AccessiblePaneView,
   void ChildPreferredSizeChanged(views::View* child) override;
 
  protected:
+  friend class BraveToolbarView;
   // AccessiblePaneView:
   bool SetPaneFocusAndFocusDefault() override;
 
@@ -242,6 +246,7 @@ class ToolbarView : public views::AccessiblePaneView,
   views::View* GetViewForDrop() override;
 
   // Loads the images for all the child views.
+  virtual
   void LoadImages();
 
   // Shows the critical notification bubble against the app menu.
