diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 6f768f7848a3885d516e337bd413d91e02e6900a..90beb01f613d62dce221195cf1310592342288de 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2261,6 +2261,7 @@ jumbo_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_win) {
