diff --git a/chrome/browser/download/download_item_model.h b/chrome/browser/download/download_item_model.h
index fafc8502127e3fea2007ba3b30b91fe708932c75..49c73246c95af21ece9bf8d9ee571c11152c6fe1 100644
--- a/chrome/browser/download/download_item_model.h
+++ b/chrome/browser/download/download_item_model.h
@@ -101,6 +101,7 @@ class DownloadItemModel : public DownloadUIModel,
   void OnDownloadDestroyed(download::DownloadItem* download) override;
 
  private:
+  friend class BraveDownloadItemModel;
   // DownloadUIModel implementation.
   std::string GetMimeType() const override;
   bool IsExtensionDownload() const override;
