diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 62b62d2ba1d51e88d184e24232ed3a5576660b3d..870fbd79f8da7e86bc2068e096e0e29d15267dbf 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -691,6 +691,7 @@ class Browser : public TabStripModelObserver,
 
   StatusBubble* GetStatusBubbleForTesting();
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
