diff --git a/chrome/browser/resources/extensions/BUILD.gn b/chrome/browser/resources/extensions/BUILD.gn
index fd049191b64dc6604ca6676f039637a99299e28b..2d20a4a68e837d70cf674f9314372daa1cfd8cfe 100644
--- a/chrome/browser/resources/extensions/BUILD.gn
+++ b/chrome/browser/resources/extensions/BUILD.gn
@@ -21,6 +21,7 @@ if (optimize_webui) {
     deps = [
       ":unpak",
       "../../../../ui/webui/resources:modulize",
+      "//brave/browser/resources/extensions:unpak",
     ]
     excludes = [ "chrome://resources/js/cr.m.js" ]
   }
