diff --git a/chrome/browser/ui/browser_window/BUILD.gn b/chrome/browser/ui/browser_window/BUILD.gn
index a21773edf33ab1f016ee67e90d0a3a4ef00569df..a13d21cfbed6dfeb74e2ab553b1042cffd3aa43a 100644
--- a/chrome/browser/ui/browser_window/BUILD.gn
+++ b/chrome/browser/ui/browser_window/BUILD.gn
@@ -39,6 +39,7 @@ source_set("browser_window") {
       "//ui/base:types",
     ]
   }
+  deps += [ "//brave/browser/ui/browser_window" ]
 }
 
 source_set("create_browser_window") {
