diff --git a/chrome/browser/resources/password_manager/BUILD.gn b/chrome/browser/resources/password_manager/BUILD.gn
index 7ec2ef22a1c79b0a593d263dc4d4c6291e975701..618e5fbf0ac7c132dafe2d76142d0e3d8c7ff4c4 100644
--- a/chrome/browser/resources/password_manager/BUILD.gn
+++ b/chrome/browser/resources/password_manager/BUILD.gn
@@ -154,4 +154,5 @@ build_webui("build") {
               "$root_gen_dir/chrome/browser/resources/settings_shared/tsc",
               root_build_dir) ]
   }
+  import("//brave/browser/resources/password_manager/sources.gni") non_web_component_files += brave_password_manager_non_web_component_files exclude_ts_preprocess_files = brave_password_manager_ts_local_files preprocess_deps = brave_password_manager_preprocess_extra_deps
 }
