diff --git a/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks.mojom b/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks.mojom
index f1d68e24a5f30d2bf8abdf5a38794fc57e68db85..889e571d18a80c00648380ec48d8c2f00f13c680 100644
--- a/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks.mojom
+++ b/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks.mojom
@@ -23,9 +23,10 @@ enum SortOrder {
   kLastOpened = 2,
   kAlphabetical = 3,
   kReverseAlphabetical = 4,
+  kCustom = 5,
 
   // Must be last. May add new values before this one.
-  kCount = 5,
+  kCount = 6,
 };
 
 // Used to indicate the order bookmarks are sorted in the side panel.
