diff --git a/chrome/browser/ui/views/extensions/extension_popup.cc b/chrome/browser/ui/views/extensions/extension_popup.cc
index 3f0ed849c90098e9cbc0a64d580879b87b9468bf..b3f38c8025d2b3a305d6612f606480d36f987cd2 100644
--- a/chrome/browser/ui/views/extensions/extension_popup.cc
+++ b/chrome/browser/ui/views/extensions/extension_popup.cc
@@ -67,7 +67,7 @@ gfx::Size ExtensionPopup::CalculatePreferredSize() const {
 
 void ExtensionPopup::AddedToWidget() {
   BubbleDialogDelegateView::AddedToWidget();
-  const int radius = GetBubbleFrameView()->corner_radius();
+  BRAVE_ADDED_TO_WIDGET
   const bool contents_has_rounded_corners =
       GetExtensionView()->holder()->SetCornerRadius(radius);
   SetBorder(views::CreateEmptyBorder(
