diff --git a/chrome/browser/notifications/BUILD.gn b/chrome/browser/notifications/BUILD.gn
index 6a6492f576b690406acc5b36f31e49fd50576b0b..3d6e66a35f53610c62d792a8420746ee5eb6ccfa 100644
--- a/chrome/browser/notifications/BUILD.gn
+++ b/chrome/browser/notifications/BUILD.gn
@@ -82,6 +82,7 @@ if (is_android) {
       "//third_party/androidx:androidx_core_core_java",
       "//ui/android:ui_no_recycler_view_java",
     ]
+    deps += [ "//brave/browser/notifications/android:brave_java", ]
 
     srcjar_deps = [ ":notifications_java_enum_srcjar" ]
     resources_package = "org.chromium.chrome.browser.notifications"
