diff --git a/chrome/browser/password_manager/android/BUILD.gn b/chrome/browser/password_manager/android/BUILD.gn
index 3dd2dbc738bfb7e8f3bdf9487a1bc21cb3e8183a..b88b7a49148b17ca450839d97cae033b9554febf 100644
--- a/chrome/browser/password_manager/android/BUILD.gn
+++ b/chrome/browser/password_manager/android/BUILD.gn
@@ -190,11 +190,13 @@ android_library("java") {
   ]
 
   resources_package = "org.chromium.chrome.browser.password_manager"
+  import("//brave/browser/password_manager/android/java_sources.gni") sources += brave_browser_password_manager_java_sources deps += brave_browser_password_manager_deps
 }
 
 generate_jni("jni_headers") {
   visibility = [
     ":*",
+    "//brave/browser:browser_impl",
     "//chrome/browser",
   ]
   sources = [
@@ -210,6 +212,7 @@ generate_jni("jni_headers") {
     "java/src/org/chromium/chrome/browser/password_manager/PasswordStoreCredential.java",
     "java/src/org/chromium/chrome/browser/password_manager/PasswordSyncControllerDelegateBridgeImpl.java",
   ]
+  import("//brave/browser/password_manager/android/java_sources.gni") sources += brave_browser_password_manager_jni_headers
 }
 
 android_library("utils_java") {
