diff --git a/chrome/browser/ui/views/toolbar/app_menu.h b/chrome/browser/ui/views/toolbar/app_menu.h
index ecb3acb155e0c15802c396a870e52608ebf6a341..8a02355e43e9fe639527d9408542739a45f76495 100644
--- a/chrome/browser/ui/views/toolbar/app_menu.h
+++ b/chrome/browser/ui/views/toolbar/app_menu.h
@@ -32,7 +32,7 @@ class MenuRunner;
 }
 
 // AppMenu adapts the AppMenuModel to view's menu related classes.
-class AppMenu final : public views::MenuDelegate,
+class AppMenu : public views::MenuDelegate,
                       public bookmarks::BaseBookmarkModelObserver,
                       public GlobalErrorObserver {
  public:
