diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index 9cbaf6dfd5e8fb4ec9f09480208260b410766693..aa292d4445430b77768f7f91f9565e7d19136167 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -319,7 +319,9 @@ cr.define('settings', function() {
 
       // TODO(tommycli): Find a way to refactor these repetitive category
       // routes.
+      // <if expr="_google_chrome">
       r.SITE_SETTINGS_ADS = r.SITE_SETTINGS.createChild('ads');
+      // </if>
       r.SITE_SETTINGS_AUTOMATIC_DOWNLOADS =
           r.SITE_SETTINGS.createChild('automaticDownloads');
       r.SITE_SETTINGS_BACKGROUND_SYNC =
@@ -379,7 +381,9 @@ cr.define('settings', function() {
       }
 
       r.PRINTING = r.ADVANCED.createSection('/printing', 'printing');
+      // <if expr="_google_chrome">
       r.CLOUD_PRINTERS = r.PRINTING.createChild('/cloudPrinters');
+      // </if>
 
       r.ACCESSIBILITY = r.ADVANCED.createSection('/accessibility', 'a11y');
 
