diff --git a/chrome/browser/ui/views/page_info/page_info_bubble_view.h b/chrome/browser/ui/views/page_info/page_info_bubble_view.h
index 54693cdc062a17f9c3f4e7254f01ba15781c52eb..b19302476b8f9f5942f686d32ad19d7bec5c0b79 100644
--- a/chrome/browser/ui/views/page_info/page_info_bubble_view.h
+++ b/chrome/browser/ui/views/page_info/page_info_bubble_view.h
@@ -54,6 +54,7 @@ class PageInfoBubbleView : public PageInfoBubbleViewBase,
   PageInfo* presenter_for_testing() { return presenter_.get(); }
 
  private:
+  friend class BravePageInfoBubbleView;
   PageInfoBubbleView(
       views::BubbleAnchor anchor,
       const gfx::Rect& anchor_rect,
@@ -73,7 +74,7 @@ class PageInfoBubbleView : public PageInfoBubbleViewBase,
   void WebContentsDestroyed() override;
   void ChildPreferredSizeChanged(views::View* child) override;
 
-  void AnnouncePageOpened(std::u16string announcement);
+  virtual void AnnouncePageOpened(std::u16string announcement);
 
   // The presenter that controls the Page Info UI.
   std::unique_ptr<PageInfo> presenter_;
