diff --git a/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelDefinitions.java b/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelDefinitions.java
index d5b4877e96934b06e80bd5d120868197c39e5bdf..6615dacd9592f3cf780b5363736220f93651cad8 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelDefinitions.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelDefinitions.java
@@ -55,6 +55,7 @@ public class ChannelDefinitions {
      * more detailed instructions.
      */
     @StringDef({ChannelId.BROWSER, ChannelId.DOWNLOADS, ChannelId.INCOGNITO, ChannelId.MEDIA,
+            BraveChannelDefinitions.ChannelId.BRAVE_ADS,
             ChannelId.SCREEN_CAPTURE, ChannelId.CONTENT_SUGGESTIONS, ChannelId.WEBAPP_ACTIONS,
             ChannelId.SITES, ChannelId.SHARING, ChannelId.UPDATES, ChannelId.COMPLETED_DOWNLOADS,
             ChannelId.PERMISSION_REQUESTS, ChannelId.PERMISSION_REQUESTS_HIGH})
@@ -78,6 +79,7 @@ public class ChannelDefinitions {
     }
 
     @StringDef({
+            BraveChannelDefinitions.ChannelGroupId.BRAVE_ADS,
             ChannelGroupId.GENERAL, ChannelGroupId.SITES
     })
     @Retention(RetentionPolicy.SOURCE)
@@ -108,6 +110,7 @@ public class ChannelDefinitions {
             Map<String, PredefinedChannel> map = new HashMap<>();
             Set<String> startup = new HashSet<>();
 
+            BraveChannelDefinitions.addBraveChannels(map, startup);
             map.put(ChannelId.BROWSER,
                     new PredefinedChannel(ChannelId.BROWSER, R.string.notification_category_browser,
                             NotificationManager.IMPORTANCE_LOW, ChannelGroupId.GENERAL));
@@ -208,6 +211,7 @@ public class ChannelDefinitions {
         static final Map<String, PredefinedChannelGroup> MAP;
         static {
             Map<String, PredefinedChannelGroup> map = new HashMap<>();
+            BraveChannelDefinitions.addBraveChannelGroups(map);
             map.put(ChannelGroupId.GENERAL,
                     new PredefinedChannelGroup(
                             ChannelGroupId.GENERAL, R.string.notification_category_group_general));
