diff --git a/chrome/browser/ui/views/side_panel/side_panel_coordinator.h b/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
index 507522f5d38ac34dfa6feadca7a7cc6bd09885e6..26d73d747105338bb0f2ce2c96678f4d9a1e954e 100644
--- a/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
+++ b/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
@@ -37,7 +37,7 @@ class View;
 // Existence and value of registries' active_entry() determines which entry is
 // visible for a given tab where the order of precedence is contextual
 // registry's active_entry() then global registry's.
-class SidePanelCoordinator final : public SidePanelUIBase,
+class SidePanelCoordinator : public SidePanelUIBase,
                                    public views::ViewObserver {
  public:
   explicit SidePanelCoordinator(BrowserView* browser_view);
