diff --git a/chrome/browser/ui/dialogs/outdated_upgrade_bubble.cc b/chrome/browser/ui/dialogs/outdated_upgrade_bubble.cc
index 00fde67c1f83e10cca9dabec89871d50122b5a91..2eca2fd7c8807916664fdfc303371552b578eea6 100644
--- a/chrome/browser/ui/dialogs/outdated_upgrade_bubble.cc
+++ b/chrome/browser/ui/dialogs/outdated_upgrade_bubble.cc
@@ -35,6 +35,8 @@ namespace {
 // we redirect to the safetyCheck page that interacts with these. On other
 // platforms it may be possible to download an updated browser via a site.
 const char* kUpdateBrowserRedirectUrl =
+    kDownloadBraveUrl;
+#if 0
 #if BUILDFLAG(IS_CHROMEOS_LACROS)
     // The URL to be used to update Lacros-Chrome when auto-update failed
     // for too long.
@@ -44,6 +46,7 @@ const char* kUpdateBrowserRedirectUrl =
     // too long.
     "https://www.google.com/chrome";
 #endif
+#endif
 
 bool g_upgrade_bubble_is_showing = false;
 
