diff --git a/chrome/browser/resources/bookmarks/toolbar.js b/chrome/browser/resources/bookmarks/toolbar.js
index 0e5cdde9cf5e0fe520147f4cf4c08b03305ec257..fb7b010b6c8d727b993adf67f2b962ebb2f24909 100644
--- a/chrome/browser/resources/bookmarks/toolbar.js
+++ b/chrome/browser/resources/bookmarks/toolbar.js
@@ -8,7 +8,7 @@ import 'chrome://resources/cr_elements/icons.m.js';
 import './shared_style.js';
 import './strings.m.js';
 
-import {CrToolbarElement} from 'chrome://resources/cr_elements/cr_toolbar/cr_toolbar.js';
+import 'chrome://brave-resources/br_elements/br_toolbar/br_toolbar.m.js';
 import {CrToolbarSearchFieldElement} from 'chrome://resources/cr_elements/cr_toolbar/cr_toolbar_search_field.js';
 import {assert} from 'chrome://resources/js/assert.m.js';
 import {StoreObserver} from 'chrome://resources/js/cr/ui/store.m.js';
