diff --git a/chrome/browser/sharing_hub/sharing_hub_model.cc b/chrome/browser/sharing_hub/sharing_hub_model.cc
index 1ddf2388a1950b980e25cae8a5b3efdcdc8aa1bf..fe698b385a735eeb18a4d0ab89ce128d75bc88b7 100644
--- a/chrome/browser/sharing_hub/sharing_hub_model.cc
+++ b/chrome/browser/sharing_hub/sharing_hub_model.cc
@@ -97,7 +97,7 @@ void SharingHubModel::PopulateFirstPartyActions() {
   DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
 
   first_party_action_list_.emplace_back(
-      IDC_COPY_URL, l10n_util::GetStringUTF16(IDS_SHARING_HUB_COPY_LINK_LABEL),
+      IDC_COPY_URL, l10n_util::GetStringUTF16(MaybeAddCopyCleanLinkItem(context_, &first_party_action_list_)),
       &(features::IsRoundedIconsEnabled() ? vector_icons::kContentCopyIcon
                                           : kCopyOldIcon),
       "SharingHubDesktop.CopyURLSelected", IDS_LINK_COPIED);
