diff --git a/chrome/installer/mini_installer/BUILD.gn b/chrome/installer/mini_installer/BUILD.gn
index ec629333b0062239ff2dec63c1a5bf79738de301..a6c74a6eecbb1b14302e5db1d42c8d594ce09032 100644
--- a/chrome/installer/mini_installer/BUILD.gn
+++ b/chrome/installer/mini_installer/BUILD.gn
@@ -182,6 +182,21 @@ template("generate_mini_installer") {
       "//third_party/icu:icudata",
       chrome_dll_target,
     ]
+    if (brave_chromium_build) {
+      inputs -= [
+        "$root_out_dir/chrome.exe",
+      ]
+      inputs += [
+        "$root_out_dir/brave.exe",
+        "$root_out_dir/brave_resources.pak",
+        "$root_out_dir/brave_100_percent.pak",
+        "$root_out_dir/brave_200_percent.pak",
+      ]
+      deps += [
+        "//brave:packed_resources",
+        "//brave/build/win:copy_exe",
+      ]
+    }
 
     if (enable_hidpi) {
       args += [ "--enable_hidpi=1" ]
