diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index af96779d25b87d1f7642ea23a42498e85707f0b5..7e3a823a20566b785b35e8d3eebb579af11559fe 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -233,6 +233,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
