diff --git a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h
index a0d15aa1a0777306b1f02d8692bafce05c4ad12f..aa544007f56b5f29107b4f8b98a2ebfdba363b1d 100644
--- a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h
+++ b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h
@@ -133,7 +133,7 @@ class TabDragController : public views::WidgetObserver,
   // and is only non-empty if the original selection isn't the same as the
   // dragging set. Returns Liveness::DELETED if `this` was deleted during this
   // call, and Liveness::ALIVE if `this` still exists.
-  [[nodiscard]] Liveness Init(TabDragContext* source_context,
+  [[nodiscard]] virtual Liveness Init(TabDragContext* source_context,
                               TabSlotView* source_view,
                               const std::vector<TabSlotView*>& dragging_views,
                               const gfx::Point& offset_from_source_view,
