diff --git a/chrome/common/webui_url_constants.h b/chrome/common/webui_url_constants.h
index bc26ce3751cbcd3aa18897353dd1bd58120f0e0c..0877f2eaab913feec9ca8fdae7293c4286462b9e 100644
--- a/chrome/common/webui_url_constants.h
+++ b/chrome/common/webui_url_constants.h
@@ -566,7 +566,7 @@ inline constexpr char kIdentityDocsSubPage[] = "identityDocs";
 inline constexpr char kTravelSubPage[] = "travel";
 inline constexpr char kShoppingSubPage[] = "shopping";
 inline constexpr char kPeopleSubPage[] = "people";
-inline constexpr char kPerformanceSubPage[] = "performance";
+inline constexpr char kPerformanceSubPage[] = "system";
 inline constexpr char kPrivacyGuideSubPage[] = "privacy/guide";
 inline constexpr char kPrivacySandboxMeasurementSubpage[] =
     "adPrivacy/measurement";
