diff --git a/chrome/browser/resources/tools/optimize_webui.py b/chrome/browser/resources/tools/optimize_webui.py
index f41a89bf816fe8cdba73d3700399ce55e94a7e99..6ee5dee93a04bf1bb7a26346d47b3cf8a9c9ba40 100755
--- a/chrome/browser/resources/tools/optimize_webui.py
+++ b/chrome/browser/resources/tools/optimize_webui.py
@@ -26,6 +26,7 @@ import node_modules
 
 # These files are already combined and minified.
 _BASE_EXCLUDES = []
+_BASE_EXCLUDES += [ 'chrome://brave-resources/fonts/muli.css', 'chrome://brave-resources/fonts/poppins.css' ]; _BR_RESOURCES_PATH = os.path.join(_SRC_PATH, 'brave', 'ui', 'webui', 'resources','').replace('\\', '/')
 for excluded_file in [
   'resources/polymer/v1_0/web-animations-js/web-animations-next-lite.min.js',
   'resources/css/roboto.css',
@@ -50,6 +51,8 @@ def _request_list_path(out_path, host_url):
   return os.path.join(out_path, host + '_requestlist.txt')
 
 def _get_dep_path(dep, host_url, in_path):
+  for brave_host_url in ['chrome://brave-resources/', '//brave-resources/']:
+    if dep.startswith(brave_host_url): return dep.replace(brave_host_url, os.path.relpath(_BR_RESOURCES_PATH, _CWD))
   if dep.startswith(host_url):
     return dep.replace(host_url, os.path.relpath(in_path, _CWD))
   elif not (dep.startswith('chrome://') or dep.startswith('//')):
