diff --git a/chrome/browser/ui/page_info/page_info_ui.cc b/chrome/browser/ui/page_info/page_info_ui.cc
index 38f48905784266a85ccb0f2decfadbd9b6d089ea..73323f92e97b8fc39615b94938ad65dcab9fa01d 100644
--- a/chrome/browser/ui/page_info/page_info_ui.cc
+++ b/chrome/browser/ui/page_info/page_info_ui.cc
@@ -30,6 +30,7 @@
 #if defined(OS_ANDROID)
 #include "chrome/browser/android/android_theme_resources.h"
 #else
+#include "brave/app/vector_icons/vector_icons.h"
 #include "chrome/app/vector_icons/vector_icons.h"
 #include "components/prefs/pref_service.h"
 #include "media/base/media_switches.h"
@@ -616,6 +617,9 @@ const gfx::ImageSkia PageInfoUI::GetPermissionIcon(const PermissionInfo& info,
     case ContentSettingsType::USB_GUARD:
       icon = &vector_icons::kUsbIcon;
       break;
+    case ContentSettingsType::AUTOPLAY:
+      icon = &kAutoplayStatusIcon;
+      break;
     case ContentSettingsType::SERIAL_GUARD:
       icon = &vector_icons::kSerialPortIcon;
       break;
