diff --git a/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc b/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
index 389c6f8c6e66768cb20a960f11ba620c7516f44b..15519835488a0e406c2571d991171274b16df399 100644
--- a/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
+++ b/chrome/browser/ui/views/frame/layout/browser_view_tabbed_layout_impl.cc
@@ -764,7 +764,7 @@ BrowserViewTabbedLayoutImpl::CalculateProposedLayout(
   }
 
   BrowserLayoutParams params = layout_data_->revised_params;
-  bool needs_exclusion = true;
+  bool needs_exclusion = !delegate().ShouldShowVerticalTabs();
   const HorizontalLayout& horizontal_layout = layout_data_->horizontal_layout;
 
   if (IsParentedTo(views().organizer_tray, views().browser_view)) {
