diff --git a/chrome/browser/ui/autofill/autofill_context_menu_manager.cc b/chrome/browser/ui/autofill/autofill_context_menu_manager.cc
index fbf05e26108fdceef9629ddba437c1570fbb523f..d2f22844e097ba17e309e027d6d7bf271a63e387 100644
--- a/chrome/browser/ui/autofill/autofill_context_menu_manager.cc
+++ b/chrome/browser/ui/autofill/autofill_context_menu_manager.cc
@@ -203,6 +203,7 @@ AutofillContextMenuManager::AutofillContextMenuManager(
 AutofillContextMenuManager::~AutofillContextMenuManager() = default;
 
 void AutofillContextMenuManager::AppendItems() {
+  [&]() -> void {
   if (params_.is_content_editable_for_autofill) {
     if (MaybeAddAtMemoryItem()) {
       menu_model_->AddSeparator(ui::NORMAL_SEPARATOR);
@@ -212,6 +213,8 @@ void AutofillContextMenuManager::AppendItems() {
 
   MaybeAddAutofillManualFallbackItems();
   MaybeAddAutofillFeedbackItem();
+  }();
+  RemoveAutofillFeedbackMenuItem(menu_model_);
 }
 
 bool AutofillContextMenuManager::IsCommandIdSupported(int command_id) {
