diff --git a/chrome/renderer/chrome_content_renderer_client.cc b/chrome/renderer/chrome_content_renderer_client.cc
index a91227b208c16cfc1d67107752727d9889301878..54b344add145e3bc5687053e93437ce006a3ceb0 100644
--- a/chrome/renderer/chrome_content_renderer_client.cc
+++ b/chrome/renderer/chrome_content_renderer_client.cc
@@ -576,7 +576,7 @@ void ChromeContentRendererClient::RenderFrameCreated(
       ChromeExtensionsRendererClient::GetInstance()->extension_dispatcher());
 #endif
   content_settings::ContentSettingsAgentImpl* content_settings =
-      new content_settings::ContentSettingsAgentImpl(
+      new content_settings::BraveContentSettingsAgentImpl(
           render_frame, should_allow_for_content_settings,
           std::move(content_settings_delegate));
   if (chrome_observer_.get()) {
@@ -765,6 +765,7 @@ void ChromeContentRendererClient::RenderFrameCreated(
                                 render_frame));
   }
 #endif
+  BRAVE_RENDER_FRAME_CREATED
 }
 
 void ChromeContentRendererClient::WebViewCreated(
