diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 4b6e4714432100f351fe3885c62749bf086aa4d7..05694d0d875f059aef3f72dfd24dca170f4a5d1d 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -671,6 +671,7 @@ static_library("ui") {
     "//ui/webui",
     "//v8:v8_version",
   ]
+  deps+= [ "//brave/browser/ui" ]
 
   # TODO(crbug/925153): Remove this circular dependency.
   allow_circular_includes_from = [
@@ -3856,10 +3857,13 @@ static_library("ui") {
       ]
       deps += [ "//google_update" ]
     } else {
-      sources += [
-        "webui/help/version_updater_basic.cc",
-        "webui/help/version_updater_basic.h",
-      ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [
+          "webui/help/version_updater_basic.cc",
+          "webui/help/version_updater_basic.h",
+        ]
+      }
     }
   } else {  # 'OS!="win"
     # The Cast Linux build sets toolkit_views to false, but many files in this
