diff --git a/chrome/browser/resources/tools/optimize_webui.gni b/chrome/browser/resources/tools/optimize_webui.gni
index fde85a57d4226f6446c832edb6a9abdee81b9beb..a48a6f3c6d145088fadbee15ecaedda48b161d02 100644
--- a/chrome/browser/resources/tools/optimize_webui.gni
+++ b/chrome/browser/resources/tools/optimize_webui.gni
@@ -69,6 +69,7 @@ template("optimize_webui") {
       "//resources/polymer/v3_0/|$polymer_path",
       "//resources/|$resources_path",
     ]
+    import("//brave/browser/resources/tools/brave_optimize_webui.gni") deps += brave_webui_deps external_paths += brave_webui_external_paths
 
     if (defined(invoker.external_paths)) {
       external_paths += invoker.external_paths
