diff --git a/chrome/browser/extensions/updater/extension_updater.cc b/chrome/browser/extensions/updater/extension_updater.cc
index 05686804f06f079eb4fee3c938ddafbe7e053a60..e7ebca3139c87f841215f79048c9f405220e40b3 100644
--- a/chrome/browser/extensions/updater/extension_updater.cc
+++ b/chrome/browser/extensions/updater/extension_updater.cc
@@ -232,6 +232,7 @@ void ExtensionUpdater::Start() {
   alive_ = true;
   // Check soon, and set up the first delayed check.
   if (!g_skip_scheduled_checks_for_tests) {
+    BRAVE_EXTENSION_UPDATER_SCHEDULED_CHECK_GUARD
     CheckSoon();
     ScheduleNextCheck();
   }
@@ -272,6 +273,7 @@ void ExtensionUpdater::NextCheck() {
   if (!alive_) {
     return;
   }
+  BRAVE_EXTENSION_UPDATER_SCHEDULED_CHECK_GUARD
   CheckNow(CheckParams());
   ScheduleNextCheck();
 }
