diff --git a/chrome/browser/resources/tools/optimize_webui.py b/chrome/browser/resources/tools/optimize_webui.py
index ddbdbf34e886c52136bf9cc8e344da2a858589ac..84fa404d251d14c1e2231e0ca5b2fb22ec486542 100755
--- a/chrome/browser/resources/tools/optimize_webui.py
+++ b/chrome/browser/resources/tools/optimize_webui.py
@@ -26,6 +26,7 @@ import node_modules
 
 # These files are already combined and minified.
 _BASE_EXCLUDES = []
+_BASE_EXCLUDES += [ 'chrome://brave-resources/fonts/muli.css', 'chrome://brave-resources/fonts/poppins.css' ]; _BR_RESOURCES_PATH = os.path.join(_SRC_PATH, 'brave', 'ui', 'webui', 'resources','').replace('\\', '/')
 for excluded_file in [
   'resources/polymer/v1_0/web-animations-js/web-animations-next-lite.min.js',
   'resources/css/roboto.css',
@@ -52,6 +53,8 @@ def _request_list_path(out_path, target_name):
   return os.path.join(out_path, target_name + '_requestlist.txt')
 
 def _get_dep_path(dep, host_url, in_path):
+  for brave_host_url in ['chrome://brave-resources/', '//brave-resources/']:
+    if dep.startswith(brave_host_url): return dep.replace(brave_host_url, os.path.relpath(_BR_RESOURCES_PATH, _CWD))
   if dep.startswith(host_url):
     return dep.replace(host_url, os.path.relpath(in_path, _CWD))
   elif not (dep.startswith('chrome://') or dep.startswith('//')):
