diff --git a/chrome/browser/resources/password_manager/BUILD.gn b/chrome/browser/resources/password_manager/BUILD.gn
index beec57d57e1b84f16deda633f03f2687144ffde7..bf5f369fb8097fb74aed7f820d2a721e53f843b4 100644
--- a/chrome/browser/resources/password_manager/BUILD.gn
+++ b/chrome/browser/resources/password_manager/BUILD.gn
@@ -149,4 +149,5 @@ build_webui("build") {
               "$root_gen_dir/chrome/browser/resources/settings_shared/tsc",
               root_build_dir) ]
   }
+  import("//brave/browser/resources/password_manager/sources.gni") non_web_component_files += brave_password_manager_non_web_component_files exclude_ts_preprocess_files = brave_password_manager_ts_local_files preprocess_deps = brave_password_manager_preprocess_extra_deps
 }
