diff --git a/chrome/browser/resources/settings_shared/privacy_page/privacy_page_browser_proxy.ts b/chrome/browser/resources/settings_shared/privacy_page/privacy_page_browser_proxy.ts
index d45222679dd87714d8e06f98061d0682dc071810..bec508e159ecaac138660fc81eaea2368224ad63 100644
--- a/chrome/browser/resources/settings_shared/privacy_page/privacy_page_browser_proxy.ts
+++ b/chrome/browser/resources/settings_shared/privacy_page/privacy_page_browser_proxy.ts
@@ -14,14 +14,14 @@ export interface MetricsReporting {
 }
 
 export interface PrivacyPageBrowserProxy {
-  // <if expr="_google_chrome and not is_chromeos">
+  // <if expr="True and not is_chromeos">
   getMetricsReporting(): Promise<MetricsReporting>;
   setMetricsReportingEnabled(enabled: boolean): void;
   // </if>
 }
 
 export class PrivacyPageBrowserProxyImpl implements PrivacyPageBrowserProxy {
-  // <if expr="_google_chrome and not is_chromeos">
+  // <if expr="True and not is_chromeos">
   getMetricsReporting() {
     return sendWithPromise<MetricsReporting>('getMetricsReporting');
   }
