diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
index af1a722acc6784746bd21b0ddd4c954cda6292b8..84568ff5b2076df30059b13d104880481e04c4c3 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
@@ -300,6 +300,7 @@ base::string16 AvatarToolbarButton::GetAvatarTooltipText() const {
   if (IsIncognito())
     return l10n_util::GetStringUTF16(IDS_AVATAR_BUTTON_INCOGNITO_TOOLTIP);
 
+  GET_AVATAR_TOOLTIP_TEXT_
   if (profile_->IsGuestSession())
     return l10n_util::GetStringUTF16(IDS_GUEST_PROFILE_NAME);
 
