diff --git a/chrome/browser/resources/settings/site_settings/constants.js b/chrome/browser/resources/settings/site_settings/constants.js
index 1de16ab5890d6d1a85b6399bd69a259130dca7af..0db6eceae952849b5fdeeb91341e421f4cbd986e 100644
--- a/chrome/browser/resources/settings/site_settings/constants.js
+++ b/chrome/browser/resources/settings/site_settings/constants.js
@@ -42,6 +42,7 @@ export const ContentSettingsTypes = {
   AR: 'ar',
   VR: 'vr',
   WINDOW_PLACEMENT: 'window-placement',
+  AUTOPLAY: 'autoplay',
 };
 
 /**
