diff --git a/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc b/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
index 1e433d614887409989ebc7a66cfb7dfc2add2ac3..bfa013b5ef14cf8d660505785d1cdc62e3d65daf 100644
--- a/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
+++ b/chrome/browser/extensions/api/webrtc_logging_private/webrtc_logging_private_api.cc
@@ -90,6 +90,7 @@ content::RenderProcessHost* WebrtcLoggingPrivateFunction::RphFromRequest(
     const api::webrtc_logging_private::RequestInfo& request,
     const std::string& security_origin,
     std::string* error) {
+  BRAVE_DISABLE_RPH_FROM_REQUEST //feature disabled in Brave
   // There are 2 ways these API functions can get called.
   //
   //  1. From an allowlisted component extension on behalf of a page with the
