diff --git a/chrome/browser/resources/settings/privacy_page/personalization_options.ts b/chrome/browser/resources/settings/privacy_page/personalization_options.ts
index 7f6e6f996c236227d2506a65a248f2cd10501080..1212f9ddaa06f3f30282aea6ce1370e6086f8343 100644
--- a/chrome/browser/resources/settings/privacy_page/personalization_options.ts
+++ b/chrome/browser/resources/settings/privacy_page/personalization_options.ts
@@ -79,7 +79,7 @@ export class SettingsPersonalizationOptionsElement extends
 
       syncStatus: Object,
 
-      // <if expr="_google_chrome and not chromeos_ash">
+      // <if expr="not chromeos_ash">
       // TODO(dbeam): make a virtual.* pref namespace and set/get this normally
       // (but handled differently in C++).
       metricsReportingPref_: {
@@ -125,7 +125,7 @@ export class SettingsPersonalizationOptionsElement extends
   declare pageVisibility: PrivacyPageVisibility;
   declare syncStatus: SyncStatus;
 
-  // <if expr="_google_chrome and not chromeos_ash">
+  // <if expr="not chromeos_ash">
   declare private metricsReportingPref_:
       chrome.settingsPrivate.PrefObject<boolean>;
   declare private showRestart_: boolean;
@@ -165,7 +165,7 @@ export class SettingsPersonalizationOptionsElement extends
   override ready() {
     super.ready();
 
-    // <if expr="_google_chrome and not chromeos_ash">
+    // <if expr="not chromeos_ash">
     const setMetricsReportingPref = (metricsReporting: MetricsReporting) =>
         this.setMetricsReportingPref_(metricsReporting);
     this.addWebUiListener('metrics-reporting-change', setMetricsReportingPref);
@@ -180,9 +180,11 @@ export class SettingsPersonalizationOptionsElement extends
         this.setChromeSigninUserChoiceInfo_.bind(this));
     // </if>
 
+    // <if expr="_google_chrome">
     this.registerHelpBubble(
         ANONYMIZED_URL_COLLECTION_ID,
         this.$.urlCollectionToggle.getBubbleAnchor(), {anchorPaddingTop: 10});
+    // </if>
   }
 
   // <if expr="chromeos_ash">
@@ -203,7 +205,7 @@ export class SettingsPersonalizationOptionsElement extends
   }
   // </if>
 
-  // <if expr="_google_chrome and not chromeos_ash">
+  // <if expr="not chromeos_ash">
   private onMetricsReportingChange_() {
     const enabled = this.$.metricsReportingControl.checked;
     this.browserProxy_.setMetricsReportingEnabled(enabled);
