diff --git a/chrome/browser/ui/android/toolbar/BUILD.gn b/chrome/browser/ui/android/toolbar/BUILD.gn
index 2a26950646bd92df2349aa6d2678979ae514ed8b..a2baf3ee0eadbe8a6b75c8fada13454031d55b5e 100644
--- a/chrome/browser/ui/android/toolbar/BUILD.gn
+++ b/chrome/browser/ui/android/toolbar/BUILD.gn
@@ -226,6 +226,7 @@ android_library("core_java") {
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/HeightTransitionHandler.java",
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/TabStripTransitionCoordinator.java",
   ]
+  sources += brave_browser_ui_android_toolbar_sources
 
   deps = [
     ":adaptive_features_java",
@@ -446,6 +447,7 @@ android_resources("java_resources") {
     "//components/browser_ui/styles/android:java_resources",
     "//ui/android:ui_java_resources",
   ]
+  deps += [ "//brave/browser/ui/android/toolbar:java_resources" ]
 }
 
 robolectric_library("junit") {
