diff --git a/chrome/browser/extensions/extension_management.cc b/chrome/browser/extensions/extension_management.cc
index 026313eb9d389ca34f597d6e00b052e5532a427b..157c89eee67009edca1fb79a9cd6a3f5dcfb374b 100644
--- a/chrome/browser/extensions/extension_management.cc
+++ b/chrome/browser/extensions/extension_management.cc
@@ -1045,7 +1045,7 @@ ExtensionManagementFactory::BuildServiceInstanceForBrowserContext(
     content::BrowserContext* context) const {
   TRACE_EVENT0("browser,startup",
                "ExtensionManagementFactory::BuildServiceInstanceFor");
-  return std::make_unique<ExtensionManagement>(
+  return std::make_unique<BraveExtensionManagement>(
       Profile::FromBrowserContext(context));
 }
 
