diff --git a/chrome/browser/net/proxy_config_monitor.cc b/chrome/browser/net/proxy_config_monitor.cc
index c9281f7abfbdeaa4f0d9fe3c199b34fcb5acaf2c..4bd06d22bad8f82a1733eda4b20d290ba498bbe1 100644
--- a/chrome/browser/net/proxy_config_monitor.cc
+++ b/chrome/browser/net/proxy_config_monitor.cc
@@ -52,6 +52,7 @@ ProxyConfigMonitor::ProxyConfigMonitor(Profile* profile) {
             profile->GetPrefs(), g_browser_process->local_state());
   }
 
+  BRAVE_PROXY_CONFIG_MONITOR
   proxy_config_service_ = ProxyServiceFactory::CreateProxyConfigService(
       pref_proxy_config_tracker_.get(), profile);
 
