diff --git a/chrome/browser/importer/external_process_importer_client.h b/chrome/browser/importer/external_process_importer_client.h
index f9834aef0068441b3267c48eb0999214b72db743..1d0c97fde7de7dfa818d6cee951de8366d9e7d65 100644
--- a/chrome/browser/importer/external_process_importer_client.h
+++ b/chrome/browser/importer/external_process_importer_client.h
@@ -84,6 +84,7 @@ class ExternalProcessImporterClient
       const std::vector<ImporterAutofillFormDataEntry>&
           autofill_form_data_entry_group) override;
 
+  BRAVE_EXTERNAL_PROCESS_IMPORTER_CLIENT_H
  protected:
   ~ExternalProcessImporterClient() override;
 
