diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index e83595adafe6e8eab7ac1467d715e9cbed49ef8d..cd84c2fa41da8fa89976ecdab66f74161e4092b4 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -7756,6 +7756,7 @@ const FeatureEntry kFeatureEntries[] = {
     // "LoginCustomFlags" in tools/metrics/histograms/enums.xml. See "Flag
     // Histograms" in tools/metrics/histograms/README.md (run the
     // AboutFlagsHistogramTest unit test to verify this process).
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 };

 class FlagsStateSingleton : public flags_ui::FlagsState::Delegate {
