diff --git a/chrome/browser/ui/hid/BUILD.gn b/chrome/browser/ui/hid/BUILD.gn
index d989e7fdcebb4123f5cdff98c282f25405ae26c6..afdb83007a0609d6ccc6eb62b6c58bf4f07a6f43 100644
--- a/chrome/browser/ui/hid/BUILD.gn
+++ b/chrome/browser/ui/hid/BUILD.gn
@@ -29,6 +29,7 @@ source_set("hid") {
     "//services/device/public/cpp/hid",
     "//ui/base",
   ]
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_hid_deps
 }
 
 source_set("unit_tests") {
