diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
index 61d7c61c094faed011b513c25cd0061bef5ab87d..9fab1d7066f01e9945ea19acdd1dec38966d4deb 100644
--- a/chrome/browser/resources/settings/route.js
+++ b/chrome/browser/resources/settings/route.js
@@ -263,6 +263,9 @@ cr.define('settings', function() {

     r.SEARCH = r.BASIC.createSection('/search', 'search');
     r.SEARCH_ENGINES = r.SEARCH.createChild('/searchEngines');
+    r.SHIELDS = r.BASIC.createSection('/shields', 'braveShieldsDefaults');
+    r.SOCIAL_BLOCKING = r.BASIC.createSection('/socialBlocking', 'socialBlocking');
+    r.BRAVE_SYNC = r.BASIC.createSection('/braveSync', 'braveSync');
     // <if expr="chromeos">
     if (loadTimeData.valueExists('assistantEnabled') &&
         loadTimeData.getBoolean('assistantEnabled')) {
@@ -341,9 +344,12 @@ cr.define('settings', function() {

       // TODO(tommycli): Find a way to refactor these repetitive category
       // routes.
+      // <if expr="_google_chrome">
       r.SITE_SETTINGS_ADS = r.SITE_SETTINGS.createChild('ads');
+      // </if>
       r.SITE_SETTINGS_AUTOMATIC_DOWNLOADS =
           r.SITE_SETTINGS.createChild('automaticDownloads');
+      r.SITE_SETTINGS_AUTOPLAY = r.SITE_SETTINGS.createChild('autoplay');
       r.SITE_SETTINGS_BACKGROUND_SYNC =
           r.SITE_SETTINGS.createChild('backgroundSync');
       r.SITE_SETTINGS_CAMERA = r.SITE_SETTINGS.createChild('camera');
@@ -401,7 +407,9 @@ cr.define('settings', function() {
       }

       r.PRINTING = r.ADVANCED.createSection('/printing', 'printing');
+      // <if expr="_google_chrome">
       r.CLOUD_PRINTERS = r.PRINTING.createChild('/cloudPrinters');
+      // </if>
       // <if expr="chromeos">
       r.CUPS_PRINTERS = r.PRINTING.createChild('/cupsPrinters');
       // </if>
@@ -419,6 +427,9 @@ cr.define('settings', function() {
         r.TRIGGERED_RESET_DIALOG =
             r.ADVANCED.createChild('/triggeredResetProfileSettings');
         r.TRIGGERED_RESET_DIALOG.isNavigableDialog = true;
+        r.RESET_REWARDS_DIALOG =
+            r.ADVANCED.createChild('/resetRewards');
+        r.RESET_REWARDS_DIALOG.isNavigableDialog = true;
         // <if expr="_google_chrome and is_win">
         r.CHROME_CLEANUP = r.RESET.createChild('/cleanup');
         if (loadTimeData.getBoolean('showIncompatibleApplications')) {
