diff --git a/chrome/browser/resources/md_bookmarks/command_manager.js b/chrome/browser/resources/md_bookmarks/command_manager.js
index e573fdf24d801ce29a8c86ca8def62549585d236..f427666f9153e70d533bb09a2f3058607019909f 100644
--- a/chrome/browser/resources/md_bookmarks/command_manager.js
+++ b/chrome/browser/resources/md_bookmarks/command_manager.js
@@ -408,7 +408,7 @@ cr.define('bookmarks', function() {
           chrome.bookmarks.export();
           break;
         case Command.HELP_CENTER:
-          window.open('https://support.google.com/chrome/?p=bookmarks');
+          window.open('https://community.brave.com');
           break;
         default:
           assert(false);
