diff --git a/chrome/browser/ui/webui/print_preview/print_preview_ui.cc b/chrome/browser/ui/webui/print_preview/print_preview_ui.cc
index 82d2bf5da99493cd244dda010c10e789f9886856..b070177ea666661091a793d9465a867e5894e5f9 100644
--- a/chrome/browser/ui/webui/print_preview/print_preview_ui.cc
+++ b/chrome/browser/ui/webui/print_preview/print_preview_ui.cc
@@ -154,7 +154,7 @@ PrintPreviewRequestIdMap& GetPrintPreviewRequestIdMap() {
 
 // PrintPreviewUI IDMap used to avoid exposing raw pointer addresses to WebUI.
 // Only accessed on the UI thread.
-base::LazyInstance<base::IDMap<PrintPreviewUI*>>::DestructorAtExit
+base::LazyInstance<base::IDMap<mojom::PrintPreviewUI*>>::DestructorAtExit
     g_print_preview_ui_id_map = LAZY_INSTANCE_INITIALIZER;
 
 void AddPrintPreviewStrings(content::WebUIDataSource* source) {
