diff --git a/chrome/browser/ui/browser_actions.h b/chrome/browser/ui/browser_actions.h
index cbdff7aa3bf0724364ef4e1572dc1a5527a71161..371c1eaeb6b68f9e1fecc4051518d1ddda89448f 100644
--- a/chrome/browser/ui/browser_actions.h
+++ b/chrome/browser/ui/browser_actions.h
@@ -24,7 +24,7 @@ class BrowserActions {
   explicit BrowserActions(BrowserWindowInterface* bwi);
   BrowserActions(const BrowserActions&) = delete;
   BrowserActions& operator=(const BrowserActions&) = delete;
-  ~BrowserActions();
+  virtual ~BrowserActions();
 
   static std::u16string GetCleanTitleAndTooltipText(std::u16string string);
 
