diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
index 5d0274b0424ecfd527962c5116370a7593d10d15..14cc90b470bcd639cdb2f6f0dfef0cbef6db5c1b 100644
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -164,7 +164,7 @@ if (!is_android && !is_mac) {
 
       if (is_win) {
         sources += [
-          "app/chrome_exe.rc",
+          "//brave/app/brave_exe.rc",
           "app/chrome_exe_load_config_win.cc",
           "app/chrome_exe_main_win.cc",
           "app/chrome_watcher_client_win.cc",
@@ -238,6 +238,8 @@ if (!is_android && !is_mac) {
           "app/chrome_main.cc",
           "app/chrome_main_delegate.cc",
           "app/chrome_main_delegate.h",
+          "//brave/app/brave_main_delegate.cc",
+          "//brave/app/brave_main_delegate.h",
         ]
 
         deps += [
@@ -308,7 +310,7 @@ if (!is_android && !is_mac) {
     if (is_win) {
       output_name = "initialexe/chrome"
     } else {
-      output_name = "chrome"
+      output_name = "brave"
     }
 
     sources = []
@@ -357,6 +359,8 @@ if (is_win) {
       "app/chrome_main.cc",
       "app/chrome_main_delegate.cc",
       "app/chrome_main_delegate.h",
+      "//brave/app/brave_main_delegate.cc",
+      "//brave/app/brave_main_delegate.h",
     ]
 
     output_name = "chrome"
@@ -465,6 +469,8 @@ if (is_win) {
         "app/chrome_main.cc",
         "app/chrome_main_delegate.cc",
         "app/chrome_main_delegate.h",
+        "//brave/app/brave_main_delegate.cc",
+        "//brave/app/brave_main_delegate.h",
       ]
 
       configs += [ "//build/config/compiler:wexit_time_destructors" ]
@@ -534,7 +540,9 @@ if (is_win) {
 } else if (is_mac) {
   chrome_helper_name = chrome_product_full_name + " Helper"
   chrome_framework_name = chrome_product_full_name + " Framework"
-  chrome_framework_version = "A"
+  # TODO(bridiver) - not sure why this has to be commented out, but otherwise
+  # it puts the framework in the wrong place for chrome_exe_main_mac.cc
+  # chrome_framework_version = "A"
 
   group("chrome") {
     deps = [
@@ -649,7 +657,7 @@ if (is_win) {
     args =
         [
           "-b",
-          "${branding_path_component}_strings",
+          _strings_file,
           "-v",
           rebase_path(chrome_version_file, root_build_dir),
           "-g",
@@ -1150,6 +1158,8 @@ if (is_win) {
       "app/chrome_main.cc",
       "app/chrome_main_delegate.cc",
       "app/chrome_main_delegate.h",
+      "//brave/app/brave_main_delegate.cc",
+      "//brave/app/brave_main_delegate.h",
       "app/chrome_main_mac.h",
       "app/chrome_main_mac.mm",
     ]
@@ -1389,6 +1399,7 @@ if (is_win) {
 
 group("browser_dependencies") {
   public_deps = [
+    "//brave:browser_dependencies",
     "//chrome/browser",
     "//chrome/common",
     "//components/sync",
@@ -1426,6 +1437,7 @@ group("browser_dependencies") {
 
 group("child_dependencies") {
   public_deps = [
+    "//brave:child_dependencies",
     "//chrome/browser/devtools",
     "//chrome/child",
     "//chrome/common",
@@ -1447,7 +1459,7 @@ group("child_dependencies") {
 if (is_win) {
   process_version_rc_template("chrome_exe_version") {
     sources = [
-      "app/chrome_exe.ver",
+      "//brave/app/brave_exe.ver",
     ]
     output = "$target_gen_dir/chrome_exe_version.rc"
   }
@@ -1715,6 +1727,8 @@ if (is_android) {
       "app/android/chrome_main_delegate_android.h",
       "app/chrome_main_delegate.cc",
       "app/chrome_main_delegate.h",
+      "//brave/app/brave_main_delegate.cc",
+      "//brave/app/brave_main_delegate.h",
     ]
 
     include_dirs = [ android_ndk_include_dir ]
@@ -1770,7 +1784,7 @@ if (is_linux) {
       dump_syms_binary =
           get_label_info(dump_syms_label, "root_out_dir") + "/" + "dump_syms"
 
-      chrome_binary = "$root_out_dir/chrome"
+      chrome_binary = "$root_out_dir/brave"
       if (current_cpu == "x86") {
         # GYP used "ia32" so keep that naming for back-compat.
         symbol_file = "$root_out_dir/chrome.breakpad.ia32"
