diff --git a/chrome/browser/ui/webui/downloads/downloads_ui.cc b/chrome/browser/ui/webui/downloads/downloads_ui.cc
index a1ddff80f28035458be64285ee801ebf74a05db2..d77ed872c4b9f00f5738beb7b90244f0f5d48a29 100644
--- a/chrome/browser/ui/webui/downloads/downloads_ui.cc
+++ b/chrome/browser/ui/webui/downloads/downloads_ui.cc
@@ -62,6 +62,7 @@ content::WebUIDataSource* CreateDownloadsUIHTMLSource(Profile* profile) {
   content::WebUIDataSource* source =
       content::WebUIDataSource::Create(chrome::kChromeUIDownloadsHost);
 
+  BraveCustomizeDownloadsDataSource(source);
 #if BUILDFLAG(OPTIMIZE_WEBUI)
   webui::SetupBundledWebUIDataSource(source, "downloads.js",
                                      IDR_DOWNLOADS_DOWNLOADS_ROLLUP_JS,
