diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index c7970e581a9760eb1a077baf1bb9e177aafacb93..0644bf1edc5b542718a3f9e8d2a0087321795662 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2449,6 +2449,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
