diff --git a/chrome/browser/ui/views/overlay/video_overlay_window_views.cc b/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
index cab32e89d40c86fa99e9e2375b968d99c7859003..733ab05df64e4cdb31ac61592968bacbc9086706 100644
--- a/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
+++ b/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
@@ -288,7 +288,7 @@ std::unique_ptr<VideoOverlayWindowViews> VideoOverlayWindowViews::Create(
   // constructor. It's important that the constructor be private, because it
   // doesn't initialize the object fully.
   auto overlay_window =
-      base::WrapUnique(new VideoOverlayWindowViews(controller));
+      base::WrapUnique(new BraveVideoOverlayWindowViews(controller));
 
   overlay_window->CalculateAndUpdateWindowBounds();
   overlay_window->SetUpViews();
@@ -745,6 +745,7 @@ void VideoOverlayWindowViews::UpdateMaxSize(const gfx::Rect& work_area) {
 
   max_size_ = new_max_size;
 
+  BRAVE_UPDATE_MAX_SIZE
   if (!native_widget())
     return;
 
