diff --git a/chrome/browser/ui/android/appmenu/internal/BUILD.gn b/chrome/browser/ui/android/appmenu/internal/BUILD.gn
index dcd1283d2b71eac80b12e788e7d186754b2687d6..8d51eab123bf094e88731fb0db10a41d5ff7212f 100644
--- a/chrome/browser/ui/android/appmenu/internal/BUILD.gn
+++ b/chrome/browser/ui/android/appmenu/internal/BUILD.gn
@@ -22,6 +22,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/ui/appmenu/AppMenuItemViewBinder.java",
     "java/src/org/chromium/chrome/browser/ui/appmenu/DividerLineMenuItemViewBinder.java",
   ]
+  sources += brave_browser_ui_android_appmenu_internal_sources
 
   deps = [
     ":java_resources",
