diff --git a/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkManagerOpenerImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkManagerOpenerImpl.java
index e3c3f6fab0d0f1128dcd7399a8c21d7d5cfc72d4..71b5d3e5b2c45ebb7f4fe2cc1290a6300fb78e9a 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkManagerOpenerImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkManagerOpenerImpl.java
@@ -102,7 +102,7 @@ public class BookmarkManagerOpenerImpl implements BookmarkManagerOpener {
         Intent intent =
                 new Intent(
                         activity == null ? ContextUtils.getApplicationContext() : activity,
-                        BookmarkActivity.class);
+                        org.chromium.chrome.browser.app.bookmarks.BraveBookmarkActivity.class);
         ProfileIntentUtils.addProfileToIntent(profile, intent);
         intent.setData(Uri.parse(url));
         if (activity != null) {
