diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 8381e254f635a1d869373b408b20ed5e80fd9054..4917adf3c1318df0f7f7b080cf85b4c9b2892751 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2519,6 +2519,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
