diff --git a/chrome/chrome_repack_locales.gni b/chrome/chrome_repack_locales.gni
index 0ec0aa2c8efb359f2f59e0316524d751381860e6..6b6a9252f693d34465bb5b8a0fbaab153f1c1560 100644
--- a/chrome/chrome_repack_locales.gni
+++ b/chrome/chrome_repack_locales.gni
@@ -43,6 +43,7 @@ template("chrome_repack_locales") {
       "${root_gen_dir}/ui/strings/ax_strings_",
       "${root_gen_dir}/ui/strings/ui_strings_",
     ]
+    source_patterns += brave_locale_source_patterns
     if (!defined(deps)) {
       deps = []
     }
@@ -68,6 +69,7 @@ template("chrome_repack_locales") {
       "//ui/strings:ax_strings",
       "//ui/strings:ui_strings",
     ]
+    deps += brave_locale_deps
     if (defined(invoker.deps)) {
       deps += invoker.deps
     }
