diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
index bcefa9166612c1bfab1647926e569aed24f474a6..98b310f97a542ebd6a8425ea0af1e3644ba14a0a 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
@@ -1049,7 +1049,7 @@ public class ChromeTabbedActivity extends ChromeActivity<ChromeActivityComponent
                                     this, getModalDialogManager()));
             NotificationPermissionController.attach(
                     getWindowAndroid(), mNotificationPermissionController);
-            mNotificationPermissionController.requestPermissionIfNeeded(false /* contextual */);
+            if(false) mNotificationPermissionController.requestPermissionIfNeeded(false /* contextual */);
             if (BackPressManager.isEnabled()) initializeBackPressHandlers();
         }
     }
