diff --git a/chrome/browser/sessions/session_tab_helper_factory.cc b/chrome/browser/sessions/session_tab_helper_factory.cc
index 6ca3ae1db1495646ff6997ce293200c7c4cd3303..b2dad92b7ef2e32768ec60d0bb4c7c1d66707040 100644
--- a/chrome/browser/sessions/session_tab_helper_factory.cc
+++ b/chrome/browser/sessions/session_tab_helper_factory.cc
@@ -49,7 +49,7 @@ void CreateSessionServiceTabHelper(content::WebContents* contents) {
 #else
   sessions::SessionTabHelper::DelegateLookup lookup;
 #endif
-  sessions::SessionTabHelper::CreateForWebContents(contents, std::move(lookup));
+  brave::BraveSessionTabHelper::CreateForWebContents(contents, std::move(lookup));
 
 #if BUILDFLAG(ENABLE_EXTENSIONS)
   auto* observer =
