diff --git a/chrome/android/java_sources.gni b/chrome/android/java_sources.gni
index b2721e0d36082c07cb7cc26f958547381c3d450c..b4ebfabf402464adc3162f67790b2961245c5861 100644
--- a/chrome/android/java_sources.gni
+++ b/chrome/android/java_sources.gni
@@ -25,6 +25,7 @@ import("//components/offline_pages/buildflags/features.gni")
 import("//device/vr/buildflags/buildflags.gni")
 
 chrome_java_sources += public_autofill_assistant_java_sources
+chrome_java_sources += brave_java_sources
 
 if (enable_vr) {
   chrome_java_sources += [ "java/src/org/chromium/chrome/browser/component_updater/VrAssetsComponentInstaller.java" ]
