diff --git a/chrome/services/qrcode_generator/qrcode_generator_service_impl.cc b/chrome/services/qrcode_generator/qrcode_generator_service_impl.cc
index 1094d0e20ed97cc62b41782426dedaeced8f3fd1..4676cb65d55f45d70fce6457f01f163a417f0908 100644
--- a/chrome/services/qrcode_generator/qrcode_generator_service_impl.cc
+++ b/chrome/services/qrcode_generator/qrcode_generator_service_impl.cc
@@ -78,7 +78,7 @@ void QRCodeGeneratorServiceImpl::DrawDino(SkCanvas* canvas,
                                           const int dino_border_px,
                                           const SkPaint& paint_foreground,
                                           const SkPaint& paint_background) {
-  int dino_width_px = pixels_per_dino_tile * dino_image::kDinoHeight;
+  int dino_width_px = pixels_per_dino_tile * dino_image::kDinoWidth;
   int dino_height_px = pixels_per_dino_tile * dino_image::kDinoHeight;
 
   // If we request too big a dino, we'll clip. In practice the dino size
