diff --git a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java
index 4cef7ae03e8cc87239e1e29a64a86443620ead2e..c00cf2c290695db7e69491629b44444240890c1d 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java
@@ -132,7 +132,7 @@ public class MostVisitedTilesCoordinator implements ConfigurationChangedObserver
         mUserEducationHelper = new UserEducationHelper(mActivity, profile, handler);
 
         mContextMenuManager =
-                new ContextMenuManager(
+                new org.chromium.chrome.browser.native_page.BraveContextMenuManager(
                         suggestionsUiDelegate.getNavigationDelegate(),
                         touchEnabledDelegate,
                         mActivity::closeContextMenu,
