diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 780a8c8efc1c1c326d6bf2470514f276795595cf..054d4f3c3f671c0c41e99a8676c1c78b5adb5c49 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -235,6 +235,7 @@ android_resources("chrome_app_java_resources") {
     "//third_party/androidx:androidx_preference_preference_java",
     "//third_party/androidx:androidx_recyclerview_recyclerview_java",
   ]
+  deps += brave_chrome_app_java_resources_deps sources += brave_java_resources resource_overlay = true
 }
 
 if (enable_vr) {
@@ -548,6 +549,7 @@ android_library("chrome_java") {
   ]
 
   deps += feed_deps
+  deps += brave_chrome_java_deps
 
   srcjar_deps = [
     ":chrome_android_java_enums_srcjar",
@@ -579,6 +581,7 @@ android_library("chrome_java") {
     "//components/ui_metrics:ui_metrics_enums_java",
     "//net:effective_connection_type_java",
   ]
+  srcjar_deps += brave_chrome_java_srcjar_deps
 
   # From java_sources.gni.
   sources = chrome_java_sources + [ app_hooks_impl ]
@@ -702,6 +705,7 @@ group("jni_headers") {
     "//chrome/browser/webauthn/android:jni_headers",
     "//components/media_router/browser/android:jni_headers",
   ]
+  public_deps += ["//brave/build/android:jni_headers"]
 }
 
 # TODO(crbug.com/1112471): Get this to run cleanly under Python 3.
@@ -1881,6 +1885,7 @@ if (current_toolchain == default_toolchain) {
         ":${_variant}_locale_pak_assets",
         ":${_variant}_paks",
       ]
+      deps += [ "//brave:brave_pak_assets" ]
       if (_is_monochrome) {
         deps += [ "//android_webview:locale_pak_assets" ]
         if (webview_includes_weblayer && !_is_bundle_module) {
