diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPage.java b/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPage.java
index 34e76fe0c4a0ab8ed374cf4b97bf044b5f349d13..1f23f46a68af28aa8ee0ebb5b933394aaafd5cc0 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPage.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPage.java
@@ -856,7 +856,7 @@ public class NewTabPage
      *     NTP.
      */
     public static boolean isInSingleUrlBarMode(boolean isLff) {
-        return !isLff;
+        return false;
     }
 
     /**
