diff --git a/chrome/installer/util/BUILD.gn b/chrome/installer/util/BUILD.gn
index 44c9a8bcb541d6a60382d7f4f02378d9b0517767..87d224258df0e9132379f98ee44f3f482d5d5f53 100644
--- a/chrome/installer/util/BUILD.gn
+++ b/chrome/installer/util/BUILD.gn
@@ -224,9 +224,24 @@ action("generate_strings") {
     "$target_gen_dir/installer_util_strings.rc",
   ]
 
+  brand = "$branding_path_component"
+
+  if (brave_chromium_build) {
+    # When brave_strings.grd is modified, outputs should be re-generated.
+    # This is workaround. Need to change the way of using |brave_strings.grd|
+    # in |create_string_rc.py|.
+    sources = [
+      "//chrome/app/brave_strings.grd"
+    ]
+    if (!is_official_build) {
+      # TODO(shong): Remove this.
+      brand = brand + "-development"
+    }
+  }
+
   args = [
     "-b",
-    "$branding_path_component",
+    brand,
     "-i",
     rebase_path(grdfile, root_build_dir) + ":resources",
     "-n",
