diff --git a/chrome/browser/resources/tab_search/BUILD.gn b/chrome/browser/resources/tab_search/BUILD.gn
index 5559ef2d0038dd8d078105f703fe23c26602216c..3db83ee312d53a82c68e60b44475ab760401dd31 100644
--- a/chrome/browser/resources/tab_search/BUILD.gn
+++ b/chrome/browser/resources/tab_search/BUILD.gn
@@ -91,4 +91,5 @@ build_webui("build") {
       "tab_search.js",
     ]
   }
+  import("//brave/browser/resources/tab_search/sources.gni") extra_grdp_deps = brave_tab_search_extra_grdp_deps extra_grdp_files = brave_tab_search_extra_grdp_files
 }
