diff --git a/chrome/browser/ui/views/translate/translate_bubble_view_unittest.cc b/chrome/browser/ui/views/translate/translate_bubble_view_unittest.cc
index a09a995f04a20432d6670ea855c6cd095c5eede9..7f87c80817dac5a7d273d63d8c062f9c61a9da7d 100644
--- a/chrome/browser/ui/views/translate/translate_bubble_view_unittest.cc
+++ b/chrome/browser/ui/views/translate/translate_bubble_view_unittest.cc
@@ -187,6 +187,7 @@ class TranslateBubbleViewTest : public ChromeViewsTestBase {
 
  protected:
   void SetUp() override {
+    scoped_feature_list_.InitFromCommandLine("BraveEnableAutoTranslate", "");
     ChromeViewsTestBase::SetUp();
 
     // The bubble needs the parent as an anchor.
