diff --git a/chrome/browser/metrics/chrome_feature_list_creator.cc b/chrome/browser/metrics/chrome_feature_list_creator.cc
index 6ede500a448685cf9b0f44ac03d43afef1ce2516..ff307c039319f7e2312574aa48666b77b24abccb 100644
--- a/chrome/browser/metrics/chrome_feature_list_creator.cc
+++ b/chrome/browser/metrics/chrome_feature_list_creator.cc
@@ -334,5 +334,6 @@ void ChromeFeatureListCreator::SetupInitialPrefs() {
     local_state_->SetTime(variations::prefs::kVariationsSeedDate,
                           base::Time::Now());
   }
+  BRAVE_CHROME_FEATURE_LIST_CREATOR_SETUP_INITIAL_PREFS
 #endif  // !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_CHROMEOS)
 }
