diff --git a/chrome/common/extensions/BUILD.gn b/chrome/common/extensions/BUILD.gn
index 94ff938a848ebb1bbec7ebad7b4f176bf0c017b9..026134f91ae49dc96187088cf52d0c0184b6b680 100644
--- a/chrome/common/extensions/BUILD.gn
+++ b/chrome/common/extensions/BUILD.gn
@@ -118,6 +118,7 @@ source_set("extensions") {
 
     sources += [ "api/file_system_provider_capabilities/file_system_provider_capabilities_handler.cc" ]
   }
+  import("//brave/common/extensions/sources.gni") public += brave_chrome_common_extensions_public sources += brave_chrome_common_extensions_sources deps += brave_chrome_common_extensions_deps public_deps += brave_chrome_common_extensions_public_deps
 }
 
 # TODO(zturner): Enable this on Windows. See
