diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 073e3d5acd478f59a6bfb79d8bd0e022731870ad..4b97363ca2c3087b927d1262a452f962203eef75 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -1866,6 +1866,7 @@ jumbo_split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
