diff --git a/chrome/browser/upgrade_detector/upgrade_detector_impl.cc b/chrome/browser/upgrade_detector/upgrade_detector_impl.cc
index 2e9c2ad3e29ed620299dadb9cd56f37ac4cfe245..45a3f00be129b9513a5152f6d0dfa2aa0834aece 100644
--- a/chrome/browser/upgrade_detector/upgrade_detector_impl.cc
+++ b/chrome/browser/upgrade_detector/upgrade_detector_impl.cc
@@ -488,7 +488,7 @@ void UpgradeDetectorImpl::Init() {
   // On Windows, only enable upgrade notifications for Google Chrome builds.
   // Chromium does not use an auto-updater.
 #if !defined(OS_WIN) || BUILDFLAG(GOOGLE_CHROME_BRANDING) || \
-    BUILDFLAG(ENABLE_CHROMIUM_UPDATER)
+    BUILDFLAG(ENABLE_CHROMIUM_UPDATER) || defined(OFFICIAL_BUILD)
 
   // On macOS, only enable upgrade notifications if the updater (Keystone) is
   // present.
