diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index f3d4c0128ee3383d683b0e70cfbe9b9285aa33bf..c96c7bbd69efe42a39a89485287ea18183731327 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -72,6 +72,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
