diff --git a/chrome/browser/ui/webui/theme_source.cc b/chrome/browser/ui/webui/theme_source.cc
index 99a12c8ab525352e8c07cae9b12f38a318743339..9310d6beca1d036e24e15b67cbc75ce717837906 100644
--- a/chrome/browser/ui/webui/theme_source.cc
+++ b/chrome/browser/ui/webui/theme_source.cc
@@ -482,6 +482,7 @@ void ThemeSource::SendColorsCss(
 #else
   const auto* theme_service =
       ThemeServiceFactory::GetForProfile(profile_->GetOriginalProfile());
+  BRAVE_THEME_SOURCE_SEND_COLORS_CSS
   bool is_grayscale = theme_service->GetIsGrayscale();
   bool is_baseline = theme_service->GetIsBaseline();
 #endif
