diff --git a/chrome/browser/webauthn/chrome_authenticator_request_delegate.cc b/chrome/browser/webauthn/chrome_authenticator_request_delegate.cc
index bb6adf030b4d9802649f6144630433d7eb7f9989..1fb25bbdffb8403f134da0bd953ab19d4f8fd988 100644
--- a/chrome/browser/webauthn/chrome_authenticator_request_delegate.cc
+++ b/chrome/browser/webauthn/chrome_authenticator_request_delegate.cc
@@ -371,6 +371,7 @@ std::string TouchIdMetadataSecret(Profile* profile) {
 ChromeAuthenticatorRequestDelegate::TouchIdAuthenticatorConfig
 ChromeAuthenticatorRequestDelegate::TouchIdAuthenticatorConfigForProfile(
     Profile* profile) {
+  BRAVE_WEBAUTHN_KEYCHAIN_ACCESS_GROUP
   return TouchIdAuthenticatorConfig{kTouchIdKeychainAccessGroup,
                                     TouchIdMetadataSecret(profile)};
 }
