diff --git a/chrome/browser/actor/execution_engine.cc b/chrome/browser/actor/execution_engine.cc
index 3751824d9e47642b87252fdddb637bf1e531cbfc..ee8d507b2bc0c99de0cff397ef3bb8f01ff3c57b 100644
--- a/chrome/browser/actor/execution_engine.cc
+++ b/chrome/browser/actor/execution_engine.cc
@@ -1430,6 +1430,14 @@ void ExecutionEngine::SafetyChecksForNextAction() {
   content::RenderFrameHost* main_frame = web_contents.GetPrimaryMainFrame();
   const GURL url = GetEffectiveUrlForGating(
       main_frame->GetLastCommittedURL(), main_frame->GetLastCommittedOrigin());
+  BRAVE_MAY_ACT_ON_URL_INTERNAL(
+      base::BindOnce([](MayActOnUrlBlockReason block_reason) {
+        return BlockReasonToResultCode(block_reason,
+                                       /*for_navigation=*/false);
+      })
+          .Then(base::BindOnce(&ExecutionEngine::DidFinishAsyncSafetyChecks,
+                               GetActionSequenceWeakPtr(),
+                               main_frame->GetLastCommittedOrigin())))
   GetOriginGatingChecker().ComputeGatingDecision(
       std::make_unique<PageActionGatingContext>(web_contents.GetWeakPtr()),
       GateableEvent(origin_gating::PageActionEvent{.destination = url}),
@@ -1740,6 +1748,7 @@ const EnterprisePolicyChecker& ExecutionEngine::GetEnterprisePolicyChecker()
 void ExecutionEngine::IsAcceptableNavigationDestination(
     const GURL& url,
     DecisionCallbackWithReason callback) {
+  BRAVE_MAY_ACT_ON_URL_INTERNAL(std::move(callback))
   GetOriginGatingChecker().ComputeGatingDecision(
       std::make_unique<OriginGatingDecisionContext>(),
       GateableEvent(origin_gating::NavigationRequestEvent{.source = GURL(),
