diff --git a/chrome/browser/resources/extensions/BUILD.gn b/chrome/browser/resources/extensions/BUILD.gn
index 8e92d58854d55a27e53d4a2b93bb299149ff4c95..aa6d8ca5cd9ec78bdbd9d93e7cbc6da60c95206c 100644
--- a/chrome/browser/resources/extensions/BUILD.gn
+++ b/chrome/browser/resources/extensions/BUILD.gn
@@ -21,6 +21,7 @@ optimize_webui("build") {
 
   deps = [
     ":unpak",
+    "//brave/browser/resources/md_extensions:unpak",
   ]
 }
 
