diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 3e75521cfba3854cc6a317b933745ba1a9e3f7b2..4c3e1a680b8a915531a4ef07e9ece189eaa27056 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2575,6 +2575,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_win || is_mac || is_linux || is_chromeos_ash) {
     deps += [ "//chrome/browser/ui/webui/connectors_internals:mojo_bindings" ]
