diff --git a/chrome/browser/ui/views/side_panel/side_panel_coordinator.h b/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
index 87f628125449dd8a9f0aa833fb9499426f1740d1..9e99a6efaa9c53f4146818f43a64845dce36e406 100644
--- a/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
+++ b/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
@@ -36,7 +36,7 @@ class View;
 // Existence and value of registries' active_entry() determines which entry is
 // visible for a given tab where the order of precedence is contextual
 // registry's active_entry() then global registry's.
-class SidePanelCoordinator final : public SidePanelUIBase,
+class SidePanelCoordinator : public SidePanelUIBase,
                                    public views::ViewObserver {
  public:
   explicit SidePanelCoordinator(BrowserView* browser_view);
