diff --git a/chrome/test/base/testing_profile.cc b/chrome/test/base/testing_profile.cc
index bd517614172dd71629f330281b7c7d8ac7f7ada3..4e7b076866163ffa0e2d865f8af9d116a45a86db 100644
--- a/chrome/test/base/testing_profile.cc
+++ b/chrome/test/base/testing_profile.cc
@@ -418,6 +418,7 @@ void TestingProfile::Init() {
       EnsureBrowserContextKeyedServiceFactoriesBuilt();
 
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
+  BRAVE_INIT
   if (!IsOffTheRecord()) {
     SupervisedUserSettingsService* settings_service =
         SupervisedUserSettingsServiceFactory::GetForKey(key_.get());
