diff --git a/chrome/browser/resources/settings/site_settings/constants.js b/chrome/browser/resources/settings/site_settings/constants.js
index 0bc63a422d1d1ca468a1222dd699d5fa9bb65d26..4cfa27279c3b54c67769a2bc9e1c0613a25ab2a7 100644
--- a/chrome/browser/resources/settings/site_settings/constants.js
+++ b/chrome/browser/resources/settings/site_settings/constants.js
@@ -35,6 +35,7 @@ cr.define('settings', function() {
     ADS: 'ads',
     CLIPBOARD: 'clipboard',
     SENSORS: 'sensors',
+    AUTOPLAY: 'autoplay',
     PAYMENT_HANDLER: 'payment-handler',
     MIXEDSCRIPT: 'mixed-script',
     BLUETOOTH_SCANNING: 'bluetooth-scanning',
