diff --git a/chrome/android/chrome_public_apk_tmpl.gni b/chrome/android/chrome_public_apk_tmpl.gni
index df3d58c8bade42ce87ff75a3ded8ce014fe6d6a9..8b40bff381fe06b4f4e31ac40b5cb5c678ac0883 100644
--- a/chrome/android/chrome_public_apk_tmpl.gni
+++ b/chrome/android/chrome_public_apk_tmpl.gni
@@ -317,6 +317,7 @@ template("chrome_common_apk_or_module_tmpl") {
       "*ic_file_download_white*",  # Bottom edge seems misaligned.
       "*ic_lock.*",  # Bottom edge seems misaligned.
     ]
+    resource_exclusion_exceptions += brave_resources_exclusion_exceptions
 
     # Most of these, with the exception of resource_exclusion_exceptions,
     # are currently duplicated in system_webview_apk_tmpl.gni.
@@ -341,6 +342,7 @@ template("chrome_common_apk_or_module_tmpl") {
 
       proguard_enabled = true
       proguard_configs = [ "//chrome/android/proguard/main.flags" ]
+      proguard_configs += [ "//brave/android/java/proguard.flags" ]
       if (_is_monochrome) {
         proguard_configs +=
             [ "//android_webview/nonembedded/java/proguard.flags" ]
@@ -729,6 +731,7 @@ template("chrome_common_apk_or_module_tmpl") {
                                  "version_code",
                                  "version_name",
                                ])
+    _wireguard_target = "//brave/third_party/android_deps:com_wireguard_android_J__unpack_aar" deps += [ _wireguard_target] _libwireguard_dir = get_label_info("//brave/third_party/android_deps:com_wireguard_android($default_toolchain)","target_out_dir") + "/com_wireguard_android_java/jni" loadable_modules +=[ "$_libwireguard_dir/$android_app_abi/libwg-go.so" ]
   }
 }
 
