diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java
index fee684b6161ccfd976df74e478f34e390331a7a8..cb89ac13dc0cfd45f6397d312e44b2d9edc22eef 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/ManageSyncSettings.java
@@ -200,7 +200,7 @@ public class ManageSyncSettings extends ChromeBaseSettingsFragment
     private boolean mShouldUpdatePrefs;
 
     /** Maps {@link UserSelectableType} to the corresponding preference. */
-    private Map<Integer, ChromeSwitchPreference> mSyncTypeSwitchPreferencesMap = new HashMap<>();
+    protected Map<Integer, ChromeSwitchPreference> mSyncTypeSwitchPreferencesMap = new HashMap<>();
 
     private Preference mGoogleActivityControls;
     private Preference mSyncEncryption;
@@ -289,6 +289,7 @@ public class ManageSyncSettings extends ChromeBaseSettingsFragment
 
     @Override
     public boolean onPreferenceChange(Preference preference, Object newValue) {
+        if (false)  // Suppress the "addresses not encrypted" warning (not applicable to Brave's sync model).
         if (mSyncService.isUsingExplicitPassphrase()
                 && preference
                         .getKey()
@@ -538,7 +539,7 @@ public class ManageSyncSettings extends ChromeBaseSettingsFragment
      * this state.
      */
     private void updateSyncPreferences() {
-        @Nullable AccountInfo primaryAccount = getIdentityManager().getPrimaryAccountInfo();
+        @Nullable AccountInfo primaryAccount = new AccountInfo.Builder("", new org.chromium.google_apis.gaia.GaiaId("")).build();
         // May happen if account is removed from the device while this screen is shown.
         if (primaryAccount == null) {
             finishCurrentSettings();
