diff --git a/chrome/browser/android/monochrome_entry_point.cc b/chrome/browser/android/monochrome_entry_point.cc
index 72d85f599682f3caa6a753dec479abaa6eafb8f4..e4aea13dd408be0434645a28dfc057eb65e34c21 100644
--- a/chrome/browser/android/monochrome_entry_point.cc
+++ b/chrome/browser/android/monochrome_entry_point.cc
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "android_webview/lib/webview_jni_onload.h"
+//#include "android_webview/lib/webview_jni_onload.h"
 #include "base/android/base_jni_onload.h"
 #include "base/android/jni_android.h"
 #include "base/android/library_loader/library_loader_hooks.h"
@@ -20,9 +20,9 @@ namespace {
 
 bool NativeInit(base::android::LibraryProcessType library_process_type) {
   switch (library_process_type) {
-    case base::android::PROCESS_WEBVIEW:
-    case base::android::PROCESS_WEBVIEW_CHILD:
-      return android_webview::OnJNIOnLoadInit();
+    // case base::android::PROCESS_WEBVIEW:
+    // case base::android::PROCESS_WEBVIEW_CHILD:
+    //   return android_webview::OnJNIOnLoadInit();
     case base::android::PROCESS_BROWSER:
     case base::android::PROCESS_CHILD:
       return android::OnJNIOnLoadInit();
