diff --git a/chrome/browser/upgrade_detector/upgrade_detector_impl.cc b/chrome/browser/upgrade_detector/upgrade_detector_impl.cc
index 53a17dc30c8768068838e81da674d6d5d374b429..f06ff76650bd35be5e0050fdadf10727e6641388 100644
--- a/chrome/browser/upgrade_detector/upgrade_detector_impl.cc
+++ b/chrome/browser/upgrade_detector/upgrade_detector_impl.cc
@@ -480,7 +480,7 @@ void UpgradeDetectorImpl::Init() {
   // On Windows, only enable upgrade notifications for Google Chrome builds.
   // Chromium does not use an auto-updater.
 #if !defined(OS_WIN) || BUILDFLAG(GOOGLE_CHROME_BRANDING) || \
-    BUILDFLAG(ENABLE_CHROMIUM_UPDATER)
+    BUILDFLAG(ENABLE_CHROMIUM_UPDATER) || defined(OFFICIAL_BUILD)
 
   // On macOS, only enable upgrade notifications if the updater (Keystone) is
   // present.
