diff --git a/chrome/browser/extensions/api/identity/identity_get_auth_token_function.cc b/chrome/browser/extensions/api/identity/identity_get_auth_token_function.cc
index 358acdc3073e374dcc64da051b091e99d155f974..91be9a9d40605488ea53eeed28e70a0a1e97bc51 100644
--- a/chrome/browser/extensions/api/identity/identity_get_auth_token_function.cc
+++ b/chrome/browser/extensions/api/identity/identity_get_auth_token_function.cc
@@ -124,7 +124,7 @@ CoreAccountInfo GetSigninPrimaryAccount(Profile* profile) {
       signin::ConsentLevel::kSignin);
 }
 
-#if BUILDFLAG(ENABLE_DESKTOP_ANDROID_EXTENSIONS)
+#if BUILDFLAG(IS_ANDROID) && BUILDFLAG(ENABLE_EXTENSIONS_CORE)
 bool IsAccountInCookieJar(const signin::AccountsInCookieJarInfo& cookie_info,
                           const CoreAccountInfo& account_info) {
   if (!cookie_info.AreAccountsFresh()) {
@@ -156,7 +156,7 @@ class IdentityGetAuthTokenFunction::RefreshTokensLoadedWaiter
       identity_manager_observation_{this};
 };
 
-#if BUILDFLAG(ENABLE_DESKTOP_ANDROID_EXTENSIONS)
+#if BUILDFLAG(IS_ANDROID) && BUILDFLAG(ENABLE_EXTENSIONS_CORE)
 class IdentityGetAuthTokenFunction::AccountsInCookieUpdatedWaiter
     : public signin::IdentityManager::Observer {
  public:
@@ -328,6 +328,10 @@ ExtensionFunction::ResponseAction IdentityGetAuthTokenFunction::Run() {
   // From here on out, results must be returned asynchronously.
   StartAsyncRun();
 
+  if (!google_apis::IsGoogleChromeAPIKeyUsed()) {
+    StartMintTokenFlow(IdentityMintRequestQueue::MINT_TYPE_NONINTERACTIVE);
+    return RespondLater();
+  }
   base::OnceCallback next_step =
       base::BindOnce(&IdentityGetAuthTokenFunction::GetAuthTokenForAccount,
                      weak_ptr_factory_.GetWeakPtr(), gaia_id);
@@ -410,7 +414,7 @@ void IdentityGetAuthTokenFunction::GetAuthTokenForAccount(
   }
 }
 
-#if BUILDFLAG(ENABLE_DESKTOP_ANDROID_EXTENSIONS)
+#if BUILDFLAG(IS_ANDROID) && BUILDFLAG(ENABLE_EXTENSIONS_CORE)
 bool IdentityGetAuthTokenFunction::ShouldDelayRemoteConsent() {
   signin::IdentityManager* identity_manager =
       IdentityManagerFactory::GetForProfile(GetProfile());
@@ -549,12 +553,16 @@ void IdentityGetAuthTokenFunction::StartSigninFlow() {
 void IdentityGetAuthTokenFunction::StartMintTokenFlow(
     IdentityMintRequestQueue::MintType type) {
 #if !BUILDFLAG(IS_CHROMEOS)
+  if (google_apis::IsGoogleChromeAPIKeyUsed()) {
   // ChromeOS in kiosk mode may start the mint token flow without account.
   DCHECK(!token_key_.account_info.IsEmpty())
       << "token_key_.account_info is empty!";
   DCHECK(IdentityManagerFactory::GetForProfile(GetProfile())
              ->HasAccountWithRefreshToken(token_key_.account_info.account_id))
       << "No Refresh token!";
+  } else {
+    DCHECK(token_key_.account_info.IsEmpty());
+  }
 #endif
   TRACE_EVENT_BEGIN("identity", "MintTokenFlow",
                     IdentityMintRequestQueue::GetRequestTrack(this), "type",
@@ -964,7 +972,7 @@ void IdentityGetAuthTokenFunction::OnIdentityAPIShutdown() {
   token_key_account_access_token_fetcher_.reset();
   refresh_tokens_loaded_waiter_.reset();
   scoped_identity_manager_observation_.Reset();
-#if BUILDFLAG(ENABLE_DESKTOP_ANDROID_EXTENSIONS)
+#if BUILDFLAG(IS_ANDROID) && BUILDFLAG(ENABLE_EXTENSIONS_CORE)
   accounts_in_cookie_updated_waiter_.reset();
 #endif
   extensions::IdentityAPI::GetFactoryInstance()
@@ -1039,7 +1047,7 @@ void IdentityGetAuthTokenFunction::ShowExtensionLoginPrompt() {
 #endif
 
 void IdentityGetAuthTokenFunction::ShowRemoteConsentDialog() {
-#if BUILDFLAG(ENABLE_DESKTOP_ANDROID_EXTENSIONS)
+#if BUILDFLAG(IS_ANDROID) && BUILDFLAG(ENABLE_EXTENSIONS_CORE)
   // On Android, Gaia session cookies are reconciled asynchronously after
   // sign-in. Defer showing the remote consent dialog until cookies are
   // ready in the cookie jar to prevent loading a blank consent page.
@@ -1196,7 +1204,7 @@ void IdentityGetAuthTokenFunction::RefreshTokensLoadedWaiter::
   std::move(callback_).Run();
 }
 
-#if BUILDFLAG(ENABLE_DESKTOP_ANDROID_EXTENSIONS)
+#if BUILDFLAG(IS_ANDROID) && BUILDFLAG(ENABLE_EXTENSIONS_CORE)
 IdentityGetAuthTokenFunction::AccountsInCookieUpdatedWaiter::
     AccountsInCookieUpdatedWaiter(signin::IdentityManager& identity_manager,
                                   const CoreAccountInfo& account_info,
