diff --git a/chrome/browser/notifications/notification_platform_bridge_mac_utils.mm b/chrome/browser/notifications/notification_platform_bridge_mac_utils.mm
index 2d50db66e64d02ef80bcd85bcb69d1f12a1c8eb8..483272a4500b8689821d3e0d0f47afc1802dcb04 100644
--- a/chrome/browser/notifications/notification_platform_bridge_mac_utils.mm
+++ b/chrome/browser/notifications/notification_platform_bridge_mac_utils.mm
@@ -60,6 +60,7 @@ void DoProcessMacNotificationResponse(
 // call this method directly as SysInfo::OperatingSystemVersionNumbers might be
 // an expensive call. Instead use SupportsAlerts which caches this value.
 bool MacOSSupportsXPCAlertsImpl() {
+  BRAVE_SUPPORTSALERTSIMPL_
   int32_t major, minor, bugfix;
   base::SysInfo::OperatingSystemVersionNumbers(&major, &minor, &bugfix);
   // Allow alerts on all versions except 10.15.0, 10.15.1 & 10.15.2.
