diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index b149e25b18d668bf2bd7a783803eb1ea1836db4c..d1fdda4881cddf87573b93d144af15e66826a989 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -519,6 +519,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
           l10n_util::GetStringUTF16(IDS_SHORT_PRODUCT_OS_NAME)));
 #endif  // BUILDFLAG(IS_CHROMEOS)
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
