diff --git a/chrome/browser/ui/views/permissions/chooser_bubble_ui.cc b/chrome/browser/ui/views/permissions/chooser_bubble_ui.cc
index 2b40d1ad6a72ebf7d92bfc9fe9162b3b6db77d05..5b6343f905e51ce19235bf4c88115427abe0fe18 100644
--- a/chrome/browser/ui/views/permissions/chooser_bubble_ui.cc
+++ b/chrome/browser/ui/views/permissions/chooser_bubble_ui.cc
@@ -117,6 +117,7 @@ ChooserBubbleUiViewDelegate::ChooserBubbleUiViewDelegate(
           GetChooserAnchorConfiguration(browser).anchor,
           contents),
       fullscreen_blocker_(std::move(fullscreen_blocker)) {
+  [&]() -> void {
   // ------------------------------------
   // | Chooser bubble title             |
   // | -------------------------------- |
@@ -153,6 +154,8 @@ ChooserBubbleUiViewDelegate::ChooserBubbleUiViewDelegate(
   SetCloseCallback(
       base::BindOnce(&DeviceChooserContentView::Close,
                      base::Unretained(device_chooser_content_view_)));
+  }();
+  SetFootnoteView(device_chooser_content_view_->CreateFootnoteView(browser));
 }
 
 ChooserBubbleUiViewDelegate::~ChooserBubbleUiViewDelegate() = default;
