diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 991e192bb1d3962460b18e5521c2f86e66ac84c9..9a4db2ed4eca1abaabe1d3bdf5ca18c9906c089e 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2083,6 +2083,7 @@ jumbo_split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
