diff --git a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list_header.ts b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list_header.ts
index cb729d6b2cacda53d3954122e67a2f53148366dc..3d788d9a010d15ebc9fc51b2885f6762dbefabb9 100644
--- a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list_header.ts
+++ b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list_header.ts
@@ -97,6 +97,7 @@ export class PowerBookmarksListHeaderElement extends CrLitElement {
       label: loadTimeData.getString('sortReverseAlphabetically'),
       lowerLabel: loadTimeData.getString('sortReverseAlphabetically'),
     },
+    { sortOrder: SortOrder.kCustom, label: loadTimeData.getString('sortCustom'), lowerLabel: loadTimeData.getString('sortCustomLower'), },
   ];
   protected accessor webuiRoundedIconsEnabled_: boolean =
       loadTimeData.getBoolean('webuiRoundedIconsEnabled');
