diff --git a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabGridDialogMediator.java b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabGridDialogMediator.java
index 822b7857affdbed70bd5be8c7187a2748487cd2c..25df19ff5ab76db6223832335f662838aa085698 100644
--- a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabGridDialogMediator.java
+++ b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabGridDialogMediator.java
@@ -615,7 +615,7 @@ public class TabGridDialogMediator
                                 Set.of(TabListEditorItemSelectionId.createTabId(tabId)));
                     };
             mTabGridContextMenuCoordinator =
-                    new TabGridContextMenuCoordinator(
+                    new BraveTabGridContextMenuCoordinator(
                             activity,
                             tabBookmarkerSupplier,
                             profile,
