diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 8500f29d7a7108f947969355d7ef302b6f5708bd..437844e024c7ec6773ed48ec4d6c1660d89fc04d 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -411,6 +411,8 @@ test("browser_tests") {
     "//third_party/blink/public:buildflags",
   ]
 
+  deps += [ "//brave/test:brave_browser_tests_deps", ]
+
   data_deps = [
     "//testing/buildbot/filters:browser_tests_filters",
     "//tools/media_engagement_preload:generator",
