diff --git a/chrome/browser/ui/views/toolbar/browser_actions_container.h b/chrome/browser/ui/views/toolbar/browser_actions_container.h
index 82b54ed855ff670d558e55cd686106ace88128eb..a5e88d9715baa3abd58bec57d3fb57dc3ae82e8f 100644
--- a/chrome/browser/ui/views/toolbar/browser_actions_container.h
+++ b/chrome/browser/ui/views/toolbar/browser_actions_container.h
@@ -262,6 +262,7 @@ class BrowserActionsContainer : public views::View,
   static views::FlexRule GetFlexRule();
 
  protected:
+  friend class BraveToolbarView;
   // Overridden from views::View:
   void ViewHierarchyChanged(
       const views::ViewHierarchyChangedDetails& details) override;
