diff --git a/chrome/browser/ui/views/profiles/incognito_menu_view.h b/chrome/browser/ui/views/profiles/incognito_menu_view.h
index c4233e1db4e5a343e1a837e095fc0ada547f4517..ae1e784f639b7e02b49d3638b6f64fbf411f680a 100644
--- a/chrome/browser/ui/views/profiles/incognito_menu_view.h
+++ b/chrome/browser/ui/views/profiles/incognito_menu_view.h
@@ -33,6 +33,7 @@ class IncognitoMenuView : public ProfileMenuViewBase {
   // ProfileMenuViewBase:
   void BuildMenu() override;
 
+  BRAVE_INCOGNITO_MENU_VIEW_H_
  private:
   // views::BubbleDialogDelegateView:
   std::u16string GetAccessibleWindowTitle() const override;
