diff --git a/chrome/browser/resources/tab_search/BUILD.gn b/chrome/browser/resources/tab_search/BUILD.gn
index 96084ec0013eb7efd522d2cb7ee9ea0113be34b3..eb3f93af7ded25f485d20d4c8f516d32a75ebf45 100644
--- a/chrome/browser/resources/tab_search/BUILD.gn
+++ b/chrome/browser/resources/tab_search/BUILD.gn
@@ -130,4 +130,5 @@ build_webui("build") {
       "tab_search.js",
     ]
   }
+  import("//brave/browser/resources/tab_search/sources.gni") extra_grdp_deps = brave_tab_search_extra_grdp_deps extra_grdp_files = brave_tab_search_extra_grdp_files
 }
