diff --git a/chrome/common/chrome_switches.h b/chrome/common/chrome_switches.h
index c8de609ea315111ef912b2c5804d5d3cea50c32a..1e2b1dbf09bdc129b82035a810f4c6134f3d668e 100644
--- a/chrome/common/chrome_switches.h
+++ b/chrome/common/chrome_switches.h
@@ -120,6 +120,7 @@ extern const char kForceAppMode[];
 extern const char kForceDevToolsAvailable[];
 #endif
 extern const char kForceFirstRun[];
+extern const char kRefreshPlatformPolicy[];
 extern const char kForceWhatsNew[];
 extern const char kHideCrashRestoreBubble[];
 extern const char kHomePage[];
