diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index b77245135219b87d53f72ca1baca69fce42645a3..01ae5efca82c77701ecbbbf6313ff46b53a80315 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -308,6 +308,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data_deps = []
 
   if (is_chromeos_lacros && also_build_ash_chrome) {
