diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 0e613a31e6a429b78d7a3994ab57b86a8d75fc55..bb0f66be4d444b33838148daa4f276502b138414 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2671,6 +2671,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
