diff --git a/chrome/browser/ui/messages/android/BUILD.gn b/chrome/browser/ui/messages/android/BUILD.gn
index 976eddf05dc43002e0bf5eb1b7f13e43ce7e2308..78eb9357339ea173f5bcd55b5b299ca72256d66c 100644
--- a/chrome/browser/ui/messages/android/BUILD.gn
+++ b/chrome/browser/ui/messages/android/BUILD.gn
@@ -33,6 +33,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarView.java",
   ]
 
+  sources += brave_browser_ui_messages_java_sources
   deps = [
     ":java_resources",
     "//base:base_java",
