diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page.ts b/chrome/browser/resources/settings/privacy_page/privacy_page.ts
index 18ced3e60d029b961d7563775e51cad86a2a509c..2cffb5508d8e6ce9433c224c38b9c3f3155561ce 100644
--- a/chrome/browser/resources/settings/privacy_page/privacy_page.ts
+++ b/chrome/browser/resources/settings/privacy_page/privacy_page.ts
@@ -16,6 +16,7 @@ import '../icons.html.js';
 import '../privacy_icons.html.js';
 import '../settings_page/settings_section.js';
 import '../settings_shared.css.js';
+import './personalization_options.js';
 import './privacy_guide/privacy_guide_dialog.js';
 
 import {PrefsMixin} from '/shared/settings/prefs/prefs_mixin.js';
