diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index fdf3a09d43732d6b1b559411f1c7544a9be16cbf..915fd54e6b550b6b2bbd4e85ce4f11e414b0abd8 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -902,6 +902,7 @@ void BookmarkBarView::Layout(PassKey) {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   const bool show_bookmarks_overflow =
       bookmark_service_->loaded() &&
