diff --git a/chrome/browser/ui/toolbar/chrome_location_bar_model_delegate.h b/chrome/browser/ui/toolbar/chrome_location_bar_model_delegate.h
index 285bab9e40b4d3c6899d7e5d8074cd9facf5b762..21e5a4534cd06be5cb08a2018b128bacd87c95a5 100644
--- a/chrome/browser/ui/toolbar/chrome_location_bar_model_delegate.h
+++ b/chrome/browser/ui/toolbar/chrome_location_bar_model_delegate.h
@@ -59,6 +59,7 @@ class ChromeLocationBarModelDelegate : public LocationBarModelDelegate {
   content::NavigationEntry* GetNavigationEntry() const;
 
  private:
+  friend class BraveLocationBarModelDelegate;
   // Returns the navigation controller used to retrieve the navigation entry
   // from which the states are retrieved. If this returns null, default values
   // are used.
