diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index ec181852d9e41441162cf153eef245cc8f9abf4d..eaa983c054fe699af1b8e782975d1bcf814d8cf4 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -203,6 +203,7 @@ source_set("internal") {
       "//chrome/browser/ui:browser_select_file_dialog_controller",
     ]
   }
+  import("//brave/browser/ui/browser_window/internal/sources.gni") deps += brave_browser_window_internal_deps
 }
 
 source_set("unit_tests") {
