diff --git a/chrome/browser/permissions/permission_uma_util.cc b/chrome/browser/permissions/permission_uma_util.cc
index e4096511164296749fa9be0eccf4fdd8ab5213f5..a0913c777d5ef17fe3d43ac510c9f4d5a36a1ab2 100644
--- a/chrome/browser/permissions/permission_uma_util.cc
+++ b/chrome/browser/permissions/permission_uma_util.cc
@@ -64,7 +64,7 @@ namespace {
 
 const int kPriorCountCap = 10;
 
-std::string GetPermissionRequestString(PermissionRequestType type) {
+std::string GetPermissionRequestString_ChromiumImpl(PermissionRequestType type) {
   switch (type) {
     case PermissionRequestType::MULTIPLE:
       return "AudioAndVideoCapture";
@@ -402,6 +402,9 @@ void PermissionUmaUtil::RecordPermissionAction(
       UMA_HISTOGRAM_ENUMERATION("Permissions.Action.PaymentHandler", action,
                                 PermissionAction::NUM);
       break;
+    case CONTENT_SETTINGS_TYPE_AUTOPLAY:
+      BraveRecordPermissionAction(permission, secure_origin, action);
+      break;
     // The user is not prompted for these permissions, thus there is no
     // permission action recorded for them.
     default:
