diff --git a/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts b/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts
index 33415d56e8108adde1714e821f85eb7f8623c7b4..86bd8062c2675e07df1fffd525af2e1b51d72745 100644
--- a/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts
+++ b/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts
@@ -9,6 +9,7 @@
  */
 
 import 'chrome://resources/cr_elements/icons.html.js';
+import '../brave_overrides/config.js'
 import 'chrome://resources/cr_elements/cr_collapse/cr_collapse.js';
 import 'chrome://resources/cr_elements/cr_expand_button/cr_expand_button.js';
 import 'chrome://resources/cr_elements/cr_link_row/cr_link_row.js';
