diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model_unittest.cc b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model_unittest.cc
index cc7261c2d4666218661b6e8ca0a1f0fdef376020..8491cd124b7397f80d8d47ebf9c997cda45fd2eb 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model_unittest.cc
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model_unittest.cc
@@ -181,6 +181,7 @@ class RecentTabsSubMenuModelTest : public BrowserWithTestWindowTest,
     helper->VerifyExport(session_sync_service_->GetOpenTabsUIDelegate());
   }
 
+  BRAVE_RECENT_TABS_SUB_MENU_MODEL_TEST
  private:
   raw_ptr<sync_sessions::SessionSyncService, DanglingUntriaged>
       session_sync_service_;
