diff --git a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_service.ts b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_service.ts
index 1db2ac8548b77d181a23ea22f5ebff6eb8802e01..6b374163f3558b6e99e44c102afa794104e7af84 100644
--- a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_service.ts
+++ b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_service.ts
@@ -265,6 +265,7 @@ export class PowerBookmarksService {
       bookmarks: chrome.bookmarks.BookmarkTreeNode[],
       activeSortIndex: number): boolean {
     let changedPosition = false;
+    if (activeSortIndex == 5) { return changedPosition }
     bookmarks.sort(function(
         a: chrome.bookmarks.BookmarkTreeNode,
         b: chrome.bookmarks.BookmarkTreeNode) {
