diff --git a/chrome/browser/policy/configuration_policy_handler_list_factory.cc b/chrome/browser/policy/configuration_policy_handler_list_factory.cc
index aea7af532667..a5864e2756d5 100644
--- a/chrome/browser/policy/configuration_policy_handler_list_factory.cc
+++ b/chrome/browser/policy/configuration_policy_handler_list_factory.cc
@@ -642,12 +642,6 @@ const PolicyToPreferenceMapEntry kSimplePolicyMap[] = {
   { key::kSuppressUnsupportedOSWarning,
     prefs::kSuppressUnsupportedOSWarning,
     base::Value::Type::BOOLEAN },
-  { key::kRelaunchNotification,
-    prefs::kRelaunchNotification,
-    base::Value::Type::INTEGER },
-  { key::kRelaunchNotificationPeriod,
-    prefs::kRelaunchNotificationPeriod,
-    base::Value::Type::INTEGER },
   { key::kAutoplayAllowed,
     prefs::kAutoplayAllowed,
     base::Value::Type::BOOLEAN },
@@ -688,6 +682,12 @@ const PolicyToPreferenceMapEntry kSimplePolicyMap[] = {
     prefs::kURLsToNotCheckComplianceOfUploadedContent,
     base::Value::Type::LIST },
 #endif  // defined(OS_ANDROID)
+  { key::kRelaunchNotification,
+    prefs::kRelaunchNotification,
+    base::Value::Type::INTEGER },
+  { key::kRelaunchNotificationPeriod,
+    prefs::kRelaunchNotificationPeriod,
+    base::Value::Type::INTEGER },
 
 #if defined(OS_CHROMEOS)
   { key::kClientCertificateManagementAllowed,
