diff --git a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java
index 2bc5486946e27da9065c3878131f3130433243c6..4b4a9cdbc2a6922ffc4063cb620d55d102b84944 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java
@@ -159,7 +159,7 @@ public class MostVisitedTilesCoordinator implements ConfigurationChangedObserver
         mUserEducationHelper = new UserEducationHelper(mActivity, profile, handler);
 
         mContextMenuManager =
-                new ContextMenuManager(
+                new org.chromium.chrome.browser.native_page.BraveContextMenuManager(
                         suggestionsUiDelegate.getNavigationDelegate(),
                         touchEnabledDelegate,
                         mActivity::closeContextMenu,
