diff --git a/chrome/browser/ui/views/translate/translate_bubble_view_unittest.cc b/chrome/browser/ui/views/translate/translate_bubble_view_unittest.cc
index eabb62a082c50fec9fcaa136089b643efb0f4fc8..78ae1f2e8c457898efc4b5fb350aba9db7cde160 100644
--- a/chrome/browser/ui/views/translate/translate_bubble_view_unittest.cc
+++ b/chrome/browser/ui/views/translate/translate_bubble_view_unittest.cc
@@ -249,6 +249,7 @@ class TranslateBubbleViewTest : public ChromeViewsTestBase {
 
  protected:
   void SetUp() override {
+    scoped_feature_list_.InitFromCommandLine("BraveEnableAutoTranslate", "");
     ChromeViewsTestBase::SetUp();
 
     // The bubble needs the parent as an anchor.
