diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 43fe1d215d3bc03b22237c069bf3823b4b54922d..e0b480f097f06b6841fce1febc360324c8e57b12 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4870,6 +4870,7 @@ constexpr char kWebiumFlag[] = "webium";
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
