diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index d64174dd2c0452d7407f35ef2df16ec4dd1ccaa4..76dfc6d72167969e528a6706d695eb332ae94bf2 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -78,6 +78,8 @@ grit("generated_resources") {
   if (is_android) {
     outputs += android_generated_java_resources
   }
+
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
@@ -100,7 +102,7 @@ grit("google_chrome_strings") {
 }
 
 grit("chromium_strings") {
-  source = "chromium_strings.grd"
+  source = "${branding_path_component}_strings.grd"
   defines = chrome_grit_defines
   output_dir = "$root_gen_dir/chrome"
   outputs =
