diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index 4a2feb82d8f5a33a855b8cd8cbfd8ee7691baba9..379dad639d4b204108bd42beab8bd5fd82dfffb0 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -978,6 +978,7 @@ void BookmarkBarView::Layout() {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   // Layout the right side buttons.
   x = max_x + bookmark_bar_button_padding;
