diff --git a/chrome/browser/password_manager/password_change_delegate_impl.cc b/chrome/browser/password_manager/password_change_delegate_impl.cc
index 3882bc4ae6b9d7c8c4b497a71a5c02702978f39a..99b060d5baffab7f5dac41cda51e0aa49230f040 100644
--- a/chrome/browser/password_manager/password_change_delegate_impl.cc
+++ b/chrome/browser/password_manager/password_change_delegate_impl.cc
@@ -117,6 +117,7 @@ std::unique_ptr<BrowserSavePasswordProgressLogger> GetLoggerIfAvailable(
 }
 
 std::unique_ptr<content::WebContents> CreateWebContents(Profile* profile,
+                                                        base::WeakPtr<content::WebContents> originator,
                                                         const GURL& url) {
   scoped_refptr<content::SiteInstance> initial_site_instance_for_new_contents =
       tab_util::GetSiteInstanceForNewTab(profile, url);
@@ -181,7 +182,7 @@ void PasswordChangeDelegateImpl::StartPasswordChange() {
   flow_start_time_ = base::Time::Now();
   UpdateState(State::kWaitingForChangePasswordForm);
 
-  executor_ = CreateWebContents(profile_, change_password_url_);
+  executor_ = CreateWebContents(profile_, originator_, change_password_url_);
   CHECK(executor_);
   logs_uploader_ = std::make_unique<ModelQualityLogsUploader>(executor_.get());
   form_finder_ = std::make_unique<ChangePasswordFormFinder>(
