diff --git a/chrome/browser/ui/views/page_action/action_ids.h b/chrome/browser/ui/views/page_action/action_ids.h
index 98b2fa389eeb2c2c2733ca423a965f665a54c438..3c4872b1726390a614c56b3f25ec8b378519a6fb 100644
--- a/chrome/browser/ui/views/page_action/action_ids.h
+++ b/chrome/browser/ui/views/page_action/action_ids.h
@@ -16,6 +16,7 @@ namespace page_actions {
 // the array.
 inline constexpr auto kActionIds = std::to_array<actions::ActionId>({
     kActionAiMode,
+    kActionShowPartitionedStorage,
     kActionSidePanelShowLensOverlayResults,
     kActionLensOverlayHomework,
     kActionShowTranslate,
