diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 72160e54d7d2b6d0f8e52960253f14041bb2a8b6..d74416b373de08426e50e1148a1e8226189fbc18 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4409,6 +4409,7 @@ const FeatureEntry::FeatureVariation kEnableCanvasNoiseVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
