diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index 18b00d539f7c0a629df04f6f670554783313442a..354bedcd261f399402301ee24cd7fc4ea17b1e42 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -310,6 +310,8 @@ content::WebUIDataSource* CreateMdExtensionsSource(Profile* profile,
   source->SetDefaultResource(IDR_EXTENSIONS_EXTENSIONS_HTML);
 #endif
 
+  BraveAddExtensionsResources(source);
+
   return source;
 }
 
