diff --git a/chrome/browser/ui/webui/settings/import_data_handler.h b/chrome/browser/ui/webui/settings/import_data_handler.h
index 445e400915201cd699feed6dee8db210ac16103a..745511a38c480f2e9cdf25969c215204c4fb2724 100644
--- a/chrome/browser/ui/webui/settings/import_data_handler.h
+++ b/chrome/browser/ui/webui/settings/import_data_handler.h
@@ -37,6 +37,7 @@ class ImportDataHandler : public SettingsPageUIHandler,
   void OnJavascriptAllowed() override {}
   void OnJavascriptDisallowed() override;
 
+  BRAVE_IMPORT_DATA_HANDLER_H
  private:
   void StartImport(const importer::SourceProfile& source_profile,
                    uint16_t imported_items);
