diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 5777c231b6bb7640695c55a636b654202885b8ed..c322c644e4d2412fdeadb4d3c5ae179125ecf71b 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2360,6 +2360,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
