diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 919e856ed7fef1fffbd917608dc954e65197f901..e0828c31139f6f96909eb1a4a8fae768fbb29bf5 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/sources.gni")
 import("//base/allocator/allocator.gni")
 import("//build/buildflag_header.gni")
 import("//build/config/buildflags_paint_preview.gni")
@@ -2046,6 +2047,7 @@ source_set("browser") {
     # should be added to browser_generated_files.
     ":browser_public_dependencies",
     ":core",
+    "//brave/browser:core",
     "//chrome/browser/accessibility/accessibility_prefs",
     "//chrome/browser/browsing_data",
     "//chrome/browser/download",
@@ -3286,7 +3288,7 @@ source_set("browser") {
 
   # TODO(https://crbug.com/356905053): Add more dependencies here and merge this
   # block with the `enable_extensions` block above.
-  if (enable_desktop_android_extensions) {
+  if (is_android && enable_extensions_core) {
     deps += [ "//chrome/browser/ui/android/extensions" ]
   }
 
@@ -3473,6 +3475,7 @@ group("browser_public_dependencies") {
     "//sql",
     "//ui/lottie",
   ]
+  public_deps += brave_chrome_browser_public_deps
   if (is_android) {
     public_deps += [
       "//chrome/android/features/dev_ui:buildflags",
