diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 49c38a50214ee7ca39cf6091902b1e14178a1aa9..3199a0f2d30236ab4f5eff34fe84f85e316aa46f 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3390,6 +3390,7 @@ const FeatureEntry::FeatureVariation kWebRtcApmDownmixMethodVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
