diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 5e69beac0bf4202b17ab1102423f0a051a8497f5..7d93894f89134349d4fce3ec68f079c93ff52f9d 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2144,6 +2144,7 @@ static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
