diff --git a/chrome/test/data/webui/settings/BUILD.gn b/chrome/test/data/webui/settings/BUILD.gn
index 75edd4b7b4b33199e5350bd5c49e9fd349a5d50c..6383fac692ae7e3509542fbcf3ff4d0146889a9b 100644
--- a/chrome/test/data/webui/settings/BUILD.gn
+++ b/chrome/test/data/webui/settings/BUILD.gn
@@ -285,6 +285,7 @@ build_webui_tests("build") {
     ts_definitions +=
         [ "//tools/typescript/definitions/quick_unlock_private.d.ts" ]
   }
+  import("//brave/browser/resources/settings/sources.gni") ts_definitions += brave_settings_ts_definitions
 
   ts_deps = [
     "//chrome/browser/resources/settings:build_ts",
