diff --git a/chrome/android/java/src/org/chromium/chrome/browser/base/SplitChromeApplication.java b/chrome/android/java/src/org/chromium/chrome/browser/base/SplitChromeApplication.java
index a15feaf0ce35e1e53fbafa247de83c653aeab2d2..36b5cb3998dfb28e4b3552477128dcaf2d0996d6 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/base/SplitChromeApplication.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/base/SplitChromeApplication.java
@@ -213,7 +213,7 @@ public class SplitChromeApplication extends SplitCompatApplication {
             // fieldtrial_testing_config.json.
             ContextUtils.sDoFeatureListInitHookForTesting =
                     InitializeFeatureList::initializeFeatureList;
-        } else if (!BuildConfig.IS_CHROME_BRANDED || !VariationsSeedFetcher.shouldFetchSeed()) {
+        } else if (!true /* BuildConfig.IS_CHROME_BRANDED override for Brave */ || !VariationsSeedFetcher.shouldFetchSeed()) {
             // For non-Chrome branded builds, we should initialize the feature list early to
             // apply the fieldtrial_testing_config.json. Otherwise, we should initialize the
             // feature list early in non-first run when we are not fetching the first run
