diff --git a/chrome/browser/extensions/extension_management.h b/chrome/browser/extensions/extension_management.h
index 8bd180cf9672c1dbec4742cda1c697da063d94b4..906060cb5c4e59fbc88e99ee4efb2edcc9d597ec 100644
--- a/chrome/browser/extensions/extension_management.h
+++ b/chrome/browser/extensions/extension_management.h
@@ -214,6 +214,7 @@ class ExtensionManagement : public KeyedService {
   bool is_child() const { return is_child_; }
 
  private:
+  friend class BraveExtensionManagement;
   using SettingsIdMap =
       std::unordered_map<ExtensionId,
                          std::unique_ptr<internal::IndividualSettings>>;
