diff --git a/chrome/browser/prefs/browser_prefs.cc b/chrome/browser/prefs/browser_prefs.cc
index 9587076ae3d6e3c95a07d60e7fb0b2f698ae82b7..d27efcba32e151859dc0f5b6b01c3ce834f4e28c 100644
--- a/chrome/browser/prefs/browser_prefs.cc
+++ b/chrome/browser/prefs/browser_prefs.cc
@@ -2026,6 +2026,7 @@ void RegisterLocalState(PrefRegistrySimple* registry) {
 #endif  // !BUILDFLAG(IS_ANDROID)
 
   // This is intentionally last.
+  brave::RegisterLocalStatePrefs(registry);
   RegisterLocalStatePrefsForMigration(registry);
 }
 
@@ -2484,6 +2485,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry,
   // the policies logic is implemented.
   registry->RegisterListPref(policy::policy_prefs::kIncognitoModeBlocklist);
   registry->RegisterListPref(policy::policy_prefs::kIncognitoModeAllowlist);
+  brave::RegisterProfilePrefs(registry);
 }
 
 void RegisterUserProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -3068,6 +3070,7 @@ void MigrateObsoleteProfilePrefs(PrefService* profile_prefs,
   profile_prefs->ClearPref(kObsoletePasswordsUseUPMLocalAndSeparateStores);
   profile_prefs->ClearPref(kObsoleteEmptyProfileStoreLoginDatabase);
   profile_prefs->ClearPref(kObsoleteUpmAutoExportCsvNeedsDeletion);
+#if 0
   base::DeleteFile(profile_path.Append(FILE_PATH_LITERAL("Login Data")));
   base::DeleteFile(
       profile_path.Append(FILE_PATH_LITERAL("Login Data For Account")));
@@ -3075,6 +3078,7 @@ void MigrateObsoleteProfilePrefs(PrefService* profile_prefs,
       profile_path.Append(FILE_PATH_LITERAL("Login Data-journal")));
   base::DeleteFile(
       profile_path.Append(FILE_PATH_LITERAL("Login Data For Account-journal")));
+#endif
 #endif  // BUILDFLAG(IS_ANDROID)
 
   // Added 09/2025.
