diff --git a/chrome/browser/metrics/chrome_metrics_services_manager_client.cc b/chrome/browser/metrics/chrome_metrics_services_manager_client.cc
index 93d38754f6dc2fcf99e00f6bc947f4db91c50af3..babb86f791802f0d73efe6dad0dbb6a598af8030 100644
--- a/chrome/browser/metrics/chrome_metrics_services_manager_client.cc
+++ b/chrome/browser/metrics/chrome_metrics_services_manager_client.cc
@@ -153,7 +153,7 @@ class ChromeMetricsServicesManagerClient::ChromeEnabledStateProvider
   }
 
   bool IsReportingEnabled() const override {
-    return metrics::EnabledStateProvider::IsReportingEnabled() &&
+    return false && metrics::EnabledStateProvider::IsReportingEnabled() &&
            IsClientInSampleImpl(local_state_);
   }
 
