diff --git a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java
index 1c0c67ebdc28b6c2b8d3488d40ba803ca882aea5..420c2159993b6d655662c671a65af40b449ff486 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/MostVisitedTilesCoordinator.java
@@ -132,7 +132,7 @@ public class MostVisitedTilesCoordinator implements ConfigurationChangedObserver
         mUserEducationHelper = new UserEducationHelper(mActivity, profile, handler);
 
         mContextMenuManager =
-                new ContextMenuManager(
+                new org.chromium.chrome.browser.native_page.BraveContextMenuManager(
                         suggestionsUiDelegate.getNavigationDelegate(),
                         touchEnabledDelegate,
                         mActivity::closeContextMenu,
