diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 1feded0d4e23dcb966674a0192984b98f55b6d15..893dcbae2bf857efecc854c6e0229561834edfaa 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -719,6 +719,7 @@ class Browser : public TabStripModelObserver,
 
   StatusBubble* GetStatusBubbleForTesting();
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
