diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 02cdf425677457ff9db420bba6d455d474b32afe..62f0f7e5d4ec35d079e784c7d7a2d78ac46ded48 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -235,6 +235,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  sources += brave_chrome_renderer_sources public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
