diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 9701b249d923e52f24e9d16c63c6ca0525a43188..1e76b463c30c35b9bc689bdf6a68901345d02943 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -913,6 +913,7 @@ if (!is_android) {
       "//v8",
     ]

+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
