diff --git a/chrome/browser/ui/views/bubble/webui_bubble_manager.h b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
index dbda307510db6bacf70a4eee109e48cdbee2ef4b..c2c3a9f1aa0175a4760c232ce0377d55c61f3753 100644
--- a/chrome/browser/ui/views/bubble/webui_bubble_manager.h
+++ b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
@@ -164,10 +164,12 @@ class WebUIBubbleManagerImpl : public WebUIBubbleManager {
 
  private:
   // WebUIBubbleManager:
+ protected:
   base::WeakPtr<WebUIBubbleDialogView> CreateWebUIBubbleDialog(
       const std::optional<gfx::Rect>& anchor,
       views::BubbleBorder::Arrow arrow) override;
   WebUIContentsWrapper* GetContentsWrapper() override;
+ private:
 
   const raw_ptr<views::View> anchor_view_;
   const raw_ptr<BrowserWindowInterface> browser_window_interface_;
