diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 6d87f470bfd7b5572802993ee6bb9723a9290cbc..558f7b093280080c6bdb7f43b6fd50b1d5f78408 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -222,6 +222,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  sources += brave_chrome_renderer_sources public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
