diff --git a/chrome/browser/resources/bookmarks/api_listener.ts b/chrome/browser/resources/bookmarks/api_listener.ts
index 588014b533bf9e45b33196763eeae96b046991fa..2d76cbf1a425caf67a6d7ae3e5ca5004c501cbec 100644
--- a/chrome/browser/resources/bookmarks/api_listener.ts
+++ b/chrome/browser/resources/bookmarks/api_listener.ts
@@ -118,6 +118,7 @@ function onChildrenReordered(
  */
 function onImportBegan() {
   chrome.bookmarks.onCreated.removeListener(onBookmarkCreated);
+  chrome.bookmarks.onMoved.removeListener(onBookmarkMoved);
   document.dispatchEvent(new CustomEvent('import-began'));
 }
 
@@ -126,6 +127,7 @@ function onImportEnded() {
     dispatch(refreshNodes(normalizeNodes(results[0]!)));
   });
   chrome.bookmarks.onCreated.addListener(onBookmarkCreated);
+  chrome.bookmarks.onMoved.addListener(onBookmarkMoved);
   document.dispatchEvent(new CustomEvent('import-ended'));
 }
 
