diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index 03f65f93fe57613431edd85fad2ac79cd5a656be..cd948b1921a1edaf86e6f3ee8c42c3791ac7b194 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -184,7 +184,7 @@ const char Profile::kProfileKey[] = "__PROFILE__";
 void Profile::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
       prefs::kSearchSuggestEnabled,
-      true,
+      false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 #if defined(OS_ANDROID)
   registry->RegisterStringPref(
