diff --git a/chrome/browser/renderer_context_menu/render_view_context_menu.h b/chrome/browser/renderer_context_menu/render_view_context_menu.h
index d5195d4b0f6c8aae36728942278a75d5ac7bed4e..a9697e02b008c2faee9c369c57f56b87c92985f6 100644
--- a/chrome/browser/renderer_context_menu/render_view_context_menu.h
+++ b/chrome/browser/renderer_context_menu/render_view_context_menu.h
@@ -83,11 +83,6 @@ class RenderViewContextMenu : public RenderViewContextMenuBase {
   void AddSpellCheckServiceItem(bool is_checked) override;
   void AddAccessibilityLabelsServiceItem(bool is_checked) override;
 
-  // Registers a one-time callback that will be called the next time a context
-  // menu is shown.
-  static void RegisterMenuShownCallbackForTesting(
-      base::OnceCallback<void(RenderViewContextMenu*)> cb);
-
  protected:
   Profile* GetProfile() const;
 
@@ -113,6 +108,7 @@ class RenderViewContextMenu : public RenderViewContextMenuBase {
   // Returns true if keyboard lock is active and requires the user to press and
   // hold escape to exit exclusive access mode.
   bool IsPressAndHoldEscRequiredToExitFullscreen() const;
+  BRAVE_RENDER_VIEW_CONTEXT_MENU_H_
 
  private:
   friend class RenderViewContextMenuTest;
