diff --git a/chrome/browser/ui/extensions/extension_action_view_controller.cc b/chrome/browser/ui/extensions/extension_action_view_controller.cc
index 10c06f59cfae4ab8582054bb36f3290dc0c167c8..057b3ee9af53feb592218e36a16045ffbcd20980 100644
--- a/chrome/browser/ui/extensions/extension_action_view_controller.cc
+++ b/chrome/browser/ui/extensions/extension_action_view_controller.cc
@@ -50,7 +50,6 @@ ExtensionActionViewController::Create(
     Browser* browser,
     ExtensionsContainer* extensions_container) {
   DCHECK(browser);
-  DCHECK(extensions_container);
 
   auto* registry = extensions::ExtensionRegistry::Get(browser->profile());
   scoped_refptr<const extensions::Extension> extension =
