diff --git a/chrome/browser/ui/dialogs/outdated_upgrade_bubble.cc b/chrome/browser/ui/dialogs/outdated_upgrade_bubble.cc
index d08dce0a2bde8d248408329e482986fa8ee176c4..a1d1fbf7f0704680087d46452d705286464fed86 100644
--- a/chrome/browser/ui/dialogs/outdated_upgrade_bubble.cc
+++ b/chrome/browser/ui/dialogs/outdated_upgrade_bubble.cc
@@ -52,7 +52,7 @@ void OnDialogAccepted(content::PageNavigator* navigator,
         base::UserMetricsAction("OutdatedUpgradeBubble.Reinstall"));
 
     navigator->OpenURL(content::OpenURLParams::CreateBrowserInitiated(
-                           GURL(update_browser_redirect_url),
+                           BraveGetUpdateUrl(),
                            WindowOpenDisposition::NEW_FOREGROUND_TAB,
                            ui::PAGE_TRANSITION_LINK),
                        /*navigation_handle_callback=*/{});
