diff --git a/chrome/browser/themes/theme_helper.h b/chrome/browser/themes/theme_helper.h
index 6dc522c40f2afcc16102227d35c0dae4a8446bd3..24eb02feb53bf5339f85b15d804daaeef1832450 100644
--- a/chrome/browser/themes/theme_helper.h
+++ b/chrome/browser/themes/theme_helper.h
@@ -95,6 +95,8 @@ class ThemeHelper {
     DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
   }
 
+  BRAVE_THEME_HELPER_H_
+
  private:
   friend class theme_service_internal::ThemeServiceTest;
 
