diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 29ca96afb505284dab74f5de18a86367093a7143..4f61f20d31cf91e57b73f7d4fc068e15c61d48cf 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -2856,6 +2856,7 @@ const FeatureEntry::FeatureVariation kTabStripImprovementsTabWidthVariations[] =
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
