diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index c524bf95656e79249463ea9898bbd7342bb06999..0f7762e546b14723952dcfd0e51a02b774d557a7 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -77,6 +77,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
