diff --git a/chrome/browser/ui/views/tabs/tab_strip.cc b/chrome/browser/ui/views/tabs/tab_strip.cc
index 2433086bcd89b854b098554e0ecced288cbdabde..d1b7fce09abb2a6259863faa47edfa5ebcc9a488 100644
--- a/chrome/browser/ui/views/tabs/tab_strip.cc
+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
@@ -1607,7 +1607,7 @@ int TabStrip::TabToFollowingNewTabButtonSpacing() const {
   // Pre-refresh, there are a variety of hardcoded values.
   const int mode = MD::GetMode();
   DCHECK_LE(mode, 2);
-  constexpr int kNewTabButtonSpacing[] = {-5, -6, 6};
+  constexpr int kNewTabButtonSpacing[] = {6, 7, 7};
   return kNewTabButtonSpacing[mode];
 }
 
