diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index c61e8b60b4c42b4c14b974cf93e61cdc666f378e..10aeecbabda3d92be6f06982b8e92bd32e0d349c 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -1835,6 +1835,7 @@ split_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_android) {
