diff --git a/chrome/common/importer/BUILD.gn b/chrome/common/importer/BUILD.gn
index a5397a8f07d7f84832944971c22d9781533ffc83..a911a6ca8b573670970f86cf85a2acd9d5ef595b 100644
--- a/chrome/common/importer/BUILD.gn
+++ b/chrome/common/importer/BUILD.gn
@@ -122,4 +122,5 @@ source_set("importer") {
     "//ui/base",
     "//url",
   ]
+  import("//brave/common/importer/sources.gni") deps += brave_common_importer_deps
 }
