diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index d67113f4fd83dc97afeba25c4c44f2092a63e94f..b2a94e986a4fc7fcde1e4508e832270e1b4d8929 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1441,6 +1441,7 @@ static_library("extensions") {
       "//components/enterprise/data_controls",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 proto_library("cws_item_service_proto") {
