diff --git a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
index 0641c5db2776ba2c7f195e8fdc7966d22bf83d60..1d5430a5957dc28ea1330013bbb558ae6ee51e57 100644
--- a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
+++ b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
@@ -193,7 +193,8 @@ export class PowerBookmarksListElement extends PolymerElement implements
                sortOrder: SortOrder.kReverseAlphabetical,
                label: loadTimeData.getString('sortReverseAlphabetically'),
                lowerLabel: loadTimeData.getString('sortReverseAlphabetically'),
-             }],
+             },
+             { sortOrder: SortOrder.kCustom, label: loadTimeData.getString('sortCustom'), lowerLabel: loadTimeData.getString('sortCustomLower'), }],
       },
 
       editing_: {
