diff --git a/chrome/browser/ui/views/frame/multi_contents_view.h b/chrome/browser/ui/views/frame/multi_contents_view.h
index c0ee7ac4b14bf6e63fd5a8c0266f2d1f9dcfe2e4..ca1b6ad1fc29257a93369bbfa71004c938e793a4 100644
--- a/chrome/browser/ui/views/frame/multi_contents_view.h
+++ b/chrome/browser/ui/views/frame/multi_contents_view.h
@@ -18,7 +18,7 @@
 
 class BrowserView;
 class ContentsWebView;
-class MultiContentsResizeArea;
+// class MultiContentsResizeArea;
 class MultiContentsViewDropTargetController;
 class MultiContentsViewMiniToolbar;
 class MultiContentsDropTargetView;
