diff --git a/chrome/browser/ui/extensions/extension_action_view_controller.cc b/chrome/browser/ui/extensions/extension_action_view_controller.cc
index f531aa84a68f011f529f1f81bb8c5a39eb4560a1..4449249d29ee39df316fecfca65449b37c0d1e5b 100644
--- a/chrome/browser/ui/extensions/extension_action_view_controller.cc
+++ b/chrome/browser/ui/extensions/extension_action_view_controller.cc
@@ -67,7 +67,6 @@ ExtensionActionViewController::Create(
     Browser* browser,
     ExtensionsContainer* extensions_container) {
   DCHECK(browser);
-  DCHECK(extensions_container);
 
   auto* registry = extensions::ExtensionRegistry::Get(browser->profile());
   scoped_refptr<const extensions::Extension> extension =
