diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageLayout.java b/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageLayout.java
index 155042713408fb054d86db4b2307ba71b3448322..d1fa5a399ae38052bd194e08d3c372eb7992f079 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageLayout.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageLayout.java
@@ -226,6 +226,7 @@ public class NewTabPageLayout extends LinearLayout implements TileGroup.Observer
                 /* observer = */ this, offlinePageBridge);
 
         int maxRows = 2;
+        maxRows = QueryTileSection.getMaxRowsForMostVisitedTiles(getContext());
         if (searchProviderIsGoogle && QueryTileUtils.isQueryTilesEnabledOnNTP()) {
             maxRows = QueryTileSection.getMaxRowsForMostVisitedTiles(getContext());
         }
