diff --git a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
index 36189521424be9a40a6208d3b469556a4f301105..c957267870c6011b39dc3ec0f6af91d61269e84e 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
@@ -262,6 +262,7 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             Action.SEND_TAB_TO_SELF,
             Action.TRANSLATE,
             Action.CREATE_QR_CODE,
+            Action.COPY_CLEAN_LINK,
         })
         @Retention(RetentionPolicy.SOURCE)
         public @interface Action {
@@ -324,7 +325,8 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             int SEND_TAB_TO_SELF = 56;
             int TRANSLATE = 57;
             int CREATE_QR_CODE = 58;
-            int NUM_ENTRIES = 59;
+            int COPY_CLEAN_LINK = 59;
+            int NUM_ENTRIES = COPY_CLEAN_LINK + 1;
         }
 
         // LINT.ThenChange(/tools/metrics/histograms/enums.xml:ContextMenuOptionAndroid)
@@ -659,6 +661,7 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             if (!MailTo.isMailTo(mParams.getLinkUrl().getSpec())
                     && !UrlUtilities.isTelScheme(mParams.getLinkUrl())) {
                 linkGroup.add(createListItem(Item.COPY_LINK_ADDRESS));
+                linkGroup.add(createListItem(Item.COPY_CLEAN_LINK));
                 if (!mParams.getLinkText().trim().isEmpty() && !mParams.isImage()) {
                     linkGroup.add(createListItem(Item.COPY_LINK_TEXT));
                 }
