diff --git a/chrome/browser/ui/browser_ui_controller/BUILD.gn b/chrome/browser/ui/browser_ui_controller/BUILD.gn
index 1170db54d65afcf1795e0ed8ec18df528140a52a..4aceed1844387e4cdfb719fc62e96bc99f79059c 100644
--- a/chrome/browser/ui/browser_ui_controller/BUILD.gn
+++ b/chrome/browser/ui/browser_ui_controller/BUILD.gn
@@ -27,6 +27,7 @@ source_set("impl") {
 
   deps = [
     "//base",
+    "//brave/browser/ui/sidebar",
     "//chrome/browser/app_mode",
     "//chrome/browser/profiles:profile",
     "//chrome/browser/ui:browser_window_header",
