diff --git a/chrome/installer/mac/signing/commands.py b/chrome/installer/mac/signing/commands.py
index 398a4cafbf4e46fe720f273bf8a4183d07bd4fc9..33ecbd824318637bb4cfb0107d66d0d198e0f441 100644
--- a/chrome/installer/mac/signing/commands.py
+++ b/chrome/installer/mac/signing/commands.py
@@ -200,3 +200,4 @@ class WorkDirectory(object):
 
     def __exit__(self, exc_type, value, traceback):
         shutil.rmtree(self._workdir)
+from brave_chromium_utils import inline_file; inline_file("//brave/chromium_src/chrome/installer/mac/signing/commands.py", globals(), locals())
\ No newline at end of file
