diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index b4fcbedc0dc5c3201789c00a2169c49c0879f879..9d38557554d8a996cbf8b91ccbf1e9c8843aa511 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -106,6 +106,7 @@ grit("generated_resources") {
   if (is_android) {
     outputs += android_generated_java_resources
   }
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
