diff --git a/chrome/browser/ui/views/extensions/extensions_menu_delegate_desktop.cc b/chrome/browser/ui/views/extensions/extensions_menu_delegate_desktop.cc
index 4f01efd742548b57a1c23e568cbc512e1e484303..a3e3b382a5650dd06e856883d05515bbc2dab5e7 100644
--- a/chrome/browser/ui/views/extensions/extensions_menu_delegate_desktop.cc
+++ b/chrome/browser/ui/views/extensions/extensions_menu_delegate_desktop.cc
@@ -406,7 +406,8 @@ void ExtensionsMenuDelegateDesktop::OnUserPermissionsSettingsChanged() {
 }
 
 void ExtensionsMenuDelegateDesktop::OpenMainPage() {
-  auto main_page = std::make_unique<ExtensionsMenuMainPageView>(browser_, this);
+  auto main_page =
+      std::make_unique<BraveExtensionsMenuMainPageView>(browser_, this);
   UpdateMainPage(main_page.get());
   PopulateMainPage(main_page.get());
   if (main_page->GetMenuEntries().empty()) {
