diff --git a/chrome/browser/sync/sync_service_factory.cc b/chrome/browser/sync/sync_service_factory.cc
index e73da2bcf1e49bd1b8021d373cc5175a94ace390..36f9a09a9731eb6ee72084df3df67cd60c901a77 100644
--- a/chrome/browser/sync/sync_service_factory.cc
+++ b/chrome/browser/sync/sync_service_factory.cc
@@ -384,7 +384,7 @@ std::unique_ptr<KeyedService> BuildSyncService(
   }
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize(CreateControllers(profile, sync_service.get()));
 
   // Notify the PasswordStore of complete initialisation to resolve a circular
