diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 0446e2ca248c7eb3227f0eca4de856c0f61e6ed2..1f09a22e7ef379bead85f7b193ceefd1cd61535c 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4690,6 +4690,7 @@ const FeatureEntry::FeatureVariation kCanvasAccessibilityChoices[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
