diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index c4a92225b7ed792fc4eab958b88482ee9a7eb8d6..19e54af7fc71d006aa6cfbcbc9fee87258532db9 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2154,6 +2154,7 @@ static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_posix || is_fuchsia) {
