diff --git a/chrome/browser/extensions/extension_tab_util.cc b/chrome/browser/extensions/extension_tab_util.cc
index d8b78c08cb2830dcb49ff166acbe2884c315e6d3..f57885ee8e2575888fbb37a0e8b16c5b5c2e0d9f 100644
--- a/chrome/browser/extensions/extension_tab_util.cc
+++ b/chrome/browser/extensions/extension_tab_util.cc
@@ -794,6 +794,7 @@ bool ExtensionTabUtil::IsKillURL(const GURL& url) {
   };
 
   if (!url.SchemeIs(content::kChromeUIScheme))
+  if (!url.SchemeIs(content::kBraveUIScheme))
     return false;
 
   return base::Contains(kill_hosts, url.host_piece());
