diff --git a/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelDefinitions.java b/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelDefinitions.java
index 23eebd17c1cf56471714da158ae9f65e7bd02551..d07591f6a25bfd5393b9108d89330c890785c1a0 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelDefinitions.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelDefinitions.java
@@ -54,6 +54,7 @@ public class ChannelDefinitions {
      * more detailed instructions.
      */
     @StringDef({ChannelId.BROWSER, ChannelId.DOWNLOADS, ChannelId.INCOGNITO, ChannelId.MEDIA,
+            BraveChannelDefinitions.ChannelId.BRAVE_ADS,
             ChannelId.SCREEN_CAPTURE, ChannelId.CONTENT_SUGGESTIONS, ChannelId.WEBAPP_ACTIONS,
             ChannelId.SITES, ChannelId.SHARING, ChannelId.UPDATES, ChannelId.COMPLETED_DOWNLOADS,
             ChannelId.PERMISSION_REQUESTS, ChannelId.PERMISSION_REQUESTS_HIGH})
@@ -77,6 +78,7 @@ public class ChannelDefinitions {
     }
 
     @StringDef({
+            BraveChannelDefinitions.ChannelGroupId.BRAVE_ADS,
             ChannelGroupId.GENERAL, ChannelGroupId.SITES
     })
     @Retention(RetentionPolicy.SOURCE)
@@ -107,6 +109,7 @@ public class ChannelDefinitions {
             Map<String, PredefinedChannel> map = new HashMap<>();
             Set<String> startup = new HashSet<>();
 
+            BraveChannelDefinitions.addBraveChannels(map, startup);
             map.put(ChannelId.BROWSER,
                     new PredefinedChannel(ChannelId.BROWSER, R.string.notification_category_browser,
                             NotificationManager.IMPORTANCE_LOW, ChannelGroupId.GENERAL));
@@ -207,6 +210,7 @@ public class ChannelDefinitions {
         static final Map<String, PredefinedChannelGroup> MAP;
         static {
             Map<String, PredefinedChannelGroup> map = new HashMap<>();
+            BraveChannelDefinitions.addBraveChannelGroups(map);
             map.put(ChannelGroupId.GENERAL,
                     new PredefinedChannelGroup(
                             ChannelGroupId.GENERAL, R.string.notification_category_group_general));
