diff --git a/chrome/browser/ui/android/theme/java/src/org/chromium/chrome/browser/theme/ThemeUtils.java b/chrome/browser/ui/android/theme/java/src/org/chromium/chrome/browser/theme/ThemeUtils.java
index b4bfb43ec3a856c15e0c9ca50b3003171160124e..24ca6f19e490da30803ff136da0c6a03b85147f0 100644
--- a/chrome/browser/ui/android/theme/java/src/org/chromium/chrome/browser/theme/ThemeUtils.java
+++ b/chrome/browser/ui/android/theme/java/src/org/chromium/chrome/browser/theme/ThemeUtils.java
@@ -74,6 +74,7 @@ public class ThemeUtils {
             Resources res, @ColorInt int color, boolean isIncognito) {
         // Text box color on default toolbar background in incognito mode is a pre-defined
         // color. We calculate the equivalent opaque color from the pre-defined translucent color.
+        if (isIncognito) {return ApiCompatibilityUtils.getColor(res, R.color.toolbar_text_box_background_incognito);}
         if (isIncognito) {
             final int overlayColor = ApiCompatibilityUtils.getColor(
                     res, R.color.toolbar_text_box_background_incognito);
