diff --git a/chrome/app/settings_strings.grdp b/chrome/app/settings_strings.grdp
index 82488d6f89030aad0fbad2558fe2de7e1bd22c76..2e1c72099f86145ea51cd1da86546d4f31a96e0c 100644
--- a/chrome/app/settings_strings.grdp
+++ b/chrome/app/settings_strings.grdp
@@ -3403,6 +3403,9 @@
     <message name="IDS_SETTINGS_IMPORT_AUTOFILL_FORM_DATA_CHECKBOX" desc="Checkbox for importing form data for autofill">
       Autofill form data
     </message>
+    <message name="IDS_SETTINGS_IMPORT_COOKIES_CHECKBOX" desc="Checkbox for importing cookies">
+      Cookies
+    </message>
 
     <message name="IDS_SETTINGS_IMPORT_CHOOSE_FILE" desc="Text for the Choose File on dialog">
       Choose File
