diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index 203ceeef18ef8160bc9ee20c68bf7aff1db96882..5a4eea9fa179125b3d81949012db989a551f5a1a 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -137,6 +137,7 @@ source_set("internal") {
       deps += [ "//chrome/browser/ui:overscroll_pref_manager" ]
     }
   }
+  import("//brave/browser/ui/browser_window/internal/sources.gni") deps += brave_browser_window_internal_deps
 }
 
 source_set("unit_tests") {
