diff --git a/chrome/browser/net/system_network_context_manager.cc b/chrome/browser/net/system_network_context_manager.cc
index c0bfebc57a0c749377f8b334855485a2731455a5..3ccab2e1b6cb74cd8d448cb8bc8fde55d535dd56 100644
--- a/chrome/browser/net/system_network_context_manager.cc
+++ b/chrome/browser/net/system_network_context_manager.cc
@@ -477,7 +477,7 @@ class SystemNetworkContextManager::URLLoaderFactoryForSystem
     if (!manager_)
       return;
     manager_->GetURLLoaderFactory()->CreateLoaderAndStart(
-        std::move(receiver), request_id, options, url_request,
+        std::move(receiver), request_id, options, BRAVE_CREATE_LOADER_AND_START,
         std::move(client), traffic_annotation);
   }
 
