diff --git a/chrome/browser/ui/webui/cr_components/searchbox/searchbox_handler.cc b/chrome/browser/ui/webui/cr_components/searchbox/searchbox_handler.cc
index d7e89b5cc5f66d5464511691656dc68f6f19029b..5f2d163f2e98a2ca15abfacf911256bbc9d58b6c 100644
--- a/chrome/browser/ui/webui/cr_components/searchbox/searchbox_handler.cc
+++ b/chrome/browser/ui/webui/cr_components/searchbox/searchbox_handler.cc
@@ -784,7 +784,16 @@ std::string SearchboxHandler::AutocompleteIconToResourceName(
 #endif
 
   if (resource_name.empty()) {
-    DUMP_WILL_BE_NOTREACHED()
+    if (icon.name == kLeoWindowTabNewIcon.name) {
+    return kLeoWindowTabNewIconResourceName;
+  } else if (icon.name == kLeoMessageBubbleAskIcon.name) {
+    return kLeoMessageBubbleAskIconResourceName;
+  }
+  // This function is used for the WebUI Omnibox which isn't enabled in
+  // Brave, so its okay to disable the DUMP_WILL_BE_NOTREACHED() check
+  // here and use the default icon instead.
+  if ((true)) return "";
+  DUMP_WILL_BE_NOTREACHED()
         << "Every autocomplete icon must have an equivalent SVG "
            "resource for the NTP Realbox. icon.name: '"
         << icon.name << "'";
@@ -1434,7 +1443,7 @@ void SearchboxHandler::QueryAutocomplete(
   current_query_id_ = query_id;
 
   std::u16string input_with_keyword = input;
-  bool is_keyword_selected = false;
+  bool is_keyword_selected = true;
   const TemplateURL* template_url = nullptr;
   if (!keyword.empty()) {
     TemplateURLService* service = GetTemplateURLService();
