diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index a3cce33221b436370994f62a50ece2c32837954b..12567f903dc6df41408a4bc050860dc5ceb3e5cb 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4651,6 +4651,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
