diff --git a/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc b/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc
index 403e25405ab54a8a6efe87fa8acb64689c670a8a..ba5e731a98b5750402444503e2fce0d540e6da5a 100644
--- a/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc
+++ b/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc
@@ -105,6 +105,8 @@ class MockPage : public tab_search::mojom::Page {
   MOCK_METHOD(void, TabUpdated, (tab_search::mojom::TabUpdateInfoPtr));
   MOCK_METHOD(void, TabsRemoved, (tab_search::mojom::TabsRemovedInfoPtr));
   MOCK_METHOD(void, TabUnsplit, ());
+  MOCK_METHOD(void, TabFocusShowFREChanged, (bool show));
+  MOCK_METHOD(void, TabFocusEnabledChanged, (bool enabled));
 };
 
 void ExpectNewTab(const tab_search::mojom::Tab* tab,
