diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 65f05c133c7d319ad2fdbbcac5d0a6d28abf05a2..dd1529e1705986b9685f05e3753993b989634f3a 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -624,6 +624,7 @@ class Browser : public TabStripModelObserver,
   // dragging.
   void SetIsInTabDragging(bool is_in_tab_dragging);
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class FullscreenControllerInteractiveTest;
