diff --git a/chrome/browser/ui/views/side_panel/side_panel_coordinator.h b/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
index fdd5da5959cdff9b83346e101fd32c2303d21f3f..48104430cfbd9904e9e073a48a20b99a49300d8c 100644
--- a/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
+++ b/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
@@ -52,7 +52,7 @@ class View;
 // registry's active_entry() then global registry's. These values are reset when
 // the side panel is closed and |last_active_global_entry_id_| is used to
 // determine what entry is seen when the panel is reopened.
-class SidePanelCoordinator final : public SidePanelUIBase,
+class SidePanelCoordinator : public SidePanelUIBase,
                                    public views::ViewObserver,
                                    public PinnedToolbarActionsModel::Observer,
                                    public ToolbarActionsModel::Observer {
@@ -71,11 +71,6 @@ class SidePanelCoordinator final : public SidePanelUIBase,
               SidePanelUtil::SidePanelOpenTrigger open_trigger) override;
   void OpenInNewTab() override;
   void UpdatePinState() override;
-  std::optional<SidePanelEntry::Id> GetCurrentEntryId() const override;
-  int GetCurrentEntryDefaultContentWidth() const override;
-  bool IsSidePanelShowing() const override;
-  bool IsSidePanelEntryShowing(
-      const SidePanelEntry::Key& entry_key) const override;
   void SetNoDelaysForTesting(bool no_delays_for_testing) override;

   // Returns the web contents in a side panel if one exists.
@@ -103,7 +98,6 @@ class SidePanelCoordinator final : public SidePanelUIBase,
   SidePanelEntry* GetLoadingEntryForTesting() const;

   // SidePanelUIBase:
-  using SidePanelUIBase::IsSidePanelEntryShowing;
   using SidePanelUIBase::Show;
   void Close(bool suppress_animations) override;
   void Show(const UniqueKey& entry,
