diff --git a/chrome/browser/shell_integration.h b/chrome/browser/shell_integration.h
index 93a9a2ecc8f7158afbe1a93157e87505619efb29..308207bd4ee376a7dc5bd6ec3fad2410f4ece6f7 100644
--- a/chrome/browser/shell_integration.h
+++ b/chrome/browser/shell_integration.h
@@ -293,6 +293,7 @@ class DefaultBrowserWorker : public DefaultWebClientWorker {
 
   static void DisableSetAsDefaultForTesting();
 
+  BRAVE_DEFAULT_BROWSER_WORKER_H
  protected:
   ~DefaultBrowserWorker() override;
 
