diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 2922768ced039ca591f088d6deb76ab67b6b9664..a236331ef43fb480ba8bb615eceda4401e6d5762 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -651,6 +651,7 @@ static_library("ui") {
     "//ui/webui/resources/cr_components/most_visited:mojom",
     "//v8:v8_version",
   ]
+  deps+= [ "//brave/browser/ui" ]
 
   # TODO(crbug/925153): Remove this circular dependency.
   allow_circular_includes_from = [
@@ -3800,10 +3801,13 @@ static_library("ui") {
       ]
       deps += [ "//google_update" ]
     } else {
-      sources += [
-        "webui/help/version_updater_basic.cc",
-        "webui/help/version_updater_basic.h",
-      ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [
+          "webui/help/version_updater_basic.cc",
+          "webui/help/version_updater_basic.h",
+        ]
+      }
     }
   } else {  # 'OS!="win"
     # The Cast Linux build sets toolkit_views to false, but many files in this
