diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 81c72c714785801548b4e15266768a96cfc69e18..ca4da00610e39e5b7c7dcd3b0235d505d41a7df7 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -308,6 +308,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data_deps = []
 
   if (is_chromeos_lacros && also_build_ash_chrome) {
