diff --git a/chrome/elevation_service/BUILD.gn b/chrome/elevation_service/BUILD.gn
index 09fc1d0d8f03af5a4158716cb40d1f0324a031f4..2c1e9603c9e07ff7233b101471039c46efad3517 100644
--- a/chrome/elevation_service/BUILD.gn
+++ b/chrome/elevation_service/BUILD.gn
@@ -11,6 +11,7 @@ import("//testing/test.gni")
 
 midl("elevation_service_idl") {
   sources = [ "elevation_service_idl.idl" ]
+  sources -= [ "elevation_service_idl.idl" ] sources += ["//brave/chromium_src/chrome/elevation_service/elevation_service_idl.idl" ]
 
   writes_tlb = true
 }
@@ -79,6 +80,7 @@ source_set("lib") {
     "crypt32.lib",
     "rpcrt4.lib",
   ]
+  deps += brave_elevation_service_lib_deps
 }
 
 source_set("service_lib") {
