diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index 233e0a48c98a1be970c83b1df0d11c04696b584a..fbdfc2fd3bcc5613755bfb75e0f59be2b50f7fb9 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -975,6 +975,7 @@ void BookmarkBarView::Layout() {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   // Only render the tab group separtor if there are groups and bookmarks
   // visible. Must done after the bookmark buttons are rendered.
