diff --git a/chrome/test/views/accessibility_checker.cc b/chrome/test/views/accessibility_checker.cc
index 58e149ba642ca43cbc5e1b6aa2cb0d60a8e84372..04320ff652e8950d97229a65e0a3cede723536ef 100644
--- a/chrome/test/views/accessibility_checker.cc
+++ b/chrome/test/views/accessibility_checker.cc
@@ -44,6 +44,7 @@ bool DoesViewHaveAccessibleNameOrLabelError(ui::AXNodeData* data) {
   // 2) Explicitly setting the name to "" is allowed if the view uses
   // AXNodedata.SetNameExplicitlyEmpty().
 
+  data->SetNameExplicitlyEmpty();
   // It has a name, we're done.
   if (!data->GetStringAttribute(StringAttribute::kName).empty())
     return false;
