diff --git a/chrome/browser/net/system_network_context_manager.cc b/chrome/browser/net/system_network_context_manager.cc
index 2fc099db479ac4bd8bd28292e30c24090d50f498..5aedd716485c29d8b310f142f716b00a5ab7fd85 100644
--- a/chrome/browser/net/system_network_context_manager.cc
+++ b/chrome/browser/net/system_network_context_manager.cc
@@ -478,7 +478,7 @@ class SystemNetworkContextManager::URLLoaderFactoryForSystem
     if (!manager_)
       return;
     manager_->GetURLLoaderFactory()->CreateLoaderAndStart(
-        std::move(receiver), request_id, options, url_request,
+        std::move(receiver), request_id, options, BRAVE_CREATE_LOADER_AND_START,
         std::move(client), traffic_annotation);
   }
 
