diff --git a/chrome/browser/resources/history/history.ts b/chrome/browser/resources/history/history.ts
index df7ce3f1cfb1adf3cad7f58cdfffb655e4547f3a..ccba80a32d4a9db2c9158dd737059cfab614e2a9 100644
--- a/chrome/browser/resources/history/history.ts
+++ b/chrome/browser/resources/history/history.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './app.js';
 
 export {BrowserProxyImpl as ShoppingBrowserProxyImpl} from 'chrome://resources/cr_components/commerce/browser_proxy.js';
