diff --git a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts
index 4115e474ac5a26badab3abf693455ced334ef8a8..d9da3566027b30fc0246d1d3087baa9756bb6036 100644
--- a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts
+++ b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.ts
@@ -131,6 +131,7 @@ export class ProfilePickerMainViewElement extends
     if (!isProfileCreationAllowed()) {
       this.$.addProfile.style.display = 'none';
     }
+    this.$.addProfile.style.borderRadius = '8px';
 
     this.addEventListener('view-enter-finish', this.onViewEnterFinish_);
 
