diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.h b/chrome/browser/ui/views/location_bar/location_bar_view.h
index f41b3198ffa02cd4d92bfe5d123f0de011396599..f4fcbb53069b9900ffc8be3cdefecbd05ab1527c 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.h
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.h
@@ -113,6 +113,7 @@ class LocationBarView : public LocationBar,
   int GetBorderRadius() const;
 
   // Initializes the LocationBarView.
+  virtual
   void Init();
 
   // True if this instance has been initialized by calling Init, which can only
@@ -182,6 +183,7 @@ class LocationBarView : public LocationBar,
 
   // Updates the controller, and, if |contents| is non-null, restores saved
   // state that the tab holds.
+  virtual
   void Update(const content::WebContents* contents);
 
   // Clears the location bar's state for |contents|.
@@ -201,6 +203,7 @@ class LocationBarView : public LocationBar,
   void GetAccessibleNodeData(ui::AXNodeData* node_data) override;
   gfx::Size GetMinimumSize() const override;
   gfx::Size CalculatePreferredSize() const override;
+  void Layout(views::View* trailing_view);
   void Layout() override;
   void OnThemeChanged() override;
   void ChildPreferredSizeChanged(views::View* child) override;
@@ -248,9 +251,11 @@ class LocationBarView : public LocationBar,
   SkColor GetLocationIconInkDropColor() const override;
 
   // Gets the theme color tint for the location bar and results.
+  virtual
   OmniboxTint CalculateTint() const;
 
  private:
+  friend class BraveLocationBarView;
   FRIEND_TEST_ALL_PREFIXES(SecurityIndicatorTest, CheckIndicatorText);
   FRIEND_TEST_ALL_PREFIXES(TouchLocationBarViewBrowserTest,
                            OmniboxViewViewsSize);
