diff --git a/chrome/browser/media/encrypted_media_supported_types_browsertest.cc b/chrome/browser/media/encrypted_media_supported_types_browsertest.cc
index 16fdfa1245b4981f55b7ee190a802a25fb0476cb..4f0935c3723324e33b5b684824acd3affe816c83 100644
--- a/chrome/browser/media/encrypted_media_supported_types_browsertest.cc
+++ b/chrome/browser/media/encrypted_media_supported_types_browsertest.cc
@@ -517,7 +517,11 @@ class EncryptedMediaSupportedTypesTest : public InProcessBrowserTest {
     // capabilities which won't indicate support when running the tests.
     // TODO(crbug.com/40226210): Fix this so that we can inject HEVC support on
     // Windows.
-    EXPECT_UNSUPPORTED(hevc_supported);
+    if (media::FFmpegSupportsHEVC()) {
+      EXPECT_ECK_PROPRIETARY(hevc_supported);
+    } else {
+      EXPECT_UNSUPPORTED(hevc_supported);
+    }
 #else
     // On other platforms, HEVC support should be available if
     // kPlatformHEVCDecoderSupport is enabled.
