diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index f5298f79a74fbd79033b666e42d8306ed631ee1a..c4fc8fa0c78b014d70f6d609392075134946201b 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3451,6 +3451,7 @@ const FeatureEntry::FeatureVariation kPasswordNotesAuthValidityVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
