diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index 8c5e577eb89cf1ee85e76fea6385f7a18a60143b..1371973b470fc370d51f93cb86875434d8a27a44 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -619,6 +619,7 @@ Browser* Browser::Create(const CreateParams& params) {
   // not possible, e.g. using the wrong profile or during shutdown. The caller
   // should handle this; see e.g. crbug.com/1141608 and crbug.com/1261628.
   CHECK_EQ(CreationStatus::kOk, GetCreationStatusForProfile(params.profile));
+  BRAVE_BROWSER_CREATE
   return new Browser(params);
 }
 
@@ -629,7 +630,7 @@ Browser::Browser(const CreateParams& params)
       window_(nullptr),
       tab_strip_model_delegate_(
           std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
-      tab_strip_model_(std::make_unique<TabStripModel>(
+      tab_strip_model_(std::make_unique<BraveTabStripModel>(
           tab_strip_model_delegate_.get(),
           params.profile,
           params.are_tab_groups_enabled ? TabGroupModelFactory::GetInstance()
@@ -3852,6 +3853,7 @@ bool Browser::ShouldShowBookmarkBar() const {
   if (!active_tab || !active_tab->GetContents()) {
     return false;
   }
+  BRAVE_BROWSER_SHOULD_SHOW_BOOKMARK_BAR
 
   bookmarks::BookmarkModel* bookmark_model =
       BookmarkModelFactory::GetForBrowserContext(
