diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index 2bb6d397511b50cf81605b53bceb648273e6546c..2daf0eeebd105ce4a5372cf469656c60f7842f44 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -106,6 +106,7 @@ grit("generated_resources") {
   if (is_android) {
     outputs += android_generated_java_resources
   }
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
