diff --git a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
index 6e1a4932b2e4b7770bb502bf0d4e3b2d08369b18..ddbbae983c0146f001dd13fb43941af05504bdfe 100644
--- a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
+++ b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
@@ -1544,7 +1544,13 @@ void TabDragController::AttachToNewContext(
 
       const bool pinned = detached_tab_collection->get()->pinned_;
 
-      if (std::holds_alternative<std::unique_ptr<tabs::TabGroupTabCollection>>(
+      if (std::holds_alternative<
+              std::unique_ptr<tabs::TreeTabNodeTabCollection>>(
+              detached_tab_collection->get()->collection_)) {
+        collection_indices =
+            attached_context_->GetTabStripModel()->InsertDetachedTreeTabNodeAt(
+                std::move(*detached_tab_collection), index);
+      } else if (std::holds_alternative<std::unique_ptr<tabs::TabGroupTabCollection>>(
               detached_tab_collection->get()->collection_)) {
         collection_indices =
             attached_context_->GetTabStripModel()->InsertDetachedTabGroupAt(
