diff --git a/chrome/installer/util/shell_util.cc b/chrome/installer/util/shell_util.cc
index 87a2c70b6cb1a0153c1afae935f316a6bb2710f9..40722d7674ea83ce7e3fe7e3431e014528e44427 100644
--- a/chrome/installer/util/shell_util.cc
+++ b/chrome/installer/util/shell_util.cc
@@ -364,6 +364,7 @@ void GetChromeProgIdEntries(
   app_info.delegate_clsid = install_static::GetLegacyCommandExecuteImplClsid();
 
   GetProgIdEntries(app_info, entries);
+  BRAVE_GET_CHROME_PROG_ID_ENTRIES
 
   if (!app_info.delegate_clsid.empty()) {
     auto delegate_execute_entries =
@@ -459,6 +460,7 @@ void GetShellIntegrationEntries(
 
   const std::wstring html_prog_id(GetBrowserProgId(suffix));
   for (int i = 0; ShellUtil::kPotentialFileAssociations[i] != nullptr; i++) {
+    BRAVE_GET_SHELL_INTEGRATION_ENTRIES
     entries->push_back(std::make_unique<RegistryEntry>(
         capabilities + L"\\FileAssociations",
         ShellUtil::kPotentialFileAssociations[i], html_prog_id));
@@ -483,6 +485,7 @@ void GetAppExtRegistrationEntries(
   std::wstring key_name =
       base::StrCat({ShellUtil::kRegClasses, kFilePathSeparator, ext,
                     kFilePathSeparator, ShellUtil::kRegOpenWithProgids});
+  BRAVE_GET_APP_EXT_REGISTRATION_ENTRIES
   entries->push_back(
       std::make_unique<RegistryEntry>(key_name, prog_id, std::wstring()));
 }
@@ -1524,8 +1527,10 @@ const wchar_t* ShellUtil::kPotentialFileAssociations[] = {
     L".htm", L".html",  L".pdf",  L".shtml", L".svg",
     L".xht", L".xhtml", L".webp", nullptr};
 const wchar_t* ShellUtil::kBrowserProtocolAssociations[] = {L"http", L"https",
+                                                            BRAVE_BROWSER_PROTOCOL_ASSOCIATIONS
                                                             nullptr};
 const wchar_t* ShellUtil::kPotentialProtocolAssociations[] = {
+    BRAVE_POTENTIAL_PROTOCOL_ASSOCIATIONS
     L"http", L"https", L"irc",   L"mailto", L"mms", L"news",   L"nntp",
     L"sms",  L"smsto", L"snews", L"tel",    L"urn", L"webcal", nullptr};
 const wchar_t* ShellUtil::kRegUrlProtocol = L"URL Protocol";
