diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index ade95f3592fc54e69ec8717c7542ef2eae645056..91a70733ca020f95ed36c4e37baeab89e65604f3 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -316,6 +316,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data_deps = []
 
   if (enable_pdf) {
@@ -5093,7 +5094,7 @@ config("disable_thinlto_cache_flags") {
 }
 
 config("pdb_larger_than_4gb") {
-  if (is_win && symbol_level == 2 && use_thin_lto) {
+  if (is_win) {
     # These binaries create PDBs larger than 4 GiB. Increasing the PDB page
     # size allows larger PDBs, but not all tools can handle such large PDBs
     # yet.
