diff --git a/chrome/browser/ui/views/toolbar/app_menu.h b/chrome/browser/ui/views/toolbar/app_menu.h
index a1492f99d0e440387cc7da51314addec9af81ba0..663bd4068ae94bef593380b0ce069c04db53ddac 100644
--- a/chrome/browser/ui/views/toolbar/app_menu.h
+++ b/chrome/browser/ui/views/toolbar/app_menu.h
@@ -31,7 +31,7 @@ class MenuRunner;
 }
 
 // AppMenu adapts the AppMenuModel to view's menu related classes.
-class AppMenu final : public views::MenuDelegate,
+class AppMenu : public views::MenuDelegate,
                       public bookmarks::BaseBookmarkModelObserver,
                       public GlobalErrorObserver {
  public:
