diff --git a/chrome/test/base/testing_browser_process.cc b/chrome/test/base/testing_browser_process.cc
index 833ef49d4160e1b0043d4bc65de4ccb3cd3923ba..a03024b3e93e01ef9f6964ce2fbb0e9446fc5122 100644
--- a/chrome/test/base/testing_browser_process.cc
+++ b/chrome/test/base/testing_browser_process.cc
@@ -155,15 +155,18 @@ void TestingBrowserProcess::CreateInstance() {
         std::move(fake_geolocation_system_permission_manager));
   }
 #endif  // BUILDFLAG(OS_LEVEL_GEOLOCATION_PERMISSION_SUPPORTED)
+  BRAVE_TESTING_BROWSER_PROCESS_CREATE_INSTANCE
 }
 
 // static
 void TestingBrowserProcess::DeleteInstance() {
+  BRAVE_TESTING_BROWSER_PROCESS_DELETE_INSTANCE
   delete g_browser_process;
 }
 
 // static
 void TestingBrowserProcess::TearDownAndDeleteInstance() {
+  BRAVE_TESTING_BROWSER_PROCESS_TEAR_DOWN_AND_DELETE_INSTANCE
   TestingBrowserProcess::DeleteInstance();
 }
 
