diff --git a/chrome/common/importer/profile_import.mojom b/chrome/common/importer/profile_import.mojom
index 625b5ddec268292acf11f3c060c07c336ca1317f..6461206ff781150ed59c66cc8ae8dffafbb5b30f 100644
--- a/chrome/common/importer/profile_import.mojom
+++ b/chrome/common/importer/profile_import.mojom
@@ -6,6 +6,7 @@ module chrome.mojom;
 
 import "components/autofill/content/common/autofill_types.mojom";
 import "mojo/public/mojom/base/string16.mojom";
+import "services/network/public/mojom/cookie_manager.mojom";
 import "url/mojom/url.mojom";
 
 const string kProfileImportServiceName = "profile_import";
@@ -64,6 +65,8 @@ interface ProfileImportObserver {
   OnAutofillFormDataImportStart(uint32 total_autofill_form_data_entry_count);
   OnAutofillFormDataImportGroup(
       array<ImporterAutofillFormDataEntry> autofill_form_data_entry_group);
+  OnCookiesImportStart(uint32 total_cookies_count);
+  OnCookiesImportGroup(array<network.mojom.CanonicalCookie> cookies_group);
   // Windows only:
   OnIE7PasswordReceived(ImporterIE7PasswordInfo importer_password_info);
 };
