diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 85ef19d4664393bf185ee09f8ba02c68b5f86c41..78c76a8000f5873aed64ca120329aea03a5a2e7d 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -411,6 +411,8 @@ test("browser_tests") {
     "//third_party/blink/public:buildflags",
   ]
 
+  deps += [ "//brave/test:brave_browser_tests_deps", ]
+
   data_deps = [
     "//testing/buildbot/filters:browser_tests_filters",
     "//tools/media_engagement_preload:generator",
