diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 7544d04546cdb9d7e9987adb9fd24e6b8018c00f..85e240a91dafa7647622312461282cb41fb6c576 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -5005,6 +5005,7 @@ const FeatureEntry::FeatureVariation kSeamlessSigninPromoTypes[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
