diff --git a/chrome/browser/tab_ui/android/BUILD.gn b/chrome/browser/tab_ui/android/BUILD.gn
index b070e2cdc2a53fd9deb18fbb1a2c8a0edcc26130..aa2a08256773bac27d153e96e37982e64ded3c98 100644
--- a/chrome/browser/tab_ui/android/BUILD.gn
+++ b/chrome/browser/tab_ui/android/BUILD.gn
@@ -23,6 +23,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/tab_ui/TabUiThemeUtils.java",
     "java/src/org/chromium/chrome/browser/tab_ui/ThumbnailProvider.java",
   ]
+  sources += brave_tab_ui_sources
   deps = [
     ":java_resources",
     "//base:base_java",
