diff --git a/chrome/android/java/src/org/chromium/chrome/browser/util/FeatureUtilities.java b/chrome/android/java/src/org/chromium/chrome/browser/util/FeatureUtilities.java
index ee8d2cf102a85b446092ef87759ca13e72f8f867..340db4f2dbe9549f13db25a460d96a8889ebd64f 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/util/FeatureUtilities.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/util/FeatureUtilities.java
@@ -378,7 +378,7 @@ public class FeatureUtilities {
      */
     public static boolean isBottomToolbarEnabled() {
         // TODO(crbug.com/944228): TabGroupsAndroid and ChromeDuet are incompatible for now.
-        return isFlagEnabled(BOTTOM_TOOLBAR_ENABLED_KEY, false)
+        return ChromePreferenceManager.getInstance().isBottomToolbarEnabled()
                 && !DeviceFormFactor.isNonMultiDisplayContextOnTablet(
                         ContextUtils.getApplicationContext())
                 && (isDuetTabStripIntegrationAndroidEnabled() || !isTabGroupsAndroidEnabled());
@@ -423,7 +423,7 @@ public class FeatureUtilities {
      *         enabled).
      */
     public static boolean isNightModeAvailable() {
-        return isFlagEnabled(NIGHT_MODE_AVAILABLE_KEY, false);
+        return isFlagEnabled(NIGHT_MODE_AVAILABLE_KEY, true);
     }
 
     /**
