diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index c64007c87236e1813a2223a5f23a075a08be01a4..6b3aed3070b77aa8c306497f49f5e542c8fed394 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -92,6 +92,7 @@ source_set("channel_info") {
   } else if (is_posix) {
     sources += [ "channel_info_posix.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -281,6 +282,7 @@ static_library("common_lib") {
     public_deps += [ "//components/cdm/common" ]
   }
 
+  import("//brave/common/sources.gni") sources += brave_chrome_common_sources deps += brave_chrome_common_deps public_deps += brave_chrome_common_public_deps
   if (enable_extensions_core) {
     public_deps += [
       "//chrome/common/extensions",
@@ -473,6 +475,7 @@ static_library("non_code_constants") {
   if (is_win) {
     sources += [ "chrome_icon_resources_win.h" ]
   }
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
@@ -578,6 +581,7 @@ static_library("constants") {
     "//third_party/widevine/cdm:buildflags",
   ]
 
+  import("//brave/common/sources.gni") deps += brave_chrome_constants_deps
   if (is_android) {
     sources += [ "chrome_paths_android.cc" ]
 
