diff --git a/chrome/common/extensions/api/developer_private.webidl b/chrome/common/extensions/api/developer_private.webidl
index f86ad741b99daa51d489b60daa0a991a54df9652..3c0f4cdfc6ffeaffbb83eb7bc4e9509277671816 100644
--- a/chrome/common/extensions/api/developer_private.webidl
+++ b/chrome/common/extensions/api/developer_private.webidl
@@ -226,6 +226,7 @@ dictionary ExtensionInfo {
   required AccessModifier incognitoAccess;
   required AccessModifier userScriptsAccess;
   required boolean incognitoAccessPendingChange;
+  required boolean isSplitMode;
   required sequence<DOMString> installWarnings;
   required boolean isCommandRegistrationHandledExternally;
   DOMString launchUrl;
