diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 2f2ac276d6b0da6c4898da4ea3d6ef2bc8230232..91eaf53c9a5fa4e844a878066b6d991009294f7f 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -64,4 +64,5 @@ source_set("configs") {
       ]
     }
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
