diff --git a/chrome/browser/ui/omnibox/omnibox_edit_model.cc b/chrome/browser/ui/omnibox/omnibox_edit_model.cc
index 8c8543c5fa22ece9e2e8c2b99015cf49d9224b6a..07e38a3cd15c39e901dbb0316f3e3ea9b6a33452 100644
--- a/chrome/browser/ui/omnibox/omnibox_edit_model.cc
+++ b/chrome/browser/ui/omnibox/omnibox_edit_model.cc
@@ -657,6 +657,7 @@ void OmniboxEditModel::StartAutocomplete(bool has_selected_text,
     input_.set_lens_overlay_suggest_inputs(*suggest_inputs);
   }
 
+  BRAVE_OMNIBOX_EDIT_MODEL_START_AUTOCOMPLETE
   controller_->StartAutocomplete(input_);
 }
 
