diff --git a/chrome/browser/ui/views/permissions/chooser_bubble_ui.cc b/chrome/browser/ui/views/permissions/chooser_bubble_ui.cc
index bbfbf51354797eac7afb24256d58e14fa198e412..a36d664f333d346698424a643682cef399f50ff3 100644
--- a/chrome/browser/ui/views/permissions/chooser_bubble_ui.cc
+++ b/chrome/browser/ui/views/permissions/chooser_bubble_ui.cc
@@ -119,6 +119,7 @@ ChooserBubbleUiViewDelegate::ChooserBubbleUiViewDelegate(
           GetChooserAnchorConfiguration(browser).anchor,
           contents),
       fullscreen_blocker_(std::move(fullscreen_blocker)) {
+  [&]() -> void {
   // ------------------------------------
   // | Chooser bubble title             |
   // | -------------------------------- |
@@ -155,6 +156,8 @@ ChooserBubbleUiViewDelegate::ChooserBubbleUiViewDelegate(
   SetCloseCallback(
       base::BindOnce(&DeviceChooserContentView::Close,
                      base::Unretained(device_chooser_content_view_)));
+  }();
+  SetFootnoteView(device_chooser_content_view_->CreateFootnoteView(browser));
 }
 
 ChooserBubbleUiViewDelegate::~ChooserBubbleUiViewDelegate() = default;
