diff --git a/chrome/browser/chrome_content_browser_client.h b/chrome/browser/chrome_content_browser_client.h
index baa07eff48e20c93099e3574c4b265f538c3341e..90697f09fd61893c7327db693d34e5c9f167bac3 100644
--- a/chrome/browser/chrome_content_browser_client.h
+++ b/chrome/browser/chrome_content_browser_client.h
@@ -693,6 +693,7 @@ class ChromeContentBrowserClient : public content::ContentBrowserClient {
   }
 
  private:
+  friend class BraveContentBrowserClient;
   friend class DisableWebRtcEncryptionFlagTest;
   friend class InProcessBrowserTest;
 
