diff --git a/chrome/browser/extensions/extension_management.h b/chrome/browser/extensions/extension_management.h
index 85d6e3667179896e3b778bd14c51bf4daf779878..0301bb1a5efaa85ae6f68783cafb4e0e34efc904 100644
--- a/chrome/browser/extensions/extension_management.h
+++ b/chrome/browser/extensions/extension_management.h
@@ -264,6 +264,7 @@ class ExtensionManagement : public KeyedService,
   }
 
  private:
+  friend class BraveExtensionManagement;
   using SettingsIdMap =
       base::flat_map<ExtensionId,
                      std::unique_ptr<internal::IndividualSettings>>;
