diff --git a/chrome/installer/setup/BUILD.gn b/chrome/installer/setup/BUILD.gn
index 6e30d38a04639c92f5c8c7eb7d34dad34481b007..6096e351761bfb5914833febe34faaf39dd65427 100644
--- a/chrome/installer/setup/BUILD.gn
+++ b/chrome/installer/setup/BUILD.gn
@@ -129,6 +129,7 @@ if (is_win) {
         "channel_override_work_item.h",
       ]
     }
+    import("//brave/chromium_src/chrome/installer/setup/sources.gni") deps += brave_chromium_src_chrome_installer_setup_deps sources += brave_chromium_src_chrome_installer_setup_sources
   }
 
   process_version_rc_template("setup_exe_version") {
