diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
index 92f749c5d24faf25343723aec7d61fd0f82d4037..b0fe866e2489a14d0d86de3e006daf771cd3a73a 100644
--- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
+++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
@@ -317,6 +317,7 @@ public class ToolbarTablet extends ToolbarLayout {
         mFixedHeightBackground.setBackgroundColor(color);
         mLocationBar.updateVisualsForState();
         setToolbarHairlineColor(color);
+        super.onThemeColorChanged(color, shouldAnimate);
 
         // Notify the StatusBarColorController of the toolbar color change. This is to match the
         // status bar's color with the toolbar color when the tab strip is hidden on a tablet.
