diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 9ab4bb62b9f7e2e74e5624ba280aec08a680df2f..2ab3259d94589e7073a962b7cd5ea2d4b6c64d04 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4057,6 +4057,7 @@ const flags_ui::FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
