diff --git a/chrome/browser/renderer_context_menu/render_view_context_menu.cc b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
index 78628f4cca9eb1aa6dce06daf0d6cc0b11e212f4..8b5e065afbd187ae824935953514cf8dee1eac8b 100644
--- a/chrome/browser/renderer_context_menu/render_view_context_menu.cc
+++ b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
@@ -1069,6 +1069,7 @@ void RenderViewContextMenu::AppendLinkItems() {
                                       IDS_CONTENT_CONTEXT_OPENLINKOFFTHERECORD);
     }
 
+    AppendBraveLinkItems();
     AppendOpenWithLinkItems();
 
     // While ChromeOS supports multiple profiles, only one can be open at a
