diff --git a/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/NotificationWrapperBuilderFactory.java b/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/NotificationWrapperBuilderFactory.java
index 28bd4b0b89cc2ff69e36f0c3bc45bc5ff98123f6..b8c651b180ff6fd0d44fd5c617eb6ac9bd8abc85 100644
--- a/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/NotificationWrapperBuilderFactory.java
+++ b/chrome/browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/NotificationWrapperBuilderFactory.java
@@ -71,7 +71,7 @@ public class NotificationWrapperBuilderFactory {
         }
 
         NotificationManagerProxyImpl notificationManagerProxy =
-                new NotificationManagerProxyImpl(context);
+                new BraveNotificationManagerProxyImpl(context);
 
         ChannelsInitializer channelsInitializer = new ChannelsInitializer(notificationManagerProxy,
                 ChromeChannelDefinitions.getInstance(), context.getResources());
