diff --git a/chrome/app/settings_strings.grdp b/chrome/app/settings_strings.grdp
index 601ef6a3d2ea9ee8065d3284d6f98c2e58d56ed7..b4a862e4ca089166ae9a90581e4a33ecd737f61a 100644
--- a/chrome/app/settings_strings.grdp
+++ b/chrome/app/settings_strings.grdp
@@ -3409,6 +3409,9 @@
     <message name="IDS_SETTINGS_IMPORT_AUTOFILL_FORM_DATA_CHECKBOX" desc="Checkbox for importing form data for autofill">
       Autofill form data
     </message>
+    <message name="IDS_SETTINGS_IMPORT_COOKIES_CHECKBOX" desc="Checkbox for importing cookies">
+      Cookies
+    </message>
 
     <message name="IDS_SETTINGS_IMPORT_CHOOSE_FILE" desc="Text for the Choose File on dialog">
       Choose File
