diff --git a/chrome/browser/ui/browser_window/internal/browser_window_features.cc b/chrome/browser/ui/browser_window/internal/browser_window_features.cc
index 4913c63016ccc548269eb5a04e5811b9ff03295c..b2463101a037f84937d58acd3a631001e777ad95 100644
--- a/chrome/browser/ui/browser_window/internal/browser_window_features.cc
+++ b/chrome/browser/ui/browser_window/internal/browser_window_features.cc
@@ -300,7 +300,7 @@ void BrowserWindowFeatures::Init(BrowserWindowInterface* browser) {
           *browser, fullscreen_controller_.get(), app_browser_controller_.get(),
           browser->GetType(),
           browser->GetBrowserForMigrationOnly()->is_trusted_source(),
-          browser->GetUnownedUserDataHost());
+          browser->GetUnownedUserDataHost(), browser->GetFeatures().vertical_tab_controller()->GetWeakPtr());
 
   side_panel_registry_ =
       GetUserDataFactory().CreateInstance<SidePanelRegistry>(*browser, browser);
