diff --git a/chrome/browser/notifications/notification_platform_bridge_android.h b/chrome/browser/notifications/notification_platform_bridge_android.h
index 68d01f04b9c5ec95ad2540b1504c75a963eb2b38..74aaac032f4017975339d0779da1c9a6e843a3ac 100644
--- a/chrome/browser/notifications/notification_platform_bridge_android.h
+++ b/chrome/browser/notifications/notification_platform_bridge_android.h
@@ -104,6 +104,7 @@ class NotificationPlatformBridgeAndroid : public NotificationPlatformBridge {
   static void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry);
 
  private:
+  friend class BraveNotificationPlatformBridgeHelperAndroid;
   // Contains information necessary in order to enable closing notifications
   // that were not created by this instance of the manager. This list may not
   // contain the notifications that have not been interacted with since the last
