diff --git a/chrome/browser/resources/signin/profile_customization/profile_customization_app.html.ts b/chrome/browser/resources/signin/profile_customization/profile_customization_app.html.ts
index 0e6e9d84fb64e65d9d1d5180493524704e6d8bb5..764a174bf946f1410925ac8e7c6f4e5a10f1d6b9 100644
--- a/chrome/browser/resources/signin/profile_customization/profile_customization_app.html.ts
+++ b/chrome/browser/resources/signin/profile_customization/profile_customization_app.html.ts
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import {html} from '//resources/lit/v3_0/lit.rollup.js';
+import {html, nothing} from '//resources/lit/v3_0/lit.rollup.js';
 import type {ProfileCustomizationAppElement} from './profile_customization_app.js';
 
 export function getHtml(this: ProfileCustomizationAppElement) {
