diff --git a/chrome/browser/ui/views/side_panel/side_panel_entry_id.h b/chrome/browser/ui/views/side_panel/side_panel_entry_id.h
index befb50f4b9b9d32543f8fe0edab5a0ebca5fecc2..816dc78d20d725eb4fb6efebb5e176fabf7e0635 100644
--- a/chrome/browser/ui/views/side_panel/side_panel_entry_id.h
+++ b/chrome/browser/ui/views/side_panel/side_panel_entry_id.h
@@ -33,6 +33,7 @@
   V(kSideSearch, kActionSidePanelShowSideSearch, "SideSearch")                \
   V(kLens, kActionSidePanelShowLens, "Lens")                                  \
   V(kAssistant, kActionSidePanelShowAssistant, "Assistant")                   \
+  V(kPlaylist, kActionSidePanelShowPlaylist, "Playlist") V(kChatUI, kActionSidePanelShowChatUI, "ChatUI") \
   V(kAboutThisSite, kActionSidePanelShowAboutThisSite, "AboutThisSite")       \
   V(kCustomizeChrome, kActionSidePanelShowCustomizeChrome, "CustomizeChrome") \
   V(kSearchCompanion, kActionSidePanelShowSearchCompanion, "Companion")       \
