diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/privacy/DoNotTrackSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/privacy/DoNotTrackSettings.java
index 940f2037c16e93e95fb66e5e7433cd7e21dd9501..3f140863b8a1c9ea753c40c0bb380720be303847 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/privacy/DoNotTrackSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/privacy/DoNotTrackSettings.java
@@ -16,7 +16,7 @@ import org.chromium.chrome.browser.settings.SettingsUtils;
 /**
  * Fragment to manage 'Do Not Track' preference and to explain to the user what it does.
  */
-public class DoNotTrackSettings extends PreferenceFragmentCompat {
+public class DoNotTrackSettings extends org.chromium.chrome.browser.settings.BravePreferenceFragment {
     // Must match key in do_not_track_preferences.xml.
     private static final String PREF_DO_NOT_TRACK_SWITCH = "do_not_track_switch";
 
