diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 4154546610dbadf30e7b121250c55e7c1b559ae3..840e56420b657e7ed33b69827b97cb8866e9ec6e 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -412,6 +412,8 @@ test("browser_tests") {
     "//third_party/blink/public:buildflags",
   ]
 
+  deps += [ "//brave/test:brave_browser_tests_deps", ]
+
   data_deps = [
     "//testing/buildbot/filters:browser_tests_filters",
     "//tools/media_engagement_preload:generator",
