diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
index 6404c81091b74413d197986558c4df007b13ea7b..822b62cbc72f0c95637fe88e8caed07f9031225c 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
@@ -154,7 +154,7 @@ import org.chromium.chrome.browser.history.HistoryManagerUtils;
 import org.chromium.chrome.browser.history.HistoryPane;
 import org.chromium.chrome.browser.history.HistoryTabHelper;
 import org.chromium.chrome.browser.homepage.HomepageManager;
-import org.chromium.chrome.browser.hub.DefaultPaneOrderController;
+import org.chromium.chrome.browser.hub.BraveDefaultPaneOrderController;
 import org.chromium.chrome.browser.hub.HubColorMixer.OverviewModeAlphaObserver;
 import org.chromium.chrome.browser.hub.HubLayoutDependencyHolder;
 import org.chromium.chrome.browser.hub.HubManager;
@@ -959,6 +959,14 @@ public class ChromeTabbedActivity extends ChromeActivity implements PreAttachInt
 
                         private void closeIfNoTabsAndHomepageEnabled(
                                 boolean isPendingClosure, boolean shouldRemoveWindowWithZeroTabs) {
+                            if (getTabModelSelector().getTotalTabCount() == 0) {
+                                shouldRemoveWindowWithZeroTabs = shouldRemoveWindowWithZeroTabs && org.chromium.chrome.browser.partnercustomizations.CloseBraveManager.shouldCloseAppWithZeroTabs();
+                            }
+                            closeIfNoTabsAndHomepageEnabled_ChromiumImpl(isPendingClosure, shouldRemoveWindowWithZeroTabs);
+                        }
+
+                        private void closeIfNoTabsAndHomepageEnabled_ChromiumImpl(
+                                boolean isPendingClosure, boolean shouldRemoveWindowWithZeroTabs) {
                             if (getTabModelSelector().getTotalTabCount() == 0) {
                                 if (shouldRemoveWindowWithZeroTabs) {
                                     mMultiInstanceManager.closeChromeWindowIfEmpty(mWindowId);
@@ -1245,7 +1253,7 @@ public class ChromeTabbedActivity extends ChromeActivity implements PreAttachInt
                 new HubProvider(
                         this,
                         getProfileProviderSupplier(),
-                        new DefaultPaneOrderController(),
+                        new BraveDefaultPaneOrderController(),
                         mBackPressManager,
                         getMenuOrKeyboardActionController(),
                         this::getSnackbarManager,
@@ -3808,6 +3816,11 @@ public class ChromeTabbedActivity extends ChromeActivity implements PreAttachInt
                 new TabModelSelectorObserver() {
                     @Override
                     public void onTabStateInitialized() {
+                        onTabStateInitialized_ChromiumImpl();
+                        onTabStateInitializedHandler();
+                    }
+
+                    private void onTabStateInitialized_ChromiumImpl() {
                         if (mMultiInstanceManager != null) {
                             mMultiInstanceManager.onTabStateInitialized();
                         }
