diff --git a/chrome/browser/resources/certificate_manager/certificate_manager.ts b/chrome/browser/resources/certificate_manager/certificate_manager.ts
index ae4ab5203dc2d7f03a5affaabcda810bcea36d5b..a9beb6433874ef875efcd52bd317037290a19d0a 100644
--- a/chrome/browser/resources/certificate_manager/certificate_manager.ts
+++ b/chrome/browser/resources/certificate_manager/certificate_manager.ts
@@ -481,4 +481,3 @@ declare global {
   }
 }
 
-customElements.define(CertificateManagerElement.is, CertificateManagerElement);
