diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 89ca3f5e0c27d8e67e7bca165bc7e79b46a3c284..bcd3fa0a15f3f5c1cf136e993580e33b66eff866 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3552,6 +3552,7 @@ const FeatureEntry::FeatureVariation kVcSegmentationModelVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
