diff --git a/chrome/installer/mini_installer/regkey.cc b/chrome/installer/mini_installer/regkey.cc
index ab0f8ab7acc306556add738c97f31a70bc2724d1..39ea0b50fbea4437ccf7d27f23583554d16a738b 100644
--- a/chrome/installer/mini_installer/regkey.cc
+++ b/chrome/installer/mini_installer/regkey.cc
@@ -89,7 +89,7 @@ LONG OpenClientsKey(HKEY root_key,
   StackString<MAX_PATH> clients_key;
   if (!clients_key.assign(kClientsKeyBase))
     return ERROR_BUFFER_OVERFLOW;
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))
   if (!clients_key.append(app_guid))
     return ERROR_BUFFER_OVERFLOW;
 #endif
@@ -104,7 +104,7 @@ LONG OpenClientStateKey(HKEY root_key,
   StackString<MAX_PATH> client_state_key;
   if (!client_state_key.assign(kClientStateKeyBase))
     return ERROR_BUFFER_OVERFLOW;
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))
   if (!client_state_key.append(app_guid))
     return ERROR_BUFFER_OVERFLOW;
 #endif
