diff --git a/chrome/browser/ui/views/translate/translate_bubble_controller.cc b/chrome/browser/ui/views/translate/translate_bubble_controller.cc
index 02b50951e03e47c62059510ba34f773a871ed208..73bad5c9d4f3a7fb44cd1a5cd7ca8bdc26ae379d 100644
--- a/chrome/browser/ui/views/translate/translate_bubble_controller.cc
+++ b/chrome/browser/ui/views/translate/translate_bubble_controller.cc
@@ -69,7 +69,7 @@ views::Widget* TranslateBubbleController::ShowTranslateBubble(
                                                        std::move(ui_delegate));
   }
 
-  auto translate_bubble_view = std::make_unique<TranslateBubbleView>(
+  auto translate_bubble_view = MAKE_BRAVE_TRANSLATE_BUBBLE_VIEW(
       anchor_view, std::move(model), error_type, web_contents,
       GetOnTranslateBubbleClosedCallback());
   translate_bubble_view_ = translate_bubble_view.get();
