diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 579d2c1b3db0685994994d3f1b941ab0f9922743..e26731d3e2a2d4397fcc44a7482881329ba4c6ed 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2535,6 +2535,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
