diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index bc15705553b5bbc8e49a4231deb2fcb3df27ce3b..95ab0b6231e78302d5cafc873dac9146d3447b53 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -937,6 +937,7 @@ void BookmarkBarView::Layout() {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   // Layout the right side buttons.
   x = max_x + bookmark_bar_button_padding;
