diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 6cf1f8748cab9a9aba1b0843143d6d16389ff7c9..b2248e9ec6bbb02d372b799403f45b3cf5dae78c 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -255,7 +255,7 @@ void ToolbarView::Init() {
 
   std::unique_ptr<AvatarToolbarButton> avatar;
   if (show_avatar_toolbar_button)
-    avatar = std::make_unique<AvatarToolbarButton>(browser_);
+    avatar = std::make_unique<BraveAvatarToolbarButton>(browser_);
 
   auto app_menu_button = std::make_unique<BrowserAppMenuButton>(this);
   app_menu_button->EnableCanvasFlippingForRTLUI(true);
