diff --git a/chrome/browser/ui/views/page_action/action_ids.h b/chrome/browser/ui/views/page_action/action_ids.h
index a5f6c92a1d8cf5d41fbe9af024143cc48401af02..4ec8463ba8ffaca66abbfe8022f366835c7f5505 100644
--- a/chrome/browser/ui/views/page_action/action_ids.h
+++ b/chrome/browser/ui/views/page_action/action_ids.h
@@ -16,6 +16,7 @@ namespace page_actions {
 // the array.
 inline constexpr auto kActionIds = std::to_array<actions::ActionId>({
     kActionAiMode,
+    kActionShowPartitionedStorage,
     kActionSidePanelShowLensOverlayResults,
     kActionLensOverlayHomework,
     kActionShowTranslate,
