diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageLayout.java b/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageLayout.java
index 1eceff519bfa8e7bac64ea65fd58ce4b4db86191..b5256e7cefc87be964a65dd32802f24163f184c7 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageLayout.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageLayout.java
@@ -256,6 +256,7 @@ public class NewTabPageLayout extends LinearLayout implements TileGroup.Observer
                 /* observer = */ this, offlinePageBridge);
 
         int maxRows = 2;
+        maxRows = QueryTileSection.getMaxRowsForMostVisitedTiles(getContext());
         if (searchProviderIsGoogle && QueryTileUtils.isQueryTilesEnabledOnNTP()) {
             maxRows = QueryTileSection.getMaxRowsForMostVisitedTiles(getContext());
         }
