diff --git a/chrome/common/importer/importer_data_types.h b/chrome/common/importer/importer_data_types.h
index ee901d48c0011f01b42cdded99f561acd65cafcf..ad044aa2e50e7922c66c3951abc65aa89692c2c9 100644
--- a/chrome/common/importer/importer_data_types.h
+++ b/chrome/common/importer/importer_data_types.h
@@ -31,7 +31,7 @@ enum ImportItem {
   SEARCH_ENGINES     = 1 << 4,
   HOME_PAGE          = 1 << 5,
   AUTOFILL_FORM_DATA = 1 << 6,
-  ALL                = (1 << 7) - 1  // All the bits should be 1, hence the -1.
+  BRAVE_IMPORT_ITEM
 };
 
 // Information about a profile needed by an importer to do import work.
@@ -111,6 +111,7 @@ enum VisitSource {
   VISIT_SOURCE_FIREFOX_IMPORTED = 1,
   VISIT_SOURCE_IE_IMPORTED = 2,
   VISIT_SOURCE_SAFARI_IMPORTED = 3,
+  BRAVE_VISIT_SOURCE
 };
 
 }  // namespace importer
