diff --git a/chrome/browser/profiles/profile_impl.h b/chrome/browser/profiles/profile_impl.h
index bafe8cf145a4e10d48a5ce2d51802929e494169b..b2328bc1927fc4716a724f6c591e1f332addabb2 100644
--- a/chrome/browser/profiles/profile_impl.h
+++ b/chrome/browser/profiles/profile_impl.h
@@ -168,6 +168,7 @@ class ProfileImpl : public Profile {
   void OnLogin() override;
   void InitChromeOSPreferences() override;
 #endif  // defined(OS_CHROMEOS)
+  BRAVE_PROFILE_IMPL_H_
 
   void SetCreationTimeForTesting(base::Time creation_time) override;
 
