diff --git a/chrome/browser/ui/safety_hub/password_status_check_service.cc b/chrome/browser/ui/safety_hub/password_status_check_service.cc
index 487d4aa5d390427d91ab8ce0efdccfc2c89852b3..c9be957b07c45c8ef70fa16bdd22110bb79228da 100644
--- a/chrome/browser/ui/safety_hub/password_status_check_service.cc
+++ b/chrome/browser/ui/safety_hub/password_status_check_service.cc
@@ -721,6 +721,7 @@ base::TimeDelta PasswordStatusCheckService::GetScheduledPasswordCheckInterval()
 
 base::DictValue PasswordStatusCheckService::GetPasswordCardData(
     bool signed_in) {
+  BRAVE_PASSWORD_STATUS_CHECK_SERVICE_GET_PASSWORD_CARD_DATA
   if (no_passwords_saved()) {
     bool password_saving_allowed = profile_->GetPrefs()->GetBoolean(
         password_manager::prefs::kCredentialsEnableService);
