diff --git a/chrome/browser/browsing_data/BUILD.gn b/chrome/browser/browsing_data/BUILD.gn
index e2bd68a88e0b2e84f9aaa22703ede2877fe724d5..8170d8a321c13b2429ef197f51249576431f8d18 100644
--- a/chrome/browser/browsing_data/BUILD.gn
+++ b/chrome/browser/browsing_data/BUILD.gn
@@ -12,6 +12,7 @@ source_set("constants") {
     "//content/public/browser",
     "//extensions/buildflags",
   ]
+  public_deps = [ "//brave/components/ai_chat/core/common/buildflags" ]
 }
 
 source_set("browsing_data") {
