diff --git a/chrome/browser/ui/extensions/icon_with_badge_image_source.cc b/chrome/browser/ui/extensions/icon_with_badge_image_source.cc
index 898bb779ee37db8415c435706af6e550a5672b22..1dca68613938bc505b2a63d2871a320dd2a11328 100644
--- a/chrome/browser/ui/extensions/icon_with_badge_image_source.cc
+++ b/chrome/browser/ui/extensions/icon_with_badge_image_source.cc
@@ -148,6 +148,7 @@ void IconWithBadgeImageSource::Draw(gfx::Canvas* canvas) {
   gfx::ImageSkiaRep rep = skia.GetRepresentation(canvas->image_scale());
   if (rep.scale() != canvas->image_scale()) {
     skia.AddRepresentation(
+        BRAVE_ICON_WITH_BADGE_IMAGE_SOURCE_DRAW_1
         ScaleImageSkiaRep(rep, extensions::ExtensionAction::ActionIconSize(),
                           canvas->image_scale()));
   }
@@ -158,6 +159,7 @@ void IconWithBadgeImageSource::Draw(gfx::Canvas* canvas) {
       (size().width() - extensions::ExtensionAction::ActionIconSize()) / 2.0);
   int y_offset = std::floor(
       (size().height() - extensions::ExtensionAction::ActionIconSize()) / 2.0);
+  BRAVE_ICON_WITH_BADGE_IMAGE_SOURCE_DRAW_2
   canvas->DrawImageInt(skia, x_offset, y_offset);
 
   // Draw a badge on the provided browser action icon's canvas.
