diff --git a/chrome/browser/ui/cocoa/main_menu_builder.mm b/chrome/browser/ui/cocoa/main_menu_builder.mm
index 2e3f95172aeab65e7a6a01c76cd2d894418c87bc..e846142a7224a932ce84d83ecc3089c48537ef61 100644
--- a/chrome/browser/ui/cocoa/main_menu_builder.mm
+++ b/chrome/browser/ui/cocoa/main_menu_builder.mm
@@ -98,6 +98,8 @@ base::scoped_nsobject<NSMenuItem> BuildFileMenu(
                 Item(IDS_NEW_INCOGNITO_WINDOW_MAC)
                     .command_id(IDC_NEW_INCOGNITO_WINDOW)
                     .remove_if(is_pwa),
+                Item(IDS_NEW_OFFTHERECORD_WINDOW_TOR)
+                    .command_id(IDC_NEW_OFFTHERECORD_WINDOW_TOR),
                 Item(IDS_REOPEN_CLOSED_TABS_MAC)
                     .command_id(IDC_RESTORE_TAB)
                     .remove_if(is_pwa),
