diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 817809f7566a6dc203a9aac8b2dff52497365262..8b82a8056b7338b4e07b55a76dd6d96b79ba479a 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -61,6 +61,7 @@ source_set("configs") {
       "//chromeos/constants",
     ]
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("webui_util") {
