diff --git a/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc b/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc
index 6bee934a63ddc1aabdd312909fda8b93bbf53b22..a4105fd43c6d7fc7c853ce73ded3597086bcc357 100644
--- a/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc
+++ b/chrome/browser/ui/webui/settings/md_settings_localized_strings_provider.cc
@@ -2890,6 +2890,7 @@ void AddLocalizedStrings(content::WebUIDataSource* html_source,
 #endif
   policy_indicator::AddLocalizedStrings(html_source);
 
+  BraveAddLocalizedStrings(html_source, profile);
   html_source->SetJsonPath(kLocalizedStringsFile);
 }
 
