diff --git a/chrome/browser/ui/navigator/browser_navigator_params_utils.cc b/chrome/browser/ui/navigator/browser_navigator_params_utils.cc
index aa6893ff719f244fd758d2117f3b6941e6d84d56..336f21fe1f457e608a7bc1ec3920a4cdf7483050 100644
--- a/chrome/browser/ui/navigator/browser_navigator_params_utils.cc
+++ b/chrome/browser/ui/navigator/browser_navigator_params_utils.cc
@@ -79,7 +79,6 @@ content::NavigationController::LoadURLParams LoadURLParamsFromNavigateParams(
   // |frame_tree_node_id| is invalid for main frame navigations.
   if (params->frame_tree_node_id.is_null() && target_contents) {
     bool force_no_https_upgrade =
-        params->url_typed_with_http_scheme ||
         params->captive_portal_window_type !=
             captive_portal::CaptivePortalWindowType::kNone;
     std::unique_ptr<ChromeNavigationUIData> navigation_ui_data =
