diff --git a/chrome/browser/resources/settings_shared/privacy_page/privacy_page_browser_proxy.ts b/chrome/browser/resources/settings_shared/privacy_page/privacy_page_browser_proxy.ts
index 2e13c777517eae71cbf8da2b4e0d0f4ac3e066bf..f9d60fb20e02f9d224e09ee85461fcc1543e6320 100644
--- a/chrome/browser/resources/settings_shared/privacy_page/privacy_page_browser_proxy.ts
+++ b/chrome/browser/resources/settings_shared/privacy_page/privacy_page_browser_proxy.ts
@@ -54,7 +54,7 @@ export interface SecureDnsSetting {
 }
 
 export interface PrivacyPageBrowserProxy {
-  // <if expr="_google_chrome and not chromeos_ash">
+  // <if expr="True and not chromeos_ash">
   getMetricsReporting(): Promise<MetricsReporting>;
   setMetricsReportingEnabled(enabled: boolean): void;
 
@@ -92,7 +92,7 @@ export interface PrivacyPageBrowserProxy {
 }
 
 export class PrivacyPageBrowserProxyImpl implements PrivacyPageBrowserProxy {
-  // <if expr="_google_chrome and not chromeos_ash">
+  // <if expr="True and not chromeos_ash">
   getMetricsReporting() {
     return sendWithPromise('getMetricsReporting');
   }
