diff --git a/chrome/browser/notifications/BUILD.gn b/chrome/browser/notifications/BUILD.gn
index 984d7e48ed6cd342c40fe5c05203df06b48fa114..e5c7f6a4cedf55534db23d3593a62ad113430080 100644
--- a/chrome/browser/notifications/BUILD.gn
+++ b/chrome/browser/notifications/BUILD.gn
@@ -82,6 +82,7 @@ if (is_android) {
       "//third_party/androidx:androidx_core_core_java",
       "//ui/android:ui_no_recycler_view_java",
     ]
+    deps += [ "//brave/browser/notifications/android:brave_java", ]
 
     srcjar_deps = [ ":notifications_java_enum_srcjar" ]
     resources_package = "org.chromium.chrome.browser.notifications"
