diff --git a/chrome/browser/extensions/extension_management.h b/chrome/browser/extensions/extension_management.h
index 457ac3b12c28e9646d1a07fcc0e677e23300d29d..7295bf70056cced59a583b973c412c4d1f7ad393 100644
--- a/chrome/browser/extensions/extension_management.h
+++ b/chrome/browser/extensions/extension_management.h
@@ -207,6 +207,7 @@ class ExtensionManagement : public KeyedService,
       const ExtensionId& extension_id);
 
  private:
+  friend class BraveExtensionManagement;
   using SettingsIdMap =
       base::flat_map<ExtensionId,
                      std::unique_ptr<internal::IndividualSettings>>;
