diff --git a/chrome/browser/ui/webui/downloads/downloads_ui.cc b/chrome/browser/ui/webui/downloads/downloads_ui.cc
index 08b737fdcc4524c650b5ad3d154e59b530f06275..fed73c59557dc49892349d2cee3beada16aafadc 100644
--- a/chrome/browser/ui/webui/downloads/downloads_ui.cc
+++ b/chrome/browser/ui/webui/downloads/downloads_ui.cc
@@ -65,6 +65,7 @@ namespace {
 content::WebUIDataSource* CreateAndAddDownloadsUIHTMLSource(Profile* profile) {
   content::WebUIDataSource* source = content::WebUIDataSource::CreateAndAdd(
       profile, chrome::kChromeUIDownloadsHost);
+  BRAVE_CREATE_DOWNLOADS_UI_HTML_SOURCE
   webui::SetupWebUIDataSource(source, kDownloadsResources,
                               IDR_DOWNLOADS_DOWNLOADS_HTML);
 
