diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index 2aaa938c7e14a03b6528aa087ff6109f8ac335eb..fe05954a7da4de3ed329a7b88d1c4ad66550fe7d 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import addBraveRoutes from './brave_routes.js'
 import {assert} from 'chrome://resources/js/assert.js';
 
 import {loadTimeData} from './i18n_setup.js';
@@ -306,6 +307,7 @@ function createRoutes(): SettingsRoutes {
         loadTimeData.getString('performancePageTitle'));
   }
 
+  addBraveRoutes(r);
   return r as unknown as SettingsRoutes;
 }
 
@@ -339,7 +341,7 @@ export let routes: SettingsRoutes = Router.getInstance().getRoutes();
 // Returns the "effective" route when at chrome://settings/.
 export function getTopLevelRoute(): Route {
   if (!loadTimeData.getBoolean('isGuest')) {
-    return routes.PEOPLE;
+    return routes.GET_STARTED;
   }
 
   let guestTopLevelRoute = routes.SEARCH;
