diff --git a/chrome/browser/ui/views/profiles/profile_menu_view.h b/chrome/browser/ui/views/profiles/profile_menu_view.h
index 0fce060877846fb499e835947d97f5d2720af75d..3b1f0c0de20fe27a224aafc16ffd0b64fffe5940 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view.h
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.h
@@ -72,6 +72,7 @@ class ProfileMenuView : public ProfileMenuViewBase {
   }
 
  private:
+  BRAVE_PROFILE_MENU_VIEW_H
   friend class ProfileMenuViewExtensionsTest;
   friend class ProfileMenuViewSigninPendingTest;
   friend class ProfileMenuViewSignoutTest;
