diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
index abbab2664254c9be1ec5074463722c1365590cd0..65fd0676cc38f853ac93eba9ae8eb57d75eb5687 100644
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -184,6 +184,10 @@ if (!is_android && !is_mac) {
           "common/crash_keys.cc",
           "common/crash_keys.h",
         ]
+        if (brave_chromium_build) {
+          sources -= [ "app/chrome_exe.rc", ]
+          sources += [ "//brave/app/brave_exe.rc", ]
+        }
 
         deps += [
           ":chrome_dll",
@@ -617,6 +621,11 @@ if (is_win) {
     ]
   }
 
+if (brave_chromium_build) {
+  group("chrome_app") {
+    public_deps = [ "//brave:chrome_app", ]
+  }
+} else {
   mac_app_bundle("chrome_app") {
     output_name = chrome_product_full_name
 
@@ -653,6 +662,7 @@ if (is_win) {
                   rebase_path("app/app.exports", root_build_dir) ]
     }
   }
+}
 
   compiled_action("chrome_app_strings") {
     tool = "//chrome/tools/build/mac:infoplist_strings_tool"
@@ -682,7 +692,7 @@ if (is_win) {
     args =
         [
           "-b",
-          "${branding_path_component}_strings",
+          _strings_file,
           "-v",
           rebase_path(chrome_version_file, root_build_dir),
           "-g",
@@ -1274,6 +1284,10 @@ if (is_win) {
     if (is_chrome_branded) {
       deps += [ ":default_apps" ]
     }
+    if (brave_chromium_build) {
+      deps += [ "//brave:framework_bundle_data", ]
+      if (defined(chrome_framework_version)) { framework_contents += [ "Frameworks", ] }
+    }
 
     ldflags = [
       "-Wl,-install_name,@executable_path/../Versions/$chrome_version_full/$chrome_framework_name.framework/$chrome_framework_name",
@@ -1433,6 +1447,7 @@ if (is_win) {
 
 group("browser_dependencies") {
   public_deps = [
+    "//brave:browser_dependencies",
     "//chrome/browser",
     "//chrome/common",
     "//components/sync",
@@ -1470,6 +1485,7 @@ group("browser_dependencies") {
 
 group("child_dependencies") {
   public_deps = [
+    "//brave:child_dependencies",
     "//chrome/browser/devtools",
     "//chrome/child",
     "//chrome/common",
@@ -1492,7 +1508,7 @@ group("child_dependencies") {
 if (is_win) {
   process_version_rc_template("chrome_exe_version") {
     sources = [
-      "app/chrome_exe.ver",
+      "//brave/app/brave_exe.ver",
     ]
     output = "$target_gen_dir/chrome_exe_version.rc"
   }
@@ -1565,6 +1581,7 @@ group("resources") {
     "//chrome/browser:resources",
     "//chrome/common:resources",
     "//chrome/renderer:resources",
+    "//brave:resources",
   ]
 }
 
