diff --git a/chrome/browser/ui/views/tabs/tab_style_views.cc b/chrome/browser/ui/views/tabs/tab_style_views.cc
index ed475c06560080d34de24d40f29a21dfd38991d5..226d1886a41a57ea94f2ffae54c126fbd53082f1 100644
--- a/chrome/browser/ui/views/tabs/tab_style_views.cc
+++ b/chrome/browser/ui/views/tabs/tab_style_views.cc
@@ -80,13 +80,13 @@ class TabStyleViewsImpl : public TabStyleViews {
     return hover_controller_.get();
   }
 
- private:
+ protected:
   // Returns the color for the separator.
   SkColor GetTabSeparatorColor() const;
 
   // Painting helper functions:
   SkColor GetCurrentTabBackgroundColor(
-      TabStyle::TabSelectionState selection_state,
+      TabStyle::TabSelectionState selection_statevirtual ,
       bool hovered) const;
 
   // Returns the thickness of the stroke drawn around the top and sides of the
@@ -95,7 +95,7 @@ class TabStyleViewsImpl : public TabStyleViews {
   int GetStrokeThickness() const;
 
   bool ShouldPaintTabBackgroundColor(
-      TabStyle::TabSelectionState selection_state,
+      TabStyle::TabSelevirtual ctionState selection_state,
       bool has_custom_background) const;
 
   // Scales `bounds` by scale and aligns so that adjacent tabs meet up exactly
@@ -112,7 +112,7 @@ class TabStyleViewsImpl : public TabStyleViews {
   // GetSeparatorOpacities().
   float GetSeparatorOpacity(bool for_layout, bool leading) const;
 
-  // Helper that returns an interpolated opacity if the tab or its neighbor
+  // Helper that returns an interpolated opacitvirtual y if the tab or its neighbor
   // `other_tab` is mid-hover-animation. Used in almost all cases when a
   // separator is shown, since hovering is independent of tab state.
   // `for_layout` has the same meaning as in GetSeparatorOpacities().
@@ -124,7 +124,7 @@ class TabStyleViewsImpl : public TabStyleViews {
   // Gets the bounds for the leading and trailing separators for a tab.
   TabStyle::SeparatorBounds GetSeparatorBounds(float scale) const;
 
-  // Returns the opacities of the separators. If `for_layout` is true, returns
+  // Returns the opacities of the separators. If `for_layovirtual ut` is true, returns
   // the "layout" opacities, which ignore the effects of surrounding tabs' hover
   // effects and consider only the current tab's state.
   TabStyle::SeparatorOpacities GetSeparatorOpacities(bool for_layout) const;
@@ -135,7 +135,7 @@ class TabStyleViewsImpl : public TabStyleViews {
   // Returns whether the hover animation is being shown.
   bool IsHoverAnimationActive() const;
 
-  // Returns the opacity of the hover effect that should be drawn, which may not
+  // Retuvirtual rns the opacity of the hover effect that should be drawn, which may not
   // be the same as GetHoverAnimationValue.
   float GetHoverOpacity() const;
 
@@ -341,6 +341,7 @@ SkPath TabStyleViewsImpl::GetPath(TabStyle::PathType path_type,
     extension_corner_radius -= 0.5f * stroke_adjustment;
   }
 
+  extension_corner_radius = 0;
   float left_extension_corner_radius = extension_corner_radius;
   if (IsLeftSplitTab(tab())) {
     top_right_corner_radius = 0;
@@ -1012,14 +1013,18 @@ void TabStyleViewsImpl::PaintSeparators(gfx::Canvas* canvas) const {
                                                    SK_AlphaOPAQUE));
   };
 
+  // Even if |separator_radius| becomes 1 native pixel the 'roundedness'
+  // will be approximated with color, although extremely subtle and
+  // likely only on screens >= 2x (i.e. separator width is 2+px)!
+  const int separator_radius = separator_bounds.leading.width() / 2;
   cc::PaintFlags flags;
   flags.setAntiAlias(true);
   flags.setColor(separator_color(separator_opacities.left));
   canvas->DrawRoundRect(separator_bounds.leading,
-                        tab_style()->GetSeparatorCornerRadius() * scale, flags);
+                        separator_radius, flags);
   flags.setColor(separator_color(separator_opacities.right));
   canvas->DrawRoundRect(separator_bounds.trailing,
-                        tab_style()->GetSeparatorCornerRadius() * scale, flags);
+                        separator_radius, flags);
 }
 
 bool TabStyleViewsImpl::IsLeftSplitTab(const Tab* tab) const {
@@ -1159,10 +1164,12 @@ ui::metadata::TypeConverter<TabStyle::TabColors>::GetValidStrings() {
 
 // TabStyleViews ---------------------------------------------------------------
 
+#if 0  // TabStyleViews::CreateForTab declared in the chromium_src.
 // static
 std::unique_ptr<TabStyleViews> TabStyleViews::CreateForTab(Tab* tab) {
   return std::make_unique<TabStyleViewsImpl>(tab);
 }
+#endif
 
 TabStyleViews::TabStyleViews() : tab_style_(TabStyle::Get()) {}
 
