diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
index be8466ad87791880f1f3c4ea61db11c466eb3356..b59176c958e2f26cc2d33fe89e48ed9322c05c14 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
@@ -172,7 +172,7 @@ import java.util.Locale;
  * This is the main activity for ChromeMobile when not running in document mode.  All the tabs
  * are accessible via a chrome specific tab switching UI.
  */
-public class ChromeTabbedActivity extends ChromeActivity implements ScreenshotMonitorDelegate {
+public class ChromeTabbedActivity extends BraveActivity implements ScreenshotMonitorDelegate {
     /**
      * The results of a system back press action.
      */
@@ -965,13 +965,13 @@ public class ChromeTabbedActivity extends ChromeActivity implements ScreenshotMo
     }
 
     @Override
-    public ChromeTabCreator getTabCreator(boolean incognito) {
-        return (ChromeTabCreator) super.getTabCreator(incognito);
+    public org.chromium.chrome.browser.tabmodel.BraveTabCreator getTabCreator(boolean incognito) {
+        return (org.chromium.chrome.browser.tabmodel.BraveTabCreator) super.getTabCreator(incognito);
     }
 
     @Override
-    public ChromeTabCreator getCurrentTabCreator() {
-        return (ChromeTabCreator) super.getCurrentTabCreator();
+    public org.chromium.chrome.browser.tabmodel.BraveTabCreator getCurrentTabCreator() {
+        return (org.chromium.chrome.browser.tabmodel.BraveTabCreator) super.getCurrentTabCreator();
     }
 
     @Override
@@ -1659,13 +1659,13 @@ public class ChromeTabbedActivity extends ChromeActivity implements ScreenshotMo
     }
 
     @Override
-    protected Pair<ChromeTabCreator, ChromeTabCreator> createTabCreators() {
+    protected Pair<org.chromium.chrome.browser.tabmodel.BraveTabCreator, org.chromium.chrome.browser.tabmodel.BraveTabCreator> createTabCreators() {
         Supplier<TabDelegateFactory> tabDelegateFactorySupplier = () -> {
             return new TabbedModeTabDelegateFactory(this);
         };
-        return Pair.create(new ChromeTabCreator(this, getWindowAndroid(), getStartupTabPreloader(),
+        return Pair.create(new org.chromium.chrome.browser.tabmodel.BraveTabCreator(this, getWindowAndroid(), getStartupTabPreloader(),
                                    tabDelegateFactorySupplier, false),
-                new ChromeTabCreator(this, getWindowAndroid(), getStartupTabPreloader(),
+                new org.chromium.chrome.browser.tabmodel.BraveTabCreator(this, getWindowAndroid(), getStartupTabPreloader(),
                         tabDelegateFactorySupplier, true));
     }
 
@@ -2091,7 +2091,7 @@ public class ChromeTabbedActivity extends ChromeActivity implements ScreenshotMo
         }
     }
 
-    private void hideOverview() {
+    public void hideOverview() {
         assert (mOverviewModeController.overviewVisible());
         Layout activeLayout = mLayoutManager.getActiveLayout();
         if (activeLayout instanceof StackLayout) {
