diff --git a/chrome/browser/profiles/profile_impl.cc b/chrome/browser/profiles/profile_impl.cc
index 97eafbff4400f74bcec284c2d28f0b38e88d1bf0..bb486e84b600ba20248fb189aad6a7b7809f53f2 100644
--- a/chrome/browser/profiles/profile_impl.cc
+++ b/chrome/browser/profiles/profile_impl.cc
@@ -363,7 +363,7 @@ std::unique_ptr<Profile> Profile::CreateProfile(const base::FilePath& path,
     NOTREACHED();
   }
 
-  std::unique_ptr<Profile> profile = base::WrapUnique(new ProfileImpl(
+  std::unique_ptr<Profile> profile = base::WrapUnique(new BraveProfileImpl(
       path, delegate, create_mode, creation_time, io_task_runner));
   return profile;
 }
