diff --git a/chrome/browser/ui/color/BUILD.gn b/chrome/browser/ui/color/BUILD.gn
index be4803fdbc2dc14f9aa0547015a679ab286b2b9e..5754cf328b34f0ec64919250d89b50adaf7695c3 100644
--- a/chrome/browser/ui/color/BUILD.gn
+++ b/chrome/browser/ui/color/BUILD.gn
@@ -38,6 +38,7 @@ source_set("mixers") {
     "//ui/color:color",
     "//ui/color:mixers",
   ]
+  sources += brave_browser_ui_color_mixers_sources deps += brave_browser_ui_color_mixers_deps
 
   if (is_chromeos) {
     sources += [ "chromeos/native_chrome_color_mixer_chromeos.cc" ]
