diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
index 25dadc14072cfa41abda8810517ad8398d087318..366a103922e6689e2f68ecc5ec01f4c7819b0c32 100644
--- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
+++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
@@ -258,6 +258,7 @@ public class ToolbarTablet extends ToolbarLayout {
         mLocationBar.getTabletCoordinator().tintBackground(textBoxColor);
         mLocationBar.updateVisualsForState();
         setToolbarHairlineColor(color);
+        super.onThemeColorChanged(color, shouldAnimate);
 
         // Notify the StatusBarColorController of the toolbar color change. This is to match the
         // status bar's color with the toolbar color when the tab strip is hidden on a tablet.
