diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 3c2b3527ca9c4af91cf8c3edc356c374ca3fb39c..ff136856a889d7b0e3eeb70c87e63b91bc8c0ab1 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2478,6 +2478,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps defines += brave_chrome_browser_defines
 
   if (is_android) {
     sources += [
