diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 5a5a48ef70c66b8314abde3d50a03208465f2d66..782166722ea75cb59fa467edd75c549a89bad078 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3569,6 +3569,7 @@ const FeatureEntry::FeatureVariation kVcSegmentationModelVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
