diff --git a/chrome/browser/preferences/BUILD.gn b/chrome/browser/preferences/BUILD.gn
index b520755f694fd7c82b0e643fc62d6de01d43a5d7..9a77b7780ac5be34d5bbbc9ea0adfc8152ee9923 100644
--- a/chrome/browser/preferences/BUILD.gn
+++ b/chrome/browser/preferences/BUILD.gn
@@ -26,6 +26,7 @@ android_library("java") {
   ]
   annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
   srcjar_deps = [ ":java_pref_names_srcjar" ]
+  sources += brave_browser_preferences_java_sources
 }
 
 # TODO(chouinard,estade): Consider breaking this target up into a different target for each pref file.
