diff --git a/chrome/browser/permissions/permission_manager.h b/chrome/browser/permissions/permission_manager.h
index 223f67e98f95ce3d5c4ff9e69d4894daf992d933..1f5928b4fd2447cc894ae62334ba702662e393dd 100644
--- a/chrome/browser/permissions/permission_manager.h
+++ b/chrome/browser/permissions/permission_manager.h
@@ -120,6 +120,7 @@ class PermissionManager : public KeyedService,
   bool IsPermissionKillSwitchOn(ContentSettingsType);
 
  private:
+  friend class BravePermissionManager;
   friend class PermissionManagerTest;
   friend class GeolocationPermissionContextTests;
 
