diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
index 795a84107a8665a7795a14cba33687d5e5e9f460..4396a998f1e9999a0e59f5b2a72bf1a2c70aec68 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
@@ -782,6 +782,7 @@ public class ToolbarManager implements ScrimObserver, ToolbarTabController, UrlF
         };
 
         final OnClickListener shareButtonListener = v -> {
+/*
             recordBottomToolbarUseForIPH();
             RecordUserAction.record("MobileBottomToolbarShareButton");
             boolean isIncognito = false;
@@ -789,6 +790,7 @@ public class ToolbarManager implements ScrimObserver, ToolbarTabController, UrlF
                 isIncognito = mTabModelSelector.getCurrentTab().isIncognito();
             }
             mActivity.onShareMenuItemSelected(false, isIncognito);
+*/          mActivity.addOrEditBookmark(mActivity.getActivityTab());
         };
 
         mBottomControlsCoordinator = new BottomControlsCoordinator(mActivity.getFullscreenManager(),
@@ -1772,6 +1774,7 @@ public class ToolbarManager implements ScrimObserver, ToolbarTabController, UrlF
         boolean editingAllowed = currentTab == null || mBookmarkBridge == null
                 || mBookmarkBridge.isEditBookmarksEnabled();
         mToolbar.updateBookmarkButton(isBookmarked, editingAllowed);
+        if (mBottomControlsCoordinator != null) {mBottomControlsCoordinator.updateBookmarkButton(isBookmarked, editingAllowed);}
     }
 
     private void updateReloadState(boolean tabCrashed) {
