diff --git a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc
index f259f0d7f12e2f12f6baf79652b363fe05d7f3a7..275466f0569320b7041ec95daaca28c61329e336 100644
--- a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc
+++ b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc
@@ -1642,6 +1642,7 @@ const char* ChromeBrowsingDataRemoverDelegate::GetHistogramSuffix(
       return "WebrtcVideoPerfHistory";
     case TracingDataType::kMediaDeviceSalts:
       return "MediaDeviceSalts";
+    CHROME_BROWSING_DATA_REMOVER_DELEGATE_GET_HISTORY_SUFFIX
   }
 }
 
