diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 8cbbd53abc7c7843dc2846d1bc17f5be8522d133..c4d0fe2c119fbdc59b86f79afba60da7131d69e1 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3283,6 +3283,7 @@ const FeatureEntry::FeatureVariation kPrivacySandboxSettings4Variations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
