diff --git a/chrome/browser/ui/messages/android/BUILD.gn b/chrome/browser/ui/messages/android/BUILD.gn
index ca802e1c9a7667b9b9302e9c42e64ba3bed9b9fd..16a5670af9a26ccf544bb26e5ded52cdffb3585b 100644
--- a/chrome/browser/ui/messages/android/BUILD.gn
+++ b/chrome/browser/ui/messages/android/BUILD.gn
@@ -31,6 +31,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarView.java",
   ]
 
+  sources += brave_browser_ui_messages_java_sources
   deps = [
     ":java_resources",
     "//base:base_java",
