diff --git a/chrome/browser/download/download_target_determiner.cc b/chrome/browser/download/download_target_determiner.cc
index 728c3ab89a592d10d2fe646c90dac0c087b8d485..56940202cd6904e6b83d3ae0001721671fd5dc36 100644
--- a/chrome/browser/download/download_target_determiner.cc
+++ b/chrome/browser/download/download_target_determiner.cc
@@ -1348,6 +1348,7 @@ DownloadFileType::DangerLevel DownloadTargetDeterminer::GetDangerLevel(
       download_->GetDownloadSource() != download::DownloadSource::DRAG_AND_DROP;
   if (HasPromptedForPath() ||
       confirmation_reason_ != DownloadConfirmationReason::NONE ||
+      BRAVE_DOWNLOAD_TARGET_DETERMINER_GET_DANGER_LEVEL
       user_approved_path) {
     return DownloadFileType::NOT_DANGEROUS;
   }
@@ -1387,6 +1388,7 @@ DownloadFileType::DangerLevel DownloadTargetDeterminer::GetDangerLevel(
         ui::PAGE_TRANSITION_FROM_ADDRESS_BAR) != 0 ||
        (download_->HasUserGesture() && visits == VISITED_REFERRER)))
     return DownloadFileType::NOT_DANGEROUS;
+  BRAVE_DOWNLOAD_TARGET_DETERMINER_GET_DANGER_LEVEL2
   return danger_level;
 }
 
