diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index 0f6308ea44ec5c8f8a76f9fd96e5387c79174db4..f11d95950a95084046ba1eab7831f2274e1dbd1a 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -251,7 +251,7 @@ grit("google_chrome_strings") {
 }
 
 grit("chromium_strings") {
-  source = "chromium_strings.grd"
+  source = "${branding_path_component}_strings.grd"
   defines = chrome_grit_defines
   output_dir = "$root_gen_dir/chrome"
   outputs = [
