diff --git a/chrome/browser/ui/views/page_info/page_info_view_factory.cc b/chrome/browser/ui/views/page_info/page_info_view_factory.cc
index 816daf0795f6342ff9a92a836cb18e2a05da3c13..7b771047c8be357d5b169c618b3e603ce2bd4663 100644
--- a/chrome/browser/ui/views/page_info/page_info_view_factory.cc
+++ b/chrome/browser/ui/views/page_info/page_info_view_factory.cc
@@ -489,6 +489,7 @@ const ui::ImageModel PageInfoViewFactory::GetPermissionIcon(
 
   icon = &gfx::VectorIcon::EmptyIcon();
   switch (permission.type) {
+    BRAVE_PAGE_INFO_VIEW_FACTORY_GET_PERMISSION_ICON
     case ContentSettingsType::COOKIES:
       icon = &vector_icons::kDatabaseOldIcon;
       break;
