diff --git a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
index 7d3ea02bde07f3585d6c3ec832f83e081251cabc..4d0173e1fdc02f4454d8cd028a3b71febaf3d659 100644
--- a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
+++ b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
@@ -47,6 +47,7 @@ class TabHoverCardBubbleView : public views::BubbleDialogDelegateView {
   void ClearPreviewImage();
   void SetPreviewImage(gfx::ImageSkia preview_image);
 
+  BRAVE_TAB_HOVER_CARD_BUBBLE_VIEW_H_
  private:
   friend class TabHoverCardBubbleViewBrowserTest;
   friend class TabHoverCardBubbleViewInteractiveUiTest;
