diff --git a/chrome/browser/ui/cocoa/new_tab_button_cocoa.mm b/chrome/browser/ui/cocoa/new_tab_button_cocoa.mm
index 767e405a0c4d5142fa1099968c6e1d0e66a6d201..f10353e69d4e1242595a38d8a937144cc3022690 100644
--- a/chrome/browser/ui/cocoa/new_tab_button_cocoa.mm
+++ b/chrome/browser/ui/cocoa/new_tab_button_cocoa.mm
@@ -25,7 +25,7 @@
   DARKEN,
 };
 
-const NSSize newTabButtonImageSize = {34, 18};
+const NSSize newTabButtonImageSize = {19, 19};
 
 NSImage* GetMaskImageFromCell(NewTabButtonCell* aCell) {
   return [aCell imageForState:image_button_cell::kDefaultState view:nil];
@@ -346,34 +346,30 @@ - (NSImage*)imageForState:(image_button_cell::ButtonState)state
 + (NSBezierPath*)newTabButtonBezierPathWithLineWidth:(CGFloat)lineWidth {
   NSBezierPath* bezierPath = [NSBezierPath bezierPath];
 
-  // This data comes straight from the SVG.
-  [bezierPath moveToPoint:NSMakePoint(15.2762236,30)];
+  const CGFloat closeBtnWidth = 35;
+  const CGFloat closeBtnHeight = 30;
+  const CGFloat closeBtnCornerRadius = 2;
 
-  [bezierPath curveToPoint:NSMakePoint(11.0354216,27.1770115)
-             controlPoint1:NSMakePoint(13.3667706,30)
-             controlPoint2:NSMakePoint(11.7297681,28.8344828)];
+  [bezierPath moveToPoint:NSMakePoint(0, 0 + closeBtnCornerRadius)];
 
-  [bezierPath curveToPoint:NSMakePoint(7.28528951e-08,2.01431416)
-             controlPoint1:NSMakePoint(11.0354216,27.1770115)
-             controlPoint2:NSMakePoint(0.000412425082,3.87955717)];
+  [bezierPath curveToPoint:NSMakePoint(0 + closeBtnCornerRadius, 0)
+              controlPoint1:NSMakePoint(0, 0)
+              controlPoint2:NSMakePoint(0, 0)];
 
-  [bezierPath curveToPoint:NSMakePoint(1.70510791,0)
-             controlPoint1:NSMakePoint(-0.000270516213,0.790325707)
-             controlPoint2:NSMakePoint(0.753255356,0)];
+  [bezierPath lineToPoint:NSMakePoint(closeBtnWidth - closeBtnCornerRadius, 0)];
+  [bezierPath curveToPoint:NSMakePoint(closeBtnWidth, 0 + closeBtnCornerRadius)
+              controlPoint1:NSMakePoint(closeBtnWidth, 0)
+              controlPoint2:NSMakePoint(closeBtnWidth, 0)];
 
-  [bezierPath lineToPoint:NSMakePoint(48.7033642,0)];
+  [bezierPath lineToPoint:NSMakePoint(closeBtnWidth, closeBtnHeight - closeBtnCornerRadius)];
+  [bezierPath curveToPoint:NSMakePoint(closeBtnWidth - closeBtnCornerRadius, closeBtnHeight)
+              controlPoint1:NSMakePoint(closeBtnWidth, closeBtnHeight)
+              controlPoint2:NSMakePoint(closeBtnWidth, closeBtnHeight)];
 
-  [bezierPath curveToPoint:NSMakePoint(52.9464653,2.82643678)
-             controlPoint1:NSMakePoint(50.6151163,0)
-             controlPoint2:NSMakePoint(52.2521188,1.16666667)];
-
-  [bezierPath curveToPoint:NSMakePoint(64.0268555,27.5961914)
-             controlPoint1:NSMakePoint(52.9464653,2.82643678)
-             controlPoint2:NSMakePoint(64.0268555,27.4111339)];
-
-  [bezierPath curveToPoint:NSMakePoint(62.2756294,30)
-             controlPoint1:NSMakePoint(64.0268555,28.5502144)
-             controlPoint2:NSMakePoint(63.227482,29.9977011)];
+  [bezierPath lineToPoint:NSMakePoint(0 + closeBtnCornerRadius, closeBtnHeight)];
+  [bezierPath curveToPoint:NSMakePoint(0, closeBtnHeight - closeBtnCornerRadius)
+              controlPoint1:NSMakePoint(0, closeBtnHeight)
+              controlPoint2:NSMakePoint(0, closeBtnHeight)];
 
   [bezierPath closePath];
 
@@ -431,9 +427,10 @@ + (void)drawNewTabButtonImage:(NewTabButtonCustomImageRep*)imageRep
   CGFloat buttonWidth = newTabButtonImageSize.width;
 
   // Bottom edge.
-  const CGFloat kBottomEdgeX = 9;
+  const CGFloat kBottomEdgeX = 1;
   const CGFloat kBottomEdgeY = 1.2825;
-  const CGFloat kBottomEdgeWidth = 22;
+  const CGFloat kBottomEdgeWidth = 17.5; //35/2 this is just for doubled line at the bottom
+
   NSPoint bottomEdgeStart = NSMakePoint(kBottomEdgeX, kBottomEdgeY);
   NSPoint bottomEdgeEnd = NSMakePoint(kBottomEdgeX + kBottomEdgeWidth,
                                       kBottomEdgeY);
@@ -454,11 +451,11 @@ + (void)drawNewTabButtonImage:(NewTabButtonCustomImageRep*)imageRep
   CGPoint shadowStart = NSZeroPoint;
   CGPoint shadowEnd = NSZeroPoint;
   NSColor* overlayColor = nil;
-  const CGFloat kBottomShadowX = 8;
+  const CGFloat kBottomShadowX = kBottomEdgeX - 1;
   const CGFloat kBottomShadowY = kBottomEdgeY - lineWidth;
   const CGFloat kTopShadowX = 1;
   const CGFloat kTopShadowY = kBottomShadowY + 15;
-  const CGFloat kShadowWidth = 24;
+  const CGFloat kShadowWidth = kBottomEdgeWidth;
   static NSColor* lightOverlayColor =
       [[NSColor colorWithCalibratedWhite:1 alpha:0.20] retain];
   static NSColor* darkOverlayColor =
