diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index 77e3ca5bb02942d0dfe4d9851a1ac57a7d3b4819..60cf7e6cc22aba212be5cd625181e0a69cd6e82f 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -9,6 +9,7 @@ import {pageVisibility} from './page_visibility.js';
 import type {SettingsRoutes} from './router.js';
 import {Route, Router} from './router.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -300,6 +301,7 @@ function createBrowserSettingsRoutes(): SettingsRoutes {
     }
     // </if>
   }
+  addBraveRoutes(r);
   return r as unknown as SettingsRoutes;
 }
 
