diff --git a/chrome/browser/ui/views/extensions/extensions_menu_main_page_view.cc b/chrome/browser/ui/views/extensions/extensions_menu_main_page_view.cc
index 51b0c7aa578e51d839bf02714e915dfaa9275a12..83890a8cd185c38312029d8016495b126a640da4 100644
--- a/chrome/browser/ui/views/extensions/extensions_menu_main_page_view.cc
+++ b/chrome/browser/ui/views/extensions/extensions_menu_main_page_view.cc
@@ -332,6 +332,7 @@ void ExtensionsMenuMainPageView::CreateAndInsertMenuItem(
     ExtensionMenuItemView::SitePermissionsButtonAccess
         site_permissions_button_access,
     int index) {
+  BRAVE_EXTENSION_MENU_MAIN_PAGE_VIEW_CREATE_AND_INSERT_MENU_ITEM
   // base::Unretained() below is safe because `menu_handler_` lifetime is
   // tied to this view lifetime by the extensions menu coordinator.
   auto item = std::make_unique<ExtensionMenuItemView>(
