diff --git a/chrome/browser/net/proxy_config_monitor.h b/chrome/browser/net/proxy_config_monitor.h
index cc48c2dcb98d49ea3737092e21fdf8a4aa52cb5b..d666014662b50db7673a2c4e8f36ac85f7361978 100644
--- a/chrome/browser/net/proxy_config_monitor.h
+++ b/chrome/browser/net/proxy_config_monitor.h
@@ -65,7 +65,7 @@ class ProxyConfigMonitor : public net::ProxyConfigService::Observer,
   // they're received, to allow tests to wait until all pending proxy
   // configuration changes have been applied.
   void FlushForTesting();
-
+  BRAVE_PROXY_CONFIG_MONITOR_H
  private:
   // net::ProxyConfigService::Observer implementation:
   void OnProxyConfigChanged(
