diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 5a32db73d363e04077fdd646d5da9c218a9078bd..2d7846728576372e04e9dee59e89789d49056b97 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -35,7 +35,7 @@ import("java_sources.gni")
 import("static_initializers.gni")
 import("trichrome.gni")
 
-chrome_public_manifest_package = "org.chromium.chrome"
+declare_args() { chrome_public_manifest_package = "org.chromium.chrome" }
 chrome_public_test_manifest_package = "org.chromium.chrome.tests"
 
 chrome_public_jinja_variables =
@@ -136,6 +136,7 @@ android_resources("chrome_app_java_resources") {
     "//third_party/android_deps:com_android_support_preference_v7_java",
     "//third_party/android_deps:com_android_support_recyclerview_v7_java",
   ]
+  deps += ["//brave/build/android:android_brave_strings_grd"]
   custom_package = "org.chromium.chrome"
 }
 
@@ -356,6 +357,7 @@ android_library("chrome_java") {
   ]
 
   deps += feed_deps
+  deps += brave_chrome_java_deps
 
   srcjar_deps = [
     ":chrome_android_java_enums_srcjar",
@@ -405,6 +407,7 @@ android_library("chrome_java") {
     "//net:effective_connection_type_java",
     ":vr_build_config",
   ]
+  srcjar_deps += brave_chrome_java_srcjar_deps
 
   android_manifest_for_lint = chrome_public_android_manifest
 
@@ -507,6 +510,7 @@ group("jni_headers") {
     "//chrome/browser/touch_to_fill/android:jni_headers",
     "//chrome/browser/util:jni_headers",
   ]
+  public_deps += ["//brave/build/android:jni_headers"]
 }
 
 action("chrome_android_java_google_api_keys_srcjar") {
@@ -1395,6 +1399,7 @@ if (current_toolchain == default_toolchain) {
         ":${_variant}_locale_pak_assets",
         ":${_variant}_paks",
       ]
+      deps += [ "//brave:brave_pak_assets" ]
       if (_is_monochrome) {
         deps += [ "//android_webview:locale_pak_assets" ]
         if (webview_includes_weblayer && !_is_bundle_module) {
