diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index 1c4c5252e82a9d9b595cbfd5423dc1ba1e9235e4..be762f8941d9e29b213cb382fbe76b4ae3fbfef3 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -8,6 +8,7 @@ import {pageVisibility} from './page_visibility.js';
 import {Route, Router} from './router.js';
 import {SettingsRoutes} from './settings_routes.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -212,6 +213,7 @@ function createBrowserSettingsRoutes(): SettingsRoutes {
       // </if>
     }
   }
+  addBraveRoutes(r);
   return r;
 }
 
