diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index 4240e3e3cc69b0cc02902ac25bb260c6e0e4987a..15d9b8a6f8251e271ad2406e225bcbcc52ed8b74 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -76,7 +76,7 @@ import org.chromium.ui.widget.ButtonCompat;
 /**
  * Settings fragment to enable Sync and other services that communicate with Google.
  */
-public class SyncAndServicesSettings extends PreferenceFragmentCompat
+public class SyncAndServicesSettings extends org.chromium.chrome.browser.settings.BravePreferenceFragment
         implements PassphraseDialogFragment.Listener, Preference.OnPreferenceChangeListener,
                    ProfileSyncService.SyncStateChangedListener,
                    SettingsActivity.OnBackPressedListener {
