diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index d1b66475ef7443bf98a82d54fa35c3fe2ca2aba2..dccf7a4bbe44be7494db95ae37e4491d2fda667d 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -7660,6 +7660,7 @@ const FeatureEntry kFeatureEntries[] = {
      SINGLE_VALUE_TYPE(switches::kPervasiveSystemAccentColor)},
 #endif
 
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 #if defined(OS_ANDROID)
     {"use-ulp-languages-in-chrome",
      flag_descriptions::kUseULPLanguagesInChromeName,
