diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 696cd2684b082358831adfce85d9b7620b0e83f7..3bed9229593eefe90d8dba51f757fb5aab6c3bd5 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -72,6 +72,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
