diff --git a/chrome/browser/ui/hats/hats_service_desktop.cc b/chrome/browser/ui/hats/hats_service_desktop.cc
index f499ca5631ae7d2c3b3ba9ead416e5eb62ed44e4..b9aeba03c9462d746767dfc144d61718b7c6f2ee 100644
--- a/chrome/browser/ui/hats/hats_service_desktop.cc
+++ b/chrome/browser/ui/hats/hats_service_desktop.cc
@@ -436,6 +436,7 @@ bool HatsServiceDesktop::CanShowSurvey(const std::string& trigger) const {
 
 HatsService::LaunchError HatsServiceDesktop::RunCommonLaunchChecks(
     const std::string& trigger) const {
+  if ((true)) return LaunchError::kError;
   // Survey should not be loaded if the corresponding survey config is
   // unavailable.
   const auto config_iterator = survey_configs_by_triggers_.find(trigger);
