diff --git a/chrome/common/extensions/api/developer_private.idl b/chrome/common/extensions/api/developer_private.idl
index 8b09424e0c4cd409f1693ba06eb69d800f948473..7341c79c9647c07320733829314638d7b7c6c799 100644
--- a/chrome/common/extensions/api/developer_private.idl
+++ b/chrome/common/extensions/api/developer_private.idl
@@ -257,6 +257,7 @@ namespace developerPrivate {
     DOMString id;
     AccessModifier incognitoAccess;
     AccessModifier userScriptsAccess;
+    boolean isSplitMode;
     DOMString[] installWarnings;
     boolean isCommandRegistrationHandledExternally;
     DOMString? launchUrl;
