diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 41583cc15dddd096ace0e5aa72d1e1828ce2cdf2..ca4d1fd0af53942ae115351ba3a2e545d7db8ace 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4333,6 +4333,7 @@ const FeatureEntry::FeatureVariation kServiceWorkerAutoPreloadVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
