diff --git a/chrome/common/importer/profile_import.mojom b/chrome/common/importer/profile_import.mojom
index a61e549f6eacfe8dbe144407a58de0c04054bb22..da56fe8e8fa9329ddb0baf1bdb2657cf5d6c8f79 100644
--- a/chrome/common/importer/profile_import.mojom
+++ b/chrome/common/importer/profile_import.mojom
@@ -6,6 +6,7 @@ module chrome.mojom;
 
 import "components/autofill/core/common/mojom/autofill_types.mojom";
 import "mojo/public/mojom/base/string16.mojom";
+import "services/network/public/mojom/cookie_manager.mojom";
 import "url/mojom/url.mojom";
 
 [Native]
@@ -29,6 +30,27 @@ struct FaviconUsageDataList;
 [Native]
 struct ImporterIE7PasswordInfo;
 
+[Native]
+struct BraveStats;
+
+[Native]
+struct BraveLedger;
+
+[Native]
+struct BraveReferral;
+
+[Native]
+struct ImportedBrowserTab;
+
+[Native]
+struct ImportedBrowserWindow;
+
+[Native]
+struct ImportedWindowState;
+
+[Native]
+struct SessionStoreSettings;
+
 [Native]
 enum ImportItem;
 
@@ -62,6 +84,13 @@ interface ProfileImportObserver {
   OnAutofillFormDataImportStart(uint32 total_autofill_form_data_entry_count);
   OnAutofillFormDataImportGroup(
       array<ImporterAutofillFormDataEntry> autofill_form_data_entry_group);
+  OnCookiesImportStart(uint32 total_cookies_count);
+  OnCookiesImportGroup(array<network.mojom.CanonicalCookie> cookies_group);
+  OnStatsImportReady(BraveStats stats);
+  OnLedgerImportReady(BraveLedger ledger);
+  OnReferralImportReady(BraveReferral referral);
+  OnWindowsImportReady(ImportedWindowState window_state);
+  OnSettingsImportReady(SessionStoreSettings settings);
 };
 
 // This interface is used to control the import process.
