diff --git a/chrome/browser/renderer_context_menu/spelling_options_submenu_observer.h b/chrome/browser/renderer_context_menu/spelling_options_submenu_observer.h
index 573c2178adce19bca86591c84d593497a8e34252..fdd896c8c70bb77a0702544bfc6957c6d33df8b9 100644
--- a/chrome/browser/renderer_context_menu/spelling_options_submenu_observer.h
+++ b/chrome/browser/renderer_context_menu/spelling_options_submenu_observer.h
@@ -15,10 +15,12 @@
 #include "ui/base/models/simple_menu_model.h"
 
 class RenderViewContextMenuProxy;
+class BraveSpellingOptionsSubMenuObserver;
 
 // A class that implements the 'spell-checker options' submenu. This class
 // creates the submenu, adds it to the parent menu, and handles events.
 class SpellingOptionsSubMenuObserver : public RenderViewContextMenuObserver {
+  friend class BraveSpellingOptionsSubMenuObserver;
  public:
   SpellingOptionsSubMenuObserver(RenderViewContextMenuProxy* proxy,
                                  ui::SimpleMenuModel::Delegate* delegate,
