diff --git a/chrome/common/pref_names.h b/chrome/common/pref_names.h
index 9550992285f95a8be26d5ad96a8e1eb9a8f3cc74..0b1eb41559da8d0b4e6c6bf4824f6dccdd5077d5 100644
--- a/chrome/common/pref_names.h
+++ b/chrome/common/pref_names.h
@@ -359,6 +359,7 @@ extern const char kImportDialogBookmarks[];
 extern const char kImportDialogHistory[];
 extern const char kImportDialogSavedPasswords[];
 extern const char kImportDialogSearchEngine[];
+extern const char kImportDialogCookies[];
 
 extern const char kProfileAvatarIndex[];
 extern const char kProfileUsingDefaultName[];
