diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
index a158d0d3652ade968a504548fa220176bea8299b..80f08d4d1b13df0569a24b8a2de4b4da6316cf32 100644
--- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
+++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
@@ -439,10 +439,12 @@ public class ToolbarTablet extends ToolbarLayout
             mOfflineDownloader.downloadPage(getContext(), getToolbarDataProvider().getTab());
             RecordUserAction.record("MobileToolbarDownloadPage");
         }
+        BraveToolbarLayout.class.cast(this).onClickImpl(v);
     }
 
     @Override
     public boolean onLongClick(View v) {
+        if (BraveToolbarLayout.class.cast(this).onLongClickImpl(v)) return true;
         String description = null;
         Context context = getContext();
         Resources resources = context.getResources();
@@ -582,6 +584,7 @@ public class ToolbarTablet extends ToolbarLayout
         mLocationBar.getTabletCoordinator().tintBackground(textBoxColor);
         mLocationBar.updateVisualsForState();
         setToolbarHairlineColor(color);
+        super.onThemeColorChanged(color, shouldAnimate);
 
         // Notify the StatusBarColorController of the toolbar color change. This is to match the
         // status bar's color with the toolbar color when the tab strip is hidden on a tablet.
