diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index fbbea351bd8a..08d29ba877a5 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -745,6 +745,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
@@ -4583,7 +4584,7 @@ test("unit_tests") {
     }
   }
 
-  if (enable_extensions) {
+  if (!is_android && enable_extensions) {
     sources += [
       "../../apps/saved_files_service_unittest.cc",
       "../../tools/json_schema_compiler/test/features_generation_unittest.cc",
