diff --git a/chrome/browser/prefs/browser_prefs.cc b/chrome/browser/prefs/browser_prefs.cc
index 812bb38a0c53e8759138fbb34936f2112341f001..0c94fdd6828a7e628c901e1ac80fcf59fb5a1d8b 100644
--- a/chrome/browser/prefs/browser_prefs.cc
+++ b/chrome/browser/prefs/browser_prefs.cc
@@ -1783,6 +1783,7 @@ void RegisterLocalState(PrefRegistrySimple* registry) {
   registry->RegisterBooleanPref(prefs::kQRCodeGeneratorEnabled, true);
 
   // This is intentionally last.
+  brave::RegisterLocalStatePrefs(registry);
   RegisterLocalStatePrefsForMigration(registry);
 }
 
@@ -2223,6 +2224,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry,
   registry->RegisterDictionaryPref(prefs::kReportingEndpoints);
 
   registry->RegisterBooleanPref(prefs::kCompactModeEnabled, false);
+  brave::RegisterProfilePrefs(registry);
 }
 
 void RegisterUserProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
