diff --git a/chrome/browser/permissions/permission_manager.h b/chrome/browser/permissions/permission_manager.h
index 5d7be3ba11e7850feb260ba31c414ac143f5fa68..c6674d6c043bdcf50f364c5a8aafc14336765d9c 100644
--- a/chrome/browser/permissions/permission_manager.h
+++ b/chrome/browser/permissions/permission_manager.h
@@ -134,6 +134,7 @@ class PermissionManager : public KeyedService,
   void Shutdown() override;
 
  private:
+  friend class BravePermissionManager;
   friend class PermissionManagerTest;
   friend class GeolocationPermissionContextTests;
   friend class NfcPermissionContextTests;
