diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 0d81f08e66b1e20f0420b257ac80ca83ca96c23b..314944a94fad4e9e5fddc7c7b1451523cf7e4a4f 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2470,6 +2470,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps defines += brave_chrome_browser_defines
 
   if (is_android) {
     sources += [
