diff --git a/chrome/browser/ui/content_settings/content_setting_image_model.cc b/chrome/browser/ui/content_settings/content_setting_image_model.cc
index afaee36cf39fae81aef937aa9fe3189ba7b1f2f6..c1ada370ccc002eac0a87282896b3dbda0b0db82 100644
--- a/chrome/browser/ui/content_settings/content_setting_image_model.cc
+++ b/chrome/browser/ui/content_settings/content_setting_image_model.cc
@@ -10,6 +10,7 @@
 #include "base/feature_list.h"
 #include "base/macros.h"
 #include "base/metrics/histogram_macros.h"
+#include "brave/browser/ui/content_settings/brave_content_setting_image_models.h"
 #include "build/build_config.h"
 #include "chrome/app/vector_icons/vector_icons.h"
 #include "chrome/browser/browser_process.h"
@@ -668,6 +669,9 @@ ContentSettingImageModel::GenerateContentSettingImageModels() {
   std::vector<std::unique_ptr<ContentSettingImageModel>> result;
   for (auto type : kContentSettingImageOrder)
     result.push_back(CreateForContentType(type));
+#if defined(BRAVE_CHROMIUM_BUILD)
+  BraveGenerateContentSettingImageModels(result);
+#endif
 
   return result;
 }
