diff --git a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_browser_proxy.ts b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_browser_proxy.ts
index f29b70ed4083cb8f0ea499f9e75b7cd409ce959c..a7c4391d37f9e246ed5c9088181d98ec0aef0f27 100644
--- a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_browser_proxy.ts
+++ b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_browser_proxy.ts
@@ -43,6 +43,9 @@ export enum BrowsingDataType {
   DOWNLOADS = 6,
   HOSTED_APPS_DATA = 7,
   // TABS = 8, Not used on Desktop.
+// <if expr="enable_ai_chat">
+  BRAVE_AI_CHAT = 9,
+// </if>
 }
 // LINT.ThenChange(/components/browsing_data/core/browsing_data_utils.h:BrowsingDataType)
 
