diff --git a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
index a81964b2b0ea8b409a8ec66d44b603fbc2319328..318a72f6d494ca4882764e212657dc7cfbb15510 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/contextmenu/ChromeContextMenuPopulator.java
@@ -218,6 +218,7 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             Action.PRINT_PAGE,
             Action.INSPECT_ELEMENT,
             Action.SHOW_INTEREST_IN_ELEMENT,
+            Action.COPY_CLEAN_LINK,
         })
         @Retention(RetentionPolicy.SOURCE)
         public @interface Action {
@@ -270,7 +271,7 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             // int RELOAD = 46;  Deprecated since 05/2025.
             int INSPECT_ELEMENT = 47;
             int SHOW_INTEREST_IN_ELEMENT = 48;
-            int NUM_ENTRIES = 49;
+            int COPY_CLEAN_LINK = 49;int NUM_ENTRIES = 50;
         }
 
         // LINT.ThenChange(/tools/metrics/histograms/enums.xml:ContextMenuOptionAndroid)
@@ -468,6 +469,7 @@ public class ChromeContextMenuPopulator implements ContextMenuPopulator {
             if (!MailTo.isMailTo(mParams.getLinkUrl().getSpec())
                     && !UrlUtilities.isTelScheme(mParams.getLinkUrl())) {
                 linkGroup.add(createListItem(Item.COPY_LINK_ADDRESS));
+                linkGroup.add(createListItem(Item.COPY_CLEAN_LINK));
                 if (!mParams.getLinkText().trim().isEmpty() && !mParams.isImage()) {
                     linkGroup.add(createListItem(Item.COPY_LINK_TEXT));
                 }
