diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java
index b2f712ede9a71f6652b1a9caa409b9cdac807917..fe71aa3cc1aeb4eead5c5a04f01564957cc9d330 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/bottom/BottomControlsCoordinator.java
@@ -222,4 +222,5 @@ public class BottomControlsCoordinator {
         if (mTabGroupUi != null) mTabGroupUi.destroy();
         mMediator.destroy();
     }
+    public void updateBookmarkButton(boolean isBookmarked, boolean editingAllowed) {if (mBottomToolbarCoordinator != null) { mBottomToolbarCoordinator.updateBookmarkButton(isBookmarked, editingAllowed);}}
 }
