diff --git a/chrome/browser/resources/extensions/service.js b/chrome/browser/resources/extensions/service.js
index df646188a53f43dc4151d96effe897b0e55e251c..be341d90f69808b1186876442e1f560acfa4baef 100644
--- a/chrome/browser/resources/extensions/service.js
+++ b/chrome/browser/resources/extensions/service.js
@@ -222,6 +222,12 @@ export class Service {
       incognitoAccess: isAllowedIncognito,
     });
   }
+  setItemAllowedTor(id, isAllowedTor) {
+    chrome.developerPrivate.updateExtensionConfiguration({
+      extensionId: id,
+      torAccess: isAllowedTor,
+    });
+  }
 
   /** @override */
   setItemAllowedOnFileUrls(id, isAllowedOnFileUrls) {
