diff --git a/chrome/browser/ui/views/side_panel/side_panel_web_ui_view.h b/chrome/browser/ui/views/side_panel/side_panel_web_ui_view.h
index 8fee06b8477a14ebcde6a588c16f0776f5cab210..82372be70459d7f57c87406a4290c69c24ce3152 100644
--- a/chrome/browser/ui/views/side_panel/side_panel_web_ui_view.h
+++ b/chrome/browser/ui/views/side_panel/side_panel_web_ui_view.h
@@ -55,6 +55,7 @@ class SidePanelWebUIView : public views::WebView,
   void HideCustomContextMenu() override;
   bool HandleKeyboardEvent(content::WebContents* source,
                            const input::NativeWebKeyboardEvent& event) override;
+  bool HandleContextMenu(content::RenderFrameHost& render_frame_host, const content::ContextMenuParams& params) override;
 
  private:
   base::RepeatingClosure on_show_cb_;
