diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/SettingsNavigationImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/SettingsNavigationImpl.java
index c7cb29e1dd81af3a0f0d00fe2962209e0b594511..2fe75fa91d63f4cd3cd3b0813e8132089cde898b 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/SettingsNavigationImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/SettingsNavigationImpl.java
@@ -18,6 +18,7 @@ import org.chromium.chrome.browser.autofill.settings.FinancialAccountsManagement
 import org.chromium.chrome.browser.autofill.settings.NonCardPaymentMethodsManagementFragment;
 import org.chromium.chrome.browser.browsing_data.ClearBrowsingDataFragment;
 import org.chromium.chrome.browser.flags.ChromeFeatureList;
+import org.chromium.chrome.browser.password_manager.settings.PasswordSettings;
 import org.chromium.chrome.browser.safety_check.SafetyCheckSettingsFragment;
 import org.chromium.chrome.browser.safety_hub.SafetyHubFragment;
 import org.chromium.chrome.browser.sync.settings.GoogleServicesSettings;
@@ -54,6 +55,7 @@ public class SettingsNavigationImpl implements SettingsNavigation {
             case SettingsFragment.PAYMENT_METHODS:
             case SettingsFragment.SITE:
             case SettingsFragment.ACCESSIBILITY:
+            case SettingsFragment.PASSWORDS:
             case SettingsFragment.GOOGLE_SERVICES:
             case SettingsFragment.MANAGE_SYNC:
             case SettingsFragment.FINANCIAL_ACCOUNTS:
@@ -117,6 +119,7 @@ public class SettingsNavigationImpl implements SettingsNavigation {
                 return SiteSettings.class;
             case SettingsFragment.ACCESSIBILITY:
                 return AccessibilitySettings.class;
+            case SettingsFragment.PASSWORDS: return PasswordSettings.class;
             case SettingsFragment.GOOGLE_SERVICES:
                 return GoogleServicesSettings.class;
             case SettingsFragment.MANAGE_SYNC:
