diff --git a/chrome/browser/ui/webui/print_preview/print_preview_ui.cc b/chrome/browser/ui/webui/print_preview/print_preview_ui.cc
index 449f6f2db551ec81fc272fcc2b5abc6d99b2dfcc..e3d271f158d59fb4ffdef547525b3d184733f6cc 100644
--- a/chrome/browser/ui/webui/print_preview/print_preview_ui.cc
+++ b/chrome/browser/ui/webui/print_preview/print_preview_ui.cc
@@ -156,7 +156,7 @@ PrintPreviewRequestIdMap& GetPrintPreviewRequestIdMap() {
 
 // PrintPreviewUI IDMap used to avoid exposing raw pointer addresses to WebUI.
 // Only accessed on the UI thread.
-base::LazyInstance<base::IDMap<PrintPreviewUI*>>::DestructorAtExit
+base::LazyInstance<base::IDMap<mojom::PrintPreviewUI*>>::DestructorAtExit
     g_print_preview_ui_id_map = LAZY_INSTANCE_INITIALIZER;
 
 void AddPrintPreviewStrings(content::WebUIDataSource* source) {
