diff --git a/chrome/browser/ui/views/tabs/common/tab_strip_view.cc b/chrome/browser/ui/views/tabs/common/tab_strip_view.cc
index 07e7de482f5f73e25b07ef0e16f07f24a665cc45..ccecb0c245682df3b60f1b301dfc26c678f7d538 100644
--- a/chrome/browser/ui/views/tabs/common/tab_strip_view.cc
+++ b/chrome/browser/ui/views/tabs/common/tab_strip_view.cc
@@ -312,6 +312,8 @@ void TabStripView::OnChildMoved(TabCollectionNode* moved_node) {
 
   if (is_active_tab || has_focus) {
     switch (moved_node->type()) {
+      case TabCollectionNode::Type::TREE_NODE:
+        break;
       case TabCollectionNode::Type::TAB: {
         const tabs::TabInterface* tab =
             std::get<tabs::ConstDanglingUntriagedTabInterface>(
