diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index a4cd3e850b1561464bfded941c65aa07fa199a75..92dc69194c54d7459675df29da6fb8e3d06ad74e 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -335,6 +335,8 @@ content::WebUIDataSource* CreateExtensionsSource(Profile* profile,
       kUseNewSiteAccessPage,
       base::FeatureList::IsEnabled(features::kExtensionsMenuAccessControl));
 
+  BraveAddExtensionsResources(source);
+
   return source;
 }
 
