diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 7197019ccc35133e75acd42dc3cd6e8f003591a2..55031c55f343495b1832ca78073dd0a6c44885ca 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4372,6 +4372,7 @@ const FeatureEntry::FeatureVariation kMerchantTrustVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
