diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index b0ca538ec58e08704922342e7d2aa90c8f3d5577..c47eaafd6921d910591e2989fb81fdf3c5e9e8f5 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -235,6 +235,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  sources += brave_chrome_renderer_sources public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
