diff --git a/chrome/installer/mini_installer/configuration.cc b/chrome/installer/mini_installer/configuration.cc
index c23d61dfd8376c9f515762abc5e50a518339e17c..454c27ed675748ed5ae893db9cb88e2b5f6115b6 100644
--- a/chrome/installer/mini_installer/configuration.cc
+++ b/chrome/installer/mini_installer/configuration.cc
@@ -43,6 +43,7 @@ Configuration::~Configuration() {
 
 bool Configuration::Initialize() {
   Clear();
+  BRAVE_READ_RESOURCES
   ReadRegistry();
   return ParseCommandLine(::GetCommandLine());
 }
