diff --git a/chrome/android/java/src/org/chromium/chrome/browser/browsing_data/ClearBrowsingDataFragment.java b/chrome/android/java/src/org/chromium/chrome/browser/browsing_data/ClearBrowsingDataFragment.java
index dea7164915a3b3afde38cc3bb9332a248c9f7a48..4b11856e97f7adebc7bb55e55856fac28be864a5 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/browsing_data/ClearBrowsingDataFragment.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/browsing_data/ClearBrowsingDataFragment.java
@@ -348,7 +348,7 @@ public class ClearBrowsingDataFragment extends ChromeBaseSettingsFragment
     }
 
     /** Called when browsing data is about to be cleared. */
-    private void onClearBrowsingData() {
+    protected void onClearBrowsingData() {
         RecordUserAction.record("ClearBrowsingData_AdvancedTab");
 
         if (ChromeFeatureList.sClearBrowsingDataAndroidSurvey.isEnabled()) {
