diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 6251976ee17278149a8e18ff6873b9a72ac2394d..50fc99b50da7b17419ac1f82823b3e1dc44395bf 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1479,6 +1479,7 @@ source_set("extensions") {
       "//components/enterprise/data_controls/core/browser",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 # KeyedService factories initialization.
