diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
index 87b244ecf219f5fe74bba538e32c3bb5bd26d845..16b27e7cdecac47e75eb6cce2d001f1606788013 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
@@ -326,7 +326,7 @@ public abstract class ChromeActivity<C extends ChromeActivityComponent>
     NightModeReparentingController mNightModeReparentingController;
 
     protected ChromeActivity() {
-        mIntentHandler = new IntentHandler(this, createIntentHandlerDelegate());
+        mIntentHandler = new org.chromium.chrome.browser.BraveIntentHandler(this, createIntentHandlerDelegate());
     }
 
     @Override
