diff --git a/chrome/browser/ui/omnibox/omnibox_edit_model.cc b/chrome/browser/ui/omnibox/omnibox_edit_model.cc
index 8a6ba1a7c125ebd4e015e455bd6eadf71436b77a..f22ba89a9274310b4327e627c30a41cb0d010886 100644
--- a/chrome/browser/ui/omnibox/omnibox_edit_model.cc
+++ b/chrome/browser/ui/omnibox/omnibox_edit_model.cc
@@ -709,6 +709,7 @@ void OmniboxEditModel::StartAutocomplete(bool prevent_inline_autocomplete) {
     input_.set_lens_overlay_suggest_inputs(*suggest_inputs);
   }
 
+  BRAVE_OMNIBOX_EDIT_MODEL_START_AUTOCOMPLETE
   controller_->StartAutocomplete(input_);
 }
 
