diff --git a/chrome/browser/ui/views/page_action/page_action_view.h b/chrome/browser/ui/views/page_action/page_action_view.h
index 78f2c272436f0effafcc0ef8fea9c34abc4efcb7..56a9c1c57bc70e902daa73cd017dfd73253f5213 100644
--- a/chrome/browser/ui/views/page_action/page_action_view.h
+++ b/chrome/browser/ui/views/page_action/page_action_view.h
@@ -35,6 +35,7 @@ struct PageActionViewParams;
 
 // PageActionView is the view displaying the page action. There is one per
 // browser, per page action.
+namespace chromium_impl {
 class PageActionView : public IconLabelBubbleView,
                        public PageActionModelObserver {
   METADATA_HEADER(PageActionView, IconLabelBubbleView)
@@ -49,7 +50,7 @@ class PageActionView : public IconLabelBubbleView,
 
   // Sets the controller for this view, and attaches this view in the
   // controller.
-  void OnNewActiveController(PageActionController* controller);
+  virtual void OnNewActiveController(PageActionController* controller);
 
   // As an alternative to OnNewActiveController(), just set the observed model.
   // TODO(crbug.com/388524315): Merge OnNewActiveController and this method.
@@ -105,6 +106,7 @@ class PageActionView : public IconLabelBubbleView,
   }
 
  private:
+  friend class ::page_actions::PageActionView;
   // The image associated with the `action_item_` size may be different from the
   // size needed for the location bar page action icon. Therefore, we should to
   // update the image size if needed.
@@ -167,6 +169,7 @@ class PageActionView : public IconLabelBubbleView,
   base::RepeatingClosure anchored_message_close_callback_ = base::DoNothing();
   base::WeakPtrFactory<PageActionView> weak_factory_{this};
 };
+}  // namespace chromium_impl
 
 }  // namespace page_actions
 
