diff --git a/chrome/browser/resources/settings/appearance_page/appearance_page.js b/chrome/browser/resources/settings/appearance_page/appearance_page.js
index 2551cf3b19168d51dd5a2432eb56871490d11d54..069915b493e46096e546d2c62204094b101b162e 100644
--- a/chrome/browser/resources/settings/appearance_page/appearance_page.js
+++ b/chrome/browser/resources/settings/appearance_page/appearance_page.js
@@ -82,6 +82,8 @@ Polymer({
         1,
         11 / 10,
         5 / 4,
+		4 / 3,
+		7 / 5,
         3 / 2,
         7 / 4,
         2,
