diff --git a/chrome/android/java_sources.gni b/chrome/android/java_sources.gni
index 38b34fd3ae9fc9e3d1f6591244e6d970c4708171..a6774ef2e8be59c4d2cc1c19f31d540822203483 100644
--- a/chrome/android/java_sources.gni
+++ b/chrome/android/java_sources.gni
@@ -25,6 +25,7 @@ import("//components/omnibox/browser/test_java_sources.gni")
 import("//device/vr/buildflags/buildflags.gni")
 
 chrome_java_sources += public_autofill_assistant_java_sources
+chrome_java_sources += brave_java_sources
 
 if (enable_vr) {
   chrome_java_sources += [ "java/src/org/chromium/chrome/browser/component_updater/VrAssetsComponentInstaller.java" ]
