diff --git a/chrome/browser/ui/views/frame/browser_view.h b/chrome/browser/ui/views/frame/browser_view.h
index 9c99d9de95f54ba303e5346028701a06c9988ecf..67c13fa0c67e4dd1b4a21ae485f45583e8f86cd4 100644
--- a/chrome/browser/ui/views/frame/browser_view.h
+++ b/chrome/browser/ui/views/frame/browser_view.h
@@ -698,6 +698,7 @@ class BrowserView : public BrowserWindow,
     return accessibility_focus_highlight_.get();
   }
 
+  BRAVE_BROWSER_VIEW_H
  private:
   // Do not friend BrowserViewLayout. Use the BrowserViewLayoutDelegate
   // interface to keep these two classes decoupled and testable.
