diff --git a/chrome/browser/ui/tabs/tab_model.cc b/chrome/browser/ui/tabs/tab_model.cc
index 6283f2549826f5f333d280dab1c3d02ce0ae2973..006a08acdc20a6501d3bbb5f11033da6a85d6b70 100644
--- a/chrome/browser/ui/tabs/tab_model.cc
+++ b/chrome/browser/ui/tabs/tab_model.cc
@@ -56,7 +56,7 @@ TabModel::TabModel(std::unique_ptr<content::WebContents> contents,
   // TODO(https://crbug.com/362038317): Tab-helpers should be created in exactly
   // one place, which is here.
   TabHelpers::AttachTabHelpers(contents_);
-  tab_features_ = std::make_unique<TabFeatures>();
+  tab_features_ = std::make_unique<BraveTabFeatures>();
   const SessionID session_id = sessions::SessionTabHelper::IdForTab(contents_);
   CHECK(session_id.is_valid());
   SetSessionId(session_id.id());
