diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 17311184c47823594b49ad07d41bd870efa7af17..611376d4d7c3bac30ecfbab8a786dc2fffd52fce 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -42,7 +42,7 @@ if (dfmify_devtools) {
   import("//chrome/android/features/devtools/devtools_module_tmpl.gni")
 }
 
-manifest_package = "org.chromium.chrome"
+declare_args() { manifest_package = "org.chromium.chrome" }
 test_manifest_package = "org.chromium.chrome.tests"
 
 chrome_public_jinja_variables = default_chrome_public_jinja_variables +
