diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 2ddc9fb6ed5784e8e4867a4126ca82153fcbbaa3..9cfafb272cde5705af3ba62d90c481828cc2b585 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -92,6 +92,7 @@ class AvatarToolbarButton : public ToolbarButton,
   void OnBoundsChanged(const gfx::Rect& previous_bounds) override;
 
  private:
+  BRAVE_AVATAR_BUTTON_H
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
