diff --git a/chrome/browser/ui/browser_command_controller.h b/chrome/browser/ui/browser_command_controller.h
index 54a4ff842bf2a687751343e3144a3b3fdadb1456..cdb2575ecc6ff26d004a370755d3e4075685ff09 100644
--- a/chrome/browser/ui/browser_command_controller.h
+++ b/chrome/browser/ui/browser_command_controller.h
@@ -121,6 +121,7 @@ class BrowserCommandController : public CommandUpdater,
       Profile* profile);
 
  private:
+  friend class BraveBrowserCommandController;
 #if BUILDFLAG(IS_CHROMEOS)
   friend class BrowserCommandControllerBrowserTestLockedFullscreen;
 #endif
@@ -200,7 +201,7 @@ class BrowserCommandController : public CommandUpdater,
 
   // Update commands whose state depends on the type of fullscreen mode the
   // window is in.
-  void UpdateCommandsForFullscreenMode();
+  virtual void UpdateCommandsForFullscreenMode();
 
   // Update commands whose state depends on whether they're available to hosted
   // app windows.
