diff --git a/chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.cc b/chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.cc
index 0aad13042f6eca92f9c6f9c033428c856cd15b1e..49fd625b744b0967678f9e71ee7850da65e3a17a 100644
--- a/chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.cc
+++ b/chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.cc
@@ -331,6 +331,7 @@ bool SaveCardBubbleControllerImpl::ShouldRequestExpirationDateFromUser() const {
 }
 
 bool SaveCardBubbleControllerImpl::ShouldShowSignInPromo() const {
+  BRAVE_SHOULD_SHOW_SIGNIN_PROMO_SHORT_CIRCUIT_TO_FALSE;
   if (is_upload_save_)
     return false;
 
