diff --git a/chrome/browser/profiles/bookmark_model_loaded_observer.h b/chrome/browser/profiles/bookmark_model_loaded_observer.h
index 27417fda3de06fc8a2ed7fe654c42e5450e1d3b3..41ff7c74a7bebdad0a2c7894149b65478aa26bdd 100644
--- a/chrome/browser/profiles/bookmark_model_loaded_observer.h
+++ b/chrome/browser/profiles/bookmark_model_loaded_observer.h
@@ -19,6 +19,7 @@ class BookmarkModelLoadedObserver
   BookmarkModelLoadedObserver& operator=(const BookmarkModelLoadedObserver&) =
       delete;
 
+  BRAVE_BOOKMARK_MODEL_LOADED_OBSERVER_H_
  private:
   void BookmarkModelChanged() override;
   void BookmarkModelLoaded(bookmarks::BookmarkModel* model,
