diff --git a/chrome/android/java/src/org/chromium/chrome/browser/webapps/AppInstallMenuHandler.java b/chrome/android/java/src/org/chromium/chrome/browser/webapps/AppInstallMenuHandler.java
index 91d43bdea1804b9bf4cadc135e2b32aead20e688..a0875161def9a26d198eaaa0f7305e3f4e34a58d 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/webapps/AppInstallMenuHandler.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/webapps/AppInstallMenuHandler.java
@@ -14,6 +14,7 @@ import org.chromium.base.ContextUtils;
 import org.chromium.build.annotations.NullMarked;
 import org.chromium.build.annotations.Nullable;
 import org.chromium.chrome.R;
+import org.chromium.components.embedder_support.util.UrlConstants;
 import org.chromium.chrome.browser.app.appmenu.AppMenuPropertiesDelegateImpl;
 import org.chromium.chrome.browser.banners.AppMenuVerbiage;
 import org.chromium.chrome.browser.tab.Tab;
@@ -80,7 +81,7 @@ public class AppInstallMenuHandler {
 
         @Nullable BottomSheetController controller =
                 BottomSheetControllerProvider.from(windowAndroid);
-        if (controller == null) {
+        if ((currentTab.getUrl() != null && UrlConstants.CHROME_SCHEME.equals(currentTab.getUrl().getScheme())) || controller == null) {
             // We have three options when this function fails. One is to abort the operation and do
             // nothing (by returning false), or we can make one of the two options of the Universal
             // Install dialog the default and go with that in case of errors. Since Install App is
