diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page.js b/chrome/browser/resources/settings/privacy_page/privacy_page.js
index ae7466afa0151c315a62b99cad760b1d04c7f72f..85659f61d1b488ab444cc0de267b10c1f15b2809 100644
--- a/chrome/browser/resources/settings/privacy_page/privacy_page.js
+++ b/chrome/browser/resources/settings/privacy_page/privacy_page.js
@@ -18,6 +18,7 @@ import '../site_settings/settings_category_default_radio_group.js';
 import '../settings_page/settings_animated_pages.m.js';
 import '../settings_page/settings_subpage.m.js';
 import '../settings_shared_css.m.js';
+import './personalization_options.m.js';
 
 import {assert} from 'chrome://resources/js/assert.m.js';
 import {focusWithoutInk} from 'chrome://resources/js/cr/ui/focus_without_ink.m.js';
