diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index 82ea4b347c005a6ec359ee22abd3947198defe7a..0bf9c1626c7e00df3e0ed347ae408b1c9663dfa1 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -9,6 +9,7 @@ import {pageVisibility} from './page_visibility.js';
 import {Route, Router} from './router.js';
 import {SettingsRoutes} from './settings_routes.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -236,6 +237,7 @@ function createBrowserSettingsRoutes(): Partial<SettingsRoutes> {
       // </if>
     }
   }
+  addBraveRoutes(r);
   return r;
 }
 
