diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
index 002333dfcc4aba1bb7122a4793200995a1d6a400..65a2efb964adb8725ba4b73a9c91867ae5dd1387 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
@@ -499,13 +499,13 @@ public class ToolbarManager implements UrlFocusChangeListener, ThemeColorObserve
         Runnable onMenuButtonClicked =
                 () -> UpdateMenuItemHelper.getInstance().onMenuButtonClicked();
         // clang-format off
-        mMenuButtonCoordinator = new MenuButtonCoordinator(appMenuCoordinatorSupplier,
+        mMenuButtonCoordinator = new org.chromium.chrome.browser.toolbar.menu_button.BraveMenuButtonCoordinator(appMenuCoordinatorSupplier,
                 mControlsVisibilityDelegate, mWindowAndroid,
                 (focus, type) -> setUrlBarFocus(focus, type), requestFocusRunnable,
                 shouldShowUpdateBadge, isInOverviewModeSupplier, menuButtonThemeColorProvider,
                 menuButtonStateSupplier, onMenuButtonClicked, R.id.menu_button_wrapper);
         if (shouldShowUpdateBadge) mMenuStateObserver = mMenuButtonCoordinator.getStateObserver();
-        MenuButtonCoordinator startSurfaceMenuButtonCoordinator = new MenuButtonCoordinator(
+        MenuButtonCoordinator startSurfaceMenuButtonCoordinator = new org.chromium.chrome.browser.toolbar.menu_button.BraveMenuButtonCoordinator(
                 appMenuCoordinatorSupplier, mControlsVisibilityDelegate, mWindowAndroid,
                 (focus, type) -> setUrlBarFocus(focus, type), requestFocusRunnable,
                 shouldShowUpdateBadge, isInOverviewModeSupplier, overviewModeThemeColorProvider,
@@ -942,7 +942,7 @@ public class ToolbarManager implements UrlFocusChangeListener, ThemeColorObserve
             boolean isTabToGtsAnimationEnabled, boolean isStartSurfaceEnabled,
             boolean isTabGroupsAndroidContinuationEnabled) {
         // clang-format off
-        TopToolbarCoordinator toolbar = new TopToolbarCoordinator(controlContainer, toolbarLayout,
+        TopToolbarCoordinator toolbar = new org.chromium.chrome.browser.toolbar.top.BraveTopToolbarCoordinator(controlContainer, toolbarLayout,
                 mLocationBarModel, mToolbarTabController,
                 new UserEducationHelper(mActivity, mHandler),
                 buttonDataProviders, mLayoutStateProviderSupplier, browsingModeThemeColorProvider,
