diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 20cbeaa28cb18ffabacba020fdc896b748741146..0aa114cce8e350031a056c9bbdb4d71268c6bc60 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2528,6 +2528,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
