diff --git a/chrome/browser/web_applications/components/web_app_shortcut_mac.mm b/chrome/browser/web_applications/components/web_app_shortcut_mac.mm
index 79f0cdfae88cf89136eefde95fc983b931317189..e18886ddd72040b70da344d97531eefa7e1062cf 100644
--- a/chrome/browser/web_applications/components/web_app_shortcut_mac.mm
+++ b/chrome/browser/web_applications/components/web_app_shortcut_mac.mm
@@ -905,6 +905,7 @@ base::FilePath WebAppShortcutCreator::GetApplicationsDirname() const {
   if (path.empty())
     return path;
 
+  return path.Append(GetLocalizableBraveAppShortcutsSubdirName());
   return path.Append(GetLocalizableAppShortcutsSubdirName());
 }
 
