diff --git a/chrome/browser/profiles/profile_shortcut_manager_win.cc b/chrome/browser/profiles/profile_shortcut_manager_win.cc
index c7b5c9e9e2a4e8e0275aff3566bef33e872303c3..39b104815a87b3b6ecf9e5dfa0b489d87895241a 100644
--- a/chrome/browser/profiles/profile_shortcut_manager_win.cc
+++ b/chrome/browser/profiles/profile_shortcut_manager_win.cc
@@ -836,7 +836,7 @@ bool ProfileShortcutManager::IsFeatureEnabled() {
 // static
 std::unique_ptr<ProfileShortcutManager> ProfileShortcutManager::Create(
     ProfileManager* manager) {
-  return std::make_unique<ProfileShortcutManagerWin>(manager);
+  return std::make_unique<BraveProfileShortcutManagerWin>(manager);
 }
 
 ProfileShortcutManagerWin::ProfileShortcutManagerWin(ProfileManager* manager)
