diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 809e0ce637ae0fc401f7b57e8ca624268ac9bdff..5feaabcc70fd07f38fc7abf4a9b92105e73d7c81 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2503,6 +2503,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
