diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 580dd0e53876204c18bf290862392e37678b34bc..4525ff68d2a38d649900ffd00b412382b79b6536 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2474,6 +2474,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps defines += brave_chrome_browser_defines
 
   if (is_android) {
     sources += [
