diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
index d7149d0526732fa4d5feca16c77fb3fa4d4433cc..d98e3ace297efda4b575c51c9fa4d7e39fc89030 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
@@ -270,6 +270,7 @@ base::string16 AvatarToolbarButton::GetAvatarTooltipText() const {
   if (IsIncognito())
     return l10n_util::GetStringUTF16(IDS_AVATAR_BUTTON_INCOGNITO_TOOLTIP);
 
+  if (profile_->IsTorProfile()) return l10n_util::GetStringUTF16(IDS_TOR_PROFILE_NAME);
   if (profile_->IsGuestSession())
     return l10n_util::GetStringUTF16(IDS_GUEST_PROFILE_NAME);
 
