diff --git a/chrome/browser/devtools/devtools_ui_bindings.h b/chrome/browser/devtools/devtools_ui_bindings.h
index ca2eb87881a19cda4bb7bb09587e6e645f60c315..86caa971128d5fcb89464638d20b13335bd18803 100644
--- a/chrome/browser/devtools/devtools_ui_bindings.h
+++ b/chrome/browser/devtools/devtools_ui_bindings.h
@@ -105,6 +105,8 @@ class DevToolsUIBindings : public DevToolsEmbedderMessageDispatcher::Delegate,
 
  private:
   using DevToolsUIBindingsList = std::vector<DevToolsUIBindings*>;
+  friend class BraveDevToolsUIBindings;
+  FRIEND_TEST_ALL_PREFIXES(BraveDevToolsUIBindingsBrowserTest, ThemeTest);
 
   void HandleMessageFromDevToolsFrontend(base::Value);
 
