diff --git a/chrome/browser/prefs/pref_service_incognito_allowlist.cc b/chrome/browser/prefs/pref_service_incognito_allowlist.cc
index 916d5c4e13c994332fd72005f22dced5399e3ff8..a1caaa7ba7551d79cf19e8b0766f381ea49b4b74 100644
--- a/chrome/browser/prefs/pref_service_incognito_allowlist.cc
+++ b/chrome/browser/prefs/pref_service_incognito_allowlist.cc
@@ -226,10 +226,16 @@ const char* const kPersistentPrefNames[] = {
 namespace prefs {
 
 std::vector<const char*> GetIncognitoPersistentPrefsAllowlist() {
+  std::vector<const char*> result = [&]() -> std::vector<const char*> {
   std::vector<const char*> allowlist;
   allowlist.insert(allowlist.end(), std::begin(kPersistentPrefNames),
                    std::end(kPersistentPrefNames));
   return allowlist;
+  }();
+  for (auto pref : brave::GetBravePersistentPrefNames()) {
+    result.push_back(pref.data());
+  }
+  return result;
 }
 
 }  // namespace prefs
