diff --git a/chrome/browser/ui/tabs/tab_style.h b/chrome/browser/ui/tabs/tab_style.h
index c2a14c6879845bc4f81da5dc9f583236157e0d18..415ae1abb7fa65401959eadc518a356646ed3321 100644
--- a/chrome/browser/ui/tabs/tab_style.h
+++ b/chrome/browser/ui/tabs/tab_style.h
@@ -111,7 +111,7 @@ class TabStyle {
 
   TabStyle(const TabStyle&) = delete;
   TabStyle& operator=(const TabStyle&) = delete;
-  ~TabStyle();
+  virtual ~TabStyle();
 
   // Returns the standard height of a single Tab.
   int GetStandardHeight() const;
