diff --git a/chrome/browser/ui/webui/settings/import_data_handler.cc b/chrome/browser/ui/webui/settings/import_data_handler.cc
index 9b70f041c6b8b0c434ffe8b8db8a39f1095e4808..5ac90b75a64930691ab1142853b0a19a25b215ae 100644
--- a/chrome/browser/ui/webui/settings/import_data_handler.cc
+++ b/chrome/browser/ui/webui/settings/import_data_handler.cc
@@ -149,6 +149,7 @@ void ImportDataHandler::HandleImportData(const base::ListValue& args) {
   if (*type_dict.FindBool(prefs::kImportDialogSearchEngine)) {
     selected_items |= user_data_importer::SEARCH_ENGINES;
   }
+  BRAVE_IMPORT_DATA
 
   const user_data_importer::SourceProfile& source_profile =
       importer_list_->GetSourceProfileAt(browser_index);
@@ -229,6 +230,7 @@ void ImportDataHandler::SendBrowserProfileData(const std::string& callback_id) {
         "autofillFormData",
         (browser_services & user_data_importer::AUTOFILL_FORM_DATA) != 0);
 
+    BRAVE_SEND_BROWSER_PROFILE_DATA
     browser_profiles.Append(std::move(browser_profile));
   }
 
