diff --git a/chrome/browser/ui/views/frame/multi_contents_view.cc b/chrome/browser/ui/views/frame/multi_contents_view.cc
index 2c009efec80c605f04d166c87d263d4159f911a6..e5be5369def616035b21cccd7caabedda62e6dd1 100644
--- a/chrome/browser/ui/views/frame/multi_contents_view.cc
+++ b/chrome/browser/ui/views/frame/multi_contents_view.cc
@@ -77,7 +77,7 @@ MultiContentsView::MultiContentsView(
       AddChildView(std::make_unique<MultiContentsBackgroundView>(browser_view));
 
   contents_container_views_.push_back(
-      AddChildView(std::make_unique<ContentsContainerView>(browser_view_)));
+      AddChildView(std::make_unique<BraveContentsContainerView>(browser_view_)));
   contents_container_views_[0]
       ->contents_view()
       ->set_is_primary_web_contents_for_window(true);
@@ -86,7 +86,7 @@ MultiContentsView::MultiContentsView(
   resize_area_->SetVisible(false);
 
   contents_container_views_.push_back(
-      AddChildView(std::make_unique<ContentsContainerView>(browser_view_)));
+      AddChildView(std::make_unique<BraveContentsContainerView>(browser_view_)));
   contents_container_views_[1]->SetVisible(false);
 
   drop_target_view_ =
