diff --git a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
index c0f4e262a9e11de91b01bd77676367bdcc026aec..5625b10b4b7643733902926d02ce4533e0431d01 100644
--- a/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
+++ b/chrome/browser/resources/signin/profile_picker/profile_picker_main_view.html.ts
@@ -25,7 +25,7 @@ export function getHtml(this: ProfilePickerMainViewElement) {
             data-index="${index}">
         </profile-card>
       `)}
-      <cr-button id="addProfile" class="profile-item"
+      <cr-button id="addProfile" class="profile-item plain"
           @click="${this.onAddProfileClick_}"
           aria-labelledby="addProfileButtonLabel">
         <div id="addProfileButtonLabel"
