diff --git a/chrome/browser/resources/extensions/BUILD.gn b/chrome/browser/resources/extensions/BUILD.gn
index aa59bce7b2f9a5b42248e53f9533dd8b5ba80ecd..1d2a1d0028ac1c81a6600e3af709ab223c05ee1b 100644
--- a/chrome/browser/resources/extensions/BUILD.gn
+++ b/chrome/browser/resources/extensions/BUILD.gn
@@ -49,6 +49,7 @@ generate_grd("build_grd") {
     deps = [ ":build_ts" ]
     manifest_files = [ "$target_gen_dir/tsconfig.manifest" ]
   }
+  deps += [ "//brave/browser/resources/extensions:resources" ]
 }
 
 preprocess_if_expr("preprocess") {
@@ -113,4 +114,5 @@ ts_library("build_ts") {
     ":preprocess",
     ":preprocess_generated",
   ]
+  in_files += brave_deps_chrome_browser_resources_extensions_in_files extra_deps += brave_deps_chrome_browser_resources_extensions_extra_deps
 }
