diff --git a/chrome/browser/ui/browser_commands.cc b/chrome/browser/ui/browser_commands.cc
index ecb927514d95aed2d223847dd3ce2c47c70a196b..ad95fa5779cad1f34d4e91dba54ed96b730b6041 100644
--- a/chrome/browser/ui/browser_commands.cc
+++ b/chrome/browser/ui/browser_commands.cc
@@ -590,6 +590,7 @@ void ReloadInternal(BrowserWindowInterface* browser,
           tab_strip_model->GetWebContentsAt(selected_index));
     }
   }
+  BRAVE_RELOAD_INTERNAL
 
   base::UmaHistogramCounts100("TabStrip.Tab.ReloadCount",
                               tabs_to_reload.size());
@@ -1172,6 +1173,7 @@ void CloseTab(BrowserWindowInterface* browser) {
   }
 
   ToastController* toast_controller = browser->GetFeatures().toast_controller();
+  BRAVE_CLOSE_TAB
   if (!toast_controller) {
     CloseSelectedTabAndRecordTabCountMetric(browser);
     return;
