diff --git a/chrome/browser/ui/page_action/page_action_controller.cc b/chrome/browser/ui/page_action/page_action_controller.cc
index 95315a1c0e0f382738cc729b2a8897382b6ea52f..49e0e76a635a198df9be91a2d95876d6990405a9 100644
--- a/chrome/browser/ui/page_action/page_action_controller.cc
+++ b/chrome/browser/ui/page_action/page_action_controller.cc
@@ -72,6 +72,7 @@ void ScopedPageActionActivity::RegisterWillDestroyControllerCallback() {
   }
 }
 
+namespace chromium_impl {
 PageActionControllerImpl::PageActionControllerImpl(
     PinnedToolbarActionsModel* pinned_actions_model,
     PageActionModelFactory* page_action_model_factory,
@@ -522,7 +523,7 @@ std::unique_ptr<PageActionModelInterface> PageActionControllerImpl::CreateModel(
   if (page_action_model_factory_ != nullptr) {
     return page_action_model_factory_->Create(action_id, is_ephemeral);
   } else {
-    return std::make_unique<PageActionModel>(action_id, is_ephemeral);
+    return std::make_unique<::page_actions::PageActionModel>(action_id, is_ephemeral);
   }
 }
 
@@ -600,6 +601,7 @@ void PageActionControllerImpl::OnImageAnimationStarted(
   FindPageActionModel(action_id).SetDidAnimateImage(PageActionPassKey());
 }
 
+}  // namespace chromium_impl
 std::ostream& operator<<(std::ostream& os, const SuggestionChipConfig& config) {
   os << "{ should_animate: " << config.should_animate
      << ", should_announce_chip: " << config.should_announce_chip << " }";
