diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index d5dc11950e0252031040315dd7fd093470597f85..fffd4e72ee5d61ae77cf0d74a5ff29595150a5a5 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -839,6 +839,7 @@ void BookmarkBarView::Layout() {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   // Layout the right side buttons.
   x = max_x + bookmark_bar_button_padding;
