diff --git a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
index b65260d0135ab5e013cd1e39f1d88d08db2068f8..46f70f3a99cec6aa7906ded9621c03f16683fbfb 100644
--- a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
+++ b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
@@ -494,6 +494,7 @@ class AppMenu implements OnKeyListener {
                         anchorView,
                         anchorViewOffset);
 
+        if (this instanceof BraveAppMenu) ((BraveAppMenu) this).updatePopup(popup, isByPermanentButton);
         int popupHeight = calculateMenuHeight();
         popup.setHeight(popupHeight);
 
