diff --git a/chrome/browser/resources/settings/settings_page/main_page_mixin.ts b/chrome/browser/resources/settings/settings_page/main_page_mixin.ts
index 3118955f9c1717cb4e3c939603ee23cfa61c2963..8848cff7a81577333bcc8af2b461b3a72c2e3411 100644
--- a/chrome/browser/resources/settings/settings_page/main_page_mixin.ts
+++ b/chrome/browser/resources/settings/settings_page/main_page_mixin.ts
@@ -38,7 +38,7 @@ guestTopLevelRoute = routes.PRIVACY;
 // </if>
 
 const TOP_LEVEL_EQUIVALENT_ROUTE: Route =
-    loadTimeData.getBoolean('isGuest') ? guestTopLevelRoute : routes.PEOPLE;
+    loadTimeData.getBoolean('isGuest') ? guestTopLevelRoute : routes.GET_STARTED;
 
 function classifyRoute(route: Route|null): RouteState {
   if (!route) {
