diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 2987967ec109f95b1f67ed2ec37748626cb118bf..0bab215ad85001890048a5f473a540fa4a987924 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -645,6 +645,7 @@ static_library("ui") {
     "//ui/webui/resources/cr_components/most_visited:mojom",
     "//v8:v8_version",
   ]
+  deps+= [ "//brave/browser/ui" ]
 
   # TODO(crbug/925153): Remove this circular dependency.
   allow_circular_includes_from = [
@@ -3632,10 +3633,13 @@ static_library("ui") {
       ]
       deps += [ "//google_update" ]
     } else {
-      sources += [
-        "webui/help/version_updater_basic.cc",
-        "webui/help/version_updater_basic.h",
-      ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [
+          "webui/help/version_updater_basic.cc",
+          "webui/help/version_updater_basic.h",
+        ]
+      }
     }
   } else {  # 'OS!="win"
     # The Cast Linux build sets toolkit_views to false, but many files in this
