diff --git a/chrome/browser/ui/views/profiles/incognito_menu_view.h b/chrome/browser/ui/views/profiles/incognito_menu_view.h
index 60c25ba033d5cf3c4720088607df6787db86db57..2462caa6c1b9222f69a027c28da0b246271b2e1a 100644
--- a/chrome/browser/ui/views/profiles/incognito_menu_view.h
+++ b/chrome/browser/ui/views/profiles/incognito_menu_view.h
@@ -30,6 +30,7 @@ class IncognitoMenuView : public ProfileMenuViewBase {
   // ProfileMenuViewBase:
   void BuildMenu() override;
 
+  BRAVE_INCOGNITO_MENU_VIEW_H_
  private:
   // views::BubbleDialogDelegateView:
   base::string16 GetAccessibleWindowTitle() const override;
