diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index ff9b86f353b224ac333e3929a51e1fd58a05d1ee..aa788283aefceedb5cda0ebc33572f726c2599e4 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -101,6 +101,8 @@ grit("generated_resources") {
   if (is_android) {
     outputs += android_generated_java_resources
   }
+
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
@@ -136,7 +138,7 @@ grit("google_chrome_strings") {
 }
 
 grit("chromium_strings") {
-  source = "chromium_strings.grd"
+  source = "${branding_path_component}_strings.grd"
   defines = chrome_grit_defines
   output_dir = "$root_gen_dir/chrome"
   outputs =
