diff --git a/chrome/browser/sync/chrome_sync_client.cc b/chrome/browser/sync/chrome_sync_client.cc
index 3776ede92f4de4f94c8617688af16a2087f284d1..4833d4d4a4288fe0c322dc6cebc8450ba3f435fb 100644
--- a/chrome/browser/sync/chrome_sync_client.cc
+++ b/chrome/browser/sync/chrome_sync_client.cc
@@ -11,6 +11,8 @@
 #include "base/feature_list.h"
 #include "base/path_service.h"
 #include "base/syslog_logging.h"
+#include "brave/browser/brave_rewards/vg_sync_service.h"
+#include "brave/browser/brave_rewards/vg_sync_service_factory.h"
 #include "build/build_config.h"
 #include "build/chromeos_buildflags.h"
 #include "chrome/browser/consent_auditor/consent_auditor_factory.h"
@@ -77,6 +79,8 @@
 #include "components/sync/base/sync_util.h"
 #include "components/sync/driver/model_type_controller.h"
 #include "components/sync/driver/sync_api_component_factory.h"
+#include "components/sync/driver/sync_service.h"
+#include "components/sync/driver/sync_user_settings.h"
 #include "components/sync/driver/syncable_service_based_model_type_controller.h"
 #include "components/sync/model/forwarding_model_type_controller_delegate.h"
 #include "components/sync/model/model_type_store.h"
@@ -575,6 +579,12 @@ ChromeSyncClient::GetControllerDelegateForModelType(syncer::ModelType type) {
     case syncer::USER_EVENTS:
       return browser_sync::UserEventServiceFactory::GetForProfile(profile_)
           ->GetControllerDelegate();
+    case syncer::VG_BODIES:
+      return VgSyncServiceFactory::GetForProfile(profile_)
+          ->GetControllerDelegateForVgBodies();
+    case syncer::VG_SPEND_STATUSES:
+      return VgSyncServiceFactory::GetForProfile(profile_)
+          ->GetControllerDelegateForVgSpendStatuses();
 #if BUILDFLAG(ENABLE_EXTENSIONS)
     case syncer::WEB_APPS: {
       auto* provider = web_app::WebAppProvider::GetForWebApps(profile_);
@@ -626,6 +636,16 @@ void ChromeSyncClient::OnLocalSyncTransportDataCleared() {
   metrics::ClearDemographicsPrefs(profile_->GetPrefs());
 }
 
+void ChromeSyncClient::SetDefaultEnabledTypes(
+    syncer::SyncService* sync_service) {
+  DCHECK(sync_service);
+
+  // Default enabled types: Bookmarks
+  syncer::UserSelectableTypeSet selected_types;
+  selected_types.Put(syncer::UserSelectableType::kBookmarks);
+  sync_service->GetUserSettings()->SetSelectedTypes(false, selected_types);
+}
+
 #if BUILDFLAG(ENABLE_EXTENSIONS)
 std::unique_ptr<syncer::ModelTypeController>
 ChromeSyncClient::CreateAppsModelTypeController(
