diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 03336cf469c9959307da60555819e8919738c6bf..611d2ddaf62ff04f041302e100b5e9e774513fbb 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("channel_info") {
   }
 
   deps = [ "//build:branding_buildflags" ]
+  if (brave_chromium_build) { deps += [ "//brave/common:channel_info" ] }
 
   public_deps = [
     "//base",
@@ -181,6 +182,7 @@ static_library("common") {
   ]
 
   public_deps = [
+    "//brave/common",
     ":available_offline_content_mojom",
     ":buildflags",
     ":channel_info",
@@ -555,6 +557,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  if (brave_chromium_build) { configs += [ "//brave/common:constants_configs" ] }
 }
 
 component("chrome_features") {
