diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
index 16da25f6a12ad0e5fd657a07f8adf3bd34f4620b..8f048c8834563019f90b1a6ad44ce6c979437ea6 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
@@ -220,6 +220,7 @@ void AvatarToolbarButton::OnHighlightChanged() {
 }
 
 base::string16 AvatarToolbarButton::GetAvatarTooltipText() const {
+  GET_AVATAR_TOOLTIP_TEXT_
   switch (delegate_->GetState()) {
     case State::kIncognitoProfile:
       return l10n_util::GetStringUTF16(IDS_AVATAR_BUTTON_INCOGNITO_TOOLTIP);
