diff --git a/chrome/browser/ui/tabs/public/tab_features.h b/chrome/browser/ui/tabs/public/tab_features.h
index e745f1c386f65510bfb2708510091acc3b5d4594..d137fe482adb9836f1e6ccf5716b3ad74291a5aa 100644
--- a/chrome/browser/ui/tabs/public/tab_features.h
+++ b/chrome/browser/ui/tabs/public/tab_features.h
@@ -369,7 +369,7 @@ class TabInterface;
 class TabFeatures {
  public:
   TabFeatures();
-  ~TabFeatures();
+  virtual ~TabFeatures();
 
   TabFeatures(const TabFeatures&) = delete;
   TabFeatures& operator=(const TabFeatures&) = delete;
@@ -507,12 +507,13 @@ class TabFeatures {
   }
 
   // Called exactly once to initialize features.
-  void Init(TabInterface& tab, Profile* profile);
+  virtual void Init(TabInterface& tab, Profile* profile);
 
   static ui::UserDataFactoryWithOwner<TabInterface>&
   GetUserDataFactoryForTesting();
 
  private:
+  friend class BraveTabFeatures;
   bool initialized_ = false;
 
   // Returns the factory used to create owned components.
