diff --git a/chrome/browser/ui/views/profiles/profile_chooser_view.cc b/chrome/browser/ui/views/profiles/profile_chooser_view.cc
index d433d4d5ed52e2d1c2063a4be839922b597573b3..34a89ffb857a94b7ad9d951b94f2193f0a6a2a55 100644
--- a/chrome/browser/ui/views/profiles/profile_chooser_view.cc
+++ b/chrome/browser/ui/views/profiles/profile_chooser_view.cc
@@ -332,7 +332,7 @@ void ProfileChooserView::ShowBubble(
     return;
 
   profile_bubble_ =
-      new ProfileChooserView(anchor_button, browser, view_mode,
+      new BraveProfileChooserView(anchor_button, browser, view_mode,
                              manage_accounts_params.service_type, access_point);
   if (!anchor_button) {
     DCHECK(parent_window);
@@ -1323,9 +1323,12 @@ views::View* ProfileChooserView::CreateOptionsView(bool display_lock,
       layout->AddView(guest_profile_button_);
     }
   }
+  static_cast<BraveProfileChooserView*>(this)->AddTorButton(layout);
 
   base::string16 text = l10n_util::GetStringUTF16(
       is_guest ? IDS_PROFILES_EXIT_GUEST : IDS_PROFILES_MANAGE_USERS_BUTTON);
+  if (browser_->profile()->IsTorProfile())
+    text = l10n_util::GetStringUTF16(IDS_PROFILES_EXIT_TOR);
   const gfx::VectorIcon& settings_icon =
       is_guest ? kCloseAllIcon : kSettingsIcon;
   users_button_ = new HoverButton(
