diff --git a/chrome/browser/updater/BUILD.gn b/chrome/browser/updater/BUILD.gn
index dff49c5ab90edb7bd8bf0371c720d6bad07a431c..671983d586d64aa49a87f8ca0858819b6d9c57ec 100644
--- a/chrome/browser/updater/BUILD.gn
+++ b/chrome/browser/updater/BUILD.gn
@@ -73,6 +73,7 @@ source_set("scheduler") {
         "//chrome/common",
         "//chrome/updater:browser_sources",
       ]
+      deps += [ "//brave/browser/mac:keystone_glue" ]
     } else {
       sources += [ "scheduler_mac_noimpl.cc" ]
     }
