diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 0592ec7a61e686628e7574fda68a0f9ba2d38346..0e2e392ec4250be2e3af2e8a54fd5dc50ec12953 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -76,6 +76,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
