diff --git a/chrome/installer/mini_installer/chrome.release b/chrome/installer/mini_installer/chrome.release
index 419d98ccc4a8dd6fe99ef705d869170dd69d6e4b..0c40b63cef66f2bc89cf2139738c2ab74c938a20 100644
--- a/chrome/installer/mini_installer/chrome.release
+++ b/chrome/installer/mini_installer/chrome.release
@@ -6,7 +6,7 @@
 #
 # Chrome Application dir entries, sorted alphabetically.
 #
-chrome.exe: %(ChromeDir)s\
+brave.exe: %(ChromeDir)s\
 #
 # Chrome version dir assembly manifest.
 # The name of this file must match the name of the version dir, so we cannot
@@ -18,6 +18,8 @@ chrome.exe: %(ChromeDir)s\
 # Chrome version dir entries, sorted alphabetically.
 #
 chrome.dll: %(VersionDir)s\
+brave_resources.pak: %(VersionDir)s\
+brave_100_percent.pak: %(VersionDir)s\
 chrome_100_percent.pak: %(VersionDir)s\
 chrome_child.dll: %(VersionDir)s\
 chrome_elf.dll: %(VersionDir)s\
@@ -76,6 +78,7 @@ swiftshader\libGLESv2.dll: %(VersionDir)s\swiftshader\
 
 [HIDPI]
 chrome_200_percent.pak: %(VersionDir)s\
+brave_200_percent.pak: %(VersionDir)s\
 
 [TOUCH]
 
