diff --git a/chrome/browser/resources/optimize_webui.gni b/chrome/browser/resources/optimize_webui.gni
index f4c1ca5ad559e285b1bfff925c4098b27c0d5888..9fc33e3b38f78cf500a293837e303b0db78890a8 100644
--- a/chrome/browser/resources/optimize_webui.gni
+++ b/chrome/browser/resources/optimize_webui.gni
@@ -44,6 +44,7 @@ template("optimize_webui") {
     }
 
     deps = invoker.deps
+    deps += [ "//brave/ui/webui/resources:modulize", ]
 
     # Note that we have to manually pass the sources to our script if the
     # script needs them as inputs.
