diff --git a/chrome/browser/resources/settings/site_settings/constants.ts b/chrome/browser/resources/settings/site_settings/constants.ts
index b51c1d25ae44512d0218030436feed867a1c216a..e03915964d1cdf887bb6b24a16e530591ab935f4 100644
--- a/chrome/browser/resources/settings/site_settings/constants.ts
+++ b/chrome/browser/resources/settings/site_settings/constants.ts
@@ -46,6 +46,7 @@ export enum ContentSettingsTypes {
   // The following item is not in the C++ kContentSettingsTypeGroupNames, but it
   // is used everywhere where ContentSettingsTypes is used in JS.
   PDF_DOCUMENTS = 'pdfDocuments',
+  AUTOPLAY = 'autoplay', ETHEREUM = 'ethereum', SOLANA = 'solana', BRAVE_SHIELDS = 'braveShields',
 }
 
 /**
