diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index d5b479bc99915992cbceefbcfac71d3f00994dbb..fe6d141fbf97c754bf4634c980f4b7f9cd5a8435 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2194,6 +2194,7 @@ static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (build_with_tflite_lib) {
