diff --git a/chrome/browser/ui/webui/theme_source.cc b/chrome/browser/ui/webui/theme_source.cc
index 04e897913a48d9321b20070cb6cffe3bf9fbfd08..0f9de8a808af1d878b70f18d1c8b6bb848e980dc 100644
--- a/chrome/browser/ui/webui/theme_source.cc
+++ b/chrome/browser/ui/webui/theme_source.cc
@@ -135,7 +135,7 @@ void ThemeSource::StartDataRequest(
   int resource_id = -1;
   if (parsed_path == "current-channel-logo") {
     switch (chrome::GetChannel()) {
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OFFICIAL_BUILD)
       case version_info::Channel::CANARY:
         resource_id = IDR_PRODUCT_LOGO_32_CANARY;
         break;
@@ -157,7 +157,10 @@ void ThemeSource::StartDataRequest(
         [[fallthrough]];
 #endif
       case version_info::Channel::UNKNOWN:
+        resource_id = IDR_PRODUCT_LOGO_32_DEVELOPMENT;
+#if 0
         resource_id = IDR_PRODUCT_LOGO_32;
+#endif
         break;
     }
   } else {
