diff --git a/chrome/browser/ui/omnibox/omnibox_popup_view.h b/chrome/browser/ui/omnibox/omnibox_popup_view.h
index 26b199944d786e8c09bbf92d890915702aad21b8..250abcb1ae697e54380625ff767f138dbc501b94 100644
--- a/chrome/browser/ui/omnibox/omnibox_popup_view.h
+++ b/chrome/browser/ui/omnibox/omnibox_popup_view.h
@@ -109,6 +109,7 @@ class OmniboxPopupView {
   base::TimeTicks construction_time() const { return construction_time_; }
 
  protected:
+  BRAVE_OMNIBOX_POPUP_VIEW_FRIEND
   friend class OmniboxResultView;
   friend class OmniboxSuggestionButtonRowView;
 
