diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 0c31ee896f98e84d40448f612c82f6306e7e3a21..e931506dceffc50e3dae462c4332f73219104171 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -722,6 +722,7 @@ class Browser : public TabStripModelObserver,
 
   StatusBubble* GetStatusBubbleForTesting();
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
