diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index e2dcc5447606a4c0ba556e42654dd978d99eb562..6abbf748c8bbc05866be36988307e0310ecea89a 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/common/sources.gni")
 import("//build/buildflag_header.gni")
 import("//build/config/chrome_build.gni")
 import("//build/util/process_version.gni")
@@ -88,6 +89,7 @@ source_set("channel_info") {
   deps = [ "//build:branding_buildflags" ]
   public_deps = [
     "//base",
+    "//brave/common:channel_info",
     "//components/version_info",
     "//components/version_info:version_string",
   ]
@@ -176,6 +178,7 @@ static_library("common_lib") {
     "webui_util.cc",
     "webui_util.h",
   ]
+  sources += brave_chrome_common_sources
   defines = []
 
   configs += [
@@ -267,6 +270,7 @@ static_library("common_lib") {
     "//ui/resources",
     "//url",
   ]
+  public_deps += brave_chrome_common_public_deps
 
   deps = [
     "//components/cdm/common:buildflags",
@@ -282,6 +286,7 @@ static_library("common_lib") {
     "//components/pdf/common:constants",
     "//components/pdf/common:util",
   ]
+  deps += brave_chrome_common_deps
 
   if (is_mac || is_win || is_linux || is_chromeos) {
     deps += [ "//components/webapps/isolated_web_apps:scheme" ]
@@ -471,6 +476,7 @@ static_library("non_code_constants") {
     "env_vars.h",
   ]
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  configs += [ "//brave/common:constants_configs" ]
 
   deps = [
     ":buildflags",
@@ -598,6 +604,7 @@ static_library("constants") {
     "//printing/buildflags",
     "//third_party/widevine/cdm:buildflags",
   ]
+  deps += brave_chrome_constants_deps
 
   if (is_android) {
     sources += [ "chrome_paths_android.cc" ]
