diff --git a/chrome/browser/profile_resetter/reset_report_uploader.cc b/chrome/browser/profile_resetter/reset_report_uploader.cc
index 65e0ce74a5e6c16a947658ff351580a5db506674..53dce32920884ec21aa975e48bea73a9f7bef6a6 100644
--- a/chrome/browser/profile_resetter/reset_report_uploader.cc
+++ b/chrome/browser/profile_resetter/reset_report_uploader.cc
@@ -49,6 +49,7 @@ void ResetReportUploader::DispatchReport(
 
 void ResetReportUploader::DispatchReportInternal(
     const std::string& request_data) {
+  if ((true)) return; // feature disabled in Brave
   // Create traffic annotation tag.
   net::NetworkTrafficAnnotationTag traffic_annotation =
       net::DefineNetworkTrafficAnnotation("profile_resetter_upload", R"(
