diff --git a/chrome/installer/setup/setup_main.cc b/chrome/installer/setup/setup_main.cc
index 10acca87dfb81dc744b4fb31fe28e58fd2a53db3..b4a34f2609b9765e6274a2dcf896de7807367f2a 100644
--- a/chrome/installer/setup/setup_main.cc
+++ b/chrome/installer/setup/setup_main.cc
@@ -706,6 +706,7 @@ installer::InstallStatus InstallProducts(InstallationState& original_state,
     }
   }
 
+  BRAVE_INSTALL_PRODUCTS
   return install_status;
 }
 
@@ -1533,6 +1534,7 @@ int SetupMain() {
                                      &exit_code)) {
     return exit_code;
   }
+  BRAVE_SETUP_MAIN
 
   if (system_install && !IsUserAnAdmin()) {
     if (!cmd_line.HasSwitch(installer::switches::kRunAsAdmin)) {
