diff --git a/chrome/browser/ui/navigator/browser_navigator.cc b/chrome/browser/ui/navigator/browser_navigator.cc
index 67fde190279e188b4c2200cbbac9771ccbc426ec..7aab910217ae5ddf6286706f1ff7517c81c18eb5 100644
--- a/chrome/browser/ui/navigator/browser_navigator.cc
+++ b/chrome/browser/ui/navigator/browser_navigator.cc
@@ -574,6 +574,7 @@ base::WeakPtr<content::NavigationHandle> Navigate(NavigateParams* params) {
       return nullptr;
     }
   }
+  BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL
 
   // Open System Apps in their standalone window if necessary.
   // TODO(crbug.com/40136163): Remove this code after we integrate with intent
@@ -813,6 +814,7 @@ base::WeakPtr<content::NavigationHandle> Navigate(NavigateParams* params) {
          params->user_gesture)) {
       tab_to_insert->set_opener(
           tabs::TabInterface::MaybeGetFromContents(params->source_contents));
+      BRAVE_NAVIGATE_MARK_OPENER_FOR_SYNC_NAVIGATION
     }
   }
 
@@ -830,6 +832,7 @@ base::WeakPtr<content::NavigationHandle> Navigate(NavigateParams* params) {
            params->user_gesture)) {
         tab_to_insert->set_opener(
             tabs::TabInterface::MaybeGetFromContents(params->source_contents));
+        BRAVE_NAVIGATE_MARK_OPENER_FOR_SYNC_NAVIGATION
       }
       contents_to_navigate_or_insert = tab_to_insert->GetContents();
 
