diff --git a/chrome/browser/lifetime/BUILD.gn b/chrome/browser/lifetime/BUILD.gn
index e214f4beeb8eaec4cbe9e94033c448edee4da723..114de0839d3db40fed21f8577cb31523472b3096 100644
--- a/chrome/browser/lifetime/BUILD.gn
+++ b/chrome/browser/lifetime/BUILD.gn
@@ -143,6 +143,7 @@ source_set("impl") {
   }
 
   if (!is_android) {
+    import("//brave/chromium_src/chrome/browser/lifetime/sources.gni") deps += brave_chrome_browser_lifetime_impl_deps
     sources += [
       "application_lifetime_desktop.cc",
       "browser_close_manager.cc",
