diff --git a/chrome/browser/extensions/updater/chrome_update_client_config.h b/chrome/browser/extensions/updater/chrome_update_client_config.h
index de6c63b9c0853df7bef89fb0121a622ea842cf4d..ee72ffd6c9f05d2ac4e9f699bdee67e32b18d3ed 100644
--- a/chrome/browser/extensions/updater/chrome_update_client_config.h
+++ b/chrome/browser/extensions/updater/chrome_update_client_config.h
@@ -86,6 +86,7 @@ class ChromeUpdateClientConfig : public update_client::Configurator {
   update_client::UpdaterStateProvider GetUpdaterStateProvider() const override;
   std::optional<base::FilePath> GetCrxCachePath() const override;
   bool IsConnectionMetered() const override;
+  BRAVE_CHROME_UPDATE_CLIENT_CONFIG_H_
 
  protected:
   friend class base::RefCountedThreadSafe<ChromeUpdateClientConfig>;
