diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 7429f324e6e856eaa8caf06999f483de3d2b6e4e..1f935c123c49d82bec9b1f40ac3e5b0e87d72e66 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -806,6 +806,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
