diff --git a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java
index 5e79f959b344a167ecd865c9ea3912886be2e2ef..1b27e7b2e66dd1bf19fcac1234a67893f61cce0d 100644
--- a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java
+++ b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/fusebox/FuseboxCoordinator.java
@@ -480,6 +480,9 @@ public class FuseboxCoordinator implements TemplateUrlServiceObserver {
     // TemplateUrlServiceObserver
     @Override
     public void onTemplateURLServiceChanged() {
+        if (!mTemplateUrlService.hasNativeService()) {
+            return;
+        }
         boolean isDseGoogle = mTemplateUrlService.isDefaultSearchEngineGoogle();
         if (isDseGoogle == mDefaultSearchEngineIsGoogle) return;
         mDefaultSearchEngineIsGoogle = isDseGoogle;
