diff --git a/chrome/browser/resources_util.cc b/chrome/browser/resources_util.cc
index eb57680b1155f8b116ee4c17b8f71a3cd98eb660..9a36b48fbfa67d804d0726f744346f484ae01b65 100644
--- a/chrome/browser/resources_util.cc
+++ b/chrome/browser/resources_util.cc
@@ -60,6 +60,7 @@ class ThemeMap {
                            kUiChromeosResources[i].id);
     }
 #endif
+    BRAVE_RESOURCES_UTIL
 
     id_map_ = StringIntMap(std::move(storage));
   }
