diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 60638f89e6aa890effa80642a3be8264f3b1613f..7cd22615b6de3a3fe00910eba48ee96c30f9b673 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -57,6 +57,7 @@ class AvatarToolbarButton : public ToolbarButton,
   static const char kAvatarToolbarButtonClassName[];
 
  private:
+  friend class BraveAvatarToolbarButton;
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
