diff --git a/chrome/renderer/worker_content_settings_client.h b/chrome/renderer/worker_content_settings_client.h
index 9829059249489c6f6fdd8c176061ce4b4ef6ee2b..00eefd3c2e833412ec8b6e80052771deef6649a8 100644
--- a/chrome/renderer/worker_content_settings_client.h
+++ b/chrome/renderer/worker_content_settings_client.h
@@ -37,6 +37,7 @@ class WorkerContentSettingsClient : public blink::WebContentSettingsClient {
   bool AllowScriptFromSource(bool enabled_per_settings,
                              const blink::WebURL& script_url) override;
   bool ShouldAutoupgradeMixedContent() override;
+  BRAVE_WORKER_CONTENT_SETTINGS_CLIENT_H
 
  private:
   explicit WorkerContentSettingsClient(
