diff --git a/chrome/browser/download/download_item_model.h b/chrome/browser/download/download_item_model.h
index 8358f379b77881ea4e87ecad8a937c209e0af7a8..c92b020320b282b0aa8dee9fbc84c7e3a9cb8e25 100644
--- a/chrome/browser/download/download_item_model.h
+++ b/chrome/browser/download/download_item_model.h
@@ -125,6 +125,7 @@ class DownloadItemModel : public DownloadUIModel,
   void OnDownloadDestroyed(download::DownloadItem* download) override;
 
  private:
+  friend class BraveDownloadItemModel;
   // DownloadUIModel implementation.
   std::string GetMimeType() const override;
   bool IsExtensionDownload() const override;
