diff --git a/chrome/browser/ui/views/page_info/page_info_view_factory.cc b/chrome/browser/ui/views/page_info/page_info_view_factory.cc
index 85c3a34c5ec00e28983502a276b9853fa897c4d7..1b7c573e2c8139039050a7ece602adb12a0d9ef7 100644
--- a/chrome/browser/ui/views/page_info/page_info_view_factory.cc
+++ b/chrome/browser/ui/views/page_info/page_info_view_factory.cc
@@ -487,6 +487,9 @@ const ui::ImageModel PageInfoViewFactory::GetPermissionIcon(
 
   icon = &gfx::VectorIcon::EmptyIcon();
   switch (permission.type) {
+    case ContentSettingsType::AUTOPLAY:
+      icon = &kAutoplayStatusIcon;
+      break;
     case ContentSettingsType::COOKIES:
       icon = &vector_icons::kDatabaseIcon;
       break;
