diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index 7db524b4a024361354bee52acea1841dadbf0835..594f6c6bc11da561d8230068a3c7a4d03f328e9c 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -174,6 +174,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   // KeepAliveStateObserver implementation
   void OnKeepAliveStateChanged(bool is_keeping_alive) override;
   void OnKeepAliveRestartStateChanged(bool can_restart) override;
