diff --git a/chrome/browser/ui/views/frame/browser_window_factory.cc b/chrome/browser/ui/views/frame/browser_window_factory.cc
index 9089e38b6a1a..b441fdc9ff82 100644
--- a/chrome/browser/ui/views/frame/browser_window_factory.cc
+++ b/chrome/browser/ui/views/frame/browser_window_factory.cc
@@ -37,7 +37,7 @@ BrowserWindow* BrowserWindow::CreateBrowserWindow(
   if (in_tab_dragging)
     browser_frame->SetTabDragKind(TabDragKind::kAllTabs);
   browser_frame->InitBrowserFrame();
-
+#if !defined(OS_ANDROID)
   view->GetWidget()->non_client_view()->SetAccessibleName(
       l10n_util::GetStringUTF16(IDS_PRODUCT_NAME));
 
@@ -46,6 +46,7 @@ BrowserWindow* BrowserWindow::CreateBrowserWindow(
   // because it requires gfx::NativeWindow to be an aura::Window*.
   view->GetWidget()->GetNativeWindow()->SetProperty(
       aura::client::kCreatedByUserGesture, user_gesture);
+#endif
 #endif
   return view;
 }
