diff --git a/chrome/browser/ui/views/side_panel/side_panel_coordinator.cc b/chrome/browser/ui/views/side_panel/side_panel_coordinator.cc
index d987aa8efd9c78c21b529ef2a9e6e0cfe22899f8..aab65cf9071fa7eb2968cecfe15a120824ee9577 100644
--- a/chrome/browser/ui/views/side_panel/side_panel_coordinator.cc
+++ b/chrome/browser/ui/views/side_panel/side_panel_coordinator.cc
@@ -175,6 +175,7 @@ SidePanelCoordinator::~SidePanelCoordinator() {
 void SidePanelCoordinator::Show(
     absl::optional<SidePanelEntry::Id> entry_id,
     absl::optional<SidePanelUtil::SidePanelOpenTrigger> open_trigger) {
+  BRAVE_SIDE_PANEL_COORDINATOR_SHOW
   if (!entry_id.has_value())
     entry_id = GetLastActiveEntryId().value_or(kDefaultEntry);
 
@@ -485,6 +486,7 @@ std::unique_ptr<views::View> SidePanelCoordinator::CreateHeader() {
       ChromeLayoutProvider::Get()->GetDistanceMetric(
           ChromeDistanceMetric::DISTANCE_SIDE_PANEL_HEADER_VECTOR_ICON_SIZE)));
 
+  BRAVE_SIDE_PANEL_COORDINATOR_CREATE_HEADER
   return header;
 }
 
