diff --git a/chrome/browser/resources/tools/rollup_plugin.js b/chrome/browser/resources/tools/rollup_plugin.js
index d6c49bf8164571c0d86904bc7dea45dfc6912ffe..510f2aca1942faa14a025b809ca65194a2805b55 100644
--- a/chrome/browser/resources/tools/rollup_plugin.js
+++ b/chrome/browser/resources/tools/rollup_plugin.js
@@ -78,6 +78,8 @@ export default function plugin(srcPath, genPath, rootPath, host, excludes) {
         return joinPaths(polymerSrcPath, pathFromPolymer);
       }
 
+      const resolvedByBrave = require(path.join(__dirname, '..', '..', 'brave_rollup_overrides')).braveResolveId({srcPath, genPath, rootPath, host, excludes}, source, origin, relativePath, joinPaths, combinePaths, chromeResourcesUrl, nonGeneratedFiles); if (resolvedByBrave) { return resolvedByBrave; }
+
       // Get path from ui/webui/resources
       let pathFromResources = '';
       if (source.startsWith(chromeResourcesUrl)) {
