diff --git a/chrome/installer/linux/BUILD.gn b/chrome/installer/linux/BUILD.gn
index 225c6fd2669818225e1877ef21774b04abb1a6f3..ae4c899764d602450d16c7479424f8369693a0e8 100644
--- a/chrome/installer/linux/BUILD.gn
+++ b/chrome/installer/linux/BUILD.gn
@@ -19,8 +19,8 @@ if (current_cpu == "x86" || current_cpu == "x64") {
 assert(is_linux)
 
 packaging_files_executables = [
-  "$root_out_dir/chrome",
-  "$root_out_dir/chrome_sandbox",
+  "$root_out_dir/brave",
+  "$root_out_dir/brave_sandbox",
 ]
 packaging_files_shlibs = []
 
@@ -153,7 +153,7 @@ action("merge_rpm_dependencies") {
 }
 
 action("strip_chrome_binary") {
-  prog_name = "$root_out_dir/chrome"
+  prog_name = "$root_out_dir/brave"
   debug_file = prog_name + ".debug"
   stripped_file = prog_name + ".stripped"
   deps = [
@@ -243,6 +243,16 @@ copy("common_packaging_files") {
       "common/chromium-browser/chromium-browser.appdata.xml",
       "common/chromium-browser/chromium-browser.info",
     ]
+
+    sources -= [
+      "common/chromium-browser/chromium-browser.appdata.xml",
+      "common/chromium-browser/chromium-browser.info",
+    ]
+
+    sources += [
+      "//brave/chromium_src/chrome/installer/linux/common/brave-browser/chromium-browser.appdata.xml",
+      "//brave/chromium_src/chrome/installer/linux/common/brave-browser/chromium-browser.info",
+    ]
   }
 
   sources += [ "//third_party/eu-strip/bin/eu-strip" ]
@@ -411,7 +421,7 @@ template("linux_package") {
     if (is_chrome_branded) {
       package = "google-chrome"
     } else {
-      package = "chromium-browser"
+      package = "brave-browser-chromium-fork"
     }
     outputs = [
       "$root_out_dir/${package}-${channel}_${chrome_version_full}-1_${deb_arch}.deb",
@@ -463,7 +473,7 @@ template("linux_package") {
       if (is_chrome_branded) {
         package = "google-chrome"
       } else {
-        package = "chromium-browser"
+        package = "brave-browser-chromium-fork"
       }
       outputs = [
         "$root_out_dir/${package}-${channel}-${chrome_version_full}-1.${rpm_arch}.rpm",
