diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 2c6382a72582a46ff806bf5d9e5271f7866e0636..76f54ec8dd6553e507b4c251b529434d765d460a 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -79,6 +79,7 @@ source_set("channel_info") {
   if (is_android) {
     public_deps += [ "//components/version_info/android:channel_getter" ]
   }
+  if (brave_chromium_build) { deps = [ "//brave/common:channel_info" ] }
 }
 
 source_set("ini_parser") {
@@ -549,6 +550,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:ui_features",
   ]
+  if (brave_chromium_build) { configs += [ "//brave/common:constants_configs" ] }
 }
 
 component("chrome_features") {
