diff --git a/chrome/updater/mac/.install.sh b/chrome/updater/mac/.install.sh
index 3b1874d09fca0cf88e361927dfea7c4273790030..b5abcdf22df138367cf2da8f3eb1cabfd796b6b8 100755
--- a/chrome/updater/mac/.install.sh
+++ b/chrome/updater/mac/.install.sh
@@ -322,7 +322,7 @@ main() {
   # --ignore-times, which is desirable, as it forces rsync to copy files even
   # when their sizes and modification times are identical, as their content
   # still may be different.
-  readonly RSYNC_FLAGS="--ignore-times --links --perms --recursive --times"
+  readonly RSYNC_FLAGS="--ignore-times --links $(if [[ ${EUID} -eq 0 ]]; then echo "--perms --recursive --times"; else echo "--no-perms --executability --chmod=u=rwX,go=rX --recursive"; fi)"
 
   note "update_dmg_mount_point = ${update_dmg_mount_point}"
 
@@ -337,7 +337,7 @@ main() {
   # The update to install.
 
   # update_app is the path to the new version of the .app.
-  local update_app="${update_dmg_mount_point}/${APP_DIR}"
+  local update_app="${update_dmg_mount_point}/${PRODUCT_NAME}.app"
   note "update_app = ${update_app}"
 
   # Make sure that it's an absolute path.
@@ -455,6 +455,30 @@ main() {
   # directory unusable even for an instant.
   if [[ -n "${update_versioned_dir}" ]]; then
     note "rsyncing versioned directory"
+    local debug_flags=0
+    if [[ ! -d "${new_versioned_dir}" ]]; then
+      debug_flags=1
+      local mkdir_stderr
+      if ! mkdir_stderr="$(mkdir -p "${new_versioned_dir}" 2>&1 1>/dev/null)"
+      then
+        case "${mkdir_stderr}" in
+          *"Read-only file system"*)
+            exit 90 ;;
+          *"No space left on device"*|*"Disc quota exceeded"*)
+            exit 91 ;;
+          *"Operation not permitted"*)
+            exit 92 ;;
+          *"Permission denied"*)
+            exit 93 ;;
+          *"File exists"*|*"Not a directory"*)
+            exit 94 ;;
+          *)
+            exit 95 ;;
+        esac
+      fi
+      debug_flags=$((debug_flags | 2))
+      rm -rf "${new_versioned_dir}"
+    fi
     if ! rsync ${RSYNC_FLAGS} --delete-before "${update_versioned_dir}/" \
                                               "${new_versioned_dir}"; then
       err "rsync of versioned directory failed, status ${PIPESTATUS[0]}"
@@ -463,7 +487,9 @@ main() {
       # The incomplete version would break code signature validation.
       note "cleaning up new_versioned_dir"
       rm -rf "${new_versioned_dir}"
-      exit 12
+      rsync ${RSYNC_FLAGS} --delete-before "${update_versioned_dir}" "${installed_versions_dir}" && (( debug_flags |= 4 ))
+      rm -rf "${new_versioned_dir}"
+      exit $((80 + debug_flags))
     fi
   fi
 
