diff --git a/chrome/browser/history/history_tab_helper.cc b/chrome/browser/history/history_tab_helper.cc
index cff7c3955c8a1063d8dfdec5b5d0e46afb7b4789..fc78f52360fcaf84de8c219655432794d7b44908 100644
--- a/chrome/browser/history/history_tab_helper.cc
+++ b/chrome/browser/history/history_tab_helper.cc
@@ -644,6 +644,9 @@ void HistoryTabHelper::TitleWasSet(NavigationEntry* entry) {
 }
 
 history::HistoryService* HistoryTabHelper::GetHistoryService() {
+  if (BraveTabRequestedOffTheRecord(web_contents())) {
+    return nullptr;
+  }
   Profile* profile =
       Profile::FromBrowserContext(web_contents()->GetBrowserContext());
   if (profile->IsOffTheRecord()) {
