diff --git a/chrome/browser/ui/browser_commands.cc b/chrome/browser/ui/browser_commands.cc
index 8dfeb6d4f0012f487db0599eaf8c409f568db8cb..78812fa24f9149a2cc8961997337d843c19ccde1 100644
--- a/chrome/browser/ui/browser_commands.cc
+++ b/chrome/browser/ui/browser_commands.cc
@@ -1182,6 +1182,7 @@ void CloseTab(BrowserWindowInterface* browser) {
   }
 
   ToastController* toast_controller = browser->GetFeatures().toast_controller();
+  BRAVE_CLOSE_TAB
   if (!toast_controller) {
     CloseSelectedTabAndRecordTabCountMetric(browser);
     return;
