diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/autofill/AutofillServerProfileFragment.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/autofill/AutofillServerProfileFragment.java
index c8119f81df2fdaa5d225d68cdf2905b0f2550af6..a88326a7b0f21f9b4e76728e8b22ab8b781a680c 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/autofill/AutofillServerProfileFragment.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/autofill/AutofillServerProfileFragment.java
@@ -20,7 +20,7 @@ import org.chromium.chrome.browser.settings.SettingsUtils;
  * Fragment for settings page that allows user to view and edit a single server-provided address.
  */
 public class AutofillServerProfileFragment
-        extends PreferenceFragmentCompat implements Preference.OnPreferenceClickListener {
+        extends org.chromium.chrome.browser.settings.BravePreferenceFragment implements Preference.OnPreferenceClickListener {
     private String mGUID;
 
     private static final String PREF_SERVER_PROFILE_DESCRIPTION = "server_profile_description";
