diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index 6e7dd0b39f10fd13f91fb4ddc2315961c2635aa2..ec1806a51f9ff59ffc921fafd91fda7a0a7eb3c3 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -76,7 +76,7 @@ import org.chromium.ui.widget.ButtonCompat;
 /**
  * Settings fragment to enable Sync and other services that communicate with Google.
  */
-public class SyncAndServicesSettings extends PreferenceFragmentCompat
+public class SyncAndServicesSettings extends org.chromium.chrome.browser.settings.BravePreferenceFragment
         implements PassphraseDialogFragment.Listener, Preference.OnPreferenceChangeListener,
                    ProfileSyncService.SyncStateChangedListener,
                    SettingsActivity.OnBackPressedListener {
