diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index b44d79d8855d8ed50ac41badc9190ca1efd73a23..780d9363c1643d07566e0573cc14904ab2fd4af3 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -147,7 +147,7 @@ const char Profile::kProfileKey[] = "__PROFILE__";
 void Profile::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
       prefs::kSearchSuggestEnabled,
-      true,
+      false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 #if defined(OS_ANDROID)
   registry->RegisterStringPref(
@@ -251,7 +251,7 @@ bool Profile::IsGuestSession() const {
           chromeos::switches::kGuestSession);
   return is_guest_session;
 #else
-  return is_guest_profile_;
+  return is_guest_profile_ || IsTorProfile();
 #endif
 }
 
