diff --git a/chrome/utility/chrome_content_utility_client.h b/chrome/utility/chrome_content_utility_client.h
index bff5ba92a14acd502466231ac3bd4f30005799a0..173ae98e6e24eb18a9152dad5af006954fc847ba 100644
--- a/chrome/utility/chrome_content_utility_client.h
+++ b/chrome/utility/chrome_content_utility_client.h
@@ -41,6 +41,7 @@ class ChromeContentUtilityClient : public content::ContentUtilityClient {
       const NetworkBinderCreationCallback& callback);
 
  private:
+  friend class BraveContentUtilityClient;
   std::unique_ptr<service_manager::Service> MaybeCreateMainThreadService(
       const std::string& service_name,
       service_manager::mojom::ServiceRequest request);
