diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 3f1940b7e597ec0d7212902057ab2c2b5be225ac..0d107e3b22d12f1e6410de8bab92610c933b2781 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2497,6 +2497,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
