diff --git a/chrome/installer/mini_installer/configuration.cc b/chrome/installer/mini_installer/configuration.cc
index 932aaedc2d8134e1f2957e867aa4e22b1d3871a5..e56c3e889d59ad9b58d6132e95b85f98001577e1 100644
--- a/chrome/installer/mini_installer/configuration.cc
+++ b/chrome/installer/mini_installer/configuration.cc
@@ -24,7 +24,11 @@ namespace {
 bool GetGoogleUpdateIsMachineEnvVar() {
   const DWORD kBufferSize = 2;
   StackString<kBufferSize> value;
+#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
+  DWORD length = ::GetEnvironmentVariableW(L"BraveSoftwareUpdateIsMachine",
+#else
   DWORD length = ::GetEnvironmentVariableW(L"GoogleUpdateIsMachine",
+#endif
                                            value.get(), kBufferSize);
   return length == 1 && *value.get() == L'1';
 }
@@ -77,7 +81,9 @@ bool Configuration::ParseCommandLine(const wchar_t* command_line) {
   for (int i = 1; i < argument_count_; ++i) {
     if (0 == ::lstrcmpi(args_[i], L"--system-level"))
       is_system_level_ = true;
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))
+    // Use original install switch because it isn't visible to user.
+    // Sync this switch with brave/.../chromium_install_modes.cc
     else if (0 == ::lstrcmpi(args_[i], L"--chrome-beta"))
       chrome_app_guid_ = google_update::kBetaAppGuid;
     else if (0 == ::lstrcmpi(args_[i], L"--chrome-dev"))
