diff --git a/chrome/browser/language/android/java/src/org/chromium/chrome/browser/language/settings/LanguageSettings.java b/chrome/browser/language/android/java/src/org/chromium/chrome/browser/language/settings/LanguageSettings.java
index 2de272e8e851996eeb823a09fc650237cd169c02..f81c3c5034b389ba09398c2590786587dd7dd113 100644
--- a/chrome/browser/language/android/java/src/org/chromium/chrome/browser/language/settings/LanguageSettings.java
+++ b/chrome/browser/language/android/java/src/org/chromium/chrome/browser/language/settings/LanguageSettings.java
@@ -37,7 +37,7 @@ import java.util.Collection;
  * Settings fragment that displays information about Chrome languages, which allow users to
  * seamlessly find and manage their languages preferences across platforms.
  */
-public class LanguageSettings extends PreferenceFragmentCompat
+public class LanguageSettings extends org.chromium.chrome.browser.settings.BravePreferenceFragment
         implements AddLanguageFragment.Launcher, FragmentSettingsLauncher {
     // Default number of items to list in a collection preference summary.
     private static final int COLLECTION_SUMMARY_ITEM_LIMIT = 3;
