diff --git a/chrome/browser/profiles/profile_impl.cc b/chrome/browser/profiles/profile_impl.cc
index 86ffb0c6cfc3c67b642d68e7320a0ea65ae1f3e5..ab59ab2f8328ad5b4135ac86cfb7a96749a87717 100644
--- a/chrome/browser/profiles/profile_impl.cc
+++ b/chrome/browser/profiles/profile_impl.cc
@@ -371,7 +371,7 @@ Profile* Profile::CreateProfile(const base::FilePath& path,
   }
 
   auto profile = base::WrapUnique(
-      new ProfileImpl(path, delegate, create_mode, io_task_runner));
+      new BraveProfileImpl(path, delegate, create_mode, io_task_runner));
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS) && !defined(OS_ANDROID) && \
     !defined(OS_CHROMEOS)
   if (create_mode == CREATE_MODE_SYNCHRONOUS && profile->IsLegacySupervised())
@@ -554,6 +554,9 @@ ProfileImpl::ProfileImpl(
   BrowserContextDependencyManager::GetInstance()
       ->RegisterProfilePrefsForServices(pref_registry_.get());
 
+  // Done here instead of browser_prefs.cc so it can override default prefs
+  brave::RegisterProfilePrefs(pref_registry_.get());
+
   SupervisedUserSettingsService* supervised_user_settings = nullptr;
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
   supervised_user_settings =
