diff --git a/chrome/android/java/src/org/chromium/chrome/browser/partnercustomizations/HomepageManager.java b/chrome/android/java/src/org/chromium/chrome/browser/partnercustomizations/HomepageManager.java
index 8aa5832190e8a11fbcb69eb570c8e7b15544f0f6..d390de93cd81bd90a3086bdee7aa2e4032025ef4 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/partnercustomizations/HomepageManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/partnercustomizations/HomepageManager.java
@@ -104,8 +104,9 @@ public class HomepageManager implements HomepagePolicyManager.HomepagePolicyStat
      * @return Whether to close the app when the user has zero tabs.
      */
     public static boolean shouldCloseAppWithZeroTabs() {
-        return HomepageManager.isHomepageEnabled()
-                && !NewTabPage.isNTPUrl(HomepageManager.getHomepageUri());
+        return CloseBraveManager.shouldCloseAppWithZeroTabs();
+        // return HomepageManager.isHomepageEnabled()
+        //         && !NewTabPage.isNTPUrl(HomepageManager.getHomepageUri());
     }
 
     /**
