diff --git a/chrome/browser/ui/autofill/BUILD.gn b/chrome/browser/ui/autofill/BUILD.gn
index 4e90419e2326a2fd2a50af63570795964badc3cf..430e614a97f19ccd502a27e268e143c540149ad5 100644
--- a/chrome/browser/ui/autofill/BUILD.gn
+++ b/chrome/browser/ui/autofill/BUILD.gn
@@ -253,6 +253,7 @@ source_set("impl") {
     "//components/variations/service",
     "//components/vector_icons",
   ]
+  import("//brave/chromium_src/chrome/browser/ui/autofill/sources.gni")  deps += brave_chromium_src_chrome_browser_ui_autofill_deps
 
   if (safe_browsing_mode != 0) {
     deps += [ "//components/safe_browsing/content/browser/password_protection" ]
@@ -279,6 +280,7 @@ source_set("impl") {
       "//components/messages/android:feature_flags",
       "//components/password_manager/core/browser/features:password_features",
     ]
+    import("//brave/browser/ui/autofill/sources.gni") deps += brave_chrome_browser_ui_autofill_deps
   } else {
     sources += [
       "address_bubbles_controller.cc",
