diff --git a/chrome/browser/ui/views/page_action/page_action_view.cc b/chrome/browser/ui/views/page_action/page_action_view.cc
index 590c15bc44e034f9ebdbde1fc038ca26590ad7ce..ee407dedf8d830f2f204651fa007be1e0e96f03b 100644
--- a/chrome/browser/ui/views/page_action/page_action_view.cc
+++ b/chrome/browser/ui/views/page_action/page_action_view.cc
@@ -156,7 +156,7 @@ void PageActionView::OnPageActionModelChanged(
   if (!visible) {
     ResetSlideAnimation(/*show=*/false);
     NotifyIsChipShowingChange();
-  } else if (model.ShouldShowSuggestionChip()) {
+  } else if (model.ShouldShowSuggestionChip() || model.GetAlwaysShowLabel()) {
     if (model.GetShouldAnimateChipIn()) {
       AnimateIn(/*string_id=*/std::nullopt);
     } else {
