diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index e0cc4ef2067e4631206939b404d3dcb0d2bb32ee..ecb8fcb3313acd449500ee1d65b40b39da5105eb 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1260,6 +1260,7 @@ static_library("extensions") {
   } else {
     sources += [ "api/braille_display_private/braille_controller_stub.cc" ]
   }
+  deps += brave_extensions_resources
 }
 
 static_library("test_support") {
