diff --git a/chrome/browser/ui/views/profiles/profile_menu_view.cc b/chrome/browser/ui/views/profiles/profile_menu_view.cc
index 3c714bfb47b3353e4275da5ea78a538a25f967a8..76abb72eeb6a83b4169014a47db00a2ee8d109f5 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view.cc
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.cc
@@ -1062,6 +1062,7 @@ void ProfileMenuView::AddOptionsView(bool display_lock,
   if (is_guest) {
     CreateAndAddButton(
         CreateVectorIcon(kCloseAllIcon),
+        BRAVE_ADDOPTIONSVIEW_ADD_TOR_EXIT_BUTTON_
         l10n_util::GetStringUTF16(IDS_PROFILES_EXIT_GUEST),
         base::BindRepeating(&ProfileMenuView::OnExitProfileButtonClicked,
                             base::Unretained(this)));
