diff --git a/chrome/browser/search_engines/template_url_service_factory.cc b/chrome/browser/search_engines/template_url_service_factory.cc
index b3f207c4d728b68fa725654e916f0d8d1e3b0c49..d93ca1e804bb0b7a83def669fa66c2ae6289982a 100644
--- a/chrome/browser/search_engines/template_url_service_factory.cc
+++ b/chrome/browser/search_engines/template_url_service_factory.cc
@@ -82,7 +82,12 @@ void TemplateURLServiceFactory::RegisterProfilePrefs(
 
 content::BrowserContext* TemplateURLServiceFactory::GetBrowserContextToUse(
     content::BrowserContext* context) const {
+#if defined(BRAVE_CHROMIUM_BUILD)
+  // To make different service for normal and incognito profile.
+  return chrome::GetBrowserContextOwnInstanceInIncognito(context);
+#else
   return chrome::GetBrowserContextRedirectedInIncognito(context);
+#endif
 }
 
 bool TemplateURLServiceFactory::ServiceIsNULLWhileTesting() const {
