diff --git a/chrome/browser/resources/tools/build_webui.gni b/chrome/browser/resources/tools/build_webui.gni
index dd8ed8d4b7c768a9d2a0d05f977828957c32d8b6..b63f37c3c72725756aecd33e4a8a94e454376cb3 100644
--- a/chrome/browser/resources/tools/build_webui.gni
+++ b/chrome/browser/resources/tools/build_webui.gni
@@ -197,6 +197,7 @@ template("build_webui") {
       }
       extra_deps += [ ":copy_mojo" ]
     }
+    if (defined(invoker.extra_build_ts_in_files)) { in_files += invoker.extra_build_ts_in_files } if (defined(invoker.extra_build_ts_extra_deps)) { extra_deps += invoker.extra_build_ts_extra_deps }
   }
 
   if (optimize) {
@@ -240,6 +241,7 @@ template("build_webui") {
       deps += invoker.extra_grdp_deps
       grdp_files = invoker.extra_grdp_files
     }
+    if (defined(invoker.extra_build_grd_deps)) { deps += invoker.extra_build_grd_deps }
   }
 
   grit("resources") {
