diff --git a/chrome/browser/extensions/extension_service.cc b/chrome/browser/extensions/extension_service.cc
index d685521a0adcbad6fcd2a3db206d28c4d8fc02ca..943be4d1ec67f5037d6b4492105d724e529c8c91 100644
--- a/chrome/browser/extensions/extension_service.cc
+++ b/chrome/browser/extensions/extension_service.cc
@@ -28,6 +28,7 @@
 #include "base/threading/thread_task_runner_handle.h"
 #include "base/time/time.h"
 #include "base/trace_event/trace_event.h"
+#include "brave/browser/extensions/brave_component_loader.h"
 #include "build/build_config.h"
 #include "chrome/browser/browser_process.h"
 #include "chrome/browser/chrome_notification_types.h"
@@ -326,7 +327,7 @@ ExtensionService::ExtensionService(Profile* profile,
                    profile)));
   }
 
-  component_loader_.reset(new ComponentLoader(
+  component_loader_.reset(new BraveComponentLoader(
       this, profile->GetPrefs(), g_browser_process->local_state(), profile));
 
   if (extensions_enabled_) {
@@ -1248,6 +1249,21 @@ void ExtensionService::AddComponentExtension(const Extension* extension) {
   }
 
   AddExtension(extension);
+#if defined(BRAVE_CHROMIUM_BUILD)
+  // ContentSettingsStore::RegisterExtension is only called for default components
+  // on the first run with a fresh profile. All restarts of the browser after that do not call it.
+  // This causes ContentSettingsStore's `entries_` to never insert the component ID
+  // and then  ContentSettingsStore::GetValueMap always returns nullptr.
+  // I don't think Chromium is affeced by this simply because they don't use content settings
+  // from default component extensions.
+  extension_prefs_->OnExtensionInstalled(extension, Extension::ENABLED,
+                                         syncer::StringOrdinal(),
+                                         extensions::kInstallFlagNone,
+                                         std::string(),
+                                         base::nullopt);
+  extensions::ContentSettingsService::Get(profile_)->OnExtensionPrefsLoaded(
+      extension->id(), extension_prefs_);
+#endif
 }
 
 void ExtensionService::CheckPermissionsIncrease(const Extension* extension,
