diff --git a/chrome/installer/linux/BUILD.gn b/chrome/installer/linux/BUILD.gn
index fbabd6c3414f5cb69cf4c7ab6ef5f957cb391376..b9a82deb60d40228dbe67bf5d91c88888d44eda6 100644
--- a/chrome/installer/linux/BUILD.gn
+++ b/chrome/installer/linux/BUILD.gn
@@ -25,11 +25,10 @@ declare_args() {
 assert(is_linux)
 
 packaging_files_executables = [
-  "$root_out_dir/chrome",
-  "$root_out_dir/chrome_sandbox",
+  "$root_out_dir/brave",
+  "$root_out_dir/brave_sandbox",
 ]
 packaging_files_shlibs = []
-
 if (enable_nacl) {
   packaging_files_executables += [
     "$root_out_dir/nacl_helper",
@@ -159,7 +158,7 @@ action("merge_rpm_dependencies") {
 }
 
 action("strip_chrome_binary") {
-  prog_name = "$root_out_dir/chrome"
+  prog_name = "$root_out_dir/brave"
   debug_file = prog_name + ".debug"
   stripped_file = prog_name + ".stripped"
   deps = [
@@ -246,8 +245,8 @@ copy("common_packaging_files") {
     ]
   } else {
     sources += [
-      "common/chromium-browser/chromium-browser.appdata.xml",
-      "common/chromium-browser/chromium-browser.info",
+      "//brave/chromium_src/chrome/installer/linux/common/brave-browser/chromium-browser.appdata.xml",
+      "//brave/chromium_src/chrome/installer/linux/common/brave-browser/chromium-browser.info",
     ]
   }
 
@@ -524,3 +523,6 @@ linux_package("beta") {
 linux_package("unstable") {
   channel = "unstable"
 }
+linux_package("nightly") {
+  channel = "nightly"
+}
