diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 43b6569625001a13904294d149109e53013e6e47..2bc7fcce46217cf7d6ee0ffd0f9d24bc54687c4a 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -853,6 +853,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
