diff --git a/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java
index a7fa3643972bde4557ba7a22cc2d6d60a55f9761..cbb55c3cc40bb2c8cf18f4b4f4be09bb7f9b6c1f 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java
@@ -50,7 +50,7 @@ import java.util.Set;
 
 /** Implements {@link MultiInstanceOrchestrator} as a singleton. */
 @NullMarked
-/* package */ final class MultiInstanceOrchestratorImpl implements MultiInstanceOrchestrator {
+/* package */ class MultiInstanceOrchestratorImpl implements MultiInstanceOrchestrator {
     private static @Nullable MultiInstanceOrchestrator sInstance;
     private static @Nullable TabReparentingDelegate sTabReparentingDelegateForTesting;
 
@@ -88,7 +88,7 @@ import java.util.Set;
         return sInstance;
     }
 
-    private MultiInstanceOrchestratorImpl(TabReparentingDelegate tabReparentingDelegate) {
+    MultiInstanceOrchestratorImpl(TabReparentingDelegate tabReparentingDelegate) {
         mTabReparentingDelegate = tabReparentingDelegate;
         ApplicationStatus.registerStateListenerForAllActivities(this::onActivityStateChange);
     }
