diff --git a/chrome/browser/renderer_context_menu/spelling_options_submenu_observer.h b/chrome/browser/renderer_context_menu/spelling_options_submenu_observer.h
index 8757b7f3deee5bdd6942123e2740cda44c1bae6f..0de51ce0d0406855bf891b39b763a8d4374f5503 100644
--- a/chrome/browser/renderer_context_menu/spelling_options_submenu_observer.h
+++ b/chrome/browser/renderer_context_menu/spelling_options_submenu_observer.h
@@ -15,10 +15,12 @@
 #include "ui/base/models/simple_menu_model.h"
 
 class RenderViewContextMenuProxy;
+class BraveSpellingOptionsSubMenuObserver;
 
 // A class that implements the 'spell-checker options' submenu. This class
 // creates the submenu, adds it to the parent menu, and handles events.
 class SpellingOptionsSubMenuObserver : public RenderViewContextMenuObserver {
+  friend class BraveSpellingOptionsSubMenuObserver;
  public:
   SpellingOptionsSubMenuObserver(RenderViewContextMenuProxy* proxy,
                                  ui::SimpleMenuModel::Delegate* delegate,
