diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.cc b/chrome/browser/ui/views/location_bar/location_bar_view.cc
index 24c769cf019feb8ea7c846d5e9bb6606011abf3b..6b488de58e06a2cbb3595ba490d1d71c263a9e95 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
@@ -881,6 +881,7 @@ void LocationBarView::Layout(PassKey) {
                                   .indent_input_when_popup_closed) &&
                              !location_icon_view_->ShouldShowLabel() &&
                              !ShouldShowKeywordBubble();
+  BRAVE_LOCATION_BAR_VIEW_LAYOUT_LEADING_DECORATIONS
   if (should_indent) {
     icon_left += 7 /*icon_indent*/;
     icon_left += omnibox_feature_configs::AdjustOmniboxIndent()
@@ -970,6 +971,7 @@ void LocationBarView::Layout(PassKey) {
       }
       selected_keyword_view_->SetCustomImage(image);
     }
+    BRAVE_LOCATION_BAR_VIEW_LAYOUT_HANDLE_SEARCH_PROMOTION_BUTTON_VISIBILITY
   } else if (location_icon_view_->GetShowText() &&
              !ShouldChipOverrideLocationIcon()) {
     location_icon_view_->SetVisible(true);
@@ -1012,6 +1014,7 @@ void LocationBarView::Layout(PassKey) {
       (info.num_legacy_page_actions_shown == 0) && !all_page_actions_migrated
           ? 4
           : trailing_decorations_edge_padding;
+  BRAVE_LOCATION_BAR_VIEW_LAYOUT_RIGHT_MOST_TRAILING_DECORATIONS
   add_trailing_decoration(page_action_icon_container_,
                           /*intra_item_padding=*/0,
                           /*edge_padding=*/
@@ -1037,6 +1040,7 @@ void LocationBarView::Layout(PassKey) {
 
   add_trailing_decoration(clear_all_button_, /*intra_item_padding=*/0,
                           /*edge_padding=*/trailing_decorations_edge_padding);
+  BRAVE_LOCATION_BAR_VIEW_LAYOUT_LEFT_MOST_TRAILING_DECORATIONS
 
   // Perform layout.
   int entry_width = width();
