diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 02d50d2a46337e298a97f011b63ed47ef60427b4..03129d2d5e3dfe6fe083d311b1cc206a5b35b79d 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -82,6 +82,7 @@ source_set("channel_info") {
   if (is_android) {
     public_deps += [ "//components/version_info/android:channel_getter" ]
   }
+  if (brave_chromium_build) { deps = [ "//brave/common:channel_info" ] }
 }
 
 source_set("ini_parser") {
@@ -554,6 +555,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  if (brave_chromium_build) { configs += [ "//brave/common:constants_configs" ] }
 }
 
 component("chrome_features") {
