diff --git a/chrome/common/extensions/api/developer_private.idl b/chrome/common/extensions/api/developer_private.idl
index bd7c2f03c906d418be6289e62e92ceefb6f6ce3f..ebe317ede8e87be29bcd832a986426e0ed3db78e 100644
--- a/chrome/common/extensions/api/developer_private.idl
+++ b/chrome/common/extensions/api/developer_private.idl
@@ -231,6 +231,7 @@ namespace developerPrivate {
     DOMString id;
     AccessModifier incognitoAccess;
     AccessModifier userScriptsAccess;
+    boolean isSplitMode;
     boolean incognitoAccessPendingChange;
     DOMString[] installWarnings;
     boolean isCommandRegistrationHandledExternally;
