diff --git a/chrome/test/base/testing_profile.cc b/chrome/test/base/testing_profile.cc
index bf7b151e7c4a7b0bcfafbcb8ace4b22168b10683..4e5212a3686de1f26b226d1cd8eaff9e6ee8d079 100644
--- a/chrome/test/base/testing_profile.cc
+++ b/chrome/test/base/testing_profile.cc
@@ -358,6 +358,7 @@ void TestingProfile::Init() {
       EnsureBrowserContextKeyedServiceFactoriesBuilt();
 
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
+  BRAVE_INIT
   if (!IsOffTheRecord()) {
     SupervisedUserSettingsService* settings_service =
         SupervisedUserSettingsServiceFactory::GetForKey(key_.get());
