diff --git a/chrome/renderer/worker_content_settings_client.h b/chrome/renderer/worker_content_settings_client.h
index a096a3417976a013ab88e90c41ea7d1c0281caed..bd0e8f6b3ef055a6b775bb0651f66aa53fa5739f 100644
--- a/chrome/renderer/worker_content_settings_client.h
+++ b/chrome/renderer/worker_content_settings_client.h
@@ -41,6 +41,7 @@ class WorkerContentSettingsClient : public blink::WebContentSettingsClient {
   bool AllowScriptFromSource(bool enabled_per_settings,
                              const blink::WebURL& script_url) override;
   bool ShouldAutoupgradeMixedContent() override;
+  BRAVE_WORKER_CONTENT_SETTINGS_CLIENT_H
 
  private:
   explicit WorkerContentSettingsClient(
