diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
index bc53c8309c5dbf52fbb6a00fb48b4129b28bf49c..37484de3e33d5b6a933d99125ee65bddf96febed 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
@@ -699,7 +699,7 @@ public abstract class ChromeActivity extends AsyncInitializationActivity
             var modalDialogManager = getModalDialogManager();
             assert modalDialogManager != null;
             SnackbarManager snackbarManager =
-                    new SnackbarManager(
+                    new org.chromium.chrome.browser.ui.messages.snackbar.BraveSnackbarManager(
                             this,
                             mBottomContainer,
                             windowAndroid,
@@ -3276,6 +3276,7 @@ public abstract class ChromeActivity extends AsyncInitializationActivity
      */
     public void onExitVr() {}
 
+    public void onTabStateInitializedHandler() {}
     private void clearToolbarResourceCache() {
         View v = findViewById(R.id.control_container);
         try {
