diff --git a/chrome/browser/ui/views/location_bar/location_icon_view.cc b/chrome/browser/ui/views/location_bar/location_icon_view.cc
index b4b4716eb7d04f1f33fbc609a91bb66e8a55f6d8..612c7d63e17bd0b8ca88068b5b66aed0bfdce1c9 100644
--- a/chrome/browser/ui/views/location_bar/location_icon_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_icon_view.cc
@@ -347,6 +347,7 @@ void LocationIconView::UpdateBackground() {
 
   SetBackgroundColor(GetColorProvider()->GetColor(id));
 
+  BRAVE_LOCATION_ICON_VIEW_UPDATE_BACKGROUND
   if (is_text_dangerous) {
     ConfigureInkDropForRefresh2023(this, kColorOmniboxSecurityChipInkDropHover,
                                    kColorOmniboxSecurityChipInkDropRipple);
