diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index fa70d696076a6bc031ce5980d291d0443c3fa2b0..c5d8d916ce0d3431cf367511ef88d5fb6ed35334 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -593,7 +593,7 @@ void ToolbarView::ChildPreferredSizeChanged(views::View* child) {
 bool ToolbarView::SetPaneFocusAndFocusDefault() {
   if (!location_bar_->HasFocus()) {
     SetPaneFocus(location_bar_);
-    location_bar_->FocusLocation();
+    location_bar_->FocusLocation(true);
     return true;
   }
 
