diff --git a/chrome/browser/browsing_topics/browsing_topics_internals_browsertest.cc b/chrome/browser/browsing_topics/browsing_topics_internals_browsertest.cc
index 49504cadb66631f16813988cd4d981ee26032fbf..1441c42027ed5407b06c008909046226b4271d46 100644
--- a/chrome/browser/browsing_topics/browsing_topics_internals_browsertest.cc
+++ b/chrome/browser/browsing_topics/browsing_topics_internals_browsertest.cc
@@ -670,7 +670,7 @@ Taxonomy version: 123
 }

 IN_PROC_BROWSER_TEST_F(BrowsingTopicsInternalsBrowserTest,
-                       ClassifierTab_ModelUnavailable) {
+                       DISABLED_ClassifierTab_ModelUnavailable) {
   fixed_browsing_topics_service()->SetWebUIGetBrowsingTopicsStateResultOverride(
       browsing_topics::mojom::WebUIGetBrowsingTopicsStateResult::
           NewOverrideStatusMessage("Failed to get the topics state."));
@@ -685,7 +685,7 @@ IN_PROC_BROWSER_TEST_F(BrowsingTopicsInternalsBrowserTest,
 )");
 }

-IN_PROC_BROWSER_TEST_F(BrowsingTopicsInternalsBrowserTest, ClassifierTab) {
+IN_PROC_BROWSER_TEST_F(BrowsingTopicsInternalsBrowserTest, DISABLED_ClassifierTab) {
   fixed_browsing_topics_service()->SetWebUIGetBrowsingTopicsStateResultOverride(
       browsing_topics::mojom::WebUIGetBrowsingTopicsStateResult::
           NewOverrideStatusMessage("Failed to get the topics state."));
