diff --git a/chrome/browser/profiles/profile_avatar_icon_util.cc b/chrome/browser/profiles/profile_avatar_icon_util.cc
index 1b71fa4cf30913956e7001c008c81fb42652ecfb..4c81be136c57e851770732809e9a4afc6c792f95 100644
--- a/chrome/browser/profiles/profile_avatar_icon_util.cc
+++ b/chrome/browser/profiles/profile_avatar_icon_util.cc
@@ -312,7 +312,7 @@ constexpr size_t kDefaultAvatarIconsCount = 1;
 #elif defined(OS_CHROMEOS)
 constexpr size_t kDefaultAvatarIconsCount = 27;
 #else
-constexpr size_t kDefaultAvatarIconsCount = 56;
+constexpr size_t kDefaultAvatarIconsCount = 56 + kBraveDefaultAvatarIconsCount;
 #endif
 
 #if !defined(OS_ANDROID)
@@ -419,6 +419,7 @@ size_t GetPlaceholderAvatarIndex() {
 
 size_t GetModernAvatarIconStartIndex() {
 #if !defined(OS_CHROMEOS) && !defined(OS_ANDROID)
+  BRAVE_GET_MODERN_AVATAR_ICON_START_INDEX
   return GetPlaceholderAvatarIndex() + 1;
 #else
   // Only use the placeholder avatar on ChromeOS and Android.
@@ -444,6 +445,7 @@ std::string GetPlaceholderAvatarIconUrl() {
 
 const IconResourceInfo* GetDefaultAvatarIconResourceInfo(size_t index) {
   CHECK_LT(index, kDefaultAvatarIconsCount);
+  BRAVE_GET_DEFAULT_AVATAR_ICON_RESOURCE_INFO
   static const IconResourceInfo resource_info[kDefaultAvatarIconsCount] = {
   // Old avatar icons:
 #if !defined(OS_ANDROID)
