diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 0c51b3af317627579e096e4aff5682ba1745b234..d1d45a5ed7ba91c6ac2e92082997c6cf939772c4 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -904,6 +904,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
