diff --git a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
index 049d2f777d3ffbb90e19f50b014d93d43946107d..cafafcdcdf70e768df14e87e063b90a601494ef9 100644
--- a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
+++ b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
@@ -558,7 +558,7 @@ void RecentTabsSubMenuModel::BuildTabsFromOtherDevices() {
   std::vector<raw_ptr<const sync_sessions::SyncedSession, VectorExperimental>>
       sessions;
   if (!open_tabs || !open_tabs->GetAllForeignSessions(&sessions)) {
-    if (open_tabs) {
+    if ((true)) {
       AddItemWithStringId(IDC_RECENT_TABS_NO_DEVICE_TABS,
                           IDS_RECENT_TABS_NO_DEVICE_TABS);
     } else if (syncer::IsReplaceSyncPromosWithSignInPromosEnabled()) {
@@ -601,6 +601,8 @@ void RecentTabsSubMenuModel::BuildTabsFromOtherDevices() {
     DCHECK(!session->GetSessionName().empty());
     std::unique_ptr<ui::SimpleMenuModel> device_menu_model =
         CreateOtherDeviceSubMenu(session, tabs_in_session);
+    MaybeAppendOverflowStubTab(tabs_in_session, kMaxSessionsToShow,
+                               device_menu_model.get());
     const int command_id = GetAndIncrementNextMenuID();
     AddSubMenu(command_id, base::UTF8ToUTF16(session->GetSessionName()),
                device_menu_model.get());
