diff --git a/chrome/browser/themes/theme_helper.h b/chrome/browser/themes/theme_helper.h
index faaf3c0c98c2edeeff3b06c5781a363b93ae1774..d0063c019a9e91c0c972ea45d37b664c2b2fd121 100644
--- a/chrome/browser/themes/theme_helper.h
+++ b/chrome/browser/themes/theme_helper.h
@@ -108,6 +108,7 @@ class ThemeHelper {
       bool incognito,
       const CustomThemeSupplier* theme_supplier) const;
 
+  BRAVE_THEME_HELPER_H_
  private:
   friend class theme_service_internal::ThemeServiceTest;
 
