diff --git a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog_v2.ts b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog_v2.ts
index 60e4e4c2e89a30581df5a189064d2010986656e3..868e2716e96e15ceca03aa269a3d95a48593fed5 100644
--- a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog_v2.ts
+++ b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog_v2.ts
@@ -81,10 +81,14 @@ export interface SettingsClearBrowsingDataDialogV2Element {
  * The list of all available Browsing Data types in the default order they
  * should appear in the dialog.
  */
+export
 const ALL_BROWSING_DATATYPES_LIST: BrowsingDataType[] = [
   BrowsingDataType.HISTORY,
   BrowsingDataType.SITE_DATA,
   BrowsingDataType.CACHE,
+// <if expr="enable_ai_chat">
+  BrowsingDataType.BRAVE_AI_CHAT,
+// </if>
   BrowsingDataType.DOWNLOADS,
   BrowsingDataType.FORM_DATA,
   BrowsingDataType.SITE_SETTINGS,
@@ -98,12 +102,14 @@ const DEFAULT_BROWSING_DATATYPES_LIST: BrowsingDataType[] = [
   BrowsingDataType.CACHE,
 ];
 
+export
 interface BrowsingDataTypeOption {
   label: string;
   subLabel?: string;
   pref: chrome.settingsPrivate.PrefObject;
 }
 
+export
 function getDataTypeLabel(datatypes: BrowsingDataType) {
   switch (datatypes) {
     case BrowsingDataType.HISTORY:
@@ -120,6 +126,9 @@ function getDataTypeLabel(datatypes: BrowsingDataType) {
       return loadTimeData.getString('clearDownloadHistory');
     case BrowsingDataType.HOSTED_APPS_DATA:
       return loadTimeData.getString('clearHostedAppData');
+// <if expr="enable_ai_chat">
+    case BrowsingDataType.BRAVE_AI_CHAT: return loadTimeData.getString('aiChatClearHistoryData');
+// </if>
     default:
       assertNotReachedCase(datatypes);
   }
@@ -141,6 +150,9 @@ export function getDataTypePrefName(datatypes: BrowsingDataType) {
       return 'browser.clear_data.download_history';
     case BrowsingDataType.HOSTED_APPS_DATA:
       return 'browser.clear_data.hosted_apps_data';
+// <if expr="enable_ai_chat">
+    case BrowsingDataType.BRAVE_AI_CHAT: return 'browser.clear_data.brave_leo';
+// </if>
     default:
       assertNotReachedCase(datatypes);
   }
