diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 49df54a9e05165ae32c15cde88f6a8bc988cf28d..1d3fc88dbfbcfa0e9adc903915c127489e19a5ab 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -144,6 +144,7 @@ class AvatarToolbarButton : public ToolbarButton {
   CreateScopedZeroDelayOverrideSigninPendingTextForTesting();
 
  private:
+  BRAVE_AVATAR_BUTTON_H
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
