diff --git a/chrome/browser/ui/views/tabs/tab_group_views.cc b/chrome/browser/ui/views/tabs/tab_group_views.cc
index 48a87284ea5a289ca9d95d65badc2e4f84065dfc..f659a87a4c29bf229e9b12136a193154644a2060 100644
--- a/chrome/browser/ui/views/tabs/tab_group_views.cc
+++ b/chrome/browser/ui/views/tabs/tab_group_views.cc
@@ -168,6 +168,7 @@ TabGroupViews::GetLeadingTrailingGroupViews(
   views::View* trailing_child = nullptr;
   gfx::Rect trailing_child_widget_bounds;
 
+  BRAVE_TAB_GROUP_VIEWS_GET_LEADING_TRAILING_GROUP_VIEWS
   for (views::View* child : children) {
     TabSlotView* tab_slot_view = views::AsViewClass<TabSlotView>(child);
     if (!tab_slot_view || tab_slot_view->group() != group_ ||
