diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index 0baa5aac365c46bd0c77d7a0ecc39b405cf1ec29..35cc3fd902dbb82fbeb0dfe0302932b756994d0e 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -242,6 +242,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
