diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index 8fd6e51b8f078acc2de2d29c2277bab2e617ea17..46212415aaa9f07b5bf47bfd542c21d73addccc6 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -223,6 +223,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
