diff --git a/chrome/browser/resources/tools/optimize_webui.py b/chrome/browser/resources/tools/optimize_webui.py
index d7bade19933acfbcd31bbce4231d9e20743f4c91..180e7f20ae355f6e152e697361a95d90cc98cb9c 100755
--- a/chrome/browser/resources/tools/optimize_webui.py
+++ b/chrome/browser/resources/tools/optimize_webui.py
@@ -56,8 +56,12 @@ _POLYMER_PATH = os.path.join(
     '').replace('\\', '/')
 
 
+_BR_RESOURCES_PATH = os.path.join(_SRC_PATH, 'brave', 'ui', 'webui', 'resources','').replace('\\', '/')
+
 # These files are already combined and minified.
 _BASE_EXCLUDES = [
+  'chrome://brave-resources/fonts/muli.css',
+  'chrome://brave-resources/fonts/poppins.css'
   # Excludes applying only to Polymer 2.
   'chrome://resources/html/polymer.html',
   'chrome://resources/polymer/v1_0/polymer/polymer.html',
@@ -102,6 +106,7 @@ for (redirect_url, file_path) in [
     ('resources/js/', _JS_RESOURCES_PATH),
     ('resources/polymer/v1_0/', _POLYMER_PATH),
     ('resources/images/', _IMAGES_RESOURCES_PATH),
+    ('brave-resources/', _BR_RESOURCES_PATH)
 ]:
   # Redirect both the chrome://resources form and the scheme-relative form.
   _URL_MAPPINGS.append(('chrome://' + redirect_url, file_path))
