diff --git a/chrome/browser/offline_pages/prefetch/prefetch_service_factory.cc b/chrome/browser/offline_pages/prefetch/prefetch_service_factory.cc
index d2c8ad6ca350d7ac7eaceb8a0b71d8fa551740ec..f25aebb0450e625cd5c8b482400e5c45b5a69556 100644
--- a/chrome/browser/offline_pages/prefetch/prefetch_service_factory.cc
+++ b/chrome/browser/offline_pages/prefetch/prefetch_service_factory.cc
@@ -51,6 +51,7 @@ namespace {
 image_fetcher::ImageFetcher* GetImageFetcher(
     ProfileKey* key,
     image_fetcher::ImageFetcherConfig config) {
+  BRAVE_GET_IMAGE_FETCHER
   image_fetcher::ImageFetcherService* image_fetcher_service =
       ImageFetcherServiceFactory::GetForKey(key);
   DCHECK(image_fetcher_service);
