diff --git a/chrome/browser/shell_integration.h b/chrome/browser/shell_integration.h
index 76815052bc5c07e6bc37b29e4c749f69483377fc..4642f3885d035d10bc7b7ff041ceefe32c6c199f 100644
--- a/chrome/browser/shell_integration.h
+++ b/chrome/browser/shell_integration.h
@@ -231,6 +231,7 @@ class DefaultBrowserWorker : public DefaultWebClientWorker {
   DefaultBrowserWorker(const DefaultBrowserWorker&) = delete;
   DefaultBrowserWorker& operator=(const DefaultBrowserWorker&) = delete;
 
+  BRAVE_DEFAULT_BROWSER_WORKER_H
  protected:
   ~DefaultBrowserWorker() override;
 
