diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index dba7a18d82b656a904d415b06e0925d0eeb507cf..43aa8c250e63198a97b8401124017bc091eef81f 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -4598,6 +4598,7 @@ source_set("browser") {
   if (!is_android) {
     deps += [ "//chrome/browser/contextual_cueing/internals" ]
   }
+  public_deps += [ "//brave/browser:browser_impl" ]
 
   # This target intentionally has no sources of its own and no
   # allow_circular_includes_from; it only aggregates dependencies. Do NOT add
@@ -4769,6 +4770,8 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni")
+  public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
