diff --git a/chrome/browser/sync/sync_service_factory.cc b/chrome/browser/sync/sync_service_factory.cc
index 99690c171e12f76efb4f7a702b841579c78269b1..57e69316fc5f5955101afc4b86820190d37aeac8 100644
--- a/chrome/browser/sync/sync_service_factory.cc
+++ b/chrome/browser/sync/sync_service_factory.cc
@@ -171,7 +171,7 @@ std::unique_ptr<KeyedService> BuildSyncService(
           init_params.sync_client.get());
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize(
       client_ptr->CreateDataTypeControllers(sync_service.get()));
 
