diff --git a/chrome/browser/hub/internal/BUILD.gn b/chrome/browser/hub/internal/BUILD.gn
index b93a8de5307b8189042509414e56654afa6f612a..bd6d4e4e858742430088967941d975fdf49e887b 100644
--- a/chrome/browser/hub/internal/BUILD.gn
+++ b/chrome/browser/hub/internal/BUILD.gn
@@ -107,6 +107,7 @@ android_library("java") {
     "//ui/edge_to_edge:java",
     "//url:url_java",
   ]
+  sources += brave_browser_hub_internal_java_sources deps += brave_browser_hub_internal_java_deps
 }
 
 android_resources("java_resources") {
