diff --git a/chrome/browser/ui/cocoa/main_menu_builder.mm b/chrome/browser/ui/cocoa/main_menu_builder.mm
index 04198bee3ea102ecfb512aceb91d830b2bfc6a2b..ea2d7eff04d6529348941d35280617c4c04b29be 100644
--- a/chrome/browser/ui/cocoa/main_menu_builder.mm
+++ b/chrome/browser/ui/cocoa/main_menu_builder.mm
@@ -100,6 +100,7 @@ base::scoped_nsobject<NSMenuItem> BuildFileMenu(
                 Item(IDS_NEW_INCOGNITO_WINDOW_MAC)
                     .command_id(IDC_NEW_INCOGNITO_WINDOW)
                     .remove_if(is_pwa),
+                BRAVE_BUILD_FILE_MENU
                 Item(IDS_REOPEN_CLOSED_TABS_MAC)
                     .command_id(IDC_RESTORE_TAB)
                     .remove_if(is_pwa),
@@ -457,6 +458,7 @@ base::scoped_nsobject<NSMenuItem> BuildHelpMenu(
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
             Item(IDS_FEEDBACK_MAC).command_id(IDC_FEEDBACK),
 #endif
+                BRAVE_BUILD_HELP_MENU
                 Item(IDS_HELP_MAC)
                     .string_format_1(product_name)
                     .command_id(IDC_HELP_PAGE_VIA_MENU),
