diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 01d5593c76ee0982714456a49929caeecc0a9771..ab47048b6c28c51e6d79148dc65712e9042fedfd 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -31,7 +31,7 @@ import("java_sources.gni")
 import("static_initializers.gni")
 import("trichrome.gni")
 
-chrome_public_manifest_package = "org.chromium.chrome"
+declare_args() { chrome_public_manifest_package = "org.chromium.chrome" }
 chrome_public_test_manifest_package = "org.chromium.chrome.tests"
 
 chrome_public_jinja_variables =
