diff --git a/chrome/browser/permissions/permission_request_impl.h b/chrome/browser/permissions/permission_request_impl.h
index 4639136abb332864391bd69989c82f77c9d54477..53a831dd86304eeb23bcce922134e88fef2dae85 100644
--- a/chrome/browser/permissions/permission_request_impl.h
+++ b/chrome/browser/permissions/permission_request_impl.h
@@ -32,6 +32,7 @@ class PermissionRequestImpl : public PermissionRequest {
  private:
   // PermissionRequest:
   IconId GetIconId() const override;
+  IconId GetIconId_ChromiumImpl() const;
 #if defined(OS_ANDROID)
   base::string16 GetTitleText() const override;
   base::string16 GetMessageText() const override;
@@ -39,6 +40,7 @@ class PermissionRequestImpl : public PermissionRequest {
   base::string16 GetQuietMessageText() const override;
 #endif
   base::string16 GetMessageTextFragment() const override;
+  base::string16 GetMessageTextFragment_ChromiumImpl() const;
   GURL GetOrigin() const override;
   void PermissionGranted() override;
   void PermissionDenied() override;
