diff --git a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
index dd3afef2ef41bc9f1444290cf12df8804f7ce73d..e21f6e5112b7ed4f856441f65ae460df63f93c69 100644
--- a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
+++ b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
@@ -52,6 +52,7 @@ content::WebUIDataSource* CreateBookmarksUIHTMLSource(Profile* profile) {
   content::WebUIDataSource* source =
       content::WebUIDataSource::Create(chrome::kChromeUIBookmarksHost);
 
+  BraveCustomizeBookmarksDataSource(source);
 #if BUILDFLAG(OPTIMIZE_WEBUI)
   webui::SetupBundledWebUIDataSource(source, "bookmarks.js",
                                      IDR_BOOKMARKS_BOOKMARKS_ROLLUP_JS,
