diff --git a/chrome/browser/extensions/extension_management.cc b/chrome/browser/extensions/extension_management.cc
index 5496b0336f8c5403b5d46fceff577bbf9061672c..f0df5c6649ea42e56df1d8b0cbde6dac42ffb41e 100644
--- a/chrome/browser/extensions/extension_management.cc
+++ b/chrome/browser/extensions/extension_management.cc
@@ -1086,6 +1086,7 @@ ExtensionManagementFactory::BuildServiceInstanceForBrowserContext(
     content::BrowserContext* context) const {
   TRACE_EVENT0("browser,startup",
                "ExtensionManagementFactory::BuildServiceInstanceFor");
+  BRAVE_EXTENSION_MANAGEMENT_FACTORY_BUILD_SERVICE_INSTANCE_FOR
   return std::make_unique<ExtensionManagement>(
       Profile::FromBrowserContext(context));
 }
