diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 4893f1d64919b29c53cc8c63e94c2583967b874e..82fddaf6725980bccbf896a827866f7e28ad19a0 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -29,6 +29,7 @@ static_library("utility") {
 
   public_deps = []
   deps = [
+    "//brave/utility",
     "//base",
     "//build:chromeos_buildflags",
     "//chrome:resources",
