diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index bcf675038912851dfd0f702a369bc33ea949df82..73c97deeef8ae1c597fbda6d5bf540778a6026cb 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2503,6 +2503,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps defines += brave_chrome_browser_defines
 
   if (is_android) {
     sources += [
