diff --git a/chrome/test/base/testing_profile.cc b/chrome/test/base/testing_profile.cc
index d160f84e5e4a0a89e3aba47b9b75bff9fc418115..907d7c6f6805d7a699ef1d3d60730ab37b84751e 100644
--- a/chrome/test/base/testing_profile.cc
+++ b/chrome/test/base/testing_profile.cc
@@ -350,6 +350,7 @@ void TestingProfile::Init() {
       EnsureBrowserContextKeyedServiceFactoriesBuilt();
 
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
+  BRAVE_INIT
   if (!IsOffTheRecord()) {
     SupervisedUserSettingsService* settings_service =
         SupervisedUserSettingsServiceFactory::GetForKey(key_.get());
