diff --git a/chrome/browser/ui/views/profiles/profile_menu_view.h b/chrome/browser/ui/views/profiles/profile_menu_view.h
index ef9396b94243b68fb77dd93d0aa8d1f53ee6af8a..af6e2e62c32df16a34973e5dde2bdbd8fc91c857 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view.h
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.h
@@ -73,6 +73,7 @@ class ProfileMenuView : public ProfileMenuViewBase {
   }
 
  private:
+  BRAVE_PROFILE_MENU_VIEW_H
   friend class ProfileMenuViewExtensionsTest;
   friend class ProfileMenuViewSigninPendingTest;
   friend class ProfileMenuViewSignoutTest;
