diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index ff490f2ffb3f5af65d8af96ff708f9292ffb5e33..949a451d9369766c0c56fb4985f2ee2f632b52af 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2587,6 +2587,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
