diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 028d269850e66db444591cda7005a0f8434193e9..5ebaaa29519642c488db4b84d9ae24a1392a6664 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -303,6 +303,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data_deps = []
 
   if (enable_pdf) {
