diff --git a/chrome/browser/ui/views/omnibox/omnibox_popup_view_views.cc b/chrome/browser/ui/views/omnibox/omnibox_popup_view_views.cc
index bbf17cc2c5c7f9aac2544fe368ca2d022fa46ace..df12e57839a4f604f37c83b8ea70c11ee0af637a 100644
--- a/chrome/browser/ui/views/omnibox/omnibox_popup_view_views.cc
+++ b/chrome/browser/ui/views/omnibox/omnibox_popup_view_views.cc
@@ -359,7 +359,7 @@ void OmniboxPopupViewViews::UpdatePopupAppearance() {
       if (children().size() == i) {
         AddChildView(std::make_unique<OmniboxRowView>(
             i, edit_model_,
-            std::make_unique<OmniboxResultView>(this, edit_model_, i),
+            std::make_unique<BraveOmniboxResultView>(this, edit_model_, i),
             pref_service));
       }
 
