diff --git a/chrome/browser/ui/tab_helpers.cc b/chrome/browser/ui/tab_helpers.cc
index ed24f9a0b67a342284fa9f09035f51c3bbab161e..73f63faf5b32ef76dd2d3c1e96a0f8be0da5e1a5 100644
--- a/chrome/browser/ui/tab_helpers.cc
+++ b/chrome/browser/ui/tab_helpers.cc
@@ -499,4 +499,5 @@ void TabHelpers::AttachTabHelpers(WebContents* web_contents) {
 
   // This is common code for all of us. PLEASE DO YOUR PART to keep it tidy and
   // organized.
+  BRAVE_TAB_HELPERS
 }
