diff --git a/chrome/browser/plugins/chrome_plugin_service_filter.cc b/chrome/browser/plugins/chrome_plugin_service_filter.cc
index 3c2de25de6dfbb132e2d3ddb2e7721d0aef09c16..f75c26ab366d874f49e7c2be42ae9f367de9434d 100644
--- a/chrome/browser/plugins/chrome_plugin_service_filter.cc
+++ b/chrome/browser/plugins/chrome_plugin_service_filter.cc
@@ -52,7 +52,7 @@ class ProfileContentSettingObserver : public content_settings::Observer {
       PluginService::GetInstance()->PurgePluginListCache(profile_, false);
 
     const GURL primary(primary_pattern.ToString());
-    if (primary.is_valid()) {
+    if (primary.is_valid() && resource_identifier.length() == 0) {
       DCHECK_EQ(ContentSettingsPattern::Relation::IDENTITY,
                 ContentSettingsPattern::Wildcard().Compare(secondary_pattern));
       PluginUtils::RememberFlashChangedForSite(map, primary);
