diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index bde7030b6eeb6497c55ccd8b64118ab9176248d4..69a8008f65e9718d3b8c2e772011e8d827a9ea33 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -3343,6 +3343,7 @@ source_set("browser") {
       "//chrome/browser/safe_browsing/suspicious_site_warnings",
     ]
   }
+  public_deps += [ "//brave/browser:core" ]
 
   # This target intentionally has no sources of its own and no
   # allow_circular_includes_from; it only aggregates dependencies. Do NOT add
@@ -3512,6 +3513,7 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
