diff --git a/chrome/android/features/tab_ui/BUILD.gn b/chrome/android/features/tab_ui/BUILD.gn
index 11643c8b3247e167a82b5cb8cd6ae9ddb60a91e7..4744766984f074a3c245c6064a036f8458d5135d 100644
--- a/chrome/android/features/tab_ui/BUILD.gn
+++ b/chrome/android/features/tab_ui/BUILD.gn
@@ -189,6 +189,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/tasks/tab_management/suggestions/TabSuggestionsRanker.java",
     "java/src/org/chromium/chrome/browser/tasks/tab_management/suggestions/TabSuggestionsServerFetcher.java",
   ]
+  sources += brave_android_features_tab_ui_java_sources
 
   deps = [
     ":java_resources",
