diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 69079f66c2b8de75b13fbd714dddc27a3928c614..b6f436098595e973b30e36c34f5509a2d42a772b 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2479,6 +2479,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps defines += brave_chrome_browser_defines
 
   if (is_android) {
     sources += [
