diff --git a/chrome/browser/external_protocol/external_protocol_handler.cc b/chrome/browser/external_protocol/external_protocol_handler.cc
index 92de40320204953760f6a566219fba33be00eb43..2313cd2f3f836dee681e8200dcabdead2d3cf5d6 100644
--- a/chrome/browser/external_protocol/external_protocol_handler.cc
+++ b/chrome/browser/external_protocol/external_protocol_handler.cc
@@ -367,7 +367,7 @@ ExternalProtocolHandler::BlockState ExternalProtocolHandler::GetBlockState(
   if (scheme == "mailto") {
     base::UmaHistogramEnumeration(kBlockStateMetric,
                                   BlockStateMetric::kAllowedDefaultMail);
-    return DONT_BLOCK;
+    return UNKNOWN;
   }
 
   PrefService* profile_prefs = profile->GetPrefs();
