diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index fc952779121dd645e061a44366ab6c1f04681f60..b73ffc395bf77df23fb9b4273864d36713d1b142 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -732,6 +732,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
