diff --git a/chrome/browser/ui/startup/BUILD.gn b/chrome/browser/ui/startup/BUILD.gn
index 9f6a1995cf47bdf8bd47f8f24c2ca00d325c8105..b3f4fe11f17360bd49c00f9925d370a1ef4d77e1 100644
--- a/chrome/browser/ui/startup/BUILD.gn
+++ b/chrome/browser/ui/startup/BUILD.gn
@@ -28,6 +28,7 @@ source_set("startup_tab") {
   ]
 
   public_deps = [ "//url" ]
+  import("//brave/browser/ui/startup/sources.gni") public_deps += brave_browser_ui_startup_public_deps
 }
 
 source_set("startup") {
