diff --git a/chrome/renderer/chrome_content_renderer_client.cc b/chrome/renderer/chrome_content_renderer_client.cc
index 78925a9dc0f085ad01d92392dc885240020d4e3b..9bde7df7d47c505d30e039cf4c4a30e0cfc6492c 100644
--- a/chrome/renderer/chrome_content_renderer_client.cc
+++ b/chrome/renderer/chrome_content_renderer_client.cc
@@ -19,6 +19,7 @@
 #include "base/strings/utf_string_conversions.h"
 #include "base/time/time.h"
 #include "base/values.h"
+#include "brave/renderer/brave_content_settings_observer.h"
 #include "build/build_config.h"
 #include "chrome/common/buildflags.h"
 #include "chrome/common/channel_info.h"
@@ -512,7 +513,7 @@ void ChromeContentRendererClient::RenderFrameCreated(
   ext_dispatcher =
       ChromeExtensionsRendererClient::GetInstance()->extension_dispatcher();
 #endif
-  ContentSettingsObserver* content_settings = new ContentSettingsObserver(
+  ContentSettingsObserver* content_settings = new BraveContentSettingsObserver(
       render_frame, ext_dispatcher, should_whitelist_for_content_settings,
       registry);
   if (chrome_observer_.get()) {
