diff --git a/chrome/browser/profiles/off_the_record_profile_impl.h b/chrome/browser/profiles/off_the_record_profile_impl.h
index 4c34e6c7d772dcbfe17da39573851b9430fec406..07bc7b89364301e5ba913e77623bb6146ac0fa7b 100644
--- a/chrome/browser/profiles/off_the_record_profile_impl.h
+++ b/chrome/browser/profiles/off_the_record_profile_impl.h
@@ -17,10 +17,10 @@
 #include "components/domain_reliability/clear_mode.h"
 #include "content/public/browser/content_browser_client.h"
 
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
 #include "chrome/browser/ui/zoom/chrome_zoom_level_prefs.h"
 #include "content/public/browser/host_zoom_map.h"
-#endif
+//#endif
 
 namespace sync_preferences {
 class PrefServiceSyncable;
@@ -93,10 +93,10 @@ class OffTheRecordProfileImpl : public Profile {
   base::FilePath GetPath() override;
   base::FilePath GetPath() const override;
   base::Time GetCreationTime() const override;
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
   std::unique_ptr<content::ZoomLevelDelegate> CreateZoomLevelDelegate(
       const base::FilePath& partition_path) override;
-#endif  // !defined(OS_ANDROID)
+//#endif  // !defined(OS_ANDROID)
   scoped_refptr<base::SequencedTaskRunner> GetIOTaskRunner() override;
   bool IsOffTheRecord() override;
   bool IsOffTheRecord() const override;
@@ -127,28 +127,28 @@ class OffTheRecordProfileImpl : public Profile {
   GetNativeFileSystemPermissionContext() override;
 
  private:
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
   // Allows a profile to track changes in zoom levels in its parent profile.
   void TrackZoomLevelsFromParent();
-#endif  // !defined(OS_ANDROID)
+//#endif  // !defined(OS_ANDROID)
 
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
   // Callback function for tracking parent's zoom level changes.
   void OnParentZoomLevelChanged(
       const content::HostZoomMap::ZoomLevelChange& change);
   void UpdateDefaultZoomLevel();
-#endif  // !defined(OS_ANDROID)
+//#endif  // !defined(OS_ANDROID)
 
   // The real underlying profile.
   Profile* profile_;
 
   std::unique_ptr<sync_preferences::PrefServiceSyncable> prefs_;
 
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
   std::unique_ptr<content::HostZoomMap::Subscription> track_zoom_subscription_;
   std::unique_ptr<ChromeZoomLevelPrefs::DefaultZoomLevelSubscription>
       parent_default_zoom_level_subscription_;
-#endif  // !defined(OS_ANDROID)
+//#endif  // !defined(OS_ANDROID)
   ProfileIODataHandle io_data_;
 
   // Time we were started.
