diff --git a/chrome/browser/extensions/api/developer_private/extension_info_generator.cc b/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
index b3c2a407a5216feefb136efae49b45c047a612ae..c4c4a51a8cb5c1ffa385c4435b218c2132a93f58 100644
--- a/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
+++ b/chrome/browser/extensions/api/developer_private/extension_info_generator.cc
@@ -718,6 +718,7 @@ void ExtensionInfoGenerator::FillExtensionInfo(const Extension& extension,
 #else
   info.incognito_access_pending_change = false;
 #endif
+  BRAVE_CREATE_EXTENSION_INFO_HELPER
 
   // User Scripts toggle.
   info.user_scripts_access.is_enabled =
