diff --git a/chrome/browser/incognito/BUILD.gn b/chrome/browser/incognito/BUILD.gn
index eccefc91c69b95f6401baec6c092ed1289fd2533..9a1b28337d36275a8cc0fdb1781b13b906a0b302 100644
--- a/chrome/browser/incognito/BUILD.gn
+++ b/chrome/browser/incognito/BUILD.gn
@@ -62,6 +62,7 @@ android_library("java") {
     "//url:gurl_java",
   ]
 
+  sources += brave_private_tab_reauth_java_sources deps += brave_private_tab_reauth_java_deps
   resources_package = "org.chromium.chrome.browser.incognito"
 }
 
