diff --git a/chrome/browser/ui/chrome_pages.cc b/chrome/browser/ui/chrome_pages.cc
index f890f886c50eaaea4ebbdb0608f611dbad9ba7e6..63bfa5f1c9a1b4d29e43d4e274de516a2b50ee3c 100644
--- a/chrome/browser/ui/chrome_pages.cc
+++ b/chrome/browser/ui/chrome_pages.cc
@@ -436,7 +436,7 @@ void ShowSlow(Browser* browser) {
 #endif
 }
 
-GURL GetSettingsUrl(std::string_view sub_page) {
+BRAVE_GET_SETTINGS_URL
   return GURL(base::StrCat({kChromeUISettingsURL, sub_page}));
 }
 
