diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 0b0b0cb0ea84f06cbd02f8a430e9200cc5113718..c67c109ae028e524a4397d052df220f6c84c20a6 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -615,6 +615,7 @@ class Browser : public TabStripModelObserver,
   // dragging.
   void SetIsInTabDragging(bool is_in_tab_dragging);
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class FullscreenControllerInteractiveTest;
