diff --git a/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc b/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
index f827abf5fcb02c6dfa9625bd8bd67aae60b891c1..cabd9ae93b50f979fe0b08161f407362ad76f75f 100644
--- a/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
+++ b/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
@@ -88,6 +88,7 @@ PermissionPromptBubbleBaseView::~PermissionPromptBubbleBaseView() = default;
 void PermissionPromptBubbleBaseView::CreatePermissionButtons(
     const std::u16string& allow_always_text,
     const std::u16string& block_text) {
+  [&]() -> void {
   if (delegate_) {
     if (std::optional<std::u16string> explanation =
             GetSubAppsPermissionExplanation(
@@ -179,6 +180,20 @@ void PermissionPromptBubbleBaseView::CreatePermissionButtons(
     SetButtonStyle(ui::mojom::DialogButton::kOk, ui::ButtonStyle::kTonal);
     SetButtonStyle(ui::mojom::DialogButton::kCancel, ui::ButtonStyle::kTonal);
   }
+  }();
+  BrowserWindowInterface* const brave_browser = GetBrowser();
+  AddAdditionalWidevineViewControlsIfNeeded(this, delegate_->Requests());
+  auto* permission_lifetime_view =
+      AddPermissionLifetimeComboboxIfNeeded(this, delegate_.get());
+  AddFootnoteViewIfNeeded(this, delegate_->Requests(), brave_browser);
+  if (permission_lifetime_view) {
+    set_fixed_width(
+        std::max(GetPreferredSize().width(),
+                 permission_lifetime_view->GetPreferredSize().width()) +
+        margins().width());
+    set_should_ignore_snapping(true);
+  }
+  AddGeolocationDescriptionIfNeeded(this, delegate_.get(), brave_browser);
 }
 
 void PermissionPromptBubbleBaseView::CreateExtraTextLabel(
@@ -201,6 +216,7 @@ void PermissionPromptBubbleBaseView::Show() {
 }
 
 void PermissionPromptBubbleBaseView::CreateWidget() {
+  [&]() -> void {
   CHECK(GetNativeWindow());
 
   UpdateAnchorPosition();
@@ -215,6 +231,8 @@ void PermissionPromptBubbleBaseView::CreateWidget() {
   }
 
   widget->SetZOrderSublevel(ChromeWidgetSublevel::kSublevelSecurity);
+  }();
+  z_order_manager_ = std::make_unique<PermissionPromptBubbleZOrderManager>(*this);
 }
 
 void PermissionPromptBubbleBaseView::ShowWidget() {
