diff --git a/chrome/chrome_repack_locales.gni b/chrome/chrome_repack_locales.gni
index e9dfe6bb49914fcfa7202cfba9d2408e686ba7f1..b9ae4b54e462f83185c26e8cf0d92e20395d7e16 100644
--- a/chrome/chrome_repack_locales.gni
+++ b/chrome/chrome_repack_locales.gni
@@ -37,6 +37,7 @@ template("chrome_repack_locales") {
       "${root_gen_dir}/ui/strings/app_locale_settings_",
       "${root_gen_dir}/ui/strings/ui_strings_",
     ]
+    source_patterns += brave_locale_source_patterns
     if (!defined(deps)) {
       deps = []
     }
@@ -54,6 +55,7 @@ template("chrome_repack_locales") {
       "//ui/strings:app_locale_settings",
       "//ui/strings:ui_strings",
     ]
+    deps += brave_locale_deps
     if (defined(invoker.deps)) {
       deps += invoker.deps
     }
