diff --git a/chrome/browser/ui/android/toolbar/BUILD.gn b/chrome/browser/ui/android/toolbar/BUILD.gn
index 8c4a4061ea049c7806d18c98498f1d2b25f6d387..7198d6173a89dafd8ff03be70f17f44089a51b0b 100644
--- a/chrome/browser/ui/android/toolbar/BUILD.gn
+++ b/chrome/browser/ui/android/toolbar/BUILD.gn
@@ -135,6 +135,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/StripVisibilityState.java",
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/TabStripTransitionCoordinator.java",
   ]
+  sources += brave_browser_ui_android_toolbar_sources
 
   deps = [
     ":java_resources",
@@ -372,6 +373,7 @@ android_resources("java_resources") {
     "//chrome/browser/ui/android/theme:java_resources",
     "//components/browser_ui/styles/android:java_resources",
   ]
+  import("//brave/browser/ui/android/toolbar/brave_java_resources.gni") sources += brave_browser_ui_android_toolbar_java_resources sources -= brave_browser_ui_android_toolbar_java_resources_to_exclude
 }
 
 robolectric_library("junit") {
