diff --git a/chrome/browser/profiles/profile_impl.h b/chrome/browser/profiles/profile_impl.h
index 5a6ea18c6fd98b2b89861259c844740e8cc1a6c5..4e14ad1d73f36a8c006ab007616a436cef077dd7 100644
--- a/chrome/browser/profiles/profile_impl.h
+++ b/chrome/browser/profiles/profile_impl.h
@@ -157,6 +157,7 @@ class ProfileImpl : public Profile {
   void OnLogin() override;
   void InitChromeOSPreferences() override;
 #endif  // BUILDFLAG(IS_CHROMEOS_ASH)
+  BRAVE_PROFILE_IMPL_H_
 
   bool IsNewProfile() const override;
 
