diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index 645cf9d7c67f659c78b4d90c813114bbebd70760..67c01c113b8b970dfad1ccfec7569a65f5e11b69 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -308,6 +308,8 @@ content::WebUIDataSource* CreateMdExtensionsSource(Profile* profile,
   source->SetDefaultResource(IDR_EXTENSIONS_EXTENSIONS_HTML);
 #endif
 
+  BraveAddExtensionsResources(source);
+
   return source;
 }
 
