diff --git a/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java
index a6d8b55b10f10afdb8a4d05c452dc55451718d8b..2f7af3f6ccc440a439826bd0d378c8bce4293ea9 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/multiwindow/MultiInstanceOrchestratorImpl.java
@@ -26,7 +26,7 @@ import java.util.Map;
 
 /** Implements {@link MultiInstanceOrchestrator} as a singleton. */
 @NullMarked
-/* package */ final class MultiInstanceOrchestratorImpl implements MultiInstanceOrchestrator {
+/* package */ class MultiInstanceOrchestratorImpl implements MultiInstanceOrchestrator {
     private static @Nullable MultiInstanceOrchestrator sInstance;
     private static @Nullable TabReparentingDelegate sTabReparentingDelegateForTesting;
 
@@ -47,7 +47,7 @@ import java.util.Map;
         return sInstance;
     }
 
-    private MultiInstanceOrchestratorImpl(TabReparentingDelegate tabReparentingDelegate) {
+    MultiInstanceOrchestratorImpl(TabReparentingDelegate tabReparentingDelegate) {
         mTabReparentingDelegate = tabReparentingDelegate;
         ApplicationStatus.registerStateListenerForAllActivities(this::onActivityStateChange);
     }
