diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/developer/TracingCategoriesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/developer/TracingCategoriesSettings.java
index 9e04f8f1af83d156b19148b75307096a5d0bd63d..64056fcce4b440b24ccb464458e7beccde6d65c5 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/developer/TracingCategoriesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/developer/TracingCategoriesSettings.java
@@ -26,7 +26,7 @@ import java.util.Set;
  * passed to the fragment via an extra (EXTRA_CATEGORY_TYPE).
  */
 public class TracingCategoriesSettings
-        extends PreferenceFragmentCompat implements Preference.OnPreferenceChangeListener {
+        extends org.chromium.chrome.browser.settings.BravePreferenceFragment implements Preference.OnPreferenceChangeListener {
     public static final String EXTRA_CATEGORY_TYPE = "type";
 
     // Non-translated strings:
