diff --git a/chrome/browser/ui/location_bar/location_bar.h b/chrome/browser/ui/location_bar/location_bar.h
index cee5096ba56e509b3480e112d98b32bbf1324462..2d06db885dc79a85d588e6cb6b709b09bdb03827 100644
--- a/chrome/browser/ui/location_bar/location_bar.h
+++ b/chrome/browser/ui/location_bar/location_bar.h
@@ -81,7 +81,7 @@ class LocationBar {
   virtual ~LocationBar();
 
   // Checks if any extension has requested that the bookmark star be hidden.
-  bool IsBookmarkStarHiddenByExtension() const;
+  virtual bool IsBookmarkStarHiddenByExtension() const;
 
  private:
   class ExtensionLoadObserver;
