diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index a2212a9e2b6364b8f36b7b74c056d936db62b353..1967b0ce06b0e8ee666705e4364dabf2f640aaf8 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -7668,6 +7668,7 @@ const FeatureEntry kFeatureEntries[] = {
     // "LoginCustomFlags" in tools/metrics/histograms/enums.xml. See "Flag
     // Histograms" in tools/metrics/histograms/README.md (run the
     // AboutFlagsHistogramTest unit test to verify this process).
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 };
 
 class FlagsStateSingleton : public flags_ui::FlagsState::Delegate {
