diff --git a/chrome/browser/lifetime/BUILD.gn b/chrome/browser/lifetime/BUILD.gn
index 0d9434fb479df37a81302660e922b81e002d5710..0f41e5881cdc8d17c0f0c55b76b6c67a6f7fe505 100644
--- a/chrome/browser/lifetime/BUILD.gn
+++ b/chrome/browser/lifetime/BUILD.gn
@@ -151,6 +151,7 @@ source_set("impl") {
   }
 
   if (!is_android) {
+    import("//brave/chromium_src/chrome/browser/lifetime/sources.gni") deps += brave_chrome_browser_lifetime_impl_deps
     sources += [
       "application_lifetime_desktop.cc",
       "browser_close_manager.cc",
