diff --git a/chrome/browser/ui/browser_command_controller.cc b/chrome/browser/ui/browser_command_controller.cc
index a6d7b2cf05b0d2cb127c7a4b4f8d21edfeb41b1c..1c70cdd35d15616bba1be2af56c4c86e1f56694d 100644
--- a/chrome/browser/ui/browser_command_controller.cc
+++ b/chrome/browser/ui/browser_command_controller.cc
@@ -948,6 +948,7 @@ void BrowserCommandController::InitCommandState() {
   UpdateCommandsForContentRestrictionState();
   UpdateCommandsForBookmarkEditing();
   UpdateCommandsForIncognitoAvailability();
+  UpdateCommandsForTor();
 }
 
 // static
@@ -997,6 +998,7 @@ void BrowserCommandController::UpdateCommandsForIncognitoAvailability() {
     command_updater_.UpdateCommandEnabled(IDC_IMPORT_SETTINGS, false);
     command_updater_.UpdateCommandEnabled(IDC_OPTIONS, false);
   }
+  UpdateCommandsForTor();
 }
 
 void BrowserCommandController::UpdateCommandsForTabState() {
