diff --git a/chrome/browser/ui/views/side_panel/side_panel_ui.h b/chrome/browser/ui/views/side_panel/side_panel_ui.h
index 4ce4de8e56d0ef5d292c334761ac362bc226ca00..f49c7e1c582c9e2361a35ffa2a8e3d1cf01ddb28 100644
--- a/chrome/browser/ui/views/side_panel/side_panel_ui.h
+++ b/chrome/browser/ui/views/side_panel/side_panel_ui.h
@@ -8,6 +8,7 @@
 #include "chrome/browser/ui/views/side_panel/side_panel_entry_id.h"
 #include "chrome/browser/ui/views/side_panel/side_panel_entry_key.h"
 #include "chrome/browser/ui/views/side_panel/side_panel_enums.h"
+#include "ui/gfx/geometry/rect.h"
 
 namespace content {
 class WebContents;
