diff --git a/chrome/browser/resources/password_manager/password_manager.ts b/chrome/browser/resources/password_manager/password_manager.ts
index 75cc76b96c41057322729a603b95bcf8c7f64814..fb0aa89966e1bdb0f6bee4b0f9f9c5d429397f34 100644
--- a/chrome/browser/resources/password_manager/password_manager.ts
+++ b/chrome/browser/resources/password_manager/password_manager.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './password_manager_app.js';
 
 export {SettingsPrefsElement} from '/shared/settings/prefs/prefs.js';
