diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index aa28c2ec1efb284ec1d76f8dc43e13c138a2efec..79f1d323b9fc95c7fddca1cb8bab336009365e21 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -450,6 +450,7 @@ Browser* Browser::Create(const CreateParams& params) {
   // not possible, e.g. using the wrong profile or during shutdown. The caller
   // should handle this; see e.g. crbug.com/1141608 and crbug.com/1261628.
   CHECK_EQ(CreationStatus::kOk, GetCreationStatusForProfile(params.profile));
+  BRAVE_BROWSER_CREATE
   return new Browser(params);
 }
 
@@ -460,7 +461,7 @@ Browser::Browser(const CreateParams& params)
       window_(nullptr),
       tab_strip_model_delegate_(
           std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
-      tab_strip_model_(std::make_unique<TabStripModel>(
+      tab_strip_model_(std::make_unique<BraveTabStripModel>(
           tab_strip_model_delegate_.get(),
           params.profile,
           params.are_tab_groups_enabled ? TabGroupModelFactory::GetInstance()
