diff --git a/chrome/installer/setup/BUILD.gn b/chrome/installer/setup/BUILD.gn
index 9635a435d6fdc568240daae2ce69c62c54d44904..461133effe196c684b10efa7ffda2e8c42458655 100644
--- a/chrome/installer/setup/BUILD.gn
+++ b/chrome/installer/setup/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/chromium_src/chrome/installer/setup/sources.gni")
 import("//build/buildflag_header.gni")
 import("//build/config/chrome_build.gni")
 import("//chrome/process_version_rc_template.gni")
@@ -98,6 +99,7 @@ if (is_win) {
       "wof_compression.cc",
       "wof_compression.h",
     ]
+    sources += brave_chromium_src_chrome_installer_setup_sources
 
     public_deps = [
       "//base",
@@ -122,6 +124,7 @@ if (is_win) {
       "//third_party/crashpad/crashpad/util",
       "//ui/base:fullscreen_win",
     ]
+    deps += brave_chromium_src_chrome_installer_setup_deps
 
     if (webnn_install_runtime_in_chrome_installer) {
       sources += [
