diff --git a/chrome/browser/resource_coordinator/resource_coordinator_parts.h b/chrome/browser/resource_coordinator/resource_coordinator_parts.h
index 788e4bd6f9a2633d99a2c6abff289b69331e5bc6..39124c059203891c3d4b75a699ac96feee09dcef 100644
--- a/chrome/browser/resource_coordinator/resource_coordinator_parts.h
+++ b/chrome/browser/resource_coordinator/resource_coordinator_parts.h
@@ -10,10 +10,10 @@
 #include "chrome/browser/resource_coordinator/tab_load_tracker.h"
 #include "chrome/browser/resource_coordinator/tab_memory_metrics_reporter.h"
 
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
 #include "chrome/browser/resource_coordinator/tab_lifecycle_unit_source.h"
 #include "chrome/browser/resource_coordinator/tab_manager.h"
-#endif
+//#endif
 
 namespace resource_coordinator {
 
@@ -39,19 +39,19 @@ class ResourceCoordinatorParts {
   TabLoadTracker* tab_load_tracker() { return &tab_load_tracker_; }
 
   TabManager* tab_manager() {
-#if defined(OS_ANDROID)
-    return nullptr;
-#else
+//#if defined(OS_ANDROID)
+    //return nullptr;
+//#else
     return &tab_manager_;
-#endif  // defined(OS_ANDROID)
+//#endif  // defined(OS_ANDROID)
   }
 
   TabLifecycleUnitSource* tab_lifecycle_unit_source() {
-#if defined(OS_ANDROID)
-    return nullptr;
-#else
+//#if defined(OS_ANDROID)
+//    return nullptr;
+//#else
     return &tab_lifecycle_unit_source_;
-#endif  // defined(OS_ANDROID)
+//#endif  // defined(OS_ANDROID)
   }
 
  private:
@@ -62,7 +62,7 @@ class ResourceCoordinatorParts {
   // Created on demand the first time it's being accessed.
   std::unique_ptr<TabMemoryMetricsReporter> tab_memory_metrics_reporter_;
 
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
   // Any change to this #ifdef must be reflected as well in
   // chrome/browser/resource_coordinator/tab_manager_browsertest.cc
   //
@@ -71,7 +71,7 @@ class ResourceCoordinatorParts {
   // owned by TabManager.
   TabManager tab_manager_;
   TabLifecycleUnitSource tab_lifecycle_unit_source_;
-#endif
+//#endif
 
   DISALLOW_COPY_AND_ASSIGN(ResourceCoordinatorParts);
 };
