diff --git a/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_bubble_view_impl.cc b/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_bubble_view_impl.cc
index 2a13c892c12a5aa491c5e29f86113ef45ebc00ab..4068cf2a7d20417249324520a6a8234b5f4133ee 100644
--- a/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_bubble_view_impl.cc
+++ b/chrome/browser/ui/views/send_tab_to_self/send_tab_to_self_bubble_view_impl.cc
@@ -140,6 +140,7 @@ void SendTabToSelfBubbleViewImpl::Init() {
   CreateHintTextLabel();
   CreateDevicesScrollView();
 
+  BRAVE_SENT_TAB_TO_SELF_BUBBLE_VIEW_IMPL_INIT
   AddChildView(std::make_unique<views::Separator>());
   CreateManageDevicesLink();
 }
