diff --git a/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts b/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts
index 148f0d9c610a632f4cdf32c565c763786c967b37..6144a6fc2d6cdd7fd7e8fbcd9ab7ac04848738bf 100644
--- a/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts
+++ b/chrome/browser/resources/settings/site_settings_page/site_settings_page.ts
@@ -9,6 +9,7 @@
  */
 
 import 'chrome://resources/cr_elements/icons_lit.html.js';
+import '../brave_overrides/config.js'
 import 'chrome://resources/cr_elements/cr_collapse/cr_collapse.js';
 import 'chrome://resources/cr_elements/cr_expand_button/cr_expand_button.js';
 import 'chrome://resources/cr_elements/cr_link_row/cr_link_row.js';
