diff --git a/chrome/browser/flags/BUILD.gn b/chrome/browser/flags/BUILD.gn
index df003b051ed5945780f8affc7ee748aa57b9a1f3..d282fae68a58e732a019b85af37c40cd451e43ff 100644
--- a/chrome/browser/flags/BUILD.gn
+++ b/chrome/browser/flags/BUILD.gn
@@ -34,6 +34,7 @@ android_library("java") {
     "//third_party/android_deps:guava_android_java",
     "//third_party/androidx:androidx_annotation_annotation_java",
   ]
+  sources += brave_flags_java_sources
   srcjar_deps = [
     ":chrome_android_java_switches_srcjar",
     ":chrome_browser_flags_enums_srcjar",
