diff --git a/chrome/browser/themes/theme_service_factory.cc b/chrome/browser/themes/theme_service_factory.cc
index caec77bbe3ba59b030622d3a9ff4fb1edd0a476d..7fdb129e8f78a51d0c055ec2191e04e03ae28fb3 100644
--- a/chrome/browser/themes/theme_service_factory.cc
+++ b/chrome/browser/themes/theme_service_factory.cc
@@ -70,13 +70,16 @@ ThemeServiceFactory::ThemeServiceFactory()
           BrowserContextDependencyManager::GetInstance()) {
   DependsOn(extensions::ExtensionRegistryFactory::GetInstance());
   DependsOn(extensions::ExtensionPrefsFactory::GetInstance());
+#if !defined(OS_ANDROID)
   DependsOn(extensions::ExtensionSystemFactory::GetInstance());
+#endif
 }
 
 ThemeServiceFactory::~ThemeServiceFactory() {}
 
 KeyedService* ThemeServiceFactory::BuildServiceInstanceFor(
     content::BrowserContext* profile) const {
+  BRAVE_THEMESERVICEFACTORY_BUILDSERVICEINSTANCEFOR
 #if defined(OS_LINUX) && !defined(OS_CHROMEOS)
   using ThemeService = ThemeServiceAuraLinux;
 #endif
