diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index ffbec39c3e56433eb9f9680060dd06ced1ced81c..4417f54ce6930d023016ba67f43cc304f981284f 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2289,6 +2289,7 @@ static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_win) {
