diff --git a/chrome/browser/ui/autofill/chrome_autofill_client.cc b/chrome/browser/ui/autofill/chrome_autofill_client.cc
index b21d89cbef670e592819ab550e50225a9246e90d..2ad99f21344306a3ee5d567bfdd1de8b99233e3f 100644
--- a/chrome/browser/ui/autofill/chrome_autofill_client.cc
+++ b/chrome/browser/ui/autofill/chrome_autofill_client.cc
@@ -288,6 +288,12 @@ ui::ElementIdentifier GetElementId(AutofillClient::IphFeature iph_feature) {
 // static
 void ChromeAutofillClient::CreateForWebContents(
     content::WebContents* web_contents) {
+  if (!FromWebContents(web_contents)) {
+    web_contents->SetUserData(
+        UserDataKey(),
+        base::WrapUnique(new BraveChromeAutofillClient(web_contents)));
+  }
+  return;
   DCHECK(web_contents);
   if (!FromWebContents(web_contents)) {
     web_contents->SetUserData(
