diff --git a/chrome/browser/ui/views/tabs/tab_container.cc b/chrome/browser/ui/views/tabs/tab_container.cc
index 2cbcaf4f997ef78d3024acc4c8c08f3bc6f694cd..cb37e87af9f0e6e5ed5c93c5e9da3b5d1e1384be 100644
--- a/chrome/browser/ui/views/tabs/tab_container.cc
+++ b/chrome/browser/ui/views/tabs/tab_container.cc
@@ -1015,6 +1015,7 @@ gfx::Rect TabContainer::GetTargetBoundsForClosingTab(
   // Compute the target bounds for animating this tab closed.  The tab's left
   // edge should stay joined to the right edge of the previous tab, if any.
   gfx::Rect target_bounds = tab->bounds();
+  TAB_CONTAINER_START_REMOVE_TAB_ANIMATION_FOR_VERTICAL_TABS
   target_bounds.set_x(
       (former_model_index > 0)
           ? (tabs_view_model_.ideal_bounds(former_model_index - 1).right() -
