diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index d2accd0c76a1348632117094dc4ada5dc8c93e95..02f533fa8909d43596bb2e1131c87e0be0f7cb33 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -3350,6 +3350,7 @@ source_set("browser") {
       "//chrome/browser/safe_browsing/suspicious_site_warnings",
     ]
   }
+  public_deps += [ "//brave/browser:core" ]
 
   # This target intentionally has no sources of its own and no
   # allow_circular_includes_from; it only aggregates dependencies. Do NOT add
@@ -3519,6 +3520,7 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
