diff --git a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
index 0099aba7e3000c68ce1c556ea2bc6b68872ab85c..f411eb4d0fc3aa0043e5e692ad2b33a534003d9e 100644
--- a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
+++ b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
@@ -519,6 +519,7 @@ void RecentTabsSubMenuModel::BuildTabsFromOtherDevices() {
     DCHECK(!session->GetSessionName().empty());
     std::unique_ptr<ui::SimpleMenuModel> device_menu_model =
         CreateOtherDeviceSubMenu(session, tabs_in_session);
+    BRAVE_RECENT_TABS_SUB_MENU_MODEL_BUILD_TABS_FROM_OTHER_DEVICES
     const int command_id = GetAndIncrementNextMenuID();
     AddSubMenu(command_id, base::UTF8ToUTF16(session->GetSessionName()),
                device_menu_model.get());
