diff --git a/chrome/browser/ui/browser_window/BUILD.gn b/chrome/browser/ui/browser_window/BUILD.gn
index 1492bc76d26d48f88b59140ad8a4cac6a7d25913..535d86a21b1b2e438a536930ce16bdaa2ec8885f 100644
--- a/chrome/browser/ui/browser_window/BUILD.gn
+++ b/chrome/browser/ui/browser_window/BUILD.gn
@@ -22,6 +22,7 @@ source_set("browser_window") {
     "//ui/gfx",
   ]
   deps = [
+    "//brave/browser/ui/browser_window",
     "//components/sessions:session_id",
     "//ui/base/unowned_user_data",
   ]
