diff --git a/chrome/browser/resources/settings/privacy_page/cr_lottie.ts b/chrome/browser/resources/settings/privacy_page/cr_lottie.ts
index d5295c5856d76c144bcf9617e9aba78a1ac670c1..fce0231f8b1ae96114eff63dbe0c337c07f0be67 100644
--- a/chrome/browser/resources/settings/privacy_page/cr_lottie.ts
+++ b/chrome/browser/resources/settings/privacy_page/cr_lottie.ts
@@ -344,6 +344,7 @@ export class CrLottieElement extends CrLitElement {
       this.fire('cr-lottie-paused');
     } else if (event.data.name === 'stopped') {
       this.fire('cr-lottie-stopped');
+    } else if (event.data.name === 'complete') { this.fire('cr-lottie-complete');
     } else if (event.data.name === 'resized') {
       this.fire('cr-lottie-resized', event.data.size);
     }
