diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 5ad265fbe50d64e49b8cb8e5a2a2bc40ac7ca259..c693778365afc1d4a9a39041ab9805ce352bc136 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -664,6 +664,7 @@ class Browser : public TabStripModelObserver,
   // dragging.
   void SetIsInTabDragging(bool is_in_tab_dragging);
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
