diff --git a/chrome/browser/ui/views/user_education/browser_user_education_service.cc b/chrome/browser/ui/views/user_education/browser_user_education_service.cc
index a92f680d4d9ce9f2d42a8c4a9729d4305d839f03..56e7b8a30960a2687bfaa899454b2af6d6af62cc 100644
--- a/chrome/browser/ui/views/user_education/browser_user_education_service.cc
+++ b/chrome/browser/ui/views/user_education/browser_user_education_service.cc
@@ -269,6 +269,7 @@ void MaybeRegisterChromeFeaturePromos(
           feature_engagement::kIPHWebUiHelpBubbleTestFeature)) {
     return;
   }
+  BRAVE_MAYBE_REGISTER_CHROME_FEATURE_PROMOS
 
   // kIPHAutofillCreditCardBenefitFeature:
   registry.RegisterFeature(std::move(
