diff --git a/chrome/browser/ui/navigator/browser_navigator.cc b/chrome/browser/ui/navigator/browser_navigator.cc
index f8d51faf9020f0fd045b0e04718d12ff609c96f7..50c7aa174aa279e053f0edb8d0a312f6b85f673d 100644
--- a/chrome/browser/ui/navigator/browser_navigator.cc
+++ b/chrome/browser/ui/navigator/browser_navigator.cc
@@ -191,6 +191,7 @@ bool ShouldForceCaptivePortalSigninIntoCurrentTab(const NavigateParams& params,
 // Returns true on success. Otherwise, if changing params leads the browser
 // into an erroneous state, returns false.
 bool AdjustNavigateParamsForURL(NavigateParams* params) {
+  UpdateParams(params);
   // Check for some chrome:// pages which we always want to open in a
   // non-incognito window. Note that even though a ChromeOS guest session is
   // technically an incognito window, these URLs are allowed.
@@ -485,7 +486,11 @@ std::unique_ptr<content::WebContents> CreateTargetContents(
   scoped_refptr<content::SiteInstance> initial_site_instance_for_new_contents =
       params.opener ? params.opener->GetSiteInstance()
                     : tab_util::GetSiteInstanceForNewTab(
-                          params.browser->GetProfile(), url);
+                          params.browser->GetProfile(), url
+#if BUILDFLAG(ENABLE_CONTAINERS)
+                          , GetStoragePartitionConfigToInherit(params)
+#endif  // BUILDFLAG(ENABLE_CONTAINERS)
+);
 
   WebContents::CreateParams create_params(
       params.browser->GetProfile(), initial_site_instance_for_new_contents);
@@ -877,6 +882,9 @@ base::WeakPtr<content::NavigationHandle> NavigateImpl(
          params->user_gesture)) {
       tab_to_insert->set_opener(
           tabs::TabInterface::MaybeGetFromContents(params->source_contents));
+      if (params->navigation_initiated_from_sync) {            
+        tab_to_insert->set_opener_was_set_for_empty_new_tab(); 
+      }
     }
   }
 
@@ -894,6 +902,9 @@ base::WeakPtr<content::NavigationHandle> NavigateImpl(
            params->user_gesture)) {
         tab_to_insert->set_opener(
             tabs::TabInterface::MaybeGetFromContents(params->source_contents));
+      if (params->navigation_initiated_from_sync) {            
+        tab_to_insert->set_opener_was_set_for_empty_new_tab(); 
+      }
       }
       contents_to_navigate_or_insert = tab_to_insert->GetContents();
 
