diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
index e8ec8c4e18a23d2a0eeac2fc1b1a58180abde076..dee4269690976d8f9d3d3b6a0edd77c31ce18969 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
@@ -1970,7 +1970,7 @@ public class ToolbarManager
         }
     }
 
-    private boolean shouldSuppressToolbarLongPress() {
+    protected boolean shouldSuppressToolbarLongPress() {
         return mOmniboxFocusStateSupplier.get()
                 || (mToolbarPositionController != null
                         && mToolbarPositionController.doesPrefMismatchPosition())
