diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 98b4d0bc541224e8b9304fe8e110b18f7234d263..1061e99a8678414feb8dccbec4f65b1dbab6b31f 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -116,6 +116,7 @@ class AvatarToolbarButton : public ToolbarButton {
   void NotifyShowSigninPausedDelayEnded() const;
 
  private:
+  BRAVE_AVATAR_BUTTON_H
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
