diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page_browser_proxy.ts b/chrome/browser/resources/settings/privacy_page/privacy_page_browser_proxy.ts
index 6450085f7c924172bb077454015b865cd350a7be..11e30622c22529b568781a2514e28845a14cf517 100644
--- a/chrome/browser/resources/settings/privacy_page/privacy_page_browser_proxy.ts
+++ b/chrome/browser/resources/settings/privacy_page/privacy_page_browser_proxy.ts
@@ -46,7 +46,7 @@ export interface SecureDnsSetting {
 }
 
 export interface PrivacyPageBrowserProxy {
-  // <if expr="_google_chrome and not chromeos_ash">
+  // <if expr="True and not chromeos_ash">
   getMetricsReporting(): Promise<MetricsReporting>;
   setMetricsReportingEnabled(enabled: boolean): void;
 
@@ -84,7 +84,7 @@ export interface PrivacyPageBrowserProxy {
 }
 
 export class PrivacyPageBrowserProxyImpl implements PrivacyPageBrowserProxy {
-  // <if expr="_google_chrome and not chromeos_ash">
+  // <if expr="True and not chromeos_ash">
   getMetricsReporting() {
     return sendWithPromise('getMetricsReporting');
   }
