diff --git a/chrome/browser/ui/views/bubble/webui_bubble_manager.h b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
index ea2b2c5f0e7127677cbeb1a500ca778b544c7709..5815b27a3967fdd531f14fa024e9fe701bad2307 100644
--- a/chrome/browser/ui/views/bubble/webui_bubble_manager.h
+++ b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
@@ -146,7 +146,7 @@ class WebUIBubbleManagerImpl : public WebUIBubbleManager {
         task_manager_string_id_(task_manager_string_id) {}
   ~WebUIBubbleManagerImpl() override = default;
 
- private:
+ protected:
   void MaybeInitPersistentRenderer() override;
 
   base::WeakPtr<WebUIBubbleDialogView> CreateWebUIBubbleDialog(
@@ -231,6 +231,7 @@ WebUIBubbleManagerImpl<T>::CreateWebUIBubbleDialog(
 
   auto bubble_view = std::make_unique<WebUIBubbleDialogView>(
       anchor_view_, contents_wrapper->GetWeakPtr(), anchor, arrow);
+    BRAVE_WEBUI_BUBBLE_MANAGER_T_CREATE_WEB_UI_BUBBLE_DIALOG
 
   if (!widget_initialization_callback_.is_null()) {
     bubble_view->RegisterWidgetInitializedCallback(
