diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 4d1c638120a7d4aa7f41f1532af7b4e877b0a071..6260116c5b1bf4cc2f1eed0030cfeaf63b4ab08b 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -65,6 +65,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
