diff --git a/chrome/browser/profiles/profile.h b/chrome/browser/profiles/profile.h
index 1109975d740352cbad67af0e765fbfcc1da3fd3f..9b4616ded04b9359ba0cf07bd2ead725670823a2 100644
--- a/chrome/browser/profiles/profile.h
+++ b/chrome/browser/profiles/profile.h
@@ -264,6 +264,8 @@ class Profile : public content::BrowserContext {
   // Returns whether it is a guest session.
   virtual bool IsGuestSession() const;
 
+  bool IsTorProfile() const override;
+
   // Returns whether it is a system profile.
   virtual bool IsSystemProfile() const;
 
