diff --git a/chrome/browser/extensions/extension_management.h b/chrome/browser/extensions/extension_management.h
index bee944b5d8e2f15dce9da8148660ebed4763c0aa..91795e2e3ac4846190991ee9a8e11a8a8a990223 100644
--- a/chrome/browser/extensions/extension_management.h
+++ b/chrome/browser/extensions/extension_management.h
@@ -218,6 +218,7 @@ class ExtensionManagement : public KeyedService {
   bool is_child() const { return is_child_; }
 
  private:
+  friend class BraveExtensionManagement;
   using SettingsIdMap =
       std::unordered_map<ExtensionId,
                          std::unique_ptr<internal::IndividualSettings>>;
