diff --git a/chrome/browser/ui/color/material_chrome_color_mixer.cc b/chrome/browser/ui/color/material_chrome_color_mixer.cc
index 81eae8890775261a69964dfe2964984135fb9426..c21e0c5acb46839370b8f2b75d253b38f6c405c8 100644
--- a/chrome/browser/ui/color/material_chrome_color_mixer.cc
+++ b/chrome/browser/ui/color/material_chrome_color_mixer.cc
@@ -75,6 +75,7 @@ void ApplyChromeSettingsRefresh2026ColorOverrides(ui::ColorMixer& mixer) {
 
 void AddMaterialChromeColorMixer(ui::ColorProvider* provider,
                                  const ui::ColorProviderKey& key) {
+  [&]() -> void {
   // Adds the color recipes for browser UI colors (toolbar, bookmarks bar,
   // downloads bar etc). While both design systems continue to exist, the
   // material recipes are intended to leverage the existing chrome color mixers,
@@ -560,4 +561,8 @@ void AddMaterialChromeColorMixer(ui::ColorProvider* provider,
 
   ApplyChromeRefresh2026ColorOverrides(mixer);
   ApplyChromeSettingsRefresh2026ColorOverrides(mixer);
+  }();
+  #if !BUILDFLAG(IS_ANDROID)
+  AddMaterialBraveColorMixer(provider, key);
+  #endif  // #if !BUILDFLAG(IS_ANDROID)
 }
