diff --git a/chrome/browser/ui/autofill/chrome_autofill_client.h b/chrome/browser/ui/autofill/chrome_autofill_client.h
index 22fbf3f0c1a949c272ee290f4ec7dd4b3d6459f6..acd1348b727920380257cb9bd661f70546bfd0d8 100644
--- a/chrome/browser/ui/autofill/chrome_autofill_client.h
+++ b/chrome/browser/ui/autofill/chrome_autofill_client.h
@@ -187,6 +187,8 @@ class ChromeAutofillClient : public ContentAutofillClient,
   void ConfirmSaveIbanLocally(const Iban& iban,
                               bool should_show_prompt,
                               LocalSaveIbanPromptCallback callback) override;
+  void ConfirmAutocomplete(
+      base::OnceCallback<void(bool)> callback) const override;
   void ShowWebauthnOfferDialog(
       WebauthnDialogCallback offer_dialog_callback) override;
   void ShowWebauthnVerifyPendingDialog(
