diff --git a/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java b/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
index 1e7233fae74a53403d37922e9c6046990769c242..cad5733125a9ebfc01576627b4dc78722cc331c2 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
@@ -237,7 +237,7 @@ public abstract class FullscreenHtmlApiHandlerBase
         @Override
         public void onMultiWindowModeChanged(boolean isInMultiWindowMode) {
             if (mTab != null && !mIsInMultiWindowMode && isInMultiWindowMode) {
-                onExitFullscreen(mTab);
+                if (!mActivity.isInPictureInPictureMode()) onExitFullscreen(mTab);
             }
             mIsInMultiWindowMode = isInMultiWindowMode;
         }
@@ -300,6 +300,7 @@ public abstract class FullscreenHtmlApiHandlerBase
 
                     @Override
                     public void onHidden(Tab tab, @TabHidingType int reason) {
+                        BraveFullscreenHtmlApiHandlerBase.class.cast(FullscreenHtmlApiHandlerBase.this).mTabHiddenByChangedTabs = reason == TabHidingType.CHANGED_TABS;
                         // Clean up any fullscreen state that might impact other tabs.
                         exitPersistentFullscreenMode();
                     }
