diff --git a/chrome/installer/linux/rpm/build.sh b/chrome/installer/linux/rpm/build.sh
index 8e9c7e56813d8791c46fa28c866e67dbde03e60a..0c433285add2b2dbfd06bb2fdcca0a81a07d39be 100755
--- a/chrome/installer/linux/rpm/build.sh
+++ b/chrome/installer/linux/rpm/build.sh
@@ -15,8 +15,9 @@ gen_spec() {
   # Different channels need to install to different locations so they
   # don't conflict with each other.
   local PACKAGE_ORIG="${PACKAGE}"
-  local PACKAGE_FILENAME="${PACKAGE}-${CHANNEL}"
+  local PACKAGE_FILENAME="${PACKAGE}"
   if [ "$CHANNEL" != "stable" ]; then
+    local PACKAGE_FILENAME="${PACKAGE}-${CHANNEL}"
     local INSTALLDIR="${INSTALLDIR}-${CHANNEL}"
     local PACKAGE="${PACKAGE}-${CHANNEL}"
     local MENUNAME="${MENUNAME} (${CHANNEL})"
@@ -87,6 +88,12 @@ do_package() {
   PROVIDES="${PACKAGE}"
   RPM_COMMON_DEPS="${BUILDDIR}/rpm_common.deps"
   DEPENDS=$(cat "${RPM_COMMON_DEPS}" | tr '\n' ',')
+
+  # Ensure that our signing key is up-to-date (brave/brave-browser#4205).
+  if [ "$CHANNEL" = "stable" ]; then
+    DEPENDS="brave-keyring, ${DEPENDS}"
+  fi
+
   gen_spec
 
   # Create temporary rpmbuild dirs.
@@ -108,7 +115,10 @@ do_package() {
     --define "${COMPRESSION_OPT}" \
     --define "__os_install_post  %{nil}" \
     "${SPEC}"
-  PKGNAME="${PACKAGE}-${CHANNEL}-${VERSION}-${PACKAGE_RELEASE}"
+  PKGNAME="${PACKAGE}-${VERSION}-${PACKAGE_RELEASE}"
+  if [ "$CHANNEL" != "stable" ]; then
+    PKGNAME="${PACKAGE}-${CHANNEL}-${VERSION}-${PACKAGE_RELEASE}"
+  fi
   mv "$RPMBUILD_DIR/RPMS/$ARCHITECTURE/${PKGNAME}.${ARCHITECTURE}.rpm" \
      "${OUTPUTDIR}"
   # Make sure the package is world-readable, otherwise it causes problems when
@@ -145,7 +155,10 @@ verify_channel() {
       CHANNEL=stable
       ;;
     unstable|dev|alpha )
-      CHANNEL=unstable
+      CHANNEL=dev
+      ;;
+    nightly )
+      CHANNEL=nightly
       ;;
     testing|beta )
       CHANNEL=beta
