diff --git a/chrome/browser/ui/views/permission_bubble/permission_prompt_impl.cc b/chrome/browser/ui/views/permission_bubble/permission_prompt_impl.cc
index 6a6df57f65ee899b1c20feedf1fc5ea82164e3b5..185be344c2c619cd67d28a4c3e32ee090b233a6b 100644
--- a/chrome/browser/ui/views/permission_bubble/permission_prompt_impl.cc
+++ b/chrome/browser/ui/views/permission_bubble/permission_prompt_impl.cc
@@ -141,6 +141,7 @@ PermissionsBubbleDialogDelegateView::PermissionsBubbleDialogDelegateView(
     AddChildView(label_container);
   }
 
+  AddAdditionalWidevineViewControlsIfNeeded(this, requests);
   chrome::RecordDialogCreation(chrome::DialogIdentifier::PERMISSIONS);
 }
 
