diff --git a/chrome/browser/resources/extensions/item.html.ts b/chrome/browser/resources/extensions/item.html.ts
index 7069f3ae2facef97a238223699f68e8d5c9be763..ffdcf43a5f74d30edaf87c1d4cd8ccefc09a1e93 100644
--- a/chrome/browser/resources/extensions/item.html.ts
+++ b/chrome/browser/resources/extensions/item.html.ts
@@ -18,8 +18,7 @@ export function getHtml(this: ItemElement) {
 <div id="card" class="${this.computeClasses_()}">
   <div id="main">
     <div id="icon-wrapper">
-      <img id="icon" src="${this.data.iconUrl}"
-          aria-describedby="a11yAssociation" alt="">
+      <img id="icon" src="${this.data.iconUrl}" aria-describedby="a11yAssociation" alt="">
       ${this.computeSourceIndicatorIcon_() ? html`
         <div id="source-indicator">
           <div class="source-icon-wrapper" role="img"
@@ -43,18 +42,17 @@ export function getHtml(this: ItemElement) {
           doesn't shrink (because it's not display: flex).-->
       <div>
         <div id="name-and-version" class="layout-horizontal-center">
-          <div id="name" role="heading" aria-level="3"
-              class="clippable-flex-text">
+          <div id="name" role="heading" aria-level="3" class="clippable-flex-text">
             ${this.data.name}
           </div>
-          <span id="version" class="cr-secondary-text"
-              ?hidden="${!this.inDevMode}">
+          <span id="version" class="cr-secondary-text" ?hidden="${!this.inDevMode}">
             ${this.data.version}
           </span>
-        </div>
+        </div><div ?hidden="${!this.isBraveHosted_(this.data.id)}" class="brave-hosted">
+          $i18n{braveHosted}
+       </div>
       </div>
-      <div id="description" class="cr-secondary-text multiline-clippable-text"
-          ?hidden="${!this.showDescription_()}">
+      <div id="description" class="cr-secondary-text multiline-clippable-text" ?hidden="${!this.showDescription_()}">
         ${this.data.description}
       </div>
       ${this.showSevereWarnings() ? html`
@@ -137,13 +135,10 @@ export function getHtml(this: ItemElement) {
   </div>
   <div id="button-strip" class="layout-horizontal-center cr-secondary-text">
     <div class="layout-horizontal-center flex">
-      <cr-button id="detailsButton" @click="${this.onDetailsClick_}"
-          aria-describedby="a11yAssociation">
+      <cr-button id="detailsButton" @click="${this.onDetailsClick_}" aria-describedby="a11yAssociation">
         $i18n{itemDetails}
       </cr-button>
-      <cr-button id="removeButton" @click="${this.onRemoveClick_}"
-          aria-describedby="a11yAssociation"
-          ?hidden="${this.data.mustRemainInstalled}">
+      <cr-button id="removeButton" @click="${this.onRemoveClick_}" aria-describedby="a11yAssociation" ?hidden="${this.data.mustRemainInstalled}">
         $i18n{remove}
       </cr-button>
       ${this.shouldShowErrorsButton_() ? html`
@@ -177,18 +172,9 @@ export function getHtml(this: ItemElement) {
           aria-describedby="a11yAssociation" class="action-button">
         $i18n{itemReload}
       </cr-button>` : ''}
-    <cr-tooltip-icon id="parentDisabledPermissionsToolTip"
-        ?hidden="${!this.data.disableReasons.parentDisabledPermissions}"
-        tooltip-text="$i18n{parentDisabledPermissions}"
-        icon-class="cr20:kite"
-        icon-aria-label="$i18n{parentDisabledPermissions}">
+    <cr-tooltip-icon id="parentDisabledPermissionsToolTip" ?hidden="${!this.data.disableReasons.parentDisabledPermissions}" tooltip-text="$i18n{parentDisabledPermissions}" icon-class="cr20:kite" icon-aria-label="$i18n{parentDisabledPermissions}">
     </cr-tooltip-icon>
-    <cr-toggle id="enableToggle"
-        aria-label="${this.getEnableToggleAriaLabel_()}"
-        aria-describedby="a11yAssociation"
-        ?checked="${this.isEnabled_()}" @change="${this.onEnableToggleChange_}"
-        ?disabled="${!this.isEnableToggleEnabled_()}"
-        ?hidden="${!this.showEnableToggle_()}">
+    <cr-toggle id="enableToggle" aria-label="${this.getEnableToggleAriaLabel_()}" aria-describedby="a11yAssociation" ?checked="${this.isEnabled_()}" @change="${this.onEnableToggleChange_}" ?disabled="${!this.isEnableToggleEnabled_()}" ?hidden="${!this.showEnableToggle_()}">
     </cr-toggle>
   </div>
 </div>
