diff --git a/chrome/browser/ui/android/appmenu/internal/BUILD.gn b/chrome/browser/ui/android/appmenu/internal/BUILD.gn
index bc4ee3632d13c79ded08aed6c03402f57dc392ef..b8e364388e92722097a36afc19604e9b2d752797 100644
--- a/chrome/browser/ui/android/appmenu/internal/BUILD.gn
+++ b/chrome/browser/ui/android/appmenu/internal/BUILD.gn
@@ -22,6 +22,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/ui/appmenu/AppMenuItemViewBinder.java",
     "java/src/org/chromium/chrome/browser/ui/appmenu/DividerLineMenuItemViewBinder.java",
   ]
+  sources += brave_browser_ui_android_appmenu_internal_sources
 
   deps = [
     ":java_resources",
