diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 28f9a5eab90529bd9268f681b26eef2612cf5902..0a02dfe039024a2bcf2d174c531bc001174f1cc4 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -747,6 +747,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome/browser/resources/media/mei_preload:component",
