diff --git a/chrome/browser/ui/tabs/tab_strip_model.h b/chrome/browser/ui/tabs/tab_strip_model.h
index 2f09a4f136216d6b00e87184d0b28b0cc255a3a2..3832763ad5fae3f476e54264dba2fdd5a2a5809f 100644
--- a/chrome/browser/ui/tabs/tab_strip_model.h
+++ b/chrome/browser/ui/tabs/tab_strip_model.h
@@ -202,6 +202,7 @@ class TabStripModel {
   TabStripModel(const TabStripModel&) = delete;
   TabStripModel& operator=(const TabStripModel&) = delete;
 
+  virtual
   ~TabStripModel();
 
   // Retrieves the TabStripModelDelegate associated with this TabStripModel.
