diff --git a/chrome/browser/extensions/api/cookies/cookies_api.cc b/chrome/browser/extensions/api/cookies/cookies_api.cc
index 46d1698f07cb9ed2866073489fbc4391ae23e337..e897e65fac5714736ceed081575016ac27acbe91 100644
--- a/chrome/browser/extensions/api/cookies/cookies_api.cc
+++ b/chrome/browser/extensions/api/cookies/cookies_api.cc
@@ -155,6 +155,7 @@ CookiesEventRouter::~CookiesEventRouter() = default;
 
 void CookiesEventRouter::OnCookieChange(bool otr,
                                         const net::CookieChangeInfo& change) {
+  if (otr && !profile_->GetPrimaryOTRProfile(/*create_if_needed=*/false)) return;
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
   // There is no way to represent non-serializable
   // partition keys in JS so return to prevent a crash.
@@ -223,6 +224,7 @@ void CookiesEventRouter::OnCookieChange(bool otr,
 }
 
 void CookiesEventRouter::OnOffTheRecordProfileCreated(Profile* off_the_record) {
+  if (off_the_record->IsTor()) return;
   // When an off-the-record spinoff of |profile_| is created, start listening
   // for cookie changes there. The OTR receiver should never be bound, since
   // there wasn't previously an OTR profile.
