diff --git a/chrome/app_shim/app_shim_controller.mm b/chrome/app_shim/app_shim_controller.mm
index 882808b999db0655873acb9417264cce7bcdea8b..efd55de59bc24db362c816715f25f3db5ef9bb12 100644
--- a/chrome/app_shim/app_shim_controller.mm
+++ b/chrome/app_shim/app_shim_controller.mm
@@ -633,6 +633,7 @@ void AppShimController::SendBootstrapOnShimConnected(
              app_mode::kIsNormalLaunch))
           ? chrome::mojom::AppShimLaunchType::kRegisterOnly
           : chrome::mojom::AppShimLaunchType::kNormal;
+  BRAVE_SEND_BOOTSTRAP_ON_SHIM_CONNECTED
   app_shim_info->files = launch_files_;
   app_shim_info->urls = launch_urls_;
 
