diff --git a/chrome/browser/ui/views/tabs/tab_strip.cc b/chrome/browser/ui/views/tabs/tab_strip.cc
index e36c258bf84aaf4eb176e0456140dccb351ce687..e1733260499d2901437cf4000c844d96a2fa3e9b 100644
--- a/chrome/browser/ui/views/tabs/tab_strip.cc
+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
@@ -582,6 +582,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
 
   std::vector<gfx::Rect> CalculateBoundsForDraggedViews(
       const std::vector<TabSlotView*>& views) override {
+    BRAVE_TAB_DRAG_CONTEXT_IMPL_CALCULATE_BOUNDS_FOR_DRAGGED_VIEWS
     CHECK(!views.empty()) << "The views vector must not be empty.";
 
     std::vector<gfx::Rect> bounds;
@@ -926,6 +927,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
         continue;
       }
 
+      BRAVE_TAB_DRAG_CONTEXT_IMPL_CALCULATE_INSERTION_INDEX
       // If there's a group header here, and we're dragging a group, we might
       // end up on either side of that header. Check both cases to find the
       // best option.
@@ -1506,7 +1508,7 @@ void TabStrip::SetSelection(const ui::ListSelectionModel& new_selection) {
       const tab_groups::TabGroupId new_group = selected_tab->group().value();
       // If the tab that is about to be selected is in a collapsed group,
       // automatically expand the group.
-      if (IsGroupCollapsed(new_group)) {
+      if (IsGroupCollapsed(new_group) && IsTabStripEditable()) {
         ToggleTabGroupCollapsedState(
             new_group, ToggleTabGroupCollapsedStateOrigin::kTabsSelected);
       }
