diff --git a/chrome/browser/ui/chrome_pages.h b/chrome/browser/ui/chrome_pages.h
index d52b78b1fe21..0ae306544986 100644
--- a/chrome/browser/ui/chrome_pages.h
+++ b/chrome/browser/ui/chrome_pages.h
@@ -13,9 +13,9 @@
 #include "components/content_settings/core/common/content_settings_types.h"
 #include "url/gurl.h"
 
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
 #include "chrome/browser/signin/signin_promo.h"
-#endif
+//#endif
 
 #if defined(OS_CHROMEOS)
 #include "chrome/browser/ui/webui/settings/chromeos/app_management/app_management_uma.h"
@@ -146,7 +146,7 @@ void ShowAppManagementPage(Profile* profile,
 void ShowPrintManagementApp(Profile* profile);
 #endif
 
-#if !defined(OS_ANDROID) && !defined(OS_CHROMEOS)
+#if !defined(OS_CHROMEOS)
 // Initiates signin in a new browser tab.
 void ShowBrowserSignin(Browser* browser,
                        signin_metrics::AccessPoint access_point);
