diff --git a/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java b/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
index ce52881ac226d81e4763a39cbf5c6ec081be9403..dc4a9297f5abcec758f2eb2f9e8669778355174e 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/fullscreen/FullscreenHtmlApiHandlerBase.java
@@ -320,6 +320,11 @@ public abstract class FullscreenHtmlApiHandlerBase
 
                     @Override
                     public void onHidden(Tab tab, @TabHidingType int reason) {
+                        if (BraveFullscreenHtmlApiHandlerBase.class.cast(
+                                        FullscreenHtmlApiHandlerBase.this)
+                                .maybeSkipExitFullscreenOnTabHidden(mActivity, reason)) {
+                            return;
+                        }
                         // Clean up any fullscreen state that might impact other tabs.
                         mFullscreenManagerDelegate.onExitFullscreen(null);
                     }
@@ -882,6 +887,10 @@ public abstract class FullscreenHtmlApiHandlerBase
     @Override
     public void onActivityStateChange(Activity activity, int newState) {
         if (newState == ActivityState.STOPPED && mExitFullscreenOnStop) {
+            if (BraveFullscreenHtmlApiHandlerBase.class.cast(FullscreenHtmlApiHandlerBase.this)
+                        .shouldPreservePersistentFullscreenForPictureInPicture(activity)) {
+                return;
+            }
             // Exit fullscreen in onStop to ensure the system UI flags are set
             // correctly when
             // showing again (on JB MR2+ builds, the omnibox would be covered by the
