diff --git a/chrome/browser/ui/views/frame/multi_contents_view.h b/chrome/browser/ui/views/frame/multi_contents_view.h
index 08e455d85248c522165cdccf553f93d720f7febc..9667d5d6508abc73707f4349fafe5362affd06fa 100644
--- a/chrome/browser/ui/views/frame/multi_contents_view.h
+++ b/chrome/browser/ui/views/frame/multi_contents_view.h
@@ -21,7 +21,7 @@
 class BrowserView;
 class ContentsWebView;
 class MultiContentsDropTargetView;
-class MultiContentsResizeArea;
+// class MultiContentsResizeArea;
 class MultiContentsViewDelegate;
 class MultiContentsViewDropTargetController;
 class MultiContentsViewMiniToolbar;
