diff --git a/chrome/browser/ui/views/tabs/hovercard/tab_hover_card_bubble_view.cc b/chrome/browser/ui/views/tabs/hovercard/tab_hover_card_bubble_view.cc
index 8699ee468269f708fba42d436d742e5cb58c4f1c..da29c7e534c26afb3e5115884076b35117a2efe8 100644
--- a/chrome/browser/ui/views/tabs/hovercard/tab_hover_card_bubble_view.cc
+++ b/chrome/browser/ui/views/tabs/hovercard/tab_hover_card_bubble_view.cc
@@ -373,6 +373,7 @@ END_METADATA
 // TabHoverCardBubbleView::TabCardView
 // ----------------------------------------------------------
 class TabHoverCardBubbleView::TabCardView : public views::View {
+  friend class BraveTabCardView;
   METADATA_HEADER(TabCardView, views::View)
 
  public:
@@ -443,7 +444,8 @@ class TabHoverCardBubbleView::TabCardView : public views::View {
     }
   }
 
-  void UpdateContent(const HoverCardAnchorTarget* anchor_target) {
+  
+  virtual void UpdateContent(const HoverCardAnchorTarget* anchor_target) {
     CHECK(std::holds_alternative<TabCardData>(anchor_target->data()));
     const TabCardData& card_data = std::get<TabCardData>(anchor_target->data());
 
@@ -680,7 +682,7 @@ TabHoverCardBubbleView::TabHoverCardBubbleView(
 
   SetProperty(views::kElementIdentifierKey, kHoverCardBubbleElementId);
 
-  tab_card_view_ = AddChildView(std::make_unique<TabCardView>(this));
+  tab_card_view_ = AddChildView(CreateTabCardView());
 
   if (std::holds_alternative<TabCardData>(anchor_target->data())) {
     const TabCardData& card_data = std::get<TabCardData>(anchor_target->data());
