diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index a18d538abdba06accad985ab9d49b3aee0c1900c..31950f87e95590a311a4cecb9d70dadc084d0688 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1316,6 +1316,7 @@ static_library("extensions") {
   } else {
     sources += [ "api/braille_display_private/braille_controller_stub.cc" ]
   }
+  deps += brave_extensions_resources
 }
 
 static_library("test_support") {
