diff --git a/components/content_settings/core/browser/content_settings_pref_provider.h b/components/content_settings/core/browser/content_settings_pref_provider.h
index 081d0c2b18cd817ca153628669ea777df5a70249..b13834488239f900ed8f474f7e41cd737ec1f474 100644
--- a/components/content_settings/core/browser/content_settings_pref_provider.h
+++ b/components/content_settings/core/browser/content_settings_pref_provider.h
@@ -66,6 +66,7 @@ class PrefProvider : public UserModifiableProvider {
   void SetClockForTesting(base::Clock* clock);
 
  private:
+  friend class BravePrefProvider;
   friend class DeadlockCheckerObserver;  // For testing.
 
   void Notify(const ContentSettingsPattern& primary_pattern,
