diff --git a/components/strings/BUILD.gn b/components/strings/BUILD.gn
index d408e22cc89bf87981eea2a2780cf8da4f46b670..91ddf4c7b9be75de43e700271bfd3f6013856d29 100644
--- a/components/strings/BUILD.gn
+++ b/components/strings/BUILD.gn
@@ -43,6 +43,7 @@ grit_strings("components_strings") {
   if (is_android) {
     create_android_resources = true
   }
+  deps = [ "//brave/components/resources:strings" ]
 }
 
 if (is_android) {
