diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 53b261069b3712a14eada688901bc6a5a3291d63..0b36ed6bcb69ded7f7a8b389b9ab1998dfa362cf 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -49,6 +49,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 typedef int64_t VisitID;
