diff --git a/components/autofill/core/browser/payments/credit_card_access_manager.cc b/components/autofill/core/browser/payments/credit_card_access_manager.cc
index b826e3f1b7dee681f73c53ee1f615e96f1172c58..94cfdcbb2aae6a6c3d612ec31ea65d8655fc71da 100644
--- a/components/autofill/core/browser/payments/credit_card_access_manager.cc
+++ b/components/autofill/core/browser/payments/credit_card_access_manager.cc
@@ -380,10 +380,10 @@ void CreditCardAccessManager::FetchCreditCard(
   // Log the server card unmasking attempt, and differentiate based on server
   // card or virtual card.
   if (ShouldLogServerCardUnmaskAttemptMetrics(record_type)) {
-    autofill_metrics::LogServerCardUnmaskAttempt(
-        record_type == CreditCard::RecordType::kVirtualCard
-            ? PaymentsRpcCardType::kVirtualCard
-            : PaymentsRpcCardType::kServerCard);
+    if (record_type == CreditCard::RecordType::kVirtualCard) {
+      autofill_metrics::LogServerCardUnmaskAttempt(
+          PaymentsRpcCardType::kVirtualCard);
+    }
   }
 
   // If the to-be-unmasked card is a virtual card eligible card (this also
