diff --git a/components/os_crypt/os_crypt_win.cc b/components/os_crypt/os_crypt_win.cc
index 91f2c6d4f79cf233770abbbcaaedbdd246ee044e..84b8a28a8f18a074ce02bdbdc9821693f22d854d 100644
--- a/components/os_crypt/os_crypt_win.cc
+++ b/components/os_crypt/os_crypt_win.cc
@@ -177,6 +177,7 @@ bool OSCryptImpl::DecryptString16(const std::string& ciphertext,
 // static
 bool OSCryptImpl::EncryptString(const std::string& plaintext,
                             std::string* ciphertext) {
+  if (IsEncryptionDisabled(plaintext, ciphertext)) { return true; }
   if (g_use_legacy)
     return EncryptStringWithDPAPI(plaintext, ciphertext);
 
@@ -203,6 +204,7 @@ bool OSCryptImpl::EncryptString(const std::string& plaintext,
 // static
 bool OSCryptImpl::DecryptString(const std::string& ciphertext,
                             std::string* plaintext) {
+  if (IsEncryptionDisabled(ciphertext, plaintext)) { return true; }
   if (!base::StartsWith(ciphertext, kEncryptionVersionPrefix,
                         base::CompareCase::SENSITIVE))
     return DecryptStringWithDPAPI(ciphertext, plaintext);
