diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index 44a59ca4e57f3977c9e9ed178e2cac73aa029c9a..636fb19de1a38e2600cfe885861a28d9d4a1172c 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -102,6 +102,7 @@ source_set("permissions") {
     "//ui/base",
     "//url",
   ]
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
