diff --git a/components/component_updater/component_updater_service.h b/components/component_updater/component_updater_service.h
index 622a95f1206e0f9bcd2dc925c2f3350717e767f1..3931777b6e5a5591f30eaa4b9def56bd677165d1 100644
--- a/components/component_updater/component_updater_service.h
+++ b/components/component_updater/component_updater_service.h
@@ -145,6 +145,7 @@ class ComponentUpdateService {
 
   friend class speech::SodaInstallerImpl;
   friend class ::ComponentsHandler;
+  BRAVE_COMPONENT_UPDATER_SERVICE_H_
   FRIEND_TEST_ALL_PREFIXES(ComponentInstallerTest, RegisterComponent);
 };
 
@@ -158,6 +159,7 @@ class OnDemandUpdater {
   enum class Priority { BACKGROUND = 0, FOREGROUND = 1 };
 
   virtual ~OnDemandUpdater() = default;
+  BRAVE_COMPONENT_UPDATER_SERVICE_H_ON_DEMAND_UPDATER
 
  private:
   friend class OnDemandTester;
