diff --git a/components/translate/core/browser/translate_ui_delegate.h b/components/translate/core/browser/translate_ui_delegate.h
index 25950625178a310f52f8bc29db29e9ee8acc1c14..42190a328b7e2a41a6969a042c5a2adf690157d0 100644
--- a/components/translate/core/browser/translate_ui_delegate.h
+++ b/components/translate/core/browser/translate_ui_delegate.h
@@ -154,6 +154,7 @@ class TranslateUIDelegate {
   void MaybeSetContentLanguages();
 
  private:
+  friend class BraveTranslateUIDelegate;
   FRIEND_TEST_ALL_PREFIXES(TranslateUIDelegateTest, GetPageHost);
   FRIEND_TEST_ALL_PREFIXES(TranslateUIDelegateTest, MaybeSetContentLanguages);
 
