diff --git a/components/infobars/core/infobar_delegate.h b/components/infobars/core/infobar_delegate.h
index cc6ac572da5ea5f8444ad431f2021cd3676fbe96..7dff1514cab4f6dd6c8294edc6221baef77490fc 100644
--- a/components/infobars/core/infobar_delegate.h
+++ b/components/infobars/core/infobar_delegate.h
@@ -164,6 +164,7 @@ class InfoBarDelegate {
     SYNC_ERROR_INFOBAR_DELEGATE_ANDROID = 97,
     MIXED_CONTENT_DOWNLOAD_INFOBAR_DELEGATE_ANDROID = 98,
     CONDITIONAL_TAB_STRIP_INFOBAR_ANDROID = 99,
+    BRAVE_INFOBAR_DELEGATE_IDENTIFIERS
   };
 
   // Describes navigation events, used to decide whether infobars should be
