diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 0d76b6a84ebed1c856e6c3bbd2bf448363c154ee..4dad49cc7039938f16a7d1987f811110921ad739 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -275,6 +275,7 @@ class SyncServiceImpl : public SyncService,
   size_t GetQueuedLocalDataMigrationItemCountForTest() const;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
