diff --git a/components/pdf/renderer/pdf_view_web_plugin_client.cc b/components/pdf/renderer/pdf_view_web_plugin_client.cc
index 3d73ca3d6d204d39a7c84c5bab06b06006735c61..5b57425ca6a6b53436b04056517585b93d1ea395 100644
--- a/components/pdf/renderer/pdf_view_web_plugin_client.cc
+++ b/components/pdf/renderer/pdf_view_web_plugin_client.cc
@@ -46,10 +46,12 @@
 namespace pdf {
 
 PdfViewWebPluginClient::PdfViewWebPluginClient(
-    content::RenderFrame* render_frame)
+    content::RenderFrame* render_frame,
+    service_manager::BinderRegistry* registry)
     : render_frame_(render_frame),
       v8_value_converter_(content::V8ValueConverter::Create()),
-      isolate_(blink::MainThreadIsolate()) {
+      isolate_(blink::MainThreadIsolate()),
+      registry_(registry) {
   DCHECK(render_frame_);
 }
 
@@ -256,7 +258,7 @@ void PdfViewWebPluginClient::RecordComputedAction(const std::string& action) {
 std::unique_ptr<chrome_pdf::PdfAccessibilityDataHandler>
 PdfViewWebPluginClient::CreateAccessibilityDataHandler(
     chrome_pdf::PdfAccessibilityActionHandler* action_handler) {
-  return std::make_unique<PdfAccessibilityTree>(render_frame_, action_handler);
+  return std::make_unique<PdfAccessibilityTree>(render_frame_, action_handler, registry_);
 }
 
 }  // namespace pdf
