diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 6bf19b81e988f6a97723d33c880132a7f438b9ef..eae8611dd05fb1b290f33b41d9811015ab0d8b49 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -49,6 +49,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 typedef int64_t VisitID;
