diff --git a/components/browser_sync/profile_sync_service.cc b/components/browser_sync/profile_sync_service.cc
index 787e2d24941cbaa469373ed5baad14d1ac9f3722..f8d2fa4e53602d61fb777d113489af5413a11d0a 100644
--- a/components/browser_sync/profile_sync_service.cc
+++ b/components/browser_sync/profile_sync_service.cc
@@ -480,6 +480,8 @@ void ProfileSyncService::StartUpSlowEngineComponents() {
                                ->GetSyncUserAgent();
   params.http_factory_getter = MakeHttpPostProviderFactoryGetter();
   params.credentials = auth_manager_->GetCredentials();
+  if (IsBraveSyncEnabled())
+    BraveEngineParamsInit(&params);
   DCHECK(!params.credentials.account_id.empty() || IsLocalSyncEnabled());
   if (!base::FeatureList::IsEnabled(switches::kSyncE2ELatencyMeasurement)) {
     invalidation::InvalidationService* invalidator =
