diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index afa60922b31d68e5013e06c5e0807852edabced5..e2d8b60aed4a89b9c31c7e759b0c9b8d2158e19a 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -112,6 +112,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
