diff --git a/components/permissions/permission_util.cc b/components/permissions/permission_util.cc
index afb346fc7191e9ccd6a14f04dc7dacbeadc7aca7..11df3b6cd1a4aa82e89774ce905cec15cacccd47 100644
--- a/components/permissions/permission_util.cc
+++ b/components/permissions/permission_util.cc
@@ -311,6 +311,7 @@ ContentSettingsType PermissionUtil::PermissionTypeToContentSettingTypeSafe(
       return ContentSettingsType::DISPLAY_CAPTURE;
     case PermissionType::NUM:
       break;
+    PERMISSION_UTIL_PERMISSION_TYPE_TO_CONTENT_SETTINGS_TYPE
   }
 
   return ContentSettingsType::DEFAULT;
