diff --git a/components/history_embeddings/content/history_embeddings_service.h b/components/history_embeddings/content/history_embeddings_service.h
index f09226eaa4669d77c4d88247f0fa69fb995dabb4..dafa76c3738ea473aee460405cedc0596171042b 100644
--- a/components/history_embeddings/content/history_embeddings_service.h
+++ b/components/history_embeddings/content/history_embeddings_service.h
@@ -111,6 +111,7 @@ class HistoryEmbeddingsService
                       std::optional<base::Time> time_range_start,
                       size_t count,
                       bool skip_answering,
+                      std::vector<history::URLID> url_id_filter,
                       SearchResultCallback callback) override;
 
   // Weak `this` provider method.
