diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 5de6bdd3a518dd7d43648afec9f55cca557d1011..30087b34fb6d77c0ad149564957797ac59135b17 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -55,6 +55,8 @@ enum VisitSource {
   SOURCE_SAFARI_IMPORTED = 5,
   SOURCE_ACTOR = 6,
   SOURCE_OS_MIGRATION_IMPORTED = 7,
+  SOURCE_CHROME_IMPORTED = 8,
+  SOURCE_BRAVE_IMPORTED = 9,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
