diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index b8ef1a5a8200d422e4cd8997b42f8a5bc5b2cfac..abaaa2e558632f8eacd67a436f86985098d4c3cf 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -56,6 +56,8 @@ enum VisitSource {
   SOURCE_SAFARI_IMPORTED = 5,
   SOURCE_ACTOR = 6,  // Added by the GLIC actor.
   SOURCE_OS_MIGRATION_IMPORTED = 7,
+  SOURCE_CHROME_IMPORTED = 8,
+  SOURCE_BRAVE_IMPORTED = 9,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
