diff --git a/components/page_info/BUILD.gn b/components/page_info/BUILD.gn
index c29bced9e24a432caa3e147a262c3cded7db3f9e..3ec21248f92c4f8d42ccaaad253598dc5c678844 100644
--- a/components/page_info/BUILD.gn
+++ b/components/page_info/BUILD.gn
@@ -44,6 +44,7 @@ static_library("page_info") {
     "//services/device/public/cpp:device_features",
     "//services/metrics/public/cpp:ukm_builders",
   ]
+  deps += [ "//brave/components/page_info" ]
 
   if (is_android) {
     deps += [
