diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 4d970f45ddc7eae734fac2f5b1efc5fbba7815d9..237b63de75b6b7b91a9473b206b58b5591b8c282 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -979,6 +979,7 @@ void PageInfo::PresentSitePermissions() {
       }
     }
 
+    BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
     if (ShouldShowPermission(
             permission_info, site_url_, content_settings, web_contents(),
             HasContentSettingChangedViaPageInfo(permission_info.type),
