diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index c489339039b629e0c08eebb9fe67e58b16e92abe..0318d0e3b4560392e96081d24742f0891bac1884 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -142,6 +142,7 @@ source_set("permissions") {
   if (!is_android) {
     deps += [ "//components/guest_view/browser" ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
