diff --git a/components/content_settings/core/browser/BUILD.gn b/components/content_settings/core/browser/BUILD.gn
index fe0850fd12e21fa1dd99c78cd4a237ad750d0021..30872a6305fc99fee3460f047e27f2b7c7f21808 100644
--- a/components/content_settings/core/browser/BUILD.gn
+++ b/components/content_settings/core/browser/BUILD.gn
@@ -63,6 +63,7 @@ jumbo_static_library("browser") {
     "//services/preferences/public/cpp",
     "//url",
   ]
+  deps += [ "//brave/components/content_settings/core/browser" ]
 
   if (is_android) {
     deps += [ "//media" ]
