diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 82fbe89ede3170e1d802704a77fcaff1bd70ed7b..ce255a500ee8dd87142d8a00c02c409d7aca35ab 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -379,6 +379,7 @@ static_library("browser") {
     "//ui/base",
     "//ui/gfx",
   ]
+  sources += brave_components_omnibox_browser_sources deps += brave_components_omnibox_browser_deps public_deps += brave_components_omnibox_browser_public_deps
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
 
@@ -1005,6 +1006,7 @@ fuzzer_test("search_suggestion_parser_fuzzer") {
 
 mojom("mojo_bindings") {
   sources = [ "omnibox.mojom" ]
+  generate_legacy_js_bindings = true
 
   webui_module_path = "chrome://resources/mojo/components/omnibox/browser"
 
