diff --git a/components/content_settings/core/browser/BUILD.gn b/components/content_settings/core/browser/BUILD.gn
index dbb8fbafd1e301f898b5edb79036004765f2e2f7..c8f3cba57f8d93dff4cff01010424044cd7915a4 100644
--- a/components/content_settings/core/browser/BUILD.gn
+++ b/components/content_settings/core/browser/BUILD.gn
@@ -83,6 +83,7 @@ static_library("browser") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_content_settings_core_browser_sources deps += brave_components_content_settings_core_browser_deps
 }
 
 source_set("unit_tests") {
