diff --git a/components/download/public/background_service/clients.h b/components/download/public/background_service/clients.h
index 9943c8c5c2b230a427b13a9b32eeffeb42dae4bf..31a65ea21f9d4a15caf0ecfcd4b68e77005be39a 100644
--- a/components/download/public/background_service/clients.h
+++ b/components/download/public/background_service/clients.h
@@ -46,6 +46,7 @@ enum class DownloadClient {
 
   // New clients should be added above here.
   BOUNDARY = 7,
+  BRAVE_DOWNLOAD_CLIENT
 };
 
 // Get a string that represents a particular client. Used in histograms and
