diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 8b228b1c338ba528080fa8a58499e74202846244..3eb1ccfa2d76cc1f93cebdf4ecfce0c9d71cb2e7 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1248,6 +1248,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
