diff --git a/components/component_updater/component_updater_service.h b/components/component_updater/component_updater_service.h
index 2c9bf5a3d16f1a1ecabee9c95402ceaf41968d8e..207b2de127e740aae16422df7e3cf5e96b12dfca 100644
--- a/components/component_updater/component_updater_service.h
+++ b/components/component_updater/component_updater_service.h
@@ -163,6 +163,7 @@ class OnDemandUpdater {
   enum class Priority { BACKGROUND = 0, FOREGROUND = 1 };
 
   virtual ~OnDemandUpdater() = default;
+  BRAVE_COMPONENT_UPDATER_SERVICE_H_
 
  private:
   friend class OnDemandTester;
