diff --git a/components/sync/service/sync_user_settings_impl.cc b/components/sync/service/sync_user_settings_impl.cc
index 554ae78028cdcfe7784a644ad9c53455fcacb5d2..1b11d559f540d5959c43151f65287593ee5bd2da 100644
--- a/components/sync/service/sync_user_settings_impl.cc
+++ b/components/sync/service/sync_user_settings_impl.cc
@@ -415,6 +415,7 @@ DataTypeSet SyncUserSettingsImpl::GetPreferredDataTypes() const {
     types.Remove(THEMES_IOS);
     types.Remove(THEMES_ANDROID);
   }
+  BraveAdjustPreferredDataTypes(&types);
   return types;
 }
 
