diff --git a/components/omnibox/resources/BUILD.gn b/components/omnibox/resources/BUILD.gn
index 8cb7892830cdb9a86a0aa8c026f7946399099080..042efdc3ddff7b564843d50f8c2b526e09e9fed0 100644
--- a/components/omnibox/resources/BUILD.gn
+++ b/components/omnibox/resources/BUILD.gn
@@ -14,6 +14,7 @@ grit_strings("omnibox_pedal_synonyms") {
         "ur",
         "nb",
       ]
+  _locales -= brave_extended_locales_exclusions
   if (is_android) {
     _locales -= [ "cy" ]
   }
