diff --git a/components/permissions/permission_request_manager.cc b/components/permissions/permission_request_manager.cc
index 37aa2d24e7426d7b67c252c173ba5ad133091d15..aa23f3d4b8749d05857209205fe3e72e81190a11 100644
--- a/components/permissions/permission_request_manager.cc
+++ b/components/permissions/permission_request_manager.cc
@@ -581,6 +581,7 @@ GURL PermissionRequestManager::GetRequestingOrigin() const {
   GURL origin = requests_.front()->requesting_origin();
   if (DCHECK_IS_ON()) {
     for (permissions::PermissionRequest* request : requests_) {
+      BRAVE_PERMISSION_REQUEST_MANAGER_GET_REQUESTING_ORIGIN
       DCHECK_EQ(origin, request->requesting_origin());
     }
   }
