diff --git a/components/search_engines/default_search_manager.cc b/components/search_engines/default_search_manager.cc
index bc76087fd6fac263e2b254f4ef0f0176a705d512..78adcc92832a2b69df00a094620196a41a474204 100644
--- a/components/search_engines/default_search_manager.cc
+++ b/components/search_engines/default_search_manager.cc
@@ -353,6 +353,7 @@ void DefaultSearchManager::LoadDefaultSearchEngineFromPrefs() {
   // Check if default search preference is overridden by extension.
   if (pref->IsExtensionControlled()) {
     extension_default_search_ = std::move(turl_data);
+  LOADDEFAULTSEARCHENGINEFROMPREFS_BRAVE
   } else {
     prefs_default_search_.SetAndReconcile(std::move(turl_data));
   }
