diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index e5879d2f3c597f0c0949a282bcabb34c92e20208..b294621b52fb6334eaeca735d8d3cdf0c6ddc255 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1277,6 +1277,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
