diff --git a/components/password_manager/core/browser/ui/BUILD.gn b/components/password_manager/core/browser/ui/BUILD.gn
index dbbaeb737b3ad3745b1f6e3b8d3035a6e9cb0c79..873e13bcfc08a81170b2392cbe8e6f8745535462 100644
--- a/components/password_manager/core/browser/ui/BUILD.gn
+++ b/components/password_manager/core/browser/ui/BUILD.gn
@@ -70,7 +70,7 @@ source_set("ui") {
     "//url",
   ]
 
-  if (!is_android) {
+  if (true) {
     sources += [
       "reuse_check_utility.cc",
       "reuse_check_utility.h",
