diff --git a/components/autofill/core/common/autofill_features.cc b/components/autofill/core/common/autofill_features.cc
index 15fd8bd7b0b4fcbc52b8429402755dfc7ca4c11f..77f0bd64261f66e7b22824fa520c08ba06fd1500 100644
--- a/components/autofill/core/common/autofill_features.cc
+++ b/components/autofill/core/common/autofill_features.cc
@@ -275,7 +275,7 @@ BASE_FEATURE_PARAM(
 
 // If enabled, the client may trigger the server model for AutofillAI type
 // predictions.
-DEFINE_FEATURE_WITH_MOBILE_COUNTRY_RESTRICTION(kAutofillAiServerModel);
+BASE_OVERRIDDEN_FEATURE(kAutofillAiServerModel, base::FEATURE_DISABLED_BY_DEFAULT);
 
 // The maximum duration for which an AutofillAI server model response is kept in
 // the local cache. NOTE: It is advisable to choose a value that is at least as
@@ -415,7 +415,7 @@ DEFINE_FEATURE_WITH_MOBILE_COUNTRY_RESTRICTION(
     kAutofillAiWalletVehicleRegistration);
 
 // Enables the second iteration AutofillAI.
-DEFINE_FEATURE_WITH_MOBILE_COUNTRY_RESTRICTION(kAutofillAiWithDataSchema);
+BASE_OVERRIDDEN_FEATURE(kAutofillAiWithDataSchema, base::FEATURE_DISABLED_BY_DEFAULT);
 
 // When enabled, autofill will fill not skip filling fields that had an initial
 // value which was modified.
