diff --git a/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java b/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java
index dc7fbc009009996876b1019a9709dcc8aa630fd6..41448769c93af485629a02b45c42106c4cf27745 100644
--- a/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java
+++ b/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java
@@ -26,7 +26,7 @@ import java.util.zip.GZIPOutputStream;
  */
 public class MinidumpUploader {
     /* package */
-    static final String CRASH_URL_STRING = "https://clients2.google.com/cr/report";
+    static final String CRASH_URL_STRING = "https://cr.brave.com";
     /* package */
     static final String CONTENT_TYPE_TMPL = "multipart/form-data; boundary=%s";
 
