diff --git a/components/bookmarks/browser/bookmark_model.h b/components/bookmarks/browser/bookmark_model.h
index ef48bc6ae64b90caf5ce1c48363cd20090997404..97c3c86c89dd18744d21357d9375e95cb49a9709 100644
--- a/components/bookmarks/browser/bookmark_model.h
+++ b/components/bookmarks/browser/bookmark_model.h
@@ -323,6 +323,7 @@ class BookmarkModel : public BookmarkUndoProvider,
     return weak_factory_.GetWeakPtr();
   }
 
+  BRAVE_BOOKMARK_MODEL_H
  private:
   friend class BookmarkCodecTest;
   friend class BookmarkModelFaviconTest;
