diff --git a/components/sync/service/sync_user_settings_impl.cc b/components/sync/service/sync_user_settings_impl.cc
index 634bef915355eece0c48e9642f911aeb9e245bcc..885bc889861276af25c9d3cfcf52cdf256e65b78 100644
--- a/components/sync/service/sync_user_settings_impl.cc
+++ b/components/sync/service/sync_user_settings_impl.cc
@@ -394,6 +394,7 @@ DataTypeSet SyncUserSettingsImpl::GetPreferredDataTypes() const {
   if (prefs_->IsLocalSyncEnabled()) {
     types.RetainAll(LocalSyncSupportedTypes());
   }
+  BraveAdjustPreferredDataTypes(&types, prefs_->IsLocalSyncEnabled());
   return types;
 }
 
