diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index f25f8110a3c22ad4786382d83af0a8d06eb3d24a..a9ccd94a13bf463d9c12e28cb38e9b3fcf64968e 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -153,6 +153,7 @@ android_library("java") {
     "//third_party/jni_zero:jni_zero_java",
     "//ui/android:ui_java",
   ]
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
