diff --git a/components/sync/driver/BUILD.gn b/components/sync/driver/BUILD.gn
index bdafcc7f8f686664ca6bf7db7ac5145fae8465e5..b729afb29e36873138ff4b0f14206bf90a7a2bb4 100644
--- a/components/sync/driver/BUILD.gn
+++ b/components/sync/driver/BUILD.gn
@@ -107,4 +107,5 @@ static_library("driver") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
