diff --git a/components/browsing_data/core/browsing_data_policies_utils.cc b/components/browsing_data/core/browsing_data_policies_utils.cc
index 200ef0ccf42763ad89cc052c7957ae3bfd19fab1..48a89503286fa357e52ec001526de9d40fc3f788 100644
--- a/components/browsing_data/core/browsing_data_policies_utils.cc
+++ b/components/browsing_data/core/browsing_data_policies_utils.cc
@@ -65,7 +65,7 @@ void AppendSyncTypesIfRequired(const base::Value& browsing_data_type,
   // When a new sync type or browsing data type is introduced in the code,
   // kDataToSyncTypesMap should be updated if needed to ensure that browsing
   // data that can be cleared by policy is not already synced across devices.
-  static_assert(static_cast<int>(syncer::UserSelectableType::kLastType) == 13,
+  static_assert(static_cast<int>(syncer::UserSelectableType::kLastType) == 14,
                 "It looks like a sync type was added or removed. Please update "
                 "`kDataToSyncTypesMap` value maps above if it affects any of "
                 "the browsing data types.");
