diff --git a/components/sync/engine_impl/syncer_proto_util.cc b/components/sync/engine_impl/syncer_proto_util.cc
index 6706d0225207d266b1ed74cc712462052b3180a6..a58ea213c65714d4841d656e85d320c856ebc266 100644
--- a/components/sync/engine_impl/syncer_proto_util.cc
+++ b/components/sync/engine_impl/syncer_proto_util.cc
@@ -434,7 +434,8 @@ void SyncerProtoUtil::AddRequiredFieldsToClientToServerMessage(
   DCHECK(msg);
   SetProtocolVersion(msg);
   AddRequestBirthday(cycle->context()->directory(), msg);
-  DCHECK(msg->has_store_birthday() || !IsBirthdayRequired(*msg));
+  // TODO(darkdh): deal with a6aca7bcee196866fc53d487f0025c502df7611a
+  // DCHECK(msg->has_store_birthday() || !IsBirthdayRequired(*msg));
   AddBagOfChips(cycle->context()->directory(), msg);
   msg->set_api_key(google_apis::GetAPIKey());
   msg->mutable_client_status()->CopyFrom(cycle->context()->client_status());
