diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index a3b39667cf27b8ba9ac5fd35444570036abd3bbb..d7433958e1de7c85711c9489e3d0747e86f3798a 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -262,6 +262,7 @@ class SyncServiceImpl : public SyncService,
   void ReportDataTypeErrorForTest(ModelType type);
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
