diff --git a/components/signin/public/identity_manager/BUILD.gn b/components/signin/public/identity_manager/BUILD.gn
index 6aaf413ab7b92eb27ad1bd8c4e073297e343857d..3085013d1beefb21c2d0adbced5b12744ba664b2 100644
--- a/components/signin/public/identity_manager/BUILD.gn
+++ b/components/signin/public/identity_manager/BUILD.gn
@@ -5,6 +5,7 @@
 if (is_android) {
   import("//build/config/android/rules.gni")
 }
+import("//brave/components/signin/public/identity_manager/sources.gni")
 
 source_set("identity_manager") {
   sources = [
@@ -75,6 +76,7 @@ source_set("identity_manager") {
     # together and include headers from each other.
     "//components/signin/internal/identity_manager",
   ]
+  sources += brave_components_signin_public_identity_manager_sources
 }
 
 if (is_android) {
