diff --git a/components/private_insights/private_insights_features.cc b/components/private_insights/private_insights_features.cc
index 2dfa7a13dfb7e4b0db4ffb30a678ed77c236ba6e..f053d46df48ee4bb6c99fe394763dbad71be42ae 100644
--- a/components/private_insights/private_insights_features.cc
+++ b/components/private_insights/private_insights_features.cc
@@ -8,7 +8,7 @@
 
 namespace private_insights {
 
-BASE_FEATURE(kPrivateInsightsFeature, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kPrivateInsightsFeature, base::FEATURE_DISABLED_BY_DEFAULT);
 
 const base::FeatureParam<base::TimeDelta> kPrivateInsightsUploadInterval{
     &kPrivateInsightsFeature, "upload_interval", base::Minutes(30)};
