diff --git a/components/content_settings/core/browser/BUILD.gn b/components/content_settings/core/browser/BUILD.gn
index ef410e799aff8363450ca2122f282a68309b8ce7..39cc5f5cf9c44d3ce54db99e9a88408c83e4efbb 100644
--- a/components/content_settings/core/browser/BUILD.gn
+++ b/components/content_settings/core/browser/BUILD.gn
@@ -62,6 +62,7 @@ static_library("browser") {
     "//services/preferences/public/cpp",
     "//url",
   ]
+  deps += [ "//brave/components/content_settings/core/browser" ]
 
   if (is_android) {
     deps += [ "//media" ]
