diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index 0f1bba5b240500bf06383ee608db09a8acb874b8..138192c70fbb60880ea5dc0676a6175c7df3df8f 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -512,7 +512,7 @@ CookieSettingsBase::DecideAccess(const GURL& url,
   if (!IsAllowed(setting)) {
     return BlockAllCookies{};
   }
-
+  BRAVE_COOKIE_SETTINGS_BASE_DECIDE_ACCESS
   if (!ShouldBlockThirdPartyCookies() &&
       !Are3pcsForceDisabledByOverride(overrides)) {
     return AllowAllCookies{
@@ -610,6 +610,7 @@ CookieSettingsBase::GetCookieSettingInternal(
   const ContentSetting cookie_setting = GetContentSetting(
       url, first_party_url, ContentSettingsType::COOKIES, &setting_info);
 
+  BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL
   const bool is_explicit_setting =
       !setting_info.primary_pattern.MatchesAllHosts() ||
       !setting_info.secondary_pattern.MatchesAllHosts();
