diff --git a/components/os_crypt/BUILD.gn b/components/os_crypt/BUILD.gn
index 8448ff2d912ed8664ba1117397a2407c08e9a578..1d578f479085b4e06ab814f734e47bb84df7bbd0 100644
--- a/components/os_crypt/BUILD.gn
+++ b/components/os_crypt/BUILD.gn
@@ -115,6 +115,7 @@ component("os_crypt") {
       defines += [ "USE_KWALLET" ]
     }
   }
+  import("//brave/components/os_crypt/sources.gni") deps += brave_components_os_crypt_deps
 }
 
 static_library("test_support") {
