diff --git a/components/rappor/log_uploader.cc b/components/rappor/log_uploader.cc
index cee35a966d821f98453ec185a8209b062ad51593..0a7a2c2656fea1068c7d6196b04aa6b8dedd94e6 100644
--- a/components/rappor/log_uploader.cc
+++ b/components/rappor/log_uploader.cc
@@ -69,6 +69,7 @@ LogUploader::LogUploader(
 LogUploader::~LogUploader() {}
 
 void LogUploader::Start() {
+  return; // feature disabled in Brave
   is_running_ = true;
   StartScheduledUpload();
 }
