diff --git a/components/download/internal/common/download_item_impl.cc b/components/download/internal/common/download_item_impl.cc
index 363e54e74ea574638f4aa3294733bd08473116dc..83bf2af09a579cac410846624e0fae54ea3d533e 100644
--- a/components/download/internal/common/download_item_impl.cc
+++ b/components/download/internal/common/download_item_impl.cc
@@ -1824,8 +1824,8 @@ void DownloadItemImpl::OnDownloadCompleting() {
                      base::Unretained(download_file_.get()),
                      GetTargetFilePath(),
                      delegate_->GetApplicationClientIdForFileScanning(),
-                     delegate_->IsOffTheRecord() ? GURL() : GetURL(),
-                     delegate_->IsOffTheRecord() ? GURL() : GetReferrerUrl(),
+                     GURL(),  // Never leak download URLs in metadata.
+                     GURL(),  // See brave-browser#2766.
                      std::move(quarantine), std::move(callback)));
 }
 
