diff --git a/components/sync/driver/profile_sync_service.h b/components/sync/driver/profile_sync_service.h
index b71429377b2068f103656834068a079fded3822d..2072bb06f5cf782664d31a610b901c5ee57a6156 100644
--- a/components/sync/driver/profile_sync_service.h
+++ b/components/sync/driver/profile_sync_service.h
@@ -521,6 +521,7 @@ class ProfileSyncService : public SyncService,
 
   base::WeakPtrFactory<ProfileSyncService> weak_factory_{this};
 
+  BRAVE_PROFILE_SYNC_SERVICE_H
   DISALLOW_COPY_AND_ASSIGN(ProfileSyncService);
 };
 
