diff --git a/components/cast_channel/cast_transport.cc b/components/cast_channel/cast_transport.cc
index 340318f6bca1ff926a691bdac11d31a7a3789457..4c088f7022eb400422cc682630f9a711288a7775 100644
--- a/components/cast_channel/cast_transport.cc
+++ b/components/cast_channel/cast_transport.cc
@@ -54,7 +54,7 @@ CastTransportImpl::CastTransportImpl(Channel* channel,
       channel_id_(channel_id),
       ip_endpoint_(ip_endpoint),
       logger_(logger) {
-  DCHECK(socket);
+  DCHECK(channel);
 
   // Buffer is reused across messages to minimize unnecessary buffer
   // [re]allocations.
