diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 1fe7f40cfccf93526e2133e2e87eaa8d16ad5935..2061e660db8fe293ad1158ef68f687f48dd68ac3 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -523,6 +523,9 @@ static_library("browser") {
     ]
     deps += [ "//content/public/common" ]
   }
+  sources += brave_components_omnibox_browser_sources
+  public_deps += brave_components_omnibox_browser_public_deps
+  deps += brave_components_omnibox_browser_deps
 }
 
 source_set("pref_names") {
@@ -1122,6 +1125,7 @@ fuzzer_test("search_suggestion_parser_fuzzer") {
 }
 
 mojom("mojo_bindings") {
+  generate_legacy_js_bindings = true
   sources = [ "omnibox.mojom" ]
   webui_module_path = "chrome://resources/mojo/components/omnibox/browser"
 
@@ -1143,6 +1147,7 @@ mojom("mojo_bindings") {
 
 if (!is_ios) {
   mojom("fusebox_action_mojo_bindings") {
+    generate_legacy_js_bindings = true
     sources = [ "fusebox_action.mojom" ]
     webui_module_path = "chrome://resources/mojo/components/omnibox/browser"
     public_deps = [ "//components/omnibox/composebox:mojo_bindings" ]
