diff --git a/components/content_settings/core/common/cookie_settings_base.h b/components/content_settings/core/common/cookie_settings_base.h
index bf48b04a598747db19adc6b899b394852dc1e340..76ded883b974f8582966ff9c8253423d97e4a03b 100644
--- a/components/content_settings/core/common/cookie_settings_base.h
+++ b/components/content_settings/core/common/cookie_settings_base.h
@@ -77,7 +77,7 @@ class CookieSettingsBase {
   CookieSettingsBase(const CookieSettingsBase&) = delete;
   CookieSettingsBase& operator=(const CookieSettingsBase&) = delete;
 
-  virtual ~CookieSettingsBase() = default;
+  virtual ~CookieSettingsBase();
 
   // Enum for measuring the mechanism for re-enabling third-party cookies when
   // applying 3PCD experiment. These values are persisted to logs. Entries
