diff --git a/components/sync_device_info/device_info.h b/components/sync_device_info/device_info.h
index 058ded9abc1aea8554e3d8aae2be93609fbed2da..8725061fb97cb0933561248e7f34017a153d47c3 100644
--- a/components/sync_device_info/device_info.h
+++ b/components/sync_device_info/device_info.h
@@ -221,7 +221,9 @@ class DeviceInfo {
              GlicExperimentalTriggeringState glic_experimental_triggering_state,
              std::optional<int> glic_experimental_triggering_version,
              std::optional<std::string> android_os_build_fingerprint_prefix,
-             std::optional<PersonalContextInfo> personal_context_info);
+             std::optional<PersonalContextInfo> personal_context_info,
+             SelfDeleteSupport self_delete_support =
+                 SelfDeleteSupport::kNotSupported);
 
   DeviceInfo& operator=(const DeviceInfo&) = delete;
 
@@ -363,6 +365,22 @@ class DeviceInfo {
   // Glic. Pass std::nullopt if unavailable.
   void set_glic_experimental_triggering_version(std::optional<int> version);
 
+  DeviceInfo(DeviceInfo&&);
+
+  SelfDeleteSupport self_delete_support() const {
+    return self_delete_support_;
+  }
+  void set_self_delete_support(SelfDeleteSupport self_delete_support) {
+    self_delete_support_ = self_delete_support;
+  }
+  // Gets the OS in string form.
+  std::string GetOSString() const;
+  // Gets the device type in string form.
+  std::string GetDeviceTypeString() const;
+  // Converts the DeviceInfo values to a JS friendly DictionaryValue,
+  // which extension APIs can expose to third party apps.
+  base::DictValue ToValue() const;
+
  private:
   // Used by DeepCopyForTesting().
   DeviceInfo(const DeviceInfo& other);
@@ -444,6 +462,7 @@ class DeviceInfo {
   // |IsStoredLocalDeviceInfoStillAccurate| in device_info_sync_bridge.cc or
   // else changes in that member might not trigger uploads of updated
   // DeviceInfos.
+  SelfDeleteSupport self_delete_support_;
 };
 
 }  // namespace syncer
