diff --git a/components/translate/core/browser/translate_download_manager.h b/components/translate/core/browser/translate_download_manager.h
index f16c4fe95aa5ceaeaeb9f2ef80da6897e5c3495b..39384b15ed7a847d5a2efee6ccbf01011254063d 100644
--- a/components/translate/core/browser/translate_download_manager.h
+++ b/components/translate/core/browser/translate_download_manager.h
@@ -94,6 +94,7 @@ class TranslateDownloadManager {
   void SetTranslateScriptExpirationDelay(int delay_ms);
 
  private:
+  friend class BraveTranslateDownloadManager;
   friend struct base::DefaultSingletonTraits<TranslateDownloadManager>;
   TranslateDownloadManager();
   virtual ~TranslateDownloadManager();
