diff --git a/components/search_engines/default_search_manager.cc b/components/search_engines/default_search_manager.cc
index ddac938ffcf08bbd1ddbc56d9ab024b88e105eef..96fe9676371ced61e3d85afbc65ef69a0211ea15 100644
--- a/components/search_engines/default_search_manager.cc
+++ b/components/search_engines/default_search_manager.cc
@@ -305,6 +305,7 @@ void DefaultSearchManager::LoadDefaultSearchEngineFromPrefs() {
   // Check if default search preference is overriden by extension.
   if (pref->IsExtensionControlled()) {
     extension_default_search_ = std::move(turl_data);
+  LOADDEFAULTSEARCHENGINEFROMPREFS_BRAVE
   } else {
     prefs_default_search_ = std::move(turl_data);
     MergePrefsDataWithPrepopulated();
