diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index a33f8f6cb41cbe0263660d3d04d8a9e8305086e4..22b632567143761fc1a9b9f7d1eae8fe24cbe2f2 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -960,6 +960,7 @@ void PageInfo::PresentSitePermissions() {
       }
     }
 
+    BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
     if (ShouldShowPermission(
             permission_info, site_url_, content_settings, web_contents(),
             HasContentSettingChangedViaPageInfo(permission_info.type),
