diff --git a/components/crx_file/crx_creator.cc b/components/crx_file/crx_creator.cc
index 2f7b3f47232d16472c581beec1d67ab52afe4c83..3a8c5952c348f4e81b609d83d8e44153462c4108 100644
--- a/components/crx_file/crx_creator.cc
+++ b/components/crx_file/crx_creator.cc
@@ -13,6 +13,9 @@
 #include "crypto/sha2.h"
 #include "crypto/signature_creator.h"
 
+#include "base/files/file_util.h"
+#include "base/logging.h"
+
 namespace crx_file {
 
 namespace {
@@ -68,7 +71,7 @@ CreatorResult SignArchiveAndCreateHeader(const base::FilePath& output_path,
 
   // Assemble SignedData section.
   SignedData signed_header_data;
-  signed_header_data.set_crx_id(GetCrxId(public_key_str));
+  signed_header_data.set_crx_id(BraveGetCrxId(public_key_str, header));
   const std::string signed_header_data_str =
       signed_header_data.SerializeAsString();
   const int signed_header_size = signed_header_data_str.size();
