diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index 7841b7a574a773ca43623ffc5d658da56e7c01b8..03b050fdb1fc496c9548733e5217327b16ca81cc 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -291,6 +291,7 @@ void PermissionManager::RequestPermissionsFromCurrentDocument(
     base::OnceCallback<void(const std::vector<PermissionStatus>&)>
         permission_status_callback) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
+  BRAVE_REQUEST_PERMISSION_FROM_CURRENT_DOCUMENT
   RequestPermissionsInternal(render_frame_host, request_description,
                              std::move(permission_status_callback));
 }
@@ -342,6 +343,7 @@ PermissionManager::GetPermissionResultForCurrentDocument(
       PermissionUtil::PermissionTypeToContentSettingType(permission);
 
   const GURL requesting_origin =
+      BRAVE_GET_PERMISSION_RESULT_FOR_CURRENT_DOCUMENT
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   const GURL embedding_origin =
       GetEmbeddingOrigin(render_frame_host, requesting_origin);
