diff --git a/components/sync/service/sync_stopped_reporter.h b/components/sync/service/sync_stopped_reporter.h
index 489a9fba1fe9d6a2f0e69f5cebb40616e698ff68..a693988ee8fabaec4338a36355605437f432c82b 100644
--- a/components/sync/service/sync_stopped_reporter.h
+++ b/components/sync/service/sync_stopped_reporter.h
@@ -30,7 +30,7 @@ class SyncStoppedReporter {
   SyncStoppedReporter(const SyncStoppedReporter&) = delete;
   SyncStoppedReporter& operator=(const SyncStoppedReporter&) = delete;
 
-  ~SyncStoppedReporter();
+  virtual ~SyncStoppedReporter();
 
   // Inform the sync server that sync was stopped on this device.
   // `access_token`, `cache_guid`, and `birthday` must not be empty.
