diff --git a/components/policy/tools/generate_policy_source.py b/components/policy/tools/generate_policy_source.py
index f582b5bf4c4cd6163aae55b477f924c5d1c85e68..b326e157aeeb07033c375ee3a565726c3048bc04 100755
--- a/components/policy/tools/generate_policy_source.py
+++ b/components/policy/tools/generate_policy_source.py
@@ -52,6 +52,7 @@ PLATFORM_STRINGS = {
     'chrome.win7': ['win']
 }
 
+from policy_source_helper import AddBravePolicies, CHROMIUM_POLICY_KEY
 
 class PolicyDetails:
   """Parses a policy template and caches all its details."""
@@ -363,6 +364,7 @@ def main():
     chrome_major_version = ParseVersionFile(version_path)
 
   template_file_contents = _LoadJSONFile(template_file_name)
+  AddBravePolicies(template_file_contents)
   risk_tags = RiskTags(template_file_contents)
   policy_details = [
       PolicyDetails(policy, chrome_major_version, target_platform,
