diff --git a/components/content_settings/core/browser/cookie_settings.h b/components/content_settings/core/browser/cookie_settings.h
index 7a15d8dfe2e0807ff1180e8e4b9c2a72134a5081..b11eaa1794e612be54c6e11cca2904ce4bcd2d93 100644
--- a/components/content_settings/core/browser/cookie_settings.h
+++ b/components/content_settings/core/browser/cookie_settings.h
@@ -99,6 +99,7 @@ class CookieSettings : public RefcountedKeyedService {
   void ShutdownOnUIThread() override;
 
   // A helper for applying third party cookie blocking rules.
+  virtual
   void GetCookieSetting(const GURL& url,
                         const GURL& first_party_url,
                         content_settings::SettingSource* source,
@@ -107,6 +108,7 @@ class CookieSettings : public RefcountedKeyedService {
   static void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry);
 
  private:
+  friend class BraveCookieSettings;
   ~CookieSettings() override;
 
   void OnBlockThirdPartyCookiesChanged();
