diff --git a/components/keyed_service/content/browser_context_keyed_service_factory.h b/components/keyed_service/content/browser_context_keyed_service_factory.h
index dda3f0c6163114dab45f1ece5ab5ec74ed206b41..5d0e3e030e94a9caa862e57a0c0217633cdee322 100644
--- a/components/keyed_service/content/browser_context_keyed_service_factory.h
+++ b/components/keyed_service/content/browser_context_keyed_service_factory.h
@@ -164,6 +164,7 @@ class KEYED_SERVICE_EXPORT BrowserContextKeyedServiceFactory
   // and the default implementation removes it from |mapping_| and deletes
   // the pointer.
   virtual void BrowserContextShutdown(content::BrowserContext* context);
+  virtual
   void BrowserContextDestroyed(content::BrowserContext* context);
 
  private:
