diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index 6655c7f06ee208eb542f4dc0943909d6296d31d3..a6b01e88de2dd27070ac562a6a0b0905157f617f 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -142,6 +142,7 @@ android_library("java") {
     "//ui/android:ui_java",
   ]
 
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
