diff --git a/components/sync/engine/sync_manager.h b/components/sync/engine/sync_manager.h
index fd197f7f2b1df63a8f4253348a5bf83d5396efa5..62bdaf719e450da862387d674ac7b0c41a7a2dec 100644
--- a/components/sync/engine/sync_manager.h
+++ b/components/sync/engine/sync_manager.h
@@ -255,6 +255,7 @@ class SyncManager {
     std::string cache_guid;
     std::string birthday;
     std::string bag_of_chips;
+    BRAVE_SYNC_MANAGER_INIT_ARGS_H
   };
 
   // The state of sync the feature. If the user turned on sync explicitly, it
