diff --git a/components/sync/base/features.h b/components/sync/base/features.h
index db8a212966289e23a12997f35ea7577f7ebccc80..514420887c000cca496d985047ab48bf43b5fb6e 100644
--- a/components/sync/base/features.h
+++ b/components/sync/base/features.h
@@ -117,7 +117,7 @@ DECLARE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuable);
 #endif
 
 // Enables syncing of usage metadata from Google Wallet passes.
-DECLARE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuableMetadata);
+BASE_DECLARE_FEATURE(kSyncAutofillValuableMetadata);
 
 // Enables syncing of flight reservations coming from Google Wallet.
 DECLARE_SYNC_AUTOFILL_AI_FEATURE(kSyncWalletFlightReservations);
