diff --git a/components/sync/engine/model_type_worker.h b/components/sync/engine/model_type_worker.h
index 256aa18ff886241d43c94e4cc67bb53520f030e0..325393f4fb712502d4b3e47e864253e7408175ba 100644
--- a/components/sync/engine/model_type_worker.h
+++ b/components/sync/engine/model_type_worker.h
@@ -155,6 +155,7 @@ class ModelTypeWorker : public UpdateHandler,
     min_gu_responses_to_ignore_key_ = min_gu_responses_to_ignore_key;
   }
 
+ BRAVE_MODEL_TYPE_WORKER_H_
  private:
   struct UnknownEncryptionKeyInfo {
     // Not increased if the cryptographer knows it's in a pending state
