diff --git a/components/contextual_tasks/public/features.cc b/components/contextual_tasks/public/features.cc
index ff8ee24d4b475bce53d535d1053da31cb84fc9d0..add1151abcba16d62022b3205a859471712b4f9a 100644
--- a/components/contextual_tasks/public/features.cc
+++ b/components/contextual_tasks/public/features.cc
@@ -35,7 +35,7 @@ std::optional<bool> g_sticky_conversation_enabled_override;
 namespace contextual_tasks {
 
 // Enables the contextual tasks side panel while browsing.
-BASE_FEATURE(kContextualTasks, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kContextualTasks, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kContextualTasksPrivateApiNoAnimation,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
@@ -181,7 +181,7 @@ BASE_FEATURE(kContextualTasksOverrideShowBottomSheetOnLargeScreen,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Enables prefetching of cookies for contextual tasks.
-BASE_FEATURE(kContextualTasksCookiePrefetch, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kContextualTasksCookiePrefetch, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kAimTriggeredThreadLinks, base::FEATURE_ENABLED_BY_DEFAULT);
 
