diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index f7f10b8c0c745c0ca17429f442887914a61e40a5..254af9670f838539d583a15ce7298ea95cdd60db 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -49,6 +49,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 typedef int64_t VisitID;
