diff --git a/components/rappor/log_uploader.cc b/components/rappor/log_uploader.cc
index 20f5d86a80038be882aaec084c22084916dba0f8..c195e17d397517080ba709ba66aa3db054ca1925 100644
--- a/components/rappor/log_uploader.cc
+++ b/components/rappor/log_uploader.cc
@@ -68,6 +68,7 @@ LogUploader::LogUploader(
 LogUploader::~LogUploader() {}
 
 void LogUploader::Start() {
+  return; // feature disabled in Brave
   is_running_ = true;
   StartScheduledUpload();
 }
