diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index c33148686e11c59232e6493e15a520e16ee51134..6e5b7dacc7f07b1cf7ed2f1b439099fb5c7f15c8 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -54,6 +54,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 typedef int64_t VisitID;
