diff --git a/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java b/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java
index 5f30ab591586f986da1f26359d9b5cf7bb763772..a430483effebcee3887d0957f0eb81e871855daa 100644
--- a/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java
+++ b/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java
@@ -26,7 +26,7 @@ import java.util.zip.GZIPOutputStream;
  */
 public class MinidumpUploader {
     /* package */
-    static final String CRASH_URL_STRING = "https://clients2.google.com/cr/report";
+    static final String CRASH_URL_STRING = "https://cr.brave.com";
     /* package */
     static final String CONTENT_TYPE_TMPL = "multipart/form-data; boundary=%s";
 
