diff --git a/components/sessions/content/content_serialized_navigation_builder.cc b/components/sessions/content/content_serialized_navigation_builder.cc
index d6ec93a25a0ff6f85d394f1eda624be458c5dd26..cd429c1ee9a459d0bbea0754958d37b74cbd80cd 100644
--- a/components/sessions/content/content_serialized_navigation_builder.cc
+++ b/components/sessions/content/content_serialized_navigation_builder.cc
@@ -65,6 +65,7 @@ ContentSerializedNavigationBuilder::FromNavigationEntry(
       navigation.extended_info_map_[handler_entry.first] = value;
   }
 
+  MaybeSaveStoragePartitionKey(entry, &navigation);
   return navigation;
 }
 
@@ -208,6 +209,7 @@ ContentSerializedNavigationBuilder::ToNavigationEntry(
   DCHECK_EQ(SerializedNavigationEntry::STATE_INVALID,
             navigation->blocked_state_);
 
+  MaybeRestoreStoragePartitionKey(navigation, entry.get());
   return entry;
 }
 
