diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index cb4bba853cbdd7bcecd2bcca55cce65f55b50d84..8d2be8f48c9cfe57a82b057fe19495fd3bf19ac8 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -124,6 +124,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kTopLevelStorageAccess:
       return RequestTypeForUma::PERMISSION_TOP_LEVEL_STORAGE_ACCESS;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
