diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index 1f192146092457ae47797cb78497ed681c5c1003..dc53f11d6c580c729d1b1774a27020cc936cab58 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -285,6 +285,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
