diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index 49c341978ed7c67dd82c8dd9bc820539307a05e1..179e12688d87569ecb4247b33d853ec6ee59e81b 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -333,6 +333,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
