diff --git a/components/search_engines/default_search_manager.cc b/components/search_engines/default_search_manager.cc
index 9a7f8b6047a9099258fbc088aa716854489f27f7..27313cd3af2da017cf99f5755cd000c0ed337aa8 100644
--- a/components/search_engines/default_search_manager.cc
+++ b/components/search_engines/default_search_manager.cc
@@ -307,6 +307,7 @@ void DefaultSearchManager::LoadDefaultSearchEngineFromPrefs() {
   // Check if default search preference is overriden by extension.
   if (pref->IsExtensionControlled()) {
     extension_default_search_ = std::move(turl_data);
+  LOADDEFAULTSEARCHENGINEFROMPREFS_BRAVE
   } else {
     prefs_default_search_ = std::move(turl_data);
     MergePrefsDataWithPrepopulated();
