diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index e4844f6153bab368e843b7bfd2ab3aeb51496f39..ad305b89942b3d262c90d11d84c9f388e92f7023 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -19,6 +19,7 @@ group("sync") {
   if (is_chromeos) {
     public_deps += [ "//components/sync/chromeos" ]
   }
+  import("//brave/build/config/brave_build.gni") public_deps += brave_components_sync_deps
 }
 
 static_library("test_support") {
