diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index a1c03fa5ff6002988c85c4bd15473fa2bca41f29..5a30c77383d43369dec6845feb4bb086a6e2ceeb 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -321,6 +321,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
