diff --git a/components/sync/syncable/write_transaction.h b/components/sync/syncable/write_transaction.h
index 3d0a353ac09b1dcbfab0d5afc3345d78eb4ee8bd..eafebbf205de8691700162151ced152a4d5ba679 100644
--- a/components/sync/syncable/write_transaction.h
+++ b/components/sync/syncable/write_transaction.h
@@ -63,6 +63,7 @@ class WriteTransaction : public BaseTransaction {
     transaction_ = trans;
   }
 
+ BRAVE_SYNC_ATTACHABLE_WRITE_TRANSACTION_H
  private:
   void* operator new(size_t size);  // Transaction is meant for stack use only.
 
