diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index fcf50789293645d7ad8a4f85a1675132a4dd4a0e..397693b98c42d19365386f9cb40a8ce62d617e9b 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -142,6 +142,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kCapturedSurfaceControl:
       return RequestTypeForUma::CAPTURED_SURFACE_CONTROL;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
