diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index 958cacdd47e63f09692ad610ab2711fddeafd4d5..82e6083d952f176aa49b0892d050478bbd2ff966 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -126,6 +126,7 @@ android_library("java") {
     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelPrompt.java",
     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelSetting.java",
   ]
+  sources += brave_components_permissions_java_sources
   deps = [
     ":core_java",
     ":java_resources",
