diff --git a/components/history/core/browser/BUILD.gn b/components/history/core/browser/BUILD.gn
index 5932063196fc6716b51ab68bca6db9e06e612702..d562fce34f537db4a57a90c3e355a2d3e9919543 100644
--- a/components/history/core/browser/BUILD.gn
+++ b/components/history/core/browser/BUILD.gn
@@ -151,6 +151,7 @@ static_library("browser") {
       "android/visit_sql_handler.h",
     ]
   }
+  import("//brave/components/history/core/browser/sources.gni") sources += brave_components_history_core_browser_sources deps += brave_components_history_core_browser_deps
 }
 
 source_set("unit_tests") {
