diff --git a/components/history/core/browser/history_backend.cc b/components/history/core/browser/history_backend.cc
index 9a8cd38cd00831675c1b7cd2fea7ff4fd7194ac8..852f2d82eedd838f1daf742a2c3bcc84f878209f 100644
--- a/components/history/core/browser/history_backend.cc
+++ b/components/history/core/browser/history_backend.cc
@@ -1988,6 +1988,17 @@ QueryURLResult HistoryBackend::QueryURL(const GURL& url) {
   return result;
 }
 
+std::vector<URLID> HistoryBackend::QueryUrlIds(const std::vector<GURL>& urls) {
+  std::vector<URLID> result(urls.size(), 0);
+  if (!db_) {
+    return result;
+  }
+  for (size_t i = 0; i < urls.size(); ++i) {
+    result[i] = db_->GetRowForURL(urls[i], nullptr);
+  }
+  return result;
+}
+
 QueryURLAndVisitsResult HistoryBackend::QueryURLAndVisits(
     const GURL& url,
     VisitQuery404sPolicy policy_for_404s) {
