diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index c3a64b3fdbecd507994cc852cf7abdf28d4ae785..f02f40c16278248252a22d119a5bb276a0a02e85 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1400,6 +1400,9 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // second section, the default behavior used when no per-type exception applies.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  if (!delegate_->BraveShouldShowPermission(info.type)) {
+    return false;
+  }
 #if BUILDFLAG(IS_ANDROID)
   if (info.type == ContentSettingsType::NOTIFICATIONS) {
     // `is_requested` is only populated if
@@ -1636,6 +1639,9 @@ void PageInfo::PresentSitePermissions() {
 
 std::set<net::SchemefulSite> PageInfo::GetTwoSitePermissionRequesters(
     ContentSettingsType type) {
+  if (type == ContentSettingsType::STORAGE_ACCESS) {
+    return {};
+  }
   std::set<net::SchemefulSite> requesters;
   // Collect sites that have tried to request a permission.
   auto* pscs = GetPageSpecificContentSettings();
