diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index 334f527bfbefddbe19883d24d738e00a0d385d3c..03fc051ad0b2f93894827890bcf8ce3d2012fe71 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -192,6 +192,7 @@ static_library("metrics") {
   if (is_fuchsia) {
     sources += [ "drive_metrics_provider_fuchsia.cc" ]
   }
+  import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
 }
 
 if (is_android) {
