diff --git a/components/content_settings/core/common/content_settings.cc b/components/content_settings/core/common/content_settings.cc
index fcd6d94a41f75c0b7fbc6578d50bda340381b3dd..6d0f2a90388a35d38ba3a5b03988c448bae80872 100644
--- a/components/content_settings/core/common/content_settings.cc
+++ b/components/content_settings/core/common/content_settings.cc
@@ -119,6 +119,7 @@ constexpr HistogramValue kHistogramValue[] = {
     {ContentSettingsType::ANTI_ABUSE, 96},
     {ContentSettingsType::THIRD_PARTY_STORAGE_PARTITIONING, 97},
     {ContentSettingsType::HTTPS_ENFORCED, 98},
+    BRAVE_HISTOGRAM_VALUE_LIST
 };
 
 void FilterRulesForType(ContentSettingsForOneType& settings,
