diff --git a/components/visited_url_ranking/internal/BUILD.gn b/components/visited_url_ranking/internal/BUILD.gn
index abe26cb61a9ac333bd37a9ccbe0ba53e5959dbcd..4465b92927397c034868d89f5f2327178d261a4c 100644
--- a/components/visited_url_ranking/internal/BUILD.gn
+++ b/components/visited_url_ranking/internal/BUILD.gn
@@ -121,6 +121,7 @@ source_set("test_support") {
     "//testing/gmock",
     "//testing/gtest",
   ]
+  import("//brave/components/sync_device_info/sources.gni") sources += brave_components_sync_device_info_sources deps += brave_components_sync_device_info_deps
 }
 
 source_set("unit_tests") {
