diff --git a/components/crash/content/browser/crash_handler_host_linux.cc b/components/crash/content/browser/crash_handler_host_linux.cc
index 5297dd2997fb9065a9d9a77f39c8ed0bb4fde054..d883b51f17120d15e14eb9199a9fa497c70bcfd3 100644
--- a/components/crash/content/browser/crash_handler_host_linux.cc
+++ b/components/crash/content/browser/crash_handler_host_linux.cc
@@ -479,7 +479,7 @@ void CrashHandlerHostLinux::QueueCrashDumpTask(
   msg.msg_iov = &done_iov;
   msg.msg_iovlen = 1;
 
-  HANDLE_EINTR(sendmsg(signal_fd, &msg, MSG_DONTWAIT | MSG_NOSIGNAL));
+  ANALYZER_ALLOW_UNUSED(HANDLE_EINTR(sendmsg(signal_fd, &msg, MSG_DONTWAIT | MSG_NOSIGNAL)));
   close(signal_fd);
 
   uploader_thread_->task_runner()->PostTask(
