diff --git a/components/content_settings/core/common/content_settings_uma_constants.h b/components/content_settings/core/common/content_settings_uma_constants.h
index 112bcef590e4554c0159fc52df90e67d05bd0376..73d6141ca6502a920e31d4017ae8600912db1306 100644
--- a/components/content_settings/core/common/content_settings_uma_constants.h
+++ b/components/content_settings/core/common/content_settings_uma_constants.h
@@ -172,6 +172,51 @@ inline constexpr auto kContentSettingsTypeToHistogramValue =
 
         // As mentioned at the top, please don't forget to update ContentType in
         // enums.xml when you add entries here!
+        {ContentSettingsType::BRAVE_ADS, brave_value(0)},
+        {ContentSettingsType::BRAVE_COSMETIC_FILTERING, brave_value(1)},
+        {ContentSettingsType::BRAVE_TRACKERS, brave_value(2)},
+        {ContentSettingsType::BRAVE_HTTP_UPGRADABLE_RESOURCES, brave_value(3)},
+        {ContentSettingsType::BRAVE_FINGERPRINTING_V2, brave_value(4)},
+        {ContentSettingsType::BRAVE_SHIELDS, brave_value(5)},
+        {ContentSettingsType::BRAVE_REFERRERS, brave_value(6)},
+        {ContentSettingsType::BRAVE_COOKIES, brave_value(7)},
+        {ContentSettingsType::BRAVE_SPEEDREADER, brave_value(8)},
+        {ContentSettingsType::BRAVE_ETHEREUM, brave_value(9)},
+        {ContentSettingsType::BRAVE_SOLANA, brave_value(10)},
+        {ContentSettingsType::BRAVE_GOOGLE_SIGN_IN, brave_value(11)},
+        {ContentSettingsType::BRAVE_HTTPS_UPGRADE, brave_value(12)},
+        {ContentSettingsType::BRAVE_REMEMBER_1P_STORAGE, brave_value(13)},
+        {ContentSettingsType::BRAVE_OPEN_AI_CHAT, brave_value(15)},
+        {ContentSettingsType::BRAVE_AUTO_SHRED, brave_value(16)},
+        // Begin webcompat items
+        {ContentSettingsType::BRAVE_WEBCOMPAT_NONE, brave_value(50)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_AUDIO, brave_value(51)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_CANVAS, brave_value(52)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_DEVICE_MEMORY, brave_value(53)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_EVENT_SOURCE_POOL,
+         brave_value(54)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_FONT, brave_value(55)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_HARDWARE_CONCURRENCY,
+         brave_value(56)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_KEYBOARD, brave_value(57)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_LANGUAGE, brave_value(58)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_MEDIA_DEVICES, brave_value(59)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_PLUGINS, brave_value(60)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_SCREEN, brave_value(61)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_SPEECH_SYNTHESIS,
+         brave_value(62)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_USB_DEVICE_SERIAL_NUMBER,
+         brave_value(63)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_USER_AGENT, brave_value(64)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_WEBGL, brave_value(65)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_WEBGL2, brave_value(66)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_WEBGPU, brave_value(67)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_WEB_SOCKETS_POOL,
+         brave_value(68)},
+        {ContentSettingsType::BRAVE_WEBCOMPAT_ALL, brave_value(69)},
+        {ContentSettingsType::BRAVE_SHIELDS_METADATA, brave_value(70)},
+        {ContentSettingsType::BRAVE_CARDANO, brave_value(71)},
+        {ContentSettingsType::BRAVE_PSST, brave_value(72)},
     });
 // LINT.ThenChange(//tools/metrics/histograms/enums.xml:ContentType)
 
