diff --git a/components/policy/core/common/policy_types.h b/components/policy/core/common/policy_types.h
index 0755d18f0027b3085400a401a6caa6722573e5e2..8f6042db2f5bbb79c37081e4196ea0d8f7b5bfe2 100644
--- a/components/policy/core/common/policy_types.h
+++ b/components/policy/core/common/policy_types.h
@@ -42,6 +42,8 @@ enum PolicySource {
   // The policy was set because we are running in an enterprise environment.
   POLICY_SOURCE_ENTERPRISE_DEFAULT,
 
+  POLICY_SOURCE_BRAVE_ORIGIN,
+
   // The policy was set by command line flag for testing purpose.
   POLICY_SOURCE_COMMAND_LINE,
 
