diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index d530d786c0640defb9c890eb1ff15eab3b04d23a..710a99d1fd9cb60b5d9a048d652778a5042f3429 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -110,6 +110,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
