diff --git a/components/sync/engine/sync_engine_host.h b/components/sync/engine/sync_engine_host.h
index c345abb5d7da17554c4a55bcbedd716a9b9c8e0d..3049f6158b70f209871a38d30c3d94fab278a179 100644
--- a/components/sync/engine/sync_engine_host.h
+++ b/components/sync/engine/sync_engine_host.h
@@ -87,6 +87,9 @@ class SyncEngineHost {
 
   // Called when the sync cycle returns there is an user actionable error.
   virtual void OnActionableError(const SyncProtocolError& error) = 0;
+
+  virtual void OnNudgeSyncCycle(brave_sync::RecordsListPtr records_list) {}
+  virtual void OnPollSyncCycle(brave_sync::GetRecordsCallback cb, base::WaitableEvent* wevent) {}
 };
 
 }  // namespace syncer
