diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index a60bf7393480c989572ff83498db9e87530a028a..f9acec455d7ea836bc2f319ebc365bc45c57e18a 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -236,6 +236,7 @@ jumbo_static_library("browser") {
   if ((!is_android || enable_vr) && !is_ios) {
     deps += [ ":vector_icons" ]
   }
+  if (brave_chromium_build) {  deps += [ "//brave/components/omnibox/browser" ] }
 
   if (is_android) {
     deps += [ ":jni_headers" ]
