diff --git a/components/download/internal/common/download_item_impl.cc b/components/download/internal/common/download_item_impl.cc
index acc02bc1416fdd0250616274eace4daa4e3ea9f7..8bf1f4f27a5656db7a5b953e053bbd6128c5f521 100644
--- a/components/download/internal/common/download_item_impl.cc
+++ b/components/download/internal/common/download_item_impl.cc
@@ -1961,8 +1961,8 @@ void DownloadItemImpl::OnDownloadCompleting() {
                      base::Unretained(download_file_.get()),
                      GetTargetFilePath(),
                      delegate_->GetApplicationClientIdForFileScanning(),
-                     delegate_->IsOffTheRecord() ? GURL() : GetURL(),
-                     delegate_->IsOffTheRecord() ? GURL() : GetReferrerUrl(),
+                     GURL(),  // Never leak download URLs in metadata.
+                     GURL(),  // See brave-browser#2766.
                      std::move(quarantine), std::move(callback)));
 }
 
