diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index dc789bba5081c5ab7ee1af0ce05013906bc3244b..683f12b9215b7b08ac9a6d863b1ccc6f2616e3c3 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -112,6 +112,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
