diff --git a/components/skills/features.cc b/components/skills/features.cc
index b03b424873eb36945d0e9bdcd2e8b1a35bf0d6ac..c556b27c47120c4eb8e242acecd208ad729450a7 100644
--- a/components/skills/features.cc
+++ b/components/skills/features.cc
@@ -7,13 +7,8 @@
 
 namespace features {
 
-BASE_FEATURE(kSkillsEnabled,
-#if BUILDFLAG(IS_MAC) || BUILDFLAG(IS_WIN) || BUILDFLAG(IS_CHROMEOS)
-             base::FEATURE_ENABLED_BY_DEFAULT
-#else
-             base::FEATURE_DISABLED_BY_DEFAULT
-#endif
-);
+BASE_OVERRIDDEN_FEATURE(kSkillsEnabled,
+base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kSkillsMetricsProviderEnabled, base::FEATURE_ENABLED_BY_DEFAULT);
 BASE_FEATURE(kSkillsRefinementEnabled, base::FEATURE_ENABLED_BY_DEFAULT);
 BASE_FEATURE(kSkillsAutocomplete, base::FEATURE_DISABLED_BY_DEFAULT);
