diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index bd71040ecc8e76266d648e71c624758c33777f0a..17dca4dd2663c737a84f8ca3f82707ebc4849702 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -139,6 +139,7 @@ source_set("permissions") {
       "contexts/geolocation_permission_context_mac.h",
     ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
