diff --git a/components/os_crypt/key_storage_libsecret.cc b/components/os_crypt/key_storage_libsecret.cc
index 312570612ccb6ec4480a6f8d4a74accf5ba79ff8..a0538085e6cf248cf0f904ab3906f6235bb1b44b 100644
--- a/components/os_crypt/key_storage_libsecret.cc
+++ b/components/os_crypt/key_storage_libsecret.cc
@@ -17,7 +17,7 @@ namespace {
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
 const char kApplicationName[] = "chrome";
 #else
-const char kApplicationName[] = "chromium";
+BRAVE_KEY_STORAGE_LIBSECRET_K_APPLICATION_NAME
 #endif
 
 const SecretSchema kKeystoreSchemaV2 = {
@@ -88,6 +88,7 @@ KeyStorageLibsecret::AddRandomPasswordInLibsecret() {
 
 base::Optional<std::string> KeyStorageLibsecret::GetKeyImpl() {
   LibsecretAttributesBuilder attrs;
+  BRAVE_KEY_STORAGE_LIBSECRET_GET_KEY_IMPL
   attrs.Append("application", kApplicationName);
 
   LibsecretLoader::SearchHelper helper;
