diff --git a/components/password_manager/core/browser/password_store_factory_util.h b/components/password_manager/core/browser/password_store_factory_util.h
index 25290d3bb3750c36ceedc9ada5e74f28edfa352c..a642a538ecdbe790db3bcd3e013253ba19130bee 100644
--- a/components/password_manager/core/browser/password_store_factory_util.h
+++ b/components/password_manager/core/browser/password_store_factory_util.h
@@ -13,7 +13,7 @@
 #include "build/build_config.h"
 #include "components/password_manager/core/browser/password_store/password_store_interface.h"
 
-#if !BUILDFLAG(IS_ANDROID)
+#if !BUILDFLAG(IS_ANDROID) || true
 #include "components/password_manager/core/browser/password_store/login_database.h"
 #endif  // !BUILDFLAG(IS_ANDROID)
 
@@ -27,7 +27,7 @@ namespace password_manager {
 
 class CredentialsCleanerRunner;
 
-#if !BUILDFLAG(IS_ANDROID)
+#if !BUILDFLAG(IS_ANDROID) || true
 // Creates a LoginDatabase. Looks in |db_directory| for the database file.
 // Does not call LoginDatabase::Init() -- to avoid UI jank, that needs to be
 // called by PasswordStore::Init() on the background thread.
