diff --git a/components/history/core/browser/BUILD.gn b/components/history/core/browser/BUILD.gn
index 3fe01e3af095229f0e942b287b526a249bff5e6b..714137c5b5031a8e249d9197446d3c50b304d887 100644
--- a/components/history/core/browser/BUILD.gn
+++ b/components/history/core/browser/BUILD.gn
@@ -134,6 +134,7 @@ static_library("browser") {
     "//ui/gfx",
     "//url",
   ]
+  import("//brave/components/history/core/browser/sources.gni") sources += brave_components_history_core_browser_sources deps += brave_components_history_core_browser_deps
 }
 
 source_set("unit_tests") {
