diff --git a/components/strings/BUILD.gn b/components/strings/BUILD.gn
index 253da9837310e302752c079f9668f4cd818d499f..e07ecf1b313f6a27514d38a6593a2e2008686a8f 100644
--- a/components/strings/BUILD.gn
+++ b/components/strings/BUILD.gn
@@ -45,6 +45,7 @@ grit("components_strings") {
   if (is_android) {
     outputs += android_components_strings_java_resources
   }
+  deps = [ "//brave/components/resources:strings" ]
 }
 
 if (is_android) {
@@ -57,7 +58,7 @@ if (is_android) {
 }
 
 grit("components_chromium_strings") {
-  source = "../components_chromium_strings.grd"
+  source = "../components_${branding_path_component}_strings.grd"
   outputs = [ "grit/components_chromium_strings.h" ]
   foreach(locale, locales_with_fake_bidi) {
     outputs += [ "components_chromium_strings_$locale.pak" ]
