diff --git a/components/component_updater/component_updater_service.h b/components/component_updater/component_updater_service.h
index 7d4220d05f91295dc5599915f17e401803b9a55c..001fd80dbea2fca54ae452a961f2e197517966d6 100644
--- a/components/component_updater/component_updater_service.h
+++ b/components/component_updater/component_updater_service.h
@@ -184,6 +184,7 @@ class ComponentUpdateService {
 
   friend class speech::SodaInstallerImpl;
   friend class ::ComponentsHandler;
+  BRAVE_COMPONENT_UPDATER_SERVICE_H_
   FRIEND_TEST_ALL_PREFIXES(ComponentInstallerTest, RegisterComponent);
 };
 
@@ -197,6 +198,7 @@ class OnDemandUpdater {
   enum class Priority { BACKGROUND = 0, FOREGROUND = 1 };
 
   virtual ~OnDemandUpdater() = default;
+  BRAVE_COMPONENT_UPDATER_SERVICE_H_ON_DEMAND_UPDATER
 
  private:
   friend class OnDemandTester;
