diff --git a/components/permissions/request_type.h b/components/permissions/request_type.h
index 70ce0b07da74da50a292c88f7f360442f77cc0b4..5921ba1f86f6dd0a0451ed46342cb5571e1b8f6b 100644
--- a/components/permissions/request_type.h
+++ b/components/permissions/request_type.h
@@ -74,7 +74,16 @@ enum class RequestType {
   kWebPrinting,
 #endif
   kWindowManagement,
-  kMaxValue = kWindowManagement
+  kWidevine,
+  kBraveEthereum,
+  kBraveSolana,
+  kBraveOpenAIChat,
+  kBraveGoogleSignInPermission,
+  kBraveCardano,
+  kBraveOnDeviceSpeechModel,
+  kBraveMinValue = kWidevine,
+  kBraveMaxValue = kBraveOnDeviceSpeechModel,
+  kMaxValue = kBraveOnDeviceSpeechModel
 };
 
 #if BUILDFLAG(IS_ANDROID)
