diff --git a/components/version_info/BUILD.gn b/components/version_info/BUILD.gn
index f5e8f2af7cc50cb36828d6ad9ee7f2db160ad89e..597ae7eba2c61bed619f39ccc64665fe775350bc 100644
--- a/components/version_info/BUILD.gn
+++ b/components/version_info/BUILD.gn
@@ -12,6 +12,7 @@ declare_args() {
 }
 
 static_library("version_info") {
+  configs += [ "//brave/build:version" ]
   sources = [
     "version_info.cc",
     "version_info.h",
