diff --git a/components/sync/protocol/entity_specifics.proto b/components/sync/protocol/entity_specifics.proto
index 6a0c27e0f0a52de4bd95b6c5f3dd3f851144a282..f0fa874a56b4743f9ee024de39a47a73cce6512d 100644
--- a/components/sync/protocol/entity_specifics.proto
+++ b/components/sync/protocol/entity_specifics.proto
@@ -11,6 +11,7 @@ option optimize_for = LITE_RUNTIME;
 
 package sync_pb;
 
+import "brave/components/sync/protocol/ai_chat_specifics.proto";
 import "components/sync/protocol/ai_thread_specifics.proto";
 import "components/sync/protocol/contextual_task_specifics.proto";
 import "components/sync/protocol/encryption.proto";
@@ -181,6 +182,7 @@ message EntitySpecifics {
         1656053;
     EncryptedTabContextItemSpecifics encrypted_tab_context_item = 1661069;
     NotebookSpecifics notebook = 1670544;
+    AIChatConversationSpecifics ai_chat_conversation = 10000001;
     // When adding a new type, follow the docs below and keep this comment as
     // the last entry.
     // https://www.chromium.org/developers/design-documents/sync/integration-checklist/
