diff --git a/components/content_settings/core/common/content_settings_mojom_traits.h b/components/content_settings/core/common/content_settings_mojom_traits.h
index 2192c55d2b760d967f213902cf0df1760d280fc8..1cefc7113e543b8ab528ae98ca2cf824055461fd 100644
--- a/components/content_settings/core/common/content_settings_mojom_traits.h
+++ b/components/content_settings/core/common/content_settings_mojom_traits.h
@@ -152,6 +152,7 @@ struct StructTraits<
   static bool Read(
       content_settings::mojom::RendererContentSettingRulesDataView data,
       RendererContentSettingRules* out);
+  BRAVE_CONTENT_SETTINGS_MOJOM_TRAITS_H
 };
 
 }  // namespace mojo
