diff --git a/components/safe_browsing/core/common/features.cc b/components/safe_browsing/core/common/features.cc
index 13a01c06a849e6f680073f3655d38fc5d9a2227d..7fc6bfc09bd09afc0846c0cd21ef73a48ac0449f 100644
--- a/components/safe_browsing/core/common/features.cc
+++ b/components/safe_browsing/core/common/features.cc
@@ -41,7 +41,7 @@ BASE_FEATURE(kAdSamplerTriggerFeature,
 BASE_FEATURE(kAddWarningShownTSToClientSafeBrowsingReport,
              base::FEATURE_ENABLED_BY_DEFAULT);
 
-BASE_FEATURE(kAntivirusTelemetryForDownloads,
+BASE_OVERRIDDEN_FEATURE(kAntivirusTelemetryForDownloads,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kAutoRevokeSuspiciousNotification,
@@ -73,8 +73,8 @@ constexpr base::FeatureParam<std::string> kClientSideDetectionBypassTiersList{
     &kClientSideDetectionBypassTiers, "ClientSideDetectionBypassTiersList",
     /*default_value=*/""};
 
-BASE_FEATURE(kClientSideDetectionClipboardCopyApi,
-             base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kClientSideDetectionClipboardCopyApi,
+             base::FEATURE_DISABLED_BY_DEFAULT);
 constexpr base::FeatureParam<double> kCsdClipboardCopyApiHCAcceptanceRate{
     &kClientSideDetectionClipboardCopyApi, "HCAcceptanceRate",
     /*default_value=*/1.0};
@@ -313,9 +313,9 @@ constexpr base::FeatureParam<double>
 BASE_FEATURE(kGlobalCacheListForGatingNotificationProtections,
              base::FEATURE_ENABLED_BY_DEFAULT);
 
-BASE_FEATURE(kGooglePlayProtectInApkTelemetry,
+BASE_OVERRIDDEN_FEATURE(kGooglePlayProtectInApkTelemetry,
              "SafeBrowsingGooglePlayProtectInApkTelemetry",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kGooglePlayProtectReducesWarnings,
              "SafeBrowsingGooglePlayProtectReducesWarnings",
@@ -404,7 +404,7 @@ BASE_FEATURE(kMovePasswordLeakDetectionToggleIos,
 
 BASE_FEATURE(kNoticeQueueForEsb, base::FEATURE_ENABLED_BY_DEFAULT);
 
-BASE_FEATURE(kNotificationTelemetry, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kNotificationTelemetry, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kProactivePasswordProtection, base::FEATURE_DISABLED_BY_DEFAULT);
 const base::FeatureParam<double> kCsdProactivePasswordProtectionSampleRate{
