diff --git a/components/permissions/permission_request_manager.cc b/components/permissions/permission_request_manager.cc
index cca424a9f925e638cc028cfa9d695ecfaa1afdbb..e71904318e9401e8a369a233ab4d96867772e4b2 100644
--- a/components/permissions/permission_request_manager.cc
+++ b/components/permissions/permission_request_manager.cc
@@ -599,6 +599,7 @@ void PermissionRequestManager::OnVisibilityChanged(
   }
   bool prior_tab_is_active_ = tab_is_active_;
   tab_is_active_ = visibility != content::Visibility::HIDDEN;
+  BRAVE_PERMISSION_REQUEST_MANAGER_ON_VISIBILITY_CHANGED
   if (prior_tab_is_active_ != tab_is_active_) {
     OnTabActiveChanged();
   }
@@ -614,6 +615,7 @@ GURL PermissionRequestManager::GetRequestingOrigin() const {
   GURL origin = requests_.front()->requesting_origin();
   if (DCHECK_IS_ON()) {
     for (const auto& request : requests_) {
+      BRAVE_PERMISSION_REQUEST_MANAGER_GET_REQUESTING_ORIGIN
       DCHECK_EQ(origin, request->requesting_origin());
     }
   }
@@ -982,7 +984,7 @@ void PermissionRequestManager::DequeueRequestIfNeeded() {
   // only after the camera request is resolved. This is caused by code in
   // PermissionBubbleMediaAccessHandler and UserMediaClient. We probably don't
   // need two permission queues, so resolve the duplication.
-  if (web_contents()->HasUncommittedNavigationInPrimaryMainFrame() || view_ ||
+  if (!web_contents()->IsDocumentOnLoadCompletedInPrimaryMainFrame() || view_ ||
       IsRequestInProgress()) {
     return;
   }
@@ -1111,7 +1113,7 @@ void PermissionRequestManager::ShowPrompt() {
     return;
   }
 
-  DCHECK(!web_contents()->HasUncommittedNavigationInPrimaryMainFrame());
+  DCHECK(web_contents()->IsDocumentOnLoadCompletedInPrimaryMainFrame());
   DCHECK(current_request_ui_to_use_);
 
   if (!tab_is_active_) {
@@ -1963,7 +1965,7 @@ void PermissionRequestManager::OnTabActiveChanged() {
     return;
   }
 
-  if (web_contents()->HasUncommittedNavigationInPrimaryMainFrame()) {
+  if (!web_contents()->IsDocumentOnLoadCompletedInPrimaryMainFrame()) {
     return;
   }
 
