diff --git a/components/content_settings/core/browser/content_settings_policy_provider.cc b/components/content_settings/core/browser/content_settings_policy_provider.cc
index 174e7f0c9e714bc7a792d9dc69fbc8c01d064146..e924faa3b30c8908e59fae2786043e54d9aff5af 100644
--- a/components/content_settings/core/browser/content_settings_policy_provider.cc
+++ b/components/content_settings/core/browser/content_settings_policy_provider.cc
@@ -51,6 +51,7 @@ struct PrefsForManagedContentSettingsMapEntry {
 // verifies this invariant or documents any necessary deviation.
 constexpr PrefsForManagedContentSettingsMapEntry
     kPrefsForManagedContentSettingsMap[] = {
+        BRAVE_MANAGED_CONTENT_SETTINGS
         {prefs::kManagedAutomaticFullscreenAllowedForUrls,
          ContentSettingsType::AUTOMATIC_FULLSCREEN, CONTENT_SETTING_ALLOW},
         {prefs::kManagedAutomaticFullscreenBlockedForUrls,
@@ -172,6 +173,7 @@ constexpr PrefsForManagedContentSettingsMapEntry
 };
 
 constexpr const char* kManagedPrefs[] = {
+    BRAVE_MANAGED_PREFS
     prefs::kManagedAutomaticFullscreenAllowedForUrls,
     prefs::kManagedAutomaticFullscreenBlockedForUrls,
     prefs::kManagedAutoSelectCertificateForUrls,
@@ -240,6 +242,7 @@ constexpr const char* kManagedPrefs[] = {
 // the preference default content settings. If a default content settings type
 // is managed any user defined exceptions (patterns) for this type are ignored.
 constexpr const char* kManagedDefaultPrefs[] = {
+    BRAVE_MANAGED_DEFAULT_PREFS
     prefs::kManagedDefaultAdsSetting,
     prefs::kManagedDefaultClipboardSetting,
     prefs::kManagedDefaultCookiesSetting,
@@ -335,6 +338,7 @@ struct PolicyProvider::PrefsForManagedDefaultMapEntry {
 // static
 const PolicyProvider::PrefsForManagedDefaultMapEntry
     PolicyProvider::kPrefsForManagedDefault[] = {
+        BRAVE_MANAGED_DEFAULT
         {ContentSettingsType::ADS, prefs::kManagedDefaultAdsSetting},
         {ContentSettingsType::CLIPBOARD_READ_WRITE,
          prefs::kManagedDefaultClipboardSetting},
