diff --git a/components/sync/driver/BUILD.gn b/components/sync/driver/BUILD.gn
index 3608ed36aaf8753f6d3f546e6ea921d30dc9d63a..bff402a76740ae5b04336babf23e0b78be20117d 100644
--- a/components/sync/driver/BUILD.gn
+++ b/components/sync/driver/BUILD.gn
@@ -111,6 +111,7 @@ static_library("driver") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
 
 static_library("test_support") {
