diff --git a/components/password_manager/core/browser/password_form_filling.cc b/components/password_manager/core/browser/password_form_filling.cc
index 7f6c5f760f5f7aa777da0689d91da4bd1416a0da..0d795a9d16eb5cd62d797bf5f39737c14a0dd917 100644
--- a/components/password_manager/core/browser/password_form_filling.cc
+++ b/components/password_manager/core/browser/password_form_filling.cc
@@ -202,6 +202,9 @@ LikelyFormFilling SendFillInformationToRenderer(
 
   bool wait_for_username =
       wait_for_username_reason != WaitForUsernameReason::kDontWait;
+#if defined(BRAVE_CHROMIUM_BUILD)
+  wait_for_username = true;
+#endif
 
   if (wait_for_username) {
     metrics_recorder->SetManagerAction(
