diff --git a/components/permissions/permission_util.cc b/components/permissions/permission_util.cc
index 6f4d9651862e90abaa22b93e1dcf97ac07592c5c..d8c95b3135daf5dea0e6416dad80cce4550f9ece 100644
--- a/components/permissions/permission_util.cc
+++ b/components/permissions/permission_util.cc
@@ -391,6 +391,7 @@ ContentSettingsType PermissionUtil::PermissionTypeToContentSettingsTypeSafe(
       return ContentSettingsType::LOCAL_NETWORK_ACCESS;
     case PermissionType::NUM:
       break;
+    PERMISSION_UTIL_PERMISSION_TYPE_TO_CONTENT_SETTINGS_TYPE
   }
 
   return ContentSettingsType::DEFAULT;
