diff --git a/components/translate/core/browser/BUILD.gn b/components/translate/core/browser/BUILD.gn
index a9c4a78b217a90a3186360dd17921dac16892bd4..874a0ca0312714b0e1f3375a4dc196c4d2b9013c 100644
--- a/components/translate/core/browser/BUILD.gn
+++ b/components/translate/core/browser/BUILD.gn
@@ -85,6 +85,7 @@ static_library("browser") {
     ]
     deps += [ "//components/infobars/core" ]
   }
+  deps += ["//brave/components/translate/core/browser"]
 }
 
 source_set("translate_pref_names") {
