diff --git a/components/password_manager/core/browser/password_form_filling.cc b/components/password_manager/core/browser/password_form_filling.cc
index d234228194e800046f8db8ee90e8baa97af8dc0f..2cc1cd29e69a91f7fdd9c3e2d1b66a730c48a7ac 100644
--- a/components/password_manager/core/browser/password_form_filling.cc
+++ b/components/password_manager/core/browser/password_form_filling.cc
@@ -190,6 +190,9 @@ LikelyFormFilling SendFillInformationToRenderer(
 
   bool wait_for_username =
       wait_for_username_reason != WaitForUsernameReason::kDontWait;
+#if defined(BRAVE_CHROMIUM_BUILD)
+  wait_for_username = true;
+#endif
 
   if (wait_for_username) {
     metrics_recorder->SetManagerAction(
