diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index 13ecd738384ecdc89a644b4abcaa96faed333336..9f4240d7ca46ec69c0ebe4a2f53c5e86f1026587 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -115,6 +115,7 @@ android_library("java") {
     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelPrompt.java",
     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelSetting.java",
   ]
+  sources += brave_components_permissions_java_sources
   deps = [
     ":core_java",
     ":java_resources",
