diff --git a/components/sync_device_info/device_info_sync_bridge_unittest.cc b/components/sync_device_info/device_info_sync_bridge_unittest.cc
index c98a2f417ab60c0c22f9e19b7f5fe877e543749a..a69b5eaf53e08d04449a88ba08c3cd16aad7e4ff 100644
--- a/components/sync_device_info/device_info_sync_bridge_unittest.cc
+++ b/components/sync_device_info/device_info_sync_bridge_unittest.cc
@@ -874,7 +874,7 @@ TEST_F(DeviceInfoSyncBridgeTest, ApplySyncChangesWithLocalGuid) {
   auto error_on_delete = bridge()->ApplySyncChanges(
       bridge()->CreateMetadataChangeList(), std::move(entity_change_list));
   EXPECT_FALSE(error_on_delete);
-  EXPECT_EQ(1, change_count());
+  EXPECT_EQ(2, change_count());  // Brave can delete other device in chain, delete record should be also processed
 }
 
 TEST_F(DeviceInfoSyncBridgeTest, ApplyDeleteNonexistent) {
