diff --git a/components/download/public/background_service/BUILD.gn b/components/download/public/background_service/BUILD.gn
index f5b1cb64a876718afde6f5c8a880c043fc4db7fd..1f0c3d061d6b4abee833bb75fbce127162e30510 100644
--- a/components/download/public/background_service/BUILD.gn
+++ b/components/download/public/background_service/BUILD.gn
@@ -35,6 +35,7 @@ component("public") {
       "logger.h",
       "navigation_monitor.h",
       "url_loader_factory_getter.h",
+      "//brave/chromium_src/components/download/public/background_service/url_loader_factory_getter.h",
     ]
   }
 
