diff --git a/components/browser_actuator/public/features.cc b/components/browser_actuator/public/features.cc
index c254ff63474c85e82986b0c31cfd87fc5ad467fb..ceeafd4a18a1aecd5d0860e02314ea0b87a6bda8 100644
--- a/components/browser_actuator/public/features.cc
+++ b/components/browser_actuator/public/features.cc
@@ -6,7 +6,7 @@
 
 namespace browser_actuator {
 
-BASE_FEATURE(kBrowserActuator, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kBrowserActuator, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kBrowserActuatorInternals, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kEnableBrowserActuatorForGlicExperimentalTriggering,
              base::FEATURE_DISABLED_BY_DEFAULT);
