diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index 5a1fa5feefec9015b4d2fc7146b5e2ff458eeefc..f5bd750a380f8f70873b65dc0ee96c8948b7e874 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -292,6 +292,7 @@ void PermissionManager::RequestPermissionsFromCurrentDocument(
     base::OnceCallback<void(const std::vector<PermissionStatus>&)>
         permission_status_callback) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
+  BRAVE_REQUEST_PERMISSION_FROM_CURRENT_DOCUMENT
   RequestPermissionsInternal(render_frame_host, request_description,
                              std::move(permission_status_callback));
 }
@@ -344,6 +345,7 @@ PermissionManager::GetPermissionResultForCurrentDocument(
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
 
   const GURL requesting_origin =
+      BRAVE_GET_PERMISSION_RESULT_FOR_CURRENT_DOCUMENT
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   const GURL embedding_origin =
       GetEmbeddingOrigin(render_frame_host, requesting_origin);
