diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index 1e1d23d40c3239c86e5810eb9978aaf51b0827f7..700b0668da1a69412ba0230a4aa1158d9df44ae6 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -408,6 +408,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
         return "window_placement";
       }
 #endif
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
