diff --git a/components/sync/base/BUILD.gn b/components/sync/base/BUILD.gn
index 6277a377e0902bb957eea7c4650850d845b29b4e..22b2f6d558168361d420823940bac72ffbb585a7 100644
--- a/components/sync/base/BUILD.gn
+++ b/components/sync/base/BUILD.gn
@@ -90,6 +90,7 @@ static_library("base") {
   ]
 
   defines = [ "SYNC_USER_AGENT_PRODUCT=$sync_user_agent_product" ]
+  import("//brave/components/sync/base/sources.gni") public_deps += brave_components_sync_base_public_deps
 }
 
 fuzzer_test("unique_position_fuzzer") {
