diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index c96d635788062d0e124ceea4c1a58c8a663ee428..45801b35b2ce6cf4a2f1c31f70958260a683ec63 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -276,6 +276,7 @@ class SyncServiceImpl : public SyncService,
   size_t GetQueuedLocalDataMigrationItemCountForTest() const;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
