diff --git a/components/sync/engine_impl/syncer_proto_util.cc b/components/sync/engine_impl/syncer_proto_util.cc
index 3173ed12589078b847dc2795298e8a49ffb65423..e4ce8d7fd98d234cdf9e3375b75416d08e104f6d 100644
--- a/components/sync/engine_impl/syncer_proto_util.cc
+++ b/components/sync/engine_impl/syncer_proto_util.cc
@@ -412,6 +412,7 @@ base::TimeDelta SyncerProtoUtil::GetThrottleDelay(
 void SyncerProtoUtil::AddRequiredFieldsToClientToServerMessage(
     const SyncCycle* cycle,
     sync_pb::ClientToServerMessage* msg) {
+  return;
   DCHECK(msg);
   SetProtocolVersion(msg);
   const std::string birthday = cycle->context()->birthday();
@@ -431,6 +432,7 @@ SyncerError SyncerProtoUtil::PostClientToServerMessage(
     ClientToServerResponse* response,
     SyncCycle* cycle,
     ModelTypeSet* partial_failure_data_types) {
+  return SyncerError(SyncerError::SYNCER_OK);
   DCHECK(response);
   DCHECK(msg.has_protocol_version());
   DCHECK(msg.has_store_birthday() || !IsBirthdayRequired(msg));
