diff --git a/components/password_manager/core/browser/features/password_features.cc b/components/password_manager/core/browser/features/password_features.cc
index c5bd250efe8fa0822e962bf96b823659e41238d6..89e558abc74fd3ed943eaac4974a50a1ff08955b 100644
--- a/components/password_manager/core/browser/features/password_features.cc
+++ b/components/password_manager/core/browser/features/password_features.cc
@@ -214,11 +214,11 @@ const base::FeatureParam<int> kCapturePageContentRetryCount = {
 BASE_FEATURE(kShowTabWithPasswordChangeOnSuccess,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
-BASE_FEATURE(kSkipUndecryptablePasswords,
-#if BUILDFLAG(IS_WIN)
-             base::FEATURE_ENABLED_BY_DEFAULT
+BASE_OVERRIDDEN_FEATURE(kSkipUndecryptablePasswords,
+#if BUILDFLAG(IS_IOS) || BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_WIN)
+  base::FEATURE_ENABLED_BY_DEFAULT
 #else
-             base::FEATURE_DISABLED_BY_DEFAULT
+  base::FEATURE_DISABLED_BY_DEFAULT
 #endif
 );
 
