diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index a95dbdcc477faaa0b00bb38e60b8f25c0c59322a..1154a9209ad86e30be17d4534f9c57aa9c8d910f 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -405,6 +405,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
         return "window_management";
       }
 #endif
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
