diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 18c28fe8ca48fc6b8a3e4cd3286315bfff18e00e..138e565777f0c5f62c5376230ea2064ecc01a21f 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -51,6 +51,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
