diff --git a/components/sync/driver/profile_sync_service.h b/components/sync/driver/profile_sync_service.h
index bf74cf0bdba095372f0a8682c7ef7b72b4617b09..efdfe4fdf7813ccd59e6ce5cb8164d8d7f10c4a0 100644
--- a/components/sync/driver/profile_sync_service.h
+++ b/components/sync/driver/profile_sync_service.h
@@ -255,6 +255,7 @@ class ProfileSyncService : public SyncService,
 
   SyncClient* GetSyncClientForTest();
 
+ BRAVE_PROFILE_SYNC_SERVICE_H_
  private:
   // Passed as an argument to StopImpl to control whether or not the sync
   // engine should clear its data when it shuts down. See StopImpl for more
