diff --git a/components/password_manager/core/browser/password_store/login_database.cc b/components/password_manager/core/browser/password_store/login_database.cc
index 324c3fe13daf34a9c3261af435ffb8e95faac513..91afd760c0253667967021ceeaf09a08a08b0616 100644
--- a/components/password_manager/core/browser/password_store/login_database.cc
+++ b/components/password_manager/core/browser/password_store/login_database.cc
@@ -2277,6 +2277,9 @@ FormRetrievalResult LoginDatabase::StatementToStoredCredentials(
 
     return FormRetrievalResult::kEncryptionServiceFailure;
   }
+  if (!credentials->empty()) {
+    brave::os_crypt_key_history_recorder::NotifyDecryptedStoredData();
+  }
   return FormRetrievalResult::kSuccess;
 }
 
