diff --git a/components/sync_device_info/device_info.h b/components/sync_device_info/device_info.h
index 980032a34b76c3e4f25c923812b37b2f3da57dc3..2d552fb0a68960a0c28bc9488e237100189f441e 100644
--- a/components/sync_device_info/device_info.h
+++ b/components/sync_device_info/device_info.h
@@ -208,11 +208,28 @@ class DeviceInfo {
              const MobilePromoOnDesktopPromoTypeSet&
                  desktop_to_ios_promo_receiving_types,
              GlicExperimentalTriggeringState glic_experimental_triggering_state,
-             std::optional<int> glic_experimental_triggering_version);
+             std::optional<int> glic_experimental_triggering_version,
+             SelfDeleteSupport self_delete_support =
+                 SelfDeleteSupport::kNotSupported);
 
   DeviceInfo& operator=(const DeviceInfo&) = delete;
 
+  DeviceInfo(DeviceInfo&&);
+
   ~DeviceInfo();
+  SelfDeleteSupport self_delete_support() const {
+    return self_delete_support_;
+  }
+  void set_self_delete_support(SelfDeleteSupport self_delete_support) {
+    self_delete_support_ = self_delete_support;
+  }
+  // Gets the OS in string form.
+  std::string GetOSString() const;
+  // Gets the device type in string form.
+  std::string GetDeviceTypeString() const;
+  // Converts the DeviceInfo values to a JS friendly DictionaryValue,
+  // which extension APIs can expose to third party apps.
+  base::DictValue ToValue() const;
 
   std::unique_ptr<DeviceInfo> DeepCopyForTesting() const;
 
@@ -414,6 +431,7 @@ class DeviceInfo {
   // NOTE: when adding a member, don't forget to update
   // |StoredDeviceInfoStillAccurate| in device_info_sync_bridge.cc or else
   // changes in that member might not trigger uploads of updated DeviceInfos.
+  SelfDeleteSupport self_delete_support_;
 };
 
 }  // namespace syncer
