diff --git a/components/sync/engine/model_type_worker.h b/components/sync/engine/model_type_worker.h
index d14aa2e11bddeadf0b5e8a9f513f0fb83a6f0257..eac1e0e21089961d415114aec7ded9beb8310766 100644
--- a/components/sync/engine/model_type_worker.h
+++ b/components/sync/engine/model_type_worker.h
@@ -203,6 +203,7 @@ class ModelTypeWorker : public UpdateHandler,
 
   static constexpr size_t kMaxPendingInvalidations = 10u;
 
+ BRAVE_MODEL_TYPE_WORKER_H_
  private:
   struct UnknownEncryptionKeyInfo {
     // Not increased if the cryptographer knows it's in a pending state
