diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index b7b391c1f166c3e67001413a1a0eea49fc530eb3..308bfdbc2118be0384f8246fe048bb98106398a1 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -347,6 +347,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
