diff --git a/components/permissions/permission_context_base.h b/components/permissions/permission_context_base.h
index 9266ced5ba3da3a6f493b5d438df85113038f8d0..90d5e77a70a44d2f00b3d2bfdfc491218cbb2384 100644
--- a/components/permissions/permission_context_base.h
+++ b/components/permissions/permission_context_base.h
@@ -126,6 +126,7 @@ class PermissionContextBase : public KeyedService,
   void AddObserver(permissions::Observer* permission_observer);
   void RemoveObserver(permissions::Observer* permission_observer);
 
+  BRAVE_PERMISSION_CONTEXT_BASE_H_
  protected:
   virtual ContentSetting GetPermissionStatusInternal(
       content::RenderFrameHost* render_frame_host,
