diff --git a/components/infobars/core/infobar_delegate.h b/components/infobars/core/infobar_delegate.h
index f9e4f43ce34f45f87b20fb8658b51db2637f6ee2..740bf57f300c2d76fbbb325100f35f3546bae649 100644
--- a/components/infobars/core/infobar_delegate.h
+++ b/components/infobars/core/infobar_delegate.h
@@ -163,6 +163,7 @@ class InfoBarDelegate {
     KNOWN_INTERCEPTION_DISCLOSURE_INFOBAR_DELEGATE = 96,
     SYNC_ERROR_INFOBAR_DELEGATE_ANDROID = 97,
     MIXED_CONTENT_DOWNLOAD_INFOBAR_DELEGATE_ANDROID = 98,
+    BRAVE_INFOBAR_DELEGATE_IDENTIFIERS
   };
 
   // Describes navigation events, used to decide whether infobars should be
