diff --git a/components/sync/engine/sync_engine.h b/components/sync/engine/sync_engine.h
index 41b2d996b520d4775bb0a9feab6b7e4f72bc4d2f..ed094db2ac96a3b7fad84df83c46b5d4bcf87ca1 100644
--- a/components/sync/engine/sync_engine.h
+++ b/components/sync/engine/sync_engine.h
@@ -84,6 +84,7 @@ class SyncEngine : public ModelTypeConfigurer {
 
     // Define the polling interval. Must not be zero.
     base::TimeDelta poll_interval;
+    BRAVE_SYNC_ENGINE_INIT_PARAMS_H
 
    private:
     DISALLOW_COPY_AND_ASSIGN(InitParams);
