diff --git a/components/crx_file/crx_verifier.cc b/components/crx_file/crx_verifier.cc
index 4f38c5eb90f677d9281d2b44086e43de366b910f..77ef7ab0d74ebf5712fa345ebe3eb335974a8241 100644
--- a/components/crx_file/crx_verifier.cc
+++ b/components/crx_file/crx_verifier.cc
@@ -205,6 +205,7 @@ VerifierResult VerifyCrx3(
       found_publisher_key =
           found_publisher_key || key_hash == publisher_key ||
           (accept_publisher_test_key && key_hash == *publisher_test_key);
+      found_publisher_key = found_publisher_key || IsBravePublisher(key_hash);
       auto v = std::make_unique<crypto::SignatureVerifier>();
       static_assert(sizeof(unsigned char) == sizeof(uint8_t),
                     "Unsupported char size.");
