diff --git a/components/search_engines/default_search_manager.cc b/components/search_engines/default_search_manager.cc
index b809b288604c4d11813cc265515ad367ca4aa9a0..d62284adc2a8aebbe4613cff7f65dae7a261d21c 100644
--- a/components/search_engines/default_search_manager.cc
+++ b/components/search_engines/default_search_manager.cc
@@ -338,6 +338,7 @@ void DefaultSearchManager::LoadDefaultSearchEngineFromPrefs() {
   // Check if default search preference is overridden by extension.
   if (pref->IsExtensionControlled()) {
     extension_default_search_ = std::move(turl_data);
+  LOADDEFAULTSEARCHENGINEFROMPREFS_BRAVE
   } else {
     prefs_default_search_.SetAndReconcile(std::move(turl_data));
   }
