diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index cfadbd5701c8d47142ab7f7549bbeeba118033a0..d27a3aaa9cae1c2e0f45f1eb505292d41e57d9f5 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -126,6 +126,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kTopLevelStorageAccess:
       return RequestTypeForUma::PERMISSION_TOP_LEVEL_STORAGE_ACCESS;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
