diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 1bd804371adab2e3db9eca7fbfb0502a5edf52a5..f6f007f37a25f3ae55eeeb1b5a2e54e816d986fc 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -282,6 +282,7 @@ static_library("browser") {
     "//ui/base",
     "//ui/gfx",
   ]
+  sources += brave_components_omnibox_browser_sources deps += brave_components_omnibox_browser_deps
 
   if ((!is_android || enable_vr) && !is_ios) {
     deps += [ ":vector_icons" ]
