diff --git a/components/content_settings/core/browser/content_settings_default_provider.cc b/components/content_settings/core/browser/content_settings_default_provider.cc
index 52fb93a67d41dcf4ff409c224595b0cd532de52f..e2713386c85c5506475189977fe5c892c8216cbd 100644
--- a/components/content_settings/core/browser/content_settings_default_provider.cc
+++ b/components/content_settings/core/browser/content_settings_default_provider.cc
@@ -373,6 +373,7 @@ void DefaultProvider::DiscardOrMigrateObsoletePreferences() {
     prefs_->SetBoolean(kBug364820109AlreadyWorkedAroundPref, true);
   }
 #endif  // !BUILDFLAG(IS_IOS)
+BRAVE_DISCARD_OR_MIGRATE_OBSOLETE_PREFERENCES
 }
 
 void DefaultProvider::RecordHistogramMetrics() {
