diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 354ea4167a014e1b374e613c2017c6fe15a925a5..ec5b9997e163673d194becbdd9dcdee8e4f223ad 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -380,6 +380,7 @@ static_library("browser") {
     "//ui/base",
     "//ui/gfx",
   ]
+  sources += brave_components_omnibox_browser_sources deps += brave_components_omnibox_browser_deps
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
 
@@ -976,6 +977,7 @@ fuzzer_test("search_suggestion_parser_fuzzer") {
 
 mojom("mojo_bindings") {
   sources = [ "omnibox.mojom" ]
+  generate_legacy_js_bindings = true
   webui_module_path = "/"
 
   public_deps = [ "//mojo/public/mojom/base" ]
