diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index e23e2e11f0da9ff3967488466b20b1b66bb28ab2..c21a432a91ca1764bf7bdee44371c63a05453116 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -711,6 +711,7 @@ std::string GetPermissionStringForUma(
     case ContentSettingsType::LOOPBACK_NETWORK:
       return "LoopbackNetwork";
     default:
+      BRAVE_GET_PERMISSION_STRING_FOR_UMA
       break;
   }
   // The user is not prompted for these permissions thus there is no
