diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index d2e5e3cc81d12d8dd1c5d2cc6e85c9f79ce23f67..d4d7e73f5d05b6cc520ea5af0a889d7f898a8e8e 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -637,6 +637,7 @@ CookieSettingsBase::DecideAccess(const GURL& url,
     return BlockAllCookies{};
   }
 
+  BRAVE_COOKIE_SETTINGS_BASE_DECIDE_ACCESS
   if (!is_third_party_request) {
     return AllowAllCookies{ThirdPartyCookieAllowMechanism::kNone};
   }
@@ -815,6 +816,7 @@ CookieSettingsBase::GetCookieSettingInternal(
   if (std::holds_alternative<AllowPartitionedCookies>(choice)) {
     CHECK(is_third_party_request);
     CHECK(block_third_party_cookies);
+    BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL_IS_EXPLICIT_SETTING
     CHECK(!is_explicit_setting);
 
     FireStorageAccessHistogram(StorageAccessResult::ACCESS_BLOCKED);
