diff --git a/components/search_engines/search_engine_type.h b/components/search_engines/search_engine_type.h
index 707c5c8785b9c1a7d8f7b5f686024afba7e8f322..bd9f2ee0e0caa9773cffd544e2d6b1ce4647b500 100644
--- a/components/search_engines/search_engine_type.h
+++ b/components/search_engines/search_engine_type.h
@@ -64,9 +64,7 @@ enum SearchEngineType {
   SEARCH_ENGINE_ZOZNAM,
   SEARCH_ENGINE_360,
   SEARCH_ENGINE_COCCOC,
-  SEARCH_ENGINE_DUCKDUCKGO,
   SEARCH_ENGINE_PARSIJOO,
-  SEARCH_ENGINE_QWANT,
   SEARCH_ENGINE_GIVERO,
   SEARCH_ENGINE_GMX,
   SEARCH_ENGINE_INFO_COM,
