diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index ab08cd903dd8d5686f33d595fdb1f5b67f4322ea..b31b8f429634ec35ce1186a4ed74e223ad358954 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -621,6 +621,7 @@ CookieSettingsBase::DecideAccess(const GURL& url,
     return BlockAllCookies{};
   }
 
+  BRAVE_COOKIE_SETTINGS_BASE_DECIDE_ACCESS
   if (!is_third_party_request) {
     return AllowAllCookies{ThirdPartyCookieAllowMechanism::kNone};
   }
@@ -799,6 +800,7 @@ CookieSettingsBase::GetCookieSettingInternal(
   if (std::holds_alternative<AllowPartitionedCookies>(choice)) {
     CHECK(is_third_party_request);
     CHECK(block_third_party_cookies);
+    BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL_IS_EXPLICIT_SETTING
     CHECK(!is_explicit_setting);
 
     FireStorageAccessHistogram(StorageAccessResult::ACCESS_BLOCKED);
