diff --git a/components/sync/driver/profile_sync_service.h b/components/sync/driver/profile_sync_service.h
index 8be1975304765f837853d7ef33320111efd76c95..5526e1efacc00f2364c2dd88ff1ea482bb163c50 100644
--- a/components/sync/driver/profile_sync_service.h
+++ b/components/sync/driver/profile_sync_service.h
@@ -497,6 +497,7 @@ class ProfileSyncService : public SyncService,
 
   base::WeakPtrFactory<ProfileSyncService> weak_factory_{this};
 
+  BRAVE_PROFILE_SYNC_SERVICE_H
   DISALLOW_COPY_AND_ASSIGN(ProfileSyncService);
 };
 
