diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index f10a1dafa23f9afe4c636bb8452f14f723144a9f..ce84feaea763d45c29fa0ec5d2dd64e737d1f4e9 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -398,6 +398,7 @@ AutocompleteController::AutocompleteController(
     if (provider_types & AutocompleteProvider::TYPE_SEARCH) {
       search_provider_ = new SearchProvider(provider_client_.get(), this);
       providers_.push_back(search_provider_.get());
+      BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
     }
     // Providers run in the order they're added.  Add `HistoryURLProvider` after
     // `SearchProvider` because:
@@ -971,6 +972,8 @@ void AutocompleteController::UpdateResult(
   metrics_.OnUpdateResult(last_result_for_logging,
                           result_.GetMatchDedupComparators());
 
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
+
   // Below are all annotations after the match list is ready.
 
   // Only produce Pedals for the default focus case (not on focus or on delete).
