diff --git a/components/content_settings/core/browser/content_settings_pref_provider.cc b/components/content_settings/core/browser/content_settings_pref_provider.cc
index 9ffac847adcfb667277910430579d007be7cba09..b89a5e7636c6cbd3f8ffa6a546615e016f51c504 100644
--- a/components/content_settings/core/browser/content_settings_pref_provider.cc
+++ b/components/content_settings/core/browser/content_settings_pref_provider.cc
@@ -118,6 +118,7 @@ PrefProvider::PrefProvider(PrefService* prefs,
       clock_(base::DefaultClock::GetInstance()) {
   TRACE_EVENT_BEGIN0("startup", "PrefProvider::PrefProvider");
   DCHECK(prefs_);
+
   // Verify preferences version.
   if (!prefs_->HasPrefPath(prefs::kContentSettingsVersion)) {
     prefs_->SetInteger(prefs::kContentSettingsVersion,
