diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index b8ad5820d942e220d5897b1ed2e24716a20ad693..15feaadd1a175d5e54a5d8000cd627ecf385c749 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -259,6 +259,7 @@ class SyncServiceImpl : public SyncService,
   void ReportDataTypeErrorForTest(DataType type);
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
