diff --git a/components/page_info/BUILD.gn b/components/page_info/BUILD.gn
index d933a85f7844bb746483478f0247d6a8f1c62e8e..f7298343596de09136dd36abe08de204f1d5e1fe 100644
--- a/components/page_info/BUILD.gn
+++ b/components/page_info/BUILD.gn
@@ -47,6 +47,7 @@ static_library("page_info") {
     "//services/device/public/cpp:device_features",
     "//services/metrics/public/cpp:ukm_builders",
   ]
+  deps += [ "//brave/components/page_info" ]
 
   if (is_android) {
     deps += [
