diff --git a/components/history_embeddings/core/vector_database.cc b/components/history_embeddings/core/vector_database.cc
index b380c2a1a8af209d7efd3e94e38a7a9b28a1b6d0..ffb3c0d78a5ec3e5267e620fe3e29559dd505830 100644
--- a/components/history_embeddings/core/vector_database.cc
+++ b/components/history_embeddings/core/vector_database.cc
@@ -245,7 +245,7 @@ SearchInfo VectorDatabase::FindNearest(
   }
 
   std::unique_ptr<UrlDataIterator> iterator =
-      MakeUrlDataIterator(time_range_start);
+      MakeUrlDataIterator(time_range_start, search_params.url_id_filter);
   if (!iterator) {
     return {};
   }
@@ -378,41 +378,47 @@ bool VectorDatabaseInMemory::AddUrlData(UrlData url_data) {
 
 std::unique_ptr<VectorDatabase::UrlDataIterator>
 VectorDatabaseInMemory::MakeUrlDataIterator(
-    std::optional<base::Time> time_range_start) {
+    std::optional<base::Time> time_range_start,
+    const std::vector<history::URLID>& url_id_filter) {
   struct SimpleIterator : public UrlDataIterator {
     explicit SimpleIterator(const std::vector<UrlData>& source,
-                            std::optional<base::Time> time_range_start)
+                            std::optional<base::Time> time_range_start,
+                            std::vector<history::URLID> url_id_filter)
         : iterator_(source.cbegin()),
           end_(source.cend()),
-          time_range_start_(time_range_start) {}
+          time_range_start_(time_range_start),
+          url_id_filter_(std::move(url_id_filter)) {}
     ~SimpleIterator() override = default;
 
     const UrlData* Next() override {
-      if (time_range_start_.has_value()) {
-        while (iterator_ != end_) {
-          if (iterator_->visit_time >= time_range_start_.value()) {
-            break;
-          }
+      while (iterator_ != end_) {
+        if (time_range_start_.has_value() &&
+            iterator_->visit_time < time_range_start_.value()) {
           iterator_++;
+          continue;
         }
+        if (!url_id_filter_.empty() &&
+            !std::ranges::contains(url_id_filter_, iterator_->url_id)) {
+          iterator_++;
+          continue;
+        }
+        return &(*iterator_++);
       }
-
-      if (iterator_ == end_) {
-        return nullptr;
-      }
-      return &(*iterator_++);
+      return nullptr;
     }
 
     std::vector<UrlData>::const_iterator iterator_;
     std::vector<UrlData>::const_iterator end_;
     const std::optional<base::Time> time_range_start_;
+    const std::vector<history::URLID> url_id_filter_;
   };
 
   if (data_.empty()) {
     return nullptr;
   }
 
-  return std::make_unique<SimpleIterator>(data_, time_range_start);
+  return std::make_unique<SimpleIterator>(data_, time_range_start,
+                                          url_id_filter);
 }
 
 std::vector<std::string> SplitQueryToTerms(
