diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index 74b6a7a9ee39f35090e4b7bc2107554b7f156220..09defb74bf37256e5bbe6843faa4bb441337a7a1 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -253,6 +253,7 @@ static_library("metrics") {
       "motherboard_metrics_provider.h",
     ]
   }
+  import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
 }
 
 # The component metrics provider is a separate target because it depends upon
