diff --git a/components/safe_browsing/core/db/v4_protocol_manager_util.cc b/components/safe_browsing/core/db/v4_protocol_manager_util.cc
index c7f7915a745f055a9af76d6dad48ff6339f02d02..d3da69ff70f2b8feac875ba2e3032570620afb60 100644
--- a/components/safe_browsing/core/db/v4_protocol_manager_util.cc
+++ b/components/safe_browsing/core/db/v4_protocol_manager_util.cc
@@ -29,7 +29,7 @@ namespace safe_browsing {
 // Can be overriden by tests.
 const char* g_sbv4_url_prefix_for_testing = nullptr;
 
-const char kSbV4UrlPrefix[] = "https://safebrowsing.googleapis.com/v4";
+const char kSbV4UrlPrefix[] = "https://safebrowsing.brave.com/v4";
 
 const base::FilePath::CharType kStoreSuffix[] = FILE_PATH_LITERAL(".store");
 
