diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index 592bb70c753934cb3a512a9c247ef8e46debc4d5..423071627ed277a4a38577197f28095c264f456f 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -469,6 +469,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
       return "window_management";
     case permissions::RequestType::kIdentityProvider:
       return "identity_provider";
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
