diff --git a/components/sync/driver/glue/sync_engine_impl.h b/components/sync/driver/glue/sync_engine_impl.h
index 9d900d5b9219a8cecb02aa6ae9555ccac375ac5d..403696d12411c39ebc6ecc0d82b3e5c719e11d75 100644
--- a/components/sync/driver/glue/sync_engine_impl.h
+++ b/components/sync/driver/glue/sync_engine_impl.h
@@ -225,6 +225,7 @@ class SyncEngineImpl : public SyncEngine, public InvalidationHandler {
 
   base::WeakPtrFactory<SyncEngineImpl> weak_ptr_factory_{this};
 
+  BRAVE_SYNC_ENGINE_IMPL_H
   DISALLOW_COPY_AND_ASSIGN(SyncEngineImpl);
 };
 
