diff --git a/components/sync_preferences/BUILD.gn b/components/sync_preferences/BUILD.gn
index 0008d5fbe2496d20ccbfdb196ebd89bf103737f8..7301d50a4016a8ed91e44226b484d8efdf18abc7 100644
--- a/components/sync_preferences/BUILD.gn
+++ b/components/sync_preferences/BUILD.gn
@@ -79,6 +79,7 @@ source_set("common_syncable_prefs_database") {
     "//components/translate/core/browser",
     "//components/variations/service",
   ]
+  import("//brave/components/sync_preferences/sources.gni") public_deps = brave_common_syncable_prefs_database_public_deps
 }
 
 static_library("test_support") {
