diff --git a/components/content_settings/core/browser/content_settings_default_provider.cc b/components/content_settings/core/browser/content_settings_default_provider.cc
index 2af9a964b89beb29f9a7073beaba751a9e070d8b..ae246dca1f81e888f794c0f3c8248b399ba7c897 100644
--- a/components/content_settings/core/browser/content_settings_default_provider.cc
+++ b/components/content_settings/core/browser/content_settings_default_provider.cc
@@ -407,6 +407,7 @@ void DefaultProvider::DiscardOrMigrateObsoletePreferences() {
   }
   prefs_->ClearPref(kDeprecatedEnableDRM);
 #endif  // BUILDFLAG(IS_CHROMEOS_ASH) || defined(OS_WIN)
+BRAVE_DISCARD_OR_MIGRATE_OBSOLETE_PREFERENCES
 }
 
 }  // namespace content_settings
