diff --git a/components/history_embeddings/history_embeddings_features.cc b/components/history_embeddings/history_embeddings_features.cc
index 96a567a11f4c631e6b187cf1b96146cf72317c15..c6fb88a13f9a90f3bd2d58b1c6b3539c9a72765b 100644
--- a/components/history_embeddings/history_embeddings_features.cc
+++ b/components/history_embeddings/history_embeddings_features.cc
@@ -175,7 +175,7 @@ const base::FeatureParam<int> kMaxAnswererContextUrlCount(
 const base::FeatureParam<double> kWordMatchMinEmbeddingScore(
     &kHistoryEmbeddings,
     "WordMatchMinEmbeddingScore",
-    0.7);
+    0.5);
 
 const base::FeatureParam<int> kWordMatchMinTermLength(&kHistoryEmbeddings,
                                                       "WordMatchMinTermLength",
