diff --git a/components/omnibox/browser/autocomplete_controller.h b/components/omnibox/browser/autocomplete_controller.h
index 1639844d493166e264432cfb0286dad0579aa996..66021361dfe719a8776bb4dbd62d4c09b7563be1 100644
--- a/components/omnibox/browser/autocomplete_controller.h
+++ b/components/omnibox/browser/autocomplete_controller.h
@@ -142,6 +142,7 @@ class AutocompleteController : public AutocompleteProviderListener,
   }
 
  private:
+  friend class BraveAutocompleteController;
   friend class AutocompleteProviderTest;
   FRIEND_TEST_ALL_PREFIXES(AutocompleteProviderTest,
                            RedundantKeywordsIgnoredInResult);
