diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index c19a286d3cc5ccefffe24ac7d2049db1034cb64c..3ef18750b1aa0c25e1b2ac77d8af637948b31315 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -114,6 +114,7 @@ android_library("java") {
     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelPrompt.java",
     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelSetting.java",
   ]
+  sources += brave_components_permissions_java_sources
   deps = [
     ":core_java",
     ":java_resources",
