diff --git a/components/sync/engine_impl/sync_manager_impl.cc b/components/sync/engine_impl/sync_manager_impl.cc
index 477dca4f7b12921e1b931a4f87fce55a841f5d33..de84116803c38db496a846686c55dd24f8da404a 100644
--- a/components/sync/engine_impl/sync_manager_impl.cc
+++ b/components/sync/engine_impl/sync_manager_impl.cc
@@ -364,8 +364,9 @@ void SyncManagerImpl::Init(InitArgs* args) {
       args->poll_interval);
   scheduler_ = args->engine_components_factory->BuildScheduler(
       name_, cycle_context_.get(), args->cancelation_signal,
-      args->enable_local_sync_backend);
+      true);
 
+  BRAVE_SYNC_MANAGER_IMPL_INIT
   scheduler_->Start(SyncScheduler::CONFIGURATION_MODE, base::Time());
 
   initialized_ = true;
