diff --git a/components/sync_bookmarks/bookmark_remote_updates_handler.cc b/components/sync_bookmarks/bookmark_remote_updates_handler.cc
index 5ccdcae3d0d9b330be2fb3d0603955e8e11f0ce0..39eb36f267d0ae34064ea3a5d5cf80187e762dcf 100644
--- a/components/sync_bookmarks/bookmark_remote_updates_handler.cc
+++ b/components/sync_bookmarks/bookmark_remote_updates_handler.cc
@@ -162,6 +162,7 @@ void ApplyRemoteUpdate(
   // Compute index information before updating the |tracker|.
   const size_t old_index = size_t{old_parent->GetIndexOf(node)};
   const size_t new_index =
+    BRAVE_APPLY_REMOTE_UPDATE
       ComputeChildNodeIndex(new_parent, update_entity.unique_position, tracker);
   tracker->Update(tracked_entity, update.response_version,
                   update_entity.modification_time,
@@ -538,9 +539,11 @@ BookmarkRemoteUpdatesHandler::ProcessCreate(
     LogProblematicBookmark(RemoteBookmarkUpdateError::kParentNotFolder);
     return nullptr;
   }
+  BRAVE_PROCESS_CREATE_1
   const bookmarks::BookmarkNode* bookmark_node =
       CreateBookmarkNodeFromSpecifics(
           update_entity.specifics.bookmark(), parent_node,
+          BRAVE_PROCESS_CREATE_2
           ComputeChildNodeIndex(parent_node, update_entity.unique_position,
                                 bookmark_tracker_),
           update_entity.is_folder, bookmark_model_, favicon_service_);
