diff --git a/components/crx_file/crx_creator.cc b/components/crx_file/crx_creator.cc
index b7c808638e8501eabab23d46ead24b75dbfa04fd..abac3171549210f51b7d43e7c873e3875ad302d6 100644
--- a/components/crx_file/crx_creator.cc
+++ b/components/crx_file/crx_creator.cc
@@ -67,7 +67,7 @@ CreatorResult SignArchiveAndCreateHeader(const base::FilePath& output_path,
 
   // Assemble SignedData section.
   SignedData signed_header_data;
-  signed_header_data.set_crx_id(GetCrxId(public_key_str));
+  signed_header_data.set_crx_id(GetCrxId_BraveImpl(public_key_str, header));
   const std::string signed_header_data_str =
       signed_header_data.SerializeAsString();
   const int signed_header_size = signed_header_data_str.size();
