diff --git a/components/os_crypt/sync/BUILD.gn b/components/os_crypt/sync/BUILD.gn
index db26459da52479751d042e89935d97c74c329c54..81d372eb39299c766d97137ea7f33de6d70e484d 100644
--- a/components/os_crypt/sync/BUILD.gn
+++ b/components/os_crypt/sync/BUILD.gn
@@ -89,6 +89,7 @@ component("os_crypt") {
       defines += [ "USE_KWALLET" ]
     }
   }
+  import("//brave/components/os_crypt/sources.gni") deps += brave_components_os_crypt_deps
 }
 
 static_library("test_support") {
