diff --git a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsDelegate.java b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsDelegate.java
index 53fba7df5d9659b23d6503f2a7a28943238ada9a..e364c229cd22c230826da1e43a5d8494fe11b910 100644
--- a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsDelegate.java
+++ b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsDelegate.java
@@ -119,4 +119,5 @@ public interface SiteSettingsDelegate {
      * Dismisses the Privacy Sandbox snackbar, if active.
      */
     void dismissPrivacySandboxSnackbar();
+    default void closeButton() {};
 }
