diff --git a/components/page_info/page_info_ui.h b/components/page_info/page_info_ui.h
index 07907da920d1..57772950b297 100644
--- a/components/page_info/page_info_ui.h
+++ b/components/page_info/page_info_ui.h
@@ -18,9 +18,9 @@
 #include "components/safe_browsing/buildflags.h"
 #include "ui/gfx/native_widget_types.h"
 
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
 #include "ui/gfx/image/image_skia.h"
-#endif
+//#endif
 
 class GURL;
 class PageInfo;
@@ -201,7 +201,7 @@ class PageInfoUI {
 
   // Returns the connection icon ID for the given connection |status|.
   static int GetConnectionIconID(PageInfo::SiteConnectionStatus status);
-#else  // !defined(OS_ANDROID)
+#endif
   // Returns icons for the given PermissionInfo |info|. If |info|'s current
   // setting is CONTENT_SETTING_DEFAULT, it will return the icon for |info|'s
   // default setting.
@@ -225,7 +225,6 @@ class PageInfoUI {
 
   // Returns the icon for VR settings.
   static const gfx::ImageSkia GetVrSettingsIcon(SkColor related_text_color);
-#endif
 
   // Return true if the given ContentSettingsType is in PageInfoUI.
   static bool ContentSettingsTypeInPageInfo(ContentSettingsType type);
