diff --git a/components/private_insights/private_insights_features.cc b/components/private_insights/private_insights_features.cc
index 4e52046a2438425606cb5bd9fe6c3a29e312175f..d69e96c85b55950d8faef84fd5b18b10b1d9726e 100644
--- a/components/private_insights/private_insights_features.cc
+++ b/components/private_insights/private_insights_features.cc
@@ -8,7 +8,7 @@
 
 namespace private_insights {
 
-BASE_FEATURE(kPrivateInsightsFeature, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kPrivateInsightsFeature, base::FEATURE_DISABLED_BY_DEFAULT);
 
 const base::FeatureParam<base::TimeDelta> kPrivateInsightsUploadInterval{
     &kPrivateInsightsFeature, "upload_interval", base::Minutes(30)};
