diff --git a/components/sync/engine_impl/commit.cc b/components/sync/engine_impl/commit.cc
index 3c4eaa05c9f10d8a4ba858afd42ea5741332127f..72ba3fbe94c8661c4f0e828444d492d0ae77a66c 100644
--- a/components/sync/engine_impl/commit.cc
+++ b/components/sync/engine_impl/commit.cc
@@ -132,6 +132,7 @@ SyncerError Commit::PostAndProcessResponse(
   sync_pb::ClientToServerResponse response;
   const SyncerError post_result = SyncerProtoUtil::PostClientToServerMessage(
       message_, &response, cycle, nullptr);
+  BRAVE_COMMIT_POST_AND_PROCESS_RESPONSE
   TRACE_EVENT_END0("sync", "PostCommit");
 
   // TODO(rlarocque): Use result that includes errors captured later?
