diff --git a/components/sync/engine/syncer.cc b/components/sync/engine/syncer.cc
index e8228e827e62496725d339e1527c58f94ac836fc..5153672fe14f7180fabaeef29ca870e6dd683279 100644
--- a/components/sync/engine/syncer.cc
+++ b/components/sync/engine/syncer.cc
@@ -245,6 +245,7 @@ SyncerError Syncer::BuildAndPostCommits(const ModelTypeSet& request_types,
     }
     nudge_tracker->RecordSuccessfulCommitMessage(
         commit->GetContributingDataTypes());
+    BRAVE_SYNCER_BUILD_AND_POST_COMMITS_POLLER_CHECK
   }
 
   return SyncerError::Success();
