diff --git a/components/browser_ui/site_settings/android/website_preference_bridge.cc b/components/browser_ui/site_settings/android/website_preference_bridge.cc
index a5316d02726f1..0a587c71d8268 100644
--- a/components/browser_ui/site_settings/android/website_preference_bridge.cc
+++ b/components/browser_ui/site_settings/android/website_preference_bridge.cc
@@ -813,6 +813,7 @@ static void JNI_WebsitePreferenceBridge_SetContentSettingEnabled(
       case ContentSettingsType::POPUPS:
       case ContentSettingsType::SENSORS:
       case ContentSettingsType::SOUND:
+      case ContentSettingsType::AUTOPLAY:
         value = CONTENT_SETTING_ALLOW;
         break;
       default:
