diff --git a/components/sync/driver/profile_sync_service.h b/components/sync/driver/profile_sync_service.h
index 8d138686695c6bd13bdacc4b64bee91af1db344a..d0fbb0d439834ffe114c55e0e4bcc76197652d16 100644
--- a/components/sync/driver/profile_sync_service.h
+++ b/components/sync/driver/profile_sync_service.h
@@ -246,6 +246,7 @@ class ProfileSyncService : public SyncService,
 
   static std::string GenerateCacheGUIDForTest();
 
+ BRAVE_PROFILE_SYNC_SERVICE_H_
  private:
   // Passed as an argument to StopImpl to control whether or not the sync
   // engine should clear its data when it shuts down. See StopImpl for more
