diff --git a/components/content_settings/core/browser/cookie_settings.cc b/components/content_settings/core/browser/cookie_settings.cc
index 6c14e1dd0ec9df4f313dd8866d88682e66edf81e..986d9e6f7e1d43e6d6e860443e0ab13f2d008218 100644
--- a/components/content_settings/core/browser/cookie_settings.cc
+++ b/components/content_settings/core/browser/cookie_settings.cc
@@ -192,6 +192,7 @@ ContentSetting CookieSettings::GetCookieSettingInternal(
   DCHECK(value.is_int());
   ContentSetting setting = ValueToContentSetting(value);
   bool block = block_third && is_third_party_request;
+  BRAVE_COOKIE_SETTINGS_GET_COOKIES_SETTINGS_INTERNAL
 
   if (!block) {
     FireStorageAccessHistogram(
