diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index c3cd15fbcea2a926a4c1c41b29136e82fff45ece..0f1d6d17f35f7374a656cb728ff476c39f8070d4 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -52,6 +52,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
