diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index 31e3cb2be59ac76f0a630155d0e5edc4290555f7..9445f12de224a0fc44dce281d2310d8dd5e8ae8a 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -292,6 +292,7 @@ CookieSettingsBase::GetCookieSettingInternal(
       ShouldBlockThirdPartyCookies() &&
       !IsThirdPartyCookiesAllowedScheme(first_party_url.scheme());
 
+  BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL
   if (IsAllowed(setting) && !block_third) {
     FireStorageAccessHistogram(
         net::cookie_util::StorageAccessResult::ACCESS_ALLOWED);
