diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index b6560da76322c0fd7812684b3407effa7973318a..9d4bae8fa88b75d1d22c8a97c381dc9a02edbb2b 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -201,6 +201,7 @@ android_library("java") {
     "//third_party/jni_zero:jni_zero_java",
     "//ui/android:ui_java",
   ]
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
