diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 85746ba70fbbb875899356393d6401ee732a620d..1bb4dc8d3780d35487371aa23b2a386317a660b9 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1315,6 +1315,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
