diff --git a/components/sync/engine_impl/sync_scheduler_impl.cc b/components/sync/engine_impl/sync_scheduler_impl.cc
index 97b4e2a32e3e9c859d27a2e41580858455c9c4a9..3999232fba71fead404ab092df20527c9cb28a86 100644
--- a/components/sync/engine_impl/sync_scheduler_impl.cc
+++ b/components/sync/engine_impl/sync_scheduler_impl.cc
@@ -710,6 +710,7 @@ void SyncSchedulerImpl::TryCanaryJob() {
 }
 
 void SyncSchedulerImpl::TrySyncCycleJob() {
+  BRAVE_SYNC_SCHEDULER_IMPL_TRY_SYNC_CYCLE_JOB
   // Post call to TrySyncCycleJobImpl on current sequence. Later request for
   // access token will be here.
   base::SequencedTaskRunnerHandle::Get()->PostTask(
