diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index f5ab648eb1ac83046b1fc55afebb01d70f088eac..7a4a68418ba578c7fd5d411411de4b86a35f28e5 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -109,6 +109,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
