diff --git a/components/sessions/BUILD.gn b/components/sessions/BUILD.gn
index 8fdb01e9efaf2b351ebbcdc24e50c5917f9d0031..7fc572f137ee16c11b744498522f80cfe12d9bdd 100644
--- a/components/sessions/BUILD.gn
+++ b/components/sessions/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/components/sessions/sources.gni")
 import("//build/config/apple/mobile_config.gni")
 import("//build/config/features.gni")
 
@@ -47,6 +48,7 @@ if (use_blink && !(is_ios && target_platform == "iphoneos")) {
       "//ui/base",
       "//url",
     ]
+    deps += brave_components_sessions_deps
   }
 } else {
   source_set("sessions") {
@@ -145,6 +147,7 @@ source_set("shared") {
     "//components/os_crypt/common",
     "//skia",
   ]
+  public_deps += brave_components_sessions_shared_public_deps
 
   deps = [
     "//base",
