diff --git a/components/os_crypt/sync/BUILD.gn b/components/os_crypt/sync/BUILD.gn
index ff1e356ff696d3830d02644969c36a71fdf32ff6..fa5f4f45644327db02c4b49a573ba6e7bf87849a 100644
--- a/components/os_crypt/sync/BUILD.gn
+++ b/components/os_crypt/sync/BUILD.gn
@@ -77,6 +77,7 @@ component("sync") {
   } else if (is_posix || is_fuchsia) {
     sources += [ "os_crypt_posix.cc" ]
   }
+  import("//brave/components/os_crypt/sources.gni") deps += brave_components_os_crypt_deps
 }
 
 static_library("test_support") {
