diff --git a/components/os_crypt/sync/BUILD.gn b/components/os_crypt/sync/BUILD.gn
index f75238dedb47ff7927c715f21befede9cdc97615..bd0909cf9a6396eaa38a16ecd4a880bb1c760f67 100644
--- a/components/os_crypt/sync/BUILD.gn
+++ b/components/os_crypt/sync/BUILD.gn
@@ -92,6 +92,7 @@ component("sync") {
   } else if (is_posix || is_fuchsia) {
     sources += [ "os_crypt_posix.cc" ]
   }
+  import("//brave/components/os_crypt/sources.gni") deps += brave_components_os_crypt_deps visibility += brave_components_os_crypt_visibility
 }
 
 static_library("test_support") {
