diff --git a/components/sync/base/data_type.h b/components/sync/base/data_type.h
index 6943010c8d9afabc94e7f1a145ff8f44264d91d8..71b17feec04d12fb6e23519084a72651c908841f 100644
--- a/components/sync/base/data_type.h
+++ b/components/sync/base/data_type.h
@@ -202,7 +202,8 @@ enum DataType {
   // A theme object specifically for Android devices.
   THEMES_ANDROID,
 
-  LAST_USER_DATA_TYPE = THEMES_ANDROID,
+  AI_CHAT_CONVERSATION,
+  LAST_USER_DATA_TYPE = AI_CHAT_CONVERSATION,
 
   // ---- Control Types ----
   // An object representing a set of Nigori keys.
@@ -312,7 +313,8 @@ enum class DataTypeForHistograms {
   kThemesIos = 78,
   // kDeprecatedAccessibilityAnnotation = 79,
   kThemesAndroid = 80,
-  kMaxValue = kThemesAndroid,
+  kAIChatConversation = 81,
+  kMaxValue = kAIChatConversation,
 };
 // LINT.ThenChange(/tools/metrics/histograms/metadata/sync/enums.xml:SyncDataTypes)
 
