diff --git a/components/external_intents/android/BUILD.gn b/components/external_intents/android/BUILD.gn
index 995965064f85e180a21d2800688aaaa4e6a847dd..bb5fa91a4289b737fd8448b8ce1b6d469aa0a1a1 100644
--- a/components/external_intents/android/BUILD.gn
+++ b/components/external_intents/android/BUILD.gn
@@ -67,6 +67,7 @@ static_library("android") {
     "//components/navigation_interception",
     "//content/public/browser",
   ]
+  deps += [ "//brave/components/external_intents/android" ]
 }
 
 android_library("unit_device_javatests") {
