diff --git a/components/history/core/browser/history_backend.h b/components/history/core/browser/history_backend.h
index 38f7fb922dd5e4858daaacbebd84c027c15d63c9..da586b7fba6c08fd6277be570c1f24da0f98bef1 100644
--- a/components/history/core/browser/history_backend.h
+++ b/components/history/core/browser/history_backend.h
@@ -300,6 +300,8 @@ class HistoryBackend : public base::RefCountedThreadSafe<HistoryBackend>,
   // Querying ------------------------------------------------------------------
 
   QueryURLResult QueryURL(const GURL& url);
+  // Returns a vector parallel to `urls`; unknown URLs map to 0.
+  std::vector<URLID> QueryUrlIds(const std::vector<GURL>& urls);
   QueryURLAndVisitsResult QueryURLAndVisits(
       const GURL& url,
       VisitQuery404sPolicy policy_for_404s);
