diff --git a/components/content_settings/core/common/content_settings.h b/components/content_settings/core/common/content_settings.h
index 4eb3cf18e06d5c9169c69d6cb313de62692f8b22..c7a3917e0bc4cc54cb1203f43ab1f4279cdda988 100644
--- a/components/content_settings/core/common/content_settings.h
+++ b/components/content_settings/core/common/content_settings.h
@@ -76,6 +76,8 @@ struct RendererContentSettingRules {
   ContentSettingsForOneType client_hints_rules;
   ContentSettingsForOneType popup_redirect_rules;
   ContentSettingsForOneType mixed_content_rules;
+  ContentSettingsForOneType fingerprinting_rules;
+  ContentSettingsForOneType brave_shields_rules;
 };
 
 namespace content_settings {
