diff --git a/components/search_engines/BUILD.gn b/components/search_engines/BUILD.gn
index 56e40464d01fdc8e75d87b51f63f3264aa47fc4c..d35c141041f59048f39369b67e8822958e75b5e3 100644
--- a/components/search_engines/BUILD.gn
+++ b/components/search_engines/BUILD.gn
@@ -163,6 +163,7 @@ static_library("search_engines") {
       "enterprise/site_search_policy_handler.h",
     ]
   }
+  import("//brave/components/search_engines/sources.gni") deps += brave_components_search_engines_deps
 }
 
 source_set("search_engine_utils") {
