diff --git a/components/policy/tools/template_writers/writer_configuration.py b/components/policy/tools/template_writers/writer_configuration.py
index 283ea725c5a1874923244d716015a394a270281f..58b96878166f71c64564ece8d5b63f6ceca38771 100755
--- a/components/policy/tools/template_writers/writer_configuration.py
+++ b/components/policy/tools/template_writers/writer_configuration.py
@@ -137,3 +137,4 @@ def GetConfigurationForBuild(defines):
     config['mac_bundle_id'] = defines['mac_bundle_id']
   config['android_webview_restriction_prefix'] = 'com.android.browser:'
   return config
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
