diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 3a1a47a8ed9447ce0ac3ece6d03dddbe23e821a1..384b239264f9f762f70a0e172a730698167215c7 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -381,6 +381,7 @@ static_library("browser") {
     "//ui/base",
     "//ui/gfx",
   ]
+  sources += brave_components_omnibox_browser_sources deps += brave_components_omnibox_browser_deps public_deps += brave_components_omnibox_browser_public_deps
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
 
@@ -1008,6 +1009,7 @@ fuzzer_test("search_suggestion_parser_fuzzer") {
 
 mojom("mojo_bindings") {
   sources = [ "omnibox.mojom" ]
+  generate_legacy_js_bindings = true
 
   webui_module_path = "chrome://resources/mojo/components/omnibox/browser"
 
