diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 2f15596165fe68c5d1709f3a3858fb7fba7515a7..178b61a62a001dde64ecb74263ee36d75425b00c 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -249,6 +249,7 @@ jumbo_static_library("browser") {
   if ((!is_android || enable_vr) && !is_ios) {
     deps += [ ":vector_icons" ]
   }
+  if (brave_chromium_build) {  deps += [ "//brave/components/omnibox/browser" ] }
 
   if (is_android) {
     deps += [ ":jni_headers" ]
