diff --git a/components/sync/base/features.h b/components/sync/base/features.h
index 800632a8eaf3ffe9f7e71e09cbc48840d21891f7..3e0622d9ca1ca2481e903cd3589e9b94adb2dfac 100644
--- a/components/sync/base/features.h
+++ b/components/sync/base/features.h
@@ -114,7 +114,7 @@ DECLARE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuable);
 #endif
 
 // Enables syncing of usage metadata from Google Wallet passes.
-DECLARE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuableMetadata);
+BASE_DECLARE_FEATURE(kSyncAutofillValuableMetadata);
 
 // Enables syncing of flight reservations coming from Google Wallet.
 DECLARE_SYNC_AUTOFILL_AI_FEATURE(kSyncWalletFlightReservations);
