diff --git a/components/autofill/core/browser/suggestions/suggestion.h b/components/autofill/core/browser/suggestions/suggestion.h
index 73e3af5b7985d711969556417913df9df6dd7535..b647f49d3bb8206e7e92f5054fb4b745541bb79b 100644
--- a/components/autofill/core/browser/suggestions/suggestion.h
+++ b/components/autofill/core/browser/suggestions/suggestion.h
@@ -742,6 +742,8 @@ struct Suggestion {
 
   // Returns whether the user is able to focus or select the suggestion.
   bool IsSelectable() const;
+  bool brave_new_email_alias_suggestion = false;
+  bool brave_email_alias_suggestion = false;
 };
 
 void PrintTo(const Suggestion& suggestion, std::ostream* os);
