diff --git a/components/policy/tools/generate_policy_source.py b/components/policy/tools/generate_policy_source.py
index 41f2f032eeaa6d30893fc5f55c0466151c36653f..1135c5db39f71b3f33071f1cddaa70a2fa189947 100755
--- a/components/policy/tools/generate_policy_source.py
+++ b/components/policy/tools/generate_policy_source.py
@@ -1668,5 +1668,6 @@ def _WriteAppRestrictions(policies, policy_atomic_groups, target_platform, f,
   f.write('</restrictions>')
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
   sys.exit(main())
