diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index f362cab65a2b76ff19f2505a9205f7d830f077b0..73109dba6e7a62681fa07a9f7e799576f56f31ec 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -48,6 +48,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 typedef int64_t VisitID;
