diff --git a/components/content_settings/renderer/content_settings_agent_impl.cc b/components/content_settings/renderer/content_settings_agent_impl.cc
index fdc02b8b6554420917db2636b5fcdc84bf770824..916e9d7edbfe8430e53354e384ff971346b24059 100644
--- a/components/content_settings/renderer/content_settings_agent_impl.cc
+++ b/components/content_settings/renderer/content_settings_agent_impl.cc
@@ -360,6 +360,7 @@ bool ContentSettingsAgentImpl::IsAllowlistedForContentSettings() const {
   const WebDocument& document = render_frame()->GetWebFrame()->GetDocument();
   WebSecurityOrigin origin = document.GetSecurityOrigin();
   WebURL document_url = document.Url();
+  BRAVE_CONTENT_SETTINGS_AGENT_IMPL_IS_WHITELISTED_FOR_CONTENT_SETTINGS
   if (document_url.GetString() == content::kUnreachableWebDataURL)
     return true;
 
