diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 6c6c8106d5f61d246c1f811bd8ead9f6dca680eb..2d8a41c3b8d71874a4b577a79c6f823ddc3396af 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1247,6 +1247,7 @@ void PageInfo::PresentSitePermissions() {
       }
     }
 
+    BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
     if (ShouldShowPermission(
             permission_info, site_url_, content_settings, web_contents_.get(),
             HasContentSettingChangedViaPageInfo(permission_info.type),
