diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 5a428e737ff8e8280eeb57f16f943e6311e72011..8294726394b69891dcda6d98b68ea8ae0f9808ee 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -151,6 +151,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kIdentityProvider:
       return RequestTypeForUma::PERMISSION_IDENTITY_PROVIDER;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
@@ -176,6 +177,7 @@ RequestTypeForUma GetUmaValueForRequests(
     return RequestTypeForUma::MULTIPLE_KEYBOARD_AND_POINTER_LOCK;
   }
 #endif
+  BRAVE_GET_UMA_VALUE_FOR_REQUESTS
   return RequestTypeForUma::UNKNOWN;
 }
 
