diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index 990f9abaef15d6a9414db3aaa162e18897551eb6..c76b1030c831c3f10fe03f73aadbf93811395e9b 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -19,6 +19,7 @@ group("sync") {
   if (is_chromeos) {
     public_deps += [ "//components/sync/chromeos" ]
   }
+  import("//brave/build/config/brave_build.gni") public_deps += brave_components_sync_deps
 }
 
 static_library("test_support") {
