diff --git a/components/policy/core/browser/BUILD.gn b/components/policy/core/browser/BUILD.gn
index f0d467381da5e21372edefc9a68ecc95b7d0cf23..59bc9ef3df5b324702a500ef47ff499cffaaaee6 100644
--- a/components/policy/core/browser/BUILD.gn
+++ b/components/policy/core/browser/BUILD.gn
@@ -121,6 +121,7 @@ source_set("internal") {
     "//net",
     "//third_party/icu",
   ]
+  deps += [ "//brave/components/brave_policy:policy_pref_interceptor" ]
 }
 
 static_library("test_support") {
