diff --git a/components/tabs/public/tab_collection.h b/components/tabs/public/tab_collection.h
index e850bf2121f32eda9692a1c8e0f5507f9bec5918..070a4b8e2a0996dad937b0b29e95992ef586cd33 100644
--- a/components/tabs/public/tab_collection.h
+++ b/components/tabs/public/tab_collection.h
@@ -233,7 +233,7 @@ class TabCollection : public SupportsHandles<TabCollectionHandleFactory> {
   // This should be called either when this collection is added to another
   // collection or it is removed from another collection. The child collection
   // should not try to call this internally and set its parent.
-  void OnReparented(TabCollection* new_parent);
+  virtual void OnReparented(TabCollection* new_parent);
 
   const ChildrenVector& GetChildren() const { return impl_->GetChildren(); }
   const ChildrenVector& GetChildren(
