diff --git a/components/sync/engine/syncer_proto_util.cc b/components/sync/engine/syncer_proto_util.cc
index 31aac4dc4cf48498cbfabbde027695106f48216f..4c046a40cdedbeaa591603fdfbb7b80a561cca15 100644
--- a/components/sync/engine/syncer_proto_util.cc
+++ b/components/sync/engine/syncer_proto_util.cc
@@ -392,6 +392,7 @@ SyncerError SyncerProtoUtil::PostClientToServerMessage(
 
   SyncProtocolError sync_protocol_error =
       GetProtocolErrorFromResponse(*response, cycle->context());
+  BRAVE_SYNCER_PROTO_UTIL_POST_CLIENT_TO_SERVER_MESSAGE
 
   // Inform the delegate of the error we got.
   cycle->delegate()->OnSyncProtocolError(sync_protocol_error);
