diff --git a/components/component_updater/component_updater_service.h b/components/component_updater/component_updater_service.h
index f926034a2c86b5bc7c826683de67ff56bec7e21c..418974f2071df6fcfa6adf082e4a7aac2c9afbd5 100644
--- a/components/component_updater/component_updater_service.h
+++ b/components/component_updater/component_updater_service.h
@@ -162,6 +162,7 @@ class OnDemandUpdater {
   enum class Priority { BACKGROUND = 0, FOREGROUND = 1 };
 
   virtual ~OnDemandUpdater() = default;
+  BRAVE_COMPONENT_UPDATER_SERVICE_H_
 
  private:
   friend class OnDemandTester;
