diff --git a/components/sync_device_info/device_info.h b/components/sync_device_info/device_info.h
index c16d57e97fbb4315ef03a9efd2091680ae056d18..d7773ccfe225133690146408c22bece38c6d8efd 100644
--- a/components/sync_device_info/device_info.h
+++ b/components/sync_device_info/device_info.h
@@ -105,6 +105,7 @@ class DeviceInfo {
              const base::Optional<PhoneAsASecurityKeyInfo>& paask_info,
              const std::string& fcm_registration_token,
              const ModelTypeSet& interested_data_types);
+  virtual
   ~DeviceInfo();
 
   // Sync specific unique identifier for the device. Note if a device
