diff --git a/components/content_settings/core/common/content_settings_types.h b/components/content_settings/core/common/content_settings_types.h
index e238be9d538dd257f74101c413cbb69cf04f39f5..fdede737f382949a667b56301edaf618af09f2db 100644
--- a/components/content_settings/core/common/content_settings_types.h
+++ b/components/content_settings/core/common/content_settings_types.h
@@ -331,6 +331,22 @@ enum class ContentSettingsType : int32_t {
   // https://github.com/screen-share/capture-all-screens
   ALL_SCREEN_CAPTURE,
 
+  BRAVE_START,
+  BRAVE_ADS = BRAVE_START,
+  BRAVE_COSMETIC_FILTERING,
+  BRAVE_TRACKERS,
+  BRAVE_HTTP_UPGRADABLE_RESOURCES,
+  BRAVE_FINGERPRINTING_V2,
+  BRAVE_SHIELDS,
+  BRAVE_REFERRERS,
+  BRAVE_COOKIES,
+  BRAVE_SPEEDREADER,
+  BRAVE_ETHEREUM,
+  BRAVE_SOLANA,
+  BRAVE_GOOGLE_SIGN_IN,
+  BRAVE_HTTPS_UPGRADE,
+  BRAVE_REMEMBER_1P_STORAGE,
+  BRAVE_LOCALHOST_ACCESS,
   NUM_TYPES,
 };
 
