diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index 0606b7e68670877100114732174af25ad0b42f77..a95184ad1613aeca838ec51fc254509878d34f03 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -99,6 +99,7 @@ jumbo_static_library("base") {
   ]
   deps = [
     "//base:i18n",
+    "//brave/components/brave_sync:syncer_export",
     "//components/os_crypt",
     "//components/pref_registry",
     "//components/prefs",
