diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index b23f5e34a3db7c397616941ed110a901ec9b8f97..e9e9fb6e50c3d5599561b0455aaf4d8733c5331b 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -48,6 +48,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 typedef int64_t VisitID;
