diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index a99e3c5ec02c7b07f76996bda9f3de33f9928c98..c9803d3f0911002062f10893e0d3a78f812d6caa 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -1265,6 +1265,7 @@ void AutocompleteController::InitializeAsyncProviders(int provider_types) {
         base::MakeRefCounted<SearchProvider>(provider_client_.get(), this);
     search_provider_ = search_provider.get();
     providers_.push_back(std::move(search_provider));
+    BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
   }
   // Providers run in the order they're added.  Add `HistoryURLProvider` after
   // `SearchProvider` because:
@@ -1541,6 +1542,7 @@ void AutocompleteController::UpdateResult(UpdateType update_type,
     StartStopTimer();
   }
 
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
   PostProcessMatches();
 
   const bool is_lens_enabled = autocomplete_provider_client()->IsLensEnabled();
