diff --git a/components/password_manager/core/browser/password_form_filling.cc b/components/password_manager/core/browser/password_form_filling.cc
index 5910626fdeb8e31487903a9133777ac03d89f5b7..b72b452b509cd1dea6f381e25ef480e2e843a173 100644
--- a/components/password_manager/core/browser/password_form_filling.cc
+++ b/components/password_manager/core/browser/password_form_filling.cc
@@ -196,6 +196,9 @@ LikelyFormFilling SendFillInformationToRenderer(
 
   bool wait_for_username =
       wait_for_username_reason != WaitForUsernameReason::kDontWait;
+#if defined(BRAVE_CHROMIUM_BUILD)
+  wait_for_username = true;
+#endif
 
   if (wait_for_username) {
     metrics_recorder->SetManagerAction(
