diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 9fc81d3e309767e50982f708d8639b30ea1e1e82..1ab0342fe531499029b106902bc022481d6e5082 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -110,6 +110,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
