diff --git a/components/content_settings/core/browser/content_settings_uma_util.cc b/components/content_settings/core/browser/content_settings_uma_util.cc
index a7351217e246b6ec28e0631762468a4575c06760..5398530df31b31d7ba7424a21d89fd2d3828a568 100644
--- a/components/content_settings/core/browser/content_settings_uma_util.cc
+++ b/components/content_settings/core/browser/content_settings_uma_util.cc
@@ -20,6 +20,8 @@ std::string GetProviderNameForHistograms(
   using ProviderType = content_settings::ProviderType;
 
   switch (provider_type) {
+    case ProviderType::kRemoteListProvider:
+      return "RemoteListProvider";
     // Update the `ContentAllProviderTypes` variants in
     // https://chromium.googlesource.com/chromium/src.git/+/HEAD/tools/metrics/histograms/metadata/content/histograms.xml
     // when new providers are added.
