diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index 519d89d616047a40bea707edd6234a10010cb2d2..4cf1cb73613c74b9207b8a5a28d5ec373a36122f 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -113,6 +113,7 @@ android_library("java") {
     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelPrompt.java",
     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelSetting.java",
   ]
+  sources += brave_components_permissions_java_sources
   deps = [
     ":core_java",
     ":java_resources",
