diff --git a/components/sync_preferences/BUILD.gn b/components/sync_preferences/BUILD.gn
index 68321658c9c7f62021e59ee0c1d96590aef1bada..3a8cd28ddbcfcbc5fd747f0defb70424e5c336ee 100644
--- a/components/sync_preferences/BUILD.gn
+++ b/components/sync_preferences/BUILD.gn
@@ -84,6 +84,7 @@ source_set("common_syncable_prefs_database") {
     "//components/translate/core/browser",
     "//components/variations/service",
   ]
+  import("//brave/components/sync_preferences/sources.gni") public_deps = brave_common_syncable_prefs_database_public_deps
 }
 
 static_library("test_support") {
