diff --git a/components/content_settings/core/common/content_settings_types.h b/components/content_settings/core/common/content_settings_types.h
index f117097edd87db9ad15123bceeb88af12b813879..00b12caac7465f403e82449367b0eafe7ed531be 100644
--- a/components/content_settings/core/common/content_settings_types.h
+++ b/components/content_settings/core/common/content_settings_types.h
@@ -248,6 +248,7 @@ enum class ContentSettingsType : int32_t {
   // the corresponding "guard" setting.
   FILE_SYSTEM_ACCESS_CHOOSER_DATA,
 
+  BRAVE_CONTENT_SETTINGS_TYPES_LIST,
   NUM_TYPES,
 };
 
