diff --git a/components/sync_device_info/BUILD.gn b/components/sync_device_info/BUILD.gn
index 1ee3436ad9a1a8c1d9941c41fce2fd9d56558c81..d1f79f0761a926143791b9dd6cda90b829c42f0c 100644
--- a/components/sync_device_info/BUILD.gn
+++ b/components/sync_device_info/BUILD.gn
@@ -89,6 +89,7 @@ static_library("sync_device_info") {
   if (is_win) {
     sources += [ "local_device_info_util_win.cc" ]
   }
+  sources += brave_components_sync_device_info_sources deps += brave_components_sync_device_info_deps
 }
 
 static_library("test_support") {
