diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index e1f33848f20284ff367c8105ef7e295ee4d45481..7dc5d37b44b5d1c0d44c459edd1d709601058a4b 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -133,6 +133,7 @@ source_set("permissions") {
       "contexts/geolocation_permission_context_mac.h",
     ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
