diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index 46a85b0b5255eaee321c0ff462c5b54d6ea84dc1..1e4012a02e5f14753f5df0b1e6c920f2b09f9f7c 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -144,6 +144,7 @@ source_set("permissions") {
       "contexts/geolocation_permission_context_system.h",
     ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
