diff --git a/components/passage_embeddings/core/passage_embeddings_service_controller.h b/components/passage_embeddings/core/passage_embeddings_service_controller.h
index d8ef6e1164bd801491aca4b3fbfd3b7c66458744..5630c9bde60d3e4476e9f8feee4bf28787d5b5dd 100644
--- a/components/passage_embeddings/core/passage_embeddings_service_controller.h
+++ b/components/passage_embeddings/core/passage_embeddings_service_controller.h
@@ -66,7 +66,7 @@ class PassageEmbeddingsServiceController : public EmbedderMetadataProvider {
       base::RepeatingCallback<void(std::vector<std::string> passages,
                                    PassagePriority priority,
                                    GetEmbeddingsResultCallback callback)>;
-  void GetEmbeddings(std::vector<std::string> passages,
+  virtual void GetEmbeddings(std::vector<std::string> passages,
                      PassagePriority priority,
                      GetEmbeddingsResultCallback callback);
 
