diff --git a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsCategory.java b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsCategory.java
index d335c6e2ea9edc45acd7af8e25d2d356a0063e30..f368f79e242159c31e205ff44e2fb7849fcd8677 100644
--- a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsCategory.java
+++ b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsCategory.java
@@ -83,6 +83,7 @@ public class SiteSettingsCategory {
         Type.AUTO_PICTURE_IN_PICTURE,
         Type.LOCAL_NETWORK,
         Type.LOOPBACK_NETWORK,
+        Type.AUTOPLAY, Type.BRAVE_GOOGLE_SIGN_IN, Type.BRAVE_LOCALHOST_ACCESS,
         Type.NUM_ENTRIES
     })
     @Retention(RetentionPolicy.SOURCE)
@@ -130,7 +131,7 @@ public class SiteSettingsCategory {
         int LOOPBACK_NETWORK = 38;
 
         /** Number of handled categories used for calculating array sizes. */
-        int NUM_ENTRIES = 39;
+        int AUTOPLAY = 39; int BRAVE_GOOGLE_SIGN_IN = 40; int BRAVE_LOCALHOST_ACCESS = 41; int NUM_ENTRIES = 42;
     }
 
     private final BrowserContextHandle mBrowserContextHandle;
