diff --git a/components/policy/tools/template_writers/writer_configuration.py b/components/policy/tools/template_writers/writer_configuration.py
index 8cb77158561d326b1640adef8ebf6031f7389d08..6bc460b0a44830afaffd1f7e26d744f1b9aed80d 100755
--- a/components/policy/tools/template_writers/writer_configuration.py
+++ b/components/policy/tools/template_writers/writer_configuration.py
@@ -132,3 +132,4 @@ def GetConfigurationForBuild(defines):
     config['mac_bundle_id'] = defines['mac_bundle_id']
   config['android_webview_restriction_prefix'] = 'com.android.browser:'
   return config
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
