diff --git a/components/webapps/browser/BUILD.gn b/components/webapps/browser/BUILD.gn
index 29c38033a783f96e26fea6097ed08f394efc37a5..e0128eefaaf9735424a538c109d9981317247e9d 100644
--- a/components/webapps/browser/BUILD.gn
+++ b/components/webapps/browser/BUILD.gn
@@ -191,6 +191,7 @@ source_set("browser") {
       "//ui/gfx",
     ]
   }
+  import("//brave/components/webapps/browser/sources.gni") deps += brave_components_webapps_browser_deps
 }
 
 source_set("test_support") {
