diff --git a/components/sync/engine/cycle/data_type_tracker.cc b/components/sync/engine/cycle/data_type_tracker.cc
index 7d1ad2be0f77f1f370cbf332d9a491da379fad19..e40c9020e0b1186e3ddb2eaf9fc4983e8a8d6a84 100644
--- a/components/sync/engine/cycle/data_type_tracker.cc
+++ b/components/sync/engine/cycle/data_type_tracker.cc
@@ -120,6 +120,7 @@ base::TimeDelta GetDefaultLocalChangeNudgeDelay(DataType data_type) {
     case GEMINI_THREAD:
     case ACCESSIBILITY_ANNOTATION:
     case THEMES_ANDROID:
+    case AI_CHAT_CONVERSATION:
       return kMediumLocalChangeNudgeDelay;
     case UNSPECIFIED:
       NOTREACHED();
@@ -199,6 +200,7 @@ bool CanGetCommitsFromExtensions(DataType data_type) {
     case GEMINI_THREAD:
     case ACCESSIBILITY_ANNOTATION:
     case THEMES_ANDROID:
+    case AI_CHAT_CONVERSATION:
       return false;
     case UNSPECIFIED:
       NOTREACHED();
