diff --git a/components/vector_icons/aggregate_vector_icons.py b/components/vector_icons/aggregate_vector_icons.py
index 1974b6fc9eaa2a9f42c7059c25ab8d41108338ef..33587f91977b874c728cb077dbccabf404a9ce4f 100644
--- a/components/vector_icons/aggregate_vector_icons.py
+++ b/components/vector_icons/aggregate_vector_icons.py
@@ -251,5 +251,6 @@ def main():
                        options.output_test_h)
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == "__main__":
   main()
