diff --git a/components/permissions/android/permission_prompt/permission_dialog_delegate.cc b/components/permissions/android/permission_prompt/permission_dialog_delegate.cc
index c97f4070e82c5233d04badc725a415d54305f642..6b0b4f8c85ae5a6a9acdfea912cd4cc8f31c971b 100644
--- a/components/permissions/android/permission_prompt/permission_dialog_delegate.cc
+++ b/components/permissions/android/permission_prompt/permission_dialog_delegate.cc
@@ -172,6 +172,7 @@ PermissionDialogDelegate* PermissionDialogDelegate::CreateForTesting(
 void PermissionDialogDelegate::Accept(JNIEnv* env,
                                       const JavaParamRef<jobject>& obj) {
   CHECK(permission_prompt_);
+  BRAVE_PERMISSION_DIALOG_DELEGATE_ACCEPT
   permission_prompt_->Accept();
 }
 
@@ -185,6 +186,7 @@ void PermissionDialogDelegate::AcceptThisTime(
 void PermissionDialogDelegate::Cancel(JNIEnv* env,
                                       const JavaParamRef<jobject>& obj) {
   CHECK(permission_prompt_);
+  BRAVE_PERMISSION_DIALOG_DELEGATE_CANCEL
   permission_prompt_->Deny();
 }
 
