diff --git a/components/content_settings/core/browser/content_settings_utils.cc b/components/content_settings/core/browser/content_settings_utils.cc
index 991543e4c6f23831957909b5e60a143c1dd8f091..fba1d4924a3fa10aaa8283efef792692af39d34f 100644
--- a/components/content_settings/core/browser/content_settings_utils.cc
+++ b/components/content_settings/core/browser/content_settings_utils.cc
@@ -143,6 +143,14 @@ void GetRendererContentSettingRules(const HostContentSettingsMap* map,
   map->GetSettingsForOneType(CONTENT_SETTINGS_TYPE_CLIENT_HINTS,
                              ResourceIdentifier(),
                              &(rules->client_hints_rules));
+  map->GetSettingsForOneType(
+      CONTENT_SETTINGS_TYPE_PLUGINS,
+      "fingerprinting",
+      &(rules->fingerprinting_rules));
+  map->GetSettingsForOneType(
+      CONTENT_SETTINGS_TYPE_PLUGINS,
+      "referers",
+      &(rules->referrer_rules));
 }
 
 bool IsMorePermissive(ContentSetting a, ContentSetting b) {
