diff --git a/components/autofill/core/browser/merchant_promo_code_manager.cc b/components/autofill/core/browser/merchant_promo_code_manager.cc
index 7c4d05c8b56375daa44988c7dff16c6f4116c16d..de2fa32c9e6dedc5fc9f631886d677d23d0a04ac 100644
--- a/components/autofill/core/browser/merchant_promo_code_manager.cc
+++ b/components/autofill/core/browser/merchant_promo_code_manager.cc
@@ -49,7 +49,7 @@ bool MerchantPromoCodeManager::OnGetSingleFieldSuggestions(
 
 void MerchantPromoCodeManager::OnWillSubmitFormWithFields(
     const std::vector<FormFieldData>& fields,
-    bool is_autocomplete_enabled) {}
+    const AutofillClient& client) {}
 
 void MerchantPromoCodeManager::CancelPendingQueries(
     const SuggestionsHandler* handler) {}
