diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 6160ba458cec4782db805cddc21a74d85d052591..ed56677ffd3493e0e12125fd924212909ca418e9 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1269,6 +1269,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
