diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index 4d7d8fa63e8313a8c97a3b8a5b4371c7415e5e56..9df52f0bd7284f962972e359817bd3a193b68fb8 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -103,6 +103,7 @@ const gfx::VectorIcon& GetIconIdDesktop(RequestType type) {
       return vector_icons::kCookieIcon;
     case RequestType::kWindowPlacement:
       return vector_icons::kSelectWindowIcon;
+    BRAVE_GET_ICON_ID_DESKTOP
   }
   NOTREACHED();
   return gfx::kNoneIcon;
