diff --git a/components/invalidation/impl/fcm_network_handler.cc b/components/invalidation/impl/fcm_network_handler.cc
index 7d6e308a7755c8527d915056c43559102d34f236..941def01683470af8725662f9b0ded165c8eb7ab 100644
--- a/components/invalidation/impl/fcm_network_handler.cc
+++ b/components/invalidation/impl/fcm_network_handler.cc
@@ -148,6 +148,7 @@ std::unique_ptr<syncer::FCMNetworkHandler> FCMNetworkHandler::Create(
 }
 
 void FCMNetworkHandler::StartListening() {
+  return; // feature disabled in Brave
   if (IsListening()) {
     StopListening();
   }
