diff --git a/components/password_manager/core/browser/password_store/BUILD.gn b/components/password_manager/core/browser/password_store/BUILD.gn
index 53f1a288f53939f35f7e5a64f337676051b1cc5e..b7fdf0740c02ae9141a30f14975243891c0d968c 100644
--- a/components/password_manager/core/browser/password_store/BUILD.gn
+++ b/components/password_manager/core/browser/password_store/BUILD.gn
@@ -76,6 +76,7 @@ source_set("password_store_impl") {
       "password_data_type_controller_delegate_android.cc",
       "password_data_type_controller_delegate_android.h",
     ]
+    import("//brave/components/password_manager/core/browser/password_store/sources.gni") sources += components_password_manager_core_browser_password_store_sources
     deps += [
       "//components/sync/base",
       "//components/sync/service",
