diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 2c01f0bf88a9c94e0bce718771a732d8fa40adbc..313093563faedced56c1fa86aeddb9beba16ec6a 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1386,6 +1386,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // via `HasContentSettingChangedViaPageInfo(type)`.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
