diff --git a/components/version_info/BUILD.gn b/components/version_info/BUILD.gn
index 931da8429699964ffe18dc37bf257edd577cf830..e3d7907fdf808e3fbfa5f4f179df0070d89537c3 100644
--- a/components/version_info/BUILD.gn
+++ b/components/version_info/BUILD.gn
@@ -11,6 +11,7 @@ declare_args() {
 }
 
 static_library("version_info") {
+  configs += [ "//brave/build:version" ]
   sources = [
     "version_info.cc",
     "version_info.h",
