diff --git a/components/invalidation/impl/fcm_network_handler.cc b/components/invalidation/impl/fcm_network_handler.cc
index 5383e8872b71bd5d18a839b2a623407d5aeb1368..e31e19f057a528cdeb6b1d8e0df1b5f3b9a621b2 100644
--- a/components/invalidation/impl/fcm_network_handler.cc
+++ b/components/invalidation/impl/fcm_network_handler.cc
@@ -126,6 +126,7 @@ std::unique_ptr<syncer::FCMNetworkHandler> FCMNetworkHandler::Create(
 }
 
 void FCMNetworkHandler::StartListening() {
+  return; // feature disabled in Brave
   if (IsListening()) {
     StopListening();
   }
