diff --git a/components/os_crypt/BUILD.gn b/components/os_crypt/BUILD.gn
index 9de162e23e0cfb70263b993693fde93f5fbd88bc..5cadd146eb68c5d05cc99acc097d880e232b26e1 100644
--- a/components/os_crypt/BUILD.gn
+++ b/components/os_crypt/BUILD.gn
@@ -114,6 +114,7 @@ component("os_crypt") {
       defines += [ "USE_KWALLET" ]
     }
   }
+  import("//brave/components/os_crypt/sources.gni") deps += brave_components_os_crypt_deps
 }
 
 static_library("test_support") {
