diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index fcac2d737c3da9fe9b42996e040248164623a395..d74cf3e35c40ff02905b98ee4fb5875076b937a0 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -379,6 +379,7 @@ static_library("browser") {
     "//ui/base",
     "//ui/gfx",
   ]
+  sources += brave_components_omnibox_browser_sources deps += brave_components_omnibox_browser_deps public_deps += brave_components_omnibox_browser_public_deps
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
 
@@ -998,6 +999,7 @@ fuzzer_test("search_suggestion_parser_fuzzer") {
 
 mojom("mojo_bindings") {
   sources = [ "omnibox.mojom" ]
+  generate_legacy_js_bindings = true
 
   webui_module_path = "chrome://resources/mojo/components/omnibox/browser"
 
