diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index d787d25a8613c8360c8a9eecbbacdaced393b783..23830d5db2b3a2ce10a4b6576ce85801919701cb 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -253,6 +253,7 @@ static_library("metrics") {
       "motherboard_metrics_provider.h",
     ]
   }
+  import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
 }
 
 # The component metrics provider is a separate target because it depends upon
