diff --git a/components/omnibox/browser/autocomplete_match.h b/components/omnibox/browser/autocomplete_match.h
index b9a16e98e6fe0af4622ef721bb8250dd4610f323..a66cbd8a75ea7ca7b5c2f4104a691b6b24772c02 100644
--- a/components/omnibox/browser/autocomplete_match.h
+++ b/components/omnibox/browser/autocomplete_match.h
@@ -712,7 +712,7 @@ struct AutocompleteMatch {
   // The provider of this match, used to remember which provider the user had
   // selected when the input changes. This may be NULL, in which case there is
   // no provider (or memory of the user's selection).
-  raw_ptr<AutocompleteProvider> provider = nullptr;
+  raw_ptr<AutocompleteProvider, DanglingUntriaged> provider = nullptr;
 
   // The relevance of this match. See table in autocomplete_provider.h for
   // scores returned by various providers. This is used to rank matches among
