diff --git a/components/policy/resources/webui/policy_base.ts b/components/policy/resources/webui/policy_base.ts
index 973695e7d0611dcf512648bccfaddb558b340c8f..ee72d03855d4169431e16602bbcb6f010cfe26ed 100644
--- a/components/policy/resources/webui/policy_base.ts
+++ b/components/policy/resources/webui/policy_base.ts
@@ -211,7 +211,7 @@ export class Page {
                       policyNames['precedence']?.policyNames,
                     ].includes(knownPolicyNames) &&
                             knownPolicyNamesSet.has(name) ?
-                        `https://chromeenterprise.google/policies/?policy=${
+                        `https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy?policy=${
                             name}` :
                         undefined,
                   },
