diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 9f1bfb3c97896193fd9c9e4328ccc465f908dbba..860340ebd22a6db1f9eac840224dc93a7fd35777 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -303,6 +303,7 @@ class SyncServiceImpl : public SyncService,
   size_t GetQueuedLocalDataMigrationItemCountForTest() const;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
