diff --git a/components/search_engines/BUILD.gn b/components/search_engines/BUILD.gn
index e4d0814de08d9ba0aa1a0baf62927074e9ece916..c18df9ae3dd44a39c72479f10cfded325f6b396a 100644
--- a/components/search_engines/BUILD.gn
+++ b/components/search_engines/BUILD.gn
@@ -201,6 +201,7 @@ json_to_struct("prepopulated_engines") {
   schema_file = "prepopulated_engines_schema.json"
   namespace = "TemplateURLPrepopulateData"
   deps = [ ":search_engine_type" ]
+  additional_sources = brave_components_search_engines_sources
 }
 
 if (is_android) {
