diff --git a/components/permissions/permission_util.cc b/components/permissions/permission_util.cc
index a061fc3a72df9f0e4f642343d21caf3bba0689a2..ea5973009edfc363bc590ebca231b93fcbe1b60f 100644
--- a/components/permissions/permission_util.cc
+++ b/components/permissions/permission_util.cc
@@ -290,6 +290,7 @@ ContentSettingsType PermissionUtil::PermissionTypeToContentSettingTypeSafe(
       return ContentSettingsType::DISPLAY_CAPTURE;
     case PermissionType::NUM:
       break;
+    PERMISSION_UTIL_PERMISSION_TYPE_TO_CONTENT_SETTINGS_TYPE
   }
 
   return ContentSettingsType::DEFAULT;
