diff --git a/components/crx_file/crx_verifier.cc b/components/crx_file/crx_verifier.cc
index a79b96206023b30f9e310e06fbdd90c6b9fb3ac4..75afca845310527c206a874c413c550fec5ef5ab 100644
--- a/components/crx_file/crx_verifier.cc
+++ b/components/crx_file/crx_verifier.cc
@@ -181,6 +181,7 @@ VerifierResult VerifyCrx3(
       found_publisher_key =
           found_publisher_key || key_hash == publisher_key ||
           (accept_publisher_test_key && key_hash == *publisher_test_key);
+      found_publisher_key = found_publisher_key || IsBravePublisher(key_hash);
       auto v = std::make_unique<crypto::SignatureVerifier>();
       static_assert(sizeof(unsigned char) == sizeof(uint8_t),
                     "Unsupported char size.");
