diff --git a/components/strings/BUILD.gn b/components/strings/BUILD.gn
index 3119b6bd0607dea7bf52ecf27432fb245038fcf1..3528a619d4e3eb395b28f188bef524926505c668 100644
--- a/components/strings/BUILD.gn
+++ b/components/strings/BUILD.gn
@@ -52,6 +52,7 @@ grit("components_strings") {
   if (is_android) {
     outputs += android_components_strings_java_resources
   }
+  deps = [ "//brave/components/resources:strings" ]
 }
 
 if (is_android) {
