diff --git a/components/download/internal/common/download_item_impl.cc b/components/download/internal/common/download_item_impl.cc
index fa74c65138cf1fe277490d8c62aee3b7990e5334..c0f1f3205bf54587959c35db2ad0530bff33b1f3 100644
--- a/components/download/internal/common/download_item_impl.cc
+++ b/components/download/internal/common/download_item_impl.cc
@@ -2061,8 +2061,8 @@ void DownloadItemImpl::OnDownloadCompleting() {
                      base::Unretained(download_file_.get()),
                      GetTargetFilePath(),
                      delegate_->GetApplicationClientIdForFileScanning(),
-                     delegate_->IsOffTheRecord() ? GURL() : GetURL(),
-                     delegate_->IsOffTheRecord() ? GURL() : GetReferrerUrl(),
+                     GURL(),  // Never leak download URLs in metadata.
+                     GURL(),  // See brave-browser#2766.
                      std::move(quarantine), std::move(rename_callback)));
 }
 
