diff --git a/components/crash/core/browser/resources/crashes.ts b/components/crash/core/browser/resources/crashes.ts
index 0d3658a8c854e8053c46836395502a3434a1b4b4..9f451948ba2eea9a3e9030ad148a286fbd7fa66f 100644
--- a/components/crash/core/browser/resources/crashes.ts
+++ b/components/crash/core/browser/resources/crashes.ts
@@ -141,7 +141,7 @@ function updateCrashList({
         crashLink.href = `https://goto.google.com/crash/${crash.id}`;
         crashLink.target = '_blank';
         crashLink.textContent = crash.id;
-        uploadIdValue.appendChild(crashLink);
+        // uploadIdValue.appendChild(crashLink); // feature disabled in Brave
       } else {
         uploadIdValue.textContent = crash.id;
       }
