diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index d5fccbc656d743ecc42700c744471c7f6a1d4764..0b81a247157e31c5d90d1952a9afbaaba93cf643 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1070,6 +1070,7 @@ void PageInfo::PresentSitePermissions() {
       }
     }
 
+    BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
     if (ShouldShowPermission(
             permission_info, site_url_, content_settings, web_contents_.get(),
             HasContentSettingChangedViaPageInfo(permission_info.type),
