diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index e088d1fd1a783f6d220817c1d50da9a321401402..62a0bef12097a8879d61f9992fc3bf7741a0ff21 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -6,7 +6,7 @@ import("//components/guest_view/buildflags/buildflags.gni")
 import("//device/vr/buildflags/buildflags.gni")
 import("//services/device/public/cpp/geolocation/buildflags.gni")
 
-source_set("permissions_common") {
+component("permissions_common") {
   sources = [
     "constants.cc",
     "constants.h",
@@ -159,6 +159,7 @@ source_set("permissions") {
   if (enable_guest_view) {
     deps += [ "//components/guest_view/browser" ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
