diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index aacb27b6a4ea4fe0727aa90588e66190b88038af..662cb03fb0474e847ef106ad045227397b4d27bb 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -345,6 +345,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
