diff --git a/components/download/internal/common/download_item_impl.cc b/components/download/internal/common/download_item_impl.cc
index f0248e7c544714ff716c9b91e1937d33758c4e9b..dc799c5b71b2852345e718ece7419bebb33cee87 100644
--- a/components/download/internal/common/download_item_impl.cc
+++ b/components/download/internal/common/download_item_impl.cc
@@ -1987,8 +1987,8 @@ void DownloadItemImpl::OnDownloadCompleting() {
           &DownloadFile::RenameAndAnnotate,
           base::Unretained(download_file_.get()), GetTargetFilePath(),
           delegate_->GetApplicationClientIdForFileScanning(),
-          delegate_->IsOffTheRecord() ? GURL() : GetURL(),
-          delegate_->IsOffTheRecord() ? GURL() : GetReferrerUrl(),
+          GURL(),  // Never leak download URLs in metadata.
+          GURL(),  // See brave-browser#2766.
           delegate_->IsOffTheRecord() ? std::nullopt : GetRequestInitiator(),
           std::move(quarantine), std::move(rename_callback)));
 }
