diff --git a/components/browser_actuator/public/features.cc b/components/browser_actuator/public/features.cc
index 88a4a36008bcbb030419d724e8846f27ce65707a..82d8d2f9fc6739203d242d19cef7af53e7cab382 100644
--- a/components/browser_actuator/public/features.cc
+++ b/components/browser_actuator/public/features.cc
@@ -24,6 +24,7 @@ const base::FeatureParam<std::string> kWatchSessionsPathParam{
 
 }  // namespace
 
+// kBrowserActuator feature state is enforced via plaster rewrite.
 BASE_FEATURE(kBrowserActuator, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kEnableBrowserActuatorForGlicExperimentalTriggering,
              base::FEATURE_DISABLED_BY_DEFAULT);
