diff --git a/components/embedder_support/user_agent_utils.cc b/components/embedder_support/user_agent_utils.cc
index 279a6ae5a54e5bec7422a65d9e01c2c88ebde22f..f389492327e61e5055e2cec588b582ef08cb68a8 100644
--- a/components/embedder_support/user_agent_utils.cc
+++ b/components/embedder_support/user_agent_utils.cc
@@ -193,10 +193,11 @@ const blink::UserAgentBrandList GetUserAgentBrandList(
 #if !BUILDFLAG(CHROMIUM_BRANDING)
   brand = version_info::GetProductName();
 #endif
+  BRAVE_GET_USER_AGENT_BRAND_LIST
 
   std::string brand_version =
       output_version_type == blink::UserAgentBrandVersionType::kFullVersion
-          ? full_version
+          ? BRAVE_BRAND_VERSION_OVERRIDE_FOR_FULL_BRAND_VERSION_TYPE
           : major_version;
 
   return GenerateBrandVersionList(major_version_number, brand, brand_version,
