diff --git a/components/component_updater/component_updater_service.h b/components/component_updater/component_updater_service.h
index c98bc5da30406725a0d20e64d7b9c38004fc56c0..f4c2c7c0f65e567afe8547427d9f72472a550b7e 100644
--- a/components/component_updater/component_updater_service.h
+++ b/components/component_updater/component_updater_service.h
@@ -162,6 +162,7 @@ class OnDemandUpdater {
   enum class Priority { BACKGROUND = 0, FOREGROUND = 1 };
 
   virtual ~OnDemandUpdater() = default;
+  BRAVE_COMPONENT_UPDATER_SERVICE_H_
 
  private:
   friend class OnDemandTester;
