diff --git a/components/tabs/impl/tab_strip_collection.cc b/components/tabs/impl/tab_strip_collection.cc
index bfb0b512753a58c0486b3a82669c17c024076e54..278b2bec3e988c591fb9b66cfbc6b9b07d1ef240 100644
--- a/components/tabs/impl/tab_strip_collection.cc
+++ b/components/tabs/impl/tab_strip_collection.cc
@@ -314,6 +314,24 @@ ChildrenPtrs TabStripCollection::GetTabsAndCollectionsForMove(
 }
 
 ScopedTab TabStripCollection::RemoveTabAtIndexRecursive(size_t index) {
+  #if !BUILDFLAG(IS_ANDROID)
+  {
+    TabInterface* tab_to_be_removed = GetTabAtIndexRecursive(index);
+    TabCollection* parent_collection =
+        tab_to_be_removed->GetParentCollection(GetPassKey());
+    CHECK(parent_collection);
+    if (parent_collection->type() == TabCollection::Type::TREE_NODE) {
+      auto tab = RemoveTabImpl(tab_to_be_removed);
+      if (auto* grand_parent = parent_collection->GetParentCollection();
+          grand_parent &&
+          grand_parent->type() == TabCollection::Type::GROUP &&
+          grand_parent->TabCountRecursive() == 0) {
+        RemoveTabCollectionImpl(grand_parent);
+      }
+      return tab;
+    }
+  }
+  #endif  // !BUILDFLAG(IS_ANDROID)
   TabInterface* tab_to_be_removed = GetTabAtIndexRecursive(index);
   TabCollection* parent_collection =
       tab_to_be_removed->GetParentCollection(GetPassKey());
