diff --git a/components/sync_preferences/BUILD.gn b/components/sync_preferences/BUILD.gn
index 094d0195f79d8a3646151c677dfaee29e7961de0..1e14d2c687f65ba8c7fc7cc25edecc42628e8e87 100644
--- a/components/sync_preferences/BUILD.gn
+++ b/components/sync_preferences/BUILD.gn
@@ -68,6 +68,7 @@ source_set("common_syncable_prefs_database") {
     "//components/translate/core/browser",
     "//components/variations/service",
   ]
+  import("//brave/components/sync_preferences/sources.gni") public_deps = brave_common_syncable_prefs_database_public_deps
 }
 
 static_library("test_support") {
