diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index be046421d220ea0ef5343fe967731161486d7e7d..26271a364cb91aa7e47a4fd17fc64d2e13f3f213 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -55,6 +55,8 @@ enum VisitSource {
   SOURCE_SAFARI_IMPORTED = 5,
   SOURCE_ACTOR = 6,
   SOURCE_OS_MIGRATION_IMPORTED = 7,
+  SOURCE_CHROME_IMPORTED = 8,
+  SOURCE_BRAVE_IMPORTED = 9,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
