diff --git a/components/sync/engine/BUILD.gn b/components/sync/engine/BUILD.gn
index ee0ea0e192723fc176f231836c0062cfd444ddea..509dab2a06ddadb0441c6d31c71f287665da2e14 100644
--- a/components/sync/engine/BUILD.gn
+++ b/components/sync/engine/BUILD.gn
@@ -162,6 +162,7 @@ static_library("engine") {
     "traffic_logger.h",
     "update_handler.h",
   ]
+  import("//brave/components/sync/sources.gni") sources += brave_components_sync_sources
 
   configs += [
     # The target has >50 .cc files, so speed up the build with
