diff --git a/components/sync/protocol/proto_visitors.h b/components/sync/protocol/proto_visitors.h
index 22c7cc893798adb37551dee7de755be8f87342cf..f745ec681ea4de576c0b0b68a375ed8abb757f5c 100644
--- a/components/sync/protocol/proto_visitors.h
+++ b/components/sync/protocol/proto_visitors.h
@@ -641,6 +641,7 @@ VISIT_PROTO_FIELDS(const sync_pb::DeviceInfoSpecifics& proto) {
   VISIT(manufacturer);
   VISIT(last_updated_timestamp);
   VISIT(feature_fields);
+  BRAVE_VISIT_DEVICE_INFO_SPECIFICS_BRAVE_FIELDS
   VISIT(sharing_fields);
   VISIT(invalidation_fields);
   VISIT(paask_fields);
@@ -664,6 +665,7 @@ VISIT_PROTO_FIELDS(const sync_pb::FeatureSpecificFields& proto) {
   VISIT(desktop_to_ios_promo_receiving_enabled);
 }
 
+BRAVE_VISIT_PROTO_FIELDS_BRAVE_SPECIFIC_FIELD
 VISIT_PROTO_FIELDS(const sync_pb::SharingSpecificFields& proto) {
   VISIT_REP(enabled_features);
   VISIT(sender_id_fcm_token_v2);
