diff --git a/components/password_manager/core/browser/password_manager_constants.h b/components/password_manager/core/browser/password_manager_constants.h
index 1342362f073e3fd2f5811440899102c9eb8b9b3e..324a00c79ce2d9f4793d1dd7140b71a55dae730a 100644
--- a/components/password_manager/core/browser/password_manager_constants.h
+++ b/components/password_manager/core/browser/password_manager_constants.h
@@ -11,12 +11,10 @@
 
 namespace password_manager {
 
-#if !BUILDFLAG(IS_ANDROID)
 extern const base::FilePath::CharType kLoginDataForProfileFileName[];
 extern const base::FilePath::CharType kLoginDataForAccountFileName[];
 extern const base::FilePath::CharType kLoginDataJournalForProfileFileName[];
 extern const base::FilePath::CharType kLoginDataJournalForAccountFileName[];
-#endif  // !BUILDFLAG(IS_ANDROID)
 
 // URL to the password manager account dashboard.
 extern const char kPasswordManagerAccountDashboardURL[];
