diff --git a/components/content_settings/core/browser/content_settings_policy_provider.cc b/components/content_settings/core/browser/content_settings_policy_provider.cc
index 088c159f20ebd8a5bf1b6440a4b39ada079f80a0..512eb3302d53b072eb2958e7ce536952dad1a75a 100644
--- a/components/content_settings/core/browser/content_settings_policy_provider.cc
+++ b/components/content_settings/core/browser/content_settings_policy_provider.cc
@@ -38,6 +38,7 @@ struct PrefsForManagedContentSettingsMapEntry {
 
 constexpr PrefsForManagedContentSettingsMapEntry
     kPrefsForManagedContentSettingsMap[] = {
+        BRAVE_MANAGED_CONTENT_SETTINGS
         {prefs::kManagedAutomaticFullscreenAllowedForUrls,
          ContentSettingsType::AUTOMATIC_FULLSCREEN, CONTENT_SETTING_ALLOW},
         {prefs::kManagedAutomaticFullscreenBlockedForUrls,
@@ -124,6 +125,7 @@ constexpr PrefsForManagedContentSettingsMapEntry
 };
 
 constexpr const char* kManagedPrefs[] = {
+    BRAVE_MANAGED_PREFS
     prefs::kManagedAutomaticFullscreenAllowedForUrls,
     prefs::kManagedAutomaticFullscreenBlockedForUrls,
     prefs::kManagedAutoSelectCertificateForUrls,
