diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index a17658bd32f42b9582f28ccb3e7b68855caa82b2..bfb0518fe0aa1c60c0dec1d741fbf6131dd05460 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -309,6 +309,7 @@ void PermissionManager::RequestPermissionsFromCurrentDocument(
     base::OnceCallback<void(const std::vector<PermissionStatus>&)>
         permission_status_callback) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
+  BRAVE_REQUEST_PERMISSION_FROM_CURRENT_DOCUMENT
   RequestPermissionsInternal(render_frame_host, request_description,
                              std::move(permission_status_callback));
 }
@@ -356,6 +357,7 @@ PermissionManager::GetPermissionResultForCurrentDocument(
       PermissionUtil::PermissionTypeToContentSettingType(permission);
 
   const GURL requesting_origin =
+      BRAVE_GET_PERMISSION_RESULT_FOR_CURRENT_DOCUMENT
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   const GURL embedding_origin =
       GetEmbeddingOrigin(render_frame_host, requesting_origin);
