diff --git a/components/autofill/core/browser/single_field_form_filler.h b/components/autofill/core/browser/single_field_form_filler.h
index 1bb17506e58d0383e197f9fd6bd51c53115da5ea..cea994ad060f40e9d4c831cda4ec266f34ca8b21 100644
--- a/components/autofill/core/browser/single_field_form_filler.h
+++ b/components/autofill/core/browser/single_field_form_filler.h
@@ -68,7 +68,7 @@ class SingleFieldFormFiller {
   // incognito and if Autocomplete is enabled or not. |fields| can be empty.
   virtual void OnWillSubmitFormWithFields(
       const std::vector<FormFieldData>& fields,
-      bool is_autocomplete_enabled) = 0;
+      const AutofillClient& client) = 0;
 
   // Cancels the currently pending WebDataService queries associated with the
   // given |handler|.
