diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 9a95559b2900c0273a66e03db37de4d548d2e038..a1b5f2fce904c0ac53bc2ab5bc8169a58e51cbbb 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -52,6 +52,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
