diff --git a/components/history/core/browser/BUILD.gn b/components/history/core/browser/BUILD.gn
index 4ac82e35ccdfcf1b13a776cdc7c7fa4c26281308..71c26758920884aabbcbca083e309a4233a251b6 100644
--- a/components/history/core/browser/BUILD.gn
+++ b/components/history/core/browser/BUILD.gn
@@ -2,6 +2,8 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/components/history/core/browser/sources.gni")
+
 component("features") {
   sources = [
     "features.cc",
@@ -127,6 +129,7 @@ static_library("browser") {
     "web_history_service.h",
     "web_history_service_observer.h",
   ]
+  sources += brave_components_history_core_browser_sources
 
   public_deps = [
     ":features",
@@ -165,6 +168,7 @@ static_library("browser") {
     "//ui/gfx",
     "//url",
   ]
+  deps += brave_components_history_core_browser_deps
 }
 
 source_set("unit_tests") {
