diff --git a/components/content_settings/core/common/content_settings_types.h b/components/content_settings/core/common/content_settings_types.h
index e465ffdd518f4de20983ea2a561150230b33cd1c..4051520fec56f40956da351370d39f77bcc1d9cc 100644
--- a/components/content_settings/core/common/content_settings_types.h
+++ b/components/content_settings/core/common/content_settings_types.h
@@ -365,6 +365,22 @@ enum class ContentSettingsType : int32_t {
   // request.
   FILE_SYSTEM_ACCESS_EXTENDED_PERMISSION,
 
+  BRAVE_START,
+  BRAVE_ADS = BRAVE_START,
+  BRAVE_COSMETIC_FILTERING,
+  BRAVE_TRACKERS,
+  BRAVE_HTTP_UPGRADABLE_RESOURCES,
+  BRAVE_FINGERPRINTING_V2,
+  BRAVE_SHIELDS,
+  BRAVE_REFERRERS,
+  BRAVE_COOKIES,
+  BRAVE_SPEEDREADER,
+  BRAVE_ETHEREUM,
+  BRAVE_SOLANA,
+  BRAVE_GOOGLE_SIGN_IN,
+  BRAVE_HTTPS_UPGRADE,
+  BRAVE_REMEMBER_1P_STORAGE,
+  BRAVE_LOCALHOST_ACCESS,
   NUM_TYPES,
 };
 
