diff --git a/components/sync/driver/glue/sync_engine_impl.h b/components/sync/driver/glue/sync_engine_impl.h
index 9fce092e22ca630fa3f5efa0bc1e963a86659625..6b1c580ca9be2d3bdb3bf30221e13922629cd250 100644
--- a/components/sync/driver/glue/sync_engine_impl.h
+++ b/components/sync/driver/glue/sync_engine_impl.h
@@ -102,6 +102,12 @@ class SyncEngineImpl : public SyncEngine, public InvalidationHandler {
   std::string GetOwnerName() const override;
   void OnInvalidatorClientIdChange(const std::string& client_id) override;
 
+  void HandleNudgeSyncCycle(brave_sync::RecordsListPtr records_list);
+  void HandlePollSyncCycle(GetRecordsCallback cb,
+                           base::WaitableEvent* wevent);
+  void DispatchGetRecordsCallback(
+      GetRecordsCallback cb, std::unique_ptr<RecordsList> records) override;
+
  protected:
   // The types and functions below are protected so that test
   // subclasses can use them.
