diff --git a/components/sync/driver/sync_service_impl.h b/components/sync/driver/sync_service_impl.h
index bcfafe1580ad1b3ab0012921b021b8b75dcc54af..710b14b8dd0edd28ae100cc8e9dca401dd16d8f1 100644
--- a/components/sync/driver/sync_service_impl.h
+++ b/components/sync/driver/sync_service_impl.h
@@ -238,6 +238,7 @@ class SyncServiceImpl : public SyncService,
 
   SyncClient* GetSyncClientForTest();
 
+  BRAVE_SYNC_SERVICE_IMPL_H_
  private:
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
