diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index a00a7d0aff66317761782d4b1c0903fc80c8ec34..116ed9d7816b4fa6a213665cd9783bb66c8a08ac 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -262,6 +262,7 @@ jumbo_static_library("browser") {
   if ((!is_android || enable_vr) && !is_ios) {
     deps += [ ":vector_icons" ]
   }
+  if (brave_chromium_build) {  deps += [ "//brave/components/omnibox/browser" ] }
 
   if (is_android) {
     sources += [
