diff --git a/components/password_manager/core/browser/generation/password_generator.cc b/components/password_manager/core/browser/generation/password_generator.cc
--- a/components/password_manager/core/browser/generation/password_generator.cc
+++ b/components/password_manager/core/browser/generation/password_generator.cc
@@ -39,7 +39,7 @@ constexpr char kAlphabeticChars[] =
     "abcdefghijkmnpqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ";
 constexpr char kDigits[] = "23456789";
-constexpr char kSymbols[] = "-_.:!";
+constexpr char kSymbols[] = "-_.:!@#$%^&*()+=?";
 
 // Returns a default password requirements specification that requires:
@@ -74,8 +74,8 @@ PasswordRequirementsSpec BuildDefaultSpec() {
 
   spec.mutable_symbols()->set_character_set(kSymbols);
-  spec.mutable_symbols()->set_min(0);
-  spec.mutable_symbols()->set_max(0);
+  spec.mutable_symbols()->set_min(1);
+  spec.mutable_symbols()->set_max(std::numeric_limits<int32_t>::max());
   return spec;
 }
