diff --git a/components/content_settings/core/common/content_settings.cc b/components/content_settings/core/common/content_settings.cc
index bca95d5b6b3b2d6576d14a8a5ed973c2156d063a..0eae3d4cc354a7a339172f99db0fe20969aa2474 100644
--- a/components/content_settings/core/common/content_settings.cc
+++ b/components/content_settings/core/common/content_settings.cc
@@ -100,6 +100,7 @@ constexpr HistogramValue kHistogramValue[] = {
     {ContentSettingsType::JAVASCRIPT_JIT, 79},
     {ContentSettingsType::HTTP_ALLOWED, 80},
     {ContentSettingsType::FORMFILL_METADATA, 81},
+    BRAVE_HISTOGRAM_VALUE_LIST
 };
 
 }  // namespace
