diff --git a/components/signin/public/base/session_binding_utils.cc b/components/signin/public/base/session_binding_utils.cc
index 8894174ebc3b38a4680f11cf0ee92edd5512684f..4a0f446c42459a8d0ff7590ca36f80027c5a5670 100644
--- a/components/signin/public/base/session_binding_utils.cc
+++ b/components/signin/public/base/session_binding_utils.cc
@@ -71,6 +71,8 @@ std::optional<std::string_view> SignatureAlgorithmToString(
       return "ML-DSA-65";
     case crypto::SignatureVerifier::MLDSA_87:
       return "ML-DSA-87";
+    case crypto::SignatureVerifier::ECDSA_SHA384:
+      return "SHA384";
   }
 }
 
