diff --git a/components/sync_device_info/device_info.h b/components/sync_device_info/device_info.h
index 0de973ef3cfb970cfdbc773cc8be98ae6d384750..23f5d53a8e07152299850f6998abd9c08d164792 100644
--- a/components/sync_device_info/device_info.h
+++ b/components/sync_device_info/device_info.h
@@ -197,6 +197,7 @@ class DeviceInfo {
   DeviceInfo(const DeviceInfo&) = delete;
   DeviceInfo& operator=(const DeviceInfo&) = delete;
 
+  virtual
   ~DeviceInfo();
 
   // Sync specific unique identifier for the device. Note if a device
