diff --git a/components/gcm_driver/gcm_client_impl.cc b/components/gcm_driver/gcm_client_impl.cc
index f4580329d84ef5c2c364d87601d982f069607454..b59532e673c977f69dbe1511ab266f685aa1e5d8 100644
--- a/components/gcm_driver/gcm_client_impl.cc
+++ b/components/gcm_driver/gcm_client_impl.cc
@@ -356,6 +356,7 @@ void GCMClientImpl::Initialize(
 }
 
 void GCMClientImpl::Start(StartMode start_mode) {
+  return; // GCM disabled in Brave
   DCHECK_NE(UNINITIALIZED, state_);
 
   if (state_ == LOADED) {
@@ -863,6 +864,7 @@ void GCMClientImpl::ResetCache() {
 
 void GCMClientImpl::Register(
     scoped_refptr<RegistrationInfo> registration_info) {
+  return; // GCM disabled in Brave
   DCHECK_EQ(state_, READY);
 
   // Registrations should never pass as an app_id the special category used
