diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index ec4e134b36aa2fbebf5705d28c566625a497983e..c7f77ddf35c10f5568aafe62239b03035f2cd465 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -199,6 +199,7 @@ android_library("java") {
     "//third_party/jni_zero:jni_zero_java",
     "//ui/android:ui_java",
   ]
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
