diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index a5df7891cce0c939e6a24b68f134fb15a94ca6d9..2691da12d54f002e54a66a7796bcc2585854d8ec 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -291,6 +291,7 @@ void PermissionManager::RequestPermissionsFromCurrentDocument(
     base::OnceCallback<void(const std::vector<PermissionStatus>&)>
         permission_status_callback) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
+  BRAVE_REQUEST_PERMISSION_FROM_CURRENT_DOCUMENT
   RequestPermissionsInternal(render_frame_host, request_description,
                              std::move(permission_status_callback));
 }
@@ -343,6 +344,7 @@ PermissionManager::GetPermissionResultForCurrentDocument(
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
 
   const GURL requesting_origin =
+      BRAVE_GET_PERMISSION_RESULT_FOR_CURRENT_DOCUMENT
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   const GURL embedding_origin =
       GetEmbeddingOrigin(render_frame_host, requesting_origin);
