diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index e6b4c24e2e7c716023c32ff669ec0554e1d63d51..324337f8ec41b491b3955b8c95ce729b56fe542d 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -6,7 +6,7 @@ import("//components/guest_view/buildflags/buildflags.gni")
 import("//device/vr/buildflags/buildflags.gni")
 import("//services/device/public/cpp/geolocation/buildflags.gni")
 
-source_set("permissions_common") {
+component("permissions_common") {
   sources = [
     "constants.cc",
     "constants.h",
@@ -167,6 +167,7 @@ source_set("permissions") {
   if (enable_guest_view) {
     deps += [ "//components/guest_view/browser" ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
