diff --git a/components/sync_preferences/BUILD.gn b/components/sync_preferences/BUILD.gn
index 1f1ce89f1b38e41e57fd0e694258a61f7f2549d0..ee9072caa6ec0654eeb8553e9b01720d7b4daa8f 100644
--- a/components/sync_preferences/BUILD.gn
+++ b/components/sync_preferences/BUILD.gn
@@ -92,6 +92,7 @@ source_set("common_syncable_prefs_database") {
     "//components/translate/core/browser",
     "//components/variations/service",
   ]
+  import("//brave/components/sync_preferences/sources.gni") public_deps = brave_common_syncable_prefs_database_public_deps
 }
 
 static_library("test_support") {
