diff --git a/components/sync/base/model_type.h b/components/sync/base/model_type.h
index d751ce7b3c9618cca3bf48e2cc9c640be9fa9d22..3b18b631a8db839c9a0ee820fea9f4b52b2b9890 100644
--- a/components/sync/base/model_type.h
+++ b/components/sync/base/model_type.h
@@ -244,6 +244,7 @@ constexpr ModelTypeSet ProxyTypes() {
 // - They support custom update application and conflict resolution logic.
 // - All change processing occurs on the sync thread (GROUP_PASSIVE).
 constexpr ModelTypeSet ControlTypes() {
+  return ModelTypeSet(UNSPECIFIED);
   return ModelTypeSet(NIGORI);
 }
 
