diff --git a/components/history/core/browser/BUILD.gn b/components/history/core/browser/BUILD.gn
index 30c7b55cf4fda17c4928410b0912d3c64c127403..b2f4b0e648921e8afe7cf8696e3348d3ec65b4bd 100644
--- a/components/history/core/browser/BUILD.gn
+++ b/components/history/core/browser/BUILD.gn
@@ -157,6 +157,7 @@ static_library("browser") {
     "//ui/gfx",
     "//url",
   ]
+  import("//brave/components/history/core/browser/sources.gni") sources += brave_components_history_core_browser_sources deps += brave_components_history_core_browser_deps
 }
 
 source_set("unit_tests") {
