diff --git a/components/crash/core/app/crash_reporter_client.cc b/components/crash/core/app/crash_reporter_client.cc
index 4abce141b758eabd8a16f1cccb0016d5f10c0cae..9ce974445becb07b3534a57bbafea46a73972e3d 100644
--- a/components/crash/core/app/crash_reporter_client.cc
+++ b/components/crash/core/app/crash_reporter_client.cc
@@ -194,6 +194,7 @@ void CrashReporterClient::GetSanitizationInformation(
 #endif
 
 std::string CrashReporterClient::GetUploadUrl() {
+  BRAVE_CRASH_REPORTER_CLIENT_GET_UPLOAD_URL
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING) && defined(OFFICIAL_BUILD)
   return kDefaultUploadURL;
 #else
