diff --git a/components/sync_device_info/device_info_sync_bridge.h b/components/sync_device_info/device_info_sync_bridge.h
index 6a9020e3d7a4fbe5788deb8d7a3e5c897115ed78..4b6911f88ee65bbf235675476cfaadfebd93bb88 100644
--- a/components/sync_device_info/device_info_sync_bridge.h
+++ b/components/sync_device_info/device_info_sync_bridge.h
@@ -61,6 +61,11 @@ class DeviceInfoSyncBridge : public DataTypeSyncBridge,
   DeviceInfoSyncBridge& operator=(const DeviceInfoSyncBridge&) = delete;
 
   ~DeviceInfoSyncBridge() override;
+  void DeleteDeviceInfo(const std::string& client_id,
+                        base::OnceClosure callback) override;
+  std::vector<DeviceInfo> GetAllBraveDeviceInfo() const override;
+  void OnDeviceInfoDeleted(const std::string& client_id, const int attempt,
+                           base::OnceClosure callback);
 
   LocalDeviceInfoProvider* GetLocalDeviceInfoProvider();
 
