diff --git a/components/sync/driver/sync_service_impl.h b/components/sync/driver/sync_service_impl.h
index aabb4c49b4d6a64777646e4d3b9a5bfcff5a3624..75a757286d902e983768f16f7c101ebdf816e6e2 100644
--- a/components/sync/driver/sync_service_impl.h
+++ b/components/sync/driver/sync_service_impl.h
@@ -237,6 +237,7 @@ class SyncServiceImpl : public SyncService,
 
   SyncClient* GetSyncClientForTest();
 
+  BRAVE_SYNC_SERVICE_IMPL_H_
  private:
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
