diff --git a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsCategory.java b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsCategory.java
index 89a35331bc43fd2814974e8d46b5e63abbc3e848..6e27dc18a75665616ba8653a1ba261b733dd5da7 100644
--- a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsCategory.java
+++ b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsCategory.java
@@ -84,6 +84,7 @@ public class SiteSettingsCategory {
         Type.AUTO_PICTURE_IN_PICTURE,
         Type.LOCAL_NETWORK,
         Type.LOOPBACK_NETWORK,
+        Type.AUTOPLAY, Type.BRAVE_GOOGLE_SIGN_IN, Type.BRAVE_LOCALHOST_ACCESS,
         Type.NUM_ENTRIES
     })
     @Retention(RetentionPolicy.SOURCE)
@@ -132,7 +133,7 @@ public class SiteSettingsCategory {
         int LOOPBACK_NETWORK = 39;
 
         /** Number of handled categories used for calculating array sizes. */
-        int NUM_ENTRIES = 40;
+        int AUTOPLAY = 40; int BRAVE_GOOGLE_SIGN_IN = 41; int BRAVE_LOCALHOST_ACCESS = 42; int NUM_ENTRIES = 43;
     }
 
     private final BrowserContextHandle mBrowserContextHandle;
