diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 2196cb3e2c620652a5cefe77bc7369a63f19ad06..1675fb8807744863e7d085c3fb46afabce6a5f7b 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1160,6 +1160,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
