diff --git a/components/sync_device_info/device_info_sync_bridge.cc b/components/sync_device_info/device_info_sync_bridge.cc
index 65d3fa026cb568ab73d59504151a4cc42dde65ca..10731f92514ef741d7bb5c1c4b572813cf49d1fa 100644
--- a/components/sync_device_info/device_info_sync_bridge.cc
+++ b/components/sync_device_info/device_info_sync_bridge.cc
@@ -215,7 +215,8 @@ DeviceInfo SpecificsToModel(const DeviceInfoSpecifics& specifics) {
       SpecificsToAutoSignOutLastSigninTimestamp(specifics),
       specifics.feature_fields().desktop_to_ios_promo_receiving_enabled(),
       SpecificsToDesktopToIOSPromoReceivingTypes(specifics),
-      SpecificsToGlicExperimentalTriggeringState(specifics));
+      SpecificsToGlicExperimentalTriggeringState(specifics),
+      SpecificsToSelfDeleteSupport(specifics));
 }
 
 // Allocate a EntityData and copies |specifics| into it.
@@ -241,7 +242,7 @@ sync_pb::PhoneAsASecurityKeySpecificFields PhoneAsASecurityKeyInfoToProto(
 }
 
 // Converts a local DeviceInfo into a freshly allocated DeviceInfoSpecifics.
-std::unique_ptr<DeviceInfoSpecifics> MakeLocalDeviceSpecifics(
+std::unique_ptr<DeviceInfoSpecifics> MakeLocalDeviceSpecifics_ChromiumImpl(
     const DeviceInfo& info) {
   auto specifics = std::make_unique<DeviceInfoSpecifics>();
   specifics->set_cache_guid(info.guid());
@@ -438,7 +439,7 @@ LocalDeviceInfoProvider* DeviceInfoSyncBridge::GetLocalDeviceInfoProvider() {
   return local_device_info_provider_.get();
 }
 
-void DeviceInfoSyncBridge::RefreshLocalDeviceInfoIfNeeded() {
+void DeviceInfoSyncBridge::RefreshLocalDeviceInfoIfNeeded_ChromiumImpl() {
   // Device info cannot be synced if the provider is not initialized. When it
   // gets initialized, local device info will be sent.
   if (!local_device_info_provider_->GetLocalDeviceInfo()) {
@@ -528,17 +529,19 @@ std::optional<ModelError> DeviceInfoSyncBridge::ApplyIncrementalSyncChanges(
   for (const std::unique_ptr<EntityChange>& change : entity_changes) {
     const std::string guid = change->storage_key();
 
+#if 0  // Brave: re-upload defeats removing other devices in the sync chain
     // Reupload local device if it was deleted from the server.
     if (local_cache_guid_ == guid &&
         change->type() == EntityChange::ACTION_DELETE) {
       has_tombstone_for_local_device = true;
       continue;
     }
+#endif
 
     // Ignore any remote changes that have a cache guid that is or was this
     // local device.
     if (device_info_prefs_->IsRecentLocalCacheGuid(guid)) {
-      continue;
+      if (change->type() != EntityChange::ACTION_DELETE) continue;
     }
 
     if (change->type() == EntityChange::ACTION_DELETE) {
@@ -890,6 +893,10 @@ void DeviceInfoSyncBridge::OnReadAllMetadata(
     return;
   }
 
+  if (!device_info_prefs_->IsResetDevicesProgressTokenDone()) {
+    metadata_batch->ClearProgressToken();
+    device_info_prefs_->SetResetDevicesProgressTokenDone();
+  }
   // In the regular case for sync being disabled, wait for MergeFullSyncData()
   // before initializing the LocalDeviceInfoProvider.
   if (!syncer::IsInitialSyncDone(
@@ -1132,6 +1139,7 @@ DeviceInfoSyncBridge::CountActiveDevicesByType() const {
 }
 
 void DeviceInfoSyncBridge::ExpireOldEntries() {
+  return;
   DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
   CHECK(store_);
   TRACE_EVENT0("sync", "DeviceInfoSyncBridge::ExpireOldEntries");
