diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 2e71fc91ec9b92704326cb4ab4dd274d8223f654..493f2cec0359e6d5bd299eab00c5adb68b037c6f 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -52,6 +52,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
