diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index 40a9b5551fed1910b43272bb3b15c990d55b061e..04e0ada7fcfe5cd6bc2eb2ae80c6e9d4984f64d6 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -130,6 +130,7 @@ ContentSettingsType PermissionTypeToContentSettingSafe(
       return ContentSettingsType::WINDOW_PLACEMENT;
     case PermissionType::FONT_ACCESS:
       return ContentSettingsType::FONT_ACCESS;
+    BRAVE_PERMISSION_MANAGER_PERMISSION_TYPE_TO_CONTENT_SETTING_SAFE
     case PermissionType::NUM:
       break;
   }
