diff --git a/components/permissions/permission_request_manager.cc b/components/permissions/permission_request_manager.cc
index a53d9cabc20384a73e8855ba8000f98cfed3139a..ec8057060b2d2fe639034283e9eff29f9ca891dd 100644
--- a/components/permissions/permission_request_manager.cc
+++ b/components/permissions/permission_request_manager.cc
@@ -581,6 +581,7 @@ GURL PermissionRequestManager::GetRequestingOrigin() const {
   GURL origin = requests_.front()->requesting_origin();
   if (DCHECK_IS_ON()) {
     for (permissions::PermissionRequest* request : requests_) {
+      BRAVE_PERMISSION_REQUEST_MANAGER_GET_REQUESTING_ORIGIN
       DCHECK_EQ(origin, request->requesting_origin());
     }
   }
