diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index d6187539a6521c4c9caa669ea8211e034a5dacb7..117884cbed192907f30f751d3201fc8f8d78edfe 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -298,6 +298,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
