diff --git a/components/sync_device_info/device_info.h b/components/sync_device_info/device_info.h
index 0de973ef3cfb970cfdbc773cc8be98ae6d384750..7605a23c9386910e5f8ed76616309a47e21fbcd2 100644
--- a/components/sync_device_info/device_info.h
+++ b/components/sync_device_info/device_info.h
@@ -192,12 +192,27 @@ class DeviceInfo {
              std::optional<base::Time> auto_sign_out_last_signin_timestamp,
              bool desktop_to_ios_promo_receiving_enabled,
              const MobilePromoOnDesktopPromoTypeSet&
-                 desktop_to_ios_promo_receiving_types = {});
+                 desktop_to_ios_promo_receiving_types = {},
+             SelfDeleteSupport self_delete_support =
+                 SelfDeleteSupport::kNotSupported);
 
   DeviceInfo(const DeviceInfo&) = delete;
   DeviceInfo& operator=(const DeviceInfo&) = delete;
 
   ~DeviceInfo();
+  SelfDeleteSupport self_delete_support() const {
+    return self_delete_support_;
+  }
+  void set_self_delete_support(SelfDeleteSupport self_delete_support) {
+    self_delete_support_ = self_delete_support;
+  }
+  // Gets the OS in string form.
+  std::string GetOSString() const;
+  // Gets the device type in string form.
+  std::string GetDeviceTypeString() const;
+  // Converts the DeviceInfo values to a JS friendly DictionaryValue,
+  // which extension APIs can expose to third party apps.
+  base::DictValue ToValue() const;
 
   // Sync specific unique identifier for the device. Note if a device
   // is wiped and sync is set up again this id WILL be different.
@@ -372,6 +387,7 @@ class DeviceInfo {
   // NOTE: when adding a member, don't forget to update
   // |StoredDeviceInfoStillAccurate| in device_info_sync_bridge.cc or else
   // changes in that member might not trigger uploads of updated DeviceInfos.
+  SelfDeleteSupport self_delete_support_;
 };
 
 }  // namespace syncer
