diff --git a/components/permissions/request_type.h b/components/permissions/request_type.h
index f272cfcfe249a45b25c5705261e725639a78136d..a388a07ee326ae5819060b99d49b9f6472f85cba 100644
--- a/components/permissions/request_type.h
+++ b/components/permissions/request_type.h
@@ -50,6 +50,7 @@ enum class RequestType {
 #endif
   kStorageAccess,
   kVrSession,
+  BRAVE_REQUEST_TYPES
 #if !defined(OS_ANDROID)
   kWindowPlacement,
   kMaxValue = kWindowPlacement
