diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 8064739c6ef26e5d16f631ea419a4f5fe590496c..0804357e6bb700b2ab2879ef64506d3c603c849c 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -248,6 +248,7 @@ static_library("browser") {
     "//ui/base",
     "//ui/gfx",
   ]
+  if (brave_chromium_build) {  deps += [ "//brave/components/omnibox/browser" ] }
 
   if ((!is_android || enable_vr) && !is_ios) {
     deps += [ ":vector_icons" ]
