diff --git a/components/contextual_tasks/public/features.cc b/components/contextual_tasks/public/features.cc
index 4882bda98d1efa215759d03d7bfcf8383eec49d0..d52c2ea8de74969a8639c000c18fd3695128f7ea 100644
--- a/components/contextual_tasks/public/features.cc
+++ b/components/contextual_tasks/public/features.cc
@@ -35,6 +35,7 @@ std::optional<bool> g_sticky_conversation_enabled_override;
 namespace contextual_tasks {
 
 // Enables the contextual tasks side panel while browsing.
+// kContextualTasks feature state is enforced via plaster rewrite.
 BASE_FEATURE(kContextualTasks, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kContextualTasksPrivateApiNoAnimation,
              base::FEATURE_DISABLED_BY_DEFAULT);
@@ -175,6 +176,7 @@ BASE_FEATURE(kContextualTasksOverrideShowBottomSheetOnLargeScreen,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Enables prefetching of cookies for contextual tasks.
+// kContextualTasksCookiePrefetch feature state is enforced via plaster rewrite.
 BASE_FEATURE(kContextualTasksCookiePrefetch, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kAimTriggeredThreadLinks, base::FEATURE_ENABLED_BY_DEFAULT);
