diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index 9778fe4153893b9fdd75c27fb84bd71728410e6b..2842ba36b463ef7184567d43b912949328cb29b6 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/components/metrics/sources.gni")
 import("//build/config/cronet/config.gni")
 import("//build/config/features.gni")
 import("//testing/test.gni")
@@ -339,6 +340,7 @@ if (!is_cronet_build || is_ios) {
       "//ui/base",
       "//url",
     ]
+    deps += brave_components_metrics_deps
 
     if (is_android) {
       sources += [
