diff --git a/components/history/core/browser/features.cc b/components/history/core/browser/features.cc
index 8fed971b9cd0ca520a331e5b109a55875d15ffc6..861a0dad23914bd5317b52ea9fe25b57bc9eaef1 100644
--- a/components/history/core/browser/features.cc
+++ b/components/history/core/browser/features.cc
@@ -26,7 +26,7 @@ constexpr int kMaxNumRepeatableQueriesDefaultValue =
 
 // If enabled, the most repeated queries from the user browsing history are
 // shown in the Most Visited tiles.
-BASE_FEATURE(kOrganicRepeatableQueries, kOrganicRepeatableQueriesDefaultValue);
+BASE_OVERRIDDEN_FEATURE(kOrganicRepeatableQueries, base::FEATURE_DISABLED_BY_DEFAULT);
 
 // The maximum number of repeatable queries to show in the Most Visited tiles.
 const base::FeatureParam<int> kMaxNumRepeatableQueries(
