diff --git a/components/omnibox/browser/aim_eligibility_service_features.cc b/components/omnibox/browser/aim_eligibility_service_features.cc
index c23cef8903ea8fba76e17a03558a1f4580144300..8702b20ebfae5aa21b43ced6e3b086df4ad35f72 100644
--- a/components/omnibox/browser/aim_eligibility_service_features.cc
+++ b/components/omnibox/browser/aim_eligibility_service_features.cc
@@ -8,7 +8,7 @@
 
 namespace omnibox {
 
-BASE_FEATURE(kAimEnabled, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kAimEnabled, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kAimServerEligibilityEnabled, base::FEATURE_ENABLED_BY_DEFAULT);
 
