diff --git a/components/sync_preferences/BUILD.gn b/components/sync_preferences/BUILD.gn
index 31e2d14b1d3b979121951e843099bc7b528e5960..4b6ff666c62b8f598b4df99a949c2c10ba636bd3 100644
--- a/components/sync_preferences/BUILD.gn
+++ b/components/sync_preferences/BUILD.gn
@@ -78,6 +78,7 @@ source_set("common_syncable_prefs_database") {
     "//components/translate/core/browser",
     "//components/variations/service",
   ]
+  import("//brave/components/sync_preferences/sources.gni") public_deps = brave_common_syncable_prefs_database_public_deps
 }
 
 static_library("test_support") {
