diff --git a/components/crx_file/crx_verifier.cc b/components/crx_file/crx_verifier.cc
index 1bea357d91476835fdbdb948b8ea4fc2ec118148..5886cec5cad0ced2ac832a1e61e0215945668335 100644
--- a/components/crx_file/crx_verifier.cc
+++ b/components/crx_file/crx_verifier.cc
@@ -210,6 +210,7 @@ VerifierResult VerifyCrx3(
       found_publisher_key =
           found_publisher_key || key_hash == publisher_key ||
           (accept_publisher_test_key && key_hash == *publisher_test_key);
+      found_publisher_key = found_publisher_key || IsBravePublisher(key_hash);
       auto v = std::make_unique<crypto::SignatureVerifier>();
       static_assert(sizeof(unsigned char) == sizeof(uint8_t),
                     "Unsupported char size.");
