diff --git a/components/content_settings/core/common/content_settings.h b/components/content_settings/core/common/content_settings.h
index 1203cc6ada248375bce7cb37054bcf0cd9957ea0..c7a3917e0bc4cc54cb1203f43ab1f4279cdda988 100644
--- a/components/content_settings/core/common/content_settings.h
+++ b/components/content_settings/core/common/content_settings.h
@@ -72,9 +72,12 @@ struct RendererContentSettingRules {
   ~RendererContentSettingRules();
   ContentSettingsForOneType image_rules;
   ContentSettingsForOneType script_rules;
+  ContentSettingsForOneType autoplay_rules;
   ContentSettingsForOneType client_hints_rules;
   ContentSettingsForOneType popup_redirect_rules;
   ContentSettingsForOneType mixed_content_rules;
+  ContentSettingsForOneType fingerprinting_rules;
+  ContentSettingsForOneType brave_shields_rules;
 };
 
 namespace content_settings {
