diff --git a/components/sync_device_info/device_info.h b/components/sync_device_info/device_info.h
index fbaa5c9b1b2ec5184372058be00c890d2e565428..f93da86da5aaf60526d56adc78e1b4fa63b5b9a6 100644
--- a/components/sync_device_info/device_info.h
+++ b/components/sync_device_info/device_info.h
@@ -112,6 +112,7 @@ class DeviceInfo {
              const absl::optional<PhoneAsASecurityKeyInfo>& paask_info,
              const std::string& fcm_registration_token,
              const ModelTypeSet& interested_data_types);
+  virtual
   ~DeviceInfo();
 
   // Sync specific unique identifier for the device. Note if a device
