diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 567cb04ba6f2ce6b2e4ef92da89197f42f426e24..66270f0b8197962b94ff0f4032b954f6cd7a4725 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -51,6 +51,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
