diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index e262be3db60cec07cbff24ee38d17e8dca49ddf6..61de3fdbe43e876e0f92cb3a4d38364a10e6bf17 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1204,6 +1204,7 @@ void PageInfo::PresentSitePermissions() {
       }
     }
 
+    BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
     if (ShouldShowPermission(
             permission_info, site_url_, content_settings, web_contents_.get(),
             HasContentSettingChangedViaPageInfo(permission_info.type),
