diff --git a/components/content_settings/core/common/content_settings_types.h b/components/content_settings/core/common/content_settings_types.h
index 97015187c0251c20eff8361e33f9e05e89a0246f..2a1de61762efecc877dd12a19eae443534df7afb 100644
--- a/components/content_settings/core/common/content_settings_types.h
+++ b/components/content_settings/core/common/content_settings_types.h
@@ -384,6 +384,24 @@ enum class ContentSettingsType : int32_t {
   // Content settings for access to printers for the Web Printing API.
   WEB_PRINTING,
 
+  // Note: This enum is annotated with GENERATED_JAVA_ENUM_PACKAGE, and cannot
+  // be extended with an override.
+  BRAVE_START,
+  BRAVE_ADS = BRAVE_START,
+  BRAVE_COSMETIC_FILTERING,
+  BRAVE_TRACKERS,
+  BRAVE_HTTP_UPGRADABLE_RESOURCES,
+  BRAVE_FINGERPRINTING_V2,
+  BRAVE_SHIELDS,
+  BRAVE_REFERRERS,
+  BRAVE_COOKIES,
+  BRAVE_SPEEDREADER,
+  BRAVE_ETHEREUM,
+  BRAVE_SOLANA,
+  BRAVE_GOOGLE_SIGN_IN,
+  BRAVE_HTTPS_UPGRADE,
+  BRAVE_REMEMBER_1P_STORAGE,
+  BRAVE_LOCALHOST_ACCESS,
   NUM_TYPES,
 };
 
