diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index 129a82836e0b07552504b6f98cd1ae2c7a4e941e..5565f9aa8e3ba6618481dc1bd5cf5bb538bd41a8 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -574,6 +574,7 @@ jumbo_static_library("sync") {
   ]
   deps = [
     "//base:i18n",
+    "//brave/components/brave_sync:jslib_messages",
     "//components/data_use_measurement/core",
     "//components/invalidation/impl:feature_list",
     "//components/invalidation/public",
