diff --git a/components/update_client/BUILD.gn b/components/update_client/BUILD.gn
index 033f4b4a64eabb2d88be077a62da8c43387f97c9..88e39c8ddb483dd01b5cc9fa7c58cc22dd21d456 100644
--- a/components/update_client/BUILD.gn
+++ b/components/update_client/BUILD.gn
@@ -181,6 +181,7 @@ static_library("update_client") {
       "background_downloader_win.h",
     ]
   }
+  deps += brave_components_update_client_deps public_deps = brave_components_update_client_public_deps
 }
 
 source_set("test_support") {
