diff --git a/components/autofill/core/browser/suggestions/suggestion.h b/components/autofill/core/browser/suggestions/suggestion.h
index c754da17ae359fbcbf8b5e67262e580b747253e6..aa62a5bc9b872ef197415f2a1653bc74e8327314 100644
--- a/components/autofill/core/browser/suggestions/suggestion.h
+++ b/components/autofill/core/browser/suggestions/suggestion.h
@@ -697,6 +697,7 @@ struct Suggestion {
 
   // Returns whether the user is able to focus or select the suggestion.
   bool IsSelectable() const;
+  bool brave_new_email_alias_suggestion = false;
 };
 
 void PrintTo(const Suggestion& suggestion, std::ostream* os);
