diff --git a/components/version_info/BUILD.gn b/components/version_info/BUILD.gn
index 786766d86d533a4b178ca66d0e0ed0854ec45614..b35ffe7f7f8dd64947121aa2c146635dc92c66d6 100644
--- a/components/version_info/BUILD.gn
+++ b/components/version_info/BUILD.gn
@@ -17,6 +17,7 @@ source_set("version_info") {
     "//base",
     "//base/version_info",
   ]
+  configs += [ "//brave/build:version" ] deps += [ "//brave/components/version_info" ]
 }
 
 # Facade for "//base/version_info:version_string"
