diff --git a/components/variations/variations_seed_store.h b/components/variations/variations_seed_store.h
index f43ac22c13d56797e4db9d39f2557ac4c89c4864..867d9b99291edcb025d289caa33c7447cf195e75 100644
--- a/components/variations/variations_seed_store.h
+++ b/components/variations/variations_seed_store.h
@@ -302,6 +302,8 @@ class COMPONENT_EXPORT(VARIATIONS) VariationsSeedStore {
       base::OnceCallback<void(StoredSeedInfo)> done_callback,
       SeedType seed_type);
 
+  void SetSessionCountry(std::string_view country_code);
+
  protected:
   // Stores the serial number of the latest seed.
   void StoreLatestSerialNumber(std::string_view serial_number);
