diff --git a/components/content_settings/core/browser/content_settings_pref.cc b/components/content_settings/core/browser/content_settings_pref.cc
index d8e7595f22c15e097aee9462a2a6f41cd69de556..4e26da9ea19c224b969ecc8e9b7f21571d0d37ee 100644
--- a/components/content_settings/core/browser/content_settings_pref.cc
+++ b/components/content_settings/core/browser/content_settings_pref.cc
@@ -201,6 +201,7 @@ void ContentSettingsPref::SetWebsiteSetting(
   DCHECK(thread_checker_.CalledOnValidThread());
   DCHECK(prefs_);
   DCHECK(primary_pattern != ContentSettingsPattern::Wildcard() ||
+         BRAVE_SET_WEBSITE_SETTING
          secondary_pattern != ContentSettingsPattern::Wildcard());
 
   // Update in memory value map.
