diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index db8b0850a30ce0e9c43e35486d14e2c4e6abb4b1..18ea7a0ac672e80e13eeb44fcf9493070d0fa853 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -518,6 +518,9 @@ static_library("browser") {
     ]
     deps += [ "//content/public/common" ]
   }
+  sources += brave_components_omnibox_browser_sources
+  public_deps += brave_components_omnibox_browser_public_deps
+  deps += brave_components_omnibox_browser_deps
 }
 
 source_set("pref_names") {
@@ -1091,6 +1094,7 @@ fuzzer_test("search_suggestion_parser_fuzzer") {
 }
 
 mojom("mojo_bindings") {
+  generate_legacy_js_bindings = true
   sources = [ "omnibox.mojom" ]
   webui_module_path = "chrome://resources/mojo/components/omnibox/browser"
 
