diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index 8ee5e2d2c0c399516ff5dc8187124e91a943594c..38b616a6bd3a3e812255d66291f023e0e9b835be 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -382,6 +382,7 @@ jumbo_static_library("rest_of_sync") {
     "syncable/write_transaction_info.cc",
     "syncable/write_transaction_info.h",
   ]
+  sources += brave_components_sync_sources
 
   configs += [ "//build/config:precompiled_headers" ]
 
