diff --git a/components/search_engines/default_search_manager.cc b/components/search_engines/default_search_manager.cc
index 24cdc40decba2aa2b3d012cea905d67c0b34005a..c15e8b64f26550d7cce5010286d6b7fd2f1a2079 100644
--- a/components/search_engines/default_search_manager.cc
+++ b/components/search_engines/default_search_manager.cc
@@ -316,6 +316,7 @@ void DefaultSearchManager::LoadDefaultSearchEngineFromPrefs() {
   // Check if default search preference is overriden by extension.
   if (pref->IsExtensionControlled()) {
     extension_default_search_ = std::move(turl_data);
+  LOADDEFAULTSEARCHENGINEFROMPREFS_BRAVE
   } else {
     prefs_default_search_ = std::move(turl_data);
     MergePrefsDataWithPrepopulated();
