diff --git a/components/signin/public/identity_manager/BUILD.gn b/components/signin/public/identity_manager/BUILD.gn
index a28cbfd17ca7d8f3e24f0d9690dfffdb06829d8f..66559c07b501292ea635dab053beb1498a33ba4e 100644
--- a/components/signin/public/identity_manager/BUILD.gn
+++ b/components/signin/public/identity_manager/BUILD.gn
@@ -7,6 +7,7 @@ import("//build/config/chromeos/ui_mode.gni")
 if (is_android) {
   import("//build/config/android/rules.gni")
 }
+import("//brave/components/signin/public/identity_manager/sources.gni")
 
 source_set("identity_manager") {
   sources = [
@@ -82,6 +83,7 @@ source_set("identity_manager") {
     # together and include headers from each other.
     "//components/signin/internal/identity_manager",
   ]
+  sources += brave_components_signin_public_identity_manager_sources
 }
 
 if (is_android) {
