diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index de497055da9fda1ed7e5b95ab41ba86324bdca27..136182f2c3bef7efd345ebb885b21ae4a684270f 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1240,6 +1240,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
