diff --git a/components/password_manager/core/browser/password_store/BUILD.gn b/components/password_manager/core/browser/password_store/BUILD.gn
index 6a3d1a50290900f8a5be82c7abda1959ca4436a4..bf9e54128c1cae27979f2d37bb987791100470ef 100644
--- a/components/password_manager/core/browser/password_store/BUILD.gn
+++ b/components/password_manager/core/browser/password_store/BUILD.gn
@@ -49,6 +49,7 @@ source_set("password_store_impl") {
   public_deps = [ ":password_store_interface" ]
   deps = [
     "//base",
+    "//brave/components/os_crypt_key_history",
     "//build:blink_buildflags",
     "//components/affiliations/core/browser:affiliations",
     "//components/affiliations/core/browser:sql_builder",
