diff --git a/components/os_crypt/key_storage_libsecret.cc b/components/os_crypt/key_storage_libsecret.cc
index a9e87cf0318e952721010ad6cef196b80f8fd173..c8ce4d05bdc4afed97c889b3635eb44adab66ff4 100644
--- a/components/os_crypt/key_storage_libsecret.cc
+++ b/components/os_crypt/key_storage_libsecret.cc
@@ -85,6 +85,7 @@ KeyStorageLibsecret::AddRandomPasswordInLibsecret() {
 
 absl::optional<std::string> KeyStorageLibsecret::GetKeyImpl() {
   LibsecretAttributesBuilder attrs;
+  BRAVE_KEY_STORAGE_LIBSECRET_GET_KEY_IMPL
   attrs.Append("application", application_name_);
 
   LibsecretLoader::SearchHelper helper;
