diff --git a/components/browsing_data/core/browsing_data_utils.h b/components/browsing_data/core/browsing_data_utils.h
index 312262adb03cc73465bee03210f2fde60e015430..d3d194c1195e9ac536ca0c7dd4a9258e4a4b295a 100644
--- a/components/browsing_data/core/browsing_data_utils.h
+++ b/components/browsing_data/core/browsing_data_utils.h
@@ -24,6 +24,7 @@ enum class BrowsingDataType {
   PASSWORDS,
   FORM_DATA,
   SITE_SETTINGS,
+  SHIELDS_SETTINGS,
   // Only for Android:
   BOOKMARKS,
   // Only for Desktop:
