diff --git a/components/update_client/op_download.cc b/components/update_client/op_download.cc
index 28a7d0f9bdfcb472340542e65716617bb56c0eed..bcea2db93dc645dd5bec2e4243793c0d90b8478f 100644
--- a/components/update_client/op_download.cc
+++ b/components/update_client/op_download.cc
@@ -90,7 +90,6 @@ base::Value::Dict MakeEvent(const CrxDownloader::DownloadMetrics& dm) {
   }
   if (dm.download_time_ms >= 0 &&
       dm.download_time_ms < protocol_request::kProtocolMaxInt) {
-    event.Set("download_time_ms", static_cast<double>(dm.download_time_ms));
   }
   return event;
 }
