diff --git a/components/content_settings/core/common/content_settings.cc b/components/content_settings/core/common/content_settings.cc
index 55be84ee488d0a8277e22acc6bf11617c17208e0..64516af6f772bc7effd23ece4340e4b7f45dc028 100644
--- a/components/content_settings/core/common/content_settings.cc
+++ b/components/content_settings/core/common/content_settings.cc
@@ -166,6 +166,7 @@ bool RendererContentSettingRules::IsRendererContentSetting(
          content_type == ContentSettingsType::JAVASCRIPT ||
          content_type == ContentSettingsType::CLIENT_HINTS ||
          content_type == ContentSettingsType::POPUPS ||
+         BRAVE_IS_RENDERER_CONTENT_SETTING
          content_type == ContentSettingsType::MIXEDSCRIPT;
 }
 
