diff --git a/components/browsing_data/content/browsing_data_helper.cc b/components/browsing_data/content/browsing_data_helper.cc
index 7d9f7945a59b302a5eab7d0e1fb4d0855ba2eab1..ab064f3b8e129d59e4956545c7928c3854aa8836 100644
--- a/components/browsing_data/content/browsing_data_helper.cc
+++ b/components/browsing_data/content/browsing_data_helper.cc
@@ -39,7 +39,7 @@ bool WebsiteSettingsFilterAdapter(
   // Website settings only use origin-scoped patterns. The only content setting
   // this filter is used for is DURABLE_STORAGE, which also only uses
   // origin-scoped patterns. Such patterns can be directly translated to a GURL.
-  GURL url(primary_pattern.ToString());
+  GURL url(primary_pattern.ToRepresentativeUrl());
   DCHECK(url.is_valid()) << "url: '" << url.possibly_invalid_spec() << "' "
                          << "pattern: '" << primary_pattern.ToString() << "'";
   return predicate.Run(url);
