diff --git a/components/history/core/browser/sync/typed_url_sync_bridge_unittest.cc b/components/history/core/browser/sync/typed_url_sync_bridge_unittest.cc
index 14ba4a045798528940a6fb1d74a505ad328c0f74..8779503e659601c3e4b81fb1f2c0a6c043f22138 100644
--- a/components/history/core/browser/sync/typed_url_sync_bridge_unittest.cc
+++ b/components/history/core/browser/sync/typed_url_sync_bridge_unittest.cc
@@ -346,6 +346,7 @@ class TypedURLSyncBridgeTest : public testing::Test {
     typed_url_sync_bridge_->Init();
     typed_url_sync_bridge_->history_backend_observation_.Reset();
     fake_history_backend_->SetTypedURLSyncBridgeForTest(std::move(bridge));
+    BRAVE_TEST_MEMBERS_INIT
   }
 
   void TearDown() override { fake_history_backend_->Closing(); }
@@ -553,6 +554,7 @@ class TypedURLSyncBridgeTest : public testing::Test {
   scoped_refptr<TestHistoryBackendForSync> fake_history_backend_;
   raw_ptr<TypedURLSyncBridge> typed_url_sync_bridge_ = nullptr;
   NiceMock<MockModelTypeChangeProcessor> mock_processor_;
+  BRAVE_TEST_MEMBERS_DECLARE
 };
 
 // Add two typed urls locally and verify bridge can get them from GetAllData.
