diff --git a/components/omnibox/browser/omnibox_popup_selection.cc b/components/omnibox/browser/omnibox_popup_selection.cc
index 20840ad0dbb5c7ddba3c7ecd82fe0e4286383e3b..47ee8fc2abaa7fe4153b24d949e75fb4582fd641 100644
--- a/components/omnibox/browser/omnibox_popup_selection.cc
+++ b/components/omnibox/browser/omnibox_popup_selection.cc
@@ -77,6 +77,7 @@ bool OmniboxPopupSelection::IsControlPresentOnMatch(
     }
     case FOCUSED_BUTTON_THUMBS_UP:
     case FOCUSED_BUTTON_THUMBS_DOWN:
+      if ((true)) return false;
       return match.type == AutocompleteMatchType::HISTORY_EMBEDDINGS;
     case FOCUSED_BUTTON_REMOVE_SUGGESTION:
       return match.SupportsDeletion();
