diff --git a/components/webapps/browser/BUILD.gn b/components/webapps/browser/BUILD.gn
index e13bcbab82ebd521e023431b7eaba00b9c83c4e9..5534525ca01d1a42296bdc17f1a2f09abcda3689 100644
--- a/components/webapps/browser/BUILD.gn
+++ b/components/webapps/browser/BUILD.gn
@@ -193,6 +193,7 @@ source_set("browser") {
       "//ui/gfx",
     ]
   }
+  import("//brave/components/webapps/browser/sources.gni") deps += brave_components_webapps_browser_deps
 }
 
 source_set("test_support") {
