diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index 2c68e0606be5e1e89f3b5e95abe4a1feb2cc79b9..97c91853edf0cb189bfa34c02720c120a4cf6bf9 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -352,6 +352,7 @@ CookieSettingsBase::DecideAccess(const GURL& url,
   if (!IsAllowed(setting)) {
     return BlockAllCookies{};
   }
+  BRAVE_COOKIE_SETTINGS_BASE_DECIDE_ACCESS
   if (is_explicit_setting) {
     if (setting_source == SettingSource::SETTING_SOURCE_POLICY) {
       return AllowAllCookies{ThirdPartyCookieAllowMechanism::
@@ -460,6 +461,7 @@ CookieSettingsBase::GetCookieSettingInternal(
   const ContentSetting cookie_setting = GetContentSetting(
       url, first_party_url, ContentSettingsType::COOKIES, &setting_info);
 
+  BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL
   const bool is_explicit_setting =
       !setting_info.primary_pattern.MatchesAllHosts() ||
       !setting_info.secondary_pattern.MatchesAllHosts();
