diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index c840171d5df2a5a4e53aafd02c4ebe1724d153a4..2bb68d4b290d4713d9e2d089ac133b07c080a970 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -301,6 +301,7 @@ static_library("location_bar") {
   if ((!is_android || enable_vr) && !is_ios) {
     deps += [ ":vector_icons" ]
   }
+  if (brave_chromium_build) {  deps += [ "//brave/components/omnibox/browser" ] }
 
   if (is_android) {
     sources += [
