diff --git a/components/download/internal/background_service/proto/entry.proto b/components/download/internal/background_service/proto/entry.proto
index 955323219619ada808758048c9d423905f25a8eb..82e51550903349bdb6aab3da6063d989e368d914 100644
--- a/components/download/internal/background_service/proto/entry.proto
+++ b/components/download/internal/background_service/proto/entry.proto
@@ -27,6 +27,7 @@ enum DownloadClient {
 
   // New clients should be added above here.
   BOUNDARY = 10;
+  CUSTOM_LIST_SUBSCRIPTIONS = 256;
 }
 
 // Custom key value pair provided by the client and will be sent back to client
