diff --git a/components/external_intents/android/BUILD.gn b/components/external_intents/android/BUILD.gn
index e3ab9740539547ba4c427019809682d0920710b5..53cd1bd03e565f13e8acef0326cb152148233852 100644
--- a/components/external_intents/android/BUILD.gn
+++ b/components/external_intents/android/BUILD.gn
@@ -64,6 +64,7 @@ static_library("android") {
     "//components/navigation_interception",
     "//content/public/browser",
   ]
+  deps += [ "//brave/components/external_intents/android" ]
 }
 
 android_library("javatests") {
