diff --git a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsDelegate.java b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsDelegate.java
index e378b8733cd6bd4b4d295bd67465ce1b7977a502..c84fa61046ec03c1f290c84baa8b1872b527bbc7 100644
--- a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsDelegate.java
+++ b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettingsDelegate.java
@@ -124,9 +124,13 @@ public interface SiteSettingsDelegate {
      * Dismisses the Privacy Sandbox snackbar, if active.
      */
     void dismissPrivacySandboxSnackbar();
+<<<<<<< ours
 
     /**
      * Called when the view this delegate is assigned to gets destroyed.
      */
     void onDestroyView();
+=======
+    default void closeButton() {};
+>>>>>>> theirs
 }
