diff --git a/components/sync/base/user_selectable_type.cc b/components/sync/base/user_selectable_type.cc
index 6a3ce4c91a9e1cf130062fc4753c3dc6caf47cf6..3b58e7b8c4a09668a94fdeb191061dfdc7bd6a56 100644
--- a/components/sync/base/user_selectable_type.cc
+++ b/components/sync/base/user_selectable_type.cc
@@ -42,6 +42,7 @@ constexpr char kSavedTabGroupsTypeName[] = "savedTabGroups";
 constexpr char kPaymentsTypeName[] = "payments";
 constexpr char kProductComparisonTypeName[] = "productComparison";
 constexpr char kCookiesTypeName[] = "cookies";
+constexpr char kBraveNewsTypeName[] = "braveNews";
 
 UserSelectableTypeInfo GetUserSelectableTypeInfo(
     UserSelectableType type,
@@ -60,7 +61,7 @@ UserSelectableTypeInfo GetUserSelectableTypeInfo(
   // TODO(crbug.com/488439751): In CL #3, map THEMES_ANDROID to an existing
   // selectable type or to a new one. The first option should be trivial, the
   // second requires touching UI code across platforms.
-  static_assert(64 == syncer::GetNumDataTypes(),
+  static_assert(65 == syncer::GetNumDataTypes(),
                 "Almost always when adding a new Data, you must tie it to "
                 "a UserSelectableType below (new or existing) so the user can "
                 "disable syncing of that data. Today you must also update the "
@@ -157,6 +158,8 @@ UserSelectableTypeInfo GetUserSelectableTypeInfo(
           kProductComparisonTypeName, PRODUCT_COMPARISON, {PRODUCT_COMPARISON}};
     case UserSelectableType::kCookies:
       return {kCookiesTypeName, COOKIES, {COOKIES}};
+    case UserSelectableType::kBraveNews:
+      return {kBraveNewsTypeName, BRAVE_NEWS, {BRAVE_NEWS}};
   }
   NOTREACHED();
 }
@@ -212,6 +215,7 @@ std::optional<UserSelectableType> GetUserSelectableTypeFromString(
           {kSavedTabGroupsTypeName, UserSelectableType::kSavedTabGroups},
           {kProductComparisonTypeName, UserSelectableType::kProductComparison},
           {kCookiesTypeName, UserSelectableType::kCookies},
+          {kBraveNewsTypeName, UserSelectableType::kBraveNews},
       });
   if (auto it = kTypeMap.find(type); it != kTypeMap.end()) {
     return it->second;
