diff --git a/components/translate/content/renderer/translate_agent.cc b/components/translate/content/renderer/translate_agent.cc
index 4ac3e40649743c1d4a47b50d7ff1737c10de5a1f..050d05e7ed5d93ccd70b74dc6105c069ee31beb8 100644
--- a/components/translate/content/renderer/translate_agent.cc
+++ b/components/translate/content/renderer/translate_agent.cc
@@ -57,7 +57,7 @@ const int kTranslateInitCheckDelayMs = 150;
 
 // The maximum number of times we'll check to see if the translate library
 // injected in the page is ready.
-const int kMaxTranslateInitCheckAttempts = 5;
+const int kMaxTranslateInitCheckAttempts = 11;
 
 // The delay we wait in milliseconds before checking whether the translation has
 // finished.
