diff --git a/components/sync/engine/model_type_worker.h b/components/sync/engine/model_type_worker.h
index c76f8f409f2e14a3418fb1a4505edc0d492575ce..2e1342ece251e6e29a5017500664f2c640040c86 100644
--- a/components/sync/engine/model_type_worker.h
+++ b/components/sync/engine/model_type_worker.h
@@ -162,6 +162,7 @@ class ModelTypeWorker : public UpdateHandler,
 
   bool IsEncryptionEnabledForTest() const { return encryption_enabled_; }
 
+ BRAVE_MODEL_TYPE_WORKER_H_
  private:
   struct UnknownEncryptionKeyInfo {
     // Not increased if the cryptographer knows it's in a pending state
