diff --git a/components/sync/driver/sync_stopped_reporter.h b/components/sync/driver/sync_stopped_reporter.h
index ff9bad2c35972d850b42762ac07cb22105225177..502e22db47d8ecff6d6437fe97ba244850189a7c 100644
--- a/components/sync/driver/sync_stopped_reporter.h
+++ b/components/sync/driver/sync_stopped_reporter.h
@@ -29,7 +29,7 @@ class SyncStoppedReporter {
   SyncStoppedReporter(const SyncStoppedReporter&) = delete;
   SyncStoppedReporter& operator=(const SyncStoppedReporter&) = delete;
 
-  ~SyncStoppedReporter();
+  virtual ~SyncStoppedReporter();
 
   // Inform the sync server that sync was stopped on this device.
   // |access_token|, |cache_guid|, and |birthday| must not be empty.
