diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 5b3b205c765b132d169eab88a34e06bd0c88ba6a..141ab38e6780103ad63c7fddd1826b03895f588f 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -111,6 +111,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
