diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 1b732ccb8802d55cc31cef9b28bb07bb6f8515c8..8c2996a1df9bae33188977b2a2b2276ea8bac355 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -711,6 +711,7 @@ std::string GetPermissionStringForUma(
     case ContentSettingsType::LOOPBACK_NETWORK:
       return "LoopbackNetwork";
     default:
+      BRAVE_GET_PERMISSION_STRING_FOR_UMA
       break;
   }
   // The user is not prompted for these permissions thus there is no
