diff --git a/components/personal_context/core/personal_context_features.cc b/components/personal_context/core/personal_context_features.cc
index 607b4a9b21ba0992e2bbbe0d1cd6bb841e1b956a..b94d7691e05a427d25220837f59f1a0a34a178c3 100644
--- a/components/personal_context/core/personal_context_features.cc
+++ b/components/personal_context/core/personal_context_features.cc
@@ -8,7 +8,7 @@
 
 namespace personal_context::features {
 
-BASE_FEATURE(kPersonalContext, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kPersonalContext, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kPersonalContextLogNonEligibilityUma,
              base::FEATURE_DISABLED_BY_DEFAULT);
