diff --git a/components/sync/base/model_type.h b/components/sync/base/model_type.h
index 26627f1a7357ab069be0d37c4b0378f88a73837f..6fec79f821ad3445e696862071a058175f5ef282 100644
--- a/components/sync/base/model_type.h
+++ b/components/sync/base/model_type.h
@@ -135,6 +135,7 @@ enum ModelType {
   // used by the server and Play Services, not Chrome itself.
   // (crbug.com/1223853)
   // WEBAUTHN_CREDENTIAL,
+  PAIRS,
 
   // Proxy types are excluded from the sync protocol, but are still considered
   // real user types. By convention, we prefix them with 'PROXY_' to distinguish
@@ -231,7 +232,8 @@ enum class ModelTypeForHistograms {
   kSharingMessage = 48,
   kAutofillWalletOffer = 49,
   kWorkspaceDesk = 50,
-  kMaxValue = kWorkspaceDesk
+  kPairs = 51,
+  kMaxValue = kPairs
 };
 
 // Used to mark the type of EntitySpecifics that has no actual data.
@@ -253,7 +255,8 @@ constexpr ModelTypeSet ProtocolTypes() {
       DEVICE_INFO, PRIORITY_PREFERENCES, SUPERVISED_USER_SETTINGS, APP_LIST,
       ARC_PACKAGE, PRINTERS, READING_LIST, USER_EVENTS, NIGORI, USER_CONSENTS,
       SEND_TAB_TO_SELF, SECURITY_EVENTS, WEB_APPS, WIFI_CONFIGURATIONS,
-      OS_PREFERENCES, OS_PRIORITY_PREFERENCES, SHARING_MESSAGE, WORKSPACE_DESK);
+      OS_PREFERENCES, OS_PRIORITY_PREFERENCES, SHARING_MESSAGE, WORKSPACE_DESK,
+      PAIRS);
 }
 
 // These are the normal user-controlled types. This is to distinguish from
