diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index beec01938252455afefb40e78d4f3084d6288916..cb4ea8bfd934bc45b9d6f801856d9c5e305484f3 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -162,6 +162,7 @@ android_library("java") {
     "//third_party/jni_zero:jni_zero_java",
     "//ui/android:ui_java",
   ]
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
