diff --git a/components/content_settings/core/browser/content_settings_pref.cc b/components/content_settings/core/browser/content_settings_pref.cc
index bcb3e1f917e2c69f64bcdc5b52fb405b3f421e0c..bca21d9cbd8a9367d25c858dea847fea92f90488 100644
--- a/components/content_settings/core/browser/content_settings_pref.cc
+++ b/components/content_settings/core/browser/content_settings_pref.cc
@@ -157,6 +157,7 @@ void ContentSettingsPref::SetWebsiteSetting(
   DCHECK(thread_checker_.CalledOnValidThread());
   DCHECK(prefs_);
   DCHECK(primary_pattern != ContentSettingsPattern::Wildcard() ||
+         BRAVE_SET_WEBSITE_SETTING
          secondary_pattern != ContentSettingsPattern::Wildcard());
 
   // Update in memory value map.
