diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index 62fd32b9874f21a6074c1d4cbb3479cbffba52da..066f9d09b52d48a27e814e2b507bcd382894d472 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -274,6 +274,7 @@ static_library("rest_of_sync") {
     "nigori/pending_local_nigori_commit.cc",
     "nigori/pending_local_nigori_commit.h",
   ]
+  sources += brave_components_sync_sources
 
   configs += [ "//build/config:precompiled_headers" ]
 
