diff --git a/components/translate/core/browser/BUILD.gn b/components/translate/core/browser/BUILD.gn
index 0c629705b6dd370611bbc30db1d78e70e1a51d3e..b50781446fd1d8559085a9c79aa64ee5ca47133d 100644
--- a/components/translate/core/browser/BUILD.gn
+++ b/components/translate/core/browser/BUILD.gn
@@ -82,6 +82,7 @@ static_library("browser") {
     ]
     deps += [ "//components/infobars/core" ]
   }
+  deps += ["//brave/components/translate/core/browser"]
 }
 
 source_set("translate_pref_names") {
