diff --git a/components/unexportable_keys/mojom/unexportable_keys_mojom_traits.cc b/components/unexportable_keys/mojom/unexportable_keys_mojom_traits.cc
index 2f16b92303fdba637f6a907d5eed592af394558e..e37f4b9c3ba368b3cab80dc333770528ebd28836 100644
--- a/components/unexportable_keys/mojom/unexportable_keys_mojom_traits.cc
+++ b/components/unexportable_keys/mojom/unexportable_keys_mojom_traits.cc
@@ -29,6 +29,7 @@ EnumTraits<unexportable_keys::mojom::SignatureAlgorithm,
       return unexportable_keys::mojom::SignatureAlgorithm::ECDSA_SHA256;
     case crypto::SignatureVerifier::SignatureAlgorithm::RSA_PSS_SHA256:
       return unexportable_keys::mojom::SignatureAlgorithm::RSA_PSS_SHA256;
+    default: NOTREACHED();
   }
 }
 
