diff --git a/components/permissions/permission_request.h b/components/permissions/permission_request.h
index 6e8dbb8bd7652c66b5f65361e57920586d76595e..60f4196592e690ae037a2a00faee113e3e3ac05e 100644
--- a/components/permissions/permission_request.h
+++ b/components/permissions/permission_request.h
@@ -54,6 +54,7 @@ enum class PermissionRequestType {
   PERMISSION_VR = 21,
   PERMISSION_AR = 22,
   PERMISSION_STORAGE_ACCESS = 23,
+  BRAVE_PERMISSION_REQUEST_TYPES
   // NUM must be the last value in the enum.
   NUM
 };
