diff --git a/components/sync/driver/BUILD.gn b/components/sync/driver/BUILD.gn
index 9b6db577a7b936b771c0c49f38297c38c2f7ea69..b7179f9e14236d466c622c55f495c5452d8ffa22 100644
--- a/components/sync/driver/BUILD.gn
+++ b/components/sync/driver/BUILD.gn
@@ -107,6 +107,7 @@ static_library("driver") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps defines = brave_service_key_defines
 }
 
 static_library("test_support") {
