diff --git a/components/component_updater/component_updater_service.cc b/components/component_updater/component_updater_service.cc
index 1e6dc0e114a1f87b60688a264adad20c072795bc..b8210dce96d8013c9a8f8e06f9647b1c17b71ede 100644
--- a/components/component_updater/component_updater_service.cc
+++ b/components/component_updater/component_updater_service.cc
@@ -206,6 +206,9 @@ bool CrxUpdateService::RegisterComponent(
     return false;
   }
 
+  // Cancel any pending unregistration for this component.
+  std::erase(components_pending_unregistration_, component.app_id);
+
   // Update the registration data if the component has been registered before.
   auto it = components_.find(component.app_id);
   if (it != components_.end()) {
