diff --git a/components/external_intents/android/BUILD.gn b/components/external_intents/android/BUILD.gn
index 9e2f0dd12e96b9e6968fd6368c4983e14cd6d486..d188cba8301db087d0e9424632f46d6d696c178d 100644
--- a/components/external_intents/android/BUILD.gn
+++ b/components/external_intents/android/BUILD.gn
@@ -64,6 +64,7 @@ static_library("android") {
     "//components/navigation_interception",
     "//content/public/browser",
   ]
+  deps += [ "//brave/components/external_intents/android" ]
 }
 
 android_library("javatests") {
