diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index a4c68d2bf10a85a1c778e36d76949be4dfb8bed7..1ca9689868a56eb46f5093ea4145d23c9b7e71a6 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1401,6 +1401,9 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // second section, the default behavior used when no per-type exception applies.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  if (!delegate_->BraveShouldShowPermission(info.type)) {
+    return false;
+  }
 #if BUILDFLAG(IS_ANDROID)
   if (info.type == ContentSettingsType::NOTIFICATIONS) {
     // `is_requested` is only populated if
@@ -1637,6 +1640,9 @@ void PageInfo::PresentSitePermissions() {
 
 std::set<net::SchemefulSite> PageInfo::GetTwoSitePermissionRequesters(
     ContentSettingsType type) {
+  if (type == ContentSettingsType::STORAGE_ACCESS) {
+    return {};
+  }
   std::set<net::SchemefulSite> requesters;
   // Collect sites that have tried to request a permission.
   auto* pscs = GetPageSpecificContentSettings();
