diff --git a/components/translate/content/renderer/per_frame_translate_agent.cc b/components/translate/content/renderer/per_frame_translate_agent.cc
index 8a84116c8c09605b84a7fdc1c7a44248033d606c..4374d8ab070e184d5cdb10c3d072383062648ce4 100644
--- a/components/translate/content/renderer/per_frame_translate_agent.cc
+++ b/components/translate/content/renderer/per_frame_translate_agent.cc
@@ -50,7 +50,7 @@ const int kTranslateInitCheckDelayMs = 150;
 
 // The maximum number of times we'll check to see if the translate library
 // injected in the page is ready.
-const int kMaxTranslateInitCheckAttempts = 5;
+const int kMaxTranslateInitCheckAttempts = 11;
 
 // The delay we wait in milliseconds before checking whether the translation has
 // finished.
