diff --git a/components/permissions/android/permission_prompt/permission_dialog_delegate.cc b/components/permissions/android/permission_prompt/permission_dialog_delegate.cc
index 81fa9ed68fa68132f95cdd1bca130f90a85627d2..d7f98c64cb45ba54602857c60e730d85d24f6eed 100644
--- a/components/permissions/android/permission_prompt/permission_dialog_delegate.cc
+++ b/components/permissions/android/permission_prompt/permission_dialog_delegate.cc
@@ -165,6 +165,7 @@ PermissionDialogDelegate::CreateForTesting(
 
 void PermissionDialogDelegate::Accept(JNIEnv* env) {
   CHECK(permission_prompt_);
+  BRAVE_PERMISSION_DIALOG_DELEGATE_ACCEPT
   permission_prompt_->Accept();
 }
 
@@ -180,6 +181,7 @@ void PermissionDialogDelegate::Acknowledge(JNIEnv* env) {
 
 void PermissionDialogDelegate::Deny(JNIEnv* env) {
   CHECK(permission_prompt_);
+  BRAVE_PERMISSION_DIALOG_DELEGATE_CANCEL
   permission_prompt_->Deny();
 }
 
