diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index d78a53c731822d02a75b9df3fb6383b2889470f6..04a7f633285959dfc0a0a757658dcec5f6c24d05 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -193,6 +193,7 @@ static_library("metrics") {
   if (is_fuchsia) {
     sources += [ "drive_metrics_provider_fuchsia.cc" ]
   }
+  import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
 }
 
 if (is_android) {
