diff --git a/components/strings/BUILD.gn b/components/strings/BUILD.gn
index 4f264ebc35e4b48b91c89ccd48413a83bb8442dd..9d8c20242d95524cbfc5f392c0520225582e4edd 100644
--- a/components/strings/BUILD.gn
+++ b/components/strings/BUILD.gn
@@ -52,7 +52,7 @@ if (is_android) {
 }
 
 grit("components_chromium_strings") {
-  source = "../components_chromium_strings.grd"
+  source = "../components_${branding_path_component}_strings.grd"
   outputs = [ "grit/components_chromium_strings.h" ]
   foreach(locale, locales_with_fake_bidi) {
     outputs += [ "components_chromium_strings_$locale.pak" ]
