diff --git a/components/page_info/BUILD.gn b/components/page_info/BUILD.gn
index d85a1b75df6ced6118af0e00382863c60bba7d6b..11fc49a106304af054033c1a2c3c315e6b01267e 100644
--- a/components/page_info/BUILD.gn
+++ b/components/page_info/BUILD.gn
@@ -45,6 +45,7 @@ static_library("page_info") {
     "//services/device/public/cpp:device_features",
     "//services/metrics/public/cpp:ukm_builders",
   ]
+  deps += [ "//brave/components/page_info" ]
 
   if (is_android) {
     deps += [
