diff --git a/components/browser_sync/profile_sync_service.h b/components/browser_sync/profile_sync_service.h
index 63f934732987944e7aace3252c1efa7095ebbce1..4818c513864a4f84277393e4a7827c51d9ad349f 100644
--- a/components/browser_sync/profile_sync_service.h
+++ b/components/browser_sync/profile_sync_service.h
@@ -45,6 +45,7 @@
 #include "components/version_info/version_info.h"
 #include "google_apis/gaia/google_service_auth_error.h"
 #include "url/gurl.h"
+#include "brave/components/sync/driver/brave_sync_service_base.h"
 
 class ProfileOAuth2TokenService;
 class SigninManagerWrapper;
@@ -159,7 +160,8 @@ class SyncAuthManager;
 //   Once first setup has completed and there are no outstanding
 //   setup-in-progress handles, CanConfigureDataTypes() will return true and
 //   datatype configuration can begin.
-class ProfileSyncService : public syncer::SyncService,
+class ProfileSyncService : public syncer::BraveSyncServiceBase,
+                           public syncer::SyncService,
                            public syncer::SyncEngineHost,
                            public syncer::SyncPrefObserver,
                            public syncer::DataTypeManagerObserver,
