diff --git a/components/metrics/machine_id_provider_win.cc b/components/metrics/machine_id_provider_win.cc
index f26a7ba523bc2e43dd0f441bc38653bfde74ff46..04d7bbbb9ecb25e43a231585d12970f2902cacac 100644
--- a/components/metrics/machine_id_provider_win.cc
+++ b/components/metrics/machine_id_provider_win.cc
@@ -20,7 +20,7 @@ namespace metrics {
 
 // static
 bool MachineIdProvider::HasId() {
-  return true;
+  return !IsMachineIdDisabled();
 }
 
 // On windows, the machine id is based on the serial number of the drive Chrome
