diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 94ee0b4ff21dc048060ca28d2040706d2df3a103..c55e7b6d1b60f02993d34660cf8e80bd8a7747d5 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -49,6 +49,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 typedef int64_t VisitID;
