diff --git a/components/search_engines/template_url_prepopulate_data.cc b/components/search_engines/template_url_prepopulate_data.cc
index e8042c05028470dd069b993a78412cd32185656c..ab73d148bac1eb92e0c214ec14412e79c86a7578 100644
--- a/components/search_engines/template_url_prepopulate_data.cc
+++ b/components/search_engines/template_url_prepopulate_data.cc
@@ -44,7 +44,6 @@ const PrepopulatedEngine* const engines_AE[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Albania
@@ -60,7 +59,6 @@ const PrepopulatedEngine* const engines_AR[] = {
     &google,
     &bing,
     &yahoo_ar,
-    &duckduckgo,
 };
 
 // Austria
@@ -68,7 +66,6 @@ const PrepopulatedEngine* const engines_AT[] = {
     &google,
     &bing,
     &yahoo_at,
-    &duckduckgo,
 };
 
 // Australia
@@ -76,7 +73,6 @@ const PrepopulatedEngine* const engines_AU[] = {
     &google,
     &bing,
     &yahoo_au,
-    &duckduckgo,
 };
 
 // Bosnia and Herzegovina
@@ -92,7 +88,6 @@ const PrepopulatedEngine* const engines_BE[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Bulgaria
@@ -124,7 +119,6 @@ const PrepopulatedEngine* const engines_BN[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Bolivia
@@ -132,7 +126,6 @@ const PrepopulatedEngine* const engines_BO[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Brazil
@@ -140,7 +133,6 @@ const PrepopulatedEngine* const engines_BR[] = {
     &google,
     &bing,
     &yahoo_br,
-    &duckduckgo,
 };
 
 // Belarus
@@ -156,19 +148,17 @@ const PrepopulatedEngine* const engines_BZ[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Canada
 const PrepopulatedEngine* const engines_CA[] = {
-    &google, &bing, &yahoo_ca, &yahoo_qc, &duckduckgo,
+    &google, &bing, &yahoo_ca, &yahoo_qc,
 };
 
 // Switzerland
 const PrepopulatedEngine* const engines_CH[] = {
     &google,
     &bing,
-    &duckduckgo,
     &yahoo_ch,
 };
 
@@ -177,7 +167,6 @@ const PrepopulatedEngine* const engines_CL[] = {
     &google,
     &bing,
     &yahoo_cl,
-    &duckduckgo,
 };
 
 // China
@@ -190,7 +179,6 @@ const PrepopulatedEngine* const engines_CO[] = {
     &google,
     &yahoo_co,
     &bing,
-    &duckduckgo,
 };
 
 // Costa Rica
@@ -198,7 +186,6 @@ const PrepopulatedEngine* const engines_CR[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Czech Republic
@@ -214,7 +201,6 @@ const PrepopulatedEngine* const engines_DE[] = {
     &google,
     &bing,
     &yahoo_de,
-    &duckduckgo,
 };
 
 // Denmark
@@ -222,7 +208,6 @@ const PrepopulatedEngine* const engines_DK[] = {
     &google,
     &bing,
     &yahoo_dk,
-    &duckduckgo,
 };
 
 // Dominican Republic
@@ -230,7 +215,6 @@ const PrepopulatedEngine* const engines_DO[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Algeria
@@ -246,7 +230,6 @@ const PrepopulatedEngine* const engines_EC[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Estonia
@@ -270,7 +253,6 @@ const PrepopulatedEngine* const engines_ES[] = {
     &google,
     &bing,
     &yahoo_es,
-    &duckduckgo,
 };
 
 // Finland
@@ -278,7 +260,6 @@ const PrepopulatedEngine* const engines_FI[] = {
     &google,
     &bing,
     &yahoo_fi,
-    &duckduckgo,
 };
 
 // Faroe Islands
@@ -286,7 +267,6 @@ const PrepopulatedEngine* const engines_FO[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // France
@@ -294,7 +274,6 @@ const PrepopulatedEngine* const engines_FR[] = {
     &google,
     &bing,
     &yahoo_fr,
-    &qwant,
 };
 
 // United Kingdom
@@ -302,7 +281,6 @@ const PrepopulatedEngine* const engines_GB[] = {
     &google,
     &bing,
     &yahoo_uk,
-    &duckduckgo,
 };
 
 // Greece
@@ -310,7 +288,6 @@ const PrepopulatedEngine* const engines_GR[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Guatemala
@@ -318,7 +295,6 @@ const PrepopulatedEngine* const engines_GT[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Hong Kong
@@ -334,7 +310,6 @@ const PrepopulatedEngine* const engines_HN[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Croatia
@@ -342,7 +317,6 @@ const PrepopulatedEngine* const engines_HR[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Hungary
@@ -350,7 +324,6 @@ const PrepopulatedEngine* const engines_HU[] = {
     &google,
     &yahoo,
     &bing,
-    &duckduckgo,
 };
 
 // Indonesia
@@ -358,7 +331,6 @@ const PrepopulatedEngine* const engines_ID[] = {
     &google,
     &yahoo_id,
     &bing,
-    &duckduckgo,
 };
 
 // Ireland
@@ -366,7 +338,6 @@ const PrepopulatedEngine* const engines_IE[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Israel
@@ -382,7 +353,6 @@ const PrepopulatedEngine* const engines_IN[] = {
     &google,
     &yahoo_in,
     &bing,
-    &duckduckgo,
 };
 
 // Iraq
@@ -406,7 +376,6 @@ const PrepopulatedEngine* const engines_IS[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Italy
@@ -414,7 +383,6 @@ const PrepopulatedEngine* const engines_IT[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Jamaica
@@ -422,7 +390,6 @@ const PrepopulatedEngine* const engines_JM[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Jordan
@@ -446,7 +413,6 @@ const PrepopulatedEngine* const engines_KE[] = {
     &google,
     &yahoo,
     &bing,
-    &duckduckgo,
 };
 
 // South Korea
@@ -478,14 +444,12 @@ const PrepopulatedEngine* const engines_LB[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Liechtenstein
 const PrepopulatedEngine* const engines_LI[] = {
     &google,
     &bing,
-    &duckduckgo,
     &yahoo,
 };
 
@@ -501,7 +465,6 @@ const PrepopulatedEngine* const engines_LT[] = {
 const PrepopulatedEngine* const engines_LU[] = {
     &google,
     &bing,
-    &duckduckgo,
     &yahoo,
 };
 
@@ -534,7 +497,6 @@ const PrepopulatedEngine* const engines_MC[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Moldova
@@ -542,7 +504,6 @@ const PrepopulatedEngine* const engines_MD[] = {
     &google,
     &yandex_ru,
     &mail_ru,
-    &duckduckgo,
 };
 
 // Montenegro
@@ -558,7 +519,6 @@ const PrepopulatedEngine* const engines_MK[] = {
     &google,
     &yahoo,
     &bing,
-    &duckduckgo,
 };
 
 // Mexico
@@ -566,7 +526,6 @@ const PrepopulatedEngine* const engines_MX[] = {
     &google,
     &bing,
     &yahoo_mx,
-    &duckduckgo,
 };
 
 // Malaysia
@@ -582,7 +541,6 @@ const PrepopulatedEngine* const engines_NI[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Netherlands
@@ -590,7 +548,6 @@ const PrepopulatedEngine* const engines_NL[] = {
     &google,
     &bing,
     &yahoo_nl,
-    &duckduckgo,
 };
 
 // Norway
@@ -598,14 +555,12 @@ const PrepopulatedEngine* const engines_NO[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // New Zealand
 const PrepopulatedEngine* const engines_NZ[] = {
     &google,
     &bing,
-    &duckduckgo,
     &yahoo_nz,
 };
 
@@ -622,7 +577,6 @@ const PrepopulatedEngine* const engines_PA[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Peru
@@ -630,7 +584,6 @@ const PrepopulatedEngine* const engines_PE[] = {
     &google,
     &yahoo_pe,
     &bing,
-    &duckduckgo,
 };
 
 // Philippines
@@ -638,7 +591,6 @@ const PrepopulatedEngine* const engines_PH[] = {
     &google,
     &yahoo_ph,
     &bing,
-    &duckduckgo,
 };
 
 // Pakistan
@@ -654,7 +606,6 @@ const PrepopulatedEngine* const engines_PL[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Puerto Rico
@@ -662,7 +613,6 @@ const PrepopulatedEngine* const engines_PR[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Portugal
@@ -670,7 +620,6 @@ const PrepopulatedEngine* const engines_PT[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Paraguay
@@ -678,7 +627,6 @@ const PrepopulatedEngine* const engines_PY[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Qatar
@@ -694,7 +642,6 @@ const PrepopulatedEngine* const engines_RO[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Serbia
@@ -702,7 +649,6 @@ const PrepopulatedEngine* const engines_RS[] = {
     &google,
     &yahoo,
     &bing,
-    &duckduckgo,
 };
 
 // Russia
@@ -734,7 +680,6 @@ const PrepopulatedEngine* const engines_SE[] = {
     &google,
     &bing,
     &yahoo_se,
-    &duckduckgo,
 };
 
 // Singapore
@@ -750,7 +695,6 @@ const PrepopulatedEngine* const engines_SI[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Slovakia
@@ -758,7 +702,6 @@ const PrepopulatedEngine* const engines_SK[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // El Salvador
@@ -766,7 +709,6 @@ const PrepopulatedEngine* const engines_SV[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Syria
@@ -806,7 +748,6 @@ const PrepopulatedEngine* const engines_TT[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Taiwan
@@ -838,7 +779,6 @@ const PrepopulatedEngine* const engines_US[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Uruguay
@@ -846,7 +786,6 @@ const PrepopulatedEngine* const engines_UY[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Venezuela
@@ -854,7 +793,6 @@ const PrepopulatedEngine* const engines_VE[] = {
     &google,
     &yahoo_ve,
     &bing,
-    &duckduckgo,
 };
 
 // Vietnam
@@ -878,7 +816,6 @@ const PrepopulatedEngine* const engines_ZA[] = {
     &google,
     &bing,
     &yahoo,
-    &duckduckgo,
 };
 
 // Zimbabwe
@@ -898,12 +835,10 @@ const PrepopulatedEngine* const kAllEngines[] = {
     &bing,
     &coccoc,
     &daum,
-    &duckduckgo,
     &google,
     &mail_ru,
     &naver,
     &parsijoo,
-    &qwant,
     &seznam,
     &sogou,
     &yahoo,
