diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 02fe3bd9aaaa84a57aa4f9a3cac840d0ebe54b0b..62fea1fd2ecf4bc001d8e047a4508250949afe59 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -995,6 +995,7 @@ void PageInfo::PresentSitePermissions() {
       }
     }
 
+    BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
     if (ShouldShowPermission(
             permission_info, site_url_, content_settings, web_contents_.get(),
             HasContentSettingChangedViaPageInfo(permission_info.type),
