diff --git a/components/content_settings/core/common/content_settings_mojom_traits.h b/components/content_settings/core/common/content_settings_mojom_traits.h
index e58b9a69537b2e49c90504fbdfe7e3444e853772..fea7416c7ada6702a2001afe6fb05d8d35579c07 100644
--- a/components/content_settings/core/common/content_settings_mojom_traits.h
+++ b/components/content_settings/core/common/content_settings_mojom_traits.h
@@ -148,6 +148,7 @@ struct StructTraits<
   static bool Read(
       content_settings::mojom::RendererContentSettingRulesDataView data,
       RendererContentSettingRules* out);
+  BRAVE_CONTENT_SETTINGS_MOJOM_TRAITS_H
 };
 
 }  // namespace mojo
