diff --git a/components/sync/service/sync_user_settings_impl.cc b/components/sync/service/sync_user_settings_impl.cc
index 2cc72205ef6538f2c183cb215a5fbb4113ba6eed..0ac8ca32914dc6e1213c1680279cee2035e5afbe 100644
--- a/components/sync/service/sync_user_settings_impl.cc
+++ b/components/sync/service/sync_user_settings_impl.cc
@@ -394,6 +394,7 @@ DataTypeSet SyncUserSettingsImpl::GetPreferredDataTypes() const {
   if (prefs_->IsLocalSyncEnabled()) {
     types.RetainAll(LocalSyncSupportedTypes());
   }
+  BraveAdjustPreferredDataTypes(&types, prefs_->IsLocalSyncEnabled());
   return types;
 }
 
