diff --git a/components/bookmarks/browser/bookmark_model.h b/components/bookmarks/browser/bookmark_model.h
index fce7bfdc9f21395fd7e03428abc499dbbbbf2568..637411bcf11ddb322c73ea01d69b53db79331c5f 100644
--- a/components/bookmarks/browser/bookmark_model.h
+++ b/components/bookmarks/browser/bookmark_model.h
@@ -310,6 +310,7 @@ class BookmarkModel : public BookmarkUndoProvider,
     return weak_factory_.GetWeakPtr();
   }
 
+  BRAVE_BOOKMARK_MODEL_H
  private:
   friend class BookmarkCodecTest;
   friend class BookmarkModelFaviconTest;
