diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index 11caf7038ef5a9c22d05dd05e1b4561e0db1b8d2..4859e14ebe43a745f969c9aa7776a46f6f47de10 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -301,6 +301,7 @@ void PermissionManager::RequestPermissionsFromCurrentDocument(
     base::OnceCallback<void(const std::vector<content::PermissionResult>&)>
         permission_status_callback) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
+  BRAVE_REQUEST_PERMISSION_FROM_CURRENT_DOCUMENT
   RequestPermissionsInternal(render_frame_host, request_description,
                              std::move(permission_status_callback));
 }
@@ -343,6 +344,7 @@ PermissionManager::GetPermissionResultForCurrentDocument(
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
 
   const GURL requesting_origin =
+      BRAVE_GET_PERMISSION_RESULT_FOR_CURRENT_DOCUMENT
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   const GURL embedding_origin =
       GetEmbeddingOrigin(render_frame_host, requesting_origin);
