diff --git a/components/permissions/permission_util.cc b/components/permissions/permission_util.cc
index 7a06b95b30f4c3c2f4f16323afe4855a9e1bd1a4..7f22d574499c4f1cf83cc1ed393dc4b872a6849f 100644
--- a/components/permissions/permission_util.cc
+++ b/components/permissions/permission_util.cc
@@ -365,6 +365,7 @@ ContentSettingsType PermissionUtil::PermissionTypeToContentSettingsTypeSafe(
       return ContentSettingsType::WEB_APP_INSTALLATION;
     case PermissionType::NUM:
       break;
+    PERMISSION_UTIL_PERMISSION_TYPE_TO_CONTENT_SETTINGS_TYPE
   }
 
   return ContentSettingsType::DEFAULT;
