diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 1d64d3ccd60b7aa9e5b0cc6eb6bb37801846f133..46157560c3da3a0734b454780eaca550f5484deb 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -50,7 +50,7 @@ namespace {
 
 const int kPriorCountCap = 10;
 
-std::string GetPermissionRequestString(PermissionRequestType type) {
+std::string GetPermissionRequestString_ChromiumImpl(PermissionRequestType type) {
   switch (type) {
     case PermissionRequestType::MULTIPLE:
       return "AudioAndVideoCapture";
