diff --git a/components/safe_browsing/core/common/safe_browsing_prefs.cc b/components/safe_browsing/core/common/safe_browsing_prefs.cc
index 12d10cc3059df7946a436561fc8b7b08bfe4af1a..f6b195d894ff3646c521455ada712e40b509ed4c 100644
--- a/components/safe_browsing/core/common/safe_browsing_prefs.cc
+++ b/components/safe_browsing/core/common/safe_browsing_prefs.cc
@@ -150,6 +150,7 @@ bool IsSafeBrowsingEnabled(const PrefService& prefs) {
 }
 
 bool IsEnhancedProtectionEnabled(const PrefService& prefs) {
+  BRAVE_IS_ENHANCED_PROTECTION_ENABLED
   // SafeBrowsingEnabled is checked too due to devices being out
   // of sync or not on a version that includes SafeBrowsingEnhanced pref.
   return base::FeatureList::IsEnabled(kEnhancedProtection) &&
