diff --git a/components/content_settings/renderer/content_settings_agent_impl.cc b/components/content_settings/renderer/content_settings_agent_impl.cc
index 34871864cf8d70ca570d6d9a055d04f98b31b654..76229296df48b50d340f3596b7ed30856a6dc3dc 100644
--- a/components/content_settings/renderer/content_settings_agent_impl.cc
+++ b/components/content_settings/renderer/content_settings_agent_impl.cc
@@ -409,6 +409,7 @@ bool ContentSettingsAgentImpl::IsAllowlistedForContentSettings() const {
   const WebDocument& document = render_frame()->GetWebFrame()->GetDocument();
   WebSecurityOrigin origin = document.GetSecurityOrigin();
   WebURL document_url = document.Url();
+  BRAVE_CONTENT_SETTINGS_AGENT_IMPL_IS_WHITELISTED_FOR_CONTENT_SETTINGS
   if (document_url.GetString() == content::kUnreachableWebDataURL)
     return true;
 
