diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index 5b6894233a31a1c989c4c97a6dd6591ecf9c48b1..209b63455ba0ff21b031f5df69beee464eb788da 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -127,6 +127,7 @@ source_set("permissions") {
       "contexts/geolocation_permission_context_mac.h",
     ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
