diff --git a/components/policy/BUILD.gn b/components/policy/BUILD.gn
index 5f345583e6c6f2bfdc486ebcc0a0112c2b953a3f..1ab0d7289e1dee9ab183024113d8852a246dd070 100644
--- a/components/policy/BUILD.gn
+++ b/components/policy/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/components/policy/resources/templates/policy_definitions/brave_policies.gni")
 import("//build/apple/convert_plist.gni")
 import("//build/config/chrome_build.gni")
 import("//build/config/python.gni")
@@ -94,6 +95,7 @@ policy_templates_grd_file = "resources/policy_templates.build.grd"
 
 # Generates a single policy_templates.json with a real JSON format.
 action("generate_policy_templates") {
+  inputs = brave_generate_policy_templates_inputs
   script = "resources/policy_templates.py"
   depfile = policy_templates_deps_file
   outputs = [
