diff --git a/components/unexportable_keys/mojom/unexportable_keys_mojom_traits.cc b/components/unexportable_keys/mojom/unexportable_keys_mojom_traits.cc
index a9ce797cf74e14b9612e9bffc50e5ec86fb5b832..2b0ffe01dc32a8e8f6657d3ee04d3ac29b36d0fa 100644
--- a/components/unexportable_keys/mojom/unexportable_keys_mojom_traits.cc
+++ b/components/unexportable_keys/mojom/unexportable_keys_mojom_traits.cc
@@ -29,6 +29,7 @@ EnumTraits<unexportable_keys::mojom::SignatureAlgorithm,
       return unexportable_keys::mojom::SignatureAlgorithm::ECDSA_SHA256;
     case crypto::SignatureVerifier::SignatureAlgorithm::RSA_PSS_SHA256:
       return unexportable_keys::mojom::SignatureAlgorithm::RSA_PSS_SHA256;
+    default: NOTREACHED();
   }
 }
 
