diff --git a/components/os_crypt/async/browser/freedesktop_secret_key_provider.h b/components/os_crypt/async/browser/freedesktop_secret_key_provider.h
index 41e3415c233b57f55addce520f797ae13bc13dc7..06fab06682f3254be417dba8367251be7ff18910 100644
--- a/components/os_crypt/async/browser/freedesktop_secret_key_provider.h
+++ b/components/os_crypt/async/browser/freedesktop_secret_key_provider.h
@@ -176,9 +176,9 @@ class FreedesktopSecretKeyProvider : public KeyProvider {
   static constexpr char kKeyName[] = "Chrome Safe Storage";
   static constexpr char kAppName[] = "chrome";
 #else
-  static constexpr char kKWalletFolder[] = "Chromium Keys";
-  static constexpr char kKeyName[] = "Chromium Safe Storage";
-  static constexpr char kAppName[] = "chromium";
+  static constexpr char kKWalletFolder[] = "Brave Keys";
+  static constexpr char kKeyName[] = "Brave Safe Storage";
+  static constexpr char kAppName[] = "brave";
 #endif
 
   void InitializeFreedesktopSecretService();
