diff --git a/components/os_crypt/key_storage_keyring.cc b/components/os_crypt/key_storage_keyring.cc
index 409bd27cbc0877634d7d9809575cfa5f60ba04c2..311c3f4e65dd18ab3c84ac989cc85e4527cd0c19 100644
--- a/components/os_crypt/key_storage_keyring.cc
+++ b/components/os_crypt/key_storage_keyring.cc
@@ -18,7 +18,7 @@ namespace {
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
 const char kApplicationName[] = "chrome";
 #else
-const char kApplicationName[] = "chromium";
+BRAVE_KEY_STORAGE_KEYRING_K_APPLICATION_NAME
 #endif
 
 const GnomeKeyringPasswordSchema kSchema = {
@@ -49,6 +49,7 @@ base::Optional<std::string> KeyStorageKeyring::GetKeyImpl() {
   gchar* password_c = nullptr;
   GnomeKeyringResult result =
       GnomeKeyringLoader::gnome_keyring_find_password_sync_ptr(
+          BRAVE_KEY_STORAGE_KEYRING_GET_KEY_IMPL
           &kSchema, &password_c, "application", kApplicationName, nullptr);
   if (result == GNOME_KEYRING_RESULT_OK) {
     password = password_c;
