diff --git a/components/browser_sync/profile_sync_service.h b/components/browser_sync/profile_sync_service.h
index 6c01395b954ac146084865b0268992f277b4914c..734f5623f889f9e576a7e4d5fb39dae0a278290a 100644
--- a/components/browser_sync/profile_sync_service.h
+++ b/components/browser_sync/profile_sync_service.h
@@ -346,6 +346,14 @@ class ProfileSyncService : public syncer::SyncService,
 
   syncer::SyncClient* GetSyncClientForTest();
 
+ protected:
+  syncer::SyncClient* GetSyncClient();
+  syncer::SyncEngine* GetSyncEngine();
+  virtual void OnNudgeSyncCycle(brave_sync::RecordsListPtr records_list);
+  virtual void OnPollSyncCycle(brave_sync::GetRecordsCallback cb,
+                       base::WaitableEvent* wevent);
+  virtual bool IsBraveSyncEnabled() const;
+
  private:
   // Passed as an argument to StopImpl to control whether or not the sync
   // engine should clear its data directory when it shuts down. See StopImpl
@@ -355,6 +363,8 @@ class ProfileSyncService : public syncer::SyncService,
     CLEAR_DATA,
   };
 
+  void BraveEngineParamsInit(syncer::SyncEngine::InitParams*);
+
   // Shorthand for user_settings_.IsFirstSetupComplete().
   bool IsFirstSetupComplete() const;
 
