diff --git a/components/download/internal/background_service/proto_conversions.cc b/components/download/internal/background_service/proto_conversions.cc
index be74ae46d79fc94962e720069fc37ba9af3ff48a..a25695be644346eeab64eda68a0febf5fa7823e3 100644
--- a/components/download/internal/background_service/proto_conversions.cc
+++ b/components/download/internal/background_service/proto_conversions.cc
@@ -77,6 +77,7 @@ protodb::DownloadClient ProtoConversions::DownloadClientToProto(
       return protodb::DownloadClient::OPTIMIZATION_GUIDE_PREDICTION_MODELS;
     case DownloadClient::BOUNDARY:
       return protodb::DownloadClient::BOUNDARY;
+    BRAVE_DOWNLOAD_CLIENT_TO_PROTO
   }
 
   NOTREACHED();
@@ -108,6 +109,7 @@ DownloadClient ProtoConversions::DownloadClientFromProto(
       return DownloadClient::OPTIMIZATION_GUIDE_PREDICTION_MODELS;
     case protodb::DownloadClient::BOUNDARY:
       return DownloadClient::BOUNDARY;
+    BRAVE_DOWNLOAD_CLIENT_FROM_PROTO
   }
 
   NOTREACHED();
