diff --git a/components/sync/driver/profile_sync_service.h b/components/sync/driver/profile_sync_service.h
index b809a57c5a691a043eee6b370bf123169d3b9835..cf62448fbd230a8df9796e72996741728b772012 100644
--- a/components/sync/driver/profile_sync_service.h
+++ b/components/sync/driver/profile_sync_service.h
@@ -257,6 +257,7 @@ class ProfileSyncService : public SyncService,
 
   static std::string GenerateCacheGUIDForTest();
 
+ BRAVE_PROFILE_SYNC_SERVICE_H_
  private:
   // Passed as an argument to StopImpl to control whether or not the sync
   // engine should clear its data when it shuts down. See StopImpl for more
