diff --git a/components/history/core/browser/BUILD.gn b/components/history/core/browser/BUILD.gn
index 427566c438713b0edf03cd20b4160f2ada2694d7..b3cb6b2400673048e8a18821281b678e82f17cb5 100644
--- a/components/history/core/browser/BUILD.gn
+++ b/components/history/core/browser/BUILD.gn
@@ -156,6 +156,7 @@ static_library("browser") {
       "android/visit_sql_handler.h",
     ]
   }
+  import("//brave/components/history/core/browser/sources.gni") sources += brave_components_history_core_browser_sources deps += brave_components_history_core_browser_deps
 }
 
 bundle_data("unit_tests_bundle_data") {
