diff --git a/components/browser_ui/settings/android/BUILD.gn b/components/browser_ui/settings/android/BUILD.gn
index 8dcf5d842176f359f8ed475b8747dff3b10ad995..b764bc3ad2fd99b1bc36ff0603917c00efd9b53b 100644
--- a/components/browser_ui/settings/android/BUILD.gn
+++ b/components/browser_ui/settings/android/BUILD.gn
@@ -50,6 +50,7 @@ android_library("java") {
     "//ui/android:ui_no_recycler_view_java",
   ]
   resources_package = "org.chromium.components.browser_ui.settings"
+  import("//brave/components/browser_ui/settings/android/sources.gni") sources += brave_components_browser_ui_settings_android_java_sources
 }
 
 android_resources("java_resources") {
