diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index f72c437850adeba75825fc3cdeeb42f95a55852a..814dca90f91688947e7cf3e0a230ef0f134c8ec0 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1265,6 +1265,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
