diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index a885babf07ef6c3f7d97269665ac0c60688fd328..d1ec740258afbb14e5685168c5c9b99c51c03c72 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -6,7 +6,7 @@ import("//components/guest_view/buildflags/buildflags.gni")
 import("//device/vr/buildflags/buildflags.gni")
 import("//services/device/public/cpp/geolocation/buildflags.gni")
 
-source_set("permissions_common") {
+component("permissions_common") {
   sources = [
     "constants.cc",
     "constants.h",
@@ -181,6 +181,7 @@ source_set("permissions") {
   if (enable_guest_view) {
     deps += [ "//components/guest_view/browser" ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
