diff --git a/components/content_settings/core/common/cookie_settings_base.h b/components/content_settings/core/common/cookie_settings_base.h
index 509fbf0fa3adb1a249c5bf56d1657a52571eebad..d11098c20051e92ab79344eeebfe44c375cbf37c 100644
--- a/components/content_settings/core/common/cookie_settings_base.h
+++ b/components/content_settings/core/common/cookie_settings_base.h
@@ -169,6 +169,7 @@ class CookieSettingsBase {
   // Determines whether |setting| is a valid content setting for legacy cookie
   // access.
   static bool IsValidSettingForLegacyAccess(ContentSetting setting);
+  BRAVE_COOKIE_SETTINGS_BASE_H
 
  private:
   virtual ContentSetting GetCookieSettingInternal(
