diff --git a/components/permissions/permission_request_manager.cc b/components/permissions/permission_request_manager.cc
index 1c5faee4432ce4bf260a6799bfd2b0df26157dd3..0efe476b74c0d5b9c20438686a9b436feea64249 100644
--- a/components/permissions/permission_request_manager.cc
+++ b/components/permissions/permission_request_manager.cc
@@ -546,6 +546,7 @@ void PermissionRequestManager::OnVisibilityChanged(
     content::Visibility visibility) {
   bool tab_was_hidden = tab_is_hidden_;
   tab_is_hidden_ = visibility == content::Visibility::HIDDEN;
+  BRAVE_PERMISSION_REQUEST_MANAGER_ON_VISIBILITY_CHANGED
   if (tab_was_hidden == tab_is_hidden_) {
     return;
   }
@@ -597,6 +598,7 @@ GURL PermissionRequestManager::GetRequestingOrigin() const {
   GURL origin = requests_.front()->requesting_origin();
   if (DCHECK_IS_ON()) {
     for (const auto& request : requests_) {
+      BRAVE_PERMISSION_REQUEST_MANAGER_GET_REQUESTING_ORIGIN
       DCHECK_EQ(origin, request->requesting_origin());
     }
   }
