diff --git a/components/content_settings/core/browser/content_settings_utils.cc b/components/content_settings/core/browser/content_settings_utils.cc
index c6c382a59d9cccdd249754f2687dc69614c74535..26f3ee28b0f9e1563aff846ce5c1f24fb781c389 100644
--- a/components/content_settings/core/browser/content_settings_utils.cc
+++ b/components/content_settings/core/browser/content_settings_utils.cc
@@ -184,6 +184,7 @@ bool IsConstraintPersistent(const ContentSettingConstraints& constraints) {
 
 bool CanTrackLastVisit(ContentSettingsType type) {
   DCHECK(WebsiteSettingsRegistry::GetInstance()->Get(type)) << type;
+  BRAVE_CAN_TRACK_LAST_VISIT
 
   // Chooser based content settings will not be tracked by default.
   // Only allowlisted ones should be tracked.
