diff --git a/components/gcm_driver/gcm_driver.h b/components/gcm_driver/gcm_driver.h
index 96eeebe5d5027c49e14ecf6efc2f0f2fc6e87aee..014a91b502127de974850fccba6a3962861219e9 100644
--- a/components/gcm_driver/gcm_driver.h
+++ b/components/gcm_driver/gcm_driver.h
@@ -296,6 +296,8 @@ class GCMDriver {
                                     int interval_ms) = 0;
   virtual void RemoveHeartbeatInterval(const std::string& scope) = 0;
 
+  virtual void SetEnabled(bool enabled) {}
+
  protected:
   // Ensures that the GCM service starts (if necessary conditions are met).
   virtual GCMClient::Result EnsureStarted(GCMClient::StartMode start_mode) = 0;
