diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 576eca10b24b4355e3030dc15f6afba23bd12ef7..d666ac3b3182e53df5f17f8c9f5eb5ccb730d5f4 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -300,6 +300,7 @@ class SyncServiceImpl : public SyncService,
   size_t GetQueuedLocalDataMigrationItemCountForTest() const;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
