diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index 9f6fcbc10802bd28de72d68edd1fd7c6bc67fc10..c5c8b611c117e07fbe84f4f8b580545258bc71d8 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -1193,6 +1193,7 @@ void AutocompleteController::InitializeAsyncProviders(int provider_types) {
   if (provider_types & AutocompleteProvider::TYPE_SEARCH) {
     search_provider_ = new SearchProvider(provider_client_.get(), this);
     providers_.push_back(search_provider_.get());
+      BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
   }
   // Providers run in the order they're added.  Add `HistoryURLProvider` after
   // `SearchProvider` because:
@@ -1411,6 +1412,7 @@ void AutocompleteController::UpdateResult(UpdateType update_type,
     StartStopTimer();
   }
 
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
   PostProcessMatches();
 
   bool default_match_changed = CheckWhetherDefaultMatchChanged(
