diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 1d8eb7c2c2aed6c9848fec881cd9cb8009f202d8..d12f3c0ea4d9414a93de9e866ba23751f074d71c 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1403,6 +1403,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // via `HasContentSettingChangedViaPageInfo(type)`.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
