diff --git a/components/content_settings/core/common/features.h b/components/content_settings/core/common/features.h
index 817b2f1cc784f71a5174613dfe8a419a31aa755e..846d495ef9d157afc6f7d1f01dedb6f36d0ce8db 100644
--- a/components/content_settings/core/common/features.h
+++ b/components/content_settings/core/common/features.h
@@ -5,6 +5,8 @@
 #ifndef COMPONENTS_CONTENT_SETTINGS_CORE_COMMON_FEATURES_H_
 #define COMPONENTS_CONTENT_SETTINGS_CORE_COMMON_FEATURES_H_
 
+#include "base/component_export.h"
+
 namespace base {
 struct Feature;
 }  // namespace base
@@ -12,6 +14,7 @@ struct Feature;
 namespace content_settings {
 
 // Feature to enable a better cookie controls ui.
+COMPONENT_EXPORT(CONTENT_SETTINGS_FEATURES)
 extern const base::Feature kImprovedCookieControls;
 
 }  // namespace content_settings
