diff --git a/components/component_updater/BUILD.gn b/components/component_updater/BUILD.gn
index 3312bd62fc60feda28bcb775a3c7af6b0c6b36ca..ad406fa9be30cb89ab8cca7e40053bb5815d15e6 100644
--- a/components/component_updater/BUILD.gn
+++ b/components/component_updater/BUILD.gn
@@ -53,6 +53,7 @@ static_library("component_updater") {
     "//ui/base",
     "//url",
   ]
+  deps += [ "//brave/components/update_client" ]
 }
 
 static_library("test_support") {
