diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 32c67dcbcc4fdf39d1318f98b5fa67fd0e064f95..e5ad8fc951904847cd6dfb0082a6a020daea7b6a 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -112,6 +112,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
