diff --git a/components/sync/driver/profile_sync_service.h b/components/sync/driver/profile_sync_service.h
index 99ece8b31fc49ae4492efd49944ae9c45e639af6..be6a449da333a2c84b0282ecc26b32a8596373bd 100644
--- a/components/sync/driver/profile_sync_service.h
+++ b/components/sync/driver/profile_sync_service.h
@@ -245,6 +245,7 @@ class ProfileSyncService : public SyncService,
 
   static std::string GenerateCacheGUIDForTest();
 
+ BRAVE_PROFILE_SYNC_SERVICE_H_
  private:
   // Passed as an argument to StopImpl to control whether or not the sync
   // engine should clear its data when it shuts down. See StopImpl for more
