diff --git a/components/content_settings/browser/ui/cookie_controls_controller.cc b/components/content_settings/browser/ui/cookie_controls_controller.cc
index e67cbe1c50ec78602f79769ff3ddb81c72a894b0..a9cc5cdcd24fdb3b5c5ee3defab6ab1f9fb1fb4b 100644
--- a/components/content_settings/browser/ui/cookie_controls_controller.cc
+++ b/components/content_settings/browser/ui/cookie_controls_controller.cc
@@ -397,6 +397,7 @@ void CookieControlsController::RecordActivationMetrics() {
 
 CookieControlsState CookieControlsController::GetCookieControlsIconState(
     CookieControlsState controls_state) {
+  if ((true)) return CookieControlsState::kHidden;
   if (controls_state != CookieControlsState::kBlocked3pc) {
     return controls_state;
   }
