diff --git a/components/autofill/core/browser/ui/autofill_external_delegate.cc b/components/autofill/core/browser/ui/autofill_external_delegate.cc
index 3a5f442136d8e68d04362fa30ffce59f08dffd0a..bb4ebb64174c10a6875d05d43d95c08682dc11b7 100644
--- a/components/autofill/core/browser/ui/autofill_external_delegate.cc
+++ b/components/autofill/core/browser/ui/autofill_external_delegate.cc
@@ -519,6 +519,13 @@ void AutofillExternalDelegate::AttemptToDisplayAutofillSuggestions(
   // updated to match.
   InsertDataListValues(last_query_.field_datalist_options, suggestions);
 
+  if (trigger_field.has_value()) {
+    manager_->client().BraveAddSuggestions(
+        manager_->client().ClassifyAsPasswordForm(
+            *manager_, last_query_.form_id, last_query_.field_id),
+        *trigger_field, suggestions);
+  }
+
   // TODO(crbug.com/362630793): Try to eliminate this state. The controller
   // should be the one that knows about what suggestions were shown and passes
   // it on, not AED.
@@ -915,6 +922,10 @@ void AutofillExternalDelegate::DidAcceptSuggestion(
     const SuggestionMetadata& metadata,
     const FormGlobalId& form_id,
     const FieldGlobalId& field_id) {
+  if (manager_->client().BraveHandleSuggestion(suggestion,
+                                                last_query_.field_id)) {
+    return;
+  }
   CHECK(suggestion.IsAcceptable());
   // TODO(crbug.com/552871965): Extract the logging here to a separate function.
   base::UmaHistogramEnumeration("Autofill.Suggestions.AcceptedType",
