diff --git a/components/external_intents/android/BUILD.gn b/components/external_intents/android/BUILD.gn
index df7d3ee91b5bd81263ffca41f62c2fe6d415a7d6..280787788a799e46126c1bd6ab26e40d6fc2a7c6 100644
--- a/components/external_intents/android/BUILD.gn
+++ b/components/external_intents/android/BUILD.gn
@@ -65,6 +65,7 @@ static_library("android") {
   deps = [
     ":jni_headers",
     "//base",
+    "//brave/components/external_intents/android",
     "//components/navigation_interception",
     "//content/public/browser",
   ]
