diff --git a/components/sync/base/BUILD.gn b/components/sync/base/BUILD.gn
index 2986bd5d14d6574cb1bc13b2fbeea87a45ce7196..648d52f3561e8a51f67cebee33d12706a217f030 100644
--- a/components/sync/base/BUILD.gn
+++ b/components/sync/base/BUILD.gn
@@ -80,6 +80,7 @@ jumbo_static_library("base") {
   ]
   deps = [
     "//base:i18n",
+    "//brave/components/brave_sync:public",
     "//components/os_crypt",
     "//components/pref_registry",
     "//components/prefs",
