diff --git a/components/infobars/core/infobar_delegate.h b/components/infobars/core/infobar_delegate.h
index db006b910b129783c6d0798a6e94c9a6127e336b..30fff487e74dffa11cd996631ae78c7c0e94bc15 100644
--- a/components/infobars/core/infobar_delegate.h
+++ b/components/infobars/core/infobar_delegate.h
@@ -175,6 +175,7 @@ class InfoBarDelegate {
     // Removed: WEBID_PERMISSION_INFOBAR_DELEGATE = 104,
     AUTOFILL_OFFER_NOTIFICATION_INFOBAR_DELEGATE = 105,
     AUTOFILL_ADDRESS_PROFILE_INFOBAR_DELEGATE_IOS = 106,
+    BRAVE_INFOBAR_DELEGATE_IDENTIFIERS
   };
 
   // Describes navigation events, used to decide whether infobars should be
