diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index fb59f249d9b9a92c1308badf41f2e0a0312bed0d..3ac25fde7e9a292aa15a6fd83d7954fa18f64115 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -388,6 +388,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
