diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 4a8554d4863b4c96c517f395a32f017042299b85..e33092c9c1a150617c82df4b5f7abc73f5f50588 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -48,6 +48,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 typedef int64_t VisitID;
