diff --git a/components/embedder_support/user_agent_utils.cc b/components/embedder_support/user_agent_utils.cc
index 6571e2a8ebe70101890a8ea29284f7735c6dab86..08565f973e71132687c6205c085ab847d087131b 100644
--- a/components/embedder_support/user_agent_utils.cc
+++ b/components/embedder_support/user_agent_utils.cc
@@ -189,10 +189,11 @@ const blink::UserAgentBrandList GetUserAgentBrandList(
 #if !BUILDFLAG(CHROMIUM_BRANDING)
   brand = version_info::GetProductName();
 #endif
+  BRAVE_GET_USER_AGENT_BRAND_LIST
 
   std::string brand_version =
       output_version_type == blink::UserAgentBrandVersionType::kFullVersion
-          ? full_version
+          ? BRAVE_BRAND_VERSION_OVERRIDE_FOR_FULL_BRAND_VERSION_TYPE
           : major_version;
 
   return GenerateBrandVersionList(major_version_number, brand, brand_version,
@@ -900,6 +901,7 @@ std::string GetAndroidOSInfo(
 
   // Send information about the device.
   bool semicolon_inserted = false;
+  BRAVE_GET_ANDROID_OS_INFO
   if (include_android_model == IncludeAndroidModel::Include) {
     std::string android_device_name = BuildModelInfo();
     if (!android_device_name.empty()) {
