diff --git a/components/browser_ui/settings/android/java/src/org/chromium/components/browser_ui/settings/SettingsUtils.java b/components/browser_ui/settings/android/java/src/org/chromium/components/browser_ui/settings/SettingsUtils.java
index 66ca55396b33f3114b614a4b20e8c453e8ddc3a7..0327ade91115acdd6a66442f426e9d0a1bcb7e3d 100644
--- a/components/browser_ui/settings/android/java/src/org/chromium/components/browser_ui/settings/SettingsUtils.java
+++ b/components/browser_ui/settings/android/java/src/org/chromium/components/browser_ui/settings/SettingsUtils.java
@@ -147,7 +147,7 @@ public class SettingsUtils {
      * @param preference The preference to start from.
      * @param visiblePreferences The list to add visible preferences to.
      */
-    private static void addVisiblePreferences(
+    static void addVisiblePreferences(
             Preference preference, ArrayList<Preference> visiblePreferences) {
         visiblePreferences.add(preference);
         if (preference instanceof PreferenceGroup preferenceGroup) {
