diff --git a/components/content_settings/core/common/content_settings.h b/components/content_settings/core/common/content_settings.h
index d99ee45df1329825546aa2671d584ccdd254ce83..ff722c133d4204ed1548bbf833e860f3b212460e 100644
--- a/components/content_settings/core/common/content_settings.h
+++ b/components/content_settings/core/common/content_settings.h
@@ -72,6 +72,8 @@ struct RendererContentSettingRules {
   ContentSettingsForOneType autoplay_rules;
   ContentSettingsForOneType client_hints_rules;
   ContentSettingsForOneType popup_redirect_rules;
+  ContentSettingsForOneType fingerprinting_rules;
+  ContentSettingsForOneType brave_shields_rules;
 };
 
 namespace content_settings {
