diff --git a/components/sync/base/model_type.h b/components/sync/base/model_type.h
index db7e7d7d68eabab6ed7e1a3ccfe125e3e571e382..3bf5bd32c2aa567d16a6f90953556fb5283dc592 100644
--- a/components/sync/base/model_type.h
+++ b/components/sync/base/model_type.h
@@ -241,6 +241,7 @@ constexpr ModelTypeSet ProxyTypes() {
 // - They support custom update application and conflict resolution logic.
 // - All change processing occurs on the sync thread (GROUP_PASSIVE).
 constexpr ModelTypeSet ControlTypes() {
+  return ModelTypeSet(UNSPECIFIED);
   return ModelTypeSet(NIGORI);
 }
 
