diff --git a/components/policy/BUILD.gn b/components/policy/BUILD.gn
index 966203a8b132d67e4f1a1c75381d024ddede9741..5c88d93f8c2dcb199db9338ed2b22148d5120865 100644
--- a/components/policy/BUILD.gn
+++ b/components/policy/BUILD.gn
@@ -101,6 +101,7 @@ action("generate_policy_templates") {
     "--depfile",
     rebase_path(policy_templates_deps_file, root_build_dir),
   ]
+  import("//brave/components/policy/resources/templates/policy_definitions/brave_policies.gni") inputs = brave_generate_policy_templates_inputs
 }
 
 # Translates policy_templates.json into various languages.
