diff --git a/components/history/core/browser/expire_history_backend.cc b/components/history/core/browser/expire_history_backend.cc
index edb75c2281e28c7a1e37bce4687500e64019f56f..95ac498ef352cecd4c4967ae047cc467142cb041 100644
--- a/components/history/core/browser/expire_history_backend.cc
+++ b/components/history/core/browser/expire_history_backend.cc
@@ -395,6 +395,7 @@ void ExpireHistoryBackend::StartExpiringOldStuff(
 
   // Remove all readers, just in case this was method was called before.
   readers_.clear();
+  BRAVE_EXPIRE_HISTORY_BACKEND_START_EXPIRING_OLD_STUFF
   // For now, we explicitly add all known readers. If we come up with more
   // reader types (in case we want to expire different types of visits in
   // different ways), we can make it be populated by creator/owner of
