diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 699ea49d5139a538e852aa2018eb6c84e3aeaf5b..15b70a92362fbbf9ca3462e2142221806df66ecb 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -107,6 +107,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
