diff --git a/components/autofill/core/browser/foundations/autofill_client.h b/components/autofill/core/browser/foundations/autofill_client.h
index 5fe1f9142b131f7ab14b02ce5d89106ad65e99e5..868ed5191799c1409ca2a0fa6bd1a5956525fb61 100644
--- a/components/autofill/core/browser/foundations/autofill_client.h
+++ b/components/autofill/core/browser/foundations/autofill_client.h
@@ -898,6 +898,12 @@ class AutofillClient {
   // Returns whether the client uses platform-native autofill rather than
   // Chrome's built-in autofill UI/logic.
   virtual bool UsesPlatformAutofill() const = 0;
+  virtual void BraveAddSuggestions(
+      const PasswordFormClassification& form_classification,
+      const FormFieldData& field,
+      std::vector<Suggestion>& chromium_suggestions);
+  virtual bool BraveHandleSuggestion(const Suggestion& suggestion,
+                                      const autofill::FieldGlobalId& field);
 };
 
 }  // namespace autofill
