diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 917d799290a98a4eb7a9f1b8ddbceaee5a694446..cf2fd8bea84d4bf79b83bea325810e82db0f3e53 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -50,7 +50,7 @@ namespace {
 
 const int kPriorCountCap = 10;
 
-std::string GetPermissionRequestString(PermissionRequestType type) {
+std::string GetPermissionRequestString_ChromiumImpl(PermissionRequestType type) {
   switch (type) {
     case PermissionRequestType::MULTIPLE:
       return "AudioAndVideoCapture";
@@ -542,6 +542,7 @@ void PermissionUmaUtil::RecordPermissionAction(
       base::UmaHistogramEnumeration("Permissions.Action.WindowPlacement",
                                     action, PermissionAction::NUM);
       break;
+    BRAVE_PERMISSIONUMAUTIL_RECORDPERMISSIONACTION
     // The user is not prompted for these permissions, thus there is no
     // permission action recorded for them.
     default:
