diff --git a/components/sync/engine/sync_manager.h b/components/sync/engine/sync_manager.h
index 2a678f0a8d284de73dc0ef020b7869e7a570783d..1964971defdbb437b97ac44a7e8497fe566db907 100644
--- a/components/sync/engine/sync_manager.h
+++ b/components/sync/engine/sync_manager.h
@@ -257,6 +257,9 @@ class SyncManager {
     std::string cache_guid;
     std::string birthday;
     std::string bag_of_chips;
+
+    brave_sync::NudgeSyncCycleDelegate nudge_sync_cycle_delegate_function;
+    brave_sync::PollSyncCycleDelegate poll_sync_cycle_delegate_function;
   };
 
   // The state of sync the feature. If the user turned on sync explicitly, it
