diff --git a/components/browser_ui/site_settings/android/BUILD.gn b/components/browser_ui/site_settings/android/BUILD.gn
index 625717554a630601ed2051c3d090fb30fc1b3cff..caaf1c1af1b73bf1f6c5551df85ed7f0e3240106 100644
--- a/components/browser_ui/site_settings/android/BUILD.gn
+++ b/components/browser_ui/site_settings/android/BUILD.gn
@@ -75,6 +75,7 @@ android_library("java") {
     "java/src/org/chromium/components/browser_ui/site_settings/WebsitePreference.java",
     "java/src/org/chromium/components/browser_ui/site_settings/WebsitePreferenceBridge.java",
   ]
+  sources += brave_components_browser_ui_site_settings_android_java
   annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
   resources_package = "org.chromium.components.browser_ui.site_settings"
   deps = [
@@ -213,5 +214,6 @@ android_resources("java_resources") {
     "//components/permissions/android:java_resources",
     "//third_party/android_deps:android_support_v7_appcompat_java",
   ]
+  deps += brave_components_browser_ui_site_settings_android_java_resources_deps sources += brave_components_browser_ui_site_settings_android_java_resources
   create_srcjar = false
 }
