diff --git a/components/sync/base/data_type.h b/components/sync/base/data_type.h
index 9ccf04f79f6d10a790f6fbf3e6ef79ab67ddde26..73c3789e7325f43501b14b76c8190447e82cdc75 100644
--- a/components/sync/base/data_type.h
+++ b/components/sync/base/data_type.h
@@ -210,7 +210,8 @@ enum DataType {
   // Information about a notebook.
   NOTEBOOK,
 
-  LAST_USER_DATA_TYPE = NOTEBOOK,
+  AI_CHAT_CONVERSATION,
+  LAST_USER_DATA_TYPE = AI_CHAT_CONVERSATION,
 
   // ---- Control Types ----
   // An object representing a set of Nigori keys.
@@ -323,7 +324,8 @@ enum class DataTypeForHistograms {
   kEncryptedTabContextContainer = 81,
   kEncryptedTabContextItem = 82,
   kNotebook = 83,
-  kMaxValue = kNotebook,
+  kAIChatConversation = 83 + 1,
+  kMaxValue = kAIChatConversation,
 };
 // LINT.ThenChange(/tools/metrics/histograms/metadata/sync/enums.xml:SyncDataTypes)
 
