diff --git a/components/variations/variations_seed_store.h b/components/variations/variations_seed_store.h
index ef87229854b8d711a54237c86ca30d18926ecc89..67e44bfa04a7010ad9c6a92d7ca505f1f14a8264 100644
--- a/components/variations/variations_seed_store.h
+++ b/components/variations/variations_seed_store.h
@@ -301,6 +301,8 @@ class COMPONENT_EXPORT(VARIATIONS) VariationsSeedStore {
       base::OnceCallback<void(StoredSeedInfo)> done_callback,
       SeedType seed_type);
 
+  void SetSessionCountry(std::string_view country_code);
+
  protected:
   // Stores the serial number of the latest seed.
   void StoreLatestSerialNumber(std::string_view serial_number);
