diff --git a/components/content_settings/core/browser/BUILD.gn b/components/content_settings/core/browser/BUILD.gn
index 3918713bb0bba7664023146ce01311b1a6e807b7..e06c41a6dcd1f77808155105584aed5b43376250 100644
--- a/components/content_settings/core/browser/BUILD.gn
+++ b/components/content_settings/core/browser/BUILD.gn
@@ -61,6 +61,7 @@ jumbo_static_library("browser") {
     "//services/preferences/public/cpp",
     "//url",
   ]
+  deps += [ "//brave/components/content_settings/core/browser" ]
 
   if (is_android) {
     deps += [ "//media" ]
