diff --git a/components/component_updater/component_updater_service.h b/components/component_updater/component_updater_service.h
index 661c7e42e49fdd03f76979fd8297186bd9324a59..6e3c6c504a6f1b0e76f3eaf01bf9803dcaf3646c 100644
--- a/components/component_updater/component_updater_service.h
+++ b/components/component_updater/component_updater_service.h
@@ -203,6 +203,7 @@ class ComponentUpdateService {
   friend class screen_ai::ScreenAIDownloaderNonChromeOS;
   friend class speech::SodaInstallerImpl;
   friend class ::ComponentsHandler;
+  BRAVE_COMPONENT_UPDATER_SERVICE_H_
   FRIEND_TEST_ALL_PREFIXES(ComponentInstallerTest, RegisterComponent);
   FRIEND_TEST_ALL_PREFIXES(ComponentUpdaterTest, ComponentDetails);
   FRIEND_TEST_ALL_PREFIXES(ComponentUpdaterTest, UpdatesDisabled);
@@ -218,6 +219,7 @@ class OnDemandUpdater {
   enum class Priority { BACKGROUND = 0, FOREGROUND = 1 };
 
   virtual ~OnDemandUpdater() = default;
+  BRAVE_COMPONENT_UPDATER_SERVICE_H_ON_DEMAND_UPDATER
 
  private:
   friend class OnDemandTester;
