diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index f40a7416de8f62cd277c21c564c856e24e5fdf9e..3f847ab4119ed07bf7b5cb70e0dd8afaab5ac542 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -143,6 +143,7 @@ source_set("permissions") {
       "contexts/geolocation_permission_context_mac.h",
     ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
