diff --git a/components/safe_browsing/browser/safe_browsing_network_context.cc b/components/safe_browsing/browser/safe_browsing_network_context.cc
index 713f202f06f874faa1c9eb792585dd891fb8d81d..2c44ab779b8434458f65b22c9a7343d2449a0f26 100644
--- a/components/safe_browsing/browser/safe_browsing_network_context.cc
+++ b/components/safe_browsing/browser/safe_browsing_network_context.cc
@@ -73,7 +73,7 @@ class SafeBrowsingNetworkContext::SharedURLLoaderFactory
                                 traffic_annotation) override {
     DCHECK(content::BrowserThread::CurrentlyOn(content::BrowserThread::UI));
     GetURLLoaderFactory()->CreateLoaderAndStart(
-        std::move(loader), routing_id, request_id, options, request,
+        std::move(loader), routing_id, request_id, options, brave::OnBeforeSystemRequest(request),
         std::move(client), traffic_annotation);
   }
 
