diff --git a/components/contextual_tasks/public/features.cc b/components/contextual_tasks/public/features.cc
index 0ab7c8b5d8809688cc1e42ae5d4a756fcaae0b5c..32e1d15ac2518c01da109d70ab8e99a9617b469b 100644
--- a/components/contextual_tasks/public/features.cc
+++ b/components/contextual_tasks/public/features.cc
@@ -35,7 +35,7 @@ std::optional<bool> g_sticky_conversation_enabled_override;
 namespace contextual_tasks {
 
 // Enables the contextual tasks side panel while browsing.
-BASE_FEATURE(kContextualTasks, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kContextualTasks, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kContextualTasksPrivateApiNoAnimation,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
@@ -182,7 +182,7 @@ BASE_FEATURE(kContextualTasksOverrideShowBottomSheetOnLargeScreen,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Enables prefetching of cookies for contextual tasks.
-BASE_FEATURE(kContextualTasksCookiePrefetch, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kContextualTasksCookiePrefetch, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kAimTriggeredThreadLinks, base::FEATURE_ENABLED_BY_DEFAULT);
 
