diff --git a/components/autofill/core/browser/foundations/autofill_client.h b/components/autofill/core/browser/foundations/autofill_client.h
index 37965a54a0ae5944b7d7b9304408fe8a93096a8f..2dcff1b01bbc9538da7082717ae967785d9cc332 100644
--- a/components/autofill/core/browser/foundations/autofill_client.h
+++ b/components/autofill/core/browser/foundations/autofill_client.h
@@ -945,6 +945,12 @@ class AutofillClient {
   // Returns whether the client uses platform-native autofill rather than
   // Chrome's built-in autofill UI/logic.
   virtual bool UsesPlatformAutofill() const = 0;
+  virtual void BraveAddSuggestions(
+      const PasswordFormClassification& form_classification,
+      const FormFieldData& field,
+      std::vector<Suggestion>& chromium_suggestions);
+  virtual bool BraveHandleSuggestion(const Suggestion& suggestion,
+                                      const autofill::FieldGlobalId& field);
 };
 
 }  // namespace autofill
