diff --git a/components/content_settings/core/browser/content_settings_default_provider.cc b/components/content_settings/core/browser/content_settings_default_provider.cc
index 6da3be0a84c8109f1c546e6bf0f681a9813a0102..c78e237878c9b2c9f8b9fa3242c3324a188e79fa 100644
--- a/components/content_settings/core/browser/content_settings_default_provider.cc
+++ b/components/content_settings/core/browser/content_settings_default_provider.cc
@@ -400,6 +400,7 @@ void DefaultProvider::DiscardOrMigrateObsoletePreferences() {
     prefs_->SetBoolean(kBug364820109AlreadyWorkedAroundPref, true);
   }
 #endif  // !BUILDFLAG(IS_IOS)
+BRAVE_DISCARD_OR_MIGRATE_OBSOLETE_PREFERENCES
 }
 
 void DefaultProvider::RecordHistogramMetrics() {
