diff --git a/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java b/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java
index ed22448263bbe51443570f8fc8274e6557d75f62..a90c41d767021d7652fe1a000f9b002a97a570bf 100644
--- a/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java
+++ b/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java
@@ -29,7 +29,7 @@ import java.util.zip.GZIPOutputStream;
 @NullMarked
 public class MinidumpUploader {
     /* package */
-    static final String CRASH_URL_STRING = "https://clients2.google.com/cr/report";
+    static final String CRASH_URL_STRING = "https://cr.brave.com";
     /* package */
     static final String CONTENT_TYPE_TMPL = "multipart/form-data; boundary=%s";
 
