diff --git a/components/crx_file/crx_verifier.cc b/components/crx_file/crx_verifier.cc
index 4ae922a8a2844c3857960b9bfc3f32d6ef89e88f..027706b9a019d8c42c68e473180525dcdf3c2cba 100644
--- a/components/crx_file/crx_verifier.cc
+++ b/components/crx_file/crx_verifier.cc
@@ -213,6 +213,7 @@ VerifierResult VerifyCrx3(
       found_publisher_key =
           found_publisher_key || key_hash == kPublisherKeyHash ||
           (accept_publisher_test_key && key_hash == kPublisherTestKeyHash);
+      found_publisher_key = found_publisher_key || IsBravePublisher(key_hash);
       std::optional<crypto::keypair::PublicKey> pub_key =
           crypto::keypair::PublicKey::FromSubjectPublicKeyInfo(
               base::as_byte_span(key));
