diff --git a/components/safe_browsing/core/common/safe_browsing_prefs.cc b/components/safe_browsing/core/common/safe_browsing_prefs.cc
index 5edc7f2a71c6c52e711ff0e715e0eeba5323ba3b..fca1adfde36114c9509f48bf32cc924a9093e9e5 100644
--- a/components/safe_browsing/core/common/safe_browsing_prefs.cc
+++ b/components/safe_browsing/core/common/safe_browsing_prefs.cc
@@ -103,6 +103,7 @@ bool IsSafeBrowsingEnabled(const PrefService& prefs) {
 }
 
 bool IsEnhancedProtectionEnabled(const PrefService& prefs) {
+  BRAVE_IS_ENHANCED_PROTECTION_ENABLED
   // SafeBrowsingEnabled is checked too due to devices being out
   // of sync or not on a version that includes SafeBrowsingEnhanced pref.
   return prefs.GetBoolean(prefs::kSafeBrowsingEnhanced) &&
