diff --git a/components/content_settings/core/common/content_settings.h b/components/content_settings/core/common/content_settings.h
index 0565046f47c016b89815d38e54648c2fd42d4e9d..4b699e7d4c169f197a8ca4100b507cb725983d57 100644
--- a/components/content_settings/core/common/content_settings.h
+++ b/components/content_settings/core/common/content_settings.h
@@ -75,6 +75,8 @@ struct RendererContentSettingRules {
   ContentSettingsForOneType autoplay_rules;
   ContentSettingsForOneType client_hints_rules;
   ContentSettingsForOneType popup_redirect_rules;
+  ContentSettingsForOneType fingerprinting_rules;
+  ContentSettingsForOneType brave_shields_rules;
 };
 
 namespace content_settings {
