diff --git a/components/sync/engine/BUILD.gn b/components/sync/engine/BUILD.gn
index e000e2ef4699031fcf89554c1ac84406d3d86724..3a952e4137c937a85dfbfcd8b66307d9ea9c7c02 100644
--- a/components/sync/engine/BUILD.gn
+++ b/components/sync/engine/BUILD.gn
@@ -156,6 +156,7 @@ static_library("engine") {
     "traffic_logger.h",
     "update_handler.h",
   ]
+  import("//brave/components/sync/engine/sources.gni") sources += brave_components_sync_engine_sources
 
   configs += [
     # The target has >50 .cc files, so speed up the build with
