diff --git a/components/permissions/permission_context_base.cc b/components/permissions/permission_context_base.cc
index 3dcc60d1fe30dcdcb78ffb59bea4520276b0e6f1..ef5e540061be65a98ece73b2d1745cc05bd0122e 100644
--- a/components/permissions/permission_context_base.cc
+++ b/components/permissions/permission_context_base.cc
@@ -75,7 +75,7 @@
 #include "ui/android/window_android.h"
 #endif
 
-namespace permissions {
+namespace permissions::chromium_impl {
 namespace {
 
 using PermissionStatus = blink::mojom::PermissionStatus;
@@ -514,7 +514,7 @@ bool PermissionContextBase::IsPermissionAvailableToOrigins(
     // possible as this corresponds to the requirements of the secure contexts
     // spec and matches what is implemented in blink. Right now we just check
     // the top level and requesting origins.
-    if (!PermissionsClient::Get()->CanBypassEmbeddingOriginCheck(
+    if (!PermissionsClient::Get()->BraveCanBypassEmbeddingOriginCheck(content_settings_type_, 
             requesting_origin, embedding_origin) &&
         !network::IsUrlPotentiallyTrustworthy(embedding_origin)) {
       return false;
@@ -630,7 +630,7 @@ void PermissionContextBase::DecidePermission(
   // origin displayed in the prompt should never differ from the top-level
   // origin. Storage access API requests are excluded as they are expected to
   // request permissions from the frame origin needing access.
-  DCHECK(PermissionsClient::Get()->CanBypassEmbeddingOriginCheck(
+  DCHECK(PermissionsClient::Get()->BraveCanBypassEmbeddingOriginCheck(content_settings_type_, 
              request_data->requesting_origin, request_data->embedding_origin) ||
          request_data->requesting_origin == request_data->embedding_origin ||
          content_settings_type_ == ContentSettingsType::STORAGE_ACCESS);
@@ -935,4 +935,4 @@ void PermissionContextBase::NotifyObservers(
   }
 }
 
-}  // namespace permissions
+}  // namespace permissions::chromium_impl
