diff --git a/components/sync_bookmarks/BUILD.gn b/components/sync_bookmarks/BUILD.gn
index 40ee9b5de6eb017facd08aa639e29d72e787562c..efdf37f4691e6de19fb9cd4474a435dafd1bf2e1 100644
--- a/components/sync_bookmarks/BUILD.gn
+++ b/components/sync_bookmarks/BUILD.gn
@@ -28,6 +28,8 @@ static_library("sync_bookmarks") {
     "bookmark_sync_service.h",
     "synced_bookmark_tracker.cc",
     "synced_bookmark_tracker.h",
+    "//brave/components/sync_bookmarks/brave_bookmark_model_observer_impl.cc",
+    "//brave/components/sync_bookmarks/brave_bookmark_model_observer_impl.h",
   ]
 
   deps = [
