diff --git a/components/os_crypt/BUILD.gn b/components/os_crypt/BUILD.gn
index 9d7a8ad7a54e26e3517e613a85aa2ec90f598beb..eccf5dd0e69323177a4f4b8395a15372d4cf54e9 100644
--- a/components/os_crypt/BUILD.gn
+++ b/components/os_crypt/BUILD.gn
@@ -114,6 +114,7 @@ component("os_crypt") {
       defines += [ "USE_KWALLET" ]
     }
   }
+  import("//brave/components/os_crypt/sources.gni") deps += brave_components_os_crypt_deps
 }
 
 static_library("test_support") {
