diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index 943eb4b4c35d47cc6bddaec1bfbf2a5703e70fb7..24fc710d8c6e294983699725a2aba588ded04122 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -207,6 +207,7 @@ android_library("java") {
     "//third_party/jni_zero:jni_zero_java",
     "//ui/android:ui_java",
   ]
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
