diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index 354f19020944438c7f9e4ef61cb78b63031e1d70..b35b49d9540240f06f93cb41466ace22f1c6fa60 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -325,6 +325,7 @@ void PermissionManager::RequestPermissionsFromCurrentDocument(
         permission_status_callback) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
   const GURL requesting_origin =
+      BRAVE_FORCED_REQUESTING_ORIGIN
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   RequestPermissionsInternal(permissions_types, render_frame_host,
                              requesting_origin, user_gesture,
@@ -377,6 +378,7 @@ PermissionManager::GetPermissionResultForCurrentDocument(
       PermissionUtil::PermissionTypeToContentSettingType(permission);
 
   const GURL requesting_origin =
+      BRAVE_FORCED_REQUESTING_ORIGIN
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   const GURL embedding_origin =
       GetEmbeddingOrigin(render_frame_host, requesting_origin);
