diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index e273208cba2646054f773718d2318f465a1b4d14..dd164493199e3e91ee814c7b063a5d88f2dcc3ad 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -54,6 +54,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
