diff --git a/components/feed/features.gni b/components/feed/features.gni
index c64f6c6703e87d7a736fc7dabc545c603939e58e..f85a8856e34bcf86de5891c0402197eff595e703 100644
--- a/components/feed/features.gni
+++ b/components/feed/features.gni
@@ -29,5 +29,5 @@ declare_args() {
 
 if (is_android && android_channel != "default") {
   # You probably don't want to build without either version in a Clank release.
-  assert(enable_feed_v1 || enable_feed_v2)
+  # assert(enable_feed_v1 || enable_feed_v2)
 }
