diff --git a/components/search_engines/default_search_manager.cc b/components/search_engines/default_search_manager.cc
index db8d182f9145336b540729916daeeb8e6b682e18..9650f5c4fe18cce359e89147b1a8307a97f118f8 100644
--- a/components/search_engines/default_search_manager.cc
+++ b/components/search_engines/default_search_manager.cc
@@ -361,6 +361,7 @@ void DefaultSearchManager::LoadDefaultSearchEngineFromPrefs() {
   // Check if default search preference is overridden by extension.
   if (pref->IsExtensionControlled()) {
     extension_default_search_ = std::move(turl_data);
+  LOADDEFAULTSEARCHENGINEFROMPREFS_BRAVE
   } else {
     prefs_default_search_.SetAndReconcile(std::move(turl_data));
   }
