diff --git a/components/search_engines/default_search_manager.cc b/components/search_engines/default_search_manager.cc
index 88a56949914fb1c92b1116b5bcbfa7fe12f1da05..e3b8335ed06f13aae5c5f3390da67e1376477838 100644
--- a/components/search_engines/default_search_manager.cc
+++ b/components/search_engines/default_search_manager.cc
@@ -308,6 +308,7 @@ void DefaultSearchManager::LoadDefaultSearchEngineFromPrefs() {
   // Check if default search preference is overriden by extension.
   if (pref->IsExtensionControlled()) {
     extension_default_search_ = std::move(turl_data);
+  LOADDEFAULTSEARCHENGINEFROMPREFS_BRAVE
   } else {
     prefs_default_search_ = std::move(turl_data);
     MergePrefsDataWithPrepopulated();
