diff --git a/components/sync/engine/syncer_proto_util.cc b/components/sync/engine/syncer_proto_util.cc
index 2a174e799e26e2b8e9f447ec52246174d2c61153..42824bfa3812f62fc480ef60864655d70fdbe1e8 100644
--- a/components/sync/engine/syncer_proto_util.cc
+++ b/components/sync/engine/syncer_proto_util.cc
@@ -290,6 +290,7 @@ SyncerError SyncerProtoUtil::HandleClientToServerMessageResponse(
 
   SyncProtocolError sync_protocol_error =
       GetProtocolErrorFromResponse(response, cycle->context());
+  BRAVE_SYNCER_PROTO_UTIL_POST_CLIENT_TO_SERVER_MESSAGE
 
   // Inform the delegate of the error we got.
   cycle->delegate()->OnSyncProtocolError(sync_protocol_error);
