diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index b4f33d34ad27b7b2313dee7a3b178e10c423c88d..7bc5b9d5e611acc026c59e17c5bfdd181bbfb28d 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -313,6 +313,7 @@ AutocompleteController::AutocompleteController(
   if (provider_types & AutocompleteProvider::TYPE_SEARCH) {
     search_provider_ = new SearchProvider(provider_client_.get(), this);
     providers_.push_back(search_provider_.get());
+    BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
   }
   // It's important that the HistoryURLProvider gets added after SearchProvider:
   // AutocompleteController::Start() calls each providers' Start() function
@@ -869,6 +870,8 @@ void AutocompleteController::UpdateResult(
   metrics_.OnUpdateResult(last_result_for_logging,
                           result_.GetMatchDedupComparators());
 
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
+
   // Below are all annotations after the match list is ready.
 #if !BUILDFLAG(IS_IOS)
   // HistoryClusters is not enabled on iOS.
