diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index a98fde2bb0c3ba2da211a5197d107f354f8a0d38..bd27a3b6398e57753a1ecf0ec972f0f5e48c2bc4 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -398,6 +398,7 @@ AutocompleteController::AutocompleteController(
     if (provider_types & AutocompleteProvider::TYPE_SEARCH) {
       search_provider_ = new SearchProvider(provider_client_.get(), this);
       providers_.push_back(search_provider_.get());
+      BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
     }
     // Providers run in the order they're added.  Add `HistoryURLProvider` after
     // `SearchProvider` because:
@@ -973,6 +974,8 @@ void AutocompleteController::UpdateResult(
   metrics_.OnUpdateResult(last_result_for_logging,
                           result_.GetMatchDedupComparators());
 
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
+
   // Below are all annotations after the match list is ready.
 
   // Only produce Pedals for the default focus case (not on focus or on delete).
