diff --git a/components/component_updater/component_updater_service.h b/components/component_updater/component_updater_service.h
index 36215051767a1b1ff4d924e977b438047ccc1527..7f5666a756344fdd27baae84a0c4d323b9be9c12 100644
--- a/components/component_updater/component_updater_service.h
+++ b/components/component_updater/component_updater_service.h
@@ -186,6 +186,7 @@ class ComponentUpdateService {
 
   friend class speech::SodaInstallerImpl;
   friend class ::ComponentsHandler;
+  BRAVE_COMPONENT_UPDATER_SERVICE_H_
   FRIEND_TEST_ALL_PREFIXES(ComponentInstallerTest, RegisterComponent);
 };
 
@@ -199,6 +200,7 @@ class OnDemandUpdater {
   enum class Priority { BACKGROUND = 0, FOREGROUND = 1 };
 
   virtual ~OnDemandUpdater() = default;
+  BRAVE_COMPONENT_UPDATER_SERVICE_H_ON_DEMAND_UPDATER
 
  private:
   friend class OnDemandTester;
