diff --git a/components/assist_ranker/ranker_url_fetcher.cc b/components/assist_ranker/ranker_url_fetcher.cc
index 8d0f709499b5faf070f32dabef3d70f937751a86..92d14d11eaf347aa1029affcc2aeae5c8e65bac7 100644
--- a/components/assist_ranker/ranker_url_fetcher.cc
+++ b/components/assist_ranker/ranker_url_fetcher.cc
@@ -31,6 +31,7 @@ bool RankerURLFetcher::Request(
     const GURL& url,
     RankerURLFetcher::Callback callback,
     network::mojom::URLLoaderFactory* url_loader_factory) {
+  return false; // feature disabled in Brave
   // This function is not supposed to be called if the previous operation is not
   // finished.
   if (state_ == REQUESTING) {
