diff --git a/components/content_settings/core/common/content_settings_types.h b/components/content_settings/core/common/content_settings_types.h
index 9c6728a428ba0ec1fd058a7f29661c8e55a4bf9c..30f6bcc5b913e4ababb4dd3c63a3d0b4d98ddac0 100644
--- a/components/content_settings/core/common/content_settings_types.h
+++ b/components/content_settings/core/common/content_settings_types.h
@@ -287,6 +287,7 @@ enum class ContentSettingsType : int32_t {
   // in local time.
   NOTIFICATION_INTERACTIONS,
 
+  BRAVE_CONTENT_SETTINGS_TYPES_LIST,
   NUM_TYPES,
 };
 
