diff --git a/components/content_settings/core/browser/content_settings_utils.cc b/components/content_settings/core/browser/content_settings_utils.cc
index 9ef38f6c0d4996cf2bc098dde6244a90fd894479..f942020d7c624733872360936b4b6615a0689cb9 100644
--- a/components/content_settings/core/browser/content_settings_utils.cc
+++ b/components/content_settings/core/browser/content_settings_utils.cc
@@ -188,6 +188,7 @@ bool IsConstraintPersistent(const ContentSettingConstraints& constraints) {
 
 bool CanTrackLastVisit(ContentSettingsType type) {
   DCHECK(WebsiteSettingsRegistry::GetInstance()->Get(type)) << type;
+  BRAVE_CAN_TRACK_LAST_VISIT
 
   // Chooser based content settings will not be tracked by default.
   // Only allowlisted ones should be tracked.
