diff --git a/components/omnibox/browser/omnibox_text_util.cc b/components/omnibox/browser/omnibox_text_util.cc
index b1ac72ce3c8034e0a799680bd0ae14e4539b61a8..34a7bf829837b571b2c5c921009e919816b547c5 100644
--- a/components/omnibox/browser/omnibox_text_util.cc
+++ b/components/omnibox/browser/omnibox_text_util.cc
@@ -165,6 +165,7 @@ void AdjustTextForCopy(
   if (!has_user_modified_text) {
     *url_from_text = navigation_entry_url;
     *write_url = true;
+    BRAVE_ADJUST_TEXT_FOR_COPY
 
     // Don't let users copy Reader Mode page URLs.
     // We display the original article's URL in the omnibox, so users will
