diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index ba6c7c783565e8922c20a6dd48380ba7da41f821..e30a0abf230ff831c3b0e03c68e84eeaca8463c6 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -19,6 +19,7 @@ group("sync") {
   if (is_chromeos) {
     public_deps += [ "//components/sync/chromeos" ]
   }
+  import("//brave/build/config/brave_build.gni") public_deps += brave_components_sync_deps
 }
 
 static_library("test_support") {
