diff --git a/components/sync/engine/syncer_proto_util.cc b/components/sync/engine/syncer_proto_util.cc
index 4f0614f0553ca88ea0c3d9b7a5696892658d29ba..d253fe36dad95a6f542ed0e19c55fca072a3d0c4 100644
--- a/components/sync/engine/syncer_proto_util.cc
+++ b/components/sync/engine/syncer_proto_util.cc
@@ -279,6 +279,7 @@ SyncerError SyncerProtoUtil::HandleClientToServerMessageResponse(
 
   SyncProtocolError sync_protocol_error =
       GetProtocolErrorFromResponse(response, cycle->context());
+  BRAVE_SYNCER_PROTO_UTIL_POST_CLIENT_TO_SERVER_MESSAGE
 
   // Inform the delegate of the error we got.
   cycle->delegate()->OnSyncProtocolError(sync_protocol_error);
