diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index fba4727d020cb21f4ebc0aefc2513aec2ba4b833..e4532527eba6ca5dfbbc103f799806e834ad635b 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -312,6 +312,7 @@ AutocompleteController::AutocompleteController(
   if (provider_types & AutocompleteProvider::TYPE_SEARCH) {
     search_provider_ = new SearchProvider(provider_client_.get(), this);
     providers_.push_back(search_provider_.get());
+    BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
   }
   // It's important that the HistoryURLProvider gets added after SearchProvider:
   // AutocompleteController::Start() calls each providers' Start() function
@@ -818,6 +819,7 @@ void AutocompleteController::UpdateResult(
     AutocompleteResult::LogUpdateMetrics(last_result_for_logging, result_,
                                          in_start_);
   }
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
 
   // Below are all annotations after the match list is ready.
 #if !BUILDFLAG(IS_IOS)
