diff --git a/components/page_info/page_info_ui.cc b/components/page_info/page_info_ui.cc
index 8fe18e4fd94c65fa40c4c841b2635098fa91bb31..584626e076b7a94071c3f462feeedbb727c71b8a 100644
--- a/components/page_info/page_info_ui.cc
+++ b/components/page_info/page_info_ui.cc
@@ -652,6 +652,7 @@ const gfx::ImageSkia PageInfoUI::GetPermissionIcon(
     SkColor related_text_color) {
   const gfx::VectorIcon* icon = &gfx::kNoneIcon;
   switch (info.type) {
+    BRAVE_PAGE_INFO_UI_GET_PERMISSION_ICON
     case ContentSettingsType::COOKIES:
       icon = &vector_icons::kCookieIcon;
       break;
