diff --git a/components/content_settings/core/common/content_settings.mojom b/components/content_settings/core/common/content_settings.mojom
index 3635caff1e290b6ad3463375bf3b9d99cf3ebeb7..7ea8d332f797fabdbe254ea851cd535194f9522f 100644
--- a/components/content_settings/core/common/content_settings.mojom
+++ b/components/content_settings/core/common/content_settings.mojom
@@ -73,4 +73,6 @@ struct RendererContentSettingRules {
   array<ContentSettingPatternSource> autoplay_rules;
   array<ContentSettingPatternSource> client_hints_rules;
   array<ContentSettingPatternSource> popup_redirect_rules;
+  array<ContentSettingPatternSource> fingerprinting_rules;
+  array<ContentSettingPatternSource> brave_shields_rules;
 };
