diff --git a/components/sync/base/model_type.h b/components/sync/base/model_type.h
index 3cd09c203aee13933c43c83f6d960ce62e4fe66a..eeb0ce3faff36fff9eea92a73e85e26ee1baa0fe 100644
--- a/components/sync/base/model_type.h
+++ b/components/sync/base/model_type.h
@@ -135,6 +135,10 @@ enum ModelType {
   // (crbug.com/1223853)
   // WEBAUTHN_CREDENTIAL,
 
+  // Rewards VG-specific objects. Brave only.
+  VG_BODIES,
+  VG_SPEND_STATUSES,
+
   // Proxy types are excluded from the sync protocol, but are still considered
   // real user types. By convention, we prefix them with 'PROXY_' to distinguish
   // them from normal protocol types.
@@ -230,7 +234,9 @@ enum class ModelTypeForHistograms {
   kSharingMessage = 48,
   kAutofillWalletOffer = 49,
   kWorkspaceDesk = 50,
-  kMaxValue = kWorkspaceDesk
+  kVgBodies = 51,
+  kVgSpendStatuses = 52,
+  kMaxValue = kVgSpendStatuses
 };
 
 // Used to mark the type of EntitySpecifics that has no actual data.
@@ -252,7 +258,8 @@ constexpr ModelTypeSet ProtocolTypes() {
       DEVICE_INFO, PRIORITY_PREFERENCES, SUPERVISED_USER_SETTINGS, APP_LIST,
       ARC_PACKAGE, PRINTERS, READING_LIST, USER_EVENTS, NIGORI, USER_CONSENTS,
       SEND_TAB_TO_SELF, SECURITY_EVENTS, WEB_APPS, WIFI_CONFIGURATIONS,
-      OS_PREFERENCES, OS_PRIORITY_PREFERENCES, SHARING_MESSAGE, WORKSPACE_DESK);
+      OS_PREFERENCES, OS_PRIORITY_PREFERENCES, SHARING_MESSAGE, WORKSPACE_DESK,
+      VG_BODIES, VG_SPEND_STATUSES);
 }
 
 // These are the normal user-controlled types. This is to distinguish from
