diff --git a/components/gcm_driver/gcm_client_impl.cc b/components/gcm_driver/gcm_client_impl.cc
index 7e12cd5fdad159e1a21fa7ef7324b8dba0dc7b9b..df490a2ad476fee3e19421dba2db93e5a46a5aa5 100644
--- a/components/gcm_driver/gcm_client_impl.cc
+++ b/components/gcm_driver/gcm_client_impl.cc
@@ -357,6 +357,7 @@ void GCMClientImpl::Initialize(
 }
 
 void GCMClientImpl::Start(StartMode start_mode) {
+  return; // GCM disabled in Brave
   DCHECK_NE(UNINITIALIZED, state_);
 
   if (state_ == LOADED) {
@@ -864,6 +865,7 @@ void GCMClientImpl::ResetCache() {
 
 void GCMClientImpl::Register(
     const linked_ptr<RegistrationInfo>& registration_info) {
+  return; // GCM disabled in Brave
   DCHECK_EQ(state_, READY);
 
   // Registrations should never pass as an app_id the special category used
