diff --git a/components/sync_device_info/device_info.h b/components/sync_device_info/device_info.h
index 801a4510537393fcfc049cc9787d4db3a8947ab9..217549d95d209c734592cf9516e5ae679cf7c8c2 100644
--- a/components/sync_device_info/device_info.h
+++ b/components/sync_device_info/device_info.h
@@ -145,6 +145,7 @@ class DeviceInfo {
   DeviceInfo(DeviceInfo&&) = delete;
   DeviceInfo& operator=(const DeviceInfo&) = delete;
 
+  virtual
   ~DeviceInfo();
 
   // Sync specific unique identifier for the device. Note if a device
