diff --git a/components/sync/engine/BUILD.gn b/components/sync/engine/BUILD.gn
index 9fd0e5fab5f580fd28c12cf8ff4dc2025fcc8a9c..c4bf15915a00f26f35a0e56063295288eb252f51 100644
--- a/components/sync/engine/BUILD.gn
+++ b/components/sync/engine/BUILD.gn
@@ -162,6 +162,7 @@ static_library("engine") {
     "traffic_logger.h",
     "update_handler.h",
   ]
+  import("//brave/components/sync/sources.gni") sources += brave_components_sync_sources
 
   configs += [
     # The target has >50 .cc files, so speed up the build with
