diff --git a/components/content_settings/core/browser/content_settings_utils.cc b/components/content_settings/core/browser/content_settings_utils.cc
index e58c5d0c98f875a9e11eea2bba90fe94bc8376e1..f670943c93e06016a08518ead2ff3b3020ce6aaf 100644
--- a/components/content_settings/core/browser/content_settings_utils.cc
+++ b/components/content_settings/core/browser/content_settings_utils.cc
@@ -186,6 +186,7 @@ bool CanTrackLastVisit(ContentSettingsType type) {
   if (type == ContentSettingsType::PROTOCOL_HANDLERS) {
     return false;
   }
+  BRAVE_CAN_TRACK_LAST_VISIT
 
   // Chooser based content settings will not be tracked by default.
   // Only allowlisted ones should be tracked.
