diff --git a/components/sync/engine_impl/sync_manager_impl.cc b/components/sync/engine_impl/sync_manager_impl.cc
index 43f420ae64a80c2fda1b8f17b53d68e1ccbd3b3d..88c310568287d7abed5003cba6cf75d7c8a76e73 100644
--- a/components/sync/engine_impl/sync_manager_impl.cc
+++ b/components/sync/engine_impl/sync_manager_impl.cc
@@ -396,8 +396,9 @@ void SyncManagerImpl::Init(InitArgs* args) {
       args->poll_interval);
   scheduler_ = args->engine_components_factory->BuildScheduler(
       name_, cycle_context_.get(), args->cancelation_signal,
-      args->enable_local_sync_backend);
+      true);
 
+  BRAVE_SYNC_MANAGER_IMPL_INIT
   scheduler_->Start(SyncScheduler::CONFIGURATION_MODE, base::Time());
 
   initialized_ = true;
