diff --git a/components/infobars/core/infobar_delegate.h b/components/infobars/core/infobar_delegate.h
index c88e620cbbbdf6c526c72da2eb5fd9b847ed427f..bf109b5a5e990e759319f172d535b3501c2af076 100644
--- a/components/infobars/core/infobar_delegate.h
+++ b/components/infobars/core/infobar_delegate.h
@@ -181,6 +181,7 @@ class InfoBarDelegate {
     SUPPORTED_LINKS_INFOBAR_DELEGATE_CHROMEOS = 109,
     AUTOFILL_VIRTUAL_CARD_ENROLLMENT_INFOBAR_DELEGATE_MOBILE = 110,
     BASIC_PRINT_DEPRECATED_ACCELERATOR_DELEGATE = 111,
+    BRAVE_INFOBAR_DELEGATE_IDENTIFIERS
   };
 
   // Describes navigation events, used to decide whether infobars should be
