diff --git a/components/sync_device_info/device_info.h b/components/sync_device_info/device_info.h
index 67ea6988ac913e442094c764cf05417ece6c1568..4ab71195e014d49f7f84def6e84b8d24ec4e93c6 100644
--- a/components/sync_device_info/device_info.h
+++ b/components/sync_device_info/device_info.h
@@ -196,6 +196,7 @@ class DeviceInfo {
   DeviceInfo(const DeviceInfo&) = delete;
   DeviceInfo& operator=(const DeviceInfo&) = delete;
 
+  virtual
   ~DeviceInfo();
 
   // Sync specific unique identifier for the device. Note if a device
