diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index 88d12bf5e10ad48b713d99ba9dcafea8d06b02de..c236b8cb5f7a96ded987764a81197da44156c58c 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -538,6 +538,7 @@ CookieSettingsBase::DecideAccess(
     return BlockAllCookies{};
   }
 
+  BRAVE_COOKIE_SETTINGS_BASE_DECIDE_ACCESS
   if (!is_third_party_request) {
     return AllowAllCookies{ThirdPartyCookieAllowMechanism::kNone};
   }
@@ -712,6 +713,7 @@ CookieSettingsBase::GetCookieSettingInternal(
     CHECK(is_third_party_request, base::NotFatalUntil::M128);
     CHECK(global_setting_or_embedder_blocks_third_party_cookies,
           base::NotFatalUntil::M128);
+    BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL_IS_EXPLICIT_SETTING
     CHECK(!is_explicit_setting, base::NotFatalUntil::M128);
 
     FireStorageAccessHistogram(StorageAccessResult::ACCESS_BLOCKED);
