diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index 5bd511c6824689339cc2340306fc693d189854a7..50e8615fa8d057a5cbfdf22aced95e0c9ea6086f 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -129,6 +129,7 @@ source_set("permissions") {
       "contexts/geolocation_permission_context_mac.h",
     ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
