diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index 96d7af0319c885e883669516fc05de60417fa3a1..654a6581c6a63f00c1463d194a8089528a8e7c5b 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -396,6 +396,7 @@ void PermissionManager::RequestPermissionsFromCurrentDocument(
 
   const PermissionRequestID request_id(render_frame_host, request_local_id);
   const GURL requesting_origin =
+      BRAVE_FORCED_REQUESTING_ORIGIN
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   const GURL embedding_origin =
       GetEmbeddingOrigin(render_frame_host, requesting_origin);
@@ -455,6 +456,7 @@ PermissionStatus PermissionManager::GetPermissionStatusForCurrentDocument(
       PermissionUtil::PermissionTypeToContentSetting(permission);
 
   const GURL requesting_origin =
+      BRAVE_FORCED_REQUESTING_ORIGIN
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   const GURL embedding_origin =
       GetEmbeddingOrigin(render_frame_host, requesting_origin);
