diff --git a/components/sync/engine/sync_manager.h b/components/sync/engine/sync_manager.h
index 16aee17d067b29470ac81bc5d3ed995e4a5277a7..5f6c90bf08611a7c0922047072a1d89d892fc4a7 100644
--- a/components/sync/engine/sync_manager.h
+++ b/components/sync/engine/sync_manager.h
@@ -255,6 +255,7 @@ class SyncManager {
     std::string cache_guid;
     std::string birthday;
     std::string bag_of_chips;
+    BRAVE_SYNC_MANAGER_INIT_ARGS_H
   };
 
   // The state of sync the feature. If the user turned on sync explicitly, it
