diff --git a/components/ntp_tiles/features.cc b/components/ntp_tiles/features.cc
index 8b7090b758230ec8eb85a520cb53e07e74ce65e0..d58f67b99ad03fc29089375822d7fb34de86acff 100644
--- a/components/ntp_tiles/features.cc
+++ b/components/ntp_tiles/features.cc
@@ -29,13 +29,13 @@ AimButtonRefactorArm GetAimButtonRefactorArm() {
 
 const char kPopularSitesFieldTrialName[] = "NTPPopularSites";
 
-BASE_FEATURE(kPopularSitesBakedInContentFeature,
+BASE_OVERRIDDEN_FEATURE(kPopularSitesBakedInContentFeature,
              "NTPPopularSitesBakedInContent",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
-BASE_FEATURE(kNtpMostLikelyFaviconsFromServerFeature,
+BASE_OVERRIDDEN_FEATURE(kNtpMostLikelyFaviconsFromServerFeature,
              "NTPMostLikelyFaviconsFromServer",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kPopularSitesRefreshUs, base::FEATURE_DISABLED_BY_DEFAULT);
 
