diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index 45faf4c091036af858c596b5cb1b1e842756a12b..fd1c3db8c4e6df45e422ee24ad67d53f23084877 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -192,6 +192,7 @@ android_library("java") {
     "//third_party/jni_zero:jni_zero_java",
     "//ui/android:ui_java",
   ]
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
