diff --git a/components/os_crypt/key_storage_linux.cc b/components/os_crypt/key_storage_linux.cc
index 2e388de793af588722c857cd7b896df9734f2b6e..4034440183fc42d2abb8ece6dcb5f23680ec8153 100644
--- a/components/os_crypt/key_storage_linux.cc
+++ b/components/os_crypt/key_storage_linux.cc
@@ -32,8 +32,8 @@
 const char KeyStorageLinux::kFolderName[] = "Chrome Keys";
 const char KeyStorageLinux::kKey[] = "Chrome Safe Storage";
 #else
-const char KeyStorageLinux::kFolderName[] = "Chromium Keys";
-const char KeyStorageLinux::kKey[] = "Chromium Safe Storage";
+const char KeyStorageLinux::kFolderName[] = "Brave Keys";
+const char KeyStorageLinux::kKey[] = "Brave Safe Storage";
 #endif
 
 namespace {
