diff --git a/components/omnibox/browser/omnibox_text_util.cc b/components/omnibox/browser/omnibox_text_util.cc
index 5e2bb7aa71940318ada9477adfd4504e551c78d4..de40b3f0383a0d39ea3b8275abe71c17a841b5fc 100644
--- a/components/omnibox/browser/omnibox_text_util.cc
+++ b/components/omnibox/browser/omnibox_text_util.cc
@@ -153,6 +153,7 @@ void AdjustTextForCopy(int sel_min,
   if (!has_user_modified_text) {
     *url_from_text = client->GetNavigationEntryURL();
     *write_url = true;
+    BRAVE_ADJUST_TEXT_FOR_COPY
 
     // Don't let users copy Reader Mode page URLs.
     // We display the original article's URL in the omnibox, so users will
