diff --git a/components/sync/protocol/proto_visitors.h b/components/sync/protocol/proto_visitors.h
index 6a8f5adbceedb7822e74a51935ed38c76864b4df..3c8285c0475e727bb176ce614627fe29bbe7852e 100644
--- a/components/sync/protocol/proto_visitors.h
+++ b/components/sync/protocol/proto_visitors.h
@@ -123,6 +123,7 @@
   void VisitProtoFields(V& visitor, proto)
 
 namespace syncer {
+BRAVE_VISIT_PROTO_FIELDS
 
 VISIT_PROTO_FIELDS(const sync_pb::AppListSpecifics& proto) {
   VISIT(item_id);
@@ -623,6 +624,7 @@ VISIT_PROTO_FIELDS(const sync_pb::DeletionOrigin& proto) {
 }
 
 VISIT_PROTO_FIELDS(const sync_pb::DeviceInfoSpecifics& proto) {
+  VISIT(brave_fields);
   VISIT(cache_guid);
   VISIT(client_name);
   VISIT_ENUM(device_type);
@@ -1529,6 +1531,7 @@ VISIT_PROTO_FIELDS(const sync_pb::UserEventSpecifics::FlocIdComputed& proto) {
 }
 
 VISIT_PROTO_FIELDS(const sync_pb::TabNavigation& proto) {
+  VISIT(brave_fields);
   VISIT(virtual_url);
   VISIT(referrer);
   VISIT(title);
