diff --git a/components/version_info/BUILD.gn b/components/version_info/BUILD.gn
index 073db629d39fc986f85cbd9d3cc359ed9f79c408..0a782dd533bccaf94076696d66ab8f5444cd0371 100644
--- a/components/version_info/BUILD.gn
+++ b/components/version_info/BUILD.gn
@@ -12,6 +12,7 @@ declare_args() {
 }
 
 static_library("version_info") {
+  configs += [ "//brave/build:version" ]
   sources = [
     "version_info.cc",
     "version_info.h",
