diff --git a/components/contextual_tasks/public/features.cc b/components/contextual_tasks/public/features.cc
index 0aeb1bf4b15900c1c8cc1d78b08e9df86f787a8f..0069a071a1609f08293042d0ec3a08967ab20f04 100644
--- a/components/contextual_tasks/public/features.cc
+++ b/components/contextual_tasks/public/features.cc
@@ -35,7 +35,7 @@ std::optional<bool> g_sticky_conversation_enabled_override;
 namespace contextual_tasks {
 
 // Enables the contextual tasks side panel while browsing.
-BASE_FEATURE(kContextualTasks, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kContextualTasks, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kContextualTasksPrivateApiNoAnimation,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
@@ -186,7 +186,7 @@ BASE_FEATURE(kContextualTasksOverrideShowBottomSheetOnLargeScreen,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Enables prefetching of cookies for contextual tasks.
-BASE_FEATURE(kContextualTasksCookiePrefetch, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kContextualTasksCookiePrefetch, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kAimTriggeredThreadLinks, base::FEATURE_ENABLED_BY_DEFAULT);
 
