diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 759552cb220018c50aa5b84cd2d41d9fd5145b7c..adaebf60e3965aa21f42ce82daf4b045a53fdfc5 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -109,6 +109,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
