diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 1d64d3ccd60b7aa9e5b0cc6eb6bb37801846f133..7881c45ca26c9a05b0f569d34fb98678763f7cbf 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -50,7 +50,7 @@ namespace {
 
 const int kPriorCountCap = 10;
 
-std::string GetPermissionRequestString(PermissionRequestType type) {
+std::string GetPermissionRequestString_ChromiumImpl(PermissionRequestType type) {
   switch (type) {
     case PermissionRequestType::MULTIPLE:
       return "AudioAndVideoCapture";
@@ -644,6 +644,7 @@ void PermissionUmaUtil::RecordPermissionAction(
       base::UmaHistogramEnumeration("Permissions.Action.IdleDetection", action,
                                     PermissionAction::NUM);
       break;
+    BRAVE_PERMISSIONUMAUTIL_RECORDPERMISSIONACTION
     // The user is not prompted for these permissions, thus there is no
     // permission action recorded for them.
     default:
