diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 0798bf1f073c5853d26463e828147832e9b6a76c..1a7736d494586df67c0dcf1072a9f9e382b81fde 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -970,6 +970,7 @@ void PageInfo::PresentSitePermissions() {
       }
     }
 
+    BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
     if (ShouldShowPermission(
             permission_info, site_url_, content_settings, web_contents_.get(),
             HasContentSettingChangedViaPageInfo(permission_info.type),
