diff --git a/components/translate/core/common/BUILD.gn b/components/translate/core/common/BUILD.gn
index dee8b37dfbcd88f127dd0a15ccbc141fe6d7c011..645b983599edb08296223d3c3632f7c078972932 100644
--- a/components/translate/core/common/BUILD.gn
+++ b/components/translate/core/common/BUILD.gn
@@ -30,6 +30,8 @@ static_library("common") {
     "//third_party/metrics_proto",
     "//url",
   ]
+
+  import("//brave/components/translate/core/common/sources.gni") sources += brave_components_translate_core_common_sources public_deps = brave_components_translate_core_common_public_deps
 }
 
 source_set("unit_tests") {
