diff --git a/components/embedder_support/user_agent_utils.cc b/components/embedder_support/user_agent_utils.cc
index 0d00f1c0a8239bad8093506b65e6df0f40cecedf..f2a3d2419b5b3d48a51811da481cde7b24100794 100644
--- a/components/embedder_support/user_agent_utils.cc
+++ b/components/embedder_support/user_agent_utils.cc
@@ -166,10 +166,11 @@ const blink::UserAgentBrandList GetUserAgentBrandList(
 #if !BUILDFLAG(CHROMIUM_BRANDING)
   brand = version_info::GetProductName();
 #endif
+  BRAVE_GET_USER_AGENT_BRAND_LIST
 
   std::string brand_version =
       output_version_type == blink::UserAgentBrandVersionType::kFullVersion
-          ? full_version
+          ? BRAVE_BRAND_VERSION_OVERRIDE_FOR_FULL_BRAND_VERSION_TYPE
           : major_version;
 
   return GenerateBrandVersionList(major_version_number, brand, brand_version,
@@ -882,6 +883,7 @@ std::string GetAndroidOSInfo(
 
   // Send information about the device.
   bool semicolon_inserted = false;
+  BRAVE_GET_ANDROID_OS_INFO
   if (include_android_model == IncludeAndroidModel::Include) {
     std::string android_device_name = BuildModelInfo();
     if (!android_device_name.empty()) {
