diff --git a/components/infobars/core/infobar_delegate.h b/components/infobars/core/infobar_delegate.h
index 706319fdf5dea1cde8f5e08ecb82a9ab358e9588..2462c2b1e663e32f06125ca2f6033e05f7a83e04 100644
--- a/components/infobars/core/infobar_delegate.h
+++ b/components/infobars/core/infobar_delegate.h
@@ -191,6 +191,7 @@ class InfoBarDelegate {
     PARCEL_TRACKING_INFOBAR_DELEGATE = 117,
     TEST_THIRD_PARTY_COOKIE_PHASEOUT_DELEGATE = 118,
     ENABLE_LINK_CAPTURING_INFOBAR_DELEGATE = 119,
+    BRAVE_INFOBAR_DELEGATE_IDENTIFIERS
   };
 
   // Describes navigation events, used to decide whether infobars should be
