diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index d9229c474d10e7e2fd552ea0096a6928ef5ed02b..af9ffa48fcbc88c3a674daff7f0ce4601b9fe6a2 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -53,6 +53,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
