diff --git a/components/policy/core/browser/configuration_policy_pref_store.cc b/components/policy/core/browser/configuration_policy_pref_store.cc
index 7440287dc5a08d93af1353ec3bb65141dbfb293f..9098815d3376c831f95f8a2c9b688954fd1780da 100644
--- a/components/policy/core/browser/configuration_policy_pref_store.cc
+++ b/components/policy/core/browser/configuration_policy_pref_store.cc
@@ -155,6 +155,8 @@ ConfigurationPolicyPrefStore::CreatePreferencesFromPolicies() {
     }
   }
 
+  BRAVE_CREATE_PREFERENCES_FROM_POLICIES
+
   return prefs;
 }
 
