diff --git a/components/sync/driver/sync_stopped_reporter.h b/components/sync/driver/sync_stopped_reporter.h
index 07bc682b645a906a336dc2699008ec272498c1de..a7cd991d65d00e9884b5ac03d547a82a39804723 100644
--- a/components/sync/driver/sync_stopped_reporter.h
+++ b/components/sync/driver/sync_stopped_reporter.h
@@ -35,7 +35,7 @@ class SyncStoppedReporter {
   SyncStoppedReporter(const SyncStoppedReporter&) = delete;
   SyncStoppedReporter& operator=(const SyncStoppedReporter&) = delete;
 
-  ~SyncStoppedReporter();
+  virtual ~SyncStoppedReporter();
 
   // Inform the sync server that sync was stopped on this device.
   // |access_token|, |cache_guid|, and |birthday| must not be empty.
