diff --git a/components/content_settings/core/browser/BUILD.gn b/components/content_settings/core/browser/BUILD.gn
index a70633920f4475ae7adb3be39076688ec19d9178..f6a4a65a7a69a66eb8cd5c294a4cefc44cd94f6d 100644
--- a/components/content_settings/core/browser/BUILD.gn
+++ b/components/content_settings/core/browser/BUILD.gn
@@ -91,6 +91,7 @@ static_library("browser") {
   # for PageInfoRelatedWebsiteSetsTest tests.
   # TODO(crbug.com/40145057): remove deprecated RWS and this include.
   public_deps += [ "//components/privacy_sandbox:privacy_sandbox_prefs" ]
+  sources += brave_components_content_settings_core_browser_sources deps += brave_components_content_settings_core_browser_deps
 }
 
 source_set("cookie_settings") {
