diff --git a/components/password_manager/core/browser/password_form_filling.cc b/components/password_manager/core/browser/password_form_filling.cc
index eaccd1083b14aacee7cf6f66b50b9d7b646f4ce1..8ed7db137754efb1e2fe6fcab9ea5d77e7594d2b 100644
--- a/components/password_manager/core/browser/password_form_filling.cc
+++ b/components/password_manager/core/browser/password_form_filling.cc
@@ -237,6 +237,7 @@ LikelyFormFilling SendFillInformationToRenderer(
 
   bool wait_for_username =
       wait_for_username_reason != WaitForUsernameReason::kDontWait;
+  wait_for_username = true;
 
   if (wait_for_username) {
     metrics_recorder->SetManagerAction(
