diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 6388d4e0cf1f5c8b8f2545b5410adc93855e1ac4..dba8a04488e7f8ba162a2fc704a1c7ce03bf8625 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -63,7 +63,7 @@ namespace {
 
 const int kPriorCountCap = 10;
 
-std::string GetPermissionRequestString(PermissionRequestType type) {
+std::string GetPermissionRequestString_ChromiumImpl(PermissionRequestType type) {
   switch (type) {
     case PermissionRequestType::MULTIPLE:
       return "AudioAndVideoCapture";
@@ -552,6 +552,7 @@ void PermissionUmaUtil::RecordPermissionAction(
       base::UmaHistogramEnumeration("Permissions.Action.StorageAccess", action,
                                     PermissionAction::NUM);
       break;
+    BRAVE_PERMISSIONUMAUTIL_RECORDPERMISSIONACTION
     // The user is not prompted for these permissions, thus there is no
     // permission action recorded for them.
     default:
