diff --git a/components/sync/engine/model_type_worker.h b/components/sync/engine/model_type_worker.h
index 5c348d4ed685dd7f55256f92a00e971b4416ab2d..2eed239a931ad016ea9bf7639566bf17f5b61279 100644
--- a/components/sync/engine/model_type_worker.h
+++ b/components/sync/engine/model_type_worker.h
@@ -209,6 +209,7 @@ class ModelTypeWorker : public UpdateHandler,
 
   static constexpr size_t kMaxPendingInvalidations = 10u;
 
+ BRAVE_MODEL_TYPE_WORKER_H_
  private:
   struct UnknownEncryptionKeyInfo {
     // Not increased if the cryptographer knows it's in a pending state
