diff --git a/components/content_settings/core/common/content_settings.cc b/components/content_settings/core/common/content_settings.cc
index f09911e191160ad2f24bc47eeb63a3f6b0031990..d78420d0a3b9aa344a747efd2c48ea26fc3f7b0d 100644
--- a/components/content_settings/core/common/content_settings.cc
+++ b/components/content_settings/core/common/content_settings.cc
@@ -101,6 +101,7 @@ constexpr HistogramValue kHistogramValue[] = {
     {ContentSettingsType::HTTP_ALLOWED, 80},
     {ContentSettingsType::FORMFILL_METADATA, 81},
     {ContentSettingsType::FEDERATED_IDENTITY_ACTIVE_SESSION, 82},
+    BRAVE_HISTOGRAM_VALUE_LIST
 };
 
 }  // namespace
