diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index 8b429a9ced4b83f55832bf853d89fe6ef4ea0129..675660ae51de04ee382c68918cc942cdcf72c9d0 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -106,6 +106,7 @@ source_set("permissions") {
     "//ui/base",
     "//url",
   ]
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
