diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index 1a0311a7d9a138b703d839dd66dd85b41b9f578d..9af651cacd5342c76751bd79ac768da83834ef1a 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -308,6 +308,7 @@ void PermissionManager::RequestPermissionsFromCurrentDocument(
     base::OnceCallback<void(const std::vector<PermissionStatus>&)>
         permission_status_callback) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
+  BRAVE_REQUEST_PERMISSION_FROM_CURRENT_DOCUMENT
   RequestPermissionsInternal(render_frame_host, request_description,
                              std::move(permission_status_callback));
 }
@@ -355,6 +356,7 @@ PermissionManager::GetPermissionResultForCurrentDocument(
       PermissionUtil::PermissionTypeToContentSettingType(permission);
 
   const GURL requesting_origin =
+      BRAVE_GET_PERMISSION_RESULT_FOR_CURRENT_DOCUMENT
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   const GURL embedding_origin =
       GetEmbeddingOrigin(render_frame_host, requesting_origin);
