diff --git a/components/crash/core/app/crashpad_linux.cc b/components/crash/core/app/crashpad_linux.cc
index 733ef9eac99f228166b013883c9cb9a0e7b470af..67f2a6ae294d01a197f01bf01508f5a32ba5f680 100644
--- a/components/crash/core/app/crashpad_linux.cc
+++ b/components/crash/core/app/crashpad_linux.cc
@@ -140,7 +140,7 @@ bool PlatformCrashpadInitialization(
     const std::vector<std::string>& initial_arguments,
     const std::vector<base::FilePath>& attachments,
     base::FilePath* database_path) {
-  DCHECK_EQ(initial_client, browser_process);
+  // DCHECK_EQ(initial_client, browser_process);  // fix for Brave VPN apps
   DCHECK(initial_arguments.empty());
 
   // Not used on Linux.
