diff --git a/components/sync_bookmarks/bookmark_change_processor.cc b/components/sync_bookmarks/bookmark_change_processor.cc
index 4c50233634aaef7c3d86cffd1465351dbcc388b8..47b5e7b42adf4dd28f0f2dc7ea1ccd92f9615e04 100644
--- a/components/sync_bookmarks/bookmark_change_processor.cc
+++ b/components/sync_bookmarks/bookmark_change_processor.cc
@@ -87,6 +87,7 @@ void BookmarkChangeProcessor::UpdateSyncNodeProperties(
   bookmark_specifics.set_creation_time_us(src->date_added().ToInternalValue());
   dst->SetBookmarkSpecifics(bookmark_specifics);
   SetSyncNodeFavicon(src, model, dst);
+  BRAVE_BOOKMARK_CHANGE_PROCESSOR_UPDATE_SYNC_NODE_PROPERTIES
   SetSyncNodeMetaInfo(src, dst);
 }
 
@@ -380,7 +381,6 @@ int64_t BookmarkChangeProcessor::UpdateSyncNode(
 
 void BookmarkChangeProcessor::BookmarkMetaInfoChanged(
     BookmarkModel* model, const BookmarkNode* node) {
-  BookmarkNodeChanged(model, node);
 }
 
 void BookmarkChangeProcessor::BookmarkNodeMoved(BookmarkModel* model,
@@ -414,6 +414,7 @@ void BookmarkChangeProcessor::BookmarkNodeMoved(BookmarkModel* model,
       error_handler()->OnUnrecoverableError(error);
       return;
     }
+    BRAVE_BOOKMARK_CHANGE_PROCESSOR_BOOKMARK_NODE_MOVED
 
     if (!PlaceSyncNode(MOVE, new_parent, new_index, &trans, &sync_node,
                        model_associator_)) {
@@ -457,6 +458,7 @@ void BookmarkChangeProcessor::BookmarkNodeFaviconChanged(
     return;
   }
 
+  BRAVE_BOOKMARK_CHANGE_PROCESSOR_BOOKMARK_NODE_FAVICON_CHANGED
   CreateOrUpdateSyncNode(node);
 }
 
@@ -489,6 +491,7 @@ void BookmarkChangeProcessor::BookmarkNodeChildrenReordered(
       DCHECK_EQ(sync_child.GetParentId(),
                 model_associator_->GetSyncIdFromChromeId(node->id()));
 
+      BRAVE_BOOKMARK_CHANGE_PROCESSOR_CHILDREN_REORDERED
       if (!PlaceSyncNode(MOVE, node, i, &trans, &sync_child,
                          model_associator_)) {
         syncer::SyncError error(FROM_HERE,
@@ -732,6 +735,7 @@ void BookmarkChangeProcessor::ApplyChangesFromSyncModel(
   // sync order, left to right, moving them into their proper positions.
   for (auto it = to_reposition.begin(); it != to_reposition.end(); ++it) {
     const BookmarkNode* parent = it->second->parent();
+    BRAVE_BOOKMARK_CHANGE_PROCESSOR_APPLY_CHANGES_FROM_SYNC_MODEL
     model->Move(it->second, parent, size_t{it->first});
   }
 
