diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 36cef660afd7737f79009d80ea0d6561cfbd8b32..3d507d4031d9c6c69b6ad04f121ed01cc8b845b0 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -512,6 +512,9 @@ static_library("browser") {
     ]
     deps += [ "//content/public/common" ]
   }
+  sources += brave_components_omnibox_browser_sources
+  public_deps += brave_components_omnibox_browser_public_deps
+  deps += brave_components_omnibox_browser_deps
 }
 
 source_set("pref_names") {
@@ -1080,6 +1083,7 @@ fuzzer_test("search_suggestion_parser_fuzzer") {
 }
 
 mojom("mojo_bindings") {
+  generate_legacy_js_bindings = true
   sources = [ "omnibox.mojom" ]
   webui_module_path = "chrome://resources/mojo/components/omnibox/browser"
 
