diff --git a/components/update_client/BUILD.gn b/components/update_client/BUILD.gn
index a3d379ae2fc6ff64f248af29557e66cebd1196d3..429c66da50d417387614c0da157230b740b27741 100644
--- a/components/update_client/BUILD.gn
+++ b/components/update_client/BUILD.gn
@@ -190,6 +190,7 @@ static_library("update_client") {
       "background_downloader_win.h",
     ]
   }
+  deps += brave_components_update_client_deps public_deps = brave_components_update_client_public_deps
 }
 
 source_set("test_support") {
