diff --git a/components/sync/driver/BUILD.gn b/components/sync/driver/BUILD.gn
index 80ab10a408c7cc55a5729f881dc0078bb63ad8df..f8854ac9579323278f1f04291166210520e45590 100644
--- a/components/sync/driver/BUILD.gn
+++ b/components/sync/driver/BUILD.gn
@@ -108,6 +108,7 @@ static_library("driver") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps defines = brave_service_key_defines
 }
 
 static_library("test_support") {
