diff --git a/components/sync/driver/BUILD.gn b/components/sync/driver/BUILD.gn
index 1d02961c42ba0e6909caba3ac5926703e9d0c803..809e7f19d42603a9b92b59f173e9cd27fd06b5aa 100644
--- a/components/sync/driver/BUILD.gn
+++ b/components/sync/driver/BUILD.gn
@@ -108,6 +108,7 @@ static_library("driver") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps defines = brave_service_key_defines
 }
 
 static_library("test_support") {
