diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 3ce2d10528aff49c12b09ff8327dc738c847a28c..5c658b27db4322d66c212d3d62423a7dae8d5183 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -51,6 +51,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
