diff --git a/components/omnibox/browser/autocomplete_match.h b/components/omnibox/browser/autocomplete_match.h
index 9cae0aee2523ed979c1073f74efea64c889cbdfb..70bd61bbe024c93f9cc5ae09d8c1aee600362551 100644
--- a/components/omnibox/browser/autocomplete_match.h
+++ b/components/omnibox/browser/autocomplete_match.h
@@ -719,7 +719,7 @@ struct AutocompleteMatch {
   // The provider of this match, used to remember which provider the user had
   // selected when the input changes. This may be NULL, in which case there is
   // no provider (or memory of the user's selection).
-  raw_ptr<AutocompleteProvider> provider = nullptr;
+  raw_ptr<AutocompleteProvider, DanglingUntriaged> provider = nullptr;
 
   // The relevance of this match. See table in autocomplete_provider.h for
   // scores returned by various providers. This is used to rank matches among
