diff --git a/components/content_settings/core/common/content_settings_types.h b/components/content_settings/core/common/content_settings_types.h
index cd92cee23aa9a692ec6732e79ce8794667692024..e42630b21e2452eacf3e2dd6cd4fe042c054267b 100644
--- a/components/content_settings/core/common/content_settings_types.h
+++ b/components/content_settings/core/common/content_settings_types.h
@@ -291,6 +291,7 @@ enum class ContentSettingsType : int32_t {
   // for a given origin, to be used on future visits to the origin.
   REDUCED_ACCEPT_LANGUAGE,
 
+  BRAVE_CONTENT_SETTINGS_TYPES_LIST,
   NUM_TYPES,
 };
 
