diff --git a/components/minidump_uploader/BUILD.gn b/components/minidump_uploader/BUILD.gn
index ff635a06f614cf44f3afb198e6d5b2c093ed2aa6..3c0a88a28feb0cb6ef958d60383bca34e07238e4 100644
--- a/components/minidump_uploader/BUILD.gn
+++ b/components/minidump_uploader/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/components/minidump_uploader/sources.gni")
 import("//build/config/android/rules.gni")
 import("//testing/test.gni")
 import("//third_party/jni_zero/jni_zero.gni")
@@ -59,6 +60,7 @@ android_library("minidump_uploader_java") {
     "//third_party/androidx:androidx_annotation_annotation_java",
     "//third_party/jni_zero:jni_zero_java",
   ]
+  deps += brave_components_minidump_uploader_util_java_deps
 
   srcjar_deps = [ ":minidump_uploader_jni_headers" ]
   sources = [
@@ -75,6 +77,7 @@ android_library("minidump_uploader_java") {
     "android/java/src/org/chromium/components/minidump_uploader/util/HttpURLConnectionFactoryImpl.java",
     "android/java/src/org/chromium/components/minidump_uploader/util/NetworkPermissionUtil.java",
   ]
+  sources += brave_components_minidump_uploader_util_java_sources
 }
 
 android_library("minidump_uploader_java_test_support") {
