diff --git a/components/autofill/core/browser/ui/autofill_external_delegate.cc b/components/autofill/core/browser/ui/autofill_external_delegate.cc
index f04f7b80b7ae48529b36b464a89aaac21db47e90..9ebc115c8f1dd47c0151ca70e9fadae239819e22 100644
--- a/components/autofill/core/browser/ui/autofill_external_delegate.cc
+++ b/components/autofill/core/browser/ui/autofill_external_delegate.cc
@@ -524,6 +524,13 @@ void AutofillExternalDelegate::AttemptToDisplayAutofillSuggestions(
   // updated to match.
   InsertDataListValues(last_query_.field_datalist_options, suggestions);
 
+  if (trigger_field.has_value()) {
+    manager_->client().BraveAddSuggestions(
+        manager_->client().ClassifyAsPasswordForm(
+            *manager_, last_query_.form_id, last_query_.field_id),
+        *trigger_field, suggestions);
+  }
+
   // TODO(crbug.com/362630793): Try to eliminate this state. The controller
   // should be the one that knows about what suggestions were shown and passes
   // it on, not AED.
@@ -906,6 +913,10 @@ void AutofillExternalDelegate::DidSelectSuggestion(
 void AutofillExternalDelegate::DidAcceptSuggestion(
     const Suggestion& suggestion,
     const SuggestionMetadata& metadata) {
+  if (manager_->client().BraveHandleSuggestion(suggestion,
+                                                last_query_.field_id)) {
+    return;
+  }
   CHECK(suggestion.IsAcceptable());
   // TODO(crbug.com/552871965): Extract the logging here to a separate function.
   base::UmaHistogramEnumeration("Autofill.Suggestions.AcceptedType",
