diff --git a/components/browsing_data/core/BUILD.gn b/components/browsing_data/core/BUILD.gn
index adb48084b7514cc3323cdc8d05bba22b02726daa..cf534bb91ec08274efa58f75e777dc26750ae1dc 100644
--- a/components/browsing_data/core/BUILD.gn
+++ b/components/browsing_data/core/BUILD.gn
@@ -2,11 +2,14 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/components/browsing_data/core/sources.gni")
+
 if (is_android) {
   import("//build/config/android/rules.gni")
 }
 
 static_library("core") {
+  public_deps = brave_components_browsing_data_core_public_deps
   sources = [
     "browsing_data_policies_utils.cc",
     "browsing_data_policies_utils.h",
