diff --git a/components/omnibox/browser/omnibox_text_util.cc b/components/omnibox/browser/omnibox_text_util.cc
index 67865669203bb123983d324bfeca40c28f3fecb2..56567f1471984ceab8b9eededa57a29735897bb3 100644
--- a/components/omnibox/browser/omnibox_text_util.cc
+++ b/components/omnibox/browser/omnibox_text_util.cc
@@ -153,6 +153,7 @@ void AdjustTextForCopy(int sel_min,
   if (!has_user_modified_text) {
     *url_from_text = client->GetNavigationEntryURL();
     *write_url = true;
+    BRAVE_ADJUST_TEXT_FOR_COPY
 
     // Don't let users copy Reader Mode page URLs.
     // We display the original article's URL in the omnibox, so users will
