diff --git a/components/update_client/BUILD.gn b/components/update_client/BUILD.gn
index a904b265e18001ca82b516e147930cceb6cd88a7..607ce02e5e5e960342337d939998fe5f25c86420 100644
--- a/components/update_client/BUILD.gn
+++ b/components/update_client/BUILD.gn
@@ -193,6 +193,8 @@ static_library("update_client") {
       "puffin_patcher.h",
     ]
   }
+
+  deps += brave_components_update_client_deps public_deps = brave_components_update_client_public_deps
 }
 
 static_library("test_support") {
