diff --git a/components/sync/protocol/device_info_specifics.proto b/components/sync/protocol/device_info_specifics.proto
index 89518c1de2a99433a7dd9567e867c1d3c6c3a99d..c57bd75004c03e59341397289720266a7cb6ee7d 100644
--- a/components/sync/protocol/device_info_specifics.proto
+++ b/components/sync/protocol/device_info_specifics.proto
@@ -17,6 +17,7 @@ option optimize_for = LITE_RUNTIME;
 package sync_pb;
 
 import "components/sync/protocol/sync_enums.proto";
+import "brave/components/sync/protocol/brave_device_info_specifics.proto";
 
 // Information about a device that is running a sync-enabled Chrome browser.  We
 // are mapping the per-client cache guid to more specific information about the
@@ -98,6 +99,7 @@ message DeviceInfoSpecifics {
 
   // The form factor of the device.
   optional SyncEnums.DeviceFormFactor device_form_factor = 20;
+  optional BraveSpecificFields brave_fields = 1000;
 }
 
 // Feature specific information about the device that is running a sync-enabled
