diff --git a/components/omnibox/browser/omnibox_controller.cc b/components/omnibox/browser/omnibox_controller.cc
index e2bc11edcf6721fd36462b87ba2852e5e76385d2..a36a62afa38a3c20b856515c2887ed5e2cd4bd22 100644
--- a/components/omnibox/browser/omnibox_controller.cc
+++ b/components/omnibox/browser/omnibox_controller.cc
@@ -14,12 +14,14 @@
 #include "components/omnibox/browser/omnibox_popup_view.h"
 #include "ui/gfx/geometry/rect.h"
 
+#include "brave/components/omnibox/browser/brave_autocomplete_controller.h"
+
 OmniboxController::OmniboxController(OmniboxEditModel* omnibox_edit_model,
                                      OmniboxClient* client)
     : omnibox_edit_model_(omnibox_edit_model),
       client_(client),
       popup_(nullptr),
-      autocomplete_controller_(new AutocompleteController(
+      autocomplete_controller_(new BraveAutocompleteController(
           client_->CreateAutocompleteProviderClient(),
           this,
           AutocompleteClassifier::DefaultOmniboxProviders())),
