diff --git a/components/permissions/permission_context_base.h b/components/permissions/permission_context_base.h
index 72a307ee4d213faf8611d06442f8688cd93513bb..aae95c86a89814a6aae4dd54b9dcee4c54e20dc7 100644
--- a/components/permissions/permission_context_base.h
+++ b/components/permissions/permission_context_base.h
@@ -146,6 +146,7 @@ class PermissionContextBase : public content_settings::Observer {
     return content_settings_type_;
   }
 
+  BRAVE_PERMISSION_CONTEXT_BASE_H_
  protected:
   virtual ContentSetting GetPermissionStatusInternal(
       content::RenderFrameHost* render_frame_host,
