diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index c3c4b04d7b8b05ab1459c84352af4ee6ee9b1fb4..eda576ccc8ace25260d10d7e1393b3ce7f4d7a0b 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -56,6 +56,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 typedef int64_t VisitID;
