diff --git a/components/sync_preferences/BUILD.gn b/components/sync_preferences/BUILD.gn
index 2b9df91b988ae4e00eabdfc0f47c1c0a92281ecf..70d3b1df8c332201e791a145fe616584f22d3162 100644
--- a/components/sync_preferences/BUILD.gn
+++ b/components/sync_preferences/BUILD.gn
@@ -67,6 +67,7 @@ source_set("common_syncable_prefs_database") {
     "//components/translate/core/browser",
     "//components/variations/service",
   ]
+  import("//brave/components/sync_preferences/sources.gni") public_deps = brave_common_syncable_prefs_database_public_deps
 }
 
 static_library("test_support") {
