diff --git a/components/regional_capabilities/BUILD.gn b/components/regional_capabilities/BUILD.gn
index 9874aced1abfdc306b1df44b696fb5c03123e07d..d4d6a58d0ed1874293249eae68f4d2d50f0edb68 100644
--- a/components/regional_capabilities/BUILD.gn
+++ b/components/regional_capabilities/BUILD.gn
@@ -44,6 +44,7 @@ source_set("country_access_reason") {
     "//chrome/browser/ui:ui",
     "//components/search_engines:search_engines",
   ]
+  import("//brave/browser/search_engines/sources.gni") visibility += brave_browser_search_engines_visibility + [ "//brave/browser/brave_ads:impl" ]
 
   sources = [ "access/country_access_reason.h" ]
 
