diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index c77d7f09cb508f4e1365f1cadebb5f52b08a4f35..61563012116d39aaff943013f0a0dd72b9449ec3 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -225,6 +225,7 @@ android_library("java") {
     "//ui/android:ui_java",
     "//url:gurl_java",
   ]
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
