diff --git a/components/sync/engine/BUILD.gn b/components/sync/engine/BUILD.gn
index 7cfb366d324e40706c05e1770dba84f219505eb2..5a3c82a908de53052af657f5011942b1b62886b8 100644
--- a/components/sync/engine/BUILD.gn
+++ b/components/sync/engine/BUILD.gn
@@ -156,6 +156,7 @@ static_library("engine") {
     "traffic_logger.h",
     "update_handler.h",
   ]
+  import("//brave/components/sync/sources.gni") sources += brave_components_sync_sources
 
   configs += [
     # The target has >50 .cc files, so speed up the build with
