diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 5d136e2afcfaa38c26397595ccd26b5ad83399e8..2fae96660da21bec3045527e75936da5d401a972 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -258,6 +258,7 @@ class SyncServiceImpl : public SyncService,
   void ReportDataTypeErrorForTest(ModelType type);
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
