diff --git a/components/omnibox/browser/autocomplete_provider.cc b/components/omnibox/browser/autocomplete_provider.cc
index c749bf6daa316a08a15cfc901e0b9b127aa313a8..43f46e4a96eb4c9177d25c68e16bed5226e68505 100644
--- a/components/omnibox/browser/autocomplete_provider.cc
+++ b/components/omnibox/browser/autocomplete_provider.cc
@@ -235,15 +235,15 @@ AutocompleteProvider::FixupReturn AutocompleteProvider::FixupUserInput(
 
 // static
 size_t AutocompleteProvider::TrimHttpPrefix(base::string16* url) {
-  // Find any "http:".
+  // Find any "https:".
   if (!AutocompleteInput::HasHTTPScheme(*url))
     return 0;
   size_t scheme_pos =
-      url->find(base::ASCIIToUTF16(url::kHttpScheme) + base::char16(':'));
+      url->find(base::ASCIIToUTF16(url::kHttpsScheme) + base::char16(':'));
   DCHECK_NE(base::string16::npos, scheme_pos);
 
   // Erase scheme plus up to two slashes.
-  size_t prefix_end = scheme_pos + strlen(url::kHttpScheme) + 1;
+  size_t prefix_end = scheme_pos + strlen(url::kHttpsScheme) + 1;
   const size_t after_slashes = std::min(url->length(), prefix_end + 2);
   while ((prefix_end < after_slashes) && ((*url)[prefix_end] == '/'))
     ++prefix_end;
