diff --git a/components/omnibox/browser/featured_search_provider.cc b/components/omnibox/browser/featured_search_provider.cc
index 3af604b09d2b00d38e83c412641f7f95d761b8b4..617f9caaee93460b47e0e56bf9308fbc586b1d2e 100644
--- a/components/omnibox/browser/featured_search_provider.cc
+++ b/components/omnibox/browser/featured_search_provider.cc
@@ -290,6 +290,7 @@ void FeaturedSearchProvider::AddFeaturedKeywordMatches(
   for (TemplateURL* turl : turls) {
     if (turl->starter_pack_id() > 0 &&
         turl->is_active() == TemplateURLData::ActiveStatus::kTrue) {
+      BRAVE_FEATURED_SEARCH_PROVIDER_ADD_FEATURED_KEYWORD_MATCHES
       // Skip @gemini if feature disabled.
       if (turl->starter_pack_id() == template_url_starter_pack_data::kGemini &&
           !OmniboxFieldTrial::IsStarterPackExpansionEnabled()) {
