diff --git a/components/sync_device_info/device_info_sync_bridge_unittest.cc b/components/sync_device_info/device_info_sync_bridge_unittest.cc
index 44f5cd9e7d27e3bd34589293fff5369f0625c594..9f37ac7fa86584087e7a3f0d05ffaa9f9fb1e24a 100644
--- a/components/sync_device_info/device_info_sync_bridge_unittest.cc
+++ b/components/sync_device_info/device_info_sync_bridge_unittest.cc
@@ -864,7 +864,7 @@ TEST_F(DeviceInfoSyncBridgeTest, ApplySyncChangesWithLocalGuid) {
   auto error_on_delete = bridge()->ApplySyncChanges(
       bridge()->CreateMetadataChangeList(), std::move(entity_change_list));
   EXPECT_FALSE(error_on_delete);
-  EXPECT_EQ(1, change_count());
+  EXPECT_EQ(2, change_count());  // Brave can delete other device in chain, delete record should be also processed
 }
 
 TEST_F(DeviceInfoSyncBridgeTest, ApplyDeleteNonexistent) {
