diff --git a/components/sync/base/features.h b/components/sync/base/features.h
index eb236fc6f74f761af3c7449751b2c1a7756e6bf9..66b3c2f3931d0cc32c223d42e8752d2806ca8bbc 100644
--- a/components/sync/base/features.h
+++ b/components/sync/base/features.h
@@ -114,7 +114,7 @@ DECLARE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuable);
 #endif
 
 // Enables syncing of usage metadata from Google Wallet passes.
-DECLARE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuableMetadata);
+BASE_DECLARE_FEATURE(kSyncAutofillValuableMetadata);
 
 // Enables syncing of flight reservations coming from Google Wallet.
 DECLARE_SYNC_AUTOFILL_AI_FEATURE(kSyncWalletFlightReservations);
