diff --git a/components/permissions/request_type.h b/components/permissions/request_type.h
index fac8dfb5ff0e6cdad186749a3b7c3549323cf987..6a089b6c7b80084bfeaad331141b1c84bbd83abe 100644
--- a/components/permissions/request_type.h
+++ b/components/permissions/request_type.h
@@ -50,6 +50,7 @@ enum class RequestType {
 #endif
   kStorageAccess,
   kVrSession,
+  BRAVE_REQUEST_TYPES
 #if !defined(OS_ANDROID)
   kWindowPlacement,
   kMaxValue = kWindowPlacement
