diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index 11049c5db2990a57774dc3dc4860e7aca8e54913..554437155731e0009713c1009dd8cf8e5507b3cc 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -659,6 +659,7 @@ CookieSettingsBase::DecideAccess(const GURL& url,
     return BlockAllCookies{};
   }
 
+  BRAVE_COOKIE_SETTINGS_BASE_DECIDE_ACCESS
   if (!is_third_party_request) {
     return AllowAllCookies{ThirdPartyCookieAllowMechanism::kNone};
   }
@@ -837,6 +838,7 @@ CookieSettingsBase::GetCookieSettingInternal(
   if (std::holds_alternative<AllowPartitionedCookies>(choice)) {
     CHECK(is_third_party_request);
     CHECK(block_third_party_cookies);
+    BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL_IS_EXPLICIT_SETTING
     CHECK(!is_explicit_setting);
 
     FireStorageAccessHistogram(StorageAccessResult::ACCESS_BLOCKED);
