diff --git a/components/content_settings/core/browser/content_settings_utils.cc b/components/content_settings/core/browser/content_settings_utils.cc
index b33cb6e52a6a4075f2d6e5b348dba0da64aaaa1f..b56ad1e6f9a5a73bb678be2fc6dd8bc0647d238a 100644
--- a/components/content_settings/core/browser/content_settings_utils.cc
+++ b/components/content_settings/core/browser/content_settings_utils.cc
@@ -148,6 +148,7 @@ void GetRendererContentSettingRules(const HostContentSettingsMap* map,
                              &(rules->client_hints_rules));
   map->GetSettingsForOneType(ContentSettingsType::POPUPS, ResourceIdentifier(),
                              &(rules->popup_redirect_rules));
+  BRAVE_GET_RENDER_CONTENT_SETTING_RULES
 }
 
 bool IsMorePermissive(ContentSetting a, ContentSetting b) {
