diff --git a/components/safe_browsing/core/common/safe_browsing_prefs.cc b/components/safe_browsing/core/common/safe_browsing_prefs.cc
index 637e113bbf8eb31f6e7f396f419e177747e494f2..d28b791f97dacaa6273d1fa7688764e03a77cf68 100644
--- a/components/safe_browsing/core/common/safe_browsing_prefs.cc
+++ b/components/safe_browsing/core/common/safe_browsing_prefs.cc
@@ -118,6 +118,7 @@ bool IsSafeBrowsingEnabled(const PrefService& prefs) {
 }
 
 bool IsEnhancedProtectionEnabled(const PrefService& prefs) {
+  BRAVE_IS_ENHANCED_PROTECTION_ENABLED
   // SafeBrowsingEnabled is checked too due to devices being out
   // of sync or not on a version that includes SafeBrowsingEnhanced pref.
   return prefs.GetBoolean(prefs::kSafeBrowsingEnhanced) &&
