diff --git a/components/update_client/BUILD.gn b/components/update_client/BUILD.gn
index cb2ff2bb734c214245f6ae75828df6266f11807d..a083baf2535bd6266f58b72483c1e0c87a65ebf8 100644
--- a/components/update_client/BUILD.gn
+++ b/components/update_client/BUILD.gn
@@ -181,6 +181,7 @@ static_library("update_client") {
       "background_downloader_win.h",
     ]
   }
+  deps += brave_components_update_client_deps public_deps = brave_components_update_client_public_deps
 }
 
 source_set("test_support") {
