diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index f47e805fddfc745e60d8596ce2d8402c8994d291..7ea6f1f6d708a15e4464e0de99259daab50eac2d 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -274,6 +274,7 @@ static_library("browser") {
     "//ui/base",
     "//ui/gfx",
   ]
+  sources += brave_components_omnibox_browser_sources deps += brave_components_omnibox_browser_deps
 
   if ((!is_android || enable_vr) && !is_ios) {
     deps += [ ":vector_icons" ]
