diff --git a/components/sync/base/data_type.h b/components/sync/base/data_type.h
index 95eb11fd40ced293b89b1476366b9477d163c8f4..ab11583c0aa99b2c4c828b4e5e9d56b915d89c4d 100644
--- a/components/sync/base/data_type.h
+++ b/components/sync/base/data_type.h
@@ -207,7 +207,8 @@ enum DataType {
   // User entity suppression records for Autofill AI.
   AUTOFILL_ENTITY_SUPPRESSION,
 
-  LAST_USER_DATA_TYPE = AUTOFILL_ENTITY_SUPPRESSION,
+  AI_CHAT_CONVERSATION,
+  LAST_USER_DATA_TYPE = AI_CHAT_CONVERSATION,
 
   // ---- Control Types ----
   // An object representing a set of Nigori keys.
@@ -322,7 +323,8 @@ enum class DataTypeForHistograms {
   kNotebook = 83,
   kJourney = 84,
   kAutofillEntitySuppression = 85,
-  kMaxValue = kAutofillEntitySuppression,
+  kAIChatConversation = 85 + 1,
+  kMaxValue = kAIChatConversation,
 };
 // LINT.ThenChange(/tools/metrics/histograms/metadata/sync/enums.xml:SyncDataTypes)
 
