diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index d074d52b184a5db03775a01ba313f76ffedb5f6c..ae6cf5e4618b3d33ef078487c00da0c8e4791a99 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -107,6 +107,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
