diff --git a/components/sync/protocol/proto_visitors.h b/components/sync/protocol/proto_visitors.h
index c03cda9054ec826530fdbc6fea38452772a791dd..3cd726a52ed8070cec9ffc5c5ac99c99f72639b6 100644
--- a/components/sync/protocol/proto_visitors.h
+++ b/components/sync/protocol/proto_visitors.h
@@ -377,6 +377,7 @@ VISIT_PROTO_FIELDS(const sync_pb::DeviceInfoSpecifics& proto) {
   VISIT(manufacturer);
   VISIT(last_updated_timestamp);
   VISIT(feature_fields);
+  BRAVE_VISIT_DEVICE_INFO_SPECIFICS_BRAVE_FIELDS
   VISIT(sharing_fields);
   VISIT(invalidation_fields);
 }
@@ -384,7 +385,7 @@ VISIT_PROTO_FIELDS(const sync_pb::DeviceInfoSpecifics& proto) {
 VISIT_PROTO_FIELDS(const sync_pb::FeatureSpecificFields& proto) {
   VISIT(send_tab_to_self_receiving_enabled);
 }
-
+BRAVE_VISIT_PROTO_FIELDS_BRAVE_SPECIFIC_FIELD
 VISIT_PROTO_FIELDS(const sync_pb::SharingSpecificFields& proto) {
   VISIT(vapid_fcm_token);
   VISIT_BYTES(vapid_p256dh);
