diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 5d99c0bb0c93585d0e4801aaafd31fc46a05f3b3..bb04f28b2aca31b86e61b4c125383272dae192b8 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -714,6 +714,7 @@ std::string GetPermissionStringForUma(
     case ContentSettingsType::LOOPBACK_NETWORK:
       return "LoopbackNetwork";
     default:
+      BRAVE_GET_PERMISSION_STRING_FOR_UMA
       break;
   }
   // The user is not prompted for these permissions thus there is no
