diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 358bf2a9d1558424bb22cf2bed2e0dd84ab037dd..8da697b3f049752cd449fd41545ad5f8be6d5264 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1239,6 +1239,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
