diff --git a/components/sync/driver/BUILD.gn b/components/sync/driver/BUILD.gn
index 60b8b3018ee9f2ef059be10eb4ae8bfc2cb56e94..da09a1bce445fd40494de848dee7a25d778a6427 100644
--- a/components/sync/driver/BUILD.gn
+++ b/components/sync/driver/BUILD.gn
@@ -110,6 +110,7 @@ static_library("driver") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps defines = brave_service_key_defines
 }
 
 static_library("test_support") {
