diff --git a/components/permissions/permission_context_base.h b/components/permissions/permission_context_base.h
index a6e895712f4a952581ea1c2ce7077a1ab703945f..789999766a5ee04926c1e205d57ac62186824f66 100644
--- a/components/permissions/permission_context_base.h
+++ b/components/permissions/permission_context_base.h
@@ -88,6 +88,7 @@ using BrowserPermissionCallback =
 // See midi_permission_context.h/cc or push_permission_context.cc/h for some
 // examples.
 
+namespace chromium_impl {
 class PermissionContextBase : public content_settings::Observer {
  public:
   PermissionContextBase(
@@ -301,6 +302,7 @@ class PermissionContextBase : public content_settings::Observer {
 #endif  // BUILDFLAG(IS_ANDROID)
 
  private:
+  friend class ::permissions::PermissionContextBase;
   friend class PermissionContextBaseTests;
 
   // Called when a request is no longer used so it can be cleaned up.
@@ -315,7 +317,7 @@ class PermissionContextBase : public content_settings::Observer {
 
   // This is the callback for PermissionRequest and is called once the user
   // allows/blocks/dismisses a permission prompt.
-  void PermissionDecided(const permissions::PermissionPromptDecision& decision,
+  virtual void PermissionDecided(const permissions::PermissionPromptDecision& decision,
                          const PermissionRequestData& request_data);
 
   raw_ptr<content::BrowserContext> browser_context_;
@@ -336,6 +338,7 @@ class PermissionContextBase : public content_settings::Observer {
   base::WeakPtrFactory<PermissionContextBase> weak_factory_{this};
 };
 
+}  // namespace chromium_impl
 }  // namespace permissions
 
 #endif  // COMPONENTS_PERMISSIONS_PERMISSION_CONTEXT_BASE_H_
