diff --git a/components/sync/engine_impl/sync_scheduler_impl.h b/components/sync/engine_impl/sync_scheduler_impl.h
index 4bab39098da187109f291f1c0ab03fd0cc34d326..c29568b405112bab3b74be98a8855d4b5817943c 100644
--- a/components/sync/engine_impl/sync_scheduler_impl.h
+++ b/components/sync/engine_impl/sync_scheduler_impl.h
@@ -281,6 +281,7 @@ class SyncSchedulerImpl : public SyncScheduler {
   // Used to prevent changing nudge delays by the server in integration tests.
   bool force_short_nudge_delay_for_test_ = false;
 
+  BRAVE_SYNC_SCHEDULER_IMPL_H
   SEQUENCE_CHECKER(sequence_checker_);
 
   base::WeakPtrFactory<SyncSchedulerImpl> weak_ptr_factory_;
