diff --git a/components/sync/base/sync_prefs.cc b/components/sync/base/sync_prefs.cc
index fb2c460af57f05e2fa7550d857c301bd61849ee6..20e122b61c553a8f4e36c79bff3d11b62f24d334 100644
--- a/components/sync/base/sync_prefs.cc
+++ b/components/sync/base/sync_prefs.cc
@@ -303,6 +303,12 @@ const char* SyncPrefs::GetPrefNameForType(UserSelectableType type) {
       return prefs::kSyncTabs;
     case UserSelectableType::kWifiConfigurations:
       return prefs::kSyncWifiConfigurations;
+    case UserSelectableType::kPairs:
+      return prefs::kSyncPairs;
+    case UserSelectableType::kVgBodies:
+      return prefs::kSyncVgBodies;
+    case UserSelectableType::kVgSpendStatuses:
+      return prefs::kSyncVgSpendStatuses;
   }
   NOTREACHED();
   return nullptr;
