diff --git a/components/sync/driver/BUILD.gn b/components/sync/driver/BUILD.gn
index a70a0cd53d6ca4fcc5ae7782061339ef29e067a8..d77b88576d80145f586cff4980de11e89733630e 100644
--- a/components/sync/driver/BUILD.gn
+++ b/components/sync/driver/BUILD.gn
@@ -107,6 +107,7 @@ static_library("driver") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps defines = brave_service_key_defines
 }
 
 static_library("test_support") {
