diff --git a/components/web_modal/web_contents_modal_dialog_manager.cc b/components/web_modal/web_contents_modal_dialog_manager.cc
index 0b94aac417f52faa27512f7543036c0e3bbaad1e..7992096dda4139862107dbe12e30be57e10e64f8 100644
--- a/components/web_modal/web_contents_modal_dialog_manager.cc
+++ b/components/web_modal/web_contents_modal_dialog_manager.cc
@@ -193,6 +193,7 @@ void WebContentsModalDialogManager::OnVisibilityChanged(
   const content::Visibility previous_web_contents_visibility =
       web_contents_visibility_;
   web_contents_visibility_ = visibility;
+  BRAVE_WEB_CONTENTS_MODAL_DIALOG_MANAGER_ON_VISIBILITY_CHANGED
   if (child_dialogs_.empty()) {
     return;
   }
