diff --git a/components/sync/driver/BUILD.gn b/components/sync/driver/BUILD.gn
index 5397307dac013ac06f539d209c8f0b46721832c9..dc545ddd954822b6f705db328c4c51d173495012 100644
--- a/components/sync/driver/BUILD.gn
+++ b/components/sync/driver/BUILD.gn
@@ -112,6 +112,7 @@ static_library("driver") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
 
 static_library("test_support") {
