diff --git a/components/sync/driver/profile_sync_service.h b/components/sync/driver/profile_sync_service.h
index c2761cef61443ebc0c991fe203717c2ac07afc8c..b36adc7a3016f8fc73240de8807801abd51fa9c1 100644
--- a/components/sync/driver/profile_sync_service.h
+++ b/components/sync/driver/profile_sync_service.h
@@ -515,6 +515,7 @@ class ProfileSyncService : public SyncService,
 
   base::WeakPtrFactory<ProfileSyncService> weak_factory_{this};
 
+  BRAVE_PROFILE_SYNC_SERVICE_H
   DISALLOW_COPY_AND_ASSIGN(ProfileSyncService);
 };
 
