diff --git a/components/content_settings/core/browser/BUILD.gn b/components/content_settings/core/browser/BUILD.gn
index 20296786c31e99c62cf28777cef2c06b23cf708c..935b09cb74d9bb68648f770e0916c4c9622b4cff 100644
--- a/components/content_settings/core/browser/BUILD.gn
+++ b/components/content_settings/core/browser/BUILD.gn
@@ -88,6 +88,7 @@ static_library("browser") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_content_settings_core_browser_sources deps += brave_components_content_settings_core_browser_deps
 }
 
 source_set("unit_tests") {
