diff --git a/components/sync/engine_impl/cycle/sync_cycle.h b/components/sync/engine_impl/cycle/sync_cycle.h
index 0971b6014231f8046eca9202840acc6167960057..6f7adf18b33be49f18469026838cbd9ad64bb8c5 100644
--- a/components/sync/engine_impl/cycle/sync_cycle.h
+++ b/components/sync/engine_impl/cycle/sync_cycle.h
@@ -79,6 +79,7 @@ class SyncCycle {
 
     // Called when server requests a migration.
     virtual void OnReceivedMigrationRequest(ModelTypeSet types) = 0;
+    BRAVE_SYNC_CYCLE_DELEGATE_H
 
    protected:
     virtual ~Delegate() {}
