diff --git a/components/sync_device_info/device_info_sync_bridge_unittest.cc b/components/sync_device_info/device_info_sync_bridge_unittest.cc
index c70d096f11cee13cfa0921c9d42bf76da548dbb6..fb1f93bda2264242761f9fe6ff335e1a9a31ed61 100644
--- a/components/sync_device_info/device_info_sync_bridge_unittest.cc
+++ b/components/sync_device_info/device_info_sync_bridge_unittest.cc
@@ -827,7 +827,7 @@ TEST_F(DeviceInfoSyncBridgeTest, ApplySyncChangesWithLocalGuid) {
   auto error_on_delete = bridge()->ApplySyncChanges(
       bridge()->CreateMetadataChangeList(), std::move(entity_change_list));
   EXPECT_FALSE(error_on_delete);
-  EXPECT_EQ(1, change_count());
+  EXPECT_EQ(2, change_count());  // Brave can delete other device in chain, delete record should be also processed
 }
 
 TEST_F(DeviceInfoSyncBridgeTest, ApplyDeleteNonexistent) {
