diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 709c14c0c851fcebf026a798806af9b8d02afa07..bca6acc9bd1bc04b5eeea76076539ca72d15e70d 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1409,6 +1409,9 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // second section, the default behavior used when no per-type exception applies.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  if (!delegate_->BraveShouldShowPermission(info.type)) {
+    return false;
+  }
 #if BUILDFLAG(IS_ANDROID)
   if (info.type == ContentSettingsType::NOTIFICATIONS) {
     // `is_requested` is only populated if
@@ -1639,6 +1642,9 @@ void PageInfo::PresentSitePermissions() {
 
 std::set<net::SchemefulSite> PageInfo::GetTwoSitePermissionRequesters(
     ContentSettingsType type) {
+  if (type == ContentSettingsType::STORAGE_ACCESS) {
+    return {};
+  }
   std::set<net::SchemefulSite> requesters;
   // Collect sites that have tried to request a permission.
   auto* pscs = GetPageSpecificContentSettings();
