diff --git a/components/search_engines/BUILD.gn b/components/search_engines/BUILD.gn
index c9b4e5ef21dde6b83bfebfdf7534fa852e8ac7c5..0ff1b36e906ebe3d639a260fe73304d0993a077b 100644
--- a/components/search_engines/BUILD.gn
+++ b/components/search_engines/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/components/search_engines/sources.gni")
 import("//build/config/chrome_build.gni")
 import("//tools/json_to_struct/json_to_struct.gni")
 if (is_android) {
@@ -129,6 +130,7 @@ static_library("search_engines") {
     "//ui/gfx/geometry",
     "//url",
   ]
+  deps += brave_components_search_engines_deps
 
   if (!is_fuchsia) {
     deps += [ "//components/variations/service" ]
