diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 7d98e5dc5fa57cbb9da7dfada58124c2fc4bf439..138f50f96b8a70523aa0e8670d504c4d7365301e 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -113,6 +113,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kTopLevelStorageAccess:
       return RequestTypeForUma::PERMISSION_TOP_LEVEL_STORAGE_ACCESS;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
