diff --git a/components/sync/protocol/protocol_sources.gni b/components/sync/protocol/protocol_sources.gni
index 3924613440fcb34ade97bf0b7bcdde99ae79c24c..b5044cdf03b9b9116ddc06cbac51a0650ea3918e 100644
--- a/components/sync/protocol/protocol_sources.gni
+++ b/components/sync/protocol/protocol_sources.gni
@@ -80,3 +80,4 @@ sync_protocol_sources = [
   "wifi_configuration_specifics.proto",
   "workspace_desk_specifics.proto",
 ]
+import("//brave/components/sync/protocol/protocol_sources.gni") sync_protocol_sources += brave_sync_protocol_sources
