diff --git a/components/autofill/core/browser/ui/autofill_external_delegate.cc b/components/autofill/core/browser/ui/autofill_external_delegate.cc
index 4c3a24622553d688378ab822f2570bd3c1a47d92..b6d7293d550703d1f719bfc2ef68d50dd1a76e5a 100644
--- a/components/autofill/core/browser/ui/autofill_external_delegate.cc
+++ b/components/autofill/core/browser/ui/autofill_external_delegate.cc
@@ -455,6 +455,13 @@ void AutofillExternalDelegate::AttemptToDisplayAutofillSuggestions(
   // updated to match.
   InsertDataListValues(last_query_.field_datalist_options, suggestions);
 
+  if (trigger_field.has_value()) {
+    manager_->client().BraveAddSuggestions(
+        manager_->client().ClassifyAsPasswordForm(
+            *manager_, last_query_.form_id, last_query_.field_id),
+        *trigger_field, suggestions);
+  }
+
   // TODO(crbug.com/362630793): Try to eliminate this state. The controller
   // should be the one that knows about what suggestions were shown and passes
   // it on, not AED.
@@ -825,6 +832,10 @@ void AutofillExternalDelegate::DidSelectSuggestion(
 void AutofillExternalDelegate::DidAcceptSuggestion(
     const Suggestion& suggestion,
     const SuggestionMetadata& metadata) {
+  if (manager_->client().BraveHandleSuggestion(suggestion,
+                                                last_query_.field_id)) {
+    return;
+  }
   CHECK(suggestion.IsAcceptable());
   base::UmaHistogramEnumeration("Autofill.Suggestions.AcceptedType",
                                 suggestion.type);
