diff --git a/components/sync/engine/data_type_worker.h b/components/sync/engine/data_type_worker.h
index 52d754cf99f8b1e672f669345805064e3a40cceb..cc94ba717c94e64dfd3f1500eebe287b44833c60 100644
--- a/components/sync/engine/data_type_worker.h
+++ b/components/sync/engine/data_type_worker.h
@@ -216,6 +216,7 @@ class DataTypeWorker : public UpdateHandler,
 
   static constexpr size_t kMaxPendingInvalidations = 10u;
 
+ BRAVE_MODEL_TYPE_WORKER_H_
  private:
   struct UnknownEncryptionKeyInfo {
     // Not increased if the cryptographer knows it's in a pending state
