diff --git a/components/permissions/request_type.h b/components/permissions/request_type.h
index 8cd80a3c07d2ff5683f42826dcb6433803ad2ae0..24dc7dcfde8a01a5e5cb0d5a7a183cbc21c74586 100644
--- a/components/permissions/request_type.h
+++ b/components/permissions/request_type.h
@@ -56,6 +56,7 @@ enum class RequestType {
 #else
   kMaxValue = kVrSession
 #endif
+  BRAVE_REQUEST_TYPES
 };
 
 #if defined(OS_ANDROID)
