diff --git a/components/page_info/BUILD.gn b/components/page_info/BUILD.gn
index 5d69b2e6cbac8b2fd7aca9314bb72fd4dec42074..0bdc230bed47fd9e1d05f043bf85f8a202b4507f 100644
--- a/components/page_info/BUILD.gn
+++ b/components/page_info/BUILD.gn
@@ -70,6 +70,7 @@ static_library("page_info") {
     "//services/device/public/cpp:device_features",
     "//services/metrics/public/cpp:ukm_builders",
   ]
+  deps += [ "//brave/components/page_info" ]
 
   if (is_android) {
     deps += [
