diff --git a/components/crash/content/app/crashpad_mac.mm b/components/crash/content/app/crashpad_mac.mm
index a76d264648d23e7e228dd54f00a01035a29a76ad..ee7d94be5982353fced6174afef990713228af37 100644
--- a/components/crash/content/app/crashpad_mac.mm
+++ b/components/crash/content/app/crashpad_mac.mm
@@ -136,6 +136,8 @@ base::FilePath PlatformCrashpadInitialization(
       // Only allow the possibility of report upload in official builds. This
       // crash server won't have symbols for any other build types.
       std::string url = "https://clients2.google.com/cr/report";
+#elif defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
+      std::string url = "https://laptop-updates.brave.com/1/bc-crashes";
 #else
       std::string url;
 #endif
