diff --git a/components/update_client/BUILD.gn b/components/update_client/BUILD.gn
index de3060e8479f81aaf7733562780ccfbf7b86cb35..ae2c1c97397c2403570ab4663ea19e80f048ff2f 100644
--- a/components/update_client/BUILD.gn
+++ b/components/update_client/BUILD.gn
@@ -180,6 +180,7 @@ static_library("update_client") {
       "background_downloader_win.h",
     ]
   }
+  deps += brave_components_update_client_deps public_deps = brave_components_update_client_public_deps
 }
 
 source_set("test_support") {
