diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 6cfe37ef2ec121c2196e2144c733bf21b60ee6b8..6cd3ef293ecc2c6b4a66d28fe4109e53cecb8fa3 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -109,6 +109,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
