diff --git a/components/crx_file/BUILD.gn b/components/crx_file/BUILD.gn
index 31edb2e296189a6d8619a4294032cf424599504c..3c009cfba4b7d46ed6fad87d5f5d3ab7b32aee0a 100644
--- a/components/crx_file/BUILD.gn
+++ b/components/crx_file/BUILD.gn
@@ -76,7 +76,6 @@ proto_library("crx3_proto") {
 
 if (host_toolchain == current_toolchain) {
   executable("crx3_build_action") {
-    testonly = true
 
     sources = [ "crx_build_action_main.cc" ]
 
