diff --git a/components/page_info/BUILD.gn b/components/page_info/BUILD.gn
index b9c9b5f8d8ec256b1064a93414ec819b01cd2e25..da6b20696ad46e2aa693b3c9fc1eef7500048e9b 100644
--- a/components/page_info/BUILD.gn
+++ b/components/page_info/BUILD.gn
@@ -47,6 +47,7 @@ static_library("page_info") {
     "//services/device/public/cpp:device_features",
     "//services/metrics/public/cpp:ukm_builders",
   ]
+  deps += [ "//brave/components/page_info" ]
 
   if (is_android) {
     deps += [
