diff --git a/components/sync/base/BUILD.gn b/components/sync/base/BUILD.gn
index a7e85ebc47c9e40263e5572f7491fd6d14906512..397809fc8bfaaf46774ec8030278af0524e92a36 100644
--- a/components/sync/base/BUILD.gn
+++ b/components/sync/base/BUILD.gn
@@ -88,5 +88,7 @@ static_library("base") {
     "//ui/base",
   ]
 
+  import("//brave/components/sync/base/sources.gni") public_deps += brave_components_sync_base_public_deps
+
   defines = [ "SYNC_USER_AGENT_PRODUCT=$sync_user_agent_product" ]
 }
