diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 28679d82699013f90fdea19bbba6b59ded686738..e3296259864c065d96d379cff7e05bf59bffef3e 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -107,6 +107,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
