diff --git a/components/page_info/BUILD.gn b/components/page_info/BUILD.gn
index 090e99cf272e59c744598c5ce121bc9cd7aec590..cbdc37eb8f8cbdc5e9b2b46f0396937e04e75dbb 100644
--- a/components/page_info/BUILD.gn
+++ b/components/page_info/BUILD.gn
@@ -43,6 +43,7 @@ static_library("page_info") {
     "//services/device/public/cpp:device_features",
     "//services/metrics/public/cpp:ukm_builders",
   ]
+  deps += [ "//brave/components/page_info" ]
 
   if (is_android) {
     deps += [
