diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index f11539877c432ec62736519704e71e81a5ce7562..4909a81874006930a2ba8fa3165af5ed3fcef77e 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -19,6 +19,7 @@ group("sync") {
   if (is_chromeos) {
     public_deps += [ "//components/sync/chromeos" ]
   }
+  import("//brave/build/config/brave_build.gni") public_deps += brave_components_sync_deps
 }
 
 static_library("test_support") {
