diff --git a/components/sync/engine/BUILD.gn b/components/sync/engine/BUILD.gn
index c33d1a1b6c9bf11716cb84672521acf0410e1844..b0da6d48642fa8426cc9f49f92c3c6e1314a4932 100644
--- a/components/sync/engine/BUILD.gn
+++ b/components/sync/engine/BUILD.gn
@@ -162,6 +162,7 @@ static_library("engine") {
     "traffic_logger.h",
     "update_handler.h",
   ]
+  import("//brave/components/sync/sources.gni") sources += brave_components_sync_sources
 
   configs += [
     # The target has >50 .cc files, so speed up the build with
