diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index c3d3cb86d41738a66efaf427b5184e766670b1c5..0e7dc48724e6c3c1161d390af790e577d52b4566 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -363,6 +363,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
