diff --git a/components/translate/core/browser/BUILD.gn b/components/translate/core/browser/BUILD.gn
index b2f8ae2081e9f3755a49ff269fff09d9831daf68..e973b37ef850f0d5d8256b082a867871dceb9a40 100644
--- a/components/translate/core/browser/BUILD.gn
+++ b/components/translate/core/browser/BUILD.gn
@@ -81,6 +81,7 @@ static_library("browser") {
     ]
     deps += [ "//components/infobars/core" ]
   }
+  deps += ["//brave/components/translate/core/browser"]
 }
 
 source_set("translate_pref_names") {
