diff --git a/components/permissions/permission_request_manager.cc b/components/permissions/permission_request_manager.cc
index 30d020a934feab640cf74d00cdb9049fe1eda4b0..4b0eae7643d7eaea4b7cb4869be79caab2d5a607 100644
--- a/components/permissions/permission_request_manager.cc
+++ b/components/permissions/permission_request_manager.cc
@@ -448,6 +448,7 @@ const std::vector<PermissionRequest*>& PermissionRequestManager::Requests() {
 }
 
 GURL PermissionRequestManager::GetRequestingOrigin() const {
+  if (requests_.size() > 0 && requests_[0]->request_type() == permissions::RequestType::kBraveEthereum) return requests_.front()->requesting_origin();
   CHECK(!requests_.empty());
   GURL origin = requests_.front()->requesting_origin();
   if (DCHECK_IS_ON()) {
