diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index 27d9b80c532c83c8985a04f0f107462bcb2b7cd4..6db6551a1a1c2d4b6613d0bc10af4493fb858091 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -1267,6 +1267,7 @@ void AutocompleteController::InitializeAsyncProviders(int provider_types) {
         base::MakeRefCounted<SearchProvider>(provider_client_.get(), this);
     search_provider_ = search_provider.get();
     providers_.push_back(std::move(search_provider));
+    BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
   }
   // Providers run in the order they're added.  Add `HistoryURLProvider` after
   // `SearchProvider` because:
@@ -1543,6 +1544,7 @@ void AutocompleteController::UpdateResult(UpdateType update_type,
     StartStopTimer();
   }
 
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
   PostProcessMatches();
 
   const bool is_lens_enabled = autocomplete_provider_client()->IsLensEnabled();
