diff --git a/components/sync/android/BUILD.gn b/components/sync/android/BUILD.gn
index 8482acd6aeef393fe8ebd0990091e6414d3850e4..6c63b3a5c8133fb907f8ed9d32421e11e5b675c5 100644
--- a/components/sync/android/BUILD.gn
+++ b/components/sync/android/BUILD.gn
@@ -32,6 +32,7 @@ android_library("sync_java") {
     "java/src/org/chromium/components/sync/notifier/InvalidationPreferences.java",
     "java/src/org/chromium/components/sync/notifier/RandomizedInvalidationClientNameGenerator.java",
   ]
+  java_files += brave_sync_java_sources
 }
 
 android_library("sync_javatests") {
