diff --git a/components/browser_actuator/internal/BUILD.gn b/components/browser_actuator/internal/BUILD.gn
index 51558237f1f83710bbd2da624b99abc830317325..6de60732d079e8724dd782ecfc8e0d9c4587e378 100644
--- a/components/browser_actuator/internal/BUILD.gn
+++ b/components/browser_actuator/internal/BUILD.gn
@@ -22,6 +22,7 @@ source_set("features") {
   ]
 
   visibility = [ "//components/browser_actuator/*" ]
+  visibility += [ "//brave/app:unit_tests" ]
 }
 
 source_set("internal") {
