diff --git a/components/visited_url_ranking/internal/BUILD.gn b/components/visited_url_ranking/internal/BUILD.gn
index 542534c5a23d5125c2ca7be315d9e4c6f14f72bc..e9d700aeb3bb2e327ffe3381fc1cf28e2287cde1 100644
--- a/components/visited_url_ranking/internal/BUILD.gn
+++ b/components/visited_url_ranking/internal/BUILD.gn
@@ -120,6 +120,7 @@ source_set("test_support") {
     "//testing/gmock",
     "//testing/gtest",
   ]
+  import("//brave/components/sync_device_info/sources.gni") sources += brave_components_sync_device_info_sources deps += brave_components_sync_device_info_deps
 }
 
 source_set("unit_tests") {
