diff --git a/components/autofill/core/browser/payments/credit_card_access_manager.cc b/components/autofill/core/browser/payments/credit_card_access_manager.cc
index 59fe4cb977bbf6e6dc4283c72c4952007c4c2c3d..fb64e6d4ee3e98aac4018e655fb1aeaa11b96734 100644
--- a/components/autofill/core/browser/payments/credit_card_access_manager.cc
+++ b/components/autofill/core/browser/payments/credit_card_access_manager.cc
@@ -370,10 +370,10 @@ void CreditCardAccessManager::FetchCreditCard(
   // Log the server card unmasking attempt, and differentiate based on server
   // card or virtual card.
   if (ShouldLogServerCardUnmaskAttemptMetrics(record_type)) {
-    autofill_metrics::LogServerCardUnmaskAttempt(
-        record_type == CreditCard::RecordType::kVirtualCard
-            ? PaymentsRpcCardType::kVirtualCard
-            : PaymentsRpcCardType::kServerCard);
+    if (record_type == CreditCard::RecordType::kVirtualCard) {
+      autofill_metrics::LogServerCardUnmaskAttempt(
+          PaymentsRpcCardType::kVirtualCard);
+    }
   }
 
   // If the to-be-unmasked card is a virtual card eligible card (this also
