diff --git a/components/browser_ui/settings/android/BUILD.gn b/components/browser_ui/settings/android/BUILD.gn
index 09dc3c0c2788d88a0ac6c8575b274cb46e2d8f80..a1a2448172f84d343b9c8de22c37316dc1d3b0b4 100644
--- a/components/browser_ui/settings/android/BUILD.gn
+++ b/components/browser_ui/settings/android/BUILD.gn
@@ -49,6 +49,7 @@ android_library("java") {
     "//ui/android:ui_no_recycler_view_java",
   ]
   resources_package = "org.chromium.components.browser_ui.settings"
+  import("//brave/components/browser_ui/settings/android/sources.gni") sources += brave_components_browser_ui_settings_android_java_sources
 }
 
 android_resources("java_resources") {
