diff --git a/components/crash/core/browser/resources/crashes.js b/components/crash/core/browser/resources/crashes.js
index 0d21ef1b4fa164495e7bba3f4e6f9aeac2e14253..0ed75777da7ae404c8630ef9a483ea684530a457 100644
--- a/components/crash/core/browser/resources/crashes.js
+++ b/components/crash/core/browser/resources/crashes.js
@@ -108,7 +108,7 @@ function updateCrashList({
         crashLink.href = `https://goto.google.com/crash/${crash.id}`;
         crashLink.target = '_blank';
         crashLink.textContent = crash.id;
-        uploadIdValue.appendChild(crashLink);
+        //uploadIdValue.appendChild(crashLink); // feature disabled in Brave
       } else {
         uploadIdValue.textContent = crash.id;
       }
