diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 38306eefa1a0d5993868e6b71584631cae262e0c..9b27f73d50d897292a152374cf5e123e397ec4c4 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -969,6 +969,7 @@ void PageInfo::PresentSitePermissions() {
       }
     }
 
+    BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
     if (ShouldShowPermission(
             permission_info, site_url_, content_settings, web_contents(),
             HasContentSettingChangedViaPageInfo(permission_info.type))) {
