diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index 2317ae10e006cce53bc2ac000487bcd81c2556c1..5bcbf81d45f5aec2af797d38294712f5e407466b 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -460,6 +460,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
 #endif
     case permissions::RequestType::kIdentityProvider:
       return "identity_provider";
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
