diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 410887f40e413aa20d8564341ed567d82b48ce66..26533ec69bd295b1027727b7f8558f62c9b3a6d6 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -288,6 +288,7 @@ class SyncServiceImpl : public SyncService,
   size_t GetQueuedLocalDataMigrationItemCountForTest() const;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
