diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index dbd1ce114a06b6d83ba40b520e04afa80dbe3568..a0be0cc2b35038928fb7ef49dd249e33f7aab016 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -357,6 +357,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
