diff --git a/components/sync_bookmarks/bookmark_change_processor.cc b/components/sync_bookmarks/bookmark_change_processor.cc
index acb064faac3ee975ec07be4977fc0970309dfb15..76eec7bcf64a9bb28b4709f9ae01352933e1197e 100644
--- a/components/sync_bookmarks/bookmark_change_processor.cc
+++ b/components/sync_bookmarks/bookmark_change_processor.cc
@@ -85,6 +85,7 @@ void BookmarkChangeProcessor::UpdateSyncNodeProperties(
     bookmark_specifics.set_url(src->url().spec());
   }
   bookmark_specifics.set_creation_time_us(src->date_added().ToInternalValue());
+  AddBraveMetaInfo(src, model);
   dst->SetBookmarkSpecifics(bookmark_specifics);
   SetSyncNodeFavicon(src, model, dst);
   SetSyncNodeMetaInfo(src, dst);
@@ -244,6 +245,7 @@ void BookmarkChangeProcessor::CreateOrUpdateSyncNode(const BookmarkNode* node) {
     // Acquire a scoped write lock via a transaction.
     syncer::WriteTransaction trans(FROM_HERE, share_handle(), &new_version);
     sync_id = model_associator_->GetSyncIdFromChromeId(node->id());
+    bookmark_model_->RemoveObserver(this);
     if (sync_id != syncer::kInvalidId) {
       UpdateSyncNode(
           node, bookmark_model_, &trans, model_associator_, error_handler());
@@ -257,6 +259,7 @@ void BookmarkChangeProcessor::CreateOrUpdateSyncNode(const BookmarkNode* node) {
                                model_associator_,
                                error_handler());
     }
+    bookmark_model_->AddObserver(this);
   }
 
   if (syncer::kInvalidId != sync_id) {
@@ -383,7 +386,8 @@ int64_t BookmarkChangeProcessor::UpdateSyncNode(
 
 void BookmarkChangeProcessor::BookmarkMetaInfoChanged(
     BookmarkModel* model, const BookmarkNode* node) {
-  BookmarkNodeChanged(model, node);
+  // TODO(darkdh): make sure the meta info changed is what we want to sync
+  // BookmarkNodeChanged(model, node);
 }
 
 void BookmarkChangeProcessor::BookmarkNodeMoved(BookmarkModel* model,
@@ -416,6 +420,11 @@ void BookmarkChangeProcessor::BookmarkNodeMoved(BookmarkModel* model,
       return;
     }
 
+    bookmark_model_->RemoveObserver(this);
+    AddBraveMetaInfo(child, model);
+    SetSyncNodeMetaInfo(child, &sync_node);
+    bookmark_model_->AddObserver(this);
+
     if (!PlaceSyncNode(MOVE, new_parent, new_index, &trans, &sync_node,
                        model_associator_)) {
       syncer::SyncError error(FROM_HERE,
@@ -458,6 +467,11 @@ void BookmarkChangeProcessor::BookmarkNodeFaviconChanged(
     return;
   }
 
+  model->RemoveObserver(this);
+  bool first_loaded_favicon = IsFirstLoadedFavicon(node);
+  model->AddObserver(this);
+  if (first_loaded_favicon) return;
+
   CreateOrUpdateSyncNode(node);
 }
 
@@ -718,6 +732,8 @@ void BookmarkChangeProcessor::ApplyChangesFromSyncModel(
                    << src.GetBookmarkSpecifics().url();
         continue;
       }
+      BookmarkNode* mutable_node = const_cast<BookmarkNode*>(dst);
+      mutable_node->SetMetaInfo("FirstLoadedFavicon", "true");
       model_associator_->Associate(dst, src);
     }
 
