diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index ff27b0d74068df5f661bdb6ceb25cff6f61f1ed6..49738a07a4ae0faa11ee73b6004e8869322bdff5 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -55,6 +55,7 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
 };
 
 typedef int64_t VisitID;
