diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 88a658e662620b4c0cc3e4a54d8cc0b44f5df873..0b53c2ae0dbbd3a4cefceb6f95b08a57d0da4e07 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -370,6 +370,7 @@ static_library("browser") {
     "zero_suggest_verbatim_match_provider.cc",
     "zero_suggest_verbatim_match_provider.h",
   ]
+  sources += brave_components_omnibox_browser_sources
 
   public_deps = [
     ":aim_eligibility_service_features",
@@ -398,6 +399,7 @@ static_library("browser") {
     "//third_party/tflite_support:tflite_support_proto",
     "//url",
   ]
+  public_deps += brave_components_omnibox_browser_public_deps
 
   deps = [
     "//build:branding_buildflags",
@@ -444,6 +446,7 @@ static_library("browser") {
     "//ui/base",
     "//ui/gfx",
   ]
+  deps += brave_components_omnibox_browser_deps
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
 
@@ -1080,6 +1083,7 @@ fuzzer_test("search_suggestion_parser_fuzzer") {
 }
 
 mojom("mojo_bindings") {
+  generate_legacy_js_bindings = true
   sources = [ "omnibox.mojom" ]
   if (!is_ios) {
     sources += [ "searchbox.mojom" ]
