diff --git a/components/gcm_driver/gcm_client_impl.cc b/components/gcm_driver/gcm_client_impl.cc
index 7e12cd5fdad159e1a21fa7ef7324b8dba0dc7b9b..f55f862875c7af40f1663760eed33b6cd3fe1fe2 100644
--- a/components/gcm_driver/gcm_client_impl.cc
+++ b/components/gcm_driver/gcm_client_impl.cc
@@ -864,6 +864,7 @@ void GCMClientImpl::ResetCache() {
 
 void GCMClientImpl::Register(
     const linked_ptr<RegistrationInfo>& registration_info) {
+  return; // GCM disabled in Brave
   DCHECK_EQ(state_, READY);
 
   // Registrations should never pass as an app_id the special category used
