diff --git a/components/policy/resources/webui/policy_app.ts b/components/policy/resources/webui/policy_app.ts
index 19b69762153d8ff57545c1285f11ceef35e3ac96..29aa7a35980bd8776686e7640dc81e923a40b43d 100644
--- a/components/policy/resources/webui/policy_app.ts
+++ b/components/policy/resources/webui/policy_app.ts
@@ -193,7 +193,7 @@ export class PolicyAppElement extends CrLitElement {
                   policyNames['precedence']?.policyNames,
                 ].includes(knownPolicyNames) &&
                         knownPolicyNamesSet.has(name) ?
-                    `https://chromeenterprise.google/policies/?policy=${name}` :
+                    `https://support.brave.app/hc/en-us/articles/360039248271-Group-Policy?policy=${name}` :
                     undefined,
                 isExtension: value.isExtension || false,
               },
