diff --git a/components/omnibox/browser/featured_search_provider.cc b/components/omnibox/browser/featured_search_provider.cc
index b9a486f7d11fc478c3652472da07b3a647ff5461..38603b3cbb10146f3def2121ccf323754384faa4 100644
--- a/components/omnibox/browser/featured_search_provider.cc
+++ b/components/omnibox/browser/featured_search_provider.cc
@@ -294,6 +294,7 @@ void FeaturedSearchProvider::AddFeaturedKeywordMatches(
   for (TemplateURL* turl : turls) {
     if (turl->starter_pack_id() > 0 &&
         turl->is_active() == TemplateURLData::ActiveStatus::kTrue) {
+      BRAVE_FEATURED_SEARCH_PROVIDER_ADD_FEATURED_KEYWORD_MATCHES
       // Skip @gemini if feature disabled.
       if (turl->starter_pack_id() == template_url_starter_pack_data::kGemini &&
           !OmniboxFieldTrial::IsStarterPackExpansionEnabled()) {
