diff --git a/components/download/public/background_service/clients.cc b/components/download/public/background_service/clients.cc
index 969153db7eb99cf2a700770243e0a51532e85243..2e830d1463ae8b8461c905ee0ba1fc2974797510 100644
--- a/components/download/public/background_service/clients.cc
+++ b/components/download/public/background_service/clients.cc
@@ -28,6 +28,7 @@ std::string BackgroundDownloadClientToString(DownloadClient client) {
       return "OptimizationGuidePredictionModels";
     case DownloadClient::BRUSCHETTA:
       return "Bruschetta";
+    BRAVE_BACKGROUND_DOWNLOAD_CLIENT_TO_STRING
     case DownloadClient::BOUNDARY:
       break;
   }
