diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index 20df13a5bd75e01cb127632448d6feaa52360bed..c62ca5702b24929ccad3dfa7bb184e85463b1a83 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -472,6 +472,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
 #endif
     case permissions::RequestType::kIdentityProvider:
       return "identity_provider";
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
