diff --git a/components/download/internal/common/download_item_impl.cc b/components/download/internal/common/download_item_impl.cc
index 48b41ef91ab2c3c70a0b0bccb12dd6a7319f5416..33c0f52f3e00e4f4af4a8dc0548342c3be641570 100644
--- a/components/download/internal/common/download_item_impl.cc
+++ b/components/download/internal/common/download_item_impl.cc
@@ -2038,8 +2038,8 @@ void DownloadItemImpl::OnDownloadCompleting() {
           &DownloadFile::RenameAndAnnotate,
           base::Unretained(download_file_.get()), GetTargetFilePath(),
           delegate_->GetApplicationClientIdForFileScanning(),
-          delegate_->IsOffTheRecord() ? GURL() : GetURL(),
-          delegate_->IsOffTheRecord() ? GURL() : GetReferrerUrl(),
+          GURL(),  // Never leak download URLs in metadata.
+          GURL(),  // See brave-browser#2766.
           delegate_->IsOffTheRecord() ? std::nullopt : GetRequestInitiator(),
           std::move(quarantine), std::move(rename_callback)));
 }
