diff --git a/components/sync/engine/sync_engine.h b/components/sync/engine/sync_engine.h
index e4d2ab4d38f6fffd2929277be51ec618180e0086..721df77d28d5d5a2c43660df09edd0697492f071 100644
--- a/components/sync/engine/sync_engine.h
+++ b/components/sync/engine/sync_engine.h
@@ -87,6 +87,9 @@ class SyncEngine : public ModelTypeConfigurer {
     // Define the polling interval. Must not be zero.
     base::TimeDelta poll_interval;
 
+    brave_sync::NudgeSyncCycleDelegate nudge_sync_cycle_delegate_function;
+    brave_sync::PollSyncCycleDelegate poll_sync_cycle_delegate_function;
+
    private:
     DISALLOW_COPY_AND_ASSIGN(InitParams);
   };
@@ -199,6 +202,10 @@ class SyncEngine : public ModelTypeConfigurer {
   virtual std::unique_ptr<ModelTypeControllerDelegate>
   GetNigoriControllerDelegate() = 0;
 
+  virtual void DispatchGetRecordsCallback(
+      brave_sync::GetRecordsCallback cb,
+      std::unique_ptr<brave_sync::RecordsList> records) {}
+
  private:
   DISALLOW_COPY_AND_ASSIGN(SyncEngine);
 };
