diff --git a/components/sync/engine/model_type_worker.h b/components/sync/engine/model_type_worker.h
index 7c782ebecb658247f8d3ae2cdeb29389a91e853e..c3f2d7508839517358caadac5d4e01a32b9535f0 100644
--- a/components/sync/engine/model_type_worker.h
+++ b/components/sync/engine/model_type_worker.h
@@ -210,6 +210,7 @@ class ModelTypeWorker : public UpdateHandler,
 
   static constexpr size_t kMaxPendingInvalidations = 10u;
 
+ BRAVE_MODEL_TYPE_WORKER_H_
  private:
   struct UnknownEncryptionKeyInfo {
     // Not increased if the cryptographer knows it's in a pending state
