diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 49c25b9a417aee5cce43099b811cbc1f64a1a962..fb590bbd86add3afd8ad709bd00645df94ad1863 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1223,6 +1223,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
