diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 18b58c189f5036eaac8baff0c0b5adb9e17720e4..39e0b187fd24fef108a9c0020bb7d68805dcc4f1 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -262,6 +262,7 @@ class SyncServiceImpl : public SyncService,
   void ReportDataTypeErrorForTest(ModelType type);
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
