diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index 46b564885f1c3be5e45759953e385689d0e465fd..78250b3f0aece162933444ec36aa5e01fc2374b8 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -305,6 +305,7 @@ static_library("rest_of_sync") {
     "nigori/pending_local_nigori_commit.cc",
     "nigori/pending_local_nigori_commit.h",
   ]
+  sources += brave_components_sync_sources
 
   configs += [ "//build/config:precompiled_headers" ]
 
