diff --git a/components/webapps/browser/BUILD.gn b/components/webapps/browser/BUILD.gn
index 124364b7eb6eae7f5b6ba07f7749af0f10986a66..96c2d1b56d94b84b0d95387044e32170eded9bfd 100644
--- a/components/webapps/browser/BUILD.gn
+++ b/components/webapps/browser/BUILD.gn
@@ -191,6 +191,7 @@ source_set("browser") {
       "//ui/gfx",
     ]
   }
+  import("//brave/components/webapps/browser/sources.gni") deps += brave_components_webapps_browser_deps
 }
 
 source_set("test_support") {
