diff --git a/components/policy/tools/generate_policy_source.py b/components/policy/tools/generate_policy_source.py
index 6e225a2f92a2ff7ed0652f79ede64544ea08c7b0..97158cbb6e89f94776dccb1f36acbc39fb46ae7c 100755
--- a/components/policy/tools/generate_policy_source.py
+++ b/components/policy/tools/generate_policy_source.py
@@ -53,6 +53,7 @@ PLATFORM_STRINGS = {
     'chrome.win7': ['win'],
 }
 
+from policy_source_helper import AddBravePolicies, CHROMIUM_POLICY_KEY
 
 class PolicyDetails:
   """Parses a policy template and caches all its details."""
@@ -364,6 +365,7 @@ def main():
     chrome_major_version = ParseVersionFile(version_path)
 
   template_file_contents = _LoadJSONFile(template_file_name)
+  AddBravePolicies(template_file_contents)
   risk_tags = RiskTags(template_file_contents)
   policy_details = [
       PolicyDetails(policy, chrome_major_version, target_platform,
