diff --git a/components/javascript_dialogs/tab_modal_dialog_manager.cc b/components/javascript_dialogs/tab_modal_dialog_manager.cc
index 7c6cf376decfd31b108d1b93081c9c2b910edcdf..0db7cbf2d223800a7e497f07610d81b907f13712 100644
--- a/components/javascript_dialogs/tab_modal_dialog_manager.cc
+++ b/components/javascript_dialogs/tab_modal_dialog_manager.cc
@@ -253,6 +253,7 @@ void TabModalDialogManager::CancelDialogs(content::WebContents* web_contents,
 
 void TabModalDialogManager::OnVisibilityChanged(
     content::Visibility visibility) {
+  BRAVE_TAB_MODAL_DIALOG_MANAGER_ON_VISIBILITY_CHANGED
   if (visibility == content::Visibility::HIDDEN) {
     HandleTabSwitchAway(DismissalCause::kTabHidden);
   } else if (pending_dialog_) {
