diff --git a/components/signin/public/identity_manager/BUILD.gn b/components/signin/public/identity_manager/BUILD.gn
index 9a65599febe11afca5ce30b3b18401720c83174f..b1710a4b72078a8e93b418f6528d577e6799b4a5 100644
--- a/components/signin/public/identity_manager/BUILD.gn
+++ b/components/signin/public/identity_manager/BUILD.gn
@@ -7,6 +7,7 @@ import("//build/config/chromeos/ui_mode.gni")
 if (is_android) {
   import("//build/config/android/rules.gni")
 }
+import("//brave/components/signin/public/identity_manager/sources.gni")
 
 source_set("identity_manager") {
   sources = [
@@ -78,6 +79,7 @@ source_set("identity_manager") {
     # together and include headers from each other.
     "//components/signin/internal/identity_manager",
   ]
+  sources += brave_components_signin_public_identity_manager_sources
 }
 
 if (is_android) {
