diff --git a/components/gcm_driver/gcm_client_impl.cc b/components/gcm_driver/gcm_client_impl.cc
index 168bef640e08..38e749438233 100644
--- a/components/gcm_driver/gcm_client_impl.cc
+++ b/components/gcm_driver/gcm_client_impl.cc
@@ -355,6 +355,7 @@ void GCMClientImpl::Initialize(
 }
 
 void GCMClientImpl::Start(StartMode start_mode) {
+  return; // GCM disabled in Brave
   DCHECK_NE(UNINITIALIZED, state_);
 
   if (state_ == LOADED) {
@@ -865,6 +866,7 @@ void GCMClientImpl::ResetCache() {
 
 void GCMClientImpl::Register(
     const linked_ptr<RegistrationInfo>& registration_info) {
+  return; // GCM disabled in Brave
   DCHECK_EQ(state_, READY);
 
   // Registrations should never pass as an app_id the special category used
