diff --git a/components/password_manager/core/browser/login_database.cc b/components/password_manager/core/browser/login_database.cc
index 9828119390c70a3eb642ddd63e0e92a346998c4a..20e6c819cc15872236fb2d36d043ade303c64a09 100644
--- a/components/password_manager/core/browser/login_database.cc
+++ b/components/password_manager/core/browser/login_database.cc
@@ -2050,6 +2050,7 @@ FormRetrievalResult LoginDatabase::StatementToForms(
     EncryptionResult result = InitPasswordFormFromStatement(
         *statement, /*decrypt_and_fill_password_value=*/true, &primary_key,
         new_form.get());
+    BRAVE_STATEMENT_TO_FORMS
     if (result == ENCRYPTION_RESULT_SERVICE_FAILURE)
       return FormRetrievalResult::kEncrytionServiceFailure;
     if (result == ENCRYPTION_RESULT_ITEM_FAILURE) {
