diff --git a/components/omnibox/browser/autocomplete_match.h b/components/omnibox/browser/autocomplete_match.h
index dd6ee1f6a44e2c2157cf84165e1f7a775a0a68ce..31645fb15638e00627168fa241c18976f86fa218 100644
--- a/components/omnibox/browser/autocomplete_match.h
+++ b/components/omnibox/browser/autocomplete_match.h
@@ -772,7 +772,7 @@ struct AutocompleteMatch {
   // The provider of this match, used to remember which provider the user had
   // selected when the input changes. This may be NULL, in which case there is
   // no provider (or memory of the user's selection).
-  raw_ptr<AutocompleteProvider> provider = nullptr;
+  raw_ptr<AutocompleteProvider, DanglingUntriaged> provider = nullptr;
 
   // The relevance of this match. See table in autocomplete_provider.h for
   // scores returned by various providers. This is used to rank matches among
