diff --git a/components/bookmarks/browser/bookmark_model.h b/components/bookmarks/browser/bookmark_model.h
index 8258c2b992e8ceafa4dcb7a7906ad8d7dc063c3d..976921b7061126d1fa32de2a6607432d1be4bc1d 100644
--- a/components/bookmarks/browser/bookmark_model.h
+++ b/components/bookmarks/browser/bookmark_model.h
@@ -385,6 +385,7 @@ class BookmarkModel final : public BookmarkUndoProvider,
   // this is a no-op. Deletion is done asynchronously on a background thread.
   static void WipeAccountStorageForRollback(const base::FilePath& profile_path);
 
+  BRAVE_BOOKMARK_MODEL_H
  private:
   friend class BookmarkCodecTest;
   friend class BookmarkModelFaviconTest;
