diff --git a/components/signin/internal/identity_manager/BUILD.gn b/components/signin/internal/identity_manager/BUILD.gn
index a34c6c2fae0f2c3799e3f7d2f8fde7376fe03037..e439d91a2f47b01cfe41281404620d985f8ec6db 100644
--- a/components/signin/internal/identity_manager/BUILD.gn
+++ b/components/signin/internal/identity_manager/BUILD.gn
@@ -144,6 +144,7 @@ source_set("identity_manager") {
       "device_accounts_synchronizer_impl.h",
     ]
   }
+  import("//brave/components/signin/internal/identity_manager/sources.gni") sources += brave_components_signin_internal_identity_manager_sources
 }
 
 source_set("capabilities") {
