diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index ebf229de524b7d0eedf7e41a0b0140bcf3c9a5b2..4bb6fd1cddfd051b7b9f5d90bdbca84491080f02 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -223,6 +223,7 @@ android_library("java") {
     "//ui/android:ui_java",
     "//url:gurl_java",
   ]
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
