diff --git a/components/crash/core/browser/resources/crashes.js b/components/crash/core/browser/resources/crashes.js
index 287eed28ae76ef4655441f376937ea395a405c57..d83e733830f46406c7a8d9af8e3e54bf1b91e81e 100644
--- a/components/crash/core/browser/resources/crashes.js
+++ b/components/crash/core/browser/resources/crashes.js
@@ -91,7 +91,7 @@ function updateCrashList(
         crashLink.href = `https://goto.google.com/crash/${crash.id}`;
         crashLink.target = '_blank';
         crashLink.textContent = crash.id;
-        uploadIdValue.appendChild(crashLink);
+        //uploadIdValue.appendChild(crashLink); // feature disabled in Brave
       } else {
         uploadIdValue.textContent = crash.id;
       }
@@ -100,6 +100,7 @@ function updateCrashList(
 
       sendNowButton.remove();
       fileBugButton.onclick = () => fileBug(crash.id, os, version);
+      fileBugButton.remove();
     } else {
       uploadId.remove();
       uploadTime.remove();
