diff --git a/components/download/internal/common/download_item_impl.cc b/components/download/internal/common/download_item_impl.cc
index b7558098bc4e23025c69c41491b1044e256c8eb9..c7e7783e25b453cd5ec4ddb0d57b066768dca3f1 100644
--- a/components/download/internal/common/download_item_impl.cc
+++ b/components/download/internal/common/download_item_impl.cc
@@ -2041,8 +2041,8 @@ void DownloadItemImpl::OnDownloadCompleting() {
           &DownloadFile::RenameAndAnnotate,
           base::Unretained(download_file_.get()), GetTargetFilePath(),
           delegate_->GetApplicationClientIdForFileScanning(),
-          delegate_->IsOffTheRecord() ? GURL() : GetURL(),
-          delegate_->IsOffTheRecord() ? GURL() : GetReferrerUrl(),
+          GURL(),  // Never leak download URLs in metadata.
+          GURL(),  // See brave-browser#2766.
           delegate_->IsOffTheRecord() ? std::nullopt : GetRequestInitiator(),
           std::move(quarantine), std::move(rename_callback)));
 }
