diff --git a/components/permissions/permission_util.cc b/components/permissions/permission_util.cc
index cf2e9f7d7e6e954893efd765a0eb45fcfdae6a98..563ec491a2ca4d842b09d51ddce8c56f2bebd828 100644
--- a/components/permissions/permission_util.cc
+++ b/components/permissions/permission_util.cc
@@ -114,6 +114,7 @@ RequestTypeForUma PermissionUtil::GetUmaValueForRequests(
   if (requests.size() == 1) {
     return GetUmaValueForRequestType(request_type);
   }
+  BRAVE_GET_UMA_VALUE_FOR_REQUESTS
   return permissions::GetUmaValueForRequests(request_type);
 }
 
@@ -210,6 +211,7 @@ RequestTypeForUma PermissionUtil::GetUmaValueForRequestType(
 #endif
     case RequestType::kIdentityProvider:
       return RequestTypeForUma::PERMISSION_IDENTITY_PROVIDER;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
@@ -522,6 +524,7 @@ ContentSettingsType PermissionUtil::PermissionTypeToContentSettingsTypeSafe(
       return ContentSettingsType::LOCAL_NETWORK_ACCESS;
     case PermissionType::NUM:
       break;
+    PERMISSION_UTIL_PERMISSION_TYPE_TO_CONTENT_SETTINGS_TYPE
   }
 
   return ContentSettingsType::DEFAULT;
