diff --git a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettings.java b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettings.java
index 3a3019fddf6168107ffa121edb892e1ea4b7e1ae..60ca66002e28e3e5731ed9afe4afdb5bf64074e6 100644
--- a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettings.java
+++ b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/SiteSettings.java
@@ -24,7 +24,7 @@ import org.chromium.components.user_prefs.UserPrefs;
  * browser-wide.
  */
 public class SiteSettings
-        extends SiteSettingsPreferenceFragment implements Preference.OnPreferenceClickListener {
+        extends BraveSiteSettingsPreferencesBase implements Preference.OnPreferenceClickListener {
     // The keys for each category shown on the Site Settings page
     // are defined in the SiteSettingsCategory.
 
