diff --git a/components/history/core/browser/features.cc b/components/history/core/browser/features.cc
index 8d23737d7c9ed4c5ef1f693d1ead43c4e236f3d7..ab97deba9c4c152e325a4bbb5a6d7d000259dc15 100644
--- a/components/history/core/browser/features.cc
+++ b/components/history/core/browser/features.cc
@@ -26,7 +26,8 @@ constexpr int kMaxNumRepeatableQueriesDefaultValue =
 
 // If enabled, the most repeated queries from the user browsing history are
 // shown in the Most Visited tiles.
-BASE_FEATURE(kOrganicRepeatableQueries, kOrganicRepeatableQueriesDefaultValue);
+// kOrganicRepeatableQueries feature state is enforced via plaster rewrite.
+BASE_FEATURE(kOrganicRepeatableQueries, base::FEATURE_DISABLED_BY_DEFAULT);
 
 // The maximum number of repeatable queries to show in the Most Visited tiles.
 const base::FeatureParam<int> kMaxNumRepeatableQueries(
