diff --git a/components/content_settings/core/browser/BUILD.gn b/components/content_settings/core/browser/BUILD.gn
index 6c6fdcc82a7838853f355c5c207f9658c55a59e1..8221914d7d22cedb8ce7be6b9f895485be60679e 100644
--- a/components/content_settings/core/browser/BUILD.gn
+++ b/components/content_settings/core/browser/BUILD.gn
@@ -84,6 +84,7 @@ static_library("browser") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_content_settings_core_browser_sources deps += brave_components_content_settings_core_browser_deps
 }
 
 source_set("cookie_settings") {
