diff --git a/components/sync/engine/model_type_worker.h b/components/sync/engine/model_type_worker.h
index 8f224468d3c34a11738ac468572fb77e52a7d815..1d8183048347576f0095cf4893df61df14a1f762 100644
--- a/components/sync/engine/model_type_worker.h
+++ b/components/sync/engine/model_type_worker.h
@@ -207,6 +207,7 @@ class ModelTypeWorker : public UpdateHandler,
 
   static constexpr size_t kMaxPendingInvalidations = 10u;
 
+ BRAVE_MODEL_TYPE_WORKER_H_
  private:
   struct UnknownEncryptionKeyInfo {
     // Not increased if the cryptographer knows it's in a pending state
