diff --git a/components/crash/core/browser/resources/crashes.js b/components/crash/core/browser/resources/crashes.js
index e5d6a728d973aae10c033177fea4ecaade532d58..3d8a27076642d57c26fcd4a2ffcb1efdef052915 100644
--- a/components/crash/core/browser/resources/crashes.js
+++ b/components/crash/core/browser/resources/crashes.js
@@ -130,7 +130,7 @@ function updateCrashList(
       link.target = '_blank';
       link.textContent = loadTimeData.getString('bugLinkText');
       linkBlock.appendChild(link);
-      crashBlock.appendChild(linkBlock);
+      //crashBlock.appendChild(linkBlock); // feature disabled in Brave
     } else {
       let textContentKey;
       if (crash.state == 'pending_user_requested') {
