diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 800c487cf76a6c6b0ce50aff351807ab64be6d18..7cad93bb17ae25b1a1b261db4f97e353c40c431d 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -704,6 +704,7 @@ std::string GetPermissionStringForUma(
     case ContentSettingsType::LOOPBACK_NETWORK:
       return "LoopbackNetwork";
     default:
+      BRAVE_GET_PERMISSION_STRING_FOR_UMA
       break;
   }
   // The user is not prompted for these permissions thus there is no
