diff --git a/components/autofill/core/browser/foundations/autofill_client.h b/components/autofill/core/browser/foundations/autofill_client.h
index 588f6cab50e28c382185e4683834c6db644e4415..ea68e6d2d85e60df7d654c9737af2265f98d520a 100644
--- a/components/autofill/core/browser/foundations/autofill_client.h
+++ b/components/autofill/core/browser/foundations/autofill_client.h
@@ -999,6 +999,12 @@ class AutofillClient {
   // Returns whether the client uses platform-native autofill rather than
   // Chrome's built-in autofill UI/logic.
   virtual bool UsesPlatformAutofill() const = 0;
+  virtual void BraveAddSuggestions(
+      const PasswordFormClassification& form_classification,
+      const FormFieldData& field,
+      std::vector<Suggestion>& chromium_suggestions);
+  virtual bool BraveHandleSuggestion(const Suggestion& suggestion,
+                                      const autofill::FieldGlobalId& field);
 };
 
 }  // namespace autofill
