diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 803799da3c1f1f11b02c0a45a88c3e1bea7dc4bb..fca7a0cbd352b68bd26bb280ae4687682c6a8f39 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -49,6 +49,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 typedef int64_t VisitID;
