diff --git a/components/permissions/permission_util.cc b/components/permissions/permission_util.cc
index befb62e853d5ee43562876b82ae60598fd5f9473..09770e25013c764eb5bdd8bf09f1f8bf840bda0d 100644
--- a/components/permissions/permission_util.cc
+++ b/components/permissions/permission_util.cc
@@ -366,6 +366,7 @@ ContentSettingsType PermissionUtil::PermissionTypeToContentSettingTypeSafe(
       return ContentSettingsType::WEB_APP_INSTALLATION;
     case PermissionType::NUM:
       break;
+    PERMISSION_UTIL_PERMISSION_TYPE_TO_CONTENT_SETTINGS_TYPE
   }
 
   return ContentSettingsType::DEFAULT;
