diff --git a/components/signin/internal/identity_manager/BUILD.gn b/components/signin/internal/identity_manager/BUILD.gn
index 8f54d7a1cc96c6db7022c8b7cad38ff22b706547..a8cc5ec84da7d44d082b13879970ba8717283967 100644
--- a/components/signin/internal/identity_manager/BUILD.gn
+++ b/components/signin/internal/identity_manager/BUILD.gn
@@ -127,6 +127,7 @@ source_set("identity_manager") {
       "device_accounts_synchronizer_impl.h",
     ]
   }
+  import("//brave/components/signin/internal/identity_manager/sources.gni") sources += brave_components_signin_internal_identity_manager_sources
 }

 source_set("unit_tests") {
