diff --git a/components/content_settings/renderer/content_settings_agent_impl.h b/components/content_settings/renderer/content_settings_agent_impl.h
index e618979c959c3d0f6c78bbcdc14cd184c976bb34..96f0575b2c9239b082258750bd07bd80307aa76f 100644
--- a/components/content_settings/renderer/content_settings_agent_impl.h
+++ b/components/content_settings/renderer/content_settings_agent_impl.h
@@ -123,6 +123,7 @@ class ContentSettingsAgentImpl
     content_settings_manager_ = std::move(manager);
   }
 
+  BRAVE_CONTENT_SETTINGS_AGENT_IMPL_H_
  protected:
   // Allow this to be overridden by tests.
   virtual void BindContentSettingsManager(
