diff --git a/components/signin/public/base/session_binding_utils.cc b/components/signin/public/base/session_binding_utils.cc
index a9702d14da2ccd37a8998aa462659e123b21722d..c0671739d0b3a45cab5848e2becf860576d4ff2b 100644
--- a/components/signin/public/base/session_binding_utils.cc
+++ b/components/signin/public/base/session_binding_utils.cc
@@ -46,6 +46,8 @@ std::string SignatureAlgorithmToString(
       return "PS256";
     case crypto::SignatureVerifier::RSA_PKCS1_SHA1:
       return "RS1";
+    case crypto::SignatureVerifier::ECDSA_SHA384:
+      return "SHA384";
   }
 }
 
