diff --git a/components/sync/driver/profile_sync_service.h b/components/sync/driver/profile_sync_service.h
index adbca0d720e47ea720cf29c4a292fc1ce6855dd2..99b03cc04acecfa580d1b06d140c7746c9167bcf 100644
--- a/components/sync/driver/profile_sync_service.h
+++ b/components/sync/driver/profile_sync_service.h
@@ -254,6 +254,7 @@ class ProfileSyncService : public SyncService,
 
   SyncClient* GetSyncClientForTest();
 
+ BRAVE_PROFILE_SYNC_SERVICE_H_
  private:
   // Passed as an argument to StopImpl to control whether or not the sync
   // engine should clear its data when it shuts down. See StopImpl for more
