diff --git a/components/page_info/page_info_ui.cc b/components/page_info/page_info_ui.cc
index 359f9a2da42ae3f386d165d33ea8df4ee7bcc368..f6ee2ab1bbbf32313eb2bfcd4d5ecfd4e6f27aa9 100644
--- a/components/page_info/page_info_ui.cc
+++ b/components/page_info/page_info_ui.cc
@@ -652,6 +652,7 @@ const gfx::ImageSkia PageInfoUI::GetPermissionIcon(
     SkColor related_text_color) {
   const gfx::VectorIcon* icon = &gfx::kNoneIcon;
   switch (info.type) {
+    BRAVE_PAGE_INFO_UI_GET_PERMISSION_ICON
     case ContentSettingsType::COOKIES:
       icon = &vector_icons::kCookieIcon;
       break;
