diff --git a/components/javascript_dialogs/tab_modal_dialog_manager.cc b/components/javascript_dialogs/tab_modal_dialog_manager.cc
index bffeb732ae2c825efe9b7968d54537cd275a05bd..e8fdc81485d641807d92e502c253423f064ee5a9 100644
--- a/components/javascript_dialogs/tab_modal_dialog_manager.cc
+++ b/components/javascript_dialogs/tab_modal_dialog_manager.cc
@@ -254,6 +254,7 @@ void TabModalDialogManager::CancelDialogs(content::WebContents* web_contents,
 
 void TabModalDialogManager::OnVisibilityChanged(
     content::Visibility visibility) {
+  BRAVE_TAB_MODAL_DIALOG_MANAGER_ON_VISIBILITY_CHANGED
   if (visibility == content::Visibility::HIDDEN) {
     HandleTabSwitchAway(DismissalCause::kTabHidden);
   } else if (pending_dialog_) {
