diff --git a/components/sessions/BUILD.gn b/components/sessions/BUILD.gn
index 004a99346441e6d34166939546262d8f5432fb1c..9d53cf46acafca7701f68b48b48e314803029124 100644
--- a/components/sessions/BUILD.gn
+++ b/components/sessions/BUILD.gn
@@ -44,6 +44,7 @@ if (!is_ios) {
       "//ui/base",
       "//url",
     ]
+    import("//brave/components/sessions/sources.gni") deps += brave_components_sessions_deps
   }
 } else {
   source_set("sessions") {
@@ -156,6 +157,7 @@ source_set("shared") {
     "//ui/gfx",
     "//url",
   ]
+  import("//brave/components/sessions/sources.gni") public_deps += brave_components_sessions_shared_public_deps
 }
 
 static_library("test_support") {
