diff --git a/components/password_manager/core/browser/password_store/password_store.cc b/components/password_manager/core/browser/password_store/password_store.cc
index 99de2961aed0c23fc1cfffb1e3d6c161fbcf7ccc..ccdc6d262f2f78d17db0df8ded04058ddd5eb306 100644
--- a/components/password_manager/core/browser/password_store/password_store.cc
+++ b/components/password_manager/core/browser/password_store/password_store.cc
@@ -10,7 +10,7 @@
 #include <string>
 #include <utility>
 #include <variant>
-
+                                      #include "base/logging.h"
 #include "base/barrier_callback.h"
 #include "base/functional/bind.h"
 #include "base/functional/callback.h"
@@ -121,7 +121,9 @@ void PasswordStore::AddLogins(std::vector<StoredCredential> forms,
                             this, LoginsChangedTrigger::Addition))
                         .Then(std::move(completion)));
 
+LOG(ERROR) << "[BraveSync] PasswordStore::AddLogin 001 forms.size()="<<forms.size();
   for (auto& form : forms) {
+LOG(ERROR) << "[BraveSync] PasswordStore::AddLogin 002 form.username_value="<<form.username_value;
     CHECK(!form.blocked_by_user ||
           (form.username_value.empty() && form.password_value.empty()));
     backend_->AddLoginAsync(std::move(form), barrier_callback);
@@ -229,7 +231,7 @@ void PasswordStore::RemoveLogin(const base::Location& location,
                                  CloneStoredCredential(form)));
     return;
   }
-
+LOG(ERROR) << "[BraveSync] PasswordStore::RemoveLogin 001 form.username_value="<<form.username_value;
   backend_->RemoveLoginAsync(
       location, CloneStoredCredential(form),
       base::BindOnce(&PasswordStore::NotifyLoginsChangedOnMainSequence, this,
