diff --git a/components/content_settings/renderer/content_settings_agent_impl.cc b/components/content_settings/renderer/content_settings_agent_impl.cc
index 9b9f37709511a369087129b1476eebbb5b66fcaf..edf7c021ce212bf832821d7cde6ba3458af9bc2d 100644
--- a/components/content_settings/renderer/content_settings_agent_impl.cc
+++ b/components/content_settings/renderer/content_settings_agent_impl.cc
@@ -439,6 +439,7 @@ bool ContentSettingsAgentImpl::IsAllowlistedForContentSettings() const {
   const WebDocument& document = render_frame()->GetWebFrame()->GetDocument();
   WebSecurityOrigin origin = document.GetSecurityOrigin();
   WebURL document_url = document.Url();
+  BRAVE_CONTENT_SETTINGS_AGENT_IMPL_IS_WHITELISTED_FOR_CONTENT_SETTINGS
   if (document_url.GetString() == content::kUnreachableWebDataURL)
     return true;
 
