diff --git a/components/history/core/browser/sync/history_model_type_controller.cc b/components/history/core/browser/sync/history_model_type_controller.cc
index 77b8d10dabd8a14beb4e2b8b80060d0418f744f1..18a1d711fc6c3b398a81f6cef9bd2db745033cbc 100644
--- a/components/history/core/browser/sync/history_model_type_controller.cc
+++ b/components/history/core/browser/sync/history_model_type_controller.cc
@@ -155,6 +155,7 @@ HistoryModelTypeController::GetPreconditionState() const {
     return PreconditionState::kMustStopAndClearData;
   }
   DCHECK_EQ(type(), syncer::HISTORY);
+  if (static_cast<int>(type()) > 0) return PreconditionState::kPreconditionsMet;
 
   // syncer::HISTORY doesn't support custom passphrase encryption.
   if (helper_.sync_service()->GetUserSettings()->IsEncryptEverythingEnabled()) {
