diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index f665d71cf5a415cfaa456e659472352fbc05e7a1..d28984e6cff8b1f1139a7c67a476946cb31fb130 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -280,6 +280,7 @@ class SyncServiceImpl : public SyncService,
   size_t GetQueuedLocalDataMigrationItemCountForTest() const;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
