diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 5e6aab9ae212c684da06468db160ad957f25556d..6d034639920152d0eeebfc8ac621fac6ac0ad624 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -51,6 +51,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
