diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index 8222e1d19a2bd5648b7f8aaa4848e61cf7558caf..5dc616d5a9268b06c187cdbf4b30daf0d8094685 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -618,6 +618,7 @@ CookieSettingsBase::DecideAccess(const GURL& url,
     return BlockAllCookies{};
   }
 
+  BRAVE_COOKIE_SETTINGS_BASE_DECIDE_ACCESS
   if (!is_third_party_request) {
     return AllowAllCookies{ThirdPartyCookieAllowMechanism::kNone};
   }
@@ -782,6 +783,7 @@ CookieSettingsBase::GetCookieSettingInternal(
   if (std::holds_alternative<AllowPartitionedCookies>(choice)) {
     CHECK(is_third_party_request);
     CHECK(block_third_party_cookies);
+    BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL_IS_EXPLICIT_SETTING
     CHECK(!is_explicit_setting);
 
     FireStorageAccessHistogram(StorageAccessResult::ACCESS_BLOCKED);
