diff --git a/components/sync/driver/sync_service_impl.h b/components/sync/driver/sync_service_impl.h
index b62804a5e4ced73ecda1aec06c1a431247326a10..32d249aa978b691a46c067503ef315f7577dfd08 100644
--- a/components/sync/driver/sync_service_impl.h
+++ b/components/sync/driver/sync_service_impl.h
@@ -239,6 +239,7 @@ class SyncServiceImpl : public SyncService,
 
   SyncClient* GetSyncClientForTest();
 
+  BRAVE_SYNC_SERVICE_IMPL_H_
  private:
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
