diff --git a/components/media_message_center/media_notification_view_impl.h b/components/media_message_center/media_notification_view_impl.h
index 9102267a4402447969ef8d2fc70ab84b7ae7397d..236e4002c4a6ea498b36e55ac8a2dd5205733556 100644
--- a/components/media_message_center/media_notification_view_impl.h
+++ b/components/media_message_center/media_notification_view_impl.h
@@ -182,6 +182,7 @@ class COMPONENT_EXPORT(MEDIA_MESSAGE_CENTER) MediaNotificationViewImpl
   raw_ptr<views::View> pip_button_separator_view_ = nullptr;
   raw_ptr<views::ToggleImageButton> play_pause_button_ = nullptr;
   raw_ptr<views::ToggleImageButton> picture_in_picture_button_ = nullptr;
+  raw_ptr<views::ToggleImageButton> download_button_ = nullptr;
   raw_ptr<views::View> title_artist_row_ = nullptr;
   raw_ptr<views::Label> title_label_ = nullptr;
   raw_ptr<views::Label> artist_label_ = nullptr;
