diff --git a/components/webapps/browser/BUILD.gn b/components/webapps/browser/BUILD.gn
index 473f3d118d0ba3de05db2cdcd424a5c94e48f2e7..24a8a5008b7022e98bef5866e57113a3ecf767fb 100644
--- a/components/webapps/browser/BUILD.gn
+++ b/components/webapps/browser/BUILD.gn
@@ -197,6 +197,7 @@ source_set("browser") {
       "//ui/gfx",
     ]
   }
+  import("//brave/components/webapps/browser/sources.gni") deps += brave_components_webapps_browser_deps
 }
 
 source_set("test_support") {
