diff --git a/components/permissions/permission_util.cc b/components/permissions/permission_util.cc
index 49aa4e42a81a35851d8fa34805ad861a27a33553..2184263d4083fbb54d894cf4077c9b6b749b072b 100644
--- a/components/permissions/permission_util.cc
+++ b/components/permissions/permission_util.cc
@@ -338,6 +338,7 @@ ContentSettingsType PermissionUtil::PermissionTypeToContentSettingTypeSafe(
       return ContentSettingsType::WEB_PRINTING;
     case PermissionType::NUM:
       break;
+    PERMISSION_UTIL_PERMISSION_TYPE_TO_CONTENT_SETTINGS_TYPE
   }
 
   return ContentSettingsType::DEFAULT;
