diff --git a/components/crash/core/app/crashpad.cc b/components/crash/core/app/crashpad.cc
index 573d95aa49cfe2dbd2b61b55d3180f6d2ba9dbb2..dc59472cc3eb53eeca664e42b06573446d444ce3 100644
--- a/components/crash/core/app/crashpad.cc
+++ b/components/crash/core/app/crashpad.cc
@@ -96,6 +96,7 @@ bool InitializeCrashpadImpl(bool initial_client,
     // as processed by the backend.
     DCHECK(browser_process || process_type == "Chrome Installer" ||
            process_type == "notification-helper" ||
+           BRAVE_INITIALIZE_CRASHPAD_IMPL_PROCESS_TYPE
            process_type == "GCPW Installer" || process_type == "GCPW DLL");
 #elif BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_CHROMEOS) || BUILDFLAG(IS_ANDROID)
     DCHECK(browser_process);
