diff --git a/components/omnibox/browser/autocomplete_match.h b/components/omnibox/browser/autocomplete_match.h
index 2c577ee559f5cc83cefbc2648f5a6c8679a41fec..315cb2f855e9955979c11740a58f6ded7dc581cb 100644
--- a/components/omnibox/browser/autocomplete_match.h
+++ b/components/omnibox/browser/autocomplete_match.h
@@ -811,7 +811,7 @@ struct AutocompleteMatch {
   // The provider of this match, used to remember which provider the user had
   // selected when the input changes. This may be NULL, in which case there is
   // no provider (or memory of the user's selection).
-  raw_ptr<AutocompleteProvider> provider = nullptr;
+  raw_ptr<AutocompleteProvider, DanglingUntriaged> provider = nullptr;
 
   // The relevance of this match. See table in autocomplete_provider.h for
   // scores returned by various providers. This is used to rank matches among
