diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index f66f790dd83fcf27e865eec1d8a2a98838d35f48..8ebf68e3522f53104630600d4efe6b788a7d4437 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -468,6 +468,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
         return "window_management";
       }
 #endif
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
