diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 57ec08b826a66f14cd8ae3c742bafb58c8e4202b..cb2fe696f6bc0b88c1347e56c181aa7d91e98770 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1242,6 +1242,7 @@ void PageInfo::PresentSitePermissions() {
       }
     }
 
+    BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
     if (ShouldShowPermission(
             permission_info, site_url_, content_settings, web_contents_.get(),
             HasContentSettingChangedViaPageInfo(permission_info.type),
