diff --git a/components/browser_ui/settings/android/java/src/org/chromium/components/browser_ui/settings/SettingsNavigation.java b/components/browser_ui/settings/android/java/src/org/chromium/components/browser_ui/settings/SettingsNavigation.java
index 37cdb5aa52e6911ed6282e32a2078a45e48e1735..1cf37988b3a2d3cec4b130468948f6ebe3fedcd1 100644
--- a/components/browser_ui/settings/android/java/src/org/chromium/components/browser_ui/settings/SettingsNavigation.java
+++ b/components/browser_ui/settings/android/java/src/org/chromium/components/browser_ui/settings/SettingsNavigation.java
@@ -28,6 +28,7 @@ public interface SettingsNavigation {
         SettingsFragment.SAFETY_CHECK,
         SettingsFragment.SITE,
         SettingsFragment.ACCESSIBILITY,
+        SettingsFragment.PASSWORDS,
         SettingsFragment.GOOGLE_SERVICES,
         SettingsFragment.MANAGE_SYNC,
         SettingsFragment.FINANCIAL_ACCOUNTS,
@@ -55,6 +56,7 @@ public interface SettingsNavigation {
         int FINANCIAL_ACCOUNTS = 8;
         /// Non-card payment methods.
         int NON_CARD_PAYMENT_METHODS = 9;
+        int PASSWORDS = 10;
     }
 
     /**
