diff --git a/components/sync_device_info/device_info.h b/components/sync_device_info/device_info.h
index f97947a6cb5183af03450db40ae3c56df39c1a3d..2a1ff24a4f9e7ebdee573a861eeddbfb303c2816 100644
--- a/components/sync_device_info/device_info.h
+++ b/components/sync_device_info/device_info.h
@@ -145,6 +145,7 @@ class DeviceInfo {
   DeviceInfo(const DeviceInfo&) = delete;
   DeviceInfo& operator=(const DeviceInfo&) = delete;
 
+  virtual
   ~DeviceInfo();
 
   // Sync specific unique identifier for the device. Note if a device
