diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index 3b3278741f7fd936c2471938cd57c055b02acfd0..aef4155080bcb2598073c9435910ff4232d30fb1 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -133,6 +133,7 @@ android_library("java") {
     "//ui/android:ui_java",
   ]
 
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
