diff --git a/components/payments/content/BUILD.gn b/components/payments/content/BUILD.gn
index cd3fc2081227..b605ffd94156 100644
--- a/components/payments/content/BUILD.gn
+++ b/components/payments/content/BUILD.gn
@@ -53,7 +53,7 @@ jumbo_static_library("content") {
     "//url",
   ]
 
-  if (!is_android) {
+  #if (!is_android) {
     sources += [
       "content_payment_request_delegate.h",
       "payment_request.cc",
@@ -68,7 +68,7 @@ jumbo_static_library("content") {
       "payment_response_helper.cc",
       "payment_response_helper.h",
     ]
-  }
+  #}
 }
 
 # Files used by content and utility.
