diff --git a/components/sync/engine/sync_scheduler_impl.cc b/components/sync/engine/sync_scheduler_impl.cc
index 70cfe169ac82ee7bad02ddb86e2d7c3c0011fa78..7e792e7407f6ed0fb1fab3b4e55a2d5db60dd30f 100644
--- a/components/sync/engine/sync_scheduler_impl.cc
+++ b/components/sync/engine/sync_scheduler_impl.cc
@@ -507,6 +507,7 @@ void SyncSchedulerImpl::HandleFailure(
     SDVLOG(2) << "Sync cycle failed.  Will back off for "
               << wait_interval_->length.InMilliseconds() << "ms.";
   }
+  BRAVE_SYNC_SCHEDULER_IMPL_HANDLE_FAILURE
 }
 
 void SyncSchedulerImpl::DoPollSyncCycleJob() {
