diff --git a/components/content_settings/core/common/BUILD.gn b/components/content_settings/core/common/BUILD.gn
index f396acdb2e4c1cb114c00bfb7e33c5cdf1ae8995..78123bb06391651e82acf9e77e372eca39c807d4 100644
--- a/components/content_settings/core/common/BUILD.gn
+++ b/components/content_settings/core/common/BUILD.gn
@@ -18,14 +18,16 @@ jumbo_static_library("common") {
     "content_settings_utils.h",
     "cookie_settings_base.cc",
     "cookie_settings_base.h",
-    "features.cc",
-    "features.h",
     "pref_names.cc",
     "pref_names.h",
   ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
 
+  public_deps = [
+    ":features",
+  ]
+
   deps = [
     "//base",
     "//mojo/public/cpp/base",
@@ -62,3 +64,18 @@ mojom("mojo_bindings") {
     "//mojo/public/mojom/base",
   ]
 }
+
+component("features") {
+  output_name = "content_settings_features"
+
+  defines = [ "IS_CONTENT_SETTINGS_FEATURES_IMPL" ]
+
+  sources = [
+    "features.cc",
+    "features.h",
+  ]
+
+  deps = [
+    "//base",
+  ]
+}
