diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 961b28d70c9e13beba2bc88684ebf4a563767da8..3b9ed58ae6fe099c0ac37d2ce8fc1871bdaf346e 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1397,6 +1397,9 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // second section, the default behavior used when no per-type exception applies.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  if (!delegate_->BraveShouldShowPermission(info.type)) {
+    return false;
+  }
 #if BUILDFLAG(IS_ANDROID)
   if (info.type == ContentSettingsType::NOTIFICATIONS) {
     // `is_requested` is only populated if
@@ -1633,6 +1636,9 @@ void PageInfo::PresentSitePermissions() {
 
 std::set<net::SchemefulSite> PageInfo::GetTwoSitePermissionRequesters(
     ContentSettingsType type) {
+  if (type == ContentSettingsType::STORAGE_ACCESS) {
+    return {};
+  }
   std::set<net::SchemefulSite> requesters;
   // Collect sites that have tried to request a permission.
   auto* pscs = GetPageSpecificContentSettings();
