diff --git a/components/permissions/permission_util.cc b/components/permissions/permission_util.cc
index a6af470d7955f3bb1a6b5d4e6929cf55bbff0fbc..f564646b512912bf6439573cee0a1b02c668de8e 100644
--- a/components/permissions/permission_util.cc
+++ b/components/permissions/permission_util.cc
@@ -338,6 +338,7 @@ ContentSettingsType PermissionUtil::PermissionTypeToContentSettingTypeSafe(
       return ContentSettingsType::WEB_PRINTING;
     case PermissionType::NUM:
       break;
+    PERMISSION_UTIL_PERMISSION_TYPE_TO_CONTENT_SETTINGS_TYPE
   }
 
   return ContentSettingsType::DEFAULT;
