diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index 4ad3db9a76c334c268e24c715f376f745608ecdd..8b9449a6285a5e99ca344a637fc36dd3b7a2cb6b 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -195,6 +195,7 @@ android_library("java") {
     "//third_party/jni_zero:jni_zero_java",
     "//ui/android:ui_java",
   ]
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
