diff --git a/components/update_client/BUILD.gn b/components/update_client/BUILD.gn
index 9180067cdebe7ff59d12d2081717528e282c91f7..0aebc062ebeb9bc40e79f8351e4a29e212b32846 100644
--- a/components/update_client/BUILD.gn
+++ b/components/update_client/BUILD.gn
@@ -198,6 +198,7 @@ static_library("update_client") {
       "background_downloader_win.h",
     ]
   }
+  sources += brave_components_update_client_sources deps += brave_components_update_client_deps public_deps = brave_components_update_client_public_deps
 }
 
 if (is_mac) {
