diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index f6a85f48e5743e42788c7d244dd1d7a91174c0c9..1c0c28e851fda8a107faa2180bc7b125c06f3a4f 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -355,6 +355,7 @@ AutocompleteController::AutocompleteController(
     if (provider_types & AutocompleteProvider::TYPE_SEARCH) {
       search_provider_ = new SearchProvider(provider_client_.get(), this);
       providers_.push_back(search_provider_.get());
+      BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
     }
     // Providers run in the order they're added.  Add `HistoryURLProvider` after
     // `SearchProvider` because:
@@ -933,6 +934,8 @@ void AutocompleteController::UpdateResult(
   metrics_.OnUpdateResult(last_result_for_logging,
                           result_.GetMatchDedupComparators());
 
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
+
   // Below are all annotations after the match list is ready.
 #if !BUILDFLAG(IS_IOS)
   // HistoryClusters is not enabled on iOS.
