diff --git a/components/sync/base/features.cc b/components/sync/base/features.cc
index 245205cee06b8409560fbe42f8400296a302a985..f52833e68507356b6b7e327f5150beeb38faf972 100644
--- a/components/sync/base/features.cc
+++ b/components/sync/base/features.cc
@@ -118,7 +118,7 @@ DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncAccountSettings);
 DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuable);
 #endif
 
-DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuableMetadata);
+BASE_FEATURE(kSyncAutofillValuableMetadata, base::FEATURE_DISABLED_BY_DEFAULT);
 
 DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncWalletFlightReservations);
 
