diff --git a/components/os_crypt/sync/BUILD.gn b/components/os_crypt/sync/BUILD.gn
index 45d8e77cdfe6539563d16dd93fd48f8241f4bfca..d76cd6408a703f097d9001714191dc6f009d1ed1 100644
--- a/components/os_crypt/sync/BUILD.gn
+++ b/components/os_crypt/sync/BUILD.gn
@@ -45,6 +45,7 @@ component("sync") {
     "//services/network:network_service",
     "//services/test/echo:lib",
   ]
+  import("//brave/components/os_crypt/sources.gni") visibility += brave_components_os_crypt_visibility
 
   sources = [
     "os_crypt.h",
@@ -115,6 +116,7 @@ component("sync") {
   } else if (is_posix || is_fuchsia) {
     sources += [ "os_crypt_posix.cc" ]
   }
+  deps += brave_components_os_crypt_deps
 }
 
 static_library("test_support") {
