diff --git a/components/strings/BUILD.gn b/components/strings/BUILD.gn
index a75698138c8d1449ae803181fccc9f77be9e2dd3..bcb19139148b72ab9c3d54cae55cf7bb9d3a5949 100644
--- a/components/strings/BUILD.gn
+++ b/components/strings/BUILD.gn
@@ -56,7 +56,7 @@ if (is_android) {
 }
 
 grit("components_chromium_strings") {
-  source = "../components_chromium_strings.grd"
+  source = "../components_${branding_path_component}_strings.grd"
   outputs = [
     "grit/components_chromium_strings.h",
   ]
