diff --git a/components/infobars/core/infobar_delegate.h b/components/infobars/core/infobar_delegate.h
index b50703c15cf9dcb49a5c8d732cccd25b1bdee8ea..753801e092e0f833eb5f8b214597bf5c2a35e4c5 100644
--- a/components/infobars/core/infobar_delegate.h
+++ b/components/infobars/core/infobar_delegate.h
@@ -190,6 +190,7 @@ class InfoBarDelegate {
     CREDENTIAL_PROVIDER_INFOBAR_DELEGATE_IOS = 122,
     PDF_INFOBAR_DELEGATE = 123,
     INSTALLER_DOWNLOADER_INFOBAR_DELEGATE = 124,
+    BRAVE_INFOBAR_DELEGATE_IDENTIFIERS
   };
   // LINT.ThenChange(//tools/metrics/histograms/metadata/browser/enums.xml:InfoBarIdentifier)
 
