diff --git a/components/content_settings/core/common/content_settings.cc b/components/content_settings/core/common/content_settings.cc
index a27c4b550910e10365e002a4494aab3a00264107..b40ad3cccf9f7365953e1eac58cdc21a241399de 100644
--- a/components/content_settings/core/common/content_settings.cc
+++ b/components/content_settings/core/common/content_settings.cc
@@ -176,6 +176,7 @@ bool RendererContentSettingRules::IsRendererContentSetting(
          content_type == ContentSettingsType::JAVASCRIPT ||
          content_type == ContentSettingsType::CLIENT_HINTS ||
          content_type == ContentSettingsType::POPUPS ||
+         BRAVE_IS_RENDERER_CONTENT_SETTING
          content_type == ContentSettingsType::MIXEDSCRIPT;
 }
 
