diff --git a/components/infobars/core/infobar_delegate.h b/components/infobars/core/infobar_delegate.h
index 1bbe2fca2f777edf2bfddc3da24bc0b0b4cd0dd1..3497cfc529535c12f8b34e4f817562697780a744 100644
--- a/components/infobars/core/infobar_delegate.h
+++ b/components/infobars/core/infobar_delegate.h
@@ -207,6 +207,18 @@ class InfoBarDelegate {
     AUTOFILL_AI_SAVE_ENTITY_INFOBAR_DELEGATE_IOS = 132,
     JS_OPTIMIZATIONS_INFOBAR_DELEGATE = 133,
     WEB_APP_BLOCKED_MIGRATION_INFOBAR_DELEGATE = 134,
+    BRAVE_CONFIRM_P3A_INFOBAR_DELEGATE = 500,
+    // WAYBACK_MACHINE_INFOBAR_DELEGATE = 502
+    // SYNC_V2_MIGRATE_INFOBAR_DELEGATE = 503
+    // ANDROID_SYSTEM_SYNC_DISABLED_INFOBAR = 504
+    SYNC_CANNOT_RUN_INFOBAR = 505,
+    WEB_DISCOVERY_INFOBAR_DELEGATE = 506,
+    BRAVE_SYNC_ACCOUNT_DELETED_INFOBAR = 507,
+    BRAVE_REQUEST_OTR_INFOBAR_DELEGATE = 508,
+    // DEV_CHANNEL_DEPRECATION_INFOBAR_DELEGATE = 509
+    SEARCH_RESULT_AD_CLICKED_INFOBAR_DELEGATE = 510,
+    NEW_TAB_TAKEOVER_INFOBAR_DELEGATE = 511,
+    BRAVE_PSST_INFOBAR_DELEGATE = 512,
   };
   // LINT.ThenChange(//tools/metrics/histograms/metadata/browser/enums.xml:InfoBarIdentifier)
 
