diff --git a/components/page_info/page_info_ui.cc b/components/page_info/page_info_ui.cc
index d34cecf07f07bd7bd7fafec529a54cf41a258d47..91f15f9087e514ec126f7e91587d1ae9beb1e834 100644
--- a/components/page_info/page_info_ui.cc
+++ b/components/page_info/page_info_ui.cc
@@ -658,6 +658,7 @@ const ui::ImageModel PageInfoUI::GetPermissionIcon(
     const PageInfo::PermissionInfo& info) {
   const gfx::VectorIcon* icon = &gfx::kNoneIcon;
   switch (info.type) {
+    BRAVE_PAGE_INFO_UI_GET_PERMISSION_ICON
     case ContentSettingsType::COOKIES:
       icon = &vector_icons::kCookieIcon;
       break;
