diff --git a/components/sync/driver/sync_stopped_reporter.cc b/components/sync/driver/sync_stopped_reporter.cc
index 53b919d9703a5f06134c85e6bb795891ef54c47f..def6671dea043a24aafd89a51ada55fb9a68a10c 100644
--- a/components/sync/driver/sync_stopped_reporter.cc
+++ b/components/sync/driver/sync_stopped_reporter.cc
@@ -94,9 +94,10 @@ void SyncStoppedReporter::ReportSyncStopped(const std::string& access_token,
       net::LOAD_BYPASS_CACHE | net::LOAD_DISABLE_CACHE;
   resource_request->credentials_mode = network::mojom::CredentialsMode::kOmit;
   resource_request->method = "POST";
-  resource_request->headers.SetHeader(
-      net::HttpRequestHeaders::kAuthorization,
-      base::StringPrintf("Bearer %s", access_token.c_str()));
+  // resource_request->headers.SetHeader(
+  //     net::HttpRequestHeaders::kAuthorization,
+  //     base::StringPrintf("Bearer %s", access_token.c_str()));
+  BRAVE_REPORT_SYNC_STOPPED
   resource_request->headers.SetHeader(net::HttpRequestHeaders::kUserAgent,
                                       user_agent_);
   simple_url_loader_ = network::SimpleURLLoader::Create(
