diff --git a/components/password_manager/core/browser/password_store/password_store_built_in_backend.cc b/components/password_manager/core/browser/password_store/password_store_built_in_backend.cc
index 2aa2dfb47f73ce9ce13a9a809b58b454f0d62c8e..b383f9eae16bde02b11148222880e8ad7be744d8 100644
--- a/components/password_manager/core/browser/password_store/password_store_built_in_backend.cc
+++ b/components/password_manager/core/browser/password_store/password_store_built_in_backend.cc
@@ -86,7 +86,7 @@ bool ShouldForwardSyncErrorToStore(
     case SyncError::kTrustedVaultRecoverabilityDegradedForPasswords:
     case SyncError::kTrustedVaultRecoverabilityDegradedForEverything:
     case SyncError::kBookmarksLimitExceeded:
-#if BUILDFLAG(IS_IOS)
+#if BUILDFLAG(IS_IOS) || BUILDFLAG(IS_ANDROID)
     case SyncError::kDeviceManagementError:
       // TODO(crbug.com/539816393): Update this case if it needs to block saving
 #endif  // BUILDFLAG(IS_IOS)
@@ -120,7 +120,7 @@ SyncErrorToBackendError(syncer::SyncService::UserActionableError error) {
     case SyncError::kBookmarksLimitExceeded:
     case SyncError::kTrustedVaultRecoverabilityDegradedForPasswords:
     case SyncError::kTrustedVaultRecoverabilityDegradedForEverything:
-#if BUILDFLAG(IS_IOS)
+#if BUILDFLAG(IS_IOS) || BUILDFLAG(IS_ANDROID)
     case SyncError::kDeviceManagementError:
 #endif  // BUILDFLAG(IS_IOS)
 
@@ -174,7 +174,7 @@ ActionableError SyncErrorToActionableError(
 #endif
     case SyncError::kNeedsClientUpgrade:
     case SyncError::kBookmarksLimitExceeded:
-#if BUILDFLAG(IS_IOS)
+#if BUILDFLAG(IS_IOS) || BUILDFLAG(IS_ANDROID)
     case SyncError::kDeviceManagementError:
 #endif  // BUILDFLAG(IS_IOS)
       return ActionableError::kInactionable;
