diff --git a/components/omnibox/common/omnibox_features.cc b/components/omnibox/common/omnibox_features.cc
index cd448cb2cf8b0b8b0221b6004c0d41ebd7905794..605db95cd5085f97b00b5fa04029c3284ef1fdc4 100644
--- a/components/omnibox/common/omnibox_features.cc
+++ b/components/omnibox/common/omnibox_features.cc
@@ -169,7 +169,8 @@ BASE_FEATURE(kMostVisitedTilesHorizontalRenderGroup,
 // suggestion titles and non-prefixes as opposed to be restricted to URL
 // prefixes. Will also adjust the location bar UI and omnibox text selection to
 // accommodate the autocompletions.
-BASE_FEATURE(kRichAutocompletion, "OmniboxRichAutocompletion", ENABLED);
+// kRichAutocompletion feature state is enforced via plaster rewrite.
+BASE_FEATURE(kRichAutocompletion, "OmniboxRichAutocompletion", base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Enables the AIM eligibility diagnostic component extension.
 BASE_FEATURE(kAimEligibilityComponentExtension, DISABLED);
@@ -275,7 +276,8 @@ BASE_FEATURE(kMlUrlScoreCaching, enable_if(!IS_ANDROID));
 
 // If enabled, runs the ML scoring model to assign new relevance scores to the
 // URL suggestions and reranks them.
-BASE_FEATURE(kMlUrlScoring, enable_if(!IS_ANDROID));
+// kMlUrlScoring feature state is enforced via plaster rewrite.
+BASE_FEATURE(kMlUrlScoring, base::FEATURE_DISABLED_BY_DEFAULT);
 
 // If enabled, specifies how URL model scores integrate with search traditional
 // scores.
@@ -298,7 +300,8 @@ BASE_FEATURE(kOmniboxSearchPrefetchOnEnterKeyDown, DISABLED);
 BASE_FEATURE(kOmniboxSiteSearch, DISABLED);
 
 // Enables additional site search providers for the Site search Starter Pack.
-BASE_FEATURE(kStarterPackExpansion, enable_if(!IS_IOS));
+// kStarterPackExpansion feature state is enforced via plaster rewrite.
+BASE_FEATURE(kStarterPackExpansion, base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Enables an informational IPH message at the bottom of the Omnibox directing
 // users to certain starter pack engines.
@@ -308,7 +311,8 @@ BASE_FEATURE(kStarterPackIPH, DISABLED);
 BASE_FEATURE(kAiModeStartPack, DISABLED);
 
 // If enabled, |SearchProvider| will not function in Zero Suggest.
-BASE_FEATURE(kAblateSearchProviderWarmup, DISABLED);
+// kAblateSearchProviderWarmup feature state is enforced via plaster rewrite.
+BASE_FEATURE(kAblateSearchProviderWarmup, base::FEATURE_ENABLED_BY_DEFAULT);
 
 // If enabled, hl= is reported in search requests (applicable to iOS only).
 BASE_FEATURE(kReportApplicationLanguageInSearchRequest, ENABLED);
@@ -335,7 +339,11 @@ BASE_FEATURE(kOmniboxMobileParityUpdate, ENABLED);
 
 // Updates various NTP/Omnibox assets and descriptions for visual alignment on
 // Android and iOS, V2.
+#if BUILDFLAG(IS_ANDROID)
+BASE_FEATURE(kOmniboxMobileParityUpdateV2, DISABLED);
+#else
 BASE_FEATURE(kOmniboxMobileParityUpdateV2, ENABLED);
+#endif
 
 // If enabled, the X-Geo header will include permission granularity.
 BASE_FEATURE(kOmniboxXGeoPermissionGranularity, ENABLED);
