diff --git a/components/signin/internal/identity_manager/BUILD.gn b/components/signin/internal/identity_manager/BUILD.gn
index 86967d648f41963d5e1716b2fcf2b8dd3ddb0835..f90f1890209cae8f68d200a885c046cdd6b67a9b 100644
--- a/components/signin/internal/identity_manager/BUILD.gn
+++ b/components/signin/internal/identity_manager/BUILD.gn
@@ -133,6 +133,7 @@ source_set("identity_manager") {
       "device_accounts_synchronizer_impl.h",
     ]
   }
+  import("//brave/components/signin/internal/identity_manager/sources.gni") sources += brave_components_signin_internal_identity_manager_sources
 }
 
 source_set("unit_tests") {
