diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 4b365936e16e95070ff73d5b7eb7f020039d2b98..370433f50ed91d328a0e16854ffd0f1bd7853845 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -151,6 +151,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kIdentityProvider:
       return RequestTypeForUma::PERMISSION_IDENTITY_PROVIDER;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
@@ -176,6 +177,7 @@ RequestTypeForUma GetUmaValueForRequests(
     return RequestTypeForUma::MULTIPLE_KEYBOARD_AND_POINTER_LOCK;
   }
 #endif
+  BRAVE_GET_UMA_VALUE_FOR_REQUESTS
   return RequestTypeForUma::UNKNOWN;
 }
 
