diff --git a/components/sync_preferences/BUILD.gn b/components/sync_preferences/BUILD.gn
index 952d70810eb36c73a12fca4a545ccddabca21a4f..a38ac1a9fb98d2857adb1a69c7f81ddd3db777fc 100644
--- a/components/sync_preferences/BUILD.gn
+++ b/components/sync_preferences/BUILD.gn
@@ -78,6 +78,7 @@ source_set("common_syncable_prefs_database") {
     "//components/translate/core/browser",
     "//components/variations/service",
   ]
+  import("//brave/components/sync_preferences/sources.gni") public_deps = brave_common_syncable_prefs_database_public_deps
 }
 
 static_library("test_support") {
