diff --git a/components/sync/base/features.cc b/components/sync/base/features.cc
index 245205cee06b8409560fbe42f8400296a302a985..68567813fc641ae453bc1747d626177ab8c55ea2 100644
--- a/components/sync/base/features.cc
+++ b/components/sync/base/features.cc
@@ -89,6 +89,7 @@ BASE_FEATURE(kReplaceSyncPromosWithSigninPromosNewSignin,
 );
 
 bool IsReplaceSyncPromosWithSignInPromosEnabled() {
+  if ((true)) return false;
   return base::FeatureList::IsEnabled(kReplaceSyncPromosWithSignInPromos) ||
          base::FeatureList::IsEnabled(
              kReplaceSyncPromosWithSigninPromosNewSignin);
@@ -118,7 +119,7 @@ DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncAccountSettings);
 DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuable);
 #endif
 
-DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncAutofillValuableMetadata);
+BASE_OVERRIDDEN_FEATURE(kSyncAutofillValuableMetadata, base::FEATURE_DISABLED_BY_DEFAULT);
 
 DEFINE_SYNC_AUTOFILL_AI_FEATURE(kSyncWalletFlightReservations);
 
