diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index 00db9b31085208523b92fdac06d29943425f2ccf..57ee48f08a78c5e3c3f31b851ecb1b39ccdade13 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -462,6 +462,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
 #endif
     case permissions::RequestType::kIdentityProvider:
       return "identity_provider";
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
