diff --git a/components/password_manager/core/browser/password_form_filling.cc b/components/password_manager/core/browser/password_form_filling.cc
index 2095ccb7ca13ed49fb71b94b1af1e95b061fd28e..ebf24f9155de6570c07ea353b494681c47b7854d 100644
--- a/components/password_manager/core/browser/password_form_filling.cc
+++ b/components/password_manager/core/browser/password_form_filling.cc
@@ -194,6 +194,9 @@ LikelyFormFilling SendFillInformationToRenderer(
 
   bool wait_for_username =
       wait_for_username_reason != WaitForUsernameReason::kDontWait;
+#if defined(BRAVE_CHROMIUM_BUILD)
+  wait_for_username = true;
+#endif
 
   if (wait_for_username) {
     metrics_recorder->SetManagerAction(
