diff --git a/components/content_settings/core/browser/content_settings_global_value_map.cc b/components/content_settings/core/browser/content_settings_global_value_map.cc
index b2cd549dee50f9d3b0c9b7c09a1af06420056817..3229061b7d725cb67ab6c35b3ded29ce781e0a07 100644
--- a/components/content_settings/core/browser/content_settings_global_value_map.cc
+++ b/components/content_settings/core/browser/content_settings_global_value_map.cc
@@ -44,6 +44,7 @@ GlobalValueMap::~GlobalValueMap() {}
 
 std::unique_ptr<RuleIterator> GlobalValueMap::GetRuleIterator(
     ContentSettingsType content_type) const {
+  BRAVE_EARLY_RETURN_ON_SHIELDS_CONTENT_SETTINGS_TYPE
   auto it = settings_.find(content_type);
   if (it == settings_.end())
     return nullptr;
