diff --git a/components/web_modal/web_contents_modal_dialog_manager.cc b/components/web_modal/web_contents_modal_dialog_manager.cc
index 8264ce48468260d58a4b57ed154ec3f10c6f30a6..81e73cddaee77cbd7a21ebc87bd886b653944148 100644
--- a/components/web_modal/web_contents_modal_dialog_manager.cc
+++ b/components/web_modal/web_contents_modal_dialog_manager.cc
@@ -179,6 +179,7 @@ void WebContentsModalDialogManager::OnVisibilityChanged(
   const content::Visibility previous_web_contents_visibility =
       web_contents_visibility_;
   web_contents_visibility_ = visibility;
+  BRAVE_WEB_CONTENTS_MODAL_DIALOG_MANAGER_ON_VISIBILITY_CHANGED
   if (child_dialogs_.empty()) {
     return;
   }
