diff --git a/components/download/internal/common/download_item_impl.cc b/components/download/internal/common/download_item_impl.cc
index 906a126260b98c085d95617ff50ba544c22630ee..e59a8acbc395c4e564c78d139066afda398c55f4 100644
--- a/components/download/internal/common/download_item_impl.cc
+++ b/components/download/internal/common/download_item_impl.cc
@@ -1966,8 +1966,8 @@ void DownloadItemImpl::OnDownloadCompleting() {
           &DownloadFile::RenameAndAnnotate,
           base::Unretained(download_file_.get()), GetTargetFilePath(),
           delegate_->GetApplicationClientIdForFileScanning(),
-          delegate_->IsOffTheRecord() ? GURL() : GetURL(),
-          delegate_->IsOffTheRecord() ? GURL() : GetReferrerUrl(),
+          GURL(),  // Never leak download URLs in metadata.
+          GURL(),  // See brave-browser#2766.
           delegate_->IsOffTheRecord() ? std::nullopt : GetRequestInitiator(),
           std::move(quarantine), std::move(rename_callback)));
 }
