diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index fbf5178558298c1fe272027d4dac617064ec2413..647e3cb9f15078b6eae08fc3b70934efbcd453af 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -305,6 +305,7 @@ static_library("rest_of_sync") {
     "nigori/pending_local_nigori_commit.cc",
     "nigori/pending_local_nigori_commit.h",
   ]
+  sources += brave_components_sync_sources
 
   configs += [ "//build/config:precompiled_headers" ]
 
