diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 5f9810305de9f66d49acf58d0a1fa202cbb8d80d..78d8afdb0722d50ac8c5992c9626ae478df4c32c 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -53,6 +53,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
