diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 903a39103566495f25a88cfe74adcbb110f67275..558198c9ed93c6039faa10077aeede7dbbb86d9c 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -284,6 +284,7 @@ class SyncServiceImpl : public SyncService,
   size_t GetQueuedLocalDataMigrationItemCountForTest() const;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
