diff --git a/components/infobars/core/infobar_delegate.h b/components/infobars/core/infobar_delegate.h
index e2050a34db75bd190594f7fa488dc7cc93ed0c96..6f4f058a47f982d1e6d125a3e2905aa7bc3d76ed 100644
--- a/components/infobars/core/infobar_delegate.h
+++ b/components/infobars/core/infobar_delegate.h
@@ -193,6 +193,7 @@ class InfoBarDelegate {
     COLLABORATION_GROUP_UPDATE_INFOBAR_DELEGATE = 125,
     COLLABORATION_OUT_OF_DATE_INFOBAR_DELEGATE = 126,
     PIN_INFOBAR_DELEGATE = 127,
+    BRAVE_INFOBAR_DELEGATE_IDENTIFIERS
   };
   // LINT.ThenChange(//tools/metrics/histograms/metadata/browser/enums.xml:InfoBarIdentifier)
 
