diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 7e7b86b435be882ea69a1dd0c15579a97a07cc87..bdc47a486cdd9da8b45141e7c43638d884c30e1d 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -979,6 +979,7 @@ void PageInfo::PresentSitePermissions() {
       }
     }
 
+    BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
     if (ShouldShowPermission(
             permission_info, site_url_, content_settings, web_contents(),
             HasContentSettingChangedViaPageInfo(permission_info.type),
