diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index 9ca6045386ea4f39528c6581d6074af238bc0e25..e6edfe4b98c80ceb612032bfad4b3eb525616083 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -193,6 +193,7 @@ android_library("java") {
     "//third_party/jni_zero:jni_zero_java",
     "//ui/android:ui_java",
   ]
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
