diff --git a/components/permissions/permission_context_base.h b/components/permissions/permission_context_base.h
index 506e1f7c8571fe91330041e1caf3eca5d733f8c1..ab7d541fb673172d060ccaa7f8aa758c22370a3d 100644
--- a/components/permissions/permission_context_base.h
+++ b/components/permissions/permission_context_base.h
@@ -136,6 +136,7 @@ class PermissionContextBase : public content_settings::Observer {
     return content_settings_type_;
   }
 
+  BRAVE_PERMISSION_CONTEXT_BASE_H_
  protected:
   virtual ContentSetting GetPermissionStatusInternal(
       content::RenderFrameHost* render_frame_host,
