diff --git a/components/variations/variations_seed_store.h b/components/variations/variations_seed_store.h
index e3041ab83e82d704876271218b0589a4f9872def..7b1c9a92b562237908c8009065bb0a8cd63fbf93 100644
--- a/components/variations/variations_seed_store.h
+++ b/components/variations/variations_seed_store.h
@@ -301,6 +301,8 @@ class COMPONENT_EXPORT(VARIATIONS) VariationsSeedStore {
       base::OnceCallback<void(StoredSeedInfo)> done_callback,
       SeedType seed_type);
 
+  void SetSessionCountry(std::string_view country_code);
+
  protected:
   // Stores the serial number of the latest seed.
   void StoreLatestSerialNumber(std::string_view serial_number);
