diff --git a/components/content_settings/browser/content_settings_manager_impl.h b/components/content_settings/browser/content_settings_manager_impl.h
index 09a41c1061e1a150180abd22e8b99ea65306a531..a7f98f9a72e2557061df09eb5f4da442775c1305 100644
--- a/components/content_settings/browser/content_settings_manager_impl.h
+++ b/components/content_settings/browser/content_settings_manager_impl.h
@@ -65,6 +65,7 @@ class ContentSettingsManagerImpl
   void OnContentBlocked(int32_t render_frame_id,
                         ContentSettingsType type) override;
 
+  BRAVE_CONTENT_SETTINGS_MANAGER_IMPL_H_
  private:
   ContentSettingsManagerImpl(content::RenderProcessHost* render_process_host,
                              std::unique_ptr<Delegate> delegate);
