diff --git a/components/visited_url_ranking/internal/BUILD.gn b/components/visited_url_ranking/internal/BUILD.gn
index f41f4c6445c12351d2c653315769202fdc411dbb..387b007b64e262131a50052332472e5a65bf3af2 100644
--- a/components/visited_url_ranking/internal/BUILD.gn
+++ b/components/visited_url_ranking/internal/BUILD.gn
@@ -119,6 +119,7 @@ source_set("test_support") {
     "//testing/gmock",
     "//testing/gtest",
   ]
+  import("//brave/components/sync_device_info/sources.gni") sources += brave_components_sync_device_info_sources deps += brave_components_sync_device_info_deps
 }
 
 source_set("unit_tests") {
