diff --git a/components/autofill/core/browser/foundations/autofill_client.h b/components/autofill/core/browser/foundations/autofill_client.h
index 8207188fa9af1f2994fe89dfec101912eaeeec54..83957d88f01113b3e6fd5f5e102a1dd288858244 100644
--- a/components/autofill/core/browser/foundations/autofill_client.h
+++ b/components/autofill/core/browser/foundations/autofill_client.h
@@ -983,6 +983,12 @@ class AutofillClient {
   // Returns whether the client uses platform-native autofill rather than
   // Chrome's built-in autofill UI/logic.
   virtual bool UsesPlatformAutofill() const = 0;
+  virtual void BraveAddSuggestions(
+      const PasswordFormClassification& form_classification,
+      const FormFieldData& field,
+      std::vector<Suggestion>& chromium_suggestions);
+  virtual bool BraveHandleSuggestion(const Suggestion& suggestion,
+                                      const autofill::FieldGlobalId& field);
 };
 
 }  // namespace autofill
