diff --git a/components/password_manager/core/browser/password_store/login_database_posix.cc b/components/password_manager/core/browser/password_store/login_database_posix.cc
index 993acc2244de27bea31826a407ca50d64b88bdc4..c9fe6495c512a62a432d773d1cc43670e2b5585b 100644
--- a/components/password_manager/core/browser/password_store/login_database_posix.cc
+++ b/components/password_manager/core/browser/password_store/login_database_posix.cc
@@ -45,6 +45,7 @@ EncryptionResult LoginDatabase::DecryptedString(
     std::u16string* plain_text) const {
 #if !BUILDFLAG(IS_FUCHSIA)
 #if BUILDFLAG(IS_CHROMEOS)
+|| BUILDFLAG(IS_ANDROID)
   // On ChromeOS, we have a mix of obfuscated and plain-text
   // passwords. Obfuscated passwords always start with "v10", therefore anything
   // else is plain-text.
@@ -67,6 +68,7 @@ EncryptionResult LoginDatabase::DecryptedString(
       encryptor_ ? encryptor_->DecryptString16(cipher_text, plain_text)
                  : OSCrypt::DecryptString16(cipher_text, plain_text);
 #if BUILDFLAG(IS_CHROMEOS)
+|| BUILDFLAG(IS_ANDROID)
   // If decryption failed, we assume it was because the value was actually a
   // plain-text password which started with "v10".
   // TODO(crbug.com/41457193): Remove this when there isn't a mix of plain-text
