diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
index d9c45663b018236aa7c7d406210b550e271bcbfe..c64995f5a29d07401d98c77f35ac69b9831d9ec9 100644
--- a/components/permissions/BUILD.gn
+++ b/components/permissions/BUILD.gn
@@ -146,6 +146,7 @@ source_set("permissions") {
   if (!is_android) {
     deps += [ "//components/guest_view/browser" ]
   }
+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
   if (is_android) {
     sources += [
       "android/android_permission_util.cc",
