diff --git a/components/variations/seed_reader_writer.h b/components/variations/seed_reader_writer.h
index 08ed89c1f8715f7d3cd4d87f36c796780f6aaad7..0a5d06626330869657bf4c7fad7da08d634577ae 100644
--- a/components/variations/seed_reader_writer.h
+++ b/components/variations/seed_reader_writer.h
@@ -268,6 +268,8 @@ class COMPONENT_EXPORT(VARIATIONS) SeedReaderWriter
   // Converts a time from StoredSeedInfo proto format (int64_t) to a base::Time.
   static base::Time ProtoTimeToTime(int64_t proto_time);
 
+  void SetSessionCountry(std::string_view country_code);
+
  private:
   // The storage format of the seed data.
   // - kCompressed: the seed is compressed. This is used for backward
