diff --git a/components/sync/driver/profile_sync_service.h b/components/sync/driver/profile_sync_service.h
index f914eac83acdb69fe6f15864e0f1c20f91006b90..332a6477d10c4394302a04b16930c17819e5cb12 100644
--- a/components/sync/driver/profile_sync_service.h
+++ b/components/sync/driver/profile_sync_service.h
@@ -249,6 +249,7 @@ class ProfileSyncService : public SyncService,
 
   SyncClient* GetSyncClientForTest();
 
+ BRAVE_PROFILE_SYNC_SERVICE_H_
  private:
   // Passed as an argument to StopImpl to control whether or not the sync
   // engine should clear its data when it shuts down. See StopImpl for more
