diff --git a/components/search_engines/BUILD.gn b/components/search_engines/BUILD.gn
index dbfbd7eeb271a94b50572c01dd9531a1ed9047ac..2afa2d056228c19ce24cdc0af36cf0e04b1f64d2 100644
--- a/components/search_engines/BUILD.gn
+++ b/components/search_engines/BUILD.gn
@@ -203,6 +203,7 @@ json_to_struct("prepopulated_engines") {
   schema_file = "prepopulated_engines_schema.json"
   namespace = "TemplateURLPrepopulateData"
   deps = [ ":search_engine_type" ]
+  additional_sources = brave_components_search_engines_sources
 }
 
 if (is_android) {
