diff --git a/components/permissions/permission_request_manager.cc b/components/permissions/permission_request_manager.cc
index b58f4a34ba6484314ecedc9ad19254d2ffb3dd03..bc7b0c68b68c5ca02b86ea622ca262dc4fd9fd87 100644
--- a/components/permissions/permission_request_manager.cc
+++ b/components/permissions/permission_request_manager.cc
@@ -546,6 +546,7 @@ void PermissionRequestManager::OnVisibilityChanged(
     content::Visibility visibility) {
   bool tab_was_hidden = tab_is_hidden_;
   tab_is_hidden_ = visibility == content::Visibility::HIDDEN;
+  BRAVE_PERMISSION_REQUEST_MANAGER_ON_VISIBILITY_CHANGED
   if (tab_was_hidden == tab_is_hidden_) {
     return;
   }
@@ -597,6 +598,7 @@ GURL PermissionRequestManager::GetRequestingOrigin() const {
   GURL origin = requests_.front()->requesting_origin();
   if (DCHECK_IS_ON()) {
     for (const auto& request : requests_) {
+      BRAVE_PERMISSION_REQUEST_MANAGER_GET_REQUESTING_ORIGIN
       DCHECK_EQ(origin, request->requesting_origin());
     }
   }
