diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index 30d3301b4bd3e0ee3f466776c303e5ba87895755..1bf4798887cbd3c8393583b0c932c6488973a93b 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -378,6 +378,7 @@ AutocompleteController::AutocompleteController(
     if (provider_types & AutocompleteProvider::TYPE_SEARCH) {
       search_provider_ = new SearchProvider(provider_client_.get(), this);
       providers_.push_back(search_provider_.get());
+      BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
     }
     // Providers run in the order they're added.  Add `HistoryURLProvider` after
     // `SearchProvider` because:
@@ -953,6 +954,8 @@ void AutocompleteController::UpdateResult(
   metrics_.OnUpdateResult(last_result_for_logging,
                           result_.GetMatchDedupComparators());
 
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
+
   // Below are all annotations after the match list is ready.
 #if !BUILDFLAG(IS_IOS)
   // HistoryClusters is not enabled on iOS.
