diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index 0d791f4e9ddfbc46ad517f9543b6fbc5888bafab..ca63a1b6452de2c1910edc6970a0d2cd74d3b8e3 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -340,6 +340,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
