diff --git a/components/system_media_controls/linux/system_media_controls_linux.cc b/components/system_media_controls/linux/system_media_controls_linux.cc
index 29e75bd680f3528533271d1e8a0efbf8974da8d3..e841a3fb097eea29614892057ae201ad1aa39645 100644
--- a/components/system_media_controls/linux/system_media_controls_linux.cc
+++ b/components/system_media_controls/linux/system_media_controls_linux.cc
@@ -64,7 +64,7 @@ constexpr base::TimeDelta kUpdatePositionInterval = base::Milliseconds(100);
 const char kMprisAPINoTrackPath[] = "/org/mpris/MediaPlayer2/TrackList/NoTrack";
 
 const char kMprisAPICurrentTrackPathFormatString[] =
-    "/org/chromium/MediaPlayer2/TrackList/Track%s";
+    "/com/brave/MediaPlayer2/TrackList/Track%s";
 
 // Writes `bitmap` to a new temporary PNG file and returns a a pair of the file
 // path and a managed base::ScopedTempFile bound to this sequence.  This should
@@ -104,7 +104,7 @@ WriteBitmapToTmpFile(const SkBitmap& bitmap) {
 }  // namespace
 
 const char kMprisAPIServiceNameFormatString[] =
-    "org.mpris.MediaPlayer2.chromium.instance%i";
+    "org.mpris.MediaPlayer2.brave.instance%i";
 const char kMprisAPIObjectPath[] = "/org/mpris/MediaPlayer2";
 const char kMprisAPIInterfaceName[] = "org.mpris.MediaPlayer2";
 const char kMprisAPIPlayerInterfaceName[] = "org.mpris.MediaPlayer2.Player";
