diff --git a/content/browser/resources/gpu/info_view.ts b/content/browser/resources/gpu/info_view.ts
index a6d024ba04a95bbfdadb7dd536fc9287ae18aae9..5f139f9b983b3d273c3c2176a9c21b7fb018255d 100644
--- a/content/browser/resources/gpu/info_view.ts
+++ b/content/browser/resources/gpu/info_view.ts
@@ -537,6 +537,7 @@ export class InfoViewElement extends CustomElement {
           value: clientInfo.graphics_backend,
         },
         {description: 'Command Line', value: clientInfo.command_line},
+        {description: 'Executable path', value: clientInfo.executable_path || ''},
       ]);
     } else {
       sections.clientInfo.list.textContent = '... loading ...';
