diff --git a/content/browser/media/cdm_registry_impl.h b/content/browser/media/cdm_registry_impl.h
index 303f3c59b3201ab78c84ae3803b48b37d9f8c998..5dde1606f0dc421467144d813fc0f7b9431e3fe1 100644
--- a/content/browser/media/cdm_registry_impl.h
+++ b/content/browser/media/cdm_registry_impl.h
@@ -26,6 +26,11 @@ class CONTENT_EXPORT CdmRegistryImpl : public CdmRegistry {
   void RegisterCdm(const CdmInfo& info) override;
   const std::vector<CdmInfo>& GetAllRegisteredCdms() override;
 
+#if defined(BRAVE_CHROMIUM_BUILD) && defined(OS_LINUX)
+  const CdmInfo& cached_widevine_cdm_info() const;
+  void Init_ChromiumImpl();
+#endif
+
  private:
   friend class CdmRegistryImplTest;
 
