diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 3a5834f68dadd0de9071c8e7c7bd416d59d40d55..bb2e2a4a88dca2bfc78c156ddf6670a3394bde4a 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -949,6 +949,7 @@ void RenderThreadImpl::InitializeWebKit(mojo::BinderMap* binders) {
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
