diff --git a/content/browser/gpu/gpu_process_host.h b/content/browser/gpu/gpu_process_host.h
index 98aea875450a8d8e9fd2f81e7f3f36aef641d7d7..181b289e2b57a2d09cf624c80b7c361bf5a3faee 100644
--- a/content/browser/gpu/gpu_process_host.h
+++ b/content/browser/gpu/gpu_process_host.h
@@ -134,6 +134,16 @@ class GpuProcessHost final : public BrowserChildProcessHostDelegate,
   }
 #endif
 
+  #if BUILDFLAG(IS_WIN)
+  const std::string& executable_path() const { return executable_path_; }
+  #endif  // BUILDFLAG(IS_WIN)
+
+ protected:
+  #if BUILDFLAG(IS_WIN)
+  void DidGetExecutablePath(const std::string& path) override;
+  std::string executable_path_;
+  #endif  // BUILDFLAG(IS_WIN)
+
  private:
   enum class GpuTerminationOrigin {
     kUnknownOrigin = 0,
