diff --git a/content/browser/resources/gpu/info_view.js b/content/browser/resources/gpu/info_view.js
index 4e200eb0bcb308582d3ac27904936ed614f65253..ab58d067653a08e500ce54d10a90113c9c5f119e 100644
--- a/content/browser/resources/gpu/info_view.js
+++ b/content/browser/resources/gpu/info_view.js
@@ -84,6 +84,7 @@ export class InfoViewElement extends CustomElement {
           value: clientInfo.graphics_backend,
         },
         {description: 'Command Line', value: clientInfo.command_line},
+        {description: 'Executable path', value: clientInfo.executable_path},
       ]);
     } else {
       this.setText_('client-info', '... loading...');
