diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 7a02c260cc997340fadfe019a88cae13b4564a38..5da2b0ad1a40ddf3f568132f01124634abc37b75 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1658,6 +1658,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if defined(OS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
