diff --git a/content/browser/web_contents/web_contents_impl.cc b/content/browser/web_contents/web_contents_impl.cc
index 64bc568b6ee9ac47588358bc4120a3a5646a24a7..36d8575fc22174513613b72cbcc510276c91be11 100644
--- a/content/browser/web_contents/web_contents_impl.cc
+++ b/content/browser/web_contents/web_contents_impl.cc
@@ -5345,6 +5345,7 @@ FrameTree* WebContentsImpl::CreateNewWindow(
   // SiteInstance is created in a new BrowsingInstance.
   scoped_refptr<SiteInstance> site_instance;
   if (params.opener_suppressed) {
+    BRAVE_WEB_CONTENTS_IMPL_CREATE_NEW_WINDOW_OPENER_SUPPRESSED
     if (is_guest) {
       // For guests, noopener windows can be created in a new BrowsingInstance
       // as long as they preserve the guest's StoragePartition.
@@ -5442,7 +5443,7 @@ FrameTree* WebContentsImpl::CreateNewWindow(
 
   // Save the window for later if we're not suppressing the opener (since it
   // will be shown immediately).
-  if (!params.opener_suppressed) {
+  if (!params.opener_suppressed && !params.no_javascript_access) {
     if (!is_guest) {
       WebContentsView* new_view = new_contents_impl->view_.get();
 
