diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 775996a26f581c43b54d5834ae2cb37d7ea74af3..522aace4a0e4f8510a890f6444fed30e49885ff1 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -31,7 +31,7 @@ source_set("common") {
   # In addition, targets outside of the content component (shell and tests)
   # must not link to this because it will duplicate the code in the component
   # build.
-  visibility = [ "//content/*" ]
+  visibility = [ "//content/*", "//brave/content/*" ]
 
   sources = [
     "accessibility_messages.h",
