diff --git a/content/browser/web_contents/web_contents_impl.h b/content/browser/web_contents/web_contents_impl.h
index d79ef098c00aeda3dbd1a5a352d5fb522c5fdbd2..b851c2b6b41f0f79bcba339d361aba66de73671f 100644
--- a/content/browser/web_contents/web_contents_impl.h
+++ b/content/browser/web_contents/web_contents_impl.h
@@ -1693,6 +1693,8 @@ class CONTENT_EXPORT WebContentsImpl
   // changed.
   void OnCapturerCountChanged();
 
+  bool GetShouldDoLearningForTesting() override;
+
  private:
   using FrameTreeIterationCallback = base::FunctionRef<void(FrameTree&)>;
   using RenderViewHostIterationCallback =
