diff --git a/content/browser/webui/web_ui_impl.cc b/content/browser/webui/web_ui_impl.cc
index d8e7829bdf990d18f8dc39efb5e62a871ac1798a..8540dac31fd5a0724a90c9e0589e48c8608a1afc 100644
--- a/content/browser/webui/web_ui_impl.cc
+++ b/content/browser/webui/web_ui_impl.cc
@@ -74,6 +74,7 @@ WebUIImpl::WebUIImpl(WebContentsImpl* contents, RenderFrameHostImpl* frame_host)
              RenderFrameHostImpl::LifecycleStateImpl::kActive ||
          frame_host->lifecycle_state() ==
              RenderFrameHostImpl::LifecycleStateImpl::kSpeculative);
+  BRAVE_WEB_UI_IMPL
 }
 
 WebUIImpl::~WebUIImpl() {
