diff --git a/content/browser/browser_main_runner_impl.cc b/content/browser/browser_main_runner_impl.cc
index efda461212d08e1a3d766021d321e3dbbccb4669..aeb506f0fee13a335e1b0825a71f3e607c328497 100644
--- a/content/browser/browser_main_runner_impl.cc
+++ b/content/browser/browser_main_runner_impl.cc
@@ -16,6 +16,7 @@
 #include "base/time/time.h"
 #include "base/trace_event/heap_profiler_allocation_context_tracker.h"
 #include "base/trace_event/trace_event.h"
+#include "brave/browser/brave_browser_main_loop.h"
 #include "build/build_config.h"
 #include "components/tracing/common/trace_startup_config.h"
 #include "components/tracing/common/tracing_switches.h"
@@ -94,8 +95,8 @@ int BrowserMainRunnerImpl::Initialize(const MainFunctionParams& parameters) {
 
     gfx::InitializeFonts();
 
-    main_loop_.reset(
-        new BrowserMainLoop(parameters, std::move(scoped_execution_fence_)));
+    main_loop_.reset(new brave::BraveBrowserMainLoop(
+        parameters, std::move(scoped_execution_fence_)));
 
     main_loop_->Init();
 
