diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index d6be4532c620861d77c900ac3aa67db26dbdf6ba..e23bc4b3cc657f1c121f6f7b04052330c040cf21 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4787,6 +4787,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (web_view->MainFrame()->IsWebRemoteFrame())
+    BRAVE_RENDER_FRAME_IMPL_SHOULD_USE_USER_AGENT_OVERRIDE
     return false;
   const WebLocalFrame* main_frame = web_view->MainFrame()->ToWebLocalFrame();
 
