diff --git a/content/browser/media/media_web_contents_observer.cc b/content/browser/media/media_web_contents_observer.cc
index 8800516e7f5c65be740445cb1d324b49b029c798..47b41efa869361d37a8ea530478bd6497de38c98 100644
--- a/content/browser/media/media_web_contents_observer.cc
+++ b/content/browser/media/media_web_contents_observer.cc
@@ -577,6 +577,12 @@ void MediaWebContentsObserver::MediaPlayerObserverHostImpl::
       ->OnVideoVisibilityChanged(media_player_id_, meets_visibility_threshold);
 }
 
+void MediaWebContentsObserver::MediaPlayerObserverHostImpl::OnMediaLoaded(
+    const GURL& url) {
+  media_web_contents_observer_->session_controllers_manager()->OnMediaLoaded(
+      media_player_id_, url);
+}
+
 bool MediaWebContentsObserver::IsMediaPlayerRemoteAvailable(
     const MediaPlayerId& player_id) {
   return media_player_remotes_.contains(player_id);
