diff --git a/content/browser/gpu/gpu_internals_ui.cc b/content/browser/gpu/gpu_internals_ui.cc
index 6f43ee799d28d2876aee494e4d9cf51f80d42548..dc62d67fbba7834da891657ecea082b7296baf93 100644
--- a/content/browser/gpu/gpu_internals_ui.cc
+++ b/content/browser/gpu/gpu_internals_ui.cc
@@ -744,6 +744,7 @@ base::DictValue GpuMessageHandler::GetClientInfo() {
            std::string("Skia/" STRINGIZE(SK_MILESTONE) " " SKIA_COMMIT_HASH));
   dict.Set("revision_identifier", GPU_LISTS_VERSION);
 
+  BRAVE_GPU_MESSAGE_HANDLER_ON_REQUEST_CLIENT_INFO
   return dict;
 }
 
