diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 0738636a7462fd973c12d0e26e298c3b767f0c53..0727d334c8eb959d731f9189ab294e3dc0c3aa50 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
