diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 2413228f605776322caff19098c6b60183005df5..83fb6710c709af43a0081d3c21b71fd7169c12d5 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3657,6 +3657,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
