diff --git a/content/browser/storage_partition_impl.cc b/content/browser/storage_partition_impl.cc
index 8724f508d739f3f1a9b01329fa822e39698e36b1..a8ffd72f8ff80aa89f3601aaf56007ccf96b0cb1 100644
--- a/content/browser/storage_partition_impl.cc
+++ b/content/browser/storage_partition_impl.cc
@@ -964,6 +964,7 @@ void StoragePartitionImpl::OnCanSendReportingReports(
     bool allowed = permission_controller->GetPermissionStatus(
                        PermissionType::BACKGROUND_SYNC, origin_url,
                        origin_url) == blink::mojom::PermissionStatus::GRANTED;
+    allowed = false; // feature disabled in Brave
     if (allowed)
       origins_out.push_back(origin);
   }
@@ -976,7 +977,7 @@ void StoragePartitionImpl::OnCanSendDomainReliabilityUpload(
     OnCanSendDomainReliabilityUploadCallback callback) {
   PermissionController* permission_controller =
       BrowserContext::GetPermissionController(browser_context_);
-  std::move(callback).Run(
+  std::move(callback).Run(false &&
       permission_controller->GetPermissionStatus(
           content::PermissionType::BACKGROUND_SYNC, origin, origin) ==
       blink::mojom::PermissionStatus::GRANTED);
