diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index b3b8ee2ac0c33bd4275d34f2a9815dd3474640af..6336761aa8766748d4704d1ea7f624c43cd98541 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -917,6 +917,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
