diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 6db720bfafadb4a771bc4e506f31585291cf9cfe..a33db470ea88a25f8afdd6e1fa67f9dcffcd1256 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -70,6 +70,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
