diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 9340b34b4fd6d102870f27b1575e2d2c9d8d8e49..d6dfb6d4f6fcf121c3ecbfd303098f1beee09457 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3691,6 +3691,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
