diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 6aaca3b6bccf7a2e2525aec44f7e779406fa1730..24cf797fc5b61b2793368db9bbef9cd332b28f4d 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3660,6 +3660,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants:constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
