diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 8ec3d837d7b0215aec28efa96827ac0046d94eaf..e8ceb368d4e38cf292e3dcdecb0e7642c1068bb3 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -31,7 +31,7 @@ source_set("common") {
   # In addition, targets outside of the content component (shell and tests)
   # must not link to this because it will duplicate the code in the component
   # build.
-  visibility = [ "//content/*" ]
+  visibility = [ "//content/*", "//brave/content/*" ]
 
   sources = [
     "accessibility_messages.h",
