diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 62b68955246583e21aa3c049e9727d0a1366c0d5..55a23726ce1cfae14505e8eb58de869746cdfee5 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -2307,6 +2307,12 @@ jumbo_source_set("browser") {
     if (use_pangocairo) {
       sources += [ "renderer_host/pepper/pepper_truetype_font_list_pango.cc" ]
     }
+    if (brave_chromium_build) {
+      sources += [
+        "//brave/content/browser/renderer_host/brave_plugin_registry_impl.cc",
+        "//brave/content/browser/renderer_host/brave_plugin_registry_impl.h",
+      ]
+    }
     if (is_linux) {
       deps += [ "//components/services/font:ppapi_fontconfig_matching" ]
     }
