diff --git a/content/browser/resources/gpu/info_view.ts b/content/browser/resources/gpu/info_view.ts
index 04621af27197fa87ae0760480274a320057d3bfd..45638ca99b7a37d55fdf4cf2bf3040d46037add3 100644
--- a/content/browser/resources/gpu/info_view.ts
+++ b/content/browser/resources/gpu/info_view.ts
@@ -91,6 +91,7 @@ export class InfoViewElement extends CustomElement {
           value: clientInfo.graphics_backend,
         },
         {description: 'Command Line', value: clientInfo.command_line},
+        {description: 'Executable path', value: clientInfo.executable_path},
       ]);
     } else {
       this.setText_('client-info', '... loading...');
