diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index dc67a5f511601fe10ceec97edc8e900a4df2e895..3c66dd6f9309387c6d1ef6a7369f60ac36cdc9b3 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -32,7 +32,7 @@ source_set("common") {
   # In addition, targets outside of the content component (shell and tests)
   # must not link to this because it will duplicate the code in the component
   # build.
-  visibility = [ "//content/*" ]
+  visibility = [ "//content/*", "//brave/content/*" ]
 
   sources = [
     "accessibility_messages.h",
