diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index 55a2ea42966e7bca9495aabbc73183bfc7df9de2..5c1f00b8c4fa5dadddaeb209e30862b5205741cc 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4624,7 +4624,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (render_view_->GetWebView()->MainFrame()->IsWebRemoteFrame())
-    return false;
+    return true;
   WebLocalFrame* main_frame =
       render_view_->GetWebView()->MainFrame()->ToWebLocalFrame();
 
