diff --git a/content/browser/child_process_security_policy_impl.cc b/content/browser/child_process_security_policy_impl.cc
index 79b62092482102a41fe03f7a33b172ba85c97891..ab958521d94777b1c4704fd39ee09349d839fa36 100644
--- a/content/browser/child_process_security_policy_impl.cc
+++ b/content/browser/child_process_security_policy_impl.cc
@@ -773,6 +773,9 @@ void ChildProcessSecurityPolicyImpl::GrantWebUIBindings(int child_id,
 
   // Web UI bindings need the ability to request chrome: URLs.
   state->second->GrantRequestScheme(kChromeUIScheme);
+#if defined(BRAVE_CHROMIUM_BUILD)
+  state->second->GrantRequestScheme(kBraveUIScheme);
+#endif
 
   // Web UI pages can contain links to file:// URLs.
   state->second->GrantRequestScheme(url::kFileScheme);
