diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index ecf345a180bb9515d8fe5469d74faaef14330121..9ebb8dca1d86256cba8261f69fd51cde393940b5 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -1020,6 +1020,7 @@ void RenderThreadImpl::InitializeWebKit(mojo::BinderMap* binders) {
 }

 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
