diff --git a/content/common/user_agent.cc b/content/common/user_agent.cc
index 25facd610551cefa60a445152e72a9d519f007ca..59fd65496f3a88e3891eaecc9f4d077dc2e29112 100644
--- a/content/common/user_agent.cc
+++ b/content/common/user_agent.cc
@@ -311,6 +311,7 @@ std::string GetAndroidOSInfo(
 
   // Send information about the device.
   bool semicolon_inserted = false;
+  BRAVE_GET_ANDROID_OS_INFO
   if (include_android_model == IncludeAndroidModel::Include) {
     std::string android_device_name = BuildModelInfo();
     if (!android_device_name.empty()) {
