diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index e503df625f2b9319e80cadd22562215ab7985271..cf6587027f9ca9f2c97b18f2bc08665171a4315e 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3722,6 +3722,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants:constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
