diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index 0cec8faf6da8cf6c2a3e3bf8df1126be94865939..2f38edbbf93f86b38deadb0d858331f9eae4b093 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4514,6 +4514,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (web_view->MainFrame()->IsWebRemoteFrame())
+    BRAVE_RENDER_FRAME_IMPL_SHOULD_USE_USER_AGENT_OVERRIDE
     return false;
   const WebLocalFrame* main_frame = web_view->MainFrame()->ToWebLocalFrame();
 
