diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 5500a0cec01ec6b48c0f8faadf1528ccc9eea120..cdf81d97d478b12a76dc1920428be57c4a887530 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2079,6 +2079,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
