diff --git a/content/browser/webui/web_ui_data_source_impl.cc b/content/browser/webui/web_ui_data_source_impl.cc
index 6014c557b90abeadf619436a409e646b1943904d..7c4536493dd79896adc54693f26fb71be79c3f40 100644
--- a/content/browser/webui/web_ui_data_source_impl.cc
+++ b/content/browser/webui/web_ui_data_source_impl.cc
@@ -328,7 +328,7 @@ void WebUIDataSourceImpl::OverrideContentSecurityPolicy(
     network::mojom::CSPDirectiveName directive,
     const std::string& value) {
   CHECK(!resources_frozen_);
-  csp_overrides_.insert_or_assign(directive, value);
+  csp_overrides_.insert_or_assign(directive, GetBraveTrustedTypesValue(directive, value));
 }
 
 void WebUIDataSourceImpl::OverrideCrossOriginOpenerPolicy(
