diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index da33f8025c9e4c0a6c1e452acc5616aed4890986..6aa83998c82ca70b185cc22c3855e76f94ef78c9 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -824,6 +824,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
