diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 96e882ea83d47e24a19a290e41cf225665025e08..16e16ff329ddafd912af5385410696837bdcc2a5 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -761,6 +761,7 @@ source_set("browser_sources") {
   if (enable_video_effects) {
     public_deps += [ "//services/video_effects/public/mojom:mojom" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
