diff --git a/content/public/browser/url_data_source.cc b/content/public/browser/url_data_source.cc
index 406ecfd3f97e618a0392661549523872c0ee42c2..f544b17b8313a9047b4cd6685e31a016ada2ddd2 100644
--- a/content/public/browser/url_data_source.cc
+++ b/content/public/browser/url_data_source.cc
@@ -85,7 +85,7 @@ bool URLDataSource::ShouldAddContentSecurityPolicy() {
 std::string URLDataSource::GetContentSecurityPolicyScriptSrc() {
   // Note: Do not add 'unsafe-eval' here. Instead override CSP for the
   // specific pages that need it, see context http://crbug.com/525224.
-  return "script-src chrome://resources 'self';";
+  return "script-src chrome://resources chrome://brave-resources 'self';";
 }
 
 std::string URLDataSource::GetContentSecurityPolicyObjectSrc() {
