diff --git a/content/common/frame.mojom b/content/common/frame.mojom
index 15a83f61ed4e31ba34cbc19995cd9d68b1599f1d..12338a534c0267e4ca46416984599ea17894a754 100644
--- a/content/common/frame.mojom
+++ b/content/common/frame.mojom
@@ -601,6 +601,7 @@ struct CreateNewWindowParams {
   // Whether the opener will be suppressed in the new window, in which case
   // scripting the new window is not allowed.
   bool opener_suppressed;
+  bool no_javascript_access;
 
   // Whether new window is created for a form submission.
   bool is_form_submission;
