diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 42e8d9febf2404064d16d1db1790f50df6ae3ea6..bf68979045bed40fb342f00095eaaf7838eca893 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2050,6 +2050,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
