diff --git a/content/child/child_thread_impl.cc b/content/child/child_thread_impl.cc
index 08dc17945bd07faacb03379be716939bcd971359..924884655595cefb6e2f20a8b4a73b2d9e88a609 100644
--- a/content/child/child_thread_impl.cc
+++ b/content/child/child_thread_impl.cc
@@ -163,6 +163,9 @@ class SuicideOnChannelErrorFilter : public IPC::MessageFilter {
     __lsan_do_leak_check();
 #endif
 #else
+#if defined(OS_MACOSX)
+    CleanupTor();
+#endif // defined(OS_MACOSX)
     base::Process::TerminateCurrentProcessImmediately(0);
 #endif
   }
