diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 5a1b57da76eb52af8c36784afa6c793892d7534f..c1299fca44598c2dc65cc9f548ea2ff6812f35ba 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3695,6 +3695,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
