diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index fb23dfd8f90ad3eeb55e4db373ca67f75d8e5ce1..ac293c9ea059066b0594eee73ca1da14b48af06b 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -68,6 +68,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
