diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index c0a4466595dc5a7a345ebf87a820ef64a9eff895..861fb50473f7d41952a4f3b99efa73fb4cc00a76 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3698,6 +3698,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
