diff --git a/content/browser/gpu/gpu_process_host.cc b/content/browser/gpu/gpu_process_host.cc
index ab3f702bae17e5079ca4d398c978d24e66e5d9fe..cc3a05a0a9af79824f06204b840d0b02eaf40464 100644
--- a/content/browser/gpu/gpu_process_host.cc
+++ b/content/browser/gpu/gpu_process_host.cc
@@ -450,6 +450,8 @@ class GpuSandboxedProcessLauncherDelegate
 
     // Block this DLL even if it is not loaded by the browser process.
     config->AddDllToUnload(L"cmsetac.dll");
+    config->SetShouldPatchModuleFileName(base::FeatureList::IsEnabled(
+        sandbox::policy::features::kModuleFileNamePatch));
 
     return true;
   }
