diff --git a/content/browser/web_contents/web_contents_impl.cc b/content/browser/web_contents/web_contents_impl.cc
index 70215dfb8251bafe6d03214ea1ece0e5eb5a355b..620f2aa7857e615370baf5073148e4b407b77eed 100644
--- a/content/browser/web_contents/web_contents_impl.cc
+++ b/content/browser/web_contents/web_contents_impl.cc
@@ -2994,7 +2994,11 @@ int WebContentsImpl::GetVirtualKeyboardResizeHeight() {
 }
 
 bool WebContentsImpl::ShouldDoLearning() {
+  bool should_do_learning = [&]() -> bool {
   return !GetBrowserContext()->IsOffTheRecord();
+  }();
+  return should_do_learning &&
+         !GetContentClient()->browser()->IsWindowsRecallDisabled();
 }
 
 void WebContentsImpl::OnAudioStateChanged() {
@@ -5604,6 +5608,12 @@ FrameTree* WebContentsImpl::CreateNewWindow(
   // SiteInstance is created in a new BrowsingInstance.
   scoped_refptr<SiteInstance> site_instance;
   if (params.opener_suppressed) {
+    if (!is_guest &&
+        GetContentClient()->browser()->ShouldInheritStoragePartition(
+            partition_config)) {
+      site_instance = SiteInstance::CreateForFixedStoragePartition(
+          GetBrowserContext(), GURL(), partition_config);
+    } else /* NOLINT(readability/braces) */
     if (is_guest) {
       // For guests, noopener windows can be created in a new BrowsingInstance
       // as long as they preserve the guest's StoragePartition.
@@ -5701,7 +5711,7 @@ FrameTree* WebContentsImpl::CreateNewWindow(
 
   // Save the window for later if we're not suppressing the opener (since it
   // will be shown immediately).
-  if (!params.opener_suppressed) {
+  if (!params.opener_suppressed && !params.no_javascript_access) {
     if (!is_guest) {
       WebContentsView* new_view = new_contents_impl->view_.get();
 
