diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 2a38c8929258489e4dbe43c0fd6dfc3a96c08cff..d4fe96b05ae340c7bb975bf66e1c7153c8330ed5 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -67,6 +67,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
