diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 99c99d6ab2422ab24af0699cc99fe4b7a9d1a3a3..98b3f433627576237e81960fd91c67238f626c92 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -778,6 +778,7 @@ source_set("browser_sources") {
   if (use_clang_profiling_inside_sandbox) {
     sources += [ "profiling_utils.h" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
