diff --git a/content/browser/browser_main_loop.h b/content/browser/browser_main_loop.h
index 99e43f559d00bf575e78fd7285dbb66731179e0a..75e7edf97f2f7b756b3ef2ded5d87a7636508c24 100644
--- a/content/browser/browser_main_loop.h
+++ b/content/browser/browser_main_loop.h
@@ -137,6 +137,7 @@ class CONTENT_EXPORT BrowserMainLoop {
   void PreMainMessageLoopStart();
   void MainMessageLoopStart();
   void PostMainMessageLoopStart();
+  virtual
   void PreShutdown();
 
   // Create and start running the tasks we need to complete startup. Note that
