diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 7490e10b3741a0680b0fbe7b069bd5803dccfd42..2ee71a8ec6d392e1147a0da58e96f1ff6617c786 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -67,6 +67,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
