diff --git a/content/public/browser/web_contents_receiver_set.h b/content/public/browser/web_contents_receiver_set.h
index b0fa64b5e1642a596df697a4a2c8d772db4730e2..5f3e1dc086bc22f61990f7211439ceae6eb7d17a 100644
--- a/content/public/browser/web_contents_receiver_set.h
+++ b/content/public/browser/web_contents_receiver_set.h
@@ -53,6 +53,11 @@ namespace subresource_redirect {
 class SubresourceRedirectObserver;
 }
 
+class BraveDrmTabHelper;
+namespace brave_shields {
+class BraveShieldsWebContentsObserver;
+}
+
 namespace content {
 
 class ConversionHost;
@@ -138,6 +143,8 @@ class WebContentsFrameReceiverSetPassKey {
   friend class DisplayCutoutHostImpl;
   friend class ScreenOrientationProvider;
   friend class TestFrameInterfaceBinder;
+  friend class ::BraveDrmTabHelper;
+  friend class ::brave_shields::BraveShieldsWebContentsObserver;
   FRIEND_TEST_ALL_PREFIXES(WebContentsReceiverSetBrowserTest,
                            OverrideForTesting);
 };
