diff --git a/content/browser/frame_host/navigation_request.cc b/content/browser/frame_host/navigation_request.cc
index 41c09cc8e7cddef94ca7cb561ff0096d4cd0c5e1..9c7bd2ec2c69c67d7b421f002917a9c7e168f7e3 100644
--- a/content/browser/frame_host/navigation_request.cc
+++ b/content/browser/frame_host/navigation_request.cc
@@ -1724,6 +1724,12 @@ void NavigationRequest::OnStartChecksComplete(
   headers.MergeFrom(navigation_handle_->TakeModifiedRequestHeaders());
   begin_params_->headers = headers.ToString();
 
+  GetContentClient()->browser()->MaybeHideReferrer(browser_context,
+                                                   common_params_.url,
+                                                   top_document_url,
+                                                   frame_tree_node_->IsMainFrame(),
+                                                   &common_params_.referrer);
+
   loader_ = NavigationURLLoader::Create(
       browser_context->GetResourceContext(), partition,
       std::make_unique<NavigationRequestInfo>(
