diff --git a/content/common/frame.mojom b/content/common/frame.mojom
index 0dfcbbd0e3274d01d938f843530b61a52028771d..b5f387029948ec11cfc8317a544dbc2483704351 100644
--- a/content/common/frame.mojom
+++ b/content/common/frame.mojom
@@ -592,6 +592,7 @@ struct CreateNewWindowParams {
   // Whether the opener will be suppressed in the new window, in which case
   // scripting the new window is not allowed.
   bool opener_suppressed;
+  bool no_javascript_access;
 
   // Whether new window is created for a form submission.
   bool is_form_submission;
