diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 59b0db6fe97c077b598d484187e22b1512235d14..9d217ee6f0e1b2014cf4e98afa166adfe895879d 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -938,6 +938,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
