diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 93ee33b10227da37b9404865df15eae16de2f9a9..21f6bdd370b32017ec7bfb3da5f9975ab59a93f8 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3696,6 +3696,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
