diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 5d96364e36f9a4cf63b9e15c0b27c36faf58b8a8..b9747059446ca45c8456c4ff4d1ac0b7e903dcad 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3666,6 +3666,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
