diff --git a/content/public/browser/permission_type.h b/content/public/browser/permission_type.h
index 187712cd58149a068dc5c0d4c74564f9884c0152..e2c797d5235b6dac2351d5286a2c8c8697688b6a 100644
--- a/content/public/browser/permission_type.h
+++ b/content/public/browser/permission_type.h
@@ -47,6 +47,7 @@ enum class PermissionType {
   VR = 25,
   AR = 26,
 
+  AUTOPLAY,
   // Always keep this at the end.
   NUM,
 };
