diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 9b4c78c06f49ade480d956025e68cb7efac336db..c8a1bff60bf0a452e80b00dcb8cd3ba52c9fa3fc 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -67,6 +67,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
