diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 8cae82ed6e46fbea8eca170f2699122dd45b200b..5ae3b10d8926294160df0104e7eba1e53d130499 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3538,6 +3538,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
