diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index dab9185e742cf427295ea57cd85d6d83c7cc28c2..aa9756ebc23bdb106281bffafef4497d9dc2bccc 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4491,6 +4491,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (web_view->MainFrame()->IsWebRemoteFrame())
+    BRAVE_RENDER_FRAME_IMPL_SHOULD_USE_USER_AGENT_OVERRIDE
     return false;
   const WebLocalFrame* main_frame = web_view->MainFrame()->ToWebLocalFrame();
 
