diff --git a/content/browser/child_process_security_policy_impl.cc b/content/browser/child_process_security_policy_impl.cc
index b7a29716b8aae57e13a450159c946718914879a3..45eb249c11404f793abcdd5ea5a3da1e54ba54d8 100644
--- a/content/browser/child_process_security_policy_impl.cc
+++ b/content/browser/child_process_security_policy_impl.cc
@@ -837,6 +837,9 @@ void ChildProcessSecurityPolicyImpl::GrantWebUIBindings(int child_id,
 
   // Web UI bindings need the ability to request chrome: URLs.
   state->second->GrantRequestScheme(kChromeUIScheme);
+#if defined(BRAVE_CHROMIUM_BUILD)
+  state->second->GrantRequestScheme(kBraveUIScheme);
+#endif
 
   // Web UI pages can contain links to file:// URLs.
   state->second->GrantRequestScheme(url::kFileScheme);
