diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index c3e208bdf94dac33d96bbc551310ebc1c5316b02..6fbfcfcda85dd860ca0603ebf493a12656678814 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -750,6 +750,7 @@ source_set("browser_sources") {
   if (enable_video_effects) {
     public_deps += [ "//services/video_effects/public/mojom" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
