diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 7c72b996ace8b4d5bef9788158061168ed20d201..e6c3ed58c417ccea165a10a73d2a22f57a79b675 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -829,6 +829,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
