diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 54df36f673af12688e0575664c44c30be0cb6584..d7ab483ab86e25cca47fc65ba454b834de4223b1 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -71,6 +71,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
