diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 765c9d898b2ce44425a4dec668ae9d2077f70864..bf205320deaf5731dbda7058a0dd6f6973c05c06 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -2177,6 +2177,12 @@ jumbo_source_set("browser") {
       "//ppapi/proxy:ipc",
       "//ppapi/shared_impl",
     ]
+    if (brave_chromium_build) {
+      sources += [
+        "//brave/content/browser/renderer_host/brave_plugin_registry_impl.cc",
+        "//brave/content/browser/renderer_host/brave_plugin_registry_impl.h",
+      ]
+    }
     if (use_ozone) {
       sources += [ "renderer_host/pepper/pepper_truetype_font_list_ozone.cc" ]
     }
