diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index c652c7e71d39384c4161e6eaa35792baf192107a..660480139fe39d228ffe2a0f46a7a69667d49d20 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -37,6 +37,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
