diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 081f2a1f61f96154e5b41679369edc255aef7918..2c23e72b9c5f46f3ed471d7da8d84631d895c26a 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -758,6 +758,7 @@ source_set("browser_sources") {
   if (enable_video_effects) {
     public_deps += [ "//services/video_effects/public/mojom:mojom" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
