diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index bd83dfa0a32658c5b45ca001bfd04cc051d8bc73..48d08aefda3e696d897b8e73a4a853a52f88aa27 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -491,6 +491,7 @@ source_set("browser_sources") {
     "//ui/shell_dialogs",
     "//ui/surface",
   ]
+  deps += ["//brave/content/public/browser:browser",]
 
   allow_circular_includes_from = [
     # This target is a pair with content/browser. They always go together and
