diff --git a/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm b/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm
index 767c532efd6cf1fa39a33ed54da325ca1e9a5d95..6502d47d05fea49a32a2663dbd70e3661a6f4f18 100644
--- a/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm
+++ b/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm
@@ -1980,6 +1980,7 @@ void ExtractUnderlines(NSAttributedString* string,
 }
 
 - (id)accessibilityParent {
+  BRAVE_ACCESSIBILITY_PARENT
   if (_accessibilityParent)
     return NSAccessibilityUnignoredAncestor(_accessibilityParent);
   return [super accessibilityParent];
