diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index d6ad990189520f7fc15e04c1c4bd03588d3ee81e..b6ff0c62332235d9ea397e6a15e2af9be377b216 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1970,6 +1970,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
