diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index aca90ffd227b07c792d03ecdfebc577131c09533..db38d6f753731a6e1898b2226e53a262aa2e2d8d 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -70,6 +70,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
