diff --git a/content/browser/permissions/permission_service_impl.cc b/content/browser/permissions/permission_service_impl.cc
index c6345e0119f80319683267332f278cbb4a473c77..85088dadc171464ab413df53d5504bec929d5634 100644
--- a/content/browser/permissions/permission_service_impl.cc
+++ b/content/browser/permissions/permission_service_impl.cc
@@ -107,6 +107,9 @@ bool PermissionDescriptorToPermissionType(
         return true;
       }
       break;
+    case PermissionName::AUTOPLAY:
+      *permission_type  = PermissionType::AUTOPLAY;
+      return true;
   }
 
   NOTREACHED();
