diff --git a/content/browser/gpu/gpu_internals_ui.cc b/content/browser/gpu/gpu_internals_ui.cc
index 1c8b95a4b1530a8aff0b9387c83da05025a06025..d19b0bb42d695a19de89be9d78b89d3dcc716581 100644
--- a/content/browser/gpu/gpu_internals_ui.cc
+++ b/content/browser/gpu/gpu_internals_ui.cc
@@ -733,6 +733,7 @@ void GpuMessageHandler::HandleGetGpuInfo(const base::ListValue& args) {
 }
 
 base::DictValue GpuMessageHandler::GetClientInfo() {
+  base::DictValue dict = [&]() -> base::DictValue {
   CHECK_CURRENTLY_ON(BrowserThread::UI, base::NotFatalUntil::M159);
 
   base::DictValue dict;
@@ -773,6 +774,13 @@ base::DictValue GpuMessageHandler::GetClientInfo() {
   dict.Set("revision_identifier", GPU_LISTS_VERSION);
 
   return dict;
+  }();
+  #if BUILDFLAG(IS_WIN)
+  if (auto* host = GpuProcessHost::Get()) {
+    dict.Set("executable_path", host->executable_path());
+  }
+  #endif  // BUILDFLAG(IS_WIN)
+  return dict;
 }
 
 base::ListValue GpuMessageHandler::GetLogMessages() {
