diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 30870710ca42d8f709510ddea300536a9261fb49..0f71a2203d813e28775ebe7c7434e2e291330e47 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -758,6 +758,7 @@ source_set("browser_sources") {
   if (use_clang_profiling_inside_sandbox) {
     sources += [ "profiling_utils.h" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
