diff --git a/content/browser/find_request_manager_browsertest.cc b/content/browser/find_request_manager_browsertest.cc
index 5ed6a16141296c809b9131ee0dc49df420cb4298..f9954054273e8b5732db9e3fbd6ebb81bc9e91f8 100644
--- a/content/browser/find_request_manager_browsertest.cc
+++ b/content/browser/find_request_manager_browsertest.cc
@@ -176,6 +176,7 @@ class FindRequestManagerTestBase : public ContentBrowserTest {
   int last_request_id_;
 };
 
+//// TODO: REVISIT - This is very strange place. 👀
 class FindRequestManagerTest : public FindRequestManagerTestBase,
                                public testing::WithParamInterface<bool> {
  protected:
