diff --git a/content/child/child_thread_impl.cc b/content/child/child_thread_impl.cc
index adbdcf956f4f79d038b6127423843fcb778ebf0e..d59124d6a3ad051dcd8dca8f2a9d13524480b8b4 100644
--- a/content/child/child_thread_impl.cc
+++ b/content/child/child_thread_impl.cc
@@ -172,6 +172,9 @@ class SuicideOnChannelErrorFilter : public IPC::MessageFilter {
     __lsan_do_leak_check();
 #endif
 #else
+#if defined(OS_MACOSX)
+    CleanupTor();
+#endif // defined(OS_MACOSX)
     base::Process::TerminateCurrentProcessImmediately(0);
 #endif
   }
