diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 8f47984d72b93e5178ec93a68711f03d6eca3cdb..651c62a7a283a3fd97d656971777c4e083479d1d 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -37,6 +37,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
