diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 2ececf64b5dad823ccab6e59aba9021605bfa216..53caff14920f1aa073b6d64d7a5d0c696decea56 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -1910,6 +1910,7 @@ jumbo_source_set("browser") {
     "worker_host/worker_script_loader_factory.cc",
     "worker_host/worker_script_loader_factory.h",
   ]
+  if (brave_chromium_build) { deps += [ "//brave/browser:content_browser" ] }
 
   if (toolkit_views) {
     deps += [ "//ui/events" ]
