diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 11759a9fcbe02d60499e7d46d76a62de28002db2..f775ce4a962328ac6d4523a9b33a94d0c0eba2d5 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -956,6 +956,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
