diff --git a/content/browser/gpu/gpu_process_host.h b/content/browser/gpu/gpu_process_host.h
index 0388647bbc751d251b8c2f2bcca6f3f70158ed3a..23e4991e7da2a61997813779eee2fdc7db178d39 100644
--- a/content/browser/gpu/gpu_process_host.h
+++ b/content/browser/gpu/gpu_process_host.h
@@ -146,6 +146,16 @@ class GpuProcessHost final : public BrowserChildProcessHostDelegate,
   }
 #endif
 
+  #if BUILDFLAG(IS_WIN)
+  const std::string& executable_path() const { return executable_path_; }
+  #endif  // BUILDFLAG(IS_WIN)
+
+ protected:
+  #if BUILDFLAG(IS_WIN)
+  void DidGetExecutablePath(const std::string& path) override;
+  std::string executable_path_;
+  #endif  // BUILDFLAG(IS_WIN)
+
  private:
   enum class GpuTerminationOrigin {
     kUnknownOrigin = 0,
