diff --git a/content/browser/resources/gpu/info_view.ts b/content/browser/resources/gpu/info_view.ts
index efb0bba3e95f5bda08b68209fc7d4b0233327f9f..abb368f9d97677a6de0d8d4829470b5b34dc98ee 100644
--- a/content/browser/resources/gpu/info_view.ts
+++ b/content/browser/resources/gpu/info_view.ts
@@ -527,6 +527,7 @@ export class InfoViewElement extends CustomElement {
           value: clientInfo.graphics_backend,
         },
         {description: 'Command Line', value: clientInfo.command_line},
+        {description: 'Executable path', value: clientInfo.executable_path || ''},
       ]);
     } else {
       sections.clientInfo.list.textContent = '... loading ...';
