diff --git a/content/public/browser/browser_context.h b/content/public/browser/browser_context.h
index d2570b2329086933e5b0b74f3dbdbc5ac8854e29..c84f122219d2a5f5fa823844cc632cf58a6c4cca 100644
--- a/content/public/browser/browser_context.h
+++ b/content/public/browser/browser_context.h
@@ -126,6 +126,11 @@ class CONTENT_EXPORT BrowserContext : public base::SupportsUserData {
   static StoragePartition* GetStoragePartition(BrowserContext* browser_context,
                                                SiteInstance* site_instance,
                                                bool can_create = true);
+  static StoragePartition* GetStoragePartitionForSite(
+      BrowserContext* browser_context,
+      const GURL& site,
+      const GURL& first_party,
+      bool can_create = true);
   static StoragePartition* GetStoragePartitionForSite(
       BrowserContext* browser_context,
       const GURL& site,
@@ -251,6 +256,9 @@ class CONTENT_EXPORT BrowserContext : public base::SupportsUserData {
   // also off the record.
   virtual bool IsOffTheRecord() const = 0;
 
+  // Return whether this context uses Tor. Default is false.
+  virtual bool IsTorProfile() const;
+
   // Returns the resource context.
   virtual ResourceContext* GetResourceContext() = 0;
 
