diff --git a/content/public/browser/navigation_handle.h b/content/public/browser/navigation_handle.h
index ec647ee9ba22d4b7bdd9379ed85fe527092f1ad7..14a193322ce7cb6c10d328d6589ca719a668be30 100644
--- a/content/public/browser/navigation_handle.h
+++ b/content/public/browser/navigation_handle.h
@@ -977,6 +977,12 @@ class CONTENT_EXPORT NavigationHandle : public base::SupportsUserData {
   // Returns true if the next redirect should bypass redirect checks, and resets
   // the flag to false.
   virtual bool ConsumeBypassRedirectChecksForNextRedirect() = 0;
+
+  // Returns true if the next redirect should bypass redirect checks,
+  // leaving the flag set for `ConsumeBypassRedirectChecksForNextRedirect`.
+  static bool HasBypassRedirectChecksForNextRedirect(
+      FrameTreeNodeId frame_tree_node_id,
+      int64_t navigation_id);
 };
 
 }  // namespace content
