diff --git a/content/public/browser/browser_context.h b/content/public/browser/browser_context.h
index 0a6913f2ca560e318f4aebe40189144bcef1bf4f..b4bbbeb04506e17402e3447d11f3879d575029cf 100644
--- a/content/public/browser/browser_context.h
+++ b/content/public/browser/browser_context.h
@@ -249,6 +249,9 @@ class CONTENT_EXPORT BrowserContext : public base::SupportsUserData {
   // Return whether this context is incognito. Default is false.
   virtual bool IsOffTheRecord() const = 0;
 
+  // Return whether this context uses Tor. Default is false.
+  virtual bool IsTorProfile() const;
+
   // Returns the resource context.
   virtual ResourceContext* GetResourceContext() = 0;
 
