diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index da4a8410a6c5d2eb7e35eb10b1629dca8d4011e1..0c8fbbb55d3b9562960aa926f0782293a118de88 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -887,6 +887,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
