diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 44d963793f143530704a0a06273e51d7c68766cc..3e39b6da2722a77df31509cd2b13602002663649 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -1233,6 +1233,14 @@ void RenderThreadImpl::InitializeWebKit(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+#if defined(BRAVE_CHROMIUM_BUILD)
+  // brave:
+  WebString brave_scheme(WebString::FromASCII(kBraveUIScheme));
+  WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(brave_scheme);
+  WebSecurityPolicy::RegisterURLSchemeAsNotAllowingJavascriptURLs(
+      brave_scheme);
+#endif
+
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
