diff --git a/content/browser/renderer_host/clipboard_host_impl.cc b/content/browser/renderer_host/clipboard_host_impl.cc
index 0def58240ad672d089f66772a18c7fa8ebfe709d..7c28fd343a252e4cc19890e3f0cf2e1c5af44516 100644
--- a/content/browser/renderer_host/clipboard_host_impl.cc
+++ b/content/browser/renderer_host/clipboard_host_impl.cc
@@ -250,6 +250,10 @@ class ClipboardHostDocumentHelper
   }
 #endif
 
+  void SanitizeOnNextWriteText() override {
+    impl_->SanitizeOnNextWriteText();
+  }
+
  private:
   std::unique_ptr<ClipboardHostImpl> impl_;
 };
@@ -787,6 +791,9 @@ void ClipboardHostImpl::WriteText(const std::u16string& text) {
   ClipboardPasteData data;
   data.text = text;
   ++pending_writes_;
+  MaybeSanitizeClipboardText(
+      &sanitize_on_next_write_text_, GetContentClient()->browser(),
+      render_frame_host_->GetMainFrame(), &data.text);
   GetContentClient()->browser()->IsClipboardCopyAllowedByPolicy(
       context_->CreateClipboardEndpoint(),
       {
