diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 20b991b675d6767c4d9903b9d70575b12700a2cd..5e82fb4f98caadcd661502c627831a413b2cfde0 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3690,6 +3690,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants:constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
