diff --git a/content/public/browser/navigation_handle.h b/content/public/browser/navigation_handle.h
index 52dcbe87c75cbd623a41c0038dfcb103d17d4ca0..c2f22edac5ec37367eee627fc3dcea99f44aabf0 100644
--- a/content/public/browser/navigation_handle.h
+++ b/content/public/browser/navigation_handle.h
@@ -980,6 +980,12 @@ class CONTENT_EXPORT NavigationHandle : public base::SupportsUserData {
   // Returns true if the next redirect should bypass redirect checks, and resets
   // the flag to false.
   virtual bool ConsumeBypassRedirectChecksForNextRedirect() = 0;
+
+  // Returns true if the next redirect should bypass redirect checks,
+  // leaving the flag set for `ConsumeBypassRedirectChecksForNextRedirect`.
+  static bool HasBypassRedirectChecksForNextRedirect(
+      FrameTreeNodeId frame_tree_node_id,
+      int64_t navigation_id);
 };
 
 }  // namespace content
