diff --git a/content/browser/worker_host/shared_worker_host.h b/content/browser/worker_host/shared_worker_host.h
index 6e1930d3ed50692ce5a0daea77648aea0b52e0b0..da4a49057e4dd3fb8a17cc47d4e50d3669fb48a7 100644
--- a/content/browser/worker_host/shared_worker_host.h
+++ b/content/browser/worker_host/shared_worker_host.h
@@ -197,6 +197,7 @@ class CONTENT_EXPORT SharedWorkerHost : public blink::mojom::SharedWorkerHost,
   // Creates a network factory params for subresource requests from this worker.
   network::mojom::URLLoaderFactoryParamsPtr
   CreateNetworkFactoryParamsForSubresources();
+  BRAVE_SHARED_WORKER_HOST_H
 
  private:
   friend class SharedWorkerHostTest;
