diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index 2ab5e79b71cc8d5fa5c5690da0ac4fe3bf9a0def..d10dc535d38bf24cd35096b6fa0014fbe06cdcd1 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -2198,6 +2198,10 @@ bool RenderFrameImpl::OnMessageReceived(const IPC::Message& msg) {
 #if BUILDFLAG(ENABLE_PLUGINS)
     IPC_MESSAGE_HANDLER(FrameMsg_SetPepperVolume, OnSetPepperVolume)
 #endif
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+    IPC_MESSAGE_HANDLER(FrameMsg_RegisterResourceBlockAd, OnRegisterResourceBlockAd)
+    IPC_MESSAGE_HANDLER(FrameMsg_RegisterResourceBlockTracker, OnRegisterResourceBlockTracker)
+// #endif
     IPC_MESSAGE_HANDLER(FrameMsg_VisualStateRequest,
                         OnVisualStateRequest)
     IPC_MESSAGE_HANDLER(FrameMsg_Reload, OnReload)
@@ -2472,6 +2476,20 @@ void RenderFrameImpl::OnClipboardHostError() {
 }
 #endif
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+void RenderFrameImpl::OnRegisterResourceBlockAd(
+    const GURL& url,
+    const std::string& rule) {
+  RegisterResourceBlockAd(url, rule);
+}
+
+void RenderFrameImpl::OnRegisterResourceBlockTracker(
+    const GURL& url,
+    const std::string& host) {
+  RegisterResourceBlockTracker(url, host);
+}
+// #endif
+
 void RenderFrameImpl::JavaScriptExecuteRequest(
     const base::string16& javascript,
     bool wants_result,
@@ -2973,6 +2991,20 @@ void RenderFrameImpl::AddMessageToConsole(
   AddMessageToConsoleImpl(level, message, false /* discard_duplicates */);
 }
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+void RenderFrameImpl::RegisterResourceBlockAd(
+    const GURL& url,
+    const std::string& rule) {
+  frame_->GetDocument().RegisterResourceBlockAd(url, rule);
+}
+
+void RenderFrameImpl::RegisterResourceBlockTracker(
+    const GURL& url,
+    const std::string& host) {
+  frame_->GetDocument().RegisterResourceBlockTracker(url, host);
+}
+// #endif
+
 PreviewsState RenderFrameImpl::GetPreviewsState() {
   WebDocumentLoader* document_loader = frame_->GetDocumentLoader();
   return document_loader ? document_loader->GetPreviewsState()
