diff --git a/content/common/user_agent.cc b/content/common/user_agent.cc
index 761d0650f514b71c7a0de32254ac39dc711253e1..9d0433ca4bbadab20ca6a56fc1621f6bc8e07315 100644
--- a/content/common/user_agent.cc
+++ b/content/common/user_agent.cc
@@ -390,6 +390,7 @@ std::string GetAndroidOSInfo(
 
   // Send information about the device.
   bool semicolon_inserted = false;
+  BRAVE_GET_ANDROID_OS_INFO
   if (include_android_model == IncludeAndroidModel::Include) {
     std::string android_device_name = BuildModelInfo();
     if (!android_device_name.empty()) {
