diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 22a541257d76322732747cfd4c781fcef2261897..b7c09a31b07440bc5ced163774629b07386b6062 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -67,6 +67,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
