diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index ce69e43769b267ba0e96a7006fa4352639b49a1e..20843bac0b47ceff80c329207c5c9c49f034b559 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1774,6 +1774,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if defined(OS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
