diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 9fe4f5cf47fe860688b150290672ae5fc44170d9..484f7aa9af350aeb86c64686b50fe7c6bdc0862e 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3533,6 +3533,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
