diff --git a/content/public/browser/web_contents_receiver_set.h b/content/public/browser/web_contents_receiver_set.h
index 725e64c675465d2beb6a3db4763230d75bf7bb93..743cdc5db25b817a37faac5be026472972520bb5 100644
--- a/content/public/browser/web_contents_receiver_set.h
+++ b/content/public/browser/web_contents_receiver_set.h
@@ -55,6 +55,11 @@ namespace subresource_redirect {
 class SubresourceRedirectObserver;
 }
 
+class BraveDrmTabHelper;
+namespace brave_shields {
+class BraveShieldsWebContentsObserver;
+}
+
 namespace content {
 
 class ConversionHost;
@@ -142,6 +147,8 @@ class WebContentsFrameReceiverSetPassKey {
   friend class DisplayCutoutHostImpl;
   friend class ScreenOrientationProvider;
   friend class TestFrameInterfaceBinder;
+  friend class ::BraveDrmTabHelper;
+  friend class ::brave_shields::BraveShieldsWebContentsObserver;
   FRIEND_TEST_ALL_PREFIXES(WebContentsReceiverSetBrowserTest,
                            OverrideForTesting);
 };
