diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 39174b67d1d3613a8cd7a02cc49d911ba8686a40..7199c240ebf763640b93e87671568a238af60718 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1554,6 +1554,13 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  // Brave-speicific: allows the embedder to modify the referrer string
+  // according to user preferences.
+  virtual void MaybeHideReferrer(BrowserContext* browser_context,
+                                 const GURL& request_url,
+                                 const GURL& document_url,
+                                 Referrer* referrer) {}
+
 #if defined(OS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
