diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index ea493573a2c8c149f8dd944f8ca4edaa44e675ca..b1971aaca7f65df162c7828ee787e528d4607e40 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -68,6 +68,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
