diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index 956085eaaf10e59a6b229fc37c17b4903e35ff20..e49a0f19b3b75a8b80b15cb17aea7dd7f5d84ca9 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -40,6 +40,7 @@
 #include "base/threading/thread_task_runner_handle.h"
 #include "base/time/time.h"
 #include "base/trace_event/trace_event.h"
+#include "brave/content/renderer/brave_render_frame_impl.h"
 #include "build/build_config.h"
 #include "cc/base/switches.h"
 #include "content/common/accessibility_messages.h"
@@ -1049,7 +1050,7 @@ RenderFrameImpl* RenderFrameImpl::Create(
   if (g_create_render_frame_impl)
     return g_create_render_frame_impl(std::move(params));
   else
-    return new RenderFrameImpl(std::move(params));
+    return new BraveRenderFrameImpl(std::move(params));
 }
 
 // static
