diff --git a/content/browser/renderer_host/navigation_controller_impl.cc b/content/browser/renderer_host/navigation_controller_impl.cc
index 87a0dd9f4584b3518292e59d87766d396abd13a6..816b115718eff7aab775f8869cd91a6ee4c81221 100644
--- a/content/browser/renderer_host/navigation_controller_impl.cc
+++ b/content/browser/renderer_host/navigation_controller_impl.cc
@@ -455,6 +455,7 @@ void RewriteUrlForNavigation(const GURL& original_url,
   *url_to_load = *virtual_url = original_url;
   BrowserURLHandlerImpl::GetInstance()->RewriteURLIfNecessary(
       url_to_load, browser_context, reverse_on_redirect);
+  MaybeRewriteVirtualURL(virtual_url);
 }
 
 #if DCHECK_IS_ON()
