diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 24e6a9599f178858cb39d3bb2a78764d97c538bd..158ec2a2dd4ee4456122d50c7b53ea3df630adeb 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2071,6 +2071,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
