diff --git a/content/browser/web_contents/web_contents_impl.h b/content/browser/web_contents/web_contents_impl.h
index c2c99afba11dad48ba0731c22e052fc20544b41c..f6e0131dc823f5fa502295368761fe9408fd16c1 100644
--- a/content/browser/web_contents/web_contents_impl.h
+++ b/content/browser/web_contents/web_contents_impl.h
@@ -1711,6 +1711,8 @@ class CONTENT_EXPORT WebContentsImpl
   // base::trace_event::TraceSessionObserver implementation:
   void OnStart(const perfetto::DataSourceBase::StartArgs&) override;
 
+  bool GetShouldDoLearningForTesting() override;
+
  private:
   using FrameTreeIterationCallback = base::FunctionRef<void(FrameTree&)>;
   using RenderViewHostIterationCallback =
