diff --git a/content/renderer/render_frame_impl.h b/content/renderer/render_frame_impl.h
index d841189173ce88675fece0fface17d990229cfea..0255f76ad52664f0819c05dd80ee8263911ef2b1 100644
--- a/content/renderer/render_frame_impl.h
+++ b/content/renderer/render_frame_impl.h
@@ -875,6 +875,7 @@ class CONTENT_EXPORT RenderFrameImpl
   explicit RenderFrameImpl(CreateParams params);
 
  private:
+  friend class BraveRenderFrameImpl;
   friend class RenderFrameImplTest;
   friend class RenderFrameObserver;
   friend class RenderAccessibilityImplTest;
