diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index f848ce09f7565249f0c075c8ea8d7a6178f81332..81d04c5a6582332b52e1a93feb471c3e707213b9 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3627,6 +3627,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
