diff --git a/content/browser/hid/hid_service.cc b/content/browser/hid/hid_service.cc
index e2edcbceed244e422151f69d5fe08b50eb495abc..8d5c3a05f1feb9f0350514dd2ce8c7dfc6f97aec 100644
--- a/content/browser/hid/hid_service.cc
+++ b/content/browser/hid/hid_service.cc
@@ -446,11 +446,13 @@ void HidService::RequestDevice(
   // activation to prevent stale/pending-deletion frames from opening choosers
   // or consuming user gestures from newly committed documents.
   if (!render_frame_host_ || !render_frame_host_->IsActive() ||
-      !FrameTreeNode::From(render_frame_host_)
+      (!delegate->AllowRequestDeviceWithoutTransientActivation(
+           render_frame_host_) &&
+       !FrameTreeNode::From(render_frame_host_)
            ->UpdateUserActivationState(
                blink::mojom::UserActivationUpdateType::
                    kConsumeTransientActivation,
-               blink::mojom::UserActivationNotificationType::kNone)) {
+               blink::mojom::UserActivationNotificationType::kNone))) {
     std::move(callback).Run(std::vector<device::mojom::HidDeviceInfoPtr>());
     return;
   }
