diff --git a/content/child/child_thread_impl.cc b/content/child/child_thread_impl.cc
index 36574e68cb09076492902dd0a6e399472d568f7c..ff522b4319e6f1e19562f3ba8e0a2aedf9cf44fa 100644
--- a/content/child/child_thread_impl.cc
+++ b/content/child/child_thread_impl.cc
@@ -180,6 +180,9 @@ class SuicideOnChannelErrorFilter : public IPC::MessageFilter {
     __lsan_do_leak_check();
 #endif
 #else
+#if defined(OS_MACOSX)
+    CleanupTor();
+#endif // defined(OS_MACOSX)
     base::Process::TerminateCurrentProcessImmediately(0);
 #endif
   }
