diff --git a/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm b/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm
index 4b9bdb8e9e9ca116703461fa0917673acec144cf..8eceea5dbbe68d59241055d2393b7c93333d0098 100644
--- a/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm
+++ b/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm
@@ -2003,6 +2003,7 @@ void ExtractUnderlines(NSAttributedString* string,
 }
 
 - (id)accessibilityParent {
+  BRAVE_ACCESSIBILITY_PARENT
   if (_accessibilityParent)
     return NSAccessibilityUnignoredAncestor(_accessibilityParent);
   return [super accessibilityParent];
