diff --git a/content/browser/gpu/gpu_internals_ui.cc b/content/browser/gpu/gpu_internals_ui.cc
index a9544c8ef69bb9587e3f01de54016e4cc709e2da..55604b10b9b30a0a839d560d75822f2fdcb4ea87 100644
--- a/content/browser/gpu/gpu_internals_ui.cc
+++ b/content/browser/gpu/gpu_internals_ui.cc
@@ -723,6 +723,7 @@ void GpuMessageHandler::HandleGetGpuInfo(const base::ListValue& args) {
 }
 
 base::DictValue GpuMessageHandler::GetClientInfo() {
+  base::DictValue dict = [&]() -> base::DictValue {
   CHECK_CURRENTLY_ON(BrowserThread::UI, base::NotFatalUntil::M159);
 
   base::DictValue dict;
@@ -763,6 +764,13 @@ base::DictValue GpuMessageHandler::GetClientInfo() {
   dict.Set("revision_identifier", GPU_LISTS_VERSION);
 
   return dict;
+  }();
+  #if BUILDFLAG(IS_WIN)
+  if (auto* host = GpuProcessHost::Get()) {
+    dict.Set("executable_path", host->executable_path());
+  }
+  #endif  // BUILDFLAG(IS_WIN)
+  return dict;
 }
 
 base::ListValue GpuMessageHandler::GetLogMessages() {
