diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index d42e689b0827e67b91abb12ef7dbc6844bf32239..5663ae096d8dea2d755e6ceaade5ebc400a687ba 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -1033,6 +1033,14 @@ void RenderThreadImpl::InitializeWebKit(mojo::BinderMap* binders) {
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+#if defined(BRAVE_CHROMIUM_BUILD)
+  // brave:
+  WebString brave_scheme(WebString::FromASCII(kBraveUIScheme));
+  WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(brave_scheme);
+  WebSecurityPolicy::RegisterURLSchemeAsNotAllowingJavascriptURLs(
+      brave_scheme);
+#endif
+
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
