diff --git a/content/child/child_thread_impl.cc b/content/child/child_thread_impl.cc
index 4aec5b12802a7347695826235870cbc655c57cdf..31a4bc1e4ec8df5617f09bac9f7a8e0b56da6c31 100644
--- a/content/child/child_thread_impl.cc
+++ b/content/child/child_thread_impl.cc
@@ -180,6 +180,9 @@ class SuicideOnChannelErrorFilter : public IPC::MessageFilter {
     __lsan_do_leak_check();
 #endif
 #else
+#if defined(OS_MACOSX)
+    CleanupTor();
+#endif // defined(OS_MACOSX)
     base::Process::TerminateCurrentProcessImmediately(0);
 #endif
   }
