diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index b374d6b28aea8103000e3a9a13bd17ce4a550e1b..4235f4eecde126c79b3469e95a50ced10bddfbf5 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2016,6 +2016,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
