diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index f6e7f72966f7bb744da4b2df999020250b5c0dc3..695786bb4575ccf52001f0edfd1e80a5282cf893 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -752,6 +752,7 @@ source_set("browser_sources") {
   if (enable_video_effects) {
     public_deps += [ "//services/video_effects/public/mojom:mojom" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
