diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 1c7ac93d0629cd20fef14db85dc222ab2d6377a1..3f4702a140f7efdeb11786250ce090e12c2475ce 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3725,6 +3725,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
