diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index e2cb076d3a690bf8ef1978b1f0bb935f2ddcb100..9f8bb6dce985cff3a562090c3f4eb2af3351880c 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -1222,6 +1222,14 @@ void RenderThreadImpl::InitializeWebKit(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+#if defined(BRAVE_CHROMIUM_BUILD)
+  // brave:
+  WebString brave_scheme(WebString::FromASCII(kBraveUIScheme));
+  WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(brave_scheme);
+  WebSecurityPolicy::RegisterURLSchemeAsNotAllowingJavascriptURLs(
+      brave_scheme);
+#endif
+
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
