diff --git a/content/browser/permissions/permission_controller_impl.cc b/content/browser/permissions/permission_controller_impl.cc
index 259186652ba17b85c65657865ad78a6ad997d00c..a8405e5951fc819b514788875e9084ea7625ddb7 100644
--- a/content/browser/permissions/permission_controller_impl.cc
+++ b/content/browser/permissions/permission_controller_impl.cc
@@ -67,6 +67,7 @@ PermissionToSchedulingFeature(PermissionType permission_name) {
     case PermissionType::CAMERA_PAN_TILT_ZOOM:
     case PermissionType::WINDOW_PLACEMENT:
     case PermissionType::FONT_ACCESS:
+    BRAVE_PERMISSION_TYPES
       return base::nullopt;
   }
 }
