diff --git a/content/browser/web_contents/web_contents_impl.h b/content/browser/web_contents/web_contents_impl.h
index faaa0345e58baadb8d94a11cf9e28bd11adc0536..04d4c27545fb62244e3ab74c15048a4b124c14f5 100644
--- a/content/browser/web_contents/web_contents_impl.h
+++ b/content/browser/web_contents/web_contents_impl.h
@@ -13,6 +13,7 @@
 #include <optional>
 #include <set>
 #include <string>
+#include <tuple>
 #include <utility>
 #include <vector>
 
@@ -51,6 +52,7 @@
 #include "content/common/content_export.h"
 #include "content/public/browser/fullscreen_types.h"
 #include "content/public/browser/global_routing_id.h"
+#include "content/public/browser/media_player_id.h"
 #include "content/public/browser/media_stream_request.h"
 #include "content/public/browser/mhtml_generation_result.h"
 #include "content/public/browser/web_contents.h"
@@ -938,6 +940,9 @@ class CONTENT_EXPORT WebContentsImpl : public WebContents,
       blink::mojom::MediaStreamType type,
       base::OnceCallback<void(std::vector<std::string>)> callback) override;
 
+  base::flat_map<MediaPlayerId, std::tuple<GURL, bool, double>>
+  GetMediaMetadataByMediaPlayerIds() const override;
+
   // NavigatorDelegate ---------------------------------------------------------
 
   void DidStartNavigation(NavigationHandle* navigation_handle) override;
@@ -1194,6 +1199,10 @@ class CONTENT_EXPORT WebContentsImpl : public WebContents,
     return media_web_contents_observer_.get();
   }
 
+  const MediaWebContentsObserver* media_web_contents_observer() const {
+    return media_web_contents_observer_.get();
+  }
+
   // Update the web contents visibility.
   void UpdateWebContentsVisibility(Visibility visibility) override;
 
