diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index 0755a49ee7558ee0c5e76f602c514bb8a7c3a019..ba94946864e9486453281fd3bfe6e0237dc555ea 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4853,6 +4853,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (web_view->MainFrame()->IsWebRemoteFrame())
+    BRAVE_RENDER_FRAME_IMPL_SHOULD_USE_USER_AGENT_OVERRIDE
     return false;
   const WebLocalFrame* main_frame = web_view->MainFrame()->ToWebLocalFrame();
 
