diff --git a/content/browser/web_contents/web_contents_impl.h b/content/browser/web_contents/web_contents_impl.h
index 58d38d13ab2f9efc7587fd93d2ae87432d00886b..4e8b0cac98b97ffe118827a7c338bee67b1cee1f 100644
--- a/content/browser/web_contents/web_contents_impl.h
+++ b/content/browser/web_contents/web_contents_impl.h
@@ -1711,6 +1711,8 @@ class CONTENT_EXPORT WebContentsImpl
   // base::trace_event::TraceSessionObserver implementation:
   void OnStart(const perfetto::DataSourceBase::StartArgs&) override;
 
+  bool GetShouldDoLearningForTesting() override;
+
  private:
   using FrameTreeIterationCallback = base::FunctionRef<void(FrameTree&)>;
   using RenderViewHostIterationCallback =
