diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index 4473c5e812a4a598f3e2f2bb06f78def5791af24..73b4f58de6b4e79f520c90e33f39d081fa7df113 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4516,6 +4516,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (web_view->MainFrame()->IsWebRemoteFrame())
+    BRAVE_RENDER_FRAME_IMPL_SHOULD_USE_USER_AGENT_OVERRIDE
     return false;
   const WebLocalFrame* main_frame = web_view->MainFrame()->ToWebLocalFrame();
 
