diff --git a/content/browser/resources/gpu/info_view.ts b/content/browser/resources/gpu/info_view.ts
index b61ac72d761f07d9f5540b841e6f51cee880be6a..0df75f44e2eb34e512b94858b8543b375ef8f2b8 100644
--- a/content/browser/resources/gpu/info_view.ts
+++ b/content/browser/resources/gpu/info_view.ts
@@ -523,6 +523,7 @@ export class InfoViewElement extends CustomElement {
           value: clientInfo.graphics_backend,
         },
         {description: 'Command Line', value: clientInfo.command_line},
+        {description: 'Executable path', value: clientInfo.executable_path || ''},
       ]);
     } else {
       sections.clientInfo.list.textContent = '... loading ...';
