diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index ca9b7cf53f061e3ee28051727615e915d13711e8..399653d23202e4c589a85b2868ee1d7a074bf510 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -749,6 +749,7 @@ source_set("browser_sources") {
   if (enable_video_effects) {
     public_deps += [ "//services/video_effects/public/mojom:mojom" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
