diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 66cbd429bec0740321ee12b0a5357c643e390496..a33283d5506d76dab601239f23a4ec6338fcecfe 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3669,6 +3669,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
