diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index a9111c0978542369298b17318c2a34bf20012328..e8d656dd03dc1da80ab1a53dac3963fc72fe6159 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -42,6 +42,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
