diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 4d443ca6bf205a4e39a62b09071916a1eddf3a96..500099ac8c8bad834abd6d2bf9ec174ba9ab3858 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -985,6 +985,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
