diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 993d628236d179be8ca34b9ccf48929ea051db5a..1b6aed19a06c9b1296fb68d0ca947e24c5a6ebed 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -878,6 +878,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
