diff --git a/content/common/user_agent.cc b/content/common/user_agent.cc
index bf5418337713ddd32cadbb0fcaba3179f8d70130..8a9fa41a2f976644ec91240c707fe6df87002a93 100644
--- a/content/common/user_agent.cc
+++ b/content/common/user_agent.cc
@@ -313,6 +313,7 @@ std::string GetAndroidOSInfo(
 
   // Send information about the device.
   bool semicolon_inserted = false;
+  BRAVE_GET_ANDROID_OS_INFO
   if (include_android_model == IncludeAndroidModel::Include) {
     std::string android_device_name = BuildModelInfo();
     if (!android_device_name.empty()) {
