diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 347526056c09a34c62512869b7c21b9e2b6ea0d7..a9cef43e1c7064419f963356e3d610c295d47528 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -986,6 +986,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
