diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index fe4075d91dffcc72d9f1a2cb43a249942a3c6aee..ccd00381a36ccd4c6405f5990c338950cb66c9ac 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3698,6 +3698,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
