diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index cb085a13580106f0218285dc5ad51912d60735d2..660a262e45de4153d651b945b5024984a27f73f1 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3532,6 +3532,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
