diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index 414af5f3574d3fa7c45a5b809cd99d19e5202160..58d2f9b97321dd714cabc4f72ded2eecc4217a85 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4717,6 +4717,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (web_view->MainFrame()->IsWebRemoteFrame())
+    BRAVE_RENDER_FRAME_IMPL_SHOULD_USE_USER_AGENT_OVERRIDE
     return false;
   const WebLocalFrame* main_frame = web_view->MainFrame()->ToWebLocalFrame();
 
