diff --git a/content/browser/webui/web_ui_impl.cc b/content/browser/webui/web_ui_impl.cc
index dbbe49eb0983f8e56730f123754a890292bf3980..a391eed86e269a4f56d362b4a79def1747130c35 100644
--- a/content/browser/webui/web_ui_impl.cc
+++ b/content/browser/webui/web_ui_impl.cc
@@ -128,6 +128,7 @@ WebUIImpl::WebUIImpl(WebContents* web_contents)
       web_contents_observer_(
           std::make_unique<WebUIMainFrameObserver>(this, web_contents_)) {
   DCHECK(web_contents_);
+  BRAVE_WEB_UI_IMPL
 }
 
 WebUIImpl::WebUIImpl(NavigationRequest* request)
