diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 10a4266c851aeb2ed298b06fbf7816b28dc43f36..e72d3fbd9dd42b6863e6599f7ce1e13195abb8d4 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -31,7 +31,7 @@ source_set("common") {
   # In addition, targets outside of the content component (shell and tests)
   # must not link to this because it will duplicate the code in the component
   # build.
-  visibility = [ "//content/*" ]
+  visibility = [ "//content/*", "//brave/content/*" ]
 
   sources = [
     "accessibility_messages.h",
