diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index c61a07110421e58f0c86bc6d2436b865b21845b8..dc259c1009526324b7abf87bfb7aa41f42805993 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1779,6 +1779,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if defined(OS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
