diff --git a/content/browser/media/session/media_session_controller.cc b/content/browser/media/session/media_session_controller.cc
index cc0af430e6162d75b702383e467bc521e1fa8ac9..ee9a0d1d6555e85ca01f0bb4faa53e3ace9fe4f2 100644
--- a/content/browser/media/session/media_session_controller.cc
+++ b/content/browser/media/session/media_session_controller.cc
@@ -318,4 +318,8 @@ media::MediaContentType MediaSessionController::GetMediaContentType() const {
   return media_content_type_;
 }
 
+MediaPlayerId MediaSessionController::GetMediaPlayerId() const {
+  return id_;
+}
+
 }  // namespace content
