diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 86a0536722bf79c105b0ab416f66158a00f01b06..651f51202c1f23e209b307440c04af7e68d23a79 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -777,6 +777,7 @@ source_set("browser_sources") {
   if (use_clang_profiling_inside_sandbox) {
     sources += [ "profiling_utils.h" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
