diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index b9656de7a8fef51d329bd5d102ef0eb6260a58ec..25b4b6584b79dc12985fbcde67487050098698d5 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -749,6 +749,7 @@ source_set("browser_sources") {
   if (use_clang_profiling_inside_sandbox) {
     sources += [ "profiling_utils.h" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
