diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 68017b5f9deb2f65577f8ce901ead46e6a4469ca..811dd08b32069f676a7e6f690f5cba6184265d05 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1925,6 +1925,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
