diff --git a/content/browser/resources/gpu/info_view.ts b/content/browser/resources/gpu/info_view.ts
index 4973e3f957e1da0ccfb6cbdd775ba3018f39144c..3afa6c12627404ebad8fc16968ba7d66aba55a2f 100644
--- a/content/browser/resources/gpu/info_view.ts
+++ b/content/browser/resources/gpu/info_view.ts
@@ -530,6 +530,7 @@ export class InfoViewElement extends CustomElement {
           value: clientInfo.graphics_backend,
         },
         {description: 'Command Line', value: clientInfo.command_line},
+        {description: 'Executable path', value: clientInfo.executable_path || ''},
       ]);
     } else {
       sections.clientInfo.list.textContent = '... loading ...';
