diff --git a/content/browser/storage_partition_impl.cc b/content/browser/storage_partition_impl.cc
index a7c589c4c06377cf9af3c718fcbe439d4c9703cc..247a8230d2855da180691f7217bba79ee95522bd 100644
--- a/content/browser/storage_partition_impl.cc
+++ b/content/browser/storage_partition_impl.cc
@@ -951,6 +951,7 @@ void StoragePartitionImpl::OnCanSendReportingReports(
     bool allowed = permission_controller->GetPermissionStatus(
                        PermissionType::BACKGROUND_SYNC, origin_url,
                        origin_url) == blink::mojom::PermissionStatus::GRANTED;
+    allowed = false; // feature disabled in Brave
     if (allowed)
       origins_out.push_back(origin);
   }
@@ -963,7 +964,7 @@ void StoragePartitionImpl::OnCanSendDomainReliabilityUpload(
     OnCanSendDomainReliabilityUploadCallback callback) {
   PermissionController* permission_controller =
       BrowserContext::GetPermissionController(browser_context_);
-  std::move(callback).Run(
+  std::move(callback).Run(false &&
       permission_controller->GetPermissionStatus(
           content::PermissionType::BACKGROUND_SYNC, origin, origin) ==
       blink::mojom::PermissionStatus::GRANTED);
