diff --git a/content/browser/webui/web_ui_data_source_impl.cc b/content/browser/webui/web_ui_data_source_impl.cc
index f4a2b992008e7edc1d7be98b274e8ba11387c598..d7347713f370f310e61a678d73773f119f38de27 100644
--- a/content/browser/webui/web_ui_data_source_impl.cc
+++ b/content/browser/webui/web_ui_data_source_impl.cc
@@ -349,7 +349,7 @@ void WebUIDataSourceImpl::OverrideContentSecurityPolicy(
     network::mojom::CSPDirectiveName directive,
     const std::string& value) {
   CHECK(!resources_frozen_);
-  csp_overrides_.insert_or_assign(directive, value);
+  csp_overrides_.insert_or_assign(directive, GetBraveTrustedTypesValue(directive, value));
 }
 
 void WebUIDataSourceImpl::OverrideCrossOriginOpenerPolicy(
