diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 08d52d6e799096e21251b0bb26de37147cf62f12..3a896708bc0d6da6636c4f66f86b4b54c6e9884c 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1811,6 +1811,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if defined(OS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
