diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 9a2ea366c71968b2ed56301f959ebec7e04d0fba..693057e318c7600bdc1019d4f14e28ec68313d47 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -2208,6 +2208,12 @@ jumbo_source_set("browser") {
       "//ppapi/proxy:ipc",
       "//ppapi/shared_impl",
     ]
+    if (brave_chromium_build) {
+      sources += [
+        "//brave/content/browser/renderer_host/brave_plugin_registry_impl.cc",
+        "//brave/content/browser/renderer_host/brave_plugin_registry_impl.h",
+      ]
+    }
     if (is_linux) {
       deps += [ "//components/services/font:ppapi_fontconfig_matching" ]
     }
