diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 53dd562d46baea4035051e4760616752dee69be7..92adc214c1a732eddfc86b2eb857e72b6f2992d7 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3634,6 +3634,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
