diff --git a/content/app_shim_remote_cocoa/web_contents_ns_view_bridge.h b/content/app_shim_remote_cocoa/web_contents_ns_view_bridge.h
index 7ecf4d093c6a934d8491d8d2ba24d4ad1b63fcd8..e3f9017e2305e90bcd41ebf50507f98ab03206be 100644
--- a/content/app_shim_remote_cocoa/web_contents_ns_view_bridge.h
+++ b/content/app_shim_remote_cocoa/web_contents_ns_view_bridge.h
@@ -53,7 +53,7 @@ class CONTENT_EXPORT WebContentsNSViewBridge : public mojom::WebContentsNSView {
   // mojom::WebContentsNSViewBridge:
   void SetParentNSView(uint64_t parent_ns_view_id) override;
   void ResetParentNSView() override;
-  void SetBounds(const gfx::Rect& bounds_in_superview) override;
+  void SetBounds(const gfx::Rect& bounds_in_window) override;
   void SetVisible(bool visible) override;
   void MakeFirstResponder() override;
   void TakeFocus(bool reverse) override;
@@ -73,6 +73,6 @@ class CONTENT_EXPORT WebContentsNSViewBridge : public mojom::WebContentsNSView {
   std::unique_ptr<ScopedNSViewIdMapping> view_id_;
 };
 
-}  // namespace remote_cocoa
+}  // namespace content
 
 #endif  // CONTENT_APP_SHIM_REMOTE_COCOA_WEB_CONTENTS_NS_VIEW_BRIDGE_H_
