diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index f9e1b4c6eef18ab6a0ab03755eaae4cb2f9e8804..f95087dc23e6abda096685297ca2a844a0e5aadd 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -42,6 +42,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
