diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 19c948e949386a1678767cbc606304e65400c507..613ca06202aa22357b2d36b9dcb13f9c6561bc69 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -63,6 +63,7 @@ source_set("common") {
   # must not link to this because it will duplicate the code in the component
   # build.
   visibility = [ "//content/*" ]
+  visibility += [ "//brave/content/*" ]
 
   sources = [
     "background_fetch/background_fetch_types.cc",
