diff --git a/content/browser/storage_partition_impl.cc b/content/browser/storage_partition_impl.cc
index 66696708ac5efb20916aa64591949c2c20ee81c7..f214b9015c8449fccce4a45dcead370e3118a72e 100644
--- a/content/browser/storage_partition_impl.cc
+++ b/content/browser/storage_partition_impl.cc
@@ -968,6 +968,7 @@ void StoragePartitionImpl::OnCanSendReportingReports(
     bool allowed = permission_controller->GetPermissionStatus(
                        PermissionType::BACKGROUND_SYNC, origin_url,
                        origin_url) == blink::mojom::PermissionStatus::GRANTED;
+    allowed = false; // feature disabled in Brave
     if (allowed)
       origins_out.push_back(origin);
   }
@@ -980,7 +981,7 @@ void StoragePartitionImpl::OnCanSendDomainReliabilityUpload(
     OnCanSendDomainReliabilityUploadCallback callback) {
   PermissionController* permission_controller =
       BrowserContext::GetPermissionController(browser_context_);
-  std::move(callback).Run(
+  std::move(callback).Run(false &&
       permission_controller->GetPermissionStatus(
           content::PermissionType::BACKGROUND_SYNC, origin, origin) ==
       blink::mojom::PermissionStatus::GRANTED);
