diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 1519637194b4ac3ba3433eeac6e25ddc73a77da8..bea06a8b80ba348f44db5de30af512e778f98e23 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3791,6 +3791,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
