diff --git a/content/browser/webui/web_ui_data_source_unittest.cc b/content/browser/webui/web_ui_data_source_unittest.cc
index 849718b093969144278d54141a0ed51876a14d74..06d454d90aaf0a494e4dadc9859b66338013025e 100644
--- a/content/browser/webui/web_ui_data_source_unittest.cc
+++ b/content/browser/webui/web_ui_data_source_unittest.cc
@@ -427,7 +427,7 @@ TEST_F(WebUIDataSourceTest, SetCspValues) {
                 network::mojom::CSPDirectiveName::RequireTrustedTypesFor));
   source()->OverrideContentSecurityPolicy(
       network::mojom::CSPDirectiveName::TrustedTypes, "trusted-types test;");
-  EXPECT_EQ("trusted-types test;",
+  EXPECT_EQ("trusted-types test svelte-trusted-html default;",
             url_data_source->GetContentSecurityPolicy(
                 network::mojom::CSPDirectiveName::TrustedTypes));
   source()->DisableTrustedTypesCSP();
