diff --git a/content/browser/webui/web_ui_impl.cc b/content/browser/webui/web_ui_impl.cc
index a4f08bc2f83c6bce669cf1a6e694d98a1aac9a4e..e97c17130ffb3e691f53cab548ebdad47f71d445 100644
--- a/content/browser/webui/web_ui_impl.cc
+++ b/content/browser/webui/web_ui_impl.cc
@@ -89,6 +89,7 @@ WebUIImpl::WebUIImpl(WebContentsImpl* contents, RenderFrameHost* frame_host)
       web_contents_(contents),
       web_contents_observer_(new MainFrameNavigationObserver(this, contents)) {
   DCHECK(contents);
+  BRAVE_WEB_UI_IMPL
 }
 
 WebUIImpl::~WebUIImpl() {
