diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 538a342dd825861da6d0e8a9fc2a526b3b46c701..b9894c915b8fc376b91418582d1cba66d9951990 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -31,7 +31,7 @@ source_set("common") {
   # In addition, targets outside of the content component (shell and tests)
   # must not link to this because it will duplicate the code in the component
   # build.
-  visibility = [ "//content/*" ]
+  visibility = [ "//content/*", "//brave/content/*" ]
 
   sources = [
     "accessibility_messages.h",
