diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 3bbb6a39eeb719b94d0e212ea8da5051ff55d441..c382ace2b60e06424a9cf36a4d3704618725d06a 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1968,6 +1968,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
