diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 5fa36b1a1329d99d7af656eb7114560955bf48e0..d1289b762bb5416f4dfddc629bd6fd7a54764f6c 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -929,6 +929,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
