diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index cabe1f72bdcc150496f20b117027ad4b4eff58cf..867cb9f16f2a3266f691e14a19dcf81e661f9726 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4991,7 +4991,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (render_view_->GetWebView()->MainFrame()->IsWebRemoteFrame())
-    return false;
+    return true;
   WebLocalFrame* main_frame =
       render_view_->GetWebView()->MainFrame()->ToWebLocalFrame();
 
