diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 6000d0326898e33cdf4b311c88e5400d8be71ed2..9bdfacc581b1c98b73908accf8544991c3b35f68 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1854,6 +1854,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if defined(OS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
