diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index c807cd670496195662edd1e13a6304f0727825e0..cacc7650ed0f7b941e6e787e40ec9a4c66b0bfb5 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3689,6 +3689,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants:constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
