diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 2c3a21e01a60263574121ec46d80d2e3392ac81a..e51c049794c1e91f8df918aee1c3124b0aa7d570 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3853,6 +3853,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
