diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index ead12a24cc3045ba0a732dac08b6ee4c3b70a521..57c75ba58c2b53fc83ee942579d7932f5ea2f191 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
