diff --git a/content/browser/gpu/gpu_internals_ui.cc b/content/browser/gpu/gpu_internals_ui.cc
index 6efa12b1aae409bbf9a7dd429bbad19f9007924a..2895ded09af18dc1cf6e1da3888e61d2bba9817f 100644
--- a/content/browser/gpu/gpu_internals_ui.cc
+++ b/content/browser/gpu/gpu_internals_ui.cc
@@ -830,6 +830,7 @@ base::Value::Dict GpuMessageHandler::GetClientInfo() {
            std::string("Skia/" STRINGIZE(SK_MILESTONE) " " SKIA_COMMIT_HASH));
   dict.Set("revision_identifier", GPU_LISTS_VERSION);
 
+  BRAVE_GPU_MESSAGE_HANDLER_ON_REQUEST_CLIENT_INFO
   return dict;
 }
 
