diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 18327d5fc34a81949924ae3b3a838f4419e683b2..df1d70a0b1a18a685921bd17d0146cda7e11829d 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1915,6 +1915,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if defined(OS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
