diff --git a/content/browser/renderer_host/navigation_request.cc b/content/browser/renderer_host/navigation_request.cc
index b71678c8c556d2151e6056f3b94645e1ae1d3f85..86653f7f216880873b75617c11691e42842a1881 100644
--- a/content/browser/renderer_host/navigation_request.cc
+++ b/content/browser/renderer_host/navigation_request.cc
@@ -4030,6 +4030,9 @@ void NavigationRequest::OnRequestRedirected(
   }
 
   common_params_->referrer->url = GURL(redirect_info.new_referrer);
+  GetContentClient()->browser()->MaybeHideReferrer(
+      browser_context, common_params_->url,
+      GetTopDocumentGURL(frame_tree_node_), &common_params_->referrer);
   common_params_->referrer = Referrer::SanitizeForRequest(
       common_params_->url, *common_params_->referrer);
 
@@ -6228,7 +6231,10 @@ void NavigationRequest::OnStartChecksComplete(
   bool is_visible = GetWebContents() &&
                     GetWebContents()->GetVisibility() == Visibility::VISIBLE;
 
-  loader_ = NavigationURLLoader::Create(
+  GetContentClient()->browser()->MaybeHideReferrer(
+    browser_context, common_params_->url,
+    GetTopDocumentGURL(frame_tree_node_), &common_params_->referrer);
+loader_ = NavigationURLLoader::Create(
       browser_context, partition,
       std::make_unique<NavigationRequestInfo>(
           common_params_->Clone(), begin_params_.Clone(), sandbox_flags,
