diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 26cd6d1afcb89148ed33ab733012a778e9b75c40..663599a225eb161ec4a92832c43dca2f485df24e 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1724,6 +1724,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if defined(OS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
