diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index deb960892c575e43c0f4b8af3d8343a60a1102c9..6d0f8bc19b46d6815c2a7c1bad1abac92f676c2e 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
