diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 2a0f8af4b708dca7f70cd808dfedbbbcc1b40faf..9057cdff52a94aaae196db8e013ed2df768230b4 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -748,6 +748,7 @@ source_set("browser_sources") {
   if (use_clang_profiling_inside_sandbox) {
     sources += [ "profiling_utils.h" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
