diff --git a/content/public/browser/web_contents_receiver_set.h b/content/public/browser/web_contents_receiver_set.h
index 725e64c675465d2beb6a3db4763230d75bf7bb93..b2fb29f019231997ab52ef50f86ccf3789ac1a64 100644
--- a/content/public/browser/web_contents_receiver_set.h
+++ b/content/public/browser/web_contents_receiver_set.h
@@ -55,6 +55,10 @@ namespace subresource_redirect {
 class SubresourceRedirectObserver;
 }
 
+namespace brave_shields {
+class BraveShieldsWebContentsObserver;
+}
+
 namespace content {
 
 class ConversionHost;
@@ -142,6 +146,7 @@ class WebContentsFrameReceiverSetPassKey {
   friend class DisplayCutoutHostImpl;
   friend class ScreenOrientationProvider;
   friend class TestFrameInterfaceBinder;
+  friend class ::brave_shields::BraveShieldsWebContentsObserver;
   FRIEND_TEST_ALL_PREFIXES(WebContentsReceiverSetBrowserTest,
                            OverrideForTesting);
 };
