diff --git a/content/browser/renderer_host/render_frame_host_impl.cc b/content/browser/renderer_host/render_frame_host_impl.cc
index 497033d3d5257daffe4aca2ebdd1fd71513db73f..f710a73beed45b44c822db4ee6366b7ca0a991e3 100644
--- a/content/browser/renderer_host/render_frame_host_impl.cc
+++ b/content/browser/renderer_host/render_frame_host_impl.cc
@@ -1057,6 +1057,7 @@ RenderFrameHostImpl::RenderFrameHostImpl(
   // of overrides.
   idle_manager_ =
       std::make_unique<IdleManagerImpl>(GetProcess()->GetBrowserContext());
+  cosmetic_filters_communication_impl_ = nullptr;
 }
 
 RenderFrameHostImpl::~RenderFrameHostImpl() {
@@ -9683,6 +9684,17 @@ bool RenderFrameHostImpl::DocumentUsedWebOTP() {
   return document_used_web_otp_;
 }
 
+#if defined(OS_ANDROID)
+void RenderFrameHostImpl::GetCosmeticFiltersResponder(
+      mojo::PendingReceiver<
+          cf_comm::mojom::CosmeticFiltersCommunication> receiver) {
+  CosmeticFiltersCommunicationImpl::CreateInstance(this, nullptr);
+  mojo::MakeSelfOwnedReceiver(
+      std::move(cosmetic_filters_communication_impl_),
+      std::move(receiver));
+}
+#endif
+
 std::ostream& operator<<(std::ostream& o,
                          const RenderFrameHostImpl::LifecycleState& s) {
   return o << LifecycleStateToString(s);
