diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index a23f4125b0b1aae800c8d4c4573778b8632ae5f2..708cad2c414f3d2fe6338c9359c9b42ad2b22ba9 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3673,6 +3673,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants:constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
