diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 57947a93614a66892ddcc1e7a03d1a9afa87d5f1..c07ac32d4c53861b924008fae9bfe35f52122f8f 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1860,6 +1860,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if defined(OS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
