diff --git a/content/browser/media/android/media_resource_getter_impl.cc b/content/browser/media/android/media_resource_getter_impl.cc
index 5bf938f1ca81db832c1d6b0a1b522c06f09755b1..791058e5f549d26097cff0c2fec436de60d84cf7 100644
--- a/content/browser/media/android/media_resource_getter_impl.cc
+++ b/content/browser/media/android/media_resource_getter_impl.cc
@@ -23,6 +23,7 @@
 #include "net/http/http_auth.h"
 #include "services/network/public/mojom/restricted_cookie_manager.mojom.h"
 #include "url/gurl.h"
+#include "url/origin.h"
 
 namespace content {
 
@@ -148,8 +149,11 @@ void MediaResourceGetterImpl::GetCookies(const GURL& url,
           browser_context_, url, render_process_id_, render_frame_id_);
   network::mojom::RestrictedCookieManager* cookie_manager_ptr =
       cookie_manager.get();
+  // TODO(crbug.com/988398): Same check as in mojo_renderer_service.cc. Is this
+  // correct?
+  DCHECK(!site_for_cookies.is_empty());
   cookie_manager_ptr->GetCookiesString(
-      url, site_for_cookies,
+      url, site_for_cookies, url::Origin::Create(site_for_cookies),
       base::BindOnce(&ReturnResultOnUIThreadAndClosePipe,
                      std::move(cookie_manager), std::move(callback)));
 }
