diff --git a/content/browser/child_process_security_policy_impl.cc b/content/browser/child_process_security_policy_impl.cc
index 3c18dc775e8bf46d0eedda409ce346736671fd5c..aae4736a8e6258b80dc712bfca6dad9c627057b5 100644
--- a/content/browser/child_process_security_policy_impl.cc
+++ b/content/browser/child_process_security_policy_impl.cc
@@ -827,6 +827,9 @@ void ChildProcessSecurityPolicyImpl::GrantWebUIBindings(int child_id,
 
   // Web UI bindings need the ability to request chrome: URLs.
   state->second->GrantRequestScheme(kChromeUIScheme);
+#if defined(BRAVE_CHROMIUM_BUILD)
+  state->second->GrantRequestScheme(kBraveUIScheme);
+#endif
 
   // Web UI pages can contain links to file:// URLs.
   state->second->GrantRequestScheme(url::kFileScheme);
