diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 1364b7440140848d4a7d8de61f27444af6b03cd5..601327eebd2153ce4198ef6d6345d964d33b42ab 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -36,6 +36,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
