diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 988a1d679c87b55f00b21a8ce42401206ebcf9c3..fe458969b34d69c497747859a0575d99e6bf3c3e 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -754,6 +754,7 @@ source_set("browser_sources") {
   if (use_clang_profiling_inside_sandbox) {
     sources += [ "profiling_utils.h" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
