diff --git a/content/browser/webui/web_ui_data_source_impl.cc b/content/browser/webui/web_ui_data_source_impl.cc
index 2ef84b73ac6b7e6fb5803cbdbbbc376e77950c18..8f6ae44c22a415d97862404d931d4ef336203a81 100644
--- a/content/browser/webui/web_ui_data_source_impl.cc
+++ b/content/browser/webui/web_ui_data_source_impl.cc
@@ -328,7 +328,7 @@ void WebUIDataSourceImpl::OverrideContentSecurityPolicy(
     network::mojom::CSPDirectiveName directive,
     const std::string& value) {
   CHECK(!resources_frozen_);
-  csp_overrides_.insert_or_assign(directive, value);
+  csp_overrides_.insert_or_assign(directive, GetBraveTrustedTypesValue(directive, value));
 }
 
 void WebUIDataSourceImpl::OverrideCrossOriginOpenerPolicy(
