diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 01f516fce5ebd25e0ef8fff4eee1c57135cd6d61..fccd6b9d281a9197a2c6485fb138145a040313f8 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -762,6 +762,7 @@ source_set("browser_sources") {
   if (enable_video_effects) {
     public_deps += [ "//services/video_effects/public/mojom:mojom" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
