diff --git a/content/browser/permissions/permission_service_impl.cc b/content/browser/permissions/permission_service_impl.cc
index 4c5760c91344f9e251cedb385d7beafb45da57cb..a5fa8ddcde7425e79b8672dbddc867934b264839 100644
--- a/content/browser/permissions/permission_service_impl.cc
+++ b/content/browser/permissions/permission_service_impl.cc
@@ -108,6 +108,9 @@ bool PermissionDescriptorToPermissionType(
         return true;
       }
       break;
+    case PermissionName::AUTOPLAY:
+      *permission_type  = PermissionType::AUTOPLAY;
+      return true;
   }
 
   NOTREACHED();
