diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 6da224ff8aef4d9ede9fdfb84086fb798613aedb..ec75f8d84bde053d104ea729c4cb52d0f4e45245 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3627,6 +3627,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
