diff --git a/content/browser/renderer_host/render_view_host_impl.h b/content/browser/renderer_host/render_view_host_impl.h
index 70f9ea0f650b9a4b482d823a603871808f8e125c..42f51e98a99aec21c0cd3c35219c665eec49ea3c 100644
--- a/content/browser/renderer_host/render_view_host_impl.h
+++ b/content/browser/renderer_host/render_view_host_impl.h
@@ -305,6 +305,10 @@ class CONTENT_EXPORT RenderViewHostImpl
   void OnPasteFromSelectionClipboard();
   void OnTakeFocus(bool reverse);
   void OnFocus();
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  void OnFinalPageGraph(const std::string& frame_id,
+                        const std::string& data);
+// #endif
 
  private:
   // TODO(nasko): Temporarily friend RenderFrameHostImpl, so we don't duplicate
