diff --git a/content/public/browser/permission_type.h b/content/public/browser/permission_type.h
index 16c3c2ad4ec798944aef5fba3437cd40fd73162c..644e00436164a0434f1ddf22320c2bd3ebf69967 100644
--- a/content/public/browser/permission_type.h
+++ b/content/public/browser/permission_type.h
@@ -32,6 +32,7 @@ enum class PermissionType {
   BACKGROUND_FETCH = 17,
   IDLE_DETECTION = 18,
 
+  AUTOPLAY = 18,
   // Always keep this at the end.
   NUM,
 };
