diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 0359d275187a886c06a145d3efb8f0a2c9b6db2e..83b47dd87bfd086369e5cc76e5ae3cbb8f457676 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3604,6 +3604,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
