diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 3c90a048f57f056c7c3e4f1b5d60c7ce0d4d1bfc..bbbaabe8b49057c8eb87c4bcf7fc0e2adc8db8a9 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2284,6 +2284,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
