diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 4176c771c26f67be6bb378c7b923f848bcf7cd1b..a2abe03ab6236bf861b8292d2262f3cb79ef937a 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -950,6 +950,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
