diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index b6a056296832cb34852d68d04bdf80372832de0a..cc7ba438daaf10e41548e0d2e768b19158aadf36 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -36,6 +36,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
