diff --git a/content/common/user_agent.cc b/content/common/user_agent.cc
index c58846564ef1dbc8f7cb4b12e19b5117747916ad..c670558f3a3c35af2c37d17069c31e1d237a296a 100644
--- a/content/common/user_agent.cc
+++ b/content/common/user_agent.cc
@@ -316,6 +316,7 @@ std::string GetAndroidOSInfo(
 
   // Send information about the device.
   bool semicolon_inserted = false;
+  BRAVE_GET_ANDROID_OS_INFO
   if (include_android_model == IncludeAndroidModel::Include) {
     std::string android_device_name = BuildModelInfo();
     if (!android_device_name.empty()) {
