diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 17b628006b06fd07c902ef2dcf1de4d5ac918dea..7ffd2bb12cb67fb494948a79dfc893883e5b9f92 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -921,6 +921,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
