diff --git a/content/browser/resources/gpu/browser_bridge.ts b/content/browser/resources/gpu/browser_bridge.ts
index d6d58da3120f6cc53b2e834fcb3f8e3aa675b413..85c3d62dc0800a66572430c13ec597cf43972da0 100644
--- a/content/browser/resources/gpu/browser_bridge.ts
+++ b/content/browser/resources/gpu/browser_bridge.ts
@@ -12,6 +12,7 @@ export interface ClientInfo {
   operating_system: string;
   revision_identifier: string;
   version: string;
+  executable_path: string;
 }
 
 export interface Problem {
