diff --git a/content/browser/loader/reconnectable_url_loader_factory.cc b/content/browser/loader/reconnectable_url_loader_factory.cc
index 1a82d44aade7d4c2648d9f8a370cbee97fc4163c..6dc0c60a6019411e17c1582b7f57f4d02ea771f9 100644
--- a/content/browser/loader/reconnectable_url_loader_factory.cc
+++ b/content/browser/loader/reconnectable_url_loader_factory.cc
@@ -29,7 +29,7 @@ void ReconnectableURLLoaderFactory::CreateLoaderAndStart(
     const net::MutableNetworkTrafficAnnotationTag& traffic_annotation) {
   if (network::mojom::URLLoaderFactory* factory = GetURLLoaderFactory()) {
     factory->CreateLoaderAndStart(std::move(receiver), request_id, options,
-                                  url_request, std::move(client),
+                                  BRAVE_CREATE_LOADER_AND_START, std::move(client),
                                   traffic_annotation);
   }
 }
@@ -136,7 +136,7 @@ class ReconnectableURLLoaderFactoryForIOThread::URLLoaderFactoryForIOThread
       override {
     DCHECK(BrowserThread::CurrentlyOn(BrowserThread::IO));
     factory_getter_->GetURLLoaderFactory()->CreateLoaderAndStart(
-        std::move(receiver), request_id, options, url_request,
+        std::move(receiver), request_id, options, BRAVE_CREATE_LOADER_AND_START,
         std::move(client), traffic_annotation);
   }
 
