diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 548abd4a6857cc47691c276b4b11cb34eadedcdf..9cbd6b08026e37fd67bc7980d6cceb4df72e39cb 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3706,6 +3706,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
