diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 754d7efda808d8d45728476a1c5072e90a504909..55a2fff72041c8818170e930463b56b149b77e62 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1785,6 +1785,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if defined(OS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
