diff --git a/content/public/browser/browser_context.h b/content/public/browser/browser_context.h
index e49eaaa04abb83e974d3e60add620b2c9172d1b2..c148bf537a58ff9cba138e3de54aeb421b225500 100644
--- a/content/public/browser/browser_context.h
+++ b/content/public/browser/browser_context.h
@@ -242,6 +242,9 @@ class CONTENT_EXPORT BrowserContext : public base::SupportsUserData {
   // Return whether this context is incognito. Default is false.
   virtual bool IsOffTheRecord() const = 0;
 
+  // Return whether this context uses Tor. Default is false.
+  virtual bool IsTorProfile() const;
+
   // Returns the resource context.
   virtual ResourceContext* GetResourceContext() = 0;
 
