diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 4ff52ee45512799037285e60d4d8f571c2058edf..784ba06fe9f863a1b317d0eb8e03b37135d1c729 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD - style license that can be
 # found in the LICENSE file.
 
+import("//brave/content/public/browser/sources.gni")
 import("//build/config/ui.gni")
 import("//content/common/features.gni")
 import("//content/public/common/features.gni")
@@ -554,6 +555,7 @@ source_set("browser_sources") {
     "webui_config_map.h",
     "zoom_level_delegate.h",
   ]
+  sources += brave_content_public_browser_sources
   configs += [
     "//build/config:precompiled_headers",
     "//build/config/compiler:wexit_time_destructors",
