diff --git a/content/public/browser/permission_type.cc b/content/public/browser/permission_type.cc
index 8bb88e872867e29aae689e44e3a882940e9a1ca0..d7502c196777726690d6e91c1c89f868c4c6c9c8 100644
--- a/content/public/browser/permission_type.cc
+++ b/content/public/browser/permission_type.cc
@@ -95,6 +95,7 @@ base::Optional<PermissionType> PermissionDescriptorToPermissionType(
       return PermissionType::NFC;
     case PermissionName::STORAGE_ACCESS:
       return PermissionType::STORAGE_ACCESS_GRANT;
+    BRAVE_PERMISSIONDESCRIPTORTOPERMISSIONTYPE
     case PermissionName::WINDOW_PLACEMENT:
       return PermissionType::WINDOW_PLACEMENT;
   }
