diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index c848b7f8f765002d9839c327540d1da6abde9a70..f68433e7ee0b6f0e033921d6c97f5a37011ee601 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -933,6 +933,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
