diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index d7d686ec2d419d6fbd045565813def91e5439d4a..5aa8903bd342fec96067aa36a28c2c9c930f1d0c 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -42,6 +42,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
