diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 2f86fff71f36353f1fc3d8024f16dbb4921a1db8..9a7e3866fc81ce89027c4b10f613671aa6a04c6f 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -927,6 +927,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
