diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 15b082726ad01563ad0a9dec9ac7af900c28e8a2..69ab3fed1bbb6b0175c6dee30bfbaff84dc2ecd6 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -936,6 +936,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
