diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 7c434cc53c7313d376bc45636134f78d6e3bf6d6..9955f4beecd59a6a3a652b366ac4c38897388500 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -42,6 +42,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
