diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index af1e2cc289018a8e632bb8a2417a85c115986b9f..0c06d7182e08b6b1224b1e0c0ba217e5947c87bc 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -64,6 +64,7 @@ source_set("common") {
   # must not link to this because it will duplicate the code in the component
   # build.
   visibility = [ "//content/*" ]
+  visibility += [ "//brave/content/*" ]

   sources = [
     "background_fetch/background_fetch_types.cc",
