diff --git a/content/public/browser/media_session.h b/content/public/browser/media_session.h
index fca03cc5765027c5e25dc20d309540550e29cc96..fff613b6da621c225b318766d09aee9771aeab02 100644
--- a/content/public/browser/media_session.h
+++ b/content/public/browser/media_session.h
@@ -10,6 +10,7 @@
 #include "base/time/time.h"
 #include "base/unguessable_token.h"
 #include "content/common/content_export.h"
+#include "content/public/browser/media_player_id.h"
 #include "services/media_session/public/mojom/media_session.mojom.h"
 
 namespace content {
@@ -55,6 +56,10 @@ class MediaSession : public media_session::mojom::MediaSession {
   // tests.  Static so that it can be optimized away outside of tests.
   CONTENT_EXPORT static void FlushObserversForTesting(WebContents* contents);
 
+  virtual void GetLoadedUrl(base::OnceCallback<void(const GURL&, bool)> callback) = 0;
+
+  virtual std::optional<MediaPlayerId> GetActivePlayerId() const = 0;
+
   // Tell the media session a user action has performed.
   virtual void DidReceiveAction(
       media_session::mojom::MediaSessionAction action) = 0;
