diff --git a/content/browser/web_contents/web_contents_impl.cc b/content/browser/web_contents/web_contents_impl.cc
index c92c7744be6916e2baea1a882ff2a55fd155b580..dfdc72d280cc1abff611c79aeccabb69828e218e 100644
--- a/content/browser/web_contents/web_contents_impl.cc
+++ b/content/browser/web_contents/web_contents_impl.cc
@@ -4804,7 +4804,7 @@ FrameTree* WebContentsImpl::CreateNewWindow(
 
   // Save the window for later if we're not suppressing the opener (since it
   // will be shown immediately).
-  if (!params.opener_suppressed) {
+  if (!params.opener_suppressed && !params.no_javascript_access) {
     if (!is_guest) {
       WebContentsView* new_view = new_contents_impl->view_.get();
 
