diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 56e30e001f5b4a4b0f21638931c48abaa125a863..daf895a5f92b3154b77d45079953332bb9a25800 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -1764,6 +1764,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if defined(OS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
