diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index e49ee5420a686e6d1c544452b4d202dfc7093236..f52dad852fe415b027bfb50f01b6e68da37ea5ab 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -42,6 +42,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
