diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index caa77f241da412b6136d534d94c9152c364959ae..80d39267230916b450444c421ea26ac85f0f1f4b 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3674,6 +3674,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
