diff --git a/content/public/browser/web_contents.h b/content/public/browser/web_contents.h
index 811fde7ff7d9eea62a368606bbfb646552e66793..d4d51ab3013b5c7a9fe59b1af70e40665aeeb037 100644
--- a/content/public/browser/web_contents.h
+++ b/content/public/browser/web_contents.h
@@ -11,8 +11,10 @@
 #include <memory>
 #include <optional>
 #include <string>
+#include <tuple>
 #include <vector>
 
+#include "base/containers/flat_map.h"
 #include "base/functional/callback_forward.h"
 #include "base/functional/callback_helpers.h"
 #include "base/functional/function_ref.h"
@@ -28,6 +30,7 @@
 #include "cc/input/browser_controls_state.h"
 #include "content/common/content_export.h"
 #include "content/public/browser/invalidate_type.h"
+#include "content/public/browser/media_player_id.h"
 #include "content/public/browser/mhtml_generation_result.h"
 #include "content/public/browser/navigation_controller.h"
 #include "content/public/browser/page.h"
@@ -1532,6 +1535,9 @@ class WebContents : public PageNavigator,
   virtual BackForwardTransitionAnimationManager*
   GetBackForwardTransitionAnimationManager() = 0;
 
+  virtual base::flat_map<MediaPlayerId, std::tuple<GURL, bool, double>>
+  GetMediaMetadataByMediaPlayerIds() const = 0;
+
  private:
   // This interface should only be implemented inside content.
   friend class WebContentsImpl;
