diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 5009ca7758d4dc0b002673e400b1cffb6d2c25ea..3725a8cf17974d18ca23e32144f99994fa9b6da0 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3793,6 +3793,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
