diff --git a/content/browser/webui/web_ui_impl.cc b/content/browser/webui/web_ui_impl.cc
index c1cb3032acb93880b83b76c0e3594620ad360c05..bf035bd0cff5815717748e9448639362d9bc80dd 100644
--- a/content/browser/webui/web_ui_impl.cc
+++ b/content/browser/webui/web_ui_impl.cc
@@ -75,6 +75,7 @@ WebUIImpl::WebUIImpl(WebContentsImpl* contents, RenderFrameHostImpl* frame_host)
              RenderFrameHostImpl::LifecycleStateImpl::kActive ||
          frame_host->lifecycle_state() ==
              RenderFrameHostImpl::LifecycleStateImpl::kSpeculative);
+  BRAVE_WEB_UI_IMPL
 }
 
 WebUIImpl::~WebUIImpl() {
