diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 63177e15437bdb10ce0d7b11bee036f203262f9a..46fc90d971705cdb10c62284589e3ac86e52c579 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3575,6 +3575,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
