diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 0352f12717108a0ed5263f5b2d2a6b9af05756b0..ce202c5af7aa21bd32e4ff7b970c2d935ffe7736 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -1022,6 +1022,7 @@ void RenderThreadImpl::InitializeWebKit(mojo::BinderMap* binders) {
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
