diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 39709f2de43d3ea1c78e455d3b182901ff2e9426..56bb936fdc6ac8ba299d9017b8fbed03c99dea7b 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -753,6 +753,7 @@ source_set("browser_sources") {
   if (use_clang_profiling_inside_sandbox) {
     sources += [ "profiling_utils.h" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
