diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 1ac4a63325dca9c8664c9bc1dacd125325d31c9d..b81f8e1eb759518a3de99d7d33b881d310823f2b 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -42,6 +42,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
