diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 2a4ef461fae47a8b12f143a5ed119ad3a812bc06..00ae2f924b1f4a3fb8713de8a6663809e3c1a9b3 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -747,6 +747,7 @@ source_set("browser_sources") {
   if (enable_video_effects) {
     public_deps += [ "//services/video_effects/public/mojom:mojom" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
