diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 41634a9509f07b5dccdcb1061a5a3f2e40ba5b25..3de715193a0cf98214bf5d4f30a2306886d37e81 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3769,6 +3769,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
