diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 2f57856179bffdc35ca14cc7b1419bee9d7bbf8e..dbcc1d8d7507e42107f84fcd66ccc8e8f91a9cea 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3711,6 +3711,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
