diff --git a/content/browser/renderer_host/render_view_host_impl.cc b/content/browser/renderer_host/render_view_host_impl.cc
index 01a4e3dc134b3cf267ec637dde1066db51ac78e7..a9d0a1fffec5efba7ef4e82f87f5c325827a5435 100644
--- a/content/browser/renderer_host/render_view_host_impl.cc
+++ b/content/browser/renderer_host/render_view_host_impl.cc
@@ -855,7 +855,9 @@ bool RenderViewHostImpl::OnMessageReceived(const IPC::Message& msg) {
     IPC_MESSAGE_HANDLER(ViewHostMsg_RouteCloseEvent, OnRouteCloseEvent)
     IPC_MESSAGE_HANDLER(ViewHostMsg_UpdateTargetURL, OnUpdateTargetURL)
     IPC_MESSAGE_HANDLER(ViewHostMsg_TakeFocus, OnTakeFocus)
-    IPC_MESSAGE_HANDLER(ViewHostMsg_Focus, OnFocus)
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+    IPC_MESSAGE_HANDLER(ViewHostMsg_FinalPageGraph, OnFinalPageGraph)
+// #endif
     IPC_MESSAGE_UNHANDLED(handled = false)
   IPC_END_MESSAGE_MAP()
 
@@ -922,6 +924,13 @@ void RenderViewHostImpl::OnFocus() {
   delegate_->Activate();
 }
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+void RenderViewHostImpl::OnFinalPageGraph(const std::string& frame_id,
+                                          const std::string& data) {
+  delegate_->FinalPageGraph(frame_id, data);
+}
+// #endif
+
 const mojo::AssociatedRemote<blink::mojom::PageBroadcast>&
 RenderViewHostImpl::GetAssociatedPageBroadcast() {
   return page_broadcast_;
