diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 578bf828963c717f58ada47ff875ad06c89b885d..292dd86d4ee7aa5353028ea1caa34ba84f3ca73b 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -2181,6 +2181,12 @@ jumbo_source_set("browser") {
       "//ppapi/proxy:ipc",
       "//ppapi/shared_impl",
     ]
+    if (brave_chromium_build) {
+      sources += [
+        "//brave/content/browser/renderer_host/brave_plugin_registry_impl.cc",
+        "//brave/content/browser/renderer_host/brave_plugin_registry_impl.h",
+      ]
+    }
     if (use_ozone) {
       sources += [ "renderer_host/pepper/pepper_truetype_font_list_ozone.cc" ]
     }
