diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 95fe796458dea7f1a7ab48e15198dc489cfa7298..7f037bc6e4de5b0125b21260a4266e51eb23578c 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -777,6 +777,7 @@ source_set("browser_sources") {
   if (use_clang_profiling_inside_sandbox) {
     sources += [ "profiling_utils.h" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
