diff --git a/content/browser/permissions/permission_service_impl.cc b/content/browser/permissions/permission_service_impl.cc
index 3495d3dbb33331691200179b3a67bd80b9e5811e..8b13966c38b66ccc139ff3fbf6f7eee106b36233 100644
--- a/content/browser/permissions/permission_service_impl.cc
+++ b/content/browser/permissions/permission_service_impl.cc
@@ -110,6 +110,9 @@ bool PermissionDescriptorToPermissionType(
     case PermissionName::NFC:
       *permission_type = PermissionType::NFC;
       return true;
+    case PermissionName::AUTOPLAY:
+      *permission_type  = PermissionType::AUTOPLAY;
+      return true;
   }
 
   NOTREACHED();
