diff --git a/content/browser/permissions/permission_controller_impl.cc b/content/browser/permissions/permission_controller_impl.cc
index e07765308f980564be911451875d388db8a32815..3ce425ce224b834d35d2816bf2a104d66eefe449 100644
--- a/content/browser/permissions/permission_controller_impl.cc
+++ b/content/browser/permissions/permission_controller_impl.cc
@@ -67,6 +67,7 @@ PermissionToSchedulingFeature(PermissionType permission_name) {
     case PermissionType::IDLE_DETECTION:
     case PermissionType::WAKE_LOCK_SCREEN:
     case PermissionType::WAKE_LOCK_SYSTEM:
+    BRAVE_PERMISSION_TYPES
     case PermissionType::NUM:
       return base::nullopt;
   }
