diff --git a/content/browser/renderer_host/navigation_request.cc b/content/browser/renderer_host/navigation_request.cc
index 654e3fced4f264286ef511ff1d69b80af4515df3..6218f4ab7673c31177278d08a7c0568438f26de0 100644
--- a/content/browser/renderer_host/navigation_request.cc
+++ b/content/browser/renderer_host/navigation_request.cc
@@ -302,7 +302,7 @@ void AddAdditionalRequestHeaders(
   headers->SetHeaderIfMissing(
       net::HttpRequestHeaders::kUserAgent,
       user_agent_override.empty()
-          ? GetContentClient()->browser()->GetUserAgent()
+          BRAVE_NAVIGATION_REQUEST_ADD_ADDITIONAL_REQUEST_HEADERS
           : user_agent_override);
 
   if (!render_prefs.enable_referrers) {
@@ -2080,6 +2080,7 @@ void NavigationRequest::OnRequestRedirected(
   common_params_->url = redirect_info.new_url;
   common_params_->method = redirect_info.new_method;
   common_params_->referrer->url = GURL(redirect_info.new_referrer);
+  BRAVE_ONREQUESTREDIRECTED_MAYBEHIDEREFERRER
   common_params_->referrer = Referrer::SanitizeForRequest(
       common_params_->url, *common_params_->referrer);
 
@@ -3086,6 +3087,7 @@ void NavigationRequest::OnStartChecksComplete(
   headers.MergeFrom(TakeModifiedRequestHeaders());
   begin_params_->headers = headers.ToString();
 
+  BRAVE_ONSTARTCHECKSCOMPLETE_MAYBEHIDEREFERRER
   // TODO(clamy): Avoid cloning the navigation params and create the
   // ResourceRequest directly here.
   std::vector<std::unique_ptr<NavigationLoaderInterceptor>> interceptor;
@@ -5423,7 +5425,7 @@ void NavigationRequest::SetIsOverridingUserAgent(bool override_ua) {
   auto user_agent_override = GetUserAgentOverride();
   headers.SetHeader(net::HttpRequestHeaders::kUserAgent,
                     user_agent_override.empty()
-                        ? GetContentClient()->browser()->GetUserAgent()
+                        BRAVE_NAVIGATION_REQUEST_SET_IS_OVERRIDING_USERAGENT
                         : user_agent_override);
   BrowserContext* browser_context =
       frame_tree_node_->navigator().controller().GetBrowserContext();
