diff --git a/content/browser/dom_storage/dom_storage_context_wrapper.cc b/content/browser/dom_storage/dom_storage_context_wrapper.cc
index d0426d5fafd18548651f6dada1135e65f3d32cbd..f094d9deff3e7216c345d4fce763de721dfac56b 100644
--- a/content/browser/dom_storage/dom_storage_context_wrapper.cc
+++ b/content/browser/dom_storage/dom_storage_context_wrapper.cc
@@ -300,6 +300,7 @@ bool DOMStorageContextWrapper::IsRequestValid(
       return false;
     }
     host_storage_key_did_not_match = host->storage_key() != storage_key;
+    BRAVE_DOM_STORAGE_CONTEXT_WRAPPER_IS_REQUEST_VALID
   }
   if (!security_policy_handle.CanAccessDataForOrigin(storage_key.origin())) {
     const std::string type_string =
