diff --git a/content/browser/browsing_data/browsing_data_filter_builder_impl.cc b/content/browser/browsing_data/browsing_data_filter_builder_impl.cc
index 04a97bf981a1a58e244ebf672dbc50a4e6d56858..a3bd38d3323f081cf014ceb240e99be3df0ea33a 100644
--- a/content/browser/browsing_data/browsing_data_filter_builder_impl.cc
+++ b/content/browser/browsing_data/browsing_data_filter_builder_impl.cc
@@ -58,6 +58,7 @@ bool MatchesStorageKey(const std::set<url::Origin>& origins,
         storage_key.origin() == origin) {
       return is_delete_list;
     }
+    BRAVE_MATCHES_STORAGE_KEY
   }
 
   switch (match_mode) {
@@ -81,6 +82,7 @@ bool MatchesStorageKey(const std::set<url::Origin>& origins,
       return is_delete_list ==
              base::Contains(registerable_domains, registerable_domain);
     }
+    BRAVE_MATCHES_STORAGE_KEY_SWITCH
   }
 
   return !is_delete_list;
