diff --git a/content/browser/webauth/webauth_request_security_checker.h b/content/browser/webauth/webauth_request_security_checker.h
index 017f2e28a100b911cf8aa321b2395adf14925420..dedfbda18c9c5209e1d5d2a0eddbf906ecf44f9e 100644
--- a/content/browser/webauth/webauth_request_security_checker.h
+++ b/content/browser/webauth/webauth_request_security_checker.h
@@ -23,7 +23,8 @@ class Value;
 
 namespace network {
 class SimpleURLLoader;
-}
+class SharedURLLoaderFactory;
+}  // namespace network
 
 namespace content {
 
@@ -60,6 +61,8 @@ class CONTENT_EXPORT WebAuthRequestSecurityChecker
     static std::unique_ptr<RemoteValidation> Create(
         const url::Origin& caller_origin,
         const std::string& relying_party_id,
+        scoped_refptr<network::SharedURLLoaderFactory>
+            shared_url_loader_factory,
         base::OnceCallback<void(blink::mojom::AuthenticatorStatus)> callback);
 
     // ValidateWellKnownJSON implements the core of remote validation. It isn't
@@ -154,6 +157,8 @@ class CONTENT_EXPORT WebAuthRequestSecurityChecker
   [[nodiscard]] bool DeduplicateCredentialDescriptorListAndValidateLength(
       std::vector<device::PublicKeyCredentialDescriptor>* list);
 
+  static bool& UseSystemSharedURLLoaderFactoryForTesting();
+
  protected:
   friend class base::RefCounted<WebAuthRequestSecurityChecker>;
   virtual ~WebAuthRequestSecurityChecker();
