diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 5298101d6dbf0a1e4c9a1d973ff9e58ba299ad9b..c3a9465a99eb9f3cb8ebd8c6504b734cd54b1e7e 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2294,6 +2294,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
