diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 315f5755176b5332ee61413fd904ac38d8f80e7e..0dbb817b4e7a39c2b9bf0907773edd985785c09d 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -912,6 +912,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
