diff --git a/content/child/child_thread_impl.cc b/content/child/child_thread_impl.cc
index 3a3e67fec077f0724a524067948ad6c48361fcb4..6ed053b1d36e8996b99962bd8afef315b955a217 100644
--- a/content/child/child_thread_impl.cc
+++ b/content/child/child_thread_impl.cc
@@ -180,6 +180,9 @@ class SuicideOnChannelErrorFilter : public IPC::MessageFilter {
     __lsan_do_leak_check();
 #endif
 #else
+#if defined(OS_MACOSX)
+    CleanupTor();
+#endif // defined(OS_MACOSX)
     base::Process::TerminateCurrentProcessImmediately(0);
 #endif
   }
