diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 7070753ef7274dbfbf73888caa98110c8187f315..362c34c307ae1bc666289d2db004055eec38ddfe 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -33,7 +33,7 @@ source_set("common") {
   # In addition, targets outside of the content component (shell and tests)
   # must not link to this because it will duplicate the code in the component
   # build.
-  visibility = [ "//content/*" ]
+  visibility = [ "//content/*", "//brave/content/*" ]
 
   sources = [
     "accessibility_messages.h",
