diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 53dd562d46baea4035051e4760616752dee69be7..12a7eeaf6efa6e1d852ca4329983b6b31efa5bfd 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3634,6 +3634,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
@@ -3662,6 +3663,7 @@ group("for_content_tests") {
   } else {
     public_deps = [ "//third_party/leveldatabase" ]
   }
+  import("//brave/content/browser/sources.gni") visibility += brave_content_browser_for_content_tests_visibility
 }
 
 if (is_android) {
