diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 156b7b296b64de8751d7ee44448aa28bdefdcc7a..4c3f8d286849dbeddf3235795dc924bd648d51be 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -754,6 +754,7 @@ source_set("browser_sources") {
   if (enable_video_effects) {
     public_deps += [ "//services/video_effects/public/mojom" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
