diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index 9a143cc6fd2b180212f65c53ba2b8f5d34365a25..8b8e86f3b9c0fcaea29c2e11399d370b61806e72 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4750,7 +4750,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (render_view_->GetWebView()->MainFrame()->IsWebRemoteFrame())
-    return false;
+    return true;
   WebLocalFrame* main_frame =
       render_view_->GetWebView()->MainFrame()->ToWebLocalFrame();
