diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index c92c7110c747cc575c94cf941cf9131987ef4fd6..600aa82572720cbd7edf59ebfd908970a035d560 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3682,6 +3682,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
