diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index d01ad9dbeabeef8a5679823d8b1678d0a46c3865..dec633b27e10c35b77cde80844d023ca4113de3c 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -67,6 +67,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
