diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index bf4f048c80b6aa222ddfa712868726279969c546..db200750258e7b488b046d64550ac579f4d5d0c4 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3700,6 +3700,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
