diff --git a/content/browser/renderer_host/render_view_host_impl.cc b/content/browser/renderer_host/render_view_host_impl.cc
index 8c6deced5a7a..546a5cec4bd1 100644
--- a/content/browser/renderer_host/render_view_host_impl.cc
+++ b/content/browser/renderer_host/render_view_host_impl.cc
@@ -107,9 +107,9 @@
 #include "ui/gfx/system_fonts_win.h"
 #endif
 
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
 #include "content/browser/host_zoom_map_impl.h"
-#endif
+//#endif
 
 using base::TimeDelta;
 
@@ -764,10 +764,10 @@ void RenderViewHostImpl::ClosePage() {
 
     // TODO(creis): Should this be moved to Shutdown?  It may not be called for
     // RenderViewHosts that have been swapped out.
-#if !defined(OS_ANDROID)
+//#if !defined(OS_ANDROID)
     static_cast<HostZoomMapImpl*>(HostZoomMap::Get(GetSiteInstance()))
         ->WillCloseRenderView(GetProcess()->GetID(), GetRoutingID());
-#endif
+//#endif
 
     static_cast<RenderFrameHostImpl*>(GetMainFrame())
         ->GetAssociatedLocalMainFrame()
