diff --git a/content/child/child_thread_impl.cc b/content/child/child_thread_impl.cc
index 5ffeccc4461487e551865d9cb0cbb0c2e89a06a0..6d6e27732bbffa8c32ebb9f851377c790294a923 100644
--- a/content/child/child_thread_impl.cc
+++ b/content/child/child_thread_impl.cc
@@ -183,6 +183,9 @@ class SuicideOnChannelErrorFilter : public IPC::MessageFilter {
     __lsan_do_leak_check();
 #endif
 #else
+#if defined(OS_MACOSX)
+    CleanupTor();
+#endif // defined(OS_MACOSX)
     base::Process::TerminateCurrentProcessImmediately(0);
 #endif
   }
