diff --git a/content/common/user_agent.cc b/content/common/user_agent.cc
index 8315350e3f663976066e4bedee238997c0bccb97..6b45080472e8f18a68f58980dba89c4003ca599d 100644
--- a/content/common/user_agent.cc
+++ b/content/common/user_agent.cc
@@ -232,7 +232,7 @@ std::string BuildUserAgentFromProduct(const std::string& product) {
   std::string os_info;
   base::StringAppendF(&os_info, "%s%s", GetUserAgentPlatform().c_str(),
                       BuildOSCpuInfo(IncludeAndroidBuildNumber::Exclude,
-                                     IncludeAndroidModel::Include)
+                                     IncludeAndroidModel::Exclude)
                           .c_str());
   return BuildUserAgentFromOSAndProduct(os_info, product);
 }
