diff --git a/content/browser/resources/gpu/info_view.ts b/content/browser/resources/gpu/info_view.ts
index 68b8b8e9c946cbe2eadb5b5bc89c8655ad4ffb89..066822fcafa7c43fd52270ce680f78455ef6b858 100644
--- a/content/browser/resources/gpu/info_view.ts
+++ b/content/browser/resources/gpu/info_view.ts
@@ -537,6 +537,7 @@ export class InfoViewElement extends CustomElement {
           value: clientInfo.graphics_backend,
         },
         {description: 'Command Line', value: clientInfo.command_line},
+        {description: 'Executable path', value: clientInfo.executable_path},
       ]);
     } else {
       sections.clientInfo.list.textContent = '... loading ...';
