diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index 36d2eb590daf30025570d5ab16373606b4842c6b..73fe172d4156e3ea3000ac96e0f697d9b31fff15 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -5067,7 +5067,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (render_view_->GetWebView()->MainFrame()->IsWebRemoteFrame())
-    return false;
+    return true;
   WebLocalFrame* main_frame =
       render_view_->GetWebView()->MainFrame()->ToWebLocalFrame();
 
