diff --git a/content/browser/frame_host/navigation_request.cc b/content/browser/frame_host/navigation_request.cc
index c1b56e48583fdb4079fcddb9606ca593b598012a..417347a8e6b39039550f0bcdf6c17d2f12eb252d 100644
--- a/content/browser/frame_host/navigation_request.cc
+++ b/content/browser/frame_host/navigation_request.cc
@@ -1636,6 +1636,11 @@ void NavigationRequest::OnStartChecksComplete(
   headers.MergeFrom(navigation_handle_->TakeModifiedRequestHeaders());
   begin_params_->headers = headers.ToString();
 
+  GetContentClient()->browser()->MaybeHideReferrer(browser_context,
+                                                   common_params_.url,
+                                                   top_document_url,
+                                                   &common_params_.referrer);
+
   loader_ = NavigationURLLoader::Create(
       browser_context->GetResourceContext(), partition,
       std::make_unique<NavigationRequestInfo>(
