diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 00e1452d5443ef29f6be2ddf5bda52fa01c87cf6..b2847d1bf30a6601c87fdf1bbe0a64134f4df0dd 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3710,6 +3710,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
