diff --git a/content/browser/web_contents/web_contents_impl.cc b/content/browser/web_contents/web_contents_impl.cc
index 03b5291540931c40707fbd786645721336c72dae..2baf764bbbaf1a0edbedc5dce323bbf0e87ca9f9 100644
--- a/content/browser/web_contents/web_contents_impl.cc
+++ b/content/browser/web_contents/web_contents_impl.cc
@@ -4904,7 +4904,7 @@ FrameTree* WebContentsImpl::CreateNewWindow(
 
   // Save the window for later if we're not suppressing the opener (since it
   // will be shown immediately).
-  if (!params.opener_suppressed) {
+  if (!params.opener_suppressed && !params.no_javascript_access) {
     if (!is_guest) {
       WebContentsView* new_view = new_contents_impl->view_.get();
 
