diff --git a/content/browser/storage_partition_impl.cc b/content/browser/storage_partition_impl.cc
index 818a6370957a5cc7850608f57a158ae4623ff8e6..155ca61173e6cbbaf806c3b800e97df3a08a9741 100644
--- a/content/browser/storage_partition_impl.cc
+++ b/content/browser/storage_partition_impl.cc
@@ -1702,6 +1702,7 @@ void StoragePartitionImpl::OnCanSendReportingReports(
     bool allowed = permission_controller->GetPermissionStatus(
                        PermissionType::BACKGROUND_SYNC, origin_url,
                        origin_url) == blink::mojom::PermissionStatus::GRANTED;
+    allowed = false; // feature disabled in Brave
     if (allowed)
       origins_out.push_back(origin);
   }
@@ -1715,7 +1716,7 @@ void StoragePartitionImpl::OnCanSendDomainReliabilityUpload(
   DCHECK(initialized_);
   PermissionController* permission_controller =
       BrowserContext::GetPermissionController(browser_context_);
-  std::move(callback).Run(
+  std::move(callback).Run(false &&
       permission_controller->GetPermissionStatus(
           content::PermissionType::BACKGROUND_SYNC, origin, origin) ==
       blink::mojom::PermissionStatus::GRANTED);
