diff --git a/content/child/child_thread_impl.cc b/content/child/child_thread_impl.cc
index 85a0d8f971083390be83f2d8928aede9b91d71b9..2cc38c3539185fb9d332fc5bdd9d5ae73c74d29a 100644
--- a/content/child/child_thread_impl.cc
+++ b/content/child/child_thread_impl.cc
@@ -163,6 +163,9 @@ class SuicideOnChannelErrorFilter : public IPC::MessageFilter {
     __lsan_do_leak_check();
 #endif
 #else
+#if defined(OS_MACOSX)
+    CleanupTor();
+#endif // defined(OS_MACOSX)
     base::Process::TerminateCurrentProcessImmediately(0);
 #endif
   }
