diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 1b2023ffa408c4c644d96a4cd440367730738b78..669a73d833db57d539813d711dff31afdbaffef9 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -71,6 +71,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
