diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 6337b36babbd727073a89e81a1040943a60bbc0f..0622826ee8edcd90c98e42555eb2805958d7708b 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -1012,6 +1012,7 @@ void RenderThreadImpl::InitializeWebKit(mojo::BinderMap* binders) {
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
