diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 9f48c905f4abbb0f2e184299a915232cf6a0f6b0..25560de0e4ba0ee6ec18780dca10c8e1b7ac4902 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
