diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index a164080599beb28449c6bda91a0f7373c6434019..039cb6c7cce6dfa271afbb5b33406738eddc33fc 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -67,6 +67,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
