diff --git a/content/browser/child_process_security_policy_impl.cc b/content/browser/child_process_security_policy_impl.cc
index 2b768dc8f4be2dc465ce24d298d5175887c3c4f0..a256285ef3feed202dbb70ea393de37acfa3b921 100644
--- a/content/browser/child_process_security_policy_impl.cc
+++ b/content/browser/child_process_security_policy_impl.cc
@@ -939,6 +939,9 @@ void ChildProcessSecurityPolicyImpl::GrantWebUIBindings(int child_id,
 
   // Web UI bindings need the ability to request chrome: URLs.
   state->second->GrantRequestScheme(kChromeUIScheme);
+#if defined(BRAVE_CHROMIUM_BUILD)
+  state->second->GrantRequestScheme(kBraveUIScheme);
+#endif
 
   // Web UI pages can contain links to file:// URLs.
   state->second->GrantRequestScheme(url::kFileScheme);
