diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index a17fee9e275e817f97e943a48e36fa9e67340c9d..7de443b32075e0c98a2f081d2b5de12492a689c5 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -2192,6 +2192,12 @@ jumbo_source_set("browser") {
       "//ppapi/proxy:ipc",
       "//ppapi/shared_impl",
     ]
+    if (brave_chromium_build) {
+      sources += [
+        "//brave/content/browser/renderer_host/brave_plugin_registry_impl.cc",
+        "//brave/content/browser/renderer_host/brave_plugin_registry_impl.h",
+      ]
+    }
     if (use_ozone) {
       sources += [ "renderer_host/pepper/pepper_truetype_font_list_ozone.cc" ]
     }
