diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index d8b60778cdb82a56d38ab48c94e4a9f7cd8681a2..fe273c291388e4e2a7e30f1d6760ac78abae5807 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3820,6 +3820,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
