diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index f50b1f7c18ef9802b6bdfac09c30a0a42cfbcd4c..b4541676d5abaf2b78a6e6cc6390b47a762e4ae0 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
