diff --git a/crypto/unexportable_key_software_unsecure.cc b/crypto/unexportable_key_software_unsecure.cc
index add48146309162f5f003fc3958d9d6afe439f9c6..ad501fc831854742508c26110177852e87016b58 100644
--- a/crypto/unexportable_key_software_unsecure.cc
+++ b/crypto/unexportable_key_software_unsecure.cc
@@ -83,6 +83,7 @@ class SoftwareProvider : public UnexportableKeyProvider {
           return algo;
         case SignatureVerifier::SignatureAlgorithm::RSA_PKCS1_SHA1:
         case SignatureVerifier::SignatureAlgorithm::RSA_PSS_SHA256:
+        case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA384:
           continue;  // Not supported
       }
     }
@@ -110,6 +111,7 @@ class SoftwareProvider : public UnexportableKeyProvider {
         }
         case SignatureVerifier::SignatureAlgorithm::RSA_PKCS1_SHA1:
         case SignatureVerifier::SignatureAlgorithm::RSA_PSS_SHA256:
+        case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA384:
           continue;  // Not supported
       }
     }
