diff --git a/crypto/signature_verifier.cc b/crypto/signature_verifier.cc
index f510546c58998e2c4de44b28b00cb9a618ffd41e..c3bdd6587dffa046ac84c6eca30b481983388422 100644
--- a/crypto/signature_verifier.cc
+++ b/crypto/signature_verifier.cc
@@ -44,6 +44,7 @@ bool SignatureVerifier::VerifyInit(SignatureAlgorithm signature_algorithm,
       pkey_type = EVP_PKEY_EC;
       digest = EVP_sha256();
       break;
+    BRAVE_VERIFY_INIT_ALGO_SWITCH
   }
   DCHECK_NE(EVP_PKEY_NONE, pkey_type);
   DCHECK(digest);
