diff --git a/DEPS b/DEPS
index c854104b089685c75f8e1b08f2f7fb9f7da99fe4..cc2c707c022c571fe6111660542ba5d46beab86b 100644
--- a/DEPS
+++ b/DEPS
@@ -4503,7 +4503,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
