diff --git a/DEPS b/DEPS
index 628408c4bce14c05eb4815dde94a894b446d9e35..3555182eea9929dbd76d78084b4b5e79e403ff53 100644
--- a/DEPS
+++ b/DEPS
@@ -4339,7 +4339,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
