diff --git a/DEPS b/DEPS
index 675caac06b97ee2151c0a7f8d727f8bd042a1f86..647439145b3057d484921cff9924c5d0514e051b 100644
--- a/DEPS
+++ b/DEPS
@@ -4322,7 +4322,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
