diff --git a/DEPS b/DEPS
index 0eedd97e47edd350257be8eafb2e2603439177c4..95a5bee19cc05ebb725d72313c8c876a88426457 100644
--- a/DEPS
+++ b/DEPS
@@ -4535,7 +4535,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
