diff --git a/DEPS b/DEPS
index a7a82af82dafb9599e0254b65c5f8596832f2f44..203853a0149addfa45155c7c2253276d30bbeebe 100644
--- a/DEPS
+++ b/DEPS
@@ -4357,7 +4357,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
