diff --git a/DEPS b/DEPS
index 08a44da27d04d154d055ac8a809c1dab4829c611..f3645dc6ed2d91cb207af41a6ab585c9b958eb0b 100644
--- a/DEPS
+++ b/DEPS
@@ -4375,7 +4375,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
