diff --git a/DEPS b/DEPS
index 17c7f93afbfde6a891da302c03bc1ebd82f79c68..fe4204e0997690ae71f17c31409989332b2d274b 100644
--- a/DEPS
+++ b/DEPS
@@ -4357,7 +4357,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
