diff --git a/DEPS b/DEPS
index 034904aa843934d7cbae98d4401ebbd26327d89e..b6117172b5c4b227a2936ddb142fd929e3cb4978 100644
--- a/DEPS
+++ b/DEPS
@@ -4524,7 +4524,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
