diff --git a/DEPS b/DEPS
index 7f12f69b57dfc2e385e079afb38ea2629b70a923..86a33faf54750110f918fcbcbae8319c139ea83a 100644
--- a/DEPS
+++ b/DEPS
@@ -4535,7 +4535,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
