diff --git a/DEPS b/DEPS
index 26ab933d9a66ee4ce160c979d33d817b61669912..9c9d1186dc556ecb5792d9ecdab87aa140c8e125 100644
--- a/DEPS
+++ b/DEPS
@@ -4503,7 +4503,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
