diff --git a/DEPS b/DEPS
index 449d4c64a7d3e7c66b4c20eadef7c29fe44cc5fd..91cc8a70da81e5383f67d32497be75bc2882b2fc 100644
--- a/DEPS
+++ b/DEPS
@@ -4436,7 +4436,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
