diff --git a/DEPS b/DEPS
index 924b5b93e3de6ed60a1a3454cef0173212858881..aabd87bc56987e385611700775aa022ffb3b3641 100644
--- a/DEPS
+++ b/DEPS
@@ -4503,7 +4503,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
