diff --git a/device/usb/usb_device_handle_usbfs.cc b/device/usb/usb_device_handle_usbfs.cc
index 7fc1b19d98895d63c852fc410bdb6f96048a9a44..80b0b2a56cef194fb58a2bba05d8638fec53a3b0 100644
--- a/device/usb/usb_device_handle_usbfs.cc
+++ b/device/usb/usb_device_handle_usbfs.cc
@@ -325,7 +325,7 @@ void UsbDeviceHandleUsbfs::BlockingTaskRunnerHelper::DiscardUrb(
 
   base::ScopedBlockingCall scoped_blocking_call(FROM_HERE,
                                                 base::BlockingType::MAY_BLOCK);
-  HANDLE_EINTR(ioctl(fd_.get(), USBDEVFS_DISCARDURB, &transfer->urb));
+  ANALYZER_ALLOW_UNUSED(HANDLE_EINTR(ioctl(fd_.get(), USBDEVFS_DISCARDURB, &transfer->urb)));
 
   task_runner_->PostTask(FROM_HERE,
                          base::BindOnce(&UsbDeviceHandleUsbfs::UrbDiscarded,
