diff --git a/extensions/browser/extension_event_histogram_value.h b/extensions/browser/extension_event_histogram_value.h
index 5ba39dad92c1daa65691fbc201db83b0d1b8c295..b3b7d03a3d5fd7c8e53a127dc240396da0d3a5b2 100644
--- a/extensions/browser/extension_event_histogram_value.h
+++ b/extensions/browser/extension_event_histogram_value.h
@@ -480,6 +480,20 @@ enum HistogramValue {
   INPUT_IME_ON_INPUT_METHOD_OPTIONS_CHANGED = 458,
   CERTIFICATEPROVIDER_ON_CERTIFICATES_UPDATE_REQUESTED = 459,
   CERTIFICATEPROVIDER_ON_SIGNATURE_REQUESTED = 460,
+
+  BRAVE_START = 600,
+  BRAVE_AD_BLOCKED,
+  BRAVE_WALLET_CREATED,
+  BRAVE_ON_WALLET_PROPERTIES,
+  BRAVE_ON_PUBLISHER_DATA,
+  BRAVE_ON_CURRENT_REPORT,
+  BRAVE_ON_BRAVE_THEME_TYPE_CHANGED,
+  BRAVE_REWARDS_NOTIFICATION_ADDED,
+  BRAVE_REWARDS_NOTIFICATION_DELETED,
+  BRAVE_REWARDS_ALL_NOTIFICATIONS_DELETED,
+  BRAVE_REWARDS_GET_NOTIFICATION,
+  BRAVE_REWARDS_GET_ALL_NOTIFICATIONS,
+  BRAVE_WALLET_FAILED,
   // Last entry: Add new entries above, then run:
   // python tools/metrics/histograms/update_extension_histograms.py
   ENUM_BOUNDARY
