diff --git a/extensions/browser/sandboxed_unpacker.cc b/extensions/browser/sandboxed_unpacker.cc
index 2d1725321cf515a3e6257b86414c7007b7103de7..51eb78f1654975d8623c80a1befc7a4bdc57fb10 100644
--- a/extensions/browser/sandboxed_unpacker.cc
+++ b/extensions/browser/sandboxed_unpacker.cc
@@ -1007,6 +1007,7 @@ absl::optional<base::Value> SandboxedUnpacker::RewriteManifestFile(
   DCHECK(!public_key_.empty());
   base::Value final_manifest = manifest.Clone();
   final_manifest.SetStringKey(manifest_keys::kPublicKey, public_key_);
+  BRAVE_SANDBOXEDUNPACKER_REWRITEMANIFESTFILE
 
   {
     std::string differential_fingerprint;
