diff --git a/extensions/buildflags/buildflags.gni b/extensions/buildflags/buildflags.gni
index 35d9c6cebd548e7895574b3192d31d1c97cede1a..ceb16cb6628145a0fb48b8fc40f3472e184879f1 100644
--- a/extensions/buildflags/buildflags.gni
+++ b/extensions/buildflags/buildflags.gni
@@ -2,8 +2,6 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-import("//build/config/features.gni")
-
 declare_args() {
   enable_extensions = !is_android && !is_ios && !is_fuchsia
 
