diff --git a/extensions/renderer/BUILD.gn b/extensions/renderer/BUILD.gn
index 12cffbca9c17c29d5f67930f64fe61e781a5b229..a8b1a5d25ed8d28696d9368aea0fffcde689d5a4 100644
--- a/extensions/renderer/BUILD.gn
+++ b/extensions/renderer/BUILD.gn
@@ -339,6 +339,8 @@ jumbo_source_set("renderer") {
     "//third_party/cld_3/src/src:cld_3",
   ]
 
+  if (brave_chromium_build) { deps += [ "//brave/extensions/renderer"] }
+
   if (proprietary_codecs && enable_wifi_display) {
     sources += [
       "api/display_source/wifi_display/wifi_display_audio_encoder.cc",
