diff --git a/extensions/BUILD.gn b/extensions/BUILD.gn
index 6ceee52d62aa7d4059ccc1a7e831c57499048b05..e14434155cb5f5b1b7eb76a879120435eead4ddc 100644
--- a/extensions/BUILD.gn
+++ b/extensions/BUILD.gn
@@ -206,38 +206,40 @@ repack("shell_and_test_pak") {
   ]
 }
 
-test("extensions_unittests") {
-  sources = [
-    "test/extensions_unittests_main.cc",
-    "test/logging_timer_unittest.cc",
-  ]
+if (!is_android) {
+  test("extensions_unittests") {
+    sources = [
+      "test/extensions_unittests_main.cc",
+      "test/logging_timer_unittest.cc",
+    ]
 
-  data = [
-    "test/data/",
-    "//chrome/test/data/extensions/",
-    "//components/test/data/cast_certificate/",
-    "$root_out_dir/content_shell.pak",
-    "$root_out_dir/extensions_shell_and_test.pak",
-  ]
+    data = [
+      "test/data/",
+      "//chrome/test/data/extensions/",
+      "//components/test/data/cast_certificate/",
+      "$root_out_dir/content_shell.pak",
+      "$root_out_dir/extensions_shell_and_test.pak",
+    ]
 
-  deps = [
-    ":extensions_resources",
-    ":shell_and_test_pak",
-    ":test_support",
-    "//base/test:test_support",
-    "//content/public/common",
-    "//content/test:test_support",
-    "//extensions/browser:unit_tests",
-    "//extensions/common",
-    "//extensions/common:unit_tests",
-    "//extensions/renderer:unit_tests",
-    "//extensions/shell:unit_tests",
-    "//services/data_decoder:lib",
-    "//services/service_manager/public/cpp/test:test_support",
-    "//ui/gl:test_support",
-  ]
+    deps = [
+      ":extensions_resources",
+      ":shell_and_test_pak",
+      ":test_support",
+      "//base/test:test_support",
+      "//content/public/common",
+      "//content/test:test_support",
+      "//extensions/browser:unit_tests",
+      "//extensions/common",
+      "//extensions/common:unit_tests",
+      "//extensions/renderer:unit_tests",
+      "//extensions/shell:unit_tests",
+      "//services/data_decoder:lib",
+      "//services/service_manager/public/cpp/test:test_support",
+      "//ui/gl:test_support",
+    ]
 
-  data_deps = [ "//third_party/mesa_headers" ]
+    data_deps = [ "//third_party/mesa_headers" ]
+  }
 }
 
 test("extensions_browsertests") {
