diff --git a/extensions/common/BUILD.gn b/extensions/common/BUILD.gn
index d36d73caa35cb84eae02abb0c2a6e45e8efdf5c6..a962961ef355e6ed67dddfc00c29e87b981f8bb3 100644
--- a/extensions/common/BUILD.gn
+++ b/extensions/common/BUILD.gn
@@ -63,6 +63,7 @@ source_set("constants_impl") {
     "//build:chromeos_buildflags",
     "//extensions/common:export_impl",
   ]
+  public_deps += [ "//extensions/buildflags" ]
 
   deps = [ "//build:chromecast_buildflags" ]
 
