diff --git a/extensions/common/mojom/api_permission_id.mojom b/extensions/common/mojom/api_permission_id.mojom
index af312a56135fa922cb3f0296fb765fcc3a62e5eb..b614ae6922cb8da9239fbd29e194cd2dd0c3098c 100644
--- a/extensions/common/mojom/api_permission_id.mojom
+++ b/extensions/common/mojom/api_permission_id.mojom
@@ -254,6 +254,7 @@ enum APIPermissionID {
   kSearch = 228,
   kTabGroups = 229,
   kScripting = 230,
+  kIpfs = 750,
 
   // Add new entries at the end of the enum and be sure to update the
   // "ExtensionPermission3" enum in tools/metrics/histograms/enums.xml
