diff --git a/extensions/browser/BUILD.gn b/extensions/browser/BUILD.gn
index fe870d2157c405dd0de4fdaa67862389c76ce25b..536d9c9cbbe77f20b03f5b300b85ccbfb66dcc72 100644
--- a/extensions/browser/BUILD.gn
+++ b/extensions/browser/BUILD.gn
@@ -10,7 +10,7 @@ import("//rlz/buildflags/buildflags.gni")
 import("//testing/libfuzzer/fuzzer_test.gni")
 import("//third_party/protobuf/proto_library.gni")
 
-if (enable_desktop_android_extensions) {
+if (is_android && enable_extensions_core) {
   import("//build/config/android/rules.gni")
 }
 
@@ -1323,7 +1323,7 @@ source_set("unit_tests") {
   ]
 }
 
-if (enable_desktop_android_extensions) {
+if (is_android && enable_extensions_core) {
   java_cpp_enum("extension_action_enums") {
     sources = [ "extension_action.h" ]
   }
