diff --git a/extensions/browser/extension_event_histogram_value.h b/extensions/browser/extension_event_histogram_value.h
index 8182f3cb061959af03498dcfc0f7fa05325e29c3..1c47d25391dcaf5ee995805d7197a250f262056a 100644
--- a/extensions/browser/extension_event_histogram_value.h
+++ b/extensions/browser/extension_event_histogram_value.h
@@ -517,6 +517,20 @@ enum HistogramValue {
   ENTERPRISE_REMOTE_APPS_ON_REMOTE_APP_LAUNCHED = 495,
   INPUT_METHOD_PRIVATE_ON_CARET_BOUNDS_CHANGED = 496,
   PASSWORDS_PRIVATE_ON_PASSWORD_MANAGER_AUTH_TIMEOUT = 497,
+
+  BRAVE_START = 600,
+  BRAVE_AD_BLOCKED,
+  BRAVE_WALLET_CREATED,
+  BRAVE_ON_WALLET_PROPERTIES,
+  BRAVE_ON_PUBLISHER_DATA,
+  BRAVE_ON_CURRENT_REPORT,
+  BRAVE_ON_BRAVE_THEME_TYPE_CHANGED,
+  BRAVE_REWARDS_NOTIFICATION_ADDED,
+  BRAVE_REWARDS_NOTIFICATION_DELETED,
+  BRAVE_REWARDS_ALL_NOTIFICATIONS_DELETED,
+  BRAVE_REWARDS_GET_NOTIFICATION,
+  BRAVE_REWARDS_GET_ALL_NOTIFICATIONS,
+  BRAVE_WALLET_FAILED,
   // Last entry: Add new entries above, then run:
   // tools/metrics/histograms/update_extension_histograms.py
   ENUM_BOUNDARY
