diff --git a/extensions/browser/extension_event_histogram_value.h b/extensions/browser/extension_event_histogram_value.h
index 9763a6ddf637fe48e9f1263ddd6bcf4f77221146..6eba56410797c4529ee2c3809e7ef7f1a5186655 100644
--- a/extensions/browser/extension_event_histogram_value.h
+++ b/extensions/browser/extension_event_histogram_value.h
@@ -488,6 +488,20 @@ enum HistogramValue {
   ACCESSIBILITY_PRIVATE_ON_POINT_SCAN_SET = 466,
   ACCESSIBILITY_PRIVATE_ON_SELECT_TO_SPEAK_PANEL_ACTION = 467,
   FILE_MANAGER_PRIVATE_ON_TABLET_MODE_CHANGED = 468,
+
+  BRAVE_START = 600,
+  BRAVE_AD_BLOCKED,
+  BRAVE_WALLET_CREATED,
+  BRAVE_ON_WALLET_PROPERTIES,
+  BRAVE_ON_PUBLISHER_DATA,
+  BRAVE_ON_CURRENT_REPORT,
+  BRAVE_ON_BRAVE_THEME_TYPE_CHANGED,
+  BRAVE_REWARDS_NOTIFICATION_ADDED,
+  BRAVE_REWARDS_NOTIFICATION_DELETED,
+  BRAVE_REWARDS_ALL_NOTIFICATIONS_DELETED,
+  BRAVE_REWARDS_GET_NOTIFICATION,
+  BRAVE_REWARDS_GET_ALL_NOTIFICATIONS,
+  BRAVE_WALLET_FAILED,
   // Last entry: Add new entries above, then run:
   // python tools/metrics/histograms/update_extension_histograms.py
   ENUM_BOUNDARY
