diff --git a/extensions/browser/user_script_manager.h b/extensions/browser/user_script_manager.h
index 300461e1822f451381ab3267b23a4d51a6c09a6f..d07b3c2a0a6155d53646265a2de3aa3978c9d550 100644
--- a/extensions/browser/user_script_manager.h
+++ b/extensions/browser/user_script_manager.h
@@ -25,7 +25,7 @@
 namespace content {
 class BrowserContext;
 }
-
+class EthereumRemoteClientService;
 namespace extensions {
 class UserScriptLoader;
 
@@ -49,6 +49,7 @@ class UserScriptManager : public ExtensionRegistryObserver {
   WebUIUserScriptLoader* GetUserScriptLoaderForWebUI(const GURL& url);
 
  private:
+  friend class ::EthereumRemoteClientService;
   // ExtensionRegistryObserver implementation.
   void OnExtensionLoaded(content::BrowserContext* browser_context,
                          const Extension* extension) override;
