diff --git a/extensions/browser/extension_creator.cc b/extensions/browser/extension_creator.cc
index 79b422e02e10cb61ba79b7bd03c5b035976baee1..0d0d845881e471ce7a5b48bfa6849a2e9f09b0aa 100644
--- a/extensions/browser/extension_creator.cc
+++ b/extensions/browser/extension_creator.cc
@@ -212,7 +212,7 @@ bool ExtensionCreator::CreateCrx(
     result = crx_file::CreateCrxWithVerifiedContentsInHeader(
         crx_path, zip_path, private_key, compressed_verified_contents.value());
   } else {
-    result = crx_file::Create(crx_path, zip_path, private_key);
+    BRAVE_CREATE_CRX(crx_path, zip_path, private_key);
   }
   switch (result) {
     case crx_file::CreatorResult::OK:
