diff --git a/extensions/renderer/native_extension_bindings_system.h b/extensions/renderer/native_extension_bindings_system.h
index 16553de1df6e1130c251a91f1d2698512c7c56d9..194a2b3fb6fb7347ce87a7646ccd6dba35a2c9eb 100644
--- a/extensions/renderer/native_extension_bindings_system.h
+++ b/extensions/renderer/native_extension_bindings_system.h
@@ -50,6 +50,7 @@ class NativeExtensionBindingsSystem {
   NativeExtensionBindingsSystem& operator=(
       const NativeExtensionBindingsSystem&) = delete;
 
+  virtual
   ~NativeExtensionBindingsSystem();
 
   // Called when a new ScriptContext is created.
