diff --git a/extensions/browser/extension_util.h b/extensions/browser/extension_util.h
index d439c598861dfd8b6e25f667b2d633608a59154e..3d143b806886baf617966473ee01ee83d7d692c7 100644
--- a/extensions/browser/extension_util.h
+++ b/extensions/browser/extension_util.h
@@ -36,6 +36,8 @@ bool CanBeIncognitoEnabled(const Extension* extension);
 // Returns true if |extension_id| can run in an incognito window.
 bool IsIncognitoEnabled(const std::string& extension_id,
                         content::BrowserContext* context);
+bool IsTorEnabled(const std::string& extension_id,
+                        content::BrowserContext* context);
 
 // Returns true if |extension| can see events and data from another sub-profile
 // (incognito to original profile, or vice versa).
