diff --git a/extensions/browser/updater/update_service.cc b/extensions/browser/updater/update_service.cc
index 316b0581f55ac689af16c8c0773724b14427a141..da8a0697be7e66fe0846ca7311306a1890e8888f 100644
--- a/extensions/browser/updater/update_service.cc
+++ b/extensions/browser/updater/update_service.cc
@@ -33,7 +33,7 @@ namespace {
 UpdateService* update_service_override = nullptr;
 
 // 98% of update checks have 22 or less extensions.
-constexpr size_t kMaxExtensionsPerUpdate = 22;
+constexpr size_t kMaxExtensionsPerUpdate = 1;
 
 void SendUninstallPingCompleteCallback(update_client::Error error) {}
 
