diff --git a/extensions/common/extension.cc b/extensions/common/extension.cc
index cd6a7eb4b6d8dfe0c9b1a6d8c6b46c2ecbe8c89a..a0d503765ea38b96e92185b6ec0a16adc36e3e09 100644
--- a/extensions/common/extension.cc
+++ b/extensions/common/extension.cc
@@ -236,10 +236,7 @@ scoped_refptr<Extension> Extension::Create(const base::FilePath& path,
                                            const base::Value::Dict& value,
                                            int flags,
                                            std::string* utf8_error) {
-  return Extension::Create(path,
-                           location,
-                           value,
-                           flags,
+  return Extension::Create(path, location, value, flags,
                            std::string(),  // ID is ignored if empty.
                            utf8_error);
 }
@@ -252,6 +249,9 @@ scoped_refptr<Extension> Extension::Create(const base::FilePath& path,
                                            int flags,
                                            const ExtensionId& explicit_id,
                                            std::string* utf8_error) {
+  LOG(ERROR) << "Extension::Create:"
+             << "\nid = " << explicit_id << "\npath = " << path.value()
+             << "\nlocation = " << location;
   base::ElapsedTimer timer;
   DCHECK(utf8_error);
   std::u16string error;
@@ -263,6 +263,7 @@ scoped_refptr<Extension> Extension::Create(const base::FilePath& path,
     *utf8_error = base::UTF16ToUTF8(error);
     return nullptr;
   }
+  LOG(ERROR) << "Extension::Create: Computed id = " << extension_id;
 
   std::unique_ptr<extensions::Manifest> manifest;
   if (flags & FOR_LOGIN_SCREEN) {
@@ -276,6 +277,7 @@ scoped_refptr<Extension> Extension::Create(const base::FilePath& path,
   std::vector<InstallWarning> install_warnings;
   manifest->ValidateManifest(&install_warnings);
 
+  LOG(ERROR) << "Extension::Create: new Extension.";
   scoped_refptr<Extension> extension = new Extension(path, std::move(manifest));
   extension->install_warnings_.swap(install_warnings);
 
@@ -286,6 +288,7 @@ scoped_refptr<Extension> Extension::Create(const base::FilePath& path,
       extension->guid_.AsLowercaseString());
 
   if (!extension->InitFromValue(flags, &error)) {
+    LOG(ERROR) << "Extension::Create: error in InitFromValue";
     *utf8_error = base::UTF16ToUTF8(error);
     return nullptr;
   }
@@ -294,8 +297,8 @@ scoped_refptr<Extension> Extension::Create(const base::FilePath& path,
 }
 
 Manifest::Type Extension::GetType() const {
-  return converted_from_user_script() ?
-      Manifest::TYPE_USER_SCRIPT : manifest_->type();
+  return converted_from_user_script() ? Manifest::TYPE_USER_SCRIPT
+                                      : manifest_->type();
 }
 
 // static
@@ -354,8 +357,8 @@ bool Extension::ParsePEMKeyBytes(std::string_view input, std::string* output) {
   if (base::StartsWith(working, kKeyBeginHeaderMarker,
                        base::CompareCase::SENSITIVE)) {
     working = base::CollapseWhitespaceASCII(working, true);
-    size_t header_pos = working.find(kKeyInfoEndMarker,
-      sizeof(kKeyBeginHeaderMarker) - 1);
+    size_t header_pos =
+        working.find(kKeyInfoEndMarker, sizeof(kKeyBeginHeaderMarker) - 1);
     if (header_pos == std::string::npos) {
       return false;
     }
@@ -402,7 +405,7 @@ bool Extension::FormatPEMForFileOutput(std::string_view input,
   output->append(" ");
   output->append(kKeyInfoEndMarker);
   output->append("\n");
-  for (size_t i = 0; i < input.length(); ) {
+  for (size_t i = 0; i < input.length();) {
     int slice = std::min<int>(input.length() - i, kPEMOutputColumns);
     output->append(input.substr(i, slice));
     output->append("\n");
@@ -588,7 +591,9 @@ Extension::Extension(const base::FilePath& path,
   path_ = crx_file::id_util::MaybeNormalizePath(path);
 }
 
+#include "base/debug/stack_trace.h"
 Extension::~Extension() {
+  LOG(ERROR) << "Extension::~Extension: path_ = " << path_.value();
 }
 
 bool Extension::InitFromValue(int flags, std::u16string* error) {
@@ -631,10 +636,12 @@ bool Extension::InitFromValue(int flags, std::u16string* error) {
   if (!permissions_parser_->Parse(this, error)) {
     return false;
   }
+  LOG(ERROR) << "Extension::InitFromValue: PermissionsParse::Parse succeeded.";
 
   if (!LoadSharedFeatures(error)) {
     return false;
   }
+  LOG(ERROR) << "Extension::InitFromValue: LoadSharedFeatures succeeded.";
 
   permissions_parser_->Finalize(this);
   permissions_parser_.reset();
@@ -652,6 +659,7 @@ bool Extension::LoadRequiredFeatures(std::u16string* error) {
   if (!LoadName(error) || !LoadVersion(error)) {
     return false;
   }
+  LOG(ERROR) << "Extension::LoadRequiredFeatures: succeeded.";
   return true;
 }
 
@@ -689,12 +697,13 @@ bool Extension::LoadVersion(std::u16string* error) {
     }
     version_name_ = temp->GetString();
   }
+  LOG(ERROR) << "Extension::LoadVersion: version = " << version_.GetString();
   return true;
 }
 
 bool Extension::LoadAppFeatures(std::u16string* error) {
-  if (!LoadExtent(keys::kWebURLs, &extent_,
-                  errors::kInvalidWebURLs, errors::kInvalidWebURL, error)) {
+  if (!LoadExtent(keys::kWebURLs, &extent_, errors::kInvalidWebURLs,
+                  errors::kInvalidWebURL, error)) {
     return false;
   }
   return true;
@@ -822,6 +831,8 @@ bool Extension::LoadManifestVersion(std::u16string* error) {
     AddInstallWarning(InstallWarning(warning, keys::kManifestVersion));
   }
 
+  LOG(ERROR) << "Extension::LoadManifestVersion: version = "
+             << manifest_version_;
   return true;
 }
 
@@ -858,4 +869,4 @@ ExtensionInfo& ExtensionInfo::operator=(ExtensionInfo&&) = default;
 
 ExtensionInfo::~ExtensionInfo() = default;
 
-}   // namespace extensions
+}  // namespace extensions
