diff --git a/extensions/browser/extension_event_histogram_value.h b/extensions/browser/extension_event_histogram_value.h
index 7e2bc5aa5bd4b1b2438ec5ff13d787216b9de187..e33f638adf88edcc091b15d798afa05973484fe9 100644
--- a/extensions/browser/extension_event_histogram_value.h
+++ b/extensions/browser/extension_event_histogram_value.h
@@ -513,6 +513,20 @@ enum HistogramValue {
   WEB_AUTHENTICATION_PROXY_ON_REMOTE_SESSION_STATE_CHANGE = 491,
   LOGIN_ON_REQUEST_EXTERNAL_LOGOUT = 492,
   LOGIN_ON_EXTERNAL_LOGOUT_DONE = 493,
+
+  BRAVE_START = 600,
+  BRAVE_AD_BLOCKED,
+  BRAVE_WALLET_CREATED,
+  BRAVE_ON_WALLET_PROPERTIES,
+  BRAVE_ON_PUBLISHER_DATA,
+  BRAVE_ON_CURRENT_REPORT,
+  BRAVE_ON_BRAVE_THEME_TYPE_CHANGED,
+  BRAVE_REWARDS_NOTIFICATION_ADDED,
+  BRAVE_REWARDS_NOTIFICATION_DELETED,
+  BRAVE_REWARDS_ALL_NOTIFICATIONS_DELETED,
+  BRAVE_REWARDS_GET_NOTIFICATION,
+  BRAVE_REWARDS_GET_ALL_NOTIFICATIONS,
+  BRAVE_WALLET_FAILED,
   // Last entry: Add new entries above, then run:
   // tools/metrics/histograms/update_extension_histograms.py
   ENUM_BOUNDARY
