diff --git a/extensions/renderer/native_extension_bindings_system.h b/extensions/renderer/native_extension_bindings_system.h
index 86d8f20e2981fbfea7a96ff19b4a04b6ac9bb1c8..bb6c802a94b9a8d1018d7b2f3b223f6e9ee41c1e 100644
--- a/extensions/renderer/native_extension_bindings_system.h
+++ b/extensions/renderer/native_extension_bindings_system.h
@@ -34,7 +34,7 @@ class NativeExtensionBindingsSystem {
  public:
   explicit NativeExtensionBindingsSystem(
       std::unique_ptr<IPCMessageSender> ipc_message_sender);
-  ~NativeExtensionBindingsSystem();
+  virtual ~NativeExtensionBindingsSystem();
 
   // Called when a new ScriptContext is created.
   // Initializes the bindings for a newly created |context|.
