diff --git a/extensions/browser/extension_creator.cc b/extensions/browser/extension_creator.cc
index db0302eee231a1174dcac31f1da544e7684f1ba8..9fa6ed0ba91aa777b91c8765f3fcb4168001ba26 100644
--- a/extensions/browser/extension_creator.cc
+++ b/extensions/browser/extension_creator.cc
@@ -215,7 +215,7 @@ bool ExtensionCreator::CreateCrx(
     result = crx_file::CreateCrxWithVerifiedContentsInHeader(
         crx_path, zip_path, private_key, compressed_verified_contents.value());
   } else {
-    result = crx_file::Create(crx_path, zip_path, private_key);
+    BRAVE_CREATE_CRX(crx_path, zip_path, private_key);
   }
   switch (result) {
     case crx_file::CreatorResult::OK:
