diff --git a/extensions/browser/updater/update_service.cc b/extensions/browser/updater/update_service.cc
index 967c7b0b8dea1babb0a480c4cdeb85b76ba7b0a5..e7d575c3c5a6682801f595cb501335769917e8e7 100644
--- a/extensions/browser/updater/update_service.cc
+++ b/extensions/browser/updater/update_service.cc
@@ -33,7 +33,7 @@ namespace {
 UpdateService* update_service_override = nullptr;
 
 // 98% of update checks have 22 or less extensions.
-constexpr size_t kMaxExtensionsPerUpdate = 22;
+constexpr size_t kMaxExtensionsPerUpdate = 1;
 
 void SendUninstallPingCompleteCallback(update_client::Error error) {}
 
