diff --git a/extensions/browser/api/web_request/extension_web_request_event_router.cc b/extensions/browser/api/web_request/extension_web_request_event_router.cc
index 8a4a20e81ac3893f983a740eeecc98335183ad85..55fbbe10b9febf09d3e1cbb453912197673848b1 100644
--- a/extensions/browser/api/web_request/extension_web_request_event_router.cc
+++ b/extensions/browser/api/web_request/extension_web_request_event_router.cc
@@ -1398,6 +1398,7 @@ WebRequestEventRouter::OnAuthRequired(content::BrowserContext* browser_context,
                                          *request)) {
     return AuthRequiredResponse::AUTH_REQUIRED_RESPONSE_NO_ACTION;
   }
+  BRAVE_EXTENSION_WEB_REQUEST_EVENT_ROUTER_ON_AUTH_REQUIRED
 
   int extra_info_spec = 0;
   RawListeners listeners = GetMatchingListeners(
