diff --git a/extensions/browser/extension_util.cc b/extensions/browser/extension_util.cc
index af63beaf829691d02a4139adb9eebb0ee941ccb9..20ed58a2c7fd5560e3854c5b94d4404d337e3326 100644
--- a/extensions/browser/extension_util.cc
+++ b/extensions/browser/extension_util.cc
@@ -49,6 +49,7 @@ bool CanBeIncognitoEnabled(const Extension* extension) {
 
 bool IsIncognitoEnabled(const std::string& extension_id,
                         content::BrowserContext* context) {
+  BRAVE_IS_INCOGNITO_ENABLED
   const Extension* extension =
       ExtensionRegistry::Get(context)->GetExtensionById(
           extension_id, ExtensionRegistry::ENABLED);
@@ -76,7 +77,7 @@ bool CanCrossIncognito(const Extension* extension,
   // extensions only see events for a matching profile.
   CHECK(extension);
   return IsIncognitoEnabled(extension->id(), context) &&
-         !IncognitoInfo::IsSplitMode(extension);
+         !IncognitoInfo::IsSplitMode(extension, context->IsTor());
 }
 
 const std::string& GetPartitionDomainForExtension(const Extension* extension) {
