diff --git a/extensions/common/manifest_handlers/mime_types_handler.h b/extensions/common/manifest_handlers/mime_types_handler.h
index 22dfd725f130ebbd8096f7a37174d2c1ce50ff4e..9afcf52ef12e170e588e5f90a644e350c4e99bc2 100644
--- a/extensions/common/manifest_handlers/mime_types_handler.h
+++ b/extensions/common/manifest_handlers/mime_types_handler.h
@@ -17,6 +17,7 @@ class MimeTypesHandler {
  public:
   // Returns list of extensions' ids that are allowed to use MIME type filters.
   static std::vector<std::string> GetMIMETypeWhitelist();
+  static std::vector<std::string> BraveGetMIMETypeWhitelist();
 
   static MimeTypesHandler* GetHandler(const extensions::Extension* extension);
 
