diff --git a/extensions/browser/BUILD.gn b/extensions/browser/BUILD.gn
index 6cac8701bd87ab4a4572f42d58671c3b33861cbd..0ddff030f91e5cc7d66f519432b081ec1d6d7ff5 100644
--- a/extensions/browser/BUILD.gn
+++ b/extensions/browser/BUILD.gn
@@ -407,6 +407,7 @@ jumbo_source_set("browser_sources") {
     "//third_party/blink/public/mojom/frame",
     "//ui/display",
   ]
+  if (brave_chromium_build) { deps += [ "//brave/extensions/browser"] }
 
   public_deps = [
     "//content/public/common",
