diff --git a/extensions/browser/api/web_request/extension_web_request_event_router.cc b/extensions/browser/api/web_request/extension_web_request_event_router.cc
index c2bfb7b8d18c4e93d4bdd3df938c6d663545c875..b0e5e01d5f36555d20247e62036a19c11eaa09bc 100644
--- a/extensions/browser/api/web_request/extension_web_request_event_router.cc
+++ b/extensions/browser/api/web_request/extension_web_request_event_router.cc
@@ -1468,6 +1468,7 @@ WebRequestEventRouter::OnAuthRequired(content::BrowserContext* browser_context,
                                          *request)) {
     return AuthRequiredResponse::AUTH_REQUIRED_RESPONSE_NO_ACTION;
   }
+  BRAVE_EXTENSION_WEB_REQUEST_EVENT_ROUTER_ON_AUTH_REQUIRED
 
   int extra_info_spec = 0;
   RawListeners listeners = GetMatchingListeners(
