diff --git a/extensions/browser/api/socket/tcp_socket.cc b/extensions/browser/api/socket/tcp_socket.cc
index eba3d2d74de01dcf083c9065519242e31c6ab0d8..4a574a069325289d1da090d88fc5e34bf2e9f8b7 100644
--- a/extensions/browser/api/socket/tcp_socket.cc
+++ b/extensions/browser/api/socket/tcp_socket.cc
@@ -153,8 +153,16 @@ void TCPSocket::Disconnect(bool socket_destroying) {
   client_socket_.reset();
   server_socket_.reset();
   listen_callback_.Reset();
-  connect_callback_.Reset();
-  accept_callback_.Reset();
+  if (connect_callback_) {
+    std::move(connect_callback_)
+        .Run(net::ERR_CONNECTION_CLOSED);
+  }
+  if (accept_callback_) {
+    std::move(accept_callback_)
+        .Run(net::ERR_CONNECTION_CLOSED, mojo::NullRemote(), absl::nullopt,
+            mojo::ScopedDataPipeConsumerHandle(),
+            mojo::ScopedDataPipeProducerHandle());
+  }
   // TODO(devlin): Should we do this for all callbacks?
   if (read_callback_) {
     std::move(read_callback_)
