diff --git a/extensions/browser/api/socket/tcp_socket.cc b/extensions/browser/api/socket/tcp_socket.cc
index bb4854c195702f4421366cc3776c788edcc61a62..816eb8559b5cdc0ccdf33f2462be388a124a8d03 100644
--- a/extensions/browser/api/socket/tcp_socket.cc
+++ b/extensions/browser/api/socket/tcp_socket.cc
@@ -155,8 +155,16 @@ void TCPSocket::Disconnect(bool socket_destroying) {
   client_socket_.reset();
   server_socket_.reset();
   listen_callback_.Reset();
-  connect_callback_.Reset();
-  accept_callback_.Reset();
+  if (connect_callback_) {
+    std::move(connect_callback_)
+        .Run(net::ERR_CONNECTION_CLOSED);
+  }
+  if (accept_callback_) {
+    std::move(accept_callback_)
+        .Run(net::ERR_CONNECTION_CLOSED, nullptr, base::nullopt,
+            mojo::ScopedDataPipeConsumerHandle(),
+            mojo::ScopedDataPipeProducerHandle());
+  }
   // TODO(devlin): Should we do this for all callbacks?
   if (read_callback_) {
     std::move(read_callback_)
