diff --git a/extensions/browser/BUILD.gn b/extensions/browser/BUILD.gn
index e7c3057b0e907e5d881cdf8fd9fb7f28b3b4ca84..2e05237cc6873dcecbffb02a85b04ad6d7f4e897 100644
--- a/extensions/browser/BUILD.gn
+++ b/extensions/browser/BUILD.gn
@@ -10,7 +10,7 @@ import("//rlz/buildflags/buildflags.gni")
 import("//testing/libfuzzer/fuzzer_test.gni")
 import("//third_party/protobuf/proto_library.gni")
 
-if (enable_desktop_android_extensions) {
+if (is_android && enable_extensions_core) {
   import("//build/config/android/rules.gni")
 }
 
@@ -1324,7 +1324,7 @@ source_set("unit_tests") {
   ]
 }
 
-if (enable_desktop_android_extensions) {
+if (is_android && enable_extensions_core) {
   java_cpp_enum("extension_action_enums") {
     sources = [ "extension_action.h" ]
   }
