diff --git a/extensions/renderer/native_extension_bindings_system.h b/extensions/renderer/native_extension_bindings_system.h
index 0b4503b0991dfbf15cb908c86ec0601eddfd98f0..ad3ec87f05ab97c5a5277d36a9075982ad822764 100644
--- a/extensions/renderer/native_extension_bindings_system.h
+++ b/extensions/renderer/native_extension_bindings_system.h
@@ -40,7 +40,7 @@ class NativeExtensionBindingsSystem {
   NativeExtensionBindingsSystem& operator=(
       const NativeExtensionBindingsSystem&) = delete;
 
-  ~NativeExtensionBindingsSystem();
+  virtual ~NativeExtensionBindingsSystem();
 
   // Called when a new ScriptContext is created.
   // Initializes the bindings for a newly created |context|.
