diff --git a/extensions/common/permissions/permissions_data.cc b/extensions/common/permissions/permissions_data.cc
index ffdce9beaa7c207df5ab304dbcf4d97c631139ca..ef3c853a5faaa2f58ced1ef6097644a20f6d86e5 100644
--- a/extensions/common/permissions/permissions_data.cc
+++ b/extensions/common/permissions/permissions_data.cc
@@ -649,6 +649,7 @@ PermissionsData::PageAccess PermissionsData::CanRunOnPage(
       return PageAccess::kDenied;
     }
   }
+  BRAVE_PERMISSION_DATA_CAN_RUN_ON_PAGE
 
   if (tab_url_patterns && tab_url_patterns->MatchesURL(document_url))
     return PageAccess::kAllowed;
