diff --git a/extensions/test/data/sockets_tcp_server/api/background.js b/extensions/test/data/sockets_tcp_server/api/background.js
index 8a2ff5ee95c74b1cc4e9a9e6bf62db889fe83309..f25b1e7dead44c84bc3019103a95a3c9101f9a91 100644
--- a/extensions/test/data/sockets_tcp_server/api/background.js
+++ b/extensions/test/data/sockets_tcp_server/api/background.js
@@ -43,6 +43,11 @@ var testSocketListening = function() {
   function onServerSocketCreate(socketInfo) {
     console.log("Server socket created: sd=" + socketInfo.socketId);
     socketId = socketInfo.socketId;
+    chrome.sockets.tcpServer.getInfo(socketId, onGetInfo);
+  }
+
+  function onGetInfo(socketInfo) {
+    chrome.test.assertEq(socketInfo.socketId, socketId);
     chrome.sockets.tcpServer.listen(socketId, address, port, onListen);
   }
 
