diff --git a/extensions/browser/extension_creator.cc b/extensions/browser/extension_creator.cc
index de7b81651327fcb3e2e0b1c884c5137d0a168efb..ad63cb0946277e683f5d5b0c5895bf179997711b 100644
--- a/extensions/browser/extension_creator.cc
+++ b/extensions/browser/extension_creator.cc
@@ -229,7 +229,7 @@ bool ExtensionCreator::CreateCrx(
     result = crx_file::CreateCrxWithVerifiedContentsInHeader(
         crx_path, zip_path, private_key, compressed_verified_contents.value());
   } else {
-    result = crx_file::Create(crx_path, zip_path, private_key);
+    BRAVE_CREATE_CRX(crx_path, zip_path, private_key);
   }
   switch (result) {
     case crx_file::CreatorResult::OK:
