diff --git a/extensions/browser/user_script_manager.h b/extensions/browser/user_script_manager.h
index 4b4d25e68c06835c1f7d84daa43080606527b297..b030c08839ba70144b5810019889708a501dc0ca 100644
--- a/extensions/browser/user_script_manager.h
+++ b/extensions/browser/user_script_manager.h
@@ -25,7 +25,7 @@
 namespace content {
 class BrowserContext;
 }
-
+class EthereumRemoteClientService;
 namespace extensions {
 class UserScriptLoader;
 
@@ -61,6 +61,7 @@ class UserScriptManager : public ExtensionRegistryObserver {
   void SetUserScriptPrefEnabled(const ExtensionId& extension, bool enabled);
 
  private:
+  friend class ::EthereumRemoteClientService;
   // ExtensionRegistryObserver implementation.
   void OnExtensionWillBeInstalled(content::BrowserContext* browser_context,
                                   const Extension* extension,
