diff --git a/extensions/browser/user_script_manager.h b/extensions/browser/user_script_manager.h
index 9a440b918e92f89e69822b735d9d9656b77d3b1f..d2219fd6e25239a4a705454e8898a22bc222e2a2 100644
--- a/extensions/browser/user_script_manager.h
+++ b/extensions/browser/user_script_manager.h
@@ -26,7 +26,7 @@
 namespace content {
 class BrowserContext;
 }
-
+class EthereumRemoteClientService;
 namespace extensions {
 class UserScriptLoader;
 
@@ -83,6 +83,7 @@ class UserScriptManager : public ExtensionRegistryObserver {
   void SetUserScriptPrefEnabled(const ExtensionId& extension_id, bool enabled);
 
  private:
+  friend class ::EthereumRemoteClientService;
   // ExtensionRegistryObserver implementation.
   void OnExtensionWillBeInstalled(content::BrowserContext* browser_context,
                                   const Extension* extension,
