diff --git a/extensions/browser/api/socket/tcp_socket.cc b/extensions/browser/api/socket/tcp_socket.cc
index 0e91a9b9799414be0dfb7a3f8fb33866c1edf4f7..765012467f5b8b695d4b5cde65eb6740a929dc6a 100644
--- a/extensions/browser/api/socket/tcp_socket.cc
+++ b/extensions/browser/api/socket/tcp_socket.cc
@@ -156,8 +156,16 @@ void TCPSocket::Disconnect(bool socket_destroying) {
   client_socket_.reset();
   server_socket_.reset();
   listen_callback_.Reset();
-  connect_callback_.Reset();
-  accept_callback_.Reset();
+  if (connect_callback_) {
+    std::move(connect_callback_)
+        .Run(net::ERR_CONNECTION_CLOSED);
+  }
+  if (accept_callback_) {
+    std::move(accept_callback_)
+        .Run(net::ERR_CONNECTION_CLOSED, mojo::NullRemote(), base::nullopt,
+            mojo::ScopedDataPipeConsumerHandle(),
+            mojo::ScopedDataPipeProducerHandle());
+  }
   // TODO(devlin): Should we do this for all callbacks?
   if (read_callback_) {
     std::move(read_callback_)
