diff --git a/extensions/browser/BUILD.gn b/extensions/browser/BUILD.gn
index 86076a326933d014272a35c2bdb04bb8e3bcb746..7b49cdedf0febe4a212556ddfcc426c3b05d0b3f 100644
--- a/extensions/browser/BUILD.gn
+++ b/extensions/browser/BUILD.gn
@@ -1,6 +1,7 @@
 # Copyright 2014 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
+import("//brave/build/config/brave_build.gni")

 import("//build/config/features.gni")
 import("//build/config/jumbo.gni")
@@ -406,6 +407,7 @@ jumbo_source_set("browser_sources") {
     "//third_party/blink/public/mojom/frame",
     "//ui/display",
   ]
+  if (brave_chromium_build) { deps += [ "//brave/extensions/browser"] }
 
   public_deps = [
     "//content/public/common",
