diff --git a/extensions/browser/extension_event_histogram_value.h b/extensions/browser/extension_event_histogram_value.h
index 77e13de501e051658aa06d20fe155c6048ed0bff..b6679a65c54a552ee94f1bd375b60e18fd9112d1 100644
--- a/extensions/browser/extension_event_histogram_value.h
+++ b/extensions/browser/extension_event_histogram_value.h
@@ -464,6 +464,20 @@ enum HistogramValue {
   PRINTING_METRICS_ON_PRINT_JOB_FINISHED = 442,
   AUTOTESTPRIVATE_ON_CLIPBOARD_DATA_CHANGED = 443,
   AUTOFILL_PRIVATE_ON_PERSONAL_DATA_CHANGED = 444,
+
+  BRAVE_START = 600,
+  BRAVE_AD_BLOCKED,
+  BRAVE_WALLET_CREATED,
+  BRAVE_ON_WALLET_PROPERTIES,
+  BRAVE_ON_PUBLISHER_DATA,
+  BRAVE_ON_CURRENT_REPORT,
+  BRAVE_ON_BRAVE_THEME_TYPE_CHANGED,
+  BRAVE_REWARDS_NOTIFICATION_ADDED,
+  BRAVE_REWARDS_NOTIFICATION_DELETED,
+  BRAVE_REWARDS_ALL_NOTIFICATIONS_DELETED,
+  BRAVE_REWARDS_GET_NOTIFICATION,
+  BRAVE_REWARDS_GET_ALL_NOTIFICATIONS,
+  BRAVE_WALLET_FAILED,
   // Last entry: Add new entries above, then run:
   // python tools/metrics/histograms/update_extension_histograms.py
   ENUM_BOUNDARY
