diff --git a/extensions/renderer/BUILD.gn b/extensions/renderer/BUILD.gn
index b9f2b1d71d5f8499fc289030dff320d148fa392c..c709f73b44273c650b47d74cb1ed78794f64d6e5 100644
--- a/extensions/renderer/BUILD.gn
+++ b/extensions/renderer/BUILD.gn
@@ -340,6 +340,8 @@ jumbo_source_set("renderer") {
     "//third_party/cld_3/src/src:cld_3",
   ]
 
+  if (brave_chromium_build) { deps += [ "//brave/extensions/renderer"] }
+
   if (proprietary_codecs && enable_wifi_display) {
     sources += [
       "api/display_source/wifi_display/wifi_display_audio_encoder.cc",
