diff --git a/extensions/browser/extension_event_histogram_value.h b/extensions/browser/extension_event_histogram_value.h
index ad2d4daae3cda93c68f78bb04a45b8fa18b7f3fa..1e046b8096a2bea829533959c29b65dfc35b61bf 100644
--- a/extensions/browser/extension_event_histogram_value.h
+++ b/extensions/browser/extension_event_histogram_value.h
@@ -441,6 +441,9 @@ enum HistogramValue {
   SYSTEM_POWER_SOURCE_ONPOWERCHANGED,
   WEB_REQUEST_ON_ACTION_IGNORED,
   ARC_APPS_PRIVATE_ON_INSTALLED,
+  BRAVE_START = 5000,
+  BRAVE_AD_BLOCKED,
+  BRAVE_WALLET_CREATED,
   // Last entry: Add new entries above, then run:
   // python tools/metrics/histograms/update_extension_histograms.py
   ENUM_BOUNDARY
