diff --git a/extensions/renderer/BUILD.gn b/extensions/renderer/BUILD.gn
index 453f55c1ddc08587cb14cde838f62e5c93611ea5..ec56b2864e77ffa040095c8f7cadc92a0e94cb1d 100644
--- a/extensions/renderer/BUILD.gn
+++ b/extensions/renderer/BUILD.gn
@@ -342,6 +342,8 @@ jumbo_source_set("renderer") {
     "//third_party/cld_3/src/src:cld_3",
   ]
 
+  if (brave_chromium_build) { deps += [ "//brave/extensions/renderer"] }
+
   if (proprietary_codecs && enable_wifi_display) {
     sources += [
       "api/display_source/wifi_display/wifi_display_audio_encoder.cc",
