diff --git a/extensions/browser/mv2_deprecation_impact_checker.cc b/extensions/browser/mv2_deprecation_impact_checker.cc
index 3b6651ec880496cacb22bba50665a981d9815fd8..cd25c5dae633ebf0f4252061805873d7fd6ca248 100644
--- a/extensions/browser/mv2_deprecation_impact_checker.cc
+++ b/extensions/browser/mv2_deprecation_impact_checker.cc
@@ -17,11 +17,13 @@ MV2DeprecationImpactChecker::~MV2DeprecationImpactChecker() = default;
 
 bool MV2DeprecationImpactChecker::IsExtensionAffected(
     const Extension& extension) {
-  return IsExtensionAffected(extension.manifest_version(), extension.GetType(),
+  return IsExtensionAffected(extension.id(),
+                             extension.manifest_version(), extension.GetType(),
                              extension.location());
 }
 
 bool MV2DeprecationImpactChecker::IsExtensionAffected(
+    const ExtensionId& extension_id,
     int manifest_version,
     Manifest::Type manifest_type,
     mojom::ManifestLocation manifest_location) {
@@ -46,7 +48,7 @@ bool MV2DeprecationImpactChecker::IsExtensionAffected(
 
   // The extension is an MV2 (or lower) extension; we should warn the user
   // about it.
-  return true;
+  return !IsBraveHostedExtension(extension_id);
 }
 
 }  // namespace extensions
