diff --git a/extensions/browser/api/web_request/extension_web_request_event_router.cc b/extensions/browser/api/web_request/extension_web_request_event_router.cc
index d299090f8917cede993f82a762f694dc288bdace..768a323fde7deaa58e2d09b895e12f1cd8596c6a 100644
--- a/extensions/browser/api/web_request/extension_web_request_event_router.cc
+++ b/extensions/browser/api/web_request/extension_web_request_event_router.cc
@@ -1396,6 +1396,7 @@ WebRequestEventRouter::OnAuthRequired(content::BrowserContext* browser_context,
                                          *request)) {
     return AuthRequiredResponse::AUTH_REQUIRED_RESPONSE_NO_ACTION;
   }
+  BRAVE_EXTENSION_WEB_REQUEST_EVENT_ROUTER_ON_AUTH_REQUIRED
 
   int extra_info_spec = 0;
   RawListeners listeners = GetMatchingListeners(
