diff --git a/google_apis/google_api_keys.cc b/google_apis/google_api_keys.cc
index b16448cce200df05dd1febb89306fe0de743b6dc..7f07e18e5472dbaf227a2669659a7e3169fc4592 100644
--- a/google_apis/google_api_keys.cc
+++ b/google_apis/google_api_keys.cc
@@ -200,6 +200,7 @@ class APIKeyCache {
 #if defined(OS_IOS)
   void set_api_key(const std::string& api_key) { api_key_ = api_key; }
 #endif
+  void set_api_key_for_testing(const std::string& api_key) { api_key_ = api_key; }
   std::string api_key_non_stable() const { return api_key_non_stable_; }
   std::string api_key_remoting() const { return api_key_remoting_; }
   std::string api_key_sharing() const { return api_key_sharing_; }
