diff --git a/google_update/BUILD.gn b/google_update/BUILD.gn
index 970510fd7c69833934b2eac157200f0265c91547..a874eaff3be772e1e07bf1740bc8b1a9d8dbf1b2 100644
--- a/google_update/BUILD.gn
+++ b/google_update/BUILD.gn
@@ -5,5 +5,5 @@
 import("//build/toolchain/win/midl.gni")
 
 midl("google_update") {
-  sources = [ "google_update_idl.idl" ]
+  sources = [ "//brave/chromium_src/google_update/google_update_idl.idl" ]
 }
