diff --git a/google_apis/google_api_keys.cc b/google_apis/google_api_keys.cc
index 28774f16e21f0e2871a78d2027c24e4a9b950de0..5a9f318d334ab1b1c9e22d9fb1bf892dfa714fdf 100644
--- a/google_apis/google_api_keys.cc
+++ b/google_apis/google_api_keys.cc
@@ -209,6 +209,7 @@ class APIKeyCache {
 #if defined(OS_IOS) || defined(OS_FUCHSIA)
   void set_api_key(const std::string& api_key) { api_key_ = api_key; }
 #endif
+  void set_api_key_for_testing(const std::string& api_key) { api_key_ = api_key; }
   std::string api_key_non_stable() const { return api_key_non_stable_; }
   std::string api_key_remoting() const { return api_key_remoting_; }
   std::string api_key_sharing() const { return api_key_sharing_; }
