diff --git a/google_apis/google_api_keys.cc b/google_apis/google_api_keys.cc
index 6d187b51dee3c20602da134fa35945814547412f..e5cb9936efe79499ed2339eb8ad7e0e6495e8305 100644
--- a/google_apis/google_api_keys.cc
+++ b/google_apis/google_api_keys.cc
@@ -192,6 +192,7 @@ class APIKeyCache {
 #if defined(OS_IOS)
   void set_api_key(const std::string& api_key) { api_key_ = api_key; }
 #endif
+  void set_api_key_for_testing(const std::string& api_key) { api_key_ = api_key; }
   std::string api_key_non_stable() const { return api_key_non_stable_; }
   std::string api_key_remoting_ftl() const { return api_key_remoting_ftl_; }
 
