diff --git a/google_apis/google_api_keys.cc b/google_apis/google_api_keys.cc
index 81bbb6751b41d6d8376fb0f904120d1a7cd41664..02e91ef0bac2c69d4383e1f84077d29d966cdac8 100644
--- a/google_apis/google_api_keys.cc
+++ b/google_apis/google_api_keys.cc
@@ -208,6 +208,7 @@ class APIKeyCache {
 #if BUILDFLAG(IS_IOS) || BUILDFLAG(IS_FUCHSIA)
   void set_api_key(const std::string& api_key) { api_key_ = api_key; }
 #endif
+  void set_api_key_for_testing(const std::string& api_key) { api_key_ = api_key; }
   std::string api_key_non_stable() const { return api_key_non_stable_; }
   std::string api_key_remoting() const { return api_key_remoting_; }
   std::string api_key_sharing() const { return api_key_sharing_; }
