diff --git a/ios/build/bots/scripts/test_runner.py b/ios/build/bots/scripts/test_runner.py
index 155b723cfff45b5e733f8571a33034f53d304891..ffb3d068aa468dadc0cabde8e076e0b15fb18c61 100644
--- a/ios/build/bots/scripts/test_runner.py
+++ b/ios/build/bots/scripts/test_runner.py
@@ -404,6 +404,7 @@ class TestRunner(object):
   @staticmethod
   def remove_proxy_settings():
     """removes any proxy settings which may remain from a previous run."""
+    return
     LOGGER.info('Removing any proxy settings.')
     network_services = subprocess.check_output(
         ['networksetup',
