diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index ffa14125d038bcdc7dc892032416287d538ad481..818ece015c64ac3c3ab32a94b131568dcfbb66e7 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -880,8 +880,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
