diff --git a/ios/chrome/browser/tabs/model/BUILD.gn b/ios/chrome/browser/tabs/model/BUILD.gn
index 17ae0a2ff3c0c9263f97dc359b127d5fbb225611..e80daa974f68c5efc831c1e726b586155dfca205 100644
--- a/ios/chrome/browser/tabs/model/BUILD.gn
+++ b/ios/chrome/browser/tabs/model/BUILD.gn
@@ -48,6 +48,7 @@ source_set("tabs_internal") {
     ":synced_tabs",
     ":tab_helper_attacher",
     "//base",
+    "//brave/components/brave_shields/core/common",
     "//components/autofill/ios/form_util",
     "//components/breadcrumbs/core:status",
     "//components/commerce/ios/browser",
