diff --git a/ios/chrome/browser/BUILD.gn b/ios/chrome/browser/BUILD.gn
index f2f0b50b2fbc1abdd17ea888947a09860eed7c9a..0b6e028be88fc09ca6b764aa5ebff3545ed94387 100644
--- a/ios/chrome/browser/BUILD.gn
+++ b/ios/chrome/browser/BUILD.gn
@@ -90,6 +90,7 @@ source_set("browser") {
     "//ios/chrome/browser/paths",
     "//ios/chrome/browser/prefs:pref_names",
   ]
+  public_deps += [ "//brave/ios/browser:chromium_src" ]
   allow_circular_includes_from = [
     "//ios/chrome/browser/sync/glue",
     "//ios/chrome/browser/browser_state",
