diff --git a/ios/web/webui/url_data_manager_ios_backend.mm b/ios/web/webui/url_data_manager_ios_backend.mm
index be4a5c37801e381e7cbc18e349364f15bf335925..ed99d870e3b33f7b3bfd68a9f861c05bfcff73bd 100644
--- a/ios/web/webui/url_data_manager_ios_backend.mm
+++ b/ios/web/webui/url_data_manager_ios_backend.mm
@@ -351,7 +351,7 @@ void URLRequestChromeJob::GetResponseInfo(net::HttpResponseInfo* info) {
   // that is compatible with a given WebUI URL, and append it to the existing
   // response headers.
   if (add_content_security_policy_) {
-    std::string base = kChromeURLContentSecurityPolicyHeaderBase;
+    std::string base;
     base.append(content_security_policy_object_source_);
     base.append(content_security_policy_frame_source_);
     info->headers->AddHeader(kContentSecurityPolicy, base);
