diff --git a/ios/chrome/browser/profile/model/keyed_service_factories.mm b/ios/chrome/browser/profile/model/keyed_service_factories.mm
index cc85bc146253d45fd7ad3d7d907db8cdb89c8f7e..2842656bb8b7545734219ec9822fd356a0b30c54 100644
--- a/ios/chrome/browser/profile/model/keyed_service_factories.mm
+++ b/ios/chrome/browser/profile/model/keyed_service_factories.mm
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 #import "ios/chrome/browser/profile/model/keyed_service_factories.h"
+#import "brave/ios/browser/profile/model/brave_keyed_service_factories.h"
 
 #import "ios/chrome/browser/account_settings/model/ios_account_setting_service_factory.h"
 #import "ios/chrome/browser/affiliations/model/ios_chrome_affiliation_service_factory.h"
@@ -248,6 +249,7 @@
 // required to create services that uses ServiceCreation::kCreateWithProfile
 // can be created with the profile.
 void EnsureProfileKeyedServiceFactoriesBuilt() {
+  [&]() -> void {
   // Keep this list alphabetized -- namespaced factories first, followed by
   // non-namespaced factories.
   actor::ActorServiceFactory::GetInstance();
@@ -490,4 +492,6 @@ void EnsureProfileKeyedServiceFactoriesBuilt() {
   if (web::features::IsCobaltEnabled()) {
     ios::provider::EnsureCobaltProfileKeyedServiceFactoriesBuilt();
   }
+  }();
+  brave::EnsureProfileKeyedServiceFactoriesBuilt();
 }
