diff --git a/ios/chrome/browser/shared/model/prefs/BUILD.gn b/ios/chrome/browser/shared/model/prefs/BUILD.gn
index b84873198fbcf4b783520a108d8e5530bea3043c..213f490f1d88fc0fb61462f112009ad5e638dcda 100644
--- a/ios/chrome/browser/shared/model/prefs/BUILD.gn
+++ b/ios/chrome/browser/shared/model/prefs/BUILD.gn
@@ -138,6 +138,7 @@ source_set("browser_prefs") {
     "//ios/web/common:features",
     "//ui/base",
   ]
+  import("//brave/chromium_src/ios/chrome/browser/shared/model/prefs/sources.gni") deps += brave_browser_prefs_deps
 }
 
 source_set("unit_tests") {
