diff --git a/ios/chrome/browser/sync/model/sync_service_factory_unittest.mm b/ios/chrome/browser/sync/model/sync_service_factory_unittest.mm
index bf9a609f6ff31861ead25583c22325fc8986c06b..d562765c2a1918367f1a35c0ea36c221b0cbdfd0 100644
--- a/ios/chrome/browser/sync/model/sync_service_factory_unittest.mm
+++ b/ios/chrome/browser/sync/model/sync_service_factory_unittest.mm
@@ -55,7 +55,7 @@ class SyncServiceFactoryTest : public PlatformTest {
  protected:
   // Returns the collection of default datatypes.
   syncer::DataTypeSet DefaultDatatypes() {
-    static_assert(64 == syncer::GetNumDataTypes(),
+    static_assert(64 + 1 == syncer::GetNumDataTypes(),
                   "When adding a new type, you probably want to add it here as "
                   "well (assuming it is already enabled).");
 
