diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index cd66ff1a0416573b3b6a789be181ae70cc3761df..4e7f89a78e543a982b23c818c94fbdc88ea61ddc 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -809,8 +809,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
