diff --git a/ios/web_view/BUILD.gn b/ios/web_view/BUILD.gn
index ad1314a46740124186248000ed93f734ec73bcb0..97354ade551cd41a506e839fe1d7a2f75a0b2eea 100644
--- a/ios/web_view/BUILD.gn
+++ b/ios/web_view/BUILD.gn
@@ -429,6 +429,7 @@ source_set("web_view_sources") {
   public_deps = ios_web_view_deps
   deps = [ "//components/version_info:channel" ]
   configs += [ ":config" ]
+  import("//brave/ios/web_view/sources.gni") sources -= brave_ios_web_view_excluded_sources public_deps -= brave_ios_web_view_excluded_public_deps
 }
 
 action("web_view_umbrella_header") {
