diff --git a/ios/build/bots/scripts/test_runner.py b/ios/build/bots/scripts/test_runner.py
index af94616e3c1d1cc651da487d740993cd547a4b27..5030dd40103b00ba4e856b9f97a0bb35a5691a60 100644
--- a/ios/build/bots/scripts/test_runner.py
+++ b/ios/build/bots/scripts/test_runner.py
@@ -445,6 +445,7 @@ class TestRunner(object):
   @staticmethod
   def remove_proxy_settings():
     """removes any proxy settings which may remain from a previous run."""
+    return
     LOGGER.info('Removing any proxy settings.')
     network_services = (
       subprocess.check_output(['networksetup', '-listallnetworkservices'])
