diff --git a/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm b/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm
index a25031b415769c9ec8e17d33b83a3fb6c12121a2..71282974130cf4b5512201f3adee798abfe4ed2c 100644
--- a/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm
+++ b/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm
@@ -826,6 +826,7 @@ constexpr CGFloat kBatchUploadSymbolPointSize = 22.;
     case syncer::UserSelectableType::kSavedTabGroups:
     case syncer::UserSelectableType::kProductComparison:
     case syncer::UserSelectableType::kCookies:
+    case syncer::UserSelectableType::kAIChat:
       NOTREACHED();
   }
   DCHECK_NE(itemType, 0);
