diff --git a/ios/chrome/browser/shared/model/prefs/BUILD.gn b/ios/chrome/browser/shared/model/prefs/BUILD.gn
index e725f6463af098489c7c0fe447138ce7fa9e70b9..23d96689ce270b3c437e98599cfc6785d479958f 100644
--- a/ios/chrome/browser/shared/model/prefs/BUILD.gn
+++ b/ios/chrome/browser/shared/model/prefs/BUILD.gn
@@ -150,6 +150,7 @@ source_set("browser_prefs") {
   if (target_environment != "catalyst") {
     deps += [ "//ios/chrome/browser/default_browser/model/default_status" ]
   }
+  import("//brave/chromium_src/ios/chrome/browser/shared/model/prefs/sources.gni") deps += brave_browser_prefs_deps
 }
 
 source_set("unit_tests") {
