diff --git a/ios/chrome/app/main_controller.mm b/ios/chrome/app/main_controller.mm
index c16aba6a824303e54b361619c345bcf517c11134..0b2e9f33098992f41ab8dcf731e150d86ed6ab9e 100644
--- a/ios/chrome/app/main_controller.mm
+++ b/ios/chrome/app/main_controller.mm
@@ -664,7 +664,9 @@ void MainControllerAuthenticationServiceDelegate::ClearBrowsingData(
   [self.appState addAgent:[[IncognitoUsageAppStateAgent alloc] init]];
   [self.appState addAgent:[[FirstRunAppAgent alloc] init]];
   [self.appState addAgent:[[CertificatePolicyAppAgent alloc] init]];
+#if BUILDFLAG(IOS_CREDENTIAL_PROVIDER_ENABLED)
   [self.appState addAgent:[[CredentialProviderAppAgent alloc] init]];
+#endif
 }
 
 #pragma mark - Property implementation.
