diff --git a/ios/web_view/BUILD.gn b/ios/web_view/BUILD.gn
index 7da119cff870de363007868017bdbd5b3c5f7531..11055fa04f32a624dac707e2e48b455bea88adc5 100644
--- a/ios/web_view/BUILD.gn
+++ b/ios/web_view/BUILD.gn
@@ -445,6 +445,7 @@ source_set("web_view_sources") {
     "//components/version_info:channel",
   ]
   configs += [ ":config" ]
+  import("//brave/ios/web_view/sources.gni") sources -= brave_ios_web_view_excluded_sources public_deps -= brave_ios_web_view_excluded_public_deps
 }
 
 action("web_view_umbrella_header") {
