diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index b185d2a47574cc8ed4112d69737512c387fdb43b..644c615db2b6f0ede49f400f8ecc6cff5215e97f 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -737,8 +737,7 @@ ios_app_bundle("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
