diff --git a/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm b/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm
index dde823f3a7b9cd4656374307088d23bc08556ee5..9081cbe233ee413613be1192b6ab72ceac0672fd 100644
--- a/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm
+++ b/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm
@@ -808,6 +808,7 @@ constexpr CGFloat kBatchUploadSymbolPointSize = 22.;
     case syncer::UserSelectableType::kSavedTabGroups:
     case syncer::UserSelectableType::kProductComparison:
     case syncer::UserSelectableType::kCookies:
+    case syncer::UserSelectableType::kAIChat:
       NOTREACHED();
   }
   DCHECK_NE(itemType, 0);
