diff --git a/ios/chrome/browser/prefs/browser_prefs.mm b/ios/chrome/browser/prefs/browser_prefs.mm
index bb087266b80c80aa377ec8a2eb85f5edcad79cb4..25c877d436c80d60e9979c2d6633a73fbd8fd468 100644
--- a/ios/chrome/browser/prefs/browser_prefs.mm
+++ b/ios/chrome/browser/prefs/browser_prefs.mm
@@ -198,6 +198,7 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
   // times a user should see autofill branding animation after installation.
   registry->RegisterIntegerPref(
       prefs::kAutofillBrandingIconAnimationRemainingCountPrefName, 2);
+  BRAVE_REGISTER_LOCAL_STATE_PREFS
 }
 
 void RegisterBrowserStatePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -328,6 +329,7 @@ void RegisterBrowserStatePrefs(user_prefs::PrefRegistrySyncable* registry) {
       base::to_underlying(
           prerender_prefs::NetworkPredictionSetting::kEnabledWifiOnly),
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
+  BRAVE_REGISTER_BROWSER_STATE_PREFS
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -387,4 +389,5 @@ void MigrateObsoleteBrowserStatePrefs(PrefService* prefs) {
     if (!account_id.empty())
       prefs->SetString(prefs::kGoogleServicesLastGaiaId, account_id);
   }
+  BRAVE_MIGRATE_OBSOLETE_BROWSER_STATE_PREFS
 }
