diff --git a/ios/chrome/browser/prefs/browser_prefs.mm b/ios/chrome/browser/prefs/browser_prefs.mm
index 14b1aa155424c223942b56d0189e7b7707509dd5..f6ae9e72de0bc25d86b3407328235716a1598b83 100644
--- a/ios/chrome/browser/prefs/browser_prefs.mm
+++ b/ios/chrome/browser/prefs/browser_prefs.mm
@@ -208,6 +208,7 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
       prefs::kAutofillBrandingIconAnimationRemainingCountPrefName, 2);
 
   registry->RegisterDictionaryPref(kLocalConsentsDictionary);
+  BRAVE_REGISTER_LOCAL_STATE_PREFS
 }
 
 void RegisterBrowserStatePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -338,6 +339,7 @@ void RegisterBrowserStatePrefs(user_prefs::PrefRegistrySyncable* registry) {
       base::to_underlying(
           prerender_prefs::NetworkPredictionSetting::kEnabledWifiOnly),
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
+  BRAVE_REGISTER_BROWSER_STATE_PREFS
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -400,4 +402,5 @@ void MigrateObsoleteBrowserStatePrefs(PrefService* prefs) {
     if (!account_id.empty())
       prefs->SetString(prefs::kGoogleServicesLastGaiaId, account_id);
   }
+  BRAVE_MIGRATE_OBSOLETE_BROWSER_STATE_PREFS
 }
