diff --git a/ios/chrome/browser/shared/model/prefs/BUILD.gn b/ios/chrome/browser/shared/model/prefs/BUILD.gn
index 4fb690bc75a8deb9687d56f8fc16b35f393e3501..1681a1299e84ba079dc80a4243f8abf8d6bee6a5 100644
--- a/ios/chrome/browser/shared/model/prefs/BUILD.gn
+++ b/ios/chrome/browser/shared/model/prefs/BUILD.gn
@@ -149,6 +149,7 @@ source_set("browser_prefs") {
   if (target_environment != "catalyst") {
     deps += [ "//ios/chrome/browser/default_browser/model/default_status" ]
   }
+  import("//brave/chromium_src/ios/chrome/browser/shared/model/prefs/sources.gni") deps += brave_browser_prefs_deps
 }
 
 source_set("unit_tests") {
