diff --git a/ios/chrome/browser/flags/BUILD.gn b/ios/chrome/browser/flags/BUILD.gn
index 0ee67696782dfe93eac762be93fadcaac0d6b98f..060f73c44527dc17376c3bb3a23c3452bcc79f7e 100644
--- a/ios/chrome/browser/flags/BUILD.gn
+++ b/ios/chrome/browser/flags/BUILD.gn
@@ -106,6 +106,7 @@ source_set("flags") {
   if (ios_enable_screen_time) {
     deps += [ "//ios/chrome/browser/screen_time/model:feature_flags" ]
   }
+  import("//brave/ios/browser/flags/sources.gni") deps += brave_flags_deps
 }
 
 source_set("system_flags") {
