diff --git a/ios/chrome/browser/flags/BUILD.gn b/ios/chrome/browser/flags/BUILD.gn
index b3e3ccd48cd5798047cf160025d3dd9898e10849..589e2b0868745254383db49f7e6e866f2b103576 100644
--- a/ios/chrome/browser/flags/BUILD.gn
+++ b/ios/chrome/browser/flags/BUILD.gn
@@ -118,6 +118,7 @@ source_set("flags") {
     "//ios/web/common:web_view_creation_util",
     "//ios/web/public",
   ]
+  import("//brave/ios/browser/flags/sources.gni") deps += brave_flags_deps
 }
 
 source_set("system_flags") {
