diff --git a/ios/build/bots/scripts/test_runner.py b/ios/build/bots/scripts/test_runner.py
index f66023dbc07d2bc867f89b8769d5bc0f091fc628..2309fa4a28a3f054c07adc64897df054eaa3a276 100644
--- a/ios/build/bots/scripts/test_runner.py
+++ b/ios/build/bots/scripts/test_runner.py
@@ -404,6 +404,7 @@ class TestRunner(object):
   @staticmethod
   def remove_proxy_settings():
     """removes any proxy settings which may remain from a previous run."""
+    return
     LOGGER.info('Removing any proxy settings.')
     network_services = subprocess.check_output(
         ['networksetup',
