diff --git a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
index 14d507bcb782cf096cf4e2f4bfd2e76c715b1329..d179ef4ad3a3f764a0ea4fd3b1f7675e87bdd228 100644
--- a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
+++ b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
@@ -492,6 +492,7 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
   // Deprecated 08/2026.
   registry->RegisterTimePref(kWaitingForMultiProfileForcedMigrationTimestamp,
                              base::Time());
+  brave::RegisterLocalStatePrefs(registry);
 }
 
 void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -942,6 +943,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   // Deprecated 09/2026.
   registry->RegisterDictionaryPref(kSigninHasAcceptedManagementDialog);
   registry->RegisterBooleanPref(kFingerprintingProtectionEnabled, false);
+  brave::RegisterBrowserStatePrefs(registry);
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -973,6 +975,7 @@ void MigrateObsoleteLocalStatePrefs(PrefService* prefs) {
 
   // Added 08/2026.
   prefs->ClearPref(kWaitingForMultiProfileForcedMigrationTimestamp);
+  brave::MigrateObsoleteLocalStatePrefs(prefs);
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -1027,6 +1030,7 @@ void MigrateObsoleteProfilePrefs(PrefService* prefs) {
   // Deprecated 09/2026.
   prefs->ClearPref(kSigninHasAcceptedManagementDialog);
   prefs->ClearPref(kFingerprintingProtectionEnabled);
+  brave::MigrateObsoleteProfilePrefs(prefs);
 }
 
 void MigrateObsoleteUserDefault() {
