diff --git a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
index 35c368e0cac75dbbed3dfccbca3369c7523b8f3b..b7a76a085635bbca8e1ea01991c436fe758c983f 100644
--- a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
+++ b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
@@ -508,6 +508,7 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
   registry->RegisterBooleanPref(kMetricsConsentRestructureFeatureState, false);
   registry->RegisterTimePref(kObsoleteManagementPlatformLastLogTime,
                              base::Time());
+  brave::RegisterLocalStatePrefs(registry);
 }
 
 void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -991,6 +992,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   // Deprecated 07/2026.
   registry->RegisterTimePref(kObsoleteManagementProfileLastLogTime,
                              base::Time());
+  brave::RegisterBrowserStatePrefs(registry);
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -1022,6 +1024,7 @@ void MigrateObsoleteLocalStatePrefs(PrefService* prefs) {
 
   // Added 07/2026.
   prefs->ClearPref(kObsoleteManagementPlatformLastLogTime);
+  brave::MigrateObsoleteLocalStatePrefs(prefs);
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -1098,6 +1101,7 @@ void MigrateObsoleteProfilePrefs(PrefService* prefs) {
 
   // Added 07/2026.
   prefs->ClearPref(kObsoleteManagementProfileLastLogTime);
+  brave::MigrateObsoleteProfilePrefs(prefs);
 }
 
 void MigrateObsoleteUserDefault() {
