diff --git a/ios/chrome/browser/shared/model/prefs/BUILD.gn b/ios/chrome/browser/shared/model/prefs/BUILD.gn
index 0c505cc2acf9950cf30d44c66d1612e6beed204c..ba1b6f0e910f3fcbfd7d81476b4c0f5f0f8c1a7a 100644
--- a/ios/chrome/browser/shared/model/prefs/BUILD.gn
+++ b/ios/chrome/browser/shared/model/prefs/BUILD.gn
@@ -153,6 +153,7 @@ source_set("browser_prefs") {
   if (target_environment != "catalyst") {
     deps += [ "//ios/chrome/browser/default_browser/model/default_status" ]
   }
+  import("//brave/chromium_src/ios/chrome/browser/shared/model/prefs/sources.gni") deps += brave_browser_prefs_deps
 }
 
 source_set("unit_tests") {
