diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 423f1511fa1ed9e101d4484a0fc41d7bdf7ffb41..b185c74cae3c81edbfeed5bea9096913c14d23aa 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -964,8 +964,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
