diff --git a/ios/chrome/browser/shared/model/prefs/BUILD.gn b/ios/chrome/browser/shared/model/prefs/BUILD.gn
index f54fced81bb556a0a602afc66cd977096f453139..5eb8ce2c16f20ceba028d894ef98c31266835cf6 100644
--- a/ios/chrome/browser/shared/model/prefs/BUILD.gn
+++ b/ios/chrome/browser/shared/model/prefs/BUILD.gn
@@ -128,6 +128,7 @@ source_set("browser_prefs") {
     "//ios/web/common:features",
     "//ui/base",
   ]
+  import("//brave/chromium_src/ios/chrome/browser/shared/model/prefs/sources.gni") deps += brave_browser_prefs_deps
 }
 
 source_set("unit_tests") {
