diff --git a/ios/web/web_state/web_state_impl_realized_web_state.mm b/ios/web/web_state/web_state_impl_realized_web_state.mm
index 3b0ff0ccc7fb673927be540bae3d9cb1ffc7692f..44bc327bed5cfc847202bea64dc3de7dab6132d1 100644
--- a/ios/web/web_state/web_state_impl_realized_web_state.mm
+++ b/ios/web/web_state/web_state_impl_realized_web_state.mm
@@ -1102,7 +1102,7 @@ std::unique_ptr<WebUIIOS> WebStateImpl::RealizedWebState::CreateWebUIIOS(
   if (!factory) {
     return nullptr;
   }
-  std::unique_ptr<WebUIIOS> web_ui = std::make_unique<WebUIIOSImpl>(owner_);
+  std::unique_ptr<WebUIIOS> web_ui = std::make_unique<WebUIIOSImpl>(owner_, url.GetHost());
   auto controller = factory->CreateWebUIIOSControllerForURL(web_ui.get(), url);
   if (!controller) {
     return nullptr;
