diff --git a/ios/chrome/browser/sync/model/sync_service_factory.mm b/ios/chrome/browser/sync/model/sync_service_factory.mm
index 4d14061c75603c8579701f49603d5a347c5d62a2..3210285bda9bbc6b6c129d0f7893bb1ce0badddb 100644
--- a/ios/chrome/browser/sync/model/sync_service_factory.mm
+++ b/ios/chrome/browser/sync/model/sync_service_factory.mm
@@ -221,7 +221,7 @@ std::unique_ptr<KeyedService> BuildSyncService(ProfileIOS* profile) {
   init_params.debug_identifier = profile->GetProfileName();
   init_params.os_crypt_async = GetApplicationContext()->GetOSCryptAsync();
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize(CreateControllers(profile, sync_service.get()));
 
   password_manager::PasswordReceiverService* password_receiver_service =
