diff --git a/ios/chrome/browser/sync/sync_service_factory.mm b/ios/chrome/browser/sync/sync_service_factory.mm
index 2387cb1c7a528c8384475b6b8c855383af0d8555..77f24183ebade6ad6003687cb4151cb0d2fc0ba8 100644
--- a/ios/chrome/browser/sync/sync_service_factory.mm
+++ b/ios/chrome/browser/sync/sync_service_factory.mm
@@ -149,7 +149,7 @@ std::unique_ptr<KeyedService> SyncServiceFactory::BuildServiceInstanceFor(
       policy_connector ? policy_connector->GetPolicyService() : nullptr;
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize();
 
   // Hook `sync_service` into PersonalDataManager (a circular dependency).
