diff --git a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
index d89dbd5ec0ff9192c4355238a5b7b9192b6d00a0..b4c98c2bff30e34594f635574f61f71641945acc 100644
--- a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
+++ b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
@@ -674,6 +674,7 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
 
   // Deprecated 09/2025.
   registry->RegisterBooleanPref(prefs::kBottomOmnibox, false);
+  BRAVE_REGISTER_LOCAL_STATE_PREFS
 }
 
 void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -1126,6 +1127,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   // instead.
   registry->RegisterBooleanPref(prefs::kHomeCustomizationMagicStackEnabled,
                                 true);
+  BRAVE_REGISTER_BROWSER_STATE_PREFS
 }
 
 // This method should be periodically pruned of year+ old migrations.
