diff --git a/ios/web/web_state/ui/BUILD.gn b/ios/web/web_state/ui/BUILD.gn
index ed74a7a416959ff2338bf46c8801cabd2290f319..04c116a3ae0f67d7554c36e3336b6ce98797ef43 100644
--- a/ios/web/web_state/ui/BUILD.gn
+++ b/ios/web/web_state/ui/BUILD.gn
@@ -88,6 +88,7 @@ source_set("ui") {
     "crw_wk_ui_handler.mm",
     "crw_wk_ui_handler_delegate.h",
   ]
+  import("//brave/ios/web/sources.gni") deps += brave_ios_web_web_state_ui_deps
 }
 
 source_set("web_controller_header") {
