diff --git a/ios/chrome/browser/prefs/browser_prefs.mm b/ios/chrome/browser/prefs/browser_prefs.mm
index f10103c31ab77f251a0fe166a53339341c6527b7..334177aa482a03cdc987f2c3e56744b0c043b24e 100644
--- a/ios/chrome/browser/prefs/browser_prefs.mm
+++ b/ios/chrome/browser/prefs/browser_prefs.mm
@@ -272,6 +272,7 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
 
   registry->RegisterBooleanPref(
       prefs::kIosCredentialProviderPromoHasRegisteredWithPromoManager, false);
+  BRAVE_REGISTER_LOCAL_STATE_PREFS
 }
 
 void RegisterBrowserStatePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -412,6 +413,7 @@ void RegisterBrowserStatePrefs(user_prefs::PrefRegistrySyncable* registry) {
 
   // Register pref used to determine if the Price Tracking UI has been shown.
   registry->RegisterBooleanPref(prefs::kPriceNotificationsHasBeenShown, false);
+  BRAVE_REGISTER_BROWSER_STATE_PREFS
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -557,4 +559,5 @@ void MigrateObsoleteBrowserStatePrefs(PrefService* prefs) {
   if (prefs->FindPreference(kUserClassifierLastTimeToUseSuggestions)) {
     prefs->ClearPref(kUserClassifierLastTimeToUseSuggestions);
   }
+  BRAVE_MIGRATE_OBSOLETE_BROWSER_STATE_PREFS
 }
