diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index e3d20ed372c4a9fb50dd12fc61bcb1451c0f36c7..8746cf09dc3e837a57c3c41a641e2c112b788f4e 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -890,8 +890,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
