diff --git a/ios/chrome/browser/favicon/favicon_loader.mm b/ios/chrome/browser/favicon/favicon_loader.mm
index febf7b0da8ec3e1e74f481a404a38d1d5db52458..e8c53c4570c1da132d66df6d52ba9b3f42ba99a5 100644
--- a/ios/chrome/browser/favicon/favicon_loader.mm
+++ b/ios/chrome/browser/favicon/favicon_loader.mm
@@ -49,7 +49,7 @@ const net::NetworkTrafficAnnotationTag kTrafficAnnotation =
 
 FaviconLoader::FaviconLoader(favicon::LargeIconService* large_icon_service)
     : large_icon_service_(large_icon_service),
-      favicon_cache_([[NSCache alloc] init]) {}
+      favicon_cache_(nullptr) {}
 FaviconLoader::~FaviconLoader() {}
 
 // TODO(pinkerton): How do we update the favicon if it's changed on the web?
