diff --git a/ios/chrome/browser/BUILD.gn b/ios/chrome/browser/BUILD.gn
index 831c25ca668afe10a0cebb4d5a4919592ae32183..7a88d0caf467112b524aaa218d14b8cb595984b9 100644
--- a/ios/chrome/browser/BUILD.gn
+++ b/ios/chrome/browser/BUILD.gn
@@ -130,6 +130,7 @@ source_set("browser") {
     ":pref_names",
     ":utils",
   ]
+  public_deps += [ "//brave/ios/browser:chromium_src" ]
   allow_circular_includes_from = [
     "//ios/chrome/browser/sync/glue",
     "//ios/chrome/browser/browser_state",
