diff --git a/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm b/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm
index a85d2d77d361754afc73fca7c5ce98d5fc8fe7bb..91f9febeda67076d1c432148f017f915e58a7489 100644
--- a/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm
+++ b/ios/chrome/browser/settings/manage_sync/coordinator/manage_sync_settings_mediator.mm
@@ -826,6 +826,7 @@ constexpr CGFloat kBatchUploadSymbolPointSize = 22.;
     case syncer::UserSelectableType::kSavedTabGroups:
     case syncer::UserSelectableType::kProductComparison:
     case syncer::UserSelectableType::kCookies:
+    case syncer::UserSelectableType::kAIChat:
       NOTREACHED();
   }
   DCHECK_NE(itemType, 0);
