diff --git a/ios/chrome/browser/BUILD.gn b/ios/chrome/browser/BUILD.gn
index 2d805833397abbdd8c34defd49d90a867173dcd9..2f4c123c07f00820eecdc2fe00b6cc43174bb5a7 100644
--- a/ios/chrome/browser/BUILD.gn
+++ b/ios/chrome/browser/BUILD.gn
@@ -130,6 +130,7 @@ source_set("browser") {
     ":pref_names",
     ":utils",
   ]
+  public_deps += [ "//brave/ios/browser:chromium_src" ]
   allow_circular_includes_from = [
     "//ios/chrome/browser/sync/glue",
     "//ios/chrome/browser/browser_state",
