diff --git a/ios/chrome/browser/shared/model/prefs/BUILD.gn b/ios/chrome/browser/shared/model/prefs/BUILD.gn
index 5e7050c0dcc3c6ff75aab3c7d32f8b516301bcc1..65c4939440b54befe2b58cc1cad5d0258b647a3f 100644
--- a/ios/chrome/browser/shared/model/prefs/BUILD.gn
+++ b/ios/chrome/browser/shared/model/prefs/BUILD.gn
@@ -127,6 +127,7 @@ source_set("browser_prefs") {
     "//ios/web/common:features",
     "//ui/base",
   ]
+  import("//brave/chromium_src/ios/chrome/browser/shared/model/prefs/sources.gni") deps += brave_browser_prefs_deps
 }
 
 source_set("unit_tests") {
