diff --git a/ios/chrome/browser/shared/model/prefs/BUILD.gn b/ios/chrome/browser/shared/model/prefs/BUILD.gn
index fa41d04a8697f77f52e0de2ef868ec6cd9a1a6c9..67f10a19406ba8bf53c19a25fe56efe9b13d7eaa 100644
--- a/ios/chrome/browser/shared/model/prefs/BUILD.gn
+++ b/ios/chrome/browser/shared/model/prefs/BUILD.gn
@@ -146,6 +146,7 @@ source_set("browser_prefs") {
   if (target_environment != "catalyst") {
     deps += [ "//ios/chrome/browser/default_browser/model/default_status" ]
   }
+  import("//brave/chromium_src/ios/chrome/browser/shared/model/prefs/sources.gni") deps += brave_browser_prefs_deps
 }
 
 source_set("unit_tests") {
