diff --git a/ios/chrome/browser/flags/BUILD.gn b/ios/chrome/browser/flags/BUILD.gn
index 990424a791d5c5a86fc3d30c0be6e8bdeed88073..798f8f828c273622ae104241198ad178c84b7bc2 100644
--- a/ios/chrome/browser/flags/BUILD.gn
+++ b/ios/chrome/browser/flags/BUILD.gn
@@ -126,6 +126,7 @@ source_set("flags") {
   if (ios_enable_screen_time) {
     deps += [ "//ios/chrome/browser/screen_time/model:feature_flags" ]
   }
+  import("//brave/ios/browser/flags/sources.gni") deps += brave_flags_deps
 }
 
 source_set("system_flags") {
