diff --git a/ios/chrome/browser/BUILD.gn b/ios/chrome/browser/BUILD.gn
index ba8282bb573ba40ba80cc23c048a200e65397deb..ab4f14e716aef804a3c3344f3acc5292dc17436e 100644
--- a/ios/chrome/browser/BUILD.gn
+++ b/ios/chrome/browser/BUILD.gn
@@ -102,6 +102,7 @@ source_set("browser") {
     "//ios/chrome/browser/paths",
     "//ios/chrome/browser/prefs:pref_names",
   ]
+  public_deps += [ "//brave/ios/browser:chromium_src" ]
   allow_circular_includes_from = [
     "//ios/chrome/browser/sync/glue",
     "//ios/chrome/browser/browser_state",
