diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 9f5b4ac69ed9dbde6b622adda6bbb13888997f62..eecd238824448edd69fb09873a85c97913690750 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -735,8 +735,7 @@ ios_app_bundle("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
