diff --git a/ios/chrome/browser/flags/BUILD.gn b/ios/chrome/browser/flags/BUILD.gn
index c21300755d99b35904352abb7fde533048bd29fe..3091c2a2615e661e004060e81718e34769d0428d 100644
--- a/ios/chrome/browser/flags/BUILD.gn
+++ b/ios/chrome/browser/flags/BUILD.gn
@@ -105,6 +105,7 @@ source_set("flags") {
   if (ios_enable_screen_time) {
     deps += [ "//ios/chrome/browser/screen_time/model:feature_flags" ]
   }
+  deps += [ "//brave/components/brave_rewards/common/buildflags" ]
 }
 
 source_set("system_flags") {
