diff --git a/ios/chrome/browser/net/model/ios_chrome_network_delegate.h b/ios/chrome/browser/net/model/ios_chrome_network_delegate.h
index a55a99a576e84e61d4bf847f0ba28e0e8eaf8b74..fcd644c9f6785aa6cbba1b2206e62affcfda7258 100644
--- a/ios/chrome/browser/net/model/ios_chrome_network_delegate.h
+++ b/ios/chrome/browser/net/model/ios_chrome_network_delegate.h
@@ -29,6 +29,7 @@ class IOSChromeNetworkDelegate : public net::NetworkDelegateImpl {
   ~IOSChromeNetworkDelegate() override;
 
  private:
+  BRAVE_IOS_CHROME_NETWORK_DELEGATE_H
   bool OnCancelURLRequestWithPolicyViolatingReferrerHeader(
       const net::URLRequest& request,
       const GURL& target_url,
