diff --git a/ios/chrome/browser/flags/about_flags.mm b/ios/chrome/browser/flags/about_flags.mm
index b08d322a327daf57924649bba80b614788e3109d..684a2fbc5e4fae45b2c74248109181291f126897 100644
--- a/ios/chrome/browser/flags/about_flags.mm
+++ b/ios/chrome/browser/flags/about_flags.mm
@@ -1350,6 +1350,7 @@ const FeatureEntry::FeatureVariation kCrossDeviceSigninVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 constexpr auto kFeatureEntries = std::to_array<flags_ui::FeatureEntry>({
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
     {"build-external-privacy-context",
      flag_descriptions::kBuildExternalPrivacyContextName,
      flag_descriptions::kBuildExternalPrivacyContextDescription,
