diff --git a/ios/chrome/browser/profile/model/BUILD.gn b/ios/chrome/browser/profile/model/BUILD.gn
index f18204e345126e4bcddb54112de4ec92529263ab..e8bb27bb674ea968830c9562fbf858456ff29a2f 100644
--- a/ios/chrome/browser/profile/model/BUILD.gn
+++ b/ios/chrome/browser/profile/model/BUILD.gn
@@ -280,6 +280,7 @@ source_set("keyed_service_factories") {
     "//ios/public/provider/chrome/browser/cobalt:cobalt_api",
     "//ios/web/common:features",
   ]
+  deps += [ "//brave/ios/browser/profile/model:keyed_service_factories" ]
 
   if (ios_enable_credential_provider_extension) {
     deps += [ "//ios/chrome/browser/credential_provider/model" ]
