diff --git a/ios/chrome/browser/BUILD.gn b/ios/chrome/browser/BUILD.gn
index f1e3638408e7e5a48bc6177ba5c5174e1f454c44..f0a9ec28451716852dc2bf6ce08e2fd388ee535c 100644
--- a/ios/chrome/browser/BUILD.gn
+++ b/ios/chrome/browser/BUILD.gn
@@ -96,6 +96,7 @@ source_set("browser") {
     "//ios/chrome/browser/paths",
     "//ios/chrome/browser/prefs:pref_names",
   ]
+  public_deps += [ "//brave/ios/browser:chromium_src" ]
   allow_circular_includes_from = [
     "//ios/chrome/browser/sync/glue",
     "//ios/chrome/browser/browser_state",
