diff --git a/ios/chrome/browser/flags/BUILD.gn b/ios/chrome/browser/flags/BUILD.gn
index 923e60f18cc00c26ff511f596373929f0a073c09..d27e23ba853ad8d7e558f822aba86f92234dc5c7 100644
--- a/ios/chrome/browser/flags/BUILD.gn
+++ b/ios/chrome/browser/flags/BUILD.gn
@@ -119,6 +119,7 @@ source_set("flags") {
   if (ios_enable_screen_time) {
     deps += [ "//ios/chrome/browser/screen_time/model:feature_flags" ]
   }
+  import("//brave/ios/browser/flags/sources.gni") deps += brave_flags_deps
 }
 
 source_set("system_flags") {
