diff --git a/media/blink/key_system_config_selector.cc b/media/blink/key_system_config_selector.cc
index 0acaf77db4d0c5cc8c8119f43730c65c12836d8a..5d33d674bbb735989ab35e7b604854648e5200fe 100644
--- a/media/blink/key_system_config_selector.cc
+++ b/media/blink/key_system_config_selector.cc
@@ -990,6 +990,7 @@ void KeySystemConfigSelector::SelectConfig(
 
   key_systems_->UpdateIfNeeded();
 
+  KeySystems::GetInstance();
   std::string key_system_ascii = key_system.Ascii();
   if (!key_systems_->IsSupportedKeySystem(key_system_ascii)) {
     std::move(cb).Run(Status::kUnsupportedKeySystem, nullptr, nullptr);
