diff --git a/media/audio/pulse/pulse_util.cc b/media/audio/pulse/pulse_util.cc
index f4dd3ba2539ec6322a3a1b2282cd6799bae4af09..381237b3e45dc57e8d03aea2f98dafba1432d9a7 100644
--- a/media/audio/pulse/pulse_util.cc
+++ b/media/audio/pulse/pulse_util.cc
@@ -36,7 +36,7 @@ namespace {
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
 constexpr char kBrowserDisplayName[] = "google-chrome";
 #define PRODUCT_STRING "Google Chrome"
-#else
+#elif 0
 constexpr char kBrowserDisplayName[] = "chromium-browser";
 #define PRODUCT_STRING "Chromium"
 #endif
