diff --git a/media/capture/capture_switches.cc b/media/capture/capture_switches.cc
index f2b84189bfeef8a653a09bb1c1f182318e9075da..88e818b370756ca88b6770f199cfef29e5a62097 100644
--- a/media/capture/capture_switches.cc
+++ b/media/capture/capture_switches.cc
@@ -44,7 +44,7 @@ namespace features {
 #if !BUILDFLAG(IS_ANDROID)
 BASE_FEATURE(kTabCaptureInfobarLinks,
              "TabCaptureInfobarLinks",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 #endif  // !BUILDFLAG(IS_ANDROID)
 
 #if defined(WEBRTC_USE_PIPEWIRE)
