diff --git a/media/audio/pulse/pulse_util.cc b/media/audio/pulse/pulse_util.cc
index 84b84ceea234e6b52db434434f16bc288cc652b6..6625b6037666f9a601dfb90079dfcdfa84bbb8a7 100644
--- a/media/audio/pulse/pulse_util.cc
+++ b/media/audio/pulse/pulse_util.cc
@@ -38,7 +38,7 @@ namespace {
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
 constexpr char kBrowserDisplayName[] = "google-chrome";
 #define PRODUCT_STRING "Google Chrome"
-#else
+#elif 0
 constexpr char kBrowserDisplayName[] = "chromium-browser";
 #define PRODUCT_STRING "Chromium"
 #endif
