diff --git a/media/base/supported_types_unittest.cc b/media/base/supported_types_unittest.cc
index 2ee18cebecfd0a667a2424883200f47ec685609a..045894346c502b765e5656f3a3c9f0bf6863668d 100644
--- a/media/base/supported_types_unittest.cc
+++ b/media/base/supported_types_unittest.cc
@@ -65,9 +65,10 @@ TEST(SupportedTypesTest, IsDecoderSupportedVideoTypeBasics) {
                                            VIDEO_CODEC_PROFILE_UNKNOWN,
                                            kUnspecifiedLevel, kColorSpace}));
 #else
-  EXPECT_FALSE(IsDecoderSupportedVideoType({VideoCodec::kHEVC,
-                                            VIDEO_CODEC_PROFILE_UNKNOWN,
-                                            kUnspecifiedLevel, kColorSpace}));
+  EXPECT_EQ(IsDecoderSupportedVideoType({VideoCodec::kHEVC,
+                                         VIDEO_CODEC_PROFILE_UNKNOWN,
+                                         kUnspecifiedLevel, kColorSpace}),
+            FFmpegSupportsHEVC());
 #endif
 }
 
@@ -488,7 +489,7 @@ TEST(SupportedTypesTest, IsDecoderBuiltInVideoCodec) {
   EXPECT_FALSE(IsDecoderBuiltInVideoCodec(VideoCodec::kMPEG4));
   EXPECT_FALSE(IsDecoderBuiltInVideoCodec(VideoCodec::kVC1));
   EXPECT_FALSE(IsDecoderBuiltInVideoCodec(VideoCodec::kMPEG2));
-  EXPECT_FALSE(IsDecoderBuiltInVideoCodec(VideoCodec::kHEVC));
+  EXPECT_EQ(IsDecoderBuiltInVideoCodec(VideoCodec::kHEVC), FFmpegSupportsHEVC());
   EXPECT_FALSE(IsDecoderBuiltInVideoCodec(VideoCodec::kDolbyVision));
 }
 
