diff --git a/mojo/public/tools/bindings/checks/mojom_attributes_check.py b/mojo/public/tools/bindings/checks/mojom_attributes_check.py
index e1b7a42d419a0b1185fa1aa787a2571482073832..157e5c1d5bf05ba6f352d4626e16fc3ebd0550ce 100644
--- a/mojo/public/tools/bindings/checks/mojom_attributes_check.py
+++ b/mojo/public/tools/bindings/checks/mojom_attributes_check.py
@@ -7,6 +7,7 @@ import mojom.generate.check as check
 import mojom.generate.module as module
 
 _COMMON_ATTRIBUTES = {
+    'BraveAdd', 'BraveExtend',
     'EnableIf',
     'EnableIfNot',
 }
@@ -185,6 +186,7 @@ _NATIVE_ALLOWLIST = {
     'search.mojom.OmniboxFocusState',
     'ui.mojom.EventPointerType',
     'ui.mojom.ScrollGranularity',
+    'brave.mojom.ImportItem', 'brave.mojom.SourceProfile',
 }
 
 
