diff --git a/mojo/public/tools/mojom/mojom_parser.py b/mojo/public/tools/mojom/mojom_parser.py
index 28c74c9e4ca17b2b2d2ee63900990bcabd2ec454..e2eccd172fa2beb72c1ca77a0702f738aef6cffe 100755
--- a/mojo/public/tools/mojom/mojom_parser.py
+++ b/mojo/public/tools/mojom/mojom_parser.py
@@ -494,6 +494,7 @@ already present in the provided output root.""")
   logging.info('Finished')
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
   Run(sys.argv[1:])
   # Exit without running GC, which can save multiple seconds due to the large
