diff --git a/mojo/public/tools/mojom/mojom_parser.py b/mojo/public/tools/mojom/mojom_parser.py
index aad01b48df5053771f84d372caf0144ac194fda5..7a8f82ef32a34919699568f3cd299a23e4b47b9e 100755
--- a/mojo/public/tools/mojom/mojom_parser.py
+++ b/mojo/public/tools/mojom/mojom_parser.py
@@ -177,6 +177,7 @@ def _ParseAstHelper(args):
   with codecs.open(mojom_abspath, encoding='utf-8') as f:
     ast = parser.Parse(f.read(), mojom_abspath)
     conditional_features.RemoveDisabledDefinitions(ast, enabled_features)
+    from mojo.public.tools.mojom.mojom.parse import brave_ast_patcher; brave_ast_patcher.PatchMojomAst(mojom_abspath, ast, enabled_features)
     return mojom_abspath, ast
 
 
