diff --git a/mojo/public/tools/bindings/checks/mojom_attributes_check.py b/mojo/public/tools/bindings/checks/mojom_attributes_check.py
index 64da9203da3982aca27fe71c4d1ece764c149b7c..d32f8e7b1ae21c6ba2e470d36f964b2104731be0 100644
--- a/mojo/public/tools/bindings/checks/mojom_attributes_check.py
+++ b/mojo/public/tools/bindings/checks/mojom_attributes_check.py
@@ -6,6 +6,7 @@
 import mojom.generate.check as check
 
 _COMMON_ATTRIBUTES = {
+    'BraveAdd', 'BraveExtend',
   'EnableIf',
   'EnableIfNot',
   'VendorSpecified',
@@ -164,6 +165,7 @@ _NATIVE_ALLOWLIST = {
   'search.mojom.OmniboxFocusState',
   'ui.mojom.EventPointerType',
   'ui.mojom.ScrollGranularity',
+    'brave.mojom.ImportItem', 'brave.mojom.SourceProfile',
 }
 
 
