diff --git a/mojo/public/tools/bindings/chromium_bindings_configuration.gni b/mojo/public/tools/bindings/chromium_bindings_configuration.gni
index 3a69de915176932b134c84abe1fd7652faafe00a..60a7c6bdb6a45a3e51a7c43373a1c5a4fbf8c35a 100644
--- a/mojo/public/tools/bindings/chromium_bindings_configuration.gni
+++ b/mojo/public/tools/bindings/chromium_bindings_configuration.gni
@@ -4,6 +4,7 @@
 
 _typemap_imports = [
   "//chrome/chrome_cleaner/mojom/typemaps/typemaps.gni",
+  "//brave/common/tor/typemaps.gni",
   "//chrome/common/importer/typemaps.gni",
   "//chrome/common/media_router/mojom/typemaps.gni",
   "//chrome/typemaps.gni",
