diff --git a/mojo/public/tools/bindings/checks/mojom_attributes_check.py b/mojo/public/tools/bindings/checks/mojom_attributes_check.py
index 511ba275df1f3e50b833ef2abc2655ca33a7f4e5..d61f8d0cc3e55ea1b84d3214b2db022d0ca2c93a 100644
--- a/mojo/public/tools/bindings/checks/mojom_attributes_check.py
+++ b/mojo/public/tools/bindings/checks/mojom_attributes_check.py
@@ -7,6 +7,7 @@ import mojom.generate.check as check
 import mojom.generate.module as module
 
 _COMMON_ATTRIBUTES = {
+    'BraveAdd', 'BraveExtend',
     'EnableIf',
     'EnableIfNot',
 }
@@ -188,6 +189,7 @@ _NATIVE_ALLOWLIST = {
     'search.mojom.OmniboxFocusState',
     'ui.mojom.EventPointerType',
     'ui.mojom.ScrollGranularity',
+    'brave.mojom.ImportItem', 'brave.mojom.SourceProfile',
 }
 
 
