diff --git a/mojo/public/tools/bindings/chromium_bindings_configuration.gni b/mojo/public/tools/bindings/chromium_bindings_configuration.gni
index 35840baac156045e2cb736b132da539ad926eed6..af95ca9784acca0da61425ea78f30ab7f4dc647c 100644
--- a/mojo/public/tools/bindings/chromium_bindings_configuration.gni
+++ b/mojo/public/tools/bindings/chromium_bindings_configuration.gni
@@ -5,6 +5,7 @@
 _typemap_imports = [
   "//cc/typemaps.gni",
   "//chrome/chrome_cleaner/mojom/typemaps/typemaps.gni",
+  "//brave/common/tor/typemaps.gni",
   "//chrome/common/importer/typemaps.gni",
   "//chrome/common/media_router/mojom/typemaps.gni",
   "//chrome/typemaps.gni",
