diff --git a/mojo/public/tools/bindings/chromium_bindings_configuration.gni b/mojo/public/tools/bindings/chromium_bindings_configuration.gni
index d8a1387448562056a600804c668c1bf4f544b776..efb3761eb097db091e3aec847fe9d9a0c987bc9a 100644
--- a/mojo/public/tools/bindings/chromium_bindings_configuration.gni
+++ b/mojo/public/tools/bindings/chromium_bindings_configuration.gni
@@ -4,6 +4,7 @@
 
 _typemap_imports = [
   "//chrome/chrome_cleaner/mojom/typemaps/typemaps.gni",
+  "//brave/common/tor/typemaps.gni",
   "//chrome/common/importer/typemaps.gni",
   "//chrome/common/media_router/mojom/typemaps.gni",
   "//chrome/typemaps.gni",
