diff --git a/mojo/public/tools/bindings/checks/mojom_attributes_check.py b/mojo/public/tools/bindings/checks/mojom_attributes_check.py
index 49a792cbdf1db2d7e97d5982ccb47bda89ec7064..be8c9fd2e0e1f95744d4d765b821093ed9363c31 100644
--- a/mojo/public/tools/bindings/checks/mojom_attributes_check.py
+++ b/mojo/public/tools/bindings/checks/mojom_attributes_check.py
@@ -7,6 +7,7 @@ import mojom.generate.check as check
 import mojom.generate.module as module
 
 _COMMON_ATTRIBUTES = {
+    'BraveAdd', 'BraveExtend',
     'EnableIf',
     'EnableIfNot',
     'VendorSpecified',
@@ -186,6 +187,7 @@ _NATIVE_ALLOWLIST = {
     'search.mojom.OmniboxFocusState',
     'ui.mojom.EventPointerType',
     'ui.mojom.ScrollGranularity',
+    'brave.mojom.ImportItem', 'brave.mojom.SourceProfile',
 }
 
 
