diff --git a/mojo/public/tools/bindings/chromium_bindings_configuration.gni b/mojo/public/tools/bindings/chromium_bindings_configuration.gni
index 2eb03f7ff4ff96594c8a2dbec53413c8a7b03797..a3e52d14e45829170aa0d1ef8ebe989757b58083 100644
--- a/mojo/public/tools/bindings/chromium_bindings_configuration.gni
+++ b/mojo/public/tools/bindings/chromium_bindings_configuration.gni
@@ -4,6 +4,7 @@
 
 _typemap_imports = [
   "//chrome/chrome_cleaner/mojom/typemaps/typemaps.gni",
+  "//brave/common/tor/typemaps.gni",
   "//chrome/common/importer/typemaps.gni",
   "//chrome/common/media_router/mojom/typemaps.gni",
   "//chrome/typemaps.gni",
