diff --git a/mojo/public/tools/bindings/checks/mojom_attributes_check.py b/mojo/public/tools/bindings/checks/mojom_attributes_check.py
index 6ee59ae32a1d6b119961b91029da4fa787d04c38..2450bf8bfdb59d69d1c34429be531aa65bba4668 100644
--- a/mojo/public/tools/bindings/checks/mojom_attributes_check.py
+++ b/mojo/public/tools/bindings/checks/mojom_attributes_check.py
@@ -7,6 +7,7 @@ import mojom.generate.check as check
 import mojom.generate.module as module
 
 _COMMON_ATTRIBUTES = {
+    'BraveAdd', 'BraveExtend',
     'EnableIf',
     'EnableIfNot',
     'VendorSpecified',
@@ -172,6 +173,7 @@ _NATIVE_ALLOWLIST = {
     'search.mojom.OmniboxFocusState',
     'ui.mojom.EventPointerType',
     'ui.mojom.ScrollGranularity',
+    'brave.mojom.ImportItem', 'brave.mojom.SourceProfile',
 }
 
 
