diff --git a/net/dns/host_resolver_manager_job.cc b/net/dns/host_resolver_manager_job.cc
index 7f41594d1856b17d4e07a73fd0736935f310ecde..245d82bae6bfd94bba9712747e3f9621d4d5b85a 100644
--- a/net/dns/host_resolver_manager_job.cc
+++ b/net/dns/host_resolver_manager_job.cc
@@ -1097,6 +1097,7 @@ void HostResolverManager::Job::CompleteRequests(
   }
 
   RecordJobHistograms(results, task_type);
+  BRAVE_RECORD_DNS_COUNTS
   if (results.error() == OK && had_non_speculative_request_ &&
       key_.source == HostResolverSource::ANY) {
     RecordJobHttpsHistograms();
