diff --git a/net/proxy_resolution/configured_proxy_resolution_service.cc b/net/proxy_resolution/configured_proxy_resolution_service.cc
index 11520accc8cf0c0bd2841cc0d7c8de7160f7e041..6d0a32620f96ffd2ca514499923a7936bbe58124 100644
--- a/net/proxy_resolution/configured_proxy_resolution_service.cc
+++ b/net/proxy_resolution/configured_proxy_resolution_service.cc
@@ -1006,6 +1006,7 @@ int ConfiguredProxyResolutionService::ResolveProxy(
   // using a direct connection for example).
   int rv = TryToCompleteSynchronously(url, /*bypass_override_rules=*/false,
                                       net_log, result);
+  BRAVE_CONFIGURED_PROXY_RESOLUTION_SERVICE_RESOLVE_PROXY
   if (rv != ERR_IO_PENDING) {
     rv = DidFinishResolvingProxy(url, network_anonymization_key, method, result,
                                  rv, net_log);
