diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index eb3abed4985776ecc2be6425119158ee80e89721..926ed829f197dc10189bb4ad4e6ec84e3b94e143 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -292,6 +292,7 @@ base::Time CanonicalCookie::ValidateAndAdjustExpiryDate(
     // * network_handler.cc::MakeCookieFromProtocolValues
     fixed_creation_date = base::Time::Now();
   }
+  BRAVE_CANONICAL_COOKIE_VALIDATE_AND_ADJUST_EXPIRY_DATE
   base::Time maximum_expiry_date;
   if (!cookie_util::IsTimeLimitedInsecureCookiesEnabled() ||
       scheme == net::CookieSourceScheme::kSecure) {
