diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index 92174b661b6aae117368fbc3d667d6089e59f73d..605e3867fea619c43ee5f05f367e4c1a8e858b92 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -516,6 +516,7 @@ base::Time CanonicalCookie::ValidateAndAdjustExpiryDate(
     // * network_handler.cc::MakeCookieFromProtocolValues
     fixed_creation_date = base::Time::Now();
   }
+  BRAVE_CANONICAL_COOKIE_VALIDATE_AND_ADJUST_EXPIRY_DATE
   if (base::FeatureList::IsEnabled(features::kClampCookieExpiryTo400Days)) {
     base::Time maximum_expiry_date = fixed_creation_date + base::Days(400);
     if (expiry_date > maximum_expiry_date)
