diff --git a/net/url_request/url_request_http_job.cc b/net/url_request/url_request_http_job.cc
index 4c19c0567d12061dc5cc214d6bb4212a8b61ed7a..3734511f356e7e4d2858f4fb7b0fa797a87e9083 100644
--- a/net/url_request/url_request_http_job.cc
+++ b/net/url_request/url_request_http_job.cc
@@ -727,6 +727,7 @@ void URLRequestHttpJob::SaveCookiesAndNotifyHeadersComplete(int result) {
     num_cookie_lines_left_++;
 
     std::unique_ptr<CanonicalCookie> cookie = net::CanonicalCookie::Create(
+        BRAVE_SAVECOOKIESANDNOTIFYHEADERSCOMPLETE
         request_->url(), cookie_string, base::Time::Now(), server_time,
         &returned_status);
 
