diff --git a/net/dns/host_cache.cc b/net/dns/host_cache.cc
index 482e6c0aed4878ac737293bad31694b52ed96957..c596bdc84715c64c0d6c4bf461a0325f626532e3 100644
--- a/net/dns/host_cache.cc
+++ b/net/dns/host_cache.cc
@@ -523,6 +523,7 @@ HostCache::Entry HostCache::Entry::CopyWithDefaultPort(uint16_t port) const {
       hostname.set_port(port);
     }
   }
+  BRAVE_COPY_WITH_DEFAULT_PORT
 
   return copy;
 }
