diff --git a/net/BUILD.gn b/net/BUILD.gn
index 3ee0718cb9ae162c5ec6bf3b57547def0ec2a6e0..17bf0bac2505b2310c1c560ecae141304c450c5b 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1269,6 +1269,7 @@ component("net") {
       "reporting/reporting_uploader.h",
     ]
   }
+  sources += brave_net_sources import("//brave/components/brave_wallet/sources.gni") deps += brave_net_deps
 
   if (enable_device_bound_sessions) {
     sources += [
