diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index 8019841b7172943f50b84a0369b49265ebd5c6ca..bc8d1a7c9cf096c48c945f144f14e50d98f27d0d 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -543,6 +543,7 @@ std::unique_ptr<CanonicalCookie> CanonicalCookie::Create(
   // Get the port, this will get a default value if a port isn't provided.
   int source_port = url.EffectiveIntPort();
 
+  BRAVE_CREATE
   // TODO(crbug.com/987177) Add partition key if Partitioned is present in the
   // cookie line.
   std::unique_ptr<CanonicalCookie> cc = base::WrapUnique(new CanonicalCookie(
