diff --git a/net/dns/host_cache.cc b/net/dns/host_cache.cc
index c1448788a90ece692444aa8f6083c407c6169461..b498d14d0a171d43e06159637d79a4506f976a97 100644
--- a/net/dns/host_cache.cc
+++ b/net/dns/host_cache.cc
@@ -516,6 +516,7 @@ HostCache::Entry HostCache::Entry::CopyWithDefaultPort(uint16_t port) const {
         hostname = HostPortPair(hostname.host(), port);
     }
   }
+  BRAVE_COPY_WITH_DEFAULT_PORT
 
   return copy;
 }
