diff --git a/net/socket/socks_client_socket_pool.cc b/net/socket/socks_client_socket_pool.cc
index 6e0bd90a2f3ad7ec77881e5de0fe3ddb45213fc7..47d16221b3df85d93a617c3b2852004fb63365c9 100644
--- a/net/socket/socks_client_socket_pool.cc
+++ b/net/socket/socks_client_socket_pool.cc
@@ -149,9 +149,11 @@ int SOCKSConnectJob::DoSOCKSConnect() {
 
   // Add a SOCKS connection on top of the tcp socket.
   if (socks_params_->is_socks_v5()) {
-    socket_.reset(new SOCKS5ClientSocket(std::move(transport_socket_handle_),
-                                         socks_params_->destination(),
-                                         socks_params_->traffic_annotation()));
+    socket_.reset(new SOCKS5ClientSocketAuth(
+            std::move(transport_socket_handle_),
+            socks_params_->destination(),
+            socks_params_->traffic_annotation(),
+            socks_params_->transport_params()->destination().host_port_pair()));
   } else {
     socket_.reset(new SOCKSClientSocket(
         std::move(transport_socket_handle_), socks_params_->destination(),
