diff --git a/net/dns/host_cache.cc b/net/dns/host_cache.cc
index 4dd78ebd000f675e02dc6a8c469443dc45a0ebb5..067bf1f6c05e0687be8209d7da0957add417ac6c 100644
--- a/net/dns/host_cache.cc
+++ b/net/dns/host_cache.cc
@@ -503,6 +503,7 @@ HostCache::Entry HostCache::Entry::CopyWithDefaultPort(uint16_t port) const {
       hostname.set_port(port);
     }
   }
+  BRAVE_COPY_WITH_DEFAULT_PORT
 
   return copy;
 }
