diff --git a/net/dns/host_cache.cc b/net/dns/host_cache.cc
index 2d42255a1db98628a741030da1a88d9730baad34..ad09bcb74948662fe978b7481bf6385ed60d8658 100644
--- a/net/dns/host_cache.cc
+++ b/net/dns/host_cache.cc
@@ -502,6 +502,7 @@ HostCache::Entry HostCache::Entry::CopyWithDefaultPort(uint16_t port) const {
       hostname.set_port(port);
     }
   }
+  BRAVE_COPY_WITH_DEFAULT_PORT
 
   return copy;
 }
