diff --git a/net/http/transport_security_state.cc b/net/http/transport_security_state.cc
index 6ac91fd12b3a314dafa25e55040bdbd9a3a21037..03f9a9edcf8d6b2e9e5798b53669b123c2536110 100644
--- a/net/http/transport_security_state.cc
+++ b/net/http/transport_security_state.cc
@@ -226,6 +226,7 @@ TransportSecurityState::TransportSecurityState(
 // others don't end up with pins that cannot be easily updated.
 #if !BUILDFLAG(GOOGLE_CHROME_BRANDING) || BUILDFLAG(IS_IOS)
   enable_static_pins_ = false;
+  BRAVE_ENABLE_STATIC_PINS
 #endif
   // Check that there no invalid entries in the static HSTS bypass list.
   for (auto& host : hsts_host_bypass_list) {
