diff --git a/net/dns/host_cache.cc b/net/dns/host_cache.cc
index 88dbcf54c84c6e950d96db52cf355ef650884ac5..b8f9bcb1ba81210da61163683b497fc76f366841 100644
--- a/net/dns/host_cache.cc
+++ b/net/dns/host_cache.cc
@@ -396,6 +396,7 @@ HostCache::Entry HostCache::Entry::CopyWithDefaultPort(uint16_t port) const {
         hostname = HostPortPair(hostname.host(), port);
     }
   }
+  BRAVE_COPY_WITH_DEFAULT_PORT
 
   return copy;
 }
