diff --git a/net/dns/host_resolver_manager_job.cc b/net/dns/host_resolver_manager_job.cc
index 50ee2e4ead425f2c700a69c3b671d54ddb9dba24..c1c2a94093e5aea6cc6b3a9af142a249afb14342 100644
--- a/net/dns/host_resolver_manager_job.cc
+++ b/net/dns/host_resolver_manager_job.cc
@@ -1336,6 +1336,7 @@ void HostResolverManager::Job::CompleteRequests(
   }
 
   RecordJobHistograms(results, task_type);
+  BRAVE_RECORD_DNS_COUNTS
   if (results.error() == OK && had_non_speculative_request_ &&
       key_.source == HostResolverSource::ANY) {
     RecordJobHttpsHistograms();
