diff --git a/net/dns/host_cache.cc b/net/dns/host_cache.cc
index e90fb519db76e42891301d1ca2ad1e3cbf300e68..61d06fefb6f2ffedca3a9a6cea3725dffdbdd29b 100644
--- a/net/dns/host_cache.cc
+++ b/net/dns/host_cache.cc
@@ -262,6 +262,7 @@ HostCache::Entry HostCache::Entry::CopyWithDefaultPort(uint16_t port) const {
     }
     copy.set_hostnames(std::move(hostnames_with_port));
   }
+  BRAVE_COPY_WITH_DEFAULT_PORT
 
   return copy;
 }
