diff --git a/net/BUILD.gn b/net/BUILD.gn
index 7fd0caf340e136294914e9f2ea9aefd5628fe7ff..af5b9b1fe6b44b9566ed2accebb493fe32534f3d 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1284,6 +1284,7 @@ component("net") {
       "reporting/reporting_uploader.h",
     ]
   }
+  sources += brave_net_sources import("//brave/components/brave_wallet/sources.gni") deps += brave_net_deps
 
   if (enable_device_bound_sessions) {
     sources += [
