diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index 8c50aacf9b0b9d4d668eb4c3ebe5038288a51da8..102b2e43f5ba2564e20a350c1d2c9c76fc4d6ca8 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -324,6 +324,7 @@ base::Time CanonicalCookie::ValidateAndAdjustExpiryDate(
     // * network_handler.cc::MakeCookieFromProtocolValues
     fixed_creation_date = base::Time::Now();
   }
+  BRAVE_CANONICAL_COOKIE_VALIDATE_AND_ADJUST_EXPIRY_DATE
   base::Time maximum_expiry_date;
   if (!cookie_util::IsTimeLimitedInsecureCookiesEnabled() ||
       scheme == net::CookieSourceScheme::kSecure) {
