diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index 91f78dc551befba361019ebad1e16a4106adaede..56f2edd71c81998a98b2e88a45a9a97edbf28fa2 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -325,6 +325,7 @@ base::Time CanonicalCookie::ValidateAndAdjustExpiryDate(
     // * network_handler.cc::MakeCookieFromProtocolValues
     fixed_creation_date = base::Time::Now();
   }
+  BRAVE_CANONICAL_COOKIE_VALIDATE_AND_ADJUST_EXPIRY_DATE
   base::Time maximum_expiry_date;
   if (!cookie_util::IsTimeLimitedInsecureCookiesEnabled() ||
       scheme == net::CookieSourceScheme::kSecure) {
