diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index 91d210fa1683da314922a15734d4fabbde8ab873..abed8a9427b0cfe3748ef3f31dd6c2407553fb5c 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -552,6 +552,7 @@ base::Time CanonicalCookie::ValidateAndAdjustExpiryDate(
     // * network_handler.cc::MakeCookieFromProtocolValues
     fixed_creation_date = base::Time::Now();
   }
+  BRAVE_CANONICAL_COOKIE_VALIDATE_AND_ADJUST_EXPIRY_DATE
   if (base::FeatureList::IsEnabled(features::kClampCookieExpiryTo400Days)) {
     base::Time maximum_expiry_date = fixed_creation_date + base::Days(400);
     if (expiry_date > maximum_expiry_date)
