diff --git a/net/dns/host_resolver_manager.cc b/net/dns/host_resolver_manager.cc
index 6b1dc26ea222472cf9163b2b19959465c664349c..7533854fd3c533ae38ce14dbed005bf39ea993e8 100644
--- a/net/dns/host_resolver_manager.cc
+++ b/net/dns/host_resolver_manager.cc
@@ -2788,6 +2788,7 @@ class HostResolverManager::Job : public PrioritizedDispatcher::Job,
       MaybeCacheResult(results, ttl, secure);
 
     RecordJobHistograms(results, task_type);
+    BRAVE_RECORD_DNS_COUNTS
 
     // Complete all of the requests that were attached to the job and
     // detach them.
