diff --git a/net/http/transport_security_state.cc b/net/http/transport_security_state.cc
index 76a67501a4c144a59a66de33975a1937d3e456f1..3cfd902c2bc46df1c354a8e3b45db1431a3237f7 100644
--- a/net/http/transport_security_state.cc
+++ b/net/http/transport_security_state.cc
@@ -241,6 +241,7 @@ TransportSecurityState::TransportSecurityState(
 // others don't end up with pins that cannot be easily updated.
 #if !BUILDFLAG(GOOGLE_CHROME_BRANDING) || BUILDFLAG(IS_IOS)
   enable_static_pins_ = false;
+  BRAVE_ENABLE_STATIC_PINS
 #endif
   // Check that there no invalid entries in the static HSTS bypass list.
   for (auto& host : hsts_host_bypass_list) {
