diff --git a/net/cookies/cookie_monster.cc b/net/cookies/cookie_monster.cc
index 3fe2b6bbabbcfa8b4fd4ed8a2614a7cc74b0b998..d18c10ef5699dfb604262ebac4d7bc004779a34c 100644
--- a/net/cookies/cookie_monster.cc
+++ b/net/cookies/cookie_monster.cc
@@ -146,7 +146,7 @@ enum CookieType {
   COOKIE_TYPE_LAST_ENTRY
 };
 
-void MaybeRunDeleteCallback(base::WeakPtr<net::CookieMonster> cookie_monster,
+void MaybeRunDeleteCallback(base::WeakPtr<net::chromium_impl::CookieMonster> cookie_monster,
                             base::OnceClosure callback) {
   if (cookie_monster && callback)
     std::move(callback).Run();
@@ -221,7 +221,7 @@ void LogStoredCookieToUMA(const net::CanonicalCookie& cc,
 
 }  // namespace
 
-namespace net {
+namespace net::chromium_impl {
 
 // See comments at declaration of these variables in cookie_monster.h
 // for details.
@@ -3012,4 +3012,4 @@ std::optional<bool> CookieMonster::SiteHasCookieInOtherPartition(
   });
 }
 
-}  // namespace net
+}  // namespace net::chromium_impl
