diff --git a/net/cookies/cookie_deletion_info.h b/net/cookies/cookie_deletion_info.h
index 1d824985f66a7ca15e056e48b658f9b256641076..e4bc9290674be27d41611696d9254696ea476a00 100644
--- a/net/cookies/cookie_deletion_info.h
+++ b/net/cookies/cookie_deletion_info.h
@@ -150,6 +150,7 @@ struct NET_EXPORT CookieDeletionInfo {
   // If true, third-party cookie blocking applies to the context that triggered
   // the deletion. In this case, we should only delete partitioned cookies.
   bool partitioned_state_only = false;
+  BRAVE_COOKIE_DELETION_INFO_H
 };
 
 }  // namespace net
