diff --git a/net/socket/socks_connect_job.cc b/net/socket/socks_connect_job.cc
index fb4a628126fa6651aeae8c74fc8b9509b6e48225..9b709beb9ded0768d9277fe53f2606fcb1474736 100644
--- a/net/socket/socks_connect_job.cc
+++ b/net/socket/socks_connect_job.cc
@@ -161,9 +161,11 @@ int SOCKSConnectJob::DoSOCKSConnect() {
 
   // Add a SOCKS connection on top of the tcp socket.
   if (socks_params_->is_socks_v5()) {
-    socket_.reset(new SOCKS5ClientSocket(transport_connect_job_->PassSocket(),
-                                         socks_params_->destination(),
-                                         socks_params_->traffic_annotation()));
+    socket_.reset(new SOCKS5ClientSocketAuth(
+        transport_connect_job_->PassSocket(),
+        socks_params_->destination(),
+        socks_params_->traffic_annotation(),
+        socks_params_->transport_params()->destination()));
   } else {
     socket_.reset(new SOCKSClientSocket(
         transport_connect_job_->PassSocket(), socks_params_->destination(),
