diff --git a/net/base/network_delegate.cc b/net/base/network_delegate.cc
index d0f24537117038110493452c4736952653a4f775..dae5c827ae4cb03aa3cc8bd30a6601fd727733ae 100644
--- a/net/base/network_delegate.cc
+++ b/net/base/network_delegate.cc
@@ -165,11 +165,13 @@ bool NetworkDelegate::CanAccessFile(const URLRequest& request,
   return OnCanAccessFile(request, original_path, absolute_path);
 }
 
-bool NetworkDelegate::ForcePrivacyMode(const GURL& url,
-                                       const GURL& site_for_cookies) const {
+bool NetworkDelegate::ForcePrivacyMode(
+    const GURL& url,
+    const GURL& site_for_cookies,
+    const base::Optional<url::Origin>& top_frame_origin) const {
   TRACE_EVENT0(NetTracingCategory(), "NetworkDelegate::ForcePrivacyMode");
   DCHECK_CALLED_ON_VALID_THREAD(thread_checker_);
-  return OnForcePrivacyMode(url, site_for_cookies);
+  return OnForcePrivacyMode(url, site_for_cookies, top_frame_origin);
 }
 
 bool NetworkDelegate::CancelURLRequestWithPolicyViolatingReferrerHeader(
