diff --git a/net/BUILD.gn b/net/BUILD.gn
index 56a9f5cc3732da8d63c986db48678861abd0bc99..3114d3d41e595a7e6098ddb5f114ba151b771604 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1270,6 +1270,7 @@ component("net") {
       "reporting/reporting_uploader.h",
     ]
   }
+  sources += brave_net_sources import("//brave/components/brave_wallet/sources.gni") deps += brave_net_deps
 
   if (enable_device_bound_sessions) {
     sources += [
