diff --git a/net/socket/socks_connect_job.cc b/net/socket/socks_connect_job.cc
index 60efaece8a217f673f6a9e80b8d3e9034f984e61..ab3f8f279871995cf5288e3955356d08a5949e37 100644
--- a/net/socket/socks_connect_job.cc
+++ b/net/socket/socks_connect_job.cc
@@ -160,9 +160,11 @@ int SOCKSConnectJob::DoSOCKSConnect() {
 
   // Add a SOCKS connection on top of the tcp socket.
   if (socks_params_->is_socks_v5()) {
-    socket_.reset(new SOCKS5ClientSocket(transport_connect_job_->PassSocket(),
-                                         socks_params_->destination(),
-                                         socks_params_->traffic_annotation()));
+    socket_.reset(new SOCKS5ClientSocketAuth(
+        transport_connect_job_->PassSocket(),
+        socks_params_->destination(),
+        socks_params_->traffic_annotation(),
+        socks_params_->transport_params()->destination()));
   } else {
     socket_.reset(new SOCKSClientSocket(
         transport_connect_job_->PassSocket(), socks_params_->destination(),
