diff --git a/net/dns/dns_transaction.cc b/net/dns/dns_transaction.cc
index feaa6bef907c37d2c8569c5170f3311eb1eccecb..b6476db372dcf738d34100474e4b7f81217c5ba8 100644
--- a/net/dns/dns_transaction.cc
+++ b/net/dns/dns_transaction.cc
@@ -1345,6 +1345,7 @@ class DnsTransactionImpl : public DnsTransaction,
     DCHECK(secure_);
 
     size_t doh_server_index = dns_server_iterator_->GetNextAttemptIndex();
+    BRAVE_MAKE_HTTP_ATTEMPT
 
     unsigned attempt_number = attempts_.size();
     ConstructDnsHTTPAttempt(
