diff --git a/net/cookies/cookie_monster.h b/net/cookies/cookie_monster.h
index 561f7e2642984aee37bdb66b19620873ba6ffb13..ccc871a339550d77de1d1ec5cca90e8f6b777454 100644
--- a/net/cookies/cookie_monster.h
+++ b/net/cookies/cookie_monster.h
@@ -58,6 +58,7 @@ class CookieInclusionStatus;
 // latter case, the cookie callback will be queued in tasks_pending_for_key_
 // while PermanentCookieStore loads cookies for the specified domain key on DB
 // thread.
+namespace chromium_impl {
 class NET_EXPORT CookieMonster : public CookieStore {
  public:
   class PersistentCookieStore;
@@ -978,6 +979,7 @@ class NET_EXPORT CookieMonster::PersistentCookieStore
   friend class base::RefCountedThreadSafe<PersistentCookieStore>;
 };
 
+}  // namespace chromium_impl
 }  // namespace net
 
 #endif  // NET_COOKIES_COOKIE_MONSTER_H_
