diff --git a/net/base/features.cc b/net/base/features.cc
index 981ed4ea564592e404969fe9c887c1c97d902e38..75ba2e639d29925177afbc32467a32decbe9213f 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -214,8 +214,9 @@ BASE_FEATURE(kSplitCacheByNetworkIsolationKey,
 BASE_FEATURE(kSplitCodeCacheByNetworkIsolationKey,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
+// kPartitionConnectionsByNetworkIsolationKey feature state is enforced via plaster rewrite.
 BASE_FEATURE(kPartitionConnectionsByNetworkIsolationKey,
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kSplitHostCacheByNetworkAnonymizationKey,
              base::FEATURE_ENABLED_BY_DEFAULT);
@@ -422,6 +423,7 @@ BASE_FEATURE(kForceThirdPartyCookieBlocking,
 
 BASE_FEATURE(kEnableEarlyHintsOnHttp11, base::FEATURE_DISABLED_BY_DEFAULT);
 
+// kEnableWebTransportDraft07 feature state is enforced via plaster rewrite.
 BASE_FEATURE(kEnableWebTransportDraft07, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kWebTransportFineGrainedThrottling,
@@ -733,7 +735,8 @@ BASE_FEATURE(kRestrictAbusePorts, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kRestrictAbusePortsOnLocalhost, base::FEATURE_ENABLED_BY_DEFAULT);
 
-BASE_FEATURE(kTLSTrustAnchorIDs, base::FEATURE_ENABLED_BY_DEFAULT);
+// kTLSTrustAnchorIDs feature state is enforced via plaster rewrite.
+BASE_FEATURE(kTLSTrustAnchorIDs, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kNonMtcTrustAnchorIDs, base::FEATURE_ENABLED_BY_DEFAULT);
 
