diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index 3936de245d1ecc09679f593a2563f4e8a320fcd0..e0ef74a515863a9c7fef057bcecfdccf556b5388 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -542,6 +542,7 @@ std::unique_ptr<CanonicalCookie> CanonicalCookie::Create(
   // Get the port, this will get a default value if a port isn't provided.
   int source_port = url.EffectiveIntPort();
 
+  BRAVE_CREATE
   // TODO(crbug.com/987177) Add partition key if Partitioned is present in the
   // cookie line.
   std::unique_ptr<CanonicalCookie> cc = base::WrapUnique(new CanonicalCookie(
