diff --git a/net/http/transport_security_state.cc b/net/http/transport_security_state.cc
index 73b325ddaead5190847fa40241bdd637c1babf1d..49a433f903a704fe0f0ceb317cfbbe000ed9c4a1 100644
--- a/net/http/transport_security_state.cc
+++ b/net/http/transport_security_state.cc
@@ -237,6 +237,7 @@ TransportSecurityState::TransportSecurityState(
 // others don't end up with pins that cannot be easily updated.
 #if !BUILDFLAG(GOOGLE_CHROME_BRANDING) || BUILDFLAG(IS_IOS)
   enable_static_pins_ = false;
+  BRAVE_ENABLE_STATIC_PINS
 #endif
   // Check that there no invalid entries in the static HSTS bypass list.
   for (auto& host : hsts_host_bypass_list) {
