diff --git a/net/socket/socks_connect_job.cc b/net/socket/socks_connect_job.cc
index c9d1ddf2f9d33622c3c4fa2763eafc890e798062..b58850b24a3e7859786316020931534e6381973a 100644
--- a/net/socket/socks_connect_job.cc
+++ b/net/socket/socks_connect_job.cc
@@ -179,9 +179,11 @@ int SOCKSConnectJob::DoSOCKSConnect() {
 
   // Add a SOCKS connection on top of the tcp socket.
   if (socks_params_->is_socks_v5()) {
-    socket_ = std::make_unique<SOCKS5ClientSocket>(
-        transport_connect_job_->PassSocket(), socks_params_->destination(),
-        socks_params_->traffic_annotation());
+    socket_ = std::make_unique<SOCKS5ClientSocketAuth>(
+        transport_connect_job_->PassSocket(),
+        socks_params_->destination(),
+        socks_params_->traffic_annotation(),
+        socks_params_->transport_params()->destination());
   } else {
     auto socks_socket = std::make_unique<SOCKSClientSocket>(
         transport_connect_job_->PassSocket(), socks_params_->destination(),
