diff --git a/net/dns/dns_transaction.cc b/net/dns/dns_transaction.cc
index 57d5663ae832636b927f1b4a334f69db0609c1e7..56e2a9b4497042700be2e4e1564e2b885cd9c298 100644
--- a/net/dns/dns_transaction.cc
+++ b/net/dns/dns_transaction.cc
@@ -1302,6 +1302,7 @@ class DnsTransactionImpl : public DnsTransaction,
     DCHECK(secure_);
 
     size_t doh_server_index = dns_server_iterator_->GetNextAttemptIndex();
+    BRAVE_MAKE_HTTP_ATTEMPT
 
     unsigned attempt_number = attempts_.size();
     ConstructDnsHTTPAttempt(
