diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index b9743cc9a894cc382f8a1da3d8c9588142b487b7..5a0689cbba31c1f81b9394f125344477cbda43c6 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -240,6 +240,7 @@ Time CanonicalCookie::CanonExpiration(const ParsedCookie& pc,
 
 // static
 std::unique_ptr<CanonicalCookie> CanonicalCookie::Create(
+    BRAVE_CANONICALCOOKIE_CREATE_EXTRA_PARAMS
     const GURL& url,
     const std::string& cookie_line,
     const base::Time& creation_time,
@@ -280,8 +281,7 @@ std::unique_ptr<CanonicalCookie> CanonicalCookie::Create(
     cookie_server_time = server_time.value();
 
   DCHECK(!creation_time.is_null());
-  Time cookie_expires = CanonicalCookie::CanonExpiration(
-      parsed_cookie, creation_time, cookie_server_time);
+  BRAVE_CANONICALCOOKIE_CREATE_BRAVECANONEXPIRATION
 
   CookiePrefix prefix = GetCookiePrefix(parsed_cookie.Name());
   bool is_cookie_prefix_valid = IsCookiePrefixValid(prefix, url, parsed_cookie);
