diff --git a/net/url_request/url_request_http_job.cc b/net/url_request/url_request_http_job.cc
index 1f133bb17a6523d8a8e2bdc80d8ba7639b57bb79..2717e578a8d77640c00780498fc5bf0701c27b75 100644
--- a/net/url_request/url_request_http_job.cc
+++ b/net/url_request/url_request_http_job.cc
@@ -767,6 +767,7 @@ void URLRequestHttpJob::SaveCookiesAndNotifyHeadersComplete(int result) {
     num_cookie_lines_left_++;
 
     std::unique_ptr<CanonicalCookie> cookie = net::CanonicalCookie::Create(
+        BRAVE_SAVECOOKIESANDNOTIFYHEADERSCOMPLETE
         request_->url(), cookie_string, base::Time::Now(), server_time,
         &returned_status);
 
