diff --git a/net/dns/host_cache.cc b/net/dns/host_cache.cc
index 27d379df459ea25f93cf72d842eecdb841679814..c7fbd13c5ebb75517c91d1c1707f386d63ca0bfe 100644
--- a/net/dns/host_cache.cc
+++ b/net/dns/host_cache.cc
@@ -502,6 +502,7 @@ HostCache::Entry HostCache::Entry::CopyWithDefaultPort(uint16_t port) const {
       hostname.set_port(port);
     }
   }
+  BRAVE_COPY_WITH_DEFAULT_PORT
 
   return copy;
 }
