diff --git a/net/dns/dns_transaction.cc b/net/dns/dns_transaction.cc
index f37d5fa1232bf2327cc894876cb24fa2df65054e..eb97f164ec1204d37821da05cdbf8ab7bcf898b1 100644
--- a/net/dns/dns_transaction.cc
+++ b/net/dns/dns_transaction.cc
@@ -1351,6 +1351,7 @@ class DnsTransactionImpl : public DnsTransaction,
     DCHECK(secure_);
 
     size_t doh_server_index = dns_server_iterator_->GetNextAttemptIndex();
+    BRAVE_MAKE_HTTP_ATTEMPT
 
     unsigned attempt_number = attempts_.size();
     ConstructDnsHTTPAttempt(
