diff --git a/net/http/transport_security_state.cc b/net/http/transport_security_state.cc
index 7ea8987ce6ec3eb5993187a36d1bd27f1cde8cb4..e0781e6ab922ab6fd5227f209661dc651cb8c499 100644
--- a/net/http/transport_security_state.cc
+++ b/net/http/transport_security_state.cc
@@ -392,6 +392,7 @@ TransportSecurityState::TransportSecurityState(
 // others don't end up with pins that cannot be easily updated.
 #if !BUILDFLAG(GOOGLE_CHROME_BRANDING) || BUILDFLAG(IS_IOS)
   enable_static_pins_ = false;
+  BRAVE_ENABLE_STATIC_PINS
 #endif
   // Check that there no invalid entries in the static HSTS bypass list.
   for (auto& host : hsts_host_bypass_list) {
