diff --git a/net/proxy_resolution/configured_proxy_resolution_service.cc b/net/proxy_resolution/configured_proxy_resolution_service.cc
index 5febe93426dc147285c47e6aa7ccc0058c2e4c29..6c2a7de95c90bf887e016a795a689edebaa13ee3 100644
--- a/net/proxy_resolution/configured_proxy_resolution_service.cc
+++ b/net/proxy_resolution/configured_proxy_resolution_service.cc
@@ -1005,6 +1005,7 @@ int ConfiguredProxyResolutionService::ResolveProxy(
   // using a direct connection for example).
   int rv = TryToCompleteSynchronously(url, /*bypass_override_rules=*/false,
                                       net_log, result);
+  BRAVE_CONFIGURED_PROXY_RESOLUTION_SERVICE_RESOLVE_PROXY
   if (rv != ERR_IO_PENDING) {
     rv = DidFinishResolvingProxy(url, network_anonymization_key, method, result,
                                  rv, net_log);
