diff --git a/net/base/layered_network_delegate.cc b/net/base/layered_network_delegate.cc
index 9f47088ed705af2fcaac3a5f9a771bd66e296696..f901dccc77b265cb4df0d03bc3702f29b27c1139 100644
--- a/net/base/layered_network_delegate.cc
+++ b/net/base/layered_network_delegate.cc
@@ -229,14 +229,17 @@ void LayeredNetworkDelegate::OnCanAccessFileInternal(
 
 bool LayeredNetworkDelegate::OnForcePrivacyMode(
     const GURL& url,
-    const GURL& site_for_cookies) const {
-  return OnForcePrivacyModeInternal(url, site_for_cookies) ||
-         nested_network_delegate_->ForcePrivacyMode(url, site_for_cookies);
+    const GURL& site_for_cookies,
+    const base::Optional<url::Origin>& top_frame_origin) const {
+  return OnForcePrivacyModeInternal(url, site_for_cookies, top_frame_origin) ||
+         nested_network_delegate_->ForcePrivacyMode(url, site_for_cookies,
+                                                    top_frame_origin);
 }
 
 bool LayeredNetworkDelegate::OnForcePrivacyModeInternal(
     const GURL& url,
-    const GURL& site_for_cookies) const {
+    const GURL& site_for_cookies,
+    const base::Optional<url::Origin>& top_frame_origin) const {
   return false;
 }
 
