diff --git a/net/url_request/url_request_http_job.cc b/net/url_request/url_request_http_job.cc
index 8eef86523c971209bac52e3cc325fc6afa051f57..6094407f6c3a9d7587139ba01cc4cb8ae3ceedf3 100644
--- a/net/url_request/url_request_http_job.cc
+++ b/net/url_request/url_request_http_job.cc
@@ -747,6 +747,7 @@ void URLRequestHttpJob::SaveCookiesAndNotifyHeadersComplete(int result) {
     num_cookie_lines_left_++;
 
     std::unique_ptr<CanonicalCookie> cookie = net::CanonicalCookie::Create(
+        BRAVE_SAVECOOKIESANDNOTIFYHEADERSCOMPLETE
         request_->url(), cookie_string, base::Time::Now(), server_time,
         &returned_status);
 
