diff --git a/net/BUILD.gn b/net/BUILD.gn
index 9e4f5694e9517bbd9f88f024f7e90d628c451950..7ea136e8b4ba845085aa122a858b26c2c12c5a69 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1280,6 +1280,7 @@ component("net") {
       "reporting/reporting_uploader.h",
     ]
   }
+  sources += brave_net_sources import("//brave/components/brave_wallet/sources.gni") deps += brave_net_deps
 
   if (enable_device_bound_sessions) {
     sources += [
