diff --git a/net/cookies/canonical_cookie.h b/net/cookies/canonical_cookie.h
index a29d1887e0f51e924e23847b959cc182614f3982..8c37c1fc319a92c2eaef1eff77956368be7b9a54 100644
--- a/net/cookies/canonical_cookie.h
+++ b/net/cookies/canonical_cookie.h
@@ -80,6 +80,7 @@ class NET_EXPORT CanonicalCookie {
   //
   // If a cookie is returned, |cookie->IsCanonical()| will be true.
   static std::unique_ptr<CanonicalCookie> Create(
+      BRAVE_CANONICAL_COOKIE_H_CREATE_EXTRA_PARAMS
       const GURL& url,
       const std::string& cookie_line,
       const base::Time& creation_time,
