diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index d3fe5891fa6f67f54a8be9e4dc074111eab3bf28..5f9949b94a4c94e6bb97745fc9d300506dc2cab3 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -542,6 +542,7 @@ std::unique_ptr<CanonicalCookie> CanonicalCookie::Create(
   // Get the port, this will get a default value if a port isn't provided.
   int source_port = url.EffectiveIntPort();
 
+  BRAVE_CREATE
   // TODO(crbug.com/987177) Add partition key if Partitioned is present in the
   // cookie line.
   std::unique_ptr<CanonicalCookie> cc = base::WrapUnique(new CanonicalCookie(
