diff --git a/net/url_request/url_request_http_job.cc b/net/url_request/url_request_http_job.cc
index ec9cb7509bac6255385e28de9b49b5154bf345ec..2490e0639e3c81357a5289000e2d341936b3bfd0 100644
--- a/net/url_request/url_request_http_job.cc
+++ b/net/url_request/url_request_http_job.cc
@@ -784,6 +784,7 @@ void URLRequestHttpJob::AddExtraHeaders() {
       request()->context()->enable_brotli(),
       request()->context()->enable_zstd());
 
+  BRAVE_URL_REQUEST_HTTP_JOB_ADD_EXTRA_HEADERS
   if (http_user_agent_settings_) {
     // Only add default Accept-Language if the request didn't have it
     // specified.
