diff --git a/net/BUILD.gn b/net/BUILD.gn
index be3159b3f9ab8ea147faa6be5c0c14820e0368a1..874f3810c84e841963415bd931c5e545b1c5b08f 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1282,6 +1282,7 @@ component("net") {
       "reporting/reporting_uploader.h",
     ]
   }
+  sources += brave_net_sources import("//brave/components/brave_wallet/sources.gni") deps += brave_net_deps
 
   if (enable_device_bound_sessions) {
     sources += [
