diff --git a/net/dns/host_resolver_manager.cc b/net/dns/host_resolver_manager.cc
index 7c2a44a769c92ec6c3fc483fa56658746647a95c..414bee6e8cdd67dddbfb27273c05a4de56ed9531 100644
--- a/net/dns/host_resolver_manager.cc
+++ b/net/dns/host_resolver_manager.cc
@@ -2796,6 +2796,7 @@ class HostResolverManager::Job : public PrioritizedDispatcher::Job,
       MaybeCacheResult(results, ttl, secure);
 
     RecordJobHistograms(results, task_type);
+    BRAVE_RECORD_DNS_COUNTS
 
     // Complete all of the requests that were attached to the job and
     // detach them.
