diff --git a/net/base/proxy_server.cc b/net/base/proxy_server.cc
index 16756fa513563a823fe36d9dbddd68de56d17c59..46bf636b688e59cf14757e5f59f549b75274b6e9 100644
--- a/net/base/proxy_server.cc
+++ b/net/base/proxy_server.cc
@@ -60,6 +60,7 @@ ProxyServer ProxyServer::FromSchemeHostAndPort(Scheme scheme,
 ProxyServer ProxyServer::FromSchemeHostAndPort(Scheme scheme,
                                                base::StringPiece host,
                                                absl::optional<uint16_t> port) {
+  BRAVE_PROXY_SERVER_FROM_SCHEME_HOST_AND_PORT_EXTRACT_AUTH_INFO
   // Create INVALID proxies directly using `ProxyServer()`.
   DCHECK_NE(scheme, SCHEME_INVALID);
 
@@ -96,6 +97,7 @@ ProxyServer ProxyServer::FromSchemeHostAndPort(Scheme scheme,
   // A uint16_t port is always valid and canonicalized.
   uint16_t fixed_port = port.value_or(GetDefaultPortForScheme(scheme));
 
+  BRAVE_PROXY_SERVER_FROM_SCHEME_HOST_AND_PORT_RETURN_HOST_PORT_PAIR
   return ProxyServer(scheme, HostPortPair(unbracketed_host, fixed_port));
 }
 
