diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index aec043d33920f689f59eb1f25003c878bbb28599..43c06924609569f435416c24ce6fbd64bbaa468b 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -242,9 +242,10 @@ std::unique_ptr<CanonicalCookie> CanonicalCookie::Create(
     server_time = options.server_time();
 
   DCHECK(!creation_time.is_null());
-  Time cookie_expires = CanonicalCookie::CanonExpiration(parsed_cookie,
-                                                         creation_time,
-                                                         server_time);
+  Time cookie_expires = BraveCanonExpiration(parsed_cookie,
+                                             creation_time,
+                                             server_time,
+                                             !options.exclude_httponly());
 
   CookiePrefix prefix = GetCookiePrefix(parsed_cookie.Name());
   bool is_cookie_valid = IsCookiePrefixValid(prefix, url, parsed_cookie);
