diff --git a/net/dns/host_resolver_manager_job.cc b/net/dns/host_resolver_manager_job.cc
index 70b4a94bb5aed43879c6060ce9eca995f9739b4b..e46aef0995dc64142304f239df5d4279ab3606c3 100644
--- a/net/dns/host_resolver_manager_job.cc
+++ b/net/dns/host_resolver_manager_job.cc
@@ -1096,6 +1096,7 @@ void HostResolverManager::Job::CompleteRequests(
   }
 
   RecordJobHistograms(results, task_type);
+  BRAVE_RECORD_DNS_COUNTS
   if (results.error() == OK && had_non_speculative_request_ &&
       key_.source == HostResolverSource::ANY) {
     RecordJobHttpsHistograms();
