diff --git a/net/base/host_port_pair.cc b/net/base/host_port_pair.cc
index ec8248449dbcd4156e361fcc8781ce0712f26a02..fd8af49cfa6f41c56b7218a71048dd077e072f6e 100644
--- a/net/base/host_port_pair.cc
+++ b/net/base/host_port_pair.cc
@@ -50,12 +50,14 @@ HostPortPair HostPortPair::FromString(const std::string& str) {
   return host_port_pair;
 }
 
+#if !defined(BRAVE_CHROMIUM_BUILD)
 std::string HostPortPair::ToString() const {
   std::string ret(HostForURL());
   ret += ':';
   ret += base::UintToString(port_);
   return ret;
 }
+#endif
 
 std::string HostPortPair::HostForURL() const {
   // TODO(rtenneti): Add support for |host| to have '\0'.
