diff --git a/net/dns/BUILD.gn b/net/dns/BUILD.gn
index 6bc60f6e3158bb98335e4331e7a8f513d070f225..415171eac8f61b45d4194005eab52fb9924799aa 100644
--- a/net/dns/BUILD.gn
+++ b/net/dns/BUILD.gn
@@ -24,6 +24,7 @@ source_set("dns") {
     "//net/*",
     "//services/network/*",
   ]
+  friend += brave_dns_friend
 
   public = []
   sources = [
@@ -212,6 +213,7 @@ source_set("host_resolver") {
     "//net/*",
     "//services/network/*",
   ]
+  friend += brave_dns_friend
 
   sources = [
     "dns_config.h",
@@ -308,6 +310,7 @@ source_set("dns_client") {
     "//net/*",
     "//services/network/*",
   ]
+  friend += brave_dns_friend
 
   sources = [
     "dns_client.h",
