diff --git a/net/dns/host_cache.cc b/net/dns/host_cache.cc
index 6a66063175431249387bc7b41fc6b266c0de722d..926302b1011b80d09bde3e33b59ca30c9171153c 100644
--- a/net/dns/host_cache.cc
+++ b/net/dns/host_cache.cc
@@ -259,6 +259,7 @@ HostCache::Entry HostCache::Entry::CopyWithDefaultPort(uint16_t port) const {
     }
     copy.set_hostnames(std::move(hostnames_with_port));
   }
+  BRAVE_COPY_WITH_DEFAULT_PORT
 
   return copy;
 }
