diff --git a/net/device_bound_sessions/session_binding_utils.cc b/net/device_bound_sessions/session_binding_utils.cc
index 954543aee319de08d0ecc83369eaee013d3186aa..99df9699c09c1f03834076cf9e6d8e45ace55b50 100644
--- a/net/device_bound_sessions/session_binding_utils.cc
+++ b/net/device_bound_sessions/session_binding_utils.cc
@@ -68,6 +68,8 @@ std::optional<std::string_view> SignatureAlgorithmToString(
       return "ML-DSA-65";
     case crypto::SignatureVerifier::MLDSA_87:
       return "ML-DSA-87";
+    case crypto::SignatureVerifier::ECDSA_SHA384:
+      return "SHA384";
   }
 }
 
