diff --git a/net/http/transport_security_state.cc b/net/http/transport_security_state.cc
index 6c304292f3ff154a67515b93d4f6a2abfd0eb4f7..a31a7c039f077530b4b8f421729aa5a6e5e4732d 100644
--- a/net/http/transport_security_state.cc
+++ b/net/http/transport_security_state.cc
@@ -247,6 +247,7 @@ TransportSecurityState::TransportSecurityState(
 // others don't end up with pins that cannot be easily updated.
 #if !BUILDFLAG(GOOGLE_CHROME_BRANDING) || BUILDFLAG(IS_IOS)
   enable_static_pins_ = false;
+  BRAVE_ENABLE_STATIC_PINS
 #endif
   // Check that there no invalid entries in the static HSTS bypass list.
   for (auto& host : hsts_host_bypass_list) {
