diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index 68fa41779bbe277e67df1e29517c52577632e4ec..7c57d14f97ad77c88e61e334fdd1e84a9c510a24 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -419,6 +419,7 @@ std::unique_ptr<CanonicalCookie> CanonicalCookie::Create(
                                          ? CookieSourceScheme::kSecure
                                          : CookieSourceScheme::kNonSecure;
 
+  BRAVE_CREATE
   std::unique_ptr<CanonicalCookie> cc(std::make_unique<CanonicalCookie>(
       parsed_cookie.Name(), parsed_cookie.Value(), cookie_domain, cookie_path,
       creation_time, cookie_expires, creation_time, parsed_cookie.IsSecure(),
