diff --git a/net/dns/BUILD.gn b/net/dns/BUILD.gn
index a69d9a515aaa61f440afe343d7cb704fb4d6495b..c427fc56840a4849c88bd65a3570c22fcfeab48c 100644
--- a/net/dns/BUILD.gn
+++ b/net/dns/BUILD.gn
@@ -24,6 +24,7 @@ source_set("dns") {
     "//net/*",
     "//services/network/*",
   ]
+  friend += brave_dns_friend
 
   public = []
   sources = [
@@ -212,6 +213,7 @@ source_set("host_resolver") {
     "//net/*",
     "//services/network/*",
   ]
+  friend += brave_dns_friend
 
   sources = [
     "dns_config.h",
@@ -308,6 +310,7 @@ source_set("dns_client") {
     "//net/*",
     "//services/network/*",
   ]
+  friend += brave_dns_friend
 
   sources = [
     "dns_client.h",
