diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index d51bb27abfc4880b08941c148f7f1aa46a6e33bb..f103f7807976a278b9f1805719b85b47ae8b864e 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -284,6 +284,7 @@ base::Time CanonicalCookie::ValidateAndAdjustExpiryDate(
     // * network_handler.cc::MakeCookieFromProtocolValues
     fixed_creation_date = base::Time::Now();
   }
+  BRAVE_CANONICAL_COOKIE_VALIDATE_AND_ADJUST_EXPIRY_DATE
   base::Time maximum_expiry_date;
   if (!cookie_util::IsTimeLimitedInsecureCookiesEnabled() ||
       scheme == net::CookieSourceScheme::kSecure) {
