diff --git a/net/http/transport_security_state.cc b/net/http/transport_security_state.cc
index 504465742a7bd93946b6aac651108da35b8389d4..03c5f81982b2cadc151e39bae7cb0a281e174d24 100644
--- a/net/http/transport_security_state.cc
+++ b/net/http/transport_security_state.cc
@@ -417,6 +417,7 @@ TransportSecurityState::TransportSecurityState(
 #if !BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OS_ANDROID) || defined(OS_IOS)
   enable_static_pins_ = false;
   enable_static_expect_ct_ = false;
+  BRAVE_ENABLE_STATIC_PINS
 #endif
   // Check that there no invalid entries in the static HSTS bypass list.
   for (auto& host : hsts_host_bypass_list) {
