diff --git a/net/url_request/url_request.cc b/net/url_request/url_request.cc
index 736dddbe7d647912b613c84385a2314a941fbd7e..5fec72700c49bbcf9b64713f05b1c5e53e1e96c2 100644
--- a/net/url_request/url_request.cc
+++ b/net/url_request/url_request.cc
@@ -1155,8 +1155,8 @@ net::PrivacyMode URLRequest::DeterminePrivacyMode() const {
   // with the network service - remove it.
   bool enable_privacy_mode = !g_default_can_use_cookies;
   if (network_delegate_) {
-    enable_privacy_mode =
-        network_delegate_->ForcePrivacyMode(url(), site_for_cookies_);
+    enable_privacy_mode = network_delegate_->ForcePrivacyMode(
+        url(), site_for_cookies_, network_isolation_key_.GetTopFrameOrigin());
   }
   return enable_privacy_mode ? PRIVACY_MODE_ENABLED : PRIVACY_MODE_DISABLED;
 }
