diff --git a/net/url_request/url_request_http_job.cc b/net/url_request/url_request_http_job.cc
index 221d7cd946d427ce66827bea58e3627371d87269..3a6b1f8bf6136f25019558c1262195dd9dd8de32 100644
--- a/net/url_request/url_request_http_job.cc
+++ b/net/url_request/url_request_http_job.cc
@@ -712,6 +712,7 @@ void URLRequestHttpJob::SaveCookiesAndNotifyHeadersComplete(int result) {
     num_cookie_lines_left_++;
 
     std::unique_ptr<CanonicalCookie> cookie = net::CanonicalCookie::Create(
+        BRAVE_SAVECOOKIESANDNOTIFYHEADERSCOMPLETE
         request_->url(), cookie_string, base::Time::Now(), server_time,
         &returned_status);
 
