diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index 55456b872530088753664931a49b442f87f49a35..34e1d3a06d0b5b47212d3e1753d25845c7fa605e 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -516,6 +516,7 @@ base::Time CanonicalCookie::ValidateAndAdjustExpiryDate(
     // * network_handler.cc::MakeCookieFromProtocolValues
     fixed_creation_date = base::Time::Now();
   }
+  BRAVE_CANONICAL_COOKIE_VALIDATE_AND_ADJUST_EXPIRY_DATE
   if (base::FeatureList::IsEnabled(features::kClampCookieExpiryTo400Days)) {
     base::Time maximum_expiry_date = fixed_creation_date + base::Days(400);
     if (expiry_date > maximum_expiry_date)
