diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index 8af2c1b14b443ef76fd59ad5f78152d80fadc2f8..2fdea507b8168231fb3082ab7db5ccdb57fbeb31 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -451,6 +451,7 @@ std::unique_ptr<CanonicalCookie> CanonicalCookie::Create(
   // Get the port, this will get a default value if a port isn't provided.
   int source_port = url.EffectiveIntPort();
 
+  BRAVE_CREATE
   std::unique_ptr<CanonicalCookie> cc = base::WrapUnique(new CanonicalCookie(
       parsed_cookie.Name(), parsed_cookie.Value(), cookie_domain, cookie_path,
       creation_time, cookie_expires, creation_time, parsed_cookie.IsSecure(),
