diff --git a/net/dns/BUILD.gn b/net/dns/BUILD.gn
index ecec89fd75bd82c28484fa614d9c1c32d831cc13..749c334631ec16aacb3d8a0386579b30c91b7540 100644
--- a/net/dns/BUILD.gn
+++ b/net/dns/BUILD.gn
@@ -24,6 +24,7 @@ source_set("dns") {
     "//net/*",
     "//services/network/*",
   ]
+  friend += brave_dns_friend
 
   public = []
   sources = [
@@ -210,6 +211,7 @@ source_set("host_resolver") {
     "//net/*",
     "//services/network/*",
   ]
+  friend += brave_dns_friend
 
   sources = [
     "dns_config.h",
@@ -306,6 +308,7 @@ source_set("dns_client") {
     "//net/*",
     "//services/network/*",
   ]
+  friend += brave_dns_friend
 
   sources = [
     "dns_client.h",
