diff --git a/net/dns/host_cache.cc b/net/dns/host_cache.cc
index 4e8d6eea38b3c50f863130aa6a1e7efd1f750a68..d1237667df6148de7eddb53a4aa2117f2be292ff 100644
--- a/net/dns/host_cache.cc
+++ b/net/dns/host_cache.cc
@@ -396,6 +396,7 @@ HostCache::Entry HostCache::Entry::CopyWithDefaultPort(uint16_t port) const {
         hostname = HostPortPair(hostname.host(), port);
     }
   }
+  BRAVE_COPY_WITH_DEFAULT_PORT
 
   return copy;
 }
