diff --git a/net/dns/host_cache.cc b/net/dns/host_cache.cc
index f9ea4fa082b3b03b888be09d12045a463966f3fc..b70aebac7f78ac92c62edfbe073439d2bce3edf8 100644
--- a/net/dns/host_cache.cc
+++ b/net/dns/host_cache.cc
@@ -259,6 +259,7 @@ HostCache::Entry HostCache::Entry::CopyWithDefaultPort(uint16_t port) const {
     }
     copy.set_hostnames(std::move(hostnames_with_port));
   }
+  BRAVE_COPY_WITH_DEFAULT_PORT
 
   return copy;
 }
