diff --git a/net/dns/host_cache.cc b/net/dns/host_cache.cc
index 2d63fc020114c360c2a29dab6851b243723ad9b0..07b1f6ff02a11546db72a7962fabbd4b1cf3bae6 100644
--- a/net/dns/host_cache.cc
+++ b/net/dns/host_cache.cc
@@ -496,6 +496,7 @@ HostCache::Entry HostCache::Entry::CopyWithDefaultPort(uint16_t port) const {
       hostname = HostPortPair(hostname.host(), port);
     }
   }
+  BRAVE_COPY_WITH_DEFAULT_PORT
 
   return copy;
 }
