diff --git a/pdf/pdf_features.cc b/pdf/pdf_features.cc
index 7f0e882c06e1d335394ef54f4a299151ab065a16..815c589876cc09e36e7199cd6b8a57985b4764cd 100644
--- a/pdf/pdf_features.cc
+++ b/pdf/pdf_features.cc
@@ -86,10 +86,10 @@ const base::FeatureParam<bool> kPdfInk2TextAnnotationsExtraStyles{
 
 #if BUILDFLAG(ENABLE_PDF_SAVE_TO_DRIVE)
 // Saves the PDF file to Google Drive.
-BASE_FEATURE(kPdfSaveToDrive, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kPdfSaveToDrive, base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Enables the survey for saving PDF to Google Drive.
-BASE_FEATURE(kPdfSaveToDriveSurvey, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kPdfSaveToDriveSurvey, base::FEATURE_DISABLED_BY_DEFAULT);
 // The consumer survey trigger ID.
 const base::FeatureParam<std::string> kPdfSaveToDriveSurveyConsumerTriggerId{
     &kPdfSaveToDriveSurvey, "consumer-trigger-id", ""};
