diff --git a/PRESUBMIT.py b/PRESUBMIT.py
index 22a6de9ec138e8ce2257aaadcffc4aa03e89b7de..cb54b632c06813a9a0e3423ed103a6c9184d407b 100644
--- a/PRESUBMIT.py
+++ b/PRESUBMIT.py
@@ -6106,7 +6106,6 @@ def CheckPythonShebang(input_api, output_api):
                 file))
     return result
 
-
 def CheckBatchAnnotation(input_api, output_api):
     """Checks that tests have either @Batch or @DoNotBatch annotation. If this
     is not an instrumentation test, disregard."""
@@ -6169,3 +6168,5 @@ Robolectric tests do not need a @Batch or @DoNotBatch annotations.
 """, extra_annotation_errors))
 
     return results
+
+if not globals().get('__name__'): from lib.import_inline import inline_module; inline_module('chromium_presubmit_overrides', globals(), locals())
