diff --git a/PRESUBMIT.py b/PRESUBMIT.py
index 8663125b048a4222f1a024ee0b50e4d6c059cfb0..4388ea7cd43614dc55a88feb3154ffcdb3113aef 100644
--- a/PRESUBMIT.py
+++ b/PRESUBMIT.py
@@ -6095,3 +6095,4 @@ Robolectric tests do not need a @Batch or @DoNotBatch annotations.
 """, extra_annotation_errors))
 
     return results
+from import_inline import inline_module; inline_module('chromium_presubmit_overrides', globals(), locals())
