diff --git a/PRESUBMIT.py b/PRESUBMIT.py
index d034f82606ff7970d78576052115cbb78a9ed3a0..27b79413070fa0693a906e9d04f02bba0efbbc01 100644
--- a/PRESUBMIT.py
+++ b/PRESUBMIT.py
@@ -6169,3 +6169,4 @@ Robolectric tests do not need a @Batch or @DoNotBatch annotations.
 """, extra_annotation_errors))
 
     return results
+from import_inline import inline_module; inline_module('chromium_presubmit_overrides', globals(), locals())
