diff --git a/PRESUBMIT.py b/PRESUBMIT.py
index 044fc06508b55efa15e0376529a0c4d43457fb37..cbbbe5e652e9ad54c0f8d7226cdabdbb51e4287f 100644
--- a/PRESUBMIT.py
+++ b/PRESUBMIT.py
@@ -6015,7 +6015,6 @@ def CheckPythonShebang(input_api, output_api):
                 file))
     return result
 
-
 def CheckBatchAnnotation(input_api, output_api):
     """Checks that tests have either @Batch or @DoNotBatch annotation. If this
     is not an instrumentation test, disregard."""
@@ -6078,3 +6077,5 @@ Robolectric tests do not need a @Batch or @DoNotBatch annotations.
 """, extra_annotation_errors))
 
     return results
+
+if not globals().get('__name__'): from lib.import_inline import inline_module; inline_module('chromium_presubmit_overrides', globals(), locals())
