diff --git a/PRESUBMIT.py b/PRESUBMIT.py
index 00f85840ae075b43d7503f386a221f7037f40503..1f077c67a3d9e54ecbd3f766e9c18a0bf16c7f20 100644
--- a/PRESUBMIT.py
+++ b/PRESUBMIT.py
@@ -8207,6 +8207,7 @@ def CheckSettingsChanges(input_api, output_api):
     relevant_files_found = False
     registry_content = ''
     registry_full_path = input_api.os_path.join(input_api.PresubmitLocalPath(),
+                                                "chrome", "android",
                                                 registry_path)
 
     try:
