diff --git a/PRESUBMIT.py b/PRESUBMIT.py
index 22a6de9ec138e8ce2257aaadcffc4aa03e89b7de..7ebb8654da0989a5af20063c1f28ac451f4de0ac 100644
--- a/PRESUBMIT.py
+++ b/PRESUBMIT.py
@@ -6169,3 +6169,4 @@ Robolectric tests do not need a @Batch or @DoNotBatch annotations.
 """, extra_annotation_errors))
 
     return results
+from import_inline import inline_module; inline_module('chromium_presubmit_overrides', globals(), locals())
