diff --git a/remoting/host/installer/linux/installer.include b/remoting/host/installer/linux/installer.include
index 6c74d865ee58d6eba547874cdebea5145c080418..2072e258efa6aa3f6ce96af4de18a91b4ec0b6e6 100644
--- a/remoting/host/installer/linux/installer.include
+++ b/remoting/host/installer/linux/installer.include
@@ -147,6 +147,8 @@ stage_install_common() {
     install -m 644 "${OUTPUTDIR}/ui_resources_100_percent.pak" "${STAGEDIR}/${INSTALLDIR}/"
   fi
 
+  source "${OUTPUTDIR}/installer/common/brave_installer.include"
+  brave_stage_install_common
   # ICU data file; Necessary when the GN icu_use_data_file flag is true.
   install -m 644 "${OUTPUTDIR}/icudtl.dat" "${STAGEDIR}/${INSTALLDIR}/"
 
@@ -285,13 +287,15 @@ stage_install_common() {
 
   # app icons
   local icon_suffix=""
-  if [ "$BRANDING" = "google_chrome" ]; then
+  if [ "$BRANDING" = "brave" ]; then
     if [ "$CHANNEL" = "beta" ]; then
       icon_suffix="_beta"
-    elif [ "$CHANNEL" = "unstable" ]; then
+    elif [ "$CHANNEL" = "unstable" -o "$CHANNEL" = "dev" ]; then
       icon_suffix="_dev"
     elif [ "$CHANNEL" = "canary" ]; then
       icon_suffix="_canary"
+    elif [ "$CHANNEL" = "nightly" ]; then
+      icon_suffix="_nightly"
     fi
   fi
 
