diff --git a/sandbox/policy/win/sandbox_win.cc b/sandbox/policy/win/sandbox_win.cc
index dac5e25ceccf1a7f51c983cde387840e671513ae..e3fbb281fc2c09d77f387a44911521fc72627a32 100644
--- a/sandbox/policy/win/sandbox_win.cc
+++ b/sandbox/policy/win/sandbox_win.cc
@@ -611,6 +611,7 @@ ResultCode LaunchWithoutSandbox(
   base::LaunchOptions options;
   options.handles_to_inherit = handles_to_inherit;
   options.feedback_cursor_off = true;
+  BRAVE_START_SANDBOXED_PROCESS
   // Network process runs in a job even when unsandboxed. This is to ensure it
   // does not outlive the browser, which could happen if there is a lot of I/O
   // on process shutdown, in which case TerminateProcess can fail. See
