diff --git a/services/network/public/cpp/BUILD.gn b/services/network/public/cpp/BUILD.gn
index cecc3966675edd77b775385d626b8da20bbca421..90a89216a30cd0f7190c232fd93e633f7032c01e 100644
--- a/services/network/public/cpp/BUILD.gn
+++ b/services/network/public/cpp/BUILD.gn
@@ -203,6 +203,7 @@ component("cpp") {
     ":cpp_base",
     ":flags_and_switches",
     ":is_potentially_trustworthy",
+    "//brave/services/network/public/cpp",
     "//net",
     "//services/network/public/mojom",
     "//services/network/public/mojom:url_loader_base",
