diff --git a/services/network/BUILD.gn b/services/network/BUILD.gn
index 2e6dbd0f771d03d1d5c0f41e077a16e808d87401..d782ef64d62383f73e47ffa82405f054c30b2bcc 100644
--- a/services/network/BUILD.gn
+++ b/services/network/BUILD.gn
@@ -399,6 +399,7 @@ component("network_service") {
     "//third_party/zstd:decompress",
     "//url",
   ]
+  import("//brave/services/network/sources.gni") deps += brave_services_network_deps
 
   if (is_linux || is_chromeos) {
     deps += [
