diff --git a/services/network/cookie_settings.cc b/services/network/cookie_settings.cc
index bc8267ddd131cd6c3b0472975970467524f5d64c..26decc18c44056f7dfb0e4cb9797dc7764d7be0c 100644
--- a/services/network/cookie_settings.cc
+++ b/services/network/cookie_settings.cc
@@ -260,6 +260,7 @@ CookieSettings::GetCookieSettingWithMetadata(const GURL& url,
       match) {
     cookie_setting = match->GetContentSetting();
     found_explicit_setting = IsExplicitSetting(*match);
+    BRAVE_COOKIE_SETTINGS_GET_COOKIE_SETTINGS_INTERNAL
   }
 
   if (cookie_setting != CONTENT_SETTING_BLOCK && !found_explicit_setting) {
