diff --git a/services/network/cookie_manager.cc b/services/network/cookie_manager.cc
index cf7e823d1ee21a6fa42187373fb3629a8de32a09..a67ef8db26ac606cc5ae6a1031bd9b8cd755f29a 100644
--- a/services/network/cookie_manager.cc
+++ b/services/network/cookie_manager.cc
@@ -419,6 +419,7 @@ CookieDeletionInfo DeletionFilterToInfo(mojom::CookieDeletionFilterPtr filter) {
   delete_info.name = std::move(filter->cookie_name);
   delete_info.url = std::move(filter->url);
   delete_info.host = std::move(filter->host_name);
+  BRAVE_DELETIONFILTERTOINFO
 
   switch (filter->session_control) {
     case mojom::CookieDeletionSessionControl::IGNORE_CONTROL:
