diff --git a/services/network/cors/cors_url_loader.cc b/services/network/cors/cors_url_loader.cc
index 408e10ce95fc744a0683d79093e825ba413c5afb..e09b218a4824b5e52503bf398ef38e5e5ae10f2f 100644
--- a/services/network/cors/cors_url_loader.cc
+++ b/services/network/cors/cors_url_loader.cc
@@ -939,6 +939,7 @@ void CorsURLLoader::StartRequest() {
   };
 
   if (should_include_origin_header()) {
+    BRAVE_CORS_URL_LOADER_START_REQUEST
     if (tainted_) {
       request_.headers.SetHeader(net::HttpRequestHeaders::kOrigin,
                                  url::Origin().Serialize());
