diff --git a/services/network/cors/cors_url_loader.cc b/services/network/cors/cors_url_loader.cc
index a1bce2abdfbf0b92d2af11c20d8018f498143c32..dbbc2d6fe32d71a00d896f112e6c10ed1221eb48 100644
--- a/services/network/cors/cors_url_loader.cc
+++ b/services/network/cors/cors_url_loader.cc
@@ -982,6 +982,7 @@ void CorsURLLoader::StartRequest() {
         cors::ShouldAllowUnsafeHeaders(
             *origin_access_list_, request_.request_initiator, request_.url);
 
+    BRAVE_CORS_URL_LOADER_START_REQUEST
     if (!has_custom_origin_header_with_bypass) {
       if (tainted_) {
         request_.headers.SetHeader(net::HttpRequestHeaders::kOrigin,
