diff --git a/services/network/public/cpp/permissions_policy/permissions_policy_features.json5 b/services/network/public/cpp/permissions_policy/permissions_policy_features.json5
index 30d7177263d6fbbf466c49666f9d08a21817178d..81d5804cdb7f0d67feb3ba18e8d0d876b2c74f11 100644
--- a/services/network/public/cpp/permissions_policy/permissions_policy_features.json5
+++ b/services/network/public/cpp/permissions_policy/permissions_policy_features.json5
@@ -639,6 +639,18 @@
       permissions_policy_name: "writer",
       depends_on: ["AIWriterAPI"],
     },
+    {
+      name: "Ethereum",
+      permissions_policy_name: "ethereum"
+    },
+    {
+      name: "Solana",
+      permissions_policy_name: "solana"
+    },
+    {
+      name: "Cardano",
+      permissions_policy_name: "cardano"
+    },
   ],
   // LINT.ThenChange(//third_party/blink/public/devtools_protocol/domains/Page.pdl:PermissionsPolicyFeature)
 }
