diff --git a/services/network/public/cpp/BUILD.gn b/services/network/public/cpp/BUILD.gn
index 1a96b81d4e53c7cedaa5bfcb1bd27cec99c2a440..52a881dea0ec4096519e741185320c2785cf523a 100644
--- a/services/network/public/cpp/BUILD.gn
+++ b/services/network/public/cpp/BUILD.gn
@@ -201,6 +201,7 @@ component("cpp") {
     ":cpp_base",
     ":flags_and_switches",
     ":is_potentially_trustworthy",
+    "//brave/services/network/public/cpp",
     "//net",
     "//services/network/public/mojom",
     "//services/network/public/mojom:url_loader_base",
