diff --git a/services/network/cors/cors_url_loader.cc b/services/network/cors/cors_url_loader.cc
index 81d51784db5ecf12a1808105524fd2403bfb7ed8..0de361eccb12f381efc4710b3ebab9e8eea46651 100644
--- a/services/network/cors/cors_url_loader.cc
+++ b/services/network/cors/cors_url_loader.cc
@@ -896,6 +896,7 @@ void CorsURLLoader::StartRequest() {
   };
 
   if (should_include_origin_header()) {
+    BRAVE_CORS_URL_LOADER_START_REQUEST
     if (tainted_) {
       request_.headers.SetHeader(net::HttpRequestHeaders::kOrigin,
                                  url::Origin().Serialize());
