diff --git a/services/network/BUILD.gn b/services/network/BUILD.gn
index dd91e7c38f4afc02875f8f7fda6970edf799f76f..57c8987f229d2952574d32d1a890ffc9f6dbc1b5 100644
--- a/services/network/BUILD.gn
+++ b/services/network/BUILD.gn
@@ -398,6 +398,7 @@ component("network_service") {
     "//third_party/zstd:decompress",
     "//url",
   ]
+  import("//brave/services/network/sources.gni") deps += brave_services_network_deps
 
   if (is_linux || is_chromeos) {
     deps += [
