diff --git a/services/network/tls_client_socket.cc b/services/network/tls_client_socket.cc
index 8fa33735cf7e80c6e5e6310dc8783be8e4a3bd50..d84f6035238501ac22a1ccbd44a973ebd2385277 100644
--- a/services/network/tls_client_socket.cc
+++ b/services/network/tls_client_socket.cc
@@ -24,7 +24,14 @@ TLSClientSocket::TLSClientSocket(
     const net::NetworkTrafficAnnotationTag& traffic_annotation)
     : observer_(std::move(observer)), traffic_annotation_(traffic_annotation) {}
 
-TLSClientSocket::~TLSClientSocket() {}
+TLSClientSocket::~TLSClientSocket() {
+  if (connect_callback_) {
+    std::move(connect_callback_)
+        .Run(net::ERR_ABORTED, mojo::ScopedDataPipeConsumerHandle(),
+             mojo::ScopedDataPipeProducerHandle(), base::nullopt);
+  }
+}
+
 
 void TLSClientSocket::Connect(
     const net::HostPortPair& host_port_pair,
