diff --git a/services/network/cookie_settings.cc b/services/network/cookie_settings.cc
index 8ab818b51d99734c89933be8cedcf766fbf01694..9ce23d9397e869661568935a85560cd82caebdc5 100644
--- a/services/network/cookie_settings.cc
+++ b/services/network/cookie_settings.cc
@@ -180,6 +180,7 @@ CookieSettings::GetCookieSettingWithMetadata(
       // Note: global settings are implemented as a catch-all (*, *) pattern.
       if (IsExplicitSetting(*entry))
         blocked_by_third_party_setting = false;
+      BRAVE_COOKIE_SETTINGS_GET_COOKIE_SETTINGS_INTERNAL
     }
   }
 
