diff --git a/services/preferences/tracked/BUILD.gn b/services/preferences/tracked/BUILD.gn
index 551ab8ca8da7e1d6b11ba7cd6fbd44743b2418c9..efd754da15d30dd66919f4046e787a1447d0ff5d 100644
--- a/services/preferences/tracked/BUILD.gn
+++ b/services/preferences/tracked/BUILD.gn
@@ -59,6 +59,7 @@ source_set("tracked") {
   if (is_mac) {
     frameworks = [ "IOKit.framework" ]
   }
+  import("//brave/services/preferences/tracked/sources.gni") deps += brave_services_preferences_tracked_deps
 }
 
 source_set("unit_tests") {
