diff --git a/services/network/cookie_settings.cc b/services/network/cookie_settings.cc
index d83a4bd0a341b429609e43e59985d2aa91d12f50..8f3212a9ef07f023fa540ae409fcc13a0fd21677 100644
--- a/services/network/cookie_settings.cc
+++ b/services/network/cookie_settings.cc
@@ -229,6 +229,7 @@ CookieSettings::GetCookieSettingWithMetadata(
       storage_access_result =
           net::cookie_util::StorageAccessResult::ACCESS_BLOCKED;
     }
+    BRAVE_COOKIE_SETTINGS_GET_COOKIE_SETTINGS_INTERNAL
   }
 
   if (cookie_setting != CONTENT_SETTING_BLOCK && !found_explicit_setting &&
