diff --git a/services/network/restricted_cookie_manager.cc b/services/network/restricted_cookie_manager.cc
index b0da2ecbd260f477692d17850b7f163c8c588b83..ae1184157136a8190b1968f8102826b28709a602 100644
--- a/services/network/restricted_cookie_manager.cc
+++ b/services/network/restricted_cookie_manager.cc
@@ -443,7 +443,8 @@ void RestrictedCookieManager::SetCookieFromString(
   DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
 
   std::unique_ptr<net::CanonicalCookie> parsed_cookie =
-      net::CanonicalCookie::Create(url, cookie, base::Time::Now(),
+      net::CanonicalCookie::Create(BRAVE_SETCOOKIEFROMSTRING
+                                   url, cookie, base::Time::Now(),
                                    base::nullopt /* server_time */);
   if (!parsed_cookie) {
     std::move(callback).Run();
