diff --git a/services/network/BUILD.gn b/services/network/BUILD.gn
index a029e50ac98ef9e0307dadc206ad169a7a081d41..6c43458052e370c668e2cff1c57895ac7415081f 100644
--- a/services/network/BUILD.gn
+++ b/services/network/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/services/network/sources.gni")
 import("//build/config/features.gni")
 import("//components/vrp_flags/features.gni")
 import("//mojo/public/tools/bindings/mojom.gni")
@@ -391,6 +392,7 @@ component("network_service") {
     "//third_party/zstd:decompress",
     "//url",
   ]
+  deps += brave_services_network_deps
 
   if (is_linux || is_chromeos) {
     deps += [
