diff --git a/services/network/cors/cors_url_loader.cc b/services/network/cors/cors_url_loader.cc
index 9f610d4a796a11b394169ba6c14a59be27aab24b..e1a50de3105061cc0f19a0605303cf5a912445cf 100644
--- a/services/network/cors/cors_url_loader.cc
+++ b/services/network/cors/cors_url_loader.cc
@@ -929,6 +929,7 @@ void CorsURLLoader::StartRequest() {
   };
 
   if (should_include_origin_header()) {
+    BRAVE_CORS_URL_LOADER_START_REQUEST
     if (tainted_) {
       request_.headers.SetHeader(net::HttpRequestHeaders::kOrigin,
                                  url::Origin().Serialize());
