diff --git a/services/network/public/cpp/permissions_policy/permissions_policy_features.json5 b/services/network/public/cpp/permissions_policy/permissions_policy_features.json5
index 8c049ec4e18eb207a72fdadb8db36f2f12dbca6d..fb798282e2acceaba929e028cdb6e3697fa12098 100644
--- a/services/network/public/cpp/permissions_policy/permissions_policy_features.json5
+++ b/services/network/public/cpp/permissions_policy/permissions_policy_features.json5
@@ -588,5 +588,13 @@
       permissions_policy_name: "writer",
       depends_on: ["AIWriterAPI", "BuiltInAIAPI"],
     },
+    {
+      name: "Ethereum",
+      permissions_policy_name: "ethereum"
+    },
+    {
+      name: "Solana",
+      permissions_policy_name: "solana"
+    },
   ],
 }
