diff --git a/services/network/cors/cors_url_loader.cc b/services/network/cors/cors_url_loader.cc
index d0886b406e34cf89cf08115a2bc9ea1e93bb4887..f2bc5991d77ff9b5c1c9bc01ff7692ddec5426c1 100644
--- a/services/network/cors/cors_url_loader.cc
+++ b/services/network/cors/cors_url_loader.cc
@@ -723,6 +723,7 @@ void CorsURLLoader::StartRequest() {
       (fetch_cors_flag_ ||
        (request_.method != net::HttpRequestHeaders::kGetMethod &&
         request_.method != net::HttpRequestHeaders::kHeadMethod))) {
+    BRAVE_CORS_URL_LOADER_START_REQUEST
     if (tainted_) {
       request_.headers.SetHeader(net::HttpRequestHeaders::kOrigin,
                                  url::Origin().Serialize());
