diff --git a/services/network/cookie_settings.cc b/services/network/cookie_settings.cc
index b0d1311bf71caa04016ee4b33bed383774ac4885..3777eea6bd9277390cd413c88d6b5a7006c7dd39 100644
--- a/services/network/cookie_settings.cc
+++ b/services/network/cookie_settings.cc
@@ -206,6 +206,7 @@ CookieSettings::GetCookieSettingWithMetadata(
         // TODO(dylancutler): Consider adding an enum variant for this case.
         blocked_by_third_party_setting = CookieSettings::
             ThirdPartyCookieBlockingSetting::kThirdPartyStateAllowed;
+        BRAVE_COOKIE_SETTINGS_GET_COOKIE_SETTINGS_INTERNAL
     }
   }
 
