diff --git a/services/network/cookie_settings.h b/services/network/cookie_settings.h
index 578af6b3722acc6a906b0cbe21ade34588148208..37814938c2341facb4bc9ae6a63499bf958a5637 100644
--- a/services/network/cookie_settings.h
+++ b/services/network/cookie_settings.h
@@ -59,13 +59,14 @@ class COMPONENT_EXPORT(NETWORK_SERVICE) CookieSettings
   SessionCleanupCookieStore::DeleteCookiePredicate
   CreateDeleteCookieOnExitPredicate() const;
 
+ private:
   // content_settings::CookieSettingsBase:
-  void GetCookieSetting(const GURL& url,
-                        const GURL& first_party_url,
-                        content_settings::SettingSource* source,
-                        ContentSetting* cookie_setting) const override;
+  void GetCookieSettingInternal(const GURL& url,
+                                const GURL& first_party_url,
+                                bool is_third_party_request,
+                                content_settings::SettingSource* source,
+                                ContentSetting* cookie_setting) const override;
 
- private:
   // Returns true if at least one content settings is session only.
   bool HasSessionOnlyOrigins() const;
 
