diff --git a/services/network/tls_client_socket.cc b/services/network/tls_client_socket.cc
index dd7a936913d6c989e088a580f1726df7bb01239c..5c8eee966bac535514fdd63874dbd5688c2c7aa5 100644
--- a/services/network/tls_client_socket.cc
+++ b/services/network/tls_client_socket.cc
@@ -24,7 +24,14 @@ TLSClientSocket::TLSClientSocket(
     const net::NetworkTrafficAnnotationTag& traffic_annotation)
     : observer_(std::move(observer)), traffic_annotation_(traffic_annotation) {}
 
-TLSClientSocket::~TLSClientSocket() {}
+TLSClientSocket::~TLSClientSocket() {
+  if (connect_callback_) {
+    std::move(connect_callback_)
+        .Run(net::ERR_ABORTED, mojo::ScopedDataPipeConsumerHandle(),
+             mojo::ScopedDataPipeProducerHandle(), base::nullopt);
+  }
+}
+
 
 void TLSClientSocket::Connect(
     const net::HostPortPair& host_port_pair,
