diff --git a/services/network/cors/cors_url_loader.cc b/services/network/cors/cors_url_loader.cc
index 6c23690386ea3a6004063a24e6b1d9d92b16a6e2..c8b869ff58fffae7901ea1fcf709742697302d64 100644
--- a/services/network/cors/cors_url_loader.cc
+++ b/services/network/cors/cors_url_loader.cc
@@ -972,6 +972,7 @@ void CorsURLLoader::StartRequest() {
         cors::ShouldAllowUnsafeHeaders(
             *origin_access_list_, request_.request_initiator, request_.url);
 
+    BRAVE_CORS_URL_LOADER_START_REQUEST
     if (!has_custom_origin_header_with_bypass) {
       if (tainted_) {
         request_.headers.SetHeader(net::HttpRequestHeaders::kOrigin,
