diff --git a/services/network/public/cpp/permissions_policy/permissions_policy_features.json5 b/services/network/public/cpp/permissions_policy/permissions_policy_features.json5
index 625ad1199eeb53dc7bab716388b68f2078846c98..6dd43360711fbb082a9ad6dc17ec7470d6301be3 100644
--- a/services/network/public/cpp/permissions_policy/permissions_policy_features.json5
+++ b/services/network/public/cpp/permissions_policy/permissions_policy_features.json5
@@ -620,6 +620,18 @@
       permissions_policy_name: "writer",
       depends_on: ["AIWriterAPI"],
     },
+    {
+      name: "Ethereum",
+      permissions_policy_name: "ethereum"
+    },
+    {
+      name: "Solana",
+      permissions_policy_name: "solana"
+    },
+    {
+      name: "Cardano",
+      permissions_policy_name: "cardano"
+    },
   ],
   // LINT.ThenChange(//third_party/blink/public/devtools_protocol/browser_protocol.pdl:PermissionsPolicyFeature)
 }
