diff --git a/services/network/cors/cors_url_loader.cc b/services/network/cors/cors_url_loader.cc
index c56255c509a07237fac958cc416e3e1201a7abdc..05b5b7bed1f8cf1e0dcc87cc17c77d9ca611cd98 100644
--- a/services/network/cors/cors_url_loader.cc
+++ b/services/network/cors/cors_url_loader.cc
@@ -1037,6 +1037,7 @@ void CorsURLLoader::StartRequest() {
         request_.headers.HasHeader(net::HttpRequestHeaders::kOrigin) &&
         AllowUnsafeHeaders();
 
+    BRAVE_CORS_URL_LOADER_START_REQUEST
     if (!has_custom_origin_header_with_bypass) {
       if (tainted_) {
         request_.headers.SetHeader(net::HttpRequestHeaders::kOrigin,
