diff --git a/services/network/network_context.cc b/services/network/network_context.cc
index dffa1299b1cfce8609313435769189495fd8ccb5..d85728b414187ea48e257b82462a228d50505894 100644
--- a/services/network/network_context.cc
+++ b/services/network/network_context.cc
@@ -440,6 +440,7 @@ class NetworkContext::ContextNetworkDelegate
                                const net::CookieList& cookie_list,
                                bool allowed_from_caller) override {
     return allowed_from_caller &&
+           BRAVE_ON_CAN_GET_COOKIES_INTERNAL
            network_context_->cookie_manager()
                ->cookie_settings()
                .IsCookieAccessAllowed(request.url(),
@@ -451,6 +452,7 @@ class NetworkContext::ContextNetworkDelegate
                               net::CookieOptions* options,
                               bool allowed_from_caller) override {
     return allowed_from_caller &&
+           BRAVE_ON_CAN_SET_COOKIES_INTERNAL
            network_context_->cookie_manager()
                ->cookie_settings()
                .IsCookieAccessAllowed(request.url(),
