diff --git a/services/network/public/mojom/cookie_manager.mojom b/services/network/public/mojom/cookie_manager.mojom
index 3efaabbf59b5afe8c735b5f365231407a7b51103..fe3525e7bdea969bb0ff90e412f19459fef16189 100644
--- a/services/network/public/mojom/cookie_manager.mojom
+++ b/services/network/public/mojom/cookie_manager.mojom
@@ -284,6 +284,7 @@ struct CookieDeletionFilter {
 
   // Delete session/persistent cookies.
   CookieDeletionSessionControl session_control = IGNORE_CONTROL;
+  string? ephemeral_storage_domain;
 };
 
 interface CookieChangeListener {
