diff --git a/services/network/cookie_manager.cc b/services/network/cookie_manager.cc
index 06a85c11e6604d3c458424af2286d11aa9485283..e292000051d730451da61f131361ab5ffa852051 100644
--- a/services/network/cookie_manager.cc
+++ b/services/network/cookie_manager.cc
@@ -418,6 +418,7 @@ CookieDeletionInfo DeletionFilterToInfo(mojom::CookieDeletionFilterPtr filter) {
   delete_info.name = std::move(filter->cookie_name);
   delete_info.url = std::move(filter->url);
   delete_info.host = std::move(filter->host_name);
+  BRAVE_DELETIONFILTERTOINFO
 
   switch (filter->session_control) {
     case mojom::CookieDeletionSessionControl::IGNORE_CONTROL:
