diff --git a/services/network/public/cpp/BUILD.gn b/services/network/public/cpp/BUILD.gn
index 124748fec3a5f8706752ad4821b30e897923276f..a9d23b6e6bbc277cb18453efbb305ff6b4ad2558 100644
--- a/services/network/public/cpp/BUILD.gn
+++ b/services/network/public/cpp/BUILD.gn
@@ -219,6 +219,7 @@ component("cpp") {
     "//services/proxy_resolver/public/mojom",
   ]
 
+  public_deps += [ "//brave/services/network/public/cpp" ]
   defines = [ "IS_NETWORK_CPP_IMPL" ]
 }
 
