diff --git a/services/network/cookie_settings.cc b/services/network/cookie_settings.cc
index ebb05ccaac10a42bbacfba194ea9e3be27b077c5..6e9f3dc89e5882eb437bf9d30290e9b2c119f3e8 100644
--- a/services/network/cookie_settings.cc
+++ b/services/network/cookie_settings.cc
@@ -230,6 +230,7 @@ CookieSettings::GetCookieSettingWithMetadata(
       storage_access_result =
           net::cookie_util::StorageAccessResult::ACCESS_BLOCKED;
     }
+    BRAVE_COOKIE_SETTINGS_GET_COOKIE_SETTINGS_INTERNAL
   }
 
   if (cookie_setting != CONTENT_SETTING_BLOCK && !found_explicit_setting &&
