diff --git a/services/network/resolve_host_request.cc b/services/network/resolve_host_request.cc
index ac5c87a1e5f29c7c47f97a3d99c0a91cae748ea5..f4ba13e1eb4d512967b1abc947589e9fdd1077af 100644
--- a/services/network/resolve_host_request.cc
+++ b/services/network/resolve_host_request.cc
@@ -93,6 +93,7 @@ int ResolveHostRequest::Start(
   mojo::Remote<mojom::ResolveHostClient> response_client(
       std::move(pending_response_client));
   if (rv != net::ERR_IO_PENDING) {
+    BRAVE_RESOLVE_HOST_REQUEST_START
     response_client->OnComplete(rv, GetResolveErrorInfo(), GetAddressResults(),
                                 GetAlternativeEndpoints());
     return rv;
