diff --git a/services/network/cookie_manager.cc b/services/network/cookie_manager.cc
index 5b4fe64a4374a1ac1f943b53089cbf05043acd3e..dc54a35c5ecdacb2d77e571c049a9376094a52ce 100644
--- a/services/network/cookie_manager.cc
+++ b/services/network/cookie_manager.cc
@@ -361,6 +361,7 @@ CookieDeletionInfo DeletionFilterToInfo(mojom::CookieDeletionFilterPtr filter) {
   delete_info.name = std::move(filter->cookie_name);
   delete_info.url = std::move(filter->url);
   delete_info.host = std::move(filter->host_name);
+  BRAVE_DELETIONFILTERTOINFO
 
   switch (filter->session_control) {
     case mojom::CookieDeletionSessionControl::IGNORE_CONTROL:
