diff --git a/services/network/cookie_settings.cc b/services/network/cookie_settings.cc
index 0f56483e3e6c0f8ab2c1c3502ef112567ac0e301..14c953deed882e0b84aab95e9f2f2af2f1533d88 100644
--- a/services/network/cookie_settings.cc
+++ b/services/network/cookie_settings.cc
@@ -7,8 +7,6 @@
 #include <functional>
 
 #include "base/bind.h"
-#include "net/base/net_errors.h"
-#include "net/base/static_cookie_policy.h"
 
 namespace network {
 namespace {
@@ -30,10 +28,12 @@ CookieSettings::CreateDeleteCookieOnExitPredicate() const {
                              std::cref(content_settings_));
 }
 
-void CookieSettings::GetCookieSetting(const GURL& url,
-                                      const GURL& first_party_url,
-                                      content_settings::SettingSource* source,
-                                      ContentSetting* cookie_setting) const {
+void CookieSettings::GetCookieSettingInternal(
+    const GURL& url,
+    const GURL& first_party_url,
+    bool is_third_party_request,
+    content_settings::SettingSource* source,
+    ContentSetting* cookie_setting) const {
   if (base::Contains(secure_origin_cookies_allowed_schemes_,
                      first_party_url.scheme()) &&
       url.SchemeIsCryptographic()) {
@@ -64,9 +64,7 @@ void CookieSettings::GetCookieSetting(const GURL& url,
     }
   }
 
-  net::StaticCookiePolicy policy(
-      net::StaticCookiePolicy::BLOCK_ALL_THIRD_PARTY_COOKIES);
-  if (block_third && policy.CanAccessCookies(url, first_party_url) != net::OK)
+  if (block_third && is_third_party_request)
     *cookie_setting = CONTENT_SETTING_BLOCK;
 }
 
