diff --git a/services/network/cors/cors_url_loader.cc b/services/network/cors/cors_url_loader.cc
index 773253fb7afd40e4ad6f93d9e06c915d4469f28c..57b77f1c44bb05ca4314af63187ae3ab4a3e35a1 100644
--- a/services/network/cors/cors_url_loader.cc
+++ b/services/network/cors/cors_url_loader.cc
@@ -1230,7 +1230,7 @@ void CorsURLLoader::MaybeSetOriginHeader(
     return;
   }
 
-  std::string origin_value = tainted_
+  std::string origin_value = tainted_ || IsCrossOriginOnionRequest(request_)
                                  ? url::Origin().Serialize()
                                  : request_.request_initiator->Serialize();
   if (headers_update_params) {
