diff --git a/services/network/cors/cors_url_loader.cc b/services/network/cors/cors_url_loader.cc
index 4dce4cf7cb2be2d51b08e3498a140a1a8412de75..cc74374c393f6f16eb94c8ca298a4b9858331286 100644
--- a/services/network/cors/cors_url_loader.cc
+++ b/services/network/cors/cors_url_loader.cc
@@ -1034,6 +1034,7 @@ void CorsURLLoader::StartRequest() {
         request_.headers.HasHeader(net::HttpRequestHeaders::kOrigin) &&
         AllowUnsafeHeaders();
 
+    BRAVE_CORS_URL_LOADER_START_REQUEST
     if (!has_custom_origin_header_with_bypass) {
       if (tainted_) {
         request_.headers.SetHeader(net::HttpRequestHeaders::kOrigin,
