diff --git a/services/network/public/cpp/BUILD.gn b/services/network/public/cpp/BUILD.gn
index afddb926271f2e56763e5cfdcc9ba6b58d2d2cde..47a4b4cd82c8ff7453670cb355eb86c0e7d6a6cc 100644
--- a/services/network/public/cpp/BUILD.gn
+++ b/services/network/public/cpp/BUILD.gn
@@ -137,6 +137,7 @@ jumbo_component("cpp") {
     "//services/proxy_resolver/public/mojom",
   ]
 
+  public_deps += [ "//brave/services/network/public/cpp" ]
   defines = [ "IS_NETWORK_CPP_IMPL" ]
 }
 
