diff --git a/services/network/public/cpp/network_param_mojom_traits.cc b/services/network/public/cpp/network_param_mojom_traits.cc
index 39280a9f7cba956ec878d9a07bf0da6fa33ac702..c7dd7a6ab8b6bebcc9a182aa3c172d789af3053e 100644
--- a/services/network/public/cpp/network_param_mojom_traits.cc
+++ b/services/network/public/cpp/network_param_mojom_traits.cc
@@ -56,6 +56,7 @@ bool StructTraits<network::mojom::HostPortPairDataView, net::HostPortPair>::
     return false;
   }
   *out = net::HostPortPair(std::move(host), data.port());
+  BRAVE_STRUCT_TRAITS_HOST_PORT_PAIR_READ
   return true;
 }
 
