diff --git a/services/preferences/tracked/device_id_win.cc b/services/preferences/tracked/device_id_win.cc
index ff7fbb620de3f2da7dd2e5525dc36f3b59c01ce6..8791808613bfc5f1ff28e3558cad45e7fe2220e7 100644
--- a/services/preferences/tracked/device_id_win.cc
+++ b/services/preferences/tracked/device_id_win.cc
@@ -15,6 +15,7 @@
 
 MachineIdStatus GetDeterministicMachineSpecificId(std::string* machine_id) {
   DCHECK(machine_id);
+  if (IsMachineIdDisabled()) { return MachineIdStatus::NOT_IMPLEMENTED; }
 
   wchar_t computer_name[MAX_COMPUTERNAME_LENGTH + 1] = {};
   DWORD computer_name_size = base::size(computer_name);
