diff --git a/services/network/public/cpp/cors/cors.cc b/services/network/public/cpp/cors/cors.cc
index 944cf9ee1a4d91b2b733337b3bf210eb9cff10b5..bfbabf6847f29bba408a3194cd7c4888ed89e2bf 100644
--- a/services/network/public/cpp/cors/cors.cc
+++ b/services/network/public/cpp/cors/cors.cc
@@ -423,6 +423,7 @@ bool IsCorsSafelistedHeader(
       "sec-ch-ua-platform",
       "sec-ch-ua-arch",
       "sec-ch-ua-model",
+      "x-brave-partner", // Prevent Allow-Cross-Origin-Request-Header failures
   };
   if (std::find(std::begin(safe_names), std::end(safe_names), lower_name) ==
       std::end(safe_names))
