diff --git a/services/network/public/cpp/BUILD.gn b/services/network/public/cpp/BUILD.gn
index 18ddc5105d783e860ad1472992bc102aa422d085..e44a1886d3f48896cfa68d0c119bd198a2241733 100644
--- a/services/network/public/cpp/BUILD.gn
+++ b/services/network/public/cpp/BUILD.gn
@@ -203,6 +203,7 @@ component("cpp") {
     ":cpp_base",
     ":flags_and_switches",
     ":is_potentially_trustworthy",
+    "//brave/services/network/public/cpp",
     "//net",
     "//services/network/public/mojom",
     "//services/network/public/mojom:url_loader_base",
