diff --git a/services/network/public/cpp/cors/cors.cc b/services/network/public/cpp/cors/cors.cc
index 3b57ea18aa3bb511879e8c02f5d28176ea94a363..a3072cfca02b2f3d2f21d09cbefd1be8d6fdb24d 100644
--- a/services/network/public/cpp/cors/cors.cc
+++ b/services/network/public/cpp/cors/cors.cc
@@ -469,6 +469,7 @@ bool IsCorsSafelistedHeader(const std::string& name, const std::string& value) {
       // was introduced to enable safely experimenting with sending a reduced
       // user agent string in the `User-Agent` header.
       "sec-ch-ua-reduced",
+      "x-brave-partner", // Prevent Allow-Cross-Origin-Request-Header failures
 
       // The `Sec-CH-Prefers-Color-Scheme` header field is modeled after the
       // prefers-color-scheme user preference media feature. It reflects the
