diff --git a/services/network/public/cpp/BUILD.gn b/services/network/public/cpp/BUILD.gn
index 370122e95e99f3be6ba11837f04c864f64cbceab..f2ef86b8ebe08e14b81eb29b9014f739b1f560a9 100644
--- a/services/network/public/cpp/BUILD.gn
+++ b/services/network/public/cpp/BUILD.gn
@@ -109,6 +109,7 @@ jumbo_component("cpp") {
     "//services/proxy_resolver/public/mojom",
   ]
 
+  deps += [ "//brave/services/network/public/cpp" ]
   defines = [ "IS_NETWORK_CPP_IMPL" ]
 
   if (is_chromecast) {
