diff --git a/services/preferences/tracked/BUILD.gn b/services/preferences/tracked/BUILD.gn
index 20404816cc91a0a0a670f61a60be411ce7e2a6bd..da3d33887e76e7fe3443f5e8f29c8e2e543556a2 100644
--- a/services/preferences/tracked/BUILD.gn
+++ b/services/preferences/tracked/BUILD.gn
@@ -67,6 +67,7 @@ source_set("tracked") {
   if (is_mac) {
     frameworks = [ "IOKit.framework" ]
   }
+  import("//brave/services/preferences/tracked/sources.gni") deps += brave_services_preferences_tracked_deps
 }
 
 source_set("unit_tests") {
