diff --git a/services/network/public/mojom/cookie_manager.mojom b/services/network/public/mojom/cookie_manager.mojom
index c736353207ff9b24d7751168d6df3d9375969f95..3af2199bb4227f629c0af6fd23f313f1b3048a99 100644
--- a/services/network/public/mojom/cookie_manager.mojom
+++ b/services/network/public/mojom/cookie_manager.mojom
@@ -274,6 +274,7 @@ struct CookieDeletionFilter {
 
   // Delete session/persistent cookies.
   CookieDeletionSessionControl session_control = IGNORE_CONTROL;
+  string? ephemeral_storage_domain;
 };
 
 interface CookieChangeListener {
