diff --git a/storage/browser/blob/blob_url_store_impl.cc b/storage/browser/blob/blob_url_store_impl.cc
index 75e8768b04f1f40b8a12797083094084de150dce..36d16a1192d74a613e39832cfd9e2427a48a7a92 100644
--- a/storage/browser/blob/blob_url_store_impl.cc
+++ b/storage/browser/blob/blob_url_store_impl.cc
@@ -178,6 +178,7 @@ bool BlobURLStoreImpl::BlobUrlIsValid(const GURL& url,
   } else {
     valid_origin = origin_ == url_origin;
   }
+  BRAVE_BLOB_URL_STORE_IMPL_BLOB_URL_IS_VALID
   if (!valid_origin) {
     mojo::ReportBadMessage(base::StrCat(
         {"URL with invalid origin passed to BlobURLStore::", method}));
