diff --git a/storage/browser/blob/blob_url_store_impl.cc b/storage/browser/blob/blob_url_store_impl.cc
index 8b318b4824c955c9abd14c246e7104958b29a82a..3cfc516a353eb387a0f596884e30b3100a34a8d5 100644
--- a/storage/browser/blob/blob_url_store_impl.cc
+++ b/storage/browser/blob/blob_url_store_impl.cc
@@ -198,6 +198,7 @@ bool BlobURLStoreImpl::BlobUrlIsValid(const GURL& url,
   } else {
     valid_origin = storage_key_origin == url_origin;
   }
+  BRAVE_BLOB_URL_STORE_IMPL_BLOB_URL_IS_VALID
   if (!valid_origin) {
     mojo::ReportBadMessage(base::StrCat(
         {"URL with invalid origin passed to BlobURLStore::", method}));
