diff --git a/storage/browser/blob/blob_url_store_impl.cc b/storage/browser/blob/blob_url_store_impl.cc
index 5d36c6d54102f9d6e4d747e8eeda64ac3794a749..e9ca91ac102ad92c166d9dc0858c0ce8ed846f88 100644
--- a/storage/browser/blob/blob_url_store_impl.cc
+++ b/storage/browser/blob/blob_url_store_impl.cc
@@ -198,6 +198,7 @@ bool BlobURLStoreImpl::BlobUrlIsValid(const GURL& url,
   } else {
     valid_origin = storage_key_origin == url_origin;
   }
+  BRAVE_BLOB_URL_STORE_IMPL_BLOB_URL_IS_VALID
   if (!valid_origin) {
     mojo::ReportBadMessage(base::StrCat(
         {"URL with invalid origin passed to BlobURLStore::", method}));
