diff --git a/testing/xvfb.py b/testing/xvfb.py
index 5fec960b9843eb57d7dcdc451231c6c101f1000f..af2ef7146cfb0b6815e072213f891767f9b23412 100755
--- a/testing/xvfb.py
+++ b/testing/xvfb.py
@@ -112,7 +112,7 @@ def dbus_session(env):  # pylint: disable=inconsistent-return-statements
 def run_executable(cmd,
                    env,
                    stdoutfile=None,
-                   use_openbox=True,
+                   use_openbox=False,
                    use_xcompmgr=True,
                    xvfb_whd=None,
                    cwd=None):
