diff --git a/testing/xvfb.py b/testing/xvfb.py
index 2a7febeaf56ad8bbeb8ad481ab25472026fe13fd..05dadf76d136150f25a16acc5a681a73db871743 100755
--- a/testing/xvfb.py
+++ b/testing/xvfb.py
@@ -112,7 +112,7 @@ def dbus_session(env):  # pylint: disable=inconsistent-return-statements
 def run_executable(cmd,
                    env,
                    stdoutfile=None,
-                   use_openbox=True,
+                   use_openbox=False,
                    use_xcompmgr=True,
                    xvfb_whd=None,
                    cwd=None):
