diff --git a/third_party/blink/renderer/modules/BUILD.gn b/third_party/blink/renderer/modules/BUILD.gn
index fe0ea271c406c1c722e5f301fd59e56e47261a0c..1363e216e32fb3da87ca1acd1de3accfda549f89 100644
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -16,6 +16,7 @@ if (is_android) {
 }
 
 visibility = [ "//third_party/blink/renderer/*" ]
+visibility += brave_blink_renderer_modules_visibility
 
 config("modules_implementation") {
   defines = [ "BLINK_MODULES_IMPLEMENTATION=1" ]
@@ -176,6 +177,7 @@ component("modules") {
     "//third_party/blink/renderer/modules/xr",
   ]
 
+  sub_modules += brave_blink_sub_modules
   if (is_android) {
     sub_modules += [ "//third_party/blink/renderer/modules/remote_objects" ]
   } else {
