diff --git a/third_party/blink/renderer/core/dom/events/event_listener_map.cc b/third_party/blink/renderer/core/dom/events/event_listener_map.cc
index b6f19c4a7c01dc9298e0927cf11f909b735379c8..9a70690ffe5a7413ca70da0a45d83d9471080c21 100644
--- a/third_party/blink/renderer/core/dom/events/event_listener_map.cc
+++ b/third_party/blink/renderer/core/dom/events/event_listener_map.cc
@@ -42,6 +42,8 @@
 #include "third_party/blink/renderer/platform/wtf/threading_primitives.h"
 #endif
 
+#include "brave/third_party/blink/brave_page_graph/buildflags/buildflags.h"
+
 namespace blink {
 
 #if DCHECK_IS_ON()
@@ -99,7 +101,13 @@ static bool AddListenerToVector(EventListenerVector* vector,
                                 EventListener* listener,
                                 const AddEventListenerOptionsResolved* options,
                                 RegisteredEventListener* registered_listener) {
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  int registered_listener_id = registered_listener->Id();
+#endif
   *registered_listener = RegisteredEventListener(listener, options);
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  registered_listener->SetId(registered_listener_id);
+#endif
 
   if (vector->Find(*registered_listener) != kNotFound)
     return false;  // Duplicate listener.
