diff --git a/third_party/tflite/BUILD.gn b/third_party/tflite/BUILD.gn
index 6f0bce0469136c235ab969808d03db558782db57..d2289f2fecba794731d9125a78e8c15edb3a6aac 100644
--- a/third_party/tflite/BUILD.gn
+++ b/third_party/tflite/BUILD.gn
@@ -245,6 +245,7 @@ tflite_static_library("tflite_builtin_op_resolver") {
     "//services/webnn/*",
     "//third_party/mediapipe/*",
   ]
+  import("//brave/third_party/tflite/tflite.gni") visibility += brave_tflite_builtin_op_resolver_visibility
 
   conditional_deps = [ ":tflite" ]
 }
