diff --git a/third_party/blink/renderer/modules/encryptedmedia/navigator_request_media_key_system_access.cc b/third_party/blink/renderer/modules/encryptedmedia/navigator_request_media_key_system_access.cc
index 9ac6db76044365b16464222cbcea319308411243..6acb31b57a8cb2110b61ce8f1821ca9fc2cbef1b 100644
--- a/third_party/blink/renderer/modules/encryptedmedia/navigator_request_media_key_system_access.cc
+++ b/third_party/blink/renderer/modules/encryptedmedia/navigator_request_media_key_system_access.cc
@@ -359,6 +359,9 @@ ScriptPromise NavigatorRequestMediaKeySystemAccess::requestMediaKeySystemAccess(
   media_client->RequestMediaKeySystemAccess(
       WebEncryptedMediaRequest(initializer));
 
+  // Notify Brave about the widevine availability request.
+  MaybeOnWidevineRequest(initializer, document->GetFrame());
+
   // 7. Return promise.
   return promise;
 }
