diff --git a/third_party/blink/common/permissions/permission_utils.cc b/third_party/blink/common/permissions/permission_utils.cc
index baa74bd6c7bf6350f4ef06f00d94d1d517b43943..3c1ba0373365effeae42bd30d2229b58be606492 100644
--- a/third_party/blink/common/permissions/permission_utils.cc
+++ b/third_party/blink/common/permissions/permission_utils.cc
@@ -27,6 +27,7 @@ mojom::PermissionStatus ToPermissionStatus(const std::string& status) {
 
 std::string GetPermissionString(PermissionType permission) {
   switch (permission) {
+    PERMISSION_UTIL_GET_PERMISSION_STRING
     case PermissionType::GEOLOCATION:
       return "Geolocation";
     case PermissionType::NOTIFICATIONS:
