diff --git a/third_party/androidx/BUILD.gn b/third_party/androidx/BUILD.gn
index 1239b758cd4c9759d10ce7c6d71286729cf5f8a3..27d567bcfeefbbcef20e4ab39579f1a72b15d08c 100644
--- a/third_party/androidx/BUILD.gn
+++ b/third_party/androidx/BUILD.gn
@@ -751,6 +751,7 @@ if (!limit_android_deps) {
     info_path = "../androidx/committed/libs/androidx_fragment_fragment/androidx_fragment_fragment.info"
     enable_bytecode_checks = false
     deps = [
+      ":androidx_activity_activity_java",
       ":androidx_annotation_annotation_experimental_java",
       ":androidx_annotation_annotation_java",
       ":androidx_arch_core_core_common_java",
