diff --git a/third_party/blink/renderer/platform/fonts/font_fallback_list.cc b/third_party/blink/renderer/platform/fonts/font_fallback_list.cc
index 5dc3da69afe9bede13a4d66efa41b8aafbd32bfe..8f069a54bd9c31d0170395f2a1be860343144500 100644
--- a/third_party/blink/renderer/platform/fonts/font_fallback_list.cc
+++ b/third_party/blink/renderer/platform/fonts/font_fallback_list.cc
@@ -176,6 +176,7 @@ scoped_refptr<FontData> FontFallbackList::GetFontData(
         GetFontSelector()->ReportFontLookupByUniqueOrFamilyName(
             curr_family->FamilyName(), font_description,
             DynamicTo<SimpleFontData>(result.get()));
+        BRAVE_GET_FONT_DATA
       }
       if (result) {
         GetFontSelector()->ReportSuccessfulFontFamilyMatch(
