diff --git a/third_party/blink/renderer/modules/background_fetch/background_fetch_manager_test.cc b/third_party/blink/renderer/modules/background_fetch/background_fetch_manager_test.cc
index 499d6f996657f1202b26d07f61f14d1df0ccdd18..de8e8c78f0360b59b560db28a243768ef5832cd0 100644
--- a/third_party/blink/renderer/modules/background_fetch/background_fetch_manager_test.cc
+++ b/third_party/blink/renderer/modules/background_fetch/background_fetch_manager_test.cc
@@ -14,6 +14,7 @@
 #include "third_party/blink/renderer/platform/bindings/script_state.h"
 #include "third_party/blink/renderer/platform/blob/blob_data.h"
 #include "third_party/blink/renderer/platform/testing/task_environment.h"
+#include "third_party/blink/renderer/platform/weborigin/security_origin.h"
 
 namespace blink {
 
@@ -184,4 +185,38 @@ TEST_F(BackgroundFetchManagerTest, BlobsExtracted) {
   EXPECT_FALSE(fetch_api_requests[1]->blob);
 }
 
+TEST_F(BackgroundFetchManagerTest, ParseAllowlistEmpty) {
+  auto origins = BackgroundFetchManager::ParseAllowlist("");
+  EXPECT_TRUE(origins.empty());
+}
+
+TEST_F(BackgroundFetchManagerTest, ParseAllowlistSingleOrigin) {
+  auto origins = BackgroundFetchManager::ParseAllowlist("https://example.com");
+  EXPECT_EQ(origins.size(), 1u);
+  scoped_refptr<const SecurityOrigin> expected =
+      SecurityOrigin::CreateFromString("https://example.com");
+  EXPECT_TRUE(origins.Contains(expected));
+}
+
+TEST_F(BackgroundFetchManagerTest, ParseAllowlistMultipleOrigins) {
+  auto origins = BackgroundFetchManager::ParseAllowlist(
+      "https://example.com, https://api.allowed-origin.org,http://loopback.ip");
+  EXPECT_EQ(origins.size(), 3u);
+
+  EXPECT_TRUE(origins.Contains(
+      SecurityOrigin::CreateFromString("https://example.com")));
+  EXPECT_TRUE(origins.Contains(
+      SecurityOrigin::CreateFromString("https://api.allowed-origin.org")));
+  EXPECT_TRUE(
+      origins.Contains(SecurityOrigin::CreateFromString("http://loopback.ip")));
+}
+
+TEST_F(BackgroundFetchManagerTest, ParseAllowlistInvalidAndOpaqueOrigins) {
+  auto origins = BackgroundFetchManager::ParseAllowlist(
+      "data:text/html,https://example.com,opaque");
+  EXPECT_EQ(origins.size(), 1u);
+  EXPECT_TRUE(origins.Contains(
+      SecurityOrigin::CreateFromString("https://example.com")));
+}
+
 }  // namespace blink
