diff --git a/third_party/blink/renderer/platform/fonts/font_fallback_list.cc b/third_party/blink/renderer/platform/fonts/font_fallback_list.cc
index a57983c24739fbbfc31c64dc030a9639e36fd7a6..097b8ee443f5b9e04e2367656239eae269ccb686 100644
--- a/third_party/blink/renderer/platform/fonts/font_fallback_list.cc
+++ b/third_party/blink/renderer/platform/fonts/font_fallback_list.cc
@@ -169,6 +169,7 @@ const FontData* FontFallbackList::GetFontData(
     if (!result && !curr_family->FamilyName().empty()) {
       result = FontCache::Get().GetFontData(font_description,
                                             curr_family->FamilyName());
+      BRAVE_GET_FONT_DATA
     }
     if (result) {
       return result;
