diff --git a/third_party/blink/renderer/modules/webgl/BUILD.gn b/third_party/blink/renderer/modules/webgl/BUILD.gn
index 37d20e85f9e0895e55b115e259b34ef7fadd99f2..cc25140382033723fe419c667db7f27dc3230e96 100644
--- a/third_party/blink/renderer/modules/webgl/BUILD.gn
+++ b/third_party/blink/renderer/modules/webgl/BUILD.gn
@@ -165,6 +165,7 @@ blink_modules_sources("webgl") {
 
   public_deps = [ "//device/vr/public/mojom:vr_service_blink" ]
   deps = [
+    "//brave/third_party/blink/brave_page_graph",
     "//build:chromeos_buildflags",
     "//third_party/blink/renderer/modules/xr:xr",
   ]
