diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
index b5dcead91fd623700dcbbfce91f99c708166f5e3..b9ee7e2d74a693bcd99e094bee8d34af954fb64e 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
@@ -1883,6 +1883,7 @@ ImageData* BaseRenderingContext2D::getImageData(
 }
 
 ImageData* BaseRenderingContext2D::getImageDataInternal(
+    BRAVE_GET_IMAGE_DATA_PARAMS
     int sx,
     int sy,
     int sw,
@@ -2066,6 +2067,7 @@ ImageData* BaseRenderingContext2D::getImageDataInternal(
     default:
       NOTREACHED();
   }
+  BRAVE_GET_IMAGE_DATA
   ImageData* image_data = ImageData::ValidateAndCreate(
       sw, sh, data_array, image_data_settings, exception_state,
       ImageData::Context2DErrorMode);
