diff --git a/third_party/blink/renderer/bindings/core/v8/scheduled_action.cc b/third_party/blink/renderer/bindings/core/v8/scheduled_action.cc
index 15c2ebe09f8fcf9464b42c934a003b071de27849..795bc42f55a0cd73f1cef0e89971a0c77965f427 100644
--- a/third_party/blink/renderer/bindings/core/v8/scheduled_action.cc
+++ b/third_party/blink/renderer/bindings/core/v8/scheduled_action.cc
@@ -74,6 +74,9 @@ ScheduledAction::ScheduledAction(ScriptState* script_state,
                                  const String& handler)
     : script_state_(
           MakeGarbageCollected<ScriptStateProtectingContext>(script_state)) {
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  parent_script_id_ = script_state->GetIsolate()->GetExecutingScriptId();
+// #endif
   if (script_state->World().IsWorkerWorld() ||
       BindingSecurity::ShouldAllowAccessToFrame(
           EnteredDOMWindow(script_state->GetIsolate()),
@@ -160,11 +163,20 @@ void ScheduledAction::Execute(LocalFrame* frame) {
     // behavior, but this causes failures on
     // wpt/html/webappapis/scripting/processing-model-2/compile-error-cross-origin-setTimeout.html
     // and friends.
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+    ScriptSourceCode source_code(code_,
+        ScriptSourceLocationType::kEvalForScheduledAction);
+    source_code.SetParentScriptId(parent_script_id_);
     frame->GetScriptController().ExecuteScriptAndReturnValue(
-        script_state_->GetContext(),
-        ScriptSourceCode(code_,
-                         ScriptSourceLocationType::kEvalForScheduledAction),
+        script_state_->GetContext(), source_code,
         KURL(), SanitizeScriptErrors::kDoNotSanitize);
+// #else
+//    frame->GetScriptController().ExecuteScriptAndReturnValue(
+//        script_state_->GetContext(),
+//        ScriptSourceCode(code_,
+//                         ScriptSourceLocationType::kEvalForScheduledAction),
+//        KURL(), SanitizeScriptErrors::kDoNotSanitize);
+// #endif
   }
 
   // The frame might be invalid at this point because JavaScript could have
