diff --git a/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc b/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc
index 9fad30c7c554ebfd12f2919eaf91dbd7e9334d77..f65a3286e5e1f54202fcc54d89e0e87297637430 100644
--- a/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc
+++ b/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc
@@ -181,6 +181,7 @@ void ModuleScriptLoader::FetchInternal(
     options.initiator_info.referrer = module_request.ReferrerString();
     options.initiator_info.position = module_request.GetReferrerPosition();
   }
+  BRAVE_MODULE_SCRIPT_LOADER_FETCH_INTERNAL
 
   // Note: |options| should not be modified after here.
   FetchParameters fetch_params(std::move(resource_request), options);
