diff --git a/third_party/blink/renderer/modules/modules_idl_files.gni b/third_party/blink/renderer/modules/modules_idl_files.gni
index 1bb5f1a07397e4f5c2cac8388a96976c12e21787..81919dea27f8c21e45285f1911407a0ee6228445 100644
--- a/third_party/blink/renderer/modules/modules_idl_files.gni
+++ b/third_party/blink/renderer/modules/modules_idl_files.gni
@@ -543,6 +543,7 @@ modules_idl_files =
         ],
         "abspath")
 
+modules_idl_files += brave_modules_idl_files
 if (!is_android) {
   modules_idl_files += get_path_info([
                                        "serial/serial.idl",
@@ -1030,6 +1031,7 @@ modules_dependency_idl_files =
           "xr/navigator_xr.idl",
         ],
         "abspath")
+modules_dependency_idl_files += brave_modules_dependency_idl_files
 
 if (!is_android) {
   modules_dependency_idl_files +=
