diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index 68c45c08f751b15237cf64e1e09390111181c738..277493417e62c3aa7833d282dee317c0392de357 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -80,6 +80,7 @@ ScriptPromise MediaDevices::enumerateDevices(ScriptState* script_state,
   requests_.insert(resolver);
 
   LocalFrame* frame = LocalDOMWindow::From(script_state)->GetFrame();
+  BRAVE_MEDIA_DEVICES_ENUMERATE_DEVICES
   GetDispatcherHost(frame)->EnumerateDevices(
       true /* audio input */, true /* video input */, true /* audio output */,
       true /* request_video_input_capabilities */,
