diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index 03a93fb8a5c90dc2da5d012e70c1afc9c73ad209..c87cf47ea564d1a80dbf5fa1c1992ee021b81f63 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -4,6 +4,7 @@
 
 #include "third_party/blink/renderer/modules/mediastream/media_devices.h"
 
+#include "brave/renderer/brave_content_settings_observer_helper.h"
 #include "services/service_manager/public/cpp/interface_provider.h"
 #include "third_party/blink/public/platform/platform.h"
 #include "third_party/blink/public/platform/task_type.h"
@@ -88,6 +89,9 @@ ScriptPromise MediaDevices::enumerateDevices(ScriptState* script_state) {
         script_state, DOMException::Create(DOMExceptionCode::kNotSupportedError,
                                            "Current frame is detached."));
   }
+  if (!AllowFingerprinting(frame)) {
+    return ScriptPromise::CastUndefined(script_state);
+  }
 
   ScriptPromiseResolver* resolver = ScriptPromiseResolver::Create(script_state);
   ScriptPromise promise = resolver->Promise();
