diff --git a/third_party/blink/renderer/BUILD.gn b/third_party/blink/renderer/BUILD.gn
index 6a7a81d47913a2009494081cc7d36bf7af43f8c3..2711c1f75c94862ba6be9525ed3f9363cc8d9ce6 100644
--- a/third_party/blink/renderer/BUILD.gn
+++ b/third_party/blink/renderer/BUILD.gn
@@ -9,6 +9,7 @@ if (is_clang) {
 }
 
 visibility = [ "//third_party/blink/*" ]
+visibility += brave_blink_renderer_modules_visibility
 
 # arguments --------------------------------------------------------------------
 
