diff --git a/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc b/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
index af14f519bae5db2e8b09af4b8bd270e8ded1cf76..ffb57031c5c6a1ba38195fd510baaa0fe93f3ae2 100644
--- a/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
+++ b/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
@@ -196,6 +196,7 @@ void DOMPluginArray::UpdatePluginData() {
     }
     return;
   }
+  BRAVE_DOM_PLUGINS_UPDATE_PLUGIN_DATA__RESET_PLUGIN_DATA
   PluginData* data = GetPluginData();
   if (!data) {
     dom_plugins_.clear();
@@ -217,6 +218,7 @@ void DOMPluginArray::UpdatePluginData() {
       }
     }
   }
+  BRAVE_DOM_PLUGINS_UPDATE_PLUGIN_DATA__FARBLE_PLUGIN_DATA
 }
 
 void DOMPluginArray::ContextDestroyed() {
