diff --git a/third_party/blink/renderer/core/frame/content_settings_client.cc b/third_party/blink/renderer/core/frame/content_settings_client.cc
index 388d80843737cf1ac7c7dde0941228c88ea49144..038663c0da3f2c69428e819c49b258f71d57a6f4 100644
--- a/third_party/blink/renderer/core/frame/content_settings_client.cc
+++ b/third_party/blink/renderer/core/frame/content_settings_client.cc
@@ -59,6 +59,12 @@ bool ContentSettingsClient::AllowScriptFromSource(bool enabled_per_settings,
   return enabled_per_settings;
 }
 
+bool ContentSettingsClient::AllowFingerprinting(bool enabled_per_settings) {
+  if (client_)
+    return client_->AllowFingerprinting(enabled_per_settings);
+  return enabled_per_settings;
+}
+
 void ContentSettingsClient::GetAllowedClientHintsFromSource(
     const KURL& url,
     WebEnabledClientHints* client_hints) {
