diff --git a/third_party/rust/bytes/v1/BUILD.gn b/third_party/rust/bytes/v1/BUILD.gn
index 21f0d8bfa1fbc2926484bb2535b1f5b693531c38..6c1c337dcc7a9bc5efbb1e43ffddd9f152f75cff 100644
--- a/third_party/rust/bytes/v1/BUILD.gn
+++ b/third_party/rust/bytes/v1/BUILD.gn
@@ -52,7 +52,7 @@ cargo_crate("lib") {
     "std",
   ]
 
-  testonly = true
+  # testonly = true
 
   #####################################################################
   # Tweaking which GN `config`s apply to this target.
