diff --git a/third_party/blink/renderer/core/inspector/inspector_page_agent.cc b/third_party/blink/renderer/core/inspector/inspector_page_agent.cc
index 98229c0fc7ff5c86fe0da4d20936cbcfe90c9720..f03b3e4ffa841e21618f80b2235dd527408a7b67 100644
--- a/third_party/blink/renderer/core/inspector/inspector_page_agent.cc
+++ b/third_party/blink/renderer/core/inspector/inspector_page_agent.cc
@@ -83,6 +83,8 @@
 #include "third_party/blink/renderer/platform/wtf/vector.h"
 #include "v8/include/v8-inspector.h"
 
+#include "brave/third_party/blink/brave_page_graph/buildflags/buildflags.h"
+
 namespace blink {
 
 using protocol::Response;
@@ -835,7 +837,7 @@ void InspectorPageAgent::DidClearDocumentOfWindowObject(LocalFrame* frame) {
     const String world_name = worlds_to_evaluate_on_load_.Get(key);
     if (world_name.IsEmpty()) {
       frame->GetScriptController().ExecuteScriptInMainWorld(
-          source, ScriptSourceLocationType::kUnknown,
+          source, ScriptSourceLocationType::kInspector,
           ScriptController::kExecuteScriptWhenScriptsDisabled);
       continue;
     }
@@ -864,12 +866,14 @@ void InspectorPageAgent::DidClearDocumentOfWindowObject(LocalFrame* frame) {
     // a foreign world.
     v8::HandleScope handle_scope(V8PerIsolateData::MainThreadIsolate());
     frame->GetScriptController().ExecuteScriptInIsolatedWorld(
-        world_id, source, KURL(), SanitizeScriptErrors::kSanitize);
+        world_id,
+        ScriptSourceCode(source, ScriptSourceLocationType::kInspector), KURL(),
+        SanitizeScriptErrors::kSanitize);
   }
 
   if (!script_to_evaluate_on_load_once_.IsEmpty()) {
     frame->GetScriptController().ExecuteScriptInMainWorld(
-        script_to_evaluate_on_load_once_, ScriptSourceLocationType::kUnknown,
+        script_to_evaluate_on_load_once_, ScriptSourceLocationType::kInspector,
         ScriptController::kExecuteScriptWhenScriptsDisabled);
   }
 }
@@ -1352,6 +1356,24 @@ Response InspectorPageAgent::waitForDebugger() {
   return Response::OK();
 }
 
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+Response InspectorPageAgent::generatePageGraph(String* data) {
+  LocalFrame* main_frame = inspected_frames_->Root();
+  if (!main_frame) {
+    return Response::Error("No main frame found");
+  }
+
+  Document* document = main_frame->GetDocument();
+  if (!document) {
+    return Response::Error("No Document instance to generate page graph for");
+  }
+
+  std::string page_graph(document->GetPageGraph()->ToGraphML());
+  *data = String::FromUTF8(page_graph.c_str());
+  return Response::OK();
+}
+#endif
+
 protocol::Response InspectorPageAgent::generateTestReport(const String& message,
                                                           Maybe<String> group) {
   Document* document = inspected_frames_->Root()->GetDocument();
