diff --git a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
index fad810a314c0594948b72c4c095cb6859cf96eaf..f1e56a658c4c3829000bcfb1bf9cdfd815d9a6be 100644
--- a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
+++ b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
@@ -360,6 +360,7 @@ class CORE_EXPORT HTMLCanvasElement final
 
   void ResetLayer();
 
+  BRAVE_HTML_CANVAS_ELEMENT_H_
  protected:
   void DidMoveToNewDocument(Document& old_document) override;
   void DidRecalcStyle(const StyleRecalcChange change) override;
