diff --git a/third_party/blink/renderer/core/loader/resource/css_style_sheet_resource.h b/third_party/blink/renderer/core/loader/resource/css_style_sheet_resource.h
index 869ce525c1ef06dcb9a69f6f109947ff06ff5e0a..da2dc0ee25ef326f774c87baba090417a1cfc9fe 100644
--- a/third_party/blink/renderer/core/loader/resource/css_style_sheet_resource.h
+++ b/third_party/blink/renderer/core/loader/resource/css_style_sheet_resource.h
@@ -32,6 +32,7 @@
 #include "third_party/blink/renderer/platform/loader/fetch/text_resource_decoder_options.h"
 #include "third_party/blink/renderer/platform/wtf/text/text_encoding.h"
 
+#include <string>
 namespace blink {
 
 class CSSParserContext;
@@ -105,6 +106,8 @@ struct DowncastTraits<CSSStyleSheetResource> {
   }
 };
 
+std::string StyleSheetDigest(CSSStyleSheetResource* resource);
+std::string ResourceDigest(Resource* resource);
 }  // namespace blink
 
 #endif  // THIRD_PARTY_BLINK_RENDERER_CORE_LOADER_RESOURCE_CSS_STYLE_SHEET_RESOURCE_H_
