diff --git a/third_party/blink/public/devtools_protocol/browser_protocol.pdl b/third_party/blink/public/devtools_protocol/browser_protocol.pdl
index 1b776447aa641e76ad2818ccea7c6b56599a730f..5309c335850e2ff916af5be8975d01dd3d7186e2 100644
--- a/third_party/blink/public/devtools_protocol/browser_protocol.pdl
+++ b/third_party/blink/public/devtools_protocol/browser_protocol.pdl
@@ -7166,6 +7166,8 @@ domain Page
       web-share
       window-placement
       xr-spatial-tracking
+      ethereum
+      solana
 
   # Reason for a permissions policy feature to be disabled.
   experimental type PermissionsPolicyBlockReason extends string
