diff --git a/third_party/blink/renderer/core/script/dynamic_module_resolver.cc b/third_party/blink/renderer/core/script/dynamic_module_resolver.cc
index 46533d91f87cbe9d8812676b6acf1d8f9058d109..ecc341af924665a594fd4fc70db3ce03282336e5 100644
--- a/third_party/blink/renderer/core/script/dynamic_module_resolver.cc
+++ b/third_party/blink/renderer/core/script/dynamic_module_resolver.cc
@@ -287,6 +287,7 @@ void DynamicModuleResolver::ResolveDynamically(
       referrer_info.ParserState(), referrer_info.CredentialsMode(),
       referrer_info.GetReferrerPolicy(), mojom::blink::FetchPriorityHint::kAuto,
       RenderBlockingBehavior::kNonBlocking);
+  BRAVE_DYNAMIC_MODULE_RESOLVER_RESOLVE_DYNAMICALLY
 
   // <spec label="fetch-an-import()-module-script-graph" step="3">Fetch a single
   // module script given url, settings object, "script", options, settings
