diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index 08261356c8edf6a05f4e7e38988df209aa7a4714..0e818a44cad143416c78e9d155c979d866efca19 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -297,6 +297,7 @@
     {
       // Interest Group JS API/runtimeflag.
       name: "AdInterestGroupAPI",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
       origin_trial_feature_name: "AdInterestGroupAPI",
       implied_by: ["Fledge", "Parakeet"],
@@ -412,6 +413,7 @@
     },
     {
       name: "AIPromptAPI",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       public: true,
       status: {
         "Win": "stable",
@@ -439,6 +441,7 @@
     },
     {
       name: "AIPromptAPIMultimodalInput",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: {
         "Win": "stable",
         "Mac": "stable",
@@ -493,7 +496,7 @@
       origin_trial_feature_name: "AIProofreaderAPI",
       origin_trial_os: ["win", "mac", "linux", "chromeos"],
       origin_trial_allows_third_party: true,
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -508,7 +511,7 @@
       origin_trial_feature_name: "AIRewriterAPI",
       origin_trial_os: ["win", "mac", "linux", "chromeos"],
       origin_trial_allows_third_party: true,
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -517,6 +520,7 @@
     },
     {
       name: "AISummarizationAPI",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: {
         "Win": "stable",
         "Mac": "stable",
@@ -544,7 +548,7 @@
       origin_trial_feature_name: "AIWriterAPI",
       origin_trial_os: ["win", "mac", "linux", "chromeos"],
       origin_trial_allows_third_party: true,
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -1341,7 +1345,7 @@
       // objects that live on the compositor thread.
       // [1] https://drafts.csswg.org/css-animations-2/#timeline-triggers
       name: "CompositorTimelineTrigger",
-      status: "stable",
+      status: "experimental", // Brave: revert when crbug.com/541924666 is fixed.
     },
     {
       name: "CompressionDictionaryTransport",
@@ -1432,6 +1436,7 @@
       // https://github.com/WICG/controlled-frame/blob/main/README.md for more
       // info.
       name: "ControlledFrame",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       public: true,
       status: "stable",
     },
@@ -2999,6 +3004,7 @@
     {
       // In-development features for the File System Access API.
       name: "FileSystemAccessAPIExperimental",
+      base_feature: "none",
       status: "experimental",
     },
     {
@@ -3013,6 +3019,7 @@
     {
       // Non-OPFS File System Access API.
       name: "FileSystemAccessLocal",
+      base_feature: "none",
       status: {"default": "stable"},
     },
     {
@@ -3147,6 +3154,7 @@
     },
     {
       name: "Fledge",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
       public: true,
     },
@@ -3851,7 +3859,7 @@
         "Android": "test",
         "default": "",
       },
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -4228,6 +4236,7 @@
     // "experimental" is the support on other platforms.
     {
       name: "MiddleClickAutoscroll",
+      base_feature: "none",
       status: "test",
     },
     // Killswitch for crbug.com/40062462 fix.
@@ -4672,6 +4681,7 @@
     {
       // PARAKEET ad serving runtime flag/JS API.
       name: "Parakeet",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       origin_trial_feature_name: "Parakeet",
     },
     {
@@ -4852,6 +4862,7 @@
       //
       // It also has some feature params defined throughout the codebase.
       name: "Prerender2",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
     },
     {
@@ -5091,6 +5102,7 @@
     // but we still keep this flag for future phase tests.
     {
       name: "ReduceUserAgentMinorVersion",
+      base_feature_status: "enabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
     },
     {
@@ -6567,7 +6579,7 @@
         "ChromeOS": "stable",
         "default": "",
       },
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -6756,6 +6768,7 @@
     },
     {
       name: "UserMediaElement",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
       public: true,
     },
@@ -7564,5 +7577,10 @@
       // crbug.com/435623334.
       name: "XSLTSpecialTrial",
     },
+    {
+      name: "BraveIsInTorContext",
+      public: true,
+      base_feature: "none",
+    },
   ],
 }
