diff --git a/third_party/blink/common/BUILD.gn b/third_party/blink/common/BUILD.gn
index bb178b847751d363b6bc2714f9360e33bcc3dc85..dd3e8cbb691f1711bab9c40b1baa555fcf17203b 100644
--- a/third_party/blink/common/BUILD.gn
+++ b/third_party/blink/common/BUILD.gn
@@ -353,6 +353,7 @@ source_set("common") {
   # TODO(crbug.com/40031409): Fix code that adds exit-time destructors and
   # enable the diagnostic by removing this line.
   configs += [ "//build/config/compiler:no_exit_time_destructors" ]
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_blink_common_deps
 }
 
 test("blink_common_unittests") {
@@ -497,6 +498,7 @@ source_set("common_unittests_sources") {
   # TODO(crbug.com/40031409): Fix code that adds exit-time destructors and
   # enable the diagnostic by removing this line.
   configs += [ "//build/config/compiler:no_exit_time_destructors" ]
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_blink_common_deps
 }
 
 group("common_unittests_data") {
