diff --git a/third_party/blink/renderer/core/html/html_link_element.cc b/third_party/blink/renderer/core/html/html_link_element.cc
index 5ef87c2c9138517b989e10f292c9ef3f47f9c8b9..2a69e3909069a560bda37843a53c1363410f9542 100644
--- a/third_party/blink/renderer/core/html/html_link_element.cc
+++ b/third_party/blink/renderer/core/html/html_link_element.cc
@@ -59,7 +59,11 @@ HTMLLinkElement::HTMLLinkElement(Document& document,
       referrer_policy_(network::mojom::ReferrerPolicy::kDefault),
       sizes_(MakeGarbageCollected<DOMTokenList>(*this, html_names::kSizesAttr)),
       rel_list_(MakeGarbageCollected<RelList>(this)),
-      created_by_parser_(flags.IsCreatedByParser()) {}
+      created_by_parser_(flags.IsCreatedByParser()) {
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  link_loader_->SetHTMLLinkElementOwner(this);
+#endif
+}
 
 HTMLLinkElement* HTMLLinkElement::Create(Document& document,
                                          const CreateElementFlags flags) {
