diff --git a/third_party/blink/renderer/core/loader/resource/script_resource.h b/third_party/blink/renderer/core/loader/resource/script_resource.h
index f6c5707e3edcb9e0a064a19f6ccaea4bbde7b922..b0eaba28eed06305146525cec0e6a8f3a91d13dc 100644
--- a/third_party/blink/renderer/core/loader/resource/script_resource.h
+++ b/third_party/blink/renderer/core/loader/resource/script_resource.h
@@ -37,6 +37,7 @@
 #include "third_party/blink/renderer/platform/loader/fetch/resource_loader_options.h"
 #include "third_party/blink/renderer/platform/loader/fetch/text_resource_decoder_options.h"
 
+#include <string>
 namespace blink {
 
 class FetchParameters;
@@ -205,6 +206,8 @@ struct DowncastTraits<ScriptResource> {
   }
 };
 
+std::string ScriptDigest(ScriptResource* resource);
+std::string ResourceDigest(Resource* resource);
 }  // namespace blink
 
 #endif  // THIRD_PARTY_BLINK_RENDERER_CORE_LOADER_RESOURCE_SCRIPT_RESOURCE_H_
