diff --git a/third_party/blink/renderer/platform/wtf/BUILD.gn b/third_party/blink/renderer/platform/wtf/BUILD.gn
index e8f3789f241e2466fcaa4adaecbe4fa44ae06736..e509a83a9fe41bcb4b9a19055a5a913cc4456260 100644
--- a/third_party/blink/renderer/platform/wtf/BUILD.gn
+++ b/third_party/blink/renderer/platform/wtf/BUILD.gn
@@ -19,6 +19,7 @@ buildflag_header("buildflags") {
 
 visibility = [
   ":*",
+  "//brave/third_party/blink/brave_page_graph:brave_page_graph",
   "//:gn_all",
   "//mojo/public/cpp/bindings/*",
   "//third_party/blink/*",
