diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index fc3eb84197366d770d97163641113d19991d5c89..8295a4ed68bffaa1244963dae1a6f8f092ac748c 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -806,6 +806,11 @@
       public: true,
       status: "stable",
     },
+    {
+      name: "BraveIsInTorContext",
+      public: true,
+      base_feature: "none",
+    },
     {
       name: "BrowserInitiatedAutomaticPictureInPicture",
       public: true,
@@ -2454,6 +2459,7 @@
       // In-development features for the File System Access API.
       name: "FileSystemAccessAPIExperimental",
       status: "experimental",
+      base_feature: "none",
     },
     {
       // The FileSystemHandle.getCloudIdentifiers() method (see
@@ -2468,6 +2474,7 @@
       // Non-OPFS File System Access API.
       name: "FileSystemAccessLocal",
       status: {"default": "stable"},
+      base_feature: "none",
     },
     {
       name: "FileSystemAccessLockingScheme",
@@ -3466,6 +3473,7 @@
     {
       name: "MiddleClickAutoscroll",
       status: "test",
+      base_feature: "none",
     },
     // Killswitch for crbug.com/40062462 fix.
     {
