diff --git a/third_party/lit/v3_0/BUILD.gn b/third_party/lit/v3_0/BUILD.gn
index 512a9736831262654d1fb8d6f46e10ad1fcf5776..93c382fe010817a14681caca89590e0ce69bb9c6 100644
--- a/third_party/lit/v3_0/BUILD.gn
+++ b/third_party/lit/v3_0/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/third_party/lit/v3_0/sources.gni")
 import("//tools/typescript/ts_library.gni")
 import("//ui/webui/resources/tools/bundle_js.gni")
 import("//ui/webui/resources/tools/generate_code_cache.gni")
@@ -24,6 +25,7 @@ ts_library("build_ts") {
     # dependency to CrOS-only UIs (to reduce coupling between Desktop and CrOS
     # and reduce the maintenance burden for the Desktop WebUI team).
 
+    "//brave/ui/webui/resources:build_ts",
     "//chrome/browser/resources/about_sys:build_ts",
     "//chrome/browser/resources/access_code_cast:build_ts",
     "//chrome/browser/resources/actor_overlay:build_ts",
@@ -183,6 +185,7 @@ ts_library("build_ts") {
     "//ui/webui/resources/cr_components/theme_color_picker:build_ts",
     "//ui/webui/resources/cr_elements:build_ts",
   ]
+  visibility += brave_lit_build_ts_visibility
   tsconfig_base = "tsconfig_base.json"
   composite = true
   in_files = [
