diff --git a/third_party/rust/anyhow/v1/BUILD.gn b/third_party/rust/anyhow/v1/BUILD.gn
index 8531187c964e2767fedb9bfd52dac53f4ae957a5..807fcdbdb55123d4bc443d10b56c774929afcd02 100644
--- a/third_party/rust/anyhow/v1/BUILD.gn
+++ b/third_party/rust/anyhow/v1/BUILD.gn
@@ -48,5 +48,4 @@ cargo_crate("lib") {
   build_sources =
       [ "//third_party/rust/chromium_crates_io/vendor/anyhow-1.0.94/build.rs" ]
 
-  testonly = true
 }
