diff --git a/third_party/blink/public/common/BUILD.gn b/third_party/blink/public/common/BUILD.gn
index 980cc5fb5ffcfb82ddabdd60d6e68cb82d5c1559..08dc1412c48da276aec780d362df57c6cef9e706 100644
--- a/third_party/blink/public/common/BUILD.gn
+++ b/third_party/blink/public/common/BUILD.gn
@@ -388,6 +388,7 @@ source_set("headers") {
   if (is_mac) {
     sources += [ "sandbox_support/sandbox_support_mac.h" ]
   }
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_blink_public_common_deps
 }
 
 if (is_android || is_win) {
