diff --git a/third_party/blink/common/BUILD.gn b/third_party/blink/common/BUILD.gn
index 4a5becb556b206f5912679c79d000d8e17cf018f..258addf641cb2bb1b5ed78c7e3bb65eac931c521 100644
--- a/third_party/blink/common/BUILD.gn
+++ b/third_party/blink/common/BUILD.gn
@@ -353,6 +353,7 @@ source_set("common") {
   if (is_chromeos || is_android || is_win || is_castos || is_fuchsia) {
     defines = [ "ENABLE_PROTECTED_MEDIA_IDENTIFIER_PERMISSION" ]
   }
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_blink_common_deps
 }
 
 test("blink_common_unittests") {
@@ -500,6 +501,7 @@ source_set("common_unittests_sources") {
   # TODO(crbug.com/40031409): Fix code that adds exit-time destructors and
   # enable the diagnostic by removing this line.
   configs += [ "//build/config/compiler:no_exit_time_destructors" ]
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_blink_common_deps
 }
 
 group("common_unittests_data") {
