diff --git a/third_party/blink/renderer/build/scripts/make_instrumenting_probes.py b/third_party/blink/renderer/build/scripts/make_instrumenting_probes.py
index f19596bf0adce1141f3e24a5f0e060c5a4a167a0..93bdd7d8ce58b8a66dcf16821cd570de81ed8932 100644
--- a/third_party/blink/renderer/build/scripts/make_instrumenting_probes.py
+++ b/third_party/blink/renderer/build/scripts/make_instrumenting_probes.py
@@ -306,5 +306,6 @@ def main():
         h_file.close()
 
 
+from import_inline import inline_file_from_src; inline_file_from_src("brave/chromium_src/third_party/blink/renderer/build/scripts/make_instrumenting_probes.py", globals(), locals())
 if __name__ == "__main__":
     main()
