diff --git a/third_party/blink/public/common/features.h b/third_party/blink/public/common/features.h
index 1babe5b230878..4d838ba47ac17 100644
--- a/third_party/blink/public/common/features.h
+++ b/third_party/blink/public/common/features.h
@@ -708,6 +708,8 @@ BLINK_COMMON_EXPORT BASE_DECLARE_FEATURE(kIntensiveWakeUpThrottling);
 BLINK_COMMON_EXPORT extern const char
     kIntensiveWakeUpThrottling_GracePeriodSeconds_Name[];
 
+BLINK_COMMON_EXPORT BASE_DECLARE_FEATURE(kJXL);
+
 // Don't require FCP for the page to turn interactive. Useful for testing.
 BLINK_COMMON_EXPORT BASE_DECLARE_FEATURE(kInteractiveDetectorIgnoreFcp);
