diff --git a/third_party/blink/renderer/core/html/canvas/canvas_async_blob_creator.cc b/third_party/blink/renderer/core/html/canvas/canvas_async_blob_creator.cc
index 6b1c447c605142bbf9cfddb845b6972086cd6759..1c82c747b887e470692ba35e358918304caf669c 100644
--- a/third_party/blink/renderer/core/html/canvas/canvas_async_blob_creator.cc
+++ b/third_party/blink/renderer/core/html/canvas/canvas_async_blob_creator.cc
@@ -209,6 +209,7 @@ CanvasAsyncBlobCreator::CanvasAsyncBlobCreator(
         src_data_.reset(info, src_data_.addr(), src_data_.rowBytes());
       }
     }
+    BRAVE_CANVAS_ASYNC_BLOB_CREATOR
   }
 
   idle_task_status_ = kIdleTaskNotSupported;
