diff --git a/third_party/blink/renderer/core/script/dynamic_module_resolver.cc b/third_party/blink/renderer/core/script/dynamic_module_resolver.cc
index 1e25bac62e4eb1bd1df8c1d5c16d5ec55daeca2a..e985489f8c8bbdef38dacdc5d3b0c28b285dedd6 100644
--- a/third_party/blink/renderer/core/script/dynamic_module_resolver.cc
+++ b/third_party/blink/renderer/core/script/dynamic_module_resolver.cc
@@ -287,6 +287,7 @@ void DynamicModuleResolver::ResolveDynamically(
       referrer_info.ParserState(), referrer_info.CredentialsMode(),
       referrer_info.GetReferrerPolicy(), mojom::blink::FetchPriorityHint::kAuto,
       RenderBlockingBehavior::kNonBlocking);
+  BRAVE_DYNAMIC_MODULE_RESOLVER_RESOLVE_DYNAMICALLY
 
   // <spec label="fetch-an-import()-module-script-graph" step="3">Fetch a single
   // module script given url, settings object, "script", options, settings
