diff --git a/third_party/WebKit/Source/modules/canvas/canvas2d/CanvasRenderingContext2D.cpp b/third_party/WebKit/Source/modules/canvas/canvas2d/CanvasRenderingContext2D.cpp
index 1f8a3c6c8acde3818902ef0cb9e697222e8a3de5..e608bdb7d53a8a8bd1f3f8fc3f81d797e9d7bc58 100644
--- a/third_party/WebKit/Source/modules/canvas/canvas2d/CanvasRenderingContext2D.cpp
+++ b/third_party/WebKit/Source/modules/canvas/canvas2d/CanvasRenderingContext2D.cpp
@@ -36,6 +36,7 @@
 #include "bindings/core/v8/ExceptionMessages.h"
 #include "bindings/core/v8/ExceptionState.h"
 #include "bindings/modules/v8/rendering_context.h"
+#include "brave/renderer/brave_content_settings_observer_helper.h"
 #include "core/CSSPropertyNames.h"
 #include "core/css/CSSFontSelector.h"
 #include "core/css/CSSPropertyValueSet.h"
@@ -768,6 +769,9 @@ TextMetrics* CanvasRenderingContext2D::measureText(const String& text) {
   if (!canvas()->GetDocument().GetFrame())
     return TextMetrics::Create();
 
+  if (!AllowFingerprinting(canvas()->GetDocument().GetFrame()))
+    return TextMetrics::Create();
+
   canvas()->GetDocument().UpdateStyleAndLayoutTreeForNode(canvas());
 
   const Font& font = AccessFont();
