diff --git a/third_party/android_deps/BUILD.gn b/third_party/android_deps/BUILD.gn
index 051c3b106cc9a0eff967c4d9d3c74ab1b5e9fbd5..e6619594d538019472c25d90adf5cf9e8297f4a1 100644
--- a/third_party/android_deps/BUILD.gn
+++ b/third_party/android_deps/BUILD.gn
@@ -470,6 +470,7 @@ java_prebuilt("com_google_auto_value_auto_value_annotations_java") {
     ":*",
     "//third_party/androidx:*",
   ]
+  visibility += [ "//brave/third_party/android_deps:com_google_android_datatransport_transport_runtime_java__classes" ] supports_android = true
 }
 
 # This is generated, do not edit. Update BuildConfigGenerator.groovy instead.
