diff --git a/third_party/blink/renderer/BUILD.gn b/third_party/blink/renderer/BUILD.gn
index 1806d2ed997f943ea1920ba61cc504a56a3ef2b3..466e7cb4d5a6593346ff4a79415f053220770e52 100644
--- a/third_party/blink/renderer/BUILD.gn
+++ b/third_party/blink/renderer/BUILD.gn
@@ -10,6 +10,7 @@ if (is_clang) {
 }
 
 visibility = [ "//third_party/blink/*" ]
+visibility += brave_blink_renderer_modules_visibility
 
 # arguments --------------------------------------------------------------------
 
