diff --git a/third_party/tflite/BUILD.gn b/third_party/tflite/BUILD.gn
index 3200cb360d12997fd1c792d22772ffae5a44c632..f3cf37b17f354fecf6581fb709fd68a1da34e065 100644
--- a/third_party/tflite/BUILD.gn
+++ b/third_party/tflite/BUILD.gn
@@ -396,6 +396,7 @@ if (!use_litert_tflite) {
 
     visibility = [
       ":tflite_benchmark",
+      "//brave/services/passage_embeddings/*",
       "//components/optimization_guide/internal/*",
       "//modules/*",
       "//services/webnn/*",
