diff --git a/third_party/blink/public/devtools_protocol/browser_protocol.pdl b/third_party/blink/public/devtools_protocol/browser_protocol.pdl
index ebfcc5da78f830045de9c2dc48d02bb8177dff72..311126190a9cc29a89df56a5ec737aaa2cd70db3 100644
--- a/third_party/blink/public/devtools_protocol/browser_protocol.pdl
+++ b/third_party/blink/public/devtools_protocol/browser_protocol.pdl
@@ -7159,6 +7159,8 @@ domain Page
       web-share
       window-placement
       xr-spatial-tracking
+      ethereum
+      solana
 
   # Reason for a permissions policy feature to be disabled.
   experimental type PermissionsPolicyBlockReason extends string
