diff --git a/third_party/blink/renderer/modules/storage/storage_namespace.h b/third_party/blink/renderer/modules/storage/storage_namespace.h
index 4eaca73c0f0be4e873753564699e0c813b694bf9..5f7a33b15d28e1f8ff95476bc432882a9df7d362 100644
--- a/third_party/blink/renderer/modules/storage/storage_namespace.h
+++ b/third_party/blink/renderer/modules/storage/storage_namespace.h
@@ -119,6 +119,7 @@ class MODULES_EXPORT StorageNamespace final
   // forcibly disconnects it so that it reconnects lazily when next needed.
   // Also forces all owned CachedStorageAreas to be reconnected.
   void ResetStorageAreaAndNamespaceConnections();
+  BRAVE_STORAGE_NAMESPACE_H
 
  private:
   void EnsureConnected();
