diff --git a/third_party/rust/rand/v0_8/BUILD.gn b/third_party/rust/rand/v0_8/BUILD.gn
index 55c80c0f479398e606fc1c0af01071395972d104..08343846efb5eb2b473a8cc213ba6684dba5b475 100644
--- a/third_party/rust/rand/v0_8/BUILD.gn
+++ b/third_party/rust/rand/v0_8/BUILD.gn
@@ -77,5 +77,5 @@ cargo_crate("lib") {
     "--cap-lints=allow",  # Suppress all warnings in crates.io crates
   ]
 
-  testonly = true
+  features += [ "small_rng" ]
 }
