diff --git a/third_party/blink/renderer/core/frame/navigator_id.idl b/third_party/blink/renderer/core/frame/navigator_id.idl
index 803f5e25f5a7674d625aad3256ba595de61f3e54..94e1e88cbe93361146ff3a25705d96f32d2f78a1 100644
--- a/third_party/blink/renderer/core/frame/navigator_id.idl
+++ b/third_party/blink/renderer/core/frame/navigator_id.idl
@@ -33,10 +33,10 @@
 interface mixin NavigatorID {
     readonly attribute DOMString appCodeName; // constant "Mozilla"
     readonly attribute DOMString appName; // constant "Netscape"
-    [HighEntropy=Direct, MeasureAs=NavigatorAppVersion] readonly attribute DOMString appVersion;
-    [HighEntropy=Direct, MeasureAs=NavigatorPlatform] readonly attribute DOMString platform;
+    [HighEntropy=Direct, MeasureAs=NavigatorAppVersion, TrackInPageGraph] readonly attribute DOMString appVersion;
+    [HighEntropy=Direct, MeasureAs=NavigatorPlatform, TrackInPageGraph] readonly attribute DOMString platform;
     readonly attribute DOMString product; // constant "Gecko"
     // https://www.w3.org/Bugs/Public/show_bug.cgi?id=22555
     // boolean taintEnabled(); // constant false
-    [Affects=Nothing, HighEntropy=Direct, MeasureAs=NavigatorUserAgent] readonly attribute DOMString userAgent;
+    [Affects=Nothing, HighEntropy=Direct, MeasureAs=NavigatorUserAgent, TrackInPageGraph] readonly attribute DOMString userAgent;
 };
