diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index 969d18b5d9d88fdcc9277f4b019bf4cbd1d9a37c..454f75e63982a9e673795ba1cdcf8849b8ee3bcf 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -2217,6 +2217,7 @@
       // In-development features for the File System Access API.
       name: "FileSystemAccessAPIExperimental",
       status: "experimental",
+      base_feature: "none",
     },
     {
       // The FileSystemHandle.getCloudIdentifiers() method (see
@@ -2231,6 +2232,7 @@
       // Non-OPFS File System Access API.
       name: "FileSystemAccessLocal",
       status: {"default": "stable"},
+      base_feature: "none",
     },
     {
       name: "FileSystemAccessLockingScheme",
@@ -3160,6 +3162,7 @@
     {
       name: "MiddleClickAutoscroll",
       status: "test",
+      base_feature: "none",
     },
     // Killswitch for crbug.com/40062462 fix.
     {
