diff --git a/third_party/rust/hashbrown/v0_15/BUILD.gn b/third_party/rust/hashbrown/v0_15/BUILD.gn
index 4f59f945b9b0e089def1cd79e37115da1a1f3800..e983926a805fb918970b5b3e46153bda63c4639e 100644
--- a/third_party/rust/hashbrown/v0_15/BUILD.gn
+++ b/third_party/rust/hashbrown/v0_15/BUILD.gn
@@ -45,6 +45,7 @@ cargo_crate("lib") {
   ]
   inputs = []
 
+  import("//brave/third_party/rust/hashbrown/v0_15/sources.gni") deps = [] deps += brave_third_party_rust_hashbrown_v0_15_deps features = [] features += brave_third_party_rust_hashbrown_v0_15_features
   build_native_rust_unit_tests = false
   edition = "2021"
   cargo_pkg_authors = "Amanieu d'Antras <amanieu@gmail.com>"
