diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index 4feaed1ede9739cb92860c995c1f9730ff0abe84..cb030a011d807e87ee10897c5d0896b087fedf63 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -414,6 +414,7 @@ component("core") {
       "Carbon.framework",
     ]
   }
+  visibility += brave_blink_renderer_core_visibility public_deps += brave_blink_renderer_core_public_deps sources += brave_blink_renderer_core_sources deps += brave_blink_renderer_core_deps
 }
 
 # Certain sources should be compiled with heavy optimization enabled.
