diff --git a/third_party/blink/renderer/core/frame/dom_window.h b/third_party/blink/renderer/core/frame/dom_window.h
index 0423e58ac9a851c5f4420978e8f26cd1e3dc50a7..5ae07eb3dd4549e25794b58123e069e3ab08d589 100644
--- a/third_party/blink/renderer/core/frame/dom_window.h
+++ b/third_party/blink/renderer/core/frame/dom_window.h
@@ -181,6 +181,7 @@ class CORE_EXPORT DOMWindow : public EventTargetWithInlineData {
 
   void DisconnectFromFrame() { frame_ = nullptr; }
 
+  BRAVE_DOM_WINDOW_H
  private:
   void DoPostMessage(scoped_refptr<SerializedScriptValue> message,
                      const MessagePortArray&,
