diff --git a/third_party/blink/renderer/core/frame/web_local_frame_impl.h b/third_party/blink/renderer/core/frame/web_local_frame_impl.h
index c8806c6e842ac2af397049ff94b5ebd9de08162f..771adf1cf159aba60746b2ccdd383cc9e0f2a91f 100644
--- a/third_party/blink/renderer/core/frame/web_local_frame_impl.h
+++ b/third_party/blink/renderer/core/frame/web_local_frame_impl.h
@@ -120,6 +120,7 @@ class CORE_EXPORT WebLocalFrameImpl final
   bool IsLoading() const override;
 
   // WebLocalFrame overrides:
+  void SetOriginForClearWindowNameCheck(const url::Origin&) override;
   WebLocalFrameImpl* CreateLocalChild(
       mojom::blink::TreeScopeType,
       WebLocalFrameClient*,
@@ -597,6 +598,8 @@ class CORE_EXPORT WebLocalFrameImpl final
 
   void WillSendSubmitEvent(const WebFormElement& form);
 
+  void NotifyScriptToolsChanged();
+
  protected:
   // WebLocalFrame protected overrides:
   void AddMessageToConsoleImpl(const WebConsoleMessage&,
