diff --git a/third_party/rust/serde/v1/BUILD.gn b/third_party/rust/serde/v1/BUILD.gn
index 0cc7bd91f39cd6cff8349a73791d04e6cca24509..498d55c57824a02e7bed15eee0f2c00d6e48bf26 100644
--- a/third_party/rust/serde/v1/BUILD.gn
+++ b/third_party/rust/serde/v1/BUILD.gn
@@ -30,4 +30,5 @@ cargo_crate("lib") {
   ]
   build_root = "crate/build.rs"
   build_sources = [ "crate/build.rs" ]
+  features += [ "rc" ]
 }
