diff --git a/third_party/blink/renderer/core/dom/events/event_target.cc b/third_party/blink/renderer/core/dom/events/event_target.cc
index e317e6d31fa6f01d46bb29c4d0dee8db1ab00f2b..2d8eef26ac5b447126462476c8633702e2bf220b 100644
--- a/third_party/blink/renderer/core/dom/events/event_target.cc
+++ b/third_party/blink/renderer/core/dom/events/event_target.cc
@@ -646,6 +646,7 @@ bool EventTarget::AddEventListenerInternal(
       event_type, listener, options, &registered_listener);
   if (added) {
     CHECK(registered_listener);
+    BRAVE_EVENT_TARGET_ADD_EVENT_LISTENER_INTERNAL
     if (options->hasSignal()) {
       // Instead of passing the entire |options| here, which could create a
       // circular reference due to |options| holding a Member<AbortSignal>, just
@@ -782,6 +783,7 @@ bool EventTarget::RemoveEventListenerInternal(
                                     &registered_listener)) {
     return false;
   }
+  BRAVE_EVENT_TARGET_REMOVE_EVENT_LISTENER_INTERNAL
 
   CHECK(registered_listener);
   RemovedEventListener(event_type, *registered_listener);
