diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index 40a9642a4160877f3d79dc55cfe5181fecb0049d..cc0e54c48722b2bbf6dc515c39db27a60580c811 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -6,6 +6,7 @@
 
 #include <utility>
 
+#include "brave/renderer/brave_content_settings_agent_impl_helper.h"
 #include "mojo/public/cpp/bindings/remote.h"
 #include "third_party/blink/public/common/browser_interface_broker_proxy.h"
 #include "third_party/blink/public/platform/task_type.h"
@@ -74,6 +75,9 @@ ScriptPromise MediaDevices::enumerateDevices(ScriptState* script_state,
                                       "Current frame is detached.");
     return ScriptPromise();
   }
+  if (!AllowFingerprinting(frame)) {
+    return ScriptPromise::CastUndefined(script_state);
+  }
 
   auto* resolver = MakeGarbageCollected<ScriptPromiseResolver>(script_state);
   ScriptPromise promise = resolver->Promise();
