diff --git a/third_party/rust/num_traits/v0_2/BUILD.gn b/third_party/rust/num_traits/v0_2/BUILD.gn
index fd27af8cfa04dc5538f6fcd9aca3a12e7ab9e8bb..fb3879970149b0344146c8c6bc8fdca8398def4a 100644
--- a/third_party/rust/num_traits/v0_2/BUILD.gn
+++ b/third_party/rust/num_traits/v0_2/BUILD.gn
@@ -47,7 +47,7 @@ cargo_crate("lib") {
   allow_unsafe = true
 
   build_deps = [ "//third_party/rust/autocfg/v1:buildrs_support" ]
-  features = [ "i128" ]
+  features = [ "default", "i128", "std" ]
 
   # Only for usage from third-party crates. Add the crate to
   # //third_party/rust/chromium_crates_io/Cargo.toml to use
