diff --git a/third_party/blink/renderer/platform/bindings/idl_member_installer.h b/third_party/blink/renderer/platform/bindings/idl_member_installer.h
index 095f734a2af6f8348414746208315e45a4c9222e..4a9907328353620abd594698613159e093757319 100644
--- a/third_party/blink/renderer/platform/bindings/idl_member_installer.h
+++ b/third_party/blink/renderer/platform/bindings/idl_member_installer.h
@@ -184,6 +184,7 @@ class PLATFORM_EXPORT IDLMemberInstaller final {
       v8::Local<v8::Object> interface_object,
       v8::Local<v8::Signature> signature,
       base::span<const ExposedConstructConfig> configs);
+  BRAVE_IDL_MEMBER_INSTALLER_H_
 };
 
 }  // namespace bindings
