diff --git a/third_party/blink/renderer/platform/graphics/bitmap_image_metrics.h b/third_party/blink/renderer/platform/graphics/bitmap_image_metrics.h
index df3c3fdadd3c0..dd2def9afc111 100644
--- a/third_party/blink/renderer/platform/graphics/bitmap_image_metrics.h
+++ b/third_party/blink/renderer/platform/graphics/bitmap_image_metrics.h
@@ -32,8 +32,8 @@ class PLATFORM_EXPORT BitmapImageMetrics {
     kICO = 5,
     kBMP = 6,
     kAVIF = 7,
-    kREMOVED_JXL = 8,
-    kMaxValue = kREMOVED_JXL,
+    kJXL = 8,
+    kMaxValue = kJXL,
   };
 
   // |type| is the return value of ImageDecoder::FilenameExtension().
