diff --git a/third_party/blink/renderer/modules/BUILD.gn b/third_party/blink/renderer/modules/BUILD.gn
index 631ffd545ceabb51ae6a4f3d6938a2d92ba1f8ad..8ca160514061b1f0fb54f46c9c2b2eb5876cef13 100644
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -16,6 +16,7 @@ if (is_android) {
 }
 
 visibility = [ "//third_party/blink/renderer/*" ]
+visibility += brave_blink_renderer_modules_visibility
 
 config("modules_implementation") {
   defines = [ "BLINK_MODULES_IMPLEMENTATION=1" ]
@@ -178,6 +179,7 @@ component("modules") {
     "//third_party/blink/renderer/modules/xr",
   ]
 
+  sub_modules += brave_blink_sub_modules
   if (is_android) {
     sub_modules += [ "//third_party/blink/renderer/modules/remote_objects" ]
   } else {
