diff --git a/third_party/blink/renderer/core/loader/resource/image_resource_content.h b/third_party/blink/renderer/core/loader/resource/image_resource_content.h
index 9e9fe6c6d222593c720d5264faaa98de17b9b90f..7301fcf0bf71ce08576672a87d3e7c96daae7223 100644
--- a/third_party/blink/renderer/core/loader/resource/image_resource_content.h
+++ b/third_party/blink/renderer/core/loader/resource/image_resource_content.h
@@ -21,6 +21,9 @@
 #include "third_party/blink/renderer/platform/wtf/hash_counted_set.h"
 #include "third_party/blink/renderer/platform/wtf/hash_map.h"
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+#include "brave/third_party/blink/brave_page_graph/types.h"
+// #endif
 namespace blink {
 
 class ExecutionContext;
@@ -57,6 +60,10 @@ class CORE_EXPORT ImageResourceContent final
   static ImageResourceContent* CreateLoaded(scoped_refptr<blink::Image>);
 
   static ImageResourceContent* Fetch(FetchParameters&, ResourceFetcher*);
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  static ImageResourceContent* Fetch(FetchParameters&, ResourceFetcher*,
+    brave_page_graph::InspectorId* identifier);
+// #endif
 
   explicit ImageResourceContent(scoped_refptr<blink::Image> = nullptr);
 
