diff --git a/third_party/rust/chromium_crates_io/vendor/cxxbridge-cmd-1.0.135/src/gen/include/cxx.h b/third_party/rust/chromium_crates_io/vendor/cxxbridge-cmd-1.0.135/src/gen/include/cxx.h
index 3414e4c8a5434f6d43b8bf7a76e634ac0c7d06a5..ab588972ad1236c81c9195e77eda20f387da5be5 100644
--- a/third_party/rust/chromium_crates_io/vendor/cxxbridge-cmd-1.0.135/src/gen/include/cxx.h
+++ b/third_party/rust/chromium_crates_io/vendor/cxxbridge-cmd-1.0.135/src/gen/include/cxx.h
@@ -2,6 +2,7 @@
 #include <algorithm>
 #include <array>
 #include <cassert>
+#include <concepts>
 #include <cstddef>
 #include <cstdint>
 #include <exception>
@@ -243,6 +244,11 @@ public:
   bool operator<=(const iterator &) const noexcept;
   bool operator>(const iterator &) const noexcept;
   bool operator>=(const iterator &) const noexcept;
+  using iterator_concept = std::contiguous_iterator_tag;
+  template <std::integral N>
+  friend iterator operator+(N n, const iterator& it) noexcept {
+      return it + n;
+  }
 
 private:
   friend class Slice;
