diff --git a/third_party/blink/renderer/core/context_features/context_feature_settings.h b/third_party/blink/renderer/core/context_features/context_feature_settings.h
index b0d65031f087c371de4be6b1f9ba2e7e5c879b2a..ad8c469cf5a6c168553a5cbd639766faaf1ec56c 100644
--- a/third_party/blink/renderer/core/context_features/context_feature_settings.h
+++ b/third_party/blink/renderer/core/context_features/context_feature_settings.h
@@ -44,6 +44,7 @@ class CORE_EXPORT ContextFeatureSettings final
   }
 
   void Trace(Visitor*) const override;
+  BRAVE_CONTEXT_FEATURE_SETTINGS_H_
 
  private:
   bool enable_mojo_js_ = false;
