diff --git a/third_party/blink/public/platform/web_content_settings_client.h b/third_party/blink/public/platform/web_content_settings_client.h
index d7da22cde33c3e16b4c5f3e6d724d500981ca03e..a56b70f190b0c146ec1180d643b935fb1afa2045 100644
--- a/third_party/blink/public/platform/web_content_settings_client.h
+++ b/third_party/blink/public/platform/web_content_settings_client.h
@@ -57,6 +57,11 @@ class WebContentSettingsClient {
     return enabled_per_settings;
   }
 
+  // Controls whether fingerprinting is allowed for this frame.
+  virtual bool AllowFingerprinting(bool enabled_per_settings) {
+    return enabled_per_settings;
+  }
+
   // Retrieves the client hints that should be attached to the request for the
   // given URL.
   virtual void GetAllowedClientHintsFromSource(const blink::WebURL& url,
