diff --git a/third_party/rust/serde/v1/BUILD.gn b/third_party/rust/serde/v1/BUILD.gn
index b9411a565e6d888306966da3740844ff875fa15b..74199f13a6b39b751a4a44e3b2156d205e5d3b90 100644
--- a/third_party/rust/serde/v1/BUILD.gn
+++ b/third_party/rust/serde/v1/BUILD.gn
@@ -53,6 +53,7 @@ cargo_crate("lib") {
     "serde_derive",
     "std",
   ]
+  features += [ "alloc", "rc" ]
   build_root =
       "//third_party/rust/chromium_crates_io/vendor/serde-1.0.193/build.rs"
   build_sources =
