diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index d95886012c43ab714c013c62e08475d05956a900..655ecec8c62fd3f5108fa05eb8ba94aca86e3eb1 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -126,6 +126,7 @@ component("core") {
 
   visibility = []  # Allow re-assignment of list.
   visibility = [ "//third_party/blink/*" ]
+  visibility += brave_blink_renderer_core_visibility
 
   # If you create a new subdirectory, make a new BUILD file for that directory
   # and reference it in the deps below rather than adding the sources here.
@@ -197,6 +198,7 @@ component("core") {
     "//url",
     "//v8",
   ]
+  public_deps += brave_blink_renderer_core_public_deps
   deps = [
     "//components/paint_preview/common",
     "//third_party/blink/public/common",
