diff --git a/third_party/crashpad/crashpad/snapshot/win/process_snapshot_win.cc b/third_party/crashpad/crashpad/snapshot/win/process_snapshot_win.cc
index 21df191a5d5a82e3ad815f6d67030f5acf10c87f..437c49baa3434da0d6415865c470b8739bb1e1d9 100644
--- a/third_party/crashpad/crashpad/snapshot/win/process_snapshot_win.cc
+++ b/third_party/crashpad/crashpad/snapshot/win/process_snapshot_win.cc
@@ -523,6 +523,7 @@ void ProcessSnapshotWin::AddMemorySnapshotForLdrLIST_ENTRY(
 
 WinVMSize ProcessSnapshotWin::DetermineSizeOfEnvironmentBlock(
     WinVMAddress start_of_environment_block) {
+  return 0;
   // https://blogs.msdn.microsoft.com/oldnewthing/20100203-00/?p=15083: On newer
   // OSs there's no stated limit, but in practice grabbing 32k characters should
   // be more than enough.
