diff --git a/third_party/blink/renderer/core/html/canvas/canvas_async_blob_creator.cc b/third_party/blink/renderer/core/html/canvas/canvas_async_blob_creator.cc
index ca17f562601b8fd8bf0c7a06a1af131ddb921e9d..df419f33fc623d7193345c5b58dfe3c87a2c8423 100644
--- a/third_party/blink/renderer/core/html/canvas/canvas_async_blob_creator.cc
+++ b/third_party/blink/renderer/core/html/canvas/canvas_async_blob_creator.cc
@@ -195,6 +195,7 @@ CanvasAsyncBlobCreator::CanvasAsyncBlobCreator(
   // We use pixmap to access the image pixels. Make the image unaccelerated if
   // necessary.
   image_ = image_->MakeUnaccelerated();
+  BRAVE_CANVAS_ASYNC_BLOB_CREATOR
 
   sk_sp<SkImage> skia_image =
       image_->PaintImageForCurrentFrame().GetSwSkImage();
