diff --git a/third_party/blink/public/devtools_protocol/browser_protocol.pdl b/third_party/blink/public/devtools_protocol/browser_protocol.pdl
index f5752bc3e32cd32ebbea69e03957f72215551b7d..7fc67cae2766c804828be1a6d2fd2fbfc4483111 100644
--- a/third_party/blink/public/devtools_protocol/browser_protocol.pdl
+++ b/third_party/blink/public/devtools_protocol/browser_protocol.pdl
@@ -7115,6 +7115,8 @@ domain Page
       web-share
       window-placement
       xr-spatial-tracking
+      ethereum
+      solana
 
   # Reason for a permissions policy feature to be disabled.
   experimental type PermissionsPolicyBlockReason extends string
