diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index 0c962124e7dfb868f10ae3dce73caf814137d415..7e141b2265e67c663396c34a400ec4a8e065ba0e 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -122,6 +122,7 @@ component("core") {
 
   visibility = []  # Allow re-assignment of list.
   visibility = [ "//third_party/blink/*" ]
+  visibility += brave_blink_renderer_core_visibility
 
   # If you create a new subdirectory, make a new BUILD file for that directory
   # and reference it in the deps below rather than adding the sources here.
@@ -194,6 +195,7 @@ component("core") {
     "//url",
     "//v8",
   ]
+  public_deps += brave_blink_renderer_core_public_deps
   deps = [
     "//components/paint_preview/common",
     "//third_party/blink/public/common",
