diff --git a/third_party/rust/rand_chacha/v0_3/BUILD.gn b/third_party/rust/rand_chacha/v0_3/BUILD.gn
index 3b68441d691b39ba5b4b3bb8d05cb939699dfcdf..2c9889447788737f24cb9aeff29e9685a2a7ec81 100644
--- a/third_party/rust/rand_chacha/v0_3/BUILD.gn
+++ b/third_party/rust/rand_chacha/v0_3/BUILD.gn
@@ -41,5 +41,4 @@ cargo_crate("lib") {
   # Only for usage from third-party crates. Add the crate to
   # third_party.toml to use it from first-party code.
   visibility = [ "//third_party/rust/*" ]
-  testonly = true
 }
