diff --git a/third_party/blink/common/permissions/permission_utils.cc b/third_party/blink/common/permissions/permission_utils.cc
index 8abbc34ec283562685a46e10142901110b5c0b55..daf04177f3487dbbd404f438d3b1aa2c0bd185ba 100644
--- a/third_party/blink/common/permissions/permission_utils.cc
+++ b/third_party/blink/common/permissions/permission_utils.cc
@@ -27,6 +27,7 @@ mojom::PermissionStatus ToPermissionStatus(const std::string& status) {
 
 std::string GetPermissionString(PermissionType permission) {
   switch (permission) {
+    PERMISSION_UTIL_GET_PERMISSION_STRING
     case PermissionType::GEOLOCATION:
       return "Geolocation";
     case PermissionType::NOTIFICATIONS:
@@ -249,6 +250,7 @@ std::optional<PermissionType> PermissionDescriptorInfoToPermissionType(
     bool clipboard_has_user_gesture,
     bool fullscreen_allow_without_user_gesture) {
   switch (name) {
+    BRAVE_PERMISSION_UTIL_PERMISSION_DESCRIPTOR_INFO_TO_PERMISSION_TYPE
     case PermissionName::GEOLOCATION:
       return PermissionType::GEOLOCATION;
     case PermissionName::NOTIFICATIONS:
