diff --git a/third_party/android_deps/BUILD.gn b/third_party/android_deps/BUILD.gn
index 76dc0a5bab42f24d2fa1dc738d744b06a2852fc4..ee36b3c357a9c5a301d30af1e0e5a38a6814493f 100644
--- a/third_party/android_deps/BUILD.gn
+++ b/third_party/android_deps/BUILD.gn
@@ -825,6 +825,7 @@ if (!limit_android_deps) {
         "$google_play_services_package:google_firebase_firebase_encoders_proto_java",
         "//third_party/androidx:androidx_annotation_annotation_java",
       ]
+      import("//brave/third_party/android_deps/sources.gni") visibility += brave_third_party_android_deps_visibility
     }
   }
 
