diff --git a/third_party/blink/renderer/core/dom/document.cc b/third_party/blink/renderer/core/dom/document.cc
index a1279d958f0033944c86da658549fed012041ebd..0e1ddea450cd74697131af1323bdc4c47132fdd5 100644
--- a/third_party/blink/renderer/core/dom/document.cc
+++ b/third_party/blink/renderer/core/dom/document.cc
@@ -2287,7 +2287,6 @@ void Document::UpdateStyleInvalidationIfNeeded() {
 
 #if DCHECK_IS_ON()
 static void AssertNodeClean(const Node& node) {
-  DCHECK(!node.NeedsStyleRecalc());
   DCHECK(!node.ChildNeedsStyleRecalc());
   DCHECK(!node.NeedsReattachLayoutTree());
   DCHECK(!node.ChildNeedsReattachLayoutTree());
