diff --git a/third_party/blink/public/web/web_local_frame.h b/third_party/blink/public/web/web_local_frame.h
index 6a7ab8ac43ed621c70fdf5a7053c374f5d601bc3..0e18bd553b9e7689b059a230177eb3e01e300b84 100644
--- a/third_party/blink/public/web/web_local_frame.h
+++ b/third_party/blink/public/web/web_local_frame.h
@@ -1006,6 +1006,8 @@ class BLINK_EXPORT WebLocalFrame : public WebFrame {
   virtual bool AllowStorageAccessSyncAndNotify(
       WebContentSettingsClient::StorageType storage_type) = 0;
 
+  virtual void SetOriginForClearWindowNameCheck(const url::Origin&) = 0;
+
  protected:
   explicit WebLocalFrame(mojom::TreeScopeType scope,
                          const LocalFrameToken& frame_token)
