diff --git a/third_party/rust/serde/v1/BUILD.gn b/third_party/rust/serde/v1/BUILD.gn
index 911bd0ed33e71c6c7c93cccebd5bb9ff108c0a87..c7cdd2a5ae699be0dc12ea26de9b5251203d9db6 100644
--- a/third_party/rust/serde/v1/BUILD.gn
+++ b/third_party/rust/serde/v1/BUILD.gn
@@ -58,4 +58,5 @@ cargo_crate("lib") {
   ]
   build_root = "crate/build.rs"
   build_sources = [ "crate/build.rs" ]
+  features += [ "alloc", "rc" ]
 }
