diff --git a/third_party/blink/public/web/web_remote_frame.h b/third_party/blink/public/web/web_remote_frame.h
index c684e484678c0989e1d47817f2ba385f4eb24c44..a8ed192ee2962ecddc45375537b76618e19d8361 100644
--- a/third_party/blink/public/web/web_remote_frame.h
+++ b/third_party/blink/public/web/web_remote_frame.h
@@ -16,6 +16,8 @@
 #include "ui/events/types/scroll_types.h"
 #include "v8/include/v8.h"
 
+#include "brave/third_party/blink/brave_page_graph/buildflags/buildflags.h"
+
 namespace cc {
 class Layer;
 }
@@ -130,6 +132,10 @@ class WebRemoteFrame : public WebFrame {
 
   virtual void DidStartLoading() = 0;
 
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  virtual void RegisterRemoteFrameCreatedForFrameOwner(const GURL& url) = 0;
+#endif
+
   // Returns true if this frame should be ignored during hittesting.
   virtual bool IsIgnoredForHitTest() const = 0;
 
