diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index d9760c6d4d287564e6939f2b250b2deff0cd5644..d4474a211e13724e008da69e4c3e0d59dd0b378b 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -1001,6 +1001,7 @@ void MediaDevices::DevicesEnumerated(
     }
   }
 
+  BRAVE_MEDIA_DEVICES_DEVICES_ENUMERATED
   RecordEnumeratedDevices(result_tracker->GetScriptState(), media_devices);
   // TODO(crbug.com/1395324): Remove this custom EnumerateDevices latency
   // tracking by reverting crrev.com/c/3944912/ once the
