diff --git a/third_party/blink/renderer/core/frame/reporting_observer.idl b/third_party/blink/renderer/core/frame/reporting_observer.idl
index 90d04d7f35828629d71124cdd0024baebd7d4fb3..094d82aa0d3ba6b70c112907c94edc49f1003506 100644
--- a/third_party/blink/renderer/core/frame/reporting_observer.idl
+++ b/third_party/blink/renderer/core/frame/reporting_observer.idl
@@ -7,6 +7,7 @@
 callback ReportingObserverCallback = void (sequence<Report> reports, ReportingObserver observer);
 
 [
+    ContextEnabled=ReportingObservers,
     ActiveScriptWrappable,
     Exposed=(Window,Worker)
 ] interface ReportingObserver {
