diff --git a/third_party/blink/renderer/core/dom/events/event_target.cc b/third_party/blink/renderer/core/dom/events/event_target.cc
index 606a43d2b5493e3f81c82d0a2396cbf3d7b63319..430f8a43c9bf24a70169a66da0a72b0c3567b9f1 100644
--- a/third_party/blink/renderer/core/dom/events/event_target.cc
+++ b/third_party/blink/renderer/core/dom/events/event_target.cc
@@ -478,6 +478,7 @@ bool EventTarget::AddEventListenerInternal(
       event_type, listener, options, &registered_listener);
   if (added) {
     CHECK(registered_listener);
+    BRAVE_EVENT_TARGET_ADD_EVENT_LISTENER_INTERNAL
     if (options->hasSignal()) {
       // Instead of passing the entire |options| here, which could create a
       // circular reference due to |options| holding a Member<AbortSignal>, just
@@ -638,6 +639,7 @@ bool EventTarget::RemoveEventListenerInternal(
                                     &registered_listener)) {
     return false;
   }
+  BRAVE_EVENT_TARGET_REMOVE_EVENT_LISTENER_INTERNAL
 
   CHECK(registered_listener);
   RemovedEventListener(event_type, *registered_listener);
