diff --git a/third_party/blink/renderer/core/frame/local_frame.h b/third_party/blink/renderer/core/frame/local_frame.h
index 648455136f6a70a6c6b9c93febec6f6071bd7447..cba31394e4a09ccb0767ce668224a93061ba1b10 100644
--- a/third_party/blink/renderer/core/frame/local_frame.h
+++ b/third_party/blink/renderer/core/frame/local_frame.h
@@ -193,7 +193,7 @@ class CORE_EXPORT LocalFrame final
     : public Frame,
       public FrameScheduler::Delegate,
       public BackForwardCacheLoaderHelperImpl::Delegate,
-      public Supplementable<LocalFrame, 5> {
+      public Supplementable<LocalFrame, 6> {
  public:
   enum class Supplements {
     kRemoteObjectGatewayImpl = 0,
@@ -201,6 +201,7 @@ class CORE_EXPORT LocalFrame final
     kImageDownloaderImpl = 2,
     kTextSuggestionBackendImpl = 3,
     kDevToolsFrontendImpl = 4
+    , kBravePageGraph,
   };
 
   // Returns the LocalFrame instance for the given |frame_token|.
