diff --git a/third_party/libaddressinput/chromium/chrome_metadata_source.cc b/third_party/libaddressinput/chromium/chrome_metadata_source.cc
index 1e63e01ae9e42053213e6cec5915c7690e071764..6f03446f65bb5b8a57bcc3a4564b14cfdd378a11 100644
--- a/third_party/libaddressinput/chromium/chrome_metadata_source.cc
+++ b/third_party/libaddressinput/chromium/chrome_metadata_source.cc
@@ -58,7 +58,7 @@ ChromeMetadataSource::Request::Request(
 void ChromeMetadataSource::Download(const std::string& key,
                                     const Callback& downloaded) {
   GURL resource(validation_data_url_ + key);
-  if (!resource.SchemeIsCryptographic()) {
+  if (1) { // feature disabled in Brave
     downloaded(false, key, NULL);
     return;
   }
