diff --git a/third_party/blink/renderer/bindings/scripts/web_idl/idl_compiler.py b/third_party/blink/renderer/bindings/scripts/web_idl/idl_compiler.py
index 513b7f8f18795a3f8a71a001dc1f2adde97e2957..f8381eadbc176d111b2fe17732763aafc00af1cb 100644
--- a/third_party/blink/renderer/bindings/scripts/web_idl/idl_compiler.py
+++ b/third_party/blink/renderer/bindings/scripts/web_idl/idl_compiler.py
@@ -1266,3 +1266,4 @@ class IdlCompiler(object):
     def _is_across_components(self, original_ir, extended_ir):
         return ("core" in original_ir.components
                 and "modules" in extended_ir.components)
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
