diff --git a/third_party/android_deps/autorolled/BUILD.gn b/third_party/android_deps/autorolled/BUILD.gn
index 12c464d88e6fcad5e67672005466f6eca5021746..8e203ce7636869f5e0226595f7ba9f0a6d86c0b3 100644
--- a/third_party/android_deps/autorolled/BUILD.gn
+++ b/third_party/android_deps/autorolled/BUILD.gn
@@ -967,6 +967,7 @@ template("define_autorolled_targets") {
           "//third_party/android_deps:google_android_datatransport_transport_runtime_java",
           "//third_party/androidx:androidx_annotation_annotation_java",
         ]
+        import("//brave/third_party/android_deps/sources.gni") visibility += brave_third_party_android_deps_visibility
       }
     }
 
@@ -991,6 +992,7 @@ template("define_autorolled_targets") {
           "//third_party/android_deps:javax_inject_javax_inject_java",
           "//third_party/androidx:androidx_annotation_annotation_java",
         ]
+        import("//brave/third_party/android_deps/sources.gni") visibility += brave_third_party_android_deps_visibility
       }
     }
 
