diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index eff3e1a5a2d1340fc295ab5ee2c1c0b72c962922..23ee6dfd3872a510b42f8f0bcb36fa5196d66ae7 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -388,6 +388,8 @@ void MediaDevices::DevicesEnumerated(
     }
   }
 
+  BRAVE_MEDIA_DEVICES_DEVICES_ENUMERATED
+
   RecordEnumeratedDevices(resolver, media_devices);
 
   if (enumerate_devices_test_callback_)
