diff --git a/third_party/blink/renderer/modules/BUILD.gn b/third_party/blink/renderer/modules/BUILD.gn
index 7fc5cd9c052b56b1907201bb8c310dd88f4e1062..977ac6e5051366f1beaccc7c51f1220ab558eee3 100644
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -19,6 +19,7 @@ if (is_ios) {
 }
 
 visibility = [ "//third_party/blink/renderer/*" ]
+visibility += brave_blink_renderer_modules_visibility
 
 config("modules_implementation") {
   defines = [ "BLINK_MODULES_IMPLEMENTATION=1" ]
@@ -178,6 +179,7 @@ component("modules") {
   # generating the snapshot for android, blink is compiled with
   # current_os="linux" and target_os="android". Using target_os is necessary as
   # we need to compile in the same way as would happen when current_os="android".
+  sub_modules += brave_blink_sub_modules
   if (target_os == "android") {
     sub_modules += [ "//third_party/blink/renderer/modules/remote_objects" ]
   } else {
