diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index 53563ef33604fd335b6d9aaa0483c2fdd725f2f5..e2031dc782c4b4032ea3413a808adfa9342c127d 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -1341,6 +1341,7 @@ void MediaDevices::DevicesEnumerated(
     }
   }
 
+  BRAVE_MEDIA_DEVICES_DEVICES_ENUMERATED
   RecordEnumeratedDevices(result_tracker->GetScriptState(), media_devices);
   result_tracker->Resolve(media_devices);
   tracer->End();
