diff --git a/third_party/blink/renderer/core/loader/mixed_content_checker.cc b/third_party/blink/renderer/core/loader/mixed_content_checker.cc
index 432bdb51d41e298294f48067e9b0f2f6351ab0f2..43dbed012334492a0d875b76522962b6b760d7c4 100644
--- a/third_party/blink/renderer/core/loader/mixed_content_checker.cc
+++ b/third_party/blink/renderer/core/loader/mixed_content_checker.cc
@@ -299,6 +299,7 @@ bool RequestIsSubframeSubresource(Frame* frame) {
 // static
 bool MixedContentChecker::IsMixedContent(const SecurityOrigin* security_origin,
                                          const KURL& url) {
+  BRAVE_MIXED_CONTENT_CHECKER_IS_MIXED_CONTENT
   return IsMixedContent(
       security_origin->GetOriginOrPrecursorOriginIfOpaque()->Protocol(), url);
 }
@@ -348,6 +349,7 @@ bool MixedContentChecker::IsMixedContentRestrictedInFrameContext(
               ->Protocol())) {
     return true;
   }
+  BRAVE_MIXED_CONTENT_CHECKER_IS_MIXED_CONTENT_RESTRICTED_IN_FRAME_CONTEXT
   return false;
 }
 
