diff --git a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
index 77b9dc2f4012ae8f63a2cf4d2f0a53f2b2389982..5919bdc07748b0c609cfe61fae247cb7d2611ab1 100644
--- a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
+++ b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
@@ -369,6 +369,7 @@ class CORE_EXPORT HTMLCanvasElement final
 
   void OnAxObjectCreated(bool is_ignored);
 
+  BRAVE_HTML_CANVAS_ELEMENT_H_
  protected:
   void DidMoveToNewDocument(Document& old_document) override;
   void DidRecalcStyle(const StyleRecalcChange change) override;
