diff --git a/third_party/blink/public/platform/web_content_settings_client.h b/third_party/blink/public/platform/web_content_settings_client.h
index 04a501dadac1b328137fb08f2a6011837337b2e5..6d013dca72b73db231a699b1e08d6f45e1b5e4f0 100644
--- a/third_party/blink/public/platform/web_content_settings_client.h
+++ b/third_party/blink/public/platform/web_content_settings_client.h
@@ -62,6 +62,11 @@ class WebContentSettingsClient {
     return enabled_per_settings;
   }
 
+  // Controls whether fingerprinting is allowed for this frame.
+  virtual bool AllowFingerprinting(bool enabled_per_settings) {
+    return enabled_per_settings;
+  }
+
   // Retrieves the client hints that should be attached to the request for the
   // given URL.
   virtual void GetAllowedClientHintsFromSource(const blink::WebURL& url,
