diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
index 0d526abe45fa3dc58874e1cbbb27ac3cbc8e1109..5f1a80c89cd3e7f40ab47391f8b4d8390490c33b 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
@@ -1617,6 +1617,7 @@ ImageData* BaseRenderingContext2D::getImageData(
 }
 
 ImageData* BaseRenderingContext2D::getImageDataInternal(
+    BRAVE_GET_IMAGE_DATA_PARAMS
     int sx,
     int sy,
     int sw,
@@ -1764,6 +1765,7 @@ ImageData* BaseRenderingContext2D::getImageDataInternal(
 
   // Read pixels into |contents|.
   if (snapshot) {
+    BRAVE_GET_IMAGE_DATA
     const bool read_pixels_successful =
         snapshot->PaintImageForCurrentFrame().readPixels(
             image_info, contents.Data(), image_info.minRowBytes(), sx, sy);
