diff --git a/third_party/blink/renderer/core/loader/modulescript/module_tree_linker.cc b/third_party/blink/renderer/core/loader/modulescript/module_tree_linker.cc
index 03d95aa708f754f4ca1e6b9b72bcb33c959240dc..cc70204e50976986315da02fe948f63ef7d2793f 100644
--- a/third_party/blink/renderer/core/loader/modulescript/module_tree_linker.cc
+++ b/third_party/blink/renderer/core/loader/modulescript/module_tree_linker.cc
@@ -455,6 +455,7 @@ void ModuleTreeLinker::FetchDescendants(const ModuleScript* module_script) {
         module_script->FetchOptions().GetReferrerPolicy(),
         mojom::blink::FetchPriorityHint::kAuto,
         RenderBlockingBehavior::kNonBlocking);
+    BRAVE_MODULE_TREE_LINKER_FETCH_DESCENDANTS
     // <spec step="8">... perform the internal module script graph fetching
     // procedure given moduleRequest, fetch client settings object, destination,
     // options, module script's settings object, visited set, and module
