diff --git a/third_party/rust/byteorder/v1/BUILD.gn b/third_party/rust/byteorder/v1/BUILD.gn
index 2ff8f05d8d77852a430729372715305323a600cb..9a0152f3d73af2baa9ee502c65ba4870c419ed91 100644
--- a/third_party/rust/byteorder/v1/BUILD.gn
+++ b/third_party/rust/byteorder/v1/BUILD.gn
@@ -41,4 +41,5 @@ cargo_crate("lib") {
   # //third_party/rust/chromium_crates_io/Cargo.toml to use
   # it from first-party code.
   visibility = [ "//third_party/rust/*" ]
+  features = [ "std", "i128" ]
 }
