diff --git a/third_party/blink/renderer/core/loader/mixed_content_checker.cc b/third_party/blink/renderer/core/loader/mixed_content_checker.cc
index 3ef8d6349860204999468ba07a47e549c5aac994..0b4795b108f7d893922da34c5d37eae99e4fe7e6 100644
--- a/third_party/blink/renderer/core/loader/mixed_content_checker.cc
+++ b/third_party/blink/renderer/core/loader/mixed_content_checker.cc
@@ -271,6 +271,7 @@ bool RequestIsSubframeSubresource(Frame* frame) {
 // static
 bool MixedContentChecker::IsMixedContent(const SecurityOrigin* security_origin,
                                          const KURL& url) {
+  BRAVE_MIXED_CONTENT_CHECKER_IS_MIXED_CONTENT
   return IsMixedContent(
       security_origin->GetOriginOrPrecursorOriginIfOpaque()->Protocol(), url);
 }
