diff --git a/third_party/blink/renderer/core/dom/document.cc b/third_party/blink/renderer/core/dom/document.cc
index 80e2277cde1d5a8c53d3f941ecb0e8399e99be10..61c15872a735ab912b2d4ea4acfe93046e0760aa 100644
--- a/third_party/blink/renderer/core/dom/document.cc
+++ b/third_party/blink/renderer/core/dom/document.cc
@@ -7588,6 +7588,7 @@ KURL Document::CompleteURLWithOverride(
   if (result.IsEmpty()) {
     result = Encoding().IsValid() ? KURL(base_url_override, url, Encoding())
                                   : KURL(base_url_override, url);
+    result = RewriteBraveToChrome(result);
     url_cache_.Put(base_url_override, url, result);
   }
   // If the conditions are met for
