diff --git a/third_party/blink/renderer/core/loader/resource/image_resource_content.cc b/third_party/blink/renderer/core/loader/resource/image_resource_content.cc
index ac259cb798a0d3e44be2eb8c7415e52f8244f2e5..92562381b0c846be0792539b33d5a22e839cabe0 100644
--- a/third_party/blink/renderer/core/loader/resource/image_resource_content.cc
+++ b/third_party/blink/renderer/core/loader/resource/image_resource_content.cc
@@ -26,6 +26,12 @@
 #include "third_party/blink/renderer/platform/wtf/vector.h"
 #include "v8/include/v8.h"
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+namespace brave_page_graph {
+using InspectorId = uint64_t;
+}
+// #endif
+
 namespace blink {
 
 namespace {
@@ -105,6 +111,18 @@ ImageResourceContent* ImageResourceContent::Fetch(FetchParameters& params,
   return resource->GetContent();
 }
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+ImageResourceContent* ImageResourceContent::Fetch(FetchParameters& params,
+                                                  ResourceFetcher* fetcher,
+                                                  brave_page_graph::InspectorId* identifier) {
+  ImageResource* resource = ImageResource::Fetch(params, fetcher);
+  if (!resource)
+    return nullptr;
+  *identifier = resource->InspectorId();
+  return resource->GetContent();
+}
+// #endif
+
 void ImageResourceContent::SetImageResourceInfo(ImageResourceInfo* info) {
   info_ = info;
 }
