diff --git a/third_party/crashpad/crashpad/util/net/http_transport_win.cc b/third_party/crashpad/crashpad/util/net/http_transport_win.cc
index 6bc2481d7aca413a552869a991e2c5775adeded1..60302d906531b2d73733c006f7ab952f1745ad22 100644
--- a/third_party/crashpad/crashpad/util/net/http_transport_win.cc
+++ b/third_party/crashpad/crashpad/util/net/http_transport_win.cc
@@ -158,6 +158,8 @@ bool HTTPTransportWin::ExecuteSynchronously(std::string* response_body) {
     return false;
   }
 
+  BraveSetSessionOptions(session.get());
+
   int timeout_in_ms = static_cast<int>(timeout() * 1000);
   if (!WinHttpSetTimeouts(session.get(),
                           timeout_in_ms,
