diff --git a/third_party/blink/public/mojom/permissions_policy/permissions_policy_feature.mojom b/third_party/blink/public/mojom/permissions_policy/permissions_policy_feature.mojom
index b67a6788c18f448c2d6282a089f14ecfd4933c7f..155396830cd1dbfd9895ecb295915193edf4ab52 100644
--- a/third_party/blink/public/mojom/permissions_policy/permissions_policy_feature.mojom
+++ b/third_party/blink/public/mojom/permissions_policy/permissions_policy_feature.mojom
@@ -211,6 +211,7 @@ enum PermissionsPolicyFeature {
   // Controls access to Bluetooth API.
   kBluetooth = 101,
 
+  kEthereum = 4096, kSolana = 4097,
 
   // Don't change assigned numbers of any item, and don't reuse removed slots.
   // Add new features at the end of the enum.
