diff --git a/third_party/blink/common/BUILD.gn b/third_party/blink/common/BUILD.gn
index 6f48104544022bee7c8290b832ff54ed51cb26be..1486328265fc78f1e544bed72f63cc4d0b446f79 100644
--- a/third_party/blink/common/BUILD.gn
+++ b/third_party/blink/common/BUILD.gn
@@ -354,6 +354,7 @@ source_set("common") {
   if (is_chromeos || is_android || is_win || is_castos || is_fuchsia) {
     defines = [ "ENABLE_PROTECTED_MEDIA_IDENTIFIER_PERMISSION" ]
   }
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_blink_common_deps
 }
 
 test("blink_common_unittests") {
@@ -502,6 +503,7 @@ source_set("common_unittests_sources") {
   # TODO(crbug.com/40031409): Fix code that adds exit-time destructors and
   # enable the diagnostic by removing this line.
   configs += [ "//build/config/compiler:no_exit_time_destructors" ]
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_blink_common_deps
 }
 
 group("common_unittests_data") {
