diff --git a/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc b/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc
index 7867261df845542d88e778653e4abc678f1726cf..26846d1c91bae159dcbe7c77cb3aa118cf167727 100644
--- a/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc
+++ b/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc
@@ -316,7 +316,7 @@ int GetMicrotasksScopeDepth(v8::Isolate* isolate,
 
 }  // namespace
 
-v8::MaybeLocal<v8::Script> V8ScriptRunner::CompileScript(
+v8::MaybeLocal<v8::Script> V8ScriptRunner::CompileScript_ChromiumImpl(
     ScriptState* script_state,
     const ClassicScript& classic_script,
     v8::ScriptOrigin origin,
@@ -362,7 +362,7 @@ v8::MaybeLocal<v8::Script> V8ScriptRunner::CompileScript(
   return script;
 }
 
-v8::MaybeLocal<v8::Module> V8ScriptRunner::CompileModule(
+v8::MaybeLocal<v8::Module> V8ScriptRunner::CompileModule_ChromiumImpl(
     v8::Isolate* isolate,
     const ModuleScriptCreationParams& params,
     const TextPosition& start_position,
