diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index 6fd98e6aa33c4dc51411936994e2ec3f9fe050e5..faebf46b3262a7947290d04fa2776c80b4e36447 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -359,6 +359,7 @@ component("core") {
       "Carbon.framework",
     ]
   }
+  visibility += brave_blink_renderer_core_visibility public_deps += brave_blink_renderer_core_public_deps sources += brave_blink_renderer_core_sources deps += brave_blink_renderer_core_deps
 }
 
 # Certain sources should be compiled with heavy optimization enabled.
