diff --git a/third_party/blink/renderer/core/frame/cached_permission_status.cc b/third_party/blink/renderer/core/frame/cached_permission_status.cc
index 2da5ef102e6c124b4cb76bf4ff5d383e6539bf57..2e17cf2829fa77e34d5d4b5c53e093aa2edfd5ce 100644
--- a/third_party/blink/renderer/core/frame/cached_permission_status.cc
+++ b/third_party/blink/renderer/core/frame/cached_permission_status.cc
@@ -16,7 +16,6 @@ using mojom::blink::PermissionDescriptorPtr;
 using mojom::blink::PermissionName;
 using mojom::blink::PermissionObserver;
 using mojom::blink::PermissionService;
-using mojom::blink::PermissionStatus;
 
 // static
 const char CachedPermissionStatus::kSupplementName[] = "CachedPermissionStatus";
@@ -53,7 +52,7 @@ void CachedPermissionStatus::Trace(Visitor* visitor) const {
 void CachedPermissionStatus::RegisterClient(
     Client* client,
     const Vector<PermissionDescriptorPtr>& permissions) {
-  HashMap<mojom::blink::PermissionName, mojom::blink::PermissionStatus>
+  HashMap<mojom::blink::PermissionName, PermissionStatus>
       initialized_map;
   for (const PermissionDescriptorPtr& descriptor : permissions) {
     auto status_it = permission_status_map_.find(descriptor->name);
