diff --git a/third_party/blink/renderer/core/script/script_loader.cc b/third_party/blink/renderer/core/script/script_loader.cc
index d19b7c08c4b83d9be6fc7cbdabd623e566b6dde9..dba7e932bbdcc7799c298f49eb43f3d77274ad34 100644
--- a/third_party/blink/renderer/core/script/script_loader.cc
+++ b/third_party/blink/renderer/core/script/script_loader.cc
@@ -734,6 +734,7 @@ PendingScript* ScriptLoader::PrepareScript(
   ScriptFetchOptions options(nonce, integrity_metadata, integrity_attr,
                              parser_state, credentials_mode, referrer_policy,
                              fetch_priority_hint, render_blocking_behavior);
+  BRAVE_SCRIPT_LOADER_PREPARE_SCRIPT
 
   // <spec step="30">Let settings object be el's node document's relevant
   // settings object.</spec>
