diff --git a/third_party/blink/renderer/bindings/idl_in_modules.gni b/third_party/blink/renderer/bindings/idl_in_modules.gni
index 4359c993593a9cb7da087de5e8a67f19fa8952cd..c89cae9cb81b54e802bf51048ce1f0b819ad1676 100644
--- a/third_party/blink/renderer/bindings/idl_in_modules.gni
+++ b/third_party/blink/renderer/bindings/idl_in_modules.gni
@@ -1143,6 +1143,7 @@ static_idl_files_in_modules = get_path_info(
           "//third_party/blink/renderer/modules/xr/xr_webgl_layer_init.idl",
         ],
         "abspath")
+static_idl_files_in_modules += get_path_info(brave_static_idl_files_in_modules, "abspath")
 
 # Direct Sockets
 # This uses target_os rather than current_os (which is what is_android is set
