diff --git a/third_party/blink/renderer/modules/scheduler/scheduled_action.cc b/third_party/blink/renderer/modules/scheduler/scheduled_action.cc
index ac30a53f748e69894f3d3fbc6e0acf2ee846ab45..dd77f2f2c13ad8a9acf3ad7621e29b0ce14ae089 100644
--- a/third_party/blink/renderer/modules/scheduler/scheduled_action.cc
+++ b/third_party/blink/renderer/modules/scheduler/scheduled_action.cc
@@ -91,6 +91,7 @@ ScheduledAction::ScheduledAction(ScriptState* script_state,
   } else {
     UseCounter::Count(target, WebFeature::kScheduledActionIgnored);
   }
+  BRAVE_SCHEDULED_ACTION_STRING_HANDLER_CONSTRUCTOR
 }
 
 ScheduledAction::~ScheduledAction() {
