diff --git a/third_party/blink/renderer/modules/modules_idl_files.gni b/third_party/blink/renderer/modules/modules_idl_files.gni
index 4050c62353344b0b019905ecd32bd4f985369642..385d0f34f18f48193d52718e9ce0b11c8fb66328 100644
--- a/third_party/blink/renderer/modules/modules_idl_files.gni
+++ b/third_party/blink/renderer/modules/modules_idl_files.gni
@@ -133,6 +133,7 @@ _idl_imports = [
 if (!is_android) {
   _idl_imports += [ "//third_party/blink/renderer/modules/serial/idls.gni" ]
 }
+_idl_imports += brave_idl_imports
 
 # Do not add IDL files directly to these lists. See _idl_imports above instead
 # on how to add IDL files for a module.
