diff --git a/third_party/blink/renderer/core/frame/navigator_concurrent_hardware.idl b/third_party/blink/renderer/core/frame/navigator_concurrent_hardware.idl
index 5cc78bbd3276c7798b5bbc019a8df470cfcc40e5..03c10b0c3145665db4bd9717b76edb263c9e7b4c 100644
--- a/third_party/blink/renderer/core/frame/navigator_concurrent_hardware.idl
+++ b/third_party/blink/renderer/core/frame/navigator_concurrent_hardware.idl
@@ -5,5 +5,5 @@
 // https://html.spec.whatwg.org/C/#navigator.hardwareconcurrency
 
 interface mixin NavigatorConcurrentHardware {
-    [HighEntropy=Direct, MeasureAs=NavigatorHardwareConcurrency] readonly attribute unsigned long long hardwareConcurrency;
+  [CallWith=ScriptState, HighEntropy=Direct, MeasureAs=NavigatorHardwareConcurrency] readonly attribute unsigned long long hardwareConcurrency;
 };
