diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index f6da067b16964757686863e75b3f522ace1f0172..742c4c45be3a75403f597cc4d422af91c39a45d3 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -512,6 +512,8 @@ void MediaDevices::DevicesEnumerated(
     }
   }
 
+  BRAVE_MEDIA_DEVICES_DEVICES_ENUMERATED
+
   RecordEnumeratedDevices(resolver, media_devices);
 
   if (enumerate_devices_test_callback_)
