diff --git a/third_party/blink/renderer/bindings/scripts/bind_gen/codegen_utils.py b/third_party/blink/renderer/bindings/scripts/bind_gen/codegen_utils.py
index 7f16ff4dcc2b031e9e67b8bcf2cacdfe4e4ef7d5..c63ee7e9c3df07ff74603a850059a47499d64b59 100644
--- a/third_party/blink/renderer/bindings/scripts/bind_gen/codegen_utils.py
+++ b/third_party/blink/renderer/bindings/scripts/bind_gen/codegen_utils.py
@@ -28,6 +28,23 @@ def make_copyright_header():
 """)
 
 
+def make_const_string_definitions(accumulator):
+    assert isinstance(accumulator, CodeGenAccumulator)
+
+    class ConstStringDefinitions(object):
+        def __init__(self, accumulator):
+            self._accumulator = accumulator
+
+        def __str__(self):
+            return "\n".join([
+                "static const char* const {} = \"{}\";".format(name, value)
+                for (name, value) in sorted(self._accumulator.const_strings)
+            ])
+
+    return LiteralNode(ConstStringDefinitions(accumulator))
+
+
+
 def make_forward_declarations(accumulator):
     assert isinstance(accumulator, CodeGenAccumulator)
 
