diff --git a/third_party/blink/renderer/platform/exported/web_canonical_cookie.cc b/third_party/blink/renderer/platform/exported/web_canonical_cookie.cc
index ca867911f08dbb1344ad63eccc24e5abbb57e965..6414ca6788b18c783f0959c4f10869c0570be1ea 100644
--- a/third_party/blink/renderer/platform/exported/web_canonical_cookie.cc
+++ b/third_party/blink/renderer/platform/exported/web_canonical_cookie.cc
@@ -104,6 +104,7 @@ base::Optional<WebCanonicalCookie> WebCanonicalCookie::Create(
     const WebString& cookie_line,
     base::Time creation_time) {
   std::unique_ptr<net::CanonicalCookie> cookie = net::CanonicalCookie::Create(
+      BRAVE_WEBCANONICALCOOKIE_CREATE
       ToGURL(url), cookie_line.Utf8(), creation_time,
       base::nullopt /* server_time */);
   if (!cookie)
