diff --git a/third_party/blink/renderer/core/frame/dom_window.h b/third_party/blink/renderer/core/frame/dom_window.h
index 7265b7668420c7ccd5e491af8ab8c3799bd8f458..b2baa3994a1b4627ad8d9e1cbd94e5e26d0edf86 100644
--- a/third_party/blink/renderer/core/frame/dom_window.h
+++ b/third_party/blink/renderer/core/frame/dom_window.h
@@ -152,6 +152,7 @@ class CORE_EXPORT DOMWindow : public EventTargetWithInlineData {
   // marked as "CrossOrigin" in the window.idl.
   void ReportCoopAccess(const char* property_name);
 
+  BRAVE_DOM_WINDOW_H
  protected:
   explicit DOMWindow(Frame&);
 
