diff --git a/third_party/rust/num_bigint/v0_4/BUILD.gn b/third_party/rust/num_bigint/v0_4/BUILD.gn
index 62a0907853856474bed252fcc44021bd2a94e84d..56fd73e2e69999d9b66ca288a8716d57d7655fa7 100644
--- a/third_party/rust/num_bigint/v0_4/BUILD.gn
+++ b/third_party/rust/num_bigint/v0_4/BUILD.gn
@@ -53,7 +53,7 @@ cargo_crate("lib") {
   cargo_pkg_repository = "https://github.com/rust-num/num-bigint"
   cargo_pkg_version = "0.4.6"
 
-  allow_unsafe = false
+  allow_unsafe = true
 
   deps = [
     "//third_party/rust/num_integer/v0_1:lib",
