diff --git a/third_party/blink/renderer/core/context_features/context_feature_settings.h b/third_party/blink/renderer/core/context_features/context_feature_settings.h
index 9c9a289262c01daa45e24d4cb553b7d90797f7b8..de4e8ced39169f817f6c22842013557198630137 100644
--- a/third_party/blink/renderer/core/context_features/context_feature_settings.h
+++ b/third_party/blink/renderer/core/context_features/context_feature_settings.h
@@ -35,6 +35,7 @@ class CORE_EXPORT ContextFeatureSettings final
   bool isMojoJSEnabled() const { return enable_mojo_js_; }
 
   void Trace(Visitor*) const override;
+  BRAVE_CONTEXT_FEATURE_SETTINGS_H_
 
  private:
   bool enable_mojo_js_ = false;
