diff --git a/third_party/rust/bitflags/v2/BUILD.gn b/third_party/rust/bitflags/v2/BUILD.gn
index f52c4678d14fa0b3b50610f1ee0fb59338189c7e..58a1ac318f707ed5eb10d4987d0123160c6e95d5 100644
--- a/third_party/rust/bitflags/v2/BUILD.gn
+++ b/third_party/rust/bitflags/v2/BUILD.gn
@@ -67,4 +67,5 @@ cargo_crate("lib") {
   executable_configs += [ "//build/config/compiler:no_chromium_code" ]
   proc_macro_configs -= [ "//build/config/compiler:chromium_code" ]
   proc_macro_configs += [ "//build/config/compiler:no_chromium_code" ]
+  deps = [ "//third_party/rust/serde/v1:lib" ] features = [ "serde" ]
 }
