diff --git a/third_party/blink/renderer/core/dom/document.cc b/third_party/blink/renderer/core/dom/document.cc
index 4478cae5917c9ebb83e77a186437dbcac1531bba..cbcee3b9246f4490bd20c1f4beb03db040ba4d04 100644
--- a/third_party/blink/renderer/core/dom/document.cc
+++ b/third_party/blink/renderer/core/dom/document.cc
@@ -33,7 +33,7 @@
 // instead of including more headers. If that is infeasible, adjust the limit.
 // For more info, see
 // https://chromium.googlesource.com/chromium/src/+/HEAD/docs/wmax_tokens.md
-#pragma clang max_tokens_here 950000
+#pragma clang max_tokens_here 2097152
 
 #include <memory>
 #include <utility>
