diff --git a/third_party/blink/renderer/core/style/style_fetched_image.h b/third_party/blink/renderer/core/style/style_fetched_image.h
index 843c35c61a1641573f06892b4af66cc8bf75c2b9..ae15501772d2692ad712c8c43da9aa936d0f14bf 100644
--- a/third_party/blink/renderer/core/style/style_fetched_image.h
+++ b/third_party/blink/renderer/core/style/style_fetched_image.h
@@ -29,6 +29,7 @@
 #include "third_party/blink/renderer/platform/weborigin/kurl.h"
 #include "third_party/blink/renderer/platform/wtf/casting.h"
 
+#include "brave/third_party/blink/brave_page_graph/types.h"
 namespace blink {
 
 class Document;
@@ -41,6 +42,7 @@ class StyleFetchedImage final : public StyleImage,
 
  public:
   StyleFetchedImage(ImageResourceContent* image,
+                    brave_page_graph::InspectorId inspector_id,
                     const Document&,
                     bool is_lazyload_possibly_deferred,
                     bool origin_clean,
@@ -91,6 +93,7 @@ class StyleFetchedImage final : public StyleImage,
   Member<const Document> document_;
   const KURL url_;
   const bool origin_clean_;
+  brave_page_graph::InspectorId inspector_id_;
 
   // Whether this was created by an ad-related CSSParserContext.
   const bool is_ad_related_;
