diff --git a/third_party/blink/renderer/core/script/dynamic_module_resolver.cc b/third_party/blink/renderer/core/script/dynamic_module_resolver.cc
index 9ebe091714ebb31631ce36da27e4a7b87ec32201..e37392d187a4bb19b07b3c1ff4917d5aacbc083d 100644
--- a/third_party/blink/renderer/core/script/dynamic_module_resolver.cc
+++ b/third_party/blink/renderer/core/script/dynamic_module_resolver.cc
@@ -314,6 +314,7 @@ void DynamicModuleResolver::ResolveDynamically(
       referrer_info.CredentialsMode(), referrer_info.GetReferrerPolicy(),
       mojom::blink::FetchPriorityHint::kAuto,
       RenderBlockingBehavior::kNonBlocking);
+  BRAVE_DYNAMIC_MODULE_RESOLVER_RESOLVE_DYNAMICALLY
 
   // <spec label="fetch-an-import()-module-script-graph" step="3">Fetch a single
   // module script given url, settings object, "script", options, settings
