diff --git a/third_party/blink/renderer/modules/BUILD.gn b/third_party/blink/renderer/modules/BUILD.gn
index 62c46ca6d4fcbe6db668ac6a63579b711c695edd..3585eb15402999e764e90de69c9a81117e1fa4be 100644
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -16,6 +16,7 @@ if (is_android) {
 }
 
 visibility = [ "//third_party/blink/renderer/*" ]
+visibility += brave_blink_renderer_modules_visibility
 
 config("modules_implementation") {
   defines = [ "BLINK_MODULES_IMPLEMENTATION=1" ]
@@ -164,6 +165,7 @@ component("modules") {
     "//third_party/blink/renderer/modules/xr",
   ]
 
+  sub_modules += brave_blink_sub_modules
   if (is_android) {
     sub_modules += [ "//third_party/blink/renderer/modules/remote_objects" ]
   }
