diff --git a/third_party/blink/renderer/modules/donottrack/navigator_do_not_track.cc b/third_party/blink/renderer/modules/donottrack/navigator_do_not_track.cc
index 26e258b726aa1b1b48e8d04003c0661bb755da1f..f5e4f7115eddd7e0e4565b115c1e28de9e42054c 100644
--- a/third_party/blink/renderer/modules/donottrack/navigator_do_not_track.cc
+++ b/third_party/blink/renderer/modules/donottrack/navigator_do_not_track.cc
@@ -34,6 +34,10 @@
 #include "third_party/blink/renderer/core/frame/local_frame_client.h"
 #include "third_party/blink/renderer/core/frame/navigator.h"
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+#include "brave/third_party/blink/brave_page_graph/page_graph.h"
+// #endif
+
 namespace blink {
 
 NavigatorDoNotTrack::NavigatorDoNotTrack(Navigator& navigator)
@@ -63,6 +67,23 @@ String NavigatorDoNotTrack::doNotTrack() {
   LocalFrame* frame = GetSupplementable()->GetFrame();
   if (!frame || !frame->Client())
     return String();
+
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  {
+    String result = frame->Client()->DoNotTrackValue();
+
+    brave_page_graph::PageGraph* page_graph =
+      frame->GetDocument()->GetPageGraph();
+    if (page_graph != nullptr) {
+      page_graph->RegisterWebAPICall("Navigator.doNotTrack",
+        std::vector<const String>());
+      page_graph->RegisterWebAPIResult("Navigator.doNotTrack", result);
+    }
+
+    return result;
+  }
+// #endif
+
   return frame->Client()->DoNotTrackValue();
 }
 
