diff --git a/third_party/blink/renderer/core/html/media/autoplay_policy.cc b/third_party/blink/renderer/core/html/media/autoplay_policy.cc
index e0413d62cc3c17e67f2bdc4d9b772813463838c9..bc424b6b561153bc8ffcbc2bee0af8aa3bace9a7 100644
--- a/third_party/blink/renderer/core/html/media/autoplay_policy.cc
+++ b/third_party/blink/renderer/core/html/media/autoplay_policy.cc
@@ -63,6 +63,7 @@ bool ComputeLockPendingUserGestureRequired(const Document& document) {
 // static
 AutoplayPolicy::Type AutoplayPolicy::GetAutoplayPolicyForDocument(
     const Document& document) {
+  BRAVE_GET_AUTOPLAY_POLICY_FOR_DOCUMENT
   if (!document.GetSettings())
     return Type::kNoUserGestureRequired;
 
@@ -310,6 +311,7 @@ void AutoplayPolicy::TryUnlockingUserGesture() {
 bool AutoplayPolicy::IsGestureNeededForPlayback() const {
   if (!IsLockedPendingUserGesture())
     return false;
+  BRAVE_AUTOPLAY_POLICY_IS_GESTURE_NEEDED_FOR_PLAYBACK
 
   // We want to allow muted video to autoplay if the element is allowed to
   // autoplay muted.
