diff --git a/third_party/blink/renderer/core/html/media/autoplay_policy.cc b/third_party/blink/renderer/core/html/media/autoplay_policy.cc
index c992fddf3552022839e2fd7c1d46d5c131e5da95..82a754428de28c17be3acd306413ce4db981e1d2 100644
--- a/third_party/blink/renderer/core/html/media/autoplay_policy.cc
+++ b/third_party/blink/renderer/core/html/media/autoplay_policy.cc
@@ -64,6 +64,7 @@ bool ComputeLockPendingUserGestureRequired(const Document& document) {
 // static
 AutoplayPolicy::Type AutoplayPolicy::GetAutoplayPolicyForDocument(
     const Document& document) {
+  BRAVE_GET_AUTOPLAY_POLICY_FOR_DOCUMENT
   if (!document.GetSettings())
     return Type::kNoUserGestureRequired;
 
@@ -311,6 +312,7 @@ void AutoplayPolicy::TryUnlockingUserGesture() {
 bool AutoplayPolicy::IsGestureNeededForPlayback() const {
   if (!IsLockedPendingUserGesture())
     return false;
+  BRAVE_AUTOPLAY_POLICY_IS_GESTURE_NEEDED_FOR_PLAYBACK
 
   // We want to allow muted video to autoplay if the element is allowed to
   // autoplay muted.
