diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
index 8940f363564d134f3c86c80a6bf6b965fbc3f3a7..d101968a55fbc040efc5434ea6454e55e5c2c05b 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
@@ -1814,6 +1814,7 @@ ImageData* BaseRenderingContext2D::getImageData(
 }
 
 ImageData* BaseRenderingContext2D::getImageDataInternal(
+    BRAVE_GET_IMAGE_DATA_PARAMS
     int sx,
     int sy,
     int sw,
@@ -1997,6 +1998,7 @@ ImageData* BaseRenderingContext2D::getImageDataInternal(
     default:
       NOTREACHED();
   }
+  BRAVE_GET_IMAGE_DATA
   ImageData* image_data = ImageData::ValidateAndCreate(
       sw, sh, data_array, image_data_settings, exception_state,
       ImageData::Context2DErrorMode);
