diff --git a/third_party/rust/anyhow/v1/BUILD.gn b/third_party/rust/anyhow/v1/BUILD.gn
index db07bb9536cf2497a28c32d61eb060e48699afa7..6562a086351b70f09b8907ae4b8fad23abedd99e 100644
--- a/third_party/rust/anyhow/v1/BUILD.gn
+++ b/third_party/rust/anyhow/v1/BUILD.gn
@@ -48,5 +48,4 @@ cargo_crate("lib") {
   build_sources =
       [ "//third_party/rust/chromium_crates_io/vendor/anyhow-1.0.87/build.rs" ]
 
-  testonly = true
 }
