diff --git a/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc b/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
index fcaabbb69c024240d37c056c36b874c0e6cb3a36..43532320baf4ac14ef6e903e1d20324e45affb96 100644
--- a/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
+++ b/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
@@ -197,6 +197,7 @@ void DOMPluginArray::UpdatePluginData() {
     }
     return;
   }
+  BRAVE_DOM_PLUGINS_UPDATE_PLUGIN_DATA__RESET_PLUGIN_DATA
   PluginData* data = GetPluginData();
   if (!data) {
     dom_plugins_.clear();
@@ -218,6 +219,7 @@ void DOMPluginArray::UpdatePluginData() {
       }
     }
   }
+  BRAVE_DOM_PLUGINS_UPDATE_PLUGIN_DATA__FARBLE_PLUGIN_DATA
 }
 
 void DOMPluginArray::ContextDestroyed() {
