diff --git a/third_party/blink/renderer/core/editing/commands/clipboard_commands.cc b/third_party/blink/renderer/core/editing/commands/clipboard_commands.cc
index 05d2b0b9610e0fce63baa4165cfa860aa55f7962..63b6fe3006a6cb649836965f246fed9dbe05ce4f 100644
--- a/third_party/blink/renderer/core/editing/commands/clipboard_commands.cc
+++ b/third_party/blink/renderer/core/editing/commands/clipboard_commands.cc
@@ -379,6 +379,7 @@ bool ClipboardCommands::ExecuteCopy(LocalFrame& frame,
       frame.Selection().IsHidden())
     return true;
 
+  BRAVE_CLIPBOARD_COMMANDS_CHECK_SOURCE
   if (EnclosingTextControl(
           frame.Selection().ComputeVisibleSelectionInDOMTree().Start())) {
     frame.GetSystemClipboard()->WritePlainText(frame.SelectedTextForClipboard(),
