diff --git a/third_party/blink/renderer/platform/scheduler/test/fake_task_runner.h b/third_party/blink/renderer/platform/scheduler/test/fake_task_runner.h
index 06257a927140c1ffd122a4fce925990e59d373da..97570910d8cde38631068a8233e5c9377d674605 100644
--- a/third_party/blink/renderer/platform/scheduler/test/fake_task_runner.h
+++ b/third_party/blink/renderer/platform/scheduler/test/fake_task_runner.h
@@ -42,6 +42,8 @@ class FakeTaskRunner : public base::SingleThreadTaskRunner {
   Deque<PendingTask> TakePendingTasksForTesting();
 
  protected:
+  ~FakeTaskRunner() override;
+
   bool PostDelayedTask(const base::Location& location,
                        base::OnceClosure task,
                        base::TimeDelta delay) override;
@@ -55,8 +57,6 @@ class FakeTaskRunner : public base::SingleThreadTaskRunner {
                                   base::TimeDelta delay) override;
 
  private:
-  ~FakeTaskRunner() override;
-
   class Data;
   class BaseTaskRunner;
   scoped_refptr<Data> data_;
