diff --git a/third_party/rust/rand_core/v0_6/BUILD.gn b/third_party/rust/rand_core/v0_6/BUILD.gn
index ea79a8a8c4936050d81aab3ab698b4fc05b45a28..a8d6f5a93122e46bf1ae4559448ff5d4fc2a43e5 100644
--- a/third_party/rust/rand_core/v0_6/BUILD.gn
+++ b/third_party/rust/rand_core/v0_6/BUILD.gn
@@ -52,5 +52,4 @@ cargo_crate("lib") {
   # //third_party/rust/chromium_crates_io/Cargo.toml to use
   # it from first-party code.
   visibility = [ "//third_party/rust/*" ]
-  testonly = true
 }
