diff --git a/third_party/rust/rustc_version/v0_4/BUILD.gn b/third_party/rust/rustc_version/v0_4/BUILD.gn
index 780f06036e9c4e3a268904e7618f7f36de5636bc..29df097be75a592b645ab20089ab04fb959e1b85 100644
--- a/third_party/rust/rustc_version/v0_4/BUILD.gn
+++ b/third_party/rust/rustc_version/v0_4/BUILD.gn
@@ -32,5 +32,4 @@ cargo_crate("buildrs_support") {
   # Only for usage from third-party crates. Add the crate to
   # third_party.toml to use it from first-party code.
   visibility = [ "//third_party/rust/*" ]
-  testonly = true
 }
