diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index c895ccd73ae4111c1e16d1a95c67fd2ec0adb9b1..2e6d9b303a5167b508fbe345a77a825c64009241 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -319,6 +319,7 @@
     {
       // Interest Group JS API/runtimeflag.
       name: "AdInterestGroupAPI",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
       origin_trial_feature_name: "AdInterestGroupAPI",
       implied_by: ["Fledge", "Parakeet"],
@@ -429,6 +430,7 @@
     },
     {
       name: "AIPromptAPI",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       public: true,
       status: {
         "Win": "stable",
@@ -456,6 +458,7 @@
     },
     {
       name: "AIPromptAPIMultimodalInput",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: {
         "Win": "stable",
         "Mac": "stable",
@@ -510,7 +513,7 @@
       origin_trial_feature_name: "AIProofreaderAPI",
       origin_trial_os: ["win", "mac", "linux", "chromeos"],
       origin_trial_allows_third_party: true,
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -525,7 +528,7 @@
       origin_trial_feature_name: "AIRewriterAPI",
       origin_trial_os: ["win", "mac", "linux", "chromeos"],
       origin_trial_allows_third_party: true,
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -534,6 +537,7 @@
     },
     {
       name: "AISummarizationAPI",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: {
         "Win": "stable",
         "Mac": "stable",
@@ -568,7 +572,7 @@
       origin_trial_feature_name: "AIWriterAPI",
       origin_trial_os: ["win", "mac", "linux", "chromeos"],
       origin_trial_allows_third_party: true,
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -1376,7 +1380,7 @@
       // objects that live on the compositor thread.
       // [1] https://drafts.csswg.org/css-animations-2/#timeline-triggers
       name: "CompositorTimelineTrigger",
-      status: "stable",
+      status: "experimental", // Brave: revert when crbug.com/541924666 is fixed.
     },
     {
       name: "CompressionDictionaryTransport",
@@ -1448,6 +1452,7 @@
       // https://github.com/WICG/controlled-frame/blob/main/README.md for more
       // info.
       name: "ControlledFrame",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       public: true,
       status: "stable",
     },
@@ -2919,7 +2924,8 @@
     },
     {
       name: "FencedFrames",
-      base_feature: "none",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
+      base_feature: "FencedFramesRuntime",
       // This helps enable and expose the <fencedframe> element, but note that
       // blink::features::kFencedFrames must be enabled as well, as we require
       // the support of the browser process to fully enable the feature.
@@ -3008,6 +3014,7 @@
     {
       // In-development features for the File System Access API.
       name: "FileSystemAccessAPIExperimental",
+      base_feature: "none",
       status: "experimental",
     },
     {
@@ -3022,6 +3029,7 @@
     {
       // Non-OPFS File System Access API.
       name: "FileSystemAccessLocal",
+      base_feature: "none",
       status: {"default": "stable"},
     },
     {
@@ -3147,6 +3155,7 @@
     },
     {
       name: "Fledge",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
       public: true,
     },
@@ -3906,7 +3915,7 @@
         "Android": "test",
         "default": "",
       },
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -4274,6 +4283,7 @@
     // "experimental" is the support on other platforms.
     {
       name: "MiddleClickAutoscroll",
+      base_feature: "none",
       status: "test",
     },
     // Killswitch for crbug.com/40062462 fix.
@@ -4773,6 +4783,7 @@
     {
       // PARAKEET ad serving runtime flag/JS API.
       name: "Parakeet",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       origin_trial_feature_name: "Parakeet",
     },
     {
@@ -4984,6 +4995,7 @@
       //
       // It also has some feature params defined throughout the codebase.
       name: "Prerender2",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
     },
     {
@@ -5181,6 +5193,7 @@
     // but we still keep this flag for future phase tests.
     {
       name: "ReduceUserAgentMinorVersion",
+      base_feature_status: "enabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
     },
     {
@@ -6617,7 +6630,7 @@
         "ChromeOS": "stable",
         "default": "",
       },
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -6789,6 +6802,7 @@
     },
     {
       name: "UserMediaElement",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
       public: true,
     },
@@ -7637,5 +7651,10 @@
       // crbug.com/435623334.
       name: "XSLTSpecialTrial",
     },
+    {
+      name: "BraveIsInTorContext",
+      public: true,
+      base_feature: "none",
+    },
   ],
 }
