diff --git a/third_party/blink/renderer/modules/modules_idl_files.gni b/third_party/blink/renderer/modules/modules_idl_files.gni
index 5323a7e9f2142d3ef2213956f2508a978d790751..4106aa2c4826b28bdf475ab1b7386f6ac2c8f053 100644
--- a/third_party/blink/renderer/modules/modules_idl_files.gni
+++ b/third_party/blink/renderer/modules/modules_idl_files.gni
@@ -152,6 +152,7 @@ if (!is_android) {
     "//third_party/blink/renderer/modules/serial/idls.gni",
   ]
 }
+_idl_imports += brave_idl_imports
 
 # Do not add IDL files directly to these lists. See _idl_imports above instead
 # on how to add IDL files for a module.
