diff --git a/third_party/android_deps/BUILD.gn b/third_party/android_deps/BUILD.gn
index 9bba687e9c49e86374f8389ed1ec49acfbe97266..51d1a1a2308c499507904d908218afa96a2b400d 100644
--- a/third_party/android_deps/BUILD.gn
+++ b/third_party/android_deps/BUILD.gn
@@ -464,6 +464,7 @@ java_prebuilt("net_ltgt_gradle_incap_incap_java") {
     ":*",
     "//third_party/androidx:*",
   ]
+  visibility += [ "//brave/third_party/android_deps:com_google_android_datatransport_transport_runtime_java__classes" ] supports_android = true
 }
 
 # This is generated, do not edit. Update BuildConfigGenerator.groovy instead.
