diff --git a/third_party/blink/renderer/core/frame/cached_permission_status.cc b/third_party/blink/renderer/core/frame/cached_permission_status.cc
index d17de7af08e0f54673ffd781f47ce5e891836525..d0496902ce765c8f696fe232e98ea153a495121d 100644
--- a/third_party/blink/renderer/core/frame/cached_permission_status.cc
+++ b/third_party/blink/renderer/core/frame/cached_permission_status.cc
@@ -15,7 +15,6 @@ using mojom::blink::PermissionDescriptorPtr;
 using mojom::blink::PermissionName;
 using mojom::blink::PermissionObserver;
 using mojom::blink::PermissionService;
-using mojom::blink::PermissionStatus;
 
 // static
 CachedPermissionStatus* CachedPermissionStatus::From(LocalDOMWindow* window) {
@@ -47,7 +46,7 @@ void CachedPermissionStatus::Trace(Visitor* visitor) const {
 void CachedPermissionStatus::RegisterClient(
     Client* client,
     const Vector<PermissionDescriptorPtr>& permissions) {
-  HashMap<mojom::blink::PermissionName, mojom::blink::PermissionStatus>
+  HashMap<mojom::blink::PermissionName, PermissionStatus>
       initialized_map;
   for (const PermissionDescriptorPtr& descriptor : permissions) {
     auto status_it = permission_status_map_.find(descriptor->name);
