diff --git a/third_party/blink/renderer/core/layout/ng/ng_container_fragment_builder.cc b/third_party/blink/renderer/core/layout/ng/ng_container_fragment_builder.cc
index e62ccbc378e139b7360a55c1e2f8d77f7ee5790d..5b40b775fde0d2a3e982a822fa8ebbb6a2e85ce2 100644
--- a/third_party/blink/renderer/core/layout/ng/ng_container_fragment_builder.cc
+++ b/third_party/blink/renderer/core/layout/ng/ng_container_fragment_builder.cc
@@ -200,7 +200,7 @@ NGContainerFragmentBuilder::AddOutOfFlowChildCandidate(
     const LogicalOffset& child_offset,
     base::Optional<TextDirection> container_direction) {
   DCHECK(child);
-  DCHECK(layout_object_ && !layout_object_->IsLayoutInline() ||
+  DCHECK((layout_object_ && !layout_object_->IsLayoutInline()) ||
          container_direction)
       << "container_direction must only be set for inline-level OOF children.";
 
