diff --git a/third_party/blink/renderer/core/html/media/autoplay_policy.cc b/third_party/blink/renderer/core/html/media/autoplay_policy.cc
index b876f630328ea337384d7d2456e95e95434ce422..9a88b818f9db1d49c968a43a6ad8436f52b3fb47 100644
--- a/third_party/blink/renderer/core/html/media/autoplay_policy.cc
+++ b/third_party/blink/renderer/core/html/media/autoplay_policy.cc
@@ -66,6 +66,7 @@ bool ComputeLockPendingUserGestureRequired(const Document& document) {
 // static
 AutoplayPolicy::Type AutoplayPolicy::GetAutoplayPolicyForDocument(
     const Document& document) {
+  BRAVE_GET_AUTOPLAY_POLICY_FOR_DOCUMENT
   if (!document.GetSettings())
     return Type::kNoUserGestureRequired;
 
@@ -348,6 +349,7 @@ void AutoplayPolicy::TryUnlockingUserGesture() {
 bool AutoplayPolicy::IsGestureNeededForPlayback() const {
   if (!IsLockedPendingUserGesture())
     return false;
+  BRAVE_AUTOPLAY_POLICY_IS_GESTURE_NEEDED_FOR_PLAYBACK
 
   // We want to allow muted video to autoplay if the element is allowed to
   // autoplay muted.
