diff --git a/third_party/blink/renderer/modules/BUILD.gn b/third_party/blink/renderer/modules/BUILD.gn
index 86a4c9f94670e41a5ae545862ff21a7e4f7a4f4f..487d2f798ac2e4f2333a97957c1d728d24353e9e 100644
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -20,6 +20,7 @@ if (is_ios) {
 }
 
 visibility = [ "//third_party/blink/renderer/*" ]
+visibility += brave_blink_renderer_modules_visibility
 
 config("modules_implementation") {
   defines = [ "BLINK_MODULES_IMPLEMENTATION=1" ]
@@ -174,6 +175,7 @@ component("modules") {
   # generating the snapshot for android, blink is compiled with
   # current_os="linux" and target_os="android". Using target_os is necessary as
   # we need to compile in the same way as would happen when current_os="android".
+  sub_modules += brave_blink_sub_modules
   if (target_os_is_android) {
     sub_modules += [ "//third_party/blink/renderer/modules/remote_objects" ]
   } else {
