diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index fef22a645f64c74aa168116a7c67cf9316b5dda6..96b3a2acde85d63fbfe655d0de9ef82c93288dfc 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -172,6 +172,7 @@ component("core") {
 
   visibility = []  # Allow re-assignment of list.
   visibility = [ "//third_party/blink/*" ]
+  visibility += brave_blink_renderer_core_visibility
 
   sources = [
     "core_initializer.cc",
@@ -270,6 +271,7 @@ component("core") {
     "//v8",
   ]
 
+  public_deps += brave_blink_renderer_core_public_deps
   deps = [
     "//build:chromeos_buildflags",
     "//components/paint_preview/common",
