diff --git a/third_party/blink/renderer/core/html/html_link_element.cc b/third_party/blink/renderer/core/html/html_link_element.cc
index 9f96ddbb156d1320ad322b667d5fcca51aa15d6c..592d0ae85f981f334b9cab6266341d2dd0ebd522 100644
--- a/third_party/blink/renderer/core/html/html_link_element.cc
+++ b/third_party/blink/renderer/core/html/html_link_element.cc
@@ -83,7 +83,9 @@ HTMLLinkElement::HTMLLinkElement(Document& document,
                                              html_names::kResourcesAttr)),
       scopes_(
           MakeGarbageCollected<DOMTokenList>(*this, html_names::kScopesAttr)),
-      created_by_parser_(flags.IsCreatedByParser()) {}
+      created_by_parser_(flags.IsCreatedByParser()) {
+  link_loader_->SetHTMLLinkElementOwner(this);
+}
 
 HTMLLinkElement::~HTMLLinkElement() = default;
 
