diff --git a/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc b/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc
index c288ca532d5328408e0fc784dd81783451ae2ce2..1e4b64bcb108c2ee7702fa300c4b4552fa384ffb 100644
--- a/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc
+++ b/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc
@@ -310,7 +310,7 @@ int GetMicrotasksScopeDepth(v8::Isolate* isolate,
 
 }  // namespace
 
-v8::MaybeLocal<v8::Script> V8ScriptRunner::CompileScript(
+v8::MaybeLocal<v8::Script> V8ScriptRunner::CompileScript_ChromiumImpl(
     ScriptState* script_state,
     const ClassicScript& classic_script,
     v8::ScriptOrigin origin,
@@ -356,7 +356,7 @@ v8::MaybeLocal<v8::Script> V8ScriptRunner::CompileScript(
   return script;
 }
 
-v8::MaybeLocal<v8::Module> V8ScriptRunner::CompileModule(
+v8::MaybeLocal<v8::Module> V8ScriptRunner::CompileModule_ChromiumImpl(
     v8::Isolate* isolate,
     const ModuleScriptCreationParams& params,
     const TextPosition& start_position,
