diff --git a/third_party/androidx/BUILD.gn b/third_party/androidx/BUILD.gn
index 60306dd2bb6509a271f083e8fcfcb7ba3f167ca8..b60e26c87407162731615a7f69b5222b11f0f25e 100644
--- a/third_party/androidx/BUILD.gn
+++ b/third_party/androidx/BUILD.gn
@@ -751,6 +751,7 @@ if (!limit_android_deps) {
     info_path = "../androidx/committed/libs/androidx_fragment_fragment/androidx_fragment_fragment.info"
     enable_bytecode_checks = false
     deps = [
+      ":androidx_activity_activity_java",
       ":androidx_annotation_annotation_experimental_java",
       ":androidx_annotation_annotation_java",
       ":androidx_arch_core_core_common_java",
