diff --git a/third_party/blink/renderer/modules/modules_idl_files.gni b/third_party/blink/renderer/modules/modules_idl_files.gni
index f6dae1fa5263f07051235e80d47a3c6fd0e2d30c..ec0028dc1d86074637f026b2f931a9a3e63b554b 100644
--- a/third_party/blink/renderer/modules/modules_idl_files.gni
+++ b/third_party/blink/renderer/modules/modules_idl_files.gni
@@ -556,6 +556,7 @@ modules_idl_files =
         ],
         "abspath")
 
+modules_idl_files += brave_modules_idl_files
 if (!is_android) {
   modules_idl_files += get_path_info([
                                        "serial/serial.idl",
@@ -1051,6 +1052,7 @@ modules_dependency_idl_files =
           "xr/navigator_xr.idl",
         ],
         "abspath")
+modules_dependency_idl_files += brave_modules_dependency_idl_files
 
 if (!is_android) {
   modules_dependency_idl_files +=
