diff --git a/third_party/blink/renderer/core/frame/local_frame.cc b/third_party/blink/renderer/core/frame/local_frame.cc
index 5ecfabc35eea65e17bcb13509186ac5d1db61021..81910b3a98bb9e0a1fdf9244d8db6a45a3491e59 100644
--- a/third_party/blink/renderer/core/frame/local_frame.cc
+++ b/third_party/blink/renderer/core/frame/local_frame.cc
@@ -536,6 +536,7 @@ void LocalFrame::Trace(Visitor* visitor) const {
   visitor->Trace(remote_object_gateway_factory_impl_);
   visitor->Trace(remote_object_gateway_impl_);
   visitor->Trace(text_suggestion_backend_impl_);
+  visitor->Trace(page_graph_);
   Frame::Trace(visitor);
   visitor->Trace(dev_tools_frontend_impl_);
 }
