diff --git a/third_party/blink/renderer/core/script_tools/model_context.cc b/third_party/blink/renderer/core/script_tools/model_context.cc
index 86fab626900eec3989449fc57f371a6ac6deca94..70a3478a496da3fac9e6cfbc476934643b3b8e1d 100644
--- a/third_party/blink/renderer/core/script_tools/model_context.cc
+++ b/third_party/blink/renderer/core/script_tools/model_context.cc
@@ -898,6 +898,9 @@ void ModelContext::PauseExecution() {
 }
 
 void ModelContext::NotifyToolChange() {
+  if (auto* web_frame = WebLocalFrameImpl::FromFrame(document_->GetFrame())) {
+    web_frame->NotifyScriptToolsChanged();
+  }
   DispatchEvent(*Event::Create(event_type_names::kToolchange));
 }
 
@@ -965,7 +968,11 @@ bool ModelContext::IsModelContextAllowed() const {
               url::kFileScheme &&
           key.IsUniversalFileAgent()) ||
          (document_->GetSettings() &&
-          !document_->GetSettings()->GetWebSecurityEnabled());
+          !document_->GetSettings()->GetWebSecurityEnabled()) ||
+         (document_->GetExecutionContext()->GetSecurityOrigin()->Protocol() ==
+              "chrome-untrusted" &&
+           document_->GetExecutionContext()->GetSecurityOrigin()->Host().ends_with(
+              ai_chat::kAIChatLeoWorkspaceUIHostSuffix));
 }
 
 ScriptPromise<IDLSequence<RegisteredTool>> ModelContext::getTools(
