diff --git a/third_party/blink/renderer/core/frame/location.h b/third_party/blink/renderer/core/frame/location.h
index 64013637445944e07468f6a6aa326594704889e2..9ad7f72e593f60219ac38b96055c8bcd3a0d8762 100644
--- a/third_party/blink/renderer/core/frame/location.h
+++ b/third_party/blink/renderer/core/frame/location.h
@@ -37,6 +37,8 @@
 #include "third_party/blink/renderer/platform/bindings/script_wrappable.h"
 #include "third_party/blink/renderer/platform/wtf/text/wtf_string.h"
 
+#include "brave/third_party/blink/brave_page_graph/types.h"
+#include "base/json/json_string_value_serializer.h"
 namespace blink {
 
 class Document;
@@ -116,6 +121,10 @@ class CORE_EXPORT Location final : public ScriptWrappable {
   const KURL& Url() const;
 
   const Member<DOMWindow> dom_window_;
+  // Internal helper method for PageGraph WebAPI recording
+  void PageGraphLogWebAPI(const brave_page_graph::WebAPI web_api,
+                          const String& value,
+                          bool is_set) const;
 };
 
 }  // namespace blink
