diff --git a/third_party/blink/renderer/core/css/offscreen_font_selector.h b/third_party/blink/renderer/core/css/offscreen_font_selector.h
index 630ad8fe392fda9f13b8b071f8f6da3fc80bd15e..67dff8bb4e1108868dfb5f3193920988da3a568f 100644
--- a/third_party/blink/renderer/core/css/offscreen_font_selector.h
+++ b/third_party/blink/renderer/core/css/offscreen_font_selector.h
@@ -90,6 +90,8 @@ class CORE_EXPORT OffscreenFontSelector : public FontSelector {
       const FontDescription&,
       const AtomicString& passed_family) override;
 
+  bool AllowFontFamily(const AtomicString& family) override;
+
   ExecutionContext* GetExecutionContext() const override {
     return worker_ ? worker_->GetExecutionContext() : nullptr;
   }
