diff --git a/third_party/blink/public/BUILD.gn b/third_party/blink/public/BUILD.gn
index 0c0cf1e5a9073bed1c4f1e1eea1ad85ec33a1075..4ead6f81a7d4c340d1c9900f2857d48ddfda1235 100644
--- a/third_party/blink/public/BUILD.gn
+++ b/third_party/blink/public/BUILD.gn
@@ -673,6 +673,7 @@ mojom("mojo_bindings") {
   visibility_blink =
       [ "//third_party/blink/renderer/platform:blink_platform_public_deps" ]
   sources = [
+    "../../../brave/third_party/blink/public/platform/brave_drm.mojom",
     "platform/modules/bluetooth/web_bluetooth.mojom",
   ]
   public_deps = [
