diff --git a/third_party/blink/renderer/modules/BUILD.gn b/third_party/blink/renderer/modules/BUILD.gn
index 30dcfb2b62cef2d2bed178fac002ac1f93f7924a..48d1853cf0bb61d178a6a3f9645cf18c7f99b6f1 100644
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -188,6 +188,7 @@ target("jumbo_" + modules_target_type, "modules") {
     "//third_party/webrtc_overrides:task_queue_factory",
     "//third_party/zlib",
   ]
+  deps += [ "//brave/third_party/blink/renderer" ]
 
   public_deps = sub_modules
   allow_circular_includes_from = sub_modules
