diff --git a/third_party/blink/common/BUILD.gn b/third_party/blink/common/BUILD.gn
index 30a57ed5834f9cc12fcf0a0cca71f39e7777d160..23feeb78530e68217c455fa9ecde8a64455f80d6 100644
--- a/third_party/blink/common/BUILD.gn
+++ b/third_party/blink/common/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/third_party/blink/common/sources.gni")
 import("//build/config/features.gni")
 import("//build/config/rust.gni")
 import("//build/nocompile.gni")
@@ -330,6 +331,7 @@ source_set("common") {
     "//ui/events/mojom:event_latency_metadata_mojom",
     "//ui/latency/mojom:shared_mojom_traits",
   ]
+  deps += brave_blink_common_deps
 
   # If use_blink is used then it needs to depend on //media
   if (use_blink) {
@@ -467,6 +469,7 @@ source_set("common_unittests_sources") {
     "//v8",
     "//v8:v8_libplatform",
   ]
+  deps += brave_blink_common_deps
 
   if (enable_nocompile_tests) {
     deps += [ ":blink_common_nocompile_tests" ]
