diff --git a/third_party/blink/renderer/core/loader/modulescript/document_module_script_fetcher.h b/third_party/blink/renderer/core/loader/modulescript/document_module_script_fetcher.h
index 8dd12f8bbe0ef976dcd81f6b269c195ab5894815..6b47c3c37d6d8c807877ce91edc2ae9a2d72ff38 100644
--- a/third_party/blink/renderer/core/loader/modulescript/document_module_script_fetcher.h
+++ b/third_party/blink/renderer/core/loader/modulescript/document_module_script_fetcher.h
@@ -25,6 +25,7 @@ class CORE_EXPORT DocumentModuleScriptFetcher final
   void Fetch(FetchParameters&,
              ModuleType,
              ResourceFetcher*,
+             const Modulator*,
              ModuleGraphLevel,
              Client*) override;
 
