diff --git a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
index d3d91e0abda87768ec38c81d9b90fb38e53f5f1b..2ea2de126d19a0498162992df90a9aeebaf145f5 100644
--- a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
+++ b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
@@ -390,6 +390,7 @@ class CORE_EXPORT HTMLCanvasElement final
     UpdateMemoryUsage();
   }
 
+  BRAVE_HTML_CANVAS_ELEMENT_H_
  protected:
   void DidMoveToNewDocument(Document& old_document) override;
   void DidRecalcStyle(const StyleRecalcChange change) override;
