diff --git a/third_party/blink/renderer/core/script/script_loader.cc b/third_party/blink/renderer/core/script/script_loader.cc
index 564cee6cae248658146bf756cf8e38aa5f1061a0..ac709b651e0280867a7cd25ddc0eca2ef2717f96 100644
--- a/third_party/blink/renderer/core/script/script_loader.cc
+++ b/third_party/blink/renderer/core/script/script_loader.cc
@@ -722,6 +722,7 @@ PendingScript* ScriptLoader::PrepareScript(
                              parser_state, credentials_mode, referrer_policy,
                              fetch_priority_hint, render_blocking_behavior,
                              RejectCoepUnsafeNone(false));
+  BRAVE_SCRIPT_LOADER_PREPARE_SCRIPT
 
   // <spec step="28">Let settings object be el's node document's relevant
   // settings object.</spec>
