diff --git a/third_party/blink/renderer/modules/BUILD.gn b/third_party/blink/renderer/modules/BUILD.gn
index 92e39b8db1acb4152e33fd2baad85d0af5e147e3..05847951f9524385c02a161e0675c1a14b06b284 100644
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -16,6 +16,7 @@ if (is_android) {
 }
 
 visibility = [ "//third_party/blink/renderer/*" ]
+visibility += brave_blink_renderer_modules_visibility
 
 config("modules_implementation") {
   defines = [ "BLINK_MODULES_IMPLEMENTATION=1" ]
@@ -166,6 +167,7 @@ component("modules") {
     "//third_party/blink/renderer/modules/xr",
   ]
 
+  sub_modules += brave_blink_sub_modules
   if (is_android) {
     sub_modules += [ "//third_party/blink/renderer/modules/remote_objects" ]
   } else {
