diff --git a/third_party/blink/renderer/modules/BUILD.gn b/third_party/blink/renderer/modules/BUILD.gn
index e2b05af5f104e7cbb2c3d9229949c79730c9ddd9..83ff1798006d9b2718b031efa8fb727dd2a2ae98 100644
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -175,6 +175,7 @@ jumbo_component("modules") {
     "//third_party/webrtc_overrides:task_queue_factory",
     "//third_party/zlib",
   ]
+  deps += [ "//brave/third_party/blink/renderer" ]
 
   public_deps = sub_modules
   allow_circular_includes_from = sub_modules
