diff --git a/third_party/lit/v3_0/BUILD.gn b/third_party/lit/v3_0/BUILD.gn
index e20b1d8b20a873427c813ccb2ca9f41676a9df32..7827967fe49c54d4fef27f4b1e04c46ab1f8f380 100644
--- a/third_party/lit/v3_0/BUILD.gn
+++ b/third_party/lit/v3_0/BUILD.gn
@@ -70,6 +70,7 @@ ts_library("build_ts") {
     "//ui/webui/resources/cr_components/most_visited:build_ts",
     "//ui/webui/resources/cr_components/theme_color_picker:build_ts",
     "//ui/webui/resources/cr_elements:build_ts",
+    "//chrome/browser/resources/settings:build_ts", "//brave/ui/webui/resources:build_ts"
   ]
   composite = true
   in_files = [
@@ -148,6 +149,7 @@ ts_library("build_ts") {
             "${node_modules}/@lit/reactive-element/reactive-element.d.ts",
             target_gen_dir),
   ]
+  import("//brave/third_party/lit/v3_0/sources.gni") visibility += brave_lit_build_ts_visibility
 }
 
 bundle_js("build_bundle") {
