diff --git a/third_party/blink/renderer/modules/modules_idl_files.gni b/third_party/blink/renderer/modules/modules_idl_files.gni
index 49501396fc5ba70b81ac103c1720f7f775c876cb..7b3bab95ea23f2fb7cd8140842163be0f7b4b493 100644
--- a/third_party/blink/renderer/modules/modules_idl_files.gni
+++ b/third_party/blink/renderer/modules/modules_idl_files.gni
@@ -135,6 +135,7 @@ _idl_imports = [
 if (!is_android) {
   _idl_imports += [ "//third_party/blink/renderer/modules/serial/idls.gni" ]
 }
+_idl_imports += brave_idl_imports
 
 # Do not add IDL files directly to these lists. See _idl_imports above instead
 # on how to add IDL files for a module.
