diff --git a/third_party/widevine/cdm/BUILD.gn b/third_party/widevine/cdm/BUILD.gn
index 0ab3bb63b0da0398cef54ed1316e12c7d8f3587b..7b2ba8a4e3caaba9002a5ec035b151dbbb80e155 100644
--- a/third_party/widevine/cdm/BUILD.gn
+++ b/third_party/widevine/cdm/BUILD.gn
@@ -110,6 +110,10 @@ if (widevine_cdm_binary_files != []) {
       "//build/config:exe_and_shlib_deps",
     ]
 
+    if (is_mac) {
+      ldflags = [ "-Wl,-install_name,@loader_path/libwidevinecdm.dylib" ]
+    }
+
     if (is_posix && !is_mac) {
       cflags = [ "-fvisibility=hidden" ]
     }
