diff --git a/third_party/blink/renderer/core/script/script_loader.cc b/third_party/blink/renderer/core/script/script_loader.cc
index 259c500e51c3f541957448a6158b6837d3814f37..df13a1aae49b2a2c0e0294e9a18c1a32e1b13889 100644
--- a/third_party/blink/renderer/core/script/script_loader.cc
+++ b/third_party/blink/renderer/core/script/script_loader.cc
@@ -734,6 +734,7 @@ PendingScript* ScriptLoader::PrepareScript(
   ScriptFetchOptions options(nonce, integrity_metadata, integrity_attr,
                              parser_state, credentials_mode, referrer_policy,
                              fetch_priority_hint, render_blocking_behavior);
+  BRAVE_SCRIPT_LOADER_PREPARE_SCRIPT
 
   // <spec step="30">Let settings object be el's node document's relevant
   // settings object.</spec>
