diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index f76eee2766478c6800ad8ffa4295c4236129d56b..103e9f388e65df2ed0743174d7a4dfe0b0756e2a 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -408,6 +408,8 @@ void MediaDevices::DevicesEnumerated(
     }
   }
 
+  BRAVE_MEDIA_DEVICES_DEVICES_ENUMERATED
+
   RecordEnumeratedDevices(resolver, media_devices);
 
   if (enumerate_devices_test_callback_)
