diff --git a/third_party/blink/renderer/core/script_tools/model_context.cc b/third_party/blink/renderer/core/script_tools/model_context.cc
index ee5fd7eecba5c5b1a8123f39007d7b81fcb3e6da..c925ded6d3f348beeca4517cb6d9c640f45be60a 100644
--- a/third_party/blink/renderer/core/script_tools/model_context.cc
+++ b/third_party/blink/renderer/core/script_tools/model_context.cc
@@ -877,6 +877,9 @@ void ModelContext::PauseExecution() {
 }
 
 void ModelContext::NotifyToolChange() {
+  if (auto* web_frame = WebLocalFrameImpl::FromFrame(document_->GetFrame())) {
+    web_frame->NotifyScriptToolsChanged();
+  }
   DispatchEvent(*Event::Create(event_type_names::kToolchange));
 }
 
