diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
index 53abce6c5b1987cd7be5a9464b771f37a4199aae..3ae760db0f879e153912d161ed1ada8bab7c209b 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
@@ -1637,6 +1637,7 @@ ImageData* BaseRenderingContext2D::getImageData(
 }

 ImageData* BaseRenderingContext2D::getImageDataInternal(
+    BRAVE_GET_IMAGE_DATA_PARAMS
     int sx,
     int sy,
     int sw,
@@ -1780,6 +1781,7 @@ ImageData* BaseRenderingContext2D::getImageDataInternal(

   // Read pixels into |contents|.
   if (snapshot) {
+    BRAVE_GET_IMAGE_DATA
     const bool read_pixels_successful =
         snapshot->PaintImageForCurrentFrame().readPixels(
             image_info, contents.Data(), image_info.minRowBytes(), sx, sy);
