diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
index 074bb39bb45d850993022204081d3b3e06ad64e7..dddcd8880782b903759613222ff19957228e9011 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
@@ -1550,6 +1550,7 @@ ImageData* BaseRenderingContext2D::createImageData(
 }
 
 ImageData* BaseRenderingContext2D::getImageData(
+    ScriptState* script_state,
     int sx,
     int sy,
     int sw,
@@ -1627,6 +1628,7 @@ ImageData* BaseRenderingContext2D::getImageData(
     return nullptr;
   }
 
+  BRAVE_GET_IMAGE_DATA
   bool may_have_stray_area =
       IsAccelerated()  // GPU readback may fail silently.
       || StaticBitmapImage::MayHaveStrayArea(snapshot, image_data_rect);
