diff --git a/third_party/blink/renderer/core/frame/reporting_observer.idl b/third_party/blink/renderer/core/frame/reporting_observer.idl
index 03017d16102e63567eff11c1b5066ebff899b023..5ea94e97d108d69477814eb0cf9bed1e04c15461 100644
--- a/third_party/blink/renderer/core/frame/reporting_observer.idl
+++ b/third_party/blink/renderer/core/frame/reporting_observer.idl
@@ -7,6 +7,7 @@
 callback ReportingObserverCallback = void (sequence<Report> reports, ReportingObserver observer);
 
 [
+    ContextEnabled=ReportingObservers,
     ActiveScriptWrappable,
     Exposed=(Window,Worker)
 ] interface ReportingObserver {
