diff --git a/third_party/blink/renderer/core/loader/mixed_content_checker.cc b/third_party/blink/renderer/core/loader/mixed_content_checker.cc
index 8bc524b89bac2ce4846edd8815c9fc8c8b802349..a0d4a14f3f5743c5d32bcd97b42ab291b30ab9de 100644
--- a/third_party/blink/renderer/core/loader/mixed_content_checker.cc
+++ b/third_party/blink/renderer/core/loader/mixed_content_checker.cc
@@ -299,6 +299,7 @@ bool RequestIsSubframeSubresource(Frame* frame) {
 // static
 bool MixedContentChecker::IsMixedContent(const SecurityOrigin* security_origin,
                                          const KURL& url) {
+  BRAVE_MIXED_CONTENT_CHECKER_IS_MIXED_CONTENT
   return IsMixedContent(
       security_origin->GetOriginOrPrecursorOriginIfOpaque()->Protocol(), url);
 }
@@ -348,6 +349,7 @@ bool MixedContentChecker::IsMixedContentRestrictedInFrameContext(
               ->Protocol())) {
     return true;
   }
+  BRAVE_MIXED_CONTENT_CHECKER_IS_MIXED_CONTENT_RESTRICTED_IN_FRAME_CONTEXT
   return false;
 }
 
