diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index a25e99fac5f5a171becff833464cdb3dfe7522c2..4a33f86496d7bb884586bd46f5cee91201fce4bd 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -768,6 +768,11 @@
       public: true,
       status: "experimental",
     },
+    {
+      name: "BraveIsInTorContext",
+      public: true,
+      base_feature: "none",
+    },
     {
       name: "BrowserInitiatedAutomaticPictureInPicture",
       public: true,
@@ -2274,6 +2279,7 @@
       // In-development features for the File System Access API.
       name: "FileSystemAccessAPIExperimental",
       status: "experimental",
+      base_feature: "none",
     },
     {
       // The FileSystemHandle.getCloudIdentifiers() method (see
@@ -2288,6 +2294,7 @@
       // Non-OPFS File System Access API.
       name: "FileSystemAccessLocal",
       status: {"default": "stable"},
+      base_feature: "none",
     },
     {
       name: "FileSystemAccessLockingScheme",
@@ -3257,6 +3264,7 @@
     {
       name: "MiddleClickAutoscroll",
       status: "test",
+      base_feature: "none",
     },
     // Killswitch for crbug.com/40062462 fix.
     {
