diff --git a/third_party/blink/renderer/modules/cookie_store/cookie_store.cc b/third_party/blink/renderer/modules/cookie_store/cookie_store.cc
index fc2f5c8ecd2eacf6c27473acb809657adec95b63..70a914162d72402c80197125a9d4ad52e50c96de 100644
--- a/third_party/blink/renderer/modules/cookie_store/cookie_store.cc
+++ b/third_party/blink/renderer/modules/cookie_store/cookie_store.cc
@@ -85,6 +85,7 @@ std::unique_ptr<net::CanonicalCookie> ToCanonicalCookie(
   base::Time expires = options->hasExpiresNonNull()
                            ? base::Time::FromJavaTime(options->expiresNonNull())
                            : base::Time();
+  BRAVE_TO_CANONICAL_COOKIE
 
   String cookie_url_host = cookie_url.Host();
   String domain;
