diff --git a/third_party/blink/renderer/core/frame/local_dom_window.cc b/third_party/blink/renderer/core/frame/local_dom_window.cc
index 23a95863dc888a9ffb4b09c2b034665ef1186f7c..b9734e0a3e0e95aa1f5bd19cc5e67bae3c93fbfb 100644
--- a/third_party/blink/renderer/core/frame/local_dom_window.cc
+++ b/third_party/blink/renderer/core/frame/local_dom_window.cc
@@ -513,6 +513,7 @@ bool LocalDOMWindow::CanExecuteScripts(
     return false;
   }
   bool script_enabled = GetFrame()->ScriptEnabled();
+  BRAVE_LOCAL_DOM_WINDOW_CAN_EXECUTE_SCRIPTS
   if (!script_enabled && reason == kAboutToExecuteScript) {
     WebContentSettingsClient* settings_client =
         GetFrame()->GetContentSettingsClient();
