diff --git a/third_party/lit/v3_0/BUILD.gn b/third_party/lit/v3_0/BUILD.gn
index 8eff90c5d4ffd2bf71d87144049ec66c3ca4d131..821f9ec0bb654ef437f67a6830ae8813109dea77 100644
--- a/third_party/lit/v3_0/BUILD.gn
+++ b/third_party/lit/v3_0/BUILD.gn
@@ -113,6 +113,7 @@ ts_library("build_ts") {
     "//ui/webui/resources/cr_components/most_visited:build_ts",
     "//ui/webui/resources/cr_components/theme_color_picker:build_ts",
     "//ui/webui/resources/cr_elements:build_ts",
+    "//chrome/browser/resources/settings:build_ts", "//brave/ui/webui/resources:build_ts"
   ]
   tsconfig_base = "tsconfig_base.json"
   composite = true
@@ -197,6 +198,7 @@ ts_library("build_ts") {
             "${node_modules}/@lit/reactive-element/reactive-element.d.ts",
             target_gen_dir),
   ]
+  import("//brave/third_party/lit/v3_0/sources.gni") visibility += brave_lit_build_ts_visibility
 }
 
 bundle_js("build_bundle") {
