diff --git a/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc b/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc
index 7b806773e3f68c02ab2feaa74932cd4167117a4e..ea507d8eddf3babb14531460cc574a1f6246855c 100644
--- a/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc
+++ b/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc
@@ -187,6 +187,7 @@ void ModuleScriptLoader::FetchInternal(
     options.initiator_info.referrer = module_request.ReferrerString();
     options.initiator_info.position = module_request.GetReferrerPosition();
   }
+  BRAVE_MODULE_SCRIPT_LOADER_FETCH_INTERNAL
 
   // Note: |options| should not be modified after here.
   FetchParameters fetch_params(std::move(resource_request), options);
