diff --git a/third_party/blink/common/client_hints/client_hints.cc b/third_party/blink/common/client_hints/client_hints.cc
index f7aae8012633c71b756aae95a5a7992288f2392e..c95706b735c06b22f7d486fe165264b14abae1e1 100644
--- a/third_party/blink/common/client_hints/client_hints.cc
+++ b/third_party/blink/common/client_hints/client_hints.cc
@@ -125,6 +125,7 @@ base::Optional<std::vector<network::mojom::WebClientHintsType>> FilterAcceptCH(
 }
 
 bool IsClientHintSentByDefault(network::mojom::WebClientHintsType type) {
+  BRAVE_CLIENT_HINTS_IS_CLIENT_HINT_SENT_BY_DEFAULT
   return (type == network::mojom::WebClientHintsType::kUA ||
           type == network::mojom::WebClientHintsType::kUAMobile);
 }
