diff --git a/third_party/blink/renderer/core/script/BUILD.gn b/third_party/blink/renderer/core/script/BUILD.gn
index 3b088840c4705100e0ff5d5018cdaa9b3b6978b4..5db29a32d7279c99cb56469a68fcf5b413b918d4 100644
--- a/third_party/blink/renderer/core/script/BUILD.gn
+++ b/third_party/blink/renderer/core/script/BUILD.gn
@@ -66,7 +66,10 @@ blink_core_sources("script") {
     "xml_parser_script_runner_host.h",
   ]
 
-  deps = [ "//third_party/blink/public:resources" ]
+  deps = [
+    "//third_party/blink/public:resources",
+    "//third_party/blink/renderer/core/loader",
+  ]
 
   jumbo_excluded_sources = [ "modulator.cc" ]  # https://crbug.com/716395
 }
