diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index ae24e3e92c74599d722e6a017dd1c084ea9e727d..f35c59eed08d9140553dfcb458d56a695aec69c5 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -6,6 +6,7 @@
 
 #include <utility>
 
+#include "brave/renderer/brave_content_settings_observer_helper.h"
 #include "mojo/public/cpp/bindings/remote.h"
 #include "third_party/blink/public/common/browser_interface_broker_proxy.h"
 #include "third_party/blink/public/platform/task_type.h"
@@ -73,6 +74,9 @@ ScriptPromise MediaDevices::enumerateDevices(ScriptState* script_state) {
         MakeGarbageCollected<DOMException>(DOMExceptionCode::kNotSupportedError,
                                            "Current frame is detached."));
   }
+  if (!AllowFingerprinting(frame)) {
+    return ScriptPromise::CastUndefined(script_state);
+  }
 
   auto* resolver = MakeGarbageCollected<ScriptPromiseResolver>(script_state);
   ScriptPromise promise = resolver->Promise();
