diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index 460451a1d77ede9819738e5b8413111792646540..4e8b75657a0862418115b5d72b50db5d6a849ed7 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -912,6 +912,7 @@ void MediaDevices::DevicesEnumerated(
     }
   }
 
+  BRAVE_MEDIA_DEVICES_DEVICES_ENUMERATED
   RecordEnumeratedDevices(result_tracker->GetScriptState(), media_devices);
   // TODO(crbug.com/1395324): Remove this custom EnumerateDevices latency
   // tracking by reverting crrev.com/c/3944912/ once the
