diff --git a/third_party/blink/renderer/core/execution_context/execution_context.h b/third_party/blink/renderer/core/execution_context/execution_context.h
index 8045819cc4c066432d1db1a62af48881ff9cc800..51b273a633d686a3120b31260e04a246d5502e85 100644
--- a/third_party/blink/renderer/core/execution_context/execution_context.h
+++ b/third_party/blink/renderer/core/execution_context/execution_context.h
@@ -46,6 +46,11 @@
 #include "third_party/blink/renderer/platform/supplementable.h"
 #include "v8/include/v8.h"
 
+#include "brave/third_party/blink/brave_page_graph/buildflags/buildflags.h"
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+#include "brave/third_party/blink/brave_page_graph/page_graph.h"
+#endif
+
 namespace base {
 class SingleThreadTaskRunner;
 }
@@ -292,10 +297,21 @@ class CORE_EXPORT ExecutionContext : public ContextLifecycleNotifier,
 
   v8::Isolate* GetIsolate() const { return isolate_; }
 
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  virtual brave_page_graph::PageGraph* GetPageGraph() {
+    return nullptr;
+  }
+#endif
+
  protected:
   explicit ExecutionContext(v8::Isolate* isolate);
   ~ExecutionContext() override;
 
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  brave_page_graph::PageGraph* page_graph_;
+  brave_page_graph::PageGraph* page_graph_non_owning_;
+#endif
+
  private:
   v8::Isolate* const isolate_;
 
