diff --git a/third_party/blink/renderer/bindings/core/v8/scheduled_action.cc b/third_party/blink/renderer/bindings/core/v8/scheduled_action.cc
index bb100bcf4b5c157904e6492e0005d027d885217a..1aa1d2d2f0f8eb228da8ccfde74159a7afb08a9a 100644
--- a/third_party/blink/renderer/bindings/core/v8/scheduled_action.cc
+++ b/third_party/blink/renderer/bindings/core/v8/scheduled_action.cc
@@ -89,6 +89,7 @@ ScheduledAction::ScheduledAction(ScriptState* script_state,
   } else {
     UseCounter::Count(target, WebFeature::kScheduledActionIgnored);
   }
+  BRAVE_SCHEDULED_ACTION_STRING_HANDLER_CONSTRUCTOR
 }
 
 ScheduledAction::~ScheduledAction() {
