diff --git a/third_party/rust/serde_json/v1/BUILD.gn b/third_party/rust/serde_json/v1/BUILD.gn
index d46a92bf00d559b1be9ee64329cbd54d23ffaafc..5822fd92a6caa25db254b1ad1406a03d671ecb2c 100644
--- a/third_party/rust/serde_json/v1/BUILD.gn
+++ b/third_party/rust/serde_json/v1/BUILD.gn
@@ -83,5 +83,4 @@ cargo_crate("lib") {
     "--cap-lints=allow",  # Suppress all warnings in crates.io crates
   ]
 
-  testonly = true
 }
