diff --git a/third_party/blink/renderer/core/dom/comment.cc b/third_party/blink/renderer/core/dom/comment.cc
index d7476d197b825f1a13d92a708b84db849f7aacec..ff630fe655bfc62e6684056deea4c6af41ff97d0 100644
--- a/third_party/blink/renderer/core/dom/comment.cc
+++ b/third_party/blink/renderer/core/dom/comment.cc
@@ -23,13 +23,29 @@
 
 #include "third_party/blink/renderer/core/dom/document.h"
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+#include "third_party/blink/renderer/core/dom/dom_node_ids.h"
+#include "brave/third_party/blink/brave_page_graph/page_graph.h"
+// #endif
+
 namespace blink {
 
 inline Comment::Comment(Document& document, const String& text)
     : CharacterData(document, text, kCreateOther) {}
 
-Comment* Comment::Create(Document& document, const String& text) {
-  return MakeGarbageCollected<Comment>(document, text);
+ Comment* Comment::Create(Document& document, const String& text) {
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  Comment* comment = MakeGarbageCollected<Comment>(document, text);
+  {
+    if (document.GetPageGraph() != nullptr) {
+      document.GetPageGraph()->RegisterHTMLTextNodeCreated(
+        DOMNodeIds::IdForNode(comment), text);
+    }
+  }
+  return comment;
+// #else
+//   return MakeGarbageCollected<Comment>(document, text);
+// #endif
 }
 
 String Comment::nodeName() const {
