diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index 8e37a2961459d5eb59d6e5d0dbb85aca39dd820b..3c3f1d835a179d1f076a69f3914192daf09e6541 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -121,6 +121,7 @@ component("core") {
 
   visibility = []  # Allow re-assignment of list.
   visibility = [ "//third_party/blink/*" ]
+  visibility += brave_blink_renderer_core_visibility
 
   # If you create a new subdirectory, make a new BUILD file for that directory
   # and reference it in the deps below rather than adding the sources here.
@@ -192,6 +193,7 @@ component("core") {
     "//url",
     "//v8",
   ]
+  public_deps += brave_blink_renderer_core_public_deps
   deps = [
     "//components/paint_preview/common",
     "//third_party/blink/public/common",
