diff --git a/third_party/blink/renderer/modules/webaudio/realtime_analyser.cc b/third_party/blink/renderer/modules/webaudio/realtime_analyser.cc
index dc68f422ce8af76e4304167385989bb01f7cb2f5..36306322616fa1eef269341e17d7e82177c3ce00 100644
--- a/third_party/blink/renderer/modules/webaudio/realtime_analyser.cc
+++ b/third_party/blink/renderer/modules/webaudio/realtime_analyser.cc
@@ -115,6 +115,7 @@ void RealtimeAnalyser::GetFloatFrequencyData(DOMFloat32Array* destination_array,
     const float* source = magnitude_buffer_.Data();
     float* destination = destination_array->Data();
 
+    BRAVE_REALTIMEANALYSER_CONVERTFLOATTODB
     for (unsigned i = 0; i < len; ++i) {
       const float linear_value = UNSAFE_TODO(source[i]);
       const double db_mag = audio_utilities::LinearToDecibels(linear_value);
@@ -150,6 +151,7 @@ void RealtimeAnalyser::GetByteFrequencyData(DOMUint8Array* destination_array,
     const float* source = magnitude_buffer_.Data();
     unsigned char* destination = destination_array->Data();
 
+    BRAVE_REALTIMEANALYSER_CONVERTTOBYTEDATA
     for (unsigned i = 0; i < len; ++i) {
       const float linear_value = UNSAFE_TODO(source[i]);
       const double db_mag = audio_utilities::LinearToDecibels(linear_value);
@@ -183,6 +185,7 @@ void RealtimeAnalyser::GetFloatTimeDomainData(
 
     const unsigned write_index = GetWriteIndex();
 
+    BRAVE_REALTIMEANALYSER_GETFLOATTIMEDOMAINDATA
     for (unsigned i = 0; i < len; ++i) {
       // Buffer access is protected due to modulo operation.
       float value = UNSAFE_TODO(
@@ -210,6 +213,7 @@ void RealtimeAnalyser::GetByteTimeDomainData(DOMUint8Array* destination_array) {
 
     const unsigned write_index = GetWriteIndex();
 
+    BRAVE_REALTIMEANALYSER_GETBYTETIMEDOMAINDATA
     for (unsigned i = 0; i < len; ++i) {
       // Buffer access is protected due to modulo operation.
       const float value = UNSAFE_TODO(
