diff --git a/third_party/tflite/BUILD.gn b/third_party/tflite/BUILD.gn
index 9ee7eaaad60549f9e8d42f55a7c5ebffd3e8715d..01848ee7fbd0b6296c905bfce48e94f23081d355 100644
--- a/third_party/tflite/BUILD.gn
+++ b/third_party/tflite/BUILD.gn
@@ -219,6 +219,7 @@ tflite_static_library("tflite_builtin_op_resolver") {
     "//services/webnn/*",
     "//third_party/mediapipe/*",
   ]
+  import("//brave/third_party/tflite/tflite.gni") visibility += brave_tflite_builtin_op_resolver_visibility
 
   conditional_deps = [ ":tflite" ]
 }
