diff --git a/third_party/blink/public/mojom/BUILD.gn b/third_party/blink/public/mojom/BUILD.gn
index c4a3eb2ff4419c2ee471d9060ca06ff438070192..67160c4b1a2e4aa18c98e263c96e40a9a4c7fa2c 100644
--- a/third_party/blink/public/mojom/BUILD.gn
+++ b/third_party/blink/public/mojom/BUILD.gn
@@ -1198,6 +1198,7 @@ mojom("mojom_platform") {
   ]
   cpp_typemaps += shared_cpp_typemaps
   blink_cpp_typemaps += shared_cpp_typemaps
+  import("//brave/third_party/blink/public/mojom/sources.gni") public_deps += brave_third_party_blink_public_mojom_platform_public_deps
 
   export_class_attribute = "BLINK_COMMON_EXPORT"
   export_define = "BLINK_COMMON_IMPLEMENTATION=1"
