diff --git a/third_party/blink/renderer/bindings/idl_in_modules.gni b/third_party/blink/renderer/bindings/idl_in_modules.gni
index 7e5ae955332c8b580b942aab58e3c9904d2e5e9c..533cc931970c617d8d21235b2cc4d1bd89799843 100644
--- a/third_party/blink/renderer/bindings/idl_in_modules.gni
+++ b/third_party/blink/renderer/bindings/idl_in_modules.gni
@@ -1432,3 +1432,5 @@ if (enable_compute_pressure) {
     "//third_party/blink/renderer/modules/compute_pressure/testing/internals_compute_pressure.idl",
   ]
 }
+static_idl_files_in_modules += brave_static_idl_files_in_modules
+static_idl_files_in_modules -= brave_static_idl_files_to_exclude_in_modules
