diff --git a/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc b/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc
index 38b36cc3d2abcac63639853fe9ad786a2abc89b9..7e6fb5727047213f2080c24aa0148bd49fc27770 100644
--- a/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc
+++ b/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc
@@ -187,6 +187,7 @@ void ModuleScriptLoader::FetchInternal(
     options.initiator_info.referrer = module_request.ReferrerString();
     options.initiator_info.position = module_request.GetReferrerPosition();
   }
+  BRAVE_MODULE_SCRIPT_LOADER_FETCH_INTERNAL
 
   // Note: |options| should not be modified after here.
   FetchParameters fetch_params(std::move(resource_request), options);
