diff --git a/third_party/blink/renderer/core/dom/text.cc b/third_party/blink/renderer/core/dom/text.cc
index 65c3f62815293afb33c6ccce350382c58837492c..8e2485e3331f3af13cbf3b45e48493dfc1b340a0 100644
--- a/third_party/blink/renderer/core/dom/text.cc
+++ b/third_party/blink/renderer/core/dom/text.cc
@@ -45,10 +45,26 @@
 #include "third_party/blink/renderer/platform/wtf/text/cstring.h"
 #include "third_party/blink/renderer/platform/wtf/text/string_builder.h"
 
+#include "brave/third_party/blink/brave_page_graph/buildflags/buildflags.h"
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+#include "third_party/blink/renderer/core/dom/document.h"
+#include "third_party/blink/renderer/core/dom/dom_node_ids.h"
+#include "brave/third_party/blink/brave_page_graph/page_graph.h"
+#endif
+
 namespace blink {
 
 Text* Text::Create(Document& document, const String& data) {
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  Text* text_node = MakeGarbageCollected<Text>(document, data, kCreateText);
+  if (document.GetPageGraph() != nullptr) {
+    document.GetPageGraph()->RegisterHTMLTextNodeCreated(
+      DOMNodeIds::IdForNode(text_node), data);
+  }
+  return text_node;
+#else
   return MakeGarbageCollected<Text>(document, data, kCreateText);
+#endif
 }
 
 Text* Text::CreateEditingText(Document& document, const String& data) {
