diff --git a/third_party/rust/syn/v2/BUILD.gn b/third_party/rust/syn/v2/BUILD.gn
index fc0d12cc2947cc11f1015acaa04eaa6e5be5beee..7e84cdd5886b658f3421a015afbeb4e5a47a6876 100644
--- a/third_party/rust/syn/v2/BUILD.gn
+++ b/third_party/rust/syn/v2/BUILD.gn
@@ -105,4 +105,5 @@ cargo_crate("lib") {
   rustflags = [
     "--cap-lints=allow",  # Suppress all warnings in crates.io crates
   ]
+  features += [ "extra-traits", "visit", "visit-mut" ]
 }
