diff --git a/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc b/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc
index 9f2d95bf4c94aff446f7ae23547672e3124f6ffb..8ed1152732bc0a2a9b48dc4dda5f0cd002c4b459 100644
--- a/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc
+++ b/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc
@@ -302,7 +302,7 @@ int GetMicrotasksScopeDepth(v8::Isolate* isolate,
 
 }  // namespace
 
-v8::MaybeLocal<v8::Script> V8ScriptRunner::CompileScript(
+v8::MaybeLocal<v8::Script> V8ScriptRunner::CompileScript_ChromiumImpl(
     ScriptState* script_state,
     const ClassicScript& classic_script,
     v8::ScriptOrigin origin,
@@ -346,7 +346,7 @@ v8::MaybeLocal<v8::Script> V8ScriptRunner::CompileScript(
   return script;
 }
 
-v8::MaybeLocal<v8::Module> V8ScriptRunner::CompileModule(
+v8::MaybeLocal<v8::Module> V8ScriptRunner::CompileModule_ChromiumImpl(
     v8::Isolate* isolate,
     const ModuleScriptCreationParams& params,
     const TextPosition& start_position,
