diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index 0ae7872c95e155707d1c96908b58c6e886debaa3..200b51613b6a1571a839046dc0fd1895db77d4b3 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -1163,6 +1163,7 @@ void MediaDevices::DevicesEnumerated(
     }
   }
 
+  BRAVE_MEDIA_DEVICES_DEVICES_ENUMERATED
   RecordEnumeratedDevices(result_tracker->GetScriptState(), media_devices);
   result_tracker->Resolve(media_devices);
 }
