diff --git a/third_party/blink/renderer/modules/modules_idl_files.gni b/third_party/blink/renderer/modules/modules_idl_files.gni
index 8bf1bbd11f99fd55846457524923af011f9b7dbf..5c8b6c0b3b721c18800151f3222b59447aaa64a9 100644
--- a/third_party/blink/renderer/modules/modules_idl_files.gni
+++ b/third_party/blink/renderer/modules/modules_idl_files.gni
@@ -153,6 +153,7 @@ if (!is_android) {
     "//third_party/blink/renderer/modules/serial/idls.gni",
   ]
 }
+_idl_imports += brave_idl_imports
 
 # Do not add IDL files directly to these lists. See _idl_imports above instead
 # on how to add IDL files for a module.
