diff --git a/third_party/blink/renderer/bindings/idl_in_modules.gni b/third_party/blink/renderer/bindings/idl_in_modules.gni
index 6f703f9066b3c822659a9ed87bbb2d326be22666..de8b9794b1a815b53908a783680c0a3f187bfe93 100644
--- a/third_party/blink/renderer/bindings/idl_in_modules.gni
+++ b/third_party/blink/renderer/bindings/idl_in_modules.gni
@@ -1301,6 +1301,7 @@ static_idl_files_in_modules = [
   "//third_party/blink/renderer/modules/xr/xr_webgl_layer_init.idl",
   "//third_party/blink/renderer/modules/xr/xr_webgl_sub_image.idl",
 ]
+static_idl_files_in_modules += brave_static_idl_files_in_modules static_idl_files_in_modules -= brave_static_idl_files_to_exclude_in_modules
 
 # Direct Sockets
 # This uses target_os rather than current_os (which is what is_android is set
