diff --git a/third_party/rust/winapi/v0_3/BUILD.gn b/third_party/rust/winapi/v0_3/BUILD.gn
index 80afe5d9b929448f7b3009761ec262657e930db7..cc1216e5db3dc7d69fff7a8cf45c22bea5a5966e 100644
--- a/third_party/rust/winapi/v0_3/BUILD.gn
+++ b/third_party/rust/winapi/v0_3/BUILD.gn
@@ -447,6 +447,7 @@ cargo_crate("lib") {
     "winerror",
     "winnt",
   ]
+  features += [ "basetsd", "cfg", "cfgmgr32", "devpropdef", "excpt", "handleapi", "ktmtypes", "libloaderapi", "limits", "ntdef", "ntstatus", "processthreadsapi", "reason", "rpcndr", "timezoneapi", "vadefs", "vcruntime", "wincontypes", "wingdi", "windef", "winreg", "winuser", "wtypesbase" ]
   build_root =
       "//third_party/rust/chromium_crates_io/vendor/winapi-0.3.9/build.rs"
   build_sources =
