diff --git a/third_party/blink/renderer/bindings/idl_in_modules.gni b/third_party/blink/renderer/bindings/idl_in_modules.gni
index 01ed8a95cfe143eb1f66950979dff6905f8d3ae8..44b5d18ca08d1b62c20e0ed7b4abed6a1e9565d2 100644
--- a/third_party/blink/renderer/bindings/idl_in_modules.gni
+++ b/third_party/blink/renderer/bindings/idl_in_modules.gni
@@ -1282,6 +1282,7 @@ static_idl_files_in_modules = [
   "//third_party/blink/renderer/modules/xr/xr_webgl_layer_init.idl",
   "//third_party/blink/renderer/modules/xr/xr_webgl_sub_image.idl",
 ]
+static_idl_files_in_modules += brave_static_idl_files_in_modules
 
 # Direct Sockets
 # This uses target_os rather than current_os (which is what is_android is set
