diff --git a/third_party/blink/renderer/platform/BUILD.gn b/third_party/blink/renderer/platform/BUILD.gn
index 0adcb6894f30356c933d6f77b5a9f6e4723ea6b7..f488d77ab0314a69beeca69eb4e262fee503a202 100644
--- a/third_party/blink/renderer/platform/BUILD.gn
+++ b/third_party/blink/renderer/platform/BUILD.gn
@@ -1755,6 +1755,7 @@ component("platform") {
     "widget/widget_base_client.h",
     "windows_keyboard_codes.h",
   ]
+  sources += brave_blink_renderer_platform_sources
 
   sources -= blink_platform_avx_files
 
@@ -1882,6 +1883,7 @@ component("platform") {
     "//ui/gfx",
     "//ui/native_theme/features",
   ]
+  public_deps += brave_blink_renderer_platform_public_deps
   deps = [
     ":character_property_data",
     ":font_rust_support",
@@ -1943,6 +1945,7 @@ component("platform") {
     "//ui/gl:buildflags",
     "//ui/native_theme",
   ]
+  deps += brave_blink_renderer_platform_deps
 
   if (has_zstd_compression) {
     deps += [
@@ -2064,6 +2067,7 @@ component("platform") {
 
   configs -= [ "//build/config/compiler:default_symbols" ]
   configs += blink_symbols_config
+  visibility += brave_blink_renderer_platform_visibility
 }
 
 static_library("test_support") {
