diff --git a/third_party/blink/renderer/modules/websockets/websocket_channel_impl.h b/third_party/blink/renderer/modules/websockets/websocket_channel_impl.h
index 5c8f237f387e322d425e531bca63ae3e26f8164a..3b6e13ed7eda979c6da9fdcbb0675d5c73b74d8e 100644
--- a/third_party/blink/renderer/modules/websockets/websocket_channel_impl.h
+++ b/third_party/blink/renderer/modules/websockets/websocket_channel_impl.h
@@ -79,7 +79,7 @@ class WebSocketHandshakeThrottle;
 // This is an implementation of WebSocketChannel. This is created on the main
 // thread for Document, or on the worker thread for WorkerGlobalScope. All
 // functions must be called on the execution context's thread.
-class MODULES_EXPORT WebSocketChannelImpl final
+class MODULES_EXPORT WebSocketChannelImpl
     : public WebSocketChannel,
       public network::mojom::blink::WebSocketHandshakeClient,
       public network::mojom::blink::WebSocketClient {
@@ -366,6 +366,7 @@ class MODULES_EXPORT WebSocketChannelImpl final
   void OnConnectionError(const base::Location& set_from,
                          uint32_t custom_reason,
                          const std::string& description);
+  virtual /* Explicit via patch, because Dispose() used everywhere in Blink. */
   void Dispose();
 
   const Member<WebSocketChannelClient> client_;
