diff --git a/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc b/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
index 8f4d9db3ee6ec9061c542dadb85e65e13da974fd..864933cfb0438ec698db71b2b3b753df52ed2b64 100644
--- a/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
+++ b/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
@@ -195,6 +195,7 @@ void DOMPluginArray::UpdatePluginData() {
     }
     return;
   }
+  BRAVE_DOM_PLUGINS_UPDATE_PLUGIN_DATA__RESET_PLUGIN_DATA
   PluginData* data = GetPluginData();
   if (!data) {
     dom_plugins_.clear();
@@ -216,6 +217,7 @@ void DOMPluginArray::UpdatePluginData() {
       }
     }
   }
+  BRAVE_DOM_PLUGINS_UPDATE_PLUGIN_DATA__FARBLE_PLUGIN_DATA
 }
 
 void DOMPluginArray::ContextDestroyed() {
