diff --git a/third_party/blink/common/BUILD.gn b/third_party/blink/common/BUILD.gn
index 04b9e8dd4d0c7e19db0c58c9dd176e86d5a14df3..b120231935b9d78ce042563c618d31d7639827e1 100644
--- a/third_party/blink/common/BUILD.gn
+++ b/third_party/blink/common/BUILD.gn
@@ -350,6 +350,7 @@ source_set("common") {
   if (is_chromeos || is_android || is_win || is_castos || is_fuchsia) {
     defines = [ "ENABLE_PROTECTED_MEDIA_IDENTIFIER_PERMISSION" ]
   }
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_blink_common_deps
 }
 
 test("blink_common_unittests") {
@@ -489,6 +490,7 @@ source_set("common_unittests_sources") {
   # TODO(crbug.com/40031409): Fix code that adds exit-time destructors and
   # enable the diagnostic by removing this line.
   configs += [ "//build/config/compiler:no_exit_time_destructors" ]
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_blink_common_deps
 }
 
 group("common_unittests_data") {
