diff --git a/third_party/blink/renderer/bindings/scripts/bind_gen/interface.py b/third_party/blink/renderer/bindings/scripts/bind_gen/interface.py
index a67710c9dc80d07e8f2a4e7ce7c004fa80865830..c18c054b0a97f45a07f93c8a38d00dc617d9a542 100644
--- a/third_party/blink/renderer/bindings/scripts/bind_gen/interface.py
+++ b/third_party/blink/renderer/bindings/scripts/bind_gen/interface.py
@@ -7620,3 +7620,4 @@ def generate_interfaces(task_queue):
                          "InitIDLInterfacesForTesting",
                          "init_idl_interfaces_for_testing",
                          for_testing=True)
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
\ No newline at end of file
