diff --git a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
index 689e72208e41758f5c1bb830d6fb612792aef6e7..e54ab07a3162cdaa2dc2a38bb8850559ffcaebe9 100644
--- a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
+++ b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
@@ -396,6 +396,7 @@ class CORE_EXPORT HTMLCanvasElement final
 
   void ResetLayer();
 
+  BRAVE_HTML_CANVAS_ELEMENT_H_
  protected:
   void DidMoveToNewDocument(Document& old_document) override;
   void DidRecalcStyle(const StyleRecalcChange change) override;
