diff --git a/third_party/blink/renderer/core/loader/modulescript/worker_module_script_fetcher.cc b/third_party/blink/renderer/core/loader/modulescript/worker_module_script_fetcher.cc
index 8aeea99a2c8a2975a1f26d7c8faef542412e822d..e489d5f8be9cfc01fd46449a45552f2f2e8e533a 100644
--- a/third_party/blink/renderer/core/loader/modulescript/worker_module_script_fetcher.cc
+++ b/third_party/blink/renderer/core/loader/modulescript/worker_module_script_fetcher.cc
@@ -15,6 +15,14 @@
 #include "third_party/blink/renderer/platform/network/network_utils.h"
 #include "third_party/blink/renderer/platform/weborigin/security_policy.h"
 
+#include "brave/third_party/blink/brave_page_graph/buildflags/buildflags.h"
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+#include "brave/third_party/blink/brave_page_graph/types.h"
+#include "brave/third_party/blink/brave_page_graph/page_graph.h"
+#include "brave/third_party/blink/brave_page_graph/utilities/digest.h"
+#include "brave/third_party/blink/brave_page_graph/utilities/response_metadata.h"
+#endif
+
 namespace blink {
 
 WorkerModuleScriptFetcher::WorkerModuleScriptFetcher(
@@ -42,8 +50,23 @@ void WorkerModuleScriptFetcher::Fetch(
   // <spec step="12.2">Fetch request, and asynchronously wait to run the
   // remaining steps as part of fetch's process response for the response
   // response.</spec>
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  ScriptResource* resource = ScriptResource::Fetch(fetch_params, fetch_client_settings_object_fetcher,
+                        this, ScriptResource::kNoStreaming);
+  ::brave_page_graph::PageGraph* page_graph = modulator_for_built_in_modules->GetPageGraph();
+  if (page_graph != nullptr) {
+    this->page_graph_ = page_graph;
+    const auto initiator_elm = modulator_for_built_in_modules->GetInitiatingDOMNodeId();
+    if(initiator_elm > 0) {
+      page_graph->RegisterRequestStartFromElm(initiator_elm, resource->InspectorId(), fetch_params.Url(), brave_page_graph::RequestType::kRequestTypeScriptModule);
+    } else {
+      LOG(ERROR) << "no initiator element for " << fetch_params.Url();
+    }
+  }
+#else
   ScriptResource::Fetch(fetch_params, fetch_client_settings_object_fetcher,
                         this, ScriptResource::kNoStreaming);
+#endif
 }
 
 void WorkerModuleScriptFetcher::Trace(blink::Visitor* visitor) {
@@ -136,6 +159,22 @@ void WorkerModuleScriptFetcher::NotifyFinished(Resource* resource) {
       script_resource->SourceText(), script_resource->CacheHandler(),
       script_resource->GetResourceRequest().GetCredentialsMode());
 
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  {
+    if (page_graph_ != nullptr) {
+      const ResourceResponse& rep = resource->GetResponse();
+      const ::brave_page_graph::ResponseMetadata metadata(rep);
+
+      if (resource->ErrorOccurred()) {
+        page_graph_->RegisterRequestError(resource->InspectorId(), metadata);
+      } else {
+        page_graph_->RegisterRequestComplete(resource->InspectorId(),
+            resource->GetType(), metadata,
+            ::brave_page_graph::ResourceDigest(resource));
+      }
+    }
+  }
+#endif
   // <spec step="12.7">Asynchronously complete the perform the fetch steps with
   // response.</spec>
   client_->NotifyFetchFinished(params, error_messages);
