diff --git a/third_party/android_deps/BUILD.gn b/third_party/android_deps/BUILD.gn
index 279c07239a8dd4bbea7edeb9a396dc4ad920ca68..a7c8498c7c5a37638a2cf062bed61aebc57e0262 100644
--- a/third_party/android_deps/BUILD.gn
+++ b/third_party/android_deps/BUILD.gn
@@ -465,6 +465,7 @@ java_prebuilt("com_google_auto_value_auto_value_annotations_java") {
     ":*",
     "//third_party/androidx:*",
   ]
+  visibility += [ "//brave/third_party/android_deps:com_google_android_datatransport_transport_runtime_java__classes" ] supports_android = true
 }
 
 # This is generated, do not edit. Update BuildConfigGenerator.groovy instead.
