diff --git a/third_party/blink/renderer/core/script/script_loader.cc b/third_party/blink/renderer/core/script/script_loader.cc
index 123621dbbfa2c6197f735d8fab0ac839f33d706a..cb3606ec51b12cbfae8a38f762e04e3b10d4bcab 100644
--- a/third_party/blink/renderer/core/script/script_loader.cc
+++ b/third_party/blink/renderer/core/script/script_loader.cc
@@ -767,6 +767,7 @@ PendingScript* ScriptLoader::PrepareScript(
                              parser_state, credentials_mode, referrer_policy,
                              fetch_priority_hint, render_blocking_behavior,
                              RejectCoepUnsafeNone(false));
+  BRAVE_SCRIPT_LOADER_PREPARE_SCRIPT
 
   // <spec step="28">Let settings object be el's node document's relevant
   // settings object.</spec>
