diff --git a/third_party/blink/renderer/core/script/script_loader.cc b/third_party/blink/renderer/core/script/script_loader.cc
index 150e900e88990deaac8bc7e98b0abbe5a32cf699..8f58440b2c469033a6dab81ccfaf0e43fdbe534c 100644
--- a/third_party/blink/renderer/core/script/script_loader.cc
+++ b/third_party/blink/renderer/core/script/script_loader.cc
@@ -730,6 +730,7 @@ PendingScript* ScriptLoader::PrepareScript(
   ScriptFetchOptions options(nonce, integrity_metadata, integrity_attr,
                              parser_state, credentials_mode, referrer_policy,
                              fetch_priority_hint, render_blocking_behavior);
+  BRAVE_SCRIPT_LOADER_PREPARE_SCRIPT
 
   // <spec step="30">Let settings object be el's node document's relevant
   // settings object.</spec>
