diff --git a/third_party/blink/renderer/core/loader/link_loader.h b/third_party/blink/renderer/core/loader/link_loader.h
index 10d0724c02f907635b938decbdc882cecbe71aee..b0935844a96582157f5c3a3dc99849b4914daf04 100644
--- a/third_party/blink/renderer/core/loader/link_loader.h
+++ b/third_party/blink/renderer/core/loader/link_loader.h
@@ -45,6 +45,9 @@ class LinkLoaderClient;
 class PrerenderHandle;
 class Resource;
 class ResourceClient;
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+class HTMLLinkElement;
+#endif
 
 // The LinkLoader can load link rel types icon, dns-prefetch, prefetch, and
 // prerender.
@@ -77,6 +80,10 @@ class CORE_EXPORT LinkLoader final : public SingleModuleClient,
 
   void Trace(blink::Visitor*) override;
 
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  void SetHTMLLinkElementOwner(HTMLLinkElement* link_element);
+#endif
+
  private:
   class FinishObserver;
 
@@ -88,6 +95,10 @@ class CORE_EXPORT LinkLoader final : public SingleModuleClient,
   Member<LinkLoaderClient> client_;
 
   Member<PrerenderHandle> prerender_;
+
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  Member<HTMLLinkElement> link_element_;
+#endif
 };
 
 }  // namespace blink
