diff --git a/third_party/libaddressinput/chromium/chrome_metadata_source.cc b/third_party/libaddressinput/chromium/chrome_metadata_source.cc
index 6bc02079f3330cb8c3af670c208348cd7bc7b5f6..f2a8613236f2f19ba913b0c6473050400e917138 100644
--- a/third_party/libaddressinput/chromium/chrome_metadata_source.cc
+++ b/third_party/libaddressinput/chromium/chrome_metadata_source.cc
@@ -58,7 +58,7 @@ ChromeMetadataSource::Request::Request(
 void ChromeMetadataSource::Download(const std::string& key,
                                     const Callback& downloaded) {
   GURL resource(validation_data_url_ + key);
-  if (!resource.SchemeIsCryptographic()) {
+  if (1) { // feature disabled in Brave
     downloaded(false, key, NULL);
     return;
   }
