diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index cf7fb031d669264d1bb9a033588b48a6dd0cd40c..5eb3e1c64d9958c3228d892990642bec3b57beeb 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -457,6 +457,7 @@ component("core") {
       "Carbon.framework",
     ]
   }
+  visibility += brave_blink_renderer_core_visibility public_deps += brave_blink_renderer_core_public_deps sources += brave_blink_renderer_core_sources deps += brave_blink_renderer_core_deps
 }
 
 # Certain sources should be compiled with heavy optimization enabled.
