diff --git a/third_party/android_deps/BUILD.gn b/third_party/android_deps/BUILD.gn
index f5bb67b74f052f83aed0d28b9f212880263f8958..63d72878915fc04d69892928a8d5f6122f8f1323 100644
--- a/third_party/android_deps/BUILD.gn
+++ b/third_party/android_deps/BUILD.gn
@@ -465,6 +465,7 @@ java_prebuilt("com_google_auto_value_auto_value_annotations_java") {
     ":*",
     "//third_party/androidx:*",
   ]
+  visibility += [ "//brave/third_party/android_deps:com_google_android_datatransport_transport_runtime_java__classes" ] supports_android = true
 }
 
 # This is generated, do not edit. Update BuildConfigGenerator.groovy instead.
