diff --git a/third_party/blink/renderer/BUILD.gn b/third_party/blink/renderer/BUILD.gn
index e90040bc6e21ab9553e88166a1796eeb4bf3be8e..bde2afc952ac3ea457938bf37476fdcff8404596 100644
--- a/third_party/blink/renderer/BUILD.gn
+++ b/third_party/blink/renderer/BUILD.gn
@@ -9,6 +9,7 @@ if (is_clang) {
 }
 
 visibility = [ "//third_party/blink/*" ]
+visibility += brave_blink_renderer_modules_visibility
 
 # arguments --------------------------------------------------------------------
 
