diff --git a/third_party/blink/renderer/bindings/scripts/bind_gen/task_queue.py b/third_party/blink/renderer/bindings/scripts/bind_gen/task_queue.py
index 8cd5a8c83274a1f78d21c14d54bd3f324e0576dc..7051c845c9a534d59c2426daf9ebf94028695530 100644
--- a/third_party/blink/renderer/bindings/scripts/bind_gen/task_queue.py
+++ b/third_party/blink/renderer/bindings/scripts/bind_gen/task_queue.py
@@ -130,3 +130,4 @@ class TaskQueue(object):
                 break
 
         self._pool.join()
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
\ No newline at end of file
