diff --git a/third_party/blink/public/BUILD.gn b/third_party/blink/public/BUILD.gn
index d7e025324889bb94136d05fe4d25e445eb85c1e9..0d12694524112ecd2d52b168febdd568f9a2c47b 100644
--- a/third_party/blink/public/BUILD.gn
+++ b/third_party/blink/public/BUILD.gn
@@ -671,6 +671,7 @@ mojom("mojo_bindings") {
   visibility_blink =
       [ "//third_party/blink/renderer/platform:blink_platform_public_deps" ]
   sources = [
+    "../../../brave/third_party/blink/public/platform/brave_drm.mojom",
     "platform/autoplay.mojom",
     "platform/content_security_policy.mojom",
     "platform/dedicated_worker_factory.mojom",
