diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
index b36da920302f6d13499187f98efa315ff6191c56..a01690aa079f9a329385ee7c7bf40de6f0aa6413 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
@@ -1550,6 +1550,7 @@ ImageData* BaseRenderingContext2D::createImageData(
 }
 
 ImageData* BaseRenderingContext2D::getImageData(
+    ScriptState* script_state,
     int sx,
     int sy,
     int sw,
@@ -1627,6 +1628,7 @@ ImageData* BaseRenderingContext2D::getImageData(
     return nullptr;
   }
 
+  BRAVE_GET_IMAGE_DATA
   bool may_have_stray_area =
       IsAccelerated()  // GPU readback may fail silently.
       || StaticBitmapImage::MayHaveStrayArea(snapshot, image_data_rect);
