diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d.cc
index 00f8e5b4d06a6a26f4f05e8a7b54c05ce1c39c83..9b147ba8f4d76db84bacdb8f08daec0e982f3c59 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d.cc
@@ -33,6 +33,7 @@
 
 #include "third_party/blink/renderer/modules/canvas/canvas2d/canvas_rendering_context_2d.h"
 
+#include "brave/renderer/brave_content_settings_observer_helper.h"
 #include "third_party/blink/public/common/features.h"
 #include "third_party/blink/public/platform/platform.h"
 #include "third_party/blink/public/platform/task_type.h"
@@ -73,6 +74,12 @@
 #include "third_party/blink/renderer/platform/wtf/text/string_builder.h"
 #include "third_party/blink/renderer/platform/wtf/typed_arrays/array_buffer_contents.h"
 
+#include "brave/third_party/blink/brave_page_graph/buildflags/buildflags.h"
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+#include "brave/third_party/blink/brave_page_graph/page_graph.h"
+#include <sstream>
+#endif
+
 namespace blink {
 
 static const TimeDelta kTryRestoreContextInterval =
@@ -782,6 +789,9 @@ TextMetrics* CanvasRenderingContext2D::measureText(const String& text) {
   if (!canvas()->GetDocument().GetFrame())
     return MakeGarbageCollected<TextMetrics>();
 
+  if (!AllowFingerprinting(canvas()->GetDocument().GetFrame()))
+    return MakeGarbageCollected<TextMetrics>();
+
   canvas()->GetDocument().UpdateStyleAndLayoutTreeForNode(canvas());
 
   const Font& font = AccessFont();
@@ -793,6 +803,44 @@ TextMetrics* CanvasRenderingContext2D::measureText(const String& text) {
   else
     direction = ToTextDirection(GetState().GetDirection(), canvas());
 
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  {
+    TextMetrics* result = TextMetrics::Create(font, direction,
+      GetState().GetTextBaseline(), GetState().GetTextAlign(), text);
+
+    brave_page_graph::PageGraph* page_graph =
+      canvas()->GetDocument().GetPageGraph();
+    if (page_graph != nullptr) {
+      std::vector<const String> args;
+      args.push_back(text);
+      page_graph->RegisterWebAPICall("CanvasRenderingContext2D.measureText",
+        args);
+
+      std::stringstream result_buffer;
+      result_buffer << "width: " << result->width()
+        << ", actualBoundingBoxLeft: " << result->actualBoundingBoxLeft()
+        << ", actualBoundingBoxRight: " << result->actualBoundingBoxRight()
+        << ", fontBoundingBoxAscent: " << result->fontBoundingBoxAscent()
+        << ", fontBoundingBoxDescent: " << result->fontBoundingBoxDescent()
+        << ", actualBoundingBoxAscent: " << result->actualBoundingBoxAscent()
+        << ", actualBoundingBoxDescent: " << result->actualBoundingBoxDescent()
+        << ", emHeightAscent: " << result->emHeightAscent()
+        << ", emHeightDescent: " << result->emHeightDescent();
+
+      Baselines* baselines = result->getBaselines();
+      result_buffer << ", hangingBaseline: " << baselines->hanging()
+        << ", alphabeticBaseline: " << baselines->alphabetic()
+        << ", ideographicBaseline: " << baselines->ideographic();
+
+      String page_graph_result(result_buffer.str().c_str());
+      page_graph->RegisterWebAPIResult("CanvasRenderingContext2D.measureText",
+        page_graph_result);
+    }
+
+    return result;
+  }
+#endif
+
   return TextMetrics::Create(font, direction, GetState().GetTextBaseline(),
                              GetState().GetTextAlign(), text);
 }
