diff --git a/third_party/rust/syn/v2/BUILD.gn b/third_party/rust/syn/v2/BUILD.gn
index f5a275f83c204710ab4b35ec3c1da6290c4e8231..99c07cca716cacdcf7f9fcf0cbd8dc8df1aa83dd 100644
--- a/third_party/rust/syn/v2/BUILD.gn
+++ b/third_party/rust/syn/v2/BUILD.gn
@@ -104,4 +104,5 @@ cargo_crate("lib") {
   rustflags = [
     "--cap-lints=allow",  # Suppress all warnings in crates.io crates
   ]
+  features += [ "extra-traits", "visit", "visit-mut" ]
 }
