diff --git a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
index ef0f7a1ea6dbe572d8860f0059a113d7bbccb0d9..c10adaf1fa4ed81c39eeb5acd0a285eba02ecf34 100644
--- a/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
+++ b/third_party/blink/renderer/core/html/canvas/html_canvas_element.h
@@ -342,6 +342,7 @@ class CORE_EXPORT HTMLCanvasElement final
                                  DOMMatrix* draw_transform,
                                  ExceptionState&);
 
+  BRAVE_HTML_CANVAS_ELEMENT_H_
  protected:
   void DidMoveToNewDocument(Document& old_document) override;
   void DidRecalcStyle(const StyleRecalcChange change) override;
