diff --git a/third_party/blink/renderer/modules/peerconnection/BUILD.gn b/third_party/blink/renderer/modules/peerconnection/BUILD.gn
index 00649ea5597f29122a30666bbd3eb488ad41d981..8f4ff432cc522afc88eae8be86ceea1be7dcd171 100644
--- a/third_party/blink/renderer/modules/peerconnection/BUILD.gn
+++ b/third_party/blink/renderer/modules/peerconnection/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/components/ai_chat/sources.gni")  # Brave
 import("//third_party/blink/renderer/modules/modules.gni")
 
 blink_modules_sources("peerconnection") {
@@ -190,6 +191,7 @@ blink_modules_sources("peerconnection") {
     "//third_party/blink/renderer/modules/permissions",
     "//third_party/blink/renderer/modules/webrtc",
   ]
+  deps += brave_blink_renderer_modules_peerconnection_deps  # Brave
 
   allow_circular_includes_from =
       [ "//third_party/blink/renderer/modules/mediastream" ]
