diff --git a/third_party/blink/renderer/core/loader/link_loader.h b/third_party/blink/renderer/core/loader/link_loader.h
index aa10afaed85d5eed1a3fcd2747d3040452cdfcff..fd236a26405bdeca675bf714e720aad55ee07cc6 100644
--- a/third_party/blink/renderer/core/loader/link_loader.h
+++ b/third_party/blink/renderer/core/loader/link_loader.h
@@ -44,6 +44,7 @@ class LinkLoaderClient;
 class PrerenderHandle;
 class Resource;
 class ResourceClient;
+class HTMLLinkElement;
 
 // The LinkLoader can load link rel types icon, dns-prefetch, prefetch, and
 // prerender.
@@ -64,6 +65,7 @@ class CORE_EXPORT LinkLoader final : public SingleModuleClient {
 
   Resource* GetResourceForTesting();
 
+  void SetHTMLLinkElementOwner(HTMLLinkElement* link_element);
   void Trace(Visitor*) const override;
 
  private:
@@ -77,6 +79,7 @@ class CORE_EXPORT LinkLoader final : public SingleModuleClient {
   Member<LinkLoaderClient> client_;
 
   Member<PrerenderHandle> prerender_;
+  Member<HTMLLinkElement> link_element_;
 };
 
 }  // namespace blink
