diff --git a/third_party/blink/renderer/core/frame/dom_window.h b/third_party/blink/renderer/core/frame/dom_window.h
index cc318deb8b6be303762b0c408c33209ff7f26c43..fbd6890bed7af8ffc39c36cefc8ae07547e8dc4f 100644
--- a/third_party/blink/renderer/core/frame/dom_window.h
+++ b/third_party/blink/renderer/core/frame/dom_window.h
@@ -151,6 +151,7 @@ class CORE_EXPORT DOMWindow : public EventTargetWithInlineData {
   // marked as "CrossOrigin" in the window.idl.
   void ReportCoopAccess(const char* property_name);
 
+  BRAVE_DOM_WINDOW_H
  protected:
   explicit DOMWindow(Frame&);
 
