diff --git a/third_party/blink/renderer/core/frame/navigator_language.cc b/third_party/blink/renderer/core/frame/navigator_language.cc
index 174b1a2c0490c2fb8936f242c1048dfa04c02855..da34cf199fc395260a5646131f6aab6c8f0e4e5e 100644
--- a/third_party/blink/renderer/core/frame/navigator_language.cc
+++ b/third_party/blink/renderer/core/frame/navigator_language.cc
@@ -9,6 +9,13 @@
 #include "third_party/blink/renderer/platform/runtime_enabled_features.h"
 #include "third_party/blink/renderer/platform/wtf/text/string_builder.h"
 
+#include "brave/third_party/blink/brave_page_graph/buildflags/buildflags.h"
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+#include "brave/third_party/blink/brave_page_graph/page_graph.h"
+#include <sstream>
+#include <string>
+#endif
+
 namespace blink {
 
 Vector<String> ParseAndSanitize(const String& accept_languages) {
@@ -35,14 +42,76 @@ NavigatorLanguage::NavigatorLanguage(ExecutionContext* context)
     : context_(context) {}
 
 AtomicString NavigatorLanguage::language() {
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  String result;
+  if (RuntimeEnabledFeatures::NavigatorLanguageInInsecureContextEnabled() ||
+      context_->IsSecureContext()) {
+    // We need to make sure we don't call languages(), since that would yield an
+    // incorrect graph in the end
+    EnsureUpdatedLanguage();
+    result = languages_.front();
+  }
+
+  if (context_) {
+    brave_page_graph::PageGraph* page_graph =
+      brave_page_graph::PageGraph::GetFromIsolate(*context_->GetIsolate());
+    if (page_graph != nullptr) {
+      page_graph->RegisterWebAPICall("NavigatorLanguage.language",
+        std::vector<const String>());
+      page_graph->RegisterWebAPIResult("NavigatorLanguage.language", result);
+    }
+  }
+
+  return AtomicString(result);
+#else
   if (RuntimeEnabledFeatures::NavigatorLanguageInInsecureContextEnabled() ||
       context_->IsSecureContext()) {
     return AtomicString(languages().front());
   }
   return AtomicString();
+#endif
 }
 
 const Vector<String>& NavigatorLanguage::languages() {
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  {
+    const Vector<String>* result;
+    if (RuntimeEnabledFeatures::NavigatorLanguageInInsecureContextEnabled() ||
+        context_->IsSecureContext()) {
+      EnsureUpdatedLanguage();
+      result = &languages_;
+    } else {
+      DEFINE_STATIC_LOCAL(const Vector<String>, empty_vector, {});
+      result = &empty_vector;
+    }
+
+    if (context_) {
+      brave_page_graph::PageGraph* page_graph =
+        brave_page_graph::PageGraph::GetFromIsolate(*context_->GetIsolate());
+
+      if (page_graph != nullptr) {
+        std::stringstream result_buffer;
+        bool is_first_language = true;
+        for (auto* it = result->begin(); it != result->end(); ++it) {
+          if (is_first_language) {
+            is_first_language = false;
+          } else {
+            result_buffer << ", ";
+          }
+          result_buffer << it->Utf8().data();
+        }
+        String result_string(result_buffer.str().c_str());
+
+        page_graph->RegisterWebAPICall("NavigatorLanguage.languages",
+          std::vector<const String>());
+        page_graph->RegisterWebAPIResult("NavigatorLanguage.languages",
+          result_string);
+      }
+    }
+
+    return *result;
+  }
+#else
   if (RuntimeEnabledFeatures::NavigatorLanguageInInsecureContextEnabled() ||
       context_->IsSecureContext()) {
     EnsureUpdatedLanguage();
@@ -50,6 +119,7 @@ const Vector<String>& NavigatorLanguage::languages() {
   }
   DEFINE_STATIC_LOCAL(const Vector<String>, empty_vector, {});
   return empty_vector;
+#endif
 }
 
 AtomicString NavigatorLanguage::SerializeLanguagesForClientHintHeader() {
