diff --git a/third_party/rust/hex/v0_4/BUILD.gn b/third_party/rust/hex/v0_4/BUILD.gn
index d5172116465c3d573a5c6eaf7405bf974833ae46..bdbe4e059879e512f727bc27d058f36f3049dca9 100644
--- a/third_party/rust/hex/v0_4/BUILD.gn
+++ b/third_party/rust/hex/v0_4/BUILD.gn
@@ -34,7 +34,7 @@ cargo_crate("lib") {
     "std",
   ]
 
-  testonly = true
+  # testonly = true
 
   #####################################################################
   # Tweaking which GN `config`s apply to this target.
