diff --git a/third_party/blink/renderer/core/script_tools/model_context.cc b/third_party/blink/renderer/core/script_tools/model_context.cc
index 0586b2995e7cae94bb2cc1fd2b4c36557c4baee1..d16ba72abb455fe37306be7a90f38a0025efdb1d 100644
--- a/third_party/blink/renderer/core/script_tools/model_context.cc
+++ b/third_party/blink/renderer/core/script_tools/model_context.cc
@@ -967,7 +967,11 @@ bool ModelContext::IsModelContextAllowed() const {
               url::kFileScheme &&
           key.IsUniversalFileAgent()) ||
          (document_->GetSettings() &&
-          !document_->GetSettings()->GetWebSecurityEnabled());
+          !document_->GetSettings()->GetWebSecurityEnabled()) ||
+         (document_->GetExecutionContext()->GetSecurityOrigin()->Protocol() ==
+              "chrome-untrusted" &&
+           document_->GetExecutionContext()->GetSecurityOrigin()->Host() ==
+              ai_chat::kAIChatLeoWorkspaceUIHost);
 }
 
 ScriptPromise<IDLSequence<RegisteredTool>> ModelContext::getTools(
