diff --git a/third_party/rust/getrandom/v0_2/BUILD.gn b/third_party/rust/getrandom/v0_2/BUILD.gn
index 29d2813377c200ede86cc43dd2137de8d8bb23fb..7e84aee6c7dc00a76400a62778a718ae36d86ffa 100644
--- a/third_party/rust/getrandom/v0_2/BUILD.gn
+++ b/third_party/rust/getrandom/v0_2/BUILD.gn
@@ -68,5 +68,4 @@ cargo_crate("lib") {
   # //third_party/rust/chromium_crates_io/Cargo.toml to use
   # it from first-party code.
   visibility = [ "//third_party/rust/*" ]
-  testonly = true
 }
