diff --git a/third_party/android_deps/BUILD.gn b/third_party/android_deps/BUILD.gn
index 66998181e74f8ccb295761a86f5be4fdde47d308..cdf2a2b12b17792029beee1bb9cc29a090d8bf2b 100644
--- a/third_party/android_deps/BUILD.gn
+++ b/third_party/android_deps/BUILD.gn
@@ -465,6 +465,7 @@ java_prebuilt("com_google_auto_value_auto_value_annotations_java") {
     ":*",
     "//third_party/androidx:*",
   ]
+  visibility += [ "//brave/third_party/android_deps:com_google_android_datatransport_transport_runtime_java__classes" ] supports_android = true
 }
 
 # This is generated, do not edit. Update BuildConfigGenerator.groovy instead.
