diff --git a/third_party/WebKit/Source/modules/canvas/canvas2d/CanvasRenderingContext2D.h b/third_party/WebKit/Source/modules/canvas/canvas2d/CanvasRenderingContext2D.h
index 1fface0bbb6ee9bd97ab49a0e0577a84a5228b5d..04e72634a50f13d97bc89319e7b319fb612e8da2 100644
--- a/third_party/WebKit/Source/modules/canvas/canvas2d/CanvasRenderingContext2D.h
+++ b/third_party/WebKit/Source/modules/canvas/canvas2d/CanvasRenderingContext2D.h
@@ -124,6 +124,19 @@ class MODULES_EXPORT CanvasRenderingContext2D final
   void strokeText(const String& text, double x, double y, double max_width);
   TextMetrics* measureText(const String& text);
 
+  bool isPointInPath(const double x,
+                     const double y,
+                     const String& winding = "nonzero");
+  bool isPointInPath(Path2D*,
+                     const double x,
+                     const double y,
+                     const String& winding = "nonzero");
+  bool isPointInStroke(const double x, const double y);
+  bool isPointInStroke(Path2D*, const double x, const double y);
+  ImageData* getImageData(int sx, int sy, int sw, int sh, ExceptionState&);
+  const Vector<double>& getLineDash() const;
+
+
   void getContextAttributes(CanvasRenderingContext2DSettings&) const;
 
   void drawFocusIfNeeded(Element*);
