diff --git a/third_party/blink/renderer/core/loader/mixed_content_checker.h b/third_party/blink/renderer/core/loader/mixed_content_checker.h
index ae26697491949cc06b7efb036d0ea9cbe9740e87..a6da7314b6695e77297b6f5da192324acff5b830 100644
--- a/third_party/blink/renderer/core/loader/mixed_content_checker.h
+++ b/third_party/blink/renderer/core/loader/mixed_content_checker.h
@@ -99,7 +99,6 @@ class CORE_EXPORT MixedContentChecker final {
   static bool IsWebSocketAllowed(WorkerFetchContext&, const KURL&);
 
   static bool IsMixedContent(const SecurityOrigin*, const KURL&);
-  static bool IsMixedContent(const String& origin_protocol, const KURL&);
   static bool IsMixedContent(const FetchClientSettingsObject&, const KURL&);
   static bool IsMixedFormAction(
       LocalFrame*,
