diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index 9ec2eadd6b4694252e4329922d429ffc239332fa..7e0466aa456fa5574f930371edbba114a8a8a9ae 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -168,6 +168,7 @@ component("core") {
 
   visibility = []  # Allow re-assignment of list.
   visibility = [ "//third_party/blink/*" ]
+  visibility += brave_blink_renderer_core_visibility
 
   sources = [
     "core_initializer.cc",
@@ -275,6 +276,7 @@ component("core") {
     "//v8",
   ]
 
+  public_deps += brave_blink_renderer_core_public_deps
   deps = [
     ":generated_settings_macros",
     "//build:chromeos_buildflags",
