diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index cc5e66ff6b461ecd90be39e00cea606a8a5785b3..d339306049f647b6160b66e1d56dd8fb9eeaf065 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -470,6 +470,7 @@ component("core") {
       "Carbon.framework",
     ]
   }
+  visibility += brave_blink_renderer_core_visibility public_deps += brave_blink_renderer_core_public_deps sources += brave_blink_renderer_core_sources deps += brave_blink_renderer_core_deps
 }
 
 # Certain sources should be compiled with heavy optimization enabled.
