diff --git a/third_party/blink/renderer/core/svg/svg_path_element.cc b/third_party/blink/renderer/core/svg/svg_path_element.cc
index 1f96415490b16350aa4a1dbc0714ebd27055fcc4..f06b02ec87e16714f25604659fc9577ed9703bc6 100644
--- a/third_party/blink/renderer/core/svg/svg_path_element.cc
+++ b/third_party/blink/renderer/core/svg/svg_path_element.cc
@@ -20,6 +20,7 @@
 
 #include "third_party/blink/renderer/core/svg/svg_path_element.h"
 
+#include "brave/renderer/brave_content_settings_observer_helper.h"
 #include "third_party/blink/renderer/core/layout/layout_object.h"
 #include "third_party/blink/renderer/core/svg/svg_mpath_element.h"
 #include "third_party/blink/renderer/core/svg/svg_path_query.h"
@@ -64,6 +65,9 @@ Path SVGPathElement::AsPath() const {
 }
 
 float SVGPathElement::getTotalLength() {
+  if (!AllowFingerprinting(GetDocument().GetFrame())) {
+    return 0.0f;
+  }
   GetDocument().UpdateStyleAndLayoutIgnorePendingStylesheets();
   return SVGPathQuery(PathByteStream()).GetTotalLength();
 }
