diff --git a/third_party/blink/renderer/modules/clipboard/clipboard_promise.cc b/third_party/blink/renderer/modules/clipboard/clipboard_promise.cc
index 92b72c553d8966da6d8a4a4ae2517d6fc7271f12..632622148a9e56ce42e8f5e3f2033261c35ded6c 100644
--- a/third_party/blink/renderer/modules/clipboard/clipboard_promise.cc
+++ b/third_party/blink/renderer/modules/clipboard/clipboard_promise.cc
@@ -588,6 +588,7 @@ void ClipboardPromise::RequestPermission(
   // TODO(ansollan): Remove this block as custom formats don't need both a user
   // gesture and a permission grant to use custom clipboard.
   if (!will_be_sanitized &&
+      BRAVE_CLIPBOARD_PROMISE_REQUEST_PERMISSION
       RuntimeEnabledFeatures::ClipboardCustomFormatsEnabled() &&
       !has_transient_user_activation) {
     script_promise_resolver_->Reject(MakeGarbageCollected<DOMException>(
