diff --git a/third_party/crashpad/crashpad/client/settings.cc b/third_party/crashpad/crashpad/client/settings.cc
index 0aa525f1b4c004f8de2145d15215ff368cfecef8..7d12accc502154fdfb9b432b770230ae01e2eb58 100644
--- a/third_party/crashpad/crashpad/client/settings.cc
+++ b/third_party/crashpad/crashpad/client/settings.cc
@@ -131,6 +131,7 @@ bool Settings::GetClientID(UUID* client_id) {
   if (!OpenAndReadSettings(&settings))
     return false;
 
+  settings.client_id.InitializeToZero();
   *client_id = settings.client_id;
   return true;
 }
