diff --git a/third_party/blink/renderer/modules/encryptedmedia/navigator_request_media_key_system_access.cc b/third_party/blink/renderer/modules/encryptedmedia/navigator_request_media_key_system_access.cc
index e4b71e783c2e46c5cff30fadc862315c952f6719..57a0d3e72e039cfa110bd6c7d7e8dd2178287d0a 100644
--- a/third_party/blink/renderer/modules/encryptedmedia/navigator_request_media_key_system_access.cc
+++ b/third_party/blink/renderer/modules/encryptedmedia/navigator_request_media_key_system_access.cc
@@ -176,6 +176,9 @@ ScriptPromise NavigatorRequestMediaKeySystemAccess::requestMediaKeySystemAccess(
   media_client->RequestMediaKeySystemAccess(
       WebEncryptedMediaRequest(initializer));
 
+  // Notify Brave about the widevine availability request.
+  MaybeOnWidevineRequest(initializer, document->GetFrame());
+
   // 7. Return promise.
   return promise;
 }
