diff --git a/third_party/libaddressinput/chromium/chrome_metadata_source.cc b/third_party/libaddressinput/chromium/chrome_metadata_source.cc
index 02f74d5d0a46bbccd4b9b533fce49666ae126c7e..3a2b997448d3c79fce78e52dc1429c1644287c19 100644
--- a/third_party/libaddressinput/chromium/chrome_metadata_source.cc
+++ b/third_party/libaddressinput/chromium/chrome_metadata_source.cc
@@ -58,7 +58,7 @@ ChromeMetadataSource::Request::Request(
 void ChromeMetadataSource::Download(const std::string& key,
                                     const Callback& downloaded) {
   GURL resource(validation_data_url_ + key);
-  if (!resource.SchemeIsCryptographic()) {
+  if (1) { // feature disabled in Brave
     downloaded(false, key, NULL);
     return;
   }
