diff --git a/third_party/blink/renderer/platform/bindings/idl_member_installer.h b/third_party/blink/renderer/platform/bindings/idl_member_installer.h
index b9bff532235ecb48867e2efeca504a0264974a33..6868c1274bb7f75925e42180660c3a40dd902321 100644
--- a/third_party/blink/renderer/platform/bindings/idl_member_installer.h
+++ b/third_party/blink/renderer/platform/bindings/idl_member_installer.h
@@ -212,6 +212,7 @@ class PLATFORM_EXPORT IDLMemberInstaller final {
       v8::Local<v8::Object> interface_object,
       v8::Local<v8::Signature> signature,
       base::span<const ExposedConstructConfig> configs);
+  BRAVE_IDL_MEMBER_INSTALLER_H_
 };
 
 }  // namespace bindings
