diff --git a/third_party/blink/renderer/bindings/core/v8/js_event_handler_for_content_attribute.cc b/third_party/blink/renderer/bindings/core/v8/js_event_handler_for_content_attribute.cc
index aa4d6d460259bef616cb75db864004a01019b274..428c81ca121c4fd2cda3f401df2068f3084f36d8 100644
--- a/third_party/blink/renderer/bindings/core/v8/js_event_handler_for_content_attribute.cc
+++ b/third_party/blink/renderer/bindings/core/v8/js_event_handler_for_content_attribute.cc
@@ -234,6 +234,7 @@ v8::Local<v8::Value> JSEventHandlerForContentAttribute::GetCompiledHandler(
   // and whose callback context is settings object.
   compiled_function->SetName(V8String(isolate, function_name_));
   SetCompiledHandler(script_state_of_event_target, compiled_function);
+  BRAVE_JS_EVENT_HANDLER_FOR_CONTENT_ATTRIBUTE_GET_COMPILED_HANDLER
 
   return JSEventHandler::GetListenerObject(event_target);
 }
