diff --git a/third_party/blink/renderer/core/frame/navigator.cc b/third_party/blink/renderer/core/frame/navigator.cc
index a1ca9b10d8896dd3cfa035191224e886c977d6bf..1a202b6fc6cf873bd88dd46f7ea53262f8d135eb 100644
--- a/third_party/blink/renderer/core/frame/navigator.cc
+++ b/third_party/blink/renderer/core/frame/navigator.cc
@@ -74,6 +74,7 @@ String Navigator::userAgent() const {
   if (!DomWindow())
     return String();
 
+  BRAVE_NAVIGATOR_USERAGENT
   return DomWindow()->GetFrame()->Loader().UserAgent();
 }
 
