diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index c458aeb51d09feb929abe97944e6cd47abbb95c6..3960f989e744a41ccf1e821ad79bdedab93063f6 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -1344,6 +1344,7 @@ void MediaDevices::DevicesEnumerated(
     }
   }
 
+  BRAVE_MEDIA_DEVICES_DEVICES_ENUMERATED
   ReportCompletedEnumerateDevices(result_contains_nonempty_input_device_ids);
   result_tracker->Resolve(media_devices);
   tracer->End();
