diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index 9fae347c8b156aa5a7c4e1bdb8a38a61161b2d80..f34f0e0c1c57536e92c6e2ee3a7388abde4fe3b3 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -907,6 +907,7 @@ void MediaDevices::DevicesEnumerated(
     }
   }
 
+  BRAVE_MEDIA_DEVICES_DEVICES_ENUMERATED
   RecordEnumeratedDevices(result_tracker->GetScriptState(), media_devices);
   // TODO(crbug.com/1395324): Remove this custom EnumerateDevices latency
   // tracking by reverting crrev.com/c/3944912/ once the
