diff --git a/third_party/blink/renderer/modules/modules_idl_files.gni b/third_party/blink/renderer/modules/modules_idl_files.gni
index 7544f660af9db9fb1706b32f5f044fb51bb28c3c..e17dacdef2f35ce713bca1a1b3b0b0e9487500a2 100644
--- a/third_party/blink/renderer/modules/modules_idl_files.gni
+++ b/third_party/blink/renderer/modules/modules_idl_files.gni
@@ -136,6 +136,7 @@ _idl_imports = [
 if (!is_android) {
   _idl_imports += [ "//third_party/blink/renderer/modules/serial/idls.gni" ]
 }
+_idl_imports += brave_idl_imports
 
 # Do not add IDL files directly to these lists. See _idl_imports above instead
 # on how to add IDL files for a module.
