diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index db5e7969682e4370ab9ac5a0765bfada64e74418..ed507ea81956444c00042d9db0f521d0d1030567 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -865,6 +865,11 @@
       public: true,
       status: "stable",
     },
+    {
+      name: "BraveIsInTorContext",
+      public: true,
+      base_feature: "none",
+    },
     {
       name: "BrowserInitiatedAutomaticPictureInPicture",
       public: true,
@@ -2593,6 +2598,7 @@
       // In-development features for the File System Access API.
       name: "FileSystemAccessAPIExperimental",
       status: "experimental",
+      base_feature: "none",
     },
     {
       // The FileSystemHandle.getCloudIdentifiers() method (see
@@ -2607,6 +2613,7 @@
       // Non-OPFS File System Access API.
       name: "FileSystemAccessLocal",
       status: {"default": "stable"},
+      base_feature: "none",
     },
     {
       name: "FileSystemAccessLockingScheme",
@@ -3677,6 +3684,7 @@
     {
       name: "MiddleClickAutoscroll",
       status: "test",
+      base_feature: "none",
     },
     // Killswitch for crbug.com/40062462 fix.
     {
