diff --git a/third_party/blink/renderer/modules/cookie_store/cookie_store.cc b/third_party/blink/renderer/modules/cookie_store/cookie_store.cc
index 5639449872e9ad1934ca13724a20986d013c9af7..ff5e71ea150868b1953d921059c19ccc9283e167 100644
--- a/third_party/blink/renderer/modules/cookie_store/cookie_store.cc
+++ b/third_party/blink/renderer/modules/cookie_store/cookie_store.cc
@@ -81,6 +81,7 @@ std::unique_ptr<net::CanonicalCookie> ToCanonicalCookie(
   base::Time expires = options->hasExpiresNonNull()
                            ? base::Time::FromJavaTime(options->expiresNonNull())
                            : base::Time();
+  BRAVE_TO_CANONICAL_COOKIE
 
   String cookie_url_host = cookie_url.Host();
   String domain;
