diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index cea00d857ff2f5857262be9a6d3af986ff737fb2..88f55aeba69bab04bfe4dcd2103f480a916630ee 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -223,6 +223,7 @@ component("core") {
     "//v8",
   ]
   deps = [
+    "//brave/third_party/blink/brave_page_graph",
     "//third_party/blink/public/common",
     "//third_party/blink/renderer/platform",
     "//third_party/blink/renderer/platform/wtf",
