diff --git a/third_party/blink/renderer/core/script/dynamic_module_resolver.cc b/third_party/blink/renderer/core/script/dynamic_module_resolver.cc
index c1594f238ea0513f20039a6b3ae47f330200fb80..34d41aba1b1bbc4f9bee9ba1d23f3c446b1e9374 100644
--- a/third_party/blink/renderer/core/script/dynamic_module_resolver.cc
+++ b/third_party/blink/renderer/core/script/dynamic_module_resolver.cc
@@ -288,6 +288,7 @@ void DynamicModuleResolver::ResolveDynamically(
       referrer_info.CredentialsMode(), referrer_info.GetReferrerPolicy(),
       mojom::blink::FetchPriorityHint::kAuto,
       RenderBlockingBehavior::kNonBlocking);
+  BRAVE_DYNAMIC_MODULE_RESOLVER_RESOLVE_DYNAMICALLY
 
   // <spec label="fetch-an-import()-module-script-graph" step="3">Fetch a single
   // module script given url, settings object, "script", options, settings
