diff --git a/third_party/blink/renderer/modules/permissions/BUILD.gn b/third_party/blink/renderer/modules/permissions/BUILD.gn
index 2e4b14e09755ff0194454863c4de36e99a290253..f50482466991dc87a15c829dd728cc09fecc4e07 100644
--- a/third_party/blink/renderer/modules/permissions/BUILD.gn
+++ b/third_party/blink/renderer/modules/permissions/BUILD.gn
@@ -15,4 +15,5 @@ blink_modules_sources("permissions") {
     "permissions.cc",
     "permissions.h",
   ]
+  import("//brave/components/brave_wallet/sources.gni") deps = brave_blink_renderer_modules_permissions_deps
 }
