diff --git a/third_party/blink/renderer/core/workers/worker_or_worklet_global_scope.cc b/third_party/blink/renderer/core/workers/worker_or_worklet_global_scope.cc
index 3816f71ffad48386e7a1f25f9764e2ced327a42b..15fb0aedfc7cdeeedfadad666cb6e8dafb306ed9 100644
--- a/third_party/blink/renderer/core/workers/worker_or_worklet_global_scope.cc
+++ b/third_party/blink/renderer/core/workers/worker_or_worklet_global_scope.cc
@@ -499,7 +499,7 @@ void WorkerOrWorkletGlobalScope::FetchModuleScript(
       mojom::blink::FetchImportanceMode::kImportanceAuto,
       RenderBlockingBehavior::kNonBlocking, reject_coep_unsafe_none);
 
-  Modulator* modulator = Modulator::From(ScriptController()->GetScriptState());
+  Modulator* modulator = Modulator::From(ScriptController()->GetScriptState(), nullptr, 0);
   // Step 3. "Perform the internal module script graph fetching procedure ..."
   modulator->FetchTree(
       module_url_record, ModuleType::kJavaScript,
