diff --git a/third_party/blink/renderer/core/style/style_fetched_image.cc b/third_party/blink/renderer/core/style/style_fetched_image.cc
index f6586b48641c05dec3e2171ab1c0131c9af0c6f7..07f05edc51ef9823a2e3329fa05a81c8eeb1b0d7 100644
--- a/third_party/blink/renderer/core/style/style_fetched_image.cc
+++ b/third_party/blink/renderer/core/style/style_fetched_image.cc
@@ -36,6 +36,16 @@
 #include "third_party/blink/renderer/platform/geometry/layout_size.h"
 #include "third_party/blink/renderer/platform/graphics/placeholder_image.h"
 
+#include "brave/third_party/blink/brave_page_graph/buildflags/buildflags.h"
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+#include <sstream>
+#include "third_party/blink/renderer/core/dom/dom_node_ids.h"
+#include "brave/third_party/blink/brave_page_graph/types.h"
+#include "brave/third_party/blink/brave_page_graph/page_graph.h"
+#include "brave/third_party/blink/brave_page_graph/utilities/digest.h"
+#include "brave/third_party/blink/brave_page_graph/utilities/response_metadata.h"
+#endif
+
 namespace blink {
 
 StyleFetchedImage::StyleFetchedImage(const Document& document,
@@ -45,7 +55,21 @@ StyleFetchedImage::StyleFetchedImage(const Document& document,
   is_image_resource_ = true;
   is_lazyload_possibly_deferred_ = is_lazyload_possibly_deferred;
 
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  image_ = ImageResourceContent::Fetch(params,
+      document_->Fetcher(), &inspector_id_);
+  {
+    Document* mutable_document = (Document*)(document_.Get());
+    ::brave_page_graph::PageGraph* page_graph =
+        mutable_document->GetPageGraph();
+    if (page_graph != nullptr) {
+      page_graph->RegisterRequestStartFromCSS(inspector_id_, params.Url(),
+          ::brave_page_graph::RequestType::kRequestTypeImage);
+    }
+  }
+#else
   image_ = ImageResourceContent::Fetch(params, document_->Fetcher());
+#endif
   image_->AddObserver(this);
   // ResourceFetcher is not determined from StyleFetchedImage and it is
   // impossible to send a request for refetching.
@@ -126,7 +150,26 @@ void StyleFetchedImage::RemoveClient(ImageResourceObserver* observer) {
   image_->RemoveObserver(observer);
 }
 
-void StyleFetchedImage::ImageNotifyFinished(ImageResourceContent*) {
+void StyleFetchedImage::ImageNotifyFinished(ImageResourceContent* resource) {
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  {
+    Document* mutable_document = (Document*)(document_.Get());
+    ::brave_page_graph::PageGraph* page_graph =
+        mutable_document->GetPageGraph();
+    if (page_graph != nullptr) {
+      const ResourceResponse& rep = resource->GetResponse();
+      const ::brave_page_graph::ResponseMetadata metadata(rep);
+
+      if (resource->ErrorOccurred()) {
+        page_graph->RegisterRequestError(inspector_id_, metadata);
+      } else {
+        page_graph->RegisterRequestComplete(inspector_id_,
+            ResourceType::kImage, metadata,
+            brave_page_graph::ImageDigest(resource->GetImage()));
+      }
+    }
+  }
+#endif
   if (image_ && image_->HasImage()) {
     Image& image = *image_->GetImage();
 
