diff --git a/third_party/blink/renderer/modules/cookie_store/cookie_store.cc b/third_party/blink/renderer/modules/cookie_store/cookie_store.cc
index f897df3d0fc43e713387ea9f59c6bbdbf2b75e61..ed5e091fea31627fd208dcb535624c08466a0240 100644
--- a/third_party/blink/renderer/modules/cookie_store/cookie_store.cc
+++ b/third_party/blink/renderer/modules/cookie_store/cookie_store.cc
@@ -83,6 +83,7 @@ std::unique_ptr<net::CanonicalCookie> ToCanonicalCookie(
   base::Time expires = options->hasExpiresNonNull()
                            ? base::Time::FromJavaTime(options->expiresNonNull())
                            : base::Time();
+  BRAVE_TO_CANONICAL_COOKIE
 
   String cookie_url_host = cookie_url.Host();
   String domain;
