diff --git a/third_party/blink/renderer/bindings/idl_in_modules.gni b/third_party/blink/renderer/bindings/idl_in_modules.gni
index f989217e0cfb51c382d22dd12e8d6bbdcf44ea4b..136464dd4b0a0327ca875db41b68f84b7ec51951 100644
--- a/third_party/blink/renderer/bindings/idl_in_modules.gni
+++ b/third_party/blink/renderer/bindings/idl_in_modules.gni
@@ -1279,6 +1279,7 @@ static_idl_files_in_modules = [
   "//third_party/blink/renderer/modules/xr/xr_webgl_layer_init.idl",
   "//third_party/blink/renderer/modules/xr/xr_webgl_sub_image.idl",
 ]
+static_idl_files_in_modules += brave_static_idl_files_in_modules
 
 # Direct Sockets
 # This uses target_os rather than current_os (which is what is_android is set
