diff --git a/third_party/blink/renderer/core/frame/local_dom_window.cc b/third_party/blink/renderer/core/frame/local_dom_window.cc
index 6fe961fdc8961b491d609f75a773e061739c0157..295b6ce733a98883b51bc12513fe20b3ec6a3372 100644
--- a/third_party/blink/renderer/core/frame/local_dom_window.cc
+++ b/third_party/blink/renderer/core/frame/local_dom_window.cc
@@ -491,6 +491,7 @@ bool LocalDOMWindow::CanExecuteScripts(
     return false;
   }
   bool script_enabled = GetFrame()->ScriptEnabled();
+  BRAVE_LOCAL_DOM_WINDOW_CAN_EXECUTE_SCRIPTS
   if (!script_enabled && reason == kAboutToExecuteScript) {
     WebContentSettingsClient* settings_client =
         GetFrame()->GetContentSettingsClient();
