diff --git a/third_party/blink/renderer/modules/webgl/webgl2_rendering_context_base.cc b/third_party/blink/renderer/modules/webgl/webgl2_rendering_context_base.cc
index e6aff8f7486035c3bb1a5bb0fdd443c676498a06..fd645a46a31a81f030f7b107947964ae710a9cd2 100644
--- a/third_party/blink/renderer/modules/webgl/webgl2_rendering_context_base.cc
+++ b/third_party/blink/renderer/modules/webgl/webgl2_rendering_context_base.cc
@@ -4796,6 +4796,7 @@ ScriptValue WebGL2RenderingContextBase::getParameter(ScriptState* script_state,
                                                      GLenum pname) {
   if (isContextLost())
     return ScriptValue::CreateNull(script_state->GetIsolate());
+  BRAVE_WEBGL2_RENDERING_CONTEXT_BASE_GETPARAMETER
   switch (pname) {
     case GL_SHADING_LANGUAGE_VERSION: {
       return WebGLAny(
@@ -5406,6 +5407,7 @@ ScriptValue WebGL2RenderingContextBase::getFramebufferAttachmentParameter(
   if (isContextLost() || !ValidateGetFramebufferAttachmentParameterFunc(
                              kFunctionName, target, attachment))
     return ScriptValue::CreateNull(script_state->GetIsolate());
+  BRAVE_WEBGL2_RENDERING_CONTEXT_BASE
 
   WebGLFramebuffer* framebuffer_binding = GetFramebufferBinding(target);
   DCHECK(!framebuffer_binding || framebuffer_binding->Object());
@@ -5624,6 +5626,7 @@ ScriptValue WebGL2RenderingContextBase::getTexParameter(
     GLenum pname) {
   if (isContextLost() || !ValidateTextureBinding("getTexParameter", target))
     return ScriptValue::CreateNull(script_state->GetIsolate());
+  BRAVE_WEBGL2_RENDERING_CONTEXT_BASE
 
   switch (pname) {
     case GL_TEXTURE_WRAP_R:
