diff --git a/third_party/blink/renderer/core/execution_context/execution_context.h b/third_party/blink/renderer/core/execution_context/execution_context.h
index 1d88aa6f72faa050bbe8bfcf43556bd66fd37701..0ea94ccae2658f2494faf878fdd0c42dadcbcad9 100644
--- a/third_party/blink/renderer/core/execution_context/execution_context.h
+++ b/third_party/blink/renderer/core/execution_context/execution_context.h
@@ -127,7 +127,7 @@ enum ReferrerPolicySource { kPolicySourceHttpHeader, kPolicySourceMetaTag };
 // by an extension developer, but these share an ExecutionContext (the window)
 // in common.
 class CORE_EXPORT ExecutionContext
-    : public Supplementable<ExecutionContext, 34>,
+    : public Supplementable<ExecutionContext, 35>,
       public MojoBindingContext,
       public UseCounterAndConsoleLogger,
       public FeatureContext {
@@ -167,6 +167,7 @@ class CORE_EXPORT ExecutionContext
     kExecutionContextClipboardEventState = 31,
     kCachedVideoFramePool = 32,
     kCanvasResourceProviderCache = 33
+    , kBraveSessionCache,
   };
 
   ExecutionContext(const ExecutionContext&) = delete;
