diff --git a/third_party/blink/public/common/BUILD.gn b/third_party/blink/public/common/BUILD.gn
index 8fd5a0337f247..d7a5cc92a6ec6 100644
--- a/third_party/blink/public/common/BUILD.gn
+++ b/third_party/blink/public/common/BUILD.gn
@@ -88,6 +88,7 @@ buildflag_header("buildflags") {
     "RTC_USE_H264=$rtc_use_h264",
     "RTC_USE_H265=$rtc_use_h265",
     "ENABLE_RUST_CRASH=$enable_rust_crash",
+    "ENABLE_JXL_DECODER=$enable_jxl_decoder",
   ]
 }
