diff --git a/third_party/blink/renderer/core/frame/navigator_ua.cc b/third_party/blink/renderer/core/frame/navigator_ua.cc
index d334ce3951aff6746fefdaa3857e9e99d6e7cb0b..b7dbea24865f5ff486751f7d3bc9aad19c3a49b7 100644
--- a/third_party/blink/renderer/core/frame/navigator_ua.cc
+++ b/third_party/blink/renderer/core/frame/navigator_ua.cc
@@ -14,7 +14,8 @@ NavigatorUAData* NavigatorUA::userAgentData() {
   NavigatorUAData* ua_data =
       MakeGarbageCollected<NavigatorUAData>(GetUAExecutionContext());
 
-  UserAgentMetadata metadata = GetUserAgentMetadata();
+  UserAgentMetadata metadata = brave::MaybeHideBraveBrand(
+      GetUAExecutionContext(), GetUserAgentMetadata());
   ua_data->SetBrandVersionList(metadata.brand_version_list);
   ua_data->SetMobile(metadata.mobile);
   ua_data->SetPlatform(String::FromUtf8(metadata.platform),
