diff --git a/third_party/blink/renderer/core/frame/cached_permission_status.cc b/third_party/blink/renderer/core/frame/cached_permission_status.cc
index a713098c66e768d4a4561d9a765175363753bedf..719a8748b28af27dcb2b2889a34518837154c7c4 100644
--- a/third_party/blink/renderer/core/frame/cached_permission_status.cc
+++ b/third_party/blink/renderer/core/frame/cached_permission_status.cc
@@ -15,7 +15,6 @@ using mojom::blink::PermissionDescriptorPtr;
 using mojom::blink::PermissionName;
 using mojom::blink::PermissionObserver;
 using mojom::blink::PermissionService;
-using mojom::blink::PermissionStatus;
 
 // static
 const char CachedPermissionStatus::kSupplementName[] = "CachedPermissionStatus";
@@ -49,7 +48,7 @@ void CachedPermissionStatus::Trace(Visitor* visitor) const {
 void CachedPermissionStatus::RegisterClient(
     Client* client,
     const Vector<PermissionDescriptorPtr>& permissions) {
-  HashMap<mojom::blink::PermissionName, mojom::blink::PermissionStatus>
+  HashMap<mojom::blink::PermissionName, PermissionStatus>
       initialized_map;
   for (const PermissionDescriptorPtr& descriptor : permissions) {
     auto status_it = permission_status_map_.find(descriptor->name);
