diff --git a/third_party/blink/renderer/bindings/BUILD.gn b/third_party/blink/renderer/bindings/BUILD.gn
index c11ab5aa4a70d9a3303189193fd8f8453139d018..66298e1a9497f457df816f579de528012b4d16b2 100644
--- a/third_party/blink/renderer/bindings/BUILD.gn
+++ b/third_party/blink/renderer/bindings/BUILD.gn
@@ -202,6 +202,7 @@ template("generate_bindings") {
     args += invoker.targets
 
     deps = [ ":web_idl_database" ]
+    import("//brave/third_party/blink/renderer/bindings/sources.gni") inputs += brave_third_party_blink_renderer_bindings_generate_bindings_inputs deps += brave_third_party_blink_renderer_bindings_generate_bindings_deps
   }
 }
 
