diff --git a/third_party/blink/renderer/modules/cookie_store/cookie_store.h b/third_party/blink/renderer/modules/cookie_store/cookie_store.h
index 3bb8eed6dc10d066b6233b928f81fc63b15fa832..f0556796f3fbf5fec3ffcf5c197f76ce560502a3 100644
--- a/third_party/blink/renderer/modules/cookie_store/cookie_store.h
+++ b/third_party/blink/renderer/modules/cookie_store/cookie_store.h
@@ -169,6 +169,9 @@ class CookieStore final : public EventTargetWithInlineData,
 
   // The RFC 6265bis "site for cookies" for this store's ExecutionContext.
   const KURL default_site_for_cookies_;
+
+  // The context in which cookies are accessed.
+  const scoped_refptr<SecurityOrigin> default_top_frame_origin_;
 };
 
 }  // namespace blink
