diff --git a/third_party/blink/renderer/modules/peerconnection/BUILD.gn b/third_party/blink/renderer/modules/peerconnection/BUILD.gn
index d3b5f5b924bfd94c9e4f5b9daabb9ef64de03823..5a245b3c2ca6ed9eb8610296b3a1116abffbefd1 100644
--- a/third_party/blink/renderer/modules/peerconnection/BUILD.gn
+++ b/third_party/blink/renderer/modules/peerconnection/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/components/ai_chat/sources.gni")  # Brave
 import("//third_party/blink/renderer/modules/modules.gni")
 
 blink_modules_sources("peerconnection") {
@@ -193,6 +194,7 @@ blink_modules_sources("peerconnection") {
     "//third_party/blink/renderer/modules/permissions",
     "//third_party/blink/renderer/modules/webrtc",
   ]
+  deps += brave_blink_renderer_modules_peerconnection_ai_chat_deps  # Brave
 
   allow_circular_includes_from =
       [ "//third_party/blink/renderer/modules/mediastream" ]
