diff --git a/tools/typescript/ts_library.py b/tools/typescript/ts_library.py
index 12356590b5cf090786beb509a17c1d8924b7bd24..bd60a29a9fa66b13ca84006a5189d155f1b057fc 100644
--- a/tools/typescript/ts_library.py
+++ b/tools/typescript/ts_library.py
@@ -264,5 +264,6 @@ def main(argv):
       json.dump(manifest_data, manifest_file)
 
 
+from import_inline import inline_file_from_src; inline_file_from_src("brave/chromium_src/tools/typescript/ts_library.py", globals(), locals())
 if __name__ == '__main__':
   main(sys.argv[1:])
