diff --git a/tools/clang/scripts/update.py b/tools/clang/scripts/update.py
index adbdba66a359f61d695ac5cfa33c257b3b8c4db0..4ac1511de9bc0bb305ae3d07cfa3cd303106b483 100755
--- a/tools/clang/scripts/update.py
+++ b/tools/clang/scripts/update.py
@@ -202,6 +202,8 @@ def DownloadAndUnpack(url, output_dir, path_prefixes=None, is_known_zip=False):
       # The nicest way to do this would be by passing a filter to extractall,
       # but that functionality is not available in macOS system Python (3.9.6).
       for m in members:
+        if sys.platform == 'win32':
+          continue
         # Confusingly, this checks if you're allowed to _not_ follow symlinks.
         if os.utime in os.supports_follow_symlinks:
           os.utime(os.path.join(output_dir, m.name), follow_symlinks=False)
