diff --git a/tools/gritsettings/resource_ids.spec b/tools/gritsettings/resource_ids.spec
index 342b54311138a3d156bc19e8428e79d6a1477b80..399c7e4d354e7c1243e12208d65a0289a20e5cb2 100644
--- a/tools/gritsettings/resource_ids.spec
+++ b/tools/gritsettings/resource_ids.spec
@@ -38,6 +38,9 @@
   "chrome/app/chromium_strings.grd": {
     "messages": [800],
   },
+  "chrome/app/brave_strings.grd": {
+    "messages": [800],
+  },
   "chrome/app/google_chrome_strings.grd": {
     "messages": [800],
   },
@@ -405,7 +408,7 @@
     "includes": [3900],
   },
   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/extensions/resources.grd": {
-    "META": {"sizes": {"includes": [120],}},
+    "META": {"sizes": {"includes": [123],}},
     "includes": [3920],
   },
   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/family_link_user_internals/resources.grd": {
@@ -581,7 +584,7 @@
     "includes": [4720],
   },
   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/settings/resources.grd": {
-    "META": {"sizes": {"includes": [500],}},
+    "META": {"sizes": {"includes": [650],}},
     "includes": [4740],
   },
   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/settings_shared/resources.grd": {
@@ -979,6 +982,9 @@
     # Big alignment at start of section.
     "messages": [7020],
   },
+  "components/components_brave_strings.grd": {
+    "messages": [7020],
+  },
   "components/components_google_chrome_strings.grd": {
     "messages": [7020],
   },
@@ -1479,11 +1485,11 @@
     "includes": [9980],
   },
   "<(SHARED_INTERMEDIATE_DIR)/ui/webui/resources/webui_code_cache_resources.grd": {
-    "META": {"sizes": {"includes": [250]}},
+    "META": {"sizes": {"includes": [252]}},
     "includes": [9990],
   },
   "<(SHARED_INTERMEDIATE_DIR)/ui/webui/resources/webui_resources.grd": {
-    "META": {"sizes": {"includes": [1100]}},
+    "META": {"sizes": {"includes": [1137]}},
     "includes": [10000],
   },
   "weblayer/weblayer_resources.grd": {
