diff --git a/tools/typescript/ts_library.py b/tools/typescript/ts_library.py
index cb7b9a22e691aaffe19a810c5aed296bd1039d82..9e680720c97689c23862c7758f0e290c402b2946 100644
--- a/tools/typescript/ts_library.py
+++ b/tools/typescript/ts_library.py
@@ -280,5 +280,6 @@ def main(argv):
       json.dump(manifest_data, manifest_file)
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
   main(sys.argv[1:])
