diff --git a/tools/crates/gnrt/gen.rs b/tools/crates/gnrt/gen.rs
index 0bbabb4f770cb34387e633f686c2623002fa9bd1..298990185fd8c1a36b4fb5361dcd8febcdefecd3 100644
--- a/tools/crates/gnrt/gen.rs
+++ b/tools/crates/gnrt/gen.rs
@@ -272,6 +272,7 @@ fn generate_for_third_party(args: GenCommandArgs, paths: &paths::ChromiumPaths)
     // Remove any excluded dep entries.
     dependencies
         .retain(|dep| !config.resolve.remove_crates.iter().any(|r| **r == dep.package_name));
+    dependencies.retain(|dep| !dep.is_local );
 
     // Remove dev dependencies since tests aren't run.
     dependencies.retain(|dep| {
