diff --git a/tools/typescript/ts_library.gni b/tools/typescript/ts_library.gni
index 74ca9154c551402ac60445620922a3c694c8fe4f..28e5874cffa48d2195a55501b013c72fef505c4a 100644
--- a/tools/typescript/ts_library.gni
+++ b/tools/typescript/ts_library.gni
@@ -17,6 +17,7 @@ template("ts_library") {
                              "tsconfig_base",
                              "path_mappings_file",
                              "manifest_excludes",
+                             "public_deps",
                              "testonly",
                              "visibility",
                            ])
@@ -178,6 +179,7 @@ template("ts_library") {
       ]
     }
 
+    import("//brave/tools/typescript/sources.gni") foreach(mapping, brave_path_mappings) { path_mappings += [ mapping[0] + rebase_path(mapping[1], target_gen_dir) ] }
     args += [ "--path_mappings" ] + path_mappings
     if (defined(invoker.path_mappings)) {
       args += invoker.path_mappings
