diff --git a/tools/crates/gnrt/lib/readme.rs b/tools/crates/gnrt/lib/readme.rs
index 236a9f5c9b148b60d08fe715b75deb1f24c7698f..49ae85265d7dc224fd38418e90efc40f5bea5632 100644
--- a/tools/crates/gnrt/lib/readme.rs
+++ b/tools/crates/gnrt/lib/readme.rs
@@ -121,7 +121,7 @@ pub fn readme_file_from_package<'a>(
         Vec::new()
     };
 
-    if license_files.is_empty() {
+    if license_files.is_empty() && false {
         bail!(
             "License file not found for crate {name}.\n
              \n
@@ -195,6 +195,9 @@ enum LicenseKind {
 
     /// https://spdx.org/licenses/Unicode-3.0.html
     Unicode3,
+    UnicodeDfs2016,
+    BSD2,
+    Mozzila2,
 }
 
 impl Display for LicenseKind {
@@ -209,6 +212,9 @@ impl Display for LicenseKind {
             LicenseKind::ISC => write!(f, "ISC"),
             LicenseKind::Zlib => write!(f, "Zlib"),
             LicenseKind::Unicode3 => write!(f, "Unicode-3.0"),
+            LicenseKind::UnicodeDfs2016 => write!(f, "Unicode-DFS-2016"),
+            LicenseKind::BSD2 => write!(f, "BSD-2-Clause"),
+            LicenseKind::Mozzila2 => write!(f, "Mozilla-Public-License-2.0"),
         }
     }
 }
@@ -254,6 +260,11 @@ static LICENSE_STRING_TO_LICENSE_KIND: LazyLock<HashMap<&'static str, Vec<Licens
         h.insert("BSD-2-Clause OR Apache-2.0 OR MIT", vec![LicenseKind::Apache2]);
         h.insert("Unicode-3.0", vec![LicenseKind::Unicode3]);
         h.insert("Zlib", vec![LicenseKind::Zlib]);
+        h.insert("CC0-1.0 OR MIT-0 OR Apache-2.0", vec![LicenseKind::Apache2]);
+        h.insert("MIT OR Apache-2.0 OR BSD-1-Clause", vec![LicenseKind::Apache2]);
+        h.insert("BSD-2-Clause", vec![LicenseKind::BSD2]);
+        h.insert("MPL-2.0", vec![LicenseKind::Mozzila2]);
+        h.insert("CC0-1.0 OR Apache-2.0", vec![LicenseKind::Apache2]);
         h
     });
 
@@ -282,6 +293,9 @@ static LICENSE_KIND_TO_LICENSE_FILES: LazyLock<HashMap<LicenseKind, Vec<&'static
         h.insert(LicenseKind::ISC, vec!["LICENSE-ISC", "LICENSE.md", "LICENSE"]);
         h.insert(LicenseKind::Zlib, vec!["LICENSE-ZLIB", "LICENSE.md", "LICENSE"]);
         h.insert(LicenseKind::Unicode3, vec!["LICENSE-UNICODE", "LICENSE.md", "LICENSE"]);
+        h.insert(LicenseKind::UnicodeDfs2016, vec!["LICENSE-UNICODE", "LICENSE.md", "LICENSE"]);
+        h.insert(LicenseKind::BSD2, vec!["LICENSE.md", "LICENSE"]);
+        h.insert(LicenseKind::Mozzila2, vec!["LICENSE.md", "LICENSE"]);
         h
     });
 
