diff --git a/tools/grit/grit/gather/policy_json.py b/tools/grit/grit/gather/policy_json.py
index f57b6494dda7164ff247dccc4b2556889d825d1e..f52e4ea392ecb61084a4f17c06b12d7e216902c5 100644
--- a/tools/grit/grit/gather/policy_json.py
+++ b/tools/grit/grit/gather/policy_json.py
@@ -369,3 +369,4 @@ class PolicyJson(skeleton_gatherer.SkeletonGatherer):
       }
     else:
       raise Exception('Unknown build')
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
