diff --git a/tools/search_engine_choice/search_engine_icons_utils.py b/tools/search_engine_choice/search_engine_icons_utils.py
index 418a690eeffd2c97104b87562de78c14ddf58958..095061b34846d16927b231e6ff788b6e2c802bca 100644
--- a/tools/search_engine_choice/search_engine_icons_utils.py
+++ b/tools/search_engine_choice/search_engine_icons_utils.py
@@ -94,3 +94,4 @@ try:
   import json_comment_eater
 finally:
   sys.path.pop(0)
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
