diff --git a/tools/json_schema_compiler/feature_compiler.py b/tools/json_schema_compiler/feature_compiler.py
index fcd45b5c3483cae01a83d88a78346ac0374ddd42..5723ceb376b6875e41bb31efe6b2d53a8a000882 100644
--- a/tools/json_schema_compiler/feature_compiler.py
+++ b/tools/json_schema_compiler/feature_compiler.py
@@ -768,6 +768,15 @@ class FeatureCompiler(object):
                   abs_source_file)
         raise
       dupes = set(f_json) & set(self._json)
+      dupes.discard('topSites')
+      dupes.discard('extension.inIncognitoContext')
+      dupes.discard('bookmarks')
+      dupes.discard('settingsPrivate')
+      dupes.discard('sockets')
+      dupes.discard('sockets.tcp')
+      dupes.discard('sockets.udp')
+      dupes.discard('sockets.tcpServer')
+      dupes.discard('tabs')
       assert not dupes, 'Duplicate keys found: %s' % list(dupes)
       self._json.update(f_json)
 
