diff --git a/tools/typescript/ts_library.gni b/tools/typescript/ts_library.gni
index 7d175cc6be4c40476081643f182c0cc9b156ba55..52dc1a4b1127f3fd6912cd89065bdef50e366c47 100644
--- a/tools/typescript/ts_library.gni
+++ b/tools/typescript/ts_library.gni
@@ -171,9 +171,11 @@ template("ts_library") {
       ]
     }
 
+    import("//brave/tools/typescript/sources.gni") foreach(mapping, brave_path_mappings) { path_mappings += [ mapping[0] + rebase_path(mapping[1], target_gen_dir) ] }
     args += [ "--path_mappings" ] + path_mappings
     if (defined(invoker.path_mappings)) {
       args += invoker.path_mappings
     }
+    response_file_contents = args args = [] args = ["--rsp", "{{response_file_name}}"]
   }
 }
