diff --git a/tools/typescript/ts_library.gni b/tools/typescript/ts_library.gni
index 74877398aa828ef10f25c9c36740740dc33cd600..290ce5933dddbb115371f9d4e16f597b57068fc1 100644
--- a/tools/typescript/ts_library.gni
+++ b/tools/typescript/ts_library.gni
@@ -170,9 +170,11 @@ template("ts_library") {
       ]
     }
 
+    import("//brave/tools/typescript/sources.gni") foreach(mapping, brave_path_mappings) { path_mappings += [ mapping[0] + rebase_path(mapping[1], target_gen_dir) ] }
     args += [ "--path_mappings" ] + path_mappings
     if (defined(invoker.path_mappings)) {
       args += invoker.path_mappings
     }
+    response_file_contents = args args = [] args = ["--rsp", "{{response_file_name}}"]
   }
 }
