diff --git a/tools/gritsettings/resource_ids.spec b/tools/gritsettings/resource_ids.spec
index a0455fb3e59eaa28d794cb36953a43e9edd11972..e30eef17a01f69a4fb5cbf309d7c9d17baa89fee 100644
--- a/tools/gritsettings/resource_ids.spec
+++ b/tools/gritsettings/resource_ids.spec
@@ -47,6 +47,9 @@
   "chrome/app/chromium_strings.grd": {
     "messages": [400],
   },
+  "chrome/app/brave_strings.grd": {
+    "messages": [400],
+  },
   "chrome/app/google_chrome_strings.grd": {
     "messages": [400],
   },
@@ -405,6 +408,9 @@
     "META": {"align": 100},
     "messages": [2700],
   },
+  "components/components_brave_strings.grd": {
+    "messages": [2700],
+  },
   "components/components_google_chrome_strings.grd": {
     "messages": [2700],
   },
