diff --git a/tools/gritsettings/resource_ids.spec b/tools/gritsettings/resource_ids.spec
index b5b86bb00d59ea646e95d7d355e2f11b3d7f5589..d7780a9d5756e56f3f6187dfd6e4b2019f64d105 100644
--- a/tools/gritsettings/resource_ids.spec
+++ b/tools/gritsettings/resource_ids.spec
@@ -47,6 +47,9 @@
   "chrome/app/chromium_strings.grd": {
     "messages": [400],
   },
+  "chrome/app/brave_strings.grd": {
+    "messages": [400],
+  },
   "chrome/app/google_chrome_strings.grd": {
     "messages": [400],
   },
@@ -470,6 +473,9 @@
     "META": {"align": 100},
     "messages": [2700],
   },
+  "components/components_brave_strings.grd": {
+    "messages": [2700],
+  },
   "components/components_google_chrome_strings.grd": {
     "messages": [2700],
   },
