diff --git a/tools/gritsettings/resource_ids.spec b/tools/gritsettings/resource_ids.spec
index 82fd9d2d4e1d4ba42ec525777af964de17b1fea4..d94732603d4a3816507e927a6bb4b4138a283721 100644
--- a/tools/gritsettings/resource_ids.spec
+++ b/tools/gritsettings/resource_ids.spec
@@ -41,6 +41,9 @@
   "chrome/app/chromium_strings.grd": {
     "messages": [400],
   },
+  "chrome/app/brave_strings.grd": {
+    "messages": [400],
+  },
   "chrome/app/google_chrome_strings.grd": {
     "messages": [400],
   },
@@ -228,7 +231,7 @@
     "includes": [1600],
   },
   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/history/resources.grd": {
-    "META": {"sizes": {"includes": [40]}},
+    "META": {"sizes": {"includes": [44]}},
     "includes": [1620],
   },
   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/identity_internals/resources.grd": {
@@ -335,7 +338,7 @@
     "includes": [1940],
   },
   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/settings/resources.grd": {
-    "META": {"sizes": {"includes": [500],}},
+    "META": {"sizes": {"includes": [650],}},
     "includes": [1960],
   },
   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/signin/profile_picker/resources.grd": {
@@ -801,6 +804,9 @@
   "components/components_chromium_strings.grd": {
     "messages": [3520],
   },
+  "components/components_brave_strings.grd": {
+    "messages": [3520],
+  },
   "components/components_google_chrome_strings.grd": {
     "messages": [3520],
   },
