diff --git a/tools/grit/grit/tool/build.py b/tools/grit/grit/tool/build.py
index 9ca3722faff81dc23fc3008ea9ce01130e83e3f6..aa58b01491dface76750e067ae4be99b2d11ba0c 100644
--- a/tools/grit/grit/tool/build.py
+++ b/tools/grit/grit/tool/build.py
@@ -667,3 +667,4 @@ Duplicate actual output files:
     dir = os.path.split(file)[0]
     if not os.path.exists(dir):
       os.makedirs(dir, exist_ok=True)
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
