diff --git a/tools/rust/build_rust.py b/tools/rust/build_rust.py
index f10dc3f56bdf9d714293bbbf62a91411017b46da..f8e0d24090b68e6006a5ec5038cae1c477f708ec 100755
--- a/tools/rust/build_rust.py
+++ b/tools/rust/build_rust.py
@@ -757,9 +757,22 @@ def main():
                    fail_hard=False)
         RunCommand([
             'git', '-C', RUST_SRC_DIR, 'fetch', 'github',
-            '046503e6f4318aaba649e51becc38b1db9c87ee7'
+            '4f1e147849d89f4c2324fd569a48e6be66fae5d8'
+        ])
+        RunCommand([
+            'git', '-C', RUST_SRC_DIR, 'cherry-pick',
+            '4f1e147849d89f4c2324fd569a48e6be66fae5d8'
+        ],
+                   fail_hard=False)
+        # This file is conflicted. Just remove it.
+        RunCommand([
+            'git', '-C', RUST_SRC_DIR, 'rm',
+            'compiler/rustc_target/src/spec/targets/x86_64_apple_watchos_sim.rs'
+        ])
+        RunCommand([
+            'git', '-C', RUST_SRC_DIR, 'cherry-pick', '--continue',
+            '--no-edit'
         ])
-        GitCherryPick(RUST_SRC_DIR, '046503e6f4318aaba649e51becc38b1db9c87ee7')
 
         path = FetchBetaPackage('cargo', checkout_revision)
         if sys.platform == 'win32':
