diff --git a/tools/gritsettings/resource_ids.spec b/tools/gritsettings/resource_ids.spec
index c9e9d621c7304bf8abad7f877a06a5475bd6251b..0964f788f1ca649fedb3624ff84194da5265f845 100644
--- a/tools/gritsettings/resource_ids.spec
+++ b/tools/gritsettings/resource_ids.spec
@@ -47,6 +47,9 @@
   "chrome/app/chromium_strings.grd": {
     "messages": [400],
   },
+  "chrome/app/brave_strings.grd": {
+    "messages": [400],
+  },
   "chrome/app/google_chrome_strings.grd": {
     "messages": [400],
   },
@@ -434,6 +437,9 @@
     "META": {"align": 100},
     "messages": [2700],
   },
+  "components/components_brave_strings.grd": {
+    "messages": [2700],
+  },
   "components/components_google_chrome_strings.grd": {
     "messages": [2700],
   },
