diff --git a/tools/binary_size/libsupersize/archive.py b/tools/binary_size/libsupersize/archive.py
index 354989f6d7dcdac3c8a640dee449066de130e906..edc3a9627a363f53b61871bed175f337311273de 100644
--- a/tools/binary_size/libsupersize/archive.py
+++ b/tools/binary_size/libsupersize/archive.py
@@ -1235,6 +1235,7 @@ def _FilterContainerSpecs(container_specs, container_re=None):
   ret = []
   seen_container_names = set()
   for container_spec in container_specs:
+    if container_spec.container_name.find('libwg-go.so') != -1: continue
     container_name = container_spec.container_name
     if container_name in seen_container_names:
       raise ValueError('Duplicate container name: {}'.format(container_name))
