diff --git a/tools/crates/gnrt/gen.rs b/tools/crates/gnrt/gen.rs
index c1c4b1b73e50ccd306f7c7e9773aa2bbbfff8eb3..0b1593d17d1ee223b365d0bf5f227061a811a680 100644
--- a/tools/crates/gnrt/gen.rs
+++ b/tools/crates/gnrt/gen.rs
@@ -278,6 +278,7 @@ fn generate_for_third_party(args: GenCommandArgs, paths: &paths::ChromiumPaths)
     // Remove any excluded dep entries.
     dependencies
         .retain(|dep| !config.resolve.remove_crates.iter().any(|r| **r == dep.package_name));
+    dependencies.retain(|dep| !dep.is_local );
 
     // Remove dev dependencies since tests aren't run.
     dependencies.retain(|dep| {
