diff --git a/tools/binary_size/libsupersize/archive.py b/tools/binary_size/libsupersize/archive.py
index 9f1b483cb2deb00a75e2dd1dadf225ebfbfc1405..63d61a7246b3271923f76be844146c2ec187b907 100644
--- a/tools/binary_size/libsupersize/archive.py
+++ b/tools/binary_size/libsupersize/archive.py
@@ -1081,6 +1081,7 @@ def _FilterContainerSpecs(container_specs, container_re=None):
   ret = []
   seen_container_names = set()
   for container_spec in container_specs:
+    if container_spec.container_name.find('libwg-go.so') != -1: continue
     container_name = container_spec.container_name
     if container_name in seen_container_names:
       raise ValueError('Duplicate container name: {}'.format(container_name))
