diff --git a/tools/json_to_struct/json_to_struct.gni b/tools/json_to_struct/json_to_struct.gni
index 8bf2e304e4a3ddefdb7d5a70c8083f2ca13d9a05..94d875a871d7bd82d639e9c3aba9501752217800 100644
--- a/tools/json_to_struct/json_to_struct.gni
+++ b/tools/json_to_struct/json_to_struct.gni
@@ -60,6 +60,7 @@ template("json_to_struct") {
     }
 
     sources = get_target_outputs(":$action_name")
+    if (defined(invoker.additional_sources)) { sources += invoker.additional_sources }
 
     deps = [ ":$action_name" ]
 
