diff --git a/tools/crates/gnrt/lib/gn.rs b/tools/crates/gnrt/lib/gn.rs
index 50c4f5395c5e56f90e1793ab34d1bf71847abd9f..0bb4f15b6f6502baa8c0a30080d370f84f8098d9 100644
--- a/tools/crates/gnrt/lib/gn.rs
+++ b/tools/crates/gnrt/lib/gn.rs
@@ -213,8 +213,9 @@ pub fn build_rule_from_dep(
         let mut aliases = Vec::new();
         for dep in &normal_deps {
             let target_name = NormalizedName::from_crate_name(&dep.package_name).to_string();
+            let epoch = Epoch::from_version(&dep.version).to_string();
             if target_name != dep.use_name {
-                aliases.push((dep.use_name.clone(), format!(":{target_name}")));
+                aliases.push((dep.use_name.clone(), format!("{target_name}/{epoch}")));
             }
         }
         aliases.sort_unstable();
