diff --git a/tools/perf/page_sets/system_health/system_health_stories.py b/tools/perf/page_sets/system_health/system_health_stories.py
index 9bb718effd4dd155a568d8589709a7b915bad847..91f55a3fd2cb5de492086295ae333a7f1edd9726 100644
--- a/tools/perf/page_sets/system_health/system_health_stories.py
+++ b/tools/perf/page_sets/system_health/system_health_stories.py
@@ -26,7 +26,7 @@ class SystemHealthStorySet(story.StorySet):
                take_memory_measurement=False,
                tag=None):
     super(SystemHealthStorySet, self).__init__(
-        archive_data_file=('../data/system_health_%s.json' % platform),
+        archive_data_file=('../../../../brave/tools/perf/page_sets/data/system_health_%s.json' % platform),
         cloud_storage_bucket=story.PARTNER_BUCKET)
 
     assert platform in platforms.ALL_PLATFORMS
