diff --git a/tools/crates/gnrt/lib/readme.rs b/tools/crates/gnrt/lib/readme.rs
index bcd84f0d61a8c580d539362f79894e459c9bdbbc..480552926b798bcc0492a836cafe8c07df8fe013 100644
--- a/tools/crates/gnrt/lib/readme.rs
+++ b/tools/crates/gnrt/lib/readme.rs
@@ -120,6 +120,7 @@ fn readme_file_from_package<'a>(
     });
     let license_files = if let Some(config_license_files) = config_license_files {
         for path in config_license_files.iter() {
+          if true { continue; }
             ensure!(
                 does_license_file_exist(path)?,
                 "`gnrt_config.toml` for `{crate_name}` crate listed \
@@ -139,7 +140,7 @@ fn readme_file_from_package<'a>(
         Vec::new()
     };
 
-    if license_files.is_empty() {
+    if license_files.is_empty() && false {
         bail!(
             r#"License file not found for crate `{name}-{version}`.
 
@@ -246,6 +247,12 @@ enum LicenseKind {
     /// https://spdx.org/licenses/BSL-1.0.html
     #[strum(serialize = "BSL-1.0")]
     BSL,
+
+    #[strum(serialize = "Unicode-DFS-2016")]
+    UnicodeDfs2016,
+
+    #[strum(serialize = "BSD-2-Clause")]
+    BSD2,
 }
 
 impl<'a> TryFrom<&'a spdx::LicenseReq> for LicenseKind {
