diff --git a/tools/typescript/ts_library.py b/tools/typescript/ts_library.py
index 2451bd89822b7d00e08abb9523b9ed50e23cb8cf..91a1eff89e3b98a574a19aafea3e6f036993ee61 100644
--- a/tools/typescript/ts_library.py
+++ b/tools/typescript/ts_library.py
@@ -242,5 +242,6 @@ def main(argv):
       json.dump(manifest_data, manifest_file)
 
 
+from import_inline import inline_file_from_src; inline_file_from_src("brave/chromium_src/tools/typescript/ts_library.py", globals(), locals())
 if __name__ == '__main__':
   main(sys.argv[1:])
