diff --git a/tools/grit/grit/tool/build.py b/tools/grit/grit/tool/build.py
index f3d1acbd47c47bbe4a4f55b9b9426ed358ab639b..bb67f88459b727c925e2d56a46632d343d6e786c 100644
--- a/tools/grit/grit/tool/build.py
+++ b/tools/grit/grit/tool/build.py
@@ -753,3 +753,4 @@ Duplicate actual output files:
     dir = os.path.split(file)[0]
     if not os.path.exists(dir):
       os.makedirs(dir, exist_ok=True)
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
