diff --git a/tools/typescript/ts_library.gni b/tools/typescript/ts_library.gni
index a06f3a7fcaec0ff800f618abb6b696a7b0a5da50..0708c5ee870897c7ca72b5d455facfe0cb1dd55e 100644
--- a/tools/typescript/ts_library.gni
+++ b/tools/typescript/ts_library.gni
@@ -180,6 +180,7 @@ template("ts_library") {
       ]
     }
 
+    import("//brave/tools/typescript/sources.gni") foreach(mapping, brave_path_mappings) { path_mappings += [ mapping[0] + rebase_path(mapping[1], target_gen_dir) ] }
     args += [ "--path_mappings" ] + path_mappings
     if (defined(invoker.path_mappings)) {
       args += invoker.path_mappings
