diff --git a/tools/gritsettings/resource_ids.spec b/tools/gritsettings/resource_ids.spec
index 667a25343279481ae67fa659b66c67022f9e5dfc..6fd210c4c3a32fc6bc312e5b38e490b9b6fa7fb8 100644
--- a/tools/gritsettings/resource_ids.spec
+++ b/tools/gritsettings/resource_ids.spec
@@ -51,6 +51,9 @@
   "chrome/app/chromium_strings.grd": {
     "messages": [400],
   },
+  "chrome/app/brave_strings.grd": {
+    "messages": [400],
+  },
   "chrome/app/google_chrome_strings.grd": {
     "messages": [400],
   },
@@ -571,6 +574,9 @@
   "components/components_chromium_strings.grd": {
     "messages": [3520],
   },
+  "components/components_brave_strings.grd": {
+    "messages": [3520],
+  },
   "components/components_google_chrome_strings.grd": {
     "messages": [3520],
   },
