diff --git a/tools/grit/grit/format/minifier.py b/tools/grit/grit/format/minifier.py
index 13f5adf3c7c2c0ce5278de17b3faea3ba3c0a9ba..303a6c680631c02c5f1e2b86b6ad564ce45c96dd 100644
--- a/tools/grit/grit/format/minifier.py
+++ b/tools/grit/grit/format/minifier.py
@@ -32,3 +32,4 @@ def Minify(source, filename):
     print(stderr)
     sys.exit(p.returncode)
   return stdout
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
\ No newline at end of file
