diff --git a/tools/crates/gnrt/lib/gn.rs b/tools/crates/gnrt/lib/gn.rs
index 4a61217a166dadf8a44752b92cd362105d8fa022..6a28c161f4cc2f037b8d7696d5b63df843ebafa2 100644
--- a/tools/crates/gnrt/lib/gn.rs
+++ b/tools/crates/gnrt/lib/gn.rs
@@ -201,8 +201,9 @@ pub fn build_rule_from_dep(
         let mut aliases = Vec::new();
         for dep in &normal_deps {
             let target_name = NormalizedName::from_crate_name(&dep.package_name).to_string();
+            let epoch = Epoch::from_version(&dep.version).to_string();
             if target_name != dep.use_name {
-                aliases.push((dep.use_name.clone(), format!(":{target_name}")));
+                aliases.push((dep.use_name.clone(), format!("{target_name}/{epoch}")));
             }
         }
         aliases.sort_unstable();
@@ -768,6 +769,7 @@ fn target_os_to_condition(target_os: &str) -> Condition {
     for (t, c) in &[
         ("android", "is_android"),
         ("darwin", "is_mac"),
+        ("macos", "is_mac"),
         ("fuchsia", "is_fuchsia"),
         ("ios", "is_ios"),
         ("linux", "is_linux || is_chromeos"),
