diff --git a/tools/crates/gnrt/gen.rs b/tools/crates/gnrt/gen.rs
index cb0ba4ad32456bbd6c4fe7789ad528347519de56..05b60f26ee77a096ef5030d5225c38a7073f53a4 100644
--- a/tools/crates/gnrt/gen.rs
+++ b/tools/crates/gnrt/gen.rs
@@ -270,6 +270,7 @@ fn generate_for_third_party(args: GenCommandArgs, paths: &paths::ChromiumPaths)
     let all_build_files: HashMap<PathBuf, gn::BuildFile> = {
         let mut map = HashMap::new();
         for dep in &dependencies {
+            if dep.is_local { continue; }
             let build_file = gn::build_file_from_deps(
                 std::iter::once(dep),
                 paths,
