diff --git a/tools/json_schema_compiler/feature_compiler.py b/tools/json_schema_compiler/feature_compiler.py
index 645c5fb934170e14db5c44c552db06febab7f09c..dd617177f7fc2ad276e23f39069cf705a3faef6e 100644
--- a/tools/json_schema_compiler/feature_compiler.py
+++ b/tools/json_schema_compiler/feature_compiler.py
@@ -766,6 +766,14 @@ class FeatureCompiler(object):
                   abs_source_file)
         raise
       dupes = set(f_json) & set(self._json)
+      dupes.discard('topSites')
+      dupes.discard('extension.inIncognitoContext')
+      dupes.discard('bookmarks')
+      dupes.discard('settingsPrivate')
+      dupes.discard('sockets')
+      dupes.discard('sockets.tcp')
+      dupes.discard('sockets.udp')
+      dupes.discard('sockets.tcpServer')
       assert not dupes, 'Duplicate keys found: %s' % list(dupes)
       self._json.update(f_json)
 
