diff --git a/tools/typescript/ts_library.gni b/tools/typescript/ts_library.gni
index 90e8188a965d4199d4adc72327077efa7d64549b..1461c8d85e95285c3a210635cc877b7a89a87550 100644
--- a/tools/typescript/ts_library.gni
+++ b/tools/typescript/ts_library.gni
@@ -116,8 +116,12 @@ template("ts_library") {
     path_mappings = [
       "chrome://resources/*|" +
           rebase_path("$root_gen_dir/ui/webui/resources/tsc/*", target_gen_dir),
+      "chrome://resources/brave/*|" +
+          rebase_path("$root_gen_dir/brave/ui/webui/resources/tsc/*", target_gen_dir),
       "//resources/*|" +
           rebase_path("$root_gen_dir/ui/webui/resources/tsc/*", target_gen_dir),
+      "//resources/brave/*|" +
+          rebase_path("$root_gen_dir/brave/ui/webui/resources/tsc/*", target_gen_dir),
       "chrome://resources/polymer/v3_0/*|" +
           rebase_path("//third_party/polymer/v3_0/components-chromium/*",
                       target_gen_dir),
