diff --git a/tools/gritsettings/resource_ids.spec b/tools/gritsettings/resource_ids.spec
index 1d74352b1b1a400cd991122607fffb884f8a05db..d4f856b45d762469ab71b43fc6000d1d76ea2795 100644
--- a/tools/gritsettings/resource_ids.spec
+++ b/tools/gritsettings/resource_ids.spec
@@ -41,6 +41,9 @@
   "chrome/app/chromium_strings.grd": {
     "messages": [400],
   },
+  "chrome/app/brave_strings.grd": {
+    "messages": [400],
+  },
   "chrome/app/google_chrome_strings.grd": {
     "messages": [400],
   },
@@ -221,7 +224,7 @@
     "includes": [1600],
   },
   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/history/resources.grd": {
-    "META": {"sizes": {"includes": [40]}},
+    "META": {"sizes": {"includes": [44]}},
     "includes": [1620],
   },
   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/identity_internals/resources.grd": {
@@ -332,7 +335,7 @@
     "includes": [1940],
   },
   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/settings/resources.grd": {
-    "META": {"sizes": {"includes": [500],}},
+    "META": {"sizes": {"includes": [650],}},
     "includes": [1960],
   },
   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/signin/profile_picker/resources.grd": {
@@ -790,6 +793,9 @@
   "components/components_chromium_strings.grd": {
     "messages": [3520],
   },
+  "components/components_brave_strings.grd": {
+    "messages": [3520],
+  },
   "components/components_google_chrome_strings.grd": {
     "messages": [3520],
   },
