diff --git a/tools/clang/pylib/clang/compile_db.py b/tools/clang/pylib/clang/compile_db.py
index 200512d7fbd51b1588d1b8c46ad0964479758ff2..16bbb5ad89e5a39723d108e70ed7741ed2575842 100755
--- a/tools/clang/pylib/clang/compile_db.py
+++ b/tools/clang/pylib/clang/compile_db.py
@@ -241,3 +241,4 @@ def Read(path):
   """
   with open(os.path.join(path, 'compile_commands.json'), 'rb') as db:
     return json.load(db)
+import os, sys; sys.path.append(os.path.join(os.path.dirname(__file__), *[os.pardir] * 4, 'brave', 'script')); from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
