diff --git a/tools/clang/scripts/update.py b/tools/clang/scripts/update.py
index 2b2dc1a58d62419e3008cdf95b273929cbba3049..0571278ef745a15308ab4fafd3c99d0e50c05d96 100755
--- a/tools/clang/scripts/update.py
+++ b/tools/clang/scripts/update.py
@@ -35,7 +35,7 @@ import zlib
 # https://chromium.googlesource.com/chromium/src/+/main/docs/updating_clang.md
 # Reverting problematic clang rolls is safe, though.
 # This is the output of `git describe` and is usable as a commit-ish.
-CLANG_REVISION = 'llvmorg-18-init-9505-g10664813'
+CLANG_REVISION = 'llvmorg-18-init-7785-geef35c28'
 CLANG_SUB_REVISION = 1
 
 PACKAGE_VERSION = '%s-%s' % (CLANG_REVISION, CLANG_SUB_REVISION)
