diff --git a/tools/crates/gnrt/gen.rs b/tools/crates/gnrt/gen.rs
index 6c90c1266ff82b340b82b4b656977fa812d0f182..e3e165138ed7fc3dd49bed0c7e99ec51972f4dc0 100644
--- a/tools/crates/gnrt/gen.rs
+++ b/tools/crates/gnrt/gen.rs
@@ -267,6 +267,7 @@ fn generate_for_third_party(args: GenCommandArgs, paths: &paths::ChromiumPaths)
     let all_build_files: HashMap<PathBuf, gn::BuildFile> = {
         let mut map = HashMap::new();
         for dep in &dependencies {
+            if dep.is_local { continue; }
             let build_file = gn::build_file_from_deps(
                 std::iter::once(dep),
                 paths,
