diff --git a/tools/gritsettings/resource_ids b/tools/gritsettings/resource_ids
index e95450a1ff2be9c5788930169fcfe902a4fb6353..4ca44c8fd34381935f94f358dc1e61c000f7feca 100644
--- a/tools/gritsettings/resource_ids
+++ b/tools/gritsettings/resource_ids
@@ -36,6 +36,9 @@
   "chrome/app/chromium_strings.grd": {
     "messages": [400],
   },
+  "chrome/app/brave_strings.grd": {
+    "messages": [400],
+  },
   "chrome/app/google_chrome_strings.grd": {
     "messages": [400],
   },
@@ -146,6 +149,9 @@
   # Chromium strings and Google Chrome strings must start at the same id.
   # We only use one file depending on whether we're building Chromium or
   # Google Chrome.
+  "components/components_brave_strings.grd": {
+    "messages": [15000],
+  },
   "components/components_chromium_strings.grd": {
     "messages": [15000],
   },
