diff --git a/tools/rust/update_rust.py b/tools/rust/update_rust.py
index 745582e7c6512e50378a2aaa00c1c2eb2597cc97..b92195e911d2476070e40552f25db44aea9f2b77 100755
--- a/tools/rust/update_rust.py
+++ b/tools/rust/update_rust.py
@@ -35,7 +35,7 @@ sys.path.append(
 # this back to its previous value _AND_ set `OVERRIDE_CLANG_REVISION` below
 # to the `CLANG_REVISION` that was in place before the roll.
 RUST_REVISION = 'df0295f07175acc7325ce3ca4152eb05752af1f2'
-RUST_SUB_REVISION = 1
+RUST_SUB_REVISION = 3
 
 # If not None, this overrides the `CLANG_REVISION` in
 # //tools/clang/scripts/update.py in order to download a Rust toolchain that
