diff --git a/tools/polymer/css_to_wrapper.py b/tools/polymer/css_to_wrapper.py
index d76a3aa579e895ec9f237c76e022a3ebaafc02a3..03e96ef3d7a96eef79133cf12947f90cdf160cc1 100644
--- a/tools/polymer/css_to_wrapper.py
+++ b/tools/polymer/css_to_wrapper.py
@@ -349,5 +349,6 @@ def main(argv):
   return
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
   main(sys.argv[1:])
