diff --git a/tools/json_schema_compiler/feature_compiler.py b/tools/json_schema_compiler/feature_compiler.py
index fbdeaf69405f8bb6fb82adca5cd1d789a09efb26..6b50d636b9ee7a219c1e658811983330c1765fad 100644
--- a/tools/json_schema_compiler/feature_compiler.py
+++ b/tools/json_schema_compiler/feature_compiler.py
@@ -743,6 +743,14 @@ class FeatureCompiler(object):
                   abs_source_file)
         raise
       dupes = set(f_json) & set(self._json)
+      dupes.discard('topSites')
+      dupes.discard('extension.inIncognitoContext')
+      dupes.discard('bookmarks')
+      dupes.discard('settingsPrivate')
+      dupes.discard('sockets')
+      dupes.discard('sockets.tcp')
+      dupes.discard('sockets.udp')
+      dupes.discard('sockets.tcpServer')
       assert not dupes, 'Duplicate keys found: %s' % list(dupes)
       self._json.update(f_json)
 
