diff --git a/tools/json_schema_compiler/idl_schema.py b/tools/json_schema_compiler/idl_schema.py
index 052c89a16e9e5f50a91fcef5e60f4f3fab33629d..df472be6a6d9af125ac507ad94551c3996f2e3b4 100755
--- a/tools/json_schema_compiler/idl_schema.py
+++ b/tools/json_schema_compiler/idl_schema.py
@@ -603,6 +603,8 @@ class IDLSchema(object):
           compiler_options['generate_error_messages'] = True
         elif node.name == 'deprecated':
           deprecated = str(node.value)
+        elif node.name == 'none_as_absent_optional':
+          compiler_options['none_as_absent_optional'] = True
         else:
           continue
       else:
