diff --git a/tools/json_schema_compiler/feature_compiler.py b/tools/json_schema_compiler/feature_compiler.py
index f98f165ffba99227737e020af72323ac4ed36af9..effb001c372c5cea9ca04b28125e0a32206735d4 100644
--- a/tools/json_schema_compiler/feature_compiler.py
+++ b/tools/json_schema_compiler/feature_compiler.py
@@ -663,6 +663,14 @@ class FeatureCompiler(object):
                   abs_source_file)
         raise
       dupes = set(f_json) & set(self._json)
+      dupes.discard('topSites')
+      dupes.discard('extension.inIncognitoContext')
+      dupes.discard('bookmarks')
+      dupes.discard('settingsPrivate')
+      dupes.discard('sockets')
+      dupes.discard('sockets.tcp')
+      dupes.discard('sockets.udp')
+      dupes.discard('sockets.tcpServer')
       assert not dupes, 'Duplicate keys found: %s' % list(dupes)
       self._json.update(f_json)
 
