diff --git a/tools/grit/grit_args.gni b/tools/grit/grit_args.gni
index 92be15a41a034e28d50e87fdcebbc37faf966436..45c19a5fe0ee4b58210b528bceadd2cb8bd2ccc8 100644
--- a/tools/grit/grit_args.gni
+++ b/tools/grit/grit_args.gni
@@ -51,8 +51,10 @@ _grit_environment_vars = [
 
 if (is_chrome_branded) {
   _grit_environment_vars += [ "CHROMIUM_BUILD=google_chrome" ]
+  _grit_defines += [ "_chromium=false" ]
 } else {
   _grit_environment_vars += [ "CHROMIUM_BUILD=chromium" ]
+  _grit_defines += [ "_chromium=true" ]
 }
 
 if (is_android) {
