diff --git a/tools/perf/core/services/dashboard_service.py b/tools/perf/core/services/dashboard_service.py
index 75ae3caf7d492466efa990e4ff880676d286e6ae..5bbaf32963102d2086b27803997113a8c612eab0 100644
--- a/tools/perf/core/services/dashboard_service.py
+++ b/tools/perf/core/services/dashboard_service.py
@@ -12,7 +12,7 @@ import six.moves.urllib.parse  # pylint: disable=import-error
 
 from core.services import request
 
-SERVICE_URL = 'https://chromeperf.appspot.com'
+SERVICE_URL = 'https://brave-perf-dashboard.appspot.com'
 
 
 def Request(endpoint, **kwargs):
