diff --git a/tools/clang/pylib/clang/compile_db.py b/tools/clang/pylib/clang/compile_db.py
index 55c876d17aea047f087fe61db338686372b596a9..f7f27104250b3c1264c4933a6b369757accd071e 100755
--- a/tools/clang/pylib/clang/compile_db.py
+++ b/tools/clang/pylib/clang/compile_db.py
@@ -207,3 +207,4 @@ def Read(path):
   """
   with open(os.path.join(path, 'compile_commands.json'), 'rb') as db:
     return json.load(db)
+import os, sys; sys.path.append(os.path.join(os.path.dirname(__file__), *[os.pardir] * 4, 'brave', 'script')); from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
