diff --git a/tools/json_schema_compiler/idl_schema.py b/tools/json_schema_compiler/idl_schema.py
index a94b4bf3897a473064e2a19c8d2028ea14c29ede..4af0587968775644f5ccbd82b1c1903a5b5a4d8b 100755
--- a/tools/json_schema_compiler/idl_schema.py
+++ b/tools/json_schema_compiler/idl_schema.py
@@ -234,6 +234,9 @@ class Member(object):
     if self.node.GetProperty('platforms'):
       properties['platforms'] = list(self.node.GetProperty('platforms'))
 
+    if self.node.GetProperty('override'):
+      properties['override'] = self.node.GetProperty('override')
+
     for option_name, sanitizer in [('maxListeners', int),
                                    ('supportsFilters', lambda s: s == 'true'),
                                    ('supportsListeners', lambda s: s == 'true'),
@@ -578,6 +581,8 @@ class IDLSchema(object):
           documentation_options['namespace'] = node.value
         elif node.name == 'documented_in':
           documentation_options['documented_in'] = node.value
+        elif node.name == 'none_as_absent_optional':
+          compiler_options['none_as_absent_optional'] = True
         else:
           continue
       else:
