diff --git a/tools/licenses/licenses.py b/tools/licenses/licenses.py
index 15b461326e2ce4bd255cbe779b449261a7f63f09..4ad542a3e8d1b806f7e63b7c10a4d37cd9b56b0a 100755
--- a/tools/licenses/licenses.py
+++ b/tools/licenses/licenses.py
@@ -39,6 +39,7 @@ _REPOSITORY_ROOT = os.path.abspath(
     os.path.join(os.path.dirname(__file__), '..', '..'))
 sys.path.insert(0, os.path.join(_REPOSITORY_ROOT, 'build'))
 import action_helpers
+from brave_license_helper import AddBraveCredits, BRAVE_THIRD_PARTY_DIRS, CheckBraveMissingLicense
 
 # Paths from the root of the tree to directories to skip.
 PRUNE_PATHS = set([
@@ -425,6 +426,7 @@ SPECIAL_CASES = {
         "License File": ["//third_party/selenium-atoms/LICENSE.closure"],
     },
 }
+(PRUNE_DIRS, ADDITIONAL_PATHS) = AddBraveCredits(_REPOSITORY_ROOT, PRUNE_PATHS, SPECIAL_CASES, PRUNE_DIRS, ADDITIONAL_PATHS)
 
 # These buildtools/third_party directories only contain
 # chromium build files. The actual third_party source files and their
@@ -708,7 +710,8 @@ def FindThirdPartyDirs(prune_paths, root, extra_third_party_dirs=None):
       if skip in dirs:
         dirs.remove(skip)
 
-    if os.path.basename(path) == 'third_party':
+    if (os.path.basename(path) == 'third_party' or
+        os.path.basename(path) in BRAVE_THIRD_PARTY_DIRS):
       # Add all subdirectories that are not marked for skipping.
       for dir in dirs:
         dirpath = os.path.join(path, dir)
@@ -956,7 +959,8 @@ def GenerateCredits(file_template_file,
                           enable_warnings=enable_warnings)
       if not metadata:
         continue
-    except LicenseError:
+    except LicenseError as e:
+      CheckBraveMissingLicense(target_os, path, e)
       # TODO(phajdan.jr): Convert to fatal error (http://crbug.com/39240).
       continue
     if metadata['Shipped'] == NO:
