diff --git a/tools/crates/gnrt/lib/gn.rs b/tools/crates/gnrt/lib/gn.rs
index f744a1b5fba0484fb8fb1ae4dc15e125bef8b8f0..7935cd4c725a5578bc52536b5c07d17af34d0293 100644
--- a/tools/crates/gnrt/lib/gn.rs
+++ b/tools/crates/gnrt/lib/gn.rs
@@ -201,8 +201,9 @@ pub fn build_rule_from_dep(
         let mut aliases = Vec::new();
         for dep in &normal_deps {
             let target_name = NormalizedName::from_crate_name(&dep.package_name).to_string();
+            let epoch = Epoch::from_version(&dep.version).to_string();
             if target_name != dep.use_name {
-                aliases.push((dep.use_name.clone(), format!(":{target_name}")));
+                aliases.push((dep.use_name.clone(), format!("{target_name}/{epoch}")));
             }
         }
         aliases.sort_unstable();
@@ -590,6 +591,7 @@ fn cfg_to_condition(cfg: &cargo_platform::Cfg) -> Condition {
             "target_family" => target_family_to_condition(value),
             "target_os" => target_os_to_condition(value),
             "target_vendor" => target_vendor_to_condition(value),
+            "curve25519_dalek_backend" => Condition::AlwaysFalse,
             _ => Condition::Unsupported(format!("Unknown key `{key}` in `{cfg}`")),
         },
     }
@@ -608,6 +610,10 @@ fn cfg_name_to_condition(name: &str) -> Condition {
         return Condition::AlwaysFalse;
     }
 
+    if ["tracing_unstable"].contains(&name) {
+        return Condition::AlwaysFalse;
+    }
+
     Condition::Unsupported(format!("unknown option name: `#[cfg({name})]`"))
 }
 
@@ -712,6 +718,7 @@ fn target_os_to_condition(target_os: &str) -> Condition {
     for (t, c) in &[
         ("android", "is_android"),
         ("darwin", "is_mac"),
+        ("macos", "is_mac"),
         ("fuchsia", "is_fuchsia"),
         ("ios", "is_ios"),
         ("linux", "is_linux || is_chromeos"),
