diff --git a/tools/json_schema_compiler/idl_schema.py b/tools/json_schema_compiler/idl_schema.py
index 3f33e43b4e3c12a77d85d86c37b9069d37339f74..a66896108dc81acb99560b6a665d17b989935985 100755
--- a/tools/json_schema_compiler/idl_schema.py
+++ b/tools/json_schema_compiler/idl_schema.py
@@ -603,6 +603,8 @@ class IDLSchema(object):
           compiler_options['generate_error_messages'] = True
         elif node.name == 'deprecated':
           deprecated = str(node.value)
+        elif node.name == 'none_as_absent_optional':
+          compiler_options['none_as_absent_optional'] = True
         else:
           continue
       else:
