diff --git a/tools/grit/grit/tool/build.py b/tools/grit/grit/tool/build.py
index a9b670840fc4a3efcdfaaaa0f11bc64d3c6b0827..5696c8dcd7fc1b5196e50b74368f6143ed9ef8d3 100644
--- a/tools/grit/grit/tool/build.py
+++ b/tools/grit/grit/tool/build.py
@@ -674,3 +674,4 @@ Duplicate actual output files:
     dir = os.path.split(file)[0]
     if not os.path.exists(dir):
       os.makedirs(dir, exist_ok=True)
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
