diff --git a/tools/git/move_source_file.py b/tools/git/move_source_file.py
index ab57ae721179f77bb92b808e65dfe1ced8cf605f..affc0ac8142d6575176675b413d0920a1c2139ec 100755
--- a/tools/git/move_source_file.py
+++ b/tools/git/move_source_file.py
@@ -115,6 +115,7 @@ def UpdatePostMove(from_path, to_path):
   mffr.MultiFileFindReplace(
       r'(//.*)%s' % re.escape(from_path),
       r'\1%s' % to_path,
+      ['*.gni', '*.gn'] + # Brave uses full paths (start with //) to add files
       ['*.cc', '*.h', '*.m', '*.mm', '*.cpp'])
 
   # Update references in GYP and BUILD.gn files.
@@ -152,6 +153,7 @@ def UpdatePostMove(from_path, to_path):
   visiting_directory = ''
   from_rest = from_path
   to_rest = to_path
+  from move_source_fille_utils import to_cwd_relative_path; from_rest, to_rest = to_cwd_relative_path(from_rest), to_cwd_relative_path(to_rest)
   while True:
     files_with_changed_sources = mffr.MultiFileFindReplace(
         r'([\'"])%s([\'"])' % from_rest,
@@ -186,6 +188,7 @@ def UpdateIncludeGuard(old_path, new_path):
   old_guard = MakeIncludeGuardName(old_path)
   new_guard = MakeIncludeGuardName(new_path)
 
+  from move_source_fille_utils import to_cwd_relative_path; old_path, new_path = to_cwd_relative_path(old_path), to_cwd_relative_path(new_path)
   with open(new_path) as f:
     contents = f.read()
 
@@ -244,6 +247,7 @@ def main():
     to_path = MakeDestinationPath(from_path, orig_to_path)
     if not opts.already_moved:
       MoveFile(from_path, to_path)
+    from move_source_fille_utils import to_src_relative_path; from_path, to_path = to_src_relative_path(from_path), to_src_relative_path(to_path)
     UpdatePostMove(from_path, to_path)
   return 0
 
