diff --git a/tools/clang/pylib/clang/compile_db.py b/tools/clang/pylib/clang/compile_db.py
index eff48afe2de97dc1649818008570c4af08eb465c..ceab8f819570006a325b66a3d04028dc6c3cbed4 100755
--- a/tools/clang/pylib/clang/compile_db.py
+++ b/tools/clang/pylib/clang/compile_db.py
@@ -12,7 +12,7 @@ import sys
 
 _RSP_RE = re.compile(r' (@(.+?\.rsp)) ')
 _GOMA_CMD_LINE_RE = re.compile(
-    r'^(?P<gomacc>.*gomacc(\.exe)?"?\s+)?(?P<clang>\S*clang\S*)\s+(?P<args>.*)$'
+    r'^(?P<gomacc>.*(gomacc|redirect-cc)(\.exe|\.py)?"?\s+)?(?P<clang>\S*clang\S*)\s+(?P<args>.*)$'
 )
 _debugging = False
 
