diff --git a/tools/crates/gnrt/vendor.rs b/tools/crates/gnrt/vendor.rs
index 1587a835d138928a77009aa2d704be9119c80976..66f92aae81339cc4040b2377ad26b93828bde025 100644
--- a/tools/crates/gnrt/vendor.rs
+++ b/tools/crates/gnrt/vendor.rs
@@ -141,7 +141,7 @@ fn download_crates(args: &VendorCommandArgs, paths: &paths::ChromiumPaths) -> Re
                 format!("Generating placeholder for removed crate {}", crate_dirname.display());
             println!("{msg}");
             generate_placeholder_crate(p, &crate_path).context(msg)?;
-        } else {
+        } else if !p.source().is_local() {
             let msg = format!("Downloading {}", crate_dirname.display());
             println!("{msg}");
             download_crate(p.name(), p.version(), paths).context(msg)?;
