diff --git a/ui/accessibility/platform/ax_platform_node_cocoa.mm b/ui/accessibility/platform/ax_platform_node_cocoa.mm
index d69725a1d840a6eacc813a57514591cc71db6fe7..6df3c79846dcd755b2dedb6b561efbbec79c4181 100644
--- a/ui/accessibility/platform/ax_platform_node_cocoa.mm
+++ b/ui/accessibility/platform/ax_platform_node_cocoa.mm
@@ -1439,6 +1439,7 @@ bool IsAXSetter(SEL selector) {
   if (!_node)
     return nil;
 
+  BRAVE_ACCESSIBILITY_ATTRIBUTED_STRING_FOR_RANGE
   return [self AXAttributedStringForRange:[NSValue valueWithRange:range]];
 }
 
