diff --git a/ui/accessibility/platform/ax_platform_node_cocoa.mm b/ui/accessibility/platform/ax_platform_node_cocoa.mm
index 623f996bf60670b144521e10931a87d843b81d28..6b6b56b627d9ddd15ae5f2dd2ea5e961a8892a02 100644
--- a/ui/accessibility/platform/ax_platform_node_cocoa.mm
+++ b/ui/accessibility/platform/ax_platform_node_cocoa.mm
@@ -3456,6 +3456,7 @@ const ui::CocoaActionList& GetCocoaActionListForTesting() {
   if (!_node)
     return nil;
 
+  BRAVE_ACCESSIBILITY_ATTRIBUTED_STRING_FOR_RANGE
   return [self AXAttributedStringForRange:[NSValue valueWithRange:range]];
 }
 
