diff --git a/ui/base/accelerators/accelerator.cc b/ui/base/accelerators/accelerator.cc
index e260f801eaf3ab3c2e3bbf09c94ee8a055bc8378..3f5d3c05d9c422e67c26c10d7106d7315ef26b56 100644
--- a/ui/base/accelerators/accelerator.cc
+++ b/ui/base/accelerators/accelerator.cc
@@ -388,6 +388,7 @@ std::u16string Accelerator::KeyCodeToName() const {
 
 std::u16string Accelerator::ApplyLongFormModifiers(
     const std::u16string& shortcut) const {
+  BRAVE_UI_BASE_ACCELERATOR_APPLY_LONG_FORM_MODIFIERS_
   std::u16string result = shortcut;
 
   if (IsShiftDown())
