diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
index 33b8a3bb7fae6941f0aa7af47aa048ac4a932b8f..1c9f64a73fe8497c2e1aadb40cd02629fc8d8acf 100644
--- a/ui/base/BUILD.gn
+++ b/ui/base/BUILD.gn
@@ -445,6 +445,7 @@ jumbo_component("base") {
     "//ui/strings",
     "//url",
   ]
+  deps += [ "//brave/ui/base:chromium_deps" ]
 
   if (is_debug || dcheck_always_on) {
     deps += [ "//third_party/re2" ]
