diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
index 085ef60055790df6a98f6b6c4b567e6dbf7b9785..fdc6122ad838586cf571929fd118252c2df08578 100644
--- a/ui/base/BUILD.gn
+++ b/ui/base/BUILD.gn
@@ -419,6 +419,7 @@ jumbo_component("base") {
     "//ui/strings",
     "//url",
   ]
+  deps += [ "//brave/ui/base:chromium_deps" ]
 
   if (!is_ios) {
     # iOS does not use Chromium-specific code for event handling.
