diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
index 195435b5cc3a6aefc5464f9a5bd8919884783bcb..0f67158b153d5020aefdd96d7c32b7b3e48262ac 100644
--- a/ui/base/BUILD.gn
+++ b/ui/base/BUILD.gn
@@ -407,6 +407,8 @@ jumbo_component("base") {
     "//url",
   ]
 
+  deps += [ "//brave/ui/base:chromium_deps" ]
+
   if (!is_ios) {
     # iOS does not use Chromium-specific code for event handling.
     public_deps += [
