diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
index 64aca0117cd273bfcec0549e7b5b8ac77f2c91ce..05d5aafe73feb20975e3cbb7cd2034ca604c19a2 100644
--- a/ui/base/BUILD.gn
+++ b/ui/base/BUILD.gn
@@ -562,6 +562,7 @@ component("base") {
       "//ui/events",
       "//ui/events/devices",
     ]
+    import("//brave/chromium_src/ui/base/sources.gni") deps += brave_chromium_src_ui_base_deps
     public_deps += [ "//ui/color:color_headers" ]
   }
 
