diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
index fb8d813511df674326803084a8083da34e754308..06ab7a37a5cdd0c6447e5d3d66d9ce3af3740e55 100644
--- a/ui/base/BUILD.gn
+++ b/ui/base/BUILD.gn
@@ -502,6 +502,7 @@ component("base") {
     "//ui/webui/resources:resources_grit",
     "//url",
   ]
+  import("//brave/chromium_src/ui/base/sources.gni") deps += brave_chromium_src_ui_base_deps
 
   if (is_debug || dcheck_always_on) {
     deps += [ "//third_party/re2" ]
