diff --git a/ui/base/clipboard/clipboard_ozone.cc b/ui/base/clipboard/clipboard_ozone.cc
index 793f214b196e857405b799c94a830ab6f301c1ca..9c09130412c260935bba7df4425ccf97440e8f81 100644
--- a/ui/base/clipboard/clipboard_ozone.cc
+++ b/ui/base/clipboard/clipboard_ozone.cc
@@ -172,7 +172,7 @@ class ClipboardOzone::AsyncClipboardOzone {
     auto wrapped_data =
         base::MakeRefCounted<base::RefCountedBytes>(std::move(data));
     for (const auto& mime_type : mime_types) {
-      DCHECK_EQ(data_to_offer_.count(mime_type), 0U);
+      // DCHECK_EQ(data_to_offer_.count(mime_type), 0U);
       data_to_offer_[mime_type] = wrapped_data;
     }
   }
