diff --git a/ui/base/clipboard/clipboard_ozone.cc b/ui/base/clipboard/clipboard_ozone.cc
index f28e680c4a9bd5a1c9505d1aaf6a67980a83183e..9496030fee43afe8167d9ed8b25ff87669e3b71f 100644
--- a/ui/base/clipboard/clipboard_ozone.cc
+++ b/ui/base/clipboard/clipboard_ozone.cc
@@ -197,7 +197,7 @@ class ClipboardOzone::AsyncClipboardOzone {
     auto wrapped_data =
         base::MakeRefCounted<base::RefCountedBytes>(std::move(data));
     for (const auto& mime_type : mime_types) {
-      DCHECK_EQ(data_to_offer_.count(mime_type), 0U);
+      // DCHECK_EQ(data_to_offer_.count(mime_type), 0U);
       data_to_offer_[mime_type] = wrapped_data;
     }
   }
