diff --git a/ui/base/accelerators/accelerator.cc b/ui/base/accelerators/accelerator.cc
index 3d8645bd5ddac023516578ae600ad23827cccba9..4b0cafefa9e02905c25d7e258ee6ab15313feb98 100644
--- a/ui/base/accelerators/accelerator.cc
+++ b/ui/base/accelerators/accelerator.cc
@@ -393,6 +393,7 @@ std::u16string Accelerator::KeyCodeToName() const {
 }
 
 std::vector<std::u16string> Accelerator::GetLongFormModifiers() const {
+  BRAVE_UI_BASE_ACCELERATOR_GET_LONG_FORM_MODIFIERS
   std::vector<std::u16string> modifiers;
 
   if (IsCmdDown()) {
