diff --git a/ui/color/BUILD.gn b/ui/color/BUILD.gn
index 035c15036cdcdb8df04028eef46f022f28c4c736..3480e78f17cd2323953ed32ecde2932577cb45ed 100644
--- a/ui/color/BUILD.gn
+++ b/ui/color/BUILD.gn
@@ -97,6 +97,9 @@ component("color") {
     ]
     deps += [ ":ui_color_jni_headers" ]
   }
+  import("//brave/ui/color/sources.gni")
+  sources += brave_ui_color_sources
+  public_deps += brave_ui_color_public_deps
 }
 
 source_set("test_support") {
@@ -210,6 +213,9 @@ component("mixers") {
     ]
     deps += [ ":accent_color_observer" ]
   }
+  import("//brave/ui/color/sources.gni")
+  sources += brave_ui_color_mixer_sources
+  public_deps += brave_ui_color_mixer_public_deps
 }
 
 if (is_android) {
