diff --git a/ui/color/BUILD.gn b/ui/color/BUILD.gn
index edee028ec928dedcb5fdee09b54546449a5fd756..779c463c76bb1a5bec9b062e01e710f9c2a8f7e5 100644
--- a/ui/color/BUILD.gn
+++ b/ui/color/BUILD.gn
@@ -84,6 +84,7 @@ component("color") {
     ]
     frameworks = [ "AppKit.framework" ]
   }
+  import("//brave/ui/color/sources.gni") sources += brave_ui_color_sources public_deps += brave_ui_color_public_deps
 }
 
 source_set("test_support") {
@@ -193,6 +194,7 @@ component("mixers") {
     ]
     deps += [ ":accent_color_observer" ]
   }
+  import("//brave/ui/color/sources.gni") sources += brave_ui_color_mixer_sources public_deps += brave_ui_color_mixer_public_deps
 }
 
 mojom("mojom") {
