diff --git a/ui/color/BUILD.gn b/ui/color/BUILD.gn
index 026b757a1f7504a48fc1bf7200c9483053e840f6..188913a670b54e2125586215d679f6286d9b4712 100644
--- a/ui/color/BUILD.gn
+++ b/ui/color/BUILD.gn
@@ -89,6 +89,7 @@ component("color") {
     ]
     frameworks = [ "AppKit.framework" ]
   }
+  import("//brave/ui/color/sources.gni") sources += brave_ui_color_sources public_deps += brave_ui_color_public_deps
 }
 
 source_set("test_support") {
@@ -198,6 +199,7 @@ component("mixers") {
     ]
     deps += [ ":accent_color_observer" ]
   }
+  import("//brave/ui/color/sources.gni") sources += brave_ui_color_mixer_sources public_deps += brave_ui_color_mixer_public_deps
 }
 
 mojom("mojom") {
