diff --git a/ui/gfx/image/image_skia.cc b/ui/gfx/image/image_skia.cc
index 1d56eed3beeeeb28153bac1dfc42db027964f87d..65c4b0d192af3aecf3af573a1475a12b24ff0c00 100644
--- a/ui/gfx/image/image_skia.cc
+++ b/ui/gfx/image/image_skia.cc
@@ -51,10 +51,14 @@ ImageSkiaRep ScaleImageSkiaRep(const ImageSkiaRep& rep, float target_scale) {
 
   gfx::Size scaled_size =
       gfx::ScaleToCeiledSize(rep.pixel_size(), target_scale / rep.scale());
+  SkBitmap resized = skia::ImageOperations::Resize(
+      rep.GetBitmap(), skia::ImageOperations::RESIZE_LANCZOS3,
+      scaled_size.width(), scaled_size.height());
+  if (resized.drawsNothing()) {
+    return rep;
+  }
   return ImageSkiaRep(
-      skia::ImageOperations::Resize(rep.GetBitmap(),
-                                    skia::ImageOperations::RESIZE_LANCZOS3,
-                                    scaled_size.width(), scaled_size.height()),
+      std::move(resized),
       target_scale);
 }
 
