diff --git a/ui/native_theme/native_theme.h b/ui/native_theme/native_theme.h
index c1d4ad9a27d895af32fd0311a640e2f62a30635a..4ca758d582e854590fa4c5c6a7fd2e15858ac8cc 100644
--- a/ui/native_theme/native_theme.h
+++ b/ui/native_theme/native_theme.h
@@ -461,6 +461,7 @@ class NATIVE_THEME_EXPORT NativeTheme {
 
   // Returns the system's caption style.
   virtual base::Optional<CaptionStyle> GetSystemCaptionStyle() const;
+  BRAVE_UI_NATIVE_THEME_NATIVE_THEME_H_
 
   ColorScheme GetSystemColorScheme() const;
 
