diff --git a/ui/native_theme/native_theme_mac.mm b/ui/native_theme/native_theme_mac.mm
index 7e412a4742fa8111c3cdc965449754d87ba1c7b3..a4360ffb616084e7af410a905042c7c2d5e7e8ec 100644
--- a/ui/native_theme/native_theme_mac.mm
+++ b/ui/native_theme/native_theme_mac.mm
@@ -205,6 +205,7 @@ SkColor NativeThemeMac::GetSystemColor(ColorId color_id,
   }
   // Even with --secondary-ui-md, menus use the platform colors and styling, and
   // Mac has a couple of specific color overrides, documented below.
+  GET_BRAVE_COLOR(color_id)
   switch (color_id) {
     case kColorId_EnabledMenuItemForegroundColor:
       return NSSystemColorToSkColor([NSColor controlTextColor]);
