diff --git a/ui/native_theme/native_theme_win.h b/ui/native_theme/native_theme_win.h
index 054c7976685cd0a586d2f521d5fdb72fc88e06f5..9bba0acf1bd7d5700897abb2312eef44c58813b5 100644
--- a/ui/native_theme/native_theme_win.h
+++ b/ui/native_theme/native_theme_win.h
@@ -75,6 +75,7 @@ class NATIVE_THEME_EXPORT NativeThemeWin : public NativeTheme,
   PreferredContrast CalculatePreferredContrast() const override;
   ColorScheme GetDefaultSystemColorScheme() const override;
 
+  BRAVE_UI_NATIVE_THEME_NATIVE_THEME_WIN_H_
  protected:
   friend class NativeTheme;
   friend class base::NoDestructor<NativeThemeWin>;
