diff --git a/ui/native_theme/native_theme_mac.mm b/ui/native_theme/native_theme_mac.mm
index 1fdbd41d57deda4c1ebbb8e364f333d3d5dc6700..ed198eab46fa6a5c6b42ceef86c9658ba891d721 100644
--- a/ui/native_theme/native_theme_mac.mm
+++ b/ui/native_theme/native_theme_mac.mm
@@ -176,6 +176,7 @@ absl::optional<SkColor> NativeThemeMac::GetOSColor(
 
   // Even with --secondary-ui-md, menus use the platform colors and styling, and
   // Mac has a couple of specific color overrides, documented below.
+  GET_BRAVE_COLOR(color_id)
   switch (color_id) {
     case kColorId_EnabledMenuItemForegroundColor:
     case kColorId_HighlightedMenuItemForegroundColor:
