diff --git a/ui/native_theme/native_theme_win.h b/ui/native_theme/native_theme_win.h
index 0ef829dd59f8375b2e17896be1cc512b9320d358..6de82041573a875329fbbc62dad186fda369043f 100644
--- a/ui/native_theme/native_theme_win.h
+++ b/ui/native_theme/native_theme_win.h
@@ -84,6 +84,7 @@ class NATIVE_THEME_EXPORT NativeThemeWin : public NativeTheme,
   PreferredContrast CalculatePreferredContrast() const override;
   ColorScheme GetDefaultSystemColorScheme() const override;
 
+  BRAVE_UI_NATIVE_THEME_NATIVE_THEME_WIN_H_
  protected:
   friend class NativeTheme;
   friend class base::NoDestructor<NativeThemeWin>;
