diff --git a/ui/native_theme/native_theme.h b/ui/native_theme/native_theme.h
index d78379d48c341dde6a10788919e0eb3bfb05c54e..cbad0c93bc65fec1fce555cd7d0f526562ae9189 100644
--- a/ui/native_theme/native_theme.h
+++ b/ui/native_theme/native_theme.h
@@ -463,6 +463,7 @@ class NATIVE_THEME_EXPORT NativeTheme {
 
   // Returns the system's caption style.
   virtual base::Optional<CaptionStyle> GetSystemCaptionStyle() const;
+  BRAVE_UI_NATIVE_THEME_NATIVE_THEME_H_
 
   virtual ColorScheme GetDefaultSystemColorScheme() const;
 
