diff --git a/ui/native_theme/native_theme_win.cc b/ui/native_theme/native_theme_win.cc
index c2f3c20cbe92e877acc30111314a0ee2bce539a1..e2bd868eddbcbf837049cd7fd71c817706027e58 100644
--- a/ui/native_theme/native_theme_win.cc
+++ b/ui/native_theme/native_theme_win.cc
@@ -1586,6 +1586,7 @@ void NativeThemeWin::UpdateDarkModeStatus() {
                                     &apps_use_light_theme);
     dark_mode_enabled = (apps_use_light_theme == 0);
   }
+  BRAVE_NATIVETHEMEWIN_UPDATEDARKMODESTATUS
   set_use_dark_colors(dark_mode_enabled);
   set_preferred_color_scheme(CalculatePreferredColorScheme());
   CloseHandlesInternal();
