diff --git a/ui/native_theme/native_theme.h b/ui/native_theme/native_theme.h
index 20943ab98c28b2689d3e4f548dba95ad0e40b2da..f8907c21d47bf49cad4e9bea9d9aadad87785d15 100644
--- a/ui/native_theme/native_theme.h
+++ b/ui/native_theme/native_theme.h
@@ -455,6 +455,7 @@ class NATIVE_THEME_EXPORT NativeTheme {
 
   // Returns the system's caption style.
   virtual base::Optional<CaptionStyle> GetSystemCaptionStyle() const;
+  BRAVE_UI_NATIVE_THEME_NATIVE_THEME_H_
 
   virtual ColorScheme GetDefaultSystemColorScheme() const;
 
