diff --git a/ui/native_theme/native_theme_win.cc b/ui/native_theme/native_theme_win.cc
index 74357892f7a732de2c8dd6ed018569892c60fd4b..642dd81ef272d788eaf79175081857072fc6b934 100644
--- a/ui/native_theme/native_theme_win.cc
+++ b/ui/native_theme/native_theme_win.cc
@@ -1689,6 +1689,7 @@ void NativeThemeWin::UpdateDarkModeStatus() {
                                     &apps_use_light_theme);
     dark_mode_enabled = (apps_use_light_theme == 0);
   }
+  BRAVE_NATIVETHEMEWIN_UPDATEDARKMODESTATUS
   set_use_dark_colors(dark_mode_enabled);
   set_preferred_color_scheme(CalculatePreferredColorScheme());
   NotifyObservers();
