diff --git a/ui/native_theme/native_theme_win.h b/ui/native_theme/native_theme_win.h
index ed2d981b787b225c69c651fc1a68c9a7facca167..7a8b17a0d0a6a47dee234eab67f76b239aa2eccc 100644
--- a/ui/native_theme/native_theme_win.h
+++ b/ui/native_theme/native_theme_win.h
@@ -76,6 +76,7 @@ class NATIVE_THEME_EXPORT NativeThemeWin : public NativeTheme,
   PreferredContrast CalculatePreferredContrast() const override;
   ColorScheme GetDefaultSystemColorScheme() const override;
 
+  BRAVE_UI_NATIVE_THEME_NATIVE_THEME_WIN_H_
  protected:
   friend class NativeTheme;
   friend class base::NoDestructor<NativeThemeWin>;
