diff --git a/ui/native_theme/native_theme_win.h b/ui/native_theme/native_theme_win.h
index 70084884c4221522e332d5cc96719b4a71d5f7f2..9d029464818c010b2cb030f67f840851cbad0855 100644
--- a/ui/native_theme/native_theme_win.h
+++ b/ui/native_theme/native_theme_win.h
@@ -76,6 +76,7 @@ class NATIVE_THEME_EXPORT NativeThemeWin : public NativeTheme,
   PreferredContrast CalculatePreferredContrast() const override;
   ColorScheme GetDefaultSystemColorScheme() const override;
 
+  BRAVE_UI_NATIVE_THEME_NATIVE_THEME_WIN_H_
  protected:
   friend class NativeTheme;
   friend class base::NoDestructor<NativeThemeWin>;
