diff --git a/ui/native_theme/native_theme.h b/ui/native_theme/native_theme.h
index 82751ea52daaad6346bbbe177959db938908eacf..c4007af0a0629a832a604f2a7f551ae19b470482 100644
--- a/ui/native_theme/native_theme.h
+++ b/ui/native_theme/native_theme.h
@@ -415,6 +415,7 @@ class NATIVE_THEME_EXPORT NativeTheme {
 
   // Returns the system's caption style.
   virtual base::Optional<CaptionStyle> GetSystemCaptionStyle() const;
+  BRAVE_UI_NATIVE_THEME_NATIVE_THEME_H_
 
   virtual ColorScheme GetDefaultSystemColorScheme() const;
 
