diff --git a/ui/native_theme/native_theme.h b/ui/native_theme/native_theme.h
index 0196aa21e880a517461001e7bbb1f3399bbc3b37..94164652a38591de9b9713ea0a2e135593e7bbd3 100644
--- a/ui/native_theme/native_theme.h
+++ b/ui/native_theme/native_theme.h
@@ -455,6 +455,7 @@ class NATIVE_THEME_EXPORT NativeTheme {
 
   // Returns the system's caption style.
   virtual base::Optional<CaptionStyle> GetSystemCaptionStyle() const;
+  BRAVE_UI_NATIVE_THEME_NATIVE_THEME_H_
 
   virtual ColorScheme GetDefaultSystemColorScheme() const;
 
