diff --git a/ui/native_theme/native_theme_win.h b/ui/native_theme/native_theme_win.h
index 7cb873fe61fdc08df500054ba08521bc2f6dc187..0f31806d9813c773a9dfb017ca7a6b43e1c55c18 100644
--- a/ui/native_theme/native_theme_win.h
+++ b/ui/native_theme/native_theme_win.h
@@ -77,6 +77,7 @@ class NATIVE_THEME_EXPORT NativeThemeWin : public NativeTheme,
   PreferredContrast CalculatePreferredContrast() const override;
   ColorScheme GetDefaultSystemColorScheme() const override;
 
+  BRAVE_UI_NATIVE_THEME_NATIVE_THEME_WIN_H_
  protected:
   friend class NativeTheme;
   friend class base::NoDestructor<NativeThemeWin>;
