diff --git a/ui/views/interaction/element_tracker_views.cc b/ui/views/interaction/element_tracker_views.cc
index 9237943bc2580302f514ff841e63adac6c4dd349..4fb64104b15c29e8dbfc977c5c3c9fbeec3d6ce0 100644
--- a/ui/views/interaction/element_tracker_views.cc
+++ b/ui/views/interaction/element_tracker_views.cc
@@ -307,7 +307,7 @@ class ElementTrackerViews::WidgetTracker : public WidgetObserver {
   }
 
   const raw_ptr<ElementTrackerViews> tracker_;
-  const raw_ptr<Widget> widget_;
+  const raw_ptr<Widget, DanglingUntriaged> widget_;
   bool visible_ = false;
   base::ScopedObservation<Widget, WidgetObserver> observation_{this};
 };
