diff --git a/ui/views/controls/menu/menu_item_view.cc b/ui/views/controls/menu/menu_item_view.cc
index f22907bb8d3c841c668ce1f31ad829b4062c3bdf..2d9efb8c7bb330922ac7f1ea9510cc70ba9e5cd7 100644
--- a/ui/views/controls/menu/menu_item_view.cc
+++ b/ui/views/controls/menu/menu_item_view.cc
@@ -1655,7 +1655,7 @@ void MenuItemView::UpdateSelectionBasedState(bool paint_as_selected) {
         features::IsRoundedIconsEnabled()
             ? vector_icons::kKeyboardArrowRightFlippableIcon
             : vector_icons::kSubmenuArrowChromeRefreshOldIcon,
-        colors.icon_color));
+        colors.icon_color, 16));
   }
   MenuDelegate* delegate = GetDelegate();
   if (type_ == Type::kCheckbox && delegate &&
