diff --git a/ui/views/interaction/element_tracker_views.cc b/ui/views/interaction/element_tracker_views.cc
index c587533d918928f11994c775d618d88c023a91e1..e09c1243ab37190623dbd7f52b90353d76d658b4 100644
--- a/ui/views/interaction/element_tracker_views.cc
+++ b/ui/views/interaction/element_tracker_views.cc
@@ -300,7 +300,7 @@ class ElementTrackerViews::WidgetTracker : public WidgetObserver {
   }
 
   const raw_ptr<ElementTrackerViews> tracker_;
-  const raw_ptr<Widget> widget_;
+  const raw_ptr<Widget, DanglingUntriaged> widget_;
   bool visible_ = false;
   base::ScopedObservation<Widget, WidgetObserver> observation_{this};
 };
