diff --git a/ui/views/controls/button/md_text_button.cc b/ui/views/controls/button/md_text_button.cc
index 7e42f93386e5cea6dca0a60f5005ff3d3f7298fb..ee0decd9fa572e058029f09f304d53500d6f0a57 100644
--- a/ui/views/controls/button/md_text_button.cc
+++ b/ui/views/controls/button/md_text_button.cc
@@ -250,6 +250,7 @@ void MdTextButton::UpdateColors() {
     UpdateTextColor();
     UpdateBackgroundColor();
     SchedulePaint();
+    BRAVE_MD_TEXT_BUTTON_UPDATE_COLORS
   }
 }
 
