diff --git a/ui/views/controls/button/md_text_button.cc b/ui/views/controls/button/md_text_button.cc
index 2110ef33b92bd6535bf243c03ae0fe2ee7cea5d4..8214c1cb8db1ccd3d9a6a5c347b91af13b5892b0 100644
--- a/ui/views/controls/button/md_text_button.cc
+++ b/ui/views/controls/button/md_text_button.cc
@@ -250,6 +250,7 @@ void MdTextButton::UpdateColors() {
     UpdateTextColor();
     UpdateBackgroundColor();
     SchedulePaint();
+    BRAVE_MD_TEXT_BUTTON_UPDATE_COLORS
   }
 }
 
