diff --git a/ui/views/controls/button/md_text_button.h b/ui/views/controls/button/md_text_button.h
index 81b342becf7658c68d54394eab6a79caa34be0fc..67789fcfb094f507fcf467aa59ac587f44da6172 100644
--- a/ui/views/controls/button/md_text_button.h
+++ b/ui/views/controls/button/md_text_button.h
@@ -57,11 +57,13 @@ class VIEWS_EXPORT MdTextButton : public LabelButton {
   void OnBlur() override;
 
  private:
+  friend class BraveTextButton;
   void UpdatePadding();
   gfx::Insets CalculateDefaultPadding() const;
 
   void UpdateTextColor();
   void UpdateBackgroundColor() override;
+  virtual
   void UpdateColors();
 
   // True if this button uses prominent styling (blue fill, etc.).
