diff --git a/ui/views/controls/menu/menu_separator.cc b/ui/views/controls/menu/menu_separator.cc
index dd7d1dee175a582b957eefbce24eacdda4e18fda..040d746f862ffc78470d43bc62ed8e6fc2ca678b 100644
--- a/ui/views/controls/menu/menu_separator.cc
+++ b/ui/views/controls/menu/menu_separator.cc
@@ -65,6 +65,7 @@ void MenuSeparator::OnPaint(gfx::Canvas* canvas) {
 
   gfx::Rect paint_rect(horizontal_padding, y, width() - horizontal_padding * 2,
                        separator_thickness);
+  BRAVE_MENU_SEPARATOR_ON_PAINT
   if (type_ == ui::PADDED_SEPARATOR) {
     paint_rect.Inset(
         gfx::Insets::TLBR(0, menu_config.padded_separator_start_padding, 0, 0));
