diff --git a/ui/views/controls/button/md_text_button.h b/ui/views/controls/button/md_text_button.h
index 15818a3e2e7d94ee4877c26ec467a812b6ee7833..613a5d466210cc09b44e4d3a60c53a857d0b8695 100644
--- a/ui/views/controls/button/md_text_button.h
+++ b/ui/views/controls/button/md_text_button.h
@@ -60,11 +60,13 @@ class VIEWS_EXPORT MdTextButton : public LabelButton {
   void OnBlur() override;
 
  private:
+ BRAVE_MD_TEXT_BUTTON_H_
   void UpdatePadding();
   gfx::Insets CalculateDefaultPadding() const;
 
   void UpdateTextColor();
   void UpdateBackgroundColor() override;
+  virtual
   void UpdateColors();
 
   // True if this button uses prominent styling (blue fill, etc.).
