diff --git a/ui/views/widget/widget.h b/ui/views/widget/widget.h
index 4b71f0122b45668b0e3c0be11c1b0ef909273e4d..fced4695f32bfc849a10b94861ec80754996049e 100644
--- a/ui/views/widget/widget.h
+++ b/ui/views/widget/widget.h
@@ -658,7 +658,7 @@ class VIEWS_EXPORT Widget : public internal::NativeWidgetDelegate,
 
   // Initializes the widget, and in turn, the native widget. |params| should be
   // moved to Init() by the caller.
-  void Init(InitParams params);
+  virtual void Init(InitParams params);
 
   // Returns the gfx::NativeView associated with this Widget.
   gfx::NativeView GetNativeView() const;
