diff --git a/ui/views/view.h b/ui/views/view.h
index b9f2a484669f128c27d37a31a58accfd88dfe2ae..62d2f942a37b42511199120568b7260c4716a3db 100644
--- a/ui/views/view.h
+++ b/ui/views/view.h
@@ -326,6 +326,7 @@ class VIEWS_EXPORT View : public ui::LayerDelegate,
     FRIEND_TEST_ALL_PREFIXES(WebViewUnitTest, CrashedOverlayView);
 
     OwnedByClientPassKey() = default;
+    BRAVE_VIEW_OWNED_BY_CLIENT_PASS_KEY
   };
 
   using PassKey = base::NonCopyablePassKey<View>;
