diff --git a/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc b/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
index 752736386da0357a290a61b0990b0262b935b2db..94cfd84df18fc4acbd6523b5e94385e1ae3034d2 100644
--- a/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
+++ b/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
@@ -552,10 +552,10 @@ void DesktopNativeWidgetAura::UpdateWindowTransparency() {
       desktop_window_tree_host_->ShouldWindowContentsBeTransparent();
 
   auto* window_tree_host = desktop_window_tree_host_->AsWindowTreeHost();
-  window_tree_host->compositor()->SetBackgroundColor(
+  window_tree_host->compositor()->SetBackgroundColor(desktop_window_tree_host_->GetBackgroundColor(
       transparent
           ? SK_ColorTRANSPARENT
-          : GetWidget()->GetColorProvider()->GetColor(ui::kColorFrameActive));
+          : GetWidget()->GetColorProvider()->GetColor(ui::kColorFrameActive)));
   window_tree_host->window()->SetTransparent(transparent);
 
   content_window_->SetTransparent(transparent);
