diff --git a/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc b/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
index 212bb087647fffee9b9a73a6099fb6bec47f8baa..8f9704c728da13948b4affa1f90d3e5a43507c48 100644
--- a/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
+++ b/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
@@ -549,10 +549,10 @@ void DesktopNativeWidgetAura::UpdateWindowTransparency() {
       desktop_window_tree_host_->ShouldWindowContentsBeTransparent();
 
   auto* window_tree_host = desktop_window_tree_host_->AsWindowTreeHost();
-  window_tree_host->compositor()->SetBackgroundColor(
+  window_tree_host->compositor()->SetBackgroundColor(desktop_window_tree_host_->GetBackgroundColor(
       transparent
           ? SK_ColorTRANSPARENT
-          : GetWidget()->GetColorProvider()->GetColor(ui::kColorFrameActive));
+          : GetWidget()->GetColorProvider()->GetColor(ui::kColorFrameActive)));
   window_tree_host->window()->SetTransparent(transparent);
 
   content_window_->SetTransparent(transparent);
