diff --git a/ui/views/view.h b/ui/views/view.h
index b3fa15b412fee09234839cb87d26b881697033fe..a510f7014338d4032715cbda5ca52078b95f9585 100644
--- a/ui/views/view.h
+++ b/ui/views/view.h
@@ -342,6 +342,7 @@ class VIEWS_EXPORT View : public ui::LayerDelegate,
     FRIEND_TEST_ALL_PREFIXES(WebViewUnitTest, CrashedOverlayView);
 
     OwnedByClientPassKey() = default;
+    BRAVE_VIEW_OWNED_BY_CLIENT_PASS_KEY
   };
 
   using PassKey = base::NonCopyablePassKey<View>;
