diff --git a/ui/views/layout/layout_provider.cc b/ui/views/layout/layout_provider.cc
index 3684e277668347e8d4d6d08097bd8db429dcc26c..088ce543138d3e0b3a89886dacc55e59b07d328a 100644
--- a/ui/views/layout/layout_provider.cc
+++ b/ui/views/layout/layout_provider.cc
@@ -224,7 +224,7 @@ int LayoutProvider::GetCornerRadiusMetric(ShapeContextTokens id,
       case ShapeContextTokens::kMenuTouchRadius:
         return GetCornerRadiusMetric(Emphasis::kHigh);
       case ShapeContextTokens::kOmniboxExpandedRadius:
-        return 16;
+        return 4;
       case ShapeContextTokens::kTextfieldRadius:
         return FocusRing::kDefaultCornerRadiusDp;
       case ShapeContextTokens::kSidePanelContentRadius:
