diff --git a/ui/views/bubble/bubble_frame_view.cc b/ui/views/bubble/bubble_frame_view.cc
index d576b8a33de6f80787720d06376527ec95024f9f..8245d2c7f7af7bc84ace5413d3fe06f6226d7802 100644
--- a/ui/views/bubble/bubble_frame_view.cc
+++ b/ui/views/bubble/bubble_frame_view.cc
@@ -1199,7 +1199,8 @@ int BubbleFrameView::GetFrameWidthForClientWidth(int client_width) const {
       GetWidget()->widget_delegate()->AsDialogDelegate();
   bool snapping =
       dialog_delegate && dialog_delegate->buttons() !=
-                             static_cast<int>(ui::mojom::DialogButton::kNone);
+                             static_cast<int>(ui::mojom::DialogButton::kNone) &&
+                             !dialog_delegate->should_ignore_snapping();
   return snapping ? LayoutProvider::Get()->GetSnappedDialogWidth(frame_width)
                   : frame_width;
 }
