diff --git a/ui/views/controls/button/md_text_button.h b/ui/views/controls/button/md_text_button.h
index 90ae3a47b7821f78c091c5a0d796556408f57dce..c5bcddd7b6463ab8c7e43d1f9eb709445f81a8b9 100644
--- a/ui/views/controls/button/md_text_button.h
+++ b/ui/views/controls/button/md_text_button.h
@@ -65,7 +65,9 @@ class VIEWS_EXPORT MdTextButton : public LabelButton {
   MdTextButton(ButtonListener* listener, int button_context);
 
  private:
+  friend class BraveTextButton;
   void UpdatePadding();
+  virtual
   void UpdateColors();
 
   // True if this button uses prominent styling (blue fill, etc.).
