diff --git a/ui/views/controls/menu/menu_item_view.cc b/ui/views/controls/menu/menu_item_view.cc
index f68316aec94d22ff1d21ad55aede1531777bf5d7..31d4815d0a1732b4e19067a62a00a2ee9d292ea0 100644
--- a/ui/views/controls/menu/menu_item_view.cc
+++ b/ui/views/controls/menu/menu_item_view.cc
@@ -1761,7 +1761,7 @@ void MenuItemView::UpdateSelectionBasedState(bool paint_as_selected) {
         features::IsRoundedIconsEnabled()
             ? vector_icons::kKeyboardArrowRightFlippableIcon
             : vector_icons::kSubmenuArrowChromeRefreshOldIcon,
-        colors.icon_color));
+        colors.icon_color, 16));
   }
   MenuDelegate* delegate = GetDelegate();
   if (type_ == Type::kCheckbox && delegate &&
