diff --git a/ui/views/view.h b/ui/views/view.h
index 3d016a2264721cc6e10efb506181962023b66bb0..ada755128e6ab3723759ee88a1a85bff4b85b437 100644
--- a/ui/views/view.h
+++ b/ui/views/view.h
@@ -322,6 +322,7 @@ class VIEWS_EXPORT View : public ui::LayerDelegate,
     FRIEND_TEST_ALL_PREFIXES(WebViewUnitTest, CrashedOverlayView);
 
     OwnedByClientPassKey() = default;
+    BRAVE_VIEW_OWNED_BY_CLIENT_PASS_KEY
   };
 
   using PassKey = base::NonCopyablePassKey<View>;
