diff --git a/ui/views/view.h b/ui/views/view.h
index 9860a9ec7714c69222a1d15d92b78c9bcb1df138..fcd46acf0a7406d0538f6651d9952ebd56162f1a 100644
--- a/ui/views/view.h
+++ b/ui/views/view.h
@@ -321,6 +321,7 @@ class VIEWS_EXPORT View : public ui::LayerDelegate,
     FRIEND_TEST_ALL_PREFIXES(WebViewUnitTest, CrashedOverlayView);
 
     OwnedByClientPassKey() = default;
+    BRAVE_VIEW_OWNED_BY_CLIENT_PASS_KEY
   };
 
   using PassKey = base::NonCopyablePassKey<View>;
