diff --git a/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc b/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
index 446d82985dab4486f876fb957caea143de98a7ba..508269bb9fd9c41e43c0aef00cb5ffa19abe3c71 100644
--- a/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
+++ b/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
@@ -540,10 +540,10 @@ void DesktopNativeWidgetAura::UpdateWindowTransparency() {
       desktop_window_tree_host_->ShouldWindowContentsBeTransparent();
 
   auto* window_tree_host = desktop_window_tree_host_->AsWindowTreeHost();
-  window_tree_host->compositor()->SetBackgroundColor(
+  window_tree_host->compositor()->SetBackgroundColor(desktop_window_tree_host_->GetBackgroundColor(
       transparent
           ? SK_ColorTRANSPARENT
-          : GetWidget()->GetColorProvider()->GetColor(ui::kColorFrameActive));
+          : GetWidget()->GetColorProvider()->GetColor(ui::kColorFrameActive)));
   window_tree_host->window()->SetTransparent(transparent);
 
   content_window_->SetTransparent(transparent);
