diff --git a/ui/views/interaction/element_tracker_views.cc b/ui/views/interaction/element_tracker_views.cc
index f2f6795858d41a1f6c5685e700fdb65f04938c88..bc045f9544ff50f9a4fe8ce993fde4441f2e2497 100644
--- a/ui/views/interaction/element_tracker_views.cc
+++ b/ui/views/interaction/element_tracker_views.cc
@@ -292,7 +292,7 @@ class ElementTrackerViews::WidgetTracker : public WidgetObserver {
   }
 
   const raw_ptr<ElementTrackerViews> tracker_;
-  const raw_ptr<Widget> widget_;
+  const raw_ptr<Widget, DanglingUntriaged> widget_;
   bool visible_ = false;
   base::ScopedObservation<Widget, WidgetObserver> observation_{this};
 };
