diff --git a/ui/views/accessibility/accessibility_paint_checks.cc b/ui/views/accessibility/accessibility_paint_checks.cc
index 97f92310782792e22592ff8d76b58df5fe90e651..76418cee28053fec05b43218e7b93e239f6229cf 100644
--- a/ui/views/accessibility/accessibility_paint_checks.cc
+++ b/ui/views/accessibility/accessibility_paint_checks.cc
@@ -33,6 +33,7 @@ void RunAccessibilityPaintChecks(View* view) {
   // some additional fields are processed and set there.
   ui::AXNodeData node_data;
   view->GetViewAccessibility().GetAccessibleNodeData(&node_data);
+  node_data.SetNameExplicitlyEmpty();
 
   // No checks for unfocusable items yet.
   if (!node_data.HasState(ax::mojom::State::kFocusable))
