diff --git a/ui/views/controls/menu/menu_item_view.cc b/ui/views/controls/menu/menu_item_view.cc
index d3a0cf726c99589c7a0cf10b8aac8e117e4b0b8a..59e0a8aef87bb7874aed8c84a6830640f38b91c9 100644
--- a/ui/views/controls/menu/menu_item_view.cc
+++ b/ui/views/controls/menu/menu_item_view.cc
@@ -1801,7 +1801,7 @@ void MenuItemView::UpdateSelectionBasedState(bool paint_as_selected) {
         features::IsRoundedIconsEnabled()
             ? vector_icons::kKeyboardArrowRightFlippableIcon
             : vector_icons::kSubmenuArrowChromeRefreshOldIcon,
-        colors.icon_color));
+        colors.icon_color, 16));
   }
   MenuDelegate* delegate = GetDelegate();
   if (type_ == Type::kCheckbox && delegate &&
