diff --git a/ui/views/controls/focus_ring.cc b/ui/views/controls/focus_ring.cc
index 4e684c2533128f5c6478fc183932057e85c75aec..6dc36e8975d1df78ec2a6d8d37171aedb47693f7 100644
--- a/ui/views/controls/focus_ring.cc
+++ b/ui/views/controls/focus_ring.cc
@@ -215,6 +215,7 @@ void FocusRing::OnPaint(gfx::Canvas* canvas) {
   }
 
   paint.setColor(color_.value_or(GetColor(this, !invalid_)));
+  BRAVE_FOCUS_RING_ON_PAINT_SET_COLOR_VALIDITY
   paint.setStrokeWidth(FocusRing::kHaloThickness);
   canvas->sk_canvas()->drawRRect(ring_rect, paint);
 }
