diff --git a/ui/views/controls/focus_ring.cc b/ui/views/controls/focus_ring.cc
index e93896acce4b4ac3e3f5e2916a8a0c888bd132b8..cc3bf6d852b8e758fb8fef6edfdfa62f2ae30f8f 100644
--- a/ui/views/controls/focus_ring.cc
+++ b/ui/views/controls/focus_ring.cc
@@ -204,6 +204,7 @@ void FocusRing::OnPaint(gfx::Canvas* canvas) {
   cc::PaintFlags paint;
   paint.setAntiAlias(true);
   paint.setColor(color_.value_or(GetColor(this, !invalid_)));
+  BRAVE_FOCUS_RING_ON_PAINT_SET_COLOR_VALIDITY
   paint.setStyle(cc::PaintFlags::kStroke_Style);
   paint.setStrokeWidth(PlatformStyle::kFocusHaloThickness);
 
