diff --git a/ui/views/view.h b/ui/views/view.h
index 4fada89e37b9c63f7690fbbdd2ef4ecf86b5d756..6a704e6f22fcedb6c44f73a8f85782009b43260f 100644
--- a/ui/views/view.h
+++ b/ui/views/view.h
@@ -343,6 +343,7 @@ class VIEWS_EXPORT View : public ui::LayerDelegate,
     FRIEND_TEST_ALL_PREFIXES(WebViewUnitTest, CrashedOverlayView);
 
     OwnedByClientPassKey() = default;
+    BRAVE_VIEW_OWNED_BY_CLIENT_PASS_KEY
   };
 
   using PassKey = base::NonCopyablePassKey<View>;
