diff --git a/ui/views/controls/button/md_text_button.h b/ui/views/controls/button/md_text_button.h
index ebe4f966fe6d8f4a57c79b95a4b475c7d516169d..1629ee72d652382c8365c495136ce36ab898383e 100644
--- a/ui/views/controls/button/md_text_button.h
+++ b/ui/views/controls/button/md_text_button.h
@@ -55,11 +55,13 @@ class VIEWS_EXPORT MdTextButton : public LabelButton {
   void OnBlur() override;
 
  private:
+ BRAVE_MD_TEXT_BUTTON_H_
   void UpdatePadding();
   gfx::Insets CalculateDefaultPadding() const;
 
   void UpdateTextColor();
   void UpdateBackgroundColor() override;
+  virtual
   void UpdateColors();
 
   // True if this button uses prominent styling (blue fill, etc.).
