diff --git a/ui/views/controls/button/md_text_button.h b/ui/views/controls/button/md_text_button.h
index 1d293a70166cdc702d33789ffab28674c5e6f169..4af09cdc276cf5cb1423c73f5fb8858f7b51384c 100644
--- a/ui/views/controls/button/md_text_button.h
+++ b/ui/views/controls/button/md_text_button.h
@@ -60,7 +60,9 @@ class VIEWS_EXPORT MdTextButton : public LabelButton {
   MdTextButton(ButtonListener* listener, int button_context);
 
  private:
+  friend class BraveTextButton;
   void UpdatePadding();
+  virtual
   void UpdateColors();
   gfx::Insets CalculateDefaultPadding() const;
 
