diff --git a/ui/views/view.h b/ui/views/view.h
index e0ce0ffb230380f8ad70a2645c1095adf6e31cf5..f8b479af588fdd834ea634825120a08da9c88ddd 100644
--- a/ui/views/view.h
+++ b/ui/views/view.h
@@ -321,6 +321,7 @@ class VIEWS_EXPORT View : public ui::LayerDelegate,
     FRIEND_TEST_ALL_PREFIXES(WebViewUnitTest, CrashedOverlayView);
 
     OwnedByClientPassKey() = default;
+    BRAVE_VIEW_OWNED_BY_CLIENT_PASS_KEY
   };
 
   using PassKey = base::NonCopyablePassKey<View>;
