diff --git a/ui/views/bubble/bubble_dialog_delegate_view.h b/ui/views/bubble/bubble_dialog_delegate_view.h
index 3cd8702c9f995d825274cdf3002bfb4944034ed6..4f18bb159ddfba6075900a33b417e1775958d43b 100644
--- a/ui/views/bubble/bubble_dialog_delegate_view.h
+++ b/ui/views/bubble/bubble_dialog_delegate_view.h
@@ -870,6 +870,15 @@ class VIEWS_EXPORT BubbleDialogDelegateView : public View,
   gfx::Size GetMaximumSize() const final;
 
  private:
+  friend class ::BraveNewsBubbleView;
+  friend class ::BraveHelpBubbleDelegateView;
+  friend class ::WaybackMachineBubbleView;
+  friend class ::playlist::PlaylistBubbleView;
+  friend class ::SplitViewMenuBubble;
+  friend class ::SidebarItemAddedFeedbackBubble;
+  friend class ::SidebarEditItemBubbleDelegateView;
+  friend class ::SidebarAddItemBubbleDelegateView;
+  friend class ::views::BraveBubbleDialogDelegateView;
   FRIEND_TEST_ALL_PREFIXES(BubbleDelegateTest, CreateDelegate);
   FRIEND_TEST_ALL_PREFIXES(BubbleDelegateTest, NonClientHitTest);
 
