diff --git a/ui/views/interaction/element_tracker_views.cc b/ui/views/interaction/element_tracker_views.cc
index 080b217f51f5aa0221204986b59af9624dace05a..105cdc1f8fe788001650287f3d57c6304a6090d8 100644
--- a/ui/views/interaction/element_tracker_views.cc
+++ b/ui/views/interaction/element_tracker_views.cc
@@ -295,7 +295,7 @@ class ElementTrackerViews::WidgetTracker : public WidgetObserver {
   }
 
   const raw_ptr<ElementTrackerViews> tracker_;
-  const raw_ptr<Widget> widget_;
+  const raw_ptr<Widget, DanglingUntriaged> widget_;
   bool visible_ = false;
   base::ScopedObservation<Widget, WidgetObserver> observation_{this};
 };
