diff --git a/ui/views/controls/menu/menu_model_adapter.cc b/ui/views/controls/menu/menu_model_adapter.cc
index 895745cf0f7e55af19cdf2c27dfd6c0fa8dd9ba1..76e55c966fe91bcc02fbbf770ed0d88e9b12e3ed 100644
--- a/ui/views/controls/menu/menu_model_adapter.cc
+++ b/ui/views/controls/menu/menu_model_adapter.cc
@@ -100,6 +100,9 @@ MenuItemView* MenuModelAdapter::AddMenuItemFromModelAt(ui::MenuModel* model,
     case ui::MenuModel::TYPE_HIGHLIGHTED:
       type = MenuItemView::Type::kHighlighted;
       break;
+    case ui::MenuModel::TYPE_TOGGLE:
+      type = MenuItemView::Type::kToggle;
+      break;
   }
 
   if (*type == MenuItemView::Type::kSeparator) {
