diff --git a/ui/views/view.h b/ui/views/view.h
index 5d89a7ce81b2df250c02601a45eaec93e3cc171d..43c98bda74a770562205eae2d56cf0cd013db91d 100644
--- a/ui/views/view.h
+++ b/ui/views/view.h
@@ -343,6 +343,7 @@ class VIEWS_EXPORT View : public ui::LayerDelegate,
     FRIEND_TEST_ALL_PREFIXES(WebViewUnitTest, CrashedOverlayView);
 
     OwnedByClientPassKey() = default;
+    BRAVE_VIEW_OWNED_BY_CLIENT_PASS_KEY
   };
 
   using PassKey = base::NonCopyablePassKey<View>;
