diff --git a/ui/views/view.h b/ui/views/view.h
index 07449516b6e97d971a78acb3d21e9fd2c35f6d54..4e249471e8b5eb562b83552d461e3e665b2a2284 100644
--- a/ui/views/view.h
+++ b/ui/views/view.h
@@ -326,6 +326,7 @@ class VIEWS_EXPORT View : public ui::LayerDelegate,
     FRIEND_TEST_ALL_PREFIXES(WebViewUnitTest, CrashedOverlayView);
 
     OwnedByClientPassKey() = default;
+    BRAVE_VIEW_OWNED_BY_CLIENT_PASS_KEY
   };
 
   using PassKey = base::NonCopyablePassKey<View>;
