diff --git a/ui/views/controls/button/md_text_button.cc b/ui/views/controls/button/md_text_button.cc
index 868a79a37bde8cde0c2a46d298887ac2fa40481a..9061d8fd45efc971fb3ad71268ce50e2e35f9da1 100644
--- a/ui/views/controls/button/md_text_button.cc
+++ b/ui/views/controls/button/md_text_button.cc
@@ -256,6 +256,7 @@ void MdTextButton::UpdateColors() {
   UpdateTextColor();
   UpdateBackgroundColor();
   SchedulePaint();
+  BRAVE_MD_TEXT_BUTTON_UPDATE_COLORS
 }
 
 BEGIN_METADATA(MdTextButton, LabelButton)
