diff --git a/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc b/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
index 04b22c7f1ddbb747645ae71acc599704768e2ab6..cf2641276cf4201bf98dcf2f2614d08aad37c9b5 100644
--- a/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
+++ b/ui/views/widget/desktop_aura/desktop_native_widget_aura.cc
@@ -537,10 +537,10 @@ void DesktopNativeWidgetAura::UpdateWindowTransparency() {
       desktop_window_tree_host_->ShouldWindowContentsBeTransparent();
 
   auto* window_tree_host = desktop_window_tree_host_->AsWindowTreeHost();
-  window_tree_host->compositor()->SetBackgroundColor(
+  window_tree_host->compositor()->SetBackgroundColor(desktop_window_tree_host_->GetBackgroundColor(
       transparent
           ? SK_ColorTRANSPARENT
-          : GetWidget()->GetColorProvider()->GetColor(ui::kColorFrameActive));
+          : GetWidget()->GetColorProvider()->GetColor(ui::kColorFrameActive)));
   window_tree_host->window()->SetTransparent(transparent);
 
   content_window_->SetTransparent(transparent);
