diff --git a/ui/webui/resources/BUILD.gn b/ui/webui/resources/BUILD.gn
index 57c676cf539d12adc74f2e982c1b9dbfb54d7217..307a8e3968658119c3c35551cf3d93d947fb9e0a 100644
--- a/ui/webui/resources/BUILD.gn
+++ b/ui/webui/resources/BUILD.gn
@@ -138,6 +138,7 @@ generate_grd("build_grd") {
       "$root_gen_dir/ui/webui/resources/cr_components/page_image_service/resources.grdp",
     ]
   }
+  import("//brave/ui/webui/resources/sources.gni") grdp_files += brave_resources_extra_grdps public_deps += brave_resources_extra_grdps_deps
 }
 
 if (is_chromeos) {
