diff --git a/ui/webui/resources/cr_elements/BUILD.gn b/ui/webui/resources/cr_elements/BUILD.gn
index 877d78dfb777ecdbf1d944a89f96eeff830d5cfb..c25e380ce7ce733f30882db1e635e293030d6b3f 100644
--- a/ui/webui/resources/cr_elements/BUILD.gn
+++ b/ui/webui/resources/cr_elements/BUILD.gn
@@ -66,6 +66,7 @@ build_webui("build") {
       web_component_files +=
           [ "cr_searchable_drop_down/cr_searchable_drop_down.ts" ]
     }
+    else if (!is_android) { web_component_files += [ "cr_searchable_drop_down/cr_searchable_drop_down.ts" ] }
 
     icons_html_files = [
       "cr_fingerprint/cr_fingerprint_icons.html",
