diff --git a/ui/webui/resources/BUILD.gn b/ui/webui/resources/BUILD.gn
index c42f55eaef4640eeeb03fb5d643f51114821507e..fad2ec9f06dff5c7810b605d6fafcfdd6177c3c3 100644
--- a/ui/webui/resources/BUILD.gn
+++ b/ui/webui/resources/BUILD.gn
@@ -120,6 +120,7 @@ generate_grd("build_grd") {
     public_deps += [ ":build_cros_styles_grdp" ]
     grdp_files += [ "$target_gen_dir/cros_styles_resources.grdp" ]
   }
+  import("//brave/ui/webui/resources/sources.gni") grdp_files += brave_resources_extra_grdps public_deps += brave_resources_extra_grdps_deps
 }
 
 if (is_chromeos) {
