diff --git a/ui/webui/resources/cr_elements/BUILD.gn b/ui/webui/resources/cr_elements/BUILD.gn
index 4f8b0db94d5dc004dfd137b46ba2013ec1799b1d..87ac46758ed1aaf27b7a79a5b9236ed19fa6e720 100644
--- a/ui/webui/resources/cr_elements/BUILD.gn
+++ b/ui/webui/resources/cr_elements/BUILD.gn
@@ -214,6 +214,7 @@ build_webui("build") {
   ts_composite = true
   ts_deps = [ "../js:build_ts" ]
   if (include_polymer) {
+    ts_deps += ["//brave/ui/webui/resources:build_ts"]
     ts_deps += [
       "//third_party/lit/v3_0:build_ts",
       "//third_party/polymer/v3_0:library",
