diff --git a/ui/webui/resources/cr_elements/BUILD.gn b/ui/webui/resources/cr_elements/BUILD.gn
index aaae8e16f2c2b79e3a05bc5d78077af95e836f32..fecf0b00abbc38db6ea2885644ef603f55c1cead 100644
--- a/ui/webui/resources/cr_elements/BUILD.gn
+++ b/ui/webui/resources/cr_elements/BUILD.gn
@@ -233,6 +233,7 @@ build_webui("build") {
     "//third_party/lit/v3_0:build_ts",
   ]
   if (!is_android && !is_ios) {
+    ts_deps += ["//brave/ui/webui/resources:build_ts"]
     ts_deps += [ "//third_party/polymer/v3_0:library" ]
   }
   ts_definitions = [
