diff --git a/ui/webui/resources/js/web_ui_listener_behavior.js b/ui/webui/resources/js/web_ui_listener_behavior.js
index 3c89e4ce0e936a84ca06dbc6a5e8b55274db4208..017b0b385a095c5b77d67bd96dd2b3e2c1345390 100644
--- a/ui/webui/resources/js/web_ui_listener_behavior.js
+++ b/ui/webui/resources/js/web_ui_listener_behavior.js
@@ -10,6 +10,7 @@
  */
 
 // #import {WebUIListener, addWebUIListener, removeWebUIListener} from './cr.m.js';
+// #import 'chrome://brave-resources/polymer_overriding.js';
 
 /** @polymerBehavior */
 // eslint-disable-next-line no-var
