diff --git a/ui/webui/resources/tools/build_webui.gni b/ui/webui/resources/tools/build_webui.gni
index bad7ce61ef4d1db771a05d3de020b8d7b31f4693..75a0e061aedb0bc706892a259d98c8188e79f0ab 100644
--- a/ui/webui/resources/tools/build_webui.gni
+++ b/ui/webui/resources/tools/build_webui.gni
@@ -173,6 +173,7 @@ template("build_webui") {
       enable_removal_comments = enable_source_maps
     }
     in_files = ts_files
+    if (defined(invoker.exclude_ts_preprocess_files)) { in_files -= invoker.exclude_ts_preprocess_files } if (defined(invoker.preprocess_deps)) { public_deps = invoker.preprocess_deps }
   }
 
   if (enable_source_maps) {
@@ -224,6 +225,7 @@ template("build_webui") {
       if (defined(invoker.css_files)) {
         in_files += invoker.css_files
       }
+      if (defined(invoker.exclude_html_css_preprocess_files)) { in_files -= invoker.exclude_html_css_preprocess_files }
     }
   }
 
@@ -250,6 +252,7 @@ template("build_webui") {
           }
         }
       }
+      if (defined(invoker.preprocess_deps)) { public_deps = invoker.preprocess_deps }
     }
   }
 
