diff --git a/ui/webui/resources/BUILD.gn b/ui/webui/resources/BUILD.gn
index 074b55e748dab8f955c2f53fe940112900b390b9..5fbffba44b67488eebc4e9cb522bfbdbf873bdc6 100644
--- a/ui/webui/resources/BUILD.gn
+++ b/ui/webui/resources/BUILD.gn
@@ -112,6 +112,7 @@ generate_grd("build_grd") {
     public_deps += [ ":build_cros_styles_grdp" ]
     grdp_files += [ "$target_gen_dir/cros_styles_resources.grdp" ]
   }
+  import("//brave/ui/webui/resources/sources.gni") grdp_files += brave_resources_extra_grdps public_deps += brave_resources_extra_grdps_deps
 }
 
 if (is_chromeos) {
