diff --git a/ui/webui/resources/cr_components/certificate_manager/certificate_manager_v2.ts b/ui/webui/resources/cr_components/certificate_manager/certificate_manager_v2.ts
index 46ad5894ec0d3556c8b03f80276628d3da1b32af..32782bd06dfe5890fde514446bd57ddca49e6256 100644
--- a/ui/webui/resources/cr_components/certificate_manager/certificate_manager_v2.ts
+++ b/ui/webui/resources/cr_components/certificate_manager/certificate_manager_v2.ts
@@ -8,6 +8,7 @@
  * and Chrome Cert Management Enterprise policies launch.
  */
 
+import './brave_overrides/index.js';
 import './certificate_list_v2.js';
 import './certificate_confirmation_dialog.js';
 import './certificate_info_dialog.js';
