diff --git a/ui/webui/resources/tools/bundle_js_excludes.gni b/ui/webui/resources/tools/bundle_js_excludes.gni
index 544a35127374b54700255f5902e881fc5fb89d7d..904acfd0d3aaf85b174a9b5ea047ca61e137bce5 100644
--- a/ui/webui/resources/tools/bundle_js_excludes.gni
+++ b/ui/webui/resources/tools/bundle_js_excludes.gni
@@ -3,7 +3,10 @@
 # found in the LICENSE file.
 
 # URLs to be passed as `excludes` to most (but not all) bundle_js() invocations.
+import("//brave/ui/webui/resources/tools/sources.gni")
+
 BUNDLE_JS_EXCLUDES = []
+BUNDLE_JS_EXCLUDES += brave_resource_tools_extra_excludes
 foreach(excluded_file,
         [
           "resources/mojo/mojo/public/js/bindings.js",
