diff --git a/ui/webui/resources/tools/bundle_js.gni b/ui/webui/resources/tools/bundle_js.gni
index 56dc603dc7ed6781af6961e274ec466663d9885d..3d338d333c508f05cf072c8c0d4e72512d1ba5c0 100644
--- a/ui/webui/resources/tools/bundle_js.gni
+++ b/ui/webui/resources/tools/bundle_js.gni
@@ -67,6 +67,7 @@ template("bundle_js") {
         rebase_path("$root_gen_dir/ui/webui/resources/tsc/", root_build_dir)
     is_chrome_untrusted =
         string_replace(invoker.host, "chrome-untrusted://", "") != invoker.host
+    brave_resources_path = rebase_path("$root_gen_dir/brave/ui/webui/resources/tsc/", root_build_dir)
 
     external_paths = []
     if (is_chromeos_ash) {
@@ -86,6 +87,7 @@ template("bundle_js") {
     external_paths += [ "//resources/polymer/v3_0/|$polymer_path" ]
 
     if (!is_chrome_untrusted) {
+      external_paths += [ "chrome://resources/brave/|$brave_resources_path", "//resources/brave/|$brave_resources_path" ]
       external_paths += [ "chrome://resources/|$resources_path" ]
     }
     external_paths += [ "//resources/|$resources_path" ]
