diff --git a/ui/webui/resources/BUILD.gn b/ui/webui/resources/BUILD.gn
index 8c4963be34df8f846278d609c167320057e0e2de..2251d3cf5c8bfd158e57cd4b0e082743a63804c0 100644
--- a/ui/webui/resources/BUILD.gn
+++ b/ui/webui/resources/BUILD.gn
@@ -126,6 +126,7 @@ generate_grd("build_grd") {
       "$root_gen_dir/ui/webui/resources/cr_components/page_image_service/resources.grdp",
     ]
   }
+  import("//brave/ui/webui/resources/sources.gni") grdp_files += brave_resources_extra_grdps public_deps += brave_resources_extra_grdps_deps
 }
 
 if (is_chromeos_ash) {
