diff --git a/ui/webui/resources/BUILD.gn b/ui/webui/resources/BUILD.gn
index 4942f3ff7b9756dc09bfc60dcf9cfc8df35cffe0..3d9b4f4bd7b7cd5702b9590dcb05df6cc6ff6fae 100644
--- a/ui/webui/resources/BUILD.gn
+++ b/ui/webui/resources/BUILD.gn
@@ -128,6 +128,7 @@ generate_grd("build_grd") {
       "$root_gen_dir/ui/webui/resources/cr_components/page_image_service/resources.grdp",
     ]
   }
+  import("//brave/ui/webui/resources/sources.gni") grdp_files += brave_resources_extra_grdps public_deps += brave_resources_extra_grdps_deps
 }
 
 if (is_chromeos) {
