diff --git a/ui/webui/resources/js/ios/mojo_api.js b/ui/webui/resources/js/ios/mojo_api.js
index 7494d7b5ea9c2e12e56765145edbb3db8450a7d3..474ade6f36f0ea06d0babff8d838c5faa6848900 100644
--- a/ui/webui/resources/js/ios/mojo_api.js
+++ b/ui/webui/resources/js/ios/mojo_api.js
@@ -228,6 +228,7 @@ Mojo.internal = Mojo.internal || {};
  * @return {Object=} Response from Mojo backend.
  */
 Mojo.internal.sendMessage = function(message) {
+  message.args.frameId = __gCrWeb.frameId;
   const response = window.prompt(__gCrWeb.common.JSONStringify(message));
   return response ? JSON.parse(response) : undefined;
 };
