diff --git a/ui/webui/resources/cr_components/theme_color_picker/theme_color_picker.html.ts b/ui/webui/resources/cr_components/theme_color_picker/theme_color_picker.html.ts
index 0e887e6b2966fead91e9ced9b2963ae716c015c0..f4d9918ef82dcbf880c1e9e9a24d0b8772996cb4 100644
--- a/ui/webui/resources/cr_components/theme_color_picker/theme_color_picker.html.ts
+++ b/ui/webui/resources/cr_components/theme_color_picker/theme_color_picker.html.ts
@@ -25,19 +25,6 @@ export function getHtml(this: ThemeColorPickerElement) {
       tabindex="${this.tabIndex_(this.isDefaultColorSelected_)}"
       @click="${this.onDefaultColorClick_}">
   </cr-theme-color>
-  <cr-theme-color
-      id="greyDefaultColor"
-      .backgroundColor="${this.greyDefaultColor_.background}"
-      .baseColor="${this.greyDefaultColor_.base}"
-      .foregroundColor="${this.greyDefaultColor_.foreground}"
-      title="${this.i18n('greyDefaultColorName')}"
-      aria-label="${this.i18n('greyDefaultColorName')}"
-      role="radio"
-      ?checked="${this.isGreyDefaultColorSelected_}"
-      aria-checked="${this.isGreyDefaultColorSelected_}"
-      tabindex="${this.tabIndex_(this.isGreyDefaultColorSelected_)}"
-      @click="${this.onGreyDefaultColorClick_}">
-  </cr-theme-color>
   ${this.colors_.map((item, index) => html`
     <cr-theme-color
         class="chrome-color"
