diff --git a/ui/webui/resources/tools/build_webui.gni b/ui/webui/resources/tools/build_webui.gni
index f64b09894c1ae7729e659dd4f48a8c8d174d2d6d..c870f62b65c3e29dec87f2f7179f8eebb624907b 100644
--- a/ui/webui/resources/tools/build_webui.gni
+++ b/ui/webui/resources/tools/build_webui.gni
@@ -190,6 +190,7 @@ template("build_webui") {
         enable_removal_comments = enable_source_maps
       }
       in_files = ts_files
+      if (defined(invoker.exclude_ts_preprocess_files)) { in_files -= invoker.exclude_ts_preprocess_files } if (defined(invoker.preprocess_deps)) { public_deps = invoker.preprocess_deps }
     }
   }
 
@@ -240,6 +241,7 @@ template("build_webui") {
       if (defined(invoker.css_files)) {
         in_files += invoker.css_files
       }
+      if (defined(invoker.exclude_html_css_preprocess_files)) { in_files -= invoker.exclude_html_css_preprocess_files } if (defined(invoker.preprocess_deps)) { public_deps = invoker.preprocess_deps }
     }
   }
 
@@ -266,6 +268,7 @@ template("build_webui") {
           }
         }
       }
+      if (defined(invoker.preprocess_deps)) { public_deps = invoker.preprocess_deps }
     }
   }
 
