diff --git a/ui/webui/resources/cr_elements/BUILD.gn b/ui/webui/resources/cr_elements/BUILD.gn
index f03dfe55925b8d0273076edc02597e218b2b656a..f5384c8fa4f2361212c279cc3ffb3fbe931457eb 100644
--- a/ui/webui/resources/cr_elements/BUILD.gn
+++ b/ui/webui/resources/cr_elements/BUILD.gn
@@ -227,6 +227,7 @@ build_webui("build") {
     "//third_party/lit/v3_0:build_ts",
   ]
   if (!is_android && !is_ios) {
+    ts_deps += ["//brave/ui/webui/resources:build_ts"]
     ts_deps += [ "//third_party/polymer/v3_0:library" ]
   }
   ts_definitions = [
