diff --git a/ui/webui/resources/cr_elements/BUILD.gn b/ui/webui/resources/cr_elements/BUILD.gn
index 1b8c167f8305919e80486542c8e6f87fa2da6274..40ff9fd3bbe3244eb82f2de29c0ba5d2c3009f12 100644
--- a/ui/webui/resources/cr_elements/BUILD.gn
+++ b/ui/webui/resources/cr_elements/BUILD.gn
@@ -62,6 +62,7 @@ build_webui("build") {
       web_component_files +=
           [ "cr_searchable_drop_down/cr_searchable_drop_down.ts" ]
     }
+    else if (!is_android) { web_component_files += [ "cr_searchable_drop_down/cr_searchable_drop_down.ts" ] }
 
     icons_html_files = [
       "cr_fingerprint/cr_fingerprint_icons.html",
