diff --git a/ui/webui/resources/cr_components/searchbox/BUILD.gn b/ui/webui/resources/cr_components/searchbox/BUILD.gn
index 098c048b4e6f4f03359ff446e8428cb93390698b..94ac92c1375c42833bf3d099b468f066a55985f2 100644
--- a/ui/webui/resources/cr_components/searchbox/BUILD.gn
+++ b/ui/webui/resources/cr_components/searchbox/BUILD.gn
@@ -9,6 +9,7 @@ assert(!is_android && !is_ios)
 
 mojom("mojo_bindings") {
   sources = [ "searchbox.mojom" ]
+  generate_legacy_js_bindings = true
   webui_module_path = "/"
 
   public_deps = [
