diff --git a/ui/webui/resources/cr_elements/cr_elements.gni b/ui/webui/resources/cr_elements/cr_elements.gni
index 40fe817c3f5c75c6d1f5b0ba10f52dcd54ba00e4..734bf93a06d75d01f5f0390d36ccc14357a80578 100644
--- a/ui/webui/resources/cr_elements/cr_elements.gni
+++ b/ui/webui/resources/cr_elements/cr_elements.gni
@@ -65,6 +65,7 @@ if (include_polymer) {
     web_component_files_polymer_ts +=
         [ "cr_searchable_drop_down/cr_searchable_drop_down.ts" ]
   }
+  else if (!is_android) { web_component_files_polymer_ts += [ "cr_searchable_drop_down/cr_searchable_drop_down.ts" ] }
 
   icons_html_files = [
     "cr_fingerprint/cr_fingerprint_icons.html",
