diff --git a/ui/webui/resources/cr_elements/cr_elements.gni b/ui/webui/resources/cr_elements/cr_elements.gni
index e06ea43ea90b8ef18cf26e260318c3b2feb28584..9cadf0f903b708335a4afcbe473cccd20dffa1de 100644
--- a/ui/webui/resources/cr_elements/cr_elements.gni
+++ b/ui/webui/resources/cr_elements/cr_elements.gni
@@ -68,6 +68,7 @@ if (include_polymer) {
     web_component_files_polymer +=
         [ "cr_searchable_drop_down/cr_searchable_drop_down.ts" ]
   }
+  else if (!is_android) { web_component_files_polymer += [ "cr_searchable_drop_down/cr_searchable_drop_down.ts" ] }
 
   icons_html_files = [
     "cr_fingerprint/cr_fingerprint_icons.html",
