diff --git a/ui/webui/resources/cr_components/history_embeddings/history_embeddings.html.ts b/ui/webui/resources/cr_components/history_embeddings/history_embeddings.html.ts
index fb81df3e6fc3916dce260588a28ed7124602e417..6726f8dc0a7895e7ed109bae6e32abaf2bc7959e 100644
--- a/ui/webui/resources/cr_components/history_embeddings/history_embeddings.html.ts
+++ b/ui/webui/resources/cr_components/history_embeddings/history_embeddings.html.ts
@@ -65,10 +65,6 @@ ${!this.enableAnswers_ ? html`
 
         <div class="footer">
           <div>${this.i18n('historyEmbeddingsFooter')}</div>
-          <cr-feedback-buttons selected-option="${this.feedbackState_}"
-              @selected-option-changed="${
-                  this.onFeedbackSelectedOptionChanged_}">
-          </cr-feedback-buttons>
         </div>
       </div>
     `}
@@ -195,9 +191,6 @@ ${this.enableAnswers_ ? html`
   </div>
   <div class="footer">
     <div>${this.i18n('historyEmbeddingsFooter')}</div>
-    <cr-feedback-buttons selected-option="${this.feedbackState_}"
-        @selected-option-changed="${this.onFeedbackSelectedOptionChanged_}">
-    </cr-feedback-buttons>
   </div>
 ` : ''}
 
