diff --git a/ui/webui/resources/tools/optimize_webui.gni b/ui/webui/resources/tools/optimize_webui.gni
index 697a4526bbe325d72890d9cdecdde5e01f32b042..1344c74bcee1ec89f031823a65064ca6bc02d388 100644
--- a/ui/webui/resources/tools/optimize_webui.gni
+++ b/ui/webui/resources/tools/optimize_webui.gni
@@ -64,6 +64,7 @@ template("optimize_webui") {
         rebase_path("$root_gen_dir/ui/webui/resources/tsc/", root_build_dir)
     is_chrome_untrusted =
         string_replace(invoker.host, "chrome-untrusted://", "") != invoker.host
+    brave_resources_path = rebase_path("$root_gen_dir/brave/ui/webui/resources/tsc/", root_build_dir)
 
     external_paths = []
     if (is_chromeos_ash) {
@@ -86,6 +87,7 @@ template("optimize_webui") {
       external_paths += [ "chrome://resources/|$resources_path" ]
     }
     external_paths += [ "//resources/|$resources_path" ]
+    external_paths += [ "chrome://resources/brave/|$brave_resources_path", "//resources/brave/|$brave_resources_path" ]
 
     if (defined(invoker.external_paths)) {
       external_paths += invoker.external_paths
