diff --git a/ui/webui/resources/cr_elements/BUILD.gn b/ui/webui/resources/cr_elements/BUILD.gn
index cb8f7cbc8e1e4a6bbe19c83fd59f9d252a784005..f1d38635832805a6943ddc1d0994570fd1c5d704 100644
--- a/ui/webui/resources/cr_elements/BUILD.gn
+++ b/ui/webui/resources/cr_elements/BUILD.gn
@@ -114,7 +114,7 @@ preprocess_if_expr("preprocess_src_ts") {
       "mouse_hoverable_mixin.ts",
     ]
 
-    if (is_chromeos) {
+    if (!is_android || is_chromeos) {
       in_files += [ "cr_searchable_drop_down/cr_searchable_drop_down.ts" ]
     }
   }
@@ -277,7 +277,7 @@ preprocess_if_expr("preprocess_generated_ts") {
       "search_highlight_style.css.ts",
     ]
 
-    if (is_chromeos) {
+    if (!is_android || is_chromeos) {
       in_files += [ "cr_searchable_drop_down/cr_searchable_drop_down.html.ts" ]
     }
   }
@@ -503,7 +503,7 @@ if (include_polymer) {
       "mwb_shared_icons.html",
     ]
 
-    if (is_chromeos) {
+    if (!is_android || is_chromeos) {
       in_files += [ "cr_searchable_drop_down/cr_searchable_drop_down.html" ]
     }
   }
