diff --git a/ui/webui/resources/cr_elements/BUILD.gn b/ui/webui/resources/cr_elements/BUILD.gn
index 02c0a5e1e72885e6355454feeabc296baa67aa18..afd3435ea01c343602d1e466b3ca2116ea8c2ca6 100644
--- a/ui/webui/resources/cr_elements/BUILD.gn
+++ b/ui/webui/resources/cr_elements/BUILD.gn
@@ -215,6 +215,7 @@ build_webui("build") {
   ts_composite = true
   ts_deps = [ "../js:build_ts" ]
   if (include_polymer) {
+    ts_deps += ["//brave/ui/webui/resources:build_ts"]
     ts_deps += [
       "//third_party/lit/v3_0:build_ts",
       "//third_party/polymer/v3_0:library",
