diff --git a/ui/webui/resources/cr_components/certificate_manager/certificate_manager_v2.ts b/ui/webui/resources/cr_components/certificate_manager/certificate_manager_v2.ts
index 63e5c352c1498ad687148f7e65fd172a17bc0870..e4b6d65c712fb72bb3a0313654cfb4988ec05364 100644
--- a/ui/webui/resources/cr_components/certificate_manager/certificate_manager_v2.ts
+++ b/ui/webui/resources/cr_components/certificate_manager/certificate_manager_v2.ts
@@ -532,5 +532,5 @@ declare global {
   }
 }
 
-customElements.define(
-    CertificateManagerV2Element.is, CertificateManagerV2Element);
+// customElements.define(
+//     CertificateManagerV2Element.is, CertificateManagerV2Element);
