diff --git a/ui/webui/resources/cr_elements/cr_lottie/cr_lottie.ts b/ui/webui/resources/cr_elements/cr_lottie/cr_lottie.ts
index 4bf7ec9d224ce1590bc65f43e9413ea425352c4b..9df4b5fd732af738242f4207e9157f68dfccd78d 100644
--- a/ui/webui/resources/cr_elements/cr_lottie/cr_lottie.ts
+++ b/ui/webui/resources/cr_elements/cr_lottie/cr_lottie.ts
@@ -359,6 +359,7 @@ export class CrLottieElement extends PolymerElement {
       this.fire_('cr-lottie-paused');
     } else if (event.data.name === 'stopped') {
       this.fire_('cr-lottie-stopped');
+    } else if (event.data.name === 'complete') { this.fire_('cr-lottie-complete');
     } else if (event.data.name === 'resized') {
       this.fire_('cr-lottie-resized', event.data.size);
     }
