diff --git a/ui/webui/resources/BUILD.gn b/ui/webui/resources/BUILD.gn
index b9bac52390e72642c485fc9e82d82a02b5b97894..4596f1687e2d005b0ab49a3a27f3b9da4f0a5d03 100644
--- a/ui/webui/resources/BUILD.gn
+++ b/ui/webui/resources/BUILD.gn
@@ -121,6 +121,7 @@ generate_grd("build_grd") {
       "$root_gen_dir/ui/webui/resources/cr_components/image_service/resources.grdp",
     ]
   }
+  import("//brave/ui/webui/resources/sources.gni") grdp_files += brave_resources_extra_grdps public_deps += brave_resources_extra_grdps_deps
 }
 
 if (is_chromeos_ash) {
