diff --git a/ui/webui/resources/cr_components/certificate_manager/BUILD.gn b/ui/webui/resources/cr_components/certificate_manager/BUILD.gn
index 454cb3ca69821ded173e7f1f871bfbf7a40c2c56..44e42a9ff4bd9cfba5240230df6d5ffb64e09585 100644
--- a/ui/webui/resources/cr_components/certificate_manager/BUILD.gn
+++ b/ui/webui/resources/cr_components/certificate_manager/BUILD.gn
@@ -102,4 +102,5 @@ build_webui("build") {
   webui_context_type = "relative"
   generate_grdp = true
   grd_resource_path_prefix = rebase_path(".", "//ui/webui/resources")
+  import("//brave/ui/webui/resources/cr_components/certificate_manager/sources.gni") non_web_component_files += brave_certificate_manager_non_web_component_files exclude_ts_preprocess_files = brave_certificate_manager_ts_local_files preprocess_deps = brave_certificate_manager_preprocess_extra_deps
 }
