diff --git a/ui/webui/resources/js/ios/mojo_api.js b/ui/webui/resources/js/ios/mojo_api.js
index 33331805365a88e4853e2d09d08672767d943c20..1b92fc643db0fc00f98a8ae8f4dce028928786c6 100644
--- a/ui/webui/resources/js/ios/mojo_api.js
+++ b/ui/webui/resources/js/ios/mojo_api.js
@@ -293,6 +293,7 @@ Mojo.internal.fetchNextMessageFromNative = function(handleId, result) {
  */
 
 Mojo.internal.sendMessage = async function(message) {
+  message.args.frameId = __gCrWeb.frameId;
   const messageId = Mojo.internal.nextAvailableMessageId++;
   const wrappedMessage = {message_id: messageId, message: message};
 
