diff --git a/weblayer/shell/android/BUILD.gn b/weblayer/shell/android/BUILD.gn
index 9f9d5fdd421e726eaba73fe9ef7bcecc68f539d8..b945e40c251a26f9aadaa57c5d2652639fb0e970 100644
--- a/weblayer/shell/android/BUILD.gn
+++ b/weblayer/shell/android/BUILD.gn
@@ -73,6 +73,7 @@ android_apk("weblayer_support_apk") {
     "//weblayer/browser/java",
     "//weblayer/browser/java:test_java",
   ]
+  deps -= [ "//android_webview:locale_pak_assets",  "//android_webview:pak_file_assets", "//android_webview:weblayer_webview_assets", ]
 
   # Transitive dependencies
   deps += [
