--- build/rustup/1.67.0/toolchains/1.67.0-x86_64-apple-darwin/lib/rustlib/src/rust/library/std/src/personality/gcc.rs	2023-02-09 09:48:23
+++ build/rust/gcc.rs	2023-02-08 11:48:54
@@ -178,6 +177,17 @@
             }
         }
     } else {
+        #[cfg(any(target_os = "macos", target_os = "ios"))]
+        extern "C-unwind" {
+            fn rust_eh_personality_impl(
+                version: c_int,
+                actions: uw::_Unwind_Action,
+                _exception_class: uw::_Unwind_Exception_Class,
+                exception_object: *mut uw::_Unwind_Exception,
+                context: *mut uw::_Unwind_Context,
+            ) -> uw::_Unwind_Reason_Code;
+        }
+        #[cfg(all(not(target_os = "macos"), not(target_os = "ios")))]
         // Default personality routine, which is used directly on most targets
         // and indirectly on Windows x86_64 via SEH.
         unsafe extern "C" fn rust_eh_personality_impl(
