diff --git a/common/py_utils/py_utils/cloud_storage.py b/common/py_utils/py_utils/cloud_storage.py
index 807550c6e9270e8ec20351c43c1d757bb53defa3..0e78ccbaf62e700444e2b941d6633e0fc133e0ba 100644
--- a/common/py_utils/py_utils/cloud_storage.py
+++ b/common/py_utils/py_utils/cloud_storage.py
@@ -29,7 +29,7 @@ logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
 PUBLIC_BUCKET = 'chromium-telemetry'
-PARTNER_BUCKET = 'chrome-partner-telemetry'
+PARTNER_BUCKET = 'brave-perf-data'  # is redirected to s3://brave-perf-data
 INTERNAL_BUCKET = 'chrome-telemetry'
 TELEMETRY_OUTPUT = 'chrome-telemetry-output'
 
@@ -732,3 +732,4 @@ def CalculateHash(file_path):
 def ReadHash(hash_path):
   with open(hash_path, 'rb') as f:
     return f.read(1024).rstrip().decode('utf-8')
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
