diff --git a/telemetry/telemetry/internal/backends/chrome/desktop_browser_finder.py b/telemetry/telemetry/internal/backends/chrome/desktop_browser_finder.py
index c39463c87b4503aa29044a9fb936d3f359103ce7..dab20b459b0067efb3239b958134c76ba2cf1c85 100644
--- a/telemetry/telemetry/internal/backends/chrome/desktop_browser_finder.py
+++ b/telemetry/telemetry/internal/backends/chrome/desktop_browser_finder.py
@@ -500,3 +500,4 @@ def FindAllAvailableBrowsers(finder_options, device):
     return []
 
   return browsers
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
