diff --git a/telemetry/telemetry/page/shared_page_state.py b/telemetry/telemetry/page/shared_page_state.py
index df3f42352963923e7f64961afbb2cb9169be7625..9e6c8050e9ce4dce87590f8e5dda6a6959b75bd4 100644
--- a/telemetry/telemetry/page/shared_page_state.py
+++ b/telemetry/telemetry/page/shared_page_state.py
@@ -219,6 +219,7 @@ class SharedPageState(story_module.SharedState):
     if archive_path is not None and not os.path.isfile(archive_path):
       logging.warning('WPR archive missing: %s', archive_path)
       archive_path = None
+      assert self._finder_options.use_live_sites # Brave: assert if no WPR
     self.platform.network_controller.StartReplay(
         archive_path, page.make_javascript_deterministic, self._extra_wpr_args)
 
