diff --git a/telemetry/telemetry/internal/actions/action_runner.py b/telemetry/telemetry/internal/actions/action_runner.py
index cfece5ea0d40858724186ab9b945a07a816c2640..eb7002027c5c226475b88323ea9232a243b82322 100644
--- a/telemetry/telemetry/internal/actions/action_runner.py
+++ b/telemetry/telemetry/internal/actions/action_runner.py
@@ -42,7 +42,7 @@ import py_utils
 
 # Time to wait in seconds before requesting a memory dump in deterministic
 # mode, thus allowing metric values to stabilize a bit.
-_MEMORY_DUMP_WAIT_TIME = 3
+_MEMORY_DUMP_WAIT_TIME = 10
 
 # Time to wait in seconds after forcing garbage collection to allow its
 # effects to propagate. Experimentally determined on an Android One device
