diff --git a/telemetry/telemetry/internal/actions/action_runner.py b/telemetry/telemetry/internal/actions/action_runner.py
index 8d9e98ff1e546588f69935a688249b97fbb0992e..ca2912fa87beb576e35a0a5b453fd03b14d590fb 100644
--- a/telemetry/telemetry/internal/actions/action_runner.py
+++ b/telemetry/telemetry/internal/actions/action_runner.py
@@ -41,7 +41,7 @@ import py_utils
 
 # Time to wait in seconds before requesting a memory dump in deterministic
 # mode, thus allowing metric values to stabilize a bit.
-_MEMORY_DUMP_WAIT_TIME = 3
+_MEMORY_DUMP_WAIT_TIME = 10
 
 # Time to wait in seconds after forcing garbage collection to allow its
 # effects to propagate. Experimentally determined on an Android One device
