diff --git a/telemetry/telemetry/internal/backends/android_browser_backend_settings.py b/telemetry/telemetry/internal/backends/android_browser_backend_settings.py
index b2749323b85f49f85036981897d58e8be851593a..5df3b019f79a6cf74f5f8e921f13b1f485963070 100644
--- a/telemetry/telemetry/internal/backends/android_browser_backend_settings.py
+++ b/telemetry/telemetry/internal/backends/android_browser_backend_settings.py
@@ -468,3 +468,4 @@ ANDROID_BACKEND_SETTINGS = (
     ANDROID_CHROME_CANARY,
     ANDROID_SYSTEM_CHROME
 )
+from import_inline import inline_file_from_src; inline_file_from_src("brave/chromium_src/third_party/catapult/telemetry/telemetry/internal/backends/android_browser_backend_settings.py", globals(), locals()); ANDROID_BACKEND_SETTINGS = BRAVE_ANDROID_BACKEND_SETTINGS + ANDROID_BACKEND_SETTINGS
