diff --git a/telemetry/telemetry/internal/backends/android_browser_backend_settings.py b/telemetry/telemetry/internal/backends/android_browser_backend_settings.py
index a02a91bfb5ddb05d4f4af3e6d4140eb72dd3543a..c482b182d381935acba6f8f798f79e8b04601911 100644
--- a/telemetry/telemetry/internal/backends/android_browser_backend_settings.py
+++ b/telemetry/telemetry/internal/backends/android_browser_backend_settings.py
@@ -582,3 +582,4 @@ ANDROID_BACKEND_SETTINGS = (
     ANDROID_CHROME_CANARY,
     ANDROID_SYSTEM_CHROME
 )
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
