diff --git a/tracing/tracing/value/diagnostics/reserved_infos.py b/tracing/tracing/value/diagnostics/reserved_infos.py
index ca975908acc85f10f92562d589461399c9e4d59b..7ada4b71b24973684d4aec94e8914568c5b091ac 100644
--- a/tracing/tracing/value/diagnostics/reserved_infos.py
+++ b/tracing/tracing/value/diagnostics/reserved_infos.py
@@ -107,3 +107,4 @@ def AllInfos():
 def AllNames():
   for info in AllInfos():
     yield info.name
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
