diff --git a/src/bech32.cpp b/src/bech32.cpp
index ba3c419d8b6860a69daa07a53efce8698fc4a86d..b3413fd46a0bafac2a147afe385b8947b68bb241 100644
--- a/src/bech32.cpp
+++ b/src/bech32.cpp
@@ -374,7 +374,7 @@ DecodeResult Decode(const std::string& str) {
     std::vector<int> errors;
     if (!CheckCharacters(str, errors)) return {};
     size_t pos = str.rfind('1');
-    if (str.size() > 90 || pos == str.npos || pos == 0 || pos + 7 > str.size()) {
+    if (pos == str.npos || pos == 0 || pos + 7 > str.size()) {
         return {};
     }
     data values(str.size() - 1 - pos);
