From 9d6d3944b3c2b254b5381cd0bce6d09df72041bb Mon Sep 17 00:00:00 2001
From: bridiver <34129+bridiver@users.noreply.github.com>
Date: Thu, 7 Mar 2024 14:12:13 -0700
Subject: [PATCH] disable autocfg probe

---
 .../vendor/num-bigint-0.4.3/build.rs                   | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/third_party/rust/chromium_crates_io/vendor/num-bigint-0.4.3/build.rs b/third_party/rust/chromium_crates_io/vendor/num-bigint-0.4.3/build.rs
index 3daed5e8d33..37bf985cb20 100644
--- a/third_party/rust/chromium_crates_io/vendor/num-bigint-0.4.3/build.rs
+++ b/third_party/rust/chromium_crates_io/vendor/num-bigint-0.4.3/build.rs
@@ -10,6 +10,7 @@ fn main() {
     if u64_digit {
         autocfg::emit("u64_digit");
     }
+/*
     let ac = autocfg::new();
     let std = if ac.probe_sysroot_crate("std") {
         "std"
@@ -17,17 +18,18 @@ fn main() {
         "core"
     };
     if ac.probe_path(&format!("{}::convert::TryFrom", std)) {
+*/
         autocfg::emit("has_try_from");
-    }
-
+//  }
     if let Ok(target_arch) = env::var("CARGO_CFG_TARGET_ARCH") {
         if target_arch == "x86_64" || target_arch == "x86" {
             let digit = if u64_digit { "u64" } else { "u32" };
-
+/*
             let addcarry = format!("{}::arch::{}::_addcarry_{}", std, target_arch, digit);
             if ac.probe_path(&addcarry) {
+*/
                 autocfg::emit("use_addcarry");
-            }
+//          }
         }
     }

--
2.47.0
