diff --git a/chromium/config/Chrome/linux/arm64/config.h b/chromium/config/Chrome/linux/arm64/config.h
index e3252cb887d257a48ccc37c97fc079c36139f609..ae35c7f3e48bd27add9bd280e941e8c493efd950 100644
--- a/chromium/config/Chrome/linux/arm64/config.h
+++ b/chromium/config/Chrome/linux/arm64/config.h
@@ -735,7 +735,7 @@
 #define CONFIG_AUDIO_FRAME_QUEUE 0
 #define CONFIG_AUDIODSP 0
 #define CONFIG_BLOCKDSP 0
-#define CONFIG_BSWAPDSP 0
+#define CONFIG_BSWAPDSP 1
 #define CONFIG_CABAC 1
 #define CONFIG_CBS 0
 #define CONFIG_CBS_APV 0
