diff --git a/chromium/config/Chrome/linux/x64/config.h b/chromium/config/Chrome/linux/x64/config.h
index 787c6833e6bb4acad17b95f9ceb5bec0e2794def..d8615f9c2c104565a8657ebca8b47c1e8bb8a442 100644
--- a/chromium/config/Chrome/linux/x64/config.h
+++ b/chromium/config/Chrome/linux/x64/config.h
@@ -730,7 +730,7 @@
 #define CONFIG_AUDIO_FRAME_QUEUE 0
 #define CONFIG_AUDIODSP 0
 #define CONFIG_BLOCKDSP 0
-#define CONFIG_BSWAPDSP 0
+#define CONFIG_BSWAPDSP 1
 #define CONFIG_CABAC 1
 #define CONFIG_CBS 0
 #define CONFIG_CBS_APV 0
