diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 604c38e786e41564797f74f07351940a2990e1aa..b723918907d350693d3ca7d4d74266499de36154 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2505,6 +2505,7 @@ grd_files_debug_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_debug_sources += brave_grd_files_debug_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_release_sources =
