diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index e7f03a1140e8ea865452f6a1e1f83e436d6ab1b9..c00f4cbd51bce1e1a40d6a0256f7c535c7ef3e64 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2602,6 +2602,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
