diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index f29b5dacdce3ce883de90b776cc4222146556509..c2bfd6cad4b9910d4ab2ac2d28780bc53c6ceaec 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2659,6 +2659,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
