diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 23e18683108b7bee8ba5bad08b46ac48eafaa9b5..4b912f2ac330b2ec6c59519e9919cdddfb0826b3 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2741,6 +2741,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
