diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 169976c019572889c3f82b756628fc21ac020e85..85c260f05512d0c49f9ef50abe1d22ffbc626ccd 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2797,6 +2797,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
