diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index c8f7387d94e4b357b7d8b731c027eae9221d248e..c7eee1a7e6dedd45e8b4b3609a2c92ca48dfe604 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2807,6 +2807,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
