diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 16951ba04519b867d739f17f19d4c3047fd515b4..3dfe7a3ab654cd816fc2811b768b40b6f6d27a67 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2558,6 +2558,7 @@ grd_files_debug_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_debug_sources += brave_grd_files_debug_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_release_sources =
