diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 35eaa0683f41cb496574595b438d27d043ace12a..d9af383ee59f484cb6737b4ed240b07e845a6c21 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2560,6 +2560,7 @@ grd_files_debug_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_debug_sources += brave_grd_files_debug_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_release_sources =
