diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 01a17dd3fa1d4571362da8f64e481368f29e2c9d..c9e62f4756ab06aaf896450c1d674b895159b91b 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2587,6 +2587,7 @@ grd_files_debug_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_debug_sources += brave_grd_files_debug_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_release_sources =
