diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index eb3c85ea4e434185f37f3140f3760592f9b14da0..55fb22480742b2be568a7928d217d6676fd17c5c 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -9,6 +9,8 @@
 # BUILD.gn these lists are used to ensure that no files
 # are missed from the GRD.
 
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni")
+
 grd_files_bundled_sources = [
   "front_end/Images/3d-center.svg",
   "front_end/Images/3d-pan.svg",
@@ -2862,6 +2864,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
