diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 2ce6e10231300d123244379dbb872f7cd296f273..8afe8cb3975260b4c26b3c4ba8c1ef30d73cb417 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2661,6 +2661,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
