diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 55397c01e22505b31b603c199f09f1bd18641020..36912835dc810ba967dfc45a50528c2dc7be7a5a 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2483,6 +2483,7 @@ grd_files_debug_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_debug_sources += brave_grd_files_debug_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_release_sources =
