diff --git a/front_end/panels/network/NetworkItemView.ts b/front_end/panels/network/NetworkItemView.ts
index 645aae8d1fb76a898cea88baf3f39ef5675449cb..e0a1e412b74afb01eeae2721dceff9a6927758d0 100644
--- a/front_end/panels/network/NetworkItemView.ts
+++ b/front_end/panels/network/NetworkItemView.ts
@@ -359,3 +359,4 @@ export class NetworkItemView extends UI.TabbedPane.TabbedPane {
     return this.headersViewComponent;
   }
 }
+import { PatchNetworkItemView } from './NetworkItemView.patch.js'; (NetworkItemView as any) = PatchNetworkItemView(NetworkItemView);
