diff --git a/front_end/panels/application/ApplicationPanelSidebar.ts b/front_end/panels/application/ApplicationPanelSidebar.ts
index f7e7ed84f3e668233180a7b8897ef6b98194dc86..1cab613d391f0d04f89a4920c3f18fe0bdb96dfa 100644
--- a/front_end/panels/application/ApplicationPanelSidebar.ts
+++ b/front_end/panels/application/ApplicationPanelSidebar.ts
@@ -553,6 +553,7 @@ export class ApplicationPanelSidebar extends UI.Widget.VBox implements SDK.Targe
         },
         {scoped: true});
     this.contentElement.style.contain = 'layout style';
+    storageTreeElement.removeChild(this.trustTokensTreeElement); [this.backgroundFetchTreeElement, this.backgroundSyncTreeElement, this.periodicBackgroundSyncTreeElement, this.reportingApiTreeElement].forEach(e => backgroundServiceTreeElement.removeChild(e));
   }
 
   private addSidebarSection(title: string, jslogContext: string): UI.TreeOutline.TreeElement {
