diff --git a/front_end/core/sdk/NetworkManager.ts b/front_end/core/sdk/NetworkManager.ts
index ae5f10b531dac3b7b1ad9fb50747b3612a388865..e05a27498e6818f6fa34f5bd3b4bc0150809b031 100644
--- a/front_end/core/sdk/NetworkManager.ts
+++ b/front_end/core/sdk/NetworkManager.ts
@@ -1386,6 +1386,7 @@ export class NetworkDispatcher implements ProtocolProxyApi.NetworkDispatcher {
     return request;
   }
 }
+import { PatchNetworkDispatcher } from './NetworkManager.patch.js'; (NetworkDispatcher as any) = PatchNetworkDispatcher(NetworkDispatcher);
 
 let multiTargetNetworkManagerInstance: MultitargetNetworkManager|null;
 
