diff --git a/front_end/panels/timeline/components/LiveMetricsView.ts b/front_end/panels/timeline/components/LiveMetricsView.ts
index 505b53eccf56c7ab0319e605eb517465d3321843..51447546971833819880152a3a4f3d26187312c7 100644
--- a/front_end/panels/timeline/components/LiveMetricsView.ts
+++ b/front_end/panels/timeline/components/LiveMetricsView.ts
@@ -1061,7 +1061,7 @@ export const DEFAULT_VIEW: View = (input, output, target) => {
         </main>
         <aside class="next-steps" aria-labelledby="next-steps-section-title">
           <h2 id="next-steps-section-title" class="section-title">${i18nString(UIStrings.nextSteps)}</h2>
-          <div id="field-setup" class="settings-card">
+          <div id="field-setup" class="settings-card" hidden="true">
             <h3 class="card-title">${i18nString(UIStrings.fieldMetricsTitle)}</h3>
             ${renderFieldDataMessage(input.cruxManager)}
             ${renderPageScopeSetting(input)}
