diff --git a/front_end/panels/network/BUILD.gn b/front_end/panels/network/BUILD.gn
index 7bd0655f2aa726bb9d7c1fa29e90eefb591d7f4b..0df02804451fe9e205525af603ead114059deff3 100644
--- a/front_end/panels/network/BUILD.gn
+++ b/front_end/panels/network/BUILD.gn
@@ -100,6 +100,7 @@ devtools_module("network") {
     "./components:bundle",
     "./forward:bundle",
   ]
+  import("//brave/third_party/devtools-frontend/src/front_end/panels/network/sources.gni")  sources += brave_devtools_front_end_panels_network_source
 }
 
 devtools_entrypoint("bundle") {
