diff --git a/front_end/core/sdk/NetworkManager.ts b/front_end/core/sdk/NetworkManager.ts
index 9760229f0c0025cbfd8c44d0bf8a932f59c31c7e..9b46a4f89cab105b7bc30f7fe43be6b0abde068e 100644
--- a/front_end/core/sdk/NetworkManager.ts
+++ b/front_end/core/sdk/NetworkManager.ts
@@ -1386,6 +1386,7 @@ export class NetworkDispatcher implements ProtocolProxyApi.NetworkDispatcher {
     return request;
   }
 }
+import { PatchNetworkDispatcher } from './NetworkManager.patch.js'; (NetworkDispatcher as any) = PatchNetworkDispatcher(NetworkDispatcher);
 
 let multiTargetNetworkManagerInstance: MultitargetNetworkManager|null;
 
