diff --git a/front_end/core/sdk/NetworkManager.ts b/front_end/core/sdk/NetworkManager.ts
index 9a151eda7298aa28f51c4f505c595758e102295e..39575d354b68fc767782593524ecc65d3a95b83f 100644
--- a/front_end/core/sdk/NetworkManager.ts
+++ b/front_end/core/sdk/NetworkManager.ts
@@ -1556,6 +1556,7 @@ export class NetworkDispatcher implements ProtocolProxyApi.NetworkDispatcher {
     return `${host}:${port}`;
   }
 }
+import { PatchNetworkDispatcher } from './NetworkManager.patch.js'; (NetworkDispatcher as any) = PatchNetworkDispatcher(NetworkDispatcher);
 
 export type RequestConditionsSetting = {
   url: string,
