diff --git a/front_end/panels/network/NetworkDataGridNode.ts b/front_end/panels/network/NetworkDataGridNode.ts
index 1038ceccdbfab71fe7e06b65b18c9b6b63efde22..4fdfd53e13e9a548c2935f754aefc77a58e8c7eb 100644
--- a/front_end/panels/network/NetworkDataGridNode.ts
+++ b/front_end/panels/network/NetworkDataGridNode.ts
@@ -1636,6 +1636,7 @@ export class NetworkRequestNode extends NetworkNode {
   }
 }
 
+import { PatchNetworkRequestNode } from './NetworkDataGridNode.patch.js'; (NetworkRequestNode as any) = PatchNetworkRequestNode(NetworkRequestNode);
 export class NetworkGroupNode extends NetworkNode {
   override createCells(element: Element): void {
     super.createCells(element);
