diff --git a/front_end/panels/network/BUILD.gn b/front_end/panels/network/BUILD.gn
index bc97ac1c34aea576629b64998312551a84968171..fa7ec69e35e9c3d7ef892aaadf36f93daa22f216 100644
--- a/front_end/panels/network/BUILD.gn
+++ b/front_end/panels/network/BUILD.gn
@@ -99,6 +99,7 @@ devtools_module("network") {
     "./components:bundle",
     "./forward:bundle",
   ]
+  import("//brave/third_party/devtools-frontend/src/front_end/panels/network/sources.gni")  sources += brave_devtools_front_end_panels_network_source
 }
 
 devtools_entrypoint("bundle") {
