diff --git a/front_end/core/protocol_client/InspectorBackend.ts b/front_end/core/protocol_client/InspectorBackend.ts
index f85c53b72c6d406dae6213c26ccd2dd5c7c14a92..77a75e72d52426d7109c43c155596338fd8c8a79 100644
--- a/front_end/core/protocol_client/InspectorBackend.ts
+++ b/front_end/core/protocol_client/InspectorBackend.ts
@@ -77,6 +77,7 @@ export class InspectorBackend implements InspectorBackendCommands.InspectorBacke
     globalThis.Protocol ||= {};
 
     InspectorBackendCommands.registerCommands(this);
+    this.registerEvent('Network.requestAdblockInfoReceived' as QualifiedName, ['requestId', 'info']);
   }
 
   private getOrCreateEventParameterNamesForDomain(domain: ProtocolDomainName): EventParameterNames {
