diff --git a/front_end/core/sdk/NetworkManager.ts b/front_end/core/sdk/NetworkManager.ts
index 5648a0ab0ebbd2b629cf370d36665382ac88024f..c585dda15b05afc1afc434519b9bd6f6a19c6f8e 100644
--- a/front_end/core/sdk/NetworkManager.ts
+++ b/front_end/core/sdk/NetworkManager.ts
@@ -1297,6 +1297,7 @@ export class NetworkDispatcher implements ProtocolProxyApi.NetworkDispatcher {
     return request;
   }
 }
+import { PatchNetworkDispatcher } from './NetworkManager.patch.js'; (NetworkDispatcher as any) = PatchNetworkDispatcher(NetworkDispatcher);
 
 let multiTargetNetworkManagerInstance: MultitargetNetworkManager|null;
 
