diff --git a/front_end/core/protocol_client/protocol_client.ts b/front_end/core/protocol_client/protocol_client.ts
index 6f4b3f92dbf5beb1be28b9a5d30434e29fa77b48..ddd5d9f1d86395ad72fe602f3f802c147ae1689c 100644
--- a/front_end/core/protocol_client/protocol_client.ts
+++ b/front_end/core/protocol_client/protocol_client.ts
@@ -19,3 +19,4 @@ self.Protocol = self.Protocol || {};
 
 // FIXME: This instance of InspectorBackend should not be a side effect of importing this module.
 InspectorBackendCommands.registerCommands(InspectorBackend.inspectorBackend);
+InspectorBackend.inspectorBackend.registerEvent('Network.requestAdblockInfoReceived' as InspectorBackend.QualifiedName, ['requestId', 'info']);
