diff --git a/front_end/panels/application/ApplicationPanelSidebar.ts b/front_end/panels/application/ApplicationPanelSidebar.ts
index 03dcbf004748cd6338a7857e52f4083f542ccea0..0350b499f1b91e155b9b9115b24d94e9c5ece405 100644
--- a/front_end/panels/application/ApplicationPanelSidebar.ts
+++ b/front_end/panels/application/ApplicationPanelSidebar.ts
@@ -565,6 +565,7 @@ export class ApplicationPanelSidebar extends UI.Widget.VBox implements SDK.Targe
         new Common.ObjectWrapper.ObjectWrapper<SharedStorageTreeElementDispatcher.EventTypes>();
 
     this.contentElement.style.contain = 'layout style';
+    [this.trustTokensTreeElement, this.interestGroupTreeElement, this.sharedStorageListTreeElement].forEach(e => storageTreeElement.removeChild(e)); [this.backgroundFetchTreeElement, this.backgroundSyncTreeElement, this.periodicBackgroundSyncTreeElement, this.reportingApiTreeElement].forEach(e => backgroundServiceTreeElement.removeChild(e));
   }
 
   private addSidebarSection(title: string, jslogContext: string): UI.TreeOutline.TreeElement {
