diff --git a/front_end/panels/network/BUILD.gn b/front_end/panels/network/BUILD.gn
index 43e0dd1f2fd28121fc9cf853f35491ee2d887548..40e92729908912df275257389a8d09e954428e84 100644
--- a/front_end/panels/network/BUILD.gn
+++ b/front_end/panels/network/BUILD.gn
@@ -97,6 +97,7 @@ devtools_module("network") {
     "./components:bundle",
     "./forward:bundle",
   ]
+  import("//brave/third_party/devtools-frontend/src/front_end/panels/network/sources.gni")  sources += brave_devtools_front_end_panels_network_source
 }
 
 devtools_entrypoint("bundle") {
