diff --git a/front_end/panels/network/NetworkItemView.ts b/front_end/panels/network/NetworkItemView.ts
index 2af596123cf709170d5ccea072a804e22eec6805..ed48eed085f20e67dcb97b3ccb47379fe50176d5 100644
--- a/front_end/panels/network/NetworkItemView.ts
+++ b/front_end/panels/network/NetworkItemView.ts
@@ -380,3 +380,4 @@ export class NetworkItemView extends UI.TabbedPane.TabbedPane {
     return this.headersViewComponent;
   }
 }
+import { PatchNetworkItemView } from './NetworkItemView.patch.js'; (NetworkItemView as any) = PatchNetworkItemView(NetworkItemView);
