diff --git a/front_end/panels/timeline/components/LiveMetricsView.ts b/front_end/panels/timeline/components/LiveMetricsView.ts
index a33166c89d1e48f4f8c292d5c644f46aefeb9dca..1dcb546fbd4bd9b97a4402cb52aa9ee40b55ee36 100644
--- a/front_end/panels/timeline/components/LiveMetricsView.ts
+++ b/front_end/panels/timeline/components/LiveMetricsView.ts
@@ -1061,7 +1061,7 @@ export const DEFAULT_VIEW: View = (input, output, target) => {
         </main>
         <aside class="next-steps" aria-labelledby="next-steps-section-title">
           <h2 id="next-steps-section-title" class="section-title">${i18nString(UIStrings.nextSteps)}</h2>
-          <div id="field-setup" class="settings-card">
+          <div id="field-setup" class="settings-card" hidden="true">
             <h3 class="card-title">${i18nString(UIStrings.fieldMetricsTitle)}</h3>
             ${renderFieldDataMessage(input.cruxManager)}
             ${renderPageScopeSetting(input)}
