diff --git a/front_end/panels/network/BUILD.gn b/front_end/panels/network/BUILD.gn
index 4040c8d2599ac93002e52edf992c55918dc311e8..a26d50e596e4b23efb151a467b5d2cade93370e3 100644
--- a/front_end/panels/network/BUILD.gn
+++ b/front_end/panels/network/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/third_party/devtools-frontend/src/front_end/panels/network/sources.gni")
 import("../../../scripts/build/ninja/devtools_entrypoint.gni")
 import("../../../scripts/build/ninja/devtools_module.gni")
 import("../../../scripts/build/ninja/generate_css.gni")
@@ -66,6 +67,7 @@ devtools_ui_module("network") {
     "ShowMoreDetailsWidget.ts",
     "SignedExchangeInfoView.ts",
   ]
+  sources += brave_devtools_front_end_panels_network_sources
 
   ts_deps = [
     "../../core/common:bundle",
@@ -114,6 +116,7 @@ devtools_ui_module("network") {
     "./components:bundle",
     "./forward:bundle",
   ]
+  ts_deps += brave_devtools_front_end_panels_network_ts_deps
   deps = [
     ":css_files",
     "../../ui/legacy/components/data_grid:css_files",
