diff --git a/front_end/core/sdk/NetworkManager.ts b/front_end/core/sdk/NetworkManager.ts
index b3de237238450eb9b05e1b2f042d7ac6e95bbd9f..df9bb6ae155cb43f3ed2622afb1e3de5e1d595c8 100644
--- a/front_end/core/sdk/NetworkManager.ts
+++ b/front_end/core/sdk/NetworkManager.ts
@@ -1437,6 +1437,7 @@ export class NetworkDispatcher implements ProtocolProxyApi.NetworkDispatcher {
     return request;
   }
 }
+import { PatchNetworkDispatcher } from './NetworkManager.patch.js'; (NetworkDispatcher as any) = PatchNetworkDispatcher(NetworkDispatcher);
 
 let multiTargetNetworkManagerInstance: MultitargetNetworkManager|null;
 
