diff --git a/front_end/panels/timeline/components/LiveMetricsView.ts b/front_end/panels/timeline/components/LiveMetricsView.ts
index a6f69dea2b1e4ad0a73389c5271a124562750571..e13c94b6bb8906a72edfb7216903b86d60f12357 100644
--- a/front_end/panels/timeline/components/LiveMetricsView.ts
+++ b/front_end/panels/timeline/components/LiveMetricsView.ts
@@ -1122,7 +1122,7 @@ export class LiveMetricsView extends LegacyWrapper.LegacyWrapper.WrappableCompon
           </main>
           <aside class="next-steps" aria-labelledby="next-steps-section-title">
             <h2 id="next-steps-section-title" class="section-title">${i18nString(UIStrings.nextSteps)}</h2>
-            <div id="field-setup" class="settings-card">
+            <div id="field-setup" class="settings-card" hidden="true">
               <h3 class="card-title">${i18nString(UIStrings.fieldMetricsTitle)}</h3>
               ${this.#renderFieldDataMessage()}
               ${this.#renderPageScopeSetting()}
