diff --git a/front_end/panels/network/NetworkItemView.ts b/front_end/panels/network/NetworkItemView.ts
index 1fc7927def7c3544b38656e0b5f16fb781f9543e..302a2f7539c3ed4f93881560170aae9be393923c 100644
--- a/front_end/panels/network/NetworkItemView.ts
+++ b/front_end/panels/network/NetworkItemView.ts
@@ -386,3 +386,4 @@ export class NetworkItemView extends UI.TabbedPane.TabbedPane {
     return this.#headersViewComponent;
   }
 }
+import { PatchNetworkItemView } from './NetworkItemView.patch.js'; (NetworkItemView as any) = PatchNetworkItemView(NetworkItemView);
