diff --git a/front_end/panels/network/NetworkDataGridNode.ts b/front_end/panels/network/NetworkDataGridNode.ts
index bcd9f33a7678034f5d13fd32f37b124f6f2d8e2f..74dfc581aee3aa2529153bc18a6be5d8bc8c32b3 100644
--- a/front_end/panels/network/NetworkDataGridNode.ts
+++ b/front_end/panels/network/NetworkDataGridNode.ts
@@ -1564,6 +1564,7 @@ export class NetworkRequestNode extends NetworkNode {
   }
 }
 
+import { PatchNetworkRequestNode } from './NetworkDataGridNode.patch.js'; (NetworkRequestNode as any) = PatchNetworkRequestNode(NetworkRequestNode);
 export class NetworkGroupNode extends NetworkNode {
   override createCells(element: Element): void {
     super.createCells(element);
