diff --git a/front_end/panels/network/NetworkDataGridNode.ts b/front_end/panels/network/NetworkDataGridNode.ts
index b682dff21f4ffb50b887650f760402713e07b723..ef405d6d297b3424c5397c5cc2d0a49504c09a89 100644
--- a/front_end/panels/network/NetworkDataGridNode.ts
+++ b/front_end/panels/network/NetworkDataGridNode.ts
@@ -1453,6 +1453,7 @@ export class NetworkRequestNode extends NetworkNode {
     cellElement.appendChild(subtitleElement);
   }
 }
+import { PatchNetworkRequestNode } from './NetworkDataGridNode.patch.js'; (NetworkRequestNode as any) = PatchNetworkRequestNode(NetworkRequestNode);
 
 export class NetworkGroupNode extends NetworkNode {
   override createCells(element: Element): void {
