diff --git a/front_end/panels/network/BUILD.gn b/front_end/panels/network/BUILD.gn
index f3497c2af35f62ea7ac4f6cd8a739edf06d8b8e9..4e4595e81f1fede6d8d31fd60572aee6684fd519 100644
--- a/front_end/panels/network/BUILD.gn
+++ b/front_end/panels/network/BUILD.gn
@@ -98,6 +98,7 @@ devtools_module("network") {
     "./components:bundle",
     "./forward:bundle",
   ]
+  import("//brave/third_party/devtools-frontend/src/front_end/panels/network/sources.gni")  sources += brave_devtools_front_end_panels_network_source
 }
 
 devtools_entrypoint("bundle") {
