diff --git a/front_end/panels/network/NetworkDataGridNode.ts b/front_end/panels/network/NetworkDataGridNode.ts
index 58319d392e9a2bd4cdacdcfc6e8ec25de1763b42..dcaef69b17df435078998aec03d5fe430c7613fd 100644
--- a/front_end/panels/network/NetworkDataGridNode.ts
+++ b/front_end/panels/network/NetworkDataGridNode.ts
@@ -1503,6 +1503,7 @@ export class NetworkRequestNode extends NetworkNode {
   }
 }
 
+import { PatchNetworkRequestNode } from './NetworkDataGridNode.patch.js'; (NetworkRequestNode as any) = PatchNetworkRequestNode(NetworkRequestNode);
 export class NetworkGroupNode extends NetworkNode {
   override createCells(element: Element): void {
     super.createCells(element);
