diff --git a/front_end/panels/timeline/components/LiveMetricsView.ts b/front_end/panels/timeline/components/LiveMetricsView.ts
index a8773368a38dd4c224bd66f6f2419cc69e16ed1e..a71e0ffe8e22d1e2aef645b92f26fd47be718b97 100644
--- a/front_end/panels/timeline/components/LiveMetricsView.ts
+++ b/front_end/panels/timeline/components/LiveMetricsView.ts
@@ -1043,7 +1043,7 @@ export const DEFAULT_VIEW: View = (input, output, target) => {
         </main>
         <aside class="next-steps" aria-labelledby="next-steps-section-title">
           <h2 id="next-steps-section-title" class="section-title">${i18nString(UIStrings.nextSteps)}</h2>
-          <div id="field-setup" class="settings-card">
+          <div id="field-setup" class="settings-card" hidden="true">
             <h3 class="card-title">${i18nString(UIStrings.fieldMetricsTitle)}</h3>
             ${renderFieldDataMessage(input.cruxManager)}
             ${renderPageScopeSetting(input)}
