diff --git a/front_end/panels/network/BUILD.gn b/front_end/panels/network/BUILD.gn
index e4283b71383743a67e8e10a4fc5a5b3497156e97..b99c4e6fa2d144e6ae023088bf812cc53de0a385 100644
--- a/front_end/panels/network/BUILD.gn
+++ b/front_end/panels/network/BUILD.gn
@@ -98,6 +98,7 @@ devtools_module("network") {
     "./components:bundle",
     "./forward:bundle",
   ]
+  import("//brave/third_party/devtools-frontend/src/front_end/panels/network/sources.gni")  sources += brave_devtools_front_end_panels_network_source
 }
 
 devtools_entrypoint("bundle") {
