diff --git a/front_end/panels/network/NetworkItemView.ts b/front_end/panels/network/NetworkItemView.ts
index 6f420138cacd5bf8c114eadfd9cfabab20ce2fd2..9f83f5f42bc12dd1a7dc2fd086cf101c2292bcc4 100644
--- a/front_end/panels/network/NetworkItemView.ts
+++ b/front_end/panels/network/NetworkItemView.ts
@@ -357,3 +357,4 @@ export class NetworkItemView extends UI.TabbedPane.TabbedPane {
     return this.headersViewComponent;
   }
 }
+import { PatchNetworkItemView } from './NetworkItemView.patch.js'; (NetworkItemView as any) = PatchNetworkItemView(NetworkItemView);
