diff --git a/front_end/core/protocol_client/InspectorBackend.ts b/front_end/core/protocol_client/InspectorBackend.ts
index 993f15ca70020366e9d136359131d001bbb1ad88..beffcad77003e33739a7253d0622804f28e8afaa 100644
--- a/front_end/core/protocol_client/InspectorBackend.ts
+++ b/front_end/core/protocol_client/InspectorBackend.ts
@@ -77,6 +77,7 @@ export class InspectorBackend implements InspectorBackendCommands.InspectorBacke
     globalThis.Protocol ||= {};
 
     InspectorBackendCommands.registerCommands(this);
+    this.registerEvent('Network.requestAdblockInfoReceived' as QualifiedName, ['requestId', 'info']);
   }
 
   private getOrCreateEventParameterNamesForDomain(domain: ProtocolDomainName): EventParameterNames {
