diff --git a/front_end/panels/network/BUILD.gn b/front_end/panels/network/BUILD.gn
index 6cc1e04248e592816af065313b1090480c7411dc..bf69c4e74b44efa29911bbcfa6f68c6c7fe70eb5 100644
--- a/front_end/panels/network/BUILD.gn
+++ b/front_end/panels/network/BUILD.gn
@@ -100,6 +100,7 @@ devtools_module("network") {
     "./components:bundle",
     "./forward:bundle",
   ]
+  import("//brave/third_party/devtools-frontend/src/front_end/panels/network/sources.gni")  sources += brave_devtools_front_end_panels_network_source
 }
 
 devtools_entrypoint("bundle") {
