diff --git a/front_end/panels/timeline/components/LiveMetricsView.ts b/front_end/panels/timeline/components/LiveMetricsView.ts
index 9821915aef1acbad5858287f53e6db58704bbfd1..11cd6b2b0977800f405060ccb26df6d29c5dc24e 100644
--- a/front_end/panels/timeline/components/LiveMetricsView.ts
+++ b/front_end/panels/timeline/components/LiveMetricsView.ts
@@ -1040,7 +1040,7 @@ export const DEFAULT_VIEW: View = (input, output, target) => {
         </main>
         <aside class="next-steps" aria-labelledby="next-steps-section-title">
           <h2 id="next-steps-section-title" class="section-title">${i18nString(UIStrings.nextSteps)}</h2>
-          <div id="field-setup" class="settings-card">
+          <div id="field-setup" class="settings-card" hidden="true">
             <h3 class="card-title">${i18nString(UIStrings.fieldMetricsTitle)}</h3>
             ${renderFieldDataMessage(input.cruxManager)}
             ${renderPageScopeSetting(input)}
