diff --git a/front_end/panels/timeline/components/LiveMetricsView.ts b/front_end/panels/timeline/components/LiveMetricsView.ts
index 35ef2855567e6d62a6d1704b3f6950674ce7f7ae..8b66f983019c5437e3d05dfd2d9f4d250d0c348e 100644
--- a/front_end/panels/timeline/components/LiveMetricsView.ts
+++ b/front_end/panels/timeline/components/LiveMetricsView.ts
@@ -1187,7 +1187,7 @@ export class LiveMetricsView extends LegacyWrapper.LegacyWrapper.WrappableCompon
           </main>
           <aside class="next-steps" aria-labelledby="next-steps-section-title">
             <h2 id="next-steps-section-title" class="section-title">${i18nString(UIStrings.nextSteps)}</h2>
-            <div id="field-setup" class="settings-card">
+            <div id="field-setup" class="settings-card" hidden="true">
               <h3 class="card-title">${i18nString(UIStrings.fieldMetricsTitle)}</h3>
               ${this.#renderFieldDataMessage()}
               ${this.#renderPageScopeSetting()}
