diff --git a/front_end/panels/network/BUILD.gn b/front_end/panels/network/BUILD.gn
index 9aa16dffeb778a231d18c30b741d77b7b1dac512..09345abe20a4b98d3807cba4932d787e512dfcf9 100644
--- a/front_end/panels/network/BUILD.gn
+++ b/front_end/panels/network/BUILD.gn
@@ -114,6 +114,7 @@ devtools_ui_module("network") {
     "./components:bundle",
     "./forward:bundle",
   ]
+  import("//brave/third_party/devtools-frontend/src/front_end/panels/network/sources.gni") sources += brave_devtools_front_end_panels_network_sources ts_deps += brave_devtools_front_end_panels_network_ts_deps
 }
 
 devtools_entrypoint("bundle") {
