diff --git a/front_end/panels/network/BUILD.gn b/front_end/panels/network/BUILD.gn
index 0a1fe05978f8c92bfa8952b3d7648fa623c3fdb6..3a8e9af60369ba443b5753e6ffd7f18a60044e3d 100644
--- a/front_end/panels/network/BUILD.gn
+++ b/front_end/panels/network/BUILD.gn
@@ -104,6 +104,7 @@ devtools_ui_module("network") {
     "./components:bundle",
     "./forward:bundle",
   ]
+  import("//brave/third_party/devtools-frontend/src/front_end/panels/network/sources.gni")  sources += brave_devtools_front_end_panels_network_source
 }
 
 devtools_entrypoint("bundle") {
