diff --git a/scripts/build/typescript/ts_library.py b/scripts/build/typescript/ts_library.py
index 7bd3ac037d096a0d46184994382d1c18c6ff50a0..faf3ac112613e41e756fd2c11dc5e74f1888ec49 100644
--- a/scripts/build/typescript/ts_library.py
+++ b/scripts/build/typescript/ts_library.py
@@ -338,5 +338,6 @@ def main():
     return 0
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
     sys.exit(main())
