diff --git a/scripts/build/esbuild.js b/scripts/build/esbuild.js
index 096edc81b9b1bf6fde428cff49017f893a23f7fc..708f79d35e8fc8bfc92f9c121a963cee248cb02d 100644
--- a/scripts/build/esbuild.js
+++ b/scripts/build/esbuild.js
@@ -25,7 +25,9 @@ const esbuild = require('esbuild');
 
 const entryPoints = [process.argv[2]];
 const outfile = process.argv[3];
-const useSourceMaps = process.argv.slice(4).includes('--configSourcemaps');
+const additionalArgs = process.argv.slice(4);
+const useSourceMaps = additionalArgs.includes('--configSourcemaps');
+const minify = additionalArgs.includes('--minify');
 
 const outdir = path.dirname(outfile);
 
@@ -46,6 +48,7 @@ esbuild
       platform: 'browser',
       plugins: [plugin],
       sourcemap: useSourceMaps,
+      minify,
     })
     .catch(err => {
       console.error('Failed to run esbuild:', err);
