diff --git a/scripts/build/typescript/typescript.gni b/scripts/build/typescript/typescript.gni
index 18e70ded2cac12df9074c4bad10bab09291f5304..fbe4848669d91d6d265edb3e33af1dbb1537b298 100644
--- a/scripts/build/typescript/typescript.gni
+++ b/scripts/build/typescript/typescript.gni
@@ -189,6 +189,7 @@ template("ts_library") {
 
       foreach(src, sources) {
         _extension = get_path_info(src, "extension")
+        if (label_matches(src, ["//brave/*"])) { src = get_path_info(src, "name") }
         _relative_file_name = rebase_path(src, _rootdir)
         _fileName = get_path_info(_relative_file_name, "dir") + "/" +
                     get_path_info(_relative_file_name, "name")
