diff --git a/scripts/build/typescript/ts_library.py b/scripts/build/typescript/ts_library.py
index 31e22aab147faa1c423f7d30295b60e290a9db33..7afd24a654feb6560f3238172943da27f57cec29 100644
--- a/scripts/build/typescript/ts_library.py
+++ b/scripts/build/typescript/ts_library.py
@@ -361,5 +361,6 @@ def main():
     return 0
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
     sys.exit(main())
