diff --git a/scripts/build/typescript/ts_library.py b/scripts/build/typescript/ts_library.py
index 9153860becbd760befe5cfb959fed5fa05221be8..b591c4890886ac3217f5334dc7175f12824b1096 100644
--- a/scripts/build/typescript/ts_library.py
+++ b/scripts/build/typescript/ts_library.py
@@ -334,5 +334,6 @@ def main():
     return 0
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
     sys.exit(main())
