diff --git a/scripts/build/typescript/ts_library.py b/scripts/build/typescript/ts_library.py
index fa4181c68dee9584649134f0b5edd394da276a3b..5c0a933731a4f423b0b9467f2e90c6384bed3739 100644
--- a/scripts/build/typescript/ts_library.py
+++ b/scripts/build/typescript/ts_library.py
@@ -326,5 +326,6 @@ def main():
     return 0
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
     sys.exit(main())
