diff --git a/scripts/build/typescript/ts_library.py b/scripts/build/typescript/ts_library.py
index a509f51a9eaf359185c6a2c7d353bbb00d827f1e..9d31b2a5f413fb6f7961fa5cb755b238b04fc561 100644
--- a/scripts/build/typescript/ts_library.py
+++ b/scripts/build/typescript/ts_library.py
@@ -369,5 +369,6 @@ def main():
     return 0
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
     sys.exit(main())
