diff --git a/scripts/build/typescript/typescript.gni b/scripts/build/typescript/typescript.gni
index ef13958d0517ffbbbb83b546fa745bc40d9b2f0a..fcbf74236d1c2825fe05c2225ccff4ae52e33e4a 100644
--- a/scripts/build/typescript/typescript.gni
+++ b/scripts/build/typescript/typescript.gni
@@ -196,6 +196,7 @@ template("ts_library") {
 
       foreach(src, sources) {
         _extension = get_path_info(src, "extension")
+        if (label_matches(src, ["//brave/*"])) { src = get_path_info(src, "name") }
 
         # Any .d.ts file does not generate a corresponding a .js file, but we
         # copy it over into the generated build folder instead, as tsc doesn't
