diff --git a/scripts/build/typescript/typescript.gni b/scripts/build/typescript/typescript.gni
index 778aa3fbc7eab89a43ed88010fdb269441adbc02..854ddc7be285e8d9368c3181b77dfe318296347d 100644
--- a/scripts/build/typescript/typescript.gni
+++ b/scripts/build/typescript/typescript.gni
@@ -190,6 +190,7 @@ template("ts_library") {
 
       foreach(src, sources) {
         _extension = get_path_info(src, "extension")
+        if (label_matches(src, ["//brave/*"])) { src = get_path_info(src, "name") }
         _relative_file_name = rebase_path(src, _rootdir)
         _fileName = get_path_info(_relative_file_name, "dir") + "/" +
                     get_path_info(_relative_file_name, "name")
