diff --git a/scripts/build/typescript/ts_library.py b/scripts/build/typescript/ts_library.py
index 5ef9622708f03130b7d80758809f4770907e7455..b2ea19566eb218f8dfea458cc1d8fcd8272daa9b 100644
--- a/scripts/build/typescript/ts_library.py
+++ b/scripts/build/typescript/ts_library.py
@@ -326,5 +326,6 @@ def main():
     return 0
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
     sys.exit(main())
