diff --git a/scripts/build/typescript/ts_library.py b/scripts/build/typescript/ts_library.py
index c4a2a6536f9c12afd46183c44444e3aefe0a5844..7c63a55f4d57136604ab2ccb5434c871b803a9ed 100644
--- a/scripts/build/typescript/ts_library.py
+++ b/scripts/build/typescript/ts_library.py
@@ -332,5 +332,6 @@ def main():
     return 0
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
     sys.exit(main())
