diff --git a/scripts/build/typescript/ts_library.py b/scripts/build/typescript/ts_library.py
index a5a92628a023c9e0f9626b46e8a9b7091929f96c..e12743c10aaa1dc70bf9f3d6baf7c14abcb67ff5 100644
--- a/scripts/build/typescript/ts_library.py
+++ b/scripts/build/typescript/ts_library.py
@@ -344,5 +344,6 @@ def main():
     return 0
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
     sys.exit(main())
