diff --git a/scripts/build/typescript/ts_library.py b/scripts/build/typescript/ts_library.py
index e9b19f3c53dec8f177f5e9b15ed59e1cd8bd8caa..824dbd0e61f59869105c37b81ac42a105e5fd087 100644
--- a/scripts/build/typescript/ts_library.py
+++ b/scripts/build/typescript/ts_library.py
@@ -368,5 +368,6 @@ def main():
     return 0
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
     sys.exit(main())
