diff --git a/scripts/build/typescript/ts_library.py b/scripts/build/typescript/ts_library.py
index dcbc940f5c669aabab0c5e5842d6dd8d5271de28..20f70c0dcdaa2fb82789541872ff26979d37628a 100644
--- a/scripts/build/typescript/ts_library.py
+++ b/scripts/build/typescript/ts_library.py
@@ -350,5 +350,6 @@ def main():
     return 0
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
     sys.exit(main())
