diff --git a/scripts/build/typescript/ts_library.py b/scripts/build/typescript/ts_library.py
index b8757b333b9a4e2d9b7398dd8d4f7fcf50f2df35..265c4f8155629b3f26e5ebc4d2079857d76b895b 100644
--- a/scripts/build/typescript/ts_library.py
+++ b/scripts/build/typescript/ts_library.py
@@ -332,5 +332,6 @@ def main():
     return 0
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
     sys.exit(main())
