diff --git a/BUILD.gn b/BUILD.gn
index c6132807ff6c488543b4df1f3ba8793d9dea98d5..5952ba023ed75fc50eb9af1514bbbe93c83fc202 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -4,6 +4,7 @@
 
 # LINT.IfChange
 
+import("//brave/v8/sources.gni")
 import("//build/config/android/config.gni")
 import("//build/config/arm.gni")
 import("//build/config/c++/c++.gni")
@@ -963,6 +964,9 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs
+  include_dirs = []
+  include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3847,6 +3851,7 @@ v8_header_set("v8_headers") {
     "include/v8-weak-callback-info.h",
     "include/v8.h",
   ]
+  sources += brave_v8_headers_sources
 
   sources += [
     # The following headers cannot be platform-specific. The include validation
@@ -3857,6 +3862,7 @@ v8_header_set("v8_headers") {
   ]
 
   public_deps = [ ":v8_config_headers" ]
+  public_deps += brave_v8_headers_public_deps
 
   deps = [
     ":cppgc_headers",
