diff --git a/BUILD.gn b/BUILD.gn
index 6dba5b97a8f4fd3b7b1159725f5a93f26512bba4..a6510cb3173662564a47d57cdae194b2061e5d72 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -745,6 +745,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3074,6 +3075,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
