diff --git a/BUILD.gn b/BUILD.gn
index 1d9a331f1c65c50cd98b966b3d9e826d4124192b..f2eb84310d5dbeccc259888865ab4276ed139ed3 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -629,6 +629,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -2658,6 +2659,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
