diff --git a/BUILD.gn b/BUILD.gn
index bca193c467c25f57a2be33c96366e516d3616994..db87df0dc8ba019ed15db3254c58a2e42cc5d229 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -694,6 +694,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -2779,6 +2780,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
