diff --git a/BUILD.gn b/BUILD.gn
index 818f13802c66607a676981e064437c87d1fc4e59..b3ca3b3ec8c6c7ef1e892a49ce1823ea0abb8151 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -854,6 +854,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3248,6 +3249,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
