diff --git a/BUILD.gn b/BUILD.gn
index aec481ae6496171da329a3c9577c48f789ccc8f2..250739af0d723d3bb01b7d77c6f80362f29821c7 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -4,6 +4,8 @@
 
 # LINT.IfChange
 
+import("//tools/v8_context_snapshot/v8_context_snapshot.gni")
+import("//brave/v8/sources.gni")
 import("//build/config/android/config.gni")
 import("//build/config/arm.gni")
 import("//build/config/c++/c++.gni")
@@ -988,6 +990,9 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs
+  include_dirs = []
+  include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -2547,6 +2552,7 @@ if (v8_use_metagen) {
     # define must re-run the driver even though the action command is
     # unchanged.
     deps = [ ":v8_dump_build_config" ]
+    deps += brave_v8_headers_public_deps
 
     if (v8_generate_external_defines_header) {
       # v8config.h includes the generated v8-gn.h in this configuration.
@@ -3925,6 +3931,7 @@ v8_header_set("v8_headers") {
     "include/v8-weak-callback-info.h",
     "include/v8.h",
   ]
+  sources += brave_v8_headers_sources
 
   sources += [
     # The following headers cannot be platform-specific. The include validation
@@ -3935,6 +3942,7 @@ v8_header_set("v8_headers") {
   ]
 
   public_deps = [ ":v8_config_headers" ]
+  public_deps += brave_v8_headers_public_deps
 
   deps = [
     ":cppgc_headers",
@@ -8784,6 +8792,9 @@ if (is_component_build) {
       ":v8_libplatform",
       "src/inspector:inspector_headers",
     ]
+    if (!use_v8_context_snapshot || include_both_v8_snapshots) {
+      public_deps += [ ":v8_snapshot" ]
+    }
 
     configs = [ ":internal_config" ]
     v8_add_configs += [ "//build/config/compiler:thinlto_optimize_max" ]
@@ -8865,6 +8876,9 @@ if (is_component_build) {
       ":v8_libplatform",
       "src/inspector:inspector_headers",
     ]
+    if (!use_v8_context_snapshot || include_both_v8_snapshots) {
+      public_deps += [ ":v8_snapshot" ]
+    }
 
     public_configs = [ ":external_config" ]
   }
