diff --git a/BUILD.gn b/BUILD.gn
index f0c7424ab29d2a7085d72d0667e14040451574bd..e2ebeef18430c10a7eac02eb13eca34eeb42c398 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -836,6 +836,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3192,6 +3193,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
