diff --git a/BUILD.gn b/BUILD.gn
index 87b212921bb8d51fd5457a7b4d61ee956589c282..18c6eaacd1723928019fab1ca75383da8691eceb 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -762,6 +762,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3235,6 +3236,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
