diff --git a/BUILD.gn b/BUILD.gn
index fd36d114bf635c916d383c584699d4212bec44b0..ea37d28097f19f635021e66aeee52b8d84a838da 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -666,6 +666,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -2701,6 +2702,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
