diff --git a/BUILD.gn b/BUILD.gn
index 78be992dc5bf6b87eae684f54b2d71c643e52303..ef4fa982f96c6969c66e40fee1746c90c26069d4 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -850,6 +850,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3315,6 +3316,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
