diff --git a/BUILD.gn b/BUILD.gn
index e1f23f17969f90df9becdcf7c0015b27cd973b4a..264467e7fc159eece57607180e04f05597660c10 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -659,6 +659,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -2693,6 +2694,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
