diff --git a/BUILD.gn b/BUILD.gn
index 15de2179a0e5ce50d5c659a9d15a920c50124c3e..ecc26744d337ffe76e4fa5f1606d3825010032f0 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -864,6 +864,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3484,6 +3485,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
