diff --git a/BUILD.gn b/BUILD.gn
index 79b95fb517e05978d3d72a2a8b1e7b795dd8769d..325065686cb3ddce7ffbba9f38aa8e2b8c101a67 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -747,6 +747,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3122,6 +3123,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
