diff --git a/BUILD.gn b/BUILD.gn
index c863cbfec6207101ead79609665ea0179ca387c4..24ec16bded9669ef018f9cc7eb3f141366573ca4 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -867,6 +867,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3314,6 +3315,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
