diff --git a/BUILD.gn b/BUILD.gn
index 9206f6dccc625422b9a58e38d1f63d1b19dab74b..a94b650ec54dccd3a1777119930419a19c8a9404 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -851,6 +851,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3365,6 +3366,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
