diff --git a/BUILD.gn b/BUILD.gn
index fc80d646d3593fafaae02e544d9125fc7ce88320..b63f0f5fe0a0b6a0c1549d739cb9798721e3da58 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -869,6 +869,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3309,6 +3310,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
