diff --git a/BUILD.gn b/BUILD.gn
index ef1d419a4895f83b850757b0b066efed677fea0d..45b9bbd4d5061647af3cd647455adc70708a69a8 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -814,6 +814,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3170,6 +3171,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
