diff --git a/BUILD.gn b/BUILD.gn
index 67588d899019f48a550410aedea220ba6ef1c3e4..3f9756ba011cb527fff2aa5aaa8d528774376099 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -746,6 +746,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3120,6 +3121,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
