diff --git a/BUILD.gn b/BUILD.gn
index 76391be1666b0b2e57c91bdac59070224e57ce24..96a9be5649c82e356a6f36f4c3257af844d2c651 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -938,6 +938,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3646,6 +3647,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
