diff --git a/BUILD.gn b/BUILD.gn
index 10b476b862eac1e753fb6fb09de219fe247bf264..bbff48a67637aa39619f4ed60790cd6ef882e031 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -845,6 +845,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3309,6 +3310,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
