diff --git a/BUILD.gn b/BUILD.gn
index a5c7d8477815b437cf5415ab34ca40bb1f43273a..a442346eb24353ee5740d5a775801c94ba0f2f5a 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -876,6 +876,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3539,6 +3540,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
