diff --git a/BUILD.gn b/BUILD.gn
index 7998c754fa00dba7a4bc7ce375bb37c4b9622c54..4b2a5660a1c9837ce0948b4b82fb2b5072722cca 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -756,6 +756,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3216,6 +3217,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
