diff --git a/BUILD.gn b/BUILD.gn
index 6801927b0d079b89fb3223a8e774730cd1f45bfa..cfdecc9d1e031c483e1d8b10100a0d88f4323d89 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -856,6 +856,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3389,6 +3390,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
