diff --git a/BUILD.gn b/BUILD.gn
index e5e040958d74be2c65216274b7f3f5887ef23e1f..95aa3754733aae8b993c59fab06660dcc3548ab2 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -891,6 +891,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3533,6 +3534,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
