diff --git a/BUILD.gn b/BUILD.gn
index 52a9c860841b53b6ad91f94f4a411641732a5a79..d12f6f6fcab8ff18f6565816e81700805c5a3e69 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -843,6 +843,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3331,6 +3332,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
