diff --git a/BUILD.gn b/BUILD.gn
index 61187af3053a02e62cc75ae0c6e368dd541af877..308fe76e92ba07f3638845f0a5e238e9f7c7f8ed 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -621,6 +621,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -2624,6 +2625,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
