diff --git a/BUILD.gn b/BUILD.gn
index 613f7ace670b56b879cbe4a1a84e5ba719be90b2..eaa18c3cb35432793b54e0f2e4cf4db60b058971 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -867,6 +867,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3310,6 +3311,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
