diff --git a/BUILD.gn b/BUILD.gn
index e212c18948f54fe183b37cd19b3f876dfa581b8d..786b63e23348c014e295b6f277d619992d20d221 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -745,6 +745,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3086,6 +3087,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
