diff --git a/BUILD.gn b/BUILD.gn
index aa348f18368594dbd1d7fc048570e4d68e93a80f..456d7c95fa80f30fe3011a2c2b1fd4a816724402 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -867,6 +867,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3314,6 +3315,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
