diff --git a/include/cppgc/common.h b/include/cppgc/common.h
index b6dbff3dd6f0077d98a44909c04f7b02284e63e7..a6549d66412a9617c45236696416d2c115790d40 100644
--- a/include/cppgc/common.h
+++ b/include/cppgc/common.h
@@ -5,8 +5,6 @@
 #ifndef INCLUDE_CPPGC_COMMON_H_
 #define INCLUDE_CPPGC_COMMON_H_
 
-// TODO(chromium:1056170): Remove dependency on v8.
-#include "v8config.h"  // NOLINT(build/include_directory)
 
 namespace cppgc {
 
