diff --git a/include/v8.h b/include/v8.h
index c4720ca6954da26db8f3f7e9f2ddf97e29ff75ac..90886dddcbf548bf2d984acb745b3d67caa78692 100644
--- a/include/v8.h
+++ b/include/v8.h
@@ -20,6 +20,7 @@
 #include <stdio.h>
 
 #include <atomic>
+#include <functional>
 #include <memory>
 #include <string>
 #include <type_traits>
@@ -31,6 +32,7 @@
 #include "v8-version.h"   // NOLINT(build/include_directory)
 #include "v8config.h"     // NOLINT(build/include_directory)
 
+#include <string>
 // We reserve the V8_* prefix for macros defined in V8 public API and
 // assume there are no name conflicts with the embedder's code.
 
@@ -1370,6 +1372,7 @@ class V8_EXPORT ScriptOrModule {
    * the ScriptOrigin.
    */
   Local<PrimitiveArray> GetHostDefinedOptions();
+  int GetScriptId() const;
 };
 
 /**
@@ -1698,6 +1701,7 @@ class V8_EXPORT Module : public Data {
    */
   Local<UnboundModuleScript> GetUnboundModuleScript();
 
+  int GetModuleScriptId() const;
   /**
    * Returns the underlying script's id.
    *
@@ -1847,6 +1851,10 @@ class V8_EXPORT ScriptCompiler {
     // Source takes ownership of CachedData.
     V8_INLINE Source(Local<String> source_string, const ScriptOrigin& origin,
                      CachedData* cached_data = nullptr);
+    V8_INLINE Source(Local<String> source_string,
+                     Local<Integer> parent_script_id,
+                     const ScriptOrigin& origin,
+                     CachedData* cached_data = nullptr);
     V8_INLINE explicit Source(Local<String> source_string,
                               CachedData* cached_data = nullptr);
     V8_INLINE ~Source();
@@ -1874,6 +1882,7 @@ class V8_EXPORT ScriptCompiler {
     ScriptOriginOptions resource_options;
     Local<Value> source_map_url;
     Local<PrimitiveArray> host_defined_options;
+    Local<Integer> parent_script_id;
 
     // Cached data from previous compilation (if a kConsume*Cache flag is
     // set), or hold newly generated cache data (kProduce*Cache flags) are
@@ -2065,7 +2074,7 @@ class V8_EXPORT ScriptCompiler {
    */
   static V8_WARN_UNUSED_RESULT MaybeLocal<Script> Compile(
       Local<Context> context, StreamedSource* source,
-      Local<String> full_source_string, const ScriptOrigin& origin);
+      Local<String> full_source_string, const ScriptOrigin& origin, int parent_script_id = 0);
 
   /**
    * Return a version tag for CachedData for the current V8 version & flags.
@@ -4792,6 +4801,8 @@ class V8_EXPORT Function : public Object {
    */
   Local<Value> GetBoundFunction() const;
 
+  // Unwrap any bindings to the innermost function, including nested bindings.
+  Local<Function> GetInnermostFunction() const;
   /**
    * Calls builtin Function.prototype.toString on this function.
    * This is different from Value::ToString() that may call a user-defined
@@ -8255,6 +8266,18 @@ class V8_EXPORT MeasureMemoryDelegate {
       Local<Promise::Resolver> promise_resolver, MeasureMemoryMode mode);
 };
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+typedef void (*EvalScriptCompiledFunc)(v8::Isolate& isolate,
+    const int parent_script_id, const int script_id,
+    v8::Local<v8::String> source);
+
+typedef void (*BuiltInFuncCallFunc)(v8::Isolate& isolate,
+    const char* builtin_name, const std::vector<const std::string>& args);
+
+typedef void (*BuiltInFuncResponseFunc)(v8::Isolate& isolate,
+    const char* builtin_name, const std::string& value);
+// #endif
+
 /**
  * Isolate represents an isolated instance of the V8 engine.  V8 isolates have
  * completely separate states.  Objects from one isolate must not be used in
@@ -9697,6 +9720,15 @@ class V8_EXPORT Isolate {
   void operator delete(void*, size_t) = delete;
   void operator delete[](void*, size_t) = delete;
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  int GetExecutingScriptId(int* out_script_position = nullptr);
+  void GetAllExecutingScripts(std::function<void(int, int)> callback);
+
+  void SetEvalScriptCompiledFunc(const EvalScriptCompiledFunc func);
+  void SetBuiltInFuncCallFunc(const BuiltInFuncCallFunc func);
+  void SetBuiltInFuncResponseFunc(const BuiltInFuncResponseFunc func);
+// #endif
+
  private:
   template <class K, class V, class Traits>
   friend class PersistentValueMapBase;
@@ -11557,6 +11589,21 @@ ScriptCompiler::Source::Source(Local<String> string, const ScriptOrigin& origin,
       host_defined_options(origin.HostDefinedOptions()),
       cached_data(data) {}
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+ScriptCompiler::Source::Source(Local<String> string,
+                               Local<Integer> parent_script_id,
+                               const ScriptOrigin& origin, CachedData* data)
+    : source_string(string),
+      resource_name(origin.ResourceName()),
+      resource_line_offset(origin.LineOffset()),
+      resource_column_offset(origin.ColumnOffset()),
+      resource_options(origin.Options()),
+      source_map_url(origin.SourceMapUrl()),
+      host_defined_options(origin.HostDefinedOptions()),
+      parent_script_id(parent_script_id),
+      cached_data(data) {}
+// #endif
+
 ScriptCompiler::Source::Source(Local<String> string,
                                CachedData* data)
     : source_string(string), cached_data(data) {}
