diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index a22093e1c011deb5890cec5d387ad1dbc314139f..a84b4588be143b26dd1dbf687fe25e5ee0632dd3 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3354,6 +3354,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
