diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index f0f778f062a6e7a431fd5492ae46cab40ba8bf9e..9bb8e15831542f36992ad14caa95c0c3b8037a55 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2894,6 +2894,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
