diff --git a/src/isolate.h b/src/isolate.h
index 2b1612a957237d280d8d2bf88b47cdcf28f71c88..096d7b1c16454915f7768262c56fb451157326a7 100644
--- a/src/isolate.h
+++ b/src/isolate.h
@@ -41,6 +41,8 @@ class UObject;
 }  // namespace U_ICU_NAMESPACE
 #endif  // V8_INTL_SUPPORT
 
+#include "brave/third_party/blink/brave_page_graph/buildflags/buildflags.h"
+
 namespace v8 {
 
 namespace base {
@@ -1501,6 +1503,21 @@ class Isolate final : private HiddenFactory {
     top_backup_incumbent_scope_ = top_backup_incumbent_scope;
   }
 
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  void SetRegisterScriptStartFunc(const ScriptStartFunc func) {
+    script_start_func_ = func;
+  }
+  ScriptStartFunc GetRegisterScriptStartFunc() const {
+    return script_start_func_;
+  }
+  void SetRegisterScriptEndFunc(const ScriptStartFunc func) {
+    script_end_func_ = func;
+  }
+  ScriptEndFunc GetRegisterScriptEndFunc() const {
+    return script_end_func_;
+  }
+#endif
+
   V8_EXPORT_PRIVATE void SetIdle(bool is_idle);
 
  private:
@@ -1830,6 +1847,11 @@ class Isolate final : private HiddenFactory {
   void* operator new(size_t) = delete;
   void operator delete(void*) = delete;
 
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  ScriptStartFunc script_start_func_ = nullptr;
+  ScriptEndFunc script_end_func_ = nullptr;
+#endif
+
   friend class heap::HeapTester;
   friend class TestSerializer;
 
