diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 8f23f702a2ebd0418e39741f33972eb948b532d8..9d3fbd2c075b4ca94ffc9027e2cfaee60c22cf1b 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3311,6 +3311,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
