diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index c87b78ae19af1ea7fa7b9e4908722670dfa70860..46edf27515b1a8e218373c655c3685cf0289e0a7 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2870,6 +2870,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
