diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 27d212d5002735092780c67f066b186f0dd9dac7..3d71a45bf5d442eeb77b6504a4f718eae0526bb0 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2892,6 +2892,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
