diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index cb495edc66553cc66e71ae874c8e23c4c2e2e0b9..3aa4782e8beb04448840a96cbbf5ed083ebd4bd8 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3467,6 +3467,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
