diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 9e7d2394fba06483f9fad30d440daf0abce876aa..c427b6d6b32dccc7ecfe7c054c426ea7ae657359 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2887,6 +2887,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
