diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 16d95660587dc7e5ca0558351591f68abec1150d..19b40c074304b06e6dd91d2756ed36d2777d850c 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3440,6 +3440,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
