diff --git a/src/d8/d8.cc b/src/d8/d8.cc
index 8c9a4c3ae6028029b63c498e8709e907a0f58420..87425e6bee557b0a000a147f9315e9935cf5ed8f 100644
--- a/src/d8/d8.cc
+++ b/src/d8/d8.cc
@@ -96,6 +96,10 @@
     if (i::FLAG_trace_backing_store) PrintF(__VA_ARGS__); \
   } while (false)
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+#include <string>
+// #endif
+
 namespace v8 {
 
 namespace {
@@ -1404,6 +1408,12 @@ void PerIsolateData::SetTimeout(Local<Function> callback,
                                 Local<Context> context) {
   set_timeout_callbacks_.emplace(isolate_, callback);
   set_timeout_contexts_.emplace(isolate_, context);
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  {
+    internal::Isolate i_isolate = reinterpret_cast<internal::Isolate*>(isolate_);
+    i_isolate->ReportBuiltInCallAndResponse("setTimeout", {}, "");
+  }
+// #endif
 }
 
 MaybeLocal<Function> PerIsolateData::GetTimeoutCallback() {
