diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index ef884f7d5866f8046cac6724d9cf6bfe20ba4e17..fcd66e56def11a9cc63dca6b4212523561f2311b 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3004,6 +3004,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
