diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index ff7360c10148657848440446e8d86bc4fcd51298..1acbf74b7d394670385e4df3dea0f02e6d7127bf 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2989,6 +2989,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
