diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 06639623222ce53fc3177c961fb11b027b9ed4ba..57e15d07fd38cfc065dc09933b1b214f1b88f267 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3399,6 +3399,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
