diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 038de915a5f6c86cc4cb76483e7e71966651bdbe..a80562e9e9ee8b1bf0db97354b2d2fb6a6faf965 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3485,6 +3485,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
