diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 6efa1f3558f8a29ad4d6489e76a10000425044f7..b379048312ed9112bdd8f46d609142e876a532cb 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2893,6 +2893,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
