diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 17e5cbcbd126ee272a543df19b91e8b3b5089244..2560825dcf1f358f8fa3b7d6abb6a54bbf3f36d3 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2887,6 +2887,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
