diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index b01ad28ef00b7eb6aa131ac21d2a76f4adc3478b..ad6971d75e35a601d26f1c34edca79ce0a919ef9 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3437,6 +3437,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
