diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 5431deb83e743d1a4f379627a014dae92d59cc51..800e59c03b176297095fa21cd77fc16746d2e7ee 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2888,6 +2888,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
