diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 4ced00f929671cfdc6af53cee336658d6806964b..77a8643b6064290d6bb46e9ecac1610b4db9dd6f 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3452,6 +3452,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
