diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 972898e2a95e59c036cb8a284859cb81b5f6af71..70384ae5d9c39f2b6a9e419fa68d54f5d8a5d0e3 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2961,6 +2961,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
