diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 3782bd438530435a066f6a1844316d38a3008086..5f389cf4c18f00e919da83a64a6e3a01d705d5be 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3398,6 +3398,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
