diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 7c14522bb6017744a0744889226382fe53c67749..4f6963b7ef4617d33ca5b28b9adb36d1ac1de467 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3306,6 +3306,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
