diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 3db0876787423db71fcca014841bfa54a64128b6..c5c9569a2fb0f5636c5824606e487522a64cc907 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3494,6 +3494,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
