diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index a93967addcb47bdfd688f5f2459e01e12efc5597..241fd058e9b51a1af6accf77bc84eb3ddc1d3502 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3458,6 +3458,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
