diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 2c82e865b83aee223bcb1925df059face246b546..8f5813367230a6030894c7585db81c97fd372e6a 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3489,6 +3489,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
