diff --git a/test/cctest/test-serialize.cc b/test/cctest/test-serialize.cc
index 5fbc9600e168dd862d8421a3ea2b1ba46b338344..5608096ffeea87eb1d81bd409534f3f709f6e0f7 100644
--- a/test/cctest/test-serialize.cc
+++ b/test/cctest/test-serialize.cc
@@ -1730,7 +1730,7 @@ TEST(CodeSerializerPromotedToCompilationCache) {
 
   MaybeHandle<SharedFunctionInfo> shared =
       isolate->compilation_cache()->LookupScript(
-          src, src, 0, 0, v8::ScriptOriginOptions(), LanguageMode::kSloppy);
+          src, src, 0, 0, v8::ScriptOriginOptions(), LanguageMode::kSloppy, 0);
 
   CHECK(*shared.ToHandleChecked() == *copy);
 
