diff --git a/test/cctest/test-serialize.cc b/test/cctest/test-serialize.cc
index 972b1ca7727263abee1881201816693a5dbfb3ea..f253f8d7fe68cd052b6c3430b439146880636827 100644
--- a/test/cctest/test-serialize.cc
+++ b/test/cctest/test-serialize.cc
@@ -61,6 +61,8 @@
 #include "test/cctest/heap/heap-utils.h"
 #include "test/cctest/setup-isolate-for-tests.h"
 
+#include "brave/third_party/blink/brave_page_graph/buildflags/buildflags.h"
+
 namespace v8 {
 namespace internal {
 
@@ -1710,7 +1712,11 @@ TEST(CodeSerializerPromotedToCompilationCache) {
   MaybeHandle<SharedFunctionInfo> shared =
       isolate->compilation_cache()->LookupScript(
           src, src, 0, 0, v8::ScriptOriginOptions(), isolate->native_context(),
-          LanguageMode::kSloppy);
+          LanguageMode::kSloppy
+#if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+          , 0
+#endif
+          );
 
   CHECK(*shared.ToHandleChecked() == *copy);
 
